---
title: Personal Notes on RNA-seq
author: Tianyi Shi
date: '2022-07-29'
slug: personal-notes-on-rna-seq
categories:
  - NGS
  - RNA-seq
tags:
  - RNA-seq
  - Illumina
  - NGS
bibliography: ../bib/main.bib
output:
  blogdown::html_page:
    number_sections: yes
    toc: yes
  rmarkdown::pdf_document:
    number_sections: yes
    toc: yes
nocite: ~
---


<div id="TOC">
<ul>
<li><a href="#overview" id="toc-overview"><span class="toc-section-number">1</span> Overview</a></li>
<li><a href="#trimming" id="toc-trimming"><span class="toc-section-number">2</span> Trimming</a></li>
<li><a href="#mapping" id="toc-mapping"><span class="toc-section-number">3</span> Mapping</a>
<ul>
<li><a href="#star" id="toc-star"><span class="toc-section-number">3.1</span> STAR</a></li>
<li><a href="#hisat2" id="toc-hisat2"><span class="toc-section-number">3.2</span> HISAT2</a></li>
</ul></li>
</ul>
</div>

<div id="overview" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Overview</h1>
<p>Typical RNA-seq data analysis workflow:</p>
<ol style="list-style-type: decimal">
<li><strong>Trimming</strong>, i.e. removal of the adapter sequences and poor-quality nucleotides. <strong>Quality control</strong> checks help to indicate whether trimming has been carried out appropriately.</li>
<li><strong>Alignment</strong> to a reference genome</li>
<li><strong>Gene quantification and normalization</strong> with reference to a file containing gene positions (GTF file)</li>
<li>Differential expression (DE) analysis across conditions</li>
</ol>
</div>
<div id="trimming" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Trimming</h1>
</div>
<div id="mapping" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Mapping</h1>
<div id="star" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> STAR</h2>
<p>The <code>--outSAMstrandField</code> intronMotif option adds an XS attribute to the spliced alignments in the BAM ﬁle, which is required by Cufﬂinks for unstranded RNA-seq data <span class="citation">(<a href="#ref-Dobin-2015" role="doc-biblioref">Dobin and Gingeras 2015</a>)</span>.</p>
</div>
<div id="hisat2" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> HISAT2</h2>
<p>HISAT2’s <span class="citation">(<a href="#ref-Kim-2019" role="doc-biblioref"><strong>Kim-2019?</strong></a>)</span> alignment algorithm is based on a graph Ferragina Manzini index, which is faster and more memory-efficient than STAR.</p>
<p>HISAT2 binaries and indexes for <em>H. sapians</em> and a few model organisms are available for download from the <a href="https://daehwankimlab.github.io/hisat2/download/">official website</a>.</p>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Dobin-2015" class="csl-entry">
Dobin, Alexander, and Thomas R. Gingeras. 2015. <span>“Mapping RNA-Seq Reads with STAR.”</span> <em>Current Protocols in Bioinformatics</em> 51 (1): 11.14.1–19. https://doi.org/<a href="https://doi.org/10.1002/0471250953.bi1114s51">https://doi.org/10.1002/0471250953.bi1114s51</a>.
</div>
</div>
</div>
</div>
