<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>biochemistry - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="biochemistry - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/biochemistry/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">biochemistry</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../tutorial/redox-based-modulation-of-enzyme-activity.html" class="archive-item-link">
                Redox-Based Modulation of Enzyme Activity 
              </a>
              <span class="archive-item-date">
                2020-06-17
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>Presentation see <a href="../thioredoxin-presentation/">here</a></p>
<!--
Thioredoxins are kept in the reduced state by the flavoenzyme thioredoxin reductase in a NADPH-dependent reaction. 

Plants have an unusually complex complement of Trxs composed of six well-defined types (Trxs f, m, x, y, h, and o) that reside in different cell compartments and function in an array of processes. In 2010 it was discovered for the first time that thioredoxin proteins are able to move from cell to cell, representing a novel form of cellular communication in plants.

Thioredoxins are small ubiquitous proteins with two redox-active cysteine rediuses separated by a pair of amino acids (CXXC)

TRXs *m, f, x, y, z* isoforms are located in the chloroplast, whereas TRXs *o* are mitochondrial and TRX *h* proteins are found in different subcellular compartments.

TRX *h* is the largest TRX gene family in plants, comprising nine different isoforms in Arabidopsis. First isolated from spinach leaves; outisde of the chloroplast. Lack of a transit peptide; initially assumed as solely cytosolic proteins. However, alternative locations were also observed for some of them: plasma membrane, endomembranes (ER & Golgi), mitochondria, and the nucleus. Highly redundant -> difficult to unequivocally determine the function of each TRX *h* isoform

Besides the plastid and mitochondria, Trxs are also found in the cytosol, ER and nucleus of plant cells. Little is known about the function of nucleoredoxins (Nrx) and ER-Trxs, the function of cytosolic Trx *h* has been well documented.

Trx regulates photosynthesis and the TCA cycle in chloroplasts and mitochondria, respectively.

Extraplastidial Trxs hhave been shown to be important in the regulation of different processes such as seed germination, seedling establishment, C, N, and mitochondrial metabolism as well as for plant stress tolerance.

# Trx reduction systems

In extraplastidial compartments the reducing power for Trx reduction is provided by NADPH via NADPH-Trx reductase A (NTRA) and NTRB.

The cytosolic h-type thioredoxins act as key regulators of seed germination and are recycled by NADPH-dependent thioredoxin reductase

## In seed

Seeds develop in a metabolically active state but become quiescent during maturatiion and drying. This period is characterised by low water content and low metabolic activity as well as the non-enzymatic accumulation of reactive oxygen species (roS) that oxidise proteins and RNA. Oxidation of the thiol groups of cysteine -> changes in protein structure -> lowering solubility and modifying enzyme function -> protection against denaturation and proteolysis as the seed matures and dries.

Under favourable conditions, the seed germinates and thhe seedling draws on stored reserves for growth and development. Large increase in ROS produced as side-products of metabolism. ROS have important signalling roles in germination (but also cause oxidative stress if present in high amounts). The Trx system represents an important waay to regulate ROS in seeds.

## NTR

The NTRA and NTRB are found in both cytosol and mitochondrion, while thhe first has also been found in the nucleuus. NTRA and NTRB are mainly expressed in the cytosol and mitochondrion, respectively.

The NADPH used by the NTR/Trx system is generated by the OPPP through the activity of G6PDH and 6-PGDH, while the souruce of NADPH in the mitoochondion is throuugh the activity of IDH or NADHk. 


# Methods

## Alkylation Labelling


@Kobrehel-1992 applied the fluorescnet alkylating reagent monobromobimane (mBBr) to label free thiol groups in proteins following reduction by Thx. -> discovered that gliadins and glutenin are reduced in vivo during germination

Buchanan and co-workers pioneered the use of proteomics for Trx target identiﬁcation and applied the ﬂuorescent alkylating reagent monobromobimane (mBBr) to label free sulfhydryl (SH) groups in proteins following reduction by Trx [37]. This approach can be applied not only to in vitro but to in vivo reactions as well [2,40,45,46]. To identify targets from peanut seeds, mBBr-labelled proteins were separated by two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) and ﬂuorescent spots were subjected to trypsin digestion followed by either Edman sequencing or mass spectrometry for protein identiﬁcation

Lane 1, Control: no addition [1.1]; lane 2, GSH/GR/NADPH: GSH, glutathione re- ductase(fromspinachleaves),andNADPH [1.2];lane3,NGS: NADPH, GSH, glutathione reductase (from spinach leaves), and glutaredoxin (from E. coli) [1.6]; lane 4, NTS: NADPH, NTR, and thioredoxin (both proteins from E. coli) [3.9]; lane 5, MET/T(Ec): ,B- mercaptoethanolandthioredoxin(E.co/i)[1.8];lane6,DTT [2.5]; lane7,DTT/T(Ec):DTT andthioredoxin(E.co/i)[3.0];lane8,DTT/ T(W): same as lane 7 except with wheat thioredoxin h [3.6]; lane 9, NGS,-Gliadin: same as lane 3 except without the gliadin protein fraction [0]; lane 10, NTS,-Gliadin: same as lane 4 except without thegliadinproteinfraction[0].

The above findings demonstrate that thioredoxin specifically reduces components of the wheat gliadin and glutenin fractions when tested in vitro. The results, however, provide no indication as to whether these proteins are reduced in vivo during germination-a question that, to our knowledge, has not been previously addressed (cf. 27).

To answer this question, we applied the mBBr/SDS-PAGE technique to monitor the reduction status of proteins in the germinating seed. We observed that reduction of components in the Osborne fractions increased progressively with time and reached a peak after 2 to 3 d germination (Fig. 4). The observed increase in reduction ranged from twofold with the gliadins to threefold with the albumin/globulins and fivefold with the glutenins. The net redox change was most apparent in the gliadins and glutenins found to undergo maximal reduction by thioredoxin, i.e. the 30- to 50-kD gliadins and 30- to 60-kD glutenins (cf. Figs. 1 and 3). The redox changes taking place in these fractions during germination are shown in Figure 5.

## CXXS

@Verdoucq-1999

In the thiol-disulfide exchange reaction, Trx forms a transient intermolecular disufide bond with the target protein involving the more N-terminal cysteine in the active site of Trx (CXXC). Then, the other cysteine (CXXC) attacks the disulﬁde bond of the Trx-target intermediate and releases the reduced target and oxidized Trx.

If instead this second cysteine is replaced with a serine residue through site-directed mutagenesis the Trx loses its capability to reduce the intermolecular disulﬁde bond and the covalent Trx-target complex will be stabilized. This strategy was first applied by Verdoucq et al. [49] who constructed a monocysteinic Trx mutant of A. thaliana Trxh3. The protein was heterologously expressed in Saccharomyces cerevisae and it was successfully stabilized in a complex with a peroxiredoxin target protein.

The concept was further developed by Motohashi et al. [50] who immobilized a monocysteinic Trx m on a resin designed to trap targets on a column. Non-targets were removed with the column wash and the trapped targets could subsequently be eluted with a chemical reductant such as DTT (Fig. 3). This approach has been used to identify target proteins from plant mitochondria [51–53], chloroplasts [54], cytosolic extracts [55], and has also been combined with the thiol alkylation approach described above to identify putative target proteins, for example in seeds from M. truncatula [40].

Improved by @Motohashi-2001 (initially applied to Trx m) 
Sepharose 4B resin

@Yamazaki-2004 Target Proteins of the Cytosolic Thioredoxins in Arabidopsis thaliana.
The obtained proteins were separated by two-dimensional gel electrophoresis and analyzed by matrix-assisted laser desorption ionization time-of-flight mass spectrometry. 

The results in this study and recent proteomics studies on the target proteins of chloroplast thioredoxin indicate the versatility and the physiological significance of thioredoxin as reductant in plant cell.

## Differential Labelling of Recombinant BASI (IAA and 4-VP)

- for the well studied chloroplastic thiioredoxins, target disulphide bonds in target enzymes were mainly idenitified by using strategies based on site directed mutagenesis of target proteins
- hoowever, to produce recombinant target proteiins and their mutants would be entremely time-consumiing considering the large number of potential target proteins of thioredoxin h that have been identifieid
- Using mass spectrometry techniques, peptide fragments containing modifications are recognised in hte peptide mass maps according to an exact mass shift from the expected thhiioretical values.
- the thiol and disulphiide forms of cysteine side chains, however, are usually not distinguishhed in MS because complete reduction and alkylation of cysteines are performed during regular sample preparation
- A novel strategy is developed by @Maeda-2005.
- Trypsin digesting
- According to the mass difference of cysteines modified by these two reagents, conversion of disulphides to thiols catalysed by HvTrxh1 was shown in the peptide mass maps of recombinant BASI.

# Rice

Cereals store reserve proteins in the endosperm during maturation of grain. In germination and early seedling development, these proteins are digested to provide nitrogen for growth. In rice and certain other seeds, the major storage proteins are 11S globulins housed in the protein body of the endosperm. Generally, 11S globulins are hexamers composed of identical subunits, each containing two polypeptide chains (Hara et al. 1976, Müntz 1998). The acidic chain, with one presumed intramolecular disulfide bond, is connected to the basic chain by an intermolecular disulfide bond (Staswick et al. 1984, Lawrence et al. 1994) (Fig. 1). The basic chain also contains a free sulfhydryl group. In rice, the proteins corresponding to 11S globulins, the glutelins, are unique in that they exist as large macromolecular complexes formed by disulfide and hydrophobic interactions of acidic and basic polypeptides (Katsube et al. 1999).
-->
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Geigenberger-2017" class="csl-entry">
Geigenberger, Peter, Ina Thormählen, Danilo M. Daloso, and Alisdair R. Fernie. 2017. <span>“The Unprecedented Versatility of the Plant‎ Thioredoxin System.”</span> <em>Trends in Plant Science</em> 22 (3): 249–62. https://doi.org/<a href="https://doi.org/10.1016/j.tplants.2016.12.008">https://doi.org/10.1016/j.tplants.2016.12.008</a>.
</div>
<div id="ref-Hagglund-2016" class="csl-entry">
Hägglund, Per, Christine Finnie, Hiroyuki Yano, Azar Shahpiri, Bob B. Buchanan, Anette Henriksen, and Birte Svensson. 2016. <span>“Seed Thioredoxin h.”</span> <em>Biochimica Et Biophysica Acta (BBA) - Proteins and Proteomics</em> 1864 (8): 974–82. https://doi.org/<a href="https://doi.org/10.1016/j.bbapap.2016.02.014">https://doi.org/10.1016/j.bbapap.2016.02.014</a>.
</div>
</div>

              </div>
            </article>
          
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="../tutorial/principle-of-electron-transfer-and-proton-transport.html" class="archive-item-link">
                Principle of electron transfer and proton transport 
              </a>
              <span class="archive-item-date">
                2019-10-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p><strong>Using the physical chemistry principles, discuss factors which are contributed in the favourability of an electron transfer reactions between cofactors.</strong></p>
<p>The reduction potential, <span class="math inline">\(E\)</span>, is used to quantify favourability of gaining electrons (getting reduced). <span class="math inline">\(E^{\circ\prime}\)</span> is the reduction potential measured under biological standard conditions (i.e. pH = 7, oxidised and reduced species are 1M, 298K and 1 atm). When concentrations of the oxidised and reduced species differ:</p>
<p><span class="math display">\[E=E^{\circ\prime} + \dfrac{RT}{zF}\ln\left(\dfrac{\text{[reduced]}}{\text{[oxidised]}}\right)\]</span></p>
<p>For two half equations, A<sup>+</sup> + <span class="math inline">\(e^-\rightarrow\)</span> A and <span class="math inline">\(\text{B} + e^- \rightarrow\)</span> B<sup>-</sup> with <span class="math inline">\(E\)</span> of <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> respectively, a full reaction equation can be constructed, which can be A<sup>+</sup> + B<sup>-</sup> <span class="math inline">\(\rightarrow\)</span> A + B (with <span class="math inline">\(E_\text{r}\)</span> = x - y) or the reverse (with <span class="math inline">\(E_\text{r}\)</span> = y - x); the one with positive <span class="math inline">\(E_\text{r}\)</span> (reaction potential) will actually occur.</p>
<p>Electron transfers occur because electrons are moving down to a lower energy level. In simple cases, atoms with greater electronegativity have greater affinity for electrons and thus a more positive reduction potential, and <em>vice versa</em>. This is the case for strong oxidants F<sub>2</sub> and Cl<sub>2</sub> (electronegative; highly positive <span class="math inline">\(E\)</span>) and strong reductants Li and Na (electropositive, highly negative <span class="math inline">\(E\)</span>). In most cases, however, other factors, such as change in state, conformation, or interactions with water, will significantly affect the reduction potential. For example, Cu and Ag is less electronegative than hydrogen, but their monovalent ions are more easily reduced than H<sup>+</sup>. That is mainly because the formation of metallic solid upon reduction is highly exergonic. Similarly, although the atoms involved in the redox reactions in the repiratory chain are not variable (Cu and Fe), they have different <span class="math inline">\(E\)</span> due to the enviornment (e.g. proximity to the P/N face, identity of ligands).</p>
<p>The series of cofactors along the electron transport chain have a generally steadily increasing <span class="math inline">\(E^{\circ\prime}\)</span>, so electrons are moved toward increasingly lower energy levels. There are some exceptions to this trend, but <span class="math inline">\(E_\text{r}\)</span> would be made positive in these cases by making a concentration difference (usually by means of ‘pulling’ from the next redox centre with favourable <span class="math inline">\(E\)</span>).</p>
<p>While the thermodynamic feasibility of the redox reactions is solely determined by the difference in the reduction potentials (which is proportional to Gibbs free energy), the kinetics (rates) are affected by more factors.</p>
<p>Electrons are passed from one centre to another by <em>tunnelling</em>. Specifically, the wave function for an electron in the donor shows there is a probability that the electron will be found at an acceptor some distance away. <em>Tunnelling</em> is independent of temperature and orientation of chemical groups, but is instead influenced by 1) the distance between the electron donor and acceptor, 2) the redox potential difference, <span class="math inline">\(E\)</span>, and 3) the ‘reorganisation energy’, <span class="math inline">\(\lambda\)</span>.</p>
<p>Within each respiratory complex, the redox centres are close to each other—no more than 14Å apart (the maxium 14Å is found between N5 and N6a Fe-S clusters in complex I). Ubiquinone has binding sites in complex I (II) and III in close proximity to the redox centres from which it receives and to which it delivers electrons, and the same is true for cyt <em>c</em> (which binds to complex III and IV). This arrangement makes the numerous electron transfer steps not the limiting factor in the respiratory chain.</p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p><strong>What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.</strong></p>
<div id="two-proton-translocation-models-loop-and-pump" class="section level2">
<h2>Two Proton Translocation Models: Loop and Pump</h2>
<p>Mechanisms of coupling redox reactions to proton translocation can be classified into two models: the chemical loop, and the conformational pump.</p>
<p>In the loop model shown in Figure <a href="#fig:loop">1</a>, the hydrogens (protons plus electrons) from the substrate (AH<sub>2</sub>) first reduce an mediator molecule (B) on the N side, and this mediator is reoxidised, forcing protons to be released to the P side via one pathway and electrons to flow back to the N side via another pathway. The electrons then reduce another species (C) using protons from the N side. This mechanism is employed in the respiratory chain, where ubiquinone serves as the ‘mediator’ molecule and ETC components from Rieske Fe-S protein in complex III to Cu<sub>B</sub> centre in complex IV as the electron pathway. The number of protons translocated in this way has to follow a fixed stoichiometry (i.e. 1 proton per electron). (Not shown in this diagram, the reduction of Q near the N face driven by QH<sub>2</sub> oxidation near the P face in the Q cycle is also an example of this mechanism.)</p>
<div class="figure" style="text-align: c"><span id="fig:loop"></span>
<img src="../img/etc-mech-1.png" alt="The loop model. Left, generic form; right, simplified example found in the respiratory chain." width="100%" />
<p class="caption">
Figure 1: The loop model. Left, generic form; right, simplified example found in the respiratory chain.
</p>
</div>
<p>The key idea of the pump mechanism is the conformational change driven by redox reactions. Conformational change can lead to alternating proton accessibility from the two faces and differential pKa values of proton binding sites, resulting in proton translocation (explained in Figure <a href="#fig:pump">2</a>). The number of protons pumped in this way does not have to follow a fixed stoichiometry within the thermodynamic constraints (i.e. <em>E</em><sub>r</sub>/n must be greater than ∆p). This mechanism is thought to be employed in complex I and IV of the respiratory chain. However, recent studies have shown that complex IV may adopt a unique mechanism, as explained later in this essay.</p>
<div class="figure" style="text-align: c"><span id="fig:pump"></span>
<img src="../img/etc-mech-2.jpg" alt="The conformational pump mechanism. When the &quot;pump&quot; protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation." width="100%" />
<p class="caption">
Figure 2: The conformational pump mechanism. When the “pump” protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation.
</p>
</div>
</div>
<div id="complex-i" class="section level2">
<h2>Complex I</h2>
<p>As shown in Figure <a href="#fig:3m9s">3</a>, bacterial complex I (from <em>Thermus thermophilus</em>) comprises two domains. The hydrophobic, transmembrane domain (a.k.a. membrane arm) has a proton-translocating P-module and subunit NuoH. The hydrophilic domain (a.k.a. peripheral domain) has an NADH-oxidising dehydrogenase module (N-module) which feeds electrons to the electron-transferring chain of Fe-S clusters, and the connecting Q-module, which reduces ubiquinone. Mammalian mitochondrial complex I is similar in gross organisation and function, although it possesses more subunits.</p>
<div class="figure" style="text-align: c"><span id="fig:3m9s"></span>
<img src="../img/3M9S.png" alt="Annotated structure of bacterial complex I. PDB accession code: 3M9S" width="100%" />
<p class="caption">
Figure 3: Annotated structure of bacterial complex I. PDB accession code: 3M9S
</p>
</div>
<div id="hydrophilic-domain-peripheral-arm" class="section level3">
<h3>Hydrophilic domain (peripheral arm)</h3>
<p>All FMN and the seven (bacteria have additional two redundant ones) Fe-S centres reside in the hydrophilic domain. FMN, which is located at the extremity of the hydrophilic arm, accepts two electrons from NADH<sub>2</sub> and transfers them one by one to the nearest Fe-S centre (N3). Electrons then flow from N3 through a series of isopotential Fe-S centres and finally to N2, whose <span class="math inline">\(E^\circ\)</span> is significantly less negative (-100 mV) <span class="citation">(Sazanov and Hinchliffe 2006)</span>. The ubiquinone is then reduced by two one-electron gaining from N2; the existence of at least two forms of semiquinone is identified by electron paramagnetic resonance.</p>
</div>
<div id="hydrophobic-domain-membrane-arm" class="section level3">
<h3>Hydrophobic domain (membrane arm)</h3>
<p>The hydrophobic domain comprises subunits NuoAHJKLMN, of which NuoLMN are homologous to bacterial Mrp Na<sup>+</sup>/H<sup>+</sup> antiporters (characterised by the conserved 14 transmembrane helices).</p>
<p>At the centre of each antiporter-like subunit (NuoLMN), two transmembrane helices are ‘broken’ and each have a kink known as a ‘π-bulge’, and there is a lysine residue in this region (Glu407 for NuoM) critical to proton pumping, confirmed by mutagenesis studies. There are two symmetry-related plausible proton channels lined by hydrophilic/charged residues, one connecting the central lysine on one broken TM to the P face and the other connecting lysine of the other TM to the N face.</p>
<div class="figure" style="text-align: c"><span id="fig:lys"></span>
<img src="../img/membrane-arm.png" alt="Plausible proton paths" width="100%" />
<p class="caption">
Figure 4: Plausible proton paths
</p>
</div>
<p>As shown in Figure <a href="#fig:proton-path">5</a>, NuoL/M/N may work by using conformational changes (induced from the hydrophilic domain) to transfer a proton from the lysines on the helix with N half channel to the other lysine (or Glu in NuoM) on the helix with P half channel, and if the pK<sub>a</sub>s of the two lysines differ, this leads to unidirectional proton transport. This mechanism is in accordance with the general scheme shown in Figure <a href="#fig:pump">2</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:proton-path"></span>
<img src="../img/proton-path.png" alt="Schematic diagram of proton translocation in complex I" width="100%" />
<p class="caption">
Figure 5: Schematic diagram of proton translocation in complex I
</p>
</div>
<p>The three antiporter-related subunits are thought to be responsible for the translocation of 3 protons per 2 <span class="math inline">\(e^-\)</span>. An additional channel is found at the interface of subunits NuoN,K,J,A (according to <span class="citation">Efremov and Sazanov (2011)</span> and <span class="citation">Sazanov (2015)</span> or NuoH,J,K according to <span class="citation">Hirst (2013)</span> and <span class="citation">Nicholls and Ferguson (2013)</span>) and they are thought to act together to pump another proton with a similar mechanism involving protonation/deprotonation of lysine.</p>
</div>
<div id="coupling" class="section level3">
<h3>Coupling</h3>
<p>It is debated how the redox reactions in the hydrophilic domain triggers conformational change, and how this conformational change is transmitted along the hydrophobic domain.</p>
<p>Some features of the Fe-S centre N2 (the last Fe-S centre in complex I) led to hypotheses in which N2 plays a major role in pumping protons<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><sup>,</sup><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, but currently, the mainstream idea is that the initial conformational change in the hydrophilic domain is contributed mostly by the reduction of ubiquinone.</p>
<p>As claimed by <span class="citation">Wirth et al. (2016)</span>, reduction of ubiquinone is a two-step process, where the Q binding site cycles between a conformation that permits electron transfer (E-state), and a conformation that permits proton transfer (P-state) onto intermediates of ubiquinone. The conformational change associated with this controlled process is assumed to initiate proton translocation.</p>
<p>It was originally suggested that the horizontal HL helix (see Figure <a href="#fig:3m9s">3</a>) in NuoL could move horizontally like a piston to transmit conformational change along the hydrophobic domain<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>, but now, after realising the similarity between NuoH and NuoL/M/N, it is thought that conformational changes associated with π-bulges in the mid-membrane parts are transmitted laterally to NuoN/M/L <span class="citation">(Nicholls and Ferguson 2013)</span>, and, as suggested by <span class="citation">Wirth et al. (2016)</span>, this is achieved by electrostatic coupling in the central hydrophilic axis.</p>
<p>As I mentioned last week <span class="citation">(Shi 2019)</span>, as there is no complete evidence demonstrating how complex I proton pump works, other hypotheses exists, including a radical one recently proposed by <span class="citation">Morelli et al. (2019)</span>, which emphasises the elusiveness of the ‘proton entrance half channel’ (from the middle Lys to the N face) contrasted by the clarity of the ‘proton exit half channel’ (to the P face). Together with another piece of evidence that an obvious proton tunnelling is found at the centre of complex I, Morelli drafted an lateral mechanism of proton circuiting, as shown in Figure <a href="#fig:Morelli2019">6</a></p>
<div class="figure" style="text-align: c"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side" width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side
</p>
</div>
</div>
</div>
<div id="complex-iii-cyt-bc1-complex" class="section level2">
<h2>Complex III (cyt <em>bc<sub>1</sub></em> complex)</h2>
<p>Complex III is a homodimer, with each monomer consisting of three subunits: cytochrome b, the Rieske iron-sulfur protein, and cytochrome c1. Each monomer has two binding sites for ubiquinone, called Q<sup>N</sup> and Q<sup>P</sup>, indicating their proximities to two opposite faces.</p>
<p>Uniquinone oxidation and proton translocation is coupled in the Q cycle, as illustrated in Figure <a href="#fig:ciii">7</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:ciii"></span>
<img src="../img/complexIII.png" alt="The Q cycle. Left, stage 1; right, stage 2." width="100%" />
<p class="caption">
Figure 7: The Q cycle. Left, stage 1; right, stage 2.
</p>
</div>
<p>In stage 1, QH<sub>2</sub> binds to Q<sub>N</sub> of one dimer and a Q binds to the Q<sub>P</sub> of another dimer. The QH<sub>2</sub> at Q<sub>N</sub> is oxidised by two electron acceptors in two steps: the first electron is accepted by the Rieske iron-sulfur protein and passed though cytochrome c<sub>1</sub> and finally to cytochrome c, and second other electron is accepted by cytochrome b, passed through its two hemes (b<sub>H</sub> and b<sub>L</sub>) and finally to the Q at Q<sub>P</sub> on the other monomer, forming semiquinone. The two protons of QH<sub>2</sub> are released into the P face (the molecular details is unclear).</p>
<p>In stage 2, the same process is repeated—one QH<sub>2</sub> is oxidised to Q, its protons released and electrons passed 1) onto one cytochrome c and 2) to the other side—but this time the electron acceptor on the other side is semiquinone. Acceptance of an electron by semiquinone, and the addition of two protons from the matrix, produces QH2.</p>
</div>
<div id="complex-iv" class="section level2">
<h2>Complex IV</h2>
<p>Complex IV (cytochrome oxidase) accepts electrons from cytochrome c to reduce oxygen to water and pumps 2 electrons to the P face.</p>
<p>Previous X-ray studies suggested that complex IV exists as dimers with each monomer comprising 13 subunits <span class="citation">(Tsukihara et al. 1996)</span>. However, it was recently shown by <span class="citation">Zong et al. (2018)</span> that complex IV is actually a 14-subunit monomer, and dimerisation was an artifect due to the dissociation of NDUFA4 subunit during the detergent-based purification steps before crystallisation (NDUFA4 hampers dimerisation).</p>
<div id="reduction-of-oxygen" class="section level3">
<h3>Reduction of oxygen</h3>
<p>The key catalytic activities are found in subunit I and II, as shown in Fig. <a href="#fig:complexIV">8</a></p>
<div class="figure" style="text-align: c"><span id="fig:complexIV"></span>
<img src="../img/complexIV.png" alt="Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II." width="100%" />
<p class="caption">
Figure 8: Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II.
</p>
</div>
<p>Each cytochrome c first donates one electron to Cu<sub>A</sub> located in the globular domain of subunit II. Cu<sub>A</sub> is a binulear centre, but it undergoes one-electron redox reactions. The electron is then passed onto haem <em>a</em>, and then to haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binuclear centre, where an oxygen molecule is bound. 4 electrons from cyt c and 4 protons from the N phase are required to reduce one oxygen molecule to two water molecules. Cu<sub>B</sub> has three His ligands, and the forth empty coordination site mediates oxygen reduction by holding the intermediates (its malfunction leads to generation of ROS).</p>
</div>
<div id="pump-mechanism" class="section level3">
<h3>Pump mechanism</h3>
<p>For each pair of electrons, 2 H<sup>+</sup> from N phase is consumed to make water and another 2 H<sup>+</sup> is pumped intot he P phase.</p>
<p>According to a recent review by <span class="citation">Wikström and Sharma (2018)</span>, the favoured charge-conpensation mechanism is as follows: (see Figure <a href="#fig:civ-p">9</a>)</p>
<ol style="list-style-type: decimal">
<li>when an electron is transferred from haem <em>a</em> to the haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binulear centre (BNC), one proton from Glu242 is loaded onto the ‘proton loading site’ (PLS<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>). This is a purely electrostatic event (not acid-base, as pK<sub>a</sub> of PLS is not high enough).</li>
<li>reprotonated Glu242 transfers another proton to BNC, annihilating the -ve charge in BNC, so H<sup>+</sup> on PLS can leave. There must be a barrier preventing H<sup>+</sup> flowing back to the N side, which can be achieved by raising the energy of the transition state of proton transfer between the Glu242 and PLS.</li>
</ol>
<div class="figure" style="text-align: c"><span id="fig:civ-p"></span>
<img src="../img/civ-p.png" alt="Charge-compensation mechanism of proton pumping in complex IV." width="70%" />
<p class="caption">
Figure 9: Charge-compensation mechanism of proton pumping in complex IV.
</p>
</div>
</div>
</div>
<div id="components-of-pmf-in-chloroplasts-and-mitochondria" class="section level2">
<h2>Components of pmf in chloroplasts and mitochondria</h2>
<p>In mitochondria, the major component of protomotive force (<span class="math inline">\(\Delta p\)</span>) is the membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), while in chloroplasts it is <span class="math inline">\(\Delta\)</span>pH. This is caused by the physiology of these organelles and the electrical properties of membranes.</p>
<p>In chloroplasts, when protons are being pumped into the thylakoid space, the developed membrane potential is balanced out by efflux of K<sup>+</sup> and influx of Cl<sup>–</sup> through voltage-gated channels, such as KEA3 K<sup>+</sup>/H<sup>+</sup> antiporter and voltage-gated VCCN1 Cl<sup>–</sup> channel, so it results in a pure pH difference. TPK3 was originally thought to be one of such channels but was recently shown to be not critical <span class="citation">(Höhner et al. 2019)</span>. There are not many enzymes in the thylakoid lumen, so a low pH is not unsafe. In addition, the high H<sup>+</sup> concentration has regulatroy functions. <span class="citation">(Höhner et al. 2019)</span></p>
<p>If mitochondria were allowed to develop such a pH difference (very basic matrix), this would denature the numerous enzymes, such as those involved in <span class="math inline">\(\beta\)</span>-oxidation and TCA cycle, that operate inside the matrix. In mitochondria, there are not such ion channels to counteract change in <span class="math inline">\(\Delta \Psi\)</span>, so proton translocation results in charge imbalance, leading to <span class="math inline">\(\Delta \Psi\)</span> change<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a>, and a significant membrane potential can be established with only minuscule amount of charge movement.This can be explained by the low capacitance of the membrane. Capacitance is the amount of charge separation needed to develop unit voltage difference, i.e.</p>
<p><span class="math display">\[C=\dfrac{Q}{V}\]</span></p>
<p>Given a constant <span class="math inline">\(V\)</span> (<span class="math inline">\(\Delta \Psi\)</span>), lower capacitance means smaller charge difference is required to build up that voltage difference.</p>
<p>Figure <a href="#fig:capacitance">10</a> shows a simple parallel-plate capacitor, which is analogous to mitochondrial inner membrane.</p>
<div class="figure" style="text-align: c"><span id="fig:capacitance"></span>
<img src="img/capacitance.png" alt="Illustration of a simple parallel-plate capacitor" width="70%" />
<p class="caption">
Figure 10: Illustration of a simple parallel-plate capacitor
</p>
</div>
<p>The electrical capacitance of such a plate can be calculated as:</p>
<p><span class="math display">\[C = \dfrac{\epsilon A}{d}\]</span></p>
<p>where <span class="math inline">\(\epsilon\)</span> is the permittivity of the plate, <span class="math inline">\(A\)</span> is the surface area and <span class="math inline">\(d\)</span> is the distance separating charges. For mitochondrial inner membrane, although <span class="math inline">\(\dfrac{A}{d}\)</span> is large, <span class="math inline">\(\epsilon\)</span> is very small, so the overall capacitance is small.</p>
<div id="a-failed-quantitative-analysis" class="section level4">
<h4>A failed quantitative analysis</h4>
<p>According to <span class="citation">Liang et al. (2017)</span>, the cell membrane capacitance is aroud 10 mF/m<sup>2</sup>, and in the <em>Bioenergetics</em> book <span class="citation">(Nicholls and Ferguson 2013)</span>, the capacitance of the mitochondrial inner membrane takes the same valu. According to a model built by <span class="citation">Mannella, Lederer, and Jafri (2013)</span>, one mitochondria with matrix volume of <span class="math inline">\(9.6\times10^4\)</span> nm^3 would have a inner membrane (cristae) surface area of <span class="math inline">\(5.76\times10^6\)</span> nm<sup>2</sup>.</p>
<p>With these values, to build up a <span class="math inline">\(\Delta \Psi\)</span> of -140 mV, the charge required is:</p>
<p><span class="math display">\[Q=CV=0.01\text{F m}^{-2} \times 5.76 \times 10^{-12}\text{m}^{-2}\times 0.14\text{V}=8.1 \times 10^{-15} \text{ C}\]</span></p>
<p>which corresponds to <span class="math inline">\(8.1 \times 10^{-15}/96450=8.4 \times 10^{-20}\)</span> moles of H<sup>+</sup>, or <span class="math inline">\(8.4 \times 10^{-20}/(9.6\times 10^{-20})=0.875\)</span> M (concentration), which is not realistic…</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Efremov-2011">
<p>Efremov, Rouslan G., and Leonid A. Sazanov. 2011. “Structure of the Membrane Domain of Respiratory Complex I.” <em>Nature</em> 476 (7361): 414–20. <a href="https://doi.org/10.1038/nature10330">https://doi.org/10.1038/nature10330</a>.</p>
</div>
<div id="ref-Hirst-2013">
<p>Hirst, Judy. 2013. “Mitochondrial Complex I.” <em>Annual Review of Biochemistry</em> 82 (1). Annual Reviews: 551–75. <a href="https://doi.org/10.1146/annurev-biochem-070511-103700">https://doi.org/10.1146/annurev-biochem-070511-103700</a>.</p>
</div>
<div id="ref-Hohner-2019">
<p>Höhner, Ricarda, Viviana Correa Galvis, Deserah D. Strand, Carsten Völkner, Moritz Krämer, Michaela Messer, Firdevs Dinc, et al. 2019. “Photosynthesis in Arabidopsis Is Unaffected by the Function of the Vacuolar K<span class="math inline">\(&lt;\)</span>sup<span class="math inline">\(&gt;\)</span>+<span class="math inline">\(&lt;\)</span>/Sup<span class="math inline">\(&gt;\)</span> Channel Tpk3.” <em>Plant Physiology</em> 180 (3): 1322. <a href="https://doi.org/10.1104/pp.19.00255">https://doi.org/10.1104/pp.19.00255</a>.</p>
</div>
<div id="ref-Liang:2017">
<p>Liang, Wenfeng, Yuliang Zhao, Lianqing Liu, Yuechao Wang, Wen Jung Li, and Gwo-Bin Lee. 2017. “Determination of Cell Membrane Capacitance and Conductance via Optically Induced Electrokinetics.” <em>Biophysical Journal</em> 113 (7): 1531–9. <a href="https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006">https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006</a>.</p>
</div>
<div id="ref-Mannella-2013">
<p>Mannella, Carmen A., W. Jonathan Lederer, and M. Saleet Jafri. 2013. “The Connection Between Inner Membrane Topology and Mitochondrial Function.” <em>Journal of Molecular and Cellular Cardiology</em> 62: 51–57. <a href="https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001">https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Nelson-2017">
<p>Nelson, David L., and Michael M. Cox. 2017. <em>Lehninger Principles of Biochemistry</em>. Book. 7th ed. W. H. Freeman. <a href="https://search.library.wisc.edu/catalog/999964334502121">https://search.library.wisc.edu/catalog/999964334502121</a>.</p>
</div>
<div id="ref-Nicholls-2013">
<p>Nicholls, David G., and Stuart J. Ferguson. 2013. <em>Bioenergetics</em>. Book. 4th ed. Amsterdam: Academic Press, Elsevier.</p>
</div>
<div id="ref-Roessler-2010">
<p>Roessler, Maxie M., Martin S. King, Alan J. Robinson, Fraser A. Armstrong, Jeffrey Harmer, and Judy Hirst. 2010. “Direct Assignment of Epr Spectra to Structurally Defined Iron-Sulfur Clusters in Complex I by Double Electron–Electron Resonance.” <em>Proceedings of the National Academy of Sciences</em> 107 (5): 1930. <a href="https://doi.org/10.1073/pnas.0908050107">https://doi.org/10.1073/pnas.0908050107</a>.</p>
</div>
<div id="ref-Sazanov-2015">
<p>Sazanov, Leonid A. 2015. “A Giant Molecular Proton Pump: Structure and Mechanism of Respiratory Complex I.” <em>Nature Reviews Molecular Cell Biology</em> 16. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved. SN -: 375 EP. <a href="https://doi.org/10.1038/nrm3997">https://doi.org/10.1038/nrm3997</a>.</p>
</div>
<div id="ref-Sazanov-2006">
<p>Sazanov, Leonid A., and Philip Hinchliffe. 2006. “Structure of the Hydrophilic Domain of Respiratory Complex I from Thermus Thermophilus.” <em>Science</em> 311 (5766): 1430. <a href="https://doi.org/10.1126/science.1123809">https://doi.org/10.1126/science.1123809</a>.</p>
</div>
<div id="ref-Shi-2019">
<p>Shi, Tianyi. 2019. “Principle of Chemiosmotic Mechanism.” 2019.</p>
</div>
<div id="ref-Tsukihara-1996">
<p>Tsukihara, Tomitake, Hiroshi Aoyama, Eiki Yamashita, Takashi Tomizaki, Hiroshi Yamaguchi, Kyoko Shinzawa-Itoh, Ryosuke Nakashima, Rieko Yaono, and Shinya Yoshikawa. 1996. “The Whole Structure of the 13-Subunit Oxidized Cytochrome c Oxidase at 2.8 Å.” <em>Science</em> 272 (5265): 1136. <a href="https://doi.org/10.1126/science.272.5265.1136">https://doi.org/10.1126/science.272.5265.1136</a>.</p>
</div>
<div id="ref-Verkhovskaya-2013">
<p>Verkhovskaya, Marina, and Dmitry A. Bloch. 2013. “Energy-Converting Respiratory Complex I: On the Way to the Molecular Mechanism of the Proton Pump.” <em>The International Journal of Biochemistry &amp; Cell Biology</em> 45 (2): 491–511. <a href="https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024">https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024</a>.</p>
</div>
<div id="ref-Wikstrom-2018">
<p>Wikström, Mårten, and Vivek Sharma. 2018. “Proton Pumping by Cytochrome c Oxidase – a 40 year Anniversary.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1859 (9): 692–98. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009">https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009</a>.</p>
</div>
<div id="ref-Wirth-2016">
<p>Wirth, Christophe, Ulrich Brandt, Carola Hunte, and Volker Zickermann. 2016. “Structure and Function of Mitochondrial Complex I.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1857 (7): 902–14. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013">https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013</a>.</p>
</div>
<div id="ref-Zong-2018">
<p>Zong, Shuai, Meng Wu, Jinke Gu, Tianya Liu, Runyu Guo, and Maojun Yang. 2018. “Structure of the Intact 14-Subunit Human Cytochrome c Oxidase.” <em>Cell Research</em> 28 (10): 1026–34. <a href="https://doi.org/10.1038/s41422-018-0071-1">https://doi.org/10.1038/s41422-018-0071-1</a>.</p>
</div>
<div id="ref-Zwicker-2006">
<p>Zwicker, K., A. Galkin, S. Drose, L. Grgic, S. Kerscher, and U. Brandt. 2006. “The Redox-Bohr Group Associated with Iron-Sulfur Cluster N2 of Complex I.” Journal Article. <em>J Biol Chem</em> 281 (32): 23013–7. <a href="https://doi.org/10.1074/jbc.M603442200">https://doi.org/10.1074/jbc.M603442200</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>In some papers <span class="citation">(Hirst 2013; Verkhovskaya and Bloch 2013)</span> it was suggested that coupling of electron transport to movement of protons depends on the significantly exergonic reduction of the N2 centre by the roughly isopotential wire of Fe-S centres that delivers electrons from NADH, and the large E<sub>m,7</sub> span from N3 to N2 must provide the majority of the driving force for the proton pumping events associated with reduction and reoxidation of N2. This may be incorrect, as newest measurements <span class="citation">(Sazanov 2015)</span> of redox potentials of the full Fe-S series (N3-N1b-N4-N5-N6a-N6b-N2) shows that N3, N4 and N6 are about equipotential at about -250 mV, while N1b, N5 and N6b actually have lower potentials; and such an arrangement is claimed by <span class="citation">Roessler et al. (2010)</span> to be energy efficient.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>The mid-point reduction potential of N2 is shown to be pH-dependent, so that E<sub>acid</sub> &gt; E<sub>alkaline</sub>, and indeed a proton binding site is present there: His226 in <em>Y. lipolytica</em>, and protons are more likely to bind when N2 is reduced. However, shifting the midpoint potential to a more negative and pH insensitive value by exchanging His226 with a methionine affected the functioning of complex I including proton pumping only marginally. <span class="citation">(Zwicker et al. 2006)</span><a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>The alternative role of HL would be to stabilise ends of the helices while permitting movement of the middle hydrophilic axis.<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>Although the identity of PLS is uncertain, many studies indicates that it may be the A-propionate substituent of haem <em>a</em><sub>3</sub>.<a href="#fnref4" class="footnote-back">↩</a></p></li>
<li id="fn5"><p>To be precise, when ∆p = 0, there is still (negative) membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), and H<sup>+</sup> equilibrate under this <span class="math inline">\(\Delta \Psi\)</span>, which results in a lower pH inside (more concentrated H<sup>+</sup>). The <span class="math inline">\(\Delta \Psi\)</span> that contributes to ∆p is actually the <span class="math inline">\(change\)</span> in <span class="math inline">\(\Delta \Psi\)</span> caused by proton translocation.<a href="#fnref5" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/principle-of-chemiosmotic-mechanism.html" class="archive-item-link">
                Principle of chemiosmotic mechanism 
              </a>
              <span class="archive-item-date">
                2019-10-15
              </span>
              <div class="article-summary" style="display: none;">
                
                  What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p class="topic">
<strong>Explain the difference between electrochemical gradient, pH gradient and membrane potentials. How can they be measured and how are these related to chemiosmotic mechanism.</strong>
</p>
<p>The electrochemical gradient describes the difference in chemical stability of ions (typically protons), between two aqueous compartments separated by a non-conducting and non-permeable membrane (typically a energy-transducing membrane such as mitochondrial inner membrane). It is quantified by the difference in Gibbs’ free energy, <span class="math inline">\(\Delta G\)</span>, or its molar equivalent, <em>chemical potential</em>, <span class="math inline">\(\Delta\mu\)</span>, of the ion species (proton) between the two compartments.</p>
<p>Usually, electrochemical gradient is used in studying processes coupled with <span class="math inline">\(\text{H}^+\)</span> movement across a membrane, where one side is positively charged and has a high <span class="math inline">\(\left[\text{H}^+\right]\)</span> and the other side with -ve charge and low <span class="math inline">\(\left[\text{H}^+\right]\)</span>. These are denoted by P and N sides, respectively, and the ‘gradients’ are often defined as the value of the corresponding parameter on the N side minus its value on the P side.</p>
<p>The electrochemical gradient is the sum of pH gradient (concentration gradient) and membrane potential (electrical gradient).</p>
<p>pH gradient is the (log-transformed) difference in the concentration of protons, <span class="math inline">\(\left[\text{H}^+\right]\)</span>, across the membrane. Protons tend to move down its concentration gradient to maximise the entropy of the whole system, and this tendency, i.e. potetial energy is the “chemical” part of the electrochemical potential. Intracellular/intra-orgenellar pH can be measured with pH-sensitive microelectrodes, nuclear magnetic resonance, or pH-sensitive fluorescent proteins <span class="citation">(Loiselle and Casey 2010)</span>. Low-RMM weak acids and bases can equilibrate across the membrane independent of the membrane potential. When there is an pH gradient, however, weak acids accumulate in the basic compartment and bases in the acidic compartment, and this property is used to measure pH gradient.</p>
<p>The membrane potential is the electrical potential difference across the membrane. According to Coulomb’s law, postively charge protons is repelled by the positively charged P side while attracted towards the N side, and this also results in a potential energy, which adds to the electrochemical gradient as the “electro-” part. Membrane potential is measured with microelectrodes.</p>
<p>Considering only the effect of concentration and electrical potential, the electrochemical potential for the proton on either side is defined as follows:</p>
<p><span class="math display">\[\bar{\mu}_{\text{H}^+} = \bar{\mu}_{\text{H}^+}^\circ + RT\ln\left[{\text{H}^+}\right] + z_{\text{H}^+}F\Psi\]</span></p>
<p>where <span class="math inline">\(\bar{\mu}_{\text{H}^+}^\circ\)</span> is the chemical potential under standard states, <span class="math inline">\(RT\ln\left[{\text{H}^+}\right]\)</span> and <span class="math inline">\(z_{\text{H}^+}F\Psi\)</span> are the additional chemical potential due to concentration and electrical potential, respectively.</p>
<p>The electrochemical gradient is the difference of the electrochemical potential between two compartments:</p>
<p><span class="math display">\[\Delta\bar{\mu} = \bar{\mu}_{\text{N}} - \bar{\mu}_{\text{P}} = \left(RT\ln\left[{\text{H}^+}\right]_{\text{N}} + z_{\text{H}^+}F\Psi_{\text{N}}\right) - \left(RT\ln\left[{\text{H}^+}\right]_{\text{P}} + z_{\text{H}^+}F\Psi_{\text{P}}\right)\]</span></p>
<p><span class="math display">\[ = -\ln{10}RT\left(\text{pH}_\text{N}-\text{pH}_\text{P}\right) + F(\Psi_{\text{N}}-\Psi_{\text{P}})=F\Delta\Psi- 2.3RT\Delta\text{pH} \text{  (J mol}^{-1}\text{)}\]</span></p>
<p>or it can be expressed as ‘protomotive force’ in (milli)volts:</p>
<p><span class="math display">\[\Delta\text{p}=\Delta\Psi-\dfrac{2.3RT}{F}\Delta\text{pH}\]</span></p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p class="topic">
<strong>What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?</strong> <br><sub><em>The essay MUST be equipped with at least 3 annotated diagrams and the length should be 1500-1750 words. Remember to include at least one KEY experimental evidence in the essay.</em></sub>
</p>
<div id="clues-leading-to-the-original-chemiosmosis-hypothesis" class="section level2">
<h2>Clues leading to the original chemiosmosis hypothesis</h2>
<p>The mechanism of oxidative phosphorylation were initially thought to be analogous to substrate-level phosphotylation, that the redox reactions create ‘high-energy’ intermediates, which in turn adds P<sub>i</sub> to ADP. In 1961, Peter Mitchell listed 6 facts to question this chemical mechanism hypothesis:</p>
<ol style="list-style-type: decimal">
<li>The ‘high-energy’ intermediate is not found.</li>
<li>The close association between phosphorylation and membranous structrures is not explained.</li>
<li>P/O ratio varies–it does not follow a fixed stoichiometry.</li>
<li>ATP hydrolysis outside mitochondria promotes NAD reduction inside, accentuated by succinate oxidation. (this observation is probably irrelevant or wrong?)</li>
<li>Uncoupling can be caused by reagents with distinct chemical properties–so they are unlikely to act on a single ‘intermediate’.</li>
<li>Unexplained swelling and shrinkage accompany phosphorylation.</li>
</ol>
<p>Trying to explain these phenomena, he proposed his initial chemiosmosis mechanism, with following 3 features:</p>
<ol style="list-style-type: decimal">
<li>A membrane located, reversible and anisotropic ‘ATPase’</li>
<li>An electron and hydrogen translocation system</li>
<li>A charge impermeable membrane</li>
</ol>
<p>According to this model, oxidation of substrates does not occur in a single compartment, but it instead allow protons and electrons to go in different directions, which in turn creates a concentration gradient of H<sup>+</sup> and OH<sup>-</sup>, as well as a membrane potential. ATP synthesis is driven by the one-way movement of OH<sup>-</sup> down its electrochemical gradient.</p>
<p>Mitchell’s origin model was rather different from the current model, as depicted in figure <a href="#fig:mitchell1961">1</a>, but it provided decent explanations for the phenomena listed above. (In his later review <span class="citation">(Mitchell 1966)</span>, there were some amendments made on his original model, but I have no time to read it)</p>
<div class="figure"><span id="fig:mitchell1961"></span>
<img src="img/mitchell1961.jpg" alt="Left, Mitchell's original chemiosmosis model; right, the current model" width="100%" />
<p class="caption">
Figure 1: Left, Mitchell’s original chemiosmosis model; right, the current model
</p>
</div>
<p>The principal idea was that, energy released from ‘asymmetric’ redox reactions can be used to establish a rather ‘physical’ form of energy (not the common chemical bond energy), the electrochemical gradient, and this is the first coupling. The second coupling is between the synthesis of ATP and the vectorised diffusion of some species promoted by the electrochemical gradient established by the first coupling.</p>
<p>It should be noted that the conception of first coupling was not new. Before the 1960s, it was known that redox reactions can create a electrical potential between two compartments, and it had been noted that some inhibitors of ion transport also has an effect (either stimulatory or inhibitory) on respiratory rate <span class="citation">(Robertson 1960)</span> (Fig. <a href="#fig:fig1">2</a>).</p>
<div class="figure"><span id="fig:fig1"></span>
<img src="img/chemiosmosis-inhibitor.png" alt="Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors" width="100%" />
<p class="caption">
Figure 2: Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors
</p>
</div>
</div>
<div id="proving-proton-pumping-by-the-redox-chain-and-determining-ho-ratio" class="section level2">
<h2>Proving proton pumping by the redox chain and determining H<sup>+</sup>/O ratio</h2>
<p>The ‘first coupling’ of the current chemiosmosis model means the pumping of protons into the P compartment powered by stepwise redox reactions along the ETC, with the overall oxidation of NADH or FADH<sub>2</sub> as substrate by molecular oxygen.</p>
<p>In actively respiring mitochondria, the proton concentrations (i.e. pH) on either side of the membrane is constant because the rate at which protons are pumped into the P compartment and that flowing back to the N compartment are equal. This makes it difficult to detect the presence of proton flux.</p>
<p><span class="citation">Mitchell and Moyle (1967)</span> devised the following procedure to solve the problem (Fig. <a href="#fig:mitchell1967a">3</a>):</p>
<ol style="list-style-type: decimal">
<li>mitochondria are incubated in anaerobic conditions, allowing H<sup>+</sup> to equilibrate (i.e. making <span class="math inline">\(\Delta p\)</span> fall back to zero)</li>
<li>a pH-sensitive glass electrode is used to monitor pH of the suspension (i.e. the environment outside the mitochondria)</li>
<li>a known quantity of O<sub>2</sub> is introduced into the suspension by ‘injecting pulses of air-saturated 150mM KCI solution.’</li>
<li>by measuring the pH change, the amount of protons pumped out of the mitochondria can be calculated, and with the known amount of O<sub>2</sub>, the H<sup>+</sup>/O ratio can be calculated.</li>
</ol>
<div class="figure"><span id="fig:mitchell1967a"></span>
<img src="img/mitchell1967a.jpg" alt="Determining H/O ratio (Mitchell 1967a)" width="100%" />
<p class="caption">
Figure 3: Determining H/O ratio (Mitchell 1967a)
</p>
</div>
<p>Additional procedures are needed to minimise error, and they’re described in <em>Bioenergetics</em> pp59-60</p>
</div>
<div id="proving-the-atp-synthesis-is-driven-by-the-electrochemical-gradient" class="section level2">
<h2>Proving the ATP synthesis is driven by the electrochemical gradient</h2>
<p>The electrochemical gradients is the sum of concentration gradient and electrical potential gradient, and each of them are shown to drive ATP synthesis.</p>
<p><span class="citation">Jagendorf and Uribe (1966)</span> showed ATP synthesis can be induced in dark, by artificially creating a pH difference across the thylakoid membrane in chloroplasts (Fig <a href="#fig:Sone">4</a>). In this experiment, broken chloroplasts (extracted from spinach) are first incubated in an acidic solution, allowing the thylakoid space to achieve the enviornmental pH of 4.0 (different acids are used to verify its non-specificity). Next, chloroplasts are exposed to pH 8.0 to develop the proton gradient across the thylakoid membrane. While moving out from the thylakoid space, protons drive ATP synthase. The amount of ATP produced is then determined by firefly luciferase assay. By repeating with different pHs, the authors concluded that the rate of ATP synthesis is positively correlated with the pH difference between two phases.</p>
<p><span class="citation">Sone et al. (1977)</span> used reconsitituted vesicles, made from purified <span class="math inline">\(\text{F}_o\text{F}_1\)</span> complex and the membrane of a thermophilic bacterium, to achieve ATP synthesis by induced membrane potential (Fig <a href="#fig:Sone">4</a>). They first incubated mitochondria with valinomycin at a fixed pH, then exposed the suspension to KCl. Valinomycin-mediated entry of K<sup>+</sup> shifts the membrane potential so the vesicle interior becomes more positively charged. Consequently, H<sup>+</sup> moves out and drives ATP synthase.</p>
<div class="figure"><span id="fig:Sone"></span>
<img src="img/sone.jpg" alt="Left, Jagendorf (2016); right, Sone (1977)" width="100%" />
<p class="caption">
Figure 4: Left, Jagendorf (2016); right, Sone (1977)
</p>
</div>
</div>
<div id="monitoring-the-protomotive-force-and-proton-flux-in-different-conditions" class="section level2">
<h2>Monitoring the protomotive force and proton flux in different conditions</h2>
<p>If the chemiosmotic model is corrent, it will have some properties resembling those of an electric circuit–the current (<span class="math inline">\(I\)</span>) and electromotive force (<span class="math inline">\(V\)</span>) are mirrored by the proton flux (<span class="math inline">\(J_{\text{H}^+}\)</span>) and the protomotive force (<span class="math inline">\(\Delta\text{p}\)</span>), respectively. These two terms can be monitored in different states of the proton circuit, and the results are in accordance with the chemiosmotic model.</p>
<div class="figure"><span id="fig:circuit-analogy"></span>
<img src="img/circuit_analogy.jpg" alt="Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state." width="100%" />
<p class="caption">
Figure 5: Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state.
</p>
</div>
<p>The different states shown in Fig <a href="#fig:circuit-analogy">5</a> are explained below:</p>
<ol style="list-style-type: decimal">
<li>When ETC substrate is absent, there is absolutely no oxygen consumption. No active forces act protons so they are allowed to equilibrate, and therefore the <span class="math inline">\(\Delta\text{p}\)</span> is zero. This resembles an electrical circuit without a power source.</li>
<li>When substrate (e.g. succinate) is provided, it is oxidised by ETC and oxygen is consumed. ETC starts to pump H<sup>+</sup> into the P face, building up the electrochemical gradient If the membrane was perfectly impermeable to H<sup>+</sup>, ETC would quickly stop as its provided energy for proton efflux balances the electrochemical gradient which favours protons’ influx, and there would be no oxygen consumption. However, the membrane is ‘leaky’ (and there are other processes causing influx of H<sup>+</sup>), causing constant influx of H<sup>+</sup>, which in turn allows H<sup>+</sup> to be steadily, but slowly, pumped to the P face, which is accompanied by oxygen usage. This resembles more of a closed circuit where energy is dissipated slowly through a high-resistance resistor than of an ‘open circuit’ as described in some books.</li>
<li>When ADP is added, ATP synthase can use the proton gradient to synthesise ATP. The proton flux through ATP synthase is huge, so the oxygen concentration (whose gradient parallels proton flux) decreases steeply. (I can’t give an rigorous mathematical explanation on the decrease of <span class="math inline">\(\Delta\text{p}\)</span>)</li>
<li>ATP is used up, and the condition is identical to state 2.</li>
<li>FCCP short-circuits the proton circuit by allowing H<sup>+</sup> quickly flowing back to the N face without doing ‘useful work’ (driving ATP synthase). The proton influx is so large that it outstrips the maximal efflux powered by ETC, so <span class="math inline">\(\Delta\text{p}\)</span> quickly falls back to zero.</li>
</ol>
</div>
<div id="lateral-proton-current" class="section level2">
<h2>Lateral proton current?</h2>
<p>The widely accepted view that ETC pumps protons transversally from the N to the P face may need updating for the following reasons stated by <span class="citation">Morelli et al. (2019)</span>:</p>
<ol style="list-style-type: decimal">
<li>H<sup>+</sup> almost never exist as isolated protons. Instead, they bind to water to form H<sub>3</sub>O<sup>+</sup>. Free protons in a mitochondrial periplasmic space are too few (fewer than 10) to drive ATP synthase.</li>
<li>Free protons are destructive for biological membranes</li>
<li>The pH inside mitochondria was shown to be higher by 0.5 units from what was previously believed <span class="citation">(Żurawik et al. 2016)</span></li>
<li>Phospholipid membranes are intrinsically (significantly) permeable to protons</li>
</ol>
<p>According to the classic view of proton pumping, complex I should have a continuous channel traversing the membrane. However, based on comprehensive X-ray studies, the ‘proton entrance half channel’ is not identified with certainty, while the ‘proton exit half channel’ is clearly identifiable, and an obvious proton tunnelling is found at the centre of complex I. This leads to an lateral mechanism of proton circuiting, as shown below:</p>
<div class="figure"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019)." width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019).
</p>
</div>
</div>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Jagendorf:1966">
<p>Jagendorf, A T, and E Uribe. 1966. “ATP Formation Caused by Acid-Base Transition of Spinach Chloroplasts.” <em>Proceedings of the National Academy of Sciences</em> 55 (1). National Academy of Sciences: 170–77. <a href="https://doi.org/10.1073/pnas.55.1.170">https://doi.org/10.1073/pnas.55.1.170</a>.</p>
</div>
<div id="ref-Loiselle:2010">
<p>Loiselle, Frederick B., and Joseph R. Casey. 2010. “Measurement of Intracellular pH.” In <em>Membrane Transporters in Drug Discovery and Development</em>, edited by Qing Yan. Vol. 637. Methods in Molecular Biology. Humana Press.</p>
</div>
<div id="ref-Mitchell:1966">
<p>Mitchell, Peter. 1966. “CHEMIOSMOTIC Coupling in Oxidative and Photosynthetic Phosphorylation.” Journal Article. <em>Biological Reviews</em> 41 (3): 445–501. <a href="https://doi.org/10.1111/j.1469-185X.1966.tb01501.x">https://doi.org/10.1111/j.1469-185X.1966.tb01501.x</a>.</p>
</div>
<div id="ref-Mitchell:1967a">
<p>Mitchell, Peter, and Jennifer Moyle. 1967. “Respiration-driven proton translocation in rat liver mitochondria.” <em>Biochemical Journal</em> 105 (3): 1147–62. <a href="https://doi.org/10.1042/bj1051147">https://doi.org/10.1042/bj1051147</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Robertson:1960">
<p>Robertson, R. N. 1960. “ION Transport and Respiration.” Journal Article. <em>Biological Reviews</em> 35 (2): 231–64. <a href="https://doi.org/10.1111/j.1469-185X.1960.tb01415.x">https://doi.org/10.1111/j.1469-185X.1960.tb01415.x</a>.</p>
</div>
<div id="ref-Sone:1977">
<p>Sone, N, M Yoshida, H Hirata, and Y Kagawa. 1977. “Adenosine Triphosphate Synthesis by Electrochemical Proton Gradient in Vesicles Reconstituted from Purified Adenosine Triphosphatase and Phospholipids of Thermophilic Bacterium.” <em>Journal of Biological Chemistry</em> 252: 2956–60.</p>
</div>
<div id="ref-Zurawik:2016">
<p>Żurawik, Tomasz Michał, Adam Pomorski, Agnieszka Belczyk-Ciesielska, Grażyna Goch, Katarzyna Niedźwiedzka, Róża Kucharczyk, Artur Krężel, and Wojciech Bal. 2016. “Revisiting Mitochondrial pH with an Improved Algorithm for Calibration of the Ratiometric 5(6)-Carboxy-Snarf-1 Probe Reveals Anticooperative Reaction with H+ Ions and Warrants Further Studies of Organellar pH.” <em>PLOS ONE</em> 11 (8). Public Library of Science: 1–17. <a href="https://doi.org/10.1371/journal.pone.0161353">https://doi.org/10.1371/journal.pone.0161353</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>