<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>membrane protein - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="membrane protein - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/membrane-protein/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">membrane protein</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../tutorial/insertion-of-membrane-proteins.html" class="archive-item-link">
                Insertion of Membrane Proteins 
              </a>
              <span class="archive-item-date">
                2020-12-03
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon"><span class="toc-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</a></li>
<li><a href="#single-pass-proteins"><span class="toc-section-number">2</span> Single-Pass Proteins</a>
<ul>
<li><a href="#type-i-proteins"><span class="toc-section-number">2.1</span> Type I Proteins</a></li>
<li><a href="#type-ii-and-iii-proteins"><span class="toc-section-number">2.2</span> Type II and III Proteins</a></li>
</ul></li>
<li><a href="#multi-pass-proteins"><span class="toc-section-number">3</span> Multi-Pass Proteins</a></li>
<li><a href="#insertases"><span class="toc-section-number">4</span> Insertases</a>
<ul>
<li><a href="#insertion-of-tail-anchored-proteins-by-the-get-system"><span class="toc-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</a></li>
<li><a href="#the-roles-of-yidc"><span class="toc-section-number">4.2</span> The Roles of YidC</a></li>
</ul></li>
<li><a href="#beta-barrel-proteins"><span class="toc-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>The majority of integral membrane proteins in an eukaryotic cell are made of one or more <span class="math inline">\(\alpha\)</span>-helices and are first inserted co-translationally via the Sec61 translocon into the endoplasmic reticulum (ER) membrane, assembled there, and then dispatched, via the golgi apparatus and vesicles, to their final destinations. Bacteria use the SecYEG translocon, which is a homolog of Sec61, to insert membrane proteins into their plasma membrane. There are also translocons specific to chroloplasts (TIC-TOC), mitochondria (TIM-TOM) and peroxisomes (PEX), but this essay will focus on the Sec61 translocon.</p>
<div id="targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</h1>
<p>The common co-translationally inserted membrane proteins are targeted to the Sec61 translocon on the ER membrane via the cytosolic signal recognition particle (SRP) and the SRP receptor near the translocon.</p>
<p>SRP is a ribonucleoprotein particle made up of 6 proteins bound to a 300-nucleotide RNA acting as a scaffold. The M and Alu domains of SRP are critical for its functions. The M domain contains a cleft whose inner surface is lined by methionine and other hydrophobic side chains. This cleft binds to a hydrophobic segment of the nascent peptide as soon as it emerges from the exit tunnel. Meanwhile, the Alu domain blocks the elongation factor 2 (eEF2) binding site on the ribosome, pausing translation.</p>
<p>Usually it is the first hydrophobic segment that is recognised by SRP. This segment can either be a pure ER-targeting signal sequence or be the first TM helix itself (see next section). Recognition by SRP not only serves to target the ribosome-nascent protein to the ER but also prevents exposure of the hydrophobic segment to the aqueous environment.</p>
<p>SRP and SRP receptor each has a GTP binding site. When their GTP-bound forms associate with each other, they form two GTPase active sites. The GTPase activity is inhibited by signal peptide binding. Once the nascent chain has left SRP and enters the translocon, the two GTP molecules are hydrolysed, causing dissociation of SRP and its receptor. GTP replaces GDP on both proteins, and they are ready to initiate another round of ER targeting.</p>
<p>The Sec61 translocon is made up of 3 subunits: Sec61<span class="math inline">\(\alpha\)</span>, an integral membrane protein composed of two 5-helix bundles that constitutes the central channel and two smaller proteins, Sec61<span class="math inline">\(\beta\)</span> and Sec61<span class="math inline">\(\gamma\)</span>. The central channel through Sec61<span class="math inline">\(\alpha\)</span> is sealed by a plug made of a helical peptide that only opens during traslocation. The channel can also open laterally by hinging apart the two 5-helix bundles to expose a hydrophobic binding pocket for signal sequences and/or TM helices of the nascent peptide. The verticle opening allows elongation of the nascent peptide chain through the central pore, and the lateral opening allows attachment of signal sequences and exit of TM helices into the membrane (Figure <a href="#fig:sec61">1.1</a>).</p>
<div class="figure"><span id="fig:sec61"></span>
<img src="../img/sec61.png" alt="The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation." width="100%" />
<p class="caption">
Figure 1.1: The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation.
</p>
</div>
</div>
<div id="single-pass-proteins" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Single-Pass Proteins</h1>
<p>A single-pass membrane protein use a single 20-25 residue long <span class="math inline">\(\alpha\)</span>-helix made mainly of hydrophobic amino acids to traverse the membrane. The <span class="math inline">\(\alpha\)</span>-helical structure satisfies hydrogen bonding between main chain carbonyl and amino groups and exposes hydrophobic side chains, which form favourable hydrophobic interactions with the aliphatic core of the membrane bilayer. Co-translational insertion of this helix is a natural strategy to minimise unfavourable interation between the hydrophobic helix and the aqueous cytosolic environment and between the hydrophilic regions of the peptide surrounding the helix and the hydrophobic core of the bilayer.</p>
<p>Single-pass proteins are classified into three types according to the mechanism by which they are inserted. All of them are described in the following sections.</p>
<div id="type-i-proteins" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Type I Proteins</h2>
<p>Type I membrane proteins, like soluble proteins to be translocated into the ER lumen, contain an N-terminal ER-targeting signal sequence to be cleaved. The signal sequence is recognised by SRP, causing the ribosome-nascent peptide complex to be targeted to the Sec61 translocon. Sec61<span class="math inline">\(\alpha\)</span> opens the lateral gate, and the signal peptide fits into the exposed hydrophobic bindng pocket. The sequence following the signal peptide displaces the plug and inserts into the central channel, resulting in the conformation shown in Figure <a href="#fig:mem-insert-type-i">2.1</a>. Signal peptidase cleaves off the signal peptide once it recognises a specific sequence on the C-terminal end of the signal peptide. After cleavage, translation continues and the newly synthesised sequence is threaded through the channel and enters the ER lumen, until another hydrophobic segment is encountered. This segment is the TM helix of the nascent protein, and it acts as a <em>stop-transfer signal</em> (or “stop-transfer anchor sequence”) by triggering the lateral opening of Sec61<span class="math inline">\(\alpha\)</span> and thus allowing this helix to move into the membrane. The C-terminus continues to be synthesised and loops out on the cytosolic side of the membrane. Eventually, the hydrophic N- and C- termini are on the luminal and the cytosolic face, respectively.</p>
<div class="figure"><span id="fig:mem-insert-type-i"></span>
<img src="../img/mem-insert-type-i.png" alt="Insertion of type I membrane proteins." width="100%" />
<p class="caption">
Figure 2.1: Insertion of type I membrane proteins.
</p>
</div>
</div>
<div id="type-ii-and-iii-proteins" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Type II and III Proteins</h2>
<p>Type II and III proteins do not contain a cleavable signal sequence, and they have opposite orientations: the N-terminus is on the cytosolic and the luminal face in type II and type III proteins, respectively.</p>
<p>Type II and III proteins use the sequence of their TM helix as an ER-targeting signal (“start-transfer singal”, or “signal anchor sequence”), and the targeting is also mediated by SRP and SRP receptor. The distribution of positively charged residues around the helix dictates its orientation: the positively charged residues tend to remain on the cytosolic side of the membrane, thus in type II proteins where the N-terminal side has more positively charged residues, the helix adopts the orientation with the N-terminus facing the cytosolic side; and in type III proteins where the C-terminal side has more positively charged residues the opposite orientation is adopted (<br />
Figure <a href="#fig:mem-insert-type-iii-iii">2.2</a> ).</p>

<div class="figure"><span id="fig:mem-insert-type-iii-iii"></span>
<img src="../img/mem-insert-type-iii-iii.png" alt="Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61\(\alpha\) results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol." width="100%" />
<p class="caption">
Figure 2.2: Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61<span class="math inline">\(\alpha\)</span> results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol.
</p>
</div>
</div>
</div>
<div id="multi-pass-proteins" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Multi-Pass Proteins</h1>
<p>Multi-pass membrane proteins (a.k.a. type IV proteins) contain more than one TM <span class="math inline">\(\alpha\)</span> helices and traverse the membrane multiple times. Since the adjacent helices are in opposite orientations, insertion of multipass membrane proteins can be viewed as an alternation of insertion of a type I helix and insertion of a type II helix, which is dictated by the TM helix sequences that alternatingly act as start-transfer signals (signal anchor sequences) and stop-transfer signals (stop-transfer anchor sequences).</p>
<p>In multipass proteins with N-terminus in the cytosol (type IV-A), the TM helix sequence that first emerges from the ribosome (the one closest to the N-terminus) functions in the same way as the start transfer sequence (signal-anchor sequence) of the type II protein, which interacts with SRP to target the protein to the translocon, opens the translocon channel, and allow growing polypeptide chain to pass through the channel. Unlike type II proteins, as the chain grows, the translocon will encounter another hydrophobic TM helix sequence, which then functions as the stop-transfer signal (stop-transfer anchor sequence) in the same way as in type I proteins: the channel is closed, and this helix is moved via the lateral gate to the bilayer, and the nascent chain continues to grow into the cytosol. If a third TM helix sequence is encountered, it will function as another type II-like start-transfer signal, but this time the insertion does not depend on SRP and SRP receptor. This cycle continues until translation completes.</p>
<p>The insertion of multipass proteins with N-terminus in the ER lumen (type IV-B) occurs in a similar manner. The only difference is the first TM helix sequence, which now functions in the same way as the start-transfer sequence of the type III protein, resulting in a closed translocon and the growing peptide to be synthesised in the cytosol. Then, the second, third, fourth TM helix sequences functions in the same way as the first, second, and third TM helix sequences in a type IV-A protein and so on.</p>
<div class="figure"><span id="fig:type-iv-a-b"></span>
<img src="../img/type-iv-a-b.png" alt="Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall." width="100%" />
<p class="caption">
Figure 3.1: Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall.
</p>
</div>
</div>
<div id="insertases" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Insertases</h1>
<p>Some membrane proteins are inserted into the lipid bilayer via Sec-independent mechanisms, which are mediated by factors generally termed ‘insertases’. The bacterial (YidC), mitochondrial (Oxa1), and chrloroplast (Alb3) insertases are evolutionarily related, whereas the ER membrane contains 3 seemingly unrelated insertases: Get1/2 complex, the ER membrane-protein complex (EMC), and TMCO1.</p>
<div id="insertion-of-tail-anchored-proteins-by-the-get-system" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</h2>
<p>Tail-anchored proteins also use an <span class="math inline">\(\alpha\)</span>-helix to traverse the membrane, but this helix is located near the C-terminus. Because there are only few, if any, residues succeeding the C-terminal helix, translation terminates while the helix have not yet emerged from the exit tunnel, so recognition by SRP is not possible, and its insertion into ER membrane depends on a post-translational pathway involving Get1, Get2, and Get3 proteins.</p>
<p>In this pathway, the protein is completely synthesised and released into the cytosol. ATP-bound Get3 binds to the hydrophobic C-terminal tail of the protein (facilitated by some other proteins), and then docks onto the dimeric Get1/Get2 receptor on the ER membrane. Accompanying the hydrolysis of ATP by Get3, the Get1/Get2 complex facilitates the insertion of the tail into the ER membrane. Finally, ATP displaces ADP and Get3 is released back to the cytosol.</p>
</div>
<div id="the-roles-of-yidc" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> The Roles of YidC</h2>
<p>YidC contains 5 TM helices arranged to form a partially hydrophilic groove that is open towards both the lipid bilayer and the cytosol (Figure <a href="#fig:yidc">4.1</a>). Cross-linking experiments suggest that this groove operates as a binding site for TM helices (of the protein to be inserted). YidC alone can mediate insertion of a small subset of small (single- or double-pass) proteins in either co-translationally or post-translationally, and it is also suggested that YidC may act as a chaperone in conjunction with the Sec translocon. Specifically, it may shield the hydrophilic surface of TM helice that line polar cavities/channels in the final structure.</p>

<div class="figure"><span id="fig:yidc"></span>
<img src="../img/yidc.png" alt="The structure of Bacillus halodurans Yidc." width="100%" />
<p class="caption">
Figure 4.1: The structure of <em>Bacillus halodurans</em> Yidc.
</p>
</div>
</div>
</div>
<div id="beta-barrel-proteins" class="section level1" number="5">
<h1><span class="header-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</h1>
<p>In Gram-negative bacteria, intergral OM (outer membrane) <span class="math inline">\(\beta\)</span>-barrel proteins (OMPs) are translocated in an unfolded form across the IM (inner membrane), ferried to the OM via periplasmic chaperones, and integrated into the OM by the <span class="math inline">\(\beta\)</span>-barrel assembly machine (Bam) multiprotein complex.</p>
<p>In mitochondria, the precursors of nucleus-encoded <span class="math inline">\(\beta\)</span>-barrel proteins are transferred from the TOM complex, with the help of chaperones, to the SAM (sorting and assembly machinery) complex. The SAM complex, being homologous to the Bam complex, mediates the assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins in a similar manner (Figure <a href="#fig:insert-beta-barrel">5.1</a>).</p>

<div class="figure"><span id="fig:insert-beta-barrel"></span>
<img src="../img/insert-beta-barrel.png" alt="Insertion of \(\beta\)-barrel proteins into bacterial or mitochondrial outer membrane." width="100%" />
<p class="caption">
Figure 5.1: Insertion of <span class="math inline">\(\beta\)</span>-barrel proteins into bacterial or mitochondrial outer membrane.
</p>
</div>
<p>The exact mechanism of assembly and insertion is not fully understood, but a recent proposed mechanism is shown in Figure <a href="#fig:insert-beta-barrel-mech">5.2</a> <span class="citation">(Ricci and Silhavy 2019)</span>.</p>

<div class="figure"><span id="fig:insert-beta-barrel-mech"></span>
<img src="../img/insert-beta-barrel-mech.png" alt="Proposed Bam-assisted assembly and insertion of \(\beta\)-barrel proteins. (a) Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). (b) The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the \(\beta\)-1 and \(\beta\)-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. (c) The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, (d) Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. (e) The folded client protein is released into the membrane, and the Bam complex restores to its idle state." width="100%" />
<p class="caption">
Figure 5.2: <strong>Proposed Bam-assisted assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins.</strong> <strong>(a)</strong> Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). <strong>(b)</strong> The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the <span class="math inline">\(\beta\)</span>-1 and <span class="math inline">\(\beta\)</span>-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. <strong>(c)</strong> The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, <strong>(d)</strong> Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. <strong>(e)</strong> The folded client protein is released into the membrane, and the Bam complex restores to its idle state.
</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Cymer-2015">
<p>Cymer, Florian, Gunnar von Heijne, and Stephen H. White. 2015. “Mechanisms of Integral Membrane Protein Insertion and Folding.” <em>Journal of Molecular Biology</em> 427 (5): 999–1022. <a href="https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014">https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014</a>.</p>
</div>
<div id="ref-Guna-2018">
<p>Guna, Alina, and Ramanujan S. Hegde. n.d. “Transmembrane Domain Recognition During Membrane Protein Biogenesis and Quality Control.” <em>Current Biology</em> 28 (8): R498–R511. <a href="https://doi.org/10.1016/j.cub.2018.02.004">https://doi.org/10.1016/j.cub.2018.02.004</a>.</p>
</div>
<div id="ref-Lodish-2016">
<p>Lodish, Harvey F., Arnold Berk, Chris Kaiser, Monty Krieger, Anthony Bretscher, Hidde L. Ploegh, Angelika Amon, and Kelsey C Martin. 2016. <em>Molecular Cell Biology</em>. Book. 8th ed. New York: W.H. Freeman.</p>
</div>
<div id="ref-Ricci-2019">
<p>Ricci, Dante P., and Thomas J. Silhavy. 2019. “Outer Membrane Protein Insertion by the β-Barrel Assembly Machine.” In <em>Protein Secretion in Bacteria</em>, 91–101. American Society of Microbiology. <a href="https://www.asmscience.org/content/book/10.1128/9781683670285.chap8">https://www.asmscience.org/content/book/10.1128/9781683670285.chap8</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/membrane-protein-structure-and-stability.html" class="archive-item-link">
                Membrane Protein: Structure and Stability 
              </a>
              <span class="archive-item-date">
                2020-11-12
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#the-biomembrane-environment"><span class="toc-section-number">2</span> The Biomembrane Environment</a></li>
<li><a href="#the-thermodynamic-basis-of-secondary-structure-formation"><span class="toc-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</a></li>
<li><a href="#glycine-and-proline-in-tm-helices"><span class="toc-section-number">4</span> Glycine and Proline in TM Helices</a></li>
<li><a href="#snorkel"><span class="toc-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</a></li>
<li><a href="#interactions-with-lipids"><span class="toc-section-number">6</span> Interactions with Lipids</a></li>
</ul>
</div>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Many proteins that perform vital functions are membrane proteins. These include transport proteins (channels, transporters and pumps), cell adhesion molecules, and proteins that transduces energy in the electron transport chain. Despite their diverse roles, the same set of biophysical and biochemical rules govern their structural stability. Membrane proteins fall into two broad categories: 1) integeral proteins, which are embedded within a membrane and can only be isolated with detergents, and 2) peripheral proteins, which are associated with the surface of a membrane and can be removed without detergents. The focus of this essay is on how integral proteins interact with the membrane to maintain their structural stability.</p>
</div>
<div id="the-biomembrane-environment" class="section level1" number="2">
<h1><span class="header-section-number">2</span> The Biomembrane Environment</h1>
<p>The basic structure of a biomembrane is a bilayer of amphipathic phospholipids and sphingolipids, where the polar head groups of lipids align the surfaces (each ~10 angstroms thick) and the acyl chains of lipid tails occupy the hydrophobic core (~ 30 angstroms thick). A bitopic or polytopic integral membrane protein spans the membrane entirely one or more times (Figure <a href="#fig:lipid-bilayer-diagram">2.1</a>).</p>
<div class="figure"><span id="fig:lipid-bilayer-diagram"></span>
<img src="../img/lipid-bilayer-diagram.png" alt="(a) a schematic showing a integral protein embedded within a biomembrane. (b)" width="100%" />
<p class="caption">
Figure 2.1: (a) a schematic showing a integral protein embedded within a biomembrane. (b)
</p>
</div>
<p>The hydrophobic core region has a dielectric constant (<span class="math inline">\(\epsilon_{r}\)</span>) of about 2, which is much lower than <span class="math inline">\(\epsilon_{r}\)</span> of water (about 80), which means the same pair of charges separated by the same distance experience more electrostatic force in the membrane core than in water, according to the Coulomb’s law:</p>
<p><span class="math display">\[F = \dfrac{q_1 q_2}{4\pi \epsilon_0 \epsilon_r r^2}\]</span></p>
<p>This has several implications on acidic and basic amino acid residues. First, the carboxyl group of acidic side chains are more difficult to dissociate (i.e. p<span class="math inline">\(K_a\)</span>s are shifted up), meaning they tend to remain in the uncharged (–COOH) form. Second, in <span class="math inline">\(\alpha\)</span>-helical bundles, oppositely charged residues in adjacent helices associated more strongly than they do in aqueous environment. Third, this facilitates the snorkelling of lysine and arginine residues near the interface (Section <a href="#snorkel">5</a>).</p>
</div>
<div id="the-thermodynamic-basis-of-secondary-structure-formation" class="section level1" number="3">
<h1><span class="header-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</h1>
<p>Transmembrane proteins tend to adopt a conformation that minimises the contact between their polar groups (main chain -NH, C=O, and side chains of polar residues, if any) and the hydrophobic core. This can be explained by simple thermodynamic reasoning.
The water molecules in the aqueous environment outside the membrane are capable of forming relatively strong dipole-dipole interactions and hydrogen bonds (H-bonds) with polar (including charged) amino acids, while the non-polar alkyl groups that occupy the hydrophobic core of the membrane can only provide weak van der Waals interactions.
Thus, if a protein adopts a conformation that exposes many polar groups, the loss of strong interactions with water will make this process very energetically unfavourable (very large <span class="math inline">\(\Delta G\)</span>).
On the contrary, if a protein with a hydrophobic surface were present in an aqueous environment, it would disrupt the dipole-dipole interactions and hydrogen bonding among water and other polar molecules. Thus, its insertion into the membrane is energetically favourable.</p>
<p>By forming either an <span class="math inline">\(\alpha\)</span>-helix or a <span class="math inline">\(\beta\)</span>-sheet (Figure <a href="#fig:bundle-helix">3.1</a>), the hydrogen bonds between main chain -NH and -C=O are maximised, and this is an efficient way to shield the polar groups on the main chain from the hydrophobic core.
Many transmembrane proteins, such as bacteriorhodopsin, contain multiple TM helices, which aggregates to form a <span class="math inline">\(\alpha\)</span>-helix bundle.
A <span class="math inline">\(\beta\)</span>-sheet has to twist and coil to form a ring, called a <span class="math inline">\(\beta\)</span>-barrel, in order to shield the main chain -NH and C=O groups on the first and last strands.
<span class="math inline">\(\beta\)</span>-barrels usually have a even number of strands so that every strand is anti-parallel to adjacent strands, there are exceptions: the voltage-dependent anion channels (VDACs) located on mitochondrial outer membrane has 19 strands, where the first strand is parallel with tha last strand, with weaker hydrogen bonding.
<span class="math inline">\(\beta\)</span>-barrels are less common than <span class="math inline">\(\alpha\)</span>-helix bundles, and they are predominantly present on the outer membrane of Gram-negative bacteria, mitochondria, and chloroplasts.</p>
<div class="figure"><span id="fig:bundle-helix"></span>
<img src="../img/bundle-helix.png" alt="Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively." width="100%" />
<p class="caption">
Figure 3.1: Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively.
</p>
</div>
<p>The transmembrane region of each helix in a <span class="math inline">\(\alpha\)</span>-helical bundle is composed predominantly of hydrophobic amino acid residues (Ala, Leu, Ile, Val, Phe), which are exposed on the surface and allows favourable interaction with the hydrophobic core.
By contrast, each <span class="math inline">\(\beta\)</span>-strand of a <span class="math inline">\(\beta\)</span>-barrel has an alternating pattern of hydrophobic and polar residues.
This is because two adjacent residues in a beta strand have their side chains pointing in opposite directions, and a <span class="math inline">\(\beta\)</span>-barrel formed in this way has an hydrophobic exterior that interacts with the hydrophobic core and an hydrophilic inner surface that is in contact with the aqueous environment.</p>
<p>Thus, TM helices, but not <span class="math inline">\(\beta\)</span>-barrels, can be predicted using a hydropathy plot (Figure <a href="#fig:hydropathy-plot-1">3.2</a>), in which the average hydrophobicity index of a fixed number of consecutive residues (a “window”), <span class="math inline">\(H(i)\)</span>, is plotted against the index (<span class="math inline">\(i\)</span>) of the window, i.e.</p>
<p><span class="math display">\[H(i) = \sum_{i&lt;j&lt;i+k}h(a_j)/k\]</span></p>
<p>for <span class="math inline">\(1 &lt; i &lt; n - k\)</span>, where n is the length (number of residues) of the peptide, <span class="math inline">\(h(a_j)\)</span> is the hydrophobicity index of the <span class="math inline">\(j\)</span>-th amino acid residue and <span class="math inline">\(k\)</span> is the window size.</p>
<div class="figure"><span id="fig:hydropathy-plot-1"></span>
<img src="../img/hydropathy-plot-1.png" alt="A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix." width="100%" />
<p class="caption">
Figure 3.2: A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix.
</p>
</div>
<p>A small number of polar (even charged) residues within a TM helix can be tolerated as long as the overall transmembrane segment is hydrophobic enough.
In addition, in polytopic proteins, polar residues in adjacent helices may help to stabilise each other.
These non-hydrophobic residues often have functional roles, as exemplified by voltage-gated K<sup>+</sup>, Na<sup>+</sup> and Ca<sup>2+</sup> channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
<div class="figure"><span id="fig:kv-mech"></span>
<img src="../img/kv-mech.png" alt="The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker." width="100%" />
<p class="caption">
Figure 3.3: The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker.
</p>
</div>
</div>
<div id="glycine-and-proline-in-tm-helices" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Glycine and Proline in TM Helices</h1>
<p>The side chain of proline forms a pentameric ring with the amine group on the main chain. Thus, in a helix, the amine group of a proline does not have a hydrogen that can be H-bonded to the main chain C=O of the residue above it. This introduces local flexibility within a relatively regid helix and often forms a hinge. Similarly, due to glycine’s small size, it can tolerate a much wider range of dihedral angles than other amino acids, which also makes it able to introduce flexibility. This flexibility is crucial for the function of some proteins, such as voltage-gated ion channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
</div>
<div id="snorkel" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</h1>
<p>Tryptophan and tyrosine are often found in the interfacial region in both <span class="math inline">\(\alpha\)</span>-helical bundle and <span class="math inline">\(\beta\)</span>-barrel proteins. Their hydrophobic aromatic ring interacts with the hydrophobic core, and their polar -NH (of Trp) or -OH (of Tyr) groups interacts with the lipid head groups. Similarly, lysine and arginine extend their side chain towards the surface, so that their positively charged guanidium (of Arg) or amide (of Lys) groups form strong ionic interactions with the negatively charged lipid head groups. This phenomenon is known as “snorkelling.” These interactions helps to lock the membrane in place within the membrane, preventing vertical motions.</p>
</div>
<div id="interactions-with-lipids" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Interactions with Lipids</h1>
<p>Membrane proteins interacts with lipid in various ways, and these interactions contribute to the structural stability and function of the protein to different extents.</p>
<p>A transmembrane protein interacts strongly with the shell (annulus) of lipid that surrounds it. These lipids are called annular lipids and can be distinguished experimentally from the bulk lipids of the bilayer. In addition, there is a third class of lipids, called nonannular lipids (or lipid cofactors), which are tightly bound in crevices or between subunits of the protein and are often crutial for the activity of the protein. Electron paramagnetic resonance (EPR) can be used to measure the mobility of annular lipids, and thus gives information on the selectivity for annular lipids of a protein. Most proteins are found to prefer negatively charged lipids such as phosphatidylserine and phosphatidylinositol.</p>
<p>Strong lipid-protein interactions are also evident form crystal structures, and this interaction can be specific. For example, three cardiolipin (CDL) molecules were found to bind to three sites on the ANT1 transporter. The tendency of CDL to bind to these sites can also be shown in molecular dynamics simulations. The specificity of this binding can also be shown computationally, by calculating the free energy at different protein-lipid separations for CDL and other lipids (Figure <a href="#fig:simu-lipid-protein-binding">6.1</a>).</p>
<div class="figure"><span id="fig:simu-lipid-protein-binding"></span>
<img src="../img/simu-lipid-protein-binding.png" alt="Computational analysis of a protein's preference to bound lipids. CDL binding results in a significantly lower energy state." width="100%" />
<p class="caption">
Figure 6.1: Computational analysis of a protein’s preference to bound lipids. CDL binding results in a significantly lower energy state.
</p>
</div>
<p>Inward-rectifier K<sup>+</sup> channels (K<sub>ir</sub>) are an example where the lipid-protein interaction is directly involved in the protein’s function. K<sub>ir</sub> have a Transmembrane domain (TMD) and an intracellular C-terminal domain (CTD). PIP<sub>2</sub> binding cuases the CTD to move towards the TMD, which in turn causes the helices in the TMD to bend and thus opens the gate (Figure <a href="#fig:kir">6.2</a>).</p>
<div class="figure"><span id="fig:kir"></span>
<img src="../img/kir.png" alt="Lipid-gated Kir channel" width="100%" />
<p class="caption">
Figure 6.2: Lipid-gated Kir channel
</p>
</div>
<p>Many crystal structures of GPCRs reveal cholesterol bound to the protein. Molecular dynamics simulations at different cholesterol concentrations suggests the role of cholesterol in reducing the dynamic flexibility the protein and stabilising a certain conformational state in an allosteric fashion.</p>
<!-- Lipid rafts are stable but transient domains that are enriched in cholesterol and sphingolipids, which makes them thinker than the average membrane thinkness (directly detectable using atomic force microscopy). They are specifically associated with lipid anchored proteins (e.g. GPI-, palmitoyl-, myristoyl-anchored protein) -->
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img
              src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>