<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>metabolism - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="metabolism - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/metabolism/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">metabolism</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../lab/metabolic-modelling.html" class="archive-item-link">
                Metabolic Modelling 
              </a>
              <span class="archive-item-date">
                2020-11-26
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#the-haldane-relationship"><span class="toc-section-number">1</span> The Haldane Relationship</a></li>
<li><a href="#control-points-in-a-simple-linear-pathway"><span class="toc-section-number">2</span> Control Points in A Simple Linear Pathway</a></li>
<li><a href="#linear-pathway-with-negative-feedback"><span class="toc-section-number">3</span> Linear Pathway with Negative Feedback</a>
<ul>
<li><a href="#re-analyse-the-effect-of-k_i-with-constant-flux"><span class="toc-section-number">3.1</span> Re-analyse the effect of <span class="math inline">\(K_i\)</span> with constant flux</a>
<ul>
<li><a href="#repeat-with-model-212"><span class="toc-section-number">3.1.1</span> Repeat with Model 212</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<ul>
<li><strong>Model 112 was used.</strong></li>
</ul>
<div id="the-haldane-relationship" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Haldane Relationship</h1>
<p>Using the “Parameter Scan” utility in Copasi (Figure <a href="#fig:copasi-q1-scan">1.1</a>), I collected steady state concentrations of ‘A’ and ‘B’ in <span class="math inline">\(5^4 = 625\)</span> combinations of <span class="math inline">\(V_{max(f)}\)</span>, <span class="math inline">\(K_{m(p)}\)</span>, <span class="math inline">\(V_{max(r)}\)</span>, and <span class="math inline">\(K_{m(s)}\)</span>, where each parameter vary from <span class="math inline">\(1\times10^{-2}\)</span> to <span class="math inline">\(1\times10^{2}\)</span> in a logarithmic scale.</p>
<div class="figure"><span id="fig:copasi-q1-scan"></span>
<img src="../img/copasi-q1-scan.png" alt="&quot;Parameter Scan&quot; setup" width="100%" />
<p class="caption">
Figure 1.1: “Parameter Scan” setup
</p>
</div>
<p>Here I read the data, and add two columns, where <code>keq</code> represents the experimental <span class="math inline">\(K_\text{eq}\)</span>:</p>
<p><span class="math inline">\(K_\text{eq} = \dfrac{\text{[B]}}{\text{[A]}}\)</span></p>
<p>and <code>keq_calc</code> represents the <span class="math inline">\(K_\text{eq}\)</span> calculated from the Haldane relationship:</p>
<p><span class="math inline">\(K_\text{eq} = \dfrac{V_{max(f)}\cdot K_{m(p)}}{V_{max(r)}\cdot K_{m(s)}}\)</span></p>
<pre class="r"><code>q1 &lt;- read_tsv(&#39;metabolic modelling/q1.txt&#39;, col_names = c(&#39;vf&#39;, &#39;kmp&#39;, &#39;vr&#39;, &#39;kms&#39;, &#39;A&#39;, &#39;B&#39;))
q1 &lt;- q1 %&gt;% mutate(
  keq = B/A,
  keq_calc = (vf * kmp) / (vr * kms),
  keq_diff = keq - keq_calc
)
q1</code></pre>
<pre><code>## # A tibble: 625 x 9
##       vf   kmp     vr   kms     A        B      keq keq_calc  keq_diff
##    &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt;    &lt;dbl&gt;    &lt;dbl&gt;    &lt;dbl&gt;     &lt;dbl&gt;
##  1  0.01  0.01   0.01  0.01 1     1         1         1       0.      
##  2  0.01  0.01   0.1   0.01 1.82  0.182     0.1       0.1     0.      
##  3  0.01  0.01   1     0.01 1.98  0.0198    0.01      0.01    0.      
##  4  0.01  0.01  10     0.01 2.00  0.00200   0.001     0.001   0.      
##  5  0.01  0.01 100     0.01 2.00  0.000200  0.00010   0.0001 -1.36e-20
##  6  0.1   0.01   0.01  0.01 0.182 1.82     10.0      10      -1.78e-15
##  7  0.1   0.01   0.1   0.01 1     1         1         1       0.      
##  8  0.1   0.01   1     0.01 1.82  0.182     0.1       0.1     0.      
##  9  0.1   0.01  10     0.01 1.98  0.0198    0.01      0.01    0.      
## 10  0.1   0.01 100     0.01 2.00  0.00200   0.001     0.001   0.      
## # … with 615 more rows</code></pre>
<p>Plotting <code>keq_calc</code> against <code>keq</code> shows that <span class="math inline">\(K_\text{eq}\)</span> calculated in these two ways are equal, as the linear regression line has a gradient of 1 and passes through the origin.</p>
<pre class="r"><code>q1 %&gt;% ggplot(aes(keq, keq_calc)) +
  geom_point() +
  geom_smooth()+
  scale_x_log10()+
  scale_y_log10()</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<pre class="r"><code>mod &lt;- lm(q1$keq ~ q1$keq_calc)
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = q1$keq ~ q1$keq_calc)
## 
## Residuals:
##        Min         1Q     Median         3Q        Max 
## -1.148e-08 -4.480e-10 -4.480e-10 -4.190e-10  2.880e-07 
## 
## Coefficients:
##              Estimate Std. Error   t value Pr(&gt;|t|)    
## (Intercept) 7.078e-10  4.630e-10 1.529e+00    0.127    
## q1$keq_calc 1.000e+00  1.135e-16 8.814e+15   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 1.156e-08 on 623 degrees of freedom
## Multiple R-squared:      1,  Adjusted R-squared:      1 
## F-statistic: 7.769e+31 on 1 and 623 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>The Haldane relationship shows that <span class="math inline">\(K_\text{eq}\)</span> is proportional to <span class="math inline">\(V_{max(f)}\)</span> and <span class="math inline">\(K_{m(p)}\)</span>, and inversely proportional to <span class="math inline">\(V_{max(r)}\)</span> and <span class="math inline">\(K_{m(s)}\)</span>, and this can be illustrated using a facetted heatmap:</p>

<pre class="r"><code>q1 %&gt;% ggplot(aes(x = vf, y = kmp, fill = log10(keq))) +
  geom_tile()+
  facet_grid((1/vr)~(1/kms), as.table = FALSE, labeller = label_both)+
  scale_x_log10(labels = plain, expand = c(0, 0))+
  scale_y_log10(labels = plain, expand = c(0, 0))+
  scale_fill_gradient2(low = &#39;blue&#39;, mid = &#39;yellow&#39;, high = &#39;red&#39;)+
  labs(title = &quot;Variation of Keq with vf, kmp, vr, and kms&quot;)+
  theme(axis.text.x = element_text(angle = 90))</code></pre>
<div class="figure"><span id="fig:q1"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q1-1.png" alt="Variation of \(K_\text{eq}\) with \(V_{max(f)}\), \(K_{m(p)}\), \(V_{max(r)}\), and \(K_{m(s)}\)" width="672" />
<p class="caption">
Figure 1.2: Variation of <span class="math inline">\(K_\text{eq}\)</span> with <span class="math inline">\(V_{max(f)}\)</span>, <span class="math inline">\(K_{m(p)}\)</span>, <span class="math inline">\(V_{max(r)}\)</span>, and <span class="math inline">\(K_{m(s)}\)</span>
</p>
</div>
<p>The pattern in each grid shows that <span class="math inline">\(K_\text{eq}\)</span> is proportional to <span class="math inline">\(V_{max(f)}\)</span> and <span class="math inline">\(K_{m(p)}\)</span>, and the pattern across the grids shows that it is inversely proportional to <span class="math inline">\(V_{max(r)}\)</span> and <span class="math inline">\(K_{m(s)}\)</span>.</p>
</div>
<div id="control-points-in-a-simple-linear-pathway" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Control Points in A Simple Linear Pathway</h1>
<p>I use the “Parameter Scan” function to vary the <span class="math inline">\(K_\text{eq}\)</span> of reaction 3 (that catalyses the reversible conversion between C and D) in the range <span class="math inline">\(10^{-6}\)</span> to <span class="math inline">\(10^6\)</span> in a logarithmic scale (13 samples). The variables being recorded are <span class="math inline">\(K_\text{eq}\)</span> and <span class="math inline">\(J\)</span> (flux control coefficient) of reaction 3, and the concentrations of C and D. Then, the reaction quotient, <span class="math inline">\(Q\)</span> (a.k.a. mass action ratio), of each row is calculated as:</p>
<p><span class="math display">\[Q = \dfrac{\text{[D]}}{\text{[C]}}\]</span></p>
<pre class="r"><code>q2_scan_r3 &lt;- read_tsv(&#39;metabolic modelling/Q2.txt&#39;)
q2_scan_r3 &lt;- q2_scan_r3 %&gt;% mutate(q = d/c)
q2_scan_r3</code></pre>
<pre><code>## # A tibble: 13 x 5
##            j     keq     c         d          q
##        &lt;dbl&gt;   &lt;dbl&gt; &lt;dbl&gt;     &lt;dbl&gt;      &lt;dbl&gt;
##  1 0.0000489 1.00e-6 100.   0.000100 0.00000100
##  2 0.0000489 1.00e-5 100    0.001    0.00001   
##  3 0.00470   1.00e-4  99.1  0.00987  0.0000996 
##  4 0.0353    1.00e-3  92.2  0.0885   0.000960  
##  5 0.119     1.00e-2  69.0  0.556    0.00806   
##  6 0.210     1.00e-1  44.5  2.35     0.0528    
##  7 0.309     1.00e+0  32.0  6.77     0.212     
##  8 0.369     1.00e+1  28.6 10.0      0.350     
##  9 0.380     1.00e+2  28.2 10.6      0.378     
## 10 0.381     1.00e+3  28.1 10.7      0.381     
## 11 0.381     1.00e+4  28.1 10.7      0.381     
## 12 0.381     1.00e+5  28.1 10.7      0.381     
## 13 0.381     1.00e+6  28.1 10.7      0.381</code></pre>
<p>Figure <a href="#fig:q2-keq">2.1</a> shows the variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with its <span class="math inline">\(K_\text{eq}\)</span>. The plot shows that a high <span class="math inline">\(K_\text{eq}\)</span>, i.e. high irreversibility, is correlated with a high <span class="math inline">\(J\)</span>, and in the intermediate range <span class="math inline">\(J\)</span> varies linearly with <span class="math inline">\(\ln(K_\text{eq})\)</span>, i.e. varies linearly with <span class="math inline">\(\Delta G = -RT\ln(K_\text{eq})\)</span></p>

<pre class="r"><code>q2_scan_r3 %&gt;% ggplot(aes(log(keq), j)) +
  geom_point()</code></pre>
<div class="figure"><span id="fig:q2-keq"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2-keq-1.png" alt="Variation of the flux control coefficient, \(J\), of reaction 3, with its \(K_\text{eq}\)" width="672" />
<p class="caption">
Figure 2.1: Variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with its <span class="math inline">\(K_\text{eq}\)</span>
</p>
</div>
<p>Figure <a href="#fig:q2-displacement">2.2</a> shows the variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3 with <span class="math inline">\(Q/K_\text{eq}\)</span>, which is a measure of displacement of the reaction from the equilibrium. A <span class="math inline">\(Q/K_\text{eq}\)</span> close to 1 indicates the reaction is close to equilibrium.
The plot shows that when reaction is further displaced from the equilibrium, the its flux control coefficient is higher.</p>

<pre class="r"><code>q2_scan_r3 %&gt;% ggplot(aes(q/keq, j)) +
  geom_point()+
  geom_smooth(method = &#39;lm&#39;, size = 0.5)</code></pre>
<div class="figure"><span id="fig:q2-displacement"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2-displacement-1.png" alt="Variation of the flux control coefficient, \(J\), of reaction 3, with \(Q/K_\text{eq}\)" width="672" />
<p class="caption">
Figure 2.2: Variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with <span class="math inline">\(Q/K_\text{eq}\)</span>
</p>
</div>
<p>A linear regression analysis shows that there is a strong linear correlation between <span class="math inline">\(J\)</span> and <span class="math inline">\(Q/K_\text{eq}\)</span>, with <span class="math inline">\(p = 4.02\times10^{-6} &lt; 10^{-5}\)</span></p>
<pre class="r"><code>mod &lt;- with(q2_scan_r3, lm(j ~ q/keq))
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = j ~ q/keq)
## 
## Residuals:
##       Min        1Q    Median        3Q       Max 
## -0.054438 -0.019991 -0.012931  0.001452  0.108459 
## 
## Coefficients:
##               Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)  5.448e-02  2.221e-02   2.453   0.0341 *  
## q            8.960e-01  9.010e-02   9.945 1.67e-06 ***
## q:keq       -4.196e-08  1.569e-07  -0.267   0.7946    
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.05374 on 10 degrees of freedom
## Multiple R-squared:  0.9167, Adjusted R-squared:    0.9 
## F-statistic:    55 on 2 and 10 DF,  p-value: 4.02e-06</code></pre>
<p>In order to vary <span class="math inline">\(Q/K_\text{eq}\)</span> without directly varying <span class="math inline">\(K_\text{eq}\)</span>, and to see the effect not only on reaction 3 but also on all other reactions, I vary <span class="math inline">\(V_f\)</span>, not <span class="math inline">\(K_\text{eq}\)</span>, of reaction 3 from <span class="math inline">\(10^{-6}\)</span> to <span class="math inline">\(10^6\)</span> with 1000 intervals, and collected the flux control coefficients of all 6 reactions, the <span class="math inline">\(K_\text{eq}\)</span> of reaction 3, as well as concentrations of all species, which are then used to calculate the mass action ratio of each reaction. Then, the displacement of each reaction is calculated. The resulting dataframe contains three columns: 1) the reaction number; 2) the flux control coefficient and 3) displacement from equilibrium of this reaction</p>
<pre class="r"><code>q2_scan_r3_vf &lt;- read_tsv(&#39;metabolic modelling/Q2-1.txt&#39;)
q2_scan_r3_vf &lt;- q2_scan_r3_vf %&gt;% 
  mutate(
    d1 = (B/A) / 10,
    d2 = (C/B) / 1,
    d3 = (D/C) / 0.5,
    d4 = (E/D) / 5,
    d5 = (F/E) / 2,
    d6 = (G/F) / 10,
    c1 = A + B,
    c2 = B + C,
    c3 = C + D,
    c4 = D + E,
    c5 = E + F,
    c6 = F + G,
  ) %&gt;% select(!(1:7))
q2_scan_r3_vf_cleaned &lt;- tibble(
  reaction = integer(),
  displacement = double(),
  j = double()
  )
for (i in 1:6) {
  q2_scan_r3_vf_cleaned &lt;- add_row(
    q2_scan_r3_vf_cleaned, 
    reaction = i, j = q2_scan_r3_vf[[paste0(&#39;j&#39;, i)]], displacement = q2_scan_r3_vf[[paste0(&#39;d&#39;, i)]])
}
q2_scan_r3_vf_cleaned </code></pre>
<pre><code>## # A tibble: 6,006 x 3
##    reaction displacement            j
##       &lt;int&gt;        &lt;dbl&gt;        &lt;dbl&gt;
##  1        1         1.00 0.0000000343
##  2        1         1.00 0.0000000353
##  3        1         1.00 0.0000000363
##  4        1         1.00 0.0000000373
##  5        1         1.00 0.0000000383
##  6        1         1.00 0.0000000394
##  7        1         1.00 0.0000000405
##  8        1         1.00 0.0000000416
##  9        1         1.00 0.0000000428
## 10        1         1.00 0.0000000440
## # … with 5,996 more rows</code></pre>
<p>Then, for all reactions, <span class="math inline">\(J\)</span> versus displacement from equilibrium is plotted:</p>
<pre class="r"><code>q2_scan_r3_vf_cleaned %&gt;% ggplot(aes(displacement, j)) + 
  geom_point()+
  facet_wrap(~reaction, scales = &#39;free&#39;, labeller = label_both)+
  xlab(&quot;MAR/Keq&quot;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2_2-1.png" width="672" /></p>
<p>The plots shows that, when disturbing <span class="math inline">\(V_{max(f)}\)</span> of reaction 3, the disequilibrium ratio of all reactions are also altered. Not only reaction 3 but also all other reactions follow the rule that, the flux control coefficent of a reaction increases with the extent of displacement from the equilibrium. However, it is hard to predict <span class="math inline">\(J\)</span> given only the value of <span class="math inline">\(Q/K_\text{eq}\)</span>, as different reactions show different patterns of variations, so <span class="math inline">\(Q/K_\text{eq}\)</span> (displacement from equilibrium) is not a robust indicator of <span class="math inline">\(J\)</span>.</p>
</div>
<div id="linear-pathway-with-negative-feedback" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Linear Pathway with Negative Feedback</h1>
<p>I chose reaction 2 to be the one sensitive to the inhibitor, and varied <span class="math inline">\(K_i\)</span> from <span class="math inline">\(10^{-18}\)</span> to <span class="math inline">\(10^{18}\)</span> with 1000 intervals, and recorded it along with flux control coefficients of all reactions. The results are shown in Figure <a href="#fig:q3">3.1</a>.</p>

<pre class="r"><code>q3 &lt;- read_tsv(&#39;metabolic modelling/Q3.txt&#39;)
q3 &lt;- q3 %&gt;% gather(reaction, &quot;j&quot;, -ki)
q3 %&gt;% ggplot(aes(ki, j)) +
  geom_point()+
  facet_wrap(~reaction, labeller = label_both)+
  scale_x_log10()</code></pre>
<div class="figure"><span id="fig:q3"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q3-1.png" alt="Effect of changing \(K_i\) of reaction 2 on the flux control coefficient of all reactions. Lower \(K_i\) means higher binding affinity to the inhibitor." width="672" />
<p class="caption">
Figure 3.1: Effect of changing <span class="math inline">\(K_i\)</span> of reaction 2 on the flux control coefficient of all reactions. Lower <span class="math inline">\(K_i\)</span> means higher binding affinity to the inhibitor.
</p>
</div>
<p>The plots show that, the flux control coefficient (<span class="math inline">\(J\)</span>) of reaction 2 increases as the binding affinity (i.e. sensitivity) to the inhibitor of the enzyme involved in this step increases (i.e. as <span class="math inline">\(K_i\)</span> decreases). <span class="math inline">\(J\)</span> of the upsteam reaction 1 also inceases slightly. For all downstream reactions, <span class="math inline">\(J\)</span> decreases.</p>
<div id="re-analyse-the-effect-of-k_i-with-constant-flux" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Re-analyse the effect of <span class="math inline">\(K_i\)</span> with constant flux</h2>
<p>The flux and values of <span class="math inline">\(J\)</span> when <span class="math inline">\(K_\text{i} = 10^{-18}\)</span> and when <span class="math inline">\(K_\text{i} = 10^18\)</span> are shown below:</p>
<pre class="r"><code>ki_low = 1e-18
flux_low = 0.476572
j_low = c(0.203839, 0.717872,   0.0342001,  0.0340061,  0.00278022, 0.007302)
ki_high = 1e18
flux_high = 1.07393
j_high = c(0.110937,    0.0822319,  0.278976,   0.319753,   0.0478304,  0.160271)</code></pre>
<p><span class="math inline">\(V_f\)</span> of reaction 2 are optimised so that the flux when <span class="math inline">\(K_\text{i} = 10^{-18}\)</span> is 1.07393 (the same as when <span class="math inline">\(K_\text{i} = 10^{18}\)</span>)</p>
<pre><code>Optimization Result:

    Objective Function Value:   1.07393
    Function Evaluations:   248
    CPU Time [s]:   0.061
    Evaluations/Second [1/s]:   4065.57

    (R2).Vf: 45.0229</code></pre>
<div class="figure"><span id="fig:copasi-q32"></span>
<img src="../img/copasi-q32.png" alt="Adjusting Vf" width="100%" />
<p class="caption">
Figure 3.2: Adjusting Vf
</p>
</div>
<p>I adjusted the <span class="math inline">\(V_f\)</span> of reaction 2 from 5 to 45.0229 (<a href="#fig:copasi-q32">3.2</a>), verified that the flux is 1.07393 (the same as in the state with negligible inhibition), and the values of <span class="math inline">\(J\)</span> are:</p>
<pre class="r"><code>j_low &lt;-    c(0.144048, 0.0858949,  0.266261,   0.30518,    0.0456503,  0.152966)</code></pre>
<p>which can be directly compared to the <span class="math inline">\(J\)</span> values in the uninhibited state:</p>
<pre class="r"><code>tibble(
  ki =c(rep(&quot;low (1e-18)&quot;, 6), rep(&quot;high (1e18)&quot;, 6)),
  reaction = rep(as.character(1:6), 2),
  j = c(j_low, j_high)
) %&gt;% ggplot(aes(reaction, j, fill = ki)) +
  geom_col(position = &#39;dodge&#39;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
<p>The plot shows that, when the flux is made constant, a lower <span class="math inline">\(K_i\)</span> (higher affinity of inhibitor binding) increases the flux control coefficent of reaction 2 and the upstream reaction 1, and decreases that of downstream reactions, which is consistent with the previous experiment. However, the amount of change is not as much as previously modelled.</p>
<div id="repeat-with-model-212" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Repeat with Model 212</h3>
<p>I repeat the last analysis with model 212, this time choosing reaction 1 as the one to be affected by the inhibitor.</p>
<pre class="r"><code>ki_low = 1e-16
j_low = c(0.982287, 0.0109131,  0.00433655, 0.00237021, 6.9396e-05, 2.40539e-05)
flux_low = 0.091902
ki_high = 1e16
flux_high = 0.807846
j_high = c(0.437942,    0.31832,    0.104533,   0.12481,    0.0105218,  0.00387277)</code></pre>
<pre><code>Optimization Result:

    Objective Function Value:   0.807846
    Function Evaluations:   82
    CPU Time [s]:   0.018
    Evaluations/Second [1/s]:   4555.56

    (R1).Vf: 25.7611</code></pre>
<p>I adjusted the <span class="math inline">\(V_f\)</span> of reaction 1 from 5 to 25.7611 (<a href="#fig:copasi-q32">3.2</a>), verified that the flux is 0.807846 (the same as in the state with negligible inhibition), and the values of <span class="math inline">\(J\)</span> are:</p>
<pre class="r"><code>j_low &lt;-    c(0.479732, 0.294652,   0.0967611,  0.11553,    0.00973947, 0.00358483)</code></pre>
<p>comparing to the <span class="math inline">\(J\)</span> values in the uninhibited state:</p>
<pre class="r"><code>tibble(
  ki =c(rep(&quot;low (1e-16)&quot;, 6), rep(&quot;high (1e16)&quot;, 6)),
  reaction = rep(as.character(1:6), 2),
  j = c(j_low, j_high)
) %&gt;% ggplot(aes(reaction, j, fill = ki)) +
  geom_col(position = &#39;dodge&#39;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-12-1.png" width="672" /></p>
<p>This time only reaction 1 has an increased <span class="math inline">\(J\)</span>, and all other downstream reactions have an lowered <span class="math inline">\(J\)</span>, which is consistent with previous observations.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/sensing-energy-status-during-exercise.html" class="archive-item-link">
                Fuel Selection During Exercise 
              </a>
              <span class="archive-item-date">
                2020-05-25
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#overview"><span class="toc-section-number">2</span> Overview</a><ul>
<li><a href="#glycogen-and-glucose"><span class="toc-section-number">2.1</span> Glycogen and Glucose</a><ul>
<li><a href="#blood-glucose"><span class="toc-section-number">2.1.1</span> Blood Glucose</a></li>
<li><a href="#regulation-of-glycogenolysis"><span class="toc-section-number">2.1.2</span> Regulation of Glycogenolysis</a></li>
</ul></li>
<li><a href="#fatty-acids"><span class="toc-section-number">2.2</span> Fatty Acids</a><ul>
<li><a href="#adipose-tissue-lipolysis-and-plasma-nefa"><span class="toc-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</a></li>
<li><a href="#intramuscular-triacylglycerol-imtg"><span class="toc-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</a></li>
<li><a href="#plasma-tg-vldl-tg"><span class="toc-section-number">2.2.3</span> Plasma TG (VLDL-TG)</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p><strong>What effects do the intensity and duration of exercise have on fuel selection by mammalian muscle?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>During exercise, glucose and fatty acids are the fuels used by the muscle. Their origin and relative contribution depend on the intensity and duration of exercise, as well as on characteristics of individuals such as the level of physical training.</p>
</div>
<div id="overview" class="section level1">
<h1><span class="header-section-number">2</span> Overview</h1>
<p>As shown in Figure 1, muscle contraction during exercise is powered by oxidation of glucose and fatty acids derived from a number of sources.</p>
<div class="figure"><span id="fig:powering-the-muscle"></span>
<img src="../img/powering-the-muscle.png" alt="Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6)." width="100%" />
<p class="caption">
Figure 2.1: Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6).
</p>
</div>
<p>Immediately after the onset of exercise (0-30 s) energy is derived predominantly from anaerobic respiration. After this period, both excercise intensity and duration determine the choice of fuel used for oxidation. As shown in Figure 2, during the first 30 minutes of a low-intensity exercise bout, the vast majority of fuel used by muscle is plasma NEFA (non-esterified fatty acids) originated from adipose lipolysis with tiny contributions from IMTG (intramuscular triglyceride) and plasma glucose. For moderately intense exercise, the contribution from fat is lower: plasma NEFA and IMTG each represent about 25% of total energy expenditure, and glycogen and plasma glucose account for the remaining 40% and 10%. For high-intensity exercise, the contribution from muscle glycogen is even higher (60%), and fat and glucose represents 30% and 10% of the total, respectively. Overall, as exercise intensity increases, carbohydrates (especially muscle glycogen) progressively becomes the preferred fuel over triglyceride-derived fuel. As for TG-derived fuel, the ratio of IMTG to plasma NEFA is greatest during moderately intense exercise and lowest during low-intensity excercise. These are explained in detail later.</p>
<p>Figure 2 also shows that, during prolonged (low- or moderate-intensity) exercise, the contribution from plasma NEFA increases and the contribution from other fuels decreases, regardless of exercise intensity. This is due to the large amount of TG reserve in adipose tissue and the gradual depletion of other fuels.</p>
<div class="figure"><span id="fig:exercise-choice-of-fuel"></span>
<img src="../img/exercise-choice-of-fuel.png" alt="Effects of exercise intensity and duration on choice of substrates. 'Percentage of VO~2max~', where VO~2max~ is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO~2max~ is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity." width="100%" />
<p class="caption">
Figure 2.2: Effects of exercise intensity and duration on choice of substrates. ‘Percentage of VO<sub>2max</sub>’, where VO<sub>2max</sub> is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO<sub>2max</sub> is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity.
</p>
</div>
<div id="glycogen-and-glucose" class="section level2">
<h2><span class="header-section-number">2.1</span> Glycogen and Glucose</h2>
<div id="blood-glucose" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Blood Glucose</h3>
<p>During exercise, catecholamines stimulates pancreatic <span class="math inline">\(\alpha\)</span> cells to secrete glucagon, which in turn promtotes hepatic glycogenolysis and gluconeogenesis. Both processes release glucose into the bloodstream.</p>
<p>Blood glucose is the predominant source of carbohydrate for oxidation in muscle. Its absolute amount of usage increases as excercise intensity increases (though proportionally remains largely unchanged), which is mediated by an increaed sensitivity to insulin (in part due to utilisation of IMTG) and thus increased translocation of GLUT4 to the cell surface membrane.</p>
</div>
<div id="regulation-of-glycogenolysis" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Regulation of Glycogenolysis</h3>
<p>Glycogen phosphorylase (GP) and phosphorylase kinase (PhK) are central to the regulation of glycogenolysis. Activated PhK phosphorylates GP, thus converting it from the inactive <em>b</em> form to the active <em>a</em> form. GP in its <em>a</em> form then phosphorolyses glycogen, releasing glucose 1-phosphase available for oxidation.</p>
<p>PhK is activated (1) by phosphorylation on <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> subunits by PKA and (2) by binding of Ca<sup>2+</sup> to <span class="math inline">\(\delta\)</span> subunit. PKA is activated upon catecholamine (mostly adrenaline) binding to cell surface receptors (via G<sub>s</sub>, adenylyl cyclase, and cAMP), and Ca<sup>2+</sup> is released upon muscle contraction (stimulated by acetylcholine).</p>
<p>Ca<sup>2+</sup> released upon muscle contraction results in a burst of glycogenolysis. The large amount of glycogen breakdown exceeds muscle’s energy requirement and capacity of TCA cycle, resulting in accumulation of lactate. This probably explains the short period of anaerobic respiration at the onset of exercise. After a few minutes, the activation of phosphorylase reverses by a mechanism that is not fully understood.</p>
<p>Myocytes also respond, via <span class="math inline">\(\beta\)</span> adrenergic receptors, to catecholamines that are produced proportionally to exercise intensity. This results in activation of PKA (via G<sub>s</sub>, AC and cAMP), which in turn activates glycogenolysis.</p>
<p>The rate of glycogenolysis also depends on availability of P<sub>i</sub>, the substrate of glycogen phosphorylase. At rest and during mild exercise, fatty acid oxidation is capable of generating sufficient ATP, resulting in low P<sub>i</sub> concentration. Availability of P<sub>i</sub> increases as exercise intensity increases because the rate of ATP hydrolysis exceeds the rate of convertion from ADP and P<sub>i</sub> back to ATP by oxidation of fatty acids (and plasma glucose).</p>
</div>
</div>
<div id="fatty-acids" class="section level2">
<h2><span class="header-section-number">2.2</span> Fatty Acids</h2>
<p>During exercise, the usage of fatty acids has an important role of delaying the onset of glycogen depletion and hypoglycemia. Its contribution to total energy expenditure increases as excercise proceeds. However, its usage is limited during strenuous exercise.</p>
<div id="adipose-tissue-lipolysis-and-plasma-nefa" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</h3>
<p>Compared to resting conditions, adipose tissue lipolysis occurs at a higher rate during mild- or moderate-intensity exercise. This is mainly mediated by increased <span class="math inline">\(\beta\)</span>-adrenergic receptor activation by catecholamines released by the adrenal gland upon sympathetic nerve stimulation. Exercise also increases blood flow in adipose tissue (which prevents toxic regional accumulation of fatty acids) and in muscle, thus promoting transport of NEFA to myocytes. Passive uptake of NEFA by myocytes, which is proportional to plasma NEFA concentration, is thus increased. AMPK, which is activated upon transition from rest to exercise and during prolonged exercise, phosphorylates ACC2 and malonyl-CoA decarboxylase, inhibiting the former and activating the latter, thus resulting in a decrease of malonyl-CoA concentration and hence releasing CPT-1 from inhibition. This promotes transport of fatty acids into mitochondria for <span class="math inline">\(\beta\)</span>-oxidation in muscle cells.</p>
<p>As the exercise intensity continue to increase, however, oxidation of NEFA derived from adipose tissue lipolysis in muscle decreases. This is in part caused by decreased rate of lipolysis due to lactate (through activation of an orphan GPCR, GPR81). Lactate can also increase re-esterification of fatty acids to TG in the liver. Raising plasma fatty acid concentrations by intravenously infusing a lipid emulsion and heparin during the exercise bout increases plasma NEFA oxidation by about 30% but does not completely restore it to the rate observed during moderate-intensity exercise. Thus, high-intensity exercise also decreases the capacity of skeletal muscle to oxidise fatty acids, which is in part caused by increased glycogen catabolism. As more acetyl-CoA concentration are produced via glycogen catabolism, they are converted to malonyl-CoA by ACC2 located on the outer mitochondrial membrane, and the malonyl-CoA then inhibits CPT-1 and hence fatty acid oxidation.</p>
</div>
<div id="intramuscular-triacylglycerol-imtg" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</h3>
<p>Within myocytes, there are lipid droplets that stores TG. Their usage during exercise can be calculated by subtracting plasma fatty acid oxidation (determined by isotope tracers) from whole-body fat oxidation (determined by indirect calorimetry).</p>
<p>IMTG use is stimulated by catecholamines via <span class="math inline">\(\beta\)</span> adrenergic receptors. This is supported by the finding that pharmarcological blockage of <span class="math inline">\(\beta_1 + \beta_2\)</span> prevents the usage of IMTG.</p>
</div>
<div id="plasma-tg-vldl-tg" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Plasma TG (VLDL-TG)</h3>
<p>The liver takes up plasma NEFA and glycerol and use some of them to resynthesise TG. These TG are packaged in VLDL and released into the bloodstream. Muscle can break down some VLDL by lipoprotein lipase and oxidise the resulting fatty acids. However, this only contributes to a small fraction of energy usage during exercise and might be more important for replenishing IMTG stores after exercise.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/supplying-fuels-to-the-brain-during-starvation.html" class="archive-item-link">
                Supplying Fuels to the Brain During Starvation 
              </a>
              <span class="archive-item-date">
                2020-05-03
              </span>
              <div class="article-summary" style="display: none;">
                
                


<!--nocite: |
  @Altarejos-2011, Bezaire-2009, Chakravarty-2005, Elia-1991, Frayn-2010, Fukao-2004, Holness-2003, Hue-2009, Hutton-2009, Lass-2011, Nuttall-2008, Okar-2001, Randle-1998, Rider-2004, Rufer-2009, Saggerson-2008, Soeters-2012, Stanley-2006, Sugden-2006, Thomson-2009, Yang-2009-->
<p><strong>Title: As starvation proceeds, how does the supply of fuels to the brain change? How is this change related to a requirement to conserve body protein?</strong></p>
<p>Due to the blood-brain barrier, The human brain can only harness energy from catabolising either glucose or ketone bodies. Under normal circumstances, glucose of dietary origin is the predominant source of fuel for the brain. As starvation begins (during the postabsorptive phase), astrocytes in the brain break down their stored glycogen to supply fuel in the form of lactate to neurons, which is followed by hepatic glycogenolysis that continue to provide glucose for the brain for a short period. As starvation proceeds, upon depletion of glycogen, muscle protein is broken down for liver gluconeogenesis, which continues to provide glucose for the brain. Meanwhile, triacylglycerol-derived ketone bodies synthesised by the liver will represent an increasingly larger proportion of fuel for the brain.</p>
<p>As shown in Figure <a href="#fig:starve-stages">1</a>, starvation can be divided into 4 stages, each having a different strategy for supplying energy to the brain.</p>
<div class="figure"><span id="fig:starve-stages"></span>
<img src="../img/starve-stages.jpg" alt="Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds." width="100%" />
<p class="caption">
Figure 1: Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds.
</p>
</div>
<div id="stage-i-absoption-and-ii-post-absorption-glycogenolysis" class="section level1">
<h1>Stage I (Absoption) and II (Post-absorption): Glycogenolysis</h1>
<p>Soon after a meal, glucose obtained from food is absorbed from the gut into the bloodstream. The abundant glucose is the exclusive fuel used by the brain. The high blood glucose level also stimulates the <span class="math inline">\(\beta\)</span> cells in the islets of Langerhans in the pancreas to release insulin, which in turn promotes uptake and storage of glucose (as glycogen, in liver, muscle, actrocytes, and other tissues) and triacylglycerol (in lipid droplets in adipocytes).</p>
<p>During the post-absorptive phase, blood glucose concentration drops. Islets of Langerhans respond by reducing production of insulin in <span class="math inline">\(\beta\)</span> cells and increasing production of glucagon in <span class="math inline">\(\alpha\)</span> cells.</p>
<p>The principal effect of glucagon is to promote glycogenolysis in liver.</p>
<p>On the plasma membrane of hepatocytes, glucagon binds to its receptors, which are G-protein coupled receptors (GPCR), on hepatocyte cell surface membrane. Upon glucagon binding, the receptor acts as a guanine nucleotide exchange factor and replace GDP bound to the G protein with GTP. The activated heterotrimeric G protein dissociates into <span class="math inline">\(\beta\gamma\)</span> and <span class="math inline">\(\alpha\)</span> subunits.</p>
<p>Some glucagon receptors are associated with G<sub>s</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates adenylyl cyclase, which produces cAMP, which in turn activates protein kinase A (PKA). PKA phosphorylates phosphorylase kinase, thus activating it. Other glucagon receptors are associated with G<sub>q</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates phospholipase C, which converts PIP2 (phosphatidylinositol bisphosphate) to IP3 (inositol trisphosphate) and DAG (diacylglycerol). IP3 stimulates Ca<sup>2+</sup> release from the smooth endoplasmic reticulum, then Ca<sup>2+</sup> binds to phosphorylase kinase, also activating it.</p>
<p>Activated phosphorylase kinase phosphorylates glycogen phosphorylase <em>b</em> (inactive form), converting it to glycogen phosphorylase <em>a</em> (active form). Then, with help from the debranching enzyme, glycogen phosphorylase <em>a</em> break down glycogen into glucose 1-phosphate, which can be converted by phosphoglucose mutase to glucose 6-phosphate, which is then dephosphorylated by glucose 6-phosphatase. The resulting glucose is released into the bloodstream via GLUT2 transporter down the concentration gradient.</p>
<p>Glycogenolysis occurs not only in liver but also in astrocytes. Unlike liver glycogenolysis, however, 1-phosphoglucose in astrocytes are converted to lactate instead of glucose. Lactate is transported via monocarboxylate transporter (MCT) and supplied to neurons.</p>
<p>During this period, the brain still uses glucose-derived fuel (glucose and lactate) as the sole fuel at the same rate, so do other parts of the body.</p>
</div>
<div id="stage-iii-early-starvation-gluconeogenesis" class="section level1">
<h1>Stage III (Early Starvation): Gluconeogenesis</h1>
<p>As the glycogen storage become depleted, most tissues start to use fatty acids released by white adipose tissue as the fuel, but the brain continue to use, almost exclusively, glucose, which are now derived from gluconeogenesis.</p>
<p>In the human body, the principal substrates for gluconeogenesis are lactate, glucogenic amino acids and glycerol. During starvation, lactate is not an important gluconeogenic substrate for net gain of glucose, because it is produced mostly by anaerobic glycolysis of glucose itself.</p>
<p>During starvation, body protein is the only source of amino acids for gluconeogenesis. These amino acids are initially supplied by hepatic proteolysis and later predominantly provided by muscle proteins. Although the composition of muscle proteins is rather heterogenous, the majority of the release amino acids are alanine and glutamine. Most alanine is taken up by the liver for gluconeogenesis, while the role of glutamine is more varied—it is used by the kidney for acid-base balance during starvation, for example.</p>
<p>Glycerol, the other source molecule for gluconeogenesis during starvation, is produced by the adipocytes in the white adipose tissue in a process known as lipolysis (fat mobilisation). Lipolysis involves the sequential removal of the three fatty acids of a TAG molecule by adipose triglyceride lipase (ATGL), hormone sensitive hormone (HSL), and monoglyceride lipase (MGL), and the removal of the first two fatty acids by ATGL and HSL is under hormonal control (MGL is always active).</p>
<p>Catecholamines (adrenaline/noradrenaline), which are released upon activation of the sympathetic nervous system at the early stages of starvation, are the major activators of lipolysis. They do so via the GPCR-G<sub>s</sub>-AC-cAMP-PKA pathway, similar to that used by glucagon. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL.</p>
<p>Insulin is the major inhibitor of lipolysis. Insulin activates PKB (Akt), which in turn activates phosphodiesterase 3B (PDE-3B). PDE-3B de-cyclise cAMP, thus antagonising the PKA activity. Thus, during starvation, the low insulin level means de-inhibition of lipolysis.</p>
<p>The resulting glycerol and fatty acids are all released into the circulation, the former being used as a substrate for gluconeogenesis and the latter being used as the principal fuel by most tissues and well as a substrate for ketone bodies synthesis in the liver.</p>
<p>There are a number of mechanisms by which the liver ensures that gluconeogenenic precursors are used for gluconeogenesis rather than oxidation. For example, the large amount of acetyl-CoA obtained via <span class="math inline">\(\beta\)</span>-oxidation inhibits pyruvate dehydrogenase complex, the activity of which would result in irreversible loss of gluconeogenic precursors. As an another example, the low insulin level has a long-term effect on gluconeogenesis by altering gene expression. Insulin leads to activation of PKB, and PKB inactivates (by phosphorylation) the transcription factor FOXO1 that promotes transcription of gluconeogenenic enzymes such as PEP carboxykinase. Thus, the low insulin level during starvation effectively activates the transcription of these gluconeogenic enzymes.</p>
</div>
<div id="stage-iv-prolonged-starvation-ketone-body-synthesis" class="section level1">
<h1>Stage IV (Prolonged Starvation): Ketone Body Synthesis</h1>
<p>During prolonged starvation, the brain gradually adopts ketone bodies (acetoacetate and 3-hydroxybutyrate) as an alternative fuel.</p>
<p>The ultimate origin of ketone bodies is the TAG stored in the white adipose tissue. As described above, low insulin level allows continuous lipolysis, resulting in release of fatty acids and glycerol into the bloodstream. Many fatty acid molecules are absorbed by adipocytes, in which they undergo <span class="math inline">\(\beta\)</span>-oxidation in mitochondria, producing acetyl-CoA in the liver. A part of acetyl-CoA is used for respiration, and the excess is converted either to acetoacetate in a 3-step sequence of reactions involving thiolase, HMG-CoA synthetase and HMG-CoA lyase or, with an additional reduction step, to 3-hydroxybutyrate (in mitochondria). Ketone bodies are released via the monocarboxylate transporter (MCT).</p>
<p>The ketone bodies can cross the blood-brain barrier and reach the neurons (via MCT). In the mitochondria of neurons, ketone bodies are converted back to acetyl-CoA, which is used for respiration. The large amount of acetyl-CoA obtained from ketone bodies inhibits pyruvate dehydrogenase complex and thus restricts glucose utilisation in much the same way acetyl-CoA derived from <span class="math inline">\(\beta\)</span>-oxidation in other tissues suppress glucose oxidation.</p>
<p>This gradual switching from glucose to ketone bodies as the fuel in the brain is important to the conservation of body proteins. As described previously, gluconeogenesis is the only source of glucose during starvation, and amino acids obtained from muscle proteolysis represent the majority of gluconeogenic precursors during starvation. Thus, conservation of body protein during starvation is equivalent to minimising the comsumption of glucose as fuel, specifically in the brain.</p>
<p>Prolonged starvation is accompanied by a gradual decrease in concentration of the active thyroid hormone, triiodothyronine (T3) over time, which in turn leads to a decrease in basal metabolic rate in most tissues. This reduces the overall consumption of energy of the body and thus also contributes to the sparing of muscle protein.</p>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p>During a period of starvation, the origin and identity of fuel molecules supplied to the brain (neurons) change, according to this order: astrocyte glycogenolysis, liver glycogenolysis, gluconeogenesis, ketone body synthesis. At the later stages, gluconeogenesis and ketone body synthesis coexist, but the amount of ketone body synthesis gradually increase, in order to spare the functional muscle proteins. This metabolic adaptation to starvation is largely under hormonal control, where insulin, glucagon, thyroid hormone and catecholamines play the most promiment roles.</p>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>