<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>radioactivity - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="radioactivity - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/radioactivity/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">radioactivity</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="../lab/2019-10-16-specific-radioactivity.html" class="archive-item-link">
                Specific Radioactivity 
              </a>
              <span class="archive-item-date">
                2019-10-20
              </span>
              <div class="article-summary" style="display: none;">
                
                  Specific radioactivity of ATP and glucose.<hr>
                
                


<div id="TOC">
<ul>
<li><a href="#aim"><span class="toc-section-number">1</span> Aim</a></li>
<li><a href="#reagents-and-apparatus-provided"><span class="toc-section-number">2</span> Reagents and Apparatus Provided</a></li>
<li><a href="#method"><span class="toc-section-number">3</span> Method</a><ul>
<li><a href="#method-atp"><span class="toc-section-number">3.1</span> ATP concentration</a><ul>
<li><a href="#dilution"><span class="toc-section-number">3.1.1</span> Dilution</a></li>
<li><a href="#results"><span class="toc-section-number">3.1.2</span> Results</a></li>
</ul></li>
<li><a href="#method-glucose"><span class="toc-section-number">3.2</span> Glucose concentration</a><ul>
<li><a href="#mechanism"><span class="toc-section-number">3.2.1</span> Mechanism</a></li>
<li><a href="#determine-the-volume-of-enzyme"><span class="toc-section-number">3.2.2</span> Determine the volume of enzyme</a></li>
<li><a href="#determining-lambda_textmax"><span class="toc-section-number">3.2.3</span> Determining <span class="math inline">\(\lambda_\text{max}\)</span></a></li>
<li><a href="#calibration"><span class="toc-section-number">3.2.4</span> Calibration</a></li>
<li><a href="#sample-assay"><span class="toc-section-number">3.2.5</span> Sample Assay</a></li>
<li><a href="#results-1"><span class="toc-section-number">3.2.6</span> Results</a></li>
</ul></li>
<li><a href="#radioactivity-counting-virtual"><span class="toc-section-number">3.3</span> Radioactivity Counting (Virtual)</a></li>
</ul></li>
<li><a href="#our-results"><span class="toc-section-number">4</span> Our Results</a><ul>
<li><a href="#our-atp"><span class="toc-section-number">4.1</span> ATP</a><ul>
<li><a href="#distinguish"><span class="toc-section-number">4.1.1</span> Distinguishing between 4.0mM and 4.5mM ATP</a></li>
</ul></li>
<li><a href="#our-glucose"><span class="toc-section-number">4.2</span> Glucose</a><ul>
<li><a href="#glucose-calibration-results"><span class="toc-section-number">4.2.1</span> Calibration</a></li>
<li><a href="#our-glucose-sample"><span class="toc-section-number">4.2.2</span> Sample result</a></li>
<li><a href="#distinguish-glucose"><span class="toc-section-number">4.2.3</span> Distinguishing between 4.0mM and 4.5mM glucose</a></li>
</ul></li>
</ul></li>
<li><a href="#class-results"><span class="toc-section-number">5</span> Class Results</a><ul>
<li><a href="#dataremoved"><span class="toc-section-number">5.1</span> Data</a></li>
<li><a href="#preliminary-graphical-analysis"><span class="toc-section-number">5.2</span> Preliminary graphical analysis</a></li>
<li><a href="#class-atp"><span class="toc-section-number">5.3</span> ATP concentration</a><ul>
<li><a href="#grouping-and-grouped-confidence-interval-of-mean"><span class="toc-section-number">5.3.1</span> Grouping and grouped confidence interval of mean</a></li>
<li><a href="#test-for-significance-of-the-difference-in-concentration-between-the-two-groups"><span class="toc-section-number">5.3.2</span> Test for significance of the difference in concentration between the two groups</a></li>
<li><a href="#test-for-diviation-from-given-concentrations"><span class="toc-section-number">5.3.3</span> Test for diviation from given concentrations</a></li>
</ul></li>
<li><a href="#class-glucose"><span class="toc-section-number">5.4</span> Glucose</a><ul>
<li><a href="#grouping-and-grouped-confidence-interval-of-mean-1"><span class="toc-section-number">5.4.1</span> Grouping and grouped confidence interval of mean</a></li>
<li><a href="#test-for-significance-of-the-difference-in-concentration-between-the-two-groups-1"><span class="toc-section-number">5.4.2</span> Test for significance of the difference in concentration between the two groups</a></li>
<li><a href="#test-for-diviation-from-given-concentrations-1"><span class="toc-section-number">5.4.3</span> Test for diviation from given concentrations</a></li>
</ul></li>
<li><a href="#radioactivity"><span class="toc-section-number">5.5</span> Radioactivity</a><ul>
<li><a href="#results-of-the-sample"><span class="toc-section-number">5.5.1</span> Results of the sample</a></li>
<li><a href="#correction-for-overlap"><span class="toc-section-number">5.5.2</span> Correction for overlap</a></li>
<li><a href="#specific-radioactivity"><span class="toc-section-number">5.5.3</span> Specific Radioactivity</a></li>
</ul></li>
</ul></li>
<li><a href="#appendix"><span class="toc-section-number">6</span> Appendix</a><ul>
<li><a href="#temp"><span class="toc-section-number">6.1</span> People should report all measurements to get a true 95% confidence interval</a></li>
<li><a href="#the-mathematical-computation-behind-r-functions-used-in-this-report"><span class="toc-section-number">6.2</span> The mathematical computation behind R functions used in this report</a></li>
</ul></li>
</ul>
</div>

<div id="aim" class="section level1">
<h1><span class="header-section-number">1</span> Aim</h1>
<ul>
<li>Given a sample containing (fake) radioactive <sup>14</sup>C glucose and <sup>3</sup>H ATP:
<ul>
<li>determine concentration of glucose and ATP.
<ul>
<li>Given that both concentrations lie between 1mM and 10mM.</li>
<li>Given that <span class="math inline">\(\epsilon^{260}_\text{ATP}=1.57\times10^4 \text{ mol}^{-1}\text{ dm}^3\text{ dm}^{-1}\)</span></li>
</ul></li>
<li>determine radioactivity of the sample (virtual practical)</li>
<li>and hence determine the specific radioactivity of ATP and glucose</li>
</ul></li>
<li>23 samples have glucose and ATP concentrations falling into two distinct classes. They are independently measured by 23 pairs of students. Determine these concentrations using the class results and assess precision and accuracy.</li>
</ul>
</div>
<div id="reagents-and-apparatus-provided" class="section level1">
<h1><span class="header-section-number">2</span> Reagents and Apparatus Provided</h1>
<ul>
<li>15mL sample solution</li>
<li>0.1 M Na<sub>3</sub>PO<sub>4</sub> buffer at pH 7.0</li>
<li>1mM ABTS</li>
<li>Glucose Oxidase (100 U/mL) and peroxidase (20 U/mL)</li>
<li>Solid glucose</li>
<li>Spectrophotometer</li>
</ul>
</div>
<div id="method" class="section level1">
<h1><span class="header-section-number">3</span> Method</h1>
<div id="method-atp" class="section level2">
<h2><span class="header-section-number">3.1</span> ATP concentration</h2>
<div id="dilution" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Dilution</h3>
<div id="determining-scale-of-dilution" class="section level4">
<h4><span class="header-section-number">3.1.1.1</span> Determining scale of dilution</h4>
<p>Given that 1mM &lt; <a href="#our-atp">ATP</a> &lt; 10mM and that <span class="math inline">\(\epsilon^{260}_\text{ATP}=1.57\times10^4 \text{ mol}^{-1}\text{ dm}^3\text{ dm}^{-1}\)</span>, according to <span class="math inline">\(A^{260}=\epsilon^{260}cl\)</span>:</p>
<p>if no dilution:</p>
<p><span class="math display">\[
\begin{aligned}
1.57\times10^4\times0.001&lt;&amp;A^{260}&lt;1.57\times10^4\times0.01 \\
15.7&lt;&amp;A^{260}_\text{raw}&lt;157 \\
\end{aligned}
\]</span></p>
<p>clearly, a 1:100 dilution would be very likely to produce a sensible <span class="math inline">\(A^{260}\)</span>:</p>
<p><span class="math display">\[0.157&lt;A^{260}_\text{1:100 dilution}&lt;1.57\]</span></p>
</div>
<div id="making-dilution" class="section level4">
<h4><span class="header-section-number">3.1.1.2</span> Making dilution</h4>
<p>5mL of 1:100 diluton is made serially by first fixing 500 <span class="math inline">\(\mu\text{L}\)</span> sample with 4.5mL buffer then mixing 500 <span class="math inline">\(\mu\text{L}\)</span> of the resulting 1:10 dilution with another 4.5mL buffer.</p>
<p>A preliminary test gives <span class="math inline">\(A^{260}_\text{1:100 dilution}=0.503\)</span>, which is within the range where the spectrophotometer is accurate, so we made 4 more 1:100 dilutions using the method described above.</p>
</div>
</div>
<div id="results" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Results</h3>
<p>See section <a href="#our-atp">4.1</a></p>
</div>
</div>
<div id="method-glucose" class="section level2">
<h2><span class="header-section-number">3.2</span> Glucose concentration</h2>
<div id="mechanism" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Mechanism</h3>
<p>Glucose itself does not have convenient absorption. Thus, glucose oxidase, ABTS and peroxidase is added to make a green solution (glucose is first oxidised by oxidase to produce gluconic acid and H<sub>2</sub>O<sub>2</sub>; peroxidase then catalyses the reaction between H<sub>2</sub>O<sub>2</sub> and ABTS to produce a green dye).</p>
</div>
<div id="determine-the-volume-of-enzyme" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Determine the volume of enzyme</h3>
<p>We want the reaction to ‘virtually’ complete (i.e. &gt; 99% conversion of substrate) in 10 min.</p>
<p>For an enzyme reaction:</p>
<p><span class="math display">\[-\dfrac{ds}{dt}=v-\dfrac{V_\text{max}s}{K_m+S}\]</span></p>
<p>As the reaction approaches completion, <span class="math inline">\(s&lt;&lt;K_m\)</span>:</p>
<p><span class="math display">\[-\dfrac{ds}{dt}=\dfrac{V_\text{max}s}{K_m}\]</span></p>
<p>Integrating to determine the total time for a given change in <span class="math inline">\(s\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
-\int_{s_0}^{s_1}{\dfrac{ds}{s}}&amp;=\dfrac{V_\text{max}}{K_m}\int_0^t dt \\
\ln\dfrac{s_0}{s_1}&amp;=\dfrac{V_\text{max}t}{K_m}
\end{aligned}
\]</span></p>
<p>For a decrease in <span class="math inline">\(s\)</span> from 100% to 1% in 10 min:</p>
<p><span class="math display">\[
\begin{aligned}
\ln\dfrac{100}{1}&amp;=\dfrac{V_\text{max}\times 10}{K_m} \\
V_\text{max}&amp;=0.4605K_m \text{ (mol L}^{-1}\text{min}^{-1}\text{)}
\end{aligned}
\]</span></p>
<p>Our assay volume will be 4.0mL, so the amound of enzyme, in U (<span class="math inline">\(\mu\text{mol min}^{-1}\)</span>), will be</p>
<p><span class="math display">\[\text{amount of enzyme in U}=0.4605\times K_m \times 4.0 \times 10^{-3}\times10^6=1842K_m\]</span></p>
<p>where <span class="math inline">\(K_m\)</span> has unit M.</p>
<p>For GOD, <span class="math inline">\(K_m=1\text{ mM}\)</span>, and the provided enzyme concentration is 100U/mL:</p>
<p><span class="math display">\[
\begin{aligned}
K_m&amp;=1\text{ mM} \\
\text{minimum amount of enzyme}&amp;=1842 \times 1 \times 10^{-3}=1.84 \text{ U} \\
\text{minimum volume of enzyme}&amp;=1.84 \div 100=0.0184 \text{ mL}
\end{aligned}
\]</span></p>
<p>For POD, <span class="math inline">\(K_m=100\text{ }\mu\text{M}\)</span>, and the provided enzyme concentration is 20U/mL:</p>
<p><span class="math display">\[
\begin{aligned}
K_m&amp;=100\text{ }\mu\text{M} \\
\text{minimum amount of enzyme}&amp;=1842 \times 100 \times 10^{-6}=0.184 \text{ U} \\
\text{minimum volume of enzyme}&amp;=0.184 \div 20=0.0092 \text{ mL}
\end{aligned}
\]</span></p>
<p>We decided to use an excess amount of enzyme (i.e. 0.1mL for both). This guarantees that the reactions will complete within 10 min, and makes it easier to make up the 4mL total volume for assay (see section <a href="#glu-assay">3.2.4.2</a>).</p>
</div>
<div id="determining-lambda_textmax" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Determining <span class="math inline">\(\lambda_\text{max}\)</span></h3>
<p>Measuring absorbance near <span class="math inline">\(\lambda_\text{max}\)</span> reduces error. An 100nmol-glucose-containing (i.e. with 1mL 0.1mM glucose solution, see section <a href="#glu-assay">3.2.4.2</a>) assay result is used. The absorbance in the range 350nm to 450nm is measured, and <span class="math inline">\(\lambda_\text{max} = 418\text{ nm}\)</span>. We then used this wavelength for all glucose assays.</p>
</div>
<div id="calibration" class="section level3">
<h3><span class="header-section-number">3.2.4</span> Calibration</h3>
<p>Calibrate <span class="math inline">\(A^{418}\)</span> with known glucose concentrations, then we can map <span class="math inline">\(A^{418}_\text{sample}\)</span> to the calibration curve to determine sample glucose concentration.</p>
<div id="dilution-1" class="section level4">
<h4><span class="header-section-number">3.2.4.1</span> Dilution</h4>
<ul>
<li>0.18 g glucose is dissolved in 100mL buffer, which gives 10mM glucose solution (which is handy, as the maximum possible concentration of glucose in the sample is also 10mM)</li>
<li>100mL 0.1mM glucose solution is made by mixing 1mL 10mM glucose solution with 99mL buffer (1:100 dilution).</li>
</ul>
</div>
<div id="glu-assay" class="section level4">
<h4><span class="header-section-number">3.2.4.2</span> Assay</h4>
<p>Assays are made on 4mL mixtures with varying amount of glucose, according to the following table:</p>
<table>
<thead>
<tr class="header">
<th align="center">ABTS</th>
<th align="center">Buffer</th>
<th align="center">0.1mM glucose</th>
<th align="center">H<sub>2</sub>O</th>
<th align="center">GOD</th>
<th align="center">POD</th>
<th align="center">nmol glucose</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">1.0mL</td>
<td align="center">0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">100</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.8mL</td>
<td align="center">0.2mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">80</td>
</tr>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.6mL</td>
<td align="center">0.4mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">60</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.4mL</td>
<td align="center">0.6mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">40</td>
</tr>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">20</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0mL</td>
<td align="center">1.0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">0</td>
</tr>
</tbody>
</table>
<p>Then the mixture is incubated at 37<span class="math inline">\(^\circ\text{C}\)</span> for 10 minutes (in a theromostatically-controlled water bath), then <span class="math inline">\(A^{418}\)</span> is measured.</p>
<p>Every row is repeated 3 times.</p>
</div>
</div>
<div id="sample-assay" class="section level3">
<h3><span class="header-section-number">3.2.5</span> Sample Assay</h3>
<p>The sample is diluted 1:100 by mixing 1mL sample with 99mL buffer.</p>
<p>4mL assays are made on 1mL 1:100 dilution 3 times, according to the following table (identical to the assays used for calibration except that an unknown concentration is used this time)</p>
<table>
<thead>
<tr class="header">
<th align="center">ABTS</th>
<th align="center">Buffer</th>
<th align="center">diluted sample</th>
<th align="center">GOD</th>
<th align="center">POD</th>
<th align="center"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">1.0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
</div>
<div id="results-1" class="section level3">
<h3><span class="header-section-number">3.2.6</span> Results</h3>
<p>See section <a href="#our-glucose">4.2</a></p>
</div>
</div>
<div id="radioactivity-counting-virtual" class="section level2">
<h2><span class="header-section-number">3.3</span> Radioactivity Counting (Virtual)</h2>
<p>0.2mL sample is mixed with 1.8mL scintillation fluid in a polyvial and placed into the scintillation counter. The count time is 2min. The results are shown in section <a href="#radioactivity">5.5</a>.</p>
</div>
</div>
<div id="our-results" class="section level1">
<h1><span class="header-section-number">4</span> Our Results</h1>
<div id="our-atp" class="section level2">
<h2><span class="header-section-number">4.1</span> ATP</h2>
<p>(Methods in section <a href="#method-atp">3.1</a>)</p>
<pre class="r"><code>A &lt;- c(0.503, 0.513, 0.508, 0.525, 0.516)
c_ATP &lt;- A/1.57e4*100*1000
kable(tibble(`$A^{260}_\\text{1:100 dilution}$`=A, `[ATP] of raw sample (mM)`=c_ATP))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right"><span class="math inline">\(A^{260}_\text{1:100 dilution}\)</span></th>
<th align="right"><a href="#our-atp">ATP</a> of raw sample (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">0.503</td>
<td align="right">3.203822</td>
</tr>
<tr class="even">
<td align="right">0.513</td>
<td align="right">3.267516</td>
</tr>
<tr class="odd">
<td align="right">0.508</td>
<td align="right">3.235669</td>
</tr>
<tr class="even">
<td align="right">0.525</td>
<td align="right">3.343949</td>
</tr>
<tr class="odd">
<td align="right">0.516</td>
<td align="right">3.286624</td>
</tr>
</tbody>
</table>
<p>Statistical summary of <a href="#our-atp">ATP</a> in sample:</p>
<pre class="r"><code>x &lt;- c_ATP; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 3.26751592 0.05309978 0.02374695 2.77644511 0.06593209</code></pre>
<p>Which gives 95% confidence interval: <span class="math inline">\(\text{[ATP]}=3.268\pm0.054\)</span> mM</p>
<div id="distinguish" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Distinguishing between 4.0mM and 4.5mM ATP</h3>
<p>The percentage error (coefficient of variance) of this experiment is:</p>
<p><span class="math display">\[c_v=\dfrac{s_x}{\bar{x}}=\dfrac{0.054}{3.268}=0.0165\]</span></p>
<p>for <span class="math inline">\(\bar{x}\)</span> = 4.0 and 4.5, <span class="math inline">\(s_x=c_v\bar{x}\)</span> = 0.0661 and 0.0744, respectively.</p>
<p>To distinguish between the two, two sample t test is used. If <span class="math inline">\(n\)</span> measurements are made on each concentration, then</p>
<p><span class="math display">\[t=\dfrac{\bar{x_1}-\bar{x_2}}{\sqrt{\dfrac{s_1^2}{n}+\dfrac{s_2^2}{n}}}=\dfrac{4.5-4.0}{\sqrt{\dfrac{0.0661^2+0.0744^2}{n}}}=\sqrt{25.26n}\]</span></p>
<p>then we can get a list of <span class="math inline">\(p\)</span> values for each <span class="math inline">\(n\)</span>:</p>
<pre class="r"><code>n &lt;- 2:8
df &lt;- n - 1
t &lt;- sqrt(25.26 * n)
p &lt;- (1 - pt(t, df)) * 2
names(p) &lt;- paste(&#39;n =&#39;, n)
round(p, 5)</code></pre>
<pre><code>##   n = 2   n = 3   n = 4   n = 5   n = 6   n = 7   n = 8 
## 0.08898 0.01294 0.00210 0.00036 0.00006 0.00001 0.00000</code></pre>
<p>as n increases, <span class="math inline">\(p\)</span> decreases. When n = 3, <span class="math inline">\(p\)</span> is already less than 0.05.</p>
</div>
</div>
<div id="our-glucose" class="section level2">
<h2><span class="header-section-number">4.2</span> Glucose</h2>
<p>(Methods in section <a href="#method-glucose">3.2</a>)</p>
<div id="glucose-calibration-results" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Calibration</h3>
<p>The results are shown below:</p>
<pre class="r"><code>A1 &lt;- c(0.952, 0.815, 0.709, 0.496, 0.272, 0)
A2 &lt;- c(1.223, 0.969, 0.704, 0.415, 0.222, 0)
A3 &lt;- c(1.194, 0.919, 0.682, 0.452, 0.236, 0)
A &lt;- c(A1, A2, A3)
glucose = c(100, 80, 60, 40, 20, 0)
kable(tibble(`nmol glucose in 4mL assay` = glucose, `$A^{418}_1$` = A1,
             `$A^{418}_2$` = A2, `$A^{418}_3$` = A3))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right">nmol glucose in 4mL assay</th>
<th align="right"><span class="math inline">\(A^{418}_1\)</span></th>
<th align="right"><span class="math inline">\(A^{418}_2\)</span></th>
<th align="right"><span class="math inline">\(A^{418}_3\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">100</td>
<td align="right">0.952</td>
<td align="right">1.223</td>
<td align="right">1.194</td>
</tr>
<tr class="even">
<td align="right">80</td>
<td align="right">0.815</td>
<td align="right">0.969</td>
<td align="right">0.919</td>
</tr>
<tr class="odd">
<td align="right">60</td>
<td align="right">0.709</td>
<td align="right">0.704</td>
<td align="right">0.682</td>
</tr>
<tr class="even">
<td align="right">40</td>
<td align="right">0.496</td>
<td align="right">0.415</td>
<td align="right">0.452</td>
</tr>
<tr class="odd">
<td align="right">20</td>
<td align="right">0.272</td>
<td align="right">0.222</td>
<td align="right">0.236</td>
</tr>
<tr class="even">
<td align="right">0</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
</tr>
</tbody>
</table>
<p>To fit a straight line, linear regression is used. The coefficients are shown below:</p>
<pre class="r"><code>summary(lm(A ~ 0 + rep(glucose, 3)))</code></pre>
<pre><code>## 
## Call:
## lm(formula = A ~ 0 + rep(glucose, 3))
## 
## Residuals:
##       Min        1Q    Median        3Q       Max 
## -0.181273 -0.000982  0.005691  0.039277  0.089727 
## 
## Coefficients:
##                  Estimate Std. Error t value Pr(&gt;|t|)    
## rep(glucose, 3) 0.0113327  0.0002387   47.48   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.06132 on 17 degrees of freedom
## Multiple R-squared:  0.9925, Adjusted R-squared:  0.9921 
## F-statistic:  2254 on 1 and 17 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>which shows there is a linear relationship <span class="math inline">\(A^{418}=0.01133n_\text{glucose}\)</span> with <span class="math inline">\(R^2=0.9921\)</span>.</p>
<p>Then we can plot it (CI added).</p>
<pre class="r"><code>ggplot(data.frame(A, glucose), aes(glucose, A))+
  geom_point()+
  geom_smooth(formula = y ~ 0 + x, method = &#39;lm&#39;)+
  labs(x = &#39;nmol glucose in 4mL assay&#39;, y = expression(A ^ 418),
       title = &#39;Calibrationn Curve for 4mL essay&#39;)</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-7-1.png" width="60%" style="display: block; margin: auto;" /></p>
</div>
<div id="our-glucose-sample" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Sample result</h3>
<p>Three independent measurements of <span class="math inline">\(A^{418}_\text{sample}\)</span> gives 0.863, 0.872 and 0.871. According to the linear relationship <span class="math inline">\(A^{418}=0.01133n_\text{glucose}\)</span> we can calculate the amount (in nmol) of glucose in 1mL 1:100 dilution, then the concentration of the raw sample can be calculated easily:</p>
<pre class="r"><code>A &lt;- c(0.863, 0.872, 0.871) # sample absorbance
n &lt;- A/0.01133 # nmol glucose in 1 mL 1:100 dilution
(c_glucose_sample &lt;- n * 1e-9 # converting to mol/mL (1:100 dilution)
       / 1e-3 # converting to M (1:100 dilution)
       * 100 # in M (raw sample)
       * 1e3) # converting to mM (raw sample)</code></pre>
<pre><code>## [1] 7.616946 7.696381 7.687555</code></pre>
<p>Statistical summary:</p>
<pre class="r"><code>x &lt;- c_glucose_sample; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t * sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 7.66696087 0.04353824 0.02513682 4.30265273 0.10815499</code></pre>
<p>Which gives 95% confidence interval: <span class="math inline">\(\text{[glucose]}=7.667\pm0.057\)</span> mM</p>
</div>
<div id="distinguish-glucose" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Distinguishing between 4.0mM and 4.5mM glucose</h3>
<p>Similar to the calculation for ATP shown in section <a href="#distinguish">4.1.1</a>, first calculate the percentage error (coefficient of variance) of this experiment:</p>
<p><span class="math display">\[c_v=\dfrac{s_x}{\bar{x}}=\dfrac{0.057}{7.667}=0.00743\]</span></p>
<p>for <span class="math inline">\(\bar{x}\)</span> = 4.0 and 4.5, <span class="math inline">\(s_x=c_v\bar{x}\)</span> = 0.0297 and 0.0335, respectively.</p>
<p>To distinguish between the two, two sample t test is used. If <span class="math inline">\(n\)</span> measurements are made on each concentration, then</p>
<p><span class="math display">\[t=\dfrac{\bar{x_1}-\bar{x_2}}{\sqrt{\dfrac{s_1^2}{n}+\dfrac{s_2^2}{n}}}=\dfrac{4.5-4.0}{\sqrt{\dfrac{0.0297^2+0.0335^2}{n}}}=\sqrt{124.7n}\]</span></p>
<p>then we can get a list of <span class="math inline">\(p\)</span> values for each <span class="math inline">\(n\)</span>:</p>
<pre class="r"><code>n &lt;- 2:8
df &lt;- n - 1
t &lt;- sqrt(124.7 * n)
p &lt;- (1 - pt(t, df)) * 2
names(p) &lt;- paste(&#39;n =&#39;, n)
round(p, 5)</code></pre>
<pre><code>##   n = 2   n = 3   n = 4   n = 5   n = 6   n = 7   n = 8 
## 0.04026 0.00266 0.00020 0.00002 0.00000 0.00000 0.00000</code></pre>
<p>as n increases, <span class="math inline">\(p\)</span> decreases. When n = 2, <span class="math inline">\(p\)</span> is already less than 0.05.</p>
<div id="what-difference-would-it-make-to-your-assay-if-you-doubled-the-concentrations-of-abts-god-and-pod" class="section level4">
<h4><span class="header-section-number">4.2.3.1</span> What difference would it make to your assay if you doubled the concentrations of ABTS, GOD and POD?</h4>
<p>Time required for incubation will be shorter and the final intensity of colors (and hence absorption) will be the same.</p>
</div>
</div>
</div>
</div>
<div id="class-results" class="section level1">
<h1><span class="header-section-number">5</span> Class Results</h1>
<div id="dataremoved" class="section level2">
<h2><span class="header-section-number">5.1</span> Data<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></h2>
<pre class="r"><code>ATP &lt;- c(3.93, 4.04, 3.01, 3.94, 3.20, 3.15, 2.81,       
         4.07, 3.17, 3.05, 4.20, 3.15, 4.17, 3.05,       
         3.90, 2.97, 3.11, 4.13, 3.27, 3.85, 2.92)
ATP_sd &lt;- c(0.164, 0.082, 0.080, 0.080, 0.140, 0.126, 0.139,       
            0.071, 0.140, 0.142, 0.002, 0.012, 0.240, 0.033,       
            0.051, 0.091, 0.187, 0.160, 0.053, 5.e-4, 0.218)
glucose &lt;- c(6.89, 6.92, 8.36, 7.38, 7.10, 7.65, 7.70,         
             9.09, 7.34, 7.60, 8.92, 8.80, 8.50, 6.60,       
             3.81, 9.80, 7.73, 9.80, 7.67, 8.50, 8.63)
glucose_sd &lt;- c(0.0880, 0.0104, 0.1560, 0.0150, 0.0603, 0.5900, 1.4580,        
                0.0470, 0.1640, 0.2810, 0.2500, 0.2000, 0.0010, 0.1180,       
                0.0170, 0.0425, 0.1500, 0.0530, 0.0440, 0.1400, 0.4800)
class_results &lt;- tibble(ATP, ATP_sd, glucose, glucose_sd)
kable(tibble(`ATP conc`=ATP, `ATP sd`=ATP_sd, 
             `Glucose conc`=glucose, `Glucose sd`=glucose_sd))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right">ATP conc</th>
<th align="right">ATP sd</th>
<th align="right">Glucose conc</th>
<th align="right">Glucose sd</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">3.93</td>
<td align="right">0.1640</td>
<td align="right">6.89</td>
<td align="right">0.0880</td>
</tr>
<tr class="even">
<td align="right">4.04</td>
<td align="right">0.0820</td>
<td align="right">6.92</td>
<td align="right">0.0104</td>
</tr>
<tr class="odd">
<td align="right">3.01</td>
<td align="right">0.0800</td>
<td align="right">8.36</td>
<td align="right">0.1560</td>
</tr>
<tr class="even">
<td align="right">3.94</td>
<td align="right">0.0800</td>
<td align="right">7.38</td>
<td align="right">0.0150</td>
</tr>
<tr class="odd">
<td align="right">3.20</td>
<td align="right">0.1400</td>
<td align="right">7.10</td>
<td align="right">0.0603</td>
</tr>
<tr class="even">
<td align="right">3.15</td>
<td align="right">0.1260</td>
<td align="right">7.65</td>
<td align="right">0.5900</td>
</tr>
<tr class="odd">
<td align="right">2.81</td>
<td align="right">0.1390</td>
<td align="right">7.70</td>
<td align="right">1.4580</td>
</tr>
<tr class="even">
<td align="right">4.07</td>
<td align="right">0.0710</td>
<td align="right">9.09</td>
<td align="right">0.0470</td>
</tr>
<tr class="odd">
<td align="right">3.17</td>
<td align="right">0.1400</td>
<td align="right">7.34</td>
<td align="right">0.1640</td>
</tr>
<tr class="even">
<td align="right">3.05</td>
<td align="right">0.1420</td>
<td align="right">7.60</td>
<td align="right">0.2810</td>
</tr>
<tr class="odd">
<td align="right">4.20</td>
<td align="right">0.0020</td>
<td align="right">8.92</td>
<td align="right">0.2500</td>
</tr>
<tr class="even">
<td align="right">3.15</td>
<td align="right">0.0120</td>
<td align="right">8.80</td>
<td align="right">0.2000</td>
</tr>
<tr class="odd">
<td align="right">4.17</td>
<td align="right">0.2400</td>
<td align="right">8.50</td>
<td align="right">0.0010</td>
</tr>
<tr class="even">
<td align="right">3.05</td>
<td align="right">0.0330</td>
<td align="right">6.60</td>
<td align="right">0.1180</td>
</tr>
<tr class="odd">
<td align="right">3.90</td>
<td align="right">0.0510</td>
<td align="right">3.81</td>
<td align="right">0.0170</td>
</tr>
<tr class="even">
<td align="right">2.97</td>
<td align="right">0.0910</td>
<td align="right">9.80</td>
<td align="right">0.0425</td>
</tr>
<tr class="odd">
<td align="right">3.11</td>
<td align="right">0.1870</td>
<td align="right">7.73</td>
<td align="right">0.1500</td>
</tr>
<tr class="even">
<td align="right">4.13</td>
<td align="right">0.1600</td>
<td align="right">9.80</td>
<td align="right">0.0530</td>
</tr>
<tr class="odd">
<td align="right">3.27</td>
<td align="right">0.0530</td>
<td align="right">7.67</td>
<td align="right">0.0440</td>
</tr>
<tr class="even">
<td align="right">3.85</td>
<td align="right">0.0005</td>
<td align="right">8.50</td>
<td align="right">0.1400</td>
</tr>
<tr class="odd">
<td align="right">2.92</td>
<td align="right">0.2180</td>
<td align="right">8.63</td>
<td align="right">0.4800</td>
</tr>
</tbody>
</table>
</div>
<div id="preliminary-graphical-analysis" class="section level2">
<h2><span class="header-section-number">5.2</span> Preliminary graphical analysis</h2>
<p>Density plots and histograms are used to examine the distribution of measurements:</p>
<pre class="r"><code>op &lt;- par(mfrow=c(2,2))
for(i in c(&#39;ATP&#39;, &#39;glucose&#39;)){
  main = paste0(&#39;[&#39;, i, &#39;] in 21 samples:&#39;); xlab &lt;- paste(i, &#39;concentration&#39;)
  {
    plot(density(class_results[[i]], bw = c(ATP = 0.1, glucose = 0.2)[i]), 
         main = paste(main, &#39;density plot&#39;), xlab = xlab) 
    rug(class_results[[i]])
  }
  {
    hist(class_results[[i]], breaks = 10, main = paste(main, &#39;histogram&#39;), xlab = xlab)
  }
}</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-12-1.png" width="100%" /></p>
<p>Clealy, <a href="#our-atp">ATP</a> has a clear bimodal distribution, with two peaks at around 3.1 and 4.0.</p>
<p>The distibution of <a href="#class-glucose">glucose</a> is affected by two outliers (the maximum and minimum). After their removal, a bimodal distribution, which peaks at 7.6 and 8.8, is also produced:</p>
<pre class="r"><code>op &lt;- par(mfrow=c(1,2))
glucose &lt;- glucose[glucose != max(glucose) &amp; glucose != min(glucose)]
main = &#39;[glucose] corrected&#39;; xlab &lt;- &#39;glucose concentration&#39;
{
  plot(density(glucose, bw = c(ATP = 0.1, glucose = 0.2)[i]), 
       main = paste(main, &#39;density plot&#39;), xlab = xlab) 
  rug(glucose)
}
{
  hist(glucose, breaks = 10, main = paste(main, &#39;histogram&#39;), xlab = xlab)
}</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-14-1.png" width="100%" height="60%" /></p>
</div>
<div id="class-atp" class="section level2">
<h2><span class="header-section-number">5.3</span> ATP concentration</h2>
<div id="grouping-and-grouped-confidence-interval-of-mean" class="section level3">
<h3><span class="header-section-number">5.3.1</span> Grouping and grouped confidence interval of mean</h3>
<p>According to the plots, evidently there are two ATP concentrations, so we need to divide the 21 measurements into two groups, and calculate their mean and CI separately.</p>
<pre class="r"><code>ATP1 &lt;- ATP[ATP &lt; 3.5] # less than 3.5mM, around 3.0mM
ATP2 &lt;- ATP[ATP &gt; 3.5] # more than 3.5mM, around 4.0mM</code></pre>
<p>For the <a href="#our-atp">ATP</a> around 3.0:</p>
<pre class="r"><code>x &lt;- ATP1; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 3.07166667 0.12995337 0.03751431 2.20098516 0.08256843</code></pre>
<p>95% confidence interval: <span class="math inline">\(\text{[ATP]}=3.07\pm0.08\)</span> mM</p>
<p>For the <a href="#our-atp">ATP</a> around 4.0:</p>
<pre class="r"><code>x &lt;- ATP2; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 4.02555556 0.12620530 0.04206843 2.30600414 0.09700998</code></pre>
<p>95% confidence interval: <span class="math inline">\(\text{[ATP]}=4.03\pm0.10\)</span> mM</p>
</div>
<div id="test-for-significance-of-the-difference-in-concentration-between-the-two-groups" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Test for significance of the difference in concentration between the two groups</h3>
<p>To test there is a significant difference between the two concentrations, t-test<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> is used:</p>
<pre class="r"><code>t.test(ATP1, ATP2)</code></pre>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  ATP1 and ATP2
## t = -16.923, df = 17.66, p-value = 2.346e-12
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -1.0724720 -0.8353058
## sample estimates:
## mean of x mean of y 
##  3.071667  4.025556</code></pre>
<p><span class="math inline">\(p=2.3\times10^{-12}&lt;&lt;0.05\)</span>; the two concentrations are significantly different</p>
</div>
<div id="test-for-diviation-from-given-concentrations" class="section level3">
<h3><span class="header-section-number">5.3.3</span> Test for diviation from given concentrations</h3>
<p>The solutions were actually made up to be 3.0mM and 4.0mM ATP. Were they made up correctly? We need two one-sample t tests using the given concentrations as the population mean, <span class="math inline">\(\mu\)</span>:</p>
<p>3.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 3.0mM)</p>
<pre class="r"><code>t.test(ATP1, mu=3.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  ATP1
## t = 1.9104, df = 11, p-value = 0.08249
## alternative hypothesis: true mean is not equal to 3
## 95 percent confidence interval:
##  2.989098 3.154235
## sample estimates:
## mean of x 
##  3.071667</code></pre>
<p><span class="math inline">\(p=0.082&gt;0.05\)</span>, failed to reject hypothesis. Solution made up correctly.</p>
<p>4.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 4.0mM)</p>
<pre class="r"><code>t.test(ATP2, mu=4.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  ATP2
## t = 0.60748, df = 8, p-value = 0.5604
## alternative hypothesis: true mean is not equal to 4
## 95 percent confidence interval:
##  3.928546 4.122566
## sample estimates:
## mean of x 
##  4.025556</code></pre>
<p><span class="math inline">\(p=0.56&gt;0.05\)</span>, failed to reject hypothesis. Solution made up correctly.</p>
</div>
</div>
<div id="class-glucose" class="section level2">
<h2><span class="header-section-number">5.4</span> Glucose</h2>
<div id="grouping-and-grouped-confidence-interval-of-mean-1" class="section level3">
<h3><span class="header-section-number">5.4.1</span> Grouping and grouped confidence interval of mean</h3>
<p>By observing the plots, the boundary between the two concentrations of glucose is 8.0mM, so we can divide them into two groups:</p>
<pre class="r"><code>glucose1 &lt;- glucose[glucose &lt; 8.0]
glucose2 &lt;- glucose[glucose &gt; 8.0]</code></pre>
<p>And calculate their 95% CIs separately:</p>
<pre class="r"><code>lapply(list(glucose1=glucose1, glucose2=glucose2), function(x){
  n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
  c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))
})</code></pre>
<pre><code>## $glucose1
##      mean        sd        SE         t      t*SE 
## 7.3254545 0.3921317 0.1182322 2.2281389 0.2634377 
## 
## $glucose2
##      mean        sd        SE         t      t*SE 
## 8.6857143 0.2612698 0.0987507 2.4469119 0.2416343</code></pre>
<p>The confidence intervals are: <span class="math inline">\(7.33\pm0.27\)</span> mM and <span class="math inline">\(8.69\pm0.22\)</span> mM</p>
</div>
<div id="test-for-significance-of-the-difference-in-concentration-between-the-two-groups-1" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Test for significance of the difference in concentration between the two groups</h3>
<p>t-test performed to test for significance:</p>
<pre class="r"><code>t.test(glucose1, glucose2)  </code></pre>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  glucose1 and glucose2
## t = -8.8301, df = 15.912, p-value = 1.573e-07
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -1.686972 -1.033548
## sample estimates:
## mean of x mean of y 
##  7.325455  8.685714</code></pre>
<p><span class="math inline">\(p=1.57\times 10^{-7}&lt;&lt;0.05\)</span>, which shows the two concentrations are significantly different.</p>
</div>
<div id="test-for-diviation-from-given-concentrations-1" class="section level3">
<h3><span class="header-section-number">5.4.3</span> Test for diviation from given concentrations</h3>
<p>The solutions were actually made up to be 7.0mM and 9.0mM ATP. Were they made up correctly? We need two one-sample t tests using the given concentrations as the population mean, <span class="math inline">\(\mu\)</span>:</p>
<p>7.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 7.0mM)</p>
<pre class="r"><code>t.test(glucose1, mu=7.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  glucose1
## t = 2.7527, df = 10, p-value = 0.02038
## alternative hypothesis: true mean is not equal to 7
## 95 percent confidence interval:
##  7.062017 7.588892
## sample estimates:
## mean of x 
##  7.325455</code></pre>
<p><span class="math inline">\(p=0.020&lt;0.05\)</span>, reject null hypothesis. Solution made up incorrectly.</p>
<p>9.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 9.0mM)</p>
<pre class="r"><code>t.test(glucose2, mu=9.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  glucose2
## t = -3.1826, df = 6, p-value = 0.01901
## alternative hypothesis: true mean is not equal to 9
## 95 percent confidence interval:
##  8.444080 8.927349
## sample estimates:
## mean of x 
##  8.685714</code></pre>
<p><span class="math inline">\(p=0.019&lt;0.05\)</span>, reject null hypothesis. Solution made up incorrectly.</p>
</div>
</div>
<div id="radioactivity" class="section level2">
<h2><span class="header-section-number">5.5</span> Radioactivity</h2>
<div id="results-of-the-sample" class="section level3">
<h3><span class="header-section-number">5.5.1</span> Results of the sample</h3>
<p>Counts in the H channel and C channel are shown below:</p>
<pre class="r"><code>H &lt;- c(14178.5, 13963, 14065, 14356.5, 13809, 14157.5, 13824.5, 14032, 14094.5, 14014)
C &lt;- c(3411, 3472.5, 3386.5, 3516, 3375.5, 3418.5, 3438, 3464.5, 3513.5, 3438.5)</code></pre>
<p>Statistical summaries are calculated:</p>
<pre class="r"><code>n &lt;- 10; df &lt;- n-1; t = qt(0.975, df)
lapply(list(`H Channel`=H, `C Channel`=C), function(i){
  c(mean=mean(i), sd=sd(i), SE=sd(i)/sqrt(n), t = t, `t*SE` = t*sd(i)/sqrt(n))
})</code></pre>
<pre><code>## $`H Channel`
##         mean           sd           SE            t         t*SE 
## 14049.450000   164.091176    51.890186     2.262157   117.383756 
## 
## $`C Channel`
##        mean          sd          SE           t        t*SE 
## 3443.450000   48.359447   15.292600    2.262157   34.594264</code></pre>
<p>95% confidence intervals:</p>
<ul>
<li>H channel: <span class="math inline">\(14049.5\pm 117.38\)</span> cpm</li>
<li>C channel: <span class="math inline">\(3443.5\pm 34.59\)</span> cpm</li>
</ul>
</div>
<div id="correction-for-overlap" class="section level3">
<h3><span class="header-section-number">5.5.2</span> Correction for overlap</h3>
<p>Let counts due to <sup>3</sup>H = <span class="math inline">\(X\)</span> and counts due to <sup>14</sup>C = <span class="math inline">\(Y\)</span>. Let <span class="math inline">\(\alpha\)</span> be the fraction in H channel of all counts due to <sup>3</sup>H and <span class="math inline">\(\beta\)</span> be the fraction in C channel of all counts due to <sup>14</sup>C, so that for a sample with mixed isotopes:</p>
<p><span class="math display">\[H = \alpha X + (1-\beta)Y\]</span>
<span class="math display">\[C = (1-\alpha)X+\beta Y\]</span></p>
<p><span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> can be determined by counting pure <sup>3</sup>H and <sup>14</sup>C, respectively. Here are the results:</p>
<pre class="r"><code>c(H_in_H = mean(c(464814.5, 462463.5, 464775.5)),
  H_in_C = mean(c(14825.5, 15057.5, 14210.0)),
  C_in_H = mean(c(10295, 9842.5, 9922.5)),
  C_in_C = mean(c(47989, 48868.5, 49133.5)))</code></pre>
<pre><code>##    H_in_H    H_in_C    C_in_H    C_in_C 
## 464017.83  14697.67  10020.00  48663.67</code></pre>
<p>which gives:</p>
<p><span class="math display">\[
\begin{aligned}
 \alpha &amp;= 464017.83/(464017.83+14697.67)=0.9693 \\
 \beta &amp;= 48663.67/(48663.67+10020.00)=0.8293
\end{aligned}
\]</span></p>
<p>Plugging in sample results:</p>
<p><span class="math display">\[
\begin{aligned}
  H &amp;= \alpha X + (1-\beta)Y = 0.9693X+(1-0.8293)Y=14049.5(\pm117.38) \\
  C &amp;= (1-\alpha)X+\beta Y = (1-0.9693)X+0.8293Y = 3443.5(\pm34.59)
\end{aligned}
\]</span></p>
<p>Simplifying…</p>
<p><span class="math display">\[
\begin{aligned}
  0.9693X+0.1707Y&amp;=14049.5(\pm117.38) \\
  0.0307X+0.8293Y &amp;= 3443.5(\pm34.59)
\end{aligned}
\]</span></p>
<p>Solving for X: (note that uncertainties are added)</p>
<p><span class="math display">\[
\begin{aligned}
  4.7075X+0.8293Y &amp;=60721.7(\pm507.31) \\
  0.0307X+0.8293Y &amp;= 3443.5(\pm34.59) \\
  4.6769X &amp;= 57278.2(\pm541.9) \\
X &amp;= 12247.2(\pm115.9)
\end{aligned}
\]</span></p>
<p>Solving for Y:</p>
<p><span class="math display">\[
\begin{aligned}
  0.9693X+0.1707Y&amp;=14049.5(\pm117.38) \\
  0.9693X+26.18Y &amp;= 108722.6(\pm1058.66) \\
  26.01Y &amp;= 94673.1(\pm1176.04) \\
  Y &amp;= 3639.4(\pm45.2)
\end{aligned}
\]</span></p>
<p>which gives:</p>
<p><span class="math display">\[
\begin{aligned}
  \text{actual H (ATP) counts in 0.2mL sample}&amp;=12247.2(\pm115.9) \\
  \text{actual C (glucose) counts in 0.2mL sample} &amp;= 3639.4(\pm45.2)
\end{aligned}
\]</span></p>
</div>
<div id="specific-radioactivity" class="section level3">
<h3><span class="header-section-number">5.5.3</span> Specific Radioactivity</h3>
<p>The radioactivity counting were done on 0.2mL sample. Our ATP concentration (section <a href="#our-atp">4.1</a>) is <span class="math inline">\(\text{[ATP]}=3.268\pm0.054\)</span> mM and glucose concentration (section <a href="#our-glucose-sample">4.2.2</a>) is <span class="math inline">\(\text{[glucose]}=7.667\pm0.057\)</span> mM. Thus, the specific radioactivity of ATP and glucose are:</p>
<p><span class="math display">\[
\begin{aligned}
  \text{ATP: }12247.2(\pm115.9)/[0.2\times10^{-3}\times3.268\times 10^{-3}(\pm0.054\times 10^{-3})]&amp;= 1.87(\pm0.05)\times 10^{10} \text{ cpm mol}^{-1}\\
  \text{glucose: } 3639.4(\pm45.2)/[0.2\times10^{-3}\times7.667\times 10^{-3}(\pm0.057\times 10^{-3})]&amp;= 2.37(\pm0.05) \times 10^{9} \text{ cpm mol}^{-1}
\end{aligned}
\]</span></p>
<p>(the <span class="math inline">\(\pm0.05\)</span> comes from the maximal possible numerator (e.g. <span class="math inline">\(12247.2 + 115.9\)</span> for ATP) over the least possible denominator (e.g. <span class="math inline">\((3.268-0.054)\times10^{-3}\)</span> for ATP))</p>
</div>
</div>
</div>
<div id="appendix" class="section level1">
<h1><span class="header-section-number">6</span> Appendix</h1>
<div id="temp" class="section level2">
<h2><span class="header-section-number">6.1</span> People should report all measurements to get a true 95% confidence interval</h2>
<p>The statistical computations perfomed in section <a href="#class-atp">5.3</a> and section <a href="#class-glucose">5.4</a> for class results are actually improper.</p>
<p>Let’s say 3 people are measuring a concentration whose true value is 4.0. Here are their results:</p>
<pre class="r"><code>c1 &lt;- rnorm(5, 4.0, 3) # 5 times, sd about 3
c2 &lt;- rnorm(6, 4.0, 2) # 6 times, sd about 2
c3 &lt;- rnorm(7, 4.0, 1) # 7 times, sd about 1</code></pre>
<p>To calculate the true mean and sd, we need to combine their results i.e. to get list of 15 independent measurements:</p>
<p>True mean:</p>
<pre class="r"><code>mean(c(c1,c2,c3))</code></pre>
<pre><code>## [1] 4.509869</code></pre>
<p>but if we take the mean of the mean (which in this practical we are supposed to do):</p>
<pre class="r"><code>mean(c(mean(c1),
       mean(c2),
       mean(c3)))</code></pre>
<pre><code>## [1] 4.564576</code></pre>
<p>This gives a different mean from the true mean!</p>
<p>similarly, the true sd calculation is not possible if only the sd of each person’s results are given:</p>
<p>true sd:</p>
<pre class="r"><code>sd(c(c1, c2, c3))</code></pre>
<pre><code>## [1] 1.948202</code></pre>
<p>we might be supposed to calculate:</p>
<pre class="r"><code>mean(c(sd(c1), sd(c2), sd(c3)))</code></pre>
<pre><code>## [1] 1.742614</code></pre>
<p>(see: <a href="https://bit.ly/2Wj505F" class="uri">https://bit.ly/2Wj505F</a> )</p>
<p>What I did was treating each person’s mean as one observation:</p>
<pre class="r"><code>c(
  mean = mean(c(mean(c1),
       mean(c2),
       mean(c3))),
  sd = sd(c(mean(c1),
     mean(c2),
     mean(c3)))
)</code></pre>
<pre><code>##      mean        sd 
## 4.5645764 0.8848836</code></pre>
</div>
<div id="the-mathematical-computation-behind-r-functions-used-in-this-report" class="section level2">
<h2><span class="header-section-number">6.2</span> The mathematical computation behind R functions used in this report</h2>
<p>Statistical computations are done automatically by R. For completeness, I include here the actual mathematical operations performed by the mathematical functions used in this report.</p>
<p>For an R vector <code>x &lt;- c(x1, x2, x3, x4, x5, ... xn)</code>:</p>
<p><code>mean(x)</code> = <span class="math inline">\(\bar{x}=\dfrac{\sum{x}}{n}\)</span>
<code>sd(x)</code> = <span class="math inline">\(s_x=\sqrt{\dfrac{\sum{(x-\bar{x})^2}}{n-1}}=\sqrt{\dfrac{\sum{x^2}-\dfrac{(\sum x)^2}{n}}{n-1}}\)</span></p>
<p><code>lm(y ~ x)</code> produces a linear model <span class="math inline">\(y = b_0 + b_1x\)</span></p>
<p><span class="math display">\[b_1=\dfrac{s_{xy}}{s_x}=\dfrac{\sum{(x-\bar{x})}\sum{(y-\bar{y})}}{\sum{(x-\bar{x})^2}} = \dfrac{n\sum{xy}-\sum x \sum y}{n\sum{x^2}-(\sum x)^2}\]</span></p>
<p>and</p>
<p><span class="math display">\[b_0 = \bar{y} - b_1\bar{x} = \dfrac{\sum y - b_1 \sum x}{n}\]</span></p>
<p>However, I forced intercept to be at the origin (<code>lm(y ~ 0 + x)</code>), so actually <span class="math inline">\(b_1\)</span> is calculated as:</p>
<p><span class="math display">\[b_1=\dfrac{\sum xy}{\sum x^2}\]</span></p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>2 pairs of students produced incomplete results, so only 21/23 sets of results are given.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Specifically, Welch’s two sample t-test with unequal variances<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../worksheet/radioactivity-worksheet.html" class="archive-item-link">
                Radioactivity Worksheet 
              </a>
              <span class="archive-item-date">
                2019-10-16
              </span>
              <div class="article-summary" style="display: none;">
                
                



<div id="guinea-pig" class="section level1">
<h1>Guinea pig</h1>
<p><strong>A guinea pig was given a single injection of <sup>24</sup>NaCl (0.5ml of 0.5<span class="math inline">\(\mu\)</span>Ci/ml). Subsequently, 0.2 ml samples of blood were taken at intervals and counted for radioactivity. The following results were obtained. The efficiency of counting <sup>24</sup>Na is 30%.</strong></p>

<p><strong>(a) Given that the radioactive half-life of <sup>24</sup>Na is 15h, calculate the biological half life of Na<sup>+</sup> in the guinea pig body. Why are these values different?</strong></p>
<p>Radioactive decay is a first-order process, so the rate of decay is proportional to the amount of the decaying substance. This can be expressed with a differential equation: <span class="math inline">\(-\frac{dA}{dt}=kA\)</span>, which can be integrated to give:</p>
<p><span class="math display">\[\ln{A} = \ln{A_0}-kt\]</span></p>
<p>where <span class="math inline">\(A_0\)</span> is the initial radioactivity, <span class="math inline">\(t\)</span> is the time elapsed, <span class="math inline">\(k\)</span> is the decay constant, and <span class="math inline">\(A\)</span> is the radioactivity at time <span class="math inline">\(t\)</span>. A linear relationship exists between <span class="math inline">\(\ln{A}\)</span> and <span class="math inline">\(t\)</span>:</p>
<pre class="r"><code>t &lt;- c(1, 3, 5, 10, 16, 24) # in hours
A &lt;- c(3604, 2928, 2376, 1412, 756, 329)/0.3 # corrected for efficiency
mod &lt;- lm(log(A) ~ t)
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = log(A) ~ t)
## 
## Residuals:
##          1          2          3          4          5          6 
##  2.578e-05  4.796e-04 -2.429e-04 -2.077e-04 -3.928e-04  3.380e-04 
## 
## Coefficients:
##               Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)  9.498e+00  2.503e-04   37948  &lt; 2e-16 ***
## t           -1.041e-01  1.972e-05   -5280 7.72e-15 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.0003878 on 4 degrees of freedom
## Multiple R-squared:      1,  Adjusted R-squared:      1 
## F-statistic: 2.787e+07 on 1 and 4 DF,  p-value: 7.722e-15</code></pre>
<pre class="r"><code>plot(t, log(A), xlab = &#39;t (h)&#39;)
abline(coef(mod))</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-4-1.png" width="100%" /></p>
<p>from linear regression, <span class="math inline">\(A_0=e^{9.498}=13333 \text{ cpm}\)</span>, and <span class="math inline">\(k=0.104 \text{ h}^{-1}\)</span></p>
<p>half life (total): <span class="math inline">\(t_{\frac{1}{2}}=\dfrac{\ln{2}}{k}=6.66 \text{ h}\)</span></p>
<p>Na<sup>+</sup> in biological systems is continuously being excreted, and excretion is generally also a first-order process. So the result is that the overall loss of radioactivity is still a first-order process with a fixed, but shorter half life (i.e. larger <span class="math inline">\(k\)</span>).</p>
<p><span class="math display">\[k_{\text{total}}=k_{\text{radioactivity}}+k_{\text{biological}}\]</span></p>
<p><span class="math display">\[\dfrac{\ln2}{6.66}=\dfrac{\ln{2}}{15}+\dfrac{\ln{2}}{t_{\frac{1}{2}{\text{bioogical}}}}\]</span></p>
<p><span class="math display">\[t_{\frac{1}{2}{\text{bioogical}}}=12.0\]</span></p>
<p><strong>(b) Assuming that Na<sup>+</sup> is essentially excluded from cell water (in the short term), calculate the volume of extracellular fluid in the guinea pig.</strong></p>
<p>from (a), <span class="math inline">\(A_0=e^{9.498}=13333.3 \text{ cpm}\)</span> (in 0.2ml sample), which gives a radioactivity per volume (of blood sample at t = 0): <span class="math inline">\(13333\div0.2=66666\text{ cpm ml}^{-1}\)</span></p>
<p>The total radioactivity is <span class="math inline">\(0.5\text{ ml}\times0.5\times10^{-6}\text{ Ci}\times2.2\times10^{12} = 550000\text{ cpm}\)</span></p>
<p>Volume: <span class="math inline">\(550000/66666=8.25\text{ ml}\)</span></p>
</div>
<div id="sulphate-transport-in-penicillium-chrysogenum" class="section level1">
<h1>Sulphate transport in <em>Penicillium chrysogenum</em></h1>
<p><strong>An experiment was carried out to investigate sulphate transport in the fungus Penicillium chrysogenum. From a culture of an ATP sulphurylase deficient mutant, 1g (wet weight of mycelium was taken and suspended in 100 ml buffer. At t=0, 1ml of 1mM K<sub>2</sub><sup>35</sup>SO<sub>4</sub> was added to the stirred suspension.</strong></p>
<p><strong>(a) 5ml aliquots were taken at 30s intervals, and the mycelium filtered, washed and counted for radioactivity. The following results were obtained.</strong></p>
<pre><code>## $`Cpm on filter after:`
##   30s   60s   90s  120s 
## 10600 21000 31200 41900</code></pre>
<p>To establish the specific radioactivity of the added sulphate, 10<span class="math inline">\(\mu\)</span>l of the stock 1mM K<sub>2</sub><sup>35</sup>SO<sub>4</sub> was also counted, yielding a value of 12,000 cpm.</p>
<p>Calculate the sulphate transport rate in terms of <span class="math inline">\(\mu\)</span>mol/min/g dry wt, assuming that the mycelium contained 15% (by weight) dry matter.</p>
<p>The measurement are made on 5mL aliquots. To calculate the cpm in the 100ml reaction mixture , multiply cpm values by 20. Then do a linear regression between cpm (of all fungus cells) and time elapsed.</p>
<pre class="r"><code>t &lt;- c(0.5, 1.0, 1.5, 2.0) # in minutes
A &lt;- 20 * c(10600, 21000, 31200, 41900)
plot(t, A, xlab = &#39;t (min)&#39;, ylab = &#39;A (cpm)&#39;); abline(lm(A~t))</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-6-1.png" width="100%" /></p>
<pre class="r"><code>coef(lm(A~t))</code></pre>
<pre><code>## (Intercept)           t 
##        3000      416400</code></pre>
<p>the coefficients gives <span class="math inline">\(A=3000 + 416400t\)</span> where <span class="math inline">\(t\)</span> (time elapsed) is in minutes and <span class="math inline">\(A\)</span> in cpm. The slope is the change in cpm per minute (<span class="math inline">\(\dfrac{dA}{dt}=416400\text{ cpm min}^{-1}\)</span>)</p>
<p>From the data of pure K<sub>2</sub>SO~4, the specific radioactivity of K<sub>2</sub>SO~4 is</p>
<p><span class="math inline">\(12000\text{ cpm}\div(10\times10^{-6}\text{ L}\times1\times10^{-3}\text{ M}=1.2\times10^{12}\text{ cpm mol}^{-1})\)</span></p>
<p><span class="math display">\[416400\text{ cpm min}^{-1}/(1.2\times10^{12}\text{ cpm mol}^{-1})\times10^6=0.347\text{ }\mu\text{mol min}^{-1}\]</span></p>
<p>1g mycelium is equivalent to 0.15g dry weight. Transport rate:</p>
<p><span class="math display">\[0.347/0.15=2.31\text{ }\mu\text{mol/min/g dry wt}\]</span></p>
<p><strong>(b) Final samples were taken after 3h and 4h. At these times, the mycelium was too ‘hot’ to count accurately, but the medium contained 220 cpm/ml at both times. Estimate the equilibrium constant for the transport of sulphate in this organism, and comment on this value in terms of the free energy of transport.</strong></p>
<p>Amount of sulfate in the medium at equilibrium: <span class="math inline">\(220\text{ cpm ml}^{-1}\times100\text{ ml}\div(1.2\times10^{12}\text{ cpm mol}^{-1})=1.833\times10^{-8}\text{ mol}\)</span></p>
<p>Total amount of sulfate: <span class="math inline">\(1\times10^{-3}\text{ L}\times1\times10^{-3}\text{ M}=1\times10^{-6}\text{ mol}\)</span></p>
<p>Amount of sulfate in fungi cells at equilibrium: <span class="math inline">\(1\times10^{-6}-1.833\times10^{-8}=9.817\times10^{-7}\)</span></p>
<p>For the reaction X<sub>out</sub><span class="math inline">\(\rightarrow\)</span>X<sub>in</sub>:</p>
<p><span class="math display">\[K=\dfrac{9.817\times10^{-7}}{1.833\times10^{-8}}=53.5\]</span></p>
<p><span class="math display">\[G^\circ=-RT\ln{K}=-9.8\text{ kJ mol}^{-1}\]</span></p>
<p>Sulfate transport into fungi cells is thermodynamically favourable, but the free energy change is not very large.</p>
<p><strong>(c) How would these observations differ if a wild type organism, containing ATP sulphurylase, was used in these experiments?</strong></p>
<p>Once ATP is sulfated, the sulfate groups are no longer free and will not affect the equilibrium between free sulfate ions on both sides. As ATP sulphurylase constantly consumes sulfate inside the cells, the equilibrium is never achieved and <span class="math inline">\(G=G^\circ+RT\ln Q\)</span> will always be negative. Finally, almost all sulfate will enter the cells (and being added to ATP) and this is limited by the equilibrium between unsulfated and sulfated ATP.</p>
</div>
<div id="rnadna-ratio" class="section level1">
<h1>RNA:DNA ratio</h1>
<p><strong>Mutant bacteria can be exploited to provide insight into the ratio of RNA to DNA within cells, if they rely on the uptake of radioactive nutrients. In one such study, a mutant E. coli with a mutation in the enzyme that synthesizes orotic acid and consequently requires uridine for growth was used. This mutant was grown in minimal medium supplemented with <sup>14</sup>C uridine at a specific activity of 30 mCi/mmol. Ribosomal RNA was isolated from the culture.</strong></p>
<p>The base composition of this RNA in mmol/200g was : Uracil = 128.4; Guanine = 189.0; Cytosine =132.4, Adenine = 163.4.</p>
<p><strong>(a) Calculate the specific activity of the RNA in mCi/mg (Hint: think about pyrimidine synthesis when working out the specific activity of the RNA)</strong></p>
<p>Purine synthesis and pyrimidine synthesis use different pathways. In this mutant <em>E. coli</em> uncapable of orotic acid synthesis, uracil and cytosine in RNA are derived from the radioactive <sup>14</sup>C. Both uracil and cytosine molecules have 4 carbon atoms, so both have a specific activity of 30 mCi/mmol.</p>
<p><span class="math display">\[(128.4+132.4)\div(200\times 1000)=1.304\times10^{-3}\text{ mmol pyrimidine/mg RNA}\]</span></p>
<p><span class="math display">\[1.304\times10^{-3}\times30=0.0391\text{ mCi/mg RNA}\]</span></p>
<p><strong>To determine the proportion of DNA complementary to the ribosomal DNA, Membrane filters were charged with 10g denatured <em>E. coli</em> DNA, and incubated with different concentrations of ribosomal RNA labelled as in part (a). A blank filter was also included in each incubation. After thorough washing and treatment with RNAase to remove unhybridised RNA, the radioactivity bound to the filters was determined by liquid scintillation counting. The results are shown in Table 1. 1 mCi = 2.2 x 10<sup>9</sup> dpm. The efficiency of counting is 80%.</strong></p>

<p><strong>(b) Determine the proportion of DNA complementary to the ribosomal RNA</strong></p>
<p>According to the table, when binding is saturated (all 10g DNA is bound complementary RNA), the cpm is around (2828-130) or (2830-140).</p>
<p><span class="math display">\[[(2928-130)+(2830-140)]\div2\div0.8=3367.5\text{ cpm}\]</span></p>
<p><span class="math display">\[3367.5\div (2.2\times10^9)=1.531\text{ mCi}\]</span></p>
<p><span class="math display">\[1.531\text{ mCi}\div0.03912\text{ mCi/mg RNA}=3.91\times10^{-5}\text{ mg} = 3.91\times10^{-8}\text{ g RNA}\]</span></p>
<p><span class="math display">\[3.91\times10^{-8}\text{ g RNA}/10\text{ g DNA}=3.91\times10^{-9}\]</span></p>
<p><strong>(c) If the total chain length of the two ribosomal RNA species is 1.5 x 10<sup>6</sup> Da, and the DNA content of a bacterial cell containing a single chromosome is 2.0 x10<sup>9</sup> Da, how many copies of the genes coding for ribosomal RNA are present in the bacterial chromosome? Speculate why the answer is not one.</strong></p>
<p>Ratio of single rRNA copy to single chromosome:</p>
<p><span class="math display">\[(1.5\times 10^6)\div(2.0\times10^9)=7.5\times10^{-4}\]</span></p>
<p>The result of (b) should be greater than <span class="math inline">\(7.5\times10^{-4}\)</span>, but it was not, so the calculation would be invalid.</p>
</div>
<div id="ip3-receptor" class="section level1">
<h1>IP<sub>3</sub> Receptor</h1>
<p><strong>An analogue of myo-inositol 1,4,5 trisphosphate (IP<sub>3</sub>) in which sulphur atoms replaced one of the oxygen atoms on each phosphate (IPS<sub>3</sub>) was used to probe membrane bound IP<sub>3</sub> receptors in a membrane fraction of rat cerebellum. Cerebellar membranes (80<span class="math inline">\(\mu\)</span>g protein) were incubated in a volume of 200<span class="math inline">\(\mu\)</span>l with increasing concentrations (0.01-2.5 <span class="math inline">\(\mu\)</span>M) of <sup>35</sup>SIPS3 (7x10<sup>9</sup> dpm/<span class="math inline">\(\mu\)</span>mol). After 30 mins, bound and free ligand were separated by centrifugation, and radioactivity measured in the pellet. To allow for non-specific binding, a parallel run was carried out at each IPS<sub>3</sub> concentration with 0.5mM IP<sub>3</sub> also present. The results obtained were as follows:</strong></p>

<p><strong>(a) Draw the structure of IP<sub>3</sub>, and briefly outline its role as an intracellular messenger.</strong></p>
<p><img src="../img/ip3.jpg" width="100%" /></p>
<p>Phospholipase C (PLC) cleaves PIP<sub>2</sub> (phosphatidylinositol 4, 5-bisphosphate) into IP<sub>3</sub> and DAG (diacylglycerol). This occurs, for example, in GPCR pathway (G<sub>s<span class="math inline">\(\alpha\)</span></sub> activates PLC-<span class="math inline">\(\beta\)</span>) and in tyrosine kinase pathway (the SH2 domain of PLC-<span class="math inline">\(\gamma\)</span> docks it to phosphorylated tyrosines in RTK intracellular domain).</p>
<p>IP<sub>3</sub> binds to its receptors located on ER membrane, which triggers release of Ca<sup>2+</sup> ions. Ca<sup>2+</sup> then activates downstream signalling pathways, such as those mediated by calmodulin.</p>
<p><strong>(b) Determine the dissociation constant for the specific binding of IPS3 to the IP3 receptor.</strong></p>
<p>The ratio of IPS<sub>3</sub>-receptor complex concentration to total receptor concentration under different IPS<sub>3</sub> concentrations can be used to calculate <span class="math inline">\(K_d\)</span>. Here’s the proof:</p>
<p>For the equilibrium: <span class="math inline">\(\text{PL}\leftrightarrow\text{P + L}\)</span>, <span class="math inline">\(K_d = \dfrac{\text{[P][L]}}{\text{[PL]}}\)</span>, i.e. <span class="math inline">\(\text{[PL]}=\dfrac{\text{[P][L]}}{K_d}\)</span></p>
<p>the ratio thus simplifies to:</p>
<p><span class="math display">\[\dfrac{\text{[PL]}}{\text{[P]}_\text{total}}=\dfrac{\text{[PL]}}{\text{[PL]}+\text{[P]}} = \dfrac{\dfrac{\text{[P][L]}}{K_d}}{\dfrac{\text{[P][L]}}{K_d}+\text{[P]}}=\dfrac{\text{[L]}}{\text{[L]}+K_d}\]</span></p>
<p>Taking double reciprocal:</p>
<p><span class="math display">\[\dfrac{1}{\text{[PL]}} = \dfrac{1}{\text{[P]}_\text{total}} + \dfrac{K_d}{\text{[P]}_\text{total}\text{[L]}}\]</span></p>
<p>Plotting <span class="math inline">\(\dfrac{1}{\text{[PL]}}\)</span> against <span class="math inline">\(\dfrac{1}{\text{[L]}}\)</span>, the intercept is <span class="math inline">\(\dfrac{1}{\text{[P]}_\text{total}}\)</span> and the gradient is <span class="math inline">\(\dfrac{K_d}{\text{[P]}_\text{total}}\)</span></p>
<p>The cpm due to specific (real PL) binding is first calculated:</p>
<pre class="r"><code>all_cpm &lt;- c(2770, 5385, 10196, 18587, 31694, 48712, 65930, 83190, 106190)
non_specific &lt;- c(110, 205, 396, 807, 1594, 3212, 6430, 12770, 25690)
(specific &lt;- all_cpm - non_specific) # unit: cpm</code></pre>
<pre><code>## [1]  2660  5180  9800 17780 30100 45500 59500 70420 80500</code></pre>
<p>and the concentrations of PL are:</p>
<pre class="r"><code>(PL &lt;- specific/7e9/200) # in M</code></pre>
<pre><code>## [1] 1.90e-09 3.70e-09 7.00e-09 1.27e-08 2.15e-08 3.25e-08 4.25e-08 5.03e-08
## [9] 5.75e-08</code></pre>
<p>the corresponding concentrations of L are:</p>
<pre class="r"><code>(L &lt;- c(.01, .02, .04, .08, .16, .32, .64, 1.28, 2.56)/1e6) # in M</code></pre>
<pre><code>## [1] 1.00e-08 2.00e-08 4.00e-08 8.00e-08 1.60e-07 3.20e-07 6.40e-07 1.28e-06
## [9] 2.56e-06</code></pre>
<p>Linear modelling of <span class="math inline">\(\dfrac{1}{\text{[PL]}}\)</span> against <span class="math inline">\(\dfrac{1}{\text{[L]}}\)</span>:</p>
<pre class="r"><code>y &lt;- 1/PL; x &lt;- 1/L
(coef &lt;- coef(lm(y~x)))</code></pre>
<pre><code>##  (Intercept)            x 
## 1.516132e+07 5.109047e+00</code></pre>
<pre class="r"><code>plot(1/L, 1/PL)
abline(coef = coef)</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-15-1.png" width="100%" /></p>
<p><span class="math inline">\(\dfrac{1}{\text{[P]}_\text{total}} = 1.516\times10^7\)</span>, <span class="math inline">\(\text{[P]}_\text{total}=6.596\times10^{-8}\text{ M}\)</span></p>
<p><span class="math inline">\(\dfrac{K_d}{\text{[P]}_\text{total}}=5.109\)</span>, <span class="math inline">\(K_d=6.596\times10^{-8}\times5.109=3.370\times10^{-7} \text{ M}\)</span></p>
<p><strong>(c) Determine the density of IP3 receptors in the membrane preparation used.</strong></p>
<p>From (b), <span class="math inline">\(\text{[P]}_\text{total}=6.596\times10^{-8}\text{ M}\)</span></p>
<p>Given mass of total protein is 80<span class="math inline">\(\mu\)</span>g and the volume is 200<span class="math inline">\(\mu\)</span>l, the density is:</p>
<p><span class="math display">\[80\times10^{-6}\div(200\times10^{-6}\times 6.596\times10^{-8})=6.06\times10^6 \text{ Da}\]</span></p>
</div>
<div id="ppiase" class="section level1">
<h1>PP<sub>i</sub>ase</h1>
<p><strong>Cytosolic pyrophosphatase (PP<sub>i</sub>ase) (molecular weight 71,000 Da) was purified from yeast, and the activity of the final preparation measured by following the release of <sup>32</sup>PP<sub>i</sub> from <sup>32</sup>PP<sub>i</sub>. Each reaction tube contained 0.5ml buffer (pH 7.4) and <sup>32</sup>PP<sub>i</sub> (400,000 cpm), together with unlabelled PP<sub>i</sub> to the final concentration shown below. (The concentration of <sup>32</sup>PP<sub>i</sub> can be assumed to be negligible.) The reaction was started by the addition of 2<span class="math inline">\(\mu\)</span>l (0.2ng) enzyme and stopped after 1 min by the addition of trichloroacetic acid. <sup>32</sup>PP<sub>i</sub> was then separated from the remaining <sup>32</sup>PP<sub>i</sub> and measured by scintillation counting.</strong></p>

<p>Take the Lineweaver-Burk equation (double reciprocal of the Michaelis-Mention equation):</p>
<p><span class="math display">\[\dfrac{1}{v}=\dfrac{K_m}{V_{\text{max}}\text{[S]}}+\dfrac{1}{V_{\text{max}}}\]</span></p>
<p>In this equation, <span class="math inline">\(v\)</span> refers to the total rate of hydrolysis of both PP<sub>i</sub> and <sup>32</sup>PP<sub>i</sub>, and [S] is the total concentration of PP<sub>i</sub> and <sup>32</sup>PP<sub>i</sub>.</p>
<p>Let concentration of <sup>32</sup>PP<sub>i</sub> be <span class="math inline">\(c_0\)</span> (which is a constant), the concentration of PP<sub>i</sub> be <span class="math inline">\(c^\text{apparent}\)</span>, the ‘apparent’ rate that is due to the hydrolysis of radioactive <sup>32</sup>PP<sub>i</sub> be <span class="math inline">\(v^\text{apparent}\)</span>, then:</p>
<p><span class="math display">\[\text{[S]}=c_0 + c^\text{apparent}\]</span></p>
<p>and because radioactivity (isotopes) does not affect chemical reactivity, the ratio of the apparent rate to the total rate should simply equal to the ratio of the concentration of radioactive <sup>32</sup>PP<sub>i</sub> to the total concentration of PP<sub>i</sub></p>
<p><span class="math display">\[\dfrac{v^\text{apparent}}{v}= \dfrac{c_0}{c^\text{apparent}+c_0}\]</span></p>
<p>Substituting into the Lineweaver-Burk equation:</p>
<p><span class="math display">\[\dfrac{1}{\left(\dfrac{v^\text{apparent}}{\left(\dfrac{c_0}{c^\text{apparent} + c_0}\right)}\right)}=\dfrac{K_m}{V_{\text{max}}( c^\text{apparent}+c_0)}+\dfrac{1}{V_{\text{max}}}\]</span></p>
<p>Rearranging and simplifying:</p>
<p><span class="math display">\[\dfrac{1}{v^\text{apparent}}=\dfrac{K_m+c_0}{V_{\text{max}}c_0}+\dfrac{c^\text{apparent}}{V_{\text{max}}c_0}\]</span></p>
<p>Obviously, there should be a linear relationship–<span class="math inline">\(\dfrac{1}{v^\text{apparent}}\)</span> against <span class="math inline">\(c^\text{apparent}\)</span>, with a gradient of <span class="math inline">\(\dfrac{1}{V_{\text{max}}c_0}\)</span> and an intercept of <span class="math inline">\(\dfrac{K_m+c_0}{V_{\text{max}}c_0}\)</span></p>
<p>Linear modelling:</p>
<pre class="r"><code>capp &lt;- c(.24, .48, .96, 1.92, 3.84, 7.68, 15.4, 30.7, 61.4, 122.9)
vapp &lt;- c(9274, 9351, 8590, 7392, 5784, 4061, 2502, 1426, 778, 437)
x &lt;- capp; y &lt;- 1/vapp
(coef &lt;- coef(lm(y~x)))</code></pre>
<pre><code>##  (Intercept)            x 
## 1.133102e-04 1.803159e-05</code></pre>
<pre class="r"><code>plot(capp, 1/vapp)
abline(coef = coef)</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-18-1.png" width="100%" /></p>
<p><span class="math inline">\(\dfrac{1}{V_{\text{max}}c_0}=1.803\times10^{-5}\rightarrow V_{\text{max}}c_0=55458\text{ cpm min}^{-1}\mu\text{M}\)</span></p>
<p><span class="math inline">\(\dfrac{K_m+c_0}{V_{\text{max}}c_0}=1.133\times10^{-4}\text{ cpm}^{-1}\text{ min}\)</span>.</p>
<p><span class="math inline">\(K_m+c_0=1.133\times10^{-4}\times 55458=6.283\text{ }\mu\text{M}\)</span></p>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img
              src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>