<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>RNA - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="RNA - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/rna/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">RNA</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="../tutorial/antibiotics-and-protein-synthesis.html" class="archive-item-link">
                Antibiotics and Protein Synthesis 
              </a>
              <span class="archive-item-date">
                2019-11-30
              </span>
              <div class="article-summary" style="display: none;">
                
                  Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.<hr>
                
                


<p><strong>Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Protein synthesis occurs in ribosomes and involves 3 phases: initiation, elongation (which is subdivided into 4 steps) and termination. Each phase depends on the correct conformation and/or catalytic activity of different domains of the ribosomal complex and soluble protein factors (IFs, EFs and RFs). The identities and mechanisms of these critical RNA/protein domains (in bacterial cells) can be investigated by genetic mutation and antibiotics, which hinder their normal functions, along with recent advances in biophysical methods, which allows high resolution structures to be determined for ribosomes in different contexts.</p>
</div>
<div id="background-information" class="section level1">
<h1>Background information</h1>
<div id="the-ribosome" class="section level2">
<h2>The Ribosome</h2>
<p>The bacterial 70S ribosome comprises two subunits, the 50S (large) and the 30S (small) subunit. The 30S subunit mediates selection of cognate aminoacyl tRNAs and the 50S subunit contains the peptidyl-transferase centre (PTC), which catalyses addition of new amino acids to the elongating polypeptide chain.</p>
</div>
<div id="the-process-of-protein-synthesis" class="section level2">
<h2>The process of protein synthesis</h2>
<p>The 3 phases of protein synthesis are initiation, elongation and termination.</p>
<p>During initiation, the 3’ end of the 16S rRNA of the 30S subunit base-pairs with the Shine-Dalgarno sequence, which positions the start codon (usually AUG) in the P site for the binding of the initiator tRNA (usually fMet-tRNA) and IF1, 2, 3. Joining of the 50S subunit and dissociations of IFs primes the elongation phase.</p>
<p>The elongation phase, where new amino acids are appended to the growing polypeptide, involves 4 steps:</p>
<ol style="list-style-type: decimal">
<li><strong>Decoding.</strong> An aminoacylated tRNA (aa-tRNA) is delivered to the A-site of the ribosome by elongation factor EF-Tu complexed with GTP. GTP hydrolysis facilitates discrimination between cognate and non-cognate tRNAs.</li>
<li><strong>Peptidyl transfer.</strong> The peptidyl-transferase centre (PTC) on the 50S subunit catalyses the formation of peptide bond between the amino acids attached to the tRNAs in the A- and P-sites by transferring the polypeptide chain from the P-site tRNA to the aa-tRNA in the A-site.</li>
<li><strong>Translocation.</strong> EF-G catalyses the movement tRNAs from the A and P sites to the P and E sites</li>
<li>the growing polypeptide chain passes through an exit tunnel</li>
</ol>
<p>Finally, when the stop codon (UAA/UAG/UGA) is encountered, it is recognised by release factors (RF1/RF2) that forces hydrolysis of the peptidyl-tRNA bond in the P-site, thus releasing the polypeptide chain from the ribosome.</p>
</div>
</div>
<div id="mutations" class="section level1">
<h1>Mutations</h1>
<p>Mutation in different components of protein synthesis (tRNA, ribosomal proteins, rRNAs) can all lead to decreased fidelity of this process.</p>
<p>The tRNA needs to undergo a structual transition that involves a 30<span class="math inline">\(^\circ\)</span> bend in order to achieve the A/T (decoding) conformation. Mutagenesis studies showed that two mutations in the D-stem, A9C and G24A, promote miscoding and these two bases are critical for distortion of the cognate tRNA that is required for decoding.</p>
<p>Mutagenesis studies also help to identify ribosomal proteins that are critical in the decoding process. The mutations can be classified into two types according to their implication on the speed and accuracy/robustness of translation: <em>ram</em> (ribosome ambiguity) mutations improve speed but has a higher frequency of failure in rejecting non-cognate tRNAs; in contrast, ribosomes with <em>str</em> (stringency) mutations are slower and more accurate, but they reject some cognate aa-tRNAs and thus waste GTP. Most of such mutations are found in the ribosomal proteins S12, S5 and S4, which are close to the decoding site, and these proteins have an important effect on codon/anticodon interactions.</p>
</div>
<div id="antibiotics" class="section level1">
<h1>Antibiotics</h1>
<p>The antibiotic puromycin can be used to demonstrate the existence of A and P sites in the ribosome and the ATP/GTP-independent nature of the peptidyl transfer reaction.</p>
<p>Puromycin (PM) is an aa-tRNA analogue which is composed of an nucleoside and an bound amino acid. Unlike the normal aa-tRNA where the 5’ OH of the nucleoside (adenosine) is linked to the rest of the tRNA, PM has a free 5’ OH. Therefore, PM can fit into the peptidyl transfer centre (PTC) and accept the polypeptide chain in the peptidyl tranfer reaction, but the new peptide with PM attached is released immediately because PM does not have an attached tRNA to anchor it to the ribosome.</p>
<p>The details of the experiment is shown below and in Figure <a href="#fig:puromycin">1</a>:</p>
<ul>
<li>Take a cell free system containing: ribosomes, Mg<sup>2+</sup>-containing buffer, poly-U mRNA, <sup>14</sup>C-Phe-tRNA, ATP/GTP-regenerating system and add varying concentrations of soluble cell extract.</li>
<li>This yields ribosomes carrying polyPhe-tRNA (not free polyPhe or poly-Phe-tRNA unbound to ribosomes). [with prior knowledge, this is because there are no stop codons on the mRNA and hence RFs are not recruited to terminate translation]</li>
<li>the ribosomes are washed to remove soluble proteins and ATP/GTP.</li>
<li>then they are incubated with puromycin (PM)</li>
<li>about 50% of the ribosomes release <sup>14</sup> labelled polyPhe as polyPhe-PM [As shown in Figure <a href="#fig:puromycin">1</a>, the convertion from state 2 to 3 and from state 3 to 1 requires GTP, and the convertion from state 1 to 2 (peptidyl transfer) does not. Therefore, when GTP is removed, protein synthesis is either trapped in state 2 or 3]</li>
<li>without a tRNA, PM cannot bind to the ribosome (at A/P sites) so the compound with polyPhe is released</li>
</ul>
<p>Conclusion: ribosome itself catalyses the formation of the peptide bond without energy supplying molecules (ATP/GTP).</p>
<ul>
<li>now wash the ribosomes to remove the PM</li>
<li>add soluble cell extract and GTP, then more PM</li>
<li>the remaining 50% ribosomes release polyPhe-PM</li>
</ul>
<p>Conclusion:</p>
<ol style="list-style-type: decimal">
<li>ribosomes have a ‘PM-reactive’ site (P site) and a ‘PM-unreactive site’ (A site)</li>
<li>soluble cell extract can translocate from RM-unreactive to PM-reactive site (driven by EF-G)</li>
</ol>
<div class="figure"><span id="fig:puromycin"></span>
<img src="../img/puromycin.jpg" alt="The puromycin experiment." width="100%" />
<p class="caption">
Figure 1: The puromycin experiment.
</p>
</div>
</div>
<div id="summary" class="section level1">
<h1>Summary</h1>
<p>Mutagenesis and ribosome-targeting antibiotics are valuable tools for studying the mechanisms of protein synthesis in the ribosome. These methods are usually aided by cryo-EM and X-ray crystallography to probe into the 3D molecular details.</p>
<p>Mutations in tRNA, rRNA and associated proteins give information on their specific roles and behaviours during protein synthesis. Antibiotics with various (and usually predictable) chemical properties is another way of studying the mechanisms of protein synthesis. Apart from the puromycin (which targets the peptidyl transfer centre) mentioned in this essay, there is a wide range of antibiotics that target different steps of protein synthesis. For example, streptomycins interferes with the delivery of tRNAs to the A-site and neomycin hinders translocation following peptidyl transfer. High resolution crystal structures were obtained for many of these antibiotics in complex with the ribosome (or part of ribosome). These structures are indicative of functionally important sites in the ribosome and they often shed light on the mechanisms.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/rna-splicing.html" class="archive-item-link">
                RNA Splicing 
              </a>
              <span class="archive-item-date">
                2019-11-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What are the roles of small nuclear RNAs (SnRNAs) in pre-mRNA splicing and what are the mechanisms that control alternative splicing?<hr>
                
                


<div id="overview" class="section level1">
<h1>Overview</h1>
<p>RNA splicing by spliceosomes removes introns from nascent pre-mRNA and joins protein-coding exon sequences, which matures the mRNA and makes it ready for translation. By chosing variable splice sites, different mature mRNAs (and hence proteins) can be produced from the same pre-mRNA.</p>
</div>
<div id="structure-of-the-intron" class="section level1">
<h1>Structure of the Intron</h1>
<p>Splice sites (i.e. exon-intron junctions) have conserved features. As shown in Figure <a href="#fig:intron">1</a>, most introns have an invariant GU at the 5’ end (5’ splice site) and AG at the 3’ end (3’ splice site), their flanking sequences are variable but the bases are found at frequencies higher than expected based on a random distribution. The branch point A is also invariant, and most introns also have a polypyrimidine tract between the branch point and the 3’ splice site. These ‘standard’ pre-mRNAs are spliced by snRNPs U1, U2, U4, U5, and U6. A small fraction of pre-mRNA does not follow the ‘GU-AG’ rule and have 5’-AU and AC-3’ instead, which are spliced by four low-abundance snRNPs together with U5.</p>
<div class="figure"><span id="fig:intron"></span>
<img src="../img/intron.jpg" alt="The structure of a typical intron" width="100%" />
<p class="caption">
Figure 1: The structure of a typical intron
</p>
</div>
</div>
<div id="spliceosome-assembly-and-the-splicing-process" class="section level1">
<h1>Spliceosome Assembly and the Splicing Process</h1>
<p>The spliceosome is a large and dynamic RNP machine whose size is comparable to a ribosome. The 5 snRNPs that constitute the spliceosome are U1, U2, U4, U5, and U6, each of which is a association of an snRNA with proteins. The key catalytic activity is attributed to RNA, so the spliceosome is an ribozyme. A spliceosome is assembled whenever an intron is to be removed and its components are recycled after splicing. The series of events are shown in Figure 2.</p>
<div class="figure"><span id="fig:spliceosome-assembly"></span>
<img src="../img/spliceosome-assembly.jpg" alt="Spliceosome assembly and the splicing process" width="100%" />
<p class="caption">
Figure 2: Spliceosome assembly and the splicing process
</p>
</div>
<ol style="list-style-type: decimal">
<li>U1 base-pairs with the 5’ splice site; SF1 (splicing factor 1) binds to the branch-point A; U2AF (U2 auxilliary factor) associates with the polypyrimidine tract and 3’ splice site. Then, U2 replaces SF1 and U2AF.</li>
<li>The U4/U6.U5 tri-snRNP complex joins, forming the spliceosome</li>
<li>Rearrangement of base-paring interactions leads to the catalytically active conformation. U6 dissociates from U4 and replaces U1. U4 is also released.</li>
<li>The catalytic core of U6/U2 catalyses the first transesterification reaction (ligating 5’ guanosine to the 2’ OH of the branch point adenosine)</li>
<li>Further rearrangements take place, and the second transesterification reaction occurs (joining the two exons)</li>
</ol>
<p>The two sequential transesterification reactions <em>per se</em> do not require energy input, as there is no net bond breaking (two phosphodiester bonds are hydrolysed and another two are formed), but ATP/GTP hydrolysis is used to drive structural transitions and as a ‘proofreading’ mechanism.</p>
<p>As shown in Figure <a href="#fig:splice-detailed-mech">3</a>, In the catalitically active state, U6 partially base-pairs with U2 and U6 forms an internal stem loop structure with three nucleotides coordinated to two Mg<sup>2+</sup> ions. Mg<sup>2+</sup> ions stabilise the transitions states of the two transesterification reactions and thus achieve catalysis. The key bases coordinated to Mg<sup>2+</sup> are confirmed by substitution of oxygen for sulfur atoms at various locations.</p>
<div class="figure"><span id="fig:splice-detailed-mech"></span>
<img src="../img/splice-detailed-mech.jpg" alt="The transesterification reactions in detail" width="100%" />
<p class="caption">
Figure 3: The transesterification reactions in detail
</p>
</div>
</div>
<div id="alternative-splicing" class="section level1">
<h1>Alternative Splicing</h1>
<p>pre-mRNAs contain <em>cis</em>-acting regulatory sequences (either splicing enhancers or silencers) to which splicing factors can bind and thus regulate splicing (Figure <a href="#fig:alternative-splicing">4</a>). RNA-RNA base pairing, which can occur either in <em>cis</em> or <em>trans</em>, can also specify splice site use.</p>
<p>RNA-binding proteins/RNPs that regulate splicing can be classified into 3 classes: the classical hnRNPs, serine-arginine repeat (SR) proteins, and tissue-specific RNA-binding proteins. Generally, hnRNPs inhibit splicing from nearby splice sites by interacting with splicing silencers while SR proteins promotes splicing by binding to enhancers.</p>
<div class="figure"><span id="fig:alternative-splicing"></span>
<img src="../img/alternative-splicing.jpg" alt="Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers." width="100%" />
<p class="caption">
Figure 4: Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers.
</p>
</div>
<p>RNA-RNA recognition can also control splice site choice, as exemplified by the exon 6 cluster of the <em>Drosophila</em> DSCAM gene, as shown in Figure <a href="#fig:alternative-splicing-rna-rna">5</a>.</p>
<div class="figure"><span id="fig:alternative-splicing-rna-rna"></span>
<img src="../img/alternative-splicing-rna-rna.jpg" alt="hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream." width="100%" />
<p class="caption">
Figure 5: hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream.
</p>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../%20images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>