<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>X-ray crystallography - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="X-ray crystallography - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../tags/x-ray-crystallography/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">X-ray crystallography</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../tutorial/crystallography-a-case-study.html" class="archive-item-link">
                Crystallography: A Case Study 
              </a>
              <span class="archive-item-date">
                2020-11-04
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>The Ca<sup>2+</sup> selectivity of voltage-gated calcium (Ca<sub>v</sub>) channels remained unclear. They were thought to use a ‘knock-off’ methanism which requires multiple ion-binding cores, but mutational analyses supported a single high-affinity Ca<sup>2+</sup> binding site. This paradox is explained by the mechanism proposed in this article, based on the crystal structures of Ca<sub>v</sub>Ab channels.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<p>In this study, the authors created several mutants of bacterial Na<sub>v</sub> channel Na<sub>v</sub>Ab by site directed mutagenesis using QuickChange at the selectivity filter region<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Some of these mutations changes the activity of the channel from allowing efflux of Na<sup>+</sup> to allowing influx of Ca<sup>2+</sup> to different extents (quantified by the relative permeability of Ca<sup>2+</sup> against Na<sup>+</sup>). Baculovirus were used as the vector to infect <em>Trichopulsia ni</em> cells. Proteins were collected from insect cells, purified, concentrated to ~20mg ml<sup>-1</sup>, and reconstituted into DMPC:CHAPSO bicelles (Figure <a href="#fig:dmpc-chapso">1</a>). Crystals were grown in a hanging-drop vapour-diffusion format.</p>
<div class="figure"><span id="fig:dmpc-chapso"></span>
<img src="../img/dmpc-chapso.png" alt="The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins' natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation." width="100%" />
<p class="caption">
Figure 1: The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins’ natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation.
</p>
</div>
<p>Before X-ray diffraction data collection, crystals were soaked with cryo-protectant solutions containing Ca<sup>2+</sup>, Mn<sup>2+</sup> or Cd<sup>2+</sup> of indicated concentrations. These ions travel through solvent channels within the crystals and chelate with amino acid residues in the selectivity filter region.</p>
<p>X-ray diffraction data were integrated and scaled with the HKL2000 packaged and further processed with the CCP4 package. THe structure of Ca<sub>v</sub>Ab and its derivatives were solved by molecular replacement by using an individual subunit of the Na<sub>v</sub>Ab structure (PDB code 3RVY) as the search template. This choice is natural since the proteins used in this study are derived from Na<sub>v</sub>Ab and differ from it by no more than 3 amino acid residues.</p>
<p>The divalent cations were identified by anomalous difference Fourier maps calculated using data collected at wavelengths of 1.75 Å for Ca<sup>2+</sup>, Cd<sup>2+</sup> and Mn<sup>2+</sup>. Anomalous scattering describes cases where a pair of structure factors <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> does not obey Friedel’s law<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, which occurs when the incident X-ray photons have an energy close to a transition energy of the diffracting atom, resulting in absorption of radiation energy and change in phase (normal scattering does not change the phase). At the wavelengths convinient for diffraction, only atoms heavier than phosphorus or sulfur behave as anomalous scatterers. While anomalous scattering data are frequently used to solve the phase problem (in techniques known as single/multiple wavelength anomalous dispersion (SAD/MAD)), it can also be used (as in this study) to calculate an anomalous difference map after phases are available to show the locations of the heavy atoms.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>Crystallography and NMR System software was used for refinement of coordinates and <em>B</em>-factors. Final models were obtained after several cycle refinement with REFMAC and PHENIX and manual re-building using COOT. The geometries of the final structural models of Ca<sub>v</sub>Ab and its derivatives were verified using PROCHECK.</p>
<p>13 crystal structures of 5 proteins (Na<sub>v</sub>Ab and 4 Ca<sub>v</sub>Ab variants) with the following divalent cation concentrations were determined and deposited into PDB:<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></p>
<ul>
<li>TLESWSM (Na<sub>v</sub>Ab) + Ca<sup>2+</sup> 15mM</li>
<li>(TLE<strong>D</strong>WSM, TLE<strong>D</strong>ES<strong>D</strong>, TL<strong>DD</strong>WSM) + Ca<sup>2+</sup> 15mM</li>
<li>TL<strong>DD</strong>WS<strong>D</strong> + (Ca<sup>2+</sup> 15mM; Mn<sup>2+</sup> 100mM; Cd<sup>2+</sup> 100mM)</li>
<li>TL<strong>DD</strong>WS<strong>N</strong> + Ca<sup>2+</sup> (0.5, 2.5, 5, 10, 15mM)</li>
</ul>
<p>Of the 13 crystal structure models, 4MVR (<sup>175</sup>TLDDWSD<sup>181</sup>) corresponds to the protein with the greatest permeability ratio <em>P</em><sub>Ca</sub>:<em>P</em><sub>Na</sub> , while <a href="https://www.rcsb.org/structure/4MS2">4MS2</a> (<sup>175</sup>TLDDWSN<sup>181</sup>) diffracted to the highest resolution (2.75 Å).</p>
</div>
<div id="analyses-of-results" class="section level1">
<h1>Analyses of Results</h1>
<div id="ca2-binding-sites" class="section level2">
<h2>Ca<sup>2+</sup> Binding Sites</h2>
<p>Using the anomalous diffraction data, the <strong>F</strong><sup>+</sup><sub>Ca</sub> - <strong>F</strong><sup>-</sup><sub>Ca</sub> anomalous difference map was calculated. Two strong peaks followed by a weaker peak were found along the ion-conduction pathway, which correspond to the three Ca<sup>2+</sup> binding sites. They are designated site 1, 2 and 3 from the extracellular side to the intracellular side.</p>
<p>Site 2 is the site with the highest affinity for Ca<sup>2+</sup>. It is surrounded by a total of 8 oxygen atoms, 4 of which coming from the carboxylate of D177 above and the other 4 from the carbonyl of L176 below. Site 1 is coordinated by the plane of 4 carboxyl groups from D178, and site 3 by the plane of 4 carbonyls from T175 (Figure <a href="#fig:cavab-schematic">2</a>). Throughout the selectivity filter, the O-Ca<sup>2+</sup> coordination distances are in the range of 4.0-5.0 Å, which is much longer than the ionic diameter of Ca<sup>2+</sup> (2.28 Å), suggesting that the bound Ca<sup>2+</sup> ion maintains its hydration shell while passing through the pore. Site 3 has the lowest affinity, consistent with its role in exit of Ca<sup>2+</sup> from the selectivity filter into the central cavity. Figure <a href="#fig:coot-4ms2">3</a> shows the molecular model and electron density near the selectivity fileter region of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong> + 15mM Ca<sup>2+</sup>) in COOT<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> (contoured at r.m.s.d = <span class="math inline">\(3.02 \sigma\)</span>).</p>

<div class="figure"><span id="fig:cavab-schematic"></span>
<img src="../img/cavab-schematic.png" alt="A schemetic showing the Ca2+ ion coordination sites of CavAb (TLDDWSD)" width="100%" />
<p class="caption">
Figure 2: A schemetic showing the Ca<sup>2+</sup> ion coordination sites of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>D</strong>)
</p>
</div>
<div class="figure"><span id="fig:coot-4ms2"></span>
<img src="../img/coot-4ms2.png" alt="The selectivity filter region of 4MS2." width="100%" />
<p class="caption">
Figure 3: The selectivity filter region of 4MS2.
</p>
</div>
<p>The relative affinities of the three sites were further confirmed by experiments on Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong>) with varying Ca<sup>2+</sup> concentrations (0.5, 2.5, 5, 10, 15mM): at low Ca<sup>2+</sup> concentration, two strong peaks of approximately equal intensity are found at Site 1 and Site 2; at high concentration the electron density is significantly enhanced in Site 2 and decreased in Site 1. The electron density at Site 3 remains low in all concentrations.</p>
</div>
<div id="ion-permeation-mechanism" class="section level2">
<h2>Ion-Permeation Mechanism</h2>
<p>Based on the properties of the three coordination sites, an ion-permeation mechanism can be deduced. The three coordination sites are separated by a distance of about 4.5 Å, which makes it energetically unfavourable for Ca<sup>2+</sup> to occupy adjacent sites simutaneously. Thus the authors suggested that the selectivity filter oscillates between two states, in which either a single hydrated Ca<sup>2+</sup> occupies Site 2, or two of them each occupies Site 1 and Site 3. The entry of Ca<sup>2+</sup> into Site 1 is promoted by it high extracellular concentration, and the exit of Ca<sup>2+</sup> is facilitated by the low affinity of Site 3.</p>
<p>Notably, this mechanism suggests that Ca<sup>2+</sup> ions are kept hydrated during its passage through the selectivity filter. This is very different from the mechanism by which potassium channels achive selectivity, which requires K<sup>+</sup> ions’ hydration shell to be removed.</p>
</div>
<div id="roles-of-key-selectivity-filter-residues" class="section level2">
<h2>Roles of Key Selectivity Filter Residues</h2>
<p>The five variants of Ca<sub>v</sub>Ab the author produced have different Ca<sup>2+</sup> selectivity ratios, which can be partially explained by directly comparing the difference in the arrangement of selectivity filter residues. This comparison is sometimes facilitated by superposition of one structure onto another. For example, the carboxyl group of D177 in TL<strong>DD</strong>WS<strong>D</strong> interacts with the Ca<sup>2+</sup> ion, while the carboxyl group of E177 in TLE<strong>D</strong>WS<strong>D</strong> swings away from the selectivity filter and forms a hydrogen bond with D181 and the main-chain nitrogen atoms of S180.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>specifically, from <sup>175</sup>TLESWSM<sup>181</sup> to TL<strong>DD</strong>WS<strong>D</strong>, TL<strong>DD</strong>WS<strong>N</strong>, TL<strong>DD</strong>WS<strong>M</strong>, TLE<strong>D</strong>WSM, TLE<strong>D</strong>WS<strong>D</strong> and TL<strong>D</strong>SWSM<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Friedel’s law states that <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> have the same amplitude but opposite phase angle.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>This map is automatically calculated in phenix.refine.<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>namely 4MS2, 4MTO, 4MTF, 4MTG, 4MVU, 4MW8, 4MVZ, 4MVM, 4MVO, 4MVQ, 4MW3, 4MVS, 4MVR<a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>COOT version 0.9.1 (packaged with CCP4), on Linux version 5.8.16-2-MANJARO<a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/assessing-structural-models.html" class="archive-item-link">
                Assessing The Quality Structural Models 
              </a>
              <span class="archive-item-date">
                2020-10-27
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#the-essence-of-x-ray-crystallography"><span class="toc-section-number">1</span> The Essence of X-Ray Crystallography</a></li>
<li><a href="#assessing-the-quality-of-a-structure-during-model-building-and-refinement"><span class="toc-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</a>
<ul>
<li><a href="#r-and-rfree"><span class="toc-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></a></li>
<li><a href="#reci"><span class="toc-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</a></li>
<li><a href="#fit"><span class="toc-section-number">2.3</span> Real-Space Refinement: Map Fitting</a></li>
</ul></li>
<li><a href="#assessing-quality-after-automatic-model-building-and-refinement"><span class="toc-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</a>
<ul>
<li><a href="#density-fit-analysis-and-local-geometry-validation"><span class="toc-section-number">3.1</span> Density fit analysis and local geometry validation</a>
<ul>
<li><a href="#structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles"><span class="toc-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</a></li>
<li><a href="#torsional-angles-and-ramachandran-plot"><span class="toc-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</a></li>
<li><a href="#undefined-regions"><span class="toc-section-number">3.1.3</span> Undefined regions</a></li>
<li><a href="#unexplained-density"><span class="toc-section-number">3.1.4</span> Unexplained Density</a></li>
<li><a href="#distortions-due-to-crystal-packing"><span class="toc-section-number">3.1.5</span> Distortions due to crystal packing</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>A high-quality protein structure model explains the experimental observations well, and conforms to physical and biochemical principles. There are some parameters indicating the quality of a structure model. Some of them are used during model building and refinement, while others apply only on a refined structure.</p>
<div id="the-essence-of-x-ray-crystallography" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Essence of X-Ray Crystallography</h1>
<p>The intensity at <span class="math inline">\((h, k, l)\)</span> in the reciprocal space, i.e. the structure factor amplitudes collected in X-ray experiments is directly related to the Fourier transform of the electron density in the real space (<span class="math inline">\(I = \mathbf{F}_{hkl}^2\)</span>), the latter being calculated as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \int_{x}\int_{y}\int_{z}\rho(x, y, z)e^{2\pi i(hx +ky + lz) }\mathrm{d}x \mathrm{d}y \mathrm{d}z
\end{equation}\]</span></p>
<p>In practice, since the number of atoms and reflections (hence <span class="math inline">\((x, y, z)\)</span> or <span class="math inline">\((h, k, l)\)</span> coordinates is limited, its discrete form, which can be efficiently computed, is used:<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \sum_{j}^{n}f_j e^{2\pi i(hx_j +ky_j + lz_j) }
\end{equation}\]</span></p>
<p>where <span class="math inline">\((x_j, y_j, z_j)\)</span> is the coordinates of the <em>j</em>-th atom in real space.</p>
<p>This is also known as the <em>structure factor equation</em>.</p>
<p>Solving the electron density from structure factors involves solving the inverse Fourier transform of the above equation, also known as the <em>electron density equation</em>, in its discrete form:</p>
<p><span class="math display">\[\begin{equation}
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}w_{hkl}|\mathbf{F}_\text{obs}|e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\alpha^\prime_{hkl}\)</span> is the phase at each set of coordinates in the reciprocal space. The phases are not directly available but are necessary (actually more important than intensities) for computing <span class="math inline">\(\rho(x,y,z)\)</span> (electron density).</p>
<p>The phases can be solved either by phasing experiments, in which case the resolution is relatively low but the data is unbiased, or by molecular replacement (MR), in which case the resolution is relatively high but the data suffer from bias (the phases resemble the search model more than the actual structure). Thus, bias minimisation measures such as maxinum likelihood map coefficients are applied before electron density map contrstruction.</p>
</div>
<div id="assessing-the-quality-of-a-structure-during-model-building-and-refinement" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</h1>
<p>Solving the electron density and ultimately the atomic structure from experimental data is an iterative process, starting from a model with low resolution and imperfect phases and progressively refining the model by applying prior knowledge on the information the current model can provide so far.</p>
<p>In general, the first step in refinement is to construct a crude electron density map (Equation (3)) using phase angles (<span class="math inline">\(\alpha^\prime\)</span>) obtained either from MR or experimental phasing. This initial map, <strong>F</strong><sub>0</sub>, is improved using techniques such as <em>density modification, solvent leveling, solvent flattening</em>. At some critical point in the iterative improvement of phases, the map becomes clear enough that a protein chain can be traced through it. From this point on, real-space refinement is also conducted at each round of iteration. Electron density map is obtained (details in Section <a href="#fit">2.3</a>), and filled with atoms, and transformed back to the reciprocal space. This process repeats until converging to a high-quality strucure. <span class="math inline">\(R\)</span> and <span class="math inline">\(R_\text{free}\)</span>, explained below, are used to assess progress in the refinement, and is also a major factor in evaluating the quality of a model.</p>
<div id="r-and-rfree" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></h2>
<p><em>R</em> is a measure of the deviation of calculated intensities from models (details in Section <a href="#reci">2.2</a>) from the observed intensities in the diffraction pattern, defined by the following equation:</p>
<p><span class="math display">\[
R = \dfrac{\sum ||\mathbf{F}_\text{obs}| - |\mathbf{F}_\text{calc}||}{\mathbf{F}_\text{obs}}
\]</span></p>
<p>Since bias can easily be introduced into the <em>R</em> value (especially by overparameterisation, see Section <a href="#reci">2.2</a>) and a reduction of <em>R</em> value sometimes does not improve the actual quality of structure (<span class="citation">Kleywegt and Brünger (n.d.)</span>), a small fraction (typically around 5%<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>) of randomly selected reflections are removed from the data used for refinement. These reflections can then be used to calculate an R factor, denoted as <span class="math inline">\(R_\text{free}\)</span>, whose reduction can be considered as an <em>unbiased estimate</em> of the improvement of the model.</p>
<div class="figure"><span id="fig:coot-phenix"></span>
<img src="../img/coot-phenix.png" alt=" R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine)." width="100%" />
<p class="caption">
Figure 2.1:  R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine).
</p>
</div>
<p>Figure <a href="#fig:coot-phenix">2.1</a> shows the decrease of <em>R</em> and <em>R</em><sub>free</sub> during a refinement task conducted in Phenix.</p>
</div>
<div id="reci" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</h2>
<p>Reciprocal-space refinement involves computerised attempts to improve agreement between <span class="math inline">\(\mathbf{F}_\text{obs}\)</span> and <span class="math inline">\(\mathbf{F}_\text{calc}\)</span> by <em>without</em> consideration of the maps and models. <em>Refinement by least squares</em> is the earliest successful technique and is discussed here.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>The goal of refinement by least squares is, find <span class="math inline">\((x_j, y_j, z_j)\)</span> for all atom <span class="math inline">\(j\)</span> whose <em>expected</em> ( i.e. computed) structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> are as close as possible to <em>observed</em> structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{obs}|\)</span>. Specifically, this means minimising the function <span class="math inline">\(\Phi\)</span>:</p>
<p><span class="math display">\[\begin{equation}
\Phi = \sum_{hkl}(w_{hkl}\mid\mathbf{F}_\text{obs}\mid - \mid\mathbf{F}_\text{calc}\mid)_{hkl}^{2}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(w_{hkl}\)</span> is the weight term that depends on the reliability of the corresponding measured intensity and <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> is a variant form of Equation (2) that can include additional parameters such as <em>B</em>-factor (<span class="math inline">\(B_j\)</span>) and occupancy <span class="math inline">\(n_j\)</span>. An equation with <span class="math inline">\(B_j\)</span> and <span class="math inline">\(n_j\)</span> included can be written as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_\text{calc} = \sum_{j}n_j f_j e^{2\pi i(hx_j +ky_j + lz_j) - B_j[(\sin\theta)/\lambda]^2}
\end{equation}\]</span></p>
<p>Note that the equatin shows that the effect of <em>B</em>-factors depends on the angle of reflection <span class="math inline">\((\sin\theta)/\lambda\)</span>.</p>
<p>Solving the minimum of <span class="math inline">\(\Phi\)</span> analytically is impractical, and instead numerical methods are used, which would lead to a minimum closest to the starting value.To prevent the refinement converging to a local minimum, it is important that the starting parameters be near the <em>global minimum</em>. Is also important <em>not</em> to include too many parameters (such as <em>B</em>-factor) at the initial stages of resolution when the resolution is low, as this would decrease the radius of convergence (Figure <a href="#fig:radius-convergece">2.2</a>).</p>
<div class="figure"><span id="fig:radius-convergece"></span>
<img src="../img/radius-convergece.png" alt="Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps." width="100%" />
<p class="caption">
Figure 2.2: Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps.
</p>
</div>
<p>To minimise the number of parameters used during early stages of refinement (and thus to increase radius of convergence), individual <span class="math inline">\((x,y,z)\)</span> coordinates are actually not used. Instead, only torional angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are allowed to change, and all bond lengths and angles are fixed to their theoretical average, side chains are assumed to be in their preferred conformation, and peptide linkages are fixed to be planar. This strategy is known as <em>restrained</em> reciprocal space refinement. As refinement proceeds, more parameters, from individual <span class="math inline">\((x,y,z)\)</span> coordinates to isotropic B-factors and finally anisotropic factors, can be added into calculation.</p>
</div>
<div id="fit" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Real-Space Refinement: Map Fitting</h2>
<p><em>Map fitting</em> or <em>model building</em> entails building a molecular model that fits realistically into the current electron density contour map.</p>
<p>To reduce the bias (towards <strong>F</strong><sub>calc</sub> (<strong>F</strong><sub>c</sub>)) when constructing the electron density map, Fourier syntheses of <strong>F</strong><sub>obs</sub> and <strong>F</strong><sub>calc</sub> are used.
A Fourier synthesis <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> is calculated as:</p>
<p><span class="math display">\[
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}(m|\mathbf{F}_\text{o}|-|\mathbf{F}_\text{c}|)e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\]</span></p>
<p>and its corresponding electron density map is called an <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> map.</p>
<p>Simply put, the 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map resembles a molecular surface, and a <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map emphasises the error (positive density implies that the unit cell contains more electron density in this region than implied by the model (<strong>F</strong><sub>c</sub>). Near the end of refinement, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map becomes rather empty except in problem areas, which may need to be corrected manually.</p>
<p>Fitting a molecular model into the electron density map depends on prior knowledge, such as average bond lengths and angles, the amino acid sequence of the protein, properties of peptide chains, etc. For example, we know that carboxyl oxygens in adjacent amino acid residues in a <span class="math inline">\(\beta\)</span>-sheet point in opposite directions. Thus, once a <span class="math inline">\(\beta\)</span>-sheet along with one or two carboxyl oxygen are discernible, we can make a sensible guess of the positions of all other carbonyl oxygens.</p>
</div>
</div>
<div id="assessing-quality-after-automatic-model-building-and-refinement" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</h1>
<div id="density-fit-analysis-and-local-geometry-validation" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Density fit analysis and local geometry validation</h2>
<p>Automatic model building and refinement use the decrease of <em>R</em> value as an indicator of progress and terminates when <em>R</em> is considered to be sufficiently low. This may lead to situations where the global <em>R</em> is favourable but local geometry can still be improved.</p>
<p>Local geometry validation programs, such as “Density fit analysis” in Coot (Figure <a href="#fig:coot-density-fit-2">3.1</a>), evaluate the model geomtry on a per-residue basis and flag outliers. These outliers can then be fixed manually. With the aid of electron density contour maps (where model atoms lie outside 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> contours, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> will often show the atoms with negativel contours, with nearby positive contours pointing to correct locations for these atoms).</p>
<div class="figure"><span id="fig:coot-density-fit-2"></span>
<img src="../img/coot-density-fit-2.png" alt="Use the 'Density fit analysis' function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the 'Rotamers' tool to fix a side chain that's pointing the wrong way." width="100%" />
<p class="caption">
Figure 3.1: Use the ‘Density fit analysis’ function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the ‘Rotamers’ tool to fix a side chain that’s pointing the wrong way.
</p>
</div>
<div id="structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</h3>
<p>During late stages of refinement, restraints on structura paramaters such as bond lengths and angles are release such that thay are allowed to change freely as long as <em>R</em> values could be made to decrease. This may lead to unrealistic models with improbable structural parameters. Thus, they need to be validated. By convention, this is done by calculating the root-mean-square deviations (RMSD) of these parameters in the model from the corresponding set of values based opon the geometry of small organic molecules. These values are recorded in PDB files in <code>REMARK 3</code> fields:</p>
<pre><code>REMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       
REMARK   3                 RMSD          COUNT                                  
REMARK   3   BOND      :  0.003           1366                                  
REMARK   3   ANGLE     :  0.675           1846                                  
REMARK   3   CHIRALITY :  0.050            186                                  
REMARK   3   PLANARITY :  0.005            246                                  
REMARK   3   DIHEDRAL  : 15.473            459                                  </code></pre>
<p>Because bond angles, lengths, chirality and planarity are restrained in early stages of refinement, they are less likely to deviate much from ideal values. Dihedral angles, which is allowed to change throughout the refinement, produce much larger RMSD and are worth closer examination.</p>
</div>
<div id="torsional-angles-and-ramachandran-plot" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</h3>
<p>Torsional (dihedral) angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are show much more variation than bond lengths and angles, but only a subset of all possible (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs are allowed so that adjacent amino acid side chains do not clash. Validation of torsional angle is achieved via a lookup table, where the keys are (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs and values are scores. A (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) is considered preferred or allowed if its score is within certain thresholds. Otherwise, it is considered an outlier. Due to glycine’s small size and proline’s cyclic structure, the preferred/allowed regions of their torsinal angle pairs are defined differently, for example in Phenix/cctbx<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>. Torsional angle validation is often visualised with a Ramachandran plot, as shown in Figure <a href="#fig:coot-ramachandran">3.2</a>.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></p>
<div class="figure"><span id="fig:coot-ramachandran"></span>
<img src="../img/coot-ramachandran.png" alt="Validating dihedral angles with Ramachandran plot in Coot" width="100%" />
<p class="caption">
Figure 3.2: Validating dihedral angles with Ramachandran plot in Coot
</p>
</div>
</div>
<div id="undefined-regions" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Undefined regions</h3>
<p>Often, some regions (often the terminal regions) of the protein are highly disordered and produce no detectable diffraction at high resolution and are thus invisible in the electron-density maps. It is not uncommon to omit these residues in the model. In some cases, however, exact coordinates are given for these atoms, along with very high <em>B</em>-factors (above 50 or 60 angstroms), and one must be careful not to over-interprete these regions.</p>
</div>
<div id="unexplained-density" class="section level3" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> Unexplained Density</h3>
<p>Empty electron density may remain after all known contents of the unit cell have been located. This can either due to an artifect of missing Fourier terms, reagents used in purification or crystallisation, or it could be due to previously unknown ligands (cofactors, inhibitors or allosteric effectors).</p>
</div>
<div id="distortions-due-to-crystal-packing" class="section level3" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Distortions due to crystal packing</h3>
<p>Although flexibility of atoms is usually reflected by the <em>B</em>-factor, in certain situations dynamic regions of a protein molecule can be rigidly fixed in a specific conformation as a result of crystal packing interactions. It should be declared in the PDB header that the protein is affected by crystallisation. One can also check whether these interactions are likely to occur by displaying all neighbouring, symmetry-related molecules in the crystal structure and examine if any intermolecular interactions are present that are a result of crystal packing.</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Blow-2002">
<p>Blow, David. 2002. <em>Outline of Crystallography for Biologists</em>. Oxford University Press.</p>
</div>
<div id="ref-Kleywegt-1996">
<p>Kleywegt, G J, and A T Brünger. n.d. “Checking Your Imagination: Applications of the Free R Value.” <em>Structure</em> 4 (8): 897–904. <a href="https://doi.org/10.1016/s0969-2126(96)00097-4">https://doi.org/10.1016/s0969-2126(96)00097-4</a>.</p>
</div>
<div id="ref-rhodes-2006">
<p>Rhodes, Gale. 2006. <em>Crystallography Made Crystal Clear Clear</em>. Academic Press.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Most crystallography softwares seem to use Fast Fourier Trasnform (FFT) algorithms provided by FFTW (<a href="http://www.fftw.org/" class="uri">http://www.fftw.org/</a>) to compute discrete fourier transforms (and its inverse)<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>In PDB files, this fraction is recorded in the field <code>FREE R VALUE TEST SET SIZE</code><a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>modern refinement softwares use techniques such as simulated annealing and beyesian methods to improve the accuracy of refinement<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>In Phenix/cctbx, a lookup table (actually implemented as an array for better performance) is defined for each of the following cases: glycine, cis-proline, trans-proline, pre-proline, isoleucine/valine, and all other amino acid residues: <a href="https://github.com/cctbx/cctbx_project/blob/master/mmtbx/validation/ramachandran/rama8000_tables.h"><code>mmtbx/validation/ramachandran/rama8000_tables.h</code></a><a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>Recently I’ve been developing a PDB file parser. It is now able to parse coordinate information reliably, which allows Ramanchandran plot analysis: <a href="https://github.com/TianyiShi2001/protein" class="uri">https://github.com/TianyiShi2001/protein</a><a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/electron-microscopy-and-x-ray-crystallography.html" class="archive-item-link">
                Electron Microscopy and X-Ray Crystallography 
              </a>
              <span class="archive-item-date">
                2020-02-13
              </span>
              <div class="article-summary" style="display: none;">
                
                  Learning objectives

To understand the methodological principle of electron microscopy in its different applications (e.g. TEM, SEM, tomography, single-particle), and X-ray crystallography  To be able to evaluate, which method of structural analysis would be most feasible with which kind of biological specimen (e.g. whole cells, viruses, protein complexes, peptides).

To gain a conception of the resolution ranges attainable by EM methods as compared to X-ray crystallography.

To be aware of the computational methods that combine EM maps and crystallogrographic data
<hr>
                
                


<p><strong>Compare the strengths and limitations of Electron Microscopy and X-ray crystallography. Which types of biological specimen can be studied with the respective methods, and how can cryo-EM and X-ray crystallography be combined to obtain novel structural information?</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Being able to solve macromolecular strucutures at atomic resolution, X-ray crystallography has long been the most powerful tool in structural biology. As of Feb 16 2019, 167943 molecular structures have been deposited into the Protein Data Bank (PDB) since its launch in 1971. However, recent improvements in the field of electron microscopy, especially the introduction of the use of cryocooling, have revolutionised the way cellular material is viewed. In conjunction with computational methods, cryo-EM can now solve macromolecular structures at sub-atomic or even atomic resolution. The structures publicised on Electron Microscopy Data Bank (EMDB) has grown exponentially over the last two decades, from only 8 in 2002 to more than 10000 on Feb 16 2020. In this breif narrative, I introduce the basic principles of both methods, their strengths and limitations, and how they can complement each other in modern structural biology studies.</p>
</div>
<div id="x-ray-crystallography" class="section level1">
<h1>X-Ray Crystallography</h1>
<p>In short, structural determination using X-ray crystallography involves making the appropriately-sized crystals of the protein of interest, gaining electron density maps by deciphering X-ray diffraction patterns obtained from these crystals, and filling the map with amino acids (and prosthetic groups or other ligands).</p>
<p>Prior to the X-ray experiment, the protein must be crystallised. This is often the most laborious and time-consuming step, involving numerous trial-and-errors. In modern high throughput methods, a wide range of crystallisation constructs are designed (which involves truncation the protein of interest, expression with a fusion partner, binding with a ligand, use of different detergents, etc.), and robots are used to deploy these crystallisation conditions in multiwell plates and to monitor the growth of crystals. <em>One of the critial factors in the sucess of crystallisation is aqueous solubility, which represents the major bottleneck in solving membrane protein structures <span class="citation">(Parker and Newstead 2016)</span>.</em></p>
<p>In the X-ray experiment, the crytal is mounted on a support which can be rotated in any direction, and is cryocooled in order to minimise radioactive damage and random thermal vibration, thus improving signal-to-noise-ratio (contrast). For every orientation of the crystal, a monochromatic X-ray is applied, and the detector records the position and intensity of the reflections (points) in the diffraction pattern.</p>
<p>Crystallisation is needed for two reasons. First, the diffraction signal given by a single protein molecule is too weak to be detected. Second, incoherent scattering causes serious chemical damage to protein molecules, and if we try to image a single molecule with X-rays, it would be destroyed as soon as it scattered one or two photons and no longer represent the native protein structure. Crystallisation forms an array of a substantial number of protein molecules arranged in the same orientation (so that their diffraction properties are the same) so the radiation damage in a small number of molecules becomes negligible. Every molecule in the array gives the same diffraction signal, thus achieving amplification.</p>
<p>X-rays with wavelengths within the range 0.5-1.6Å are used in crystallography. X-rays above this range are ‘soft X-rays’ that penerate crystals without scattering. Generally, X-rays of shorter wavelengths are more preferable because of two reasons. First, this generally gives better resolution. According to this rearranged form of Bragg’s equation, <span class="math inline">\(\dfrac{n_{max}}{d}\propto\dfrac{1}{\lambda}\)</span>, as wavelength (<span class="math inline">\(\lambda\)</span> decreases), smaller <span class="math inline">\(d\)</span> (distance between crystal lattice planes) can be distinguished with the same <span class="math inline">\(n\)</span> (number of constructive interference). Second, X-rays with shorter <span class="math inline">\(\lambda\)</span> are scattered more, thus producing stronger signals for a fixed amount of sample, which allows crytals of smaller sizes to be studied. However, using X-rays of too short wavelengths (and using too small crystals) have the disadvantage that a greater proportion of proteins would be damaged due to the higher energy of the X-ray and the smaller sample size. Traditional X-rays sources are characteristic radiations from a Cu anode (CuK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=1.54Å) or a Mo anode (MoK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=0.71Å) when bombarded by electrons beams at appropriate potential difference from the cathode. Since the 1960s, synchrotrons (e.g. Diamond Light Source in Oxfordshire) have become available as more intense and X-ray sources. In these giant devices, electrons travel on a circular track in vacuum, emitting intense X-rays in tangential directions. Electrical disturbances in synchrotron allow for production of X-rays with any <span class="math inline">\(\lambda\)</span> within the useful range 0.5-1.6Å, and this makes it possible to use multiple anomalous dispersion (MAD) method to solve the phase problem.</p>
<p>The diffraction patterns only give information on the position amplitude of every reflection, but an additional parameter, phase, is required to do the inverse Fourier transform that would give the electron density map. Single/multiple isomorphous replacements (SIR/MIR) using heavy metals represent the earliest attempts to solve the phase problem. Later, with the advent of synchrotrons, multi-wavelength anomalous dispersion (MAD) became the more popular phasing strategy. Recently, thank to the numerous experimental structures solved previously and publicised in PDB, molecular replacement (MR) has become the most efficient way to solve the phase problem.</p>
<p>After solving the first electron density map using inverse Fourier transform, the structure is refined by optimising the parameters of the model to fit the observations in a iterative process.</p>
<p>Today, abundant software packages are available for data collection and processing, structure solution, refinement and validation.</p>
</div>
<div id="cryo-electron-microscopy" class="section level1">
<h1>Cryo-Electron Microscopy</h1>
<p>Electron microscopy (EM) is mechanistically similar to light microscopy (LM). Electron beams (which is analogous to visible light beams in LM) hit the sample, and their path is regulated by a set of electromagnets (analogous to lens in LM) so that they finally converge onto a plane where an enlarged image of the sample can be detected.</p>
<p>Unstained samples have a very poor singal-to-noise ratio (contrast) under EM, and traditionally heavy metal are applied to improve contrast. However, this often leads to unwanted artefacts and can only achieve resolutions at 20-40Å. High electron dose improves contrast but causes damage to the specimen.</p>
<p>Cryo-EM partially solves this problem by reducing the effect of radiation damage using low temperature. In cryo-electron microscopy, protein solutions are applied onto a support grid, and is then plunge-frozen with liquid ethane. The process is so fast that the water adopts a vitreous form instead of crystallises into ice. The vitrified sample is then maintained at low temperature with liquid nitrogen during storage and EM studies.</p>
<p>Another challenge of cryo-EM is the movement of the particles when being hit by the electron beam, which leads to blurred images on conventional CCD/photographic films (because they are less sensitive and need a long time of exposure). This is solved by the more sensive and faster direct electron detectors which can record movies at a rate of many fps. The traces of molecules recorded in the movies can be computationally processed to give much sharper images. The motion is in part due to the different thermal expansion coefficients between the metal grid and the carbon film, and choosing metals that have similar thermal properties with carbon (e.g. titanium, molybdenum or tungsten instead of copper or gold) can minimise this effect <span class="citation">(Sgro and Costa 2018)</span>.</p>
<p>There are two major strategies of constructing 3D models using cryo-EM, namely electron tomography and single-particle cryo-EM. In electron tomography, the specimen is tilted in all directions and respective EM images are recorded, which are combined (similar to CT) into a 3D model. It is commonly used to visualise one-of-a-kind, structurally heterogeneous entities (such as viruses and whole bacterial cells) at resolutions 50-100Å. Single-particle cryo-EM are usually used to study smaller entities such as the ribosome and proteins. In this approach, a large number (tens or hunders of thousands) of 2D images are extracted from EM images. These heterogeneous low-resolution ‘snapshots’ are computationally sorted and aligned (sometimes called <em>in silico</em> purification), and finally used to synthesise the 3D model using Fourier transform.</p>
</div>
<div id="comparison-of-strengths-and-limitations" class="section level1">
<h1>Comparison of Strengths and Limitations</h1>
<p>The most significant advantage of X-ray crystallography is its resolution. 2.05Å is the median resolution for X-ray crystallographic results in the protein data bank (as of May 19, 2019, according to <a href="https://proteopedia.org/wiki/index.php/Resolution">Proteopedia</a>, but I will do the analysis myself on Monday. I have downloaded all the entire PDB repository.). The protein backbone and most sidechains can be identified unambiguously under this resolution. Strikingly<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>, however, this long unrivalled strength of X-ray crystallography is now challeged by cryo-EM—Wu el al.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> claimed cryo-EM solution of mouse heavy chain apoferritin at 1.75Å!</p>
<p>A major advantage of cryo-EM over X-ray crystallography is the ease and speed of sample preparation, as the proteins need not to be crystallised and only a small amount is needed. Another related merit is its forgiveness of heterogeneity, as robust computational methods can ‘purify’ proteins <em>in silico</em>. Furthermore, cryo-EM are suited for studying membrane proteins and multiprotein supra-assemblies/RNP machines, which are difficult to crystallise in their native states. However, it should be noted that during the process of specimen preparation, delicate protein complexes may become associated so that they no longer represent their <em>in vivo</em> state.</p>
</div>
<div id="combining-x-ray-and-cryo-em-studies" class="section level1">
<h1>Combining X-ray and Cryo-EM studies</h1>
<p>Today, it is common to combine the results of X-ray crystallography and cryo-EM studies for structural determination. There are two major ways in which these two methods can complement each other. First, a low-resolution cryo-EM map can provide an overall shape of the macromolecule, whose sub-components are solved at a high resolution by X-ray crystallograhy and docked onto the EM map. Second, the cryo-EM model may help to solve the phase problem in X-ray crystallography by serving as a search model in molecular replacement (MR).</p>
<p>The docking methods can be classified into two categories: rigid-body docking and flexible docking. Both are used to find the optimal position and orientation of sub-component X-ray structures in the cryo-EM map, but the the latter entails additional algorithms such as normal mode analysis and molecular dynamic simulation that introduce minor conformational changes within stereochemistry limits in X-ray structures to minimise local conformational discrepancy between the X-ray and cryo-EM models. The docking is useful to define the protein location and protein-protein interface within a complex and new interaction modes that are not revealed by X-ray crystallography.</p>
<p>The electron density map (strictly speaking, Coulomb potential density map) obtained by Cryo-EM, albeit at a low resolution, has information about the phase, and thus can be used as an initial phasing model for X-ray studies. Once the search EM map has been positioned, theoretical phases can be calculated by Fourier transform up to the EM model resolution.</p>
</div>
<div id="concluding-remarks" class="section level1">
<h1>Concluding Remarks</h1>
<p>The past decade has witnessed a resolution revolution in single particle cryo-EM, making them another powerful tool in solving biological macromolecules after X-ray crystallography. The different perspectives provided by these two methods is helping us to gain a more complete understanding of molecualr mechanisms that underlie the principle of life.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Cheng-2018">
<p>Cheng, Yifan. 2018. “Single-Particle Cryo-Em—How Did It Get Here and Where Will It Go.” <em>Science</em> 361 (6405). American Association for the Advancement of Science: 876–80. <a href="https://doi.org/10.1126/science.aat4346">https://doi.org/10.1126/science.aat4346</a>.</p>
</div>
<div id="ref-Earl-2017">
<p>Earl, Lesley A, Veronica Falconieri, Jacqueline LS Milne, and Sriram Subramaniam. 2017. “Cryo-Em: Beyond the Microscope.” <em>Current Opinion in Structural Biology</em> 46: 71–78. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002">https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002</a>.</p>
</div>
<div id="ref-Elmlund-2017">
<p>Elmlund, Dominika, Sarah N Le, and Hans Elmlund. 2017. “High-Resolution Cryo-Em: The Nuts and Bolts.” <em>Current Opinion in Structural Biology</em> 46: 1–6. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003">https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003</a>.</p>
</div>
<div id="ref-Jain-2010">
<p>Jain, Deepti, and Valerie Lamour. 2010. “Computational Tools in Protein Crystallography.” <em>Methods in Molecular Biology (Clifton, N.J.)</em> 673. United States: 129–56. <a href="https://doi.org/10.1007/978-1-60761-842-3_8">https://doi.org/10.1007/978-1-60761-842-3_8</a>.</p>
</div>
<div id="ref-Kuhlbrandt-2014">
<p>Kühlbrandt, Werner. 2014. “Microscopy: Cryo-Em Enters a New Era.” <em>eLife</em> 3. eLife Sciences Publications, Ltd: e03678. <a href="https://doi.org/10.7554/eLife.03678">https://doi.org/10.7554/eLife.03678</a>.</p>
</div>
<div id="ref-Milne-2013">
<p>Milne, Jacqueline L S, Mario J Borgnia, Alberto Bartesaghi, Erin E H Tran, Lesley A Earl, David M Schauder, Jeffrey Lengyel, Jason Pierson, Ardan Patwardhan, and Sriram Subramaniam. n.d. “Cryo-Electron Microscopy–a Primer for the Non-Microscopist.” <em>The FEBS Journal</em> 280 (1): 28–45. <a href="https://doi.org/10.1111/febs.12078">https://doi.org/10.1111/febs.12078</a>.</p>
</div>
<div id="ref-Nogales-2018">
<p>Nogales, Eva. 2018. “Cryo-Em.” <em>Current Biology</em> 28 (19): R1127–R1128. <a href="https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016">https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016</a>.</p>
</div>
<div id="ref-Parker-2016">
<p>Parker, Joanne L, and Simon Newstead. 2016. “Membrane Protein Crystallisation: Current Trends and Future Perspectives.” <em>Advances in Experimental Medicine and Biology</em> 922. Springer International Publishing: 61–72. <a href="https://doi.org/10.1007/978-3-319-35072-1_5">https://doi.org/10.1007/978-3-319-35072-1_5</a>.</p>
</div>
<div id="ref-Powell-2017">
<p>Powell, Harold R. n.d. “X-Ray Data Processing.” <em>Bioscience Reports</em> 37 (5). Portland Press Ltd.: BSR20170227. <a href="https://doi.org/10.1042/BSR20170227">https://doi.org/10.1042/BSR20170227</a>.</p>
</div>
<div id="ref-Rhodes-2006a">
<p>Rhodes, Gale. 2006. “An Overview of Protein Crystallography.” In <em>Crystallography Made Crystal Clear</em>, edited by Gale Rhodes, 3rd ed., 7–30. Complementary Science. Burlington: Academic Press. <a href="https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0">https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0</a>.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
<div id="ref-Sgro-2018">
<p>Sgro, Germán G, and Tiago R D Costa. 2018. “Cryo-Em Grid Preparation of Membrane Protein Samples for Single Particle Analysis.” <em>Front Mol Biosci</em> 5. Departamento de Bioquı'mica, Instituto de Quı'mica, Universidade de São Paulo, São Paulo, Brazil.; Department of Life Sciences, Imperial College London, MRC Centre for Molecular Microbiology; Infection, London, United Kingdom.: 74. <a href="https://doi.org/10.3389/fmolb.2018.00074">https://doi.org/10.3389/fmolb.2018.00074</a>.</p>
</div>
<div id="ref-Wang-2017">
<p>Wang, Hong-Wei, and Jia-Wei Wang. 2017. “How Cryo-Electron Microscopy and X-Ray Crystallography Complement Each Other.” <em>Protein Science</em> 26 (1): 32–39. <a href="https://doi.org/10.1002/pro.3022">https://doi.org/10.1002/pro.3022</a>.</p>
</div>
<div id="ref-Whitford-2005a">
<p>Whitford, David. 2005. “Physical Methods of Determining the Three-Dimensional Structure of Proteins.” Book. In <em>Proteins: Structure and Function</em>. Sussex, England: John Wiley &amp; Sons.</p>
</div>
<div id="ref-Xiong-2008">
<p>Xiong, Yong. 2008. “From electron microscopy to X-ray crystallography: molecular-replacement case studies.” <em>Acta Crystallographica Section D</em> 64 (1): 76–82. <a href="https://doi.org/10.1107/S090744490705398X">https://doi.org/10.1107/S090744490705398X</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>I was REALLY shocked when when facing the search results on EMDB website. According to reviews written in 2014, 4.5Å was still the best resolution ever achieved at that time, and in less than 6 years cryo-EM is becoming able to provide atomic resolution as does X-ray crystallography!<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>To be published. Preprint available on <a href="https://www.biorxiv.org/content/10.1101/855643v1.full">bioRxiv</a><a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="../tutorial/protein-structure-and-function.html" class="archive-item-link">
                Protein Structure and Function 
              </a>
              <span class="archive-item-date">
                2019-10-24
              </span>
              <div class="article-summary" style="display: none;">
                
                  Choose 3 structural papers. They can be crystallography, NMR, CryoEM etc. Explain why the protein is important and what the structure tells you about function (why you find it interesting) but most importantly go to the methods section (it may be supplementary information especially if you choose high profile paper) and explain how the protein was purified. The techniques will be those covered in Matt Higgins&#39; lecture series. Bring notes no essay.<hr>
                
                


<div id="principles-of-x-ray-crystallography" class="section level1">
<h1>Principles of X-ray crystallography</h1>
<div id="scattering" class="section level2">
<h2>Scattering</h2>
<ul>
<li>When light hits matter…
<ul>
<li>Vibration<span class="math inline">\(\rightarrow\)</span>scattering (all directions)</li>
<li>Energy level transitions<span class="math inline">\(\rightarrow\)</span>absorption and emission (fluorescence)</li>
<li>Photochemical reactions (e.g. photosynthesis)</li>
</ul></li>
<li>scattering can give rise to refraction and diffraction</li>
<li>experiments on scattering
<ul>
<li>turbidity (reduction in intensity)</li>
<li>angular dependence</li>
<li>changes in <span class="math inline">\(\lambda\)</span></li>
</ul></li>
</ul>
</div>
<div id="wnt3-fz8-complex-hirai-2019" class="section level2">
<h2>Wnt3-Fz8 Complex <span class="citation">(Hirai et al. 2019)</span></h2>
<ul>
<li>crystals of lysine-methylated and deglycosylated human Wnt3 (hWnt3)-mFz8 CRD complex were obtained by X-ray crystal structure was solved by molecular replacement and refined to a resolution of 2.8Å.</li>
</ul>
<div id="difficulties-in-crystallisation-and-their-solutions" class="section level3">
<h3>Difficulties in crystallisation and their solutions</h3>
<ul>
<li>Strong hydrophobic property of Wnt proteins caused by a covalent lipid modification</li>
</ul>
<p>Optimisation and chemical modifications conducted to ensure high expression yields, enhanced solubility and sample homogeneity</p>
<div id="solubility" class="section level4">
<h4>Solubility</h4>
<ul>
<li>Failed attempts on making crystallization constructs
<ul>
<li>afamin can solubilize Wnt proteins; when coexpressed and complexed, Wnt3 and 3a can be purified to homogeneity. However, diffraction-quality crystals could not be obtained after repeated trials.</li>
<li>coexpression with mFz8 CRD after Janda’s success; but found purified Wnt3a-Fz8 CRD complex still remained hydrophobic and required detergents during concentration, which hampered crystallisation</li>
</ul></li>
<li>Successful: N-terminal truncation of Wnts to mimic the cleavage of the N-terminal peptide by a metalloprotease Tiki, which has been reported to reduce the overall hydrophobicity
<ul>
<li>N-terminal 20 residues were removed from hWnt3/mWnt3a constructs</li>
<li>PA-hWnt3(<span class="math inline">\(\Delta\)</span>N)/PA-mWnt3a(<span class="math inline">\(\Delta\)</span>N) coexpressed with mFz8 CRD C-terminally fused with modified human Fc.</li>
<li>confirmed that complexes were fully soluble in aqueous buffer and could be concentrated to &gt; 5 mg/ml without detergents</li>
</ul></li>
</ul>
</div>
<div id="optimisation" class="section level4">
<h4>Optimisation</h4>
<ul>
<li>Initially (following Janda) attached normal (with hinge region) human IgG1 Fc to the C-terminal of mFz8 CRD, intervened by a TEV protease cleavage sequence for the later Fc removal
<ul>
<li>although complex formed with high yield, it could not be cleaved at all; different linker lengths showed no improvements</li>
</ul></li>
<li>decided to use IdeS protease to remove Fc.</li>
</ul>
<div id="refs" class="references">
<div id="ref-Hirai-2019">
<p>Hirai, Hidenori, Kyoko Matoba, Emiko Mihara, Takao Arimori, and Junichi Takagi. 2019. “Crystal Structure of a Mammalian Wnt–Frizzled Complex.” <em>Nature Structural &amp; Molecular Biology</em> 26 (5): 372–79. <a href="https://doi.org/10.1038/s41594-019-0216-z">https://doi.org/10.1038/s41594-019-0216-z</a>.</p>
</div>
</div>
</div>
</div>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>