<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>1-Hilary - Tianyi@Oxford</title>
<meta property="og:title" content="1-Hilary - Tianyi@Oxford">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../term/1-hilary/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi@Oxford" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../lab.html">Lab</a></li>
      
      <li><a href="../notes.html">Notes</a></li>
      
      <li><a href="../special.html">Special</a></li>
      
      <li><a href="../tutorial.html">Tutorial</a></li>
      
      <li><a href="../worksheet.html">Worksheet</a></li>
      
      <li><a href="../menu.html">菜谱</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../lab.html">Lab</a></li>
    
    <li><a href="../notes.html">Notes</a></li>
    
    <li><a href="../special.html">Special</a></li>
    
    <li><a href="../tutorial.html">Tutorial</a></li>
    
    <li><a href="../worksheet.html">Worksheet</a></li>
    
    <li><a href="../menu.html">菜谱</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">1-Hilary</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../lab/immunoassay-in-silico.html" class="archive-item-link">
                Immunoassay (in Silico) 
              </a>
              <span class="archive-item-date">
                2020-03-08
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#aim-of-the-practical"><span class="toc-section-number">1</span> Aim of the Practical</a></li>
<li><a href="#background"><span class="toc-section-number">2</span> Background</a><ul>
<li><a href="#q1">Q1</a></li>
</ul></li>
<li><a href="#introduction-to-radioimmunoassay"><span class="toc-section-number">3</span> Introduction to Radioimmunoassay</a></li>
<li><a href="#radioimmunoassay-simulation-tutorial"><span class="toc-section-number">4</span> Radioimmunoassay Simulation — Tutorial</a><ul>
<li><a href="#a-choosing-a-suitable-concentration-of-ab">(a) Choosing a suitable concentration of Ab</a></li>
<li><a href="#b-testing-effect-of-changing-ab-at-fixed-ag">(b) Testing effect of changing [Ab] at fixed [Ag]</a></li>
<li><a href="#d-testing-precision">(d) Testing precision</a></li>
</ul></li>
<li><a href="#ria-generating-your-own-data"><span class="toc-section-number">5</span> RIA: Generating Your Own Data</a><ul>
<li><a href="#a-finding-a-suitable-concentration-of-antibody">(a) Finding a suitable concentration of antibody</a></li>
<li><a href="#b-generating-the-calibartion-curve-and-c-finding-the-concentration-of-the-unknown">(b) Generating the calibartion curve and (c) Finding the concentration of the unknown</a></li>
</ul></li>
<li><a href="#enzyme-linked-immunosorbentassay"><span class="toc-section-number">6</span> Enzyme-Linked Immunosorbentassay</a><ul>
<li><a href="#a-background-to-elisa">(a) Background to ELISA</a></li>
<li><a href="#b-method-elisalab">(b) METHOD: ElisaLab</a></li>
</ul></li>
<li><a href="#etut"><span class="toc-section-number">7</span> ELISA Tutorial</a></li>
<li><a href="#elisa-experiment"><span class="toc-section-number">8</span> ELISA Experiment</a><ul>
<li><a href="#a-choosing-a-suitable-concentration-of-antibody">(a) Choosing a suitable concentration of antibody</a></li>
<li><a href="#b-generating-the-calibration-curve">(b) Generating the calibration curve</a></li>
<li><a href="#c-measuring-concentration-of-the-unknown">(c) Measuring concentration of the unknown</a></li>
</ul></li>
<li><a href="#supplementary-information"><span class="toc-section-number">9</span> Supplementary Information</a><ul>
<li><a href="#ria-supp"><span class="toc-section-number">9.1</span> RIA</a></li>
<li><a href="#elisa-absorbance"><span class="toc-section-number">9.2</span> ELISA</a></li>
</ul></li>
</ul>
</div>

<div id="aim-of-the-practical" class="section level1">
<h1><span class="header-section-number">1</span> Aim of the Practical</h1>
</div>
<div id="background" class="section level1">
<h1><span class="header-section-number">2</span> Background</h1>
<div id="q1" class="section level2 unnumbered">
<h2>Q1</h2>
<p>Draw a simple schematic diagram of an IgG molecule, labelling the relevant chains and domains.</p>
<p>Diagram see Figure <a href="#fig:ab-schematic">2.1</a></p>
<div class="figure"><span id="fig:ab-schematic"></span>
<img src="../img/ab-schematic.jpg" alt="Schematic of an IgG molecule." width="100%" />
<p class="caption">
Figure 2.1: Schematic of an IgG molecule.
</p>
</div>
<ol style="list-style-type: lower-roman">
<li>What is meant by a polyclonal antibody?
<ul>
<li>A mixture of antibodies with diifferent F<sub>v</sub> regions (antigen-binding sites) that are complementary to different epitopes (on the same antigen)</li>
</ul></li>
<li>What is a monoclonal antibody?
<ul>
<li>A group of clone of a single antibody, with identical F<sub>v</sub> region and bind to the same epitope.</li>
</ul></li>
<li>Detail the steps by which mouse monoclonal antibodies are raised.
<ul>
<li>When the antigen molecules are injected (probably with an adjunvant e.g. Aluminium salt to elicit immune response more effectively), they activate antigen-presenting cells (APCs) called dendritic cells. Activated dendritic cells then use their MHC class II to present antigens on their plasma membrane, while migrating to the lymphatic system. When they encounter B cells and T<sub>h</sub> (T helper) cells with cognate BCR/TCRs in lymph nodes, they activate these B and T cells. With the help from activated dendritic cells and T<sub>h</sub> cells, B cells undergoes monoclonal expansion, class switching and somatic hypermutation, and differentiate into plasma cells, which produce a large number of antibodies in a short period.</li>
</ul></li>
</ol>
</div>
</div>
<div id="introduction-to-radioimmunoassay" class="section level1">
<h1><span class="header-section-number">3</span> Introduction to Radioimmunoassay</h1>
<div id="q2" class="section level3 unnumbered">
<h3>Q2</h3>
<p><strong>18 tubes are set up for radioimmunoassay containing antibody with a capacity for binding 1µg antigen. Both labelled antigen and a known concentration of unlabelled antigen are added to the tube as indicated. Note that the labelled antigen has been conjugated with 125I so that approx 50,000 cpm are associated with every <span class="math inline">\(\mu\text{g}\)</span> of labelled antigen). Fill in the table below giving values rounded to the nearest integer.</strong></p>
<table>
<caption>Horizontal axis: unlabelled antigen (<span class="math inline">\(\mu\text{g}\)</span>); vertical axis: labelled antigen (<span class="math inline">\(\mu\text{g}\)</span>);</caption>
<thead>
<tr class="header">
<th align="right"></th>
<th align="right">0</th>
<th align="right">0.5</th>
<th align="right">1</th>
<th align="right">2</th>
<th align="right">4</th>
<th align="right">8</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right"><strong>0.5</strong></td>
<td align="right"><em>25000</em></td>
<td align="right">25000</td>
<td align="right">16667</td>
<td align="right">10000</td>
<td align="right">5556</td>
<td align="right">2941</td>
</tr>
<tr class="even">
<td align="right"><strong>1.0</strong></td>
<td align="right">50000</td>
<td align="right">33333</td>
<td align="right">25000</td>
<td align="right">16666</td>
<td align="right">10000</td>
<td align="right">5556</td>
</tr>
<tr class="odd">
<td align="right"><strong>2.0</strong></td>
<td align="right">50000</td>
<td align="right">40000</td>
<td align="right">33333</td>
<td align="right">25000</td>
<td align="right">16667</td>
<td align="right">10000</td>
</tr>
</tbody>
</table>
<p>When the antibodies are saturated with antigens (when the amount of total antigen is greater than <span class="math inline">\(1 \mu\text{g}\)</span>), the radioactivity is the proportion of labelled antigen multiplied by the maximal radioactivity (50000 cpm). There is one case (0 <span class="math inline">\(\mu\text{g}\)</span> unlabelled; 0.5 <span class="math inline">\(\mu\text{g}\)</span> labelled) where the antibodies are not saturated.</p>
</div>
<div id="q3" class="section level3 unnumbered">
<h3>Q3</h3>
<p><strong>Use the data from Q2 to draw curves of <em>counts precipitated</em> versus <em>amount of unlabelled antigen</em> added at each concentration of labelled antigen.</strong></p>
<p>There are two ways to plot it. The obvious way is to directly plot the radioactivity count against the amount of unlabelled antigen (Figure <a href="#fig:q2">3.1</a> <strong>A</strong>). However, by simply taking the reciprocal of the radioactivity count, a linear relationship can be obtained (Figure <a href="#fig:q2">3.1</a> <strong>B</strong>). This is explained in more detail in Q4.</p>
<pre class="r"><code># Loading data
d &lt;- tibble(labelled = rep(c(&#39;0.5 microgram&#39;, &#39;1 microgram&#39;, &#39;2 microgram&#39;), each = 6),
       unlabelled = rep(c(0, 0.5, 1, 2, 4, 8), 3),
       count = c(25000, 25000, 16667, 10000, 5556, 2941,
                 50000, 33333, 25000, 16666, 10000, 5556,
                 50000, 40000, 33333, 25000, 16667, 10000))
# Direct plot
direct &lt;- ggplot(d, aes(unlabelled, count, linetype = labelled))+
  geom_line()+
  labs(title=&#39;Direct Plotting&#39;)
# Plot after transformation
linear &lt;- ggplot(d, aes(unlabelled, 10000/count, linetype = labelled))+
  geom_line()+
  labs(title=&quot;Tranformed to Linear Relationship&quot;)
# Arrangement
ggarrange(direct, linear, ncol=2, nrow=1, common.legend = TRUE, legend=&quot;bottom&quot;, labels = c(&quot;A&quot;, &quot;B&quot;))</code></pre>
<div class="figure"><span id="fig:q2"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/q2-1.png" alt="A: direct plotting; B: plotting the reciprocal of radioactivity count against unlabelled antigen amount." width="100%" />
<p class="caption">
Figure 3.1: A: direct plotting; B: plotting the reciprocal of radioactivity count against unlabelled antigen amount.
</p>
</div>
</div>
<div id="q4" class="section level3 unnumbered">
<h3>Q4</h3>
<p><strong>Explain why these curves have a negative slope.</strong></p>
<p>When antibodies are saturated with antigens, this simple relationship holds:</p>
<p><span class="math display">\[y=\dfrac{k}{x+k}\times A\]</span></p>
<p>where <span class="math inline">\(k\)</span> is the amount of labelled antigen (constant), <span class="math inline">\(x\)</span> is the amount of unlabelled antigen, <span class="math inline">\(A\)</span> is the maximal radioactivity count (when <span class="math inline">\(x=0\)</span> and all antibodies are bound to labelled antigens). In other words, the proportion of labelled antigen (relative to the total amount of antigen that saturates the antibodies) equals the ratio between the actual radioactivity and the maximal radioactivity. Thus, according to this equation, plotting radioactivity against amount of unlabelled antigen will result in a negative slope.</p>
<p>Taking the reciprocal results in a linear relationship between <span class="math inline">\(\dfrac{1}{y}\)</span> and <span class="math inline">\(x\)</span>, and this was used to plot Q3 and later the calibration curve.</p>
<p><span class="math display">\[\dfrac{1}{y}=\dfrac{x}{Ak} + A\]</span></p>
</div>
<div id="q5" class="section level3 unnumbered">
<h3>Q5</h3>
<p><strong>From the data in Q3 and Q4 and your graph, identify the range of counts precipitated that would be useful for calibration (i.e. that will give you a significant change in counts precipitated when you add antigen).</strong></p>
<p>The slope of the curve is steeper (i.e. it’s more sensitive to small change in unlabelled antigen amount) as the radioactivity count is closer to maximum (50000). However, when the count is too close to 50000, the amount of unlabelled antigen would be too low, which would be more error-prone.</p>
<p>Thus my answer is: 5000-20000 cpm.</p>
</div>
<div id="q6" class="section level3 unnumbered">
<h3>Q6</h3>
<p><strong>How does this vary with [Ag] labelled and [Ag] unlabelled?</strong></p>
<p>A decrease in [Ag]<sub>labelled</sub> increases the sensitivity of RIA for detecting [Ag]<sub>unlabelled</sub>, but the amount of labelled antigen must be at least as much as the capacity of the antibodies (in this case, 1 microgram), otherwise it would be impossible to detect very small amount (precisely, capacity of antibodies minus amount of labelled antigen) of unlabelled antigen.</p>
<p>As [Ag]<sub>unlabelled</sub> increases, the change in cpm becomes less significant.</p>
<p><strong>What does this tell you about the dynamic range of the assay (i.e. the range of amounts of unlabelled antigen you can measure) in these particular experiments?</strong></p>
<p>The dynamic range varies with the amount of labelled antigen. Greater [Ag]<sub>labelled</sub> allows measurement of larger [Ag]<sub>unlabelled</sub> (but has less absolute precision/sensitivity); smaller [Ag]<sub>labelled</sub> allows more precise measurement of low concentrations of unlabelled antigen, but becomes insensitive when [Ag]<sub>unlabelled</sub> is high. The amount of labelled antigen must be at least as much as the capacity of the antibodies, as noted above.</p>
</div>
</div>
<div id="radioimmunoassay-simulation-tutorial" class="section level1">
<h1><span class="header-section-number">4</span> Radioimmunoassay Simulation — Tutorial</h1>
<div id="a-choosing-a-suitable-concentration-of-ab" class="section level2 unnumbered">
<h2>(a) Choosing a suitable concentration of Ab</h2>
<p><strong>7. Enter your results in the table below, using scientific notation rather than strings of zeros.</strong></p>
<table>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag dil</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ab dil (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ab vol</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>6</sup></td>
<td align="right">1</td>
<td align="right">489</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right">5525</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">44899</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">51875</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>2</sup></td>
<td align="right">1</td>
<td align="right">48750</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>1</sup></td>
<td align="right">1</td>
<td align="right">51700</td>
</tr>
</tbody>
</table>
<p><strong>14. Fill out the table below using own your own data from the results screen, making sure you use scientific notation (ie NOT like the data table on screen). Calculate how much Ag you have added.</strong></p>
<table>
<colgroup>
<col width="4%" />
<col width="6%" />
<col width="25%" />
<col width="19%" />
<col width="25%" />
<col width="6%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag dil</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Amount of Ag (pg)</th>
<th align="right">Ab dil (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ab vol</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">1000</td>
<td align="right">10<sup>6</sup></td>
<td align="right">1</td>
<td align="right"><strong>4606</strong></td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">7</td>
<td align="right">700</td>
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right"><strong>5178</strong></td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">10<sup>4</sup></td>
<td align="right">5</td>
<td align="right">700</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right"><strong>9203</strong></td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">10<sup>4</sup></td>
<td align="right">0</td>
<td align="right">100</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right"><strong>25687</strong></td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="right">10<sup>5</sup></td>
<td align="right">0</td>
<td align="right">10</td>
<td align="right">10<sup>2</sup></td>
<td align="right">1</td>
<td align="right"><strong>51613</strong></td>
</tr>
</tbody>
</table>
<p><strong>Unique Code: txt9</strong></p>
</div>
<div id="b-testing-effect-of-changing-ab-at-fixed-ag" class="section level2 unnumbered">
<h2>(b) Testing effect of changing [Ab] at fixed [Ag]</h2>
<p><strong>2. Increase the Ab concentration 10x (ie decrease the dilution factor to 103) and test against three concentrations of Ag (diluted 10<sup>3</sup>, 10<sup>4</sup> and 10<sup>5</sup>) (Keep volume of Ab and Ag at 1 <span class="math inline">\(\mu\text{l}\)</span> for now). Click Experiment and 1mg/ml to generate data. Enter the results.</strong></p>
<table>
<colgroup>
<col width="4%" />
<col width="27%" />
<col width="9%" />
<col width="14%" />
<col width="9%" />
<col width="27%" />
<col width="6%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ag dil’n</th>
<th align="right">Ag mass (pg)</th>
<th align="right">Ab dil’n</th>
<th align="right">Ab vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">1</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1000</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">48772</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">1</td>
<td align="right">10<sup>4</sup></td>
<td align="right">100</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">55150</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">1</td>
<td align="right">10<sup>5</sup></td>
<td align="right">10</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">43975</td>
</tr>
</tbody>
</table>
<p><strong>3. Now perform the same experiment but with the ‘optimal’ Ab dilution of 10<sup>4</sup>.</strong></p>
<table>
<colgroup>
<col width="4%" />
<col width="27%" />
<col width="9%" />
<col width="14%" />
<col width="9%" />
<col width="27%" />
<col width="6%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ag dil’n</th>
<th align="right">Ag mass (pg)</th>
<th align="right">Ab dil’n</th>
<th align="right">Ab vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">1</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1000</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">4122</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">1</td>
<td align="right">10<sup>4</sup></td>
<td align="right">100</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">22237</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">1</td>
<td align="right">10<sup>5</sup></td>
<td align="right">10</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">44681</td>
</tr>
</tbody>
</table>
<p><strong>4. Finally, set up three tubes with 10x LESS Ab than the ‘optimal’ i.e. diluted 10<sup>5</sup>. Again, note your data:</strong></p>
<table>
<colgroup>
<col width="5%" />
<col width="27%" />
<col width="10%" />
<col width="15%" />
<col width="10%" />
<col width="27%" />
<col width="5%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ag dil’n</th>
<th align="right">Ag mass (pg)</th>
<th align="right">Ab dil’n</th>
<th align="right">Ab vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">1</td>
<td align="right">10<sup>3</sup></td>
<td align="right">1000</td>
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right">402</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">1</td>
<td align="right">10<sup>4</sup></td>
<td align="right">100</td>
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right">2366</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">1</td>
<td align="right">10<sup>5</sup></td>
<td align="right">10</td>
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right">4872</td>
</tr>
</tbody>
</table>
<div id="q6-1" class="section level3 unnumbered">
<h3>Q6</h3>
<p><strong>How does the amount of antibody influence the number of counts precipitated?</strong></p>
<p>The number of counts increases with increasing antibody concentration until the antigen-binding capacity of antibodies exceed the amount of labelled antigen (when Ab dilution is 10<sup>3</sup>). In this case, additional unlabelled antigens will just bind free antibodies without interfering with those bound to labelled antigens.</p>
</div>
<div id="q7" class="section level3 unnumbered">
<h3>Q7</h3>
<p>See Figure <a href="#fig:immunolab-q7">4.1</a>.</p>
<div class="figure"><span id="fig:immunolab-q7"></span>
<img src="../img/immunolab-q7.jpg" alt="Q7" width="100%" />
<p class="caption">
Figure 4.1: Q7
</p>
</div>
</div>
</div>
<div id="d-testing-precision" class="section level2 unnumbered">
<h2>(d) Testing precision</h2>
<table>
<colgroup>
<col width="4%" />
<col width="27%" />
<col width="9%" />
<col width="14%" />
<col width="9%" />
<col width="27%" />
<col width="6%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ag dil’n</th>
<th align="right">Ag mass (pg)</th>
<th align="right">Ab dil’n</th>
<th align="right">Ab vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">19215</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">14307</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">14866</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">14474</td>
</tr>
<tr class="odd">
<td align="right">5</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">14607</td>
</tr>
<tr class="even">
<td align="right">6</td>
<td align="right">2</td>
<td align="right">10<sup>4</sup></td>
<td align="right">200</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">15341</td>
</tr>
<tr class="odd">
<td align="right">7</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">51798</td>
</tr>
<tr class="even">
<td align="right">8</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">45749</td>
</tr>
<tr class="odd">
<td align="right">9</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">52873</td>
</tr>
<tr class="even">
<td align="right">10</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">55798</td>
</tr>
<tr class="odd">
<td align="right">11</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">57048</td>
</tr>
<tr class="even">
<td align="right">12</td>
<td align="right">0</td>
<td align="right">-</td>
<td align="right">0</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">49474</td>
</tr>
</tbody>
</table>
<div id="q8" class="section level3 unnumbered">
<h3>Q8</h3>
<p>See Figure <a href="#fig:immunolab-q8">4.2</a></p>
<div class="figure"><span id="fig:immunolab-q8"></span>
<img src="../img/immunolab-q8.jpg" alt="Q8" width="100%" />
<p class="caption">
Figure 4.2: Q8
</p>
</div>
</div>
<div id="q9" class="section level3 unnumbered">
<h3>Q9</h3>
<p>Calculations see Figure <a href="#fig:immunolab-q9">4.3</a></p>
<div class="figure"><span id="fig:immunolab-q9"></span>
<img src="../img/immunolab-q9.jpg" alt="Q9" width="100%" />
<p class="caption">
Figure 4.3: Q9
</p>
</div>
<p><strong>Explain the possible causes of the variation, bearing in mind the experiments have been carried out by an experienced operator</strong></p>
<ul>
<li>Radioactive decay is a probablistic event, so the number of disintegrations per unit time naturally varies.</li>
<li>There can also be random errors intrintic to the Geiger counter</li>
</ul>
</div>
</div>
</div>
<div id="ria-generating-your-own-data" class="section level1">
<h1><span class="header-section-number">5</span> RIA: Generating Your Own Data</h1>
<p><strong>Unique Code:</strong> <code>yckmq9fe</code></p>
<div id="a-finding-a-suitable-concentration-of-antibody" class="section level2 unnumbered">
<h2>(a) Finding a suitable concentration of antibody</h2>
<p><strong>4. Using the grid below…</strong></p>
<p>See Figure <a href="#fig:immunolab-5a4">5.1</a></p>
<div class="figure"><span id="fig:immunolab-5a4"></span>
<img src="../img/immunolab-5a4.jpg" alt="5 (a) 4." width="100%" />
<p class="caption">
Figure 5.1: 5 (a) 4.
</p>
</div>
<p><strong>6. Note down useful data in the tables below</strong></p>
<table>
<colgroup>
<col width="4%" />
<col width="27%" />
<col width="9%" />
<col width="14%" />
<col width="9%" />
<col width="27%" />
<col width="6%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Tube</th>
<th align="right">Ag vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">Ag dil’n</th>
<th align="right">Ag mass (pg)</th>
<th align="right">Ab dil’n</th>
<th align="right">Ab vol (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">1</td>
<td align="right">1</td>
<td align="right">10<sup>2</sup></td>
<td align="right">10<sup>4</sup></td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">346</td>
</tr>
<tr class="even">
<td align="right">2</td>
<td align="right">1</td>
<td align="right">10<sup>3</sup></td>
<td align="right">10<sup>3</sup></td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">4083</td>
</tr>
<tr class="odd">
<td align="right">3</td>
<td align="right">1</td>
<td align="right">10<sup>4</sup></td>
<td align="right">10<sup>2</sup></td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">18380</td>
</tr>
<tr class="even">
<td align="right">4</td>
<td align="right">1</td>
<td align="right">10<sup>5</sup></td>
<td align="right">10</td>
<td align="right">10<sup>4</sup></td>
<td align="right">1</td>
<td align="right">35217</td>
</tr>
</tbody>
</table>
<p><strong>8. If necessary, try varying the volume at given Ab dilution to see if you can optimise further.</strong></p>
<p>I decided to use the 10<sup>4</sup> dilution.</p>
</div>
<div id="b-generating-the-calibartion-curve-and-c-finding-the-concentration-of-the-unknown" class="section level2 unnumbered">
<h2>(b) Generating the calibartion curve and (c) Finding the concentration of the unknown</h2>
<p><strong>5. Note all the relevant data in the table below</strong></p>
<p>Implicitly shown in Figure <a href="#fig:ria-cali-raw">5.2</a> and <a href="#fig:ria-cali">5.3</a>.</p>
<p><strong>Q10: How many replicates would be appropriate for your unknown solution? Explain your answer using sound statistical reasoning.</strong></p>
<p>This depends on the precision we want to achieve, the <em>p</em> value we adopt, and the variance of the results of the repeats of that particular piece of experiment (which is unknown before we have done them). For example, if we choose <span class="math inline">\(p=0.05\)</span> as the threshold, and did some experiments resulting in a mean of <span class="math inline">\(\bar{x}\)</span> and standard deviation of <span class="math inline">\(s_x\)</span>, the confidence interval of the population mean, <span class="math inline">\(\mu\)</span>, will be:</p>
<p><span class="math display">\[\mu = \bar{x} \pm t_{0.975, n} \times \dfrac{s_x}{\sqrt{n}}\]</span></p>
<p>Clearly, in order to calculate <span class="math inline">\(n\)</span>, it should be specified clearly to what precision we want to achieve for <span class="math inline">\(\mu\)</span>, and calculate <span class="math inline">\(\mu\)</span> each time we finish one repeat of the experiment (because <span class="math inline">\(\bar{x}\)</span> and <span class="math inline">\(s_x\)</span> will update).</p>
<p><strong>You may have to test a range of Ag dilutions before you obtain a cpm value that can be read of your calibration curve (remember to find appropriate range first by altering only the dilution). Show your preliminary data here:</strong></p>
<table>
<thead>
<tr class="header">
<th align="right">Unknown Ag dilution</th>
<th align="right">Unknown Ag volume (<span class="math inline">\(\mu\text{l}\)</span>)</th>
<th align="right">cpm</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">10<sup>2</sup></td>
<td align="right">1</td>
<td align="right">454</td>
</tr>
<tr class="even">
<td align="right">10<sup>3</sup></td>
<td align="right">5</td>
<td align="right">826</td>
</tr>
<tr class="odd">
<td align="right">10<sup>3</sup></td>
<td align="right">1</td>
<td align="right">3857</td>
</tr>
<tr class="even">
<td align="right">10<sup>4</sup></td>
<td align="right">5</td>
<td align="right">8245</td>
</tr>
<tr class="odd">
<td align="right"><strong>10<sup>4</sup></strong></td>
<td align="right"><strong>1</strong></td>
<td align="right"><strong>19760</strong></td>
</tr>
<tr class="even">
<td align="right">10<sup>5</sup></td>
<td align="right">5</td>
<td align="right">29922</td>
</tr>
<tr class="odd">
<td align="right">10<sup>5</sup></td>
<td align="right">1</td>
<td align="right">33886</td>
</tr>
</tbody>
</table>
<p>I decide to use Ag dilution of 10<sup>4</sup> and volume of 1 <span class="math inline">\(\mu\text{l}\)</span>, because 19760 is within the optimal range of the calibration curve (shown later, as a response to Q11)</p>
<p><strong>Q11: Read the data off your calibration curve, and calculate the concentration of antigen in your unknown solution. (Each successful measurement will generate a value for the amount of antigen estimated to be present in that particular assay tube. You are required to calculate the concentration of antigen in the unknown solution). Explain your result below. Provide a discussion of your experimental approach, the validity of your answer (conduct appropriate statistical tests) and discuss any aspects that could have been improved. How would you express the precision of your answer (see Q.8)?</strong></p>
<p>Here I load the data (and standardised variable names):</p>
<pre class="r"><code>library(tidyverse)
raw &lt;- read_tsv(&#39;./2020-03-08-immunoassay-in-silico/ria.tsv&#39;) %&gt;% 
  transmute(antigen_volume = `Ag volume`,
            antigen_dilution = `Ag dilution`,
            antibody_volume = `Ab volume`,
            antibody_dilution = factor(`Ab dilution`),
            count = Count,
            known = `Known?`)</code></pre>
<div id="calibration-curve" class="section level3 unnumbered">
<h3>Calibration Curve</h3>
<p>I extracted experiments with known amount of antigen and calculated the mass of antigen in pg:</p>
<pre class="r"><code>known &lt;- raw %&gt;%
  filter(known == 1, # experiments with known antigen amount for calibration
         antibody_dilution == 10000) %&gt;%
  mutate(antigen_amount = 
        antigen_volume / 1000 # \mu L to ml
                       * 10^9  # mg to pg
                       / antigen_dilution) %&gt;%
  filter(antigen_amount&lt;=1000)</code></pre>
<p>First, I plotted count vs antigen amount directly (the blue line is fitted using the LOESS algorithm) in Figure <a href="#fig:ria-cali-raw">5.2</a> . Although this plot is not the most useful for calibration, it shows that the random variation is large when the count is greater than 20000. Therefore I aim at a count around 20000 in the test for the unkown antigen concentration.</p>
<pre class="r"><code>known %&gt;%
  ggplot(aes(antigen_amount, count))+
  geom_point()+
  geom_smooth(span=0.9, se=FALSE)+
  labs(x = &#39;antigen amount (pg)&#39;, title = &#39;Calibration Curve (cpm vs antigen amount)&#39;)+
  theme_bw()</code></pre>
<div class="figure"><span id="fig:ria-cali-raw"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/ria-cali-raw-1.png" alt="RIA Calibration Curve" width="672" />
<p class="caption">
Figure 5.2: RIA Calibration Curve
</p>
</div>
<p>The real calibration curve is shown in Figure <a href="#fig:ria-cali">5.3</a> (The reasoning was described in Q3 and Q4).</p>
<pre class="r"><code>known %&gt;% 
  ggplot(aes(antigen_amount, 10000/count))+
  geom_point()+
  geom_smooth(span=0.9)+
  labs(x = &#39;antigen amount (pg)&#39;, title = &#39;Calibration Curve (10000/cpm vs antigen amount)&#39;)+
  theme_bw()</code></pre>
<div class="figure"><span id="fig:ria-cali"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/ria-cali-1.png" alt="RIA Calibration Curve (10000/count vs antigen amount)" width="672" />
<p class="caption">
Figure 5.3: RIA Calibration Curve (10000/count vs antigen amount)
</p>
</div>
<p>The equation of the fitted straight line can be calculated as follows:</p>
<pre class="r"><code>with(known, summary(lm(10000/count ~ antigen_amount)))</code></pre>
<pre><code>## 
## Call:
## lm(formula = 10000/count ~ antigen_amount)
## 
## Residuals:
##      Min       1Q   Median       3Q      Max 
## -0.33043 -0.05604  0.00116  0.05493  0.39587 
## 
## Coefficients:
##                 Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)    2.591e-01  1.659e-02   15.62   &lt;2e-16 ***
## antigen_amount 2.486e-03  3.819e-05   65.10   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.133 on 134 degrees of freedom
## Multiple R-squared:  0.9693, Adjusted R-squared:  0.9691 
## F-statistic:  4238 on 1 and 134 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>This gives:</p>
<p><span class="math display">\[10000/\text{cpm} = 0.2591 + 2.486\times10^{-3}\text{antigen amount}\]</span></p>
<p>or:</p>
<p><span class="math display">\[\text{antigen amount (in pg)}=4022526/\text{cpm} - 104.2237\]</span></p>
</div>
<div id="testing-for-the-unkown" class="section level3 unnumbered">
<h3>Testing for the Unkown</h3>
<p>Here is the data when I tried different antigen dilution and volumes to make the count close to 20000. <span class="math inline">\(1\mu\text{L}\)</span> of <span class="math inline">\(1\times10^4\)</span> dilution suffices:</p>
<pre class="r"><code>unknown_trial &lt;- raw %&gt;%
  filter(`known`== 0,
         `antigen_volume` != 0) %&gt;%
  slice(1:7)
unknown_trial</code></pre>
<pre><code>## # A tibble: 7 x 6
##   antigen_volume antigen_dilution antibody_volume antibody_dilution count known
##            &lt;dbl&gt;            &lt;dbl&gt;           &lt;dbl&gt; &lt;fct&gt;             &lt;dbl&gt; &lt;dbl&gt;
## 1              1              100               1 10000               454     0
## 2              5             1000               1 10000               826     0
## 3              1             1000               1 10000              3857     0
## 4              5            10000               1 10000              8245     0
## 5              1            10000               1 10000             19760     0
## 6              5           100000               1 10000             29922     0
## 7              1           100000               1 10000             33886     0</code></pre>
<p>Then I repeated <span class="math inline">\(1\mu\text{L}\)</span> of <span class="math inline">\(1\times10^4\)</span> dilution 276 times (see Section <a href="#ria-supp">9.1</a> for the data). <strong>Curiously, the counts do not follow a normal distribution. It is bimodal:</strong></p>
<pre class="r"><code>unknown_test &lt;- raw %&gt;%
  filter(`known`== 0,
         `antigen_volume` != 0) %&gt;%
  slice(8:n())
unknown_test %&gt;% 
  ggplot(aes(count))+
  geom_histogram(bins = 40, fill=&#39;white&#39;, color=&#39;black&#39;)+
  labs(title = &#39;Distribution of 276 Radioactivity Count Readings&#39;, x=&#39;radioactivity count&#39;)+
  theme_bw()</code></pre>
<p><img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/unnamed-chunk-5-1.png" width="672" /></p>
<p><strong>This is likely to be a bug in the ImmunoLab application. </strong></p>
<p>If I had to assume it’s a normal distribution, a 95% confidence interval could be calculated:</p>
<pre class="r"><code>cpm = unknown_test[[&#39;count&#39;]]
mass = 4022526/cpm - 104.2237
conc = mass /
          1e-3 / # 1e-3 ml
          1e-4 / # dilution
          1e9  # pg to mg
mean = mean(conc); sd = sd(conc);
t = qt(0.975, length(conc)-1); t_sd = t*sd
c(mean=mean, sd = sd, t = t, t_sd = t_sd, lower = mean-t_sd, upper = mean+t_sd)</code></pre>
<pre><code>##      mean        sd         t      t_sd     lower     upper 
## 0.8361621 0.1897999 1.9686279 0.3736453 0.4625168 1.2098075</code></pre>
<p>(The relationship between cpm and antigen amount has been established previously: <span class="math inline">\(\text{antigen amount (in pg)}=4022526/\text{cpm} - 104.2237\)</span>)</p>
<p>The mean is 0.8362 mg/ml and the confidence interval is <span class="math inline">\(0.8362 \pm 0.3736\)</span>, or [0.4625 1.2098] mg/ml. (This is actually invalid because the distribution was not normal).</p>
</div>
</div>
</div>
<div id="enzyme-linked-immunosorbentassay" class="section level1">
<h1><span class="header-section-number">6</span> Enzyme-Linked Immunosorbentassay</h1>
<div id="a-background-to-elisa" class="section level2 unnumbered">
<h2>(a) Background to ELISA</h2>
<div id="q12" class="section level3 unnumbered">
<h3>Q12</h3>
<p><strong>Draw a simple labelled diagram showing the principle of a sandwich ELISA.</strong></p>
<p>See Figure <a href="#fig:immunolab-elisa">6.1</a>.</p>
<div class="figure"><span id="fig:immunolab-elisa"></span>
<img src="../img/immunolab-elisa.jpg" alt="Schematic of ELISA" width="100%" />
<p class="caption">
Figure 6.1: Schematic of ELISA
</p>
</div>
</div>
<div id="q13" class="section level3 unnumbered">
<h3>Q13</h3>
<p><strong>Give an example of a chromogenic substrate for:</strong></p>
<ol style="list-style-type: lower-roman">
<li>phosphatase: BCIP/NBT</li>
<li>peroxidase: TMB (3,3’,5,5’-tetramethylbenzidine)</li>
</ol>
</div>
<div id="q14" class="section level3 unnumbered">
<h3>Q14</h3>
<p><strong>Measuring the amount of enzyme in a well is technically simpler than measuring radioactivity, but it is much more susceptible to outside influences. List the factors that might affect the accuracy of your estimation of the amount of enzyme (assuming that optical density measurements are completely accurate).</strong></p>
<ul>
<li>Variation in temperature and pH</li>
<li>Aggregation of enzyme proteins (which leads to underestimation)</li>
</ul>
</div>
</div>
<div id="b-method-elisalab" class="section level2 unnumbered">
<h2>(b) METHOD: ElisaLab</h2>
<div id="q15" class="section level3 unnumbered">
<h3>Q15</h3>
<p><strong>What is the Beer-Lambert law and why is it relevant when working out how much labelled antibody to use?</strong></p>
<p>Beer-Lambert Law is:</p>
<p><span class="math display">\[A = \epsilon cl\]</span></p>
<p>where <span class="math inline">\(A\)</span> is the absorbance (<span class="math inline">\(A=\log_{10}\left(\dfrac{I_\text{incident}}{I_\text{transmitted}}\right)\)</span>), <span class="math inline">\(\epsilon\)</span> is the molar extinction coefficient, <span class="math inline">\(c\)</span> is the concentration of the light-absorbing substance, <span class="math inline">\(l\)</span> is the path length.</p>
<p>By measuring absorbance (<span class="math inline">\(A\)</span>), the concentration of the coloured product, and hence the concentration of the enzyme-conjugated antibodies, can be deduced.</p>
<p><strong>What would happen to the dynamic range if we used more labelled Ab?</strong></p>
</div>
</div>
</div>
<div id="etut" class="section level1">
<h1><span class="header-section-number">7</span> ELISA Tutorial</h1>
<p>As I will be plot several heatmaps to show the ELISA results, and the procedures are similar, I created this helper function to prevent repetition:</p>
<pre class="r"><code>plot_elisa &lt;- function(fn, title) {
  df &lt;- read_tsv(fn) %&gt;% 
    janitor::clean_names() %&gt;% 
    mutate(ag_dilution = factor(ag_dilution),
           ab_dilution = factor(ab_dilution))
  return(ggplot(df, aes(ab_dilution, ag_dilution, fill=absorbance))+
    geom_tile()+
    labs(title = title)+
    scale_y_discrete(limits = rev(levels(df$ag_dilution)))+
    scale_fill_gradient(high=&#39;blue&#39;, low=&#39;white&#39;)+
    geom_text(aes(label=absorbance)))
}</code></pre>
<p><strong><em>A small flaw: until now I haven’t found an easy way to show exponent forms of axes values in this case. It would have been straightforward if my axes had been continuous. However, heatmaps require discrete variables so I converted numerical values (continuous) to factors (discrete).</em></strong></p>
<p>The results of the ELISA tutorial are shown in Figure <a href="#fig:elisa-tut">7.1</a>.</p>
<pre class="r"><code>plot_elisa(&#39;./2020-03-08-immunoassay-in-silico/elisa_tutorial.tsv&#39;, &quot;ELISA Tutorial&quot;)</code></pre>
<div class="figure"><span id="fig:elisa-tut"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-tut-1.png" alt="ELISA tutorial results" width="100%" />
<p class="caption">
Figure 7.1: ELISA tutorial results
</p>
</div>
</div>
<div id="elisa-experiment" class="section level1">
<h1><span class="header-section-number">8</span> ELISA Experiment</h1>
<p><strong>Reference number:</strong> <code>2nf5rpcd</code></p>
<div id="a-choosing-a-suitable-concentration-of-antibody" class="section level2 unnumbered">
<h2>(a) Choosing a suitable concentration of antibody</h2>
<div id="q16" class="section level3 unnumbered">
<h3>Q16</h3>
<p><strong>Describe the steps you took to optimise Antibody dilution for the rest of the experiment.</strong></p>
<p>First, I used a wide range of antibody dilutions (but with smaller intervals around <span class="math inline">\(10^4\)</span>, as the optimal concentration is more likely to be around here), these concentrations are <span class="math inline">\(10^2, 10^3, 4\times10^3, 10^4, 4\times10^4, 8\times10^4, 10^5, 10^6\)</span>. The antigen concentrations are evenly distributed, namely <span class="math inline">\(1\times10^n\)</span> and <span class="math inline">\(4\times10^n\)</span> for <span class="math inline">\(n\)</span> in <span class="math inline">\(\{1,2,3,4,5,6\}\)</span>. The results are shown in Figure <a href="#fig:elisa-t1">8.1</a>. The <code>plot_elisa()</code> function used here were defined in Section <a href="#etut">7</a>. I believe the plots I made are a better representation of ELISA results than screenshots or Excel spreadsheets. However, if you wish to view the raw data, you can download them from <a href="https://github.com/TianyiShi2001/ox/tree/master/content/lab/2020-03-08-immunoassay-in-silico/">https://github.com/TianyiShi2001/ox/tree/master/content/lab/2020-03-08-immunoassay-in-silico/</a>.</p>
<pre class="r"><code>plot_elisa(&#39;./2020-03-08-immunoassay-in-silico/elisa_trial_1.tsv&#39;, &quot;ELISA Trial 1 (choosing antibody concentration)&quot;)</code></pre>
<div class="figure"><span id="fig:elisa-t1"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-t1-1.png" alt="ELISA Trial 1 (choosing antibody concentration) results." width="100%" />
<p class="caption">
Figure 8.1: ELISA Trial 1 (choosing antibody concentration) results.
</p>
</div>
<p>As shown in Figure <a href="#fig:elisa-t1">8.1</a>, the optimal antibody concentration is between <span class="math inline">\(4\times10^4\)</span> and <span class="math inline">\(8\times10^4\)</span>. So in the next round of trial, I chosed antibody concentrations of <span class="math inline">\(3.2\times10^4\)</span>, <span class="math inline">\(4.0\times10^4\)</span>, <span class="math inline">\(6.4\times10^4\)</span>, and <span class="math inline">\(8.0\times10^4\)</span>. The antigen concentrations are the same as the previous trial. The experiments on these four antibody concentrations are repeated to fill 96 wells. The results are shown in Figure <a href="#fig:elisa-t2">8.2</a>.</p>
<pre class="r"><code>elisa_trial_2_1 &lt;- plot_elisa(&#39;./2020-03-08-immunoassay-in-silico/elisa_trial_2_1.tsv&#39;, &quot;ELISA Trial 2-1&quot;)
elisa_trial_2_2 &lt;- plot_elisa(&#39;./2020-03-08-immunoassay-in-silico/elisa_trial_2_2.tsv&#39;, &quot;ELISA Trial 2-2&quot;)
ggarrange(elisa_trial_2_1, elisa_trial_2_2, ncol=2, nrow=1, common.legend = TRUE, legend=&quot;bottom&quot;, labels = c(&quot;A&quot;, &quot;B&quot;))</code></pre>
<div class="figure"><span id="fig:elisa-t2"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-t2-1.png" alt="ELISA Trial 2 (choosing antibody concentration) results." width="100%" />
<p class="caption">
Figure 8.2: ELISA Trial 2 (choosing antibody concentration) results.
</p>
</div>
<p>The plot suggests that an antibody concentration of <span class="math inline">\(4.0\times10^4\)</span> would be optimal. Its maximum is close to, but does not exceed, 2.0.</p>
</div>
</div>
<div id="b-generating-the-calibration-curve" class="section level2 unnumbered">
<h2>(b) Generating the calibration curve</h2>
<div id="q17" class="section level3 unnumbered">
<h3>Q17</h3>
<p><strong>Discuss how you generated data for your calibration curve (you can append titled data sets to the end of this document)</strong></p>
</div>
<div id="q18" class="section level3 unnumbered">
<h3>Q18</h3>
<p><strong>Plot your results to generate a calibration curve. Ensure axes are labelled, amount (not dilution) of antigen is plotted and that you have provided a descriptive title of the graph.</strong></p>
<p>First, I used the same antigen concentrations as in previous two trials (<span class="math inline">\(1\times10^n\)</span> and <span class="math inline">\(4\times10^n\)</span> for <span class="math inline">\(n\)</span> in <span class="math inline">\(\{1,2,3,4,5,6\}\)</span>.), as shown below (this code reads data into dataframe <code>std1</code> while printing unique dilutions used).</p>
<pre class="r"><code>std1 &lt;- read_tsv(&#39;./2020-03-08-immunoassay-in-silico/elisa_std1.tsv&#39;) %&gt;%
  janitor::clean_names() %T&gt;%
  # print dilutions used
  (function(d){print(sort(unique(d$ag_dilution)))}) %&gt;%
  # convert antigen dilution into antigen concentration in ng/ml
  # keep only two variables: ag_conc and absorbance
  transmute(ag_conc = 10^6 / ag_dilution, absorbance)</code></pre>
<pre><code>##  [1] 1e+01 4e+01 1e+02 4e+02 1e+03 4e+03 1e+04 4e+04 1e+05 4e+05 1e+06 4e+06</code></pre>
<p>As shown in Figure <a href="#fig:elisa-std-1-raw">8.3</a>, if plotted directly, it can be observed that absorbance increases with increasing antigen concentration as expected, but the mathematical relationship is unclear and so the ‘optimal range’ is also vague.</p>
<pre class="r"><code>ggplot(std1, aes(ag_conc, absorbance))+
  geom_point()+
  labs(title = &#39;ELISA priliminary calibration curve&#39;, x = &#39;antigen concentration (ng/ml)&#39;)</code></pre>
<div class="figure"><span id="fig:elisa-std-1-raw"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-std-1-raw-1.png" alt="ELISA priliminary calibration curve" width="100%" />
<p class="caption">
Figure 8.3: ELISA priliminary calibration curve
</p>
</div>
<p>However, as shown in Figure <a href="#fig:elisa-std-1">8.4</a>, a linear relationship can be obtained by a double-log transformation. In this plot, it is clear that the useful range of antigen concentration would be <span class="math inline">\(1.0&lt;\log_{10}{c}&lt;{3.4}\)</span>, or <span class="math inline">\(10^1 &lt; c &lt; 10^{3.4}\)</span>, where <span class="math inline">\(c\)</span> is the antigen concentration in <span class="math inline">\(ng/ml\)</span>. This corresponds to a range of dilution of <span class="math inline">\(4\times10^2 &lt; x &lt; 10^5\)</span>.</p>
<pre class="r"><code>ggplot(std1, aes(log10(ag_conc), log10(absorbance)))+
  geom_point()+
  labs(title = &#39;ELISA priliminary calibration curve (double log)&#39;,
       x = &#39;log(antigen concentration (ng/ml))&#39;)+
  geom_abline(intercept = -3.4, slope = 1)+
  geom_text(x=2,y=-1, label=&#39;y = x - 3.4&#39;, family=&quot;Helvetica&quot;, size=4.5)</code></pre>
<div class="figure"><span id="fig:elisa-std-1"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-std-1-1.png" alt="ELISA priliminary calibration curve (double log)" width="100%" />
<p class="caption">
Figure 8.4: ELISA priliminary calibration curve (double log)
</p>
</div>
<p>Based on previous knowledge that the dilution should be in the range <span class="math inline">\(4\times10^2 &lt; x &lt; 10^5\)</span>, more experiments are carried out. The dilutions used are shown below.</p>
<pre class="r"><code>std2 &lt;- read_tsv(&#39;./2020-03-08-immunoassay-in-silico/elisa_std2.tsv&#39;) %&gt;%
  janitor::clean_names() %T&gt;%
  # print dilutions used
  (function(d){print(sort(unique(d$ag_dilution)))}) %&gt;%
  # convert antigen dilution into antigen concentration in ng/ml
  # keep only two variables: ag_conc and absorbance
  transmute(ag_conc = 10^6 / ag_dilution, absorbance)</code></pre>
<pre><code>##  [1]   800  1000  2000  4000  8000 10000 20000 32000 40000 64000 80000</code></pre>
<p>The calibration curve is shown in Figure <a href="#fig:elisa-std-fin">8.5</a>.</p>
<pre class="r"><code>ggplot(std2, aes(log10(ag_conc), log10(absorbance)))+
  geom_point()+
  labs(title = &#39;ELISA calibration curve&#39;,
       x = &#39;log(antigen concentration (ng/ml))&#39;)+
  geom_smooth(method=&#39;lm&#39;)</code></pre>
<div class="figure"><span id="fig:elisa-std-fin"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-std-fin-1.png" alt="ELISA calibration curve (double log)" width="100%" />
<p class="caption">
Figure 8.5: ELISA calibration curve (double log)
</p>
</div>
</div>
<div id="q19" class="section level3 unnumbered">
<h3>Q19</h3>
<p><strong>Briefly comment on the graph:</strong></p>
<pre class="r"><code>summary(lm(log10(std2$absorbance) ~ log10(std2$ag_conc)))</code></pre>
<pre><code>## 
## Call:
## lm(formula = log10(std2$absorbance) ~ log10(std2$ag_conc))
## 
## Residuals:
##      Min       1Q   Median       3Q      Max 
## -0.09315 -0.03981  0.00023  0.04625  0.08439 
## 
## Coefficients:
##                      Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)         -3.397722   0.015712  -216.3   &lt;2e-16 ***
## log10(std2$ag_conc)  0.996370   0.007425   134.2   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.04711 on 94 degrees of freedom
## Multiple R-squared:  0.9948, Adjusted R-squared:  0.9948 
## F-statistic: 1.801e+04 on 1 and 94 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>As shown in Figure <a href="#fig:elisa-std-fin">8.5</a> and in the linear model summary above, when the antigen concentration is in the range <span class="math inline">\(10^1 &lt; c &lt; 2.5\times10^3\)</span> (i.e. the absorbance is in the range <span class="math inline">\(0.004 &lt; A &lt; 1.000\)</span>), there is a significant correlation between the antigen concentration and the absorbance, which can be described by this equation:</p>
<p><span class="math display">\[\log_{10}{A} = \log_{10}{c} - 3.40\]</span></p>
<p>where <span class="math inline">\(A\)</span> is the absorbance and <span class="math inline">\(c\)</span> is the antigen concentration in <span class="math inline">\(ng/ml\)</span>.</p>
<p>As shown in Figure <a href="#fig:elisa-std-fin">8.5</a>, when log(antigen concentration) is less than 1.7, the variation within each sets of absorbance readings is slightly larger, so the optimal range should be narrow down to <span class="math inline">\(10^{1.7} &lt; c &lt; 10^{3.4}\)</span> for antigen concentration, or <span class="math inline">\(0.020 &lt; A &lt; 1.000\)</span> for absorbance.</p>
</div>
</div>
<div id="c-measuring-concentration-of-the-unknown" class="section level2 unnumbered">
<h2>(c) Measuring concentration of the unknown</h2>
<div id="q20" class="section level3 unnumbered">
<h3>Q20</h3>
<p><strong>Using appropriate dilutions and number of replicates (discuss briefly below). Calculate the concentration of antigen in the original (unknown) solution, stating your calculation process. Where relevant, you can append titled data sets to the end of this document. Esnure your answer is statistically valid (think back to the stats test used in RiaLab)</strong></p>
<p>First, I used a broad range of dilutions (again, <span class="math inline">\(1\times10^n\)</span> and <span class="math inline">\(4\times10^n\)</span> for <span class="math inline">\(n\)</span> in <span class="math inline">\(\{1,2,3,4,5,6\}\)</span>).</p>
<pre class="r"><code>df &lt;- read_tsv(&#39;./2020-03-08-immunoassay-in-silico/elisa_exp1.tsv&#39;) %&gt;%
  janitor::clean_names() %&gt;%
  transmute(ag_dilution = factor(ag_dilution), absorbance)
ggplot(df, aes(ag_dilution, absorbance))+
    geom_boxplot()+
    labs(title = &quot;ELISA Experiment for the Unknown&quot;)+
    geom_hline(yintercept = 1.000, color=&#39;red&#39;, size = 0.6)+
    geom_hline(yintercept = 0.020, color=&#39;red&#39;, size = 0.6)</code></pre>
<div class="figure"><span id="fig:elisa-exp1"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-exp1-1.png" alt="Finding the appropriate dilution of the unknown so that the absorbance fall within the linear range." width="100%" />
<p class="caption">
Figure 8.6: Finding the appropriate dilution of the unknown so that the absorbance fall within the linear range.
</p>
</div>
<pre class="r"><code>    # scale_x_discrete(limits = rev(levels(df$ag_dilution)))+</code></pre>
<p>As learnt from Q19, the linear range is where the absorbance lies between 0.020 and 1.000. According to Figure <a href="#fig:elisa-exp1">8.6</a>, a <span class="math inline">\(10^3\)</span> dilution would be the most appropriate. Therefore, I repeat 480 times with this dilution.</p>
<pre class="r"><code>elisa_absorbance &lt;- read_tsv(&#39;./2020-03-08-immunoassay-in-silico/elisa_exp_fin.tsv&#39;) %&gt;%
  janitor::clean_names() %&gt;%
  extract2(&#39;absorbance&#39;)</code></pre>
<p>Curiously, again, the distribution is bimodal (the data is shown in Section <a href="#elisa-absorbance">9.2</a>:</p>
<pre class="r"><code>qplot(elisa_absorbance, geom = &#39;histogram&#39;, bins = I(40), fill=I(&#39;white&#39;), color=I(&#39;black&#39;),
      main = &quot;distribution of 480 absorbance readings for the unknown&quot;)</code></pre>
<div class="figure"><span id="fig:elisa-abs-dist"></span>
<img src="../lab/2020-03-08-immunoassay-in-silico_files/figure-html/elisa-abs-dist-1.png" alt="Distribution of 480 absorbance readings for the ELISA assay with 1:1000 unknown sample and 1:40000 antibody" width="100%" />
<p class="caption">
Figure 8.7: Distribution of 480 absorbance readings for the ELISA assay with 1:1000 unknown sample and 1:40000 antibody
</p>
</div>
<p>If I had to assume it’s a normal distribution, a 95% confidence interval could be calculated:</p>
<pre class="r"><code>conc &lt;- 10^(log10(elisa_absorbance) + 3.40) *
                10^3 / # dilution
                10^6 # ng/ml to mg/ml
mean &lt;- mean(conc); sd = sd(conc);
t &lt;- qt(0.975, length(conc)-1); t_sd &lt;- t*sd
c(mean=mean, sd = sd, t = t, t_sd = t_sd, lower = mean-t_sd, upper = mean+t_sd)</code></pre>
<pre><code>##       mean         sd          t       t_sd      lower      upper 
## 0.80949203 0.08313395 1.96492886 0.16335230 0.64613973 0.97284433</code></pre>
<p>(Used the relationship between antigen concentration and absorbance established previously: <span class="math inline">\(\log_{10}{A} = \log_{10}{c} - 3.40\)</span>)</p>
<p>The mean is 0.8095 mg/ml and the confidence interval is <span class="math inline">\(0.8094\pm0.1634\)</span>, or [0.6461 0.9728] mg/ml. (This is actually invalid because the distribution was not normal).</p>
</div>
</div>
</div>
<div id="supplementary-information" class="section level1">
<h1><span class="header-section-number">9</span> Supplementary Information</h1>
<p>For a complete set of raw data, please go to <a href="https://github.com/TianyiShi2001/ox/tree/master/content/lab/2020-03-08-immunoassay-in-silico/">https://github.com/TianyiShi2001/ox/tree/master/content/lab/2020-03-08-immunoassay-in-silico/</a>.</p>
<div id="ria-supp" class="section level2">
<h2><span class="header-section-number">9.1</span> RIA</h2>
<p>The 276 counts for <span class="math inline">\(1\mu\text{L}\)</span> of <span class="math inline">\(1\times10^4\)</span> dilution of antigen:</p>
<pre class="r"><code>unknown_test[[&#39;count&#39;]]</code></pre>
<pre><code>##   [1] 20336 17947 20618 23919 23332 23224 21150 23007 23463 20596 25927 18685
##  [13] 20270 21910 20466 20162 19880 17502 20314 24646 20281 20151 18946 20346
##  [25] 23137 21715 20108 19858 23180 19445 18012 20477 23256 19782 23235 24407
##  [37] 23647 24320 20064 21291 19521 22735 19250 21573 23560 18631 25656 19337
##  [49] 19174 24016 19630 24136 20292 24005 17643 23365 19890 20629 24103 20173
##  [61] 25102 20933 19174 19065 23397 19880 22388 19174 23159 23658 22008 24787
##  [73] 22931 24309 24841 23484 23777 25048 20976 24700 20053 20629 20792 19565
##  [85] 19261 24527 18099 24516 23072 22279 21139 23745 21715 19782 19977 19923
##  [97] 24353 18609 23452 19000 20336 19858 18968 23951 24744 23539 24418 23810
## [109] 21009 19478 23625 19152 25211 23202 22963 22844 20379 22409 25069 17730
## [121] 20846 23452 24646 25254 24722 18533 24429 19597 19532 24483 22073 19521
## [133] 19804 25493 19956 20889 18240 17806 19445 18577 24168 19543 25395 23929
## [145] 19912 24820 24060 18707 18881 20368 19044 24114 23224 20640 23995 20227
## [157] 24049 19348 23506 24679 19152 20412 19413 23560 22442 23897 19814 23159
## [169] 22399 19586 22605 23473 23148 19337 20650 17969 23061 22789 25482 19749
## [181] 24537 23408 20564 18989 22127 19011 20596 19337 19022 19402 20455 23169
## [193] 22724 19966 20835 20216 23148 23951 19510 20650 19206 24559 23647 19434
## [205] 22073 19880 20705 19543 17849 23691 23940 17686 20813 19250 22876 19521
## [217] 23767 22225 25688 23734 18826 20683 20694 20433 21573 19206 25004 23354
## [229] 23332 24635 22540 17914 22692 23017 22149 19652 24038 20900 24744 20346
## [241] 22681 20064 19717 19109 24331 19500 23235 19825 19152 19684 19630 21421
## [253] 23365 22822 20162 22073 23202 24505 24309 22062 23799 19836 22963 24429
## [265] 19293 23962 21204 23886 18685 24147 20531 19901 19521 23734 20118 24125</code></pre>
</div>
<div id="elisa-absorbance" class="section level2">
<h2><span class="header-section-number">9.2</span> ELISA</h2>
<p>The 480 absorbance readings:</p>
<pre class="r"><code>elisa_absorbance</code></pre>
<pre><code>##   [1] 0.354 0.343 0.355 0.279 0.268 0.303 0.348 0.358 0.283 0.358 0.277 0.378
##  [13] 0.290 0.290 0.304 0.284 0.355 0.331 0.345 0.271 0.276 0.356 0.349 0.301
##  [25] 0.259 0.308 0.331 0.357 0.284 0.361 0.295 0.332 0.282 0.381 0.278 0.340
##  [37] 0.358 0.316 0.273 0.285 0.359 0.366 0.286 0.297 0.275 0.307 0.368 0.351
##  [49] 0.341 0.339 0.313 0.354 0.346 0.284 0.334 0.278 0.300 0.286 0.312 0.353
##  [61] 0.361 0.364 0.357 0.372 0.285 0.358 0.274 0.282 0.280 0.344 0.253 0.300
##  [73] 0.336 0.300 0.301 0.348 0.355 0.374 0.276 0.290 0.279 0.370 0.338 0.270
##  [85] 0.373 0.351 0.356 0.353 0.256 0.294 0.339 0.315 0.351 0.288 0.311 0.352
##  [97] 0.352 0.280 0.333 0.368 0.334 0.294 0.297 0.299 0.279 0.343 0.357 0.309
## [109] 0.366 0.363 0.310 0.311 0.358 0.335 0.361 0.296 0.257 0.334 0.308 0.293
## [121] 0.293 0.336 0.342 0.352 0.366 0.287 0.266 0.286 0.360 0.313 0.358 0.287
## [133] 0.305 0.363 0.343 0.284 0.290 0.339 0.268 0.350 0.363 0.334 0.343 0.271
## [145] 0.324 0.299 0.258 0.334 0.309 0.336 0.374 0.362 0.290 0.344 0.303 0.352
## [157] 0.295 0.314 0.286 0.300 0.287 0.292 0.369 0.349 0.292 0.349 0.360 0.334
## [169] 0.339 0.277 0.286 0.347 0.293 0.325 0.288 0.278 0.303 0.271 0.289 0.305
## [181] 0.309 0.364 0.344 0.309 0.355 0.360 0.298 0.298 0.299 0.322 0.340 0.364
## [193] 0.344 0.337 0.311 0.344 0.294 0.365 0.351 0.297 0.384 0.327 0.350 0.315
## [205] 0.326 0.301 0.316 0.292 0.300 0.366 0.346 0.329 0.291 0.355 0.290 0.300
## [217] 0.301 0.370 0.299 0.340 0.336 0.274 0.301 0.359 0.342 0.345 0.350 0.356
## [229] 0.347 0.317 0.354 0.286 0.339 0.358 0.326 0.362 0.358 0.349 0.329 0.298
## [241] 0.317 0.337 0.327 0.309 0.301 0.291 0.368 0.342 0.359 0.347 0.337 0.352
## [253] 0.292 0.294 0.370 0.377 0.287 0.292 0.279 0.349 0.315 0.370 0.365 0.280
## [265] 0.352 0.342 0.342 0.363 0.278 0.335 0.372 0.351 0.354 0.348 0.292 0.293
## [277] 0.298 0.327 0.286 0.374 0.335 0.348 0.359 0.274 0.314 0.354 0.359 0.347
## [289] 0.331 0.345 0.371 0.283 0.370 0.289 0.359 0.344 0.356 0.374 0.267 0.356
## [301] 0.304 0.333 0.279 0.344 0.355 0.366 0.291 0.306 0.352 0.361 0.372 0.270
## [313] 0.359 0.317 0.294 0.340 0.257 0.265 0.283 0.285 0.293 0.330 0.345 0.322
## [325] 0.294 0.336 0.362 0.369 0.343 0.283 0.324 0.326 0.341 0.299 0.362 0.368
## [337] 0.358 0.334 0.283 0.331 0.278 0.261 0.351 0.337 0.288 0.278 0.269 0.298
## [349] 0.361 0.311 0.355 0.361 0.287 0.343 0.267 0.294 0.343 0.293 0.286 0.312
## [361] 0.296 0.340 0.267 0.281 0.301 0.326 0.356 0.287 0.265 0.296 0.295 0.254
## [373] 0.289 0.371 0.274 0.285 0.349 0.361 0.353 0.277 0.341 0.332 0.336 0.296
## [385] 0.274 0.321 0.279 0.297 0.346 0.273 0.370 0.298 0.380 0.273 0.338 0.342
## [397] 0.369 0.308 0.361 0.294 0.312 0.353 0.371 0.288 0.354 0.278 0.367 0.280
## [409] 0.315 0.335 0.334 0.369 0.308 0.292 0.369 0.349 0.312 0.286 0.289 0.379
## [421] 0.349 0.315 0.339 0.297 0.355 0.310 0.287 0.342 0.296 0.275 0.337 0.353
## [433] 0.329 0.302 0.332 0.356 0.311 0.344 0.283 0.358 0.288 0.332 0.341 0.278
## [445] 0.367 0.284 0.287 0.321 0.343 0.286 0.307 0.271 0.311 0.266 0.357 0.348
## [457] 0.298 0.297 0.361 0.370 0.377 0.362 0.330 0.388 0.287 0.332 0.295 0.357
## [469] 0.279 0.306 0.313 0.355 0.279 0.303 0.328 0.334 0.339 0.335 0.312 0.305</code></pre>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/how-has-ras-signalling-been-studied-in-c-elegans.html" class="archive-item-link">
                How Has Ras Signalling Been Studied in C. Elegans 
              </a>
              <span class="archive-item-date">
                2020-03-03
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#the-big-picture-components-of-the-c.-elegans-egf-ras-mapk-pathway-involved-in-vulva-induction"><span class="toc-section-number">2</span> The Big Picture: Components of the <em>C. elegans</em> EGF-Ras-MAPK Pathway Involved in Vulva Induction</a></li>
<li><a href="#techniques-for-studying-egfr-ras-mapk-signalling-and-vulva-induction"><span class="toc-section-number">3</span> Techniques for Studying EGFR-Ras-MAPK Signalling and Vulva Induction</a><ul>
<li><a href="#forward-genetics-and-reverse-genetics"><span class="toc-section-number">3.1</span> Forward Genetics and Reverse Genetics</a><ul>
<li><a href="#in-early-experiments-forward-genetics-methods-were-used-entensively"><span class="toc-section-number">3.1.1</span> In Early Experiments, Forward Genetics Methods were Used Entensively</a></li>
<li><a href="#rnai-and-reverse-genetics-since-1998"><span class="toc-section-number">3.1.2</span> RNAi and Reverse Genetics Since 1998</a></li>
</ul></li>
<li><a href="#observing-cell-fates"><span class="toc-section-number">3.2</span> Observing Cell Fates</a><ul>
<li><a href="#microscopic-analysis"><span class="toc-section-number">3.2.1</span> Microscopic Analysis</a></li>
<li><a href="#transcriptional-reporters"><span class="toc-section-number">3.2.2</span> Transcriptional Reporters</a></li>
</ul></li>
<li><a href="#cross-species-genomic-analysis-reveals-homology-and-helps-to-elucidate-roles-of-proteins-with-greater-accuracy"><span class="toc-section-number">3.3</span> Cross-Species Genomic Analysis Reveals Homology and Helps to Elucidate Roles of Proteins with Greater Accuracy</a></li>
<li><a href="#studying-egfr-localisation-and-trafficking"><span class="toc-section-number">3.4</span> Studying EGFR Localisation and Trafficking</a><ul>
<li><a href="#immunostaining"><span class="toc-section-number">3.4.1</span> Immunostaining</a></li>
<li><a href="#live-imaging"><span class="toc-section-number">3.4.2</span> Live Imaging</a></li>
</ul></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">4</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>The <em>Caenorhabditis elegans</em> vulva is one of the best studied models of organogeneis. Genetic screening revealed three signalling pathways, EGFR-Ras-MAPK, Notch and Wnt<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>, that are critical to the fate determination of the 6 vulval precursor cells (VPCs), P(3-8)p.</p>
<p>VPCs adopt either 1<span class="math inline">\(^\circ\)</span> or 2<span class="math inline">\(^\circ\)</span> fate upon receiving appropriate signals, or 3<span class="math inline">\(^\circ\)</span> fate if undinduced. The 8 descentdents of the 1<span class="math inline">\(^\circ\)</span> VPC and the 14 descendents (7 each) of the 2<span class="math inline">\(^\circ\)</span> VPCs give rise to the vulva, while 3<span class="math inline">\(^\circ\)</span> VPCs produces epidermal cells which fuse with the syncytial epidermis hyp7.</p>
<p>In the L1 stage, a Wnt signal form the posterior body region selects the six Pn.p cells to become the VPCs and form the vulval competentce group. Then in the L2 stage, LIN-3 (EGF) signal from the anchor cell (AC) in the somatic gonad selects P6.p to be the single 1<span class="math inline">\(^\circ\)</span> VPC<span class="citation"><sup>1,2</sup></span>, and contact-dependent LIN-12 (Notch) signalling causes the flanking P5.p and P7.p to become 2<span class="math inline">\(^\circ\)</span> VPCs. These precise signalling events always results in an invariant spatial pattern of VPCs in wild-type (WT) <em>C. elegans</em>, namely 3<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-1<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>.</p>
<p>This essay illustrates how the LET-60 (Ras) signalling has been studied in <em>C. elegans</em> vulval development, specifically the 1<span class="math inline">\(^\circ\)</span> fate determination, according to these selected work and reviews: <span class="citation">3</span>; <span class="citation">4</span>; <span class="citation">5</span>; <span class="citation">6</span>; <span class="citation">7</span>; <span class="citation">8</span>. A recent review article by <span class="citation">9</span> and a book chapter by <span class="citation">10</span> are also taken into account (the latter was the most helpful).</p>
</div>
<div id="the-big-picture-components-of-the-c.-elegans-egf-ras-mapk-pathway-involved-in-vulva-induction" class="section level1">
<h1><span class="header-section-number">2</span> The Big Picture: Components of the <em>C. elegans</em> EGF-Ras-MAPK Pathway Involved in Vulva Induction</h1>
<p>Figure <a href="#fig:c-ele-valva-big">2.1</a> gives an overview of the pathways by which EGF-Ras-MAPK signalling leads to the 1<span class="math inline">\(^\circ\)</span> fate in P6.p and 2<span class="math inline">\(^\circ\)</span> fate in flanking P5.p and P7.p cells.</p>
<div class="figure"><span id="fig:c-ele-valva-big"></span>
<img src="../img/c-ele-valva-big.jpg" alt="P6.p is the VPC closest to the AC, and it also expresses the highest levels of LET-23 on its basolateral membrane. Thus, P6.p receives most LIN-3 signalling and commits to the 1\(^\circ\) fate. Downstram of the LET-23 (EGFR) RTK, a canocical Ras-MAPK pathway transduces the signal into the nucleus. The scaffold proteins LIN-2 (CASK), LIN-7 (Veli), and LIN-10 (Mint-1) form a complex which helps to maintain basolateral localisation of LET-23. The SEM-5 (Grb2) adaptor protein11 uses its SH2 domain to anchor to LET-23 at phosphorylated tyrosines. The guanine exchange factor (GEF) SOS-112 then binds to SEM-5 (at SEM-5’s two SH3 domains), and it activates the membrane-anchored LET-60 (Ras)3. GTP-bound LET-60 then activates the LIN-45 (Raf)13, MEK-214 and MPK-1 (MAPK) kinase cascade, which is facilitated by the scaffold protein KSR. LIN-1 and forkhead LIN-31 transcription factors are the targets of MPK-115. In their unphosphorylated state, LIN-1 and LIN-31 forms a complex which represses 1\(^\circ\)-specific transcription of lin-39 and egl-1716. Phosphorylation makes the complex to dissociate, and phospho-LIN-31 is turned into an transcriptional activator. The 1\(^\circ\) P6.p also produces LAG-2 (Delta), which is the ligand for LIN-12 (Notch), thus transducing lateral signals to P5.p and P7.p. Notch signalling inhibits MAKP, and LET-60 in P5.p and P7.p cells uses a RAL-1 dependent pathway to commit to 2\(^\circ\) fate. A number of modifiers and negative regulators of the EGFR pathway has been identified, which include the tyrosine kinase Ark-117, the RTK phosphatase Dep-118. Adapted from 10 with reference to 19 and 20." width="100%" />
<p class="caption">
Figure 2.1: P6.p is the VPC closest to the AC, and it also expresses the highest levels of LET-23 on its basolateral membrane. Thus, P6.p receives most LIN-3 signalling and commits to the 1<span class="math inline">\(^\circ\)</span> fate. Downstram of the LET-23 (EGFR) RTK, a canocical Ras-MAPK pathway transduces the signal into the nucleus. The scaffold proteins LIN-2 (CASK), LIN-7 (Veli), and LIN-10 (Mint-1) form a complex which helps to maintain basolateral localisation of LET-23. The SEM-5 (Grb2) adaptor protein<span class="citation"><sup>11</sup></span> uses its SH2 domain to anchor to LET-23 at phosphorylated tyrosines. The guanine exchange factor (GEF) SOS-1<span class="citation"><sup>12</sup></span> then binds to SEM-5 (at SEM-5’s two SH3 domains), and it activates the membrane-anchored LET-60 (Ras)<span class="citation"><sup>3</sup></span>. GTP-bound LET-60 then activates the LIN-45 (Raf)<span class="citation"><sup>13</sup></span>, MEK-2<span class="citation"><sup>14</sup></span> and MPK-1 (MAPK) kinase cascade, which is facilitated by the scaffold protein KSR. LIN-1 and <em>forkhead</em> LIN-31 transcription factors are the targets of MPK-1<span class="citation"><sup>15</sup></span>. In their unphosphorylated state, LIN-1 and LIN-31 forms a complex which represses 1<span class="math inline">\(^\circ\)</span>-specific transcription of <em>lin-39</em> and <em>egl-17</em><span class="citation"><sup>16</sup></span>. Phosphorylation makes the complex to dissociate, and phospho-LIN-31 is turned into an transcriptional activator. The 1<span class="math inline">\(^\circ\)</span> P6.p also produces LAG-2 (Delta), which is the ligand for LIN-12 (Notch), thus transducing lateral signals to P5.p and P7.p. Notch signalling inhibits MAKP, and LET-60 in P5.p and P7.p cells uses a RAL-1 dependent pathway to commit to 2<span class="math inline">\(^\circ\)</span> fate. A number of modifiers and negative regulators of the EGFR pathway has been identified, which include the tyrosine kinase Ark-1<span class="citation"><sup>17</sup></span>, the RTK phosphatase Dep-1<span class="citation"><sup>18</sup></span>. Adapted from <span class="citation">10</span> with reference to <span class="citation">19</span> and <span class="citation">20</span>.
</p>
</div>

</div>
<div id="techniques-for-studying-egfr-ras-mapk-signalling-and-vulva-induction" class="section level1">
<h1><span class="header-section-number">3</span> Techniques for Studying EGFR-Ras-MAPK Signalling and Vulva Induction</h1>
<div id="forward-genetics-and-reverse-genetics" class="section level2">
<h2><span class="header-section-number">3.1</span> Forward Genetics and Reverse Genetics</h2>
<div id="in-early-experiments-forward-genetics-methods-were-used-entensively" class="section level3">
<h3><span class="header-section-number">3.1.1</span> In Early Experiments, Forward Genetics Methods were Used Entensively</h3>
<p>Studies on the molecular mechanisms of <em>C. elegans</em> development date back to the 1980s, when genetic manipulating techniques, especially gene cloning, were emerging, but the complete genome sequencing (of <em>C. elegans</em> and other model organisms) had yet to start. Therefore, those pioineering studies relied heavily on forward genetics, where worms were subjected to mutagens, and abnomal phenotypes were identified, isolated, amplified (by self-fertilisation) and the respective genotypes were determined. Differential interference contrast (DIC)/Nomarski microscopes were commonly used to examine the phenotypes because of their superior imaging quuality for live and unstained samples. There were also cellular methods for manipulating gene expression (which especially aided in eludidating cell-cell interactions). First, observing fates of the cells among varying individuals. Second, using laser beams to ablate cells suspected to interact with the cell of interest. Third, isolation of cells of interest, or rearrangement of their surrounding cells.</p>
<p>These extensive genetic screens led to identification of a number of genes that cause abormal cell lineage (the <em>lin#</em> genes) phenotypes, notably vulvaless (Vul)/Egg-laying defective (Egl) or multivulva (Muv) (Figure <a href="#fig:egl-muv-c-ele-vulva">3.1</a>). Epistasis analyses of these phenotypes grouped these genes into distinct pathways (notably EGFR-Ras-MAPK and Notch) and helped elucidating where each component functions in the respetive pathway. The general idea is that, if the phenotype of gene A masks the phenotype of gene B, then gene A is likely to act downstream of gene B.</p>

<div class="figure"><span id="fig:egl-muv-c-ele-vulva"></span>
<img src="../img/egl-muv-c-ele-vulva.png" alt="Egg-laying defective (Egl) and Multivulva (Muv) phenotypes caused by LET-23 EGFR-Ras-MAPK signalling mutants. (a): Wild type. Eggs are neatly arranged in the uterus. (b) and (c): Defective LIN-10 (which helps to localise LET-23) can lead to disorganisation of eggs (b) and the bag-of-worms phenotype (c), in which self-fertilised eggs hatch and larvae fills up their mother’s carcass. (d): let-60 gain-of-function results in Ras signalling in all VPCs, resulting in a Muv phenotype. Reproduced from 10." width="100%" />
<p class="caption">
Figure 3.1: Egg-laying defective (Egl) and Multivulva (Muv) phenotypes caused by LET-23 EGFR-Ras-MAPK signalling mutants. (a): Wild type. Eggs are neatly arranged in the uterus. (b) and (c): Defective LIN-10 (which helps to localise LET-23) can lead to disorganisation of eggs (b) and the bag-of-worms phenotype (c), in which self-fertilised eggs hatch and larvae fills up their mother’s carcass. (d): <em>let-60</em> gain-of-function results in Ras signalling in all VPCs, resulting in a Muv phenotype. Reproduced from <span class="citation">10</span>.
</p>
</div>
<p>Figure <a href="#fig:vul-Muv-simplest">3.2</a> shows the earliest expetiments that demonstrated the role of the AC and the LET-60 protein in vulval induction.</p>

<div class="figure"><span id="fig:vul-Muv-simplest"></span>
<img src="../img/vul-mul-simplest.jpg" alt="(a) in wild-type cells, signals from the AC results in the invariant VPC pattern 3\(^\circ\)-3\(^\circ\)-2\(^\circ\)-1\(^\circ\)-2\(^\circ\)-3\(^\circ\); (b) ablation of the AC renders a Vul phenotype, where all 6 VPCs adopts the 3\(^\circ\) fate, this shows that the AC positively regulates vulval induction; (c) inactive LET-60 (Ras) also renders a Vul phenotype even in the presence of the AC, therefore LET-60 should be downstream of the signal from the AC; (d) LET-60 hyperactivity leads to a Muv phenotype, further demonstrating LET-60 as a positive regulator of vulval induction." width="100%" />
<p class="caption">
Figure 3.2: (a) in wild-type cells, signals from the AC results in the invariant VPC pattern 3<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-1<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>; (b) ablation of the AC renders a Vul phenotype, where all 6 VPCs adopts the 3<span class="math inline">\(^\circ\)</span> fate, this shows that the AC positively regulates vulval induction; (c) inactive LET-60 (Ras) also renders a Vul phenotype even in the presence of the AC, therefore LET-60 should be downstream of the signal from the AC; (d) LET-60 hyperactivity leads to a Muv phenotype, further demonstrating LET-60 as a positive regulator of vulval induction.
</p>
</div>
</div>
<div id="rnai-and-reverse-genetics-since-1998" class="section level3">
<h3><span class="header-section-number">3.1.2</span> RNAi and Reverse Genetics Since 1998</h3>
<p>Early mutagenesis studies relied on chance to generate individuals with mutations for certain genes, which is laborious and not specific—usually some genes/DNA regions other than the genes of interest are also mutated, and studies on them can lead to confounding results.</p>
<p>Reverse genetics in <em>C. elegans</em> has become much more prevalent since 1998, when <em>C. elegans</em> became the first multicellular eukaryote to have its genome sequenced<span class="citation"><sup>21</sup></span>, and it was shown that introdcution of dsRNA into worms results in inactivation of an endogenous gene with corresponding sequence<span class="citation"><sup>22</sup></span>. This technique, known as RNA interference (RNAi), enables rapid, site-specific gene inactivation and has been rapidly embraced as a reverse-genetic tool and has dramatically accelerated the pace at which new gene functions are discovered.</p>
</div>
</div>
<div id="observing-cell-fates" class="section level2">
<h2><span class="header-section-number">3.2</span> Observing Cell Fates</h2>
<div id="microscopic-analysis" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Microscopic Analysis</h3>
<p>Observing Val and Muv phenotypes is generally sufficient for ordering the components of the Ras pathway, but cell-cell interaction analyses require the ability to identify differentiated or undifferentiated individual VPCs. The boundaries between cells are not visible by DIC optics and thus cells are identified by the location, size, and morphology of the nucleus and nucleoli. Undifferentiated VPCs at the Pn.p stage each have a characteristic oblong nucleus with a round nucleolus.</p>
</div>
<div id="transcriptional-reporters" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Transcriptional Reporters</h3>
<p>Several transcriptional targets of LET-23 (EGFR) signalling in P6.p, such as <em>egl-17</em> and <em>lag-2</em>, have been identified and can be used as transcriptional reporters to identify vulval cell fates. By using a highly sentitive Ras-responsive reporter gene, <em>egl-17::CFP</em>, <span class="citation">8</span> showed that the Ras pathway is transiently activated in P5.p and P7.p, and that LIN-12 (Notch) activation in these two cells antagonises Ras signalling.</p>
<p>Single molecule fluorescence <em>in situ</em> hybridisation (smFISH) can also be used to monitor expression of cell fate determinants. It reveals subtle changes in gene expression and has been used, for example by <span class="citation">23</span>, to visualise the dynamics of Ras-induced <em>lag-2</em> expression, which demonstrates the cellular control of EGF-Ras induced gene expression by changing sensitivity to the EGF gradient .</p>
</div>
</div>
<div id="cross-species-genomic-analysis-reveals-homology-and-helps-to-elucidate-roles-of-proteins-with-greater-accuracy" class="section level2">
<h2><span class="header-section-number">3.3</span> Cross-Species Genomic Analysis Reveals Homology and Helps to Elucidate Roles of Proteins with Greater Accuracy</h2>
<p>As sequencing techniques mature, complete genome sequences of more organisms are becoming available. This is accompanied by advance in computing power and bioinformatic algorithms, and they allow identification of a significant amount of homology among species.</p>
<p>The studies on <em>C. elegans</em> Ras signalling also benefitted from these developments. For example, MPK-1 was initially identified as a suppressor of Ras and was named SUR-1 before being identified as an Erk1/2 homologue. The scaffold protein KSR-1 was originally named SUR-3 due to the same reason. With homology available, one can easily guess the function of a novel gene/protein identified in one organism, if its sequence is found homologous with another gene with known functions.</p>
</div>
<div id="studying-egfr-localisation-and-trafficking" class="section level2">
<h2><span class="header-section-number">3.4</span> Studying EGFR Localisation and Trafficking</h2>
<p>The LET-23 EGFR upstream of LET-60 Ras has a polarised localisation in VPCs: vuval induction requires that EGFR to be expressed on the basolateral membrane. Immunostaining and live imaging can be used to monitor LET-23 trafficking and identify regulators involved.</p>
<div id="immunostaining" class="section level3">
<h3><span class="header-section-number">3.4.1</span> Immunostaining</h3>
<p>Immunostaining can be used to visualise subcellular localisation of components of regulators of the EDGR-Ras-MAPK pathway. This technique was used by <span class="citation">24</span> to establish the role of LIN-2/7/10 complex binding to LET-23 C-terminuc in regulating LET-23 basolateral localisation (Figure <a href="#fig:c-ele-vulva-immunostaining">3.3</a>).</p>

<div class="figure"><span id="fig:c-ele-vulva-immunostaining"></span>
<img src="../img/c-ele-vulva-immunostaining.png" alt="Top: LET-23 are localised to the basolateral membrane of VPCs in wild-type C. elegans; Middle: lin-7 mutant shows mislocalisation of LET-23 to the apical membrane; Bottom: let-23(sy1) mutant, in which a premature stop codon removes 6 C-terminal amino acids of LET-23, similarly resulted in mislocalisation of LET-23. LET-23 staining is shownin green, and staining for the cell junctions using the monoclonal antibody MH27 is shown in red. Reproduced from 24." width="100%" />
<p class="caption">
Figure 3.3: Top: LET-23 are localised to the basolateral membrane of VPCs in wild-type <em>C. elegans</em>; Middle: <em>lin-7</em> mutant shows mislocalisation of LET-23 to the apical membrane; Bottom: <em>let-23(sy1)</em> mutant, in which a premature stop codon removes 6 C-terminal amino acids of LET-23, similarly resulted in mislocalisation of LET-23. LET-23 staining is shownin green, and staining for the cell junctions using the monoclonal antibody MH27 is shown in red. Reproduced from <span class="citation">24</span>.
</p>
</div>
</div>
<div id="live-imaging" class="section level3">
<h3><span class="header-section-number">3.4.2</span> Live Imaging</h3>
<p>Modern imaging techniques, notably confocal and epifluorescence microscopy, allow the subcellular localisation and trafficking of LET-23 to be observed <em>in vivo</em>, in real time. These studies often involve fusion proteins with fluorescence tags. For example, <span class="citation">25</span> used time-lapse imaging and fluorescence recovery after photobleaching to identify ERM-1 (the Ezrin homologue) as a temporal regulator of LET-23 localisation.</p>
</div>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">4</span> Conclusion</h1>
<p>The understanding of the EGFR-Ras-MAPK signalling pathway is importnat because of its involvement in many human diseases, especially cancer. Vulva induction in <em>C. elegans</em> has been a valuable <em>in vivo</em> model for elucidating the identities and mechanisms of the components and regulators in this pathway. Studies on <em>C. elegans</em> traditionally relied on forward genetics, but new experimental techniques and bioinformatic are now solving previously unanswered questions in this field.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Hill-1992">
<p>1. Hill, R. &amp; Sternberg, P. The gene lin-3 encodes an inductive signal for vulval development in c. Elegans. <em>Nature</em> <strong>358</strong>, 470–476 (1992).</p>
</div>
<div id="ref-Sternberg-1989">
<p>2. Sternberg, P. W. &amp; Horvitz, H. The combined action of two intercellular signaling pathways specifies three cell fates during vulval induction in c. Elegans. <em>Cell</em> <strong>58</strong>, 679–693 (1989).</p>
</div>
<div id="ref-Beitel-1990">
<p>3. Beitel, G. J., Clark, S. G. &amp; Horvitz, H. R. Caenorhabditis elegans ras gene let-60 acts as a switch in the pathway of vulval induction. <em>Nature</em> <strong>348</strong>, 503–509 (1990).</p>
</div>
<div id="ref-Thomas-1990">
<p>4. Thomas, J. H., Stern, M. J. &amp; Horvitz, H. R. Cell interactions coordinate the development of the c. Elegans egg-laying system. <em>Cell</em> <strong>62</strong>, 1041–1052 (1990).</p>
</div>
<div id="ref-Sternberg-1998">
<p>5. Sternberg, P. W. &amp; Han, M. Genetics of ras signaling in c. Elegans. <em>Trends Genet</em> <strong>14</strong>, 466–472 (1998).</p>
</div>
<div id="ref-Kornfeld-1997">
<p>6. Kornfeld, K. Vulval development in caenorhabditis elegans. <em>Trends Genet</em> <strong>13</strong>, 55–61 (1997).</p>
</div>
<div id="ref-Sundaram-2004">
<p>7. Sundaram, M. V. Vulval development: The battle between ras and notch. <em>Current Biology</em> <strong>14</strong>, R311–R313 (2004).</p>
</div>
<div id="ref-Yoo-2004">
<p>8. Yoo, A., Bais, C. &amp; Greenwald, I. Crosstalk between the egfr and lin-12/notch pathways in c. Elegans vulval development. <em>Science</em> <strong>303</strong>, 663–666 (2004).</p>
</div>
<div id="ref-Schmid-2015">
<p>9. Schmid, T. &amp; Hajnal, A. Signal transduction during c. Elegans vulval development: A neverending story. <em>Current Opinion in Genetics &amp; Development</em> <strong>32</strong>, 1–9 (2015).</p>
</div>
<div id="ref-Gauthier-2017">
<p>10. Gauthier, K. &amp; Rocheleau, C. E. C. Elegans vulva induction: An in vivo model to study epidermal growth factor receptor signaling and trafficking. in <em>ErbB receptor signaling: Methods and protocols</em> (ed. Wang, Z.) vol. 1652 43–61 (Humana Press, 2017).</p>
</div>
<div id="ref-Clark-1992">
<p>11. Clark, S., Stern, M. &amp; Horvritz, H. C. Elegans cell-signalling gene sem-5 encodes a protein with sh2 and sh3 domains. <em>Nature</em> <strong>356</strong>, 340–344 (1992).</p>
</div>
<div id="ref-Chang-2000">
<p>12. Chang, C., Hopper, N. &amp; Sternberg, P. Caenorhabditis elegans sos-1 is necessary for multiple ras-mediated developmental signals. <em>EMBO Journal</em> <strong>19</strong>, 3283–3294 (2000).</p>
</div>
<div id="ref-Han-1993">
<p>13. Han, M., Golden, A., Han, Y. &amp; Sternberg, P. C. Elegans lin-45 raf gene participates in let-60 ras-stimulated vulval differentiation. <em>Nature</em> <strong>363</strong>, 133–140 (1993).</p>
</div>
<div id="ref-Kornfeld-1995">
<p>14. Kornfeld, K., Guan, K. L. &amp; Horvitz, H. R. The caenorhabditis elegans gene mek-2 is required for vulval induction and encodes a protein similar to the protein kinase mek. <em>Genes Dev</em> <strong>9</strong>, 756–768 (1995).</p>
</div>
<div id="ref-Lackner-1994">
<p>15. Lackner, M. R., Kornfeld, K., Miller, L. M., Horvitz, H. R. &amp; Kim, S. K. A map kinase homolog, mpk-1, is involved in ras-mediated induction of vulval cell fates in caenorhabditis elegans. <em>Genes Dev</em> <strong>8</strong>, 160–173 (1994).</p>
</div>
<div id="ref-Leight-2005">
<p>16. Leight, E. R., Glossip, D. &amp; Kornfeld, K. Sumoylation of lin-1 promotes transcriptional repression and inhibition of vulval cell fates. <em>Development</em> <strong>132</strong>, 1047–1056 (2005).</p>
</div>
<div id="ref-Hopper-2000">
<p>17. Hopper, N. A., Lee, J. &amp; Sternberg, P. W. ARK-1 inhibits EGFR signaling in <em>C. elegans</em>. <em>Molecular Cell</em> <strong>6</strong>, 65–75 (2000).</p>
</div>
<div id="ref-Berset-2005">
<p>18. Berset, T. A., Hoier, E. F. &amp; Hajnal, A. The c. Elegans homolog of the mammalian tumor suppressor dep-1/scc1 inhibits egfr signaling to regulate binary cell fate decisions. <em>Genes Dev</em> <strong>19</strong>, 1328–1340 (2005).</p>
</div>
<div id="ref-Alberts-2014">
<p>19. Alberts, B. <em>et al.</em> <em>Molecular biology of the cell</em>. (Garland Science, 2014).</p>
</div>
<div id="ref-Sternberg-2005">
<p>20. Sternberg, P. W. Vulval development. in <em>WormBook</em> (ed. Research Community, T. C. elegans) (WormBook, 2005). doi:<a href="https://doi.org/doi/10.1895/wormbook.1.6.1">doi/10.1895/wormbook.1.6.1</a>.</p>
</div>
<div id="ref-Stein-1999">
<p>21. Stein, L. D. Internet access to the c. Elegans genome. <em>Trends in Genetics</em> <strong>15</strong>, 425–427 (1999).</p>
</div>
<div id="ref-Fire-1998">
<p>22. Fire, A. <em>et al.</em> Potent and specific genetic interference by double-stranded rna in caenorhabditis elegans. <em>Nature</em> <strong>391</strong>, 806–811 (1998).</p>
</div>
<div id="ref-Zon-2015">
<p>23. Zon, J. S. van, Kienle, S., Huelsz-Prince, G., Barkoulas, M. &amp; Oudenaarden, A. van. Cells change their sensitivity to an egf morphogen gradient to control egf-induced gene expression. <em>Nature Communications</em> <strong>6</strong>, 7053 (2015).</p>
</div>
<div id="ref-Kaech-1998">
<p>24. Kaech, S. M., Whitfield, C. W. &amp; Kim, S. K. The lin-2/lin-7/lin-10 complex mediates basolateral membrane localization of the c. Elegans egf receptor let-23 in vulval epithelial cells. <em>Cell</em> <strong>94</strong>, 761–771 (1998).</p>
</div>
<div id="ref-Haag-2014">
<p>25. Haag, P. A. B., Andrea AND Gutierrez. An in vivo egf receptor localization screen in c. Elegans identifies the ezrin homolog erm-1 as a temporal regulator of signaling. <em>PLOS Genetics</em> <strong>10</strong>, 1–13 (2014).</p>
</div>
<div id="ref-Gilbert-2016">
<p>26. Gilbert, S. F. &amp; Barresi, M. J. F. <em>Developmental biology</em>. (Sinauer Associates, 2016).</p>
</div>
<div id="ref-Slack-2018">
<p>27. Slack, J. M. W. <em>Essential developmental biology</em>. (WIley-Blackwell, 2018).</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>In this essay, the names of the actual <em>C. elegans</em> proteins and their (more commonly known) vertebrate/<em>Drosophila</em> homologues are used interchangeably. Here is a translation between them: (<em>Drosophila</em>: <em>C. elegans</em>) EGF: LIN-3; EGFR: LET-23; Grb2: SEM-5; Ras: LET-60; Raf: LIN-45; MEK: MEK-2; MAPK/Erk: MPK-1; Notch: LIN-12; Frizzled and Ryk (WNT-receptors): LIN-17 and LIN-18. <em>C. elegans</em> protein naming follows the <a href="https://wormbase.org//about/userguide/nomenclature#634c2j7f9gi1e05kabmlh8d--10">WormBase convention</a><a href="#fnref1" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/what-is-the-point-of-studying-developmental-biology.html" class="archive-item-link">
                What is the Point of Studying Developmental Biology 
              </a>
              <span class="archive-item-date">
                2020-02-26
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p><strong>Title: What is the point of studying developmental biology?</strong></p>
<p><a href="https://tianyishi2001.github.io/ox/developmental-biology-presentation/#/">Link to presentation</a></p>
<p>The main theme of developmental biology is the fate determination of cells, their division, differention and death, as well as the resulting higher level processes, including morphogenesis and organ formation. I’ll illustrate how this knowledge can help with preventive and public healthcare as well as modern medical research.</p>
<div id="prenatal-care-preventing-teratogenesis" class="section level1">
<h1>Prenatal Care (Preventing Teratogenesis)</h1>
<div id="teratogen-and-congenital-anomaly" class="section level2">
<h2>Teratogen and congenital anomaly</h2>
<p>According to the US’ National Research Council, ~3% of live births suffer from major developmental aberrations, ~70% of neonatal deaths are due to developmental defects.</p>
<ul>
<li>exogenous agents that cause birth defects are teratogens, they can be chemicals, radiations, or viral infections, and so on</li>
<li>thalidomide
<ul>
<li>used to treat anxiety</li>
<li>malformed arms and legs</li>
<li>thee first major evidence that drugs could induce congenital anomalies</li>
<li>led to the development of greater drug regulation and monitoring in many countries</li>
</ul></li>
<li>alcohol interferes with fetal neural development and is shown to cause mental retardation in the newborn
<ul>
<li>fetal alcohol syndrome</li>
<li>abnormal facial development</li>
<li>most prevalent type of congenital mental retardation syndrome in the US</li>
</ul></li>
<li>in pregnant women, Zika virus directly infects the neural progenitor cells of the fetal cortex (vertically transmitted infection), resulting in the death of these cells</li>
<li>Model organisms such as Xenopus and Denio, are often used to screen compound that have a high probability of being hazardous, because the early development of these organisms rely on the same basic paracrine factors and TFs as humans do.
<ul>
<li>studies on model organisms have revealed three probable pathways for alcohol teratogenesis. first, generation of superoxide redicals; second, downregulation of Sonic hedgehog; third, intereference of the cell adhesion molecule L1</li>
</ul></li>
</ul>
</div>
<div id="nutrients" class="section level2">
<h2>Nutrients</h2>
<p>to sustain normal development of the fetus, healthy diet is needed. This is quite obvious, however, there is one vitamin, that cannot be obtained with enough amount just from food, it is folic acid</p>
<ul>
<li>Folic acid
<ul>
<li>it is found to significantly reduce the chance of <strong>neural tube defects in infants</strong>, but only at high doses</li>
<li>it’s recommended, by NHS, that all pregnant women should take a daily supplement of 400 migrograms of folic acid before they’re pregnant and during the first 12 weeeks of pregnancy, when the baby’s spine is developing</li>
</ul></li>
</ul>
</div>
</div>
<div id="stem-cell" class="section level1">
<h1>Stem Cell</h1>
<p>an important subdomain of developmental biology is stem cell biology, which has many applications in research and medicine. here I’ll focus on pluripotent stem cells</p>
<div id="es-cells" class="section level2">
<h2>ES Cells</h2>
<ul>
<li>ES cells are the natural source of pluripotent cells</li>
<li>ICM (embryoblast) and trophoblast</li>
<li>the study of the normal sequence of embryonic inductions yielded the methods for induced differentiation of these cells</li>
<li>derived from the inner cell mass of blastocysts from IVF clinic</li>
<li>harvesting human ES cells means destroying human embryos (5 days after fertilisation)</li>
<li>raised ethical controversy</li>
</ul>
</div>
<div id="ips-cells" class="section level2">
<h2>iPS cells</h2>
<ul>
<li>Reverting to undifferentated state
<ul>
<li>functionallty equivalent to embryonic stem cells</li>
<li>capable of differentiating into almost all cell types</li>
</ul></li>
<li>Shinya Yamanaka factors: Oct4, Klf4, Sox2, c-Myc
<ul>
<li>screened 22 factors</li>
</ul></li>
<li>Regenerative Medicine
<ul>
<li>SMA clinical trial in Japan</li>
</ul></li>
<li>Organoids
<ul>
<li>An organoid is a miniaturized and simplified version of an organ produced in vitro in three dimensions that shows realistic micro-anatomy</li>
</ul></li>
</ul>
</div>
</div>
<div id="cancer-as-a-disease-of-development" class="section level1">
<h1>Cancer as a Disease of Development</h1>
<ul>
<li>carcinogenesis can be viewed as a disease of development because it involes aberrations of the processes that underlie cell division, differentiation and morphogenesis</li>
<li>once it was thought that carcinogenesis and metastasis were caused by cells that had acquired mutations enabling them to proliferate independent of the enviornment.</li>
<li>but it turns out that this is not the complete explanation</li>
<li>cancer cells can actuallt modify their environment, turning it into a cancer-promoting niche</li>
<li>this means the progression of many cancers depends on reciprocal interactions between hte cancer cells and the supporting cells of their tissue environment</li>
<li><p>Indeed, carcinogenesis appears to recapitulate steps of normal development, including the formation of a niche in which to proliferate, using the same or closely related signalling pathways</p></li>
<li>Defects in cell-cell conmmunication
<ul>
<li>studies have shown that tumours can be cuased by altering hte structure of the surrounding tissue, and that these tumors can be suppressed by restoring an appropriate tissue envionment</li>
<li>althogh 80% human tumors are from epithelial cells, the cells that carcinogens act on are often not the epithelial cells themselves, but the mesenchymal stromal cells that surround and sustain the epithelia</li>
<li>there was a study in which normal and carcinogen-treated epithelia and mesenchyme in rat mammary glands are <strong>recombined</strong>, and it turns out that, tumor growth occurred not in carcinogen-treated epithelia, but in epithelia placed in combination with carcinogen-treated mesenchyme</li>
<li>in this case, the mesenchyme fails to give instructions to the epithelia to form normals structures, and epithelial cells exhibited a loose control of cell proliferation</li>
</ul></li>
<li>Defects in paracrine pathways
<ul>
<li>Several key signaling pathways involved in embryogenesis, such as Hedgehog, Notch and Wnt, also have crucial roles in carcinogenesis when improperly activated in adults through sporadic mutatiions or other mechanisms</li>
<li>many tumors, secrete the paracrine signalling factor Shh, which can act in two ways</li>
<li><strong>the same chemicals that can cause teratogenesis by blocking a pathway in embryonic development may be useful in blocking the activation of cancer cells</strong> (cancer stem cells, as I will describe next)</li>
<li>Cyclopamine and other antagonists of the Shh pathway, for instance, appear to be useful in preventing hte generation and proliferation of meedulloblastoma stem cells</li>
</ul></li>
<li>Cancer stem cells
<ul>
<li>an aspect of viewing cancers as diseases of development is that the properties of tumors may emerge because of a population of cells that are analogous to adult stem cells</li>
<li>this is shown in studies on rat intestinal adenomas</li>
<li>the lumen of the mice intestine is made up of villi and crypts</li>
<li>at the bottom of the crypts there reside two important cells, Lrg5+ cells and Paneth cells, Lgr5+ is the stem cell and Paneth cell is the</li>
<li>lineage tracing revealed that
<ul>
<li>the stem cells of rat intestinal adenomas also expresses Lgr5 and has the same interact with P cell as normal stem cells do</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Gilbert-2016">
<p>Gilbert, Scott F., and Michael J. F. Barresi. 2016. <em>Developmental Biology</em>. 11th ed. Sinauer Associates.</p>
</div>
<div id="ref-Slack-2018">
<p>Slack, Jonathan M. W. 2018. <em>Essential Developmental Biology</em>. 3rd Ser. WIley-Blackwell.</p>
</div>
<div id="ref-Song-2011">
<p>Song, Wen AND Wei, Zhou AND Yue. 2011. “Sonic Hedgehog Pathway Is Essential for Maintenance of Cancer Stem-Like Cells in Human Gastric Cancer.” <em>PLOS ONE</em> 6 (3). Public Library of Science: 1–13. <a href="https://doi.org/10.1371/journal.pone.0017687">https://doi.org/10.1371/journal.pone.0017687</a>.</p>
</div>
<div id="ref-Takeo-2018">
<p>Takeo, Makoto, and Takashi Tsuji. 2018. “Organ Regeneration Based on Developmental Biology: Past and Future.” <em>Current Opinion in Genetics &amp; Development</em> 52: 42–47. <a href="https://doi.org/https://doi.org/10.1016/j.gde.2018.05.008">https://doi.org/https://doi.org/10.1016/j.gde.2018.05.008</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/random-notes-on-biioinformatics.html" class="archive-item-link">
                Random Notes on Bioinformatics 
              </a>
              <span class="archive-item-date">
                2020-02-24
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="go" class="section level1">
<h1>Go</h1>
<p><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2903-5" class="uri">https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2903-5</a></p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/pdb-stats.html" class="archive-item-link">
                Visualising the Resolution Revolution of Electron Microscopy 
              </a>
              <span class="archive-item-date">
                2020-02-23
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#getting-all-the-pdb-files"><span class="toc-section-number">1</span> Getting All the PDB Files</a></li>
<li><a href="#parsing-pdb-files"><span class="toc-section-number">2</span> Parsing PDB files</a></li>
<li><a href="#data-analysis-with-r"><span class="toc-section-number">3</span> Data Analysis with R</a><ul>
<li><a href="#prerequisites"><span class="toc-section-number">3.0.1</span> Prerequisites</a></li>
<li><a href="#load-data"><span class="toc-section-number">3.1</span> Load Data</a></li>
<li><a href="#tidying-data"><span class="toc-section-number">3.2</span> Tidying Data</a></li>
<li><a href="#plotting"><span class="toc-section-number">3.3</span> Plotting</a><ul>
<li><a href="#plot-setup"><span class="toc-section-number">3.3.1</span> Plot Setup</a></li>
<li><a href="#median-resolution-of-different-methods-over-time"><span class="toc-section-number">3.3.2</span> Median Resolution of Different Methods Over Time</a></li>
<li><a href="#all-cases-of-x-ray-and-em-structures"><span class="toc-section-number">3.3.3</span> All Cases of X-Ray and EM Structures</a></li>
<li><a href="#boxplots"><span class="toc-section-number">3.3.4</span> Boxplots</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p>Last week I wrote an essay on electron microscopy (EM) and X-ray crystallography. I originally thought X-ray’s superiority in resolution has always been unrivalled, but the situation is changing—the best resolution achived by cryo-EM is now comparable with the average/median resolution achived by X-ray crystallography, which is around 2.3Å (this resolution allows for unambiguous identification of most amino acid residues in proteins). Recently it’s even reported that 1.75Å resolution has been achieved using cryo-EM, which really struck me. Thus, I was motivated to visualise the resolution revolution of cryo-EM. This article is a record of how I downloaded all PDB files from the <a href="https://www.rcsb.org">protein data bank</a> and did some simple analyses on them.</p>
<div id="getting-all-the-pdb-files" class="section level1">
<h1><span class="header-section-number">1</span> Getting All the PDB Files</h1>
<p>All the PDB files are <a href="https://www.rcsb.org/pages/download/ftp">available via PDB’s FTP server</a>. Simply <code>cd</code> to a directory and use <code>wget</code> to download all of them.</p>
<pre class="bash"><code>mkdir pdb; cd pdb; mkdir compressed decompressed; cd compressed
wget ftp://ftp.wwpdb.org/pub/pdb/data/structures/all/pdb/*</code></pre>
<p>Depending on the traffic, The first download will take 2~4 days. After the first download, to sync the local database with the FTP server, add the <code>-N</code> option to <code>wget</code> (essentially this means downloading “new” files only).</p>
<p>The <code>*.ent.gz</code> files downloaded are compressed. You can use <code>gunzip</code> to decompress them to ready-to-use <code>*.ent</code> files (which is written in PDB format), but that will remove the <code>*.gz</code> files, which is inconvenient if you want to keep your database in sync with the remote. Use <code>gzcat</code> (or equivalently <code>gunzip -c</code>) instead to preserve the <code>*.gz</code> files while decompressing:</p>
<pre class="bash"><code>find . -name &quot;*.gz&quot; | xargs -I{} -n1 bash -c &#39;
src={}; dst=../decompressed/$(basename ${src%*.gz})
[ -f $dst ] &amp;&amp; echo &quot;$dst already exists, skipping...&quot; || ( gzcat -cv $src &gt; $dst )&#39;</code></pre>
<p>You can monitor the progress from the verbose output of <code>gzcat</code>:</p>
<pre><code>./pdb1byf.ent.gz:      78.1%
./pdb1byg.ent.gz:      76.3%
./pdb1byh.ent.gz:      77.3%
./pdb1byi.ent.gz:      75.2%
...</code></pre>
</div>
<div id="parsing-pdb-files" class="section level1">
<h1><span class="header-section-number">2</span> Parsing PDB files</h1>
<p>The decompressed <code>*.ent</code> files are in accordance with the PDB format specification, which is accessible <a href="http://www.wwpdb.org/documentation/file-format">here</a>. Many ready-to-use PDB parsers are available, such as in BioPython’s <a href="https://biopython.org/wiki/The_Biopython_Structural_Bioinformatics_FAQ">PDB module</a>. However, I am only interested in three variables: experimental technique, resolution and date, and it would be a overkill to parse entire PDB files using those standard parsers. Instead, I manually wrote a parser (with python) with reference to the format specification:</p>
<pre class="python"><code>#!/usr/bin/env python3.8
import sys
for fn in sys.argv[1:]:
    with open(fn) as f:
        HEAD = f.readline()
        date, id = HEAD[50:59], HEAD[62:66]
        while True:
            line = f.readline()
            if line[:6] == &#39;EXPDTA&#39;:
                method = line[10:].rstrip()
            if line[:6] == &#39;REMARK&#39;:
                if line[11:22] == &#39;RESOLUTION.&#39;:
                    resolution = line[23:30] if line[31:40] == &#39;ANGSTROMS&#39; else &#39;NA&#39;
                    break
    print(&#39;,&#39;.join([id, date, resolution, method]))</code></pre>
<p>Then I parsed the data, 100 files at a time (took 25 min in total):</p>
<pre class="bash"><code>echo &quot;id,date,resolution,method&quot; &gt; date_method_resolution.csv
find decompressed -name &#39;*.ent&#39; | xargs -n100 ./parse &gt;&gt; date_method_resolution.csv</code></pre>
<p>To see whether it is Python that causes this operation slow, I used Go to rewrite the parser:</p>
<details>
<summary>Expand to view</summary>
<p>
<pre class="go"><code>package main

import (
    &quot;bufio&quot;
    &quot;fmt&quot;
    &quot;os&quot;
    &quot;strings&quot;
)

func check(e error) {
    if e != nil {
        panic(e)
    }
}

func main() {

    for _, arg := range os.Args[1:] {

        f, err := os.Open(arg)
        check(err)

        b1 := make([]byte, 81)
        f.Read(b1)
        check(err)
        fmt.Print(string(b1[50:59]) + &quot;,&quot; + string(b1[62:66]) + &quot;,&quot;)

        scanner := bufio.NewScanner(f)
        var method string
        var resolution string
        for scanner.Scan() {
            var line string
            line = scanner.Text()
            if line[:6] == &quot;EXPDTA&quot; {
                method = strings.TrimSpace(line[10:])
            }
            if line[:6] == &quot;REMARK&quot; {
                if line[11:22] == &quot;RESOLUTION.&quot; {
                    if line[31:40] == &quot;ANGSTROMS&quot; {
                        resolution = strings.TrimSpace(line[23:30])
                    } else {
                        resolution = &quot;NA&quot;
                    }
                    break
                }
            }
        }
        fmt.Println(resolution + &quot;,&quot; + method)

        if err := scanner.Err(); err != nil {
            panic(err)
        }
    }
}
</code></pre>
</p>
</details>
<p>And it turns out that Go achived a similar speed as Python, meaning the speed is limited by the file system I/O instead.</p>
<p>The csv file looks like this:</p>
<pre><code>| id   | date      | resolution | method            |
| ---- | --------- | ---------- | ----------------- |
| 100D | 05-DEC-94 |       1.90 | X-RAY DIFFRACTION |
| 101D | 14-DEC-94 |       2.25 | X-RAY DIFFRACTION |
| 101M | 13-DEC-97 |       2.07 | X-RAY DIFFRACTION |</code></pre>
</div>
<div id="data-analysis-with-r" class="section level1">
<h1><span class="header-section-number">3</span> Data Analysis with R</h1>
<div id="prerequisites" class="section level3">
<h3><span class="header-section-number">3.0.1</span> Prerequisites</h3>
<p>First, load the required packages:</p>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>## ── Attaching packages ──────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>## ✓ ggplot2 3.2.1     ✓ purrr   0.3.3
## ✓ tibble  2.1.3     ✓ dplyr   0.8.4
## ✓ tidyr   1.0.2     ✓ stringr 1.4.0
## ✓ readr   1.3.1     ✓ forcats 0.4.0</code></pre>
<pre><code>## ── Conflicts ─────────────── tidyverse_conflicts() ──
## x dplyr::filter() masks stats::filter()
## x dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(lubridate)</code></pre>
<pre><code>## 
## Attaching package: &#39;lubridate&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:base&#39;:
## 
##     date</code></pre>
<pre class="r"><code>library(scales)</code></pre>
<pre><code>## 
## Attaching package: &#39;scales&#39;</code></pre>
<pre><code>## The following object is masked from &#39;package:purrr&#39;:
## 
##     discard</code></pre>
<pre><code>## The following object is masked from &#39;package:readr&#39;:
## 
##     col_factor</code></pre>
</div>
<div id="load-data" class="section level2">
<h2><span class="header-section-number">3.1</span> Load Data</h2>
<p>Parse the csv data file with <code>readr::read_csv()</code>. It is a good practice to specify the data type of each column (<code>c</code> means ‘character’, <code>d</code> means ‘double precision floating number’). Note the date column cannot be directly parsed, and it should be read as a character column and parsed by <code>lubridate::date()</code> in a separate step. The entries without resolution data are filtered out.</p>
<pre class="r"><code>pdb &lt;- read_csv(&#39;./src/pdb/date_method_resolution.csv&#39;,
                col_types = &#39;ccdc&#39;) %&gt;% 
  mutate(date = dmy(date)) %&gt;% 
  filter(!is.na(resolution))</code></pre>
</div>
<div id="tidying-data" class="section level2">
<h2><span class="header-section-number">3.2</span> Tidying Data</h2>
<p>Note that some entries have more than one experimental methods, and individual columns are separated by semicolons, meaning there could be something like <code>X-RAY DIFFRACTION; ELECTRON MICROSCOPY; SOLUTION NMR</code> in the <code>method</code> column. The following code convert such entries into entries each with the first method of the original, dropping the rest:</p>
<pre class="r"><code>single_method &lt;- pdb %&gt;% filter(!str_detect(method, &#39;;&#39;))
multiple_method &lt;- pdb %&gt;% filter(str_detect(method, &#39;;&#39;))


pdb &lt;- bind_rows(single_method, do.call(bind_rows, 
  lapply(1:length(multiple_method[[1]]),function(i) {
    original_row = multiple_method[i,]
    mutate(original_row, method=str_split(original_row[[&#39;method&#39;]], &#39;; &#39;)[[1]][1])
    }
  )
))</code></pre>
<p>You can also use a for loop (shown below) to bind new rows to <code>pdb</code>, but this is noticeably slower than the <code>lapply()</code> and <code>do.call()</code> equivalent (shown above). In R, use those vectorised and highly efficient functions whenever possible (they should be able to replace almost all for loops).</p>
<pre class="r"><code>pdb &lt;- single_method
for (i in 1:length(multiple_method[[1]])) {
  original_row = multiple_method[i,]
  bind_rows(pdb, mutate(original_row, method=str_split(original_row[[&#39;method&#39;]], &#39;; &#39;)[[1]][1]))
}</code></pre>
</div>
<div id="plotting" class="section level2">
<h2><span class="header-section-number">3.3</span> Plotting</h2>
<p>I used the <code>ggplot2</code> package for plotting.</p>
<div id="plot-setup" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Plot Setup</h3>
<p><code>ggplot2::theme_set()</code> sets global themes for all plots.</p>
<p><code>ggplot2</code> doesn’t have a built-in <code>reverse_log10</code> scaling, which will be needed, so I used <code>scales::trans_new()</code> to make this.</p>
<pre class="r"><code>theme_set(
  theme_bw()+
  theme(
    text = element_text(family = &#39;Courier&#39;),
    plot.title = element_text(hjust = 0.5)
  )
)

reverselog_trans &lt;- function(base = exp(1)) {
  trans &lt;- function(x) -log(x, base)
  inv &lt;- function(x) base^(-x)
  trans_new(paste0(&quot;reverselog-&quot;, format(base)), trans, inv, 
            log_breaks(base = base), 
            domain = c(1e-100, Inf))
}</code></pre>
</div>
<div id="median-resolution-of-different-methods-over-time" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Median Resolution of Different Methods Over Time</h3>
<p>This plot shows the median resolution of different structure-solving methods every year. Size represents <code>log(number_of_cases)</code>.</p>
<pre class="r"><code>pdb %&gt;% mutate(year = year(date)) %&gt;% 
  group_by(year, method) %&gt;% 
  summarise(
    median = median(resolution, na.rm = TRUE),
    n = n()) %&gt;% 
  mutate( alpha = ifelse(method==&#39;X-RAY DIFFRACTION&#39;, &#39;A&#39;, &#39;B&#39;)) %&gt;% 
  ggplot(aes(year, median, color=method, size=log(n), alpha=alpha))+
  geom_point()+
  scale_y_continuous(breaks = seq(0,25,5))+
  scale_alpha_discrete(range=c(0.7, 0.9), guide=FALSE)+
  labs(title=&#39;Median Resolution of Different Methods Over Time&#39;)+
  scale_color_brewer(palette = &#39;Set2&#39;)</code></pre>
<p><img src="src/pdb/plot/dot.png" width="100%" /></p>
</div>
<div id="all-cases-of-x-ray-and-em-structures" class="section level3">
<h3><span class="header-section-number">3.3.3</span> All Cases of X-Ray and EM Structures</h3>
<p>These plots show the resolution and date of all cases as dots of EM and X-Ray Crystallography, respectively. Linear fits are added.</p>
<pre class="r"><code>ggplot(filter(pdb, method ==&#39;ELECTRON MICROSCOPY&#39;, date&gt;ymd(20000101), resolution &lt; 20), aes(date, resolution))+
    geom_point(alpha=0.1)+
    labs(title = &#39;Resolution of EM Structures Over Time&#39;)+
    scale_y_continuous(limits = c(0, 20), breaks = c(2, 3, 4, seq(0, 20, 5)), minor_breaks = c(7.5, 12.5, 17.5, seq(2, 5, 0.5)))+
    geom_smooth(method = &#39;lm&#39;)</code></pre>
<p><img src="src/pdb/plot/overtime-em.png" width="100%" /></p>
<pre class="r"><code>ggplot(filter(pdb, method ==&#39;X-RAY DIFFRACTION&#39;), aes(date, resolution))+
    geom_point(alpha=0.1)+
    scale_y_continuous(limits = c(0, 10), breaks = seq(0, 10, 2), minor_breaks = c(seq(1,9,2), seq(1,3,0.1)))+
    geom_smooth(method = &#39;lm&#39;)+
    labs(title = &#39;Resolution of X-Ray Crystallography Structures Over Time&#39;)</code></pre>
<p><img src="src/pdb/plot/overtime-x-ray.png" width="100%" /></p>
<p>EM structures show a strong trend of improvement in resolution over time. For X-Ray crystallography, however, although the maximum resolution improves over time, the number of published low-resolution structures also increases, making the median virtually unchanged over time.</p>
</div>
<div id="boxplots" class="section level3">
<h3><span class="header-section-number">3.3.4</span> Boxplots</h3>
<p>Boxplots summarise the resolution, grouped by year, of different structural biology methods. Here the <code>reverse_log10</code> scale is used, so that lower resolution values correspond to ‘high positions’ in the plot, which is more intuitive:</p>
<pre class="r"><code>boxPlotScale = scale_y_continuous(
  minor_breaks = c(1:10, seq(10, 100, 10), seq(0.1, 1, 0.1)), 
  breaks=c(0.5, 1, 2, 3, 5, 10, 20, 60),
  trans=reverselog_trans(10))</code></pre>
<pre class="r"><code>ggplot(mutate(pdb, year = year(date)), aes(year, resolution, group=year))+
    geom_boxplot()+
    facet_wrap(~method)+
    boxPlotScale+
    scale_x_reverse()+
    coord_flip()+
    labs(title = &#39;Comparison of Resolution of Different Methods Over Time&#39;)</code></pre>
<p><img src="src/pdb/plot/hist_all.png" width="100%" /></p>
<p>To compare EM and X-ray crystallography:</p>
<pre class="r"><code>ggplot(mutate(pdb, year = year(date)) %&gt;% filter(method %in% c(&#39;X-RAY DIFFRACTION&#39;, &#39;ELECTRON MICROSCOPY&#39;)), aes(year, resolution, group=year))+
    geom_boxplot()+
    facet_wrap(~method, ncol = 1)+
    boxPlotScale+
    labs(title = &#39;Comparison of Resolution of EM and X-Ray Crystallography Over Time&#39;)</code></pre>
<p><img src="src/pdb/plot/hist_2.png" width="100%" /></p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/three-biophysical-methods.html" class="archive-item-link">
                Three Biophysical Methods 
              </a>
              <span class="archive-item-date">
                2020-02-22
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/htmlwidgets/htmlwidgets.js"></script>
<script src="../rmarkdown-libs/plotly-binding/plotly.js"></script>
<script src="../rmarkdown-libs/typedarray/typedarray.min.js"></script>
<script src="../rmarkdown-libs/jquery/jquery.min.js"></script>
<link href="../rmarkdown-libs/crosstalk/css/crosstalk.css" rel="stylesheet" />
<script src="../rmarkdown-libs/crosstalk/js/crosstalk.min.js"></script>
<link href="../rmarkdown-libs/plotly-htmlwidgets-css/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="../rmarkdown-libs/plotly-main/plotly-latest.min.js"></script>

<div id="TOC">
<ul>
<li><a href="#circular-dichorism-cd"><span class="toc-section-number">1</span> Circular Dichorism (CD)</a><ul>
<li><a href="#physical-principles"><span class="toc-section-number">1.1</span> Physical Principles</a></li>
<li><a href="#experimental-setup"><span class="toc-section-number">1.2</span> Experimental Setup</a></li>
<li><a href="#ss"><span class="toc-section-number">1.3</span> Application of CD</a></li>
<li><a href="#other-cd-based-techniques"><span class="toc-section-number">1.4</span> Other CD-based Techniques</a></li>
</ul></li>
<li><a href="#small-angle-x-ray-scattering-saxs"><span class="toc-section-number">2</span> Small-angle X-ray Scattering (SAXS)</a><ul>
<li><a href="#physical-principles-1"><span class="toc-section-number">2.1</span> Physical Principles</a></li>
<li><a href="#experimentation-and-data-processing"><span class="toc-section-number">2.2</span> Experimentation and Data Processing</a></li>
</ul></li>
<li><a href="#analytical-ultracentrifuge-auc"><span class="toc-section-number">3</span> Analytical Ultracentrifuge (AUC)</a><ul>
<li><a href="#choice-of-optical-system"><span class="toc-section-number">3.1</span> Choice of Optical System</a></li>
<li><a href="#experiments"><span class="toc-section-number">3.2</span> Experiments</a><ul>
<li><a href="#sedimentation-velocity-sv"><span class="toc-section-number">3.2.1</span> Sedimentation Velocity (SV)</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="circular-dichorism-cd" class="section level1">
<h1><span class="header-section-number">1</span> Circular Dichorism (CD)</h1>
<p>Reference: <span class="citation">Ranjbar and Gill (2009)</span>; <span class="citation">Kelly, Jess, and Price (2005)</span>;<span class="citation">Whitmore and Wallace (2008)</span></p>
<p>Circular dichorism measures the difference in absorption of left-hand and right-hand circularly polarised light by optically active (chiral) molecules (usually biological macromolecules). It serves as a rapid way to assess structural information on proteins, carbohydrates, nucleic acids, pharmaceuticals, liquiud crystals, etc., in the solution state.</p>
<div id="physical-principles" class="section level2">
<h2><span class="header-section-number">1.1</span> Physical Principles</h2>
<p>Electromagnetic waves (EM waves) has an electrical (E) component and a magnetic (M) component perpendicular to each other and both are perpendicular to the axis of propagation. Usually, only the E component is depicted for simplicity.</p>
<p>Most light sources have E components in all orientations, and when they pass through a slit, they become plane polarised, and can be represented by a simple sinusoidal wave. Adding up two plane polarised light perpendicular to each other produces interesting resultant waves, which can be visualised with <strong>plotly</strong> in R <span class="citation">(Sievert 2018)</span> (which is an uncommon use ‘invented’ by me<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>). Assuming we are superposing two sinusoidal waves of the same amplitude and frequency with various phase differences, we can then compute the pattern of the resultant wave as a function of the phase difference:</p>
<pre class="r"><code>library(plotly)
library(tibble)

calc_waves &lt;- function(phase_diff = 0){
  # compute y and z values for given x values of the waves 
  # first arbitrarily choose evenly-spaced points on the x_axis
  x_axis = x=seq(1,20,0.1) 
  # compute the y, z values of the first sin wave on the xy plane
  sin1 &lt;- tibble(x=x_axis,y=sin(x),z=0)
  # compute the y, z values of the second sin wave on the xz plane, 
  # taking the phase difference into account
  sin2 &lt;- tibble(x=x_axis,y=0,z=sin(x + phase_diff))
  # compute the axes of the resultant wave
  # each y and z value is the sum of corresponding values in the 2 component waves
  resultant &lt;- tibble(x=x_axis, y = sin1$y+sin2$y, z = sin1$z+sin2$z)
  
  return(list(sin1, sin2, resultant))
} 

plot_waves &lt;- function(wave_list, components = c(&#39;sin1&#39;, &#39;sin2&#39;), resultant = &#39;resultant&#39;){
  # plotting
  plot_ly() %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=components[1], data = wave_list[[1]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;) %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=components[2], data = wave_list[[2]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;) %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=resultant, data = wave_list[[3]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;)
}</code></pre>
<p>When phase difference is zero, the resultant wave is also sinusoidal, and the plane in which it resides is oriented at 45<span class="math inline">\(^\circ\)</span> to each component wave, as shown in Fig. <a href="#fig:si">1.1</a> (<em>this is an interactive HTML widget so you can rotate, drag and zoom! Unfortunately it cannot be rendered in LaTeX, so please visit ../tutorial/three-biophysical-methods.html if you are reeading the LaTeX PDF output</em>).</p>
<pre class="r"><code>calc_waves(phase_diff=0) %&gt;% plot_waves()</code></pre>
<div class="figure"><span id="fig:si"></span>
<div id="htmlwidget-1" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1">{"x":{"visdat":{"629ff04c89d":["function () ","plotlyVisDat"],"629f30ec31f":["function () ","data"],"629f3e2c29ca":["function () ","data"],"629f456baca9":["function () ","data"]},"cur_data":"629f456baca9","attrs":{"629f30ec31f":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f3e2c29ca":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f456baca9":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"name":"resultant","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.1: 0 phase difference results in a sinusoidal wave
</p>
</div>
<p>As shown in <a href="#fig:lh">1.2</a>, When the phase difference is <span class="math inline">\(\pi/2\)</span>, the resultant wave is helical and is said to be <strong>circularly polarised</strong> (for other phase differences, the pattern is helical but ellipised.</p>
<pre class="r"><code>L &lt;- calc_waves(phase_diff=pi/2)
plot_waves(L, resultant = &#39;resultant: left circularly polarised&#39;)</code></pre>
<div class="figure"><span id="fig:lh"></span>
<div id="htmlwidget-2" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2">{"x":{"visdat":{"629f2de37aaa":["function () ","plotlyVisDat"],"629f318a466f":["function () ","data"],"629f6d163ad1":["function () ","data"],"629f4e6fa74a":["function () ","data"]},"cur_data":"629f4e6fa74a","attrs":{"629f318a466f":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f6d163ad1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f4e6fa74a":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant: left circularly polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"resultant: left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.2: Formation of left circularly polarised wave.
</p>
</div>
<pre class="r"><code>L &lt;- L[[3]] # the third tibble represents the resultant, which will be used later</code></pre>
<p>Note the handedness of the resultant wave: when the phase difference is <span class="math inline">\(\pi/2\)</span>, it is left-handed and it becomes right-handed when the phase difference is <span class="math inline">\(-\pi/2\)</span>. The latter is drawn in Fig. <a href="#fig:rh">1.3</a>.</p>
<pre class="r"><code>R &lt;- calc_waves(phase_diff=-pi/2)
plot_waves(R, resultant = &#39;resultant: right circularly polarised&#39;)</code></pre>
<div class="figure"><span id="fig:rh"></span>
<div id="htmlwidget-3" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-3">{"x":{"visdat":{"629f25307c5d":["function () ","plotlyVisDat"],"629f1234d0be":["function () ","data"],"629f49747348":["function () ","data"],"629f7c3ca1a4":["function () ","data"]},"cur_data":"629f7c3ca1a4","attrs":{"629f1234d0be":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f49747348":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f7c3ca1a4":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant: right circularly polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"resultant: right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.3: Formation of right circularly polarised wave.
</p>
</div>
<pre class="r"><code>R &lt;- R[[3]]</code></pre>
<p>As shown in Fig. <a href="#fig:plane-polarised">1.4</a> If we add up the left and right circularly polarised light, the resultant is a plane polarised light. Thus it can be said that a plane polarised light can be viewed as being made up of two circularly polarised components of equal magnitude and frequency.</p>
<pre class="r"><code>plot_waves(list(L, R, 
  resultant=tibble(x=L$x, y = L$y+R$y, z = L$z+R$z)),
  components = c(&#39;left circularly polarised&#39;, &#39;right circularly polarised&#39;),
  resultant = &#39;plane polarised&#39;)</code></pre>
<div class="figure"><span id="fig:plane-polarised"></span>
<div id="htmlwidget-4" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4">{"x":{"visdat":{"629f2122d62c":["function () ","plotlyVisDat"],"629f761af3b2":["function () ","data"],"629f637567b1":["function () ","data"],"629f30eac87a":["function () ","data"]},"cur_data":"629f30eac87a","attrs":{"629f761af3b2":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"left circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f637567b1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"right circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f30eac87a":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"plane polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[1.68294196961579,1.78241472012287,1.86407817193445,1.92711637083439,1.97089945997692,1.99498997320811,1.99914720608301,1.98332962090494,1.94769526175639,1.89260017537483,1.81859485365136,1.72641873329775,1.61699280763918,1.49141042435344,1.3509263611023,1.19694428820791,1.03100274364293,0.85475976046766,0.66997630031181,0.478498658427964,0.282240016119734,0.083161324866581,-0.11674828685516,-0.315491388286497,-0.511082204053663,-0.70156645537924,-0.885040886589705,-1.05967228181699,-1.22371578188544,-1.37553231836795,-1.51360499061586,-1.63655422212882,-1.74315154482718,-1.83233187349891,-1.90320414777903,-1.95506023533019,-1.98738200726693,-1.9998465151282,-1.99232921767168,-1.96490522524866,-1.91784854932628,-1.85162936465546,-1.76690931144031,-1.6645348844478,-1.54552897511197,-1.41108065114078,-1.26253327574464,-1.10137108519528,-0.929204358827513,-0.747753329660472,-0.558830996397852,-0.36432500854419,-0.166178805634993,0.0336278009687012,0.233098409700987,0.430239976175631,0.623082727026757,0.809699841233197,0.988226702277218,1.1568795287764,1.31397319743758,1.45793808025175,1.58733572769831,1.70087324125713,1.79741619162325,1.87599995354948,1.93583934406297,1.976336467754,1.99708669074921,1.99788268367954,1.97871649324676,1.93977962169017,1.88146111335955,1.80434366751259,1.70919781617656,1.59697422524698,1.46879419574822,1.32593846016437,1.16983438578352,1.00204171291577,0.824236970483513,0.638196724698704,0.445779828200492,0.248908847014123,0.0495508509067155,-0.150302240923619,-0.348653562445959,-0.543521252821888,-0.732958258503857,-0.915071787550643,-1.08804222177874,-1.25014129778576,-1.39974937518709,-1.53537161952716,-1.65565293817131,-1.75939151994334,-1.84555084322561,-1.91327003254038,-1.96187246013298,-1.99087250661275,-1.99998041310141,-1.98910517640798,-1.95835545830263,-1.90803849980418,-1.83865705132935,-1.75090434937686,-1.64565718993742,-1.52396716783806,-1.38705016955424,-1.23627422447407,-1.07314583600087,-0.8992949290692,-0.716458564473654,-0.526463582731602,-0.331208350896619,-0.132643794702401,0.0672460944422769,0.266464082839888,0.463019650203078,0.654948878275386,0.840334073653282,1.01732292874475,1.18414702941445,1.3391395243932,1.4807517799049,1.60756885310324,1.71832362971299,1.81190948461693,1.88739133888821,1.94401500278995,1.98121471138974,1.99861877749584,1.99605330543272,1.97354392854923,1.93131555309856,1.86979011104937,1.78958234428101,1.69149366228587,1.57650413475063,1.44576269902395,1.30057568031423,1.14239373931997,0.972797377707596,0.793481146261224,0.606236713491405,0.412934963875593,0.215507304598885,0.0159263675718711,-0.183813700455363,-0.381717162748379,-0.575806633330131,-0.764142834368018,-0.944843972796939,-1.11610454257356,-1.27621336469589,-1.42357068473825,-1.5567041570686,-1.6742835560395,-1.77513406716301,-1.85824802546874,-1.92279498375911,-1.96813001016329,-1.99380013208319,-1.99954886214602,-1.98531876094127,-1.95125201093632,-1.89768899583625,-1.82516489958237,-1.73440435897116,-1.62631422332297,-1.50197449354335,-1.362627531111,-1.20966564481257,-1.04461717925346,-0.869131244143787,-0.684961236939225,-0.493947323473242,-0.297998051628398,-0.0990712817567348,0.10084537561363,0.299754419325905,0.49566841596592,0.686629857639791,0.870730720745786,1.0461315303154,1.2110797394392,1.36392724013627,1.5031468307043,1.62734747501421,1.73528820128334,1.82589050145526],"z":[1.11022302462516e-16,1.11022302462516e-16,-1.11022302462516e-16,3.33066907387547e-16,1.11022302462516e-16,1.2490009027033e-16,1.21430643318376e-16,1.11022302462516e-16,3.33066907387547e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,3.33066907387547e-16,0,0,1.11022302462516e-16,0,0,0,1.11022302462516e-16,-1.11022302462516e-16,0,-3.33066907387547e-16,-1.11022302462516e-16,2.22044604925031e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-5.55111512312578e-17,-1.38777878078145e-16,-1.2490009027033e-16,-1.21430643318376e-16,-1.2490009027033e-16,-1.38777878078145e-16,-1.11022302462516e-16,-1.11022302462516e-16,-5.55111512312578e-17,-1.11022302462516e-16,-1.11022302462516e-16,0,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,0,0,0,3.33066907387547e-16,-2.22044604925031e-16,0,1.11022302462516e-16,1.11022302462516e-16,6.66133814775094e-16,-5.55111512312578e-16,1.11022302462516e-16,1.66533453693773e-16,1.66533453693773e-16,1.0547118733939e-15,-7.7715611723761e-16,1.2490009027033e-16,1.2490009027033e-16,1.11022302462516e-16,8.32667268468867e-17,1.66533453693773e-16,1.66533453693773e-16,1.11022302462516e-16,0,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,0,1.11022302462516e-16,1.11022302462516e-16,0,0,0,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.38777878078145e-16,-1.2490009027033e-16,-1.22298005056365e-16,-1.2490009027033e-16,-1.38777878078145e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,0,0,0,0,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,0,2.22044604925031e-16,2.22044604925031e-16,2.22044604925031e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.17961196366423e-16,1.2490009027033e-16,1.11022302462516e-16,1.11022302462516e-16,-1.55431223447522e-15,-1.49880108324396e-15,2.22044604925031e-16,1.66533453693773e-15,1.11022302462516e-16,-9.99200722162641e-16,-8.88178419700125e-16,1.11022302462516e-16,7.7715611723761e-16,0,-3.33066907387547e-16,-2.22044604925031e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,5.55111512312578e-16,6.66133814775094e-16,7.7715611723761e-16,-1.11022302462516e-15,1.11022302462516e-15,1.22124532708767e-15,1.33226762955019e-15,1.33226762955019e-15,-1.66533453693773e-15,1.55431223447522e-15,1.60982338570648e-15,1.63757896132211e-15,-1.90125692967058e-15,-1.90125692967058e-15,-1.88737914186277e-15,1.60982338570648e-15,3.21964677141295e-15,-3.38618022510673e-15,-3.21964677141295e-15,-2.99760216648792e-15,2.55351295663786e-15,2.44249065417534e-15,-2.22044604925031e-15,-1.88737914186277e-15,-1.55431223447522e-15,1.11022302462516e-15,7.7715611723761e-16,-5.55111512312578e-16,-2.22044604925031e-16,2.22044604925031e-16,-5.55111512312578e-16,-8.88178419700125e-16,1.22124532708767e-15,1.66533453693773e-15,1.88737914186277e-15,-2.1094237467878e-15,-2.33146835171283e-15,2.77555756156289e-15,2.99760216648792e-15,3.21964677141295e-15,-3.10862446895044e-15],"name":"plane polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.4: Addition of two identical waves of oppsite handedness results in a plane polarised wave.
</p>
</div>
<p>CD measures differential absorption of these components. If the two components are absorbed to the same extent, clearly the resultant will still be planar as shown above. If there is differential absorption, as I simulate in Fig. <a href="#fig:diff-absorb">1.5</a>, the resultant wave will be elliptical polarised. Here the R component is absorbed more than the L component, and the resultant ellipsed wave is left-handed. The CD signal (formally <span class="math inline">\(\Delta A=\Delta A_\text{L}-\Delta A_\text{R}\)</span>) is generally reported in terms of the ellipticity, <span class="math inline">\(\theta\)</span>, of this resultant wave. <span class="math inline">\(\theta=\arctan(b/a)\)</span> where b and a are the minor and major axes of the ellipse. <span class="math inline">\(\theta\)</span> can be easily converted to <span class="math inline">\(\Delta A\)</span> by the simple relationship: <span class="math inline">\(\theta=32.98\Delta A\)</span>. The CD spectrum is obtained when the CD signal <span class="math inline">\(\theta\)</span> or <span class="math inline">\(\Delta A\)</span> is measured as a function of wavelength.</p>
<pre class="r"><code>L1 &lt;- L %&gt;% mutate(y = 0.9 * y, z = 0.9 * z) # 10% absorption of L
R1 &lt;- R %&gt;% mutate(y = 0.7 * y, z = 0.7 * z) # 30% absorption of R
plot_waves(list(L, R, 
  resultant=tibble(x=L1$x, y = L1$y+R1$y, z = L1$z+R1$z)),
  components = c(&#39;left circularly polarised&#39;, &#39;right circularly polarised&#39;),
  resultant = &#39;elliptical polarised&#39;)</code></pre>
<div class="figure"><span id="fig:diff-absorb"></span>
<div id="htmlwidget-5" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5">{"x":{"visdat":{"629f7add96f6":["function () ","plotlyVisDat"],"629f69e22f7e":["function () ","data"],"629f9c2f81":["function () ","data"],"629fdf2be77":["function () ","data"]},"cur_data":"629fdf2be77","attrs":{"629f69e22f7e":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"left circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f9c2f81":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"right circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629fdf2be77":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"elliptical polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[1.34635357569263,1.4259317760983,1.49126253754756,1.54169309666751,1.57671956798154,1.59599197856649,1.59931776486641,1.58666369672395,1.55815620940511,1.51408014029986,1.45487588292109,1.3811349866382,1.29359424611134,1.19312833948275,1.08074108888184,0.95755543056633,0.824802194914343,0.683807808374128,0.535981040249448,0.382798926742371,0.225792012895788,0.0665290598932648,-0.0933986294841281,-0.252393110629198,-0.408865763242931,-0.561253164303392,-0.708032709271764,-0.847737825453589,-0.978972625508351,-1.10042585469436,-1.21088399249269,-1.30924337770306,-1.39452123586174,-1.46586549879913,-1.52256331822323,-1.56404818826416,-1.58990560581354,-1.59987721210256,-1.59386337413734,-1.57192418019893,-1.53427883946102,-1.48130349172437,-1.41352744915224,-1.33162790755824,-1.23642318008958,-1.12886452091263,-1.01002662059571,-0.88109686815622,-0.74336348706201,-0.598202663728378,-0.447064797118281,-0.291460006835352,-0.132943044507994,0.026902240774961,0.18647872776079,0.344191980940505,0.498466181621406,0.647759872986557,0.790581361821774,0.92550362302112,1.05117855795006,1.1663504642014,1.26986858215865,1.3606985930057,1.4379329532986,1.50079996283958,1.54867147525038,1.5810691742032,1.59766935259937,1.59830614694364,1.58297319459741,1.55182369735214,1.50516889068764,1.44347493401007,1.36735825294125,1.27757938019758,1.17503535659858,1.06075076813149,0.935867508626819,0.801633370332615,0.659389576386811,0.510557379758963,0.356623862560393,0.199127077611299,0.0396406807253724,-0.120241792738895,-0.278922849956767,-0.434817002257511,-0.586366606803085,-0.732057430040514,-0.870433777422992,-1.00011303822861,-1.11979950014967,-1.22829729562173,-1.32452235053705,-1.40751321595467,-1.47644067458049,-1.5306160260323,-1.56949796810639,-1.5926980052902,-1.59998433048113,-1.59128414112638,-1.56668436664211,-1.52643079984334,-1.47092564106348,-1.40072347950149,-1.31652575194993,-1.21917373427045,-1.1096401356434,-0.989019379579253,-0.858516668800696,-0.71943594325536,-0.573166851578923,-0.421170866185282,-0.264966680717295,-0.106115035761921,0.0537968755538215,0.21317126627191,0.370415720162462,0.523959102620309,0.672267258922626,0.8138583429958,0.947317623531559,1.07131161951456,1.18460142392392,1.28605508248259,1.37465890377039,1.44952758769354,1.50991307111057,1.55521200223196,1.58497176911179,1.59889502199667,1.59684264434618,1.57883514283938,1.54505244247884,1.49583208883949,1.43166587542481,1.35319492982869,1.26120330780051,1.15661015921916,1.04046054425139,0.913914991455979,0.778237902166077,0.634784917008979,0.484989370793124,0.330347971100475,0.172405843679108,0.0127410940574969,-0.147050960364291,-0.305373730198703,-0.460645306664104,-0.611314267494415,-0.755875178237551,-0.892883634058847,-1.02097069175672,-1.1388565477906,-1.24536332565488,-1.3394268448316,-1.42010725373041,-1.48659842037499,-1.53823598700729,-1.57450400813063,-1.59504010566655,-1.59963908971682,-1.58825500875301,-1.56100160874905,-1.518151196669,-1.4601319196659,-1.38752348717693,-1.30105137865838,-1.20157959483468,-1.0901020248888,-0.967732515850055,-0.83569374340277,-0.69530499531503,-0.54796898955138,-0.395157858778593,-0.238398441302718,-0.0792570254053879,0.0806763004909036,0.239803535460724,0.396534732772736,0.549303886111833,0.696584576596629,0.836905224252319,0.968863791551361,1.09114179210902,1.20251746456344,1.30187798001137,1.38823056102667,1.4607124011642],"z":[0.108060461173628,0.0907192242851156,0.0724715508953346,0.0534997657249178,0.0339934285800483,0.0141474403335407,-0.00583990446025767,-0.0257688988591049,-0.0454404189386172,-0.0646579133727006,-0.0832293673094284,-0.100969220919971,-0.117700223451069,-0.133255204255965,-0.147478743108249,-0.160228723109387,-0.171377750673789,-0.180814428403412,-0.188444468133732,-0.194191633029918,-0.197998499320089,-0.199827030054656,-0.199658955158951,-0.197495953981773,-0.193359638515892,-0.187291337458159,-0.17935168326683,-0.169620006342082,-0.158193542382883,-0.145186460840028,-0.130728724172722,-0.114964789306654,-0.09805216426814,-0.080159834415995,-0.0614665739956839,-0.0421591598861561,-0.0224305053870111,-0.00247773269257821,0.0174997966878894,0.037302473884515,0.0567324370926452,0.0755955485425961,0.0937033342600754,0.110874867235832,0.126938575188527,0.141733954858252,0.15511317570205,0.166942556967832,0.177103903388264,0.185495686148807,0.192034057330073,0.196653687688517,0.199308419404643,0.199971727276683,0.198636983751639,0.195317525145605,0.190046518391706,0.182876629647064,0.173879498069965,0.163145020025072,0.150780450868661,0.136909333288562,0.12167026290645,0.105215503476221,0.0877094655148782,0.0693270635670053,0.0502519685164519,0.0306747724075722,0.0107910841125299,-0.00920042512790729,-0.0291000067617226,-0.0487088307471585,-0.0678309721967668,-0.0862753689941241,-0.103857730823337,-0.120402380536965,-0.135744009464003,-0.14972932911948,-0.162218602812331,-0.173087041848222,-0.182226052376935,-0.189544320426222,-0.194968724280833,-0.198445065090521,-0.199938608407041,-0.199434431239276,-0.196937571158825,-0.192472975966262,-0.186085254420951,-0.177838230525072,-0.167814305815291,-0.156113636033837,-0.14285313040544,-0.128165283518999,-0.112196851485446,-0.0951073855991986,-0.0770676381543657,-0.0582578563442689,-0.0388659812910671,-0.0190857702001902,0.000885139597610059,0.0208472053731396,0.0406009727637503,0.0599490686554029,0.0786981732695781,0.0966609517506011,0.113657925953595,0.129519267730775,0.144086495798168,0.157214059228208,0.168770791746498,0.178641222301865,0.186726728814928,0.192946523577322,0.197238460455773,0.199559655835716,0.199886917100201,0.198216974362851,0.194566513139487,0.188972007631972,0.181489356290039,0.172193323292461,0.16117679152809,0.148549834540734,0.134438616710694,0.118984132661979,0.10234079849063,0.0846749088901329,0.0661629755898095,0.0469899637079647,0.0273474436415668,0.00743167695816502,-0.0125583445848166,-0.0324228872999436,-0.0519634712427511,-0.0709848533577424,-0.0892969782824548,-0.106716877317824,-0.123070496590943,-0.138194436143825,-0.151937582571765,-0.164162618898535,-0.174747396602216,-0.183586156082858,-0.190590583377436,-0.195690692563777,-0.198835525036763,-0.199993658669868,-0.199153521774658,-0.196323508721277,-0.191531896064676,-0.184826560014625,-0.176274498072446,-0.165961159614131,-0.153989592108413,-0.140479411500542,-0.125565607049276,-0.109397192558845,-0.0921357174822741,-0.0739536527726332,-0.0550326676103179,-0.0355618142246217,-0.0157356389463697,0.0042477616347275,0.0241887199856939,0.0438879926422933,0.0631487509838513,0.0817785478797733,0.0995912405576805,0.116408850480423,0.132063341648818,0.146398299561791,0.159270494058383,0.170551310426173,0.180128034476953,0.187904978749652,0.193804438587811,0.197767468538829,0.199754471317442,0.199745593448701,0.197740923637333,0.193760491881441,0.187844069339375,0.180050770949462,0.170458464773094,0.159162993962787,0.146277219129098,0.131929890674695,0.11626436236289,0.099437158974243,0.081616412362676],"name":"elliptical polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.5: Differential absorption of circularly polarised light. Here 10% of the L component and 30% of the R component is absorbed (90% and 70% transmitted, respectively).
</p>
</div>
<p>A CD signal will be observed when a chromophore is chiral (either intrinsically chiral, bonded to a chiral atom, or due to asymmetric enviornment). In proteins, such chromophores include the peptide bond (absorption below 240 nm), aromatic amino acid side chains (absorption in the range 260~320 nm) and disulphide bonds (weak absorptoin around 260 nm).</p>
</div>
<div id="experimental-setup" class="section level2">
<h2><span class="header-section-number">1.2</span> Experimental Setup</h2>
<p>The EM wave used in CD is UV, usually with <span class="math inline">\(\lambda\)</span> in the range 170~320 nm. Traditionally Xe arc lamps have been used as the light source of UV, but they can hardly achieve a wavelength below 180 nm. Now, high frequency UV can be generated by modern synchrotrons, and extending of CD data into the far UV region improves reliability of secondary structure prediction (see Section <a href="#ss">1.3</a>).</p>
<p>There are various methods by which the CD effect can be measured in a spectropolarimeter:</p>
<ul>
<li>modulation, in which the incident readiation is continuously switched between the L and R ocmponents</li>
<li>direct subtraction, in whiich the absorbances of the 2 components are measured separately and subtracted from each other</li>
<li>ellipsometric, in which the ellipticity of the transmitted radiation is measured</li>
</ul>
<p>The modulation method is most commonly used. The experimental setup is described as follows:</p>
<ul>
<li>plane polarised light is split into the L and R components by passage through a modulator subjected to an alternating electric field</li>
<li>the modulator normally consists of a piezoelectric quartz crystal and a thin plate of isotropic material (e.g. fused silica) tightly coupled to the crystal.</li>
<li>the alternating electric field induces structural changes in the quartz crystal; these make the plate transmit circularly polarised light at the extremes of the field</li>
<li>as the transmitted radiation of switched between L and R components, these are detecred in turn by the photomultiplier.</li>
</ul>
</div>
<div id="ss" class="section level2">
<h2><span class="header-section-number">1.3</span> Application of CD</h2>
<p>Far UV and near UV regions of CD give different information about a protein.</p>
<p>Absorption in the far UV region (170-250 nm) is mainly due to the <span class="math inline">\(\pi\rightarrow\pi^*\)</span> and <span class="math inline">\(n\rightarrow\pi^*\)</span> transitions in the peptide bonds, which is dependent on <span class="math inline">\(\Psi\)</span> and <span class="math inline">\(\Phi\)</span> torsional angles. Thus different types of secondary structures, such as <span class="math inline">\(\alpha\)</span>-helices, <span class="math inline">\(\beta\)</span>-sheets, <span class="math inline">\(\beta\)</span>-turns, each have their characteristic CD spectrum (Fig. <a href="#fig:cd">1.6</a>). These standard curves can be linearly combined to estimate the proportion of each secondary structure in a protein of interest.</p>
<div class="figure"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="Experimental setup of CD; characteristic curves of some secondary structures" width="100%" />
<p class="caption">
Figure 1.6: Experimental setup of CD; characteristic curves of some secondary structures
</p>
</div>
<p>The spectra in the near-UV region (260-320 nm) arise from the aromatic amino acids, each with a characteristic CD profile. The actual shape of the near UV CD spectrum will depend on the number of each type of aromic amino acid present, their mobility, their residing environment. Thus it can serve as a fingerprint of the of the tertiary structure of a protein.</p>
<div class="figure"><span id="fig:cd-aromatic"></span>
<img src="../img/cd-aromatic.png" alt="The near UV CD spectrum of a dehydroquinase, labelled with contribution from each aromatic amino acid side chain." width="100%" />
<p class="caption">
Figure 1.7: The near UV CD spectrum of a dehydroquinase, labelled with contribution from each aromatic amino acid side chain.
</p>
</div>
<p>By combining the far- and near-UV CD spectra, we can obtain a summary of the overall structural features of the protein of interest. Although it gives little insight into the precise 3D structure of the protein, it serves at a rapid way to detect conformational differences between two similar proteins <em>in solution</em>. Specifically, it can be used to:</p>
<ul>
<li>monitor the progress of protein folding (especially the detectiion of molten globule-like structures)</li>
<li>compare the wild type and mutant forms of a protein</li>
<li>confirm a modification (tagging) will not affect the protein’s native conformation and normal function</li>
<li>assess thermal stability (unfolding at high temperature)</li>
<li>show the formation of amyloid <span class="math inline">\(\beta\)</span> protein in Alzheimer disease <span class="citation">(Barrow et al. 1992)</span></li>
</ul>
<p>CD can not only be applied to proteins but also to other chiral molecules. Such applications include:</p>
<ul>
<li>determination of nucleic acid conformations (A-RNA, A-DNA, B-DNA, Z-DNA)</li>
<li>determination of nucleic acid-ligand interactions, e.g. between cationic porphyrins and DNA <span class="citation">(Pasternack 2003)</span></li>
<li>conformational study of biomolecular interaction with nanoparticles, where the degree of protein or nucleic acid denaturation is estimated <span class="citation">(Liu and Webster 2007)</span></li>
</ul>
</div>
<div id="other-cd-based-techniques" class="section level2">
<h2><span class="header-section-number">1.4</span> Other CD-based Techniques</h2>
<p>The experiment described above is the conventional electronic circular dichorism (ECD). During the past decades, many other CD-based techniques have been developed to solve more specific questions. These include magnetic CDs (MCD, magnetic vibrational circular dichroism (MVCD), XMCD), fluorescence detected CD (FDCD), near-infrared CD (NIR-CD), vibrational CDs (VCD, FTIR-VCD), HPLC-CD, stopped-flow CD, and synchrotron radiation CD (SRCD).</p>
</div>
</div>
<div id="small-angle-x-ray-scattering-saxs" class="section level1">
<h1><span class="header-section-number">2</span> Small-angle X-ray Scattering (SAXS)</h1>
<p>Reference: <span class="citation">Kachala, Valentini, and Svergun (2015)</span>, <span class="citation">Kikhney and Svergun (2015)</span></p>
<p>Small-angle X-ray Scattering (SAXS) detects the X-ray scattering pattern of macromolecules in the solution state. Information about particle shape and size can then be obtained from the angular dependence of scattering.</p>
<p>SAXS provides low resolution information on the structure, conformation and assembly state of proteins, nucleic acids and various macromolecular complexes. Importantly, it offers powerful means for the quantitative analysis of flexible systems, including intrinsically disordered proteins (IDPs) and multi-domain proteins with ﬂexible linkers.</p>
<div id="physical-principles-1" class="section level2">
<h2><span class="header-section-number">2.1</span> Physical Principles</h2>
<p>The the X-ray used in SAXS has wavelength of about the same size as the macromolecules in the sample, which allows formation of interference pattern (Fig. <a href="#fig:saxs-principle-1">2.1</a>).</p>
<div class="figure"><span id="fig:saxs-principle-1"></span>
<img src="../img/saxs-principle-1.jpg" alt="(A) for an array (usually a crystal) of small scatterers whose size is small compared to the wavelength of the incident light, an interference pattern can be observed due to the fixed path differences (and hence phase differences) between any two scatterers; (B) in the solution state of such small particles, their random orientation and movement make scattering to occur in all directions (so that a pattern cannot be observed); (C) when the size of the scatterers is comparable to or greater than the wavelength, angle-dependent scatterings due the nuclei/electrons of pairs of atoms within individual particles can thus be observed (larger angle, larger path difference)." width="100%" />
<p class="caption">
Figure 2.1: (A) for an array (usually a crystal) of small scatterers whose size is small compared to the wavelength of the incident light, an interference pattern can be observed due to the fixed path differences (and hence phase differences) between any two scatterers; (B) in the solution state of such small particles, their random orientation and movement make scattering to occur in all directions (so that a pattern cannot be observed); (C) when the size of the scatterers is comparable to or greater than the wavelength, angle-dependent scatterings due the nuclei/electrons of pairs of atoms within individual particles can thus be observed (larger angle, larger path difference).
</p>
</div>
</div>
<div id="experimentation-and-data-processing" class="section level2">
<h2><span class="header-section-number">2.2</span> Experimentation and Data Processing</h2>
<p>SAXS uses a collimated monochromatic X-ray beam to illuminate the sample, and the intensity of the scattered X-rays a is recorded. The scattering of the pure solvent is also collected and subtracted from the sample solution scattering. The resulting 2D scattering pattern is translated into a 1D <em>I</em> vs. <em>q</em> relationship (where <em>I</em> is intensity and <span class="math inline">\(q = 4\pi\sin(\theta/2)/\lambda\)</span>), and the data is transformed and plotted in a variety of ways.</p>
<p>Several parameters can be calculated from SAXS data, including molecular weight, excluded particle volume, maximum dimension <span class="math inline">\(D_\text{max}\)</span> and the radius of gyration <span class="math inline">\(R_g\)</span>.</p>
<p><span class="math inline">\(R_g\)</span> can be directly extracted from SAXS data using the Guinier approximation, which states that, when the incident angle is small (approaching 0), the angular dependence of scattering can be described by the equation <span class="math inline">\(I(q)=I_z\exp(-q^2R_g^2/3)\)</span>, where <span class="math inline">\(R_g\)</span> is the radius of gyration of the particle. When <span class="math inline">\(\ln(I(q))\)</span> is plotted against <span class="math inline">\(q^2\)</span> (<span class="math inline">\(\ln(I(q)) = \ln{I_0}-(R_g^2/3)q^2\)</span>), <span class="math inline">\(R_g^2/3\)</span> is the slope of the resulting straight line.</p>
<p><span class="math inline">\(R_g\)</span> provides a measure of the overall size of the macromolecule. It is the average root-mean-square distance to the centre of density in the molecule weighted by the scattering length density.</p>
<p>Some plots can emphasised sample flexibility, e.g. a Kranky plot (<span class="math inline">\(IQ^2\)</span> agianst <span class="math inline">\(Q\)</span>) can help identify an unfolded protein, as shown in Fig. <a href="#fig:Kratky">2.2</a></p>
<div class="figure"><span id="fig:Kratky"></span>
<img src="../img/Kratky.png" alt="The Kratky plot" width="100%" />
<p class="caption">
Figure 2.2: The Kratky plot
</p>
</div>
<p><span class="math inline">\(I(q)\)</span> can be considered as a reciprocal space as a Fourier transform of <span class="math inline">\(p(r)\)</span>, which is the distribution of distances between pairs of atoms in real space. These two are related to each other by the equation:</p>
<p><span class="math display">\[p(r)=\dfrac{r^2}{2\pi^2}\int_0^\infty\dfrac{q^2I(q)\sin{(qr)}}{qr}ds\]</span></p>
<p><span class="math inline">\(p(r)\)</span> can be obtained from experimental data by indirect Fourier transformation. One such distribution is shown in Fig. <a href="#fig:saxs-pr">2.3</a>.</p>
<div class="figure"><span id="fig:saxs-pr"></span>
<img src="../img/saxs-pr.png" alt="Distance distribution function *p(r)* vs. r. Globular compact particles have a more symmetric distribution while unfolded particles have an skewed distribution with an extended tail." width="100%" />
<p class="caption">
Figure 2.3: Distance distribution function <em>p(r)</em> vs. r. Globular compact particles have a more symmetric distribution while unfolded particles have an skewed distribution with an extended tail.
</p>
</div>
<p>To produce meaningful results, SAXS requires that the samples to be monodisperse (non-aggregated), which can be verified by dynamic light scattering (DLS) or analytical centrifugation (AUC).</p>
</div>
</div>
<div id="analytical-ultracentrifuge-auc" class="section level1">
<h1><span class="header-section-number">3</span> Analytical Ultracentrifuge (AUC)</h1>
<p>Reference: <span class="citation">Harding et al. (2015)</span>; <span class="citation">Uchiyama, Noda, and Krayukhina (2018)</span>; <span class="citation">Unzai (2018)</span>; <span class="citation">Harding and Rowe (2010)</span></p>
<p>The analytical centrifuge (AUC) is a high speed (up to 60000 rpm) ultracentrifuge equipped with absorbance and interference detection systems, which allow the analysis of the redistribution of macromolecular solute under the influence of a centrifugal field. AUC is a convenient, matrix-free solution technique without requirement for immobilisation, columns, or membranes.</p>
<p>Typical AUC experiments can be classified into two types:</p>
<ul>
<li><strong>sedimentation velocity experiment:</strong> record the <em>change in concentration distribution</em> over time (performed at high speed)</li>
<li><strong>sedimentation equilibrium experiment:</strong> record the <em>steady state distribution</em> of the macromolecular solute following equiilibration of centrifugal and diffusive forces (performed at lower speed)</li>
</ul>
<p>These two methods can give information about a wide range of parameters of protein-protein interactions, including stoichiometry, reversibility, strength and, in some cases, dynamics.</p>
<p>Apart from protein-protein interactions, AUC is now applied to study interactions of a wider range of macromolecules, such as protein-like carbohydrate assocaition, carbohydrate-protein association (polysaccharide-gliadin), and nucleic acid protein (G-duplexes) interactions.</p>
<div id="choice-of-optical-system" class="section level2">
<h2><span class="header-section-number">3.1</span> Choice of Optical System</h2>
<p>Depending on the strengths of the interaction probed (and the concentration of the sample), different optical systems are used:</p>
<ul>
<li>for weak interactions (<span class="math inline">\(10^{-4}&lt;K_\text{d}&lt;10^{-1}\)</span>), higher concentrations (&gt;5 mg/ml) are required and the <em>Rayleigh interference</em> optical system is the most appropriate</li>
<li>for moderate-strength interactions (<span class="math inline">\(10^{-7}&lt;K_\text{d}&lt;10^{-4}\)</span>, 0.1~0.5 mg/ml), either <em>interference</em> optics or <em>UV absorption</em> optics can be used</li>
<li>for strong interactions (<span class="math inline">\(K_\text{d}&lt;10^{-7}\)</span>), <em>dye-labelled</em> proteins and <em>fluorescence optics</em> are necessary</li>
</ul>
</div>
<div id="experiments" class="section level2">
<h2><span class="header-section-number">3.2</span> Experiments</h2>
<div id="sedimentation-velocity-sv" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Sedimentation Velocity (SV)</h3>
<p>In the SV experiment, measurements on the absorption (<span class="math inline">\(A\)</span>, which is proportional to local concentration) at different radial distances (<span class="math inline">\(r\)</span>) from the rotation centre are made at fixed time intervals (<span class="math inline">\(t\)</span>), producing a series of s-shaped curves that shift to higher <span class="math inline">\(r\)</span> values as sedimentation proceeds (Fig. <a href="#fig:auc-sv-data">3.1</a>.</p>
<p>The spinning rotor generates a sedimentation force on a particle of <span class="math inline">\(m\omega^2r\)</span> (<span class="math inline">\(m\)</span> = particle mass; <span class="math inline">\(\omega\)</span> = angular velocity; <span class="math inline">\(r\)</span> = distance from the centre of rotation). In solution, the particle displaces solvent, so the sedimentation force acts on an effective mass, <span class="math inline">\(m_\text{eff}=m(1-\bar v\rho)\)</span> that is less than <span class="math inline">\(m\)</span> (<span class="math inline">\(\rho\)</span> = solvent density; <span class="math inline">\(\bar v\)</span> = partial-specific volume (in ml/g)). <span class="math inline">\(\bar v\)</span> is usually calculated (for proteins, <span class="math inline">\(\bar v\)</span> lies in the range 0.70-0.75, leading to <span class="math inline">\((1-\bar v\rho)\)</span> of around 0.27 (in water)). At terminal velocity (acceleration = 0), the sedimentation force is balanced by the frictional force and the velocity can be described by the equation <span class="math inline">\(v=m(1-\bar{v}\rho)\omega^2r/f\)</span>, where <span class="math inline">\(f\)</span> is the friction coefficient, which is related to the diffusion coefficient, <span class="math inline">\(D\)</span>, by <span class="math inline">\(f=RT/N_\text{A}D\)</span>. The equation can thus be arranged to <span class="math inline">\(v=DM(1-\bar{v}\rho)\omega^2r/RT\)</span>. By defining the <em>sedimentation coefficient</em> as <span class="math inline">\(s=v/\omega^2r\)</span>, the <em>Svedberg equation</em> can be written as <span class="math inline">\(s=\dfrac{DM(1-\bar{v}\rho)}{RT}\)</span>.</p>
<p>The sedimentation coefficient <span class="math inline">\(s\)</span>, which is experimentally determined in SV has the unit Svedberg (S) where 1 S = 10<sup>-13</sup> seconds. The greater the molecular weight or more compact/spherical (less friction) the macromolecule is, the larger its <span class="math inline">\(s\)</span> value. Usually the experimentally measured <span class="math inline">\(s_\text{exp}\)</span> is standarised to <span class="math inline">\(s_{20, \text{w}}\)</span>, which is the value that would have been observed in water at 20<span class="math inline">\(^\circ\)</span>C, using the relationship <span class="math inline">\(s_{20, \text{w}}=\dfrac{s_\text{exp}(\eta_\text{exp}(1-\bar{v}\rho_\text{20,w})}{(\eta_\text{20,w}(1-\bar{v}\rho_\text{exp})}\)</span>, where <span class="math inline">\(\eta\)</span> and <span class="math inline">\(\rho\)</span> refer to the viscosity and density of the buffer.</p>
<p>The Lamm equation, derived from Svedberg equation and Fick’s diffusion laws, describes the time dependence of the concentration:</p>
<p><span class="math display">\[\frac{\partial c}{\partial t}=D\left[\left(\frac{\partial^2 c}{\partial r^2}\right)+\frac{1}{r}\left(\frac{\partial c}{\partial r}\right)\right]-s\omega^2\left[r\left(\frac{\partial c}{\partial r}\right)+2c\right]\]</span></p>
<p>Solving this diffential equation leads to <span class="math inline">\(g(s)\)</span>, which is the distribution of <span class="math inline">\(s\)</span>. SEDFIT is one of the softwares specialised in solving this. SEDFIT can either give the uncorrected <span class="math inline">\(g(s)\)</span> versus <span class="math inline">\(s\)</span> profile, or it can give a distribution, known as <span class="math inline">\(c(s)\)</span> vs. <span class="math inline">\(s\)</span>, which has been corrected for diffusion broadening (this assumes all particles have the same frictional ratio <span class="math inline">\(f/f_o\)</span>). The <span class="math inline">\(g(s)\)</span> vs. <span class="math inline">\(s\)</span> or <span class="math inline">\(c(s)\)</span> vs. <span class="math inline">\(s\)</span> can further be converted to a molecular weight distribution, <span class="math inline">\(c(M)\)</span> vs. <span class="math inline">\(M\)</span>, which is analogous to a mass spectrum (Fig. <a href="#fig:auc-sv-data">3.1</a>). SEDFIT is particularly good at evaluating homogeneity/heterogeneity of a prepation. Where the solution is heterogeneous, it can estimate the proportion of each sedimenting species and ascertain whether there is a reversible equilibrium.</p>
<div class="figure"><span id="fig:auc-sv-data"></span>
<img src="../img/auc-sv-data.jpg" alt="Schematic of data processing in SV experiments. The raw data collected is a series of curves recorded at fixed intervals, each showing the variation of the absorbance with the distance from the rotation centre. By solving the Lamm equation, this is transformed to a *g(s)* vs. *s* plot, or *c(s)* vs. *s* plot if it is denoised by SEDFIT. It can be further transformed into a *c(M)* vs. M plot (distribution of molecular weights), assuming constant frictional ratio $f/f_o$" width="100%" />
<p class="caption">
Figure 3.1: Schematic of data processing in SV experiments. The raw data collected is a series of curves recorded at fixed intervals, each showing the variation of the absorbance with the distance from the rotation centre. By solving the Lamm equation, this is transformed to a <em>g(s)</em> vs. <em>s</em> plot, or <em>c(s)</em> vs. <em>s</em> plot if it is denoised by SEDFIT. It can be further transformed into a <em>c(M)</em> vs. M plot (distribution of molecular weights), assuming constant frictional ratio <span class="math inline">\(f/f_o\)</span>
</p>
</div>
<p>SV also gives information about the shape. The friction coefficient, <span class="math inline">\(f\)</span>, can be easily calculated from the terminal velocity (<span class="math inline">\(v=m(1-\bar{v}\rho)\omega^2r/f\)</span>), and the friction ratio, <span class="math inline">\(f/f_o\)</span> (where <span class="math inline">\(f_o=6\pi r\eta\)</span>), shows diviation of the molecular shape from the sphere.</p>
<p>SV can also be applied for interaction analysis, and the simpest case is co-sedimentation. For example, the binding of adenosylcobalamin cofactor to the methylmalonyl-CoA mutase system from <em>Propionibacterium shermanii</em> was demonstrated by AUC with UV-absoption system. At a wavelength selected to detect the ligand only, in the presence of the mutase, all ligands sediment at the same rate as the protein, confirming the ligand is 100% bound.</p>
<p>In SV analysis, the emergence of new peaks at higher concentrations, shifts in the ratios of the peak areas, and/or shifts in peak positions are indicative of protein-protein interactions.</p>
<p>Fig. <a href="#fig:sv-trap">3.2</a> shows an SV experiment which studies the binding between the <em>Bacillus stearothermophilus</em> 11-mer protein TRAP (<em>trp</em> RNA-binding attentuation protein) and Anti-TRAP (AT). As the TRAP:AT ratio increases froom 1:0 to 1:6, the peak representing TRAP shifts to higher <em>s</em>, indicating rapid and reversible binding between TRAP and AT (slow/irreversible interaction would result in separate peaks). The plot also shows that TRAP was saturated with AT at a 1:6 stoichiometry (as increasing ratio to 1:10 did not result in further shifts) and that the TRAP-AT complex was stable to excess AT (no negative feedback loop).</p>
<div class="figure"><span id="fig:sv-trap"></span>
<img src="../img/sv-trap.png" alt="SV analysis of TRAP and AT. The TRAP concentratioin was fixed at 0.5 mg/ml, and varying amounts of AT (TRAP's ligand) were added, with molar ratios of TRAP:AT labelled." width="100%" />
<p class="caption">
Figure 3.2: SV analysis of TRAP and AT. The TRAP concentratioin was fixed at 0.5 mg/ml, and varying amounts of AT (TRAP’s ligand) were added, with molar ratios of TRAP:AT labelled.
</p>
</div>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Barrow-1992">
<p>Barrow, Colin J., Akikazu Yasuda, Peter T.M. Kenny, and Michael G. Zagorski. 1992. “Solution Conformations and Aggregational Properties of Synthetic Amyloid β-Peptides of Alzheimer’s Disease: Analysis of Circular Dichroism Spectra.” <em>Journal of Molecular Biology</em> 225 (4): 1075–93. <a href="https://doi.org/https://doi.org/10.1016/0022-2836(92)90106-T">https://doi.org/https://doi.org/10.1016/0022-2836(92)90106-T</a>.</p>
</div>
<div id="ref-Harding-2015">
<p>Harding, Stephen E, Richard B Gillis, Fahad Almutairi, Tayyibe Erten, M Şamil Kök, and Gary G Adams. 2015. “Recent Advances in the Analysis of Macromolecular Interactions Using the Matrix-Free Method of Sedimentation in the Analytical Ultracentrifuge.” <em>Biology</em> 4 (1). MDPI: 237–50. <a href="https://doi.org/10.3390/biology4010237">https://doi.org/10.3390/biology4010237</a>.</p>
</div>
<div id="ref-Harding-2010">
<p>Harding, Stephen E., and Arthur J. Rowe. 2010. “Insight into protein–protein interactions from analytical ultracentrifugation.” <em>Biochemical Society Transactions</em> 38 (4): 901–7. <a href="https://doi.org/10.1042/BST0380901">https://doi.org/10.1042/BST0380901</a>.</p>
</div>
<div id="ref-Kachala-2015">
<p>Kachala, Michael, Erica Valentini, and Dmitri I Svergun. 2015. “Application of Saxs for the Structural Characterization of Idps.” <em>Advances in Experimental Medicine and Biology</em> 870. United States: 261–89. <a href="https://doi.org/10.1007/978-3-319-20164-1{\_}8">https://doi.org/10.1007/978-3-319-20164-1{\_}8</a>.</p>
</div>
<div id="ref-Kelly-2005">
<p>Kelly, Sharon M, Thomas J Jess, and Nicholas C Price. 2005. “How to Study Proteins by Circular Dichroism.” <em>Biochimica et Biophysica Acta</em> 1751 (2). Netherlands: 119–39. <a href="https://doi.org/10.1016/j.bbapap.2005.06.005">https://doi.org/10.1016/j.bbapap.2005.06.005</a>.</p>
</div>
<div id="ref-Kikhney-2015">
<p>Kikhney, Alexey G., and Dmitri I. Svergun. 2015. “A Practical Guide to Small Angle X-Ray Scattering (Saxs) of Flexible and Intrinsically Disordered Proteins.” <em>FEBS Letters</em> 589 (19, Part A): 2570–7. <a href="https://doi.org/https://doi.org/10.1016/j.febslet.2015.08.027">https://doi.org/https://doi.org/10.1016/j.febslet.2015.08.027</a>.</p>
</div>
<div id="ref-Liu-2007">
<p>Liu, Huinan, and Thomas Jay Webster. 2007. “Nanomedicine for Implants: A Review of Studies and Necessary Experimental Tools.” <em>Biomaterials</em> 28 (2): 354–69. <a href="https://doi.org/https://doi.org/10.1016/j.biomaterials.2006.08.049">https://doi.org/https://doi.org/10.1016/j.biomaterials.2006.08.049</a>.</p>
</div>
<div id="ref-Pasternack-2003">
<p>Pasternack, Robert F. 2003. “Circular Dichroism and the Interactions of Water Soluble Porphyrins with Dna—a Minireview.” <em>Chirality: The Pharmacological, Biological, and Chemical Consequences of Molecular Asymmetry</em> 15 (4). Wiley Online Library: 329–32.</p>
</div>
<div id="ref-Ranjbar-2009">
<p>Ranjbar, Bijan, and Pooria Gill. 2009. “Circular Dichroism Techniques: Biomolecular and Nanostructural Analyses—a Review.” <em>Chemical Biology &amp; Drug Design</em> 74 (2). England: 101–20. <a href="https://doi.org/10.1111/j.1747-0285.2009.00847.x">https://doi.org/10.1111/j.1747-0285.2009.00847.x</a>.</p>
</div>
<div id="ref-Sievert-2018a">
<p>Sievert, Carson. 2018. “Plotly for R.” 2018.</p>
</div>
<div id="ref-Uchiyama-2018">
<p>Uchiyama, Susumu, Masanori Noda, and Elena Krayukhina. 2018. “Sedimentation Velocity Analytical Ultracentrifugation for Characterization of Therapeutic Antibodies.” <em>Biophysical Reviews</em> 10 (2). Springer Berlin Heidelberg: 259–69. <a href="https://doi.org/10.1007/s12551-017-0374-3">https://doi.org/10.1007/s12551-017-0374-3</a>.</p>
</div>
<div id="ref-Unzai-2018">
<p>Unzai, Satoru. 2018. “Analytical Ultracentrifugation in Structural Biology.” <em>Biophysical Reviews</em> 10 (2). Springer Berlin Heidelberg: 229–33. <a href="https://doi.org/10.1007/s12551-017-0340-0">https://doi.org/10.1007/s12551-017-0340-0</a>.</p>
</div>
<div id="ref-Whitmore-2008">
<p>Whitmore, Lee, and B A Wallace. 2008. “Protein Secondary Structure Analyses from Circular Dichroism Spectroscopy: Methods and Reference Databases.” <em>Biopolymers</em> 89 (5). United States: 392–400. <a href="https://doi.org/10.1002/bip.20853">https://doi.org/10.1002/bip.20853</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>suprisingly, this graphing library originally not designed for math plotting worked pretty well, and is much easier to use (and more powerful in terms of interactivity) than other professional math plotting libraries. I even <a href="../bioinformatics/visualizing-protein-structures-using-plotly.html">used <strong>plotly</strong> to plot proteins</a> and it worked quite decently!)<a href="#fnref1" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/visualizing-protein-structures-using-plotly.html" class="archive-item-link">
                Visualizing Protein Structures Using Plotly 
              </a>
              <span class="archive-item-date">
                2020-02-20
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/htmlwidgets/htmlwidgets.js"></script>
<script src="../rmarkdown-libs/plotly-binding/plotly.js"></script>
<script src="../rmarkdown-libs/typedarray/typedarray.min.js"></script>
<script src="../rmarkdown-libs/jquery/jquery.min.js"></script>
<link href="../rmarkdown-libs/crosstalk/css/crosstalk.css" rel="stylesheet" />
<script src="../rmarkdown-libs/crosstalk/js/crosstalk.min.js"></script>
<link href="../rmarkdown-libs/plotly-htmlwidgets-css/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="../rmarkdown-libs/plotly-main/plotly-latest.min.js"></script>


<p>Am I the first one to try to visualize protein structures using Plotly?</p>
<p>Personally, I think the hitherto most powerful and elegant protein structure visualization tool is <a href="https://spdbv.vital-it.ch/content.html">SwissPdbViewer (a.k.a. DeepView)</a>, which is tightly linked to SWISS-MODEL. Apparently it’s used as a web application, but it is not written in JS. It is not open source, and it’s 32-bit architecture makes it obsolete on modern 64-bit computers.</p>
<p>Plotly has an open source graphing library with APIs for R, Python and JS, of which the R API highly compatible with the renowned <code>ggplot2</code> package. This means you can make ggplots in R and convert them into interactive svg images for web with one line of code!</p>
<p>Apart ggplot conversion, Plotly also provides its own suite of plotting functions in its R API, many of which are specifically suited for web application (e.g. 3D support).</p>
<p>Given its simplicity and rich APIs, I think Plotly might be used as a framwork for plotting protein structures. In this preliminary experiment, I plotted the atoms of a protein structure (written in a PDB file) as dots with Plotly in R, and rendered them into HTML widgets in Rmarkdown, so that the plots shown below are interactive!</p>
<p>Required Packages:</p>
<pre class="r"><code>library(tidyverse)
library(plotly)
library(Rpdb)</code></pre>
<pre class="r"><code>x &lt;- read.pdb(&#39;/Users/tianyishi/Desktop/sample.pdb&#39;)
pdb &lt;- as_tibble(x$atoms) %&gt;% 
  mutate(charge = ifelse(resname %in% c(&#39;LYS&#39;, &#39;ARG&#39;, &#39;HIS&#39;),
                         &#39;basic&#39;, ifelse(resname %in% c(&#39;ASP&#39;, &#39;GLU&#39;),
                                         &#39;acidic&#39;, &#39;neutral&#39;)))


pdb %&gt;% plot_ly(x = ~x1, y = ~x2, z = ~x3, color = ~charge, colors = c(&#39;red&#39;, &#39;blue&#39;, &#39;grey&#39;)) %&gt;%
  add_markers()</code></pre>
<div id="htmlwidget-1" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1">{"x":{"visdat":{"413e12753743":["function () ","plotlyVisDat"]},"cur_data":"413e12753743","attrs":{"413e12753743":{"x":{},"y":{},"z":{},"color":{},"colors":["red","blue","grey"],"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter3d","mode":"markers","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x1"},"yaxis":{"title":"x2"},"zaxis":{"title":"x3"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[-12.713,-12.917,-12.895,-12.411,-14.242,-14.205,-13.1,-15.286,-11.105,-9.706,-9.014,-8.327,-8.959,-9.408,-9.568,-9.589,7.99,9.416,9.611,10.416,9.993,10.119,9.746,10.608,11.165,10.743,10.654,11.156,9.39,9.052,9.779,8.041,27.762,27.225,25.741,25.153,27.393,26.76,27.283,25.747,-4.338,-5.167,-4.371,-4.936,-5.901,-6.945,-7.929,-6.781,-3.068,-2.199,-0.79,0.088,-2.182,-1.48,-0.865,-1.541,9.747,11.11,11.69,12.841,11.151,10.827,10.824,11.785,9.856,12.557,13.803,15.006,15.981,13.841,15.162,15.435,14.473,16.621,15.826,14.962,13.577,13.463,15.569,14.875,13.659,15.566,-6.773,-6.971,-6.766,-7.38,-6.022,-6.302,-5.763,-5.037,-6.065,-10.502,-10.846,-9.681,-9.348,-12.091,-13.318,-13.452,-14.188,-12.821,-2.962,-2.213,-2.303,-1.295,-2.761,-1.858,-0.621,-2.384,7.889,7.552,8.708,8.603,6.805,6.469,5.65,4.874,5.784,17.252,17.981,17.412,17.648,19.474,20.351,20.693,21.494,20.173,17.592,16.822,15.886,14.781,17.747,18.338,17.828,19.318,14.397,13.255,12.251,11.05,13.728,14.536,15.238,15.418,15.628,6.299,5.823,4.604,3.548,6.949,6.57,5.542,7.318,-5.014,-6.339,-6.939,-7.447,-7.159,-8.331,-9.039,-8.353,-10.287,-9.677,-11.039,-11.222,-12.342,-12.078,-11.816,-11.108,-12.344,-0.952,-0.365,0.555,1.611,-1.449,-1.946,-1.292,-2.983,10.602,11.515,10.844,11.521,12.609,13.617,14.028,14.006,9.521,8.807,7.525,6.552,8.479,8.109,8.841,7.086,-0.862,-2.258,-3.152,-4.048,-2.747,-4.064,-4.386,-5.401,-3.625,-6.743,-7.716,-9.17,-10.061,-7.165,-5.809,-5.16,-5.764,-4.028,-6.356,-5.149,-4.151,-4.39,-5.476,-4.282,-3.119,-4.518,-1.508,-1,-1.793,-1.281,-1.103,-0.524,0.712,-1.301,0.277,0.556,-0.458,-0.095,0.515,1.823,2.825,1.849,13.32,13.846,12.695,12.74,14.745,16.099,16.003,16.938,15.016,4.904,4.599,5.479,5.049,4.873,3.833,2.705,4.142,6.654,6.128,4.936,4.862,5.679,6.829,7.89,6.666,-4.011,-4.594,-3.51,-3.644,-5.357,-6.696,-7.128,-7.329,-0.343,0.973,1.161,1.641,2.072,3.439,3.503,4.459,-10.67,-9.246,-8.635,-9.29,-8.495,-8.507,-9.097,-7.904,-4.212,-4.138,-4.847,-4.525,-4.699,-3.837,-2.677,-4.306,-5.806,-6.548,-6.992,-8.15,-7.77,-8.486,-8.142,-9.416,-15.009,-15.818,-15.016,-15.475,-16.393,-17.503,-18.129,-18.258,-18.49,-14.352,-15.554,-15.699,-15.958,-16.787,-18.102,-18.17,-17.52,-18.893,-6.816,-5.76,-4.81,-4.346,-5.008,-4.047,-3.45,-3.877,11.804,10.959,10.997,10.883,9.505,9.266,9.677,9.922,9.748,14.132,13.275,11.806,10.96,13.605,15.06,15.949,15.794,16.798,6.537,6.508,5.736,4.861,7.931,7.968,7.312,8.652,-8.38,-9.124,-10.333,-10.466,-9.538,-10.374,-9.686,-8.437,-10.403,14.569,14.767,14.669,13.571,13.683,13.805,14.92,15.812,14.898,15.415,16.38,16.111,17.047,16.388,17.049,17.702,16.919,14.901,15.413,16.932,17.469,14.979,13.471,12.977,13.672,11.872,21.566,21.291,21.34,22.12,19.915,19.598,20.453,18.48,25.995,27.283,27.413,26.84,27.272,28.64,28.715,28.148,29.345,29.591,30.007,28.983,29.242,31.401,31.508,30.479,32.651,28.14,28.671,28.878,28.615,29.971,29.721,28.568,30.673,23.216,22.858,23.633,23.423,23.032,21.96,20.78,22.296,24.503,25.299,25.409,26.389,26.689,27.418,26.824,28.592,30.551,31.604,31.147,31.364,32.87,34.054,35.275,35.439,36.083,28.876,29.822,30.931,31.556,30.351,29.195,29.575,30.776,28.651,22.003,20.671,20.329,20.149,19.673,18.246,17.932,17.417,17.674,16.466,16.863,16.189,15.466,14.188,13.383,13.999,15.262,14.113,14.554,13.999,13.138,12.278,12.161,11.702,7.433,8.169,9.635,10.23,8.065,6.654,5.69,4.464,6.146],"y":[13.937,15.334,15.425,16.406,15.859,16.045,16.189,16.053,13.51,13.428,14.777,15.259,12.382,10.968,10.628,10.191,24.121,24.017,23.018,22.087,25.374,26.371,26.061,27.489,17.76,16.643,15.328,14.298,16.973,16.068,15.09,16.348,15.687,14.796,14.578,15.306,15.409,16.789,17.74,16.926,18.107,17.587,16.766,15.927,18.72,19.38,18.708,20.572,17.013,16.285,16.227,17.013,16.965,16.13,15.089,16.526,17.857,18.269,17.479,17.031,19.764,20.668,22.141,22.593,22.848,13.891,13.849,13.922,13.185,15.01,15.18,14.078,13.438,13.86,5.061,3.888,4.306,4.889,2.787,1.44,1.384,0.42,17.971,18.14,16.851,16.654,19.211,20.641,20.961,20.129,22.063,10.609,11.036,11.791,11.581,11.929,11.333,11.781,12.766,11.16,9.959,8.745,8.457,8.155,7.563,6.343,6.504,5.22,8.493,8.55,8.365,7.566,9.853,10.07,8.941,8.296,8.7,-8.398,-7.952,-6.592,-5.595,-7.833,-7.329,-8.411,-9.31,-8.353,-1.405,-0.974,0.182,0.272,-0.603,-1.825,-2.95,-1.659,0.679,0.122,-0.5,-0.233,-0.893,-0.25,-1.256,-2.419,-0.877,2.447,3.811,4.11,4.507,4.805,6.245,6.478,7.15,4.846,4.527,3.597,4.052,5.81,5.723,7.063,8.112,7.067,1.851,2.036,3.332,3.733,1.91,2.849,3.869,2.56,-6.519,-6.559,-7.745,-7.601,-6.525,-5.12,-4.155,-4.973,-10.921,-11.939,-13.241,-14.168,-12.259,-11.147,-10.567,-10.857,-13.304,-14.526,-14.23,-14.976,-15.292,-16.738,-17.421,-17.191,-19.363,-19.782,-18.738,-19.077,-20.066,-20.835,-21.359,-20.92,-22.214,-19.978,-21.041,-20.818,-21.426,-22.314,-22.679,-23.88,-24.479,-24.219,-11.921,-11.695,-11.041,-9.933,-10.733,-10.445,-10.681,-9.955,-9.352,-8.033,-6.97,-5.886,-7.764,-6.412,-6.254,-5.493,0.238,1.575,2.583,3.707,1.587,1.139,0.949,0.99,8.445,7.347,6.528,5.295,7.876,8.437,9.767,10.073,10.516,4.293,3.577,2.337,1.274,4.463,5.556,5.41,6.561,0.587,-0.414,-1.125,-2.355,0.231,0.811,0.154,1.927,-20.259,-21.208,-21.817,-21.853,-22.335,-21.878,-20.733,-22.686,-22.211,-22.189,-20.999,-21.171,-22.215,-22.598,-23.156,-22.348,-26.952,-26.755,-25.961,-25.735,-28.077,-29.045,-28.746,-30.132,1.097,2.551,3.273,4.42,3.012,2.569,3.015,1.764,2.6,3.177,2.054,1.632,3.961,4.732,4.583,5.496,-2.696,-3.774,-4.636,-4.942,-4.659,-4.023,-5.001,-4.64,-6.129,-2.76,-2.742,-4.085,-4.139,-2.469,-2.344,-1.114,-0.09,-1.169,-7.378,-6.439,-6.308,-7.311,-6.993,-5.991,-5.168,-6.045,-15.356,-14.181,-13.195,-11.987,-14.599,-15.402,-16.87,-17.368,-17.534,-6.386,-5.331,-5.71,-4.866,-5.054,-4.727,-5.96,-6.829,-6.065,-5.575,-5.226,-3.918,-3.818,-5.079,-5.083,-5.949,-4.224,0.999,2.228,2.485,3.573,2.297,3.518,4.839,4.893,5.83,-22.105,-23.542,-24.187,-24.33,-24.103,-25.587,-25.889,-26.701,-25.324,-30.359,-31.027,-30.72,-30.585,-32.537,-32.902,-32.031,-34.065,-27.921,-26.584,-26.599,-26.017,-25.616,-25.384,-24.681,-24.711,-24.107,-27.898,-28.358,-27.188,-27.202,-29.028,-29.657,-29.638,-30.187,-21.671,-21.292,-19.818,-18.948,-21.495,-21.389,-22.119,-21.617,-23.201,-17.368,-16.344,-16.082,-15.32,-16.634,-18.001,-18.584,-18.491,-18.164,-19.479,-19.688,-20.778,-19.771,-20.276,-20.645,-20.32,-21.943,-21.634,-20.471,-20.13,-22.87,-23.915,-23.544,-25.113,-19.841,-18.738,-17.598,-16.856,-19.263,-18.364,-17.99,-18.04,-8.48,-7.661,-7.071,-5.89,-8.501,-7.738,-8.62,-9.661,-8.261,1.454,2.53,2.802,3.862,2.394,2.466,2.15,2.002,2.054,1.384,1.316,-0.181,-0.841,2.041,2.041,1.327,2.77,-1.752,-2.222,-2.855,-3.761,-1.095,-1.595,-2.254,-1.364,-10.663,-10.925,-11.221,-12.113,-9.745,-9.706,-10.743,-8.633,-25.214,-25.664,-25.244,-24.79,-27.186,-27.71,-27.656,-27.551,-27.733],"z":[4.663,5.025,6.546,7.111,4.462,2.95,2.383,2.326,8.601,9.014,8.844,9.75,8.185,8.483,9.675,7.523,17.99,18.325,19.465,19.357,18.755,17.606,16.455,17.866,21.747,22.586,21.799,22.252,23.235,24.422,24.687,25.1,18.209,19.237,18.943,18.142,20.634,20.75,20.123,21.472,1.354,0.266,-0.745,-1.451,-0.454,0.419,0.789,0.743,-0.82,-1.739,-1.15,-1.519,-3.115,-4.181,-3.848,-5.366,-2.57,-2.239,-1.069,-1.119,-1.921,-3.099,-2.719,-2.053,-3.082,-2.181,-2.95,-2.008,-2.171,-3.951,-4.714,-5.735,-6.211,-6.076,6.061,5.998,6.52,7.599,6.879,6.734,6.448,6.937,24.35,22.913,22.126,21.072,22.356,22.819,24.207,24.792,24.712,18.744,17.385,16.759,15.591,17.378,18.062,19.515,19.765,20.402,13.303,12.995,11.488,10.84,13.806,13.748,13.825,13.632,-1.323,-2.743,-3.726,-4.659,-3.051,-4.527,-5.127,-4.389,-6.345,30.788,31.969,32.367,31.682,31.647,32.794,33.806,33.468,34.944,37.662,38.825,38.481,39.021,39.991,40.683,40.476,41.439,35.574,34.857,35.824,35.736,33.817,32.692,31.803,32.219,30.683,39.886,40.095,39.218,39.719,39.79,40.103,40.764,39.697,38.026,38.545,37.483,36.457,38.679,39.637,39.819,39.856,39.924,37.017,36.524,35.736,35.438,37.653,38.826,38.67,39.921,36.502,37.841,38.077,38.7,38.923,39.208,38.767,39.884,36.095,35.575,35.153,34.712,36.596,36.734,35.704,37.879,35.252,34.899,34.121,34.206,36.188,35.937,35.188,36.497,30.307,30.289,29.612,28.832,31.71,31.764,33.154,33.739,33.658,21.624,21.447,21.841,21.246,22.081,21.494,22.149,23.067,21.739,17.437,16.646,17.61,18.099,15.495,14.586,14.973,13.463,17.147,16.793,17.555,17.846,15.292,14.914,14.988,14.586,22.595,22.068,22.615,22.974,20.536,19.905,20.634,18.665,28.918,29.728,30.318,30.335,30.853,30.394,29.644,28.869,29.835,-14.133,-15.368,-15.42,-15.873,-16.588,-16.767,-16.245,-17.443,-12.839,-11.912,-12.531,-12.522,-10.598,-9.805,-9.709,-9.272,-9.08,-8.122,-7.24,-6.014,-8.837,-9.428,-9.183,-10.143,-2.191,-1.571,-0.623,0.499,-2.641,-2.076,-0.962,-2.749,11.17,11.422,10.269,9.246,11.691,10.51,9.449,10.656,-12,-12.118,-10.973,-10.666,-13.466,-14.632,-14.712,-15.46,-10.344,-9.226,-8.3,-8.329,-9.728,-8.615,-7.417,-8.947,-2.792,-2.227,-1.262,-0.158,-3.334,-4.153,-5.142,-6.335,-4.727,1.972,2.809,3.531,4.738,1.938,2.698,3.596,3.282,4.617,10.771,11.141,9.938,9.399,12.358,12.978,12.251,14.214,-13.809,-14.019,-12.85,-13.048,-14.275,-15.556,-15.454,-14.333,-16.511,-10.265,-10.811,-10.687,-10.383,-12.28,-12.557,-12.532,-13.417,-11.625,-5.02,-3.601,-3.417,-2.549,-3.048,-1.532,-0.918,-0.945,3.123,2.874,3.77,4.338,1.398,1.012,1.307,1.298,1.552,21.62,21.781,20.398,19.848,22.709,23.03,24.014,23.675,25.129,16.382,15.514,14.041,13.247,15.773,17.098,17.712,17.526,12.793,12.517,12.358,11.411,13.616,13.634,14.891,15.929,14.844,6.252,4.898,3.912,2.953,4.855,3.504,2.595,3.358,-2.746,-3.314,-2.948,-3.607,-4.833,-5.484,-6.82,-7.818,-6.871,-3.242,-4.197,-5.302,-6.229,-4.777,-5.437,-5.845,-5.557,8.769,9.129,10.625,11.143,8.37,6.959,6.658,6.154,19.389,20.772,21.38,22.544,21.663,21.431,21.258,21.434,20.6,21.125,20.112,20.093,21.492,22.469,23.501,22.206,18.796,18.2,16.862,16.577,18.001,17.411,17.153,17.834,16.267,15.761,16.009,14.997,15.037,17.439,18.449,19.887,20.192,20.724,9.83,10.432,10.518,9.488,9.521,10.061,11.034,9.485,10.856,10.186,8.853,8.351,9.943,9.283,9.973,8.07,4.517,3.656,2.223,1.571,3.672,4.931,5.625,5.22,6.179,7.36,7.26,8.241,7.522,7.829,6.634,6.867,5.468],"type":"scatter3d","mode":"markers","name":"acidic","marker":{"color":"rgba(255,0,0,1)","line":{"color":"rgba(255,0,0,1)"}},"textfont":{"color":"rgba(255,0,0,1)"},"error_y":{"color":"rgba(255,0,0,1)"},"error_x":{"color":"rgba(255,0,0,1)"},"line":{"color":"rgba(255,0,0,1)"},"frame":null},{"x":[-13.384,-13.398,-11.965,-11.649,-14.138,-14.233,-14.886,-14.986,-15.587,-9.216,-8.606,-9.006,-8.155,-8.932,-8.336,-8.746,-8.106,-8.603,-6.448,-6.363,-6.743,-6.089,-7.285,-6.796,-7.835,-7.273,-7.897,-9.118,-7.291,-4.712,-4.501,-3.049,-2.568,-5.357,-6.84,-7.567,-9.024,-9.735,1.301,2.099,3.278,4.288,1.212,0.553,-0.356,-1.035,-1.947,13.125,12.576,13.082,13.526,11.051,10.418,8.927,8.403,8.233,8.52,7.882,20.136,21.286,22.306,23.068,21.969,21.14,21.393,22.749,22.877,27.13,27.919,27.862,28.566,27.505,26.058,25.035,25.466,23.876,24.111,17.36,16.366,16.177,15.045,16.771,16.623,16.928,16.645,16.952,16.907,16.294,14.804,13.993,16.998,16.737,17.275,15.948,16.824,16.016,12.827,12.184,12.029,11.033,13.014,13.125,13.909,14.033,14.812,13.024,12.968,11.826,10.994,14.289,15.446,16.726,17.894,19.178,-0.666,-0.78,-1.48,-1.339,0.614,1.386,0.643,1.471,1.865,1.577,2.611,3.235,4.457,2.035,3.095,2.492,2.07,1.604,15.191,15.308,15.635,16.252,14.047,13.907,13.537,14.162,13.588,13.963,19.731,20.819,20.922,21.988,20.71,21.866,21.721,22.868,22.708,18.785,18.512,18.082,18.655,19.716,20.045,20.969,21.205,20.472,19.446,20.749,8.054,7.058,7.445,6.602,6.958,6.629,6.377,6.438,7.823,5.387,4.957,5.427,4.684,5.516,5.23,3.95,6.057,4,5.268,-5.986,-7.388,-7.711,-8.782,-7.826,-7.786,-8.392,-9.876,-10.502,-6.625,-5.579,-5.683,-4.701,-5.645,-4.553,-4.358,-3.694,-3.4,-2.965,-3.51,-3.713,-2.927,-2.298,-5.201,-6.03,-7.502,-7.673,-9.099,9.797,10.952,11.962,13,11.68,10.863,10.461,11.611,12.411,12.196,13.448,11.893,12.143,11.065,10.15,12.212,10.972,11.084,9.779,9.44,10.315,8.204,11.15,12.75,11.891,11.14,9.926,12.716,13.347,14.132,14.775,14.191,12.942,14.872,6.935,6.171,5.526,4.349,7.05,6.259,5.155,4.221,4.95,0.482,-0.188,-1.375,-2.153,0.82,1.673,2.763,3.563,4.852,-9.862,-8.972,-8.571,-8.601,-7.725,-6.741,-5.683,-6.27,-6.38,-5.934,-6.975,-13.787,-14.128,-14.568,-15.483,-12.924,-13.05,-11.8,-11.751,-12.102,-12.534,-12.02,-20.835,-21.822,-21.848,-22.696,-21.589,-22.021,-21.898,-22.354,-22.198,-15.638,-15.414,-14.255,-13.357,-16.685,-16.573,-17.803,-17.696,-17.703,-14.253,-13.187,-11.83,-10.795,-13.521,-12.564,-12.288,-11.756,-11.345,-11.005,0.173,0.985,2.359,3.344,0.239,-0.037,-1.053,-2.423,-3.501,-3.376,-4.708,4.475,4.92,6.176,7.016,3.796,3.99,2.928,2.834,2.318,-6.407,-6.626,-6.544,-7.28,-5.624,-5.971,-5.719,-6.59,-6.185,-6.724,-9.43,-10.811,-11.358,-12.427,-10.958,-12.396,-12.541,-12.299,-12.349,-9.684,-9.273,-8.931,-9.381,-10.368,-10.776,-9.92,-11.941,-10.537,-11.765,-0.167,0.799,0.062,0.501,1.56,2.928,3.28,2.65,3.026,-1.542,-1.691,-0.525,-0.746,-1.886,-1.702,-2.733,-2.501,-1.656,-0.954,-1.492,9.526,10.303,9.456,9.34,11.607,12.555,13.783,14.805,15.967,2.947,3.872,3.26,3.856,4.739,6.211,7.039,8.524,8.959,-0.214,0.299,-0.687,-0.293,0.633,1.885,2.07,1.058,1.185,2.289,0.198,-1.834,6.711,7.673,7.164,7.224,9.028,10.097,11.395,12.484,13.768,4.007,2.804,3.118,2.498,1.895,1.288,0.549,-0.036,-0.641,2.311,1.462,2.199,1.606,1.052,0,-0.314,-1.449,-1.484,-0.446,-2.554,2.919,3.67,3.183,3.007,5.153,5.815,7.174,7.772,9.069,-1.297,-1.779,-1.713,-2.466,-0.934,0.552,1.356,1.565,2.296,1.627,2.842,2.458,3.007,3.726,4.427,5.197,5.736,6.874,7.622,7.227,-4.198,-5.245,-6.514,-7.586,-4.696,-3.806,-2.721,-3.267,-2.145,-13.555,-14.882,-14.845,-15.883,-15.747,-15.089,-14.079,-15.33,-13.727,-14.471,-7.266,-7.196,-8.277,-8.007,-7.343,-6.11,-6.287,-5.038,-5.195,-9.85,-9.54,-8.39,-8.348,-9.201,-9.181,-10.312,-8.16,-9.989,-8.689,-7.823,-8.391,-9.064,-8.987,-9.393,-8.765,-9.783,-9.146,-10.121,-9.7,-10.365,-9.35,-9.526,-11.092,-12.276,-12.928,-14.099,-14.753,-7.005,-7.336,-6.116,-6.258,-8.424,-9.726,-10.294,-11.717,-11.822,-7.423,-8.405,-9.826,-10.607,-8.205,-9.049,-8.659,-9.007,-8.645,-13.304,-12.254,-12.3,-11.886,-10.876,-10.603,-11.182,-9.849,-10.78,-9.969,-16.841,-17.309,-16.228,-16.474,-17.873,-18.302,-18.725,-18.891,-19.159,-12.806,-11.998,-10.578,-10.044,-12.663,-14.041,-13.935,-15.231,-15.47,-14.504,-16.685,0.631,2.079,2.648,3.537,2.705,4.201,4.756,6.275,6.861,4.427,5.753,5.67,6.533,6.284,7.59,8.746,7.919,9.73,9.254,13.373,13.885,13.128,13.74,13.884,14.867,14.723,15.672,15.479,10.726,10.556,10.204,9.199,11.848,11.776,12.786,10.854,12.465,11.306,6.064,5.38,3.896,3.033,5.972,7.403,7.903,7.155,7.561,-11.205,-12.412,-12.27,-13.26,-13.545,-13.978,-14.796,-16.035,-16.867,-16.605,-17.973,-10.544,-10.716,-10.704,-10.583,-9.598,-8.218,-7.129,-5.837,-4.798,-4.892,-3.673,-10.795,17.617,19.074,19.518,20.412,19.603,19.467,20.008,19.889,20.769,21.849,20.582,21.546,22.801,22.673,23.551,23.205,24.491,25.662,26.967,28.075,24.492,24.159,24.247,24.164,22.782,22.775,21.436,21.529,20.196,20.074,18.615,18.179,17.276,18.026,16.499,15.963,16.24,15.386,14.178,15.755,23.141,23.743,25.164,26.017,22.851,23.454,22.496,23.161,23.104,22.406,23.751,31.474,32.786,32.835,33.914,33.864,34.098,35.273,35.534,36.694,31.56,32.539,31.956,32.147,33.797,34.92,36.195,37.359,38.646,31.23,30.646,29.477,29.222,30.223,29.845,30.68,28.694,30.058,28.852,29.346,29.57,28.255,28.242,30.275,29.49,30.367,31.222,32.049,32.152,32.747,27.236,28.085,27.644,27.815,29.533,30.549,31.931,33.025,33.324,29.059,28.037,27.461,27.166,26.911,27.268,28.017,26.984,28.164,27.553,31.147,32.167,31.505,31.792,33.033,33.998,35.211,36.495,36.774,28.031,26.81,25.682,25.055,27.106,28.233,28.18,29.463,29.328,30.124,17.059,15.812,15.506,15.068,14.65,13.437,12.149,11.6,10.304,16.416,17.438,16.901,17.248,18.548,19.51,20.344,21.23,20.887,19.671,21.758,13.216,13.429,14.923,15.327,12.948,13.076,12.565,12.798,12.381,21.689,21.764,22.715,22.403,22.01,20.759,20.99,19.798,18.54,23.516,23.228,23.432,22.961,24.114,23.532,24.51,25.58,25.442,24.276,26.475,24.054],"y":[14.379,14.329,14.325,15.046,13.08,12.978,11.68,11.605,10.314,15.39,16.679,17.779,18.549,17.084,16.145,16.522,15.598,15.884,19.663,21.123,21.749,22.682,21.666,21.41,21.812,21.765,21.271,20.772,21.276,24.263,24.329,24.156,24.909,23.274,23.384,22.127,22.18,20.902,26.452,27.429,26.78,27.434,28.169,27.254,28.016,27.073,27.796,12.063,10.763,9.706,8.638,10.77,11.596,11.597,12.45,12.064,10.819,12.957,7.068,7.957,7.852,8.797,7.767,8.325,7.623,7.919,7.28,12.397,13.524,14.76,15.741,13.885,14.251,13.341,15.425,13.939,15.203,20.508,21.573,22.095,22.267,22.709,22.369,23.577,23.292,24.473,20.493,19.434,19.322,18.96,18.095,17.099,17.224,15.999,16.251,15.494,21.77,22.832,22.396,22.714,24.117,24.752,26.055,26.758,25.957,21.686,21.212,20.225,20.401,20.568,21.534,20.882,21.824,21.213,21.363,20.59,19.246,18.325,20.352,21.623,22.471,23.659,24.531,13.867,13.43,14.629,14.8,12.451,11.688,10.719,11.423,10.444,13.13,11.908,10.666,9.73,11.696,12.686,14.001,12.551,14.612,13.774,11.958,12.444,11.747,11.265,13.958,14.532,16.023,16.562,18.017,9.063,7.63,7.089,6.132,6.867,7.279,6.301,6.702,6.304,5.479,6.754,5.697,5.103,5.381,5.725,3.587,3.155,1.649,1.106,1.155,7.775,7.286,8.258,8.555,5.885,5.39,5.165,5.115,4.777,4.738,17.045,17.34,17.443,17.017,18.612,18.485,19.702,19.849,21.021,11.482,10.465,9.632,9.517,9.572,8.547,7.834,8.098,6.968,7.088,8.56,8.329,9.369,9.053,8.408,7.226,7.367,7.356,7.525,9.1,9.024,7.924,7.826,10.375,11.547,11.337,11.032,11.943,13.239,11.555,4.282,3.155,3.021,3.866,1.871,1.642,0.407,0.05,-1.165,-2.162,-1.389,2.066,-1.287,-1.919,-0.857,-0.96,-2.772,-4.008,-4.756,-5.949,-7.14,-7.313,-8.166,0.071,0.031,1.384,1.457,-0.364,-0.485,-1.539,-1.571,-1.976,4.387,5.673,5.684,6.638,6.759,7.167,8.133,8.518,9.166,3.973,4.606,3.603,3.923,5.171,5.876,6.681,7.792,9.041,9.371,9.963,-6.42,-7.766,-8.747,-9.543,-8.361,-9.844,-10.357,-11.821,-12.567,-11.995,-13.892,-8.524,-8.56,-7.286,-7.139,-9.767,-11.103,-12.203,-13.552,-14.623,-1.86,-3.024,-3.87,-4.247,-3.873,-5.066,-5.95,-7.16,-6.773,-4.145,-4.952,-4.25,-4.902,-5.424,-6.451,-7.611,-6.459,-8.284,-7.607,-8.909,-10.105,-9.963,-10.537,-11.344,-11.353,-12.424,-11.961,-12.729,-14.01,-12.209,-7.513,-8.182,-9.033,-9.117,-9.037,-9.367,-10.322,-10.3,-8.983,-16.607,-15.651,-16.308,-15.938,-14.493,-13.471,-13.618,-12.27,-12.532,-11.682,-19.964,-19.716,-18.421,-18.413,-19.703,-19.448,-19.499,-20.909,-20.976,-16.466,-16.314,-14.864,-14.317,-16.838,-18.249,-19.318,-18.764,-20.43,-20.121,-5.505,-4.442,-3.107,-2.084,-4.723,-4.068,-3.564,-2.198,-1.635,11.01,10.993,11.645,12.4,9.546,9.309,10.043,9.819,10.524,11.513,10.217,-1.077,-2.307,-3.329,-4.494,-2.031,-3.213,-2.861,-3.978,-3.742,-15.677,-16.799,-18.18,-19.017,-16.809,-16.965,-16.784,-16.758,-17.996,-21.321,-22.405,-22.797,-23.583,-23.616,-23.426,-24.551,-24.507,-23.828,-23.13,-23.834,-22.302,2.483,1.387,0.244,-0.926,1.881,0.802,1.303,0.258,0.719,-0.342,-0.897,-1.881,-2.941,0.228,1.045,2.251,3.076,4.321,-5.868,-6.389,-7.486,-8.504,-5.273,-4.33,-3.216,-2.421,-1.092,-0.389,-0.467,-13.527,-14.773,-15.777,-16.956,-14.484,-13.589,-13.122,-12.096,-11.546,-17.791,-19.139,-20.03,-20.999,-19.767,-19.928,-20.624,-19.724,-20.411,-21.302,-20.813,-19.882,-19.968,-20.04,-20.908,-20.087,-20.959,-21.635,-21.533,-22.47,-17.891,-18.905,-18.485,-19.037,-20.216,-20.955,-21.782,-22.951,-23.795,-20.385,-20.911,-22.241,-22.852,-19.868,-19.247,-18.313,-19.394,-17.91,-18.551,-24.065,-23.958,-23.015,-22.139,-25.33,-26.202,-27.492,-28.353,-29.638,-20.662,-19.308,-18.72,-17.509,-19.266,-17.88,-17.099,-17.106,-15.901,-15.878,-19.209,-18.754,-17.395,-16.532,-19.776,-21.115,-22.116,-23.489,-24.511,-17.199,-15.935,-14.798,-13.756,-16.018,-16.963,-16.953,-17.915,-17.879,-3.263,-1.943,-1.229,-0.287,-2.09,-2.717,-1.925,-2.336,-3.752,2.063,2.487,2.232,1.538,3.968,4.463,5.878,6.897,8.286,-6.481,-6.675,-5.718,-6.071,-6.638,-7.831,-8.043,-8.933,-9.239,-9.82,-6.445,-7.81,-8.736,-9.496,-8.377,-9.837,-10.336,-11.848,-12.323,-7.621,-6.921,-7.495,-7.729,-7.014,-6.377,-4.899,-4.32,-3.015,-2.137,-2.585,-7.55,-7.491,-8.911,-9.258,-6.633,-6.426,-5.548,-5.431,-4.666,-10.984,-11.591,-13.028,-13.481,-11.589,-12.314,-11.926,-13.407,-12.746,-13.653,-17.375,-16.605,-15.301,-14.259,-17.452,-18.61,-19.533,-20.719,-21.706,-7.781,-7.833,-6.455,-6.316,-8.395,-8.653,-9.325,-8.245,-9.307,-8.666,-2.92,-1.633,-1.852,-1.269,-0.649,-0.201,0.814,2.136,3.098,1.493,1.64,1.253,1.241,0.833,1.353,2.629,2.367,3.302,4.592,2.94,2.765,3.765,5.184,5.338,3.605,3.739,3.472,3.893,4.219,4.167,4.633,6.127,-27.306,-27.409,-28.058,-27.557,-28.242,-27.598,-28.536,-27.987,-27.161,-26.767,-26.756,-26.256,-26.828,-27.239,-26.946,-28.04,-28.717,-27.739,-28.415,-27.428,-26.172,-26.245,-24.917,-24.889,-26.882,-28.359,-29.007,-30.5,-31.155,-7.685,-7.631,-6.601,-5.799,-9.009,-9.044,-10.474,-11.206,-11.334,-10.792,-11.975,3.639,4.68,4.371,5.258,4.938,5.858,6.059,6.645,7.931,8.791,8.362,4.264,3.996,2.664,2.147,4.044,5.442,5.484,6.906,6.967,-8.781,-9.141,-8.98,-9.842,-8.288,-8.665,-7.904,-8.359,-7.705,-7.892,-7.683,-8.621,-8.956,-6.237,-5.955,-6.258,-5.5,-6.011,-5.549,-18.655,-18.751,-19.004,-19.68,-17.491,-16.199,-14.98,-14.663,-13.624,-12.789,-13.397,-22.694,-23.582,-23.754,-24.833,-23.102,-24.172,-23.56,-24.617,-25.14,-2.598,-2.009,-0.726,0.221,-3.015,-4.066,-5.185,-4.143,-5.885,-5.292,1.878,2.054,2.353,3.367,0.792,0.766,1.644,0.816,0.114,2.628,3.42,2.943,3.737,4.904,5.428,5.476,5.878,5.93,6.18,-6.657,-7.382,-7.435,-8.462,-6.706,-7.61,-6.83,-6.311,-5.592,-20.513,-21.544,-22.964,-23.879,-21.239,-22.379,-22.769,-23.886,-25.166,-25.509,-26.105,-18.111,-18.112,-17.925,-17.129,-19.432,-19.533,-20.883,-21.044,-22.382,-5.892,-5.783,-4.762,-4.181,-7.151,-8.002,-9.282,-10.225,-9.616,1.478,2.329,3.803,4.648,1.932,0.823,0.375,-0.453,-1.738,-2.356,-2.405,4.102],"z":[7.205,8.665,9.208,10.162,9.149,10.658,11.088,12.6,13.044,7.684,7.375,8.354,8.811,5.935,4.906,3.495,2.477,1.106,10.942,10.869,12.211,12.674,9.773,8.359,7.318,5.971,4.907,5.012,3.73,17.392,18.836,19.285,20.134,19.54,19.266,19.697,19.285,19.575,17.92,18.655,19.389,19.656,19.661,20.691,21.637,22.609,23.542,22.349,22.736,21.781,22.195,22.658,23.706,23.586,24.635,25.892,26.257,26.805,15.786,15.89,14.761,14.529,17.25,18.398,19.719,20.29,21.625,11.782,11.288,12.193,11.958,9.852,9.703,9.867,9.382,9.654,9.357,18.7,18.687,17.268,16.81,19.626,21.095,21.96,23.423,24.284,14.48,13.693,14.024,13.164,13.921,12.836,11.575,12.8,10.807,11.525,14.688,13.922,12.464,11.816,13.983,15.357,15.262,16.602,17.586,11.942,10.561,10.36,9.468,10.149,10.14,9.663,9.855,9.441,2.991,1.757,1.978,1.171,1.17,0.834,-0.187,-0.65,0.482,-1.489,-2.423,-3.129,-3.116,-3.449,-4.214,-5.219,-6.502,-7.526,1.32,2.105,1.288,1.798,2.952,4.07,3.896,5.396,5.089,6.036,-0.904,-0.069,1.282,1.665,0.142,0.953,1.186,2.031,2.312,2.48,2.408,3.77,4.3,1.842,0.415,-0.283,-1.669,-2.708,-2.537,-3.924,10.929,11.821,13.266,14.09,11.613,10.196,10.143,8.714,8.147,16.011,17.312,18.397,19.335,17.592,18.978,19.442,20.016,20.704,21.076,26.886,26.619,25.129,24.693,27.35,28.87,29.558,29.223,29.907,17.652,17.565,16.282,15.543,18.82,18.922,20.08,17.972,19.812,18.55,10.934,9.505,8.71,7.695,9.145,9.625,9.221,7.705,7.295,-3.525,-4.419,-4.083,-4.732,-4.443,-4.991,-6.448,-7.3,-7.85,-7.648,-8.581,-4.228,-5.117,-6.182,-6.203,-4.29,-3.452,-2.589,-2.05,-1.637,-1.686,-1.212,-6.981,36.774,37.772,38.574,38.774,38.736,38.124,39.181,38.644,38.554,38.959,38.076,38.566,39.808,40.075,40.431,41.002,42.306,42.166,43.359,44.582,40.491,40.366,39.411,39.4,39.995,41.172,40.789,42.02,41.662,23.513,22.544,21.461,20.273,23.239,22.298,23.065,23.816,23.366,22.161,24.111,16.538,16.068,17.155,16.951,15.316,14.934,14.222,14.187,13.139,12.021,13.206,21.503,22.581,23.423,24.301,23.496,22.911,23.959,23.426,24.456,31.61,32.459,31.935,32.691,32.547,33.479,33.382,34.305,35.748,30.634,30.067,30.091,30.264,28.653,28.138,28.828,27.052,28.195,27.114,37.564,37.743,37.082,37.552,37.236,35.746,35.386,35.591,35.464,35.135,35.639,39.33,40.549,40.36,41.263,41.128,42.597,43.091,44.604,45.08,24.932,23.849,22.474,21.557,23.935,24.974,26.319,24.846,26.95,26.096,22.826,23.225,22.631,22.025,24.755,25.228,26.747,27.29,28.785,20.269,18.871,18.534,17.522,17.933,18.219,18.064,18.679,18.417,18.793,20.601,20.343,20.225,20.76,19.045,18.983,17.589,17.326,15.997,31.099,32.546,33.299,34.242,33.017,34.513,35.354,36.778,37.523,36.991,38.799,27.007,27.124,27.885,27.486,27.875,27.955,28.777,28.743,29.649,22.744,22.704,22.486,21.807,23.963,23.673,24.924,24.595,23.878,18.302,17.477,16.38,15.492,18.357,19.207,20.221,21.275,22.414,22.66,23.304,16.406,-14.937,-14.904,-14.019,-14.409,-14.389,-14.37,-13.772,-13.905,-13.307,-13.07,-13.682,-14.803,-14.893,-14.181,-13.06,-13.598,-12.477,-13.009,-14.716,-15.786,-16.547,-16.906,-16.752,-16.197,-17.181,-16.726,-16.712,-17.134,-16.243,-17.304,-17.262,-16.218,-16.526,-17.006,-18.028,-17.539,-18.487,-17.988,-15.121,-15.411,-14.17,-14.05,-16.532,-16.177,-17.271,-18.478,-19.595,3.417,4.067,5.2,6.297,3.087,2.095,1.092,0.058,0.166,1.26,-0.799,11.836,11.861,12.589,12.331,12.436,11.449,12.119,12.89,13.396,12.242,11.944,11.179,10.948,11.21,10.012,10.115,8.687,8.905,8.022,-0.259,-1.715,-2.231,-3.063,-2.376,-2.271,-3.055,-2.98,-3.72,-0.535,-0.102,-0.923,-1.176,1.391,1.952,2.028,2.389,2.476,2.703,-4.026,-5.29,-5.096,-5.963,-5.826,-6.133,-6.65,-6.804,-7.27,-3.944,-3.639,-3.583,-4.223,-2.296,-2.269,-0.898,-0.841,0.498,-12.204,-11.669,-11.083,-10.298,-10.595,-11.097,-12.275,-12.635,-13.065,-5.007,-4.014,-4.517,-3.864,-3.676,-2.515,-2.101,-3.176,-2.764,4.174,5.169,6.348,7.454,4.503,3.643,2.412,3.887,1.964,2.825,7.923,8.128,8.681,9.618,6.823,6.899,5.527,5.49,4.095,9.548,10.553,10.628,11.716,11.933,12.041,12.334,12.672,12.755,12.526,13.06,5.366,5.199,5.224,4.438,6.304,6.159,7.271,7.17,8.308,1.953,2.038,1.538,0.787,3.477,3.64,2.994,4.37,3.319,4.153,-12.56,-13.689,-13.919,-14.159,-14.963,-14.912,-16.112,-15.993,-17.092,-8.517,-7.067,-6.483,-5.782,-6.439,-4.96,-4.31,-4.05,-3.03,-2.821,-4.233,-4.173,-4.465,-3.809,-5.196,-4.899,-5.928,-5.824,-6.885,3.902,4.707,6.179,6.91,4.065,2.697,2.834,3.564,4.019,3.831,4.658,9.007,10.051,9.497,8.266,11.082,10.481,11.49,10.974,11.734,13.056,11.168,10.307,13.253,13.189,11.874,11.187,14.358,15.724,16.78,18.127,18.686,18.023,19.931,8.51,8.037,6.574,5.757,8.886,8.424,8.431,8.043,7.896,1.033,-0.383,-1.128,-2.356,-0.576,-0.238,-0.49,-0.219,-0.157,-4.796,-4.72,-3.677,-3.932,-4.389,-4.344,-4.296,-5.534,-6.549,-6.489,-7.649,0.969,1.803,2.273,2.325,3.029,4.093,5.258,6.421,6.762,6.031,7.84,-1.73,-2.302,-3.047,-3.334,-1.215,-0.668,0.286,0.76,1.692,-3.988,-2.971,-1.57,-0.705,-3.116,-2.167,-2.507,-1.635,-2.026,-1.341,-0.03,0.266,1.428,0.185,1.601,2.654,2.15,3.791,3.514,11.317,12.752,13.501,14.528,13.266,13.093,13.363,12.223,12.169,13.197,11.065,12.836,13.628,15.082,15.659,13.587,13.954,14.118,14.158,12.797,13.258,12.396,12.983,12.258,12.13,11.126,11.409,9.801,10.277,9.27,14.068,13.038,11.691,11.055,12.928,11.747,11.974,12.001,10.706,11.243,11.416,10.498,9.795,11.18,12.013,13.39,11.669,13.86,12.836,8.728,8.503,7.011,6.49,9.236,9.396,9.615,8.293,8.455,-5.134,-5.224,-5.015,-5.763,-4.21,-3.918,-5.125,-4.804,-4.891,-5.294,-4.557,11.248,12.692,12.944,13.787,13.29,14.799,15.291,16.78,17.296,18.756,20.211,20.817,21.854,20.852,20.941,21.725,21.606,22.131,20.915,22.063,21.734,22.521,23.247,24.099,25.173,24.623,24.299,24.469,23.8,20.694],"type":"scatter3d","mode":"markers","name":"basic","marker":{"color":"rgba(0,0,255,1)","line":{"color":"rgba(0,0,255,1)"}},"textfont":{"color":"rgba(0,0,255,1)"},"error_y":{"color":"rgba(0,0,255,1)"},"error_x":{"color":"rgba(0,0,255,1)"},"line":{"color":"rgba(0,0,255,1)"},"frame":null},{"x":[-5.717,-5.027,-5.696,-6.506,-6.605,-7.334,-8.09,-7.333,-8.734,-7.106,-7.723,-9.093,-9.24,-6.802,-5.382,-4.542,-5.436,-10.093,-11.453,-11.566,-10.635,-12.438,-12.273,-10.293,-10.789,-10.193,-9.792,-12.317,-10.107,-9.56,-8.095,-7.69,-9.725,-7.303,-5.886,-5.7,-4.896,-5.119,-3.673,-5.149,-7.803,-8.275,-7.242,-6.948,-9.606,-6.676,-5.665,-4.45,-3.953,-5.234,-4.112,-4.585,-3.665,-3.99,-2.828,-3.096,-2.195,-2.421,-1.086,-0.832,0.176,0.506,1.148,0.577,2.501,1.922,2.867,-4.337,-4.73,-4.44,-3.934,-6.224,-6.67,-2.349,-0.968,0.169,1.311,-0.72,-0.634,-1.852,-0.307,-0.14,0.87,1.819,3.007,3.151,4.204,5.054,5.764,3.592,2.878,5.051,5.806,7.289,7.792,5.618,6.055,8.859,8.939,8.573,9.223,8.038,7.537,7.1,8.219,8.514,5.818,4.642,5.472,3.352,8.868,9.942,11.136,11.708,11.513,12.636,12.348,13.163,12.865,13.991,14.006,14.946,10.049,9.88,11.202,11.376,9.071,12.133,13.431,14.277,15.047,14.173,13.549,14.151,13.767,14.129,14.875,14.428,15.249,14.674,15.251,13.022,13.452,14.864,15.066,13.338,13.641,13.626,15.219,15.834,17.212,17.472,18.425,18.236,18.016,18.143,19.114,16.63,16.77,16.102,16.733,16.144,16.097,16.951,14.822,14.041,14.462,14.455,12.524,11.697,12.129,14.835,15.261,16.666,16.851,14.29,12.832,12.096,12.186,10.75,10.841,10.131,8.803,17.688,19.083,19.64,20.599,19.839,19.001,17.606,19.008,19.468,19.297,20.045,18.761,17.268,16.361,15.216,16.867,18.35,18.1,19.262,19.388,16.796,16.888,15.593,22.3,23.252,23.106,24.077,23.172,21.85,23.372,21.94,21.904,21.645,22.212,22.287,20.141,19.572,19.848,18.795,19.37,18.309,18.603,18.126,22.627,23.174,24.621,25.153,22.297,20.837,20.397,19.912,19.055,18.57,18.143,25.272,26.662,27.597,28.725,27.103,26.92,26.996,26.863,27.993,28.543,25.538,24.319,24.262,22.969,22.959,22.144,22.376,20.755,20.992,20.198,28.343,29.423,29.061,29.93,29.532,28.163,27.845,25.149,23.732,23.149,23.131,23.519,22.065,24.425,22.709,22.156,20.82,20.579,23.147,23.511,24.404,19.928,18.601,18.67,19.541,18.069,18.723,20.132,17.753,17.694,18.903,19.028,19.785,20.977,20.61,19.554,22.034,21.592,21.482,21.231,20.888,19.981,22.556,22.973,22.712,21.607,21.362,19.964,19.261,22.401,23.639,23.611,24.738,19.57,18.251,17.139,16.095,18.144,18.946,16.689,18.367,17.287,17.248,16.552,15.683,18.659,14.451,13.062,12.247,11.121,11.767,10.718,9.323,8.427,10.941,9.763,12.237,9.14,7.836,7.475,6.297,7.766,7.891,6.917,5.26,8.491,8.254,7.697,6.828,8.199,7.729,6.291,5.467,5.989,4.639,3.676,2.562,4.608,5.483,3.171,4.987,4.104,3.279,2.944,1.799,4.018,3.948,3.772,2.767,1.887,5.111,5.088,4.548,6.489,2.88,1.972,0.525,-0.399,2.4,0.322,-1.027,-1.611,-2.776,-1.059,-2.466,-0.606,-0.791,-0.804,-1.22,-1.278,-1.48,-1.511,-2.411,-2.454,-0.158,-0.29,0.152,0.96,-2.292,-2.986,-3.851,-4.055,-3.797,-4.205,-5.005,-4.748,-0.581,0.699,1.85,2.969,0.563,-0.37,-0.592,0.212,2.391,2.867,3.458,4.571,1.72,1.172,2.222,2.73,3.17,4.523,5.256,4.872,6.14,7.217,8.217,5.899,4.876,4.76,5.269,7.029,7.976,9.44,10.277,7.645,6.52,6.149,7.476,10.892,11.326,11.609,12.594,10.27,10.233,10.18,9.034,10.766,10.943,12.266,13.023,9.773,9.844,14.923,15.982,16.245,17.373,15.584,16.413,17.859,18.148,18.776,15.289,15.532,16.957,17.606,14.548,17.449,18.777,19.946,21.044,18.759,17.864,18.401,16.479,17.58,15.649,16.204,15.388,19.809,19.789,19.627,20.236,18.694,18.719,17.49,20.012,17.055,16.514,15.525,14.509,15.781,15.168,16.738,12.511,11.127,10.706,9.784,10.318,11.21,12.559,11.372,11.065,11.184,10.473,12.022,13.376,12.067,12.289,11.162,10.933,13.636,14.788,15.152,15.332,10.446,9.328,8.245,7.624,8.777,9.725,10.552,9.8,11.44,10.684,11.507,8.729,9.25,9.009,8.398,10.762,10.961,11.401,12.411,9.437,9.263,7.787,7.425,9.927,9.901,10.345,10.799,6.933,5.498,4.947,4.157,4.743,4.998,5.464,4.695,6.665,7.209,6.447,6.224,8.695,9.723,6.037,5.278,3.92,3.492,5.092,6.439,4.058,6.389,3.259,1.966,2.117,1.293,1.36,1.013,0.34,0.1,3.179,3.468,3.644,3.034,4.738,5.189,4.456,4.452,4.696,3.397,3.118,5.685,5.791,7.052,2.597,1.334,0.409,-0.282,0.645,1.526,-0.742,1.063,0.432,-0.391,0.094,-0.719,-0.351,-1.351,1.411,1.955,1.664,1.385,3.497,4.167,3.884,1.726,1.517,0.08,-0.283,2.339,3.831,4.722,6.385,-0.736,-2.12,-3.076,-3.818,-2.28,-1.375,-1.756,-0.133,-0.91,0.72,0.332,-3.122,-4.005,-5.484,-6.155,-3.762,-3.319,-2.405,-5.889,-5.575,-6.559,-6.215,-4.131,-3.564,-3.488,-3.132,-2.992,-2.633,-2.563,-7.787,-8.857,-8.646,-7.756,-10.213,-10.191,-10.488,-9.439,-9.278,-9.542,-8.862,-10.326,-10.299,-10.296,-9.073,-7.923,-6.764,-6.048,-7.509,-6.855,-7.006,-6.718,-5.977,-8.391,-9.534,-8.443,-7.23,-6.997,-5.529,-4.979,-7.878,-9.247,-4.891,-3.486,-2.615,-1.656,-3.015,-1.625,-1.652,-1.191,-2.949,-2.243,-0.729,-0.095,-2.582,-1.694,-1.864,-0.72,-1.078,0.073,-0.106,-0.155,1.283,1.721,2.83,1.712,1.592,1.95,2.491,0.859,1.173,1.209,2.067,0.113,0.027,0.284,0.229,1.482,2.012,1.954,3.167,4.37,5.171,3.396,4.778,2.331,4.464,5.565,5.582,6.646,5.456,4.402,4.301,4.687,5.325,2.887,2.687,3.607,1.235,4.304,4.632,6.144,6.741,3.884,6.766,8.211,8.939,10.062,8.649,8.201,8.665,8.761,8.294,8.895,8.657,9.201,8.37,11.643,12.553,12.9,14.071,12.057,12.413,12.448,12.746,12.812,13.11,13.145,13.537,15.022,15.482,16.186,15.796,14.28,14.725,13.612,16.622,16.02,16.97,17.639,14.75,13.695,13.32,12.853,12.295,11.987,12.744,11.019,11.78,10.931,17.019,17.859,16.983,15.757,18.336,17.218,19.021,16.324,15.527,14.257,13.172,16.33,15.517,15.214,15.024,14.429,14.237,13.937,11.869,11.287,10.245,9.133,12.379,10.595,9.695,8.366,7.295,10.388,11.556,9.452,8.435,7.227,6.335,5.122,7.567,8.327,6.296,7.625,4.743,3.658,2.38,1.332,4.075,5.082,2.849,5.663,2.477,1.309,0.648,-0.575,1.592,1.605,0.415,2.79,0.404,2.791,1.594,1.446,1.447,0.902,0.913,0.121,0.127,-0.792,-0.766,2.025,2.048,2.998,2.721,-1.524,-2.632,-3.944,-3.988,-2.305,-1.174,-1.039,-1.442,-6.844,-7.314,-8.711,-8.901,-7.139,-5.69,-4.661,-5.347,-3.329,-4.012,-3.011,-1.688,-10.113,-10.129,-9.227,-9.652,-9.609,-9.446,-10.571,-7.998,-6.995,-7.37,-7.204,-5.602,-4.563,-5.181,-7.876,-8.279,-9.318,-9.109,-10.471,-11.537,-11.025,-11.309,-12.55,-12.431,-10.939,-10.235,-9.678,-8.812,-8.907,-8.864,-7.996,-7.101,-7.847,-7.578,-6.132,-5.303,-5.249,-8.784,-9.543,-10.405,-10.456,-10.404,-11.162,-10.201,-11.944,-11.056,-11.895,-11.07,-11.514,-12.558,-8.252,-7.857,-8.993,-8.767,-7.429,-6.771,-10.216,-11.386,-11.671,-12.249,-12.614,-12.587,-13.86,-12.427,-11.262,-11.453,-10.301,-10.532,-11.605,-11.574,-12.906,-9.065,-7.88,-7.611,-7.106,-6.623,-5.391,-6.854,-7.935,-7.736,-9.056,-9.145,-6.61,-5.307,-4.567,-4.816,-3.37,-3.618,-2.9,-1.717,-10.079,-11.439,-11.58,-12.556,-12.049,-11.329,-9.909,-10.622,-10.69,-10.664,-11.063,-9.592,-8.215,-7.308,-7.564,-6.125,-6.255,-7.959,-5.334,-7.044,-5.745,-10.245,-10.203,-11.62,-11.828,-9.313,-9.777,-7.86,-12.602,-14,-14.528,-15.568,-14.856,-14.508,-15.309,-16.3,-14.875,-13.944,-14.225,-15.256,-15.614,-12.916,-11.729,-10.93,-11.433,-9.867,-10.379,-9.6,-8.562,-15.772,-16.715,-18.136,-18.795,-16.116,-14.717,-14.49,-13.624,-13.194,-12.325,-12.11,-18.627,-19.968,-21.062,-22.111,-20.909,-20.846,-21.885,-22.134,-19.44,-18.432,-18.669,-17.251,-17.746,-16.317,-16.567,-22.486,-23.504,-23.018,-22.294,-23.419,-23.067,-21.616,-21.226,-24.026,-23.881,-22.858,-24.9,-20.831,-19.425,-19.265,-18.17,-18.517,-18.556,-17.569,-18.227,-20.364,-20.414,-19.741,-19.607,-21.868,-22.594,-23.898,-21.954,-24.545,-22.591,-23.884,-24.508,-19.323,-18.652,-17.94,-18.23,-19.642,-20.745,-20.535,-21.926,-17.005,-16.224,-17.073,-16.78,-15.329,-18.138,-19.028,-19.677,-19.658,-20.094,-20.228,-20.881,-19.885,-20.202,-21.521,-18.679,-17.626,-17.255,-17.145,-16.371,-16.708,-15.22,-15.594,-17.124,-16.783,-17.886,-17.611,-16.517,-19.133,-20.288,-20.605,-21.457,-21.522,-21.39,-22.487,-23.467,-22.319,-19.926,-20.171,-19.591,-18.374,-19.646,-20.181,-19.853,-21.013,-20.475,-20.07,-19.304,-18.413,-21.285,-19.646,-18.959,-17.522,-16.621,-19.73,-21.072,-21.931,-21.097,-17.308,-15.967,-15.14,-14.012,-15.99,-14.574,-16.613,-15.72,-15.046,-14.676,-13.532,-15.934,-16.193,-11.83,-10.583,-9.944,-8.718,-10.781,-10.318,-9.751,-8.718,-11.461,-12.026,-10.941,-10.408,-9.955,-8.553,-7.663,-10.924,-12.221,-10.423,-8.36,-7.072,-5.96,-4.902,-7.164,-7.921,-5.774,-8.212,-6.211,-5.209,-4.881,-3.715,-5.686,-4.609,-3.523,-5.241,-5.906,-5.703,-4.867,-4.081,-7.041,-7.586,-8.647,-6.864,-5.012,-4.271,-2.771,-2.019,-2.334,-0.913,-0.19,1.042,-0.729,-1.144,-1.049,-0.267,2.426,3.679,4.537,5.748,3.404,3.897,4.591,5.199,6.303,3.628,4.345,3.067,6.288,7.462,7.892,7.533,7.198,8.464,9.527,8.359,8.684,9.166,10.115,10.39,9.806,8.803,9.556,10.198,7.552,6.382,5.884,4.672,6.623,6.859,5.428,7.242,6.808,6.434,5.582,4.451,7.677,8.355,7.281,6.103,5.377,4.071,3.041,6.228,7.399,7.387,8.425,4.11,2.927,1.775,0.607,3.275,4.29,2.055,2.089,1.041,0.778,-0.048,1.331,1.057,-0.234,2.087,-0.492,1.843,0.551,0.317,1.452,1.279,-0.179,-0.688,2.132,-2.891,-3.663,-3.425,-4.331,-3.285,-4.307,-5.669,-3.798,-2.215,-1.869,-2.674,-3.215,-0.365,-0.006,-2.759,-3.51,-4.988,-5.651,-3.35,-4.277,-1.909,-5.486,-6.881,-7.221,-8.209,-7.184,-8.654,-9.442,-8.764,-5.655,-5.469,-6.605,-7.367,-4.149,-3.956,-10.598,-11.021,-10.692,-11.328,-10.434,-10.681,-9.877,-12.167,-8.125,-7.708,-6.391,-5.449,-7.553,-6.991,-8.9,-2.995,-1.98,-1.396,-0.875,-0.928,-1.039,-2.531,-3.039,-3.891,-3.305,-3.568,-5.313,-6.038,-6.857,-5.755,-2.505,-1.873,-0.893,-0.812,-1.12,-1.999,-3.273,-1.543,-4.08,-2.34,-3.609,-1.072,-1.881,-2.447,-2.448,-3.002,-2.497,-3.666,-1.529,-2.893,-3.437,-2.312,-2.506,-4.032,-4.876,-4.015,-6.001,-1.128,0.034,0.388,0.755,1.226,-1.729,-2.769,-2.54,-2.733,-4.16,-4.639,-2.091,-1.808,-0.695,-0.785,-1.387,-1.099,-2.306,-0.734,0.347,1.465,0.966,1.403,2.553,3.048,3.712,0.04,-0.532,-1.333,-1.22,-1.448,-0.613,-2.211,0.491,-2.125,-2.934,-2.05,-2.308,-3.927,-4.63,-4.957,-1.01,-0.075,0.653,0.834,0.938,2.088,0.222,1.038,1.718,0.835,1.312,2.065,-0.452,-1.407,-1.573,-1.727,-2.762,0.705,1.883,2.553,3.427,2.904,2.378,1.999,2.234,1.481,1.715,1.338,2.163,2.727,4.243,4.905,4.791,6.237,7.063,8.228,6.544,5.861,6.472,7.173,7.412,8.266,6.411,6.667,6.83,7.968,7.744,5.522,5.477,5.406,5.507,5.366,5.467,5.396,9.192,10.381,10.718,10.106,11.597,11.889,11.307,11.716,12.137,12.637,12.394,11.663,10.498,9.769,9.351,9.542,8.221,10.201,9.651,8.987,9.72,9.093,8.966,8.215,8.353,8.764,8.023,11.048,11.866,11.581,11.378,13.347,11.537,11.272,9.851,9.645,11.515,8.879,7.483,7.267,6.621,6.542,5.082,4.477,4.322,3.13,2.981,2.386,7.806,7.658,8.394,7.925,8.111,7.194,7.528,6.858,8.558,8.846,7.99,6.789,6.49,7.517,6.485,6.847,5.15,5.892,4.188,4.562,6.105,4.94,5.291,4.508,4.206,3.467,2.402,1.181,6.471,6.908,7.069,6.747,8.215,7.559,7.736,6.369,6.159,8.572,8.641,9.978,5.439,4.084,3.413,2.874,3.225,1.769,3.756,3.501,2.916,3.502,2.772,3.122,2.767,2.271,4.814,5.481,4.971,4.622,6.998,7.682,4.896,4.898,4.431,4.434,2.996,3.003,2.683,2.698,4.535,4.527,5.874,4.143,2.13,0.732,0.542,-0.437,-0.08,-0.476,-1.084,-1.46,1.468,1.34,2.228,2.213,2.071,1.447,0.547,-0.231,0.711,1.508,2.905,3.103,3.873,0.652,-0.144,0.509,1.648,-0.267,-1.31,-1.133,-2.485,-2.101,-3.456,-3.258,-4.219,3.998,3.536,4.972,3.671,2.754,3.432,4.665,2.326,3.421,2.628,3.161,3.449,2.581,2.183,1.845,0.852,3.117,4.681,5.077,4.69,4.225,6.59,7.276,4.105,4.508,5.135,4.862,5.491,5.964,6.635,5.634,5.764,7.614,4.626,3.61,2.706,2.375,2.763,1.639,3.66,3.493,4.34,4.5,4.347,5.708,4.793,4.958,3.667,3.688,5.402,5.6,6.71,5.925,2.543,1.257,0.906,0.262,0.142,-1.127,-2.303,-1.332,-3.224,-2.656,-0.524,-3.192,-1.056,-2.379,1.328,1.079,1.73,1.15,1.666,1.698,2.944,2.541,1.047,0.671,3.247,2.582,4.704,3.157,0.203,-1.233,-1.853,-2.816,-0.82,-0.645,-1.234,-0.891,0.846,1.443,-2.161,-2.744,-3.327,-3.134,-3.767,-4.835,-2.432,-1.317,-0.614,-0.212,-0.333,-0.479,0.162,-0.648,-0.086,0.33,1.306,0.858,1.428,-1.97,-2.836,-2.78,-2.708,-2.832,-2.773,-1.449,-1.436,-2.926,-2.748,-2.247,-3.16,0.736,0.87,0.135,0.727,0.402,-1.146,-1.935,-1.442,-1.452,-3.414,-4.12,-5.517,-4.158,-0.994,-0.47,0.79,0.996,-0.17,-1.366,1.491,1.042,0.613,1.114,-0.1,-0.571,-2.018,-3.329,-0.277,-0.758,0.279,0.166,-2.052,-1.803,-3.186,-2.928,1.289,2.348,3.423,3.753,3.025,4.253,2.048,3.965,5.017,4.499,5.212,5.716,6.91,6.164,3.261,2.663,1.346,0.274,2.405,3.595,3.857,4.462,4.951,5.563,5.799,6.875,1.415,0.22,-0.834,-2.004,0.803,2.095,2.633,-0.439,-1.377,-2.479,-3.58,-0.644,-0.03,1.459,2.188,1.923,-2.191,-3.169,-4.362,-5.418,-2.529,-1.978,-1.429,-6.416,-7.586,-8.706,-9.886,-7.219,-6.769,-6.144,-7.882,-8.342,-9.333,-10.113,-11.091,-8.665,-8.195,-9.112,-6.84,-8.692,-6.41,-7.344,-6.924,-9.686,-10.34,-10.805,-11.016,-9.371,-8.784,-9.497,-7.538,-8.98,-7.013,-7.737,-11.028,-11.471,-12.858,-13.298,-11.462,-12.219,-13.651,-13.482,-13.116,-12.637,-12.322,-12.448,-13.565,-11.404,-13.283,-11.964,-10.047,-11.216,-9.302,-9.892,-13.328,-12.998,-11.531,-11.193,-13.272,-14.458,-14.062,-7.412,-6.73,-5.424,-4.453,-6.436,-6.07,-7.642,-5.417,-4.243,-3.786,-4.601,-4.51,-5.66,-4.753,-2.468,-1.871,-2.418,-2.644,-0.387,-0.21,-1.435,-2.627,-3.114,-4.481,-4.835,-5.253,-6.587,-7.558,-7.277,-7.078,-7.248,-8.703,-9.702,-10.112,-10.282,-10.861,-10.142,-9.05,-10.242,-10.617,-9.546,-9.86,-10.888,-12.307,-13.217,-12.507,-8.286,-7.163,-7.043,-6.757,-5.831,-5.808,-4.624,-5.802,-9.494,-10.622,-10.336,-10.516,-11.898,-7.457,-6.329,-6.798,-6.246,-8.286,-7.238,-6.563,-6.441,-6.187,-5.015,-6.845,-6.159,-5.506,-6.449,-6.004,-4.899,-3.732,-2.514,-1.8,-7.75,-8.729,-8.708,-8.83,-8.518,-8.449,-7.209,-7.232,-6.119,-4.86,-5.029,-4.568,-3.69,-3.418,-2.427,-3.007,-5.695,-5.931,-6.744,-6.478,-6.668,-7.709,-8.524,-7.618,-7.708,-9.64,-10.549,-11.248,-11.574,-6.717,-5.78,-4.92,-4.679,-4.893,-4.456,-3.636,-4.414,-3.873,-3.091,-2.287,-2.223,-5.676,-5.676,-6.524,-6.492,-6.856,-6.834,-6.95,-6.92,-7.82,-7.774,-7.55,-8.523,-4.929,-3.661,-3.496,-2.749,-2.477,-1.172,-2.356,-0.059,-6.066,-6.329,-7.407,-8.217,-5.037,-4.022,-2.687,-4.575,-10.135,-11.454,-11.661,-12.713,-11.626,-11.372,-13.036,-10.631,-10.724,-11.029,-11.748,-10.526,-10.729,-11.71,-11.685,-9.375,-8.289,-6.985,-8.739,-12.588,-13.584,-14.384,-14.449,-14.572,-14.046,-15.237,-16.762,-13.823,-12.938,-12.555,-12.476,-11.687,-11.674,-13.046,-10.622,-12.339,-11.985,-13.146,-12.955,-11.638,-11.18,-15.491,-15.568,-14.579,-14.969,-15.264,-16.427,-16.502,-16.403,-16.665,-12.871,-12.968,-14.084,-13.841,-13.094,-15.296,-16.441,-16.939,-17.423,-17.602,-17.238,-16.685,-17.426,-16.325,-17.069,-16.517,-16.151,-15.034,-13.928,-13.072,-12.639,-13.056,-13.824,-12.92,-14.406,-9.97,-8.62,-7.574,-7.448,-8.315,-6.864,-9.261,-4.534,-3.644,-2.262,-1.592,-3.551,-4.909,-5.143,-1.836,-0.544,-0.471,0.596,-0.351,-1.453,-1.62,-1.691,-1.494,-1.029,-2.995,-3.117,-2.411,-3.967,-1.834,-1.653,-0.154,0.284,-2.259,-3.759,-4.542,-4.391,-5.928,-5.778,-6.548,2.109,2.541,2.245,3.141,1.832,2.355,1.998,1.589,1.002,0.603,1.393,1.787,-0.9,-1.475,-1.046,-2.985,1.641,2.41,3.808,4.308,2.502,1.178,0.297,1.028,4.646,4.446,4.22,4.688,3.287,3.67,3.526,3.272,4.598,4.856,2.256,0.857,2.212,-0.193,5.453,6.749,7.59,8.256,7.502,6.843,7.751,6.553,7.546,8.294,7.831,8.655,8.173,8.735,8.93,6.519,5.955,6.376,6.882,4.406,3.826,3.966,6.189,6.55,8.035,8.409,6.148,4.62,6.669,4.093,8.873,10.302,10.606,11.463,11.062,12.463,9.882,10.104,9.623,10.261,9.385,7.96,9.836,8.495,7.886,8.41,8.443,6.368,5.664,3.882,3.313,8.799,9.284,10.587,10.652,8.204,6.882,6.654,5.867,5.434,4.641,4.424,11.662,12.961,13.519,14.057,13.866,13.226,11.765,11.148,11.175,12.551,13.261,10.693,10.218,9.228,10.766,8.79,10.334,9.344,12.926,14.198,14.072,12.979,14.518,13.479,14.609,15.185,15.098,14.149,13.418,16.542,17.325,16.595,11.499,10.131,9.751,8.621,10.024,11.011,10.738,9.362,8.599,11.852,11.929,11.622,9.018,7.722,6.542,5.536,7.692,8.611,6.666,5.585,5.426,4.316,5.83,4.637,3.446,5.013,3.614,2.249,1.461,0.372,2.276,0.942,-0.07,0.704,-1.3,-0.522,-1.525,2.016,1.351,1.134,0.128,2.113,2.076,2.884,0.639,2.057,1.899,0.717,-0.104,3.161,4.251,0.631,-0.464,-1.779,-2.79,-1.769,-2.963,-3.373,-4.543,-2.732,-3.918,-2.506,-2.402,-2.676,-3.277,-4.209,-1.403,-2.759,-3.264,-4.679,-5.528,-2.318,-2.619,-2.637,-1.581,-4.938,-6.258,-7.279,-8.436,-6.224,-6.858,-7.752,-8.036,-9.104,-7.141,-7.061,-6.319,-8.461,-7.081,-7.22,-8.038,-8.391,-5.848,-11.044,-10.804,-11.081,-11.761,-9.384,-9.247,-9.642,-8.73,-9.522,-8.608,-9.004,-8.878,15.857,15.206,15.564,16.712,15.706,15.072,15.387,15.814,15.857,15.46,15.862,17.264,17.788,18.782,17.367,19.008,18.153,16.401,18.005,16.252,17.052,14.682,14.283,15.325,16.028,12.894,12.939,11.821,14.835,14.453,15.026,15.559,12.926,12.463,12.49,12.031,12.098,11.636,11.67,18.878,19.196,18.924,19.765,18.394,17.755,17.355,18.338,18.725,15.923,15.009,15.502,18.773,19.715,21.043,21.593,19.961,18.649,21.012,17.93,20.528,20.47,21.846,22.312,19.501,18.052,19.731,17.042,22.52,23.819,24.919,25.727,24.262,23.523,23.974,22.334,23.249,21.6,22.056,24.933,24.935,25.942,25.938,25.718,25.717,26.643,26.636,25.934,25.903,24.649,26.252,24.401,24.49,25.9,26.884,24.047,22.587,22.447,21.663,28.123,28.306,28.697,28.177,29.26,28.704,27.635,29.248,27.122,28.742,27.678,27.154,27.834,26.754,25.556,25,26.365,25.108,27.507,25.203,24.078,24.335,23.43,23.718,22.614,23.279,25.56,25.906,25.687,24.943,26.351,26.218,24.758,24.272,27.052,28.165,27.446,24.051,22.644,21.789,20.955,22.094,22.014,21.275,21.482,20.509,21.674,21.281,21.009,22.739,23.036,22.546,21.965,24.533,24.928,24.186,26.427,22.751,22.303,20.784,20.264,22.737,18.851,18.548,18.768,17.958,19.408,18.996,19.842,20.148,19.116,18.92,21.565,22.729,24.041,22.762,18.465,17.439,16.056,15.368,17.358,16.185,18.668,15.664,14.356,14.25,13.183,13.961,12.587,13.959,15.357,15.394,16.603,17.575,15.544,14.484,13.134,14.828,12.155,13.857,12.523,11.554,16.518,17.55,18.258,19.422,16.788,15.75,15.258,17.567,18.194,19.407,20.279,17.18,16.36,16.656,15.08,15.634,14.655,14.249,13.433,13.032,12.637,19.494,20.622,21.926,23.021,20.359,20.092,19.172,21.807,22.971,23.755,24.897,22.538,21.873,21.372,21.82,20.431,25.422,26.723,27.707,28.791,26.514,25.28,24.076,25.298,22.918,24.147,22.963,21.829,27.359,28.231,28.629,28.761,27.581,27.226,28.198,25.928,27.887,25.605,26.585,28.893,29.291,30.639,30.932,31.666,31.596,31.424,31.415,30.504,30.919,31.458,30.811,31.887,31.237,31.778,31.279,31.14,29.732,28.849,29.535,28.255,27.117,27.18,28.415,27.178,26.177,27.246,26.071,24.887,23.635,22.574,24.923,26.089,25.913,26.16,23.742,22.604,21.413,20.318,23.049,22.956,21.781,24.061,21.711,24.005,22.826,22.767,21.612,20.517,20.769,21.89,20.316,21.374,21.238,22.447,19.73,19.792,20.929,21.679,18.457,21.063,22.111,23.505,24.403,21.946,23.669,24.946,25.318,26.472,24.876,24.329,24.531,24.914,25.81,23.26,22.976,23.444,21.681,24.273,24.539,25.96,26.561,23.524,26.487,27.838,28.899,30.091,27.909,27.044,27.064,26.714,27.485,28.474,29.408,30.191,29.653,28.671,29.614,30.503,29.423,31.482,32.359,32.504,32.66,33.728,32.462,32.592,31.35,31.445,32.862,34.255,34.597,34.945,30.187,28.934,28.979,28.725,27.727,26.462,27.554,29.353,29.45,30.392,30.041,29.912,28.971,28.759,27.651,28.19,27.594,29.327,29.971,30.432,30.276,31.165,30.729,31.731,30.967,31.427,30.269,30.407,32.013,33.136,32.45,29.12,27.945,27.491,27.252,26.788,27.15,25.479,26.146,27.412,27.002,27.976,27.557,26.883,25.478,25.592,24.779,29.274,30.281,30.132,30.398,31.693,32.129,32.139,32.472,29.673,29.469,28.401,28.329,27.579,26.522,26.986,26.303,25.326,24.539,23.51,23.868,27.15,25.836,25.37,24.82,24.837,25.587,25.207,25.954,25.368,25.53,25.195,24.755,27.121,26.656,25.407,25.47,27.746,27.385,26.271,28.328,24.269,22.984,22.528,21.569,21.919,22.158,21.114,19.513,24.361,24.352,24.509,25.207,23.096,23.108,23.051,21.863,23.899,23.992,25.439,25.929,23.129,21.747,23.352,26.134,27.522,28.424,29.325,28.028,27.335,26.768,27.374,28.168,28.959,28.95,29.96,28.491,28.73,29.242,27.83,27.726,27.858,27.652,26.42,26.395,27.238,25.527,27.22,25.5,26.35,26.335,28.218,28.366,29.379,29.088,28.763,30.503,30.007,28.909,28.793,29.489,29.792,31.27,29.369,28.133,27.064,27.647,27.198,26.196,25.14,28.658,29.328,29.971,29.947,30.425,31.061,29.834,30.531,31.162,30.159,30.38,31.792,32.545,33.915,32.692,27.341,26.803,27.742,27.457,26.392,25.93,30.579,29.884,28.608,28.14,29.568,30.764,30.312,31.812,25.44,24.403,23.085,23.059,24.825,23.677,26.049,20.177,19.86,18.575,18.41,19.856,19.439,20.209,17.982,18.477,18.828,18.825,19.699,19.379,20.208,18.185,19.12,19.455,18.228,18.338,19.88,21.161,22.192,21.334,23.38,22.514,23.537,15.744,15.503,16.43,15.998,15.683,14.761,15.079,13.296,17.699,18.682,18.253,18.293,20.078,21.367,21.436,21.494,17.813,17.362,16.197,16.153,16.958,15.541,16.058,16.745,16.614,17.025,16.202,17.46,18.136,17.088,17.276,18.976,19.782,20.716,20.578,15.974,14.905,14.357,14.096,13.758,14.273,12.654,14.189,13.698,14.707,14.327,13.447,12.302,13.165,10.999,15.991,17.042,17.122,17.208,18.422,19.539,18.38,17.072,17.117,15.898,16.013,17.174,16.951,18.521,14.744,13.504,13.647,13.219,12.333,14.246,14.459,15.445,15.318,14.98,16.04,15.51,14.13,13.726,15.6,17.008,17.466,17.894,18.787,19.222,19.668,13.405,12.089,11.114,10.957,10.488,9.504,10.044,9.27,8.855,9.833,11.363,11.965,11.894,12.056,13.408,11.665,11.562,10.113,9.377,12,12.314,13.454,11.469,13.751,11.756,12.902,9.711,8.354,8.091,9.009,8.11,8.993,8.362,6.839,6.474,7.245,7.683,10.215,11.606,11.718,12.656,12.124,13.492,12.203,10.733,10.692,10.639,11.357,9.471,9.389,8.074,7.338,7.775,9.809,9.668,11.011,11.428,8.62,11.702,13.002,14.023,14.749,13.507,14.067,15.011,14.784,15.736,14.944,16.02,17.335,15.733,18.351,16.739,18.053,13.527,13.192,13.433,13.839,11.748,11.539,10.146,9.835,9.31,15.738,17.186,17.653,18.401,17.86,19.351,20.198,19.904,21.579,21.284,22.12,17.208,17.595,17.137,17.862,17.085,17.777,19.597,19.865,15.955,15.438,16.366,16.592,14.033,16.899,17.817,19.122,19.685,18.122,19.14,16.844,19.594,20.817,20.61,21.415,21.206,22.345,21.629,19.498,19.148,19.054,19.568,17.814,17.354,18.003,18.432,18.288,19.673,19.965,17.434,17.185,20.523,20.524,21.879,21.872,22.698,22.671,23.392,23.305,22.587,22.603,21.865,23.886,22.602,23.336,22.9,23.674,23.23,24.136,23.774,25.599,21.666,21.187,21.282,21.044,23.857,24.825,24.652,25.564,26.252,26.512,26.154,26.548,25.425,23.49,23.225,22.793,21.833,22.151,22.678,23.036,22.819,23.519,23.302,23.649,24.124,14.195,16.512,11.712,11.959,16.733,14.121,15.185,14.725,13.378,13.007,12.445,15.588,16.032,16.944,17.803,16.806,12.237,11.337,9.964,10.041,11.46,8.685,8.986,7.904,14.354,13.293,13.756,15.121,15.46,12.859,16.045,16.023,16.174,17.075,18.446,18.373,16.977,19.72,19.596,19.853,21.324,21.632,22.174,-7.642,-10.132,-5.966,-5.486,-9.647,-8.008,-9.051,-8.884,-7.719,-7.181,-7.118,-9.848,-9.587,-10.498,-10.048,-11.66,-5.997,-5.382,-4.122,-3.999,-5.179,-3.125,-2.941,-2.331,-7.566,-6.59,-6.84,-7.98,-8.431,-5.987,-8.602,-8.664,-9.485,-10.162,-11.497,-11.633,-10.375,-12.545,-12.916,-13.816,-15.185,-16.168,-15.28,-9.209,-12.249,-9.107,-5.864,-8.926,-10.378,-11.652,-12.28,-11.412,-10.233,-11.633,-13.688,-14.674,-16.123,-16.962,-16.431,-7.789,-7.962,-6.788,-5.868,-6.495,-6.659,-4.603,-3.599,-7.7,-6.404,-5.674,-6.535,-7.78,-4.207,-6.335,-5.184,-10.362,-10.136,-11.301,-12.217,-11.655,-11.484,-13.477,-13.126,-14.248,-15.093,-14.262,27.49,30.857,28.278,24.273,26.821,29.211,30.505,31.441,30.732,29.339,31.313,32.951,33.317,34.808,35.342,35.441,26.507,26.948,25.842,24.706,25.136,25.964,23.373,22.856,25.887,24.605,23.648,24.351,25.751,22.152,23.866,22.881,28.603,28.156,29.261,30.383,29.982,29.201,31.797,32.205,33.613,34.458,33.875,8.749,6.924,-8.196,-4.944,6.966,5.946,8.226,15.186,19.441,20.397,-1.319,-9.798,-0.234,0.628,-0.281,-4.075,7.453,-2.573,11.648,25.484,-11.385,10.286,-0.933,25.563,11.143,-4.831,3.508,-15.423,6.427,9.964,-4.274,-11.112,13.134,23.007,-6.252,-8.772,-2.984,20.646,-11.402,-4.638,19.363,-11.908,22.464,6.951,-12.511,10.066,19.149,9.707,19.383,-0.343,14.374,-3.928,14.664,-3.288,9.286,-12.685,3.921,6.7,-0.511,-3.425,9.441,1.611,13.277,-4.18,-12.983,9.626,23.775,7.178,-13.781,4.392,13.602,-12.573,-0.992,8.675,24.247,0.439,15.895,-9.914,-13.199,13.032,-2.526,9.727,-11.788,-12.808,7.239,-14.388,-1.403,3.195,4.749,6.58,-10.009,7.616,3.892,11.89,-11.722,-11.025,-4.957,23.378,6.206,-4.689,-4.758,-2.413,-2.627,0.364,3.538,-7.811,1.748,12.022,-2.835,10.063,17.457,5.159,0.895,14.65,-6.669,1.939,-8.093,8.871,-1.75,4.34,10.606,5.668,6.71,17.927,10.133,-8.986,5.099,19.113,20.376,-11.577,5.866,17.642,-12.778,-1.277,11.129,9.864,-0.46,-11.781,17.446,13.847,-27.06,1.609,-9.967,-5.248,13.455,16.573,-18.657,14.042,-10.035,-18.092,-15.524,1.024,2.268,-1.17,-22.492,-17.735,-13.529,2.252,-12.965,-0.991,-11.111,-13.99,-23.299,8.247,17.209,3.137,5.896,-5.966,1.652,-5.905,6.095,-14.987,-9.439,4.903,3.05,14.901,-7.638,-5.132,19.797,-8.271,-1.431,1.927,-8.215,-5.173,11.651,-8.916,3.59,-4.072,-2.822,-4.365,-11.668,-6.014,-0.887,2.758,-9.728,-1.541,7.404,12.674,1.196,9.124,3.281,0.175,8.775,-6.593,-2.874,10.798,-11.116,-6.159,2.029,-0.261,-5.804,-2.83,2.83,-3.467,-2.577,-10.66,-8.123,12.958,-1.241,1.67,-13.066,-13.412,-10.599,-0.244,15.866,6.593,-6.377,-11.283,-12.669,-14.439,-12.588,8.018,-2.136,2.521,1.048,-12.865,9.171,16.441,10.814,-5.878,-10.182,-6.155,-12.134,-3.201,-8.2,3.293,8.301,-13.33,5.108,5.242,-24.431,-5.809,-12.409,13.709,7.015,-4.861,5.495,2.992,-13.372,9.546,2.841,-27.539,8.085,-11.938,-13.807,-14.589,-0.125,-12.071,8.988,-2.283,10.436,0.872,-2.937,-7.949,0.314,-0.084,6.671,-1.159,-8.868,4.7,-9.713,-5.9,6.494,-10.541,-15.412,-12.482,2.13,21.56,12.623,25.297,26.16,16.927,28.164,22.265,24.798,25.427,9.776,24.716,11.849,30.948,32.726,19.035,31.064,28.021,23.926,25.053,27.213,31.459,23.614,18.728,18.307,14.007,17.547,20.017,14.926,14.798,26.033,10.588,25.138,23.278,21.822,15.345,18.273,11.242,9.771,26.3,3.502,28.823,34.246,21.601,8.954,24.253,16.242,10.452,14.64,9.005,33.897,27,15.709,9.703,12.306,31.186,27.485,8.659,28.933,8.587,13.113,17.886,33.316,32.183,8.835,10.192,10.896,18.402,28.148,22.599,15.215,32.835,12.216,16.863,18.039,34.714,13.862,29.318,16.664,21.295,34.601,19.248,32.986],"y":[6.376,7.185,4.893,6.733,8.125,8.943,8.396,8.231,8.081,10.251,11.142,11.627,12.144,12.337,11.968,13.221,11.127,11.435,11.863,13.332,13.905,11.063,11.369,17.848,18.848,18.591,19.522,18.806,17.317,16.926,17.33,17.869,15.424,17.088,17.444,18.966,19.501,16.798,17.291,15.282,21.227,21.721,21.466,22.355,21.079,20.263,19.935,20.835,21.422,18.463,18.039,18.157,16.616,20.97,21.799,23.254,23.963,21.697,22.32,23.446,21.891,23.749,22.813,20.819,22.698,20.703,21.638,23.691,25.048,25.282,26.342,25.24,26.527,23.179,22.886,23.54,23.561,21.355,20.789,20.659,19.306,24.083,24.707,25.682,25.734,25.496,24.783,23.859,22.997,23.977,24.817,24.083,23.232,23.045,21.926,23.687,25.016,23.21,22.335,20.897,19.955,22.851,20.73,19.399,18.686,17.52,19.471,20.009,18.087,20.207,19.408,18.829,18.422,17.343,19.292,19.035,17.792,16.864,20.252,20.049,20.693,19.184,15.379,14.199,13.551,12.338,14.558,14.358,13.847,13.399,12.441,14.883,15.131,16.37,13.944,14.078,13.709,12.322,11.505,14.745,15.99,10.023,9.103,8.559,7.342,9.759,8.807,9.614,10.424,9.445,8.995,8.262,7.488,10.138,11.25,10.678,12.296,8.499,7.83,6.456,5.436,8.658,7.834,9.915,6.441,5.209,4.289,3.067,5.526,4.248,6.477,4.873,4.098,4.571,5.215,4.318,4.113,5.125,2.909,4.943,2.715,3.737,3.554,4.177,4.563,4.255,4.892,3.827,2.617,3.188,3.329,2.971,4.121,4.212,1.703,1.85,1.432,1.04,1.495,5.021,6.151,7.15,7.942,6.904,7.423,5.972,6.74,6.556,7.624,7.979,5.128,4.916,4.062,5.192,8.175,9.204,10.56,11.476,9.325,8.115,7.876,7.188,6.743,6.046,5.834,4.715,10.676,11.922,11.754,12.577,12.416,12.502,13.413,11.637,13.459,11.676,12.591,10.714,10.426,11.556,11.643,9.116,9.154,14.725,15.837,15.727,14.646,15.739,16.077,16.504,16.056,16.756,16.49,15.719,16.597,15.825,16.262,16.84,16.769,15.795,15.138,18.216,18.794,18.219,13.568,13.266,13.076,11.965,11.968,11.849,11.95,14.175,14.142,14.874,15.85,14.761,16.076,13.907,14.401,14.989,16.478,16.915,14.198,12.881,13.247,17.246,18.678,19.509,20.655,18.951,19.671,20.647,20.518,18.692,18.055,21.634,22.624,21.961,22.406,23.379,23.395,21.969,20.894,20.163,19.554,19.644,19.059,19.545,19.814,19.653,18.934,18.311,19.365,19.146,17.481,16.183,17.162,15.192,22.305,22.792,21.762,22.103,23.061,19.634,19.588,20.59,20.302,19.204,18.193,18.79,18.54,17.038,16.074,16.302,19.593,20.198,21.2,21.465,20.832,19.823,18.313,18.949,21.738,22.659,21.837,22.291,20.61,19.715,19.325,19.273,19.055,18.69,19.861,19.656,18.289,17.049,18.04,15.792,21.081,22.273,22.374,22.658,23.53,22.126,22.15,21.071,21.314,21.909,21.793,23.075,21.491,19.89,18.785,19.14,18.746,17.533,19.871,20.27,21.207,21.072,20.939,21.427,19.945,22.133,22.142,23.082,23.072,22.372,22.349,22.72,22.665,24.17,24.226,24.807,23.764,19.167,17.939,17.303,16.087,18.147,16.802,19.037,16.917,15.271,15.094,14.714,15.201,14.066,14.484,13.33,15.693,15.481,16.672,17.661,18.16,17.362,16.449,18.6,17.89,18.827,18.553,19.491,17.28,16.913,16.433,15.849,15.86,16.244,15.129,17.561,16.715,16.32,16.599,15.696,17.029,16.412,17.404,16.876,17.313,16.567,15.112,14.526,16.631,17.853,19.133,17.766,14.546,13.165,12.989,12.066,12.707,11.305,14.81,14.984,13.684,13.183,16.068,17.335,17.114,16.777,17.299,10.694,9.568,9.46,8.429,9.604,10.534,10.525,10.973,10.437,11.313,10.7,9.953,10.862,9.384,10.298,9.562,9.013,11.694,11.099,9.587,8.914,11.763,13.295,13.855,13.838,7.724,7.342,6.198,6.345,8.524,8.075,9.686,4.029,4.366,3.878,4.43,3.678,3.805,3.428,2.846,2.303,3.391,3.376,1.153,1.585,4.352,5.436,6.458,7.114,6.104,5.125,4.676,4.75,6.581,7.512,7.014,7.805,7.609,8.249,9.306,7.787,9.89,8.362,9.415,5.223,5.46,6.909,7.164,5.121,3.601,5.729,3.173,7.852,9.27,9.636,10.39,10.141,11.66,12.012,12.339,9.084,9.356,8.935,9.664,8.652,9.288,10.426,8.561,8.735,9.679,11.001,11.605,9.919,8.489,11.444,12.685,12.541,13.447,13.088,13.52,14.211,13.832,11.398,11.162,11.19,11.78,9.824,9.683,8.339,10.817,10.559,10.518,11.939,12.292,9.675,9.861,8.204,12.751,14.132,14.935,15.596,14.848,16.339,14.197,14.862,15.589,15.169,16.004,15.42,16.059,16.064,15.81,13.886,13.381,13.96,14.354,11.85,11.32,14.06,14.603,16.097,16.574,14.409,15.163,12.939,16.826,18.271,18.756,19.794,18.919,18.706,19.565,19.656,18.023,18.411,17.252,16.86,18.944,20.097,21.399,19.88,22.475,20.943,22.245,16.733,15.612,15.844,14.953,15.423,16.779,17.211,15.989,14.731,13.606,12.622,14.322,13.347,13.671,12.092,12.757,11.17,11.501,13.766,12.801,11.594,11.602,13.428,13.774,14.68,10.526,9.355,9.714,9.214,8.388,8.685,10.2,12.682,13.447,12.481,12.624,14.472,9.065,8.255,9.061,8.6,7.54,8.538,6.628,10.289,11.135,11.532,11.475,12.383,12.04,11.93,12.327,11.137,11.281,12.912,13.554,14.752,13.969,10.605,11.717,11.48,11.616,13.018,14.186,14.885,14.617,16.002,15.734,16.429,11.131,10.883,9.695,9.694,10.702,11.967,11.663,13.067,8.689,7.54,8.043,7.641,6.449,6.013,8.945,9.526,10.342,10.337,11.054,11.859,10.943,11.184,12.733,13.367,13.815,9.868,8.919,8.306,8.111,7.824,8.011,7.426,8.446,8.101,6.897,6.1,4.883,5.681,9.702,10.756,10.992,11.246,12.036,10.897,11.081,9.924,10.085,11.176,12.402,12.277,13.659,8.764,7.587,7.539,6.68,6.306,7.078,6.02,4.884,4.529,5.422,6.238,7.633,5.613,8.382,6.349,7.732,8.453,-10.988,-9.586,-9.179,-9.616,-8.648,-7.207,-9.004,-6.561,-5.313,-4.521,-5.093,-5.592,-6.331,-7.627,-5.814,-7.949,-6.855,-4.568,-6.694,-4.406,-5.465,-3.207,-2.353,-1.934,-2.107,-1.093,-0.244,-1.484,1.043,2.184,1.693,2.229,3.061,4.156,5.317,4.009,6.317,5.003,6.158,0.158,1.237,2.019,2.24,2.17,2.397,3.157,2.427,3.03,3.497,4.289,4.269,1.128,0.341,0.312,0.527,-1.108,-1.08,-1.952,-0.3,3.879,4.125,3.377,3.995,3.735,4.749,3.634,4.371,2.056,1.254,1.542,1.487,-0.235,-0.621,-0.915,-0.613,-1.188,-0.884,-1.17,1.879,1.876,2.186,2.193,3.503,3.505,3.714,3.725,2.247,2.298,3.219,1.062,4.63,4.55,4.121,3.156,3.584,3.93,2.81,5.265,2.294,1.274,1.438,1.218,-0.132,-0.56,0.383,-1.904,0.002,-2.297,-1.336,-1.703,3.997,5.225,4.992,5.142,6.437,7.658,6.755,4.567,4.297,3.093,3.13,4.076,3.727,5.331,2.036,0.844,1.172,0.878,1.743,2.11,3.059,2.867,2.8,2.008,1.876,4.055,5.023,4.341,4.657,6.077,3.384,2.646,1.793,1.883,1.755,2.59,0.917,0.971,0.117,0.956,0.683,-0.892,-1.92,-2.699,-2.865,1.987,2.866,3.525,3.606,3.922,2.378,1.318,0.971,0.78,0.063,-1.283,0.915,0.598,1.686,1.409,0.354,-0.885,-0.954,-2.152,2.919,4.045,4.16,4.255,5.391,6.581,5.404,4.125,4.247,3.026,3.159,4.574,4.73,5.848,1.838,0.595,-0.165,-1.207,-0.224,0.539,0.91,0.893,1.614,1.597,1.954,2.652,0.312,-0.235,-1.737,-2.322,0.525,1.804,1.354,-2.366,-3.807,-4.596,-5.76,-4.282,-3.835,-4.502,-2.641,-3.802,-2.656,-1.561,-1.632,-0.542,-0.587,-3.948,-4.605,-4.821,-5.584,-3.825,-2.477,-3.812,-4.182,-4.32,-5.716,-6.144,-3.266,-1.827,-0.814,-1.158,0.438,-8.656,-9.581,-9.109,-9.86,-9.896,-9.976,-8.856,-11.155,-8.911,-11.223,-10.101,-10.179,-7.898,-7.35,-7.104,-6.15,-6.076,-6.265,-7.01,-5.727,-7.22,-5.931,-6.679,-8.005,-7.871,-7.853,-7.245,-6.378,-5.125,-4.117,-3.11,-4.522,-5.228,-6.52,-4.594,-7.168,-5.235,-6.523,-4.387,-3.5,-2.415,-2.68,-1.182,-0.037,0.427,0.991,1.129,2.27,2.401,3.113,0.179,0.579,1.29,1.366,-0.655,-1.625,-2.761,-0.888,1.863,2.516,3.876,4.379,2.569,1.253,1.214,0.04,0.003,-1.176,-1.186,-2.386,4.471,5.766,6.032,5.387,6.893,7.01,7.53,6.517,6.977,7.337,7.619,7.113,8.526,8.398,8.752,7.527,7.376,9.73,6.645,5.428,4.526,3.949,4.68,4.41,3.587,4.114,3.341,3.536,2.805,2.838,2.783,5.435,6.07,5.878,5.793,7.552,5.822,5.644,4.175,3.874,6.296,7.792,8.343,7.659,9.586,3.263,1.834,1.244,1.12,1.07,-0.337,-1.183,-0.598,0.812,0.239,-1.073,-1.36,0.049,-1.877,-3.147,-2.926,-3.67,-4.046,-4.506,-5.685,-7.215,-1.887,-1.55,-1.126,-1.586,-0.398,0.119,-0.879,-0.269,0.198,-0.972,-1.087,1.185,2.328,-2.932,-2.186,-2.308,-2.437,-2.26,-2.391,-3.802,-3.968,-2.09,-0.81,-2.05,-4.806,-6.196,-6.333,-6.937,-7.153,-7.083,-8.597,-5.743,-5.791,-5.141,-5.739,-5.13,-6.055,-4.803,-5.442,-3.94,-3.236,-4.034,-4.16,-1.823,-0.799,-0.742,0.569,-4.603,-5.408,-6.634,-7.111,-5.84,-4.822,-4.232,-4.617,-7.108,-8.268,-8.048,-9.01,-6.793,-6.457,-6.513,-6.561,-5.068,-4.918,-3.464,-5.801,-9.174,-8.938,-7.965,-8.145,-8.377,-6.94,-5.942,-6.57,-6.213,-4.815,-3.815,-4.112,-9.697,-10.517,-10.323,-11.105,-12,-12.845,-12.355,-14.114,-9.282,-8.948,-9.971,-9.909,-7.559,-6.448,-4.845,-4.43,-13.177,-12.768,-13.819,-14.026,-11.39,-10.331,-10.981,-8.965,-14.524,-15.548,-16.648,-16.902,-16.162,-15.15,-17.302,-17.269,-18.334,-17.835,-18.505,-18.953,-19.743,-20.206,-19.899,-16.653,-16.069,-15.848,-15.978,-14.745,-14.992,-14.17,-15.517,-15.294,-16.462,-16.348,-14.041,-12.759,-12.437,-11.868,-11.259,-10.687,-10.389,-9.23,-17.587,-18.764,-19.197,-19.356,-19.92,-17.467,-16.379,-16.321,-15.992,-15.054,-13.914,-14.399,-12.786,-16.67,-16.672,-17.757,-17.534,-16.914,-16.997,-18.928,-20.05,-19.675,-19.965,-21.32,-22.426,-21.795,-19,-18.569,-17.643,-17.863,-17.847,-17.453,-18.649,-16.306,-17.285,-18.001,-18.973,-18.785,-18.78,-19.538,-17.342,-16.038,-15.772,-14.932,-14.936,-15.145,-14.15,-15.043,-14.251,-12.866,-12.779,-13.512,-12.155,-10.744,-12.091,-11.688,-11.149,-9.775,-9.083,-12.26,-12.873,-12.95,-7.291,-6.359,-6.035,-4.965,-6.906,-7.08,-7.977,-6.213,-6.946,-6.713,-5.54,-4.675,-7.963,-9.17,-9.038,-10.446,-10.163,-11.578,-11.435,-3.131,-1.935,-1.418,-0.209,-2.22,-2.714,-2.966,-1.702,-2.338,-1.971,-1.35,-0.314,-3.207,-3.096,-2.746,-2.091,-1.96,-1.463,-0.036,0.802,-2.398,2.187,3.062,3.306,4.424,2.481,2.512,2.275,2.399,3.424,4.284,1.05,1.021,1.536,-0.393,3.334,4.271,5.705,6.627,3.924,2.598,4.916,5.881,7.194,7.697,8.862,7.149,6.844,8.459,7.843,6.813,7.173,7.521,8.5,6.033,6.335,5.86,6.721,6.941,8.265,9.036,5.77,6.128,4.526,8.539,9.788,10.988,11.966,9.811,10.904,11.973,12.133,13.244,11.672,11.376,11.913,13.146,13.745,10.795,9.666,8.461,9.82,7.426,8.789,7.591,13.518,14.696,14.777,13.838,15.892,16.133,15.052,14.848,17.483,18.533,14.38,13.327,12.077,11.25,12.969,11.938,10.784,11.088,11.346,10.498,9.138,7.989,9.008,6.728,7.748,6.604,11.079,11.367,10.241,9.172,11.577,10.351,12.663,10.477,9.463,8.23,7.104,7.218,6.544,5.791,4.645,7.55,6.876,8.118,6.431,5.837,4.569,3.552,6.821,6.306,7.223,8.378,6.703,4.633,3.488,2.306,1.186,3.854,2.562,1.506,0.962,-0.255,2.017,1.856,1.441,0.585,-0.467,2.655,2.287,1.815,2.364,1.419,1.971,1.498,1.033,0.3,-1.038,-2.027,1.152,2.343,3.104,4.083,2.669,-2.873,-3.721,-4.207,-5.401,-2.953,-3.682,-4.752,-3.292,-5.423,-3.956,-5.023,-3.284,-3.656,-4.608,-5.501,-2.417,-1.626,-2.625,-3.252,-4.44,-5.314,-6.751,-7.699,-4.812,-6.916,-8.249,-8.872,-10.062,-8.221,-9.614,-7.713,-8.069,-8.546,-8.99,-10.101,-7.44,-7.863,-7.156,-8.137,-8.405,-9.677,-10.519,-7.194,-7.557,-6.023,-9.832,-11.014,-12.268,-13.266,-10.866,-11.944,-12.19,-12.194,-13.3,-13.314,-13.711,-13.716,-14.903,-14.904,-12.927,-12.97,-12.621,-14.074,-12.727,-12.991,-13.69,-14.406,-11.694,-11.198,-9.81,-12.179,-13.482,-14.112,-15.327,-15.981,-18.416,-19.727,-19.935,-20.889,-20.101,-19.835,-20.456,-21.582,-19.725,-19.054,-19.196,-20.241,-20.074,-17.86,-16.939,-16.37,-16.652,-15.541,-15.824,-15.274,-14.456,4.549,3.414,4.842,5.572,5.398,4.567,4.517,6.752,7.437,3.916,3.088,3.915,4.637,1.955,1.037,-0.039,0.405,3.834,4.564,3.735,2.595,4.783,3.542,-1.548,-2.41,-3.696,-4.788,-1.68,-3.559,-4.704,-5.685,-6.898,-4.235,-5.162,-6.01,-6.647,-7.833,-5.225,-6.107,-4.733,-7.27,-8.231,-9.505,-10.608,-7.622,-9.354,-10.511,-11.317,-12.546,-10.09,-11.248,-12.173,-10.711,-10.629,-11.308,-12.013,-13.06,-10.341,-11.051,-11.041,-11.915,-11.85,-12.399,-12.332,-13.281,-13.208,-13.673,-11.439,-12.043,-13.431,-14.396,-11.157,-11.832,-15.31,-16.196,-16.386,-17.288,-15.784,-14.549,-15.465,-14.151,-15.559,-15.641,-17.014,-17.366,-19.693,-20.481,-19.862,-20.293,-20.76,-21.373,-18.917,-18.26,-19.191,-18.946,-17.193,-17.752,-22.278,-22.879,-21.832,-22.117,-23.515,-20.617,-19.542,-19.247,-19.135,-18.249,-18.511,-17.091,-17.36,-19.185,-18.906,-19.979,-19.68,-21.233,-22.356,-22.332,-22.385,-23.681,-24.902,-25.939,-24.79,-19.813,-18.606,-18.706,-18.475,-17.371,-19.06,-19.183,-20.321,-20.209,-19.359,-18.103,-18.487,-17.016,-21.408,-22.546,-22.124,-22.538,-23.715,-24.175,-19.01,-18.053,-18.717,-18.369,-17.197,-16.127,-15.239,-16.447,-19.702,-20.369,-21.233,-21.461,-21.183,-22.277,-20.244,-23.275,-21.697,-22.519,-21.648,-21.829,-23.439,-24.135,-24.481,-20.708,-19.813,-18.779,-18.388,-19.092,-18.291,-20.105,-18.336,-17.347,-17.926,-17.422,-16.037,-15.529,-16.004,-14.578,-15.545,-14.113,-14.6,-14.136,-18.943,-19.595,-18.68,-19.05,-20.587,-19.945,-19.439,-17.467,-16.522,-16.09,-15.732,-15.29,-14.361,-14.583,-13.646,-15.814,-16.15,-15.757,-16.719,-16.393,-15.669,-16.943,-14.614,-17.481,-17.015,-16.513,-16.603,-15.929,-14.624,-16.455,-13.601,-16.024,-15.516,-16.627,-16.369,-14.614,-13.313,-12.339,-13.068,-11.153,-11.883,-10.936,-9.771,-17.865,-19.015,-19.973,-21.162,-19.701,-18.763,-18.426,-18.174,-17.509,-17.256,-16.923,-19.426,-20.205,-20.822,-21.655,-19.333,-18.148,-22.671,-23.936,-25.008,-24.68,-23.844,-22.83,-22.145,-22.42,-21.336,-21.486,-22.756,-20.881,-22.153,-21.227,-26.3,-27.399,-27.31,-27.581,-28.634,-28.206,-26.811,-25.483,-24.699,-25.414,-25.297,-23.257,-22.36,-22.692,-26.179,-26.947,-26.633,-26.335,-28.463,-28.818,-28.838,-26.707,-26.432,-27.343,-28.532,-26.721,-26.517,-27.117,-26.77,-27.532,-28.176,-29.025,-27.774,-28.323,-27.729,-26.687,-27.992,-26.59,-28.398,-27.897,-26.451,-25.636,-28.874,-30.168,-29.761,-26.133,-24.782,-23.758,-22.671,-24.741,-25.141,-25.021,-25.611,-24.1,-23.217,-23.008,-21.901,-23.773,-23.606,-23.074,-22.153,-23.185,-22.347,-20.891,-19.989,-22.822,-19.578,-19.142,-18.58,-17.596,-15.006,-14.003,-13.69,-12.521,-14.443,-13.459,-14.54,-14.724,-14.508,-13.863,-13.191,-15.799,-16.303,-15.027,-14.715,-14.049,-13.418,-11.931,-11.086,-11.612,-10.227,-9.57,-8.394,-10.33,-9.837,-9.619,-8.615,-10.828,-10.928,-10.397,-9.619,-10.553,-10.417,-9.139,-8.389,-11.632,-8.875,-7.669,-6.449,-5.474,-7.666,-6.426,-6.234,-6.546,-6.529,-5.444,-5.119,-3.952,-5.813,-6.151,-5.944,-5.196,-4.291,-7.279,-7.024,-7.971,-5.572,-5.563,-4.935,-4.902,-3.494,-3.472,-2.609,-2.572,-5.734,-5.695,-7.107,-5.892,-1.625,-1.091,0.437,1.009,-1.814,-1.593,-1.368,-2.53,1.594,0.511,0.824,-0.042,0.107,-0.556,-0.719,-1.9,2.746,2.563,1.107,0.522,3.47,4.831,3.356,0.517,-0.854,-1.894,-2.864,-1.68,-2.637,-2.186,-2.706,-3.001,-3.468,-3.712,-4.726,-1.244,-0.726,-1.81,-1.827,0.198,1.573,2.467,2.254,-5.035,-5.855,-5.122,-5.735,-6.241,-7.605,-7.957,-7.613,-3.813,-3.017,-3.021,-3.289,-1.58,-0.849,-5.16,-6.523,-6.747,-7.112,-7.502,-8.383,-9.663,-9.643,-10.787,-4.54,-3.521,-3.721,-3.654,-2.148,-3.989,-4.138,-5.545,-5.807,-3.276,-1.818,-1.342,-0.918,-0.008,0.425,0.872,2.196,-8.671,-9.523,-8.925,-9.644,-9.883,-10.527,-10.672,-11.873,-7.72,-8.268,-7.195,-6.225,-8.899,-9.37,-10.062,-5.07,-4.807,-5.455,-5.671,-3.283,-2.885,-3.814,-5.764,-6.406,-7.716,-8.115,-6.694,-7.427,-8.363,-9.626,-9.498,-10.436,-10.341,-10.656,-11.518,-9.918,-8.339,-8.11,-8.108,-8.619,-6.769,-6.709,-7.825,-5.527,-7.767,-5.457,-6.578,-9.735,-11.122,-11.889,-12.51,-11.81,-11.24,-13.323,-11.717,-11.811,-12.506,-12.047,-12.87,-12.36,-13.164,-14.623,-13.06,-10.744,-10.213,-10.829,-11.181,-8.688,-8.013,-8.605,-6.772,-13.749,-15.133,-15.25,-16.206,-15.761,-16.024,-14.275,-14.279,-14.126,-14.853,-13.167,-13.518,-12.998,-12.43,-13.213,-13,-14.271,-14.64,-11.84,-10.46,-9.41,-10.12,-14.942,-16.179,-17.263,-17.901,-16.682,-18.096,-15.747,-17.443,-18.441,-18.166,-19.053,-18.468,-19.399,-18.918,-16.928,-16.543,-16.785,-17.213,-15.074,-14.957,-14.624,-13.546,-16.556,-16.781,-18.284,-18.669,-16.188,-16.252,-19.134,-20.579,-21.137,-22.015,-21.383,-21.246,-20.925,-20.622,-21.081,-20.451,-21.101,-20.885,-21.672,-21.383,-22.739,-19.183,-18.47,-17.717,-16.499,-17.492,-18.146,-18.768,-18.149,-19.39,-18.768,-19.389,-18.443,-17.823,-16.941,-15.866,-19.032,-20.135,-19.912,-13.711,-12.865,-12.238,-12.592,-13.657,-12.793,-11.837,-12.923,-11.023,-12.114,-11.163,-11.309,-10.605,-9.523,-9.285,-9.898,-8.95,-10.905,-8.855,-7.804,-6.679,-6.122,-7.312,-8.561,-9.163,-6.972,-7.465,-7.463,-7.137,-8.876,-5.436,-4.104,-3.594,-3.109,-3.078,-2.837,-1.774,-3.77,-3.327,-4.093,-3.493,-3.421,-2.505,-5.415,-6.21,-5.882,-5.897,-7.709,-8.589,-8.27,-10.052,-2.718,-3.037,-3.66,-3.195,-3.971,-4.594,-3.844,-5.941,-4.426,-6.532,-5.776,-4.704,-5.38,-4.442,-4.556,-6.647,-7.773,-8.976,-8.183,-3.507,-2.541,-1.642,-1.295,-1.697,-2.49,-1.283,-0.451,-1.181,-0.581,-2.487,-3.307,-3.35,-3.152,-4.739,-5.635,-4.692,-3.557,-3.608,-2.286,-2.271,-3.915,-1.177,0.136,0.362,0.933,1.239,2.653,2.683,3.61,-0.111,0.031,-0.799,-0.401,-0.383,-1.957,-2.815,-2.17,-2.374,-4.204,-5.125,-6.4,-5.445,-1.379,-0.691,0.603,1.198,-0.392,0.994,0.569,1.559,1.228,0.026,-1.427,-1.88,-2.352,-3.221,-3.692,-4.124,-5.455,-19.011,-19.767,-21.234,-21.555,-19.277,-20.084,-17.797,-24.551,-25.159,-26.63,-27.393,-25.039,-23.645,-23.098,-22.623,-21.798,-21.479,-22.561,-20.284,-21.369,-20.248,-27.026,-28.422,-29.035,-28.309,-28.576,-28.062,-27.841,-30.585,-30.263,-28.903,-28.743,-30.233,-29.769,-30.632,-28.459,-30.197,-28.013,-28.882,-29.17,-29.902,-29.059,-28.974,-31.188,-28.426,-27.584,-26.435,-26.188,-27.038,-28.14,-26.145,-25.757,-24.65,-25.146,-24.541,-23.901,-23.3,-22.805,-22.422,-26.168,-24.988,-24.367,-24.293,-23.92,-24.378,-22.549,-23.501,-23.979,-23.341,-24.154,-23.6,-22.744,-21.486,-20.25,-21.541,-19.087,-20.386,-19.157,-25.463,-25.462,-26.323,-26.327,-26.438,-26.44,-26.751,-26.774,-27.708,-27.716,-28.301,-27.647,-23.817,-22.508,-22.2,-22.458,-21.404,-21.349,-20.275,-21.062,-19.554,-18.203,-17.085,-15.973,-18.219,-18.925,-19.818,-18.708,-20.476,-19.363,-20.245,-20.888,-16.742,-16.551,-16.008,-14.961,-17.875,-17.686,-18.351,-16.7,-16.309,-14.996,-14.163,-17.445,-16.999,-18.692,-14.815,-13.602,-12.377,-11.464,-12.31,-11.187,-10.953,-9.827,-11.647,-12.385,-13.205,-12.027,-11.929,-11.297,-10.432,-13.306,-11.728,-11.227,-9.727,-8.968,-12.007,-13.378,-11.412,-9.296,-7.884,-7.032,-5.968,-7.657,-6.213,-5.743,-6.108,-7.512,-6.788,-6.624,-5.538,-7.512,-6.598,-5.662,-4.209,-3.333,-5.98,-5.053,-3.963,-2.614,-2.109,-0.901,-2.55,-2.825,-2.808,-1.807,-3.032,-2.667,-2.525,-1.522,-3.698,-3.366,-3.712,-3.514,-3.498,-2.483,-1.911,-4.918,-4.902,-5.892,-2.246,-1.301,-0.392,-0.434,-2.059,-3.117,-2.769,-4.466,-3.739,-5.443,-5.071,-6.033,0.495,1.453,2.244,2.608,2.377,1.486,0.773,2.537,3.289,2.561,3.193,3.651,2.507,1.68,2.092,0.782,1.011,2.532,0.363,1.888,0.815,1.249,0.463,0.84,0.501,-1.072,-1.441,-1.465,1.576,2.023,3.096,3.403,2.581,1.562,2.177,3.251,1.503,3.107,2.693,2.216,1.728,1.589,1.812,1.181,2.704,1.439,2.968,2.334,2.603,2.382,1.919,3.005,2.743,0.717,-0.403,-1.302,-0.534,-2.313,-1.543,-2.433,4.205,5.32,5.121,5.743,2.115,0.841,0.988,-0.003,-0.055,-0.748,-0.024,-2.13,-0.665,-2.786,-2.052,2.224,2.451,2.676,3.125,2.345,2.522,1.675,0.442,2.235,2.605,3.079,2.397,2.356,1.711,2.445,2.346,1.781,1.185,1.509,-0.314,3.152,3.915,3.102,3.64,4.956,4.544,4.754,4.014,4.45,3.708,3.93,3.636,1.814,0.978,-0.496,-0.893,1.173,0.465,-0.711,1.165,-1.304,-2.743,-3.439,-4.223,-3.38,-3.146,-3.751,-3.374,-4.221,-3.338,-2.109,-1.609,-2.31,-2.703,-0.099,-2.475,-3.136,-4.599,-5.127,-3.035,-1.565,-3.83,-1.316,-5.227,-6.621,-6.805,-7.866,-7.116,-5.774,-5.822,-5.287,-5.32,-5.009,-5.519,-4.584,-3.405,-5.101,-4.79,-4.223,-5.271,-5.926,-3.283,-2.399,-2.886,-1.092,-5.38,-6.338,-6.106,-7.059,-6.345,-4.851,-4.555,-4.998,-5.46,-3.07,-2.633,-0.945,-1.184,-4.85,-5.269,-6.785,-7.306,-4.88,-5.558,-3.375,-7.48,-8.935,-9.409,-10.292,-9.457,-9.105,-9.029,-9.945,-11.273,-11.911,-11.687,-12.932,-12.842,-13.791,-13.241,-13.213,-14.617,-11.684,-11.447,-11.621,-12.29,-10.022,-9.891,-9.73,-11.041,-11.137,-12.591,-13.046,-10.236,-8.761,-10.566,-7.783,-13.328,-14.729,-15.551,-16.386,-15.343,-15.77,-16.658,-16.53,-15.327,-16.045,-15.683,-16.503,-15.707,-16.581,-16.151,-17.822,-14.462,-13.999,-14.763,-14.649,-15.543,-16.333,-17.721,-18.384,-16.485,-15.216,-15.506,-14.686,-18.484,-18.678,-20.107,-20.766,-17.685,-20.584,-21.945,-22.957,-23.931,-22.243,-23.702,-21.294,-22.691,-22.762,-21.908,-20.756,-22.305,-22.589,-23.013,-22.352,-22.498,-21.819,-21.438,-20.681,-22.663,-22.801,-24.036,-23.249,-17.418,-16.392,-14.961,-14.151,-16.531,-17.91,-15.427,-18.223,-14.661,-13.322,-12.923,-11.917,-13.199,-13.336,-11.85,-13.745,-13.465,-13.501,-12.676,-14.458,-14.299,-13.249,-15.348,-14.449,-14.592,-13.333,-13.002,-15.824,-17.006,-15.943,-12.616,-11.398,-10.088,-9.02,-11.375,-12.327,-12.141,-13.415,-13.016,-14.297,-14.091,-14.97,-10.155,-8.944,-7.969,-6.783,-9.299,-7.901,-7.469,-6.405,-5.489,-8.681,-8.705,-8.467,-10.039,-6.509,-5.555,-4.173,-3.152,-6.067,-5.169,-4.149,-2.906,-2.277,-1.056,-3.152,-1.835,-3.843,-3.116,-2.645,-1.929,-0.789,-3.814,-3.438,-2.871,-4.657,-0.692,0.459,1.659,2.736,0.049,1.166,1.487,1.617,2.464,2.929,0.224,-0.734,-2.152,-0.253,1.637,1.019,0.931,0.498,-0.416,-1.134,-0.351,-0.719,-2.14,-2.646,-3.853,-2.312,-0.978,-0.029,-2.394,-2.914,-4.39,-5.103,-2.126,-0.671,0.212,-0.42,-4.849,-6.254,-7.117,-8.173,-6.489,-5.798,-5.626,-5.327,-4.994,-4.694,-4.528,-6.319,-6.225,-7.158,-7.853,-4.778,-3.75,-2.357,-4.096,-7.188,-8.055,-9.512,-10.321,-7.802,-8.386,-9.892,-8.005,-9.828,-11.176,-11.587,-12.721,-11.247,-10.471,-10.683,-12.041,-12.752,-9.568,-7.992,-12.408,-13.698,-14.807,-15.789,-13.842,-15.142,-15.319,-15.135,-14.638,-15.636,-15.793,-16.908,-15.259,-15.165,-16.311,-14.674,-14.697,-15.417,-16.248,-13.255,-12.588,-13.268,-13.345,-15.112,-15.737,-17.241,-18.05,-15.055,-15.821,-13.617,-17.612,-19.014,-19.749,-20.874,-19.146,-20.583,-18.658,-19.087,-19.664,-19.98,-21.043,-18.701,-19.053,-19.238,-20.381,-21.127,-17.949,-23.142,-24.465,-24.823,-25.987,-24.722,-24.688,-23.598,-25.72,-23.532,-25.663,-24.567,-23.838,-24.099,-24.81,-24.424,-25.873,-26.64,-27.263,-27.651,-27.717,-28.567,-27.373,-27.926,-26.907,-27.268,-28.334,-25.639,-24.579,-24.511,-23.572,-23.232,-22.183,-22.29,-21.093,-21.327,-20.123,-20.242,-25.529,-25.629,-24.597,-23.898,-27.028,-27.227,-28.113,-24.528,-23.603,-23.922,-23.017,-25.385,-24.985,-23.461,-22.948,-25.456,-24.863,-26.985,-22.751,-21.288,-20.823,-19.899,-20.748,-19.216,-18.706,-19.45,-17.431,-21.482,-21.132,-21.243,-20.31,-22.013,-22.366,-22.579,-21.549,-20.961,-23.989,-21.315,-20.356,-18.943,-18.265,-20.347,-19.51,-19.972,-18.243,-19.183,-17.447,-17.918,-18.504,-17.168,-17.027,-15.961,-16.801,-16.598,-16.093,-15.95,-15.813,-18.639,-18.543,-17.129,-16.516,-19.56,-19.388,-19.784,-18.828,-19.623,-18.662,-19.061,-16.611,-15.27,-14.198,-13.237,-14.962,-15.753,-15.642,-13.887,-14.375,-13.415,-13.323,-12.234,-13.798,-14.464,-14.484,-13.777,-13.033,-15.924,-15.896,-16.598,-14,-13.354,-11.833,-11.076,-13.861,-13.03,-15.321,-11.403,-9.985,-9.328,-8.227,-9.805,-8.349,-10.252,-10.028,-9.525,-9.345,-8.3,-10.493,-9.978,-10.359,-10.355,-10.333,-10.312,-9.175,-9.143,-8.477,-8.393,-11.658,-11.626,-12.749,-11.628,-8.959,-7.874,-6.484,-5.53,-7.983,-9.033,-9.187,-8.63,-6.369,-5.08,-4.868,-3.761,-4.521,-3.582,-2.119,-1.311,-4.047,-5.517,-5.898,-5.227,-7,-1.775,-0.396,0.4,0.036,-0.333,-0.631,-1.929,0.383,-2.207,0.113,-1.183,-1.464,14.549,16.868,16.877,12.475,12.479,16.544,17.301,18.576,18.566,17.297,19.714,19.768,19.866,21.059,20.963,22.095,14.644,15.646,15.264,14.028,13.654,16.03,13.203,13.552,12.755,12.059,10.861,10.86,12.049,9.818,9.901,9.208,14.647,13.676,14.086,15.309,15.662,13.276,16.125,17.059,17.221,17.661,16.904,-9.834,-11.501,-9.6,-7.536,-9.44,-10.352,-11.145,-11.511,-10.96,-10.242,-11.029,-12.344,-13.826,-14.709,-15.2,-14.914,-8.915,-9.005,-8.28,-7.688,-8.065,-8.27,-6.931,-5.788,-8.733,-7.816,-7.248,-7.847,-8.757,-6.164,-7.688,-6.459,-10.355,-10.153,-10.744,-11.288,-11.093,-10.796,-11.893,-10.869,-11.43,-10.66,-12.635,-11.782,-13.236,-11.073,-10.994,-13.274,-12.145,-12.698,-12.621,-11.998,-11.705,-11.675,-13.079,-11.944,-12.359,-11.464,-13.566,-11.142,-10.798,-10.156,-10.155,-10.775,-9.515,-9.574,-9.9,-12.098,-11.61,-11.888,-12.531,-12.665,-11.545,-12.934,-13.639,-12.961,-13.399,-14.094,-14.157,-13.395,-14.613,-15.029,-16.521,-17.419,-16.951,-18.615,-6.085,-5.444,-9.336,-6.957,-3.093,-7.209,-6.733,-7.782,-8.887,-8.519,-10.23,-7.679,-7.224,-7.233,-8.303,-6.173,-7.761,-8.985,-9.903,-9.23,-7.914,-11.285,-9.639,-10.921,-5.182,-5.706,-4.778,-3.649,-3.933,-5.045,-2.433,-2.269,-4.563,-3.376,-2.506,-3.187,-4.441,-1.093,-2.753,-3.098,-2.641,-3.508,-1.419,4.525,5.469,18.529,28.564,5.303,0.762,2.513,1.861,5.706,7.683,26.428,19.923,19.631,9.514,15.652,20.96,25.754,20.889,-0.471,8.575,8.778,6.424,6.049,10.733,22.551,13.718,5.95,12.63,13.036,26.57,11.8,20.887,22.676,3.61,12.279,17.126,6.872,20.073,11.856,19.642,18.995,15.366,11.413,27.003,18.869,1.47,24.951,1.082,3.384,13.644,1.777,23.07,0.157,9.196,2.515,14.125,8.173,15.723,3.256,26.716,24.719,4.99,-0.17,24.157,9.138,21.749,13.359,4.648,9.843,5.038,24.674,9.278,12.815,24.171,19.797,3.777,-2.116,20.575,15.862,0.834,13.267,22.009,7.506,23.031,3.078,12.568,7.66,5.15,22.246,10.553,14.974,19.747,5.037,-0.266,18.572,15.607,11.051,16.457,24.851,7.392,9.511,-21.793,-9.479,-19.29,-17.136,-19.693,-8.888,0.563,1.239,-11.808,-4.813,-12.362,-3.561,1.251,7.929,-15.206,-8.902,-16.814,-16.724,-3.605,0.742,-5.206,-7.571,1.837,11.614,9.425,-13.975,1.212,-1.985,-14.161,9.001,-7.585,4.909,-10.385,-5.385,-14.327,3.395,6.852,1.972,12.201,-22.693,-10.596,-7.907,-19.759,9.988,-9.055,-10.344,-11.29,7.103,-6.636,3.153,-3.269,-11.008,5.001,2.166,-10.271,-5.196,-24.058,-15.404,-15.709,-5.417,7.123,1.065,-3.398,4.274,-12.382,-9.627,-9.999,-5.886,-21.029,-1.689,5.059,-11.491,-12.899,-1.745,3.799,8.196,-6.492,2.263,3.639,-3.258,-1.007,1.367,-1.07,-19.501,5.723,1.122,-3.557,-1.778,-21.204,-13.338,0.108,-3.595,-23.965,1.243,3.292,-19.731,-7.795,-10.982,-9.383,1.241,-24.106,-10.449,-18.234,2.134,-2.49,-26.294,3.422,-15.983,-13.61,3.429,-19.677,-7.788,4.78,-23.084,-28.01,-8.13,-22.435,2.49,1.821,2.366,-4.653,0.753,3.209,-15.296,-7.114,-9.592,-27.807,-20.429,-24.319,-30.116,5.181,-13.589,1.53,-0.932,-14.349,-20.326,-13.955,-3.389,-22.767,-23.575,-25.072,-10.613,1.148,-21.166,-5.556,-7.905,-3.123,-13.947,-7.441,-16.736,-25.757,-21.383,-14.762,-7.74,0.777,6.8,-9.212,-16.782,3.739,-24.704,-20.541,5.641,-1.818,-20.8,-6.964,-16.744,4.866,-2.052,-10.958,-6.092,2.474,0.083,-26.26,-26.106,-25.077,-16.074,-9.114,-16.802,-5.538,-15.884,-12.212,-23.576,-11.204,-14.616,-18.696,-0.819,-18.41,-31.775,3.958,1.367,-35.195,-23.355,-16.524,-15.984,5.948,-18.5,-30.733,-29.92,-10.726,-12.782,-1.842,-16.609,-26.508,-30.471,-9.318,-18.836,-11.251,-21.927,-17.924,-6.424,-31.438,-29.535,-10.293,-25.857,-10.211,5.875,-23.584,-14.041,-26.184,-16.071,0.392,-5.884,-2.758,-29.519,-7.759,-24.668,-23.258,-11.501,-17.386,-27.612,-25.012,-13.007,-27.917,-11.298,-15.962,-13.417,5.285,-27.953,-25.252,-30.431,3.072,-23.096,-29.799,-27.836,-30.397,-13.553,-27.29,-7.08,-13.869,-5.395,-30.55,-8.116,-12.587,-25.832,-19.969,-7.801,-14.407,-13.553,-27.746,-24.628,-10.858,-9.802,3.412,-2.487,3.681,-13.633,-21.212,-18.97],"z":[3.891,4.515,4.256,2.881,2.452,3.516,4.327,1.109,1.261,3.506,4.475,4.011,2.903,4.748,5.189,5.41,6.451,4.861,4.567,4.947,5.524,5.416,6.795,8.682,9.623,11.006,11.705,9.686,11.38,12.675,12.813,13.844,12.888,11.772,11.801,11.796,12.567,10.62,10.576,10.756,12.828,14.117,15.215,16.013,14.476,15.243,16.245,16.06,17.024,16.149,17.117,18.566,16.822,14.817,14.535,14.917,15.372,13.065,12.806,12.075,13.335,12.128,12.895,14.144,13.238,14.485,14.033,14.736,15.088,16.578,16.968,14.803,15.2,18.711,19.104,18.311,18.775,19.142,17.722,19.89,17.672,17.137,16.292,16.968,16.634,19.709,20.424,19.548,20.074,21.577,22.471,18.235,17.316,17.638,17.565,15.862,15.674,20.547,21.714,21.361,21.816,22.827,20.542,20.123,19.362,19.628,19.24,20.068,18.674,19.277,18.452,17.661,18.496,18.311,19.421,20.307,21.141,21.195,21.211,22.215,23.263,21.894,20.613,19.757,19.357,19.5,18.51,18.86,18.444,19.626,19.512,17.612,16.239,15.659,15.312,20.761,21.958,22.422,22.842,23.07,22.714,20.495,19.459,19.659,19.648,18.084,16.947,15.366,15.371,19.87,20.044,21.36,21.455,19.846,20.874,18.42,20.881,22.364,23.659,23.622,23.907,24.815,26.105,25.052,23.263,23.198,22.045,22.188,23.111,23.199,24.231,20.911,19.747,19.362,18.327,18.581,18.941,19.564,18.662,19.901,18.996,19.614,19.939,20.147,19.908,18.522,18.083,21.016,21.256,21.239,17.806,16.468,15.465,14.488,15.96,15.665,16.816,16.595,18.043,15.73,14.832,14.79,13.854,15.2,16.535,15.093,14.03,12.936,11.851,11.19,12.308,11.563,13.379,10.068,11.711,10.704,11.098,10.276,10.443,9.749,8.404,10.441,7.761,9.805,8.464,7.821,12.354,12.871,13.313,14.056,14.021,13.659,12.704,14.231,12.318,13.852,12.891,12.796,13.125,12.702,13.18,12.464,11.055,13.2,14.139,15.154,15.356,14.897,14.109,12.812,14.587,12.456,13.525,15.812,13.649,15.936,14.858,15.824,16.815,17.938,18.513,17.313,17.038,15.687,19.568,19.377,20.77,21.301,18.552,18.132,17.327,21.373,22.72,22.814,22.097,23.73,23.294,23.835,23.699,23.91,24.24,24.996,25.108,24.962,24.593,23.658,23.907,23.513,23.945,22.689,22.254,21.145,20.522,21.737,20.548,20.878,19.827,18.492,17.789,19.763,21.19,21.621,18.152,16.906,16.879,15.872,16.713,15.999,14.799,16.728,17.99,18.116,18.057,17.439,19.432,19.305,19.768,18.304,16.564,15.186,14.301,13.496,14.677,15.268,15.685,14.892,14.469,11.209,11.112,11.301,10.495,12.122,12.091,11.792,12.346,12.593,11.503,11.262,13.982,15.118,14.879,15.123,10.835,9.74,8.59,7.84,8.46,7.418,7.696,6.784,8.963,9.38,9.127,8.643,10.878,11.116,11.347,10.438,9.451,9.232,7.746,7.371,9.697,6.906,5.457,5.051,4.229,4.747,3.215,2.592,2.688,5.652,5.355,5.699,4.99,6.107,6.792,7.182,6.124,5.749,8.576,8.903,9.634,5.637,5.631,4.613,4.609,3.284,3.284,2.552,2.552,4.419,4.415,5.648,3.94,3.025,3.392,2.297,2.3,4.7,5.304,4.449,6.722,-0.251,0.423,-0.502,-0.333,1.55,2.694,3.655,3.417,-3.707,-4.401,-3.398,-3.579,-5.175,-6.131,-5.9,-2.312,-1.295,-0.668,-0.347,-0.514,0.104,-0.872,-0.45,1.191,2.266,3.287,2.937,-2.161,-3.209,-2.873,-2.918,-4.527,-5.333,-6.811,-7.922,-0.018,1.158,0.783,1.242,2.268,3.194,2.389,4.126,-0.077,-0.506,-1.245,-0.95,-1.38,-1.592,-1.02,-0.029,0.73,0.763,0.979,0.923,1.336,2.485,0.393,0.006,-0.886,-1.42,-1.263,-2.048,-2.024,-2.623,-1.767,-1.9,-3.931,-4.983,-6.032,-4.929,-7.002,-5.891,-6.925,-7.89,2.002,3.331,3.366,4.202,4.168,4.115,4.806,4.723,4.326,5.621,5.445,4.767,6.303,7.632,6.518,5.746,6.106,7.487,8.086,5.01,3.835,4.401,7.991,9.309,10.38,11.389,9.633,9.588,10.139,11.086,11.102,12.118,10.824,10.924,12.012,9.784,9.989,9.92,10.878,11.587,8.491,7.509,7.895,6.197,6.99,5.284,5.68,13.565,14.906,15.326,16.367,14.974,14.919,16.226,14.72,14.49,14.791,14.95,15.854,13.716,13.941,15.359,12.925,14.092,14.17,15.527,16.138,13.04,11.69,11.606,10.625,18.28,19.255,19.243,20.297,19.004,19.384,18.058,17.928,18.63,19.345,16.437,15.839,16.301,14.339,18.451,19.097,20.617,21.32,18.657,17.17,16.918,16.742,21.111,22.541,23.097,24.107,22.831,24.271,22.57,22.423,22.841,22.909,23.908,21.885,22.235,21.968,21.853,21.809,22.953,23.539,20.433,19.352,20.438,17.936,23.297,24.384,25.724,26.567,24.405,25.258,25.905,27.151,27.299,28.4,27.276,26.259,27.162,26.189,26.193,26.077,26.637,25.081,25.193,23.879,24.621,25.332,25.115,25.348,24.445,23.688,23.37,23.673,22.783,23.399,22.504,22.812,26.588,26.926,26.608,26.083,28.429,28.894,27.784,22.63,21.96,22.301,22.958,22.266,21.285,19.935,21.701,19.012,20.787,19.441,21.827,22.048,21.134,20.281,21.669,20.276,22.499,21.322,20.454,18.983,18.087,21.001,22.466,22.481,17.537,17.066,16.785,15.797,18.117,16,14.792,13.527,12.656,14.708,14.68,13.484,13.459,12.292,12.138,11.036,12.342,12.188,13.236,13.176,12.795,12.044,14.509,14.452,13.506,15.84,9.2,8.572,8.542,7.494,9.319,8.976,7.784,9.874,7.493,9.595,8.405,9.691,9.786,8.924,8.388,11.246,12.109,13.559,11.553,8.791,7.946,6.5,5.712,8.098,9.445,6.173,4.844,4.581,3.469,5.603,5.485,5.249,4.349,6.744,6.72,6.818,6.028,5.886,4.482,3.89,6.943,3.945,2.611,1.538,0.541,2.364,1.071,1.065,0.94,1.742,0.792,0.794,-0.251,1.142,1.967,2.075,1.398,0.925,3.539,4.337,5.786,3.703,1.336,0.704,-0.808,-1.495,1.356,-3.108,-2.662,-3.612,-3.757,-1.345,-0.121,-0.164,1.075,0.953,2.195,2.13,3.236,29.801,29.611,30.898,31.973,29.357,29.328,28.045,33.436,33.904,34.791,35.649,34.71,33.971,34.161,32.936,33.313,32.547,32.3,31.549,31.307,30.943,34.595,35.426,36.607,36.565,34.666,34.376,33.36,37.564,37.113,36.422,36.648,36.144,35.506,36.212,34.211,35.641,33.63,34.346,39.033,39.825,39.037,39.518,40.327,37.81,36.941,36.689,36.776,35.595,35.85,34.68,36.407,36.16,37.402,37.306,35.709,34.377,35.588,33.269,37.916,36.968,37.344,37.562,35.517,34.972,34.602,33.628,37.461,37.777,39.118,39.225,37.555,36.1,35.437,35.372,34.066,34,33.347,40.128,40.127,41.445,41.448,41.417,41.42,42.219,42.24,42.488,42.469,42.148,42.617,38.617,37.673,38.329,39.088,36.53,35.56,34.538,34.874,37.725,36.789,36.201,35.003,37.382,37.533,37.558,37.647,37.692,37.783,37.803,37.912,35.432,34.648,33.437,32.287,35.466,34.571,36.598,33.713,32.686,31.821,30.601,33.33,32.275,34.081,32.453,31.724,30.666,29.486,31.059,30.122,29.035,27.855,31.028,32.286,32.451,29.433,28.487,27.422,26.231,29.232,27.855,26.965,25.938,24.731,27.782,28.604,26.864,26.404,25.496,24.553,23.353,26.267,25.41,24.527,26.298,25.088,24.276,23.169,22.019,25.144,21.873,20.941,19.975,18.78,21.714,20.684,20.493,19.677,18.643,17.594,20.562,21.466,22.274,20.642,18.941,18.024,17.016,15.811,18.794,17.825,19.582,17.51,16.663,15.776,14.661,17.513,16.624,18.308,16.277,15.529,15.604,16.259,16.169,16.312,15.188,17.571,15.309,17.706,16.571,16.689,14.868,14.794,14.61,15.07,13.615,13.627,13.842,13.937,13.724,15.034,15.069,12.765,13.116,13.883,12.666,13.933,13.195,11.864,12.946,11.616,12.158,16.118,17.418,17.968,18.914,18.428,18.558,17.952,17.339,17.741,17.391,17.901,17.035,17.397,16.6,15.963,16.624,18.32,19.413,20.425,21.332,20.144,19.215,18.976,18.534,18.072,17.634,17.406,16.513,20.25,21.215,20.709,21.124,21.822,22.352,23.508,21.677,23.983,22.143,23.298,19.866,19.325,20.378,20.171,23.175,23.925,23.448,24.111,23.859,24.729,25.196,25.095,26.014,25.915,26.375,22.295,21.768,20.827,19.866,21.114,20.284,20.426,21.45,20.569,19.575,18.905,19.49,19.382,19.314,17.971,17.414,19.323,20.503,20.257,21.791,17.496,16.195,16.047,14.935,15.713,15.897,16.383,15.639,16.616,15.868,16.358,16.608,17.155,17.114,18.434,19.448,16.796,17.825,18.917,17.486,18.409,19.588,20.825,21.909,19.279,20.653,21.757,22.397,23.619,21.279,21.569,22.048,22.772,23.814,20.886,22.216,22.807,24.196,25.147,21.895,20.591,22.604,19.573,24.315,25.586,26.627,27.82,25.378,26.167,27.046,27.34,28.181,26.415,26.165,25.257,24.948,24.819,26.652,26.822,28.109,28.258,25.609,25.539,26.371,24.547,29.003,30.282,30.148,30.946,31.181,29.148,28.959,28.497,28.886,27.995,28.556,27.497,27.958,27.689,27.217,28.428,28.612,26.185,25.925,24.886,29.264,30.467,31.373,31.817,31.227,30.435,29.91,29.94,31.313,31.439,32.348,33.722,33.942,34.592,34.709,34.392,36.144,33.365,33.464,32.86,33.462,32.72,33.324,32.764,31.684,30.993,31.819,32.013,29.589,28.632,29.045,27.273,32.337,33.137,34.401,34.775,33.485,33.866,32.808,34.027,35.032,36.231,35.89,36.711,36.843,37.831,37.62,38.921,34.654,34.192,34.083,33.931,34.156,34.067,35.412,35.459,33.443,31.976,31.544,31.104,36.013,35.306,36.113,36.242,33.915,36.664,37.476,38.731,39.138,37.907,38.823,36.685,39.213,38.93,37.484,36.6,39.21,39.112,38.714,39.478,37.259,35.929,35.303,34.103,35.949,36.196,36.511,34.902,33.31,32.542,31.538,31.403,31.869,32.955,31.015,32.424,30.892,29.921,30.556,30.119,29.25,28.497,28.32,31.611,32.296,32.887,32.79,33.407,32.873,31.732,33.696,33.491,34.102,33.118,33.49,34.811,35.792,35.51,31.869,30.876,29.937,29.031,30.052,30.791,31.212,31.078,31.904,31.768,32.179,32.874,30.161,29.31,29.178,28.062,29.819,29.895,29.31,27.796,27.029,29.981,29.978,30.462,30.864,27.369,25.95,25.227,24.137,25.779,24.409,25.852,25.301,25.215,24.218,26.177,25.69,26.145,26.248,26.295,25.126,24.419,27.613,27.815,28.324,28.797,22.34,21.083,20.767,19.813,21.124,19.941,22.765,22.27,20.799,20.167,23.157,24.655,25.463,24.967,19.393,19.217,18.453,18.743,20.59,20.413,21.302,17.849,18.761,18.403,19.275,18.803,17.463,17.248,17.892,18.627,19.992,20.537,18.789,17.461,17.315,16.502,20.541,21.843,21.713,22.59,22.322,22.549,23.1,22.218,23.318,22.43,22.98,19.535,19.335,20.654,20.91,18.326,16.961,16.027,16.355,21.505,22.808,23.643,24.286,23.503,24.786,25.987,24.606,23.596,24.332,23.893,24.72,24.136,22.663,23.19,24.675,25.155,22.942,21.199,25.391,26.819,27.015,27.887,27.417,28.924,29.724,29.36,26.194,26.267,26.051,26.733,25.219,25.463,25.293,25.111,24.821,26.029,26.418,23.553,22.301,23.379,22.031,26.631,27.798,29.003,29.71,28.169,29.485,27.058,29.229,30.331,30.118,31.056,30.446,31.373,30.956,28.877,28.54,28.903,29.477,27.054,28.575,28.882,30.391,30.898,28.243,32.869,33.545,32.941,33.573,33.767,34.614,34.035,35.992,34.81,36.78,36.188,31.725,31.082,31.03,30.583,31.51,31.519,32.211,31.866,32.175,31.514,33.195,33.929,33.076,33.404,35.198,31.983,31.099,30.121,28.94,30.349,29.702,30.478,28.315,29.888,27.711,28.504,30.635,29.839,28.859,28.891,30.757,31.441,31.785,28.011,27.06,27.794,27.366,30.798,31.362,30.246,30.429,32.048,32.412,33.304,29.085,27.926,27.508,27.205,26.75,25.515,24.304,24.423,23.13,27.5,27.123,28.047,27.578,27.158,29.354,30.336,30.207,30.227,31.757,30.04,29.911,28.667,28.735,29.884,29.892,31.054,28.73,31.057,28.724,29.891,27.535,26.283,26.311,25.74,25.094,24.829,23.558,23.228,22.837,28.973,29.791,28.985,28.965,31.03,31.847,32.659,31.807,33.423,32.566,33.374,28.316,27.527,26.387,26.061,27.007,28.088,29.182,28.031,25.798,24.708,25.202,24.483,24.106,26.43,26.997,27.27,27.004,28.295,28.909,28.001,27.78,28.062,26.755,26.661,28.75,28.832,30.15,25.738,24.425,23.816,23.295,23.473,22.032,23.951,23.925,23.401,24.13,23.497,23.562,22.953,25.456,25.455,26.287,26.263,25.957,25.914,25.877,25.783,27.768,27.75,28.125,28.55,25.752,25.424,24.078,23.881,25.461,26.853,26.755,27.488,23.149,21.847,21.684,20.635,23.027,22.894,21.67,21.621,24.186,25.478,25.495,25.044,26.044,20.68,19.465,18.568,18.135,18.728,19.321,20.582,18.629,21.138,19.177,20.43,20.975,-3.911,-4.069,-2.774,-4.694,-5.815,-6.899,-6.979,-6.385,-6.963,-7.727,-8.795,-10.042,-10.537,-9.116,-7.935,-8.366,-7.385,-10.526,-11.725,-12.945,-12.814,-11.727,-11.812,-15.626,-16.732,-16.198,-16.699,-17.634,-15.167,-14.553,-13.955,-14.134,-13.491,-13.265,-12.651,-13.711,-13.619,-11.62,-11.062,-10.48,-16.78,-17.483,-16.662,-17.19,-17.781,-15.37,-14.494,-14.407,-14.526,-13.088,-12.094,-12.577,-10.714,-14.219,-14.12,-15.426,-15.414,-13.725,-13.364,-14.06,-12.239,-13.44,-12.321,-11.172,-11.375,-10.232,-10.341,-16.548,-17.854,-17.894,-18.409,-18.959,-20.206,-14.996,-13.902,-13.617,-12.86,-12.585,-11.972,-12.86,-10.618,-14.226,-14.002,-14.195,-13.512,-13.243,-12.028,-10.76,-9.656,-11.802,-12.935,-10.902,-9.733,-8.668,-7.48,-10.145,-10.876,-7.868,-7.146,-6.279,-5.152,-8.126,-6.8,-6.049,-4.783,-3.694,-6.913,-8.069,-6.057,-9.067,-4.922,-3.787,-2.72,-1.522,-3.151,-2.227,-1.463,-0.229,-2.986,-2.089,-2.53,-0.834,-1.05,-0.234,1.097,2.155,-1.012,1.054,2.274,3.158,4.385,1.949,1.438,1.037,2.509,2.535,3.28,4.043,5.183,2.338,1.729,4.937,5.938,7.244,8.311,5.391,6.364,5.791,6.132,7.16,8.364,9.074,10.278,8.109,7.069,7.674,6.957,8.341,8.929,9.586,10.764,7.872,8.466,7.387,8.818,9.298,10.304,11.229,8.112,8.596,7.07,10.125,11.021,11.531,11.204,10.261,9.471,8.184,10.009,7.452,9.284,8.008,7.283,12.409,12.962,13.584,13.67,13.971,14.385,13.075,13.962,14.565,13.592,14.013,15.113,14.064,13.84,13.506,14.04,12.293,11.276,11.224,10.67,9.871,9.501,9.852,13.459,14.201,13.282,13.625,15.263,14.598,16.212,14.439,12.098,11.148,10.456,9.758,10.11,10.697,11.097,10.904,11.697,11.5,11.893,12.478,10.668,10.062,11.151,10.907,9.098,8.081,6.934,8.294,6.011,7.379,6.235,12.342,13.492,13.288,14.082,14.75,14.562,10.776,10.058,11.09,12.18,9.053,7.943,7.566,7.048,6.487,6.15,6.916,5.129,5.9,5.022,10.763,11.683,12.118,13.274,10.836,10.033,9.589,10.46,9.436,9.126,9.872,9.936,8.767,10.69,8.039,7.644,6.224,5.348,7.772,6.992,9.23,5.977,4.667,3.571,3.798,4.9,6.354,6.943,2.39,1.252,1.374,0.556,2.378,2.584,1.57,0.97,3.994,4.163,1.343,0.394,0.711,-0.196,0.589,0.816,1.795,1.998,2.429,2.072,1.58,3.935,4.286,3.466,5.51,2.336,2.03,0.518,0.062,2.606,4.13,1.973,4.606,-1.719,-2.112,-1.751,-2.576,-1.429,-1.327,-2.131,-3.456,-3.959,-2.813,-2.661,-3.996,-4.372,-1.609,-1.568,-0.232,-4.731,-6.02,-7.031,-7.964,-6.56,-5.718,-5.301,-6.911,-6.831,-7.698,-7.374,-8.263,-6.094,-5.664,-6.239,-6.613,-6.302,-6.86,-8.369,-8.914,-6.59,-5.082,-7.344,-4.43,-9.043,-10.478,-10.718,-11.664,-11.013,-9.839,-9.948,-9.757,-10.505,-8.895,-8.859,-10.202,-7.733,-8.782,-8.498,-9.722,-10.031,-7.312,-10.421,-11.609,-12.699,-13.339,-12.177,-13.447,-11.142,-12.891,-12.904,-13.897,-13.92,-13.504,-13.505,-14.36,-14.348,-14.087,-14.192,-14.333,-13.007,-11.535,-11.045,-11.092,-10.295,-11.742,-10.978,-13.191,-11.415,-7.463,-6.519,-5.492,-5.153,-5.813,-6.743,-6.04,-7.211,-5.704,-6.301,-6.718,-6.464,-7.541,-7.171,-8.119,-7.315,-7.778,-6.718,-6.982,-5.508,-4.427,-3.345,-2.23,-3.803,-4.781,-4.037,-5.528,-3.68,-2.736,-2.02,-0.791,-3.45,-3.784,-4.817,-3.826,-1.695,-0.879,0.405,1.471,-1.677,-2.379,-2.923,-3.483,0.308,1.477,2.467,3.652,1.078,2.205,2.777,3.288,4.437,5.55,2.15,1.731,2.539,3.765,1.856,6.126,7.159,8.18,9.384,6.52,7.704,8.599,8.894,9.991,8.099,7.947,6.762,8.994,6.618,8.863,7.67,7.53,8.099,8.522,9.633,10.538,7.319,6.16,4.952,6.587,9.467,9.407,9.696,8.95,8.019,7.955,7.744,9.487,8.349,8.461,7.451,8.344,8.798,9.962,9.683,9.904,9.115,8.631,11.394,11.905,8.956,8.235,6.736,6.092,8.556,10.026,10.551,10.715,6.177,4.746,4.448,3.42,4.319,4.425,4.144,4.804,4.24,4.903,4.621,6.114,6.238,4.948,4.369,7.427,8.75,7.348,9.966,4.48,3.259,2.036,1.202,3.014,1.841,1.915,1.863,1.932,0.811,0.782,-0.285,0.88,0.576,-0.054,1.026,1.983,1.569,0.066,-0.56,2.338,4.079,-0.515,-1.953,-2.712,-3.676,-2.36,-1.83,-3.876,-2.046,-2.255,-2.893,-2.818,-3.791,-2.234,-2.327,-1.711,-3.774,-1.667,-1.464,-2.441,-3.095,-0.003,0.127,0.933,-2.565,-3.476,-4.919,-5.609,-3.398,-4.47,-2.018,-5.361,-6.72,-6.992,-8.087,-7.005,-6.951,-8.371,-7.033,-5.987,-6.139,-6.294,-7.097,-4.947,-5.154,-5.565,-5.659,-6.998,-7.576,-4.525,-4.645,-3.14,-7.48,-8.727,-10.012,-11.058,-8.673,-7.584,-7.599,-6.534,-9.944,-11.121,-10.847,-11.019,-11.605,-11.898,-13.12,-10.947,-13.393,-11.207,-12.432,-10.483,-10.191,-11.31,-11.041,-9.918,-10.708,-10.428,-11.633,-10.446,-10.195,-9.252,-9.225,-8.095,-8.303,-6.821,-7.259,-5.77,-5.991,-11.066,-10.966,-9.889,-9.366,-12.299,-12.586,-13.441,-9.542,-8.525,-8.964,-8.144,-8.426,-8.669,-9.853,-10.967,-10.85,-9.364,-9,-11.43,-6.771,-6.234,-6.656,-5.82,-6.589,-8.087,-5.84,-7.931,-8.441,-7.838,-7.45,-9.965,-10.532,-7.736,-7.165,-5.684,-5.158,-7.351,-6.964,-7.867,-7.068,-5.437,-5.845,-4.703,-4.374,-7.067,-7.403,-7.998,-7.129,-8.316,-7.442,-8.035,-4.091,-2.98,-1.785,-1.081,-2.577,-3.626,-3.157,-3.914,-1.567,-0.482,-0.844,0.009,-0.31,0.126,-2.122,-2.591,-2.405,-2.041,-2.654,-2.495,-1.019,-0.689,-3.034,-2.711,-4.533,-0.132,1.301,1.775,2.581,2.076,1.259,1.646,1.107,1.794,1.17,1.676,3.203,1.138,-0.11,-0.715,0.054,0.184,-2.182,0.556,1.333,2.686,3.271,1.544,0.325,0.693,-0.168,3.169,4.447,4.345,5.359,5.026,6.627,8.009,9.128,10.099,8.166,7.782,6.524,8.68,6.172,8.341,7.085,6.742,22.647,21.543,21.727,22.015,20.171,19.057,19.999,19.831,18.503,18.489,19.368,17.905,17.823,18.584,16.914,18.202,17.179,15.898,16.469,15.188,15.481,17.484,17.33,16.401,15.689,16.663,15.325,17.461,13.684,12.309,11.908,10.813,12.159,10.802,9.711,10.61,8.447,9.348,8.266,11.528,10.306,9.067,8.17,10.248,9.033,7.909,7.65,6.508,8.123,8.22,6.96,8.707,8.547,7.972,7.044,9.887,10.401,9.704,9.387,4.17,3.311,3.075,1.937,3.902,3.728,3.259,4.444,4.151,4.026,3.357,2.614,5.361,5.714,5.257,6.437,5.509,6.695,6.23,3.59,3.593,2.983,2.982,1.473,1.473,0.72,0.724,3.639,3.622,3.579,4.992,-0.4,-1.035,-1.533,-0.841,-0.07,0.393,1.466,-0.778,-1.857,-1.328,-2.294,-2.181,-0.127,1.093,0.983,2.356,2.095,3.477,3.335,4.432,-5.21,-6.17,-5.391,-5.726,-6.873,-7.711,-7.759,-4.312,-3.47,-2.719,-2.59,-2.47,-1.505,-3.239,-2.229,-1.507,-2.376,-1.996,-3.541,-4.472,-4.865,-4.801,-5.664,-4.999,-3.958,-5.216,-5.615,-4.522,-4.796,-5.975,-3.285,-2.131,-1.907,-1.806,-0.852,-0.993,0.383,-1.844,-1.624,-2.792,-2.578,-1.38,-1.039,0.21,-0.841,-4.017,-5.204,-5.176,-5.463,-6.468,-2.527,-1.444,-1.881,-1.578,-0.214,1.299,-2.626,-3.102,-4.115,-4.258,-3.689,-2.727,-3.485,-1.598,-4.818,-5.794,-5.138,-5.339,-6.958,-7.894,-7.747,-4.335,-3.677,-2.535,-2.303,-3.183,-2.523,-4.356,-1.839,-0.726,-0.956,-0.201,0.598,0.829,0.947,0.929,1.159,1.142,1.254,1.446,-1.965,-2.388,-1.299,-1.461,-3.337,-3.906,-2.678,-0.203,0.88,1.468,2.069,1.973,2.473,3.519,1.98,3.713,2.782,0.941,2.579,0.738,1.555,1.242,1.744,1.026,1.482,1.642,0.283,2.506,-0.078,-0.841,-0.08,-0.424,-2.198,-3.113,-4.407,-4.812,-5.058,2.581,3.101,2.045,2.378,4.141,4.984,4.67,6.055,5.399,6.79,6.456,7.179,0.775,-0.297,-1.3,-2.498,-0.992,-0.045,0.389,0.445,1.297,1.355,1.783,-0.785,-1.629,-2.305,-3.327,-3.365,-4.075,-5.582,-6.314,-3.481,-2.211,-1.15,-2.093,0.009,-0.937,0.116,-6.047,-7.473,-7.987,-7.255,-9.259,-9.938,-9.371,-9.376,-11.433,-12.241,-11.696,-13.549,-8.912,-8.347,-8.833,-8.217,-6.816,-6.029,-4.55,-6.25,-9.955,-10.453,-10.955,-11.107,-11.494,-12.953,-13.685,-13.621,-15.045,-14.985,-15.69,-17.037,-11.208,-11.693,-11.4,-11.06,-13.201,-14.026,-14.342,-14.341,-11.586,-11.351,-12.105,-11.517,-11.706,-13.396,-14.219,-13.714,-13.653,-15.68,-13.332,-12.824,-11.515,-11.31,-12.616,-10.64,-9.354,-9.579,-8.912,-8.47,-8.145,-7.185,-7.412,-10.562,-10.899,-11.435,-11.265,-11.92,-12.092,-12.654,-11.695,-12.005,-13.948,-15.086,-16.286,-16.179,-17.434,-10.538,-9.572,-8.786,-7.891,-8.617,-7.827,-7.118,-7.937,-9.088,-8.423,-6.92,-6.157,-9.087,-6.486,-5.061,-4.296,-3.158,-4.832,-5.241,-4.73,-2.962,-4.921,-4.306,-4.139,-3.051,-5.177,-4.668,-5.167,-5.209,-5.185,-4.081,-3.293,-6.548,-7.553,-0.774,-0.58,-0.073,0.802,-0.626,-0.226,1.229,2.001,-1.137,-2.501,-0.828,1.604,2.966,3.952,4.98,3.098,2.219,4.531,3.619,4.479,4.64,5.76,3.958,4.167,4.663,3.573,3.535,3.599,4.439,5.241,2.204,1.773,0.554,2.887,4.245,5.015,6.481,7.362,4.526,3.37,2.217,3.668,6.737,8.098,8.867,10.091,8.167,8.807,9.264,10.049,7.859,7.521,6.441,8.777,12.969,13.58,13.317,14.201,13.014,12.095,11.727,12.597,13.067,10.241,9.901,9.332,15.686,17.067,17.229,17.67,18.043,19.498,19.808,20.395,16.891,16.963,18.37,18.529,16.261,14.766,13.965,14.031,19.321,18.288,18.808,18.192,17.401,16.728,16.35,15.932,19.951,20.527,20.825,20.308,21.799,21.442,22.488,21.606,21.956,20.733,20.589,23.001,24.343,24.648,25.152,19.841,18.63,17.754,17.13,17.827,18.603,16.518,17.73,16.926,17.696,17.12,16.131,14.965,13.872,14.951,12.792,13.877,12.803,11.751,18.974,19.777,19.164,18.999,21.203,16.049,14.751,14.915,14.099,13.966,12.465,12.223,11.87,15.987,16.258,16.579,16.04,17.411,17.706,17.447,17.831,16.59,16.419,18.904,19.316,20.119,15.722,14.494,13.588,13.189,13.734,12.456,12.812,11.57,14.303,15.006,15.104,14.575,16.421,17.161,11.292,10.013,10.055,9.015,9.456,9.446,9.138,8.454,10.509,9.688,10.455,11.608,9.248,8.527,8.353,11.742,11.916,11.25,11.067,13.439,13.934,13.043,8.3,7.031,7.145,6.144,6.55,6.233,6.437,5.716,8.361,8.592,8.285,7.651,10.047,10.44,9.527,11.74,9.902,12.125,11.206,6.33,4.895,4.123,3.2,4.428,5.1,4.579,4.845,4.514,3.866,4.076,3.143,4.459,3.853,4.037,2.387,5.293,5.632,4.735,4.245,7.093,1.732,0.381,0.3,-0.7,-0.023,-0.341,1.359,1.398,1.324,0.617,2.673,2.797,1.594,4.09,2.03,2.012,0.585,0.13,2.99,4.326,2.961,-0.116,-1.492,-2.393,-3.216,-2.03,-1.257,-3.537,-1.295,-2.221,-3.025,-2.74,-3.668,-2.79,-3.502,-3.297,-1.464,-1.059,-1.62,-2.108,0.489,0.921,0.994,-1.592,-2.113,-3.6,-4.062,-1.888,-4.347,-5.782,-6.029,-6.999,-6.411,-4.019,-3.71,-4.244,-4.175,-2.204,-1.681,-0.95,-1.971,-0.517,-1.541,-0.814,-4.762,-5.322,-4.404,-3.246,-4.909,-4.142,-2.852,-1.978,-5.023,-5.594,-2.736,-1.529,-0.388,0.78,-1.785,-0.723,0.283,0.771,0.463,-0.308,0.729,1.518,0.918,2.484,1.881,2.667,1.522,2.049,3.142,3.585,2.645,3.757,1.599,3.588,4.645,5.913,6.626,6.072,5.859,5.987,6.603,4.479,4.19,4.451,5.438,5.49,6.941,7.331,4.728,4.674,4.091,3.449,4.321,7.745,9.157,9.888,10.579,9.818,9.704,10.34,9.859,10.666,10.049,8.548,7.98,8.514,8.909,6.446,5.803,5.746,5.298,5.201,4.752,4.703,8.525,9.009,10.51,10.983,8.645,7.141,6.764,5.579,7.758,12.176,12.292,11.953,12.717,11.359,11.238,12.266,10.092,12.152,9.967,11,10.811,10.388,11.372,11.616,8.978,7.868,7.901,7.594,11.957,12.931,14.147,14.678,13.364,14.586,15.725,15.349,16.149,16.202,17.33,16.681,14.126,13.651,13.631,14.188,12.224,11.643,12.287,13.039,12.925,14.304,14.516,12.138,12.172,10.697,15.246,16.604,17.236,17.822,17.433,18.73,17.087,17.088,17.628,17.64,17.063,17.071,17.802,17.817,17.338,17.362,17.884,17.963,15.756,15.11,15.578,15.514,13.589,12.941,11.471,13.104,16.055,16.513,18.012,18.488,20.182,20.744,20.344,20.527,20.454,20.779,22.202,23.155,22.355,19.799,19.408,20.633,21.311,18.32,16.933,16.565,15.988,15.293,14.714,14.374,13.111,7.996,7.141,7.515,9.512,9.2,7.567,7.118,6.602,6.683,7.277,6.298,6.209,4.754,4.492,3.591,5.181,8.349,8.021,8.315,8.895,8.919,7.921,9.279,10.178,9.121,9.642,10.348,10.248,9.486,11.022,10.679,11.952,8.154,8.595,8.382,7.832,7.663,8.645,7.506,8.668,8.979,10.1,8.121,26.751,28.295,29.768,25.447,23.923,28.672,29.096,30.486,30.914,29.777,32.318,31.319,31.105,31.937,32.996,31.526,27.434,28.665,28.705,27.481,26.711,29.881,26.972,27.627,24.984,24.649,23.34,22.852,23.905,22.681,21.607,20.816,26.198,25.001,25.057,26.295,26.983,23.948,26.844,27.523,27.881,27.841,28.201,2.428,1.993,-0.999,2.548,5.507,0.79,0.84,-0.467,-1.3,-0.508,-2.78,-0.828,-0.583,-0.779,-1.027,-0.662,1.037,-0.287,-0.838,0.155,1.316,-2.227,0.123,-0.845,3.771,3.684,4.908,5.749,5.024,5.153,7.069,7.583,3.538,4.835,5.37,4.365,3.241,6.798,4.426,4.408,4.926,5.72,4.569,7.296,6.965,8.154,7.001,5.696,7.435,7.365,7.738,8.045,7.861,8.497,7.838,9.24,9.481,9.856,9.289,7.631,8.093,8.36,7.978,7.529,9.031,8.045,7.588,6.497,6.5,5.926,5.6,5.948,5.726,5.096,4.035,6.491,5.922,5.557,5.886,6.514,4.957,5.549,4.122,3.795,3.488,3.846,-3.343,-5.372,18.621,15.463,4.344,-1.292,-1.588,11.755,24.91,23.286,14.271,2.245,-2.112,1.275,30.388,1.056,21.064,-1.395,6.383,15.637,10.1,-6.873,4.462,9.272,7.756,1.375,11.048,5.938,-2.582,22.319,24.867,7.034,18.681,18.07,26.711,16.204,6.239,13.42,13.545,27.492,11.313,18.928,7.57,18.675,5.995,-9.208,18.231,12.465,5.695,-6.019,24.491,26.298,13.806,3.451,1.02,14.349,-1.762,27.415,14.169,12.042,12.363,17.901,18.555,11.391,7.989,0.815,6.157,7.1,22.678,8.394,20.387,12.681,-2.046,14.856,19.432,5.053,14.009,22.725,12.327,21.241,0.05,5.413,5.71,7.128,2.739,2.797,2.303,5.825,-1.092,8.942,25.744,-2.186,13.863,13.551,3.472,16.377,1.78,19.674,7.064,36.722,30.624,20.143,37.151,32.988,27.389,16.375,19.03,25.072,40.712,32.235,38.362,36.176,13.902,29.357,31.853,18.051,35.318,32.4,32.337,22.607,22.745,20.915,21.782,32.67,35.28,31.194,20.976,37.051,37.823,16.084,34.338,36.89,14.466,12.898,25.496,31.233,45.005,31.859,42.459,26.102,17.248,16.418,37.003,17.326,33.65,34.882,21.729,32.476,25.617,16.727,27.604,40.12,20.624,17.948,23.705,18.446,35.374,14.855,28.755,35.645,36.657,22.722,28.641,39.348,31.726,39.67,20.054,36.678,40.951,29.067,23.704,16.399,35.407,41.695,21.191,30.258,26.431,40.059,34.621,6.927,12.376,-18.543,-13.163,-16.676,-13.737,4.944,-2.622,-16.106,5.067,-11.851,-5.808,-14.118,5.553,12.192,-10.621,-9.003,12.298,-14.853,13.721,1.823,-17.821,11.215,-15.198,-8.683,-2.389,-11.894,-2.7,1.101,-20.799,-20.628,8.014,16.194,10.086,7.961,-10.801,4.119,14.324,-7.812,-16.871,-0.684,-0.789,-4.704,-1.28,-2.157,-8.964,-11.078,-14.068,8.11,-4.201,14.447,7.329,-14.751,-16.766,1.786,2.458,10.12,-16.039,-13.746,-14.38,-4.674,13.046,-5.927,-6.368,10.832,13.911,2.205,-14.385,-9.316,13.079,3.213,7.349,13.014,1.74,-7.371,1.213,8.746,-8.797,12.372,12.249,-10.43,1.381,15.734,-8.74,10.859,16.72,-5.554,-17.527,6.254,-16.154,-18.531,-14.647,-4.633,7.007,6.75,0.014,-4.701,11.998,-16.595,-8.658,-0.271,-19.378,-14.511,-4.049,-10.57,9.222,2.182,-21.468,19.869,15.143,16.689,19.853,13.355,1.421,21.639,22.947,8.242,11.569,5.081,1.696,16.1,6.238,17.657,9.858,18.044,2.447,19.868,17.208,19.443,-4.447,-6.033,18.846,18.638,18.884,-7.459,0.998,14.641,14.846,12.553,-8.246,12.182,-8.429,-7.26,14.676,7.911,5.169,-7.616,3.931,9.779,-0.157,-6.092,11.966,19.048,-10.474,14.202,10.078,10.637,1.81,18.462,-4.542,10.727,4.352,0.933,7.352,10.475,1.274,7.74,9.627,21.313,1.973,15.86,1.704,14.847,-0.049,-6.946,10.476,-6.205,15.768,8.487,5.556,1.003,21.624,7.488,12.144,6.49,16.592,24.669,4.422,21.683,6.081],"type":"scatter3d","mode":"markers","name":"neutral","marker":{"color":"rgba(190,190,190,1)","line":{"color":"rgba(190,190,190,1)"}},"textfont":{"color":"rgba(190,190,190,1)"},"error_y":{"color":"rgba(190,190,190,1)"},"error_x":{"color":"rgba(190,190,190,1)"},"line":{"color":"rgba(190,190,190,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>pdb %&gt;% plot_ly(x = ~x1, y = ~x2, z = ~x3, color = ~resname) %&gt;%
  add_markers()</code></pre>
<div id="htmlwidget-2" style="width:672px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2">{"x":{"visdat":{"413e4d7f192c":["function () ","plotlyVisDat"]},"cur_data":"413e4d7f192c","attrs":{"413e4d7f192c":{"x":{},"y":{},"z":{},"color":{},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"scatter3d","mode":"markers","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x1"},"yaxis":{"title":"x2"},"zaxis":{"title":"x3"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[-5.717,-5.027,-5.696,3.998,3.536,4.972],"y":[6.376,7.185,4.893,4.549,3.414,4.842],"z":[3.891,4.515,4.256,-3.911,-4.069,-2.774],"type":"scatter3d","mode":"markers","name":"ACE","marker":{"color":"rgba(102,194,165,1)","line":{"color":"rgba(102,194,165,1)"}},"textfont":{"color":"rgba(102,194,165,1)"},"error_y":{"color":"rgba(102,194,165,1)"},"error_x":{"color":"rgba(102,194,165,1)"},"line":{"color":"rgba(102,194,165,1)"},"frame":null},{"x":[-10.293,-10.789,-10.193,-9.792,-12.317,-10.107,-9.56,-8.095,-7.69,-9.725,-7.803,-8.275,-7.242,-6.948,-9.606,8.859,8.939,8.573,9.223,8.038,10.049,9.88,11.202,11.376,9.071,17.287,17.248,16.552,15.683,18.659,4.104,3.279,2.944,1.799,4.018,2.88,1.972,0.525,-0.399,2.4,15.289,15.532,16.957,17.606,14.548,-9.073,-7.923,-6.764,-6.048,-7.509,4.464,5.565,5.582,6.646,5.456,4.304,4.632,6.144,6.741,3.884,8.294,8.895,8.657,9.201,8.37,11.869,11.287,10.245,9.133,12.379,-10.235,-9.678,-8.812,-8.907,-8.864,-11.056,-11.895,-11.07,-11.514,-12.558,-17.005,-16.224,-17.073,-16.78,-15.329,-18.138,-19.028,-19.677,-19.658,-20.094,-20.228,-20.881,-19.885,-20.202,-21.521,-17.124,-16.783,-17.886,-17.611,-16.517,-20.475,-20.07,-19.304,-18.413,-21.285,2.426,3.679,4.537,5.748,3.404,1.452,1.279,-0.179,-0.688,2.132,-1.128,0.034,0.388,0.755,1.226,1.038,1.718,0.835,1.312,2.065,-0.452,-1.407,-1.573,-1.727,-2.762,6.472,7.173,7.412,8.266,6.411,11.048,11.866,11.581,11.378,13.347,11.537,11.272,9.851,9.645,11.515,6.471,6.908,7.069,6.747,8.215,4.105,4.508,5.135,4.862,5.491,5.964,6.635,5.634,5.764,7.614,3.493,4.34,4.5,4.347,5.708,-2.432,-1.317,-0.614,-0.212,-0.333,0.736,0.87,0.135,0.727,0.402,-9.494,-10.622,-10.336,-10.516,-11.898,-5.695,-5.931,-6.744,-6.478,-6.668,-6.717,-5.78,-4.92,-4.679,-4.893,-12.871,-12.968,-14.084,-13.841,-13.094,11.499,10.131,9.751,8.621,10.024,-2.402,-2.676,-3.277,-4.209,-1.403,-4.938,-6.258,-7.279,-8.436,-6.224,-7.081,-7.22,-8.038,-8.391,-5.848,18.878,19.196,18.924,19.765,18.394,24.051,22.644,21.789,20.955,22.094,22.751,22.303,20.784,20.264,22.737,19.73,19.792,20.929,21.679,18.457,21.063,22.111,23.505,24.403,21.946,23.669,24.946,25.318,26.472,24.876,24.273,24.539,25.96,26.561,23.524,31.482,32.359,32.504,32.66,33.728,27.15,25.836,25.37,24.82,24.837,28.218,28.366,29.379,29.088,28.763,17.813,17.362,16.197,16.153,16.958,14.744,13.504,13.647,13.219,12.333,14.246,14.459,15.445,15.318,14.98,11.363,11.965,11.894,12.056,13.408,9.809,9.668,11.011,11.428,8.62,11.702,13.002,14.023,14.749,13.507,15.955,15.438,16.366,16.592,14.033],"y":[17.848,18.848,18.591,19.522,18.806,17.317,16.926,17.33,17.869,15.424,21.227,21.721,21.466,22.355,21.079,23.21,22.335,20.897,19.955,22.851,15.379,14.199,13.551,12.338,14.558,22.305,22.792,21.762,22.103,23.061,21.081,22.273,22.374,22.658,23.53,19.89,18.785,19.14,18.746,17.533,10.694,9.568,9.46,8.429,9.604,12.682,13.447,12.481,12.624,14.472,9.868,8.919,8.306,8.111,7.824,9.702,10.756,10.992,11.246,12.036,8.764,7.587,7.539,6.68,6.306,0.158,1.237,2.019,2.24,2.17,4.055,5.023,4.341,4.657,6.077,1.987,2.866,3.525,3.606,3.922,6.977,7.337,7.619,7.113,8.526,8.398,8.752,7.527,7.376,9.73,6.645,5.428,4.526,3.949,4.68,5.435,6.07,5.878,5.793,7.552,0.812,0.239,-1.073,-1.36,0.049,-9.174,-8.938,-7.965,-8.145,-8.377,-17.587,-18.764,-19.197,-19.356,-19.92,-1.96,-1.463,-0.036,0.802,-2.398,8.539,9.788,10.988,11.966,9.811,10.904,11.973,12.133,13.244,11.672,14.38,13.327,12.077,11.25,12.969,4.633,3.488,2.306,1.186,3.854,2.562,1.506,0.962,-0.255,2.017,-4.44,-5.314,-6.751,-7.699,-4.812,-1.548,-2.41,-3.696,-4.788,-1.68,-3.559,-4.704,-5.685,-6.898,-4.235,-7.27,-8.231,-9.505,-10.608,-7.622,-22.278,-22.879,-21.832,-22.117,-23.515,-19.813,-18.606,-18.706,-18.475,-17.371,-23.185,-22.347,-20.891,-19.989,-22.822,-10.553,-10.417,-9.139,-8.389,-11.632,-6.529,-5.444,-5.119,-3.952,-5.813,-4.54,-3.521,-3.721,-3.654,-2.148,-6.972,-7.465,-7.463,-7.137,-8.876,-3.557,-3.608,-2.286,-2.271,-3.915,-0.111,0.031,-0.799,-0.401,-0.383,-1.379,-0.691,0.603,1.198,-0.392,-29.17,-29.902,-29.059,-28.974,-31.188,-12.027,-11.929,-11.297,-10.432,-13.306,-7.512,-6.788,-6.624,-5.538,-7.512,-1.304,-2.743,-3.439,-4.223,-3.38,-3.146,-3.751,-3.374,-4.221,-3.338,-2.109,-1.609,-2.31,-2.703,-0.099,-5.227,-6.621,-6.805,-7.866,-7.116,-5.38,-6.338,-6.106,-7.059,-6.345,-18.484,-18.678,-20.107,-20.766,-17.685,-10.155,-8.944,-7.969,-6.783,-9.299,-9.828,-11.176,-11.587,-12.721,-11.247,-19.087,-19.664,-19.98,-21.043,-18.701,-19.053,-19.238,-20.381,-21.127,-17.949,-27.373,-27.926,-26.907,-27.268,-28.334,-21.482,-21.132,-21.243,-20.31,-22.013,-22.366,-22.579,-21.549,-20.961,-23.989,-14.375,-13.415,-13.323,-12.234,-13.798],"z":[8.682,9.623,11.006,11.705,9.686,11.38,12.675,12.813,13.844,12.888,12.828,14.117,15.215,16.013,14.476,20.547,21.714,21.361,21.816,22.827,20.613,19.757,19.357,19.5,18.51,16.564,15.186,14.301,13.496,14.677,9.451,9.232,7.746,7.371,9.697,5.652,5.355,5.699,4.99,6.107,0.006,-0.886,-1.42,-1.263,-2.048,17.537,17.066,16.785,15.797,18.117,6.028,5.886,4.482,3.89,6.943,1.742,0.792,0.794,-0.251,1.142,1.336,0.704,-0.808,-1.495,1.356,39.033,39.825,39.037,39.518,40.327,29.433,28.487,27.422,26.231,29.232,25.088,24.276,23.169,22.019,25.144,18.409,19.588,20.825,21.909,19.279,20.653,21.757,22.397,23.619,21.279,21.569,22.048,22.772,23.814,20.886,24.315,25.586,26.627,27.82,25.378,29.003,30.282,30.148,30.946,31.181,36.013,35.306,36.113,36.242,33.915,30.161,29.31,29.178,28.062,29.819,23.596,24.332,23.893,24.72,24.136,28.877,28.54,28.903,29.477,27.054,28.575,28.882,30.391,30.898,28.243,33.195,33.929,33.076,33.404,35.198,27.5,27.123,28.047,27.578,27.158,29.354,30.336,30.207,30.227,31.757,25.798,24.708,25.202,24.483,24.106,-15.626,-16.732,-16.198,-16.699,-17.634,-15.167,-14.553,-13.955,-14.134,-13.491,-16.78,-17.483,-16.662,-17.19,-17.781,-7.868,-7.146,-6.279,-5.152,-8.126,-1.05,-0.234,1.097,2.155,-1.012,-1.719,-2.112,-1.751,-2.576,-1.429,-9.043,-10.478,-10.718,-11.664,-11.013,-8.782,-8.498,-9.722,-10.031,-7.312,6.126,7.159,8.18,9.384,6.52,-10.967,-10.85,-9.364,-9,-11.43,-0.132,1.301,1.775,2.581,2.076,-0.11,-0.715,0.054,0.184,-2.182,3.169,4.447,4.345,5.359,5.026,11.528,10.306,9.067,8.17,10.248,-5.216,-5.615,-4.522,-4.796,-5.975,-4.017,-5.204,-5.176,-5.463,-6.468,-11.586,-11.351,-12.105,-11.517,-11.706,-13.396,-14.219,-13.714,-13.653,-15.68,-13.332,-12.824,-11.515,-11.31,-12.616,-10.562,-10.899,-11.435,-11.265,-11.92,-9.088,-8.423,-6.92,-6.157,-9.087,12.969,13.58,13.317,14.201,13.014,18.974,19.777,19.164,18.999,21.203,5.293,5.632,4.735,4.245,7.093,-1.592,-2.113,-3.6,-4.062,-1.888,-4.347,-5.782,-6.029,-6.999,-6.411,-2.736,-1.529,-0.388,0.78,-1.785,7.745,9.157,9.888,10.579,9.818,9.704,10.34,9.859,10.666,10.049,11.957,12.931,14.147,14.678,13.364],"type":"scatter3d","mode":"markers","name":"ALA","marker":{"color":"rgba(167,180,143,1)","line":{"color":"rgba(167,180,143,1)"}},"textfont":{"color":"rgba(167,180,143,1)"},"error_y":{"color":"rgba(167,180,143,1)"},"error_x":{"color":"rgba(167,180,143,1)"},"line":{"color":"rgba(167,180,143,1)"},"frame":null},{"x":[-6.448,-6.363,-6.743,-6.089,-7.285,-6.796,-7.835,-7.273,-7.897,-9.118,-7.291,13.125,12.576,13.082,13.526,11.051,10.418,8.927,8.403,8.233,8.52,7.882,18.785,18.512,18.082,18.655,19.716,20.045,20.969,21.205,20.472,19.446,20.749,9.797,10.952,11.962,13,11.68,10.863,10.461,11.611,12.411,12.196,13.448,11.893,12.143,11.065,10.15,12.212,10.972,11.084,9.779,9.44,10.315,8.204,11.15,12.75,11.891,11.14,9.926,12.716,13.347,14.132,14.775,14.191,12.942,14.872,-9.862,-8.972,-8.571,-8.601,-7.725,-6.741,-5.683,-6.27,-6.38,-5.934,-6.975,-13.787,-14.128,-14.568,-15.483,-12.924,-13.05,-11.8,-11.751,-12.102,-12.534,-12.02,0.173,0.985,2.359,3.344,0.239,-0.037,-1.053,-2.423,-3.501,-3.376,-4.708,-1.542,-1.691,-0.525,-0.746,-1.886,-1.702,-2.733,-2.501,-1.656,-0.954,-1.492,-0.214,0.299,-0.687,-0.293,0.633,1.885,2.07,1.058,1.185,2.289,0.198,-1.834,2.311,1.462,2.199,1.606,1.052,0,-0.314,-1.449,-1.484,-0.446,-2.554,1.627,2.842,2.458,3.007,3.726,4.427,5.197,5.736,6.874,7.622,7.227,-12.806,-11.998,-10.578,-10.044,-12.663,-14.041,-13.935,-15.231,-15.47,-14.504,-16.685,-11.205,-12.412,-12.27,-13.26,-13.545,-13.978,-14.796,-16.035,-16.867,-16.605,-17.973,-10.544,-10.716,-10.704,-10.583,-9.598,-8.218,-7.129,-5.837,-4.798,-4.892,-3.673,-10.795,17.617,19.074,19.518,20.412,19.603,19.467,20.008,19.889,20.769,21.849,20.582,20.074,18.615,18.179,17.276,18.026,16.499,15.963,16.24,15.386,14.178,15.755,23.141,23.743,25.164,26.017,22.851,23.454,22.496,23.161,23.104,22.406,23.751,29.346,29.57,28.255,28.242,30.275,29.49,30.367,31.222,32.049,32.152,32.747,16.416,17.438,16.901,17.248,18.548,19.51,20.344,21.23,20.887,19.671,21.758,23.516,23.228,23.432,22.961,24.114,23.532,24.51,25.58,25.442,24.276,26.475,24.054],"y":[19.663,21.123,21.749,22.682,21.666,21.41,21.812,21.765,21.271,20.772,21.276,12.063,10.763,9.706,8.638,10.77,11.596,11.597,12.45,12.064,10.819,12.957,9.063,7.63,7.089,6.132,6.867,7.279,6.301,6.702,6.304,5.479,6.754,9.1,9.024,7.924,7.826,10.375,11.547,11.337,11.032,11.943,13.239,11.555,4.282,3.155,3.021,3.866,1.871,1.642,0.407,0.05,-1.165,-2.162,-1.389,2.066,-1.287,-1.919,-0.857,-0.96,-2.772,-4.008,-4.756,-5.949,-7.14,-7.313,-8.166,3.973,4.606,3.603,3.923,5.171,5.876,6.681,7.792,9.041,9.371,9.963,-6.42,-7.766,-8.747,-9.543,-8.361,-9.844,-10.357,-11.821,-12.567,-11.995,-13.892,-8.909,-10.105,-9.963,-10.537,-11.344,-11.353,-12.424,-11.961,-12.729,-14.01,-12.209,11.01,10.993,11.645,12.4,9.546,9.309,10.043,9.819,10.524,11.513,10.217,-21.321,-22.405,-22.797,-23.583,-23.616,-23.426,-24.551,-24.507,-23.828,-23.13,-23.834,-22.302,-5.868,-6.389,-7.486,-8.504,-5.273,-4.33,-3.216,-2.421,-1.092,-0.389,-0.467,-21.302,-20.813,-19.882,-19.968,-20.04,-20.908,-20.087,-20.959,-21.635,-21.533,-22.47,-7.621,-6.921,-7.495,-7.729,-7.014,-6.377,-4.899,-4.32,-3.015,-2.137,-2.585,1.493,1.64,1.253,1.241,0.833,1.353,2.629,2.367,3.302,4.592,2.94,2.765,3.765,5.184,5.338,3.605,3.739,3.472,3.893,4.219,4.167,4.633,6.127,-27.306,-27.409,-28.058,-27.557,-28.242,-27.598,-28.536,-27.987,-27.161,-26.767,-26.756,-7.685,-7.631,-6.601,-5.799,-9.009,-9.044,-10.474,-11.206,-11.334,-10.792,-11.975,3.639,4.68,4.371,5.258,4.938,5.858,6.059,6.645,7.931,8.791,8.362,-18.655,-18.751,-19.004,-19.68,-17.491,-16.199,-14.98,-14.663,-13.624,-12.789,-13.397,-20.513,-21.544,-22.964,-23.879,-21.239,-22.379,-22.769,-23.886,-25.166,-25.509,-26.105,1.478,2.329,3.803,4.648,1.932,0.823,0.375,-0.453,-1.738,-2.356,-2.405,4.102],"z":[10.942,10.869,12.211,12.674,9.773,8.359,7.318,5.971,4.907,5.012,3.73,22.349,22.736,21.781,22.195,22.658,23.706,23.586,24.635,25.892,26.257,26.805,2.48,2.408,3.77,4.3,1.842,0.415,-0.283,-1.669,-2.708,-2.537,-3.924,-3.525,-4.419,-4.083,-4.732,-4.443,-4.991,-6.448,-7.3,-7.85,-7.648,-8.581,-4.228,-5.117,-6.182,-6.203,-4.29,-3.452,-2.589,-2.05,-1.637,-1.686,-1.212,-6.981,36.774,37.772,38.574,38.774,38.736,38.124,39.181,38.644,38.554,38.959,38.076,23.513,22.544,21.461,20.273,23.239,22.298,23.065,23.816,23.366,22.161,24.111,16.538,16.068,17.155,16.951,15.316,14.934,14.222,14.187,13.139,12.021,13.206,37.564,37.743,37.082,37.552,37.236,35.746,35.386,35.591,35.464,35.135,35.639,31.099,32.546,33.299,34.242,33.017,34.513,35.354,36.778,37.523,36.991,38.799,18.302,17.477,16.38,15.492,18.357,19.207,20.221,21.275,22.414,22.66,23.304,16.406,-14.716,-15.786,-16.547,-16.906,-16.752,-16.197,-17.181,-16.726,-16.712,-17.134,-16.243,3.417,4.067,5.2,6.297,3.087,2.095,1.092,0.058,0.166,1.26,-0.799,9.548,10.553,10.628,11.716,11.933,12.041,12.334,12.672,12.755,12.526,13.06,3.902,4.707,6.179,6.91,4.065,2.697,2.834,3.564,4.019,3.831,4.658,9.007,10.051,9.497,8.266,11.082,10.481,11.49,10.974,11.734,13.056,11.168,10.307,13.253,13.189,11.874,11.187,14.358,15.724,16.78,18.127,18.686,18.023,19.931,-4.796,-4.72,-3.677,-3.932,-4.389,-4.344,-4.296,-5.534,-6.549,-6.489,-7.649,0.969,1.803,2.273,2.325,3.029,4.093,5.258,6.421,6.762,6.031,7.84,11.317,12.752,13.501,14.528,13.266,13.093,13.363,12.223,12.169,13.197,11.065,-5.134,-5.224,-5.015,-5.763,-4.21,-3.918,-5.125,-4.804,-4.891,-5.294,-4.557,20.915,22.063,21.734,22.521,23.247,24.099,25.173,24.623,24.299,24.469,23.8,20.694],"type":"scatter3d","mode":"markers","name":"ARG","marker":{"color":"rgba(211,164,122,1)","line":{"color":"rgba(211,164,122,1)"}},"textfont":{"color":"rgba(211,164,122,1)"},"error_y":{"color":"rgba(211,164,122,1)"},"error_x":{"color":"rgba(211,164,122,1)"},"line":{"color":"rgba(211,164,122,1)"},"frame":null},{"x":[11.513,12.636,12.348,13.163,12.865,13.991,14.006,14.946,21.607,21.362,19.964,19.261,22.401,23.639,23.611,24.738,12.067,12.289,11.162,10.933,13.636,14.788,15.152,15.332,6.933,5.498,4.947,4.157,4.743,4.998,5.464,4.695,-23.419,-23.067,-21.616,-21.226,-24.026,-23.881,-22.858,-24.9,-19.323,-18.652,-17.94,-18.23,-19.642,-20.745,-20.535,-21.926,-19.926,-20.171,-19.591,-18.374,-19.646,-20.181,-19.853,-21.013,-5.906,-5.703,-4.867,-4.081,-7.041,-7.586,-8.647,-6.864,6.288,7.462,7.892,7.533,7.198,8.464,9.527,8.359,6.103,5.377,4.071,3.041,6.228,7.399,7.387,8.425,-3.039,-3.891,-3.305,-3.568,-5.313,-6.038,-6.857,-5.755,-2.832,-2.773,-1.449,-1.436,-2.926,-2.748,-2.247,-3.16,-10.242,-10.617,-9.546,-9.86,-10.888,-12.307,-13.217,-12.507,-1.62,-1.691,-1.494,-1.029,-2.995,-3.117,-2.411,-3.967,1.641,2.41,3.808,4.308,2.502,1.178,0.297,1.028,29.535,28.255,27.117,27.18,28.415,27.178,26.177,27.246,21.612,20.517,20.769,21.89,20.316,21.374,21.238,22.447,28.474,29.408,30.191,29.653,28.671,29.614,30.503,29.423,29.274,30.281,30.132,30.398,31.693,32.129,32.139,32.472,27.121,26.656,25.407,25.47,27.746,27.385,26.271,28.328,26.134,27.522,28.424,29.325,28.028,27.335,26.768,27.374,17.982,18.477,18.828,18.825,19.699,19.379,20.208,18.185],"y":[19.292,19.035,17.792,16.864,20.252,20.049,20.693,19.184,20.894,20.163,19.554,19.644,19.059,19.545,19.814,19.653,4.352,5.436,6.458,7.114,6.104,5.125,4.676,4.75,9.084,9.356,8.935,9.664,8.652,9.288,10.426,8.561,-1.182,-0.037,0.427,0.991,1.129,2.27,2.401,3.113,4.471,5.766,6.032,5.387,6.893,7.01,7.53,6.517,3.263,1.834,1.244,1.12,1.07,-0.337,-1.183,-0.598,-4.603,-5.408,-6.634,-7.111,-5.84,-4.822,-4.232,-4.617,-9.697,-10.517,-10.323,-11.105,-12,-12.845,-12.355,-14.114,-17.269,-18.334,-17.835,-18.505,-18.953,-19.743,-20.206,-19.899,-7.291,-6.359,-6.035,-4.965,-6.906,-7.08,-7.977,-6.213,-21.233,-22.356,-22.332,-22.385,-23.681,-24.902,-25.939,-24.79,-26.133,-24.782,-23.758,-22.671,-24.741,-25.141,-25.021,-25.611,-8.363,-9.626,-9.498,-10.436,-10.341,-10.656,-11.518,-9.918,-10.744,-10.213,-10.829,-11.181,-8.688,-8.013,-8.605,-6.772,2.345,2.522,1.675,0.442,2.235,2.605,3.079,2.397,1.814,0.978,-0.496,-0.893,1.173,0.465,-0.711,1.165,-4.79,-4.223,-5.271,-5.926,-3.283,-2.399,-2.886,-1.092,-15.327,-16.045,-15.683,-16.503,-15.707,-16.581,-16.151,-17.822,-22.691,-22.762,-21.908,-20.756,-22.305,-22.589,-23.013,-22.352,-13.745,-13.465,-13.501,-12.676,-14.458,-14.299,-13.249,-15.348,-2.394,-2.914,-4.39,-5.103,-2.126,-0.671,0.212,-0.42],"z":[19.421,20.307,21.141,21.195,21.211,22.215,23.263,21.894,18.152,16.906,16.879,15.872,16.713,15.999,14.799,16.728,10.139,11.086,11.102,12.118,10.824,10.924,12.012,9.784,14.092,14.17,15.527,16.138,13.04,11.69,11.606,10.625,21.114,20.284,20.426,21.45,20.569,19.575,18.905,19.49,17.155,17.114,18.434,19.448,16.796,17.825,18.917,17.486,26.652,26.822,28.109,28.258,25.609,25.539,26.371,24.547,35.032,36.231,35.89,36.711,36.843,37.831,37.62,38.921,39.213,38.93,37.484,36.6,39.21,39.112,38.714,39.478,31.611,32.296,32.887,32.79,33.407,32.873,31.732,33.696,17.892,18.627,19.992,20.537,18.789,17.461,17.315,16.502,-3.151,-2.227,-1.463,-0.229,-2.986,-2.089,-2.53,-0.834,1.998,2.429,2.072,1.58,3.935,4.286,3.466,5.51,8.956,8.235,6.736,6.092,8.556,10.026,10.551,10.715,1.932,0.811,0.782,-0.285,0.88,0.576,-0.054,1.026,-9.259,-9.938,-9.371,-9.376,-11.433,-12.241,-11.696,-13.549,-11.208,-11.693,-11.4,-11.06,-13.201,-14.026,-14.342,-14.341,-10.538,-9.572,-8.786,-7.891,-8.617,-7.827,-7.118,-7.937,4.245,5.015,6.481,7.362,4.526,3.37,2.217,3.668,15.686,17.067,17.229,17.67,18.043,19.498,19.808,20.395,21.606,21.956,20.733,20.589,23.001,24.343,24.648,25.152,8.3,7.031,7.145,6.144,6.55,6.233,6.437,5.716],"type":"scatter3d","mode":"markers","name":"ASN","marker":{"color":"rgba(247,144,101,1)","line":{"color":"rgba(247,144,101,1)"}},"textfont":{"color":"rgba(247,144,101,1)"},"error_y":{"color":"rgba(247,144,101,1)"},"error_x":{"color":"rgba(247,144,101,1)"},"line":{"color":"rgba(247,144,101,1)"},"frame":null},{"x":[-12.713,-12.917,-12.895,-12.411,-14.242,-14.205,-13.1,-15.286,-11.105,-9.706,-9.014,-8.327,-8.959,-9.408,-9.568,-9.589,7.99,9.416,9.611,10.416,9.993,10.119,9.746,10.608,11.165,10.743,10.654,11.156,9.39,9.052,9.779,8.041,27.762,27.225,25.741,25.153,27.393,26.76,27.283,25.747,-4.338,-5.167,-4.371,-4.936,-5.901,-6.945,-7.929,-6.781,-3.068,-2.199,-0.79,0.088,-2.182,-1.48,-0.865,-1.541,15.826,14.962,13.577,13.463,15.569,14.875,13.659,15.566,-2.962,-2.213,-2.303,-1.295,-2.761,-1.858,-0.621,-2.384,17.592,16.822,15.886,14.781,17.747,18.338,17.828,19.318,6.299,5.823,4.604,3.548,6.949,6.57,5.542,7.318,-9.677,-11.039,-11.222,-12.342,-12.078,-11.816,-11.108,-12.344,-0.952,-0.365,0.555,1.611,-1.449,-1.946,-1.292,-2.983,10.602,11.515,10.844,11.521,12.609,13.617,14.028,14.006,9.521,8.807,7.525,6.552,8.479,8.109,8.841,7.086,-6.356,-5.149,-4.151,-4.39,-5.476,-4.282,-3.119,-4.518,-1.508,-1,-1.793,-1.281,-1.103,-0.524,0.712,-1.301,0.277,0.556,-0.458,-0.095,0.515,1.823,2.825,1.849,4.904,4.599,5.479,5.049,4.873,3.833,2.705,4.142,6.654,6.128,4.936,4.862,5.679,6.829,7.89,6.666,-4.011,-4.594,-3.51,-3.644,-5.357,-6.696,-7.128,-7.329,-0.343,0.973,1.161,1.641,2.072,3.439,3.503,4.459,-10.67,-9.246,-8.635,-9.29,-8.495,-8.507,-9.097,-7.904,-4.212,-4.138,-4.847,-4.525,-4.699,-3.837,-2.677,-4.306,-5.806,-6.548,-6.992,-8.15,-7.77,-8.486,-8.142,-9.416,-6.816,-5.76,-4.81,-4.346,-5.008,-4.047,-3.45,-3.877,6.537,6.508,5.736,4.861,7.931,7.968,7.312,8.652,15.415,16.38,16.111,17.047,16.388,17.049,17.702,16.919,21.566,21.291,21.34,22.12,19.915,19.598,20.453,18.48,29.591,30.007,28.983,29.242,31.401,31.508,30.479,32.651,28.14,28.671,28.878,28.615,29.971,29.721,28.568,30.673,23.216,22.858,23.633,23.423,23.032,21.96,20.78,22.296,24.503,25.299,25.409,26.389,26.689,27.418,26.824,28.592,22.003,20.671,20.329,20.149,19.673,18.246,17.932,17.417,17.674,16.466,16.863,16.189,15.466,14.188,13.383,13.999,15.262,14.113,14.554,13.999,13.138,12.278,12.161,11.702],"y":[13.937,15.334,15.425,16.406,15.859,16.045,16.189,16.053,13.51,13.428,14.777,15.259,12.382,10.968,10.628,10.191,24.121,24.017,23.018,22.087,25.374,26.371,26.061,27.489,17.76,16.643,15.328,14.298,16.973,16.068,15.09,16.348,15.687,14.796,14.578,15.306,15.409,16.789,17.74,16.926,18.107,17.587,16.766,15.927,18.72,19.38,18.708,20.572,17.013,16.285,16.227,17.013,16.965,16.13,15.089,16.526,5.061,3.888,4.306,4.889,2.787,1.44,1.384,0.42,9.959,8.745,8.457,8.155,7.563,6.343,6.504,5.22,-1.405,-0.974,0.182,0.272,-0.603,-1.825,-2.95,-1.659,2.447,3.811,4.11,4.507,4.805,6.245,6.478,7.15,1.851,2.036,3.332,3.733,1.91,2.849,3.869,2.56,-6.519,-6.559,-7.745,-7.601,-6.525,-5.12,-4.155,-4.973,-10.921,-11.939,-13.241,-14.168,-12.259,-11.147,-10.567,-10.857,-13.304,-14.526,-14.23,-14.976,-15.292,-16.738,-17.421,-17.191,-11.921,-11.695,-11.041,-9.933,-10.733,-10.445,-10.681,-9.955,-9.352,-8.033,-6.97,-5.886,-7.764,-6.412,-6.254,-5.493,0.238,1.575,2.583,3.707,1.587,1.139,0.949,0.99,4.293,3.577,2.337,1.274,4.463,5.556,5.41,6.561,0.587,-0.414,-1.125,-2.355,0.231,0.811,0.154,1.927,-20.259,-21.208,-21.817,-21.853,-22.335,-21.878,-20.733,-22.686,-22.211,-22.189,-20.999,-21.171,-22.215,-22.598,-23.156,-22.348,-26.952,-26.755,-25.961,-25.735,-28.077,-29.045,-28.746,-30.132,1.097,2.551,3.273,4.42,3.012,2.569,3.015,1.764,2.6,3.177,2.054,1.632,3.961,4.732,4.583,5.496,-7.378,-6.439,-6.308,-7.311,-6.993,-5.991,-5.168,-6.045,-5.575,-5.226,-3.918,-3.818,-5.079,-5.083,-5.949,-4.224,-30.359,-31.027,-30.72,-30.585,-32.537,-32.902,-32.031,-34.065,-27.898,-28.358,-27.188,-27.202,-29.028,-29.657,-29.638,-30.187,-17.368,-16.344,-16.082,-15.32,-16.634,-18.001,-18.584,-18.491,-18.164,-19.479,-19.688,-20.778,-19.771,-20.276,-20.645,-20.32,-21.943,-21.634,-20.471,-20.13,-22.87,-23.915,-23.544,-25.113,-19.841,-18.738,-17.598,-16.856,-19.263,-18.364,-17.99,-18.04,1.384,1.316,-0.181,-0.841,2.041,2.041,1.327,2.77,-1.752,-2.222,-2.855,-3.761,-1.095,-1.595,-2.254,-1.364,-10.663,-10.925,-11.221,-12.113,-9.745,-9.706,-10.743,-8.633],"z":[4.663,5.025,6.546,7.111,4.462,2.95,2.383,2.326,8.601,9.014,8.844,9.75,8.185,8.483,9.675,7.523,17.99,18.325,19.465,19.357,18.755,17.606,16.455,17.866,21.747,22.586,21.799,22.252,23.235,24.422,24.687,25.1,18.209,19.237,18.943,18.142,20.634,20.75,20.123,21.472,1.354,0.266,-0.745,-1.451,-0.454,0.419,0.789,0.743,-0.82,-1.739,-1.15,-1.519,-3.115,-4.181,-3.848,-5.366,6.061,5.998,6.52,7.599,6.879,6.734,6.448,6.937,13.303,12.995,11.488,10.84,13.806,13.748,13.825,13.632,37.662,38.825,38.481,39.021,39.991,40.683,40.476,41.439,39.886,40.095,39.218,39.719,39.79,40.103,40.764,39.697,37.017,36.524,35.736,35.438,37.653,38.826,38.67,39.921,36.502,37.841,38.077,38.7,38.923,39.208,38.767,39.884,36.095,35.575,35.153,34.712,36.596,36.734,35.704,37.879,35.252,34.899,34.121,34.206,36.188,35.937,35.188,36.497,17.437,16.646,17.61,18.099,15.495,14.586,14.973,13.463,17.147,16.793,17.555,17.846,15.292,14.914,14.988,14.586,22.595,22.068,22.615,22.974,20.536,19.905,20.634,18.665,-14.133,-15.368,-15.42,-15.873,-16.588,-16.767,-16.245,-17.443,-12.839,-11.912,-12.531,-12.522,-10.598,-9.805,-9.709,-9.272,-9.08,-8.122,-7.24,-6.014,-8.837,-9.428,-9.183,-10.143,-2.191,-1.571,-0.623,0.499,-2.641,-2.076,-0.962,-2.749,11.17,11.422,10.269,9.246,11.691,10.51,9.449,10.656,-12,-12.118,-10.973,-10.666,-13.466,-14.632,-14.712,-15.46,-10.344,-9.226,-8.3,-8.329,-9.728,-8.615,-7.417,-8.947,10.771,11.141,9.938,9.399,12.358,12.978,12.251,14.214,-5.02,-3.601,-3.417,-2.549,-3.048,-1.532,-0.918,-0.945,16.382,15.514,14.041,13.247,15.773,17.098,17.712,17.526,6.252,4.898,3.912,2.953,4.855,3.504,2.595,3.358,-3.242,-4.197,-5.302,-6.229,-4.777,-5.437,-5.845,-5.557,8.769,9.129,10.625,11.143,8.37,6.959,6.658,6.154,19.389,20.772,21.38,22.544,21.663,21.431,21.258,21.434,20.6,21.125,20.112,20.093,21.492,22.469,23.501,22.206,9.83,10.432,10.518,9.488,9.521,10.061,11.034,9.485,10.856,10.186,8.853,8.351,9.943,9.283,9.973,8.07,4.517,3.656,2.223,1.571,3.672,4.931,5.625,5.22],"type":"scatter3d","mode":"markers","name":"ASP","marker":{"color":"rgba(229,147,127,1)","line":{"color":"rgba(229,147,127,1)"}},"textfont":{"color":"rgba(229,147,127,1)"},"error_y":{"color":"rgba(229,147,127,1)"},"error_x":{"color":"rgba(229,147,127,1)"},"line":{"color":"rgba(229,147,127,1)"},"frame":null},{"x":[6.665,7.209,6.447,6.224,8.695,9.723,-8.252,-7.857,-8.993,-8.767,-7.429,-6.771,-1.729,-2.769,-2.54,-2.733,-4.16,-4.639,4.646,4.446,4.22,4.688,3.287,3.67,18.851,18.548,18.768,17.958,19.408,18.996,15.541,16.058,16.745,16.614,17.025,16.202],"y":[8.735,9.679,11.001,11.605,9.919,8.489,2.378,1.318,0.971,0.78,0.063,-1.283,2.187,3.062,3.306,4.424,2.481,2.512,-13.749,-15.133,-15.25,-16.206,-15.761,-16.024,-6.598,-5.662,-4.209,-3.333,-5.98,-5.053,-10.471,-10.683,-12.041,-12.752,-9.568,-7.992],"z":[18.28,19.255,19.243,20.297,19.004,19.384,21.873,20.941,19.975,18.78,21.714,20.684,22.663,23.19,24.675,25.155,22.942,21.199,1.983,1.569,0.066,-0.56,2.338,4.079,-2.527,-1.444,-1.881,-1.578,-0.214,1.299,1.732,0.381,0.3,-0.7,-0.023,-0.341],"type":"scatter3d","mode":"markers","name":"CYS","marker":{"color":"rgba(197,153,160,1)","line":{"color":"rgba(197,153,160,1)"}},"textfont":{"color":"rgba(197,153,160,1)"},"error_y":{"color":"rgba(197,153,160,1)"},"error_x":{"color":"rgba(197,153,160,1)"},"line":{"color":"rgba(197,153,160,1)"},"frame":null},{"x":[19.008,19.468,19.297,20.045,18.761,17.268,16.361,15.216,16.867,14.923,15.982,16.245,17.373,15.584,16.413,17.859,18.148,18.776,-12.602,-14,-14.528,-15.568,-14.856,-14.508,-15.309,-16.3,-14.875,-19.133,-20.288,-20.605,-21.457,-21.522,-21.39,-22.487,-23.467,-22.319,9.651,8.987,9.72,9.093,8.966,8.215,8.353,8.764,8.023,7.806,7.658,8.394,7.925,8.111,7.194,7.528,6.858,8.558,2.071,1.447,0.547,-0.231,0.711,1.508,2.905,3.103,3.873,-0.439,-1.377,-2.479,-3.58,-0.644,-0.03,1.459,2.188,1.923,-15.491,-15.568,-14.579,-14.969,-15.264,-16.427,-16.502,-16.403,-16.665,21.807,22.971,23.755,24.897,22.538,21.873,21.372,21.82,20.431,26.487,27.838,28.899,30.091,27.909,27.044,27.064,26.714,27.485,10.733,10.692,10.639,11.357,9.471,9.389,8.074,7.338,7.775,13.527,13.192,13.433,13.839,11.748,11.539,10.146,9.835,9.31,23.857,24.825,24.652,25.564,26.252,26.512,26.154,26.548,25.425],"y":[3.329,2.971,4.121,4.212,1.703,1.85,1.432,1.04,1.495,14.81,14.984,13.684,13.183,16.068,17.335,17.114,16.777,17.299,-4.182,-4.32,-5.716,-6.144,-3.266,-1.827,-0.814,-1.158,0.438,5.822,5.644,4.175,3.874,6.296,7.792,8.343,7.659,9.586,6.431,5.837,4.569,3.552,6.821,6.306,7.223,8.378,6.703,1.033,0.3,-1.038,-2.027,1.152,2.343,3.104,4.083,2.669,-18.416,-19.727,-19.935,-20.889,-20.101,-19.835,-20.456,-21.582,-19.725,-17.467,-16.522,-16.09,-15.732,-15.29,-14.361,-14.583,-13.646,-15.814,-5.16,-6.523,-6.747,-7.112,-7.502,-8.383,-9.663,-9.643,-10.787,1.576,2.023,3.096,3.403,2.581,1.562,2.177,3.251,1.503,-5.774,-5.822,-5.287,-5.32,-5.009,-5.519,-4.584,-3.405,-5.101,-22.751,-21.288,-20.823,-19.899,-20.748,-19.216,-18.706,-19.45,-17.431,-18.504,-17.168,-17.027,-15.961,-16.801,-16.598,-16.093,-15.95,-15.813,-4.521,-3.582,-2.119,-1.311,-4.047,-5.517,-5.898,-5.227,-7],"z":[17.806,16.468,15.465,14.488,15.96,15.665,16.816,16.595,18.043,-1.02,-0.029,0.73,0.763,0.979,0.923,1.336,2.485,0.393,17.339,17.741,17.391,17.901,17.035,17.397,16.6,15.963,16.624,26.167,27.046,27.34,28.181,26.415,26.165,25.257,24.948,24.819,29.085,27.926,27.508,27.205,26.75,25.515,24.304,24.423,23.13,27.535,26.283,26.311,25.74,25.094,24.829,23.558,23.228,22.837,23.027,22.894,21.67,21.621,24.186,25.478,25.495,25.044,26.044,13.962,14.565,13.592,14.013,15.113,14.064,13.84,13.506,14.04,2.777,3.288,4.437,5.55,2.15,1.731,2.539,3.765,1.856,-0.078,-0.841,-0.08,-0.424,-2.198,-3.113,-4.407,-4.812,-5.058,-12.092,-12.654,-11.695,-12.005,-13.948,-15.086,-16.286,-16.179,-17.434,5.438,5.49,6.941,7.331,4.728,4.674,4.091,3.449,4.321,8.525,9.009,10.51,10.983,8.645,7.141,6.764,5.579,7.758,20.182,20.744,20.344,20.527,20.454,20.779,22.202,23.155,22.355],"type":"scatter3d","mode":"markers","name":"GLN","marker":{"color":"rgba(156,159,194,1)","line":{"color":"rgba(156,159,194,1)"}},"textfont":{"color":"rgba(156,159,194,1)"},"error_y":{"color":"rgba(156,159,194,1)"},"error_x":{"color":"rgba(156,159,194,1)"},"line":{"color":"rgba(156,159,194,1)"},"frame":null},{"x":[9.747,11.11,11.69,12.841,11.151,10.827,10.824,11.785,9.856,12.557,13.803,15.006,15.981,13.841,15.162,15.435,14.473,16.621,-6.773,-6.971,-6.766,-7.38,-6.022,-6.302,-5.763,-5.037,-6.065,-10.502,-10.846,-9.681,-9.348,-12.091,-13.318,-13.452,-14.188,-12.821,7.889,7.552,8.708,8.603,6.805,6.469,5.65,4.874,5.784,17.252,17.981,17.412,17.648,19.474,20.351,20.693,21.494,20.173,14.397,13.255,12.251,11.05,13.728,14.536,15.238,15.418,15.628,-5.014,-6.339,-6.939,-7.447,-7.159,-8.331,-9.039,-8.353,-10.287,-0.862,-2.258,-3.152,-4.048,-2.747,-4.064,-4.386,-5.401,-3.625,-6.743,-7.716,-9.17,-10.061,-7.165,-5.809,-5.16,-5.764,-4.028,13.32,13.846,12.695,12.74,14.745,16.099,16.003,16.938,15.016,-15.009,-15.818,-15.016,-15.475,-16.393,-17.503,-18.129,-18.258,-18.49,-14.352,-15.554,-15.699,-15.958,-16.787,-18.102,-18.17,-17.52,-18.893,11.804,10.959,10.997,10.883,9.505,9.266,9.677,9.922,9.748,14.132,13.275,11.806,10.96,13.605,15.06,15.949,15.794,16.798,-8.38,-9.124,-10.333,-10.466,-9.538,-10.374,-9.686,-8.437,-10.403,14.569,14.767,14.669,13.571,13.683,13.805,14.92,15.812,14.898,14.901,15.413,16.932,17.469,14.979,13.471,12.977,13.672,11.872,25.995,27.283,27.413,26.84,27.272,28.64,28.715,28.148,29.345,30.551,31.604,31.147,31.364,32.87,34.054,35.275,35.439,36.083,28.876,29.822,30.931,31.556,30.351,29.195,29.575,30.776,28.651,7.433,8.169,9.635,10.23,8.065,6.654,5.69,4.464,6.146],"y":[17.857,18.269,17.479,17.031,19.764,20.668,22.141,22.593,22.848,13.891,13.849,13.922,13.185,15.01,15.18,14.078,13.438,13.86,17.971,18.14,16.851,16.654,19.211,20.641,20.961,20.129,22.063,10.609,11.036,11.791,11.581,11.929,11.333,11.781,12.766,11.16,8.493,8.55,8.365,7.566,9.853,10.07,8.941,8.296,8.7,-8.398,-7.952,-6.592,-5.595,-7.833,-7.329,-8.411,-9.31,-8.353,0.679,0.122,-0.5,-0.233,-0.893,-0.25,-1.256,-2.419,-0.877,4.846,4.527,3.597,4.052,5.81,5.723,7.063,8.112,7.067,-19.363,-19.782,-18.738,-19.077,-20.066,-20.835,-21.359,-20.92,-22.214,-19.978,-21.041,-20.818,-21.426,-22.314,-22.679,-23.88,-24.479,-24.219,8.445,7.347,6.528,5.295,7.876,8.437,9.767,10.073,10.516,-2.696,-3.774,-4.636,-4.942,-4.659,-4.023,-5.001,-4.64,-6.129,-2.76,-2.742,-4.085,-4.139,-2.469,-2.344,-1.114,-0.09,-1.169,-15.356,-14.181,-13.195,-11.987,-14.599,-15.402,-16.87,-17.368,-17.534,-6.386,-5.331,-5.71,-4.866,-5.054,-4.727,-5.96,-6.829,-6.065,0.999,2.228,2.485,3.573,2.297,3.518,4.839,4.893,5.83,-22.105,-23.542,-24.187,-24.33,-24.103,-25.587,-25.889,-26.701,-25.324,-27.921,-26.584,-26.599,-26.017,-25.616,-25.384,-24.681,-24.711,-24.107,-21.671,-21.292,-19.818,-18.948,-21.495,-21.389,-22.119,-21.617,-23.201,-8.48,-7.661,-7.071,-5.89,-8.501,-7.738,-8.62,-9.661,-8.261,1.454,2.53,2.802,3.862,2.394,2.466,2.15,2.002,2.054,-25.214,-25.664,-25.244,-24.79,-27.186,-27.71,-27.656,-27.551,-27.733],"z":[-2.57,-2.239,-1.069,-1.119,-1.921,-3.099,-2.719,-2.053,-3.082,-2.181,-2.95,-2.008,-2.171,-3.951,-4.714,-5.735,-6.211,-6.076,24.35,22.913,22.126,21.072,22.356,22.819,24.207,24.792,24.712,18.744,17.385,16.759,15.591,17.378,18.062,19.515,19.765,20.402,-1.323,-2.743,-3.726,-4.659,-3.051,-4.527,-5.127,-4.389,-6.345,30.788,31.969,32.367,31.682,31.647,32.794,33.806,33.468,34.944,35.574,34.857,35.824,35.736,33.817,32.692,31.803,32.219,30.683,38.026,38.545,37.483,36.457,38.679,39.637,39.819,39.856,39.924,30.307,30.289,29.612,28.832,31.71,31.764,33.154,33.739,33.658,21.624,21.447,21.841,21.246,22.081,21.494,22.149,23.067,21.739,28.918,29.728,30.318,30.335,30.853,30.394,29.644,28.869,29.835,-2.792,-2.227,-1.262,-0.158,-3.334,-4.153,-5.142,-6.335,-4.727,1.972,2.809,3.531,4.738,1.938,2.698,3.596,3.282,4.617,-13.809,-14.019,-12.85,-13.048,-14.275,-15.556,-15.454,-14.333,-16.511,-10.265,-10.811,-10.687,-10.383,-12.28,-12.557,-12.532,-13.417,-11.625,3.123,2.874,3.77,4.338,1.398,1.012,1.307,1.298,1.552,21.62,21.781,20.398,19.848,22.709,23.03,24.014,23.675,25.129,12.793,12.517,12.358,11.411,13.616,13.634,14.891,15.929,14.844,-2.746,-3.314,-2.948,-3.607,-4.833,-5.484,-6.82,-7.818,-6.871,18.796,18.2,16.862,16.577,18.001,17.411,17.153,17.834,16.267,15.761,16.009,14.997,15.037,17.439,18.449,19.887,20.192,20.724,6.179,7.36,7.26,8.241,7.522,7.829,6.634,6.867,5.468],"type":"scatter3d","mode":"markers","name":"GLU","marker":{"color":"rgba(165,156,201,1)","line":{"color":"rgba(165,156,201,1)"}},"textfont":{"color":"rgba(165,156,201,1)"},"error_y":{"color":"rgba(165,156,201,1)"},"error_x":{"color":"rgba(165,156,201,1)"},"line":{"color":"rgba(165,156,201,1)"},"frame":null},{"x":[-0.14,0.87,1.819,3.007,8.868,9.942,11.136,11.708,17.753,17.694,18.903,19.028,14.451,13.062,12.247,11.121,8.491,8.254,7.697,6.828,8.199,7.729,6.291,5.467,2.73,3.17,4.523,5.256,0.284,0.229,1.482,2.012,-7.876,-8.279,-9.318,-9.109,-18.627,-19.968,-21.062,-22.111,-22.486,-23.504,-23.018,-22.294,-11.83,-10.583,-9.944,-8.718,-5.012,-4.271,-2.771,-2.019,2.163,2.727,4.243,4.905,11.716,12.137,12.637,12.394,1.468,1.34,2.228,2.213,0.203,-1.233,-1.853,-2.816,-1.97,-2.836,-2.78,-2.708,-2.627,-3.114,-4.481,-4.835,-7.457,-6.329,-6.798,-6.246,-7.75,-8.729,-8.708,-8.83,-8.518,-8.449,-7.209,-7.232,-10.631,-10.724,-11.029,-11.748,0.631,-0.464,-1.779,-2.79,25.56,25.906,25.687,24.943,28.893,29.291,30.639,30.932,31.279,31.14,29.732,28.849,28.759,27.651,28.19,27.594,29.673,29.469,28.401,28.329,13.405,12.089,11.114,10.957,6.839,6.474,7.245,7.683,21.666,21.187,21.282,21.044],"y":[24.083,24.707,25.682,25.734,19.408,18.829,18.422,17.343,17.246,18.678,19.509,20.655,19.634,19.588,20.59,20.302,21.738,22.659,21.837,22.291,20.61,19.715,19.325,19.273,17.89,18.827,18.553,19.491,8.945,9.526,10.342,10.337,2.036,0.844,1.172,0.878,-8.005,-7.871,-7.853,-7.245,-4.387,-3.5,-2.415,-2.68,-2.932,-2.186,-2.308,-2.437,-7.108,-8.268,-8.048,-9.01,13.518,14.696,14.777,13.838,10.477,9.463,8.23,7.104,-13.482,-14.112,-15.327,-15.981,-15.559,-15.641,-17.014,-17.366,-19.185,-18.906,-19.979,-19.68,-26.77,-27.532,-28.176,-29.025,-19.578,-19.142,-18.58,-17.596,-14.049,-13.418,-11.931,-11.086,-11.612,-10.227,-9.57,-8.394,0.517,-0.854,-1.894,-2.864,-1.283,-0.451,-1.181,-0.581,-14.815,-13.602,-12.377,-11.464,4.205,5.32,5.121,5.743,2.224,2.451,2.676,3.125,-9.029,-9.945,-11.273,-11.911,-14.462,-13.999,-14.763,-14.649,-23.838,-24.099,-24.81,-24.424,-24.528,-23.603,-23.922,-23.017,-6.369,-5.08,-4.868,-3.761],"z":[17.137,16.292,16.968,16.634,18.452,17.661,18.496,18.311,23.658,23.907,23.513,23.945,15.268,15.685,14.892,14.469,10.835,9.74,8.59,7.84,8.46,7.418,7.696,6.784,-2.312,-1.295,-0.668,-0.347,6.173,4.844,4.581,3.469,32.453,31.724,30.666,29.486,19.866,19.325,20.378,20.171,22.295,21.768,20.827,19.866,29.91,29.94,31.313,31.439,34.654,34.192,34.083,33.931,31.725,31.082,31.03,30.583,28.011,27.06,27.794,27.366,23.149,21.847,21.684,20.635,-14.226,-14.002,-14.195,-13.512,-4.922,-3.787,-2.72,-1.522,2.39,1.252,1.374,0.556,-1.327,-2.131,-3.456,-3.959,-6.831,-7.698,-7.374,-8.263,-6.094,-5.664,-6.239,-6.613,-7.315,-7.778,-6.718,-6.982,-2.122,-2.591,-2.405,-2.041,-2.229,-1.507,-2.376,-1.996,-0.785,-1.629,-2.305,-3.327,-6.047,-7.473,-7.987,-7.255,-0.774,-0.58,-0.073,0.802,6.737,8.098,8.867,10.091,-4.762,-5.322,-4.404,-3.246,3.588,4.645,5.913,6.626,16.055,16.513,18.012,18.488],"type":"scatter3d","mode":"markers","name":"GLY","marker":{"color":"rgba(194,150,199,1)","line":{"color":"rgba(194,150,199,1)"}},"textfont":{"color":"rgba(194,150,199,1)"},"error_y":{"color":"rgba(194,150,199,1)"},"error_x":{"color":"rgba(194,150,199,1)"},"line":{"color":"rgba(194,150,199,1)"},"frame":null},{"x":[14.195,16.512,11.712,11.959,16.733,14.121,15.185,14.725,13.378,13.007,12.445,15.588,16.032,16.944,17.803,16.806,12.237,11.337,9.964,10.041,11.46,8.685,8.986,7.904,14.354,13.293,13.756,15.121,15.46,12.859,16.045,16.023,16.174,17.075,18.446,18.373,16.977,19.72,19.596,19.853,21.324,21.632,22.174,-7.642,-10.132,-5.966,-5.486,-9.647,-8.008,-9.051,-8.884,-7.719,-7.181,-7.118,-9.848,-9.587,-10.498,-10.048,-11.66,-5.997,-5.382,-4.122,-3.999,-5.179,-3.125,-2.941,-2.331,-7.566,-6.59,-6.84,-7.98,-8.431,-5.987,-8.602,-8.664,-9.485,-10.162,-11.497,-11.633,-10.375,-12.545,-12.916,-13.816,-15.185,-16.168,-15.28,-9.209,-12.249,-9.107,-5.864,-8.926,-10.378,-11.652,-12.28,-11.412,-10.233,-11.633,-13.688,-14.674,-16.123,-16.962,-16.431,-7.789,-7.962,-6.788,-5.868,-6.495,-6.659,-4.603,-3.599,-7.7,-6.404,-5.674,-6.535,-7.78,-4.207,-6.335,-5.184,-10.362,-10.136,-11.301,-12.217,-11.655,-11.484,-13.477,-13.126,-14.248,-15.093,-14.262,27.49,30.857,28.278,24.273,26.821,29.211,30.505,31.441,30.732,29.339,31.313,32.951,33.317,34.808,35.342,35.441,26.507,26.948,25.842,24.706,25.136,25.964,23.373,22.856,25.887,24.605,23.648,24.351,25.751,22.152,23.866,22.881,28.603,28.156,29.261,30.383,29.982,29.201,31.797,32.205,33.613,34.458,33.875],"y":[14.549,16.868,16.877,12.475,12.479,16.544,17.301,18.576,18.566,17.297,19.714,19.768,19.866,21.059,20.963,22.095,14.644,15.646,15.264,14.028,13.654,16.03,13.203,13.552,12.755,12.059,10.861,10.86,12.049,9.818,9.901,9.208,14.647,13.676,14.086,15.309,15.662,13.276,16.125,17.059,17.221,17.661,16.904,-9.834,-11.501,-9.6,-7.536,-9.44,-10.352,-11.145,-11.511,-10.96,-10.242,-11.029,-12.344,-13.826,-14.709,-15.2,-14.914,-8.915,-9.005,-8.28,-7.688,-8.065,-8.27,-6.931,-5.788,-8.733,-7.816,-7.248,-7.847,-8.757,-6.164,-7.688,-6.459,-10.355,-10.153,-10.744,-11.288,-11.093,-10.796,-11.893,-10.869,-11.43,-10.66,-12.635,-11.782,-13.236,-11.073,-10.994,-13.274,-12.145,-12.698,-12.621,-11.998,-11.705,-11.675,-13.079,-11.944,-12.359,-11.464,-13.566,-11.142,-10.798,-10.156,-10.155,-10.775,-9.515,-9.574,-9.9,-12.098,-11.61,-11.888,-12.531,-12.665,-11.545,-12.934,-13.639,-12.961,-13.399,-14.094,-14.157,-13.395,-14.613,-15.029,-16.521,-17.419,-16.951,-18.615,-6.085,-5.444,-9.336,-6.957,-3.093,-7.209,-6.733,-7.782,-8.887,-8.519,-10.23,-7.679,-7.224,-7.233,-8.303,-6.173,-7.761,-8.985,-9.903,-9.23,-7.914,-11.285,-9.639,-10.921,-5.182,-5.706,-4.778,-3.649,-3.933,-5.045,-2.433,-2.269,-4.563,-3.376,-2.506,-3.187,-4.441,-1.093,-2.753,-3.098,-2.641,-3.508,-1.419],"z":[7.996,7.141,7.515,9.512,9.2,7.567,7.118,6.602,6.683,7.277,6.298,6.209,4.754,4.492,3.591,5.181,8.349,8.021,8.315,8.895,8.919,7.921,9.279,10.178,9.121,9.642,10.348,10.248,9.486,11.022,10.679,11.952,8.154,8.595,8.382,7.832,7.663,8.645,7.506,8.668,8.979,10.1,8.121,26.751,28.295,29.768,25.447,23.923,28.672,29.096,30.486,30.914,29.777,32.318,31.319,31.105,31.937,32.996,31.526,27.434,28.665,28.705,27.481,26.711,29.881,26.972,27.627,24.984,24.649,23.34,22.852,23.905,22.681,21.607,20.816,26.198,25.001,25.057,26.295,26.983,23.948,26.844,27.523,27.881,27.841,28.201,2.428,1.993,-0.999,2.548,5.507,0.79,0.84,-0.467,-1.3,-0.508,-2.78,-0.828,-0.583,-0.779,-1.027,-0.662,1.037,-0.287,-0.838,0.155,1.316,-2.227,0.123,-0.845,3.771,3.684,4.908,5.749,5.024,5.153,7.069,7.583,3.538,4.835,5.37,4.365,3.241,6.798,4.426,4.408,4.926,5.72,4.569,7.296,6.965,8.154,7.001,5.696,7.435,7.365,7.738,8.045,7.861,8.497,7.838,9.24,9.481,9.856,9.289,7.631,8.093,8.36,7.978,7.529,9.031,8.045,7.588,6.497,6.5,5.926,5.6,5.948,5.726,5.096,4.035,6.491,5.922,5.557,5.886,6.514,4.957,5.549,4.122,3.795,3.488,3.846],"type":"scatter3d","mode":"markers","name":"HEM","marker":{"color":"rgba(220,142,196,1)","line":{"color":"rgba(220,142,196,1)"}},"textfont":{"color":"rgba(220,142,196,1)"},"error_y":{"color":"rgba(220,142,196,1)"},"error_x":{"color":"rgba(220,142,196,1)"},"line":{"color":"rgba(220,142,196,1)"},"frame":null},{"x":[27.13,27.919,27.862,28.566,27.505,26.058,25.035,25.466,23.876,24.111,16.907,16.294,14.804,13.993,16.998,16.737,17.275,15.948,16.824,16.016,15.191,15.308,15.635,16.252,14.047,13.907,13.537,14.162,13.588,13.963,5.387,4.957,5.427,4.684,5.516,5.23,3.95,6.057,4,5.268,-6.625,-5.579,-5.683,-4.701,-5.645,-4.553,-4.358,-3.694,-3.4,-2.965,-14.253,-13.187,-11.83,-10.795,-13.521,-12.564,-12.288,-11.756,-11.345,-11.005,-6.407,-6.626,-6.544,-7.28,-5.624,-5.971,-5.719,-6.59,-6.185,-6.724,-9.684,-9.273,-8.931,-9.381,-10.368,-10.776,-9.92,-11.941,-10.537,-11.765,-13.555,-14.882,-14.845,-15.883,-15.747,-15.089,-14.079,-15.33,-13.727,-14.471,-9.85,-9.54,-8.39,-8.348,-9.201,-9.181,-10.312,-8.16,-9.989,-8.689,-13.304,-12.254,-12.3,-11.886,-10.876,-10.603,-11.182,-9.849,-10.78,-9.969,4.427,5.753,5.67,6.533,6.284,7.59,8.746,7.919,9.73,9.254,10.726,10.556,10.204,9.199,11.848,11.776,12.786,10.854,12.465,11.306,31.23,30.646,29.477,29.222,30.223,29.845,30.68,28.694,30.058,28.852,29.059,28.037,27.461,27.166,26.911,27.268,28.017,26.984,28.164,27.553,28.031,26.81,25.682,25.055,27.106,28.233,28.18,29.463,29.328,30.124],"y":[12.397,13.524,14.76,15.741,13.885,14.251,13.341,15.425,13.939,15.203,20.493,19.434,19.322,18.96,18.095,17.099,17.224,15.999,16.251,15.494,13.13,11.908,10.666,9.73,11.696,12.686,14.001,12.551,14.612,13.774,7.775,7.286,8.258,8.555,5.885,5.39,5.165,5.115,4.777,4.738,11.482,10.465,9.632,9.517,9.572,8.547,7.834,8.098,6.968,7.088,-4.145,-4.952,-4.25,-4.902,-5.424,-6.451,-7.611,-6.459,-8.284,-7.607,-16.607,-15.651,-16.308,-15.938,-14.493,-13.471,-13.618,-12.27,-12.532,-11.682,-16.466,-16.314,-14.864,-14.317,-16.838,-18.249,-19.318,-18.764,-20.43,-20.121,-20.385,-20.911,-22.241,-22.852,-19.868,-19.247,-18.313,-19.394,-17.91,-18.551,-20.662,-19.308,-18.72,-17.509,-19.266,-17.88,-17.099,-17.106,-15.901,-15.878,-6.481,-6.675,-5.718,-6.071,-6.638,-7.831,-8.043,-8.933,-9.239,-9.82,-10.984,-11.591,-13.028,-13.481,-11.589,-12.314,-11.926,-13.407,-12.746,-13.653,-7.781,-7.833,-6.455,-6.316,-8.395,-8.653,-9.325,-8.245,-9.307,-8.666,-7.892,-7.683,-8.621,-8.956,-6.237,-5.955,-6.258,-5.5,-6.011,-5.549,-2.598,-2.009,-0.726,0.221,-3.015,-4.066,-5.185,-4.143,-5.885,-5.292,2.628,3.42,2.943,3.737,4.904,5.428,5.476,5.878,5.93,6.18],"z":[11.782,11.288,12.193,11.958,9.852,9.703,9.867,9.382,9.654,9.357,14.48,13.693,14.024,13.164,13.921,12.836,11.575,12.8,10.807,11.525,1.32,2.105,1.288,1.798,2.952,4.07,3.896,5.396,5.089,6.036,16.011,17.312,18.397,19.335,17.592,18.978,19.442,20.016,20.704,21.076,17.652,17.565,16.282,15.543,18.82,18.922,20.08,17.972,19.812,18.55,30.634,30.067,30.091,30.264,28.653,28.138,28.828,27.052,28.195,27.114,24.932,23.849,22.474,21.557,23.935,24.974,26.319,24.846,26.95,26.096,20.269,18.871,18.534,17.522,17.933,18.219,18.064,18.679,18.417,18.793,12.242,11.944,11.179,10.948,11.21,10.012,10.115,8.687,8.905,8.022,-0.535,-0.102,-0.923,-1.176,1.391,1.952,2.028,2.389,2.476,2.703,4.174,5.169,6.348,7.454,4.503,3.643,2.412,3.887,1.964,2.825,1.953,2.038,1.538,0.787,3.477,3.64,2.994,4.37,3.319,4.153,-8.517,-7.067,-6.483,-5.782,-6.439,-4.96,-4.31,-4.05,-3.03,-2.821,-1.341,-0.03,0.266,1.428,0.185,1.601,2.654,2.15,3.791,3.514,13.258,12.396,12.983,12.258,12.13,11.126,11.409,9.801,10.277,9.27,11.243,11.416,10.498,9.795,11.18,12.013,13.39,11.669,13.86,12.836],"type":"scatter3d","mode":"markers","name":"HIS","marker":{"color":"rgba(223,155,177,1)","line":{"color":"rgba(223,155,177,1)"}},"textfont":{"color":"rgba(223,155,177,1)"},"error_y":{"color":"rgba(223,155,177,1)"},"error_x":{"color":"rgba(223,155,177,1)"},"line":{"color":"rgba(223,155,177,1)"},"frame":null},{"x":[8.749,6.924,-8.196,-4.944,6.966,5.946,8.226,15.186,19.441,20.397,-1.319,-9.798,-0.234,0.628,-0.281,-4.075,7.453,-2.573,11.648,25.484,-11.385,10.286,-0.933,25.563,11.143,-4.831,3.508,-15.423,6.427,9.964,-4.274,-11.112,13.134,23.007,-6.252,-8.772,-2.984,20.646,-11.402,-4.638,19.363,-11.908,22.464,6.951,-12.511,10.066,19.149,9.707,19.383,-0.343,14.374,-3.928,14.664,-3.288,9.286,-12.685,3.921,6.7,-0.511,-3.425,9.441,1.611,13.277,-4.18,-12.983,9.626,23.775,7.178,-13.781,4.392,13.602,-12.573,-0.992,8.675,24.247,0.439,15.895,-9.914,-13.199,13.032,-2.526,9.727,-11.788,-12.808,7.239,-14.388,-1.403,3.195,4.749,6.58,-10.009,7.616,3.892,11.89,-11.722,-11.025,-4.957,23.378,6.206,-4.689,-4.758,-2.413,-2.627,0.364,3.538,-7.811,1.748,12.022,-2.835,10.063,17.457,5.159,0.895,14.65,-6.669,1.939,-8.093,8.871,-1.75,4.34,10.606,5.668,6.71,17.927,10.133,-8.986,5.099,19.113,20.376,-11.577,5.866,17.642,-12.778,-1.277,11.129,9.864,-0.46,-11.781,17.446,13.847,-27.06,1.609,-9.967,-5.248,13.455,16.573,-18.657,14.042,-10.035,-18.092,-15.524,1.024,2.268,-1.17,-22.492,-17.735,-13.529,2.252,-12.965,-0.991,-11.111,-13.99,-23.299,8.247,17.209,3.137,5.896,-5.966,1.652,-5.905,6.095,-14.987,-9.439,4.903,3.05,14.901,-7.638,-5.132,19.797,-8.271,-1.431,1.927,-8.215,-5.173,11.651,-8.916,3.59,-4.072,-2.822,-4.365,-11.668,-6.014,-0.887,2.758,-9.728,-1.541,7.404,12.674,1.196,9.124,3.281,0.175,8.775,-6.593,-2.874,10.798,-11.116,-6.159,2.029,-0.261,-5.804,-2.83,2.83,-3.467,-2.577,-10.66,-8.123,12.958,-1.241,1.67,-13.066,-13.412,-10.599,-0.244,15.866,6.593,-6.377,-11.283,-12.669,-14.439,-12.588,8.018,-2.136,2.521,1.048,-12.865,9.171,16.441,10.814,-5.878,-10.182,-6.155,-12.134,-3.201,-8.2,3.293,8.301,-13.33,5.108,5.242,-24.431,-5.809,-12.409,13.709,7.015,-4.861,5.495,2.992,-13.372,9.546,2.841,-27.539,8.085,-11.938,-13.807,-14.589,-0.125,-12.071,8.988,-2.283,10.436,0.872,-2.937,-7.949,0.314,-0.084,6.671,-1.159,-8.868,4.7,-9.713,-5.9,6.494,-10.541,-15.412,-12.482,2.13,21.56,12.623,25.297,26.16,16.927,28.164,22.265,24.798,25.427,9.776,24.716,11.849,30.948,32.726,19.035,31.064,28.021,23.926,25.053,27.213,31.459,23.614,18.728,18.307,14.007,17.547,20.017,14.926,14.798,26.033,10.588,25.138,23.278,21.822,15.345,18.273,11.242,9.771,26.3,3.502,28.823,34.246,21.601,8.954,24.253,16.242,10.452,14.64,9.005,33.897,27,15.709,9.703,12.306,31.186,27.485,8.659,28.933,8.587,13.113,17.886,33.316,32.183,8.835,10.192,10.896,18.402,28.148,22.599,15.215,32.835,12.216,16.863,18.039,34.714,13.862,29.318,16.664,21.295,34.601,19.248,32.986],"y":[4.525,5.469,18.529,28.564,5.303,0.762,2.513,1.861,5.706,7.683,26.428,19.923,19.631,9.514,15.652,20.96,25.754,20.889,-0.471,8.575,8.778,6.424,6.049,10.733,22.551,13.718,5.95,12.63,13.036,26.57,11.8,20.887,22.676,3.61,12.279,17.126,6.872,20.073,11.856,19.642,18.995,15.366,11.413,27.003,18.869,1.47,24.951,1.082,3.384,13.644,1.777,23.07,0.157,9.196,2.515,14.125,8.173,15.723,3.256,26.716,24.719,4.99,-0.17,24.157,9.138,21.749,13.359,4.648,9.843,5.038,24.674,9.278,12.815,24.171,19.797,3.777,-2.116,20.575,15.862,0.834,13.267,22.009,7.506,23.031,3.078,12.568,7.66,5.15,22.246,10.553,14.974,19.747,5.037,-0.266,18.572,15.607,11.051,16.457,24.851,7.392,9.511,-21.793,-9.479,-19.29,-17.136,-19.693,-8.888,0.563,1.239,-11.808,-4.813,-12.362,-3.561,1.251,7.929,-15.206,-8.902,-16.814,-16.724,-3.605,0.742,-5.206,-7.571,1.837,11.614,9.425,-13.975,1.212,-1.985,-14.161,9.001,-7.585,4.909,-10.385,-5.385,-14.327,3.395,6.852,1.972,12.201,-22.693,-10.596,-7.907,-19.759,9.988,-9.055,-10.344,-11.29,7.103,-6.636,3.153,-3.269,-11.008,5.001,2.166,-10.271,-5.196,-24.058,-15.404,-15.709,-5.417,7.123,1.065,-3.398,4.274,-12.382,-9.627,-9.999,-5.886,-21.029,-1.689,5.059,-11.491,-12.899,-1.745,3.799,8.196,-6.492,2.263,3.639,-3.258,-1.007,1.367,-1.07,-19.501,5.723,1.122,-3.557,-1.778,-21.204,-13.338,0.108,-3.595,-23.965,1.243,3.292,-19.731,-7.795,-10.982,-9.383,1.241,-24.106,-10.449,-18.234,2.134,-2.49,-26.294,3.422,-15.983,-13.61,3.429,-19.677,-7.788,4.78,-23.084,-28.01,-8.13,-22.435,2.49,1.821,2.366,-4.653,0.753,3.209,-15.296,-7.114,-9.592,-27.807,-20.429,-24.319,-30.116,5.181,-13.589,1.53,-0.932,-14.349,-20.326,-13.955,-3.389,-22.767,-23.575,-25.072,-10.613,1.148,-21.166,-5.556,-7.905,-3.123,-13.947,-7.441,-16.736,-25.757,-21.383,-14.762,-7.74,0.777,6.8,-9.212,-16.782,3.739,-24.704,-20.541,5.641,-1.818,-20.8,-6.964,-16.744,4.866,-2.052,-10.958,-6.092,2.474,0.083,-26.26,-26.106,-25.077,-16.074,-9.114,-16.802,-5.538,-15.884,-12.212,-23.576,-11.204,-14.616,-18.696,-0.819,-18.41,-31.775,3.958,1.367,-35.195,-23.355,-16.524,-15.984,5.948,-18.5,-30.733,-29.92,-10.726,-12.782,-1.842,-16.609,-26.508,-30.471,-9.318,-18.836,-11.251,-21.927,-17.924,-6.424,-31.438,-29.535,-10.293,-25.857,-10.211,5.875,-23.584,-14.041,-26.184,-16.071,0.392,-5.884,-2.758,-29.519,-7.759,-24.668,-23.258,-11.501,-17.386,-27.612,-25.012,-13.007,-27.917,-11.298,-15.962,-13.417,5.285,-27.953,-25.252,-30.431,3.072,-23.096,-29.799,-27.836,-30.397,-13.553,-27.29,-7.08,-13.869,-5.395,-30.55,-8.116,-12.587,-25.832,-19.969,-7.801,-14.407,-13.553,-27.746,-24.628,-10.858,-9.802,3.412,-2.487,3.681,-13.633,-21.212,-18.97],"z":[-3.343,-5.372,18.621,15.463,4.344,-1.292,-1.588,11.755,24.91,23.286,14.271,2.245,-2.112,1.275,30.388,1.056,21.064,-1.395,6.383,15.637,10.1,-6.873,4.462,9.272,7.756,1.375,11.048,5.938,-2.582,22.319,24.867,7.034,18.681,18.07,26.711,16.204,6.239,13.42,13.545,27.492,11.313,18.928,7.57,18.675,5.995,-9.208,18.231,12.465,5.695,-6.019,24.491,26.298,13.806,3.451,1.02,14.349,-1.762,27.415,14.169,12.042,12.363,17.901,18.555,11.391,7.989,0.815,6.157,7.1,22.678,8.394,20.387,12.681,-2.046,14.856,19.432,5.053,14.009,22.725,12.327,21.241,0.05,5.413,5.71,7.128,2.739,2.797,2.303,5.825,-1.092,8.942,25.744,-2.186,13.863,13.551,3.472,16.377,1.78,19.674,7.064,36.722,30.624,20.143,37.151,32.988,27.389,16.375,19.03,25.072,40.712,32.235,38.362,36.176,13.902,29.357,31.853,18.051,35.318,32.4,32.337,22.607,22.745,20.915,21.782,32.67,35.28,31.194,20.976,37.051,37.823,16.084,34.338,36.89,14.466,12.898,25.496,31.233,45.005,31.859,42.459,26.102,17.248,16.418,37.003,17.326,33.65,34.882,21.729,32.476,25.617,16.727,27.604,40.12,20.624,17.948,23.705,18.446,35.374,14.855,28.755,35.645,36.657,22.722,28.641,39.348,31.726,39.67,20.054,36.678,40.951,29.067,23.704,16.399,35.407,41.695,21.191,30.258,26.431,40.059,34.621,6.927,12.376,-18.543,-13.163,-16.676,-13.737,4.944,-2.622,-16.106,5.067,-11.851,-5.808,-14.118,5.553,12.192,-10.621,-9.003,12.298,-14.853,13.721,1.823,-17.821,11.215,-15.198,-8.683,-2.389,-11.894,-2.7,1.101,-20.799,-20.628,8.014,16.194,10.086,7.961,-10.801,4.119,14.324,-7.812,-16.871,-0.684,-0.789,-4.704,-1.28,-2.157,-8.964,-11.078,-14.068,8.11,-4.201,14.447,7.329,-14.751,-16.766,1.786,2.458,10.12,-16.039,-13.746,-14.38,-4.674,13.046,-5.927,-6.368,10.832,13.911,2.205,-14.385,-9.316,13.079,3.213,7.349,13.014,1.74,-7.371,1.213,8.746,-8.797,12.372,12.249,-10.43,1.381,15.734,-8.74,10.859,16.72,-5.554,-17.527,6.254,-16.154,-18.531,-14.647,-4.633,7.007,6.75,0.014,-4.701,11.998,-16.595,-8.658,-0.271,-19.378,-14.511,-4.049,-10.57,9.222,2.182,-21.468,19.869,15.143,16.689,19.853,13.355,1.421,21.639,22.947,8.242,11.569,5.081,1.696,16.1,6.238,17.657,9.858,18.044,2.447,19.868,17.208,19.443,-4.447,-6.033,18.846,18.638,18.884,-7.459,0.998,14.641,14.846,12.553,-8.246,12.182,-8.429,-7.26,14.676,7.911,5.169,-7.616,3.931,9.779,-0.157,-6.092,11.966,19.048,-10.474,14.202,10.078,10.637,1.81,18.462,-4.542,10.727,4.352,0.933,7.352,10.475,1.274,7.74,9.627,21.313,1.973,15.86,1.704,14.847,-0.049,-6.946,10.476,-6.205,15.768,8.487,5.556,1.003,21.624,7.488,12.144,6.49,16.592,24.669,4.422,21.683,6.081],"type":"scatter3d","mode":"markers","name":"HOH","marker":{"color":"rgba(205,180,144,1)","line":{"color":"rgba(205,180,144,1)"}},"textfont":{"color":"rgba(205,180,144,1)"},"error_y":{"color":"rgba(205,180,144,1)"},"error_x":{"color":"rgba(205,180,144,1)"},"line":{"color":"rgba(205,180,144,1)"},"frame":null},{"x":[-2.349,-0.968,0.169,1.311,-0.72,-0.634,-1.852,-0.307,7.537,7.1,8.219,8.514,5.818,4.642,5.472,3.352,15.834,17.212,17.472,18.425,18.236,18.016,18.143,19.114,22.3,23.252,23.106,24.077,23.172,21.85,23.372,21.94,19.57,18.251,17.139,16.095,18.144,18.946,16.689,18.367,5.989,4.639,3.676,2.562,4.608,5.483,3.171,4.987,-2.292,-2.986,-3.851,-4.055,-3.797,-4.205,-5.005,-4.748,8.729,9.25,9.009,8.398,10.762,10.961,11.401,12.411,6.037,5.278,3.92,3.492,5.092,6.439,4.058,6.389,2.597,1.334,0.409,-0.282,0.645,1.526,-0.742,1.063,8.435,7.227,6.335,5.122,7.567,8.327,6.296,7.625,4.743,3.658,2.38,1.332,4.075,5.082,2.849,5.663,-18.679,-17.626,-17.255,-17.145,-16.371,-16.708,-15.22,-15.594,-8.36,-7.072,-5.96,-4.902,-7.164,-7.921,-5.774,-8.212,7.552,6.382,5.884,4.672,6.623,6.859,5.428,7.242,0.04,-0.532,-1.333,-1.22,-1.448,-0.613,-2.211,0.491,2.944,2.541,1.047,0.671,3.247,2.582,4.704,3.157,-0.479,0.162,-0.648,-0.086,0.33,1.306,0.858,1.428,-0.277,-0.758,0.279,0.166,-2.052,-1.803,-3.186,-2.928,-6.416,-7.586,-8.706,-9.886,-7.219,-6.769,-6.144,-7.882,-8.286,-7.163,-7.043,-6.757,-5.831,-5.808,-4.624,-5.802,-6.119,-4.86,-5.029,-4.568,-3.69,-3.418,-2.427,-3.007,-4.929,-3.661,-3.496,-2.749,-2.477,-1.172,-2.356,-0.059,2.109,2.541,2.245,3.141,1.832,2.355,1.998,1.589,3.526,3.272,4.598,4.856,2.256,0.857,2.212,-0.193,6.189,6.55,8.035,8.409,6.148,4.62,6.669,4.093,18.773,19.715,21.043,21.593,19.961,18.649,21.012,17.93,20.528,20.47,21.846,22.312,19.501,18.052,19.731,17.042,24.329,24.531,24.914,25.81,23.26,22.976,23.444,21.681,29.12,27.945,27.491,27.252,26.788,27.15,25.479,26.146,24.361,24.352,24.509,25.207,23.096,23.108,23.051,21.863,14.189,13.698,14.707,14.327,13.447,12.302,13.165,10.999],"y":[23.179,22.886,23.54,23.561,21.355,20.789,20.659,19.306,20.73,19.399,18.686,17.52,19.471,20.009,18.087,20.207,9.445,8.995,8.262,7.488,10.138,11.25,10.678,12.296,6.74,6.556,7.624,7.979,5.128,4.916,4.062,5.192,18.934,18.311,19.365,19.146,17.481,16.183,17.162,15.192,19.055,18.69,19.861,19.656,18.289,17.049,18.04,15.792,19.167,17.939,17.303,16.087,18.147,16.802,19.037,16.917,5.223,5.46,6.909,7.164,5.121,3.601,5.729,3.173,11.444,12.685,12.541,13.447,13.088,13.52,14.211,13.832,14.862,15.589,15.169,16.004,15.42,16.059,16.064,15.81,1.128,0.341,0.312,0.527,-1.108,-1.08,-1.952,-0.3,3.879,4.125,3.377,3.995,3.735,4.749,3.634,4.371,4.41,3.587,4.114,3.341,3.536,2.805,2.838,2.783,-5.743,-5.791,-5.141,-5.739,-5.13,-6.055,-4.803,-5.442,-13.177,-12.768,-13.819,-14.026,-11.39,-10.331,-10.981,-8.965,5.881,7.194,7.697,8.862,7.149,6.844,8.459,7.843,-15.31,-16.196,-16.386,-17.288,-15.784,-14.549,-15.465,-14.151,-20.617,-19.542,-19.247,-19.135,-18.249,-18.511,-17.091,-17.36,-19.702,-20.369,-21.233,-21.461,-21.183,-22.277,-20.244,-23.275,-17.481,-17.015,-16.513,-16.603,-15.929,-14.624,-16.455,-13.601,-24.1,-23.217,-23.008,-21.901,-23.773,-23.606,-23.074,-22.153,-10.33,-9.837,-9.619,-8.615,-10.828,-10.928,-10.397,-9.619,-1.625,-1.091,0.437,1.009,-1.814,-1.593,-1.368,-2.53,-9.735,-11.122,-11.889,-12.51,-11.81,-11.24,-13.323,-11.717,-14.275,-14.279,-14.126,-14.853,-13.167,-13.518,-12.998,-12.43,-16.928,-16.543,-16.785,-17.213,-15.074,-14.957,-14.624,-13.546,-25.757,-24.65,-25.146,-24.541,-23.901,-23.3,-22.805,-22.422,-26.168,-24.988,-24.367,-24.293,-23.92,-24.378,-22.549,-23.501,-2.475,-3.136,-4.599,-5.127,-3.035,-1.565,-3.83,-1.316,-11.041,-11.137,-12.591,-13.046,-10.236,-8.761,-10.566,-7.783,-17.418,-16.392,-14.961,-14.151,-16.531,-17.91,-15.427,-18.223,-14.674,-14.697,-15.417,-16.248,-13.255,-12.588,-13.268,-13.345],"z":[18.711,19.104,18.311,18.775,19.142,17.722,19.89,17.672,20.542,20.123,19.362,19.628,19.24,20.068,18.674,19.277,19.87,20.044,21.36,21.455,19.846,20.874,18.42,20.881,14.03,12.936,11.851,11.19,12.308,11.563,13.379,10.068,17.99,18.116,18.057,17.439,19.432,19.305,19.768,18.304,8.963,9.38,9.127,8.643,10.878,11.116,11.347,10.438,3.025,3.392,2.297,2.3,4.7,5.304,4.449,6.722,13.565,14.906,15.326,16.367,14.974,14.919,16.226,14.72,18.058,17.928,18.63,19.345,16.437,15.839,16.301,14.339,21.853,21.809,22.953,23.539,20.433,19.352,20.438,17.936,36.407,36.16,37.402,37.306,35.709,34.377,35.588,33.269,37.916,36.968,37.344,37.562,35.517,34.972,34.602,33.628,22.216,22.807,24.196,25.147,21.895,20.591,22.604,19.573,31.684,30.993,31.819,32.013,29.589,28.632,29.045,27.273,33.31,32.542,31.538,31.403,31.869,32.955,31.015,32.424,25.111,24.821,26.029,26.418,23.553,22.301,23.379,22.031,-14.996,-13.902,-13.617,-12.86,-12.585,-11.972,-12.86,-10.618,-6.8,-6.049,-4.783,-3.694,-6.913,-8.069,-6.057,-9.067,7.16,8.364,9.074,10.278,8.109,7.069,7.674,6.957,13.459,14.201,13.282,13.625,15.263,14.598,16.212,14.439,2.336,2.03,0.518,0.062,2.606,4.13,1.973,4.606,-6.302,-6.86,-8.369,-8.914,-6.59,-5.082,-7.344,-4.43,-11.535,-11.045,-11.092,-10.295,-11.742,-10.978,-13.191,-11.415,6.114,6.238,4.948,4.369,7.427,8.75,7.348,9.966,-0.515,-1.953,-2.712,-3.676,-2.36,-1.83,-3.876,-2.046,-5.361,-6.72,-6.992,-8.087,-7.005,-6.951,-8.371,-7.033,8.707,8.547,7.972,7.044,9.887,10.401,9.704,9.387,4.17,3.311,3.075,1.937,3.902,3.728,3.259,4.444,-10.64,-9.354,-9.579,-8.912,-8.47,-8.145,-7.185,-7.412,3.619,4.479,4.64,5.76,3.958,4.167,4.663,3.573,19.321,18.288,18.808,18.192,17.401,16.728,16.35,15.932,-0.116,-1.492,-2.393,-3.216,-2.03,-1.257,-3.537,-1.295],"type":"scatter3d","mode":"markers","name":"ILE","marker":{"color":"rgba(182,204,108,1)","line":{"color":"rgba(182,204,108,1)"}},"textfont":{"color":"rgba(182,204,108,1)"},"error_y":{"color":"rgba(182,204,108,1)"},"error_x":{"color":"rgba(182,204,108,1)"},"line":{"color":"rgba(182,204,108,1)"},"frame":null},{"x":[-7.106,-7.723,-9.093,-9.24,-6.802,-5.382,-4.542,-5.436,-6.676,-5.665,-4.45,-3.953,-5.234,-4.112,-4.585,-3.665,12.133,13.431,14.277,15.047,14.173,13.549,14.151,13.767,3.948,3.772,2.767,1.887,5.111,5.088,4.548,6.489,-0.581,0.699,1.85,2.969,0.563,-0.37,-0.592,0.212,4.872,6.14,7.217,8.217,5.899,4.876,4.76,5.269,10.892,11.326,11.609,12.594,10.27,10.233,10.18,9.034,19.809,19.789,19.627,20.236,18.694,18.719,17.49,20.012,9.437,9.263,7.787,7.425,9.927,9.901,10.345,10.799,3.259,1.966,2.117,1.293,1.36,1.013,0.34,0.1,-4.891,-3.486,-2.615,-1.656,-3.015,-1.625,-1.652,-1.191,-0.155,1.283,1.721,2.83,1.712,1.592,1.95,2.491,4.402,4.301,4.687,5.325,2.887,2.687,3.607,1.235,6.766,8.211,8.939,10.062,8.649,8.201,8.665,8.761,-1.524,-2.632,-3.944,-3.988,-2.305,-1.174,-1.039,-1.442,-8.784,-9.543,-10.405,-10.456,-10.404,-11.162,-10.201,-11.944,-10.216,-11.386,-11.671,-12.249,-12.614,-12.587,-13.86,-12.427,-20.831,-19.425,-19.265,-18.17,-18.517,-18.556,-17.569,-18.227,-6.211,-5.209,-4.881,-3.715,-5.686,-4.609,-3.523,-5.241,-2.334,-0.913,-0.19,1.042,-0.729,-1.144,-1.049,-0.267,-2.891,-3.663,-3.425,-4.331,-3.285,-4.307,-5.669,-3.798,-5.486,-6.881,-7.221,-8.209,-7.184,-8.654,-9.442,-8.764,-10.598,-11.021,-10.692,-11.328,-10.434,-10.681,-9.877,-12.167,-1.072,-1.881,-2.447,-2.448,-3.002,-2.497,-3.666,-1.529,-2.893,-3.437,-2.312,-2.506,-4.032,-4.876,-4.015,-6.001,-2.091,-1.808,-0.695,-0.785,-1.387,-1.099,-2.306,-0.734,2.13,0.732,0.542,-0.437,-0.08,-0.476,-1.084,-1.46,2.628,3.161,3.449,2.581,2.183,1.845,0.852,3.117,4.793,4.958,3.667,3.688,5.402,5.6,6.71,5.925,-1.146,-1.935,-1.442,-1.452,-3.414,-4.12,-5.517,-4.158,-7.709,-8.524,-7.618,-7.708,-9.64,-10.549,-11.248,-11.574,-6.066,-6.329,-7.407,-8.217,-5.037,-4.022,-2.687,-4.575,-10.526,-10.729,-11.71,-11.685,-9.375,-8.289,-6.985,-8.739,-13.823,-12.938,-12.555,-12.476,-11.687,-11.674,-13.046,-10.622,-15.034,-13.928,-13.072,-12.639,-13.056,-13.824,-12.92,-14.406,1.002,0.603,1.393,1.787,-0.9,-1.475,-1.046,-2.985,5.453,6.749,7.59,8.256,7.502,6.843,7.751,6.553,6.666,5.585,5.426,4.316,5.83,4.637,3.446,5.013,2.016,1.351,1.134,0.128,2.113,2.076,2.884,0.639,-2.759,-3.264,-4.679,-5.528,-2.318,-2.619,-2.637,-1.581,-6.858,-7.752,-8.036,-9.104,-7.141,-7.061,-6.319,-8.461,24.401,24.49,25.9,26.884,24.047,22.587,22.447,21.663,22.739,23.036,22.546,21.965,24.533,24.928,24.186,26.427,19.842,20.148,19.116,18.92,21.565,22.729,24.041,22.762,26.071,24.887,23.635,22.574,24.923,26.089,25.913,26.16,27.412,27.002,27.976,27.557,26.883,25.478,25.592,24.779,27.579,26.522,26.986,26.303,25.326,24.539,23.51,23.868,30.503,30.007,28.909,28.793,29.489,29.792,31.27,29.369,30.531,31.162,30.159,30.38,31.792,32.545,33.915,32.692,30.579,29.884,28.608,28.14,29.568,30.764,30.312,31.812,15.744,15.503,16.43,15.998,15.683,14.761,15.079,13.296,17.699,18.682,18.253,18.293,20.078,21.367,21.436,21.494,17.46,18.136,17.088,17.276,18.976,19.782,20.716,20.578,22.602,23.336,22.9,23.674,23.23,24.136,23.774,25.599],"y":[10.251,11.142,11.627,12.144,12.337,11.968,13.221,11.127,20.263,19.935,20.835,21.422,18.463,18.039,18.157,16.616,14.358,13.847,13.399,12.441,14.883,15.131,16.37,13.944,22.126,22.15,21.071,21.314,21.909,21.793,23.075,21.491,15.271,15.094,14.714,15.201,14.066,14.484,13.33,15.693,17.28,16.913,16.433,15.849,15.86,16.244,15.129,17.561,17.313,16.567,15.112,14.526,16.631,17.853,19.133,17.766,11.694,11.099,9.587,8.914,11.763,13.295,13.855,13.838,7.852,9.27,9.636,10.39,10.141,11.66,12.012,12.339,11.398,11.162,11.19,11.78,9.824,9.683,8.339,10.817,11.93,12.327,11.137,11.281,12.912,13.554,14.752,13.969,11.131,10.883,9.695,9.694,10.702,11.967,11.663,13.067,8.011,7.426,8.446,8.101,6.897,6.1,4.883,5.681,10.897,11.081,9.924,10.085,11.176,12.402,12.277,13.659,4.63,4.55,4.121,3.156,3.584,3.93,2.81,5.265,0.971,0.117,0.956,0.683,-0.892,-1.92,-2.699,-2.865,0.915,0.598,1.686,1.409,0.354,-0.885,-0.954,-2.152,0.179,0.579,1.29,1.366,-0.655,-1.625,-2.761,-0.888,-3.94,-3.236,-4.034,-4.16,-1.823,-0.799,-0.742,0.569,-6.793,-6.457,-6.513,-6.561,-5.068,-4.918,-3.464,-5.801,-17.467,-16.379,-16.321,-15.992,-15.054,-13.914,-14.399,-12.786,-19,-18.569,-17.643,-17.863,-17.847,-17.453,-18.649,-16.306,-17.342,-16.038,-15.772,-14.932,-14.936,-15.145,-14.15,-15.043,-3.131,-1.935,-1.418,-0.209,-2.22,-2.714,-2.966,-1.702,-2.338,-1.971,-1.35,-0.314,-3.207,-3.096,-2.746,-2.091,2.275,2.399,3.424,4.284,1.05,1.021,1.536,-0.393,-12.727,-12.991,-13.69,-14.406,-11.694,-11.198,-9.81,-12.179,3.916,3.088,3.915,4.637,1.955,1.037,-0.039,0.405,-9.354,-10.511,-11.317,-12.546,-10.09,-11.248,-12.173,-10.711,-19.06,-19.183,-20.321,-20.209,-19.359,-18.103,-18.487,-17.016,-8.875,-7.669,-6.449,-5.474,-7.666,-6.426,-6.234,-6.546,1.594,0.511,0.824,-0.042,0.107,-0.556,-0.719,-1.9,-1.68,-2.637,-2.186,-2.706,-3.001,-3.468,-3.712,-4.726,-5.035,-5.855,-5.122,-5.735,-6.241,-7.605,-7.957,-7.613,-8.671,-9.523,-8.925,-9.644,-9.883,-10.527,-10.672,-11.873,-11.811,-12.506,-12.047,-12.87,-12.36,-13.164,-14.623,-13.06,-13.213,-13,-14.271,-14.64,-11.84,-10.46,-9.41,-10.12,-5.415,-6.21,-5.882,-5.897,-7.709,-8.589,-8.27,-10.052,-4.704,-5.38,-4.442,-4.556,-6.647,-7.773,-8.976,-8.183,-1.177,0.136,0.362,0.933,1.239,2.653,2.683,3.61,-1.957,-2.815,-2.17,-2.374,-4.204,-5.125,-6.4,-5.445,-23.817,-22.508,-22.2,-22.458,-21.404,-21.349,-20.275,-21.062,-9.296,-7.884,-7.032,-5.968,-7.657,-6.213,-5.743,-6.108,-3.963,-2.614,-2.109,-0.901,-2.55,-2.825,-2.808,-1.807,2.356,1.711,2.445,2.346,1.781,1.185,1.509,-0.314,-13.328,-14.729,-15.551,-16.386,-15.343,-15.77,-16.658,-16.53,-15.543,-16.333,-17.721,-18.384,-16.485,-15.216,-15.506,-14.686,-7.901,-7.469,-6.405,-5.489,-8.681,-8.705,-8.467,-10.039,-3.116,-2.645,-1.929,-0.789,-3.814,-3.438,-2.871,-4.657,1.487,1.617,2.464,2.929,0.224,-0.734,-2.152,-0.253,-6.319,-6.225,-7.158,-7.853,-4.778,-3.75,-2.357,-4.096,-7.188,-8.055,-9.512,-10.321,-7.802,-8.386,-9.892,-8.005,-12.408,-13.698,-14.807,-15.789,-13.842,-15.142,-15.319,-15.135,-8.959,-7.874,-6.484,-5.53,-7.983,-9.033,-9.187,-8.63],"z":[3.506,4.475,4.011,2.903,4.748,5.189,5.41,6.451,15.243,16.245,16.06,17.024,16.149,17.117,18.566,16.822,18.86,18.444,19.626,19.512,17.612,16.239,15.659,15.312,6.906,5.457,5.051,4.229,4.747,3.215,2.592,2.688,-0.251,0.423,-0.502,-0.333,1.55,2.694,3.655,3.417,-0.514,0.104,-0.872,-0.45,1.191,2.266,3.287,2.937,-0.018,1.158,0.783,1.242,2.268,3.194,2.389,4.126,2.002,3.331,3.366,4.202,4.168,4.115,4.806,4.723,14.49,14.791,14.95,15.854,13.716,13.941,15.359,12.925,18.451,19.097,20.617,21.32,18.657,17.17,16.918,16.742,13.236,13.176,12.795,12.044,14.509,14.452,13.506,15.84,9.691,9.786,8.924,8.388,11.246,12.109,13.559,11.553,3.945,2.611,1.538,0.541,2.364,1.071,1.065,0.94,1.967,2.075,1.398,0.925,3.539,4.337,5.786,3.703,38.617,37.673,38.329,39.088,36.53,35.56,34.538,34.874,26.404,25.496,24.553,23.353,26.267,25.41,24.527,26.298,20.493,19.677,18.643,17.594,20.562,21.466,22.274,20.642,19.382,19.314,17.971,17.414,19.323,20.503,20.257,21.791,32.337,33.137,34.401,34.775,33.485,33.866,32.808,34.027,34.156,34.067,35.412,35.459,33.443,31.976,31.544,31.104,29.895,29.31,27.796,27.029,29.981,29.978,30.462,30.864,26.248,26.295,25.126,24.419,27.613,27.815,28.324,28.797,22.765,22.27,20.799,20.167,23.157,24.655,25.463,24.967,19.535,19.335,20.654,20.91,18.326,16.961,16.027,16.355,21.505,22.808,23.643,24.286,23.503,24.786,25.987,24.606,25.391,26.819,27.015,27.887,27.417,28.924,29.724,29.36,25.752,25.424,24.078,23.881,25.461,26.853,26.755,27.488,-7.727,-8.795,-10.042,-10.537,-9.116,-7.935,-8.366,-7.385,-15.37,-14.494,-14.407,-14.526,-13.088,-12.094,-12.577,-10.714,1.054,2.274,3.158,4.385,1.949,1.438,1.037,2.509,-9.839,-9.948,-9.757,-10.505,-8.895,-8.859,-10.202,-7.733,-7.463,-6.519,-5.492,-5.153,-5.813,-6.743,-6.04,-7.211,-5.508,-4.427,-3.345,-2.23,-3.803,-4.781,-4.037,-5.528,-1.695,-0.879,0.405,1.471,-1.677,-2.379,-2.923,-3.483,8.099,8.522,9.633,10.538,7.319,6.16,4.952,6.587,4.48,3.259,2.036,1.202,3.014,1.841,1.915,1.863,-2.255,-2.893,-2.818,-3.791,-2.234,-2.327,-1.711,-3.774,-7.736,-7.165,-5.684,-5.158,-7.351,-6.964,-7.867,-7.068,-4.091,-2.98,-1.785,-1.081,-2.577,-3.626,-3.157,-3.914,1.259,1.646,1.107,1.794,1.17,1.676,3.203,1.138,0.556,1.333,2.686,3.271,1.544,0.325,0.693,-0.168,-0.4,-1.035,-1.533,-0.841,-0.07,0.393,1.466,-0.778,-1.844,-1.624,-2.792,-2.578,-1.38,-1.039,0.21,-0.841,-2.626,-3.102,-4.115,-4.258,-3.689,-2.727,-3.485,-1.598,-8.912,-8.347,-8.833,-8.217,-6.816,-6.029,-4.55,-6.25,3.535,3.599,4.439,5.241,2.204,1.773,0.554,2.887,8.167,8.807,9.264,10.049,7.859,7.521,6.441,8.777,16.049,14.751,14.915,14.099,13.966,12.465,12.223,11.87,15.722,14.494,13.588,13.189,13.734,12.456,12.812,11.57,11.292,10.013,10.055,9.015,9.456,9.446,9.138,8.454,6.33,4.895,4.123,3.2,4.428,5.1,4.579,4.845,4.514,3.866,4.076,3.143,4.459,3.853,4.037,2.387,1.359,1.398,1.324,0.617,2.673,2.797,1.594,4.09,15.756,15.11,15.578,15.514,13.589,12.941,11.471,13.104],"type":"scatter3d","mode":"markers","name":"LEU","marker":{"color":"rgba(179,216,80,1)","line":{"color":"rgba(179,216,80,1)"}},"textfont":{"color":"rgba(179,216,80,1)"},"error_y":{"color":"rgba(179,216,80,1)"},"error_x":{"color":"rgba(179,216,80,1)"},"line":{"color":"rgba(179,216,80,1)"},"frame":null},{"x":[-13.384,-13.398,-11.965,-11.649,-14.138,-14.233,-14.886,-14.986,-15.587,-9.216,-8.606,-9.006,-8.155,-8.932,-8.336,-8.746,-8.106,-8.603,-4.712,-4.501,-3.049,-2.568,-5.357,-6.84,-7.567,-9.024,-9.735,1.301,2.099,3.278,4.288,1.212,0.553,-0.356,-1.035,-1.947,20.136,21.286,22.306,23.068,21.969,21.14,21.393,22.749,22.877,17.36,16.366,16.177,15.045,16.771,16.623,16.928,16.645,16.952,12.827,12.184,12.029,11.033,13.014,13.125,13.909,14.033,14.812,13.024,12.968,11.826,10.994,14.289,15.446,16.726,17.894,19.178,-0.666,-0.78,-1.48,-1.339,0.614,1.386,0.643,1.471,1.865,1.577,2.611,3.235,4.457,2.035,3.095,2.492,2.07,1.604,19.731,20.819,20.922,21.988,20.71,21.866,21.721,22.868,22.708,8.054,7.058,7.445,6.602,6.958,6.629,6.377,6.438,7.823,-5.986,-7.388,-7.711,-8.782,-7.826,-7.786,-8.392,-9.876,-10.502,-3.51,-3.713,-2.927,-2.298,-5.201,-6.03,-7.502,-7.673,-9.099,6.935,6.171,5.526,4.349,7.05,6.259,5.155,4.221,4.95,0.482,-0.188,-1.375,-2.153,0.82,1.673,2.763,3.563,4.852,-20.835,-21.822,-21.848,-22.696,-21.589,-22.021,-21.898,-22.354,-22.198,-15.638,-15.414,-14.255,-13.357,-16.685,-16.573,-17.803,-17.696,-17.703,4.475,4.92,6.176,7.016,3.796,3.99,2.928,2.834,2.318,-9.43,-10.811,-11.358,-12.427,-10.958,-12.396,-12.541,-12.299,-12.349,-0.167,0.799,0.062,0.501,1.56,2.928,3.28,2.65,3.026,9.526,10.303,9.456,9.34,11.607,12.555,13.783,14.805,15.967,2.947,3.872,3.26,3.856,4.739,6.211,7.039,8.524,8.959,6.711,7.673,7.164,7.224,9.028,10.097,11.395,12.484,13.768,4.007,2.804,3.118,2.498,1.895,1.288,0.549,-0.036,-0.641,2.919,3.67,3.183,3.007,5.153,5.815,7.174,7.772,9.069,-1.297,-1.779,-1.713,-2.466,-0.934,0.552,1.356,1.565,2.296,-4.198,-5.245,-6.514,-7.586,-4.696,-3.806,-2.721,-3.267,-2.145,-7.266,-7.196,-8.277,-8.007,-7.343,-6.11,-6.287,-5.038,-5.195,-7.823,-8.391,-9.064,-8.987,-9.393,-8.765,-9.783,-9.146,-10.121,-9.7,-10.365,-9.35,-9.526,-11.092,-12.276,-12.928,-14.099,-14.753,-7.005,-7.336,-6.116,-6.258,-8.424,-9.726,-10.294,-11.717,-11.822,-7.423,-8.405,-9.826,-10.607,-8.205,-9.049,-8.659,-9.007,-8.645,-16.841,-17.309,-16.228,-16.474,-17.873,-18.302,-18.725,-18.891,-19.159,0.631,2.079,2.648,3.537,2.705,4.201,4.756,6.275,6.861,13.373,13.885,13.128,13.74,13.884,14.867,14.723,15.672,15.479,6.064,5.38,3.896,3.033,5.972,7.403,7.903,7.155,7.561,21.546,22.801,22.673,23.551,23.205,24.491,25.662,26.967,28.075,24.492,24.159,24.247,24.164,22.782,22.775,21.436,21.529,20.196,31.474,32.786,32.835,33.914,33.864,34.098,35.273,35.534,36.694,31.56,32.539,31.956,32.147,33.797,34.92,36.195,37.359,38.646,27.236,28.085,27.644,27.815,29.533,30.549,31.931,33.025,33.324,31.147,32.167,31.505,31.792,33.033,33.998,35.211,36.495,36.774,17.059,15.812,15.506,15.068,14.65,13.437,12.149,11.6,10.304,13.216,13.429,14.923,15.327,12.948,13.076,12.565,12.798,12.381,21.689,21.764,22.715,22.403,22.01,20.759,20.99,19.798,18.54],"y":[14.379,14.329,14.325,15.046,13.08,12.978,11.68,11.605,10.314,15.39,16.679,17.779,18.549,17.084,16.145,16.522,15.598,15.884,24.263,24.329,24.156,24.909,23.274,23.384,22.127,22.18,20.902,26.452,27.429,26.78,27.434,28.169,27.254,28.016,27.073,27.796,7.068,7.957,7.852,8.797,7.767,8.325,7.623,7.919,7.28,20.508,21.573,22.095,22.267,22.709,22.369,23.577,23.292,24.473,21.77,22.832,22.396,22.714,24.117,24.752,26.055,26.758,25.957,21.686,21.212,20.225,20.401,20.568,21.534,20.882,21.824,21.213,21.363,20.59,19.246,18.325,20.352,21.623,22.471,23.659,24.531,13.867,13.43,14.629,14.8,12.451,11.688,10.719,11.423,10.444,11.958,12.444,11.747,11.265,13.958,14.532,16.023,16.562,18.017,5.697,5.103,5.381,5.725,3.587,3.155,1.649,1.106,1.155,17.045,17.34,17.443,17.017,18.612,18.485,19.702,19.849,21.021,8.56,8.329,9.369,9.053,8.408,7.226,7.367,7.356,7.525,0.071,0.031,1.384,1.457,-0.364,-0.485,-1.539,-1.571,-1.976,4.387,5.673,5.684,6.638,6.759,7.167,8.133,8.518,9.166,-8.524,-8.56,-7.286,-7.139,-9.767,-11.103,-12.203,-13.552,-14.623,-1.86,-3.024,-3.87,-4.247,-3.873,-5.066,-5.95,-7.16,-6.773,-7.513,-8.182,-9.033,-9.117,-9.037,-9.367,-10.322,-10.3,-8.983,-19.964,-19.716,-18.421,-18.413,-19.703,-19.448,-19.499,-20.909,-20.976,-5.505,-4.442,-3.107,-2.084,-4.723,-4.068,-3.564,-2.198,-1.635,-1.077,-2.307,-3.329,-4.494,-2.031,-3.213,-2.861,-3.978,-3.742,-15.677,-16.799,-18.18,-19.017,-16.809,-16.965,-16.784,-16.758,-17.996,2.483,1.387,0.244,-0.926,1.881,0.802,1.303,0.258,0.719,-0.342,-0.897,-1.881,-2.941,0.228,1.045,2.251,3.076,4.321,-13.527,-14.773,-15.777,-16.956,-14.484,-13.589,-13.122,-12.096,-11.546,-17.791,-19.139,-20.03,-20.999,-19.767,-19.928,-20.624,-19.724,-20.411,-17.891,-18.905,-18.485,-19.037,-20.216,-20.955,-21.782,-22.951,-23.795,-24.065,-23.958,-23.015,-22.139,-25.33,-26.202,-27.492,-28.353,-29.638,-19.209,-18.754,-17.395,-16.532,-19.776,-21.115,-22.116,-23.489,-24.511,-17.199,-15.935,-14.798,-13.756,-16.018,-16.963,-16.953,-17.915,-17.879,-3.263,-1.943,-1.229,-0.287,-2.09,-2.717,-1.925,-2.336,-3.752,2.063,2.487,2.232,1.538,3.968,4.463,5.878,6.897,8.286,-6.445,-7.81,-8.736,-9.496,-8.377,-9.837,-10.336,-11.848,-12.323,-7.55,-7.491,-8.911,-9.258,-6.633,-6.426,-5.548,-5.431,-4.666,-17.375,-16.605,-15.301,-14.259,-17.452,-18.61,-19.533,-20.719,-21.706,-2.92,-1.633,-1.852,-1.269,-0.649,-0.201,0.814,2.136,3.098,-26.256,-26.828,-27.239,-26.946,-28.04,-28.717,-27.739,-28.415,-27.428,-26.172,-26.245,-24.917,-24.889,-26.882,-28.359,-29.007,-30.5,-31.155,4.264,3.996,2.664,2.147,4.044,5.442,5.484,6.906,6.967,-8.781,-9.141,-8.98,-9.842,-8.288,-8.665,-7.904,-8.359,-7.705,-22.694,-23.582,-23.754,-24.833,-23.102,-24.172,-23.56,-24.617,-25.14,1.878,2.054,2.353,3.367,0.792,0.766,1.644,0.816,0.114,-6.657,-7.382,-7.435,-8.462,-6.706,-7.61,-6.83,-6.311,-5.592,-18.111,-18.112,-17.925,-17.129,-19.432,-19.533,-20.883,-21.044,-22.382,-5.892,-5.783,-4.762,-4.181,-7.151,-8.002,-9.282,-10.225,-9.616],"z":[7.205,8.665,9.208,10.162,9.149,10.658,11.088,12.6,13.044,7.684,7.375,8.354,8.811,5.935,4.906,3.495,2.477,1.106,17.392,18.836,19.285,20.134,19.54,19.266,19.697,19.285,19.575,17.92,18.655,19.389,19.656,19.661,20.691,21.637,22.609,23.542,15.786,15.89,14.761,14.529,17.25,18.398,19.719,20.29,21.625,18.7,18.687,17.268,16.81,19.626,21.095,21.96,23.423,24.284,14.688,13.922,12.464,11.816,13.983,15.357,15.262,16.602,17.586,11.942,10.561,10.36,9.468,10.149,10.14,9.663,9.855,9.441,2.991,1.757,1.978,1.171,1.17,0.834,-0.187,-0.65,0.482,-1.489,-2.423,-3.129,-3.116,-3.449,-4.214,-5.219,-6.502,-7.526,-0.904,-0.069,1.282,1.665,0.142,0.953,1.186,2.031,2.312,10.929,11.821,13.266,14.09,11.613,10.196,10.143,8.714,8.147,26.886,26.619,25.129,24.693,27.35,28.87,29.558,29.223,29.907,10.934,9.505,8.71,7.695,9.145,9.625,9.221,7.705,7.295,38.566,39.808,40.075,40.431,41.002,42.306,42.166,43.359,44.582,40.491,40.366,39.411,39.4,39.995,41.172,40.789,42.02,41.662,21.503,22.581,23.423,24.301,23.496,22.911,23.959,23.426,24.456,31.61,32.459,31.935,32.691,32.547,33.479,33.382,34.305,35.748,39.33,40.549,40.36,41.263,41.128,42.597,43.091,44.604,45.08,22.826,23.225,22.631,22.025,24.755,25.228,26.747,27.29,28.785,20.601,20.343,20.225,20.76,19.045,18.983,17.589,17.326,15.997,27.007,27.124,27.885,27.486,27.875,27.955,28.777,28.743,29.649,22.744,22.704,22.486,21.807,23.963,23.673,24.924,24.595,23.878,-14.937,-14.904,-14.019,-14.409,-14.389,-14.37,-13.772,-13.905,-13.307,-13.07,-13.682,-14.803,-14.893,-14.181,-13.06,-13.598,-12.477,-13.009,-17.304,-17.262,-16.218,-16.526,-17.006,-18.028,-17.539,-18.487,-17.988,-15.121,-15.411,-14.17,-14.05,-16.532,-16.177,-17.271,-18.478,-19.595,11.836,11.861,12.589,12.331,12.436,11.449,12.119,12.89,13.396,-0.259,-1.715,-2.231,-3.063,-2.376,-2.271,-3.055,-2.98,-3.72,-4.026,-5.29,-5.096,-5.963,-5.826,-6.133,-6.65,-6.804,-7.27,-3.944,-3.639,-3.583,-4.223,-2.296,-2.269,-0.898,-0.841,0.498,-12.204,-11.669,-11.083,-10.298,-10.595,-11.097,-12.275,-12.635,-13.065,-5.007,-4.014,-4.517,-3.864,-3.676,-2.515,-2.101,-3.176,-2.764,7.923,8.128,8.681,9.618,6.823,6.899,5.527,5.49,4.095,5.366,5.199,5.224,4.438,6.304,6.159,7.271,7.17,8.308,-12.56,-13.689,-13.919,-14.159,-14.963,-14.912,-16.112,-15.993,-17.092,-4.233,-4.173,-4.465,-3.809,-5.196,-4.899,-5.928,-5.824,-6.885,8.51,8.037,6.574,5.757,8.886,8.424,8.431,8.043,7.896,1.033,-0.383,-1.128,-2.356,-0.576,-0.238,-0.49,-0.219,-0.157,-1.73,-2.302,-3.047,-3.334,-1.215,-0.668,0.286,0.76,1.692,-3.988,-2.971,-1.57,-0.705,-3.116,-2.167,-2.507,-1.635,-2.026,12.836,13.628,15.082,15.659,13.587,13.954,14.118,14.158,12.797,14.068,13.038,11.691,11.055,12.928,11.747,11.974,12.001,10.706,8.728,8.503,7.011,6.49,9.236,9.396,9.615,8.293,8.455,11.248,12.692,12.944,13.787,13.29,14.799,15.291,16.78,17.296,18.756,20.211,20.817,21.854,20.852,20.941,21.725,21.606,22.131],"type":"scatter3d","mode":"markers","name":"LYS","marker":{"color":"rgba(209,217,69,1)","line":{"color":"rgba(209,217,69,1)"}},"textfont":{"color":"rgba(209,217,69,1)"},"error_y":{"color":"rgba(209,217,69,1)"},"error_x":{"color":"rgba(209,217,69,1)"},"line":{"color":"rgba(209,217,69,1)"},"frame":null},{"x":[13.022,13.452,14.864,15.066,13.338,13.641,13.626,15.219,9.14,7.836,7.475,6.297,7.766,7.891,6.917,5.26,7.029,7.976,9.44,10.277,7.645,6.52,6.149,7.476,1.726,1.517,0.08,-0.283,2.339,3.831,4.722,6.385,-19.646,-18.959,-17.522,-16.621,-19.73,-21.072,-21.931,-21.097,8.684,9.166,10.115,10.39,9.806,8.803,9.556,10.198,6.105,4.94,5.291,4.508,4.206,3.467,2.402,1.181,1.491,1.042,0.613,1.114,-0.1,-0.571,-2.018,-3.329,-6.159,-5.506,-6.449,-6.004,-4.899,-3.732,-2.514,-1.8,-12.588,-13.584,-14.384,-14.449,-14.572,-14.046,-15.237,-16.762,8.495,7.886,8.41,8.443,6.368,5.664,3.882,3.313,32.462,32.592,31.35,31.445,32.862,34.255,34.597,34.945,24.269,22.984,22.528,21.569,21.919,22.158,21.114,19.513,17.208,17.595,17.137,17.862,17.085,17.777,19.597,19.865],"y":[10.023,9.103,8.559,7.342,9.759,8.807,9.614,10.424,19.593,20.198,21.2,21.465,20.832,19.823,18.313,18.949,16.715,16.32,16.599,15.696,17.029,16.412,17.404,16.876,16.826,18.271,18.756,19.794,18.919,18.706,19.565,19.656,-1.877,-3.147,-2.926,-3.67,-4.046,-4.506,-5.685,-7.215,-9.282,-8.948,-9.971,-9.909,-7.559,-6.448,-4.845,-4.43,-3.284,-3.656,-4.608,-5.501,-2.417,-1.626,-2.625,-3.252,-19.01,-18.053,-18.717,-18.369,-17.197,-16.127,-15.239,-16.447,-14.724,-14.508,-13.863,-13.191,-15.799,-16.303,-15.027,-14.715,-1.244,-0.726,-1.81,-1.827,0.198,1.573,2.467,2.254,-20.622,-21.081,-20.451,-21.101,-20.885,-21.672,-21.383,-22.739,-4.851,-4.555,-4.998,-5.46,-3.07,-2.633,-0.945,-1.184,-22.498,-21.819,-21.438,-20.681,-22.663,-22.801,-24.036,-23.249,-16.611,-15.27,-14.198,-13.237,-14.962,-15.753,-15.642,-13.887],"z":[20.495,19.459,19.659,19.648,18.084,16.947,15.366,15.371,12.346,12.593,11.503,11.262,13.982,15.118,14.879,15.123,-2.161,-3.209,-2.873,-2.918,-4.527,-5.333,-6.811,-7.922,26.189,26.193,26.077,26.637,25.081,25.193,23.879,24.621,29.148,28.959,28.497,28.886,27.995,28.556,27.497,27.958,37.259,35.929,35.303,34.103,35.949,36.196,36.511,34.902,28.316,27.527,26.387,26.061,27.007,28.088,29.182,28.031,4.937,5.938,7.244,8.311,5.391,6.364,5.791,6.132,-4.731,-6.02,-7.031,-7.964,-6.56,-5.718,-5.301,-6.911,-3.68,-2.736,-2.02,-0.791,-3.45,-3.784,-4.817,-3.826,-7.48,-8.727,-10.012,-11.058,-8.673,-7.584,-7.599,-6.534,-6.486,-5.061,-4.296,-3.158,-4.832,-5.241,-4.73,-2.962,16.891,16.963,18.37,18.529,16.261,14.766,13.965,14.031,10.811,10.388,11.372,11.616,8.978,7.868,7.901,7.594],"type":"scatter3d","mode":"markers","name":"MET","marker":{"color":"rgba(236,217,57,1)","line":{"color":"rgba(236,217,57,1)"}},"textfont":{"color":"rgba(236,217,57,1)"},"error_y":{"color":"rgba(236,217,57,1)"},"error_x":{"color":"rgba(236,217,57,1)"},"line":{"color":"rgba(236,217,57,1)"},"frame":null},{"x":[22.627,23.174,24.621,25.153,22.297,20.837,20.397,19.912,19.055,18.57,18.143,10.446,9.328,8.245,7.624,8.777,9.725,10.552,9.8,11.44,10.684,11.507,-0.736,-2.12,-3.076,-3.818,-2.28,-1.375,-1.756,-0.133,-0.91,0.72,0.332,-5.889,-5.575,-6.559,-6.215,-4.131,-3.564,-3.488,-3.132,-2.992,-2.633,-2.563,-2.949,-2.243,-0.729,-0.095,-2.582,-1.694,-1.864,-0.72,-1.078,0.073,-0.106,16.324,15.527,14.257,13.172,16.33,15.517,15.214,15.024,14.429,14.237,13.937,2.477,1.309,0.648,-0.575,1.592,1.605,0.415,2.79,0.404,2.791,1.594,-15.772,-16.715,-18.136,-18.795,-16.116,-14.717,-14.49,-13.624,-13.194,-12.325,-12.11,-20.909,-20.846,-21.885,-22.134,-19.44,-18.432,-18.669,-17.251,-17.746,-16.317,-16.567,-2.505,-1.873,-0.893,-0.812,-1.12,-1.999,-3.273,-1.543,-4.08,-2.34,-3.609,0.705,1.883,2.553,3.427,2.904,2.378,1.999,2.234,1.481,1.715,1.338,6.667,6.83,7.968,7.744,5.522,5.477,5.406,5.507,5.366,5.467,5.396,8.879,7.483,7.267,6.621,6.542,5.082,4.477,4.322,3.13,2.981,2.386,8.846,7.99,6.789,6.49,7.517,6.485,6.847,5.15,5.892,4.188,4.562,-9.686,-10.34,-10.805,-11.016,-9.371,-8.784,-9.497,-7.538,-8.98,-7.013,-7.737,-1.834,-1.653,-0.154,0.284,-2.259,-3.759,-4.542,-4.391,-5.928,-5.778,-6.548,8.799,9.284,10.587,10.652,8.204,6.882,6.654,5.867,5.434,4.641,4.424,11.148,11.175,12.551,13.261,10.693,10.218,9.228,10.766,8.79,10.334,9.344,3.614,2.249,1.461,0.372,2.276,0.942,-0.07,0.704,-1.3,-0.522,-1.525,14.835,14.453,15.026,15.559,12.926,12.463,12.49,12.031,12.098,11.636,11.67,22.52,23.819,24.919,25.727,24.262,23.523,23.974,22.334,23.249,21.6,22.056,27.359,28.231,28.629,28.761,27.581,27.226,28.198,25.928,27.887,25.605,26.585,31.666,31.596,31.424,31.415,30.504,30.919,31.458,30.811,31.887,31.237,31.778,19.12,19.455,18.228,18.338,19.88,21.161,22.192,21.334,23.38,22.514,23.537,16.04,15.51,14.13,13.726,15.6,17.008,17.466,17.894,18.787,19.222,19.668,11.665,11.562,10.113,9.377,12,12.314,13.454,11.469,13.751,11.756,12.902,14.067,15.011,14.784,15.736,14.944,16.02,17.335,15.733,18.351,16.739,18.053,15.738,17.186,17.653,18.401,17.86,19.351,20.198,19.904,21.579,21.284,22.12],"y":[10.676,11.922,11.754,12.577,12.416,12.502,13.413,11.637,13.459,11.676,12.591,6.581,7.512,7.014,7.805,7.609,8.249,9.306,7.787,9.89,8.362,9.415,18.023,18.411,17.252,16.86,18.944,20.097,21.399,19.88,22.475,20.943,22.245,15.989,14.731,13.606,12.622,14.322,13.347,13.671,12.092,12.757,11.17,11.501,10.605,11.717,11.48,11.616,13.018,14.186,14.885,14.617,16.002,15.734,16.429,1.043,2.184,1.693,2.229,3.061,4.156,5.317,4.009,6.317,5.003,6.158,2.056,1.254,1.542,1.487,-0.235,-0.621,-0.915,-0.613,-1.188,-0.884,-1.17,-7.898,-7.35,-7.104,-6.15,-6.076,-6.265,-7.01,-5.727,-7.22,-5.931,-6.679,-6.378,-5.125,-4.117,-3.11,-4.522,-5.228,-6.52,-4.594,-7.168,-5.235,-6.523,-6.946,-6.713,-5.54,-4.675,-7.963,-9.17,-9.038,-10.446,-10.163,-11.578,-11.435,11.376,11.913,13.146,13.745,10.795,9.666,8.461,9.82,7.426,8.789,7.591,11.938,10.784,11.088,11.346,10.498,9.138,7.989,9.008,6.728,7.748,6.604,1.856,1.441,0.585,-0.467,2.655,2.287,1.815,2.364,1.419,1.971,1.498,-2.873,-3.721,-4.207,-5.401,-2.953,-3.682,-4.752,-3.292,-5.423,-3.956,-5.023,-17.865,-19.015,-19.973,-21.162,-19.701,-18.763,-18.426,-18.174,-17.509,-17.256,-16.923,-8.339,-8.11,-8.108,-8.619,-6.769,-6.709,-7.825,-5.527,-7.767,-5.457,-6.578,-19.183,-18.47,-17.717,-16.499,-17.492,-18.146,-18.768,-18.149,-19.39,-18.768,-19.389,-13.711,-12.865,-12.238,-12.592,-13.657,-12.793,-11.837,-12.923,-11.023,-12.114,-11.163,-2.718,-3.037,-3.66,-3.195,-3.971,-4.594,-3.844,-5.941,-4.426,-6.532,-5.776,-30.585,-30.263,-28.903,-28.743,-30.233,-29.769,-30.632,-28.459,-30.197,-28.013,-28.882,-23.979,-23.341,-24.154,-23.6,-22.744,-21.486,-20.25,-21.541,-19.087,-20.386,-19.157,2.382,1.919,3.005,2.743,0.717,-0.403,-1.302,-0.534,-2.313,-1.543,-2.433,2.115,0.841,0.988,-0.003,-0.055,-0.748,-0.024,-2.13,-0.665,-2.786,-2.052,-4.849,-6.254,-7.117,-8.173,-6.489,-5.798,-5.626,-5.327,-4.994,-4.694,-4.528,-23.142,-24.465,-24.823,-25.987,-24.722,-24.688,-23.598,-25.72,-23.532,-25.663,-24.567,-25.639,-24.579,-24.511,-23.572,-23.232,-22.183,-22.29,-21.093,-21.327,-20.123,-20.242,-21.315,-20.356,-18.943,-18.265,-20.347,-19.51,-19.972,-18.243,-19.183,-17.447,-17.918,-18.639,-18.543,-17.129,-16.516,-19.56,-19.388,-19.784,-18.828,-19.623,-18.662,-19.061],"z":[12.354,12.871,13.313,14.056,14.021,13.659,12.704,14.231,12.318,13.852,12.891,9.989,9.92,10.878,11.587,8.491,7.509,7.895,6.197,6.99,5.284,5.68,25.332,25.115,25.348,24.445,23.688,23.37,23.673,22.783,23.399,22.504,22.812,22.63,21.96,22.301,22.958,22.266,21.285,19.935,21.701,19.012,20.787,19.441,9.2,8.572,8.542,7.494,9.319,8.976,7.784,9.874,7.493,9.595,8.405,37.564,37.113,36.422,36.648,36.144,35.506,36.212,34.211,35.641,33.63,34.346,37.461,37.777,39.118,39.225,37.555,36.1,35.437,35.372,34.066,34,33.347,20.25,21.215,20.709,21.124,21.822,22.352,23.508,21.677,23.983,22.143,23.298,23.175,23.925,23.448,24.111,23.859,24.729,25.196,25.095,26.014,25.915,26.375,20.541,21.843,21.713,22.59,22.322,22.549,23.1,22.218,23.318,22.43,22.98,32.869,33.545,32.941,33.573,33.767,34.614,34.035,35.992,34.81,36.78,36.188,31.983,31.099,30.121,28.94,30.349,29.702,30.478,28.315,29.888,27.711,28.504,30.04,29.911,28.667,28.735,29.884,29.892,31.054,28.73,31.057,28.724,29.891,28.973,29.791,28.985,28.965,31.03,31.847,32.659,31.807,33.423,32.566,33.374,10.668,10.062,11.151,10.907,9.098,8.081,6.934,8.294,6.011,7.379,6.235,6.177,4.746,4.448,3.42,4.319,4.425,4.144,4.804,4.24,4.903,4.621,-9.944,-11.121,-10.847,-11.019,-11.605,-11.898,-13.12,-10.947,-13.393,-11.207,-12.432,-11.633,-10.446,-10.195,-9.252,-9.225,-8.095,-8.303,-6.821,-7.259,-5.77,-5.991,-5.437,-5.845,-4.703,-4.374,-7.067,-7.403,-7.998,-7.129,-8.316,-7.442,-8.035,13.684,12.309,11.908,10.813,12.159,10.802,9.711,10.61,8.447,9.348,8.266,4.151,4.026,3.357,2.614,5.361,5.714,5.257,6.437,5.509,6.695,6.23,0.775,-0.297,-1.3,-2.498,-0.992,-0.045,0.389,0.445,1.297,1.355,1.783,-3.365,-4.075,-5.582,-6.314,-3.481,-2.211,-1.15,-2.093,0.009,-0.937,0.116,8.361,8.592,8.285,7.651,10.047,10.44,9.527,11.74,9.902,12.125,11.206,-4.019,-3.71,-4.244,-4.175,-2.204,-1.681,-0.95,-1.971,-0.517,-1.541,-0.814,-0.723,0.283,0.771,0.463,-0.308,0.729,1.518,0.918,2.484,1.881,2.667,8.548,7.98,8.514,8.909,6.446,5.803,5.746,5.298,5.201,4.752,4.703,12.176,12.292,11.953,12.717,11.359,11.238,12.266,10.092,12.152,9.967,11],"type":"scatter3d","mode":"markers","name":"PHE","marker":{"color":"rgba(253,215,61,1)","line":{"color":"rgba(253,215,61,1)"}},"textfont":{"color":"rgba(253,215,61,1)"},"error_y":{"color":"rgba(253,215,61,1)"},"error_x":{"color":"rgba(253,215,61,1)"},"line":{"color":"rgba(253,215,61,1)"},"frame":null},{"x":[17.688,19.083,19.64,20.599,19.839,19.001,17.606,28.343,29.423,29.061,29.93,29.532,28.163,27.845,19.928,18.601,18.67,19.541,18.069,18.723,20.132,21.482,21.231,20.888,19.981,22.556,22.973,22.712,12.511,11.127,10.706,9.784,10.318,11.21,12.559,-3.122,-4.005,-5.484,-6.155,-3.762,-3.319,-2.405,-9.439,-9.278,-9.542,-8.862,-10.326,-10.299,-10.296,-10.471,-11.537,-11.025,-11.309,-12.55,-12.431,-10.939,-10.079,-11.439,-11.58,-12.556,-12.049,-11.329,-9.909,-2.995,-1.98,-1.396,-0.875,-0.928,-1.039,-2.531,1.415,0.22,-0.834,-2.004,0.803,2.095,2.633,-13.328,-12.998,-11.531,-11.193,-13.272,-14.458,-14.062,-2.468,-1.871,-2.418,-2.644,-0.387,-0.21,-1.435,-8.703,-9.702,-10.112,-10.282,-10.861,-10.142,-9.05,-4.534,-3.644,-2.262,-1.592,-3.551,-4.909,-5.143,11.662,12.961,13.519,14.057,13.866,13.226,11.765,15.185,15.098,14.149,13.418,16.542,17.325,16.595,26.351,26.218,24.758,24.272,27.052,28.165,27.446,16.518,17.55,18.258,19.422,16.788,15.75,15.258,20.177,19.86,18.575,18.41,19.856,19.439,20.209],"y":[4.177,4.563,4.255,4.892,3.827,2.617,3.188,16.84,16.769,15.795,15.138,18.216,18.794,18.219,14.401,14.989,16.478,16.915,14.198,12.881,13.247,21.634,22.624,21.961,22.406,23.379,23.395,21.969,4.029,4.366,3.878,4.43,3.678,3.805,3.428,16.733,15.612,15.844,14.953,15.423,16.779,17.211,10.526,9.355,9.714,9.214,8.388,8.685,10.2,1.743,2.11,3.059,2.867,2.8,2.008,1.876,0.312,-0.235,-1.737,-2.322,0.525,1.804,1.354,-11.688,-11.149,-9.775,-9.083,-12.26,-12.873,-12.95,-18.943,-19.595,-18.68,-19.05,-20.587,-19.945,-19.439,-26.3,-27.399,-27.31,-27.581,-28.634,-28.206,-26.811,-26.707,-26.432,-27.343,-28.532,-26.721,-26.517,-27.117,-28.398,-27.897,-26.451,-25.636,-28.874,-30.168,-29.761,-5.07,-4.807,-5.455,-5.671,-3.283,-2.885,-3.814,-18.443,-17.823,-16.941,-15.866,-19.032,-20.135,-19.912,-8.855,-7.804,-6.679,-6.122,-7.312,-8.561,-9.163,-12.31,-11.187,-10.953,-9.827,-11.647,-12.385,-13.205,0.495,1.453,2.244,2.608,2.377,1.486,0.773,-0.719,-2.14,-2.646,-3.853,-2.312,-0.978,-0.029],"z":[20.147,19.908,18.522,18.083,21.016,21.256,21.239,15.824,16.815,17.938,18.513,17.313,17.038,15.687,23.699,23.91,24.24,24.996,25.108,24.962,24.593,20.878,19.827,18.492,17.789,19.763,21.19,21.621,5.746,6.106,7.487,8.086,5.01,3.835,4.401,26.588,26.926,26.608,26.083,28.429,28.894,27.784,21.322,20.454,18.983,18.087,21.001,22.466,22.481,31.059,30.122,29.035,27.855,31.028,32.286,32.451,14.868,14.794,14.61,15.07,13.615,13.627,13.842,17.849,18.761,18.403,19.275,18.803,17.463,17.248,12.409,12.962,13.584,13.67,13.971,14.385,13.075,10.763,11.683,12.118,13.274,10.836,10.033,9.589,5.977,4.667,3.571,3.798,4.9,6.354,6.943,1.343,0.394,0.711,-0.196,0.589,0.816,1.795,9.487,8.349,8.461,7.451,8.344,8.798,9.962,-10.483,-10.191,-11.31,-11.041,-9.918,-10.708,-10.428,-9.542,-8.525,-8.964,-8.144,-8.426,-8.669,-9.853,-3.541,-4.472,-4.865,-4.801,-5.664,-4.999,-3.958,-1.965,-2.388,-1.299,-1.461,-3.337,-3.906,-2.678,11.742,11.916,11.25,11.067,13.439,13.934,13.043],"type":"scatter3d","mode":"markers","name":"PRO","marker":{"color":"rgba(246,208,97,1)","line":{"color":"rgba(246,208,97,1)"}},"textfont":{"color":"rgba(246,208,97,1)"},"error_y":{"color":"rgba(246,208,97,1)"},"error_x":{"color":"rgba(246,208,97,1)"},"line":{"color":"rgba(246,208,97,1)"},"frame":null},{"x":[-6.506,-6.605,-7.334,-8.09,-7.333,-8.734,-10.093,-11.453,-11.566,-10.635,-12.438,-12.273,-4.337,-4.73,-4.44,-3.934,-6.224,-6.67,3.151,4.204,5.054,5.764,3.592,2.878,5.051,5.806,7.289,7.792,5.618,6.055,14.129,14.875,14.428,15.249,14.674,15.251,25.272,26.662,27.597,28.725,27.103,26.92,19.785,20.977,20.61,19.554,22.034,21.592,-0.791,-0.804,-1.22,-1.278,-1.48,-1.511,-2.411,-2.454,-0.158,-0.29,0.152,0.96,10.766,10.943,12.266,13.023,9.773,9.844,11.372,11.065,11.184,10.473,12.022,13.376,0.432,-0.391,0.094,-0.719,-0.351,-1.351,-7.23,-6.997,-5.529,-4.979,-7.878,-9.247,0.859,1.173,1.209,2.067,0.113,0.027,1.446,1.447,0.902,0.913,0.121,0.127,-0.792,-0.766,2.025,2.048,2.998,2.721,-15.72,-15.046,-14.676,-13.532,-15.934,-16.193,-2.215,-1.869,-2.674,-3.215,-0.365,-0.006,-5.655,-5.469,-6.605,-7.367,-4.149,-3.956,4.791,6.237,7.063,8.228,6.544,5.861,4.814,5.481,4.971,4.622,6.998,7.682,4.896,4.898,4.431,4.434,2.996,3.003,2.683,2.698,4.535,4.527,5.874,4.143,3.671,2.754,3.432,4.665,2.326,3.421,4.681,5.077,4.69,4.225,6.59,7.276,1.328,1.079,1.73,1.15,1.666,1.698,-0.82,-0.645,-1.234,-0.891,0.846,1.443,-2.161,-2.744,-3.327,-3.134,-3.767,-4.835,-0.994,-0.47,0.79,0.996,-0.17,-1.366,-11.028,-11.471,-12.858,-13.298,-11.462,-12.219,-5.253,-6.587,-7.558,-7.277,-7.078,-7.248,-5.676,-5.676,-6.524,-6.492,-6.856,-6.834,-6.95,-6.92,-7.82,-7.774,-7.55,-8.523,-12.339,-11.985,-13.146,-12.955,-11.638,-11.18,-1.836,-0.544,-0.471,0.596,-0.351,-1.453,8.873,10.302,10.606,11.463,11.062,12.463,9.018,7.722,6.542,5.536,7.692,8.611,2.057,1.899,0.717,-0.104,3.161,4.251,24.933,24.935,25.942,25.938,25.718,25.717,26.643,26.636,25.934,25.903,24.649,26.252,29.353,29.45,30.392,30.041,29.912,28.971,28.133,27.064,27.647,27.198,26.196,25.14,27.341,26.803,27.742,27.457,26.392,25.93,10.488,9.504,10.044,9.27,8.855,9.833,18.432,18.288,19.673,19.965,17.434,17.185,20.523,20.524,21.879,21.872,22.698,22.671,23.392,23.305,22.587,22.603,21.865,23.886],"y":[6.733,8.125,8.943,8.396,8.231,8.081,11.435,11.863,13.332,13.905,11.063,11.369,23.691,25.048,25.282,26.342,25.24,26.527,25.496,24.783,23.859,22.997,23.977,24.817,24.083,23.232,23.045,21.926,23.687,25.016,14.078,13.709,12.322,11.505,14.745,15.99,10.714,10.426,11.556,11.643,9.116,9.154,18.951,19.671,20.647,20.518,18.692,18.055,22.133,22.142,23.082,23.072,22.372,22.349,22.72,22.665,24.17,24.226,24.807,23.764,14.546,13.165,12.989,12.066,12.707,11.305,2.846,2.303,3.391,3.376,1.153,1.585,13.886,13.381,13.96,14.354,11.85,11.32,10.289,11.135,11.532,11.475,12.383,12.04,8.689,7.54,8.043,7.641,6.449,6.013,1.879,1.876,2.186,2.193,3.503,3.505,3.714,3.725,2.247,2.298,3.219,1.062,-0.269,0.198,-0.972,-1.087,1.185,2.328,-16.67,-16.672,-17.757,-17.534,-16.914,-16.997,-17.285,-18.001,-18.973,-18.785,-18.78,-19.538,15.892,16.133,15.052,14.848,17.483,18.533,-9.832,-11.014,-12.268,-13.266,-10.866,-11.944,-12.19,-12.194,-13.3,-13.314,-13.711,-13.716,-14.903,-14.904,-12.927,-12.97,-12.621,-14.074,5.572,5.398,4.567,4.517,6.752,7.437,3.834,4.564,3.735,2.595,4.783,3.542,-11.439,-12.043,-13.431,-14.396,-11.157,-11.832,-19.693,-20.481,-19.862,-20.293,-20.76,-21.373,-18.917,-18.26,-19.191,-18.946,-17.193,-17.752,-21.408,-22.546,-22.124,-22.538,-23.715,-24.175,-19.426,-20.205,-20.822,-21.655,-19.333,-18.148,-27.774,-28.323,-27.729,-26.687,-27.992,-26.59,-5.572,-5.563,-4.935,-4.902,-3.494,-3.472,-2.609,-2.572,-5.734,-5.695,-7.107,-5.892,-3.813,-3.017,-3.021,-3.289,-1.58,-0.849,-5.764,-6.406,-7.716,-8.115,-6.694,-7.427,-16.556,-16.781,-18.284,-18.669,-16.188,-16.252,-3.77,-3.327,-4.093,-3.493,-3.421,-2.505,-3.507,-2.541,-1.642,-1.295,-1.697,-2.49,-25.463,-25.462,-26.323,-26.327,-26.438,-26.44,-26.751,-26.774,-27.708,-27.716,-28.301,-27.647,-7.48,-8.935,-9.409,-10.292,-9.457,-9.105,-6.509,-5.555,-4.173,-3.152,-6.067,-5.169,-0.692,0.459,1.659,2.736,0.049,1.166,-25.873,-26.64,-27.263,-27.651,-27.717,-28.567,-10.028,-9.525,-9.345,-8.3,-10.493,-9.978,-10.359,-10.355,-10.333,-10.312,-9.175,-9.143,-8.477,-8.393,-11.658,-11.626,-12.749,-11.628],"z":[2.881,2.452,3.516,4.327,1.109,1.261,4.861,4.567,4.947,5.524,5.416,6.795,14.736,15.088,16.578,16.968,14.803,15.2,19.709,20.424,19.548,20.074,21.577,22.471,18.235,17.316,17.638,17.565,15.862,15.674,20.761,21.958,22.422,22.842,23.07,22.714,12.796,13.125,12.702,13.18,12.464,11.055,22.689,22.254,21.145,20.522,21.737,20.548,5.637,5.631,4.613,4.609,3.284,3.284,2.552,2.552,4.419,4.415,5.648,3.94,-0.077,-0.506,-1.245,-0.95,-1.38,-1.592,7.991,9.309,10.38,11.389,9.633,9.588,23.297,24.384,25.724,26.567,24.405,25.258,13.459,12.292,12.138,11.036,12.342,12.188,8.791,7.946,6.5,5.712,8.098,9.445,40.128,40.127,41.445,41.448,41.417,41.42,42.219,42.24,42.488,42.469,42.148,42.617,29.264,30.467,31.373,31.817,31.227,30.435,27.369,25.95,25.227,24.137,25.779,24.409,22.34,21.083,20.767,19.813,21.124,19.941,31.51,31.519,32.211,31.866,32.175,31.514,23.925,23.401,24.13,23.497,23.562,22.953,25.456,25.455,26.287,26.263,25.957,25.914,25.877,25.783,27.768,27.75,28.125,28.55,-4.694,-5.815,-6.899,-6.979,-6.385,-6.963,-10.526,-11.725,-12.945,-12.814,-11.727,-11.812,-16.548,-17.854,-17.894,-18.409,-18.959,-20.206,-13.243,-12.028,-10.76,-9.656,-11.802,-12.935,-10.902,-9.733,-8.668,-7.48,-10.145,-10.876,2.535,3.28,4.043,5.183,2.338,1.729,12.342,13.492,13.288,14.082,14.75,14.562,2.378,2.584,1.57,0.97,3.994,4.163,-12.891,-12.904,-13.897,-13.92,-13.504,-13.505,-14.36,-14.348,-14.087,-14.192,-14.333,-13.007,0.308,1.477,2.467,3.652,1.078,2.205,9.683,9.904,9.115,8.631,11.394,11.905,-5.987,-6.139,-6.294,-7.097,-4.947,-5.154,-7.931,-8.441,-7.838,-7.45,-9.965,-10.532,-1.567,-0.482,-0.844,0.009,-0.31,0.126,3.59,3.593,2.983,2.982,1.473,1.473,0.72,0.724,3.639,3.622,3.579,4.992,-5.209,-5.185,-4.081,-3.293,-6.548,-7.553,15.987,16.258,16.579,16.04,17.411,17.706,14.303,15.006,15.104,14.575,16.421,17.161,-4.909,-4.142,-2.852,-1.978,-5.023,-5.594,15.246,16.604,17.236,17.822,17.433,18.73,17.087,17.088,17.628,17.64,17.063,17.071,17.802,17.817,17.338,17.362,17.884,17.963],"type":"scatter3d","mode":"markers","name":"SER","marker":{"color":"rgba(237,202,125,1)","line":{"color":"rgba(237,202,125,1)"}},"textfont":{"color":"rgba(237,202,125,1)"},"error_y":{"color":"rgba(237,202,125,1)"},"error_x":{"color":"rgba(237,202,125,1)"},"line":{"color":"rgba(237,202,125,1)"},"frame":null},{"x":[18.35,18.1,19.262,19.388,16.796,16.888,15.593,22.709,22.156,20.82,20.579,23.147,23.511,24.404,2.391,2.867,3.458,4.571,1.72,1.172,2.222,1.411,1.955,1.664,1.385,3.497,4.167,3.884,-7.787,-8.857,-8.646,-7.756,-10.213,-10.191,-10.488,17.019,17.859,16.983,15.757,18.336,17.218,19.021,10.595,9.695,8.366,7.295,10.388,11.556,9.452,-7.996,-7.101,-7.847,-7.578,-6.132,-5.303,-5.249,-10.245,-10.203,-11.62,-11.828,-9.313,-9.777,-7.86,-10.781,-10.318,-9.751,-8.718,-11.461,-12.026,-10.941,-10.408,-9.955,-8.553,-7.663,-10.924,-12.221,-10.423,6.808,6.434,5.582,4.451,7.677,8.355,7.281,4.11,2.927,1.775,0.607,3.275,4.29,2.055,0.347,1.465,0.966,1.403,2.553,3.048,3.712,9.192,10.381,10.718,10.106,11.597,11.889,11.307,-2.191,-3.169,-4.362,-5.418,-2.529,-1.978,-1.429,-5.417,-4.243,-3.786,-4.601,-4.51,-5.66,-4.753,-10.135,-11.454,-11.661,-12.713,-11.626,-11.372,-13.036,9.882,10.104,9.623,10.261,9.385,7.96,9.836,12.926,14.198,14.072,12.979,14.518,13.479,14.609,14.682,14.283,15.325,16.028,12.894,12.939,11.821,17.755,17.355,18.338,18.725,15.923,15.009,15.502,22.014,21.275,21.482,20.509,21.674,21.281,21.009,19.494,20.622,21.926,23.021,20.359,20.092,19.172,29.327,29.971,30.432,30.276,31.165,30.729,31.731,30.967,31.427,30.269,30.407,32.013,33.136,32.45,23.899,23.992,25.439,25.929,23.129,21.747,23.352,28.168,28.959,28.95,29.96,28.491,28.73,29.242,15.974,14.905,14.357,14.096,13.758,14.273,12.654,9.711,8.354,8.091,9.009,8.11,8.993,8.362],"y":[5.021,6.151,7.15,7.942,6.904,7.423,5.972,14.175,14.142,14.874,15.85,14.761,16.076,13.907,15.481,16.672,17.661,18.16,17.362,16.449,18.6,14.06,14.603,16.097,16.574,14.409,15.163,12.939,13.766,12.801,11.594,11.602,13.428,13.774,14.68,-3.207,-2.353,-1.934,-2.107,-1.093,-0.244,-1.484,2.397,3.157,2.427,3.03,3.497,4.289,4.269,3.384,2.646,1.793,1.883,1.755,2.59,0.917,-3.948,-4.605,-4.821,-5.584,-3.825,-2.477,-3.812,-2.26,-2.391,-3.802,-3.968,-2.09,-0.81,-2.05,-4.806,-6.196,-6.333,-6.937,-7.153,-7.083,-8.597,-14.524,-15.548,-16.648,-16.902,-16.162,-15.15,-17.302,-16.653,-16.069,-15.848,-15.978,-14.745,-14.992,-14.17,3.334,4.271,5.705,6.627,3.924,2.598,4.916,11.079,11.367,10.241,9.172,11.577,10.351,12.663,-16.15,-15.757,-16.719,-16.393,-15.669,-16.943,-14.614,-26.179,-26.947,-26.633,-26.335,-28.463,-28.818,-28.838,2.746,2.563,1.107,0.522,3.47,4.831,3.356,-19.134,-20.579,-21.137,-22.015,-21.383,-21.246,-20.925,-11.309,-10.605,-9.523,-9.285,-9.898,-8.95,-10.905,-27.026,-28.422,-29.035,-28.309,-28.576,-28.062,-27.841,-28.426,-27.584,-26.435,-26.188,-27.038,-28.14,-26.145,-11.728,-11.227,-9.727,-8.968,-12.007,-13.378,-11.412,1.249,0.463,0.84,0.501,-1.072,-1.441,-1.465,-11.687,-12.932,-12.842,-13.791,-13.241,-13.213,-14.617,-11.684,-11.447,-11.621,-12.29,-10.022,-9.891,-9.73,-14.661,-13.322,-12.923,-11.917,-13.199,-13.336,-11.85,-14.449,-14.592,-13.333,-13.002,-15.824,-17.006,-15.943,-14.638,-15.636,-15.793,-16.908,-15.259,-15.165,-16.311,-25.529,-25.629,-24.597,-23.898,-27.028,-27.227,-28.113],"z":[15.73,14.832,14.79,13.854,15.2,16.535,15.093,21.373,22.72,22.814,22.097,23.73,23.294,23.835,-3.707,-4.401,-3.398,-3.579,-5.175,-6.131,-5.9,25.905,27.151,27.299,28.4,27.276,26.259,27.162,21.827,22.048,21.134,20.281,21.669,20.276,22.499,34.595,35.426,36.607,36.565,34.666,34.376,33.36,37.81,36.941,36.689,36.776,35.595,35.85,34.68,27.855,26.965,25.938,24.731,27.782,28.604,26.864,16.118,17.418,17.968,18.914,18.428,18.558,17.952,32.348,33.722,33.942,34.592,34.709,34.392,36.144,33.365,33.464,32.86,33.462,32.72,33.324,32.764,30.892,29.921,30.556,30.119,29.25,28.497,28.32,33.491,34.102,33.118,33.49,34.811,35.792,35.51,26.194,26.267,26.051,26.733,25.219,25.463,25.293,30.635,29.839,28.859,28.891,30.757,31.441,31.785,12.293,11.276,11.224,10.67,9.871,9.501,9.852,8.039,7.644,6.224,5.348,7.772,6.992,9.23,-5.704,-6.301,-6.718,-6.464,-7.541,-7.171,-8.119,-5.565,-5.659,-6.998,-7.576,-4.525,-4.645,-3.14,-11.066,-10.966,-9.889,-9.366,-12.299,-12.586,-13.441,17.484,17.33,16.401,15.689,16.663,15.325,17.461,9.033,7.909,7.65,6.508,8.123,8.22,6.96,-3.285,-2.131,-1.907,-1.806,-0.852,-0.993,0.383,1.242,1.744,1.026,1.482,1.642,0.283,2.506,-0.626,-0.226,1.229,2.001,-1.137,-2.501,-0.828,1.604,2.966,3.952,4.98,3.098,2.219,4.531,19.951,20.527,20.825,20.308,21.799,21.442,22.488,19.841,18.63,17.754,17.13,17.827,18.603,16.518,2.03,2.012,0.585,0.13,2.99,4.326,2.961,1.522,2.049,3.142,3.585,2.645,3.757,1.599],"type":"scatter3d","mode":"markers","name":"THR","marker":{"color":"rgba(227,195,149,1)","line":{"color":"rgba(227,195,149,1)"}},"textfont":{"color":"rgba(227,195,149,1)"},"error_y":{"color":"rgba(227,195,149,1)"},"error_x":{"color":"rgba(227,195,149,1)"},"line":{"color":"rgba(227,195,149,1)"},"frame":null},{"x":[-3.99,-2.828,-3.096,-2.195,-2.421,-1.086,-0.832,0.176,0.506,1.148,0.577,2.501,1.922,2.867,26.996,26.863,27.993,28.543,25.538,24.319,24.262,22.969,22.959,22.144,22.376,20.755,20.992,20.198,16.622,16.02,16.97,17.639,14.75,13.695,13.32,12.853,12.295,11.987,12.744,11.019,11.78,10.931,-10.622,-10.69,-10.664,-11.063,-9.592,-8.215,-7.308,-7.564,-6.125,-6.255,-7.959,-5.334,-7.044,-5.745,2.543,1.257,0.906,0.262,0.142,-1.127,-2.303,-1.332,-3.224,-2.656,-0.524,-3.192,-1.056,-2.379,-13.651,-13.482,-13.116,-12.637,-12.322,-12.448,-13.565,-11.404,-13.283,-11.964,-10.047,-11.216,-9.302,-9.892,15.814,15.857,15.46,15.862,17.264,17.788,18.782,17.367,19.008,18.153,16.401,18.005,16.252,17.052,17.567,18.194,19.407,20.279,17.18,16.36,16.656,15.08,15.634,14.655,14.249,13.433,13.032,12.637],"y":[20.97,21.799,23.254,23.963,21.697,22.32,23.446,21.891,23.749,22.813,20.819,22.698,20.703,21.638,14.725,15.837,15.727,14.646,15.739,16.077,16.504,16.056,16.756,16.49,15.719,16.597,15.825,16.262,-6.561,-5.313,-4.521,-5.093,-5.592,-6.331,-7.627,-5.814,-7.949,-6.855,-4.568,-6.694,-4.406,-5.465,-2.366,-3.807,-4.596,-5.76,-4.282,-3.835,-4.502,-2.641,-3.802,-2.656,-1.561,-1.632,-0.542,-0.587,-10.629,-11.308,-12.013,-13.06,-10.341,-11.051,-11.041,-11.915,-11.85,-12.399,-12.332,-13.281,-13.208,-13.673,-22.671,-23.936,-25.008,-24.68,-23.844,-22.83,-22.145,-22.42,-21.336,-21.486,-22.756,-20.881,-22.153,-21.227,-24.551,-25.159,-26.63,-27.393,-25.039,-23.645,-23.098,-22.623,-21.798,-21.479,-22.561,-20.284,-21.369,-20.248,2.537,3.289,2.561,3.193,3.651,2.507,1.68,2.092,0.782,1.011,2.532,0.363,1.888,0.815],"z":[14.817,14.535,14.917,15.372,13.065,12.806,12.075,13.335,12.128,12.895,14.144,13.238,14.485,14.033,13.2,14.139,15.154,15.356,14.897,14.109,12.812,14.587,12.456,13.525,15.812,13.649,15.936,14.858,33.436,33.904,34.791,35.649,34.71,33.971,34.161,32.936,33.313,32.547,32.3,31.549,31.307,30.943,13.937,13.724,15.034,15.069,12.765,13.116,13.883,12.666,13.933,13.195,11.864,12.946,11.616,12.158,-14.219,-14.12,-15.426,-15.414,-13.725,-13.364,-14.06,-12.239,-13.44,-12.321,-11.172,-11.375,-10.232,-10.341,10.776,10.058,11.09,12.18,9.053,7.943,7.566,7.048,6.487,6.15,6.916,5.129,5.9,5.022,19.831,18.503,18.489,19.368,17.905,17.823,18.584,16.914,18.202,17.179,15.898,16.469,15.188,15.481,-0.203,0.88,1.468,2.069,1.973,2.473,3.519,1.98,3.713,2.782,0.941,2.579,0.738,1.555],"type":"scatter3d","mode":"markers","name":"TRP","marker":{"color":"rgba(212,190,160,1)","line":{"color":"rgba(212,190,160,1)"}},"textfont":{"color":"rgba(212,190,160,1)"},"error_y":{"color":"rgba(212,190,160,1)"},"error_x":{"color":"rgba(212,190,160,1)"},"line":{"color":"rgba(212,190,160,1)"},"frame":null},{"x":[14.835,15.261,16.666,16.851,14.29,12.832,12.096,12.186,10.75,10.841,10.131,8.803,21.904,21.645,22.212,22.287,20.141,19.572,19.848,18.795,19.37,18.309,18.603,18.126,17.449,18.777,19.946,21.044,18.759,17.864,18.401,16.479,17.58,15.649,16.204,15.388,11.643,12.553,12.9,14.071,12.057,12.413,12.448,12.746,12.812,13.11,13.145,13.537,-6.844,-7.314,-8.711,-8.901,-7.139,-5.69,-4.661,-5.347,-3.329,-4.012,-3.011,-1.688,-7.935,-7.736,-9.056,-9.145,-6.61,-5.307,-4.567,-4.816,-3.37,-3.618,-2.9,-1.717,-13.944,-14.225,-15.256,-15.614,-12.916,-11.729,-10.93,-11.433,-9.867,-10.379,-9.6,-8.562,-20.364,-20.414,-19.741,-19.607,-21.868,-22.594,-23.898,-21.954,-24.545,-22.591,-23.884,-24.508,2.089,1.041,0.778,-0.048,1.331,1.057,-0.234,2.087,-0.492,1.843,0.551,0.317,0.652,-0.144,0.509,1.648,-0.267,-1.31,-1.133,-2.485,-2.101,-3.456,-3.258,-4.219,3.261,2.663,1.346,0.274,2.405,3.595,3.857,4.462,4.951,5.563,5.799,6.875,-8.342,-9.333,-10.113,-11.091,-8.665,-8.195,-9.112,-6.84,-8.692,-6.41,-7.344,-6.924,-15.296,-16.441,-16.939,-17.423,-17.602,-17.238,-16.685,-17.426,-16.325,-17.069,-16.517,-16.151,-11.044,-10.804,-11.081,-11.761,-9.384,-9.247,-9.642,-8.73,-9.522,-8.608,-9.004,-8.878,28.123,28.306,28.697,28.177,29.26,28.704,27.635,29.248,27.122,28.742,27.678,27.154,15.357,15.394,16.603,17.575,15.544,14.484,13.134,14.828,12.155,13.857,12.523,11.554,25.422,26.723,27.707,28.791,26.514,25.28,24.076,25.298,22.918,24.147,22.963,21.829,23.742,22.604,21.413,20.318,23.049,22.956,21.781,24.061,21.711,24.005,22.826,22.767,27.83,27.726,27.858,27.652,26.42,26.395,27.238,25.527,27.22,25.5,26.35,26.335,23.49,23.225,22.793,21.833,22.151,22.678,23.036,22.819,23.519,23.302,23.649,24.124],"y":[4.873,4.098,4.571,5.215,4.318,4.113,5.125,2.909,4.943,2.715,3.737,3.554,8.175,9.204,10.56,11.476,9.325,8.115,7.876,7.188,6.743,6.046,5.834,4.715,10.534,10.525,10.973,10.437,11.313,10.7,9.953,10.862,9.384,10.298,9.562,9.013,7.078,6.02,4.884,4.529,5.422,6.238,7.633,5.613,8.382,6.349,7.732,8.453,2.294,1.274,1.438,1.218,-0.132,-0.56,0.383,-1.904,0.002,-2.297,-1.336,-1.703,1.838,0.595,-0.165,-1.207,-0.224,0.539,0.91,0.893,1.614,1.597,1.954,2.652,-8.656,-9.581,-9.109,-9.86,-9.896,-9.976,-8.856,-11.155,-8.911,-11.223,-10.101,-10.179,1.863,2.516,3.876,4.379,2.569,1.253,1.214,0.04,0.003,-1.176,-1.186,-2.386,-15.517,-15.294,-16.462,-16.348,-14.041,-12.759,-12.437,-11.868,-11.259,-10.687,-10.389,-9.23,-19.054,-19.196,-20.241,-20.074,-17.86,-16.939,-16.37,-16.652,-15.541,-15.824,-15.274,-14.456,-18.336,-17.347,-17.926,-17.422,-16.037,-15.529,-16.004,-14.578,-15.545,-14.113,-14.6,-14.136,-16.024,-15.516,-16.627,-16.369,-14.614,-13.313,-12.339,-13.068,-11.153,-11.883,-10.936,-9.771,-3.989,-4.138,-5.545,-5.807,-3.276,-1.818,-1.342,-0.918,-0.008,0.425,0.872,2.196,0.994,0.569,1.559,1.228,0.026,-1.427,-1.88,-2.352,-3.221,-3.692,-4.124,-5.455,-19.554,-18.203,-17.085,-15.973,-18.219,-18.925,-19.818,-18.708,-20.476,-19.363,-20.245,-20.888,-2.246,-1.301,-0.392,-0.434,-2.059,-3.117,-2.769,-4.466,-3.739,-5.443,-5.071,-6.033,3.107,2.693,2.216,1.728,1.589,1.812,1.181,2.704,1.439,2.968,2.334,2.603,3.152,3.915,3.102,3.64,4.956,4.544,4.754,4.014,4.45,3.708,3.93,3.636,-12.616,-11.398,-10.088,-9.02,-11.375,-12.327,-12.141,-13.415,-13.016,-14.297,-14.091,-14.97,-1.775,-0.396,0.4,0.036,-0.333,-0.631,-1.929,0.383,-2.207,0.113,-1.183,-1.464],"z":[20.911,19.747,19.362,18.327,18.581,18.941,19.564,18.662,19.901,18.996,19.614,19.939,11.711,10.704,11.098,10.276,10.443,9.749,8.404,10.441,7.761,9.805,8.464,7.821,-2.024,-2.623,-1.767,-1.9,-3.931,-4.983,-6.032,-4.929,-7.002,-5.891,-6.925,-7.89,-3.108,-2.662,-3.612,-3.757,-1.345,-0.121,-0.164,1.075,0.953,2.195,2.13,3.236,37.725,36.789,36.201,35.003,37.382,37.533,37.558,37.647,37.692,37.783,37.803,37.912,16.277,15.529,15.604,16.259,16.169,16.312,15.188,17.571,15.309,17.706,16.571,16.689,18.32,19.413,20.425,21.332,20.144,19.215,18.976,18.534,18.072,17.634,17.406,16.513,17.496,16.195,16.047,14.935,15.713,15.897,16.383,15.639,16.616,15.868,16.358,16.608,31.869,30.876,29.937,29.031,30.052,30.791,31.212,31.078,31.904,31.768,32.179,32.874,20.68,19.465,18.568,18.135,18.728,19.321,20.582,18.629,21.138,19.177,20.43,20.975,10.125,11.021,11.531,11.204,10.261,9.471,8.184,10.009,7.452,9.284,8.008,7.283,12.098,11.148,10.456,9.758,10.11,10.697,11.097,10.904,11.697,11.5,11.893,12.478,7.704,8.599,8.894,9.991,8.099,7.947,6.762,8.994,6.618,8.863,7.67,7.53,6.627,8.009,9.128,10.099,8.166,7.782,6.524,8.68,6.172,8.341,7.085,6.742,-1.857,-1.328,-2.294,-2.181,-0.127,1.093,0.983,2.356,2.095,3.477,3.335,4.432,-1.839,-0.726,-0.956,-0.201,0.598,0.829,0.947,0.929,1.159,1.142,1.254,1.446,2.581,3.101,2.045,2.378,4.141,4.984,4.67,6.055,5.399,6.79,6.456,7.179,-9.955,-10.453,-10.955,-11.107,-11.494,-12.953,-13.685,-13.621,-15.045,-14.985,-15.69,-17.037,17.73,16.926,17.696,17.12,16.131,14.965,13.872,14.951,12.792,13.877,12.803,11.751,19.799,19.408,20.633,21.311,18.32,16.933,16.565,15.988,15.293,14.714,14.374,13.111],"type":"scatter3d","mode":"markers","name":"TYR","marker":{"color":"rgba(196,184,169,1)","line":{"color":"rgba(196,184,169,1)"}},"textfont":{"color":"rgba(196,184,169,1)"},"error_y":{"color":"rgba(196,184,169,1)"},"error_x":{"color":"rgba(196,184,169,1)"},"line":{"color":"rgba(196,184,169,1)"},"frame":null},{"x":[-7.303,-5.886,-5.7,-4.896,-5.119,-3.673,-5.149,16.63,16.77,16.102,16.733,16.144,16.097,16.951,14.822,14.041,14.462,14.455,12.524,11.697,12.129,25.149,23.732,23.149,23.131,23.519,22.065,24.425,11.767,10.718,9.323,8.427,10.941,9.763,12.237,0.322,-1.027,-1.611,-2.776,-1.059,-2.466,-0.606,17.055,16.514,15.525,14.509,15.781,15.168,16.738,3.179,3.468,3.644,3.034,4.738,5.189,4.456,4.452,4.696,3.397,3.118,5.685,5.791,7.052,-6.855,-7.006,-6.718,-5.977,-8.391,-9.534,-8.443,1.954,3.167,4.37,5.171,3.396,4.778,2.331,15.022,15.482,16.186,15.796,14.28,14.725,13.612,-10.113,-10.129,-9.227,-9.652,-9.609,-9.446,-10.571,-7.998,-6.995,-7.37,-7.204,-5.602,-4.563,-5.181,-11.262,-11.453,-10.301,-10.532,-11.605,-11.574,-12.906,-9.065,-7.88,-7.611,-7.106,-6.623,-5.391,-6.854,-17.308,-15.967,-15.14,-14.012,-15.99,-14.574,-16.613,3.897,4.591,5.199,6.303,3.628,4.345,3.067,-2.759,-3.51,-4.988,-5.651,-3.35,-4.277,-1.909,-8.125,-7.708,-6.391,-5.449,-7.553,-6.991,-8.9,-2.125,-2.934,-2.05,-2.308,-3.927,-4.63,-4.957,-1.01,-0.075,0.653,0.834,0.938,2.088,0.222,11.663,10.498,9.769,9.351,9.542,8.221,10.201,7.559,7.736,6.369,6.159,8.572,8.641,9.978,5.439,4.084,3.413,2.874,3.225,1.769,3.756,3.501,2.916,3.502,2.772,3.122,2.767,2.271,4.626,3.61,2.706,2.375,2.763,1.639,3.66,1.289,2.348,3.423,3.753,3.025,4.253,2.048,3.965,5.017,4.499,5.212,5.716,6.91,6.164,-7.412,-6.73,-5.424,-4.453,-6.436,-6.07,-7.642,-8.286,-7.238,-6.563,-6.441,-6.187,-5.015,-6.845,-4.456,-3.636,-4.414,-3.873,-3.091,-2.287,-2.223,-9.97,-8.62,-7.574,-7.448,-8.315,-6.864,-9.261,7.546,8.294,7.831,8.655,8.173,8.735,8.93,6.519,5.955,6.376,6.882,4.406,3.826,3.966,11.011,10.738,9.362,8.599,11.852,11.929,11.622,-1.769,-2.963,-3.373,-4.543,-2.732,-3.918,-2.506,15.857,15.206,15.564,16.712,15.706,15.072,15.387,27.834,26.754,25.556,25,26.365,25.108,27.507,25.203,24.078,24.335,23.43,23.718,22.614,23.279,18.465,17.439,16.056,15.368,17.358,16.185,18.668,15.664,14.356,14.25,13.183,13.961,12.587,13.959,30.187,28.934,28.979,28.725,27.727,26.462,27.554,25.587,25.207,25.954,25.368,25.53,25.195,24.755,28.658,29.328,29.971,29.947,30.425,31.061,29.834,25.44,24.403,23.085,23.059,24.825,23.677,26.049,15.991,17.042,17.122,17.208,18.422,19.539,18.38,17.072,17.117,15.898,16.013,17.174,16.951,18.521,10.215,11.606,11.718,12.656,12.124,13.492,12.203,16.899,17.817,19.122,19.685,18.122,19.14,16.844,19.594,20.817,20.61,21.415,21.206,22.345,21.629,19.498,19.148,19.054,19.568,17.814,17.354,18.003],"y":[17.088,17.444,18.966,19.501,16.798,17.291,15.282,8.499,7.83,6.456,5.436,8.658,7.834,9.915,6.441,5.209,4.289,3.067,5.526,4.248,6.477,13.568,13.266,13.076,11.965,11.968,11.849,11.95,19.204,18.193,18.79,18.54,17.038,16.074,16.302,19.871,20.27,21.207,21.072,20.939,21.427,19.945,7.724,7.342,6.198,6.345,8.524,8.075,9.686,10.559,10.518,11.939,12.292,9.675,9.861,8.204,12.751,14.132,14.935,15.596,14.848,16.339,14.197,9.065,8.255,9.061,8.6,7.54,8.538,6.628,11.054,11.859,10.943,11.184,12.733,13.367,13.815,-10.988,-9.586,-9.179,-9.616,-8.648,-7.207,-9.004,3.997,5.225,4.992,5.142,6.437,7.658,6.755,4.567,4.297,3.093,3.13,4.076,3.727,5.331,2.919,4.045,4.16,4.255,5.391,6.581,5.404,4.125,4.247,3.026,3.159,4.574,4.73,5.848,-1.887,-1.55,-1.126,-1.586,-0.398,0.119,-0.879,-6.94,-5.942,-6.57,-6.213,-4.815,-3.815,-4.112,-18.928,-20.05,-19.675,-19.965,-21.32,-22.426,-21.795,-14.251,-12.866,-12.779,-13.512,-12.155,-10.744,-12.091,6.813,7.173,7.521,8.5,6.033,6.335,5.86,6.721,6.941,8.265,9.036,5.77,6.128,4.526,7.218,6.544,5.791,4.645,7.55,6.876,8.118,-6.916,-8.249,-8.872,-10.062,-8.221,-9.614,-7.713,-8.069,-8.546,-8.99,-10.101,-7.44,-7.863,-7.156,-8.137,-8.405,-9.677,-10.519,-7.194,-7.557,-6.023,-5.162,-6.01,-6.647,-7.833,-5.225,-6.107,-4.733,-21.697,-22.519,-21.648,-21.829,-23.439,-24.135,-24.481,-20.708,-19.813,-18.779,-18.388,-19.092,-18.291,-20.105,-25.483,-24.699,-25.414,-25.297,-23.257,-22.36,-22.692,-15.006,-14.003,-13.69,-12.521,-14.443,-13.459,-14.54,-6.151,-5.944,-5.196,-4.291,-7.279,-7.024,-7.971,-7.72,-8.268,-7.195,-6.225,-8.899,-9.37,-10.062,-14.942,-16.179,-17.263,-17.901,-16.682,-18.096,-15.747,-17.443,-18.441,-18.166,-19.053,-18.468,-19.399,-18.918,-5.436,-4.104,-3.594,-3.109,-3.078,-2.837,-1.774,-2.487,-3.307,-3.35,-3.152,-4.739,-5.635,-4.692,-19.011,-19.767,-21.234,-21.555,-19.277,-20.084,-17.797,-16.742,-16.551,-16.008,-14.961,-17.875,-17.686,-18.351,-16.7,-16.309,-14.996,-14.163,-17.445,-16.999,-18.692,-3.032,-2.667,-2.525,-1.522,-3.698,-3.366,-3.712,-3.514,-3.498,-2.483,-1.911,-4.918,-4.902,-5.892,-4.85,-5.269,-6.785,-7.306,-4.88,-5.558,-3.375,-20.584,-21.945,-22.957,-23.931,-22.243,-23.702,-21.294,-4.149,-2.906,-2.277,-1.056,-3.152,-1.835,-3.843,1.637,1.019,0.931,0.498,-0.416,-1.134,-0.351,-15.112,-15.737,-17.241,-18.05,-15.055,-15.821,-13.617,-17.612,-19.014,-19.749,-20.874,-19.146,-20.583,-18.658,-25.385,-24.985,-23.461,-22.948,-25.456,-24.863,-26.985,-14.464,-14.484,-13.777,-13.033,-15.924,-15.896,-16.598,-14,-13.354,-11.833,-11.076,-13.861,-13.03,-15.321,-11.403,-9.985,-9.328,-8.227,-9.805,-8.349,-10.252],"z":[11.772,11.801,11.796,12.567,10.62,10.576,10.756,22.364,23.659,23.622,23.907,24.815,26.105,25.052,23.263,23.198,22.045,22.188,23.111,23.199,24.231,19.568,19.377,20.77,21.301,18.552,18.132,17.327,11.209,11.112,11.301,10.495,12.122,12.091,11.792,6.792,7.182,6.124,5.749,8.576,8.903,9.634,4.326,5.621,5.445,4.767,6.303,7.632,6.518,21.111,22.541,23.097,24.107,22.831,24.271,22.57,22.423,22.841,22.909,23.908,21.885,22.235,21.968,16,14.792,13.527,12.656,14.708,14.68,13.484,5.603,5.485,5.249,4.349,6.744,6.72,6.818,29.801,29.611,30.898,31.973,29.357,29.328,28.045,35.432,34.648,33.437,32.287,35.466,34.571,36.598,33.713,32.686,31.821,30.601,33.33,32.275,34.081,18.941,18.024,17.016,15.811,18.794,17.825,19.582,17.51,16.663,15.776,14.661,17.513,16.624,18.308,27.689,27.217,28.428,28.612,26.185,25.925,24.886,36.664,37.476,38.731,39.138,37.907,38.823,36.685,25.852,25.301,25.215,24.218,26.177,25.69,26.145,19.393,19.217,18.453,18.743,20.59,20.413,21.302,26.631,27.798,29.003,29.71,28.169,29.485,27.058,29.229,30.331,30.118,31.056,30.446,31.373,30.956,30.798,31.362,30.246,30.429,32.048,32.412,33.304,26.43,26.997,27.27,27.004,28.295,28.909,28.001,27.78,28.062,26.755,26.661,28.75,28.832,30.15,25.738,24.425,23.816,23.295,23.473,22.032,23.951,-13.265,-12.651,-13.711,-13.619,-11.62,-11.062,-10.48,8.341,8.929,9.586,10.764,7.872,8.466,7.387,8.818,9.298,10.304,11.229,8.112,8.596,7.07,10.46,9.436,9.126,9.872,9.936,8.767,10.69,-2.813,-2.661,-3.996,-4.372,-1.609,-1.568,-0.232,-10.421,-11.609,-12.699,-13.339,-12.177,-13.447,-11.142,9.467,9.407,9.696,8.95,8.019,7.955,7.744,-1.667,-1.464,-2.441,-3.095,-0.003,0.127,0.933,-2.565,-3.476,-4.919,-5.609,-3.398,-4.47,-2.018,-6.771,-6.234,-6.656,-5.82,-6.589,-8.087,-5.84,-2.654,-2.495,-1.019,-0.689,-3.034,-2.711,-4.533,22.647,21.543,21.727,22.015,20.171,19.057,19.999,-5.21,-6.17,-5.391,-5.726,-6.873,-7.711,-7.759,-4.312,-3.47,-2.719,-2.59,-2.47,-1.505,-3.239,-4.818,-5.794,-5.138,-5.339,-6.958,-7.894,-7.747,-4.335,-3.677,-2.535,-2.303,-3.183,-2.523,-4.356,-4.921,-4.306,-4.139,-3.051,-5.177,-4.668,-5.167,12.095,11.727,12.597,13.067,10.241,9.901,9.332,17.447,17.831,16.59,16.419,18.904,19.316,20.119,10.509,9.688,10.455,11.608,9.248,8.527,8.353,-2.221,-3.025,-2.74,-3.668,-2.79,-3.502,-3.297,-1.464,-1.059,-1.62,-2.108,0.489,0.921,0.994,6.072,5.859,5.987,6.603,4.479,4.19,4.451,14.586,15.725,15.349,16.149,16.202,17.33,16.681,14.126,13.651,13.631,14.188,12.224,11.643,12.287,13.039,12.925,14.304,14.516,12.138,12.172,10.697],"type":"scatter3d","mode":"markers","name":"VAL","marker":{"color":"rgba(179,179,179,1)","line":{"color":"rgba(179,179,179,1)"}},"textfont":{"color":"rgba(179,179,179,1)"},"error_y":{"color":"rgba(179,179,179,1)"},"error_x":{"color":"rgba(179,179,179,1)"},"line":{"color":"rgba(179,179,179,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/electron-microscopy-and-x-ray-crystallography.html" class="archive-item-link">
                Electron Microscopy and X-Ray Crystallography 
              </a>
              <span class="archive-item-date">
                2020-02-13
              </span>
              <div class="article-summary" style="display: none;">
                
                  Learning objectives

To understand the methodological principle of electron microscopy in its different applications (e.g. TEM, SEM, tomography, single-particle), and X-ray crystallography  To be able to evaluate, which method of structural analysis would be most feasible with which kind of biological specimen (e.g. whole cells, viruses, protein complexes, peptides).

To gain a conception of the resolution ranges attainable by EM methods as compared to X-ray crystallography.

To be aware of the computational methods that combine EM maps and crystallogrographic data
<hr>
                
                


<p><strong>Compare the strengths and limitations of Electron Microscopy and X-ray crystallography. Which types of biological specimen can be studied with the respective methods, and how can cryo-EM and X-ray crystallography be combined to obtain novel structural information?</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Being able to solve macromolecular strucutures at atomic resolution, X-ray crystallography has long been the most powerful tool in structural biology. As of Feb 16 2019, 167943 molecular structures have been deposited into the Protein Data Bank (PDB) since its launch in 1971. However, recent improvements in the field of electron microscopy, especially the introduction of the use of cryocooling, have revolutionised the way cellular material is viewed. In conjunction with computational methods, cryo-EM can now solve macromolecular structures at sub-atomic or even atomic resolution. The structures publicised on Electron Microscopy Data Bank (EMDB) has grown exponentially over the last two decades, from only 8 in 2002 to more than 10000 on Feb 16 2020. In this breif narrative, I introduce the basic principles of both methods, their strengths and limitations, and how they can complement each other in modern structural biology studies.</p>
</div>
<div id="x-ray-crystallography" class="section level1">
<h1>X-Ray Crystallography</h1>
<p>In short, structural determination using X-ray crystallography involves making the appropriately-sized crystals of the protein of interest, gaining electron density maps by deciphering X-ray diffraction patterns obtained from these crystals, and filling the map with amino acids (and prosthetic groups or other ligands).</p>
<p>Prior to the X-ray experiment, the protein must be crystallised. This is often the most laborious and time-consuming step, involving numerous trial-and-errors. In modern high throughput methods, a wide range of crystallisation constructs are designed (which involves truncation the protein of interest, expression with a fusion partner, binding with a ligand, use of different detergents, etc.), and robots are used to deploy these crystallisation conditions in multiwell plates and to monitor the growth of crystals. <em>One of the critial factors in the sucess of crystallisation is aqueous solubility, which represents the major bottleneck in solving membrane protein structures <span class="citation">(Parker and Newstead 2016)</span>.</em></p>
<p>In the X-ray experiment, the crytal is mounted on a support which can be rotated in any direction, and is cryocooled in order to minimise radioactive damage and random thermal vibration, thus improving signal-to-noise-ratio (contrast). For every orientation of the crystal, a monochromatic X-ray is applied, and the detector records the position and intensity of the reflections (points) in the diffraction pattern.</p>
<p>Crystallisation is needed for two reasons. First, the diffraction signal given by a single protein molecule is too weak to be detected. Second, incoherent scattering causes serious chemical damage to protein molecules, and if we try to image a single molecule with X-rays, it would be destroyed as soon as it scattered one or two photons and no longer represent the native protein structure. Crystallisation forms an array of a substantial number of protein molecules arranged in the same orientation (so that their diffraction properties are the same) so the radiation damage in a small number of molecules becomes negligible. Every molecule in the array gives the same diffraction signal, thus achieving amplification.</p>
<p>X-rays with wavelengths within the range 0.5-1.6Å are used in crystallography. X-rays above this range are ‘soft X-rays’ that penerate crystals without scattering. Generally, X-rays of shorter wavelengths are more preferable because of two reasons. First, this generally gives better resolution. According to this rearranged form of Bragg’s equation, <span class="math inline">\(\dfrac{n_{max}}{d}\propto\dfrac{1}{\lambda}\)</span>, as wavelength (<span class="math inline">\(\lambda\)</span> decreases), smaller <span class="math inline">\(d\)</span> (distance between crystal lattice planes) can be distinguished with the same <span class="math inline">\(n\)</span> (number of constructive interference). Second, X-rays with shorter <span class="math inline">\(\lambda\)</span> are scattered more, thus producing stronger signals for a fixed amount of sample, which allows crytals of smaller sizes to be studied. However, using X-rays of too short wavelengths (and using too small crystals) have the disadvantage that a greater proportion of proteins would be damaged due to the higher energy of the X-ray and the smaller sample size. Traditional X-rays sources are characteristic radiations from a Cu anode (CuK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=1.54Å) or a Mo anode (MoK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=0.71Å) when bombarded by electrons beams at appropriate potential difference from the cathode. Since the 1960s, synchrotrons (e.g. Diamond Light Source in Oxfordshire) have become available as more intense and X-ray sources. In these giant devices, electrons travel on a circular track in vacuum, emitting intense X-rays in tangential directions. Electrical disturbances in synchrotron allow for production of X-rays with any <span class="math inline">\(\lambda\)</span> within the useful range 0.5-1.6Å, and this makes it possible to use multiple anomalous dispersion (MAD) method to solve the phase problem.</p>
<p>The diffraction patterns only give information on the position amplitude of every reflection, but an additional parameter, phase, is required to do the inverse Fourier transform that would give the electron density map. Single/multiple isomorphous replacements (SIR/MIR) using heavy metals represent the earliest attempts to solve the phase problem. Later, with the advent of synchrotrons, multi-wavelength anomalous dispersion (MAD) became the more popular phasing strategy. Recently, thank to the numerous experimental structures solved previously and publicised in PDB, molecular replacement (MR) has become the most efficient way to solve the phase problem.</p>
<p>After solving the first electron density map using inverse Fourier transform, the structure is refined by optimising the parameters of the model to fit the observations in a iterative process.</p>
<p>Today, abundant software packages are available for data collection and processing, structure solution, refinement and validation.</p>
</div>
<div id="cryo-electron-microscopy" class="section level1">
<h1>Cryo-Electron Microscopy</h1>
<p>Electron microscopy (EM) is mechanistically similar to light microscopy (LM). Electron beams (which is analogous to visible light beams in LM) hit the sample, and their path is regulated by a set of electromagnets (analogous to lens in LM) so that they finally converge onto a plane where an enlarged image of the sample can be detected.</p>
<p>Unstained samples have a very poor singal-to-noise ratio (contrast) under EM, and traditionally heavy metal are applied to improve contrast. However, this often leads to unwanted artefacts and can only achieve resolutions at 20-40Å. High electron dose improves contrast but causes damage to the specimen.</p>
<p>Cryo-EM partially solves this problem by reducing the effect of radiation damage using low temperature. In cryo-electron microscopy, protein solutions are applied onto a support grid, and is then plunge-frozen with liquid ethane. The process is so fast that the water adopts a vitreous form instead of crystallises into ice. The vitrified sample is then maintained at low temperature with liquid nitrogen during storage and EM studies.</p>
<p>Another challenge of cryo-EM is the movement of the particles when being hit by the electron beam, which leads to blurred images on conventional CCD/photographic films (because they are less sensitive and need a long time of exposure). This is solved by the more sensive and faster direct electron detectors which can record movies at a rate of many fps. The traces of molecules recorded in the movies can be computationally processed to give much sharper images. The motion is in part due to the different thermal expansion coefficients between the metal grid and the carbon film, and choosing metals that have similar thermal properties with carbon (e.g. titanium, molybdenum or tungsten instead of copper or gold) can minimise this effect <span class="citation">(Sgro and Costa 2018)</span>.</p>
<p>There are two major strategies of constructing 3D models using cryo-EM, namely electron tomography and single-particle cryo-EM. In electron tomography, the specimen is tilted in all directions and respective EM images are recorded, which are combined (similar to CT) into a 3D model. It is commonly used to visualise one-of-a-kind, structurally heterogeneous entities (such as viruses and whole bacterial cells) at resolutions 50-100Å. Single-particle cryo-EM are usually used to study smaller entities such as the ribosome and proteins. In this approach, a large number (tens or hunders of thousands) of 2D images are extracted from EM images. These heterogeneous low-resolution ‘snapshots’ are computationally sorted and aligned (sometimes called <em>in silico</em> purification), and finally used to synthesise the 3D model using Fourier transform.</p>
</div>
<div id="comparison-of-strengths-and-limitations" class="section level1">
<h1>Comparison of Strengths and Limitations</h1>
<p>The most significant advantage of X-ray crystallography is its resolution. 2.05Å is the median resolution for X-ray crystallographic results in the protein data bank (as of May 19, 2019, according to <a href="https://proteopedia.org/wiki/index.php/Resolution">Proteopedia</a>, but I will do the analysis myself on Monday. I have downloaded all the entire PDB repository.). The protein backbone and most sidechains can be identified unambiguously under this resolution. Strikingly<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>, however, this long unrivalled strength of X-ray crystallography is now challeged by cryo-EM—Wu el al.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> claimed cryo-EM solution of mouse heavy chain apoferritin at 1.75Å!</p>
<p>A major advantage of cryo-EM over X-ray crystallography is the ease and speed of sample preparation, as the proteins need not to be crystallised and only a small amount is needed. Another related merit is its forgiveness of heterogeneity, as robust computational methods can ‘purify’ proteins <em>in silico</em>. Furthermore, cryo-EM are suited for studying membrane proteins and multiprotein supra-assemblies/RNP machines, which are difficult to crystallise in their native states. However, it should be noted that during the process of specimen preparation, delicate protein complexes may become associated so that they no longer represent their <em>in vivo</em> state.</p>
</div>
<div id="combining-x-ray-and-cryo-em-studies" class="section level1">
<h1>Combining X-ray and Cryo-EM studies</h1>
<p>Today, it is common to combine the results of X-ray crystallography and cryo-EM studies for structural determination. There are two major ways in which these two methods can complement each other. First, a low-resolution cryo-EM map can provide an overall shape of the macromolecule, whose sub-components are solved at a high resolution by X-ray crystallograhy and docked onto the EM map. Second, the cryo-EM model may help to solve the phase problem in X-ray crystallography by serving as a search model in molecular replacement (MR).</p>
<p>The docking methods can be classified into two categories: rigid-body docking and flexible docking. Both are used to find the optimal position and orientation of sub-component X-ray structures in the cryo-EM map, but the the latter entails additional algorithms such as normal mode analysis and molecular dynamic simulation that introduce minor conformational changes within stereochemistry limits in X-ray structures to minimise local conformational discrepancy between the X-ray and cryo-EM models. The docking is useful to define the protein location and protein-protein interface within a complex and new interaction modes that are not revealed by X-ray crystallography.</p>
<p>The electron density map (strictly speaking, Coulomb potential density map) obtained by Cryo-EM, albeit at a low resolution, has information about the phase, and thus can be used as an initial phasing model for X-ray studies. Once the search EM map has been positioned, theoretical phases can be calculated by Fourier transform up to the EM model resolution.</p>
</div>
<div id="concluding-remarks" class="section level1">
<h1>Concluding Remarks</h1>
<p>The past decade has witnessed a resolution revolution in single particle cryo-EM, making them another powerful tool in solving biological macromolecules after X-ray crystallography. The different perspectives provided by these two methods is helping us to gain a more complete understanding of molecualr mechanisms that underlie the principle of life.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Cheng-2018">
<p>Cheng, Yifan. 2018. “Single-Particle Cryo-Em—How Did It Get Here and Where Will It Go.” <em>Science</em> 361 (6405). American Association for the Advancement of Science: 876–80. <a href="https://doi.org/10.1126/science.aat4346">https://doi.org/10.1126/science.aat4346</a>.</p>
</div>
<div id="ref-Earl-2017">
<p>Earl, Lesley A, Veronica Falconieri, Jacqueline LS Milne, and Sriram Subramaniam. 2017. “Cryo-Em: Beyond the Microscope.” <em>Current Opinion in Structural Biology</em> 46: 71–78. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002">https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002</a>.</p>
</div>
<div id="ref-Elmlund-2017">
<p>Elmlund, Dominika, Sarah N Le, and Hans Elmlund. 2017. “High-Resolution Cryo-Em: The Nuts and Bolts.” <em>Current Opinion in Structural Biology</em> 46: 1–6. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003">https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003</a>.</p>
</div>
<div id="ref-Jain-2010">
<p>Jain, Deepti, and Valerie Lamour. 2010. “Computational Tools in Protein Crystallography.” <em>Methods in Molecular Biology (Clifton, N.J.)</em> 673. United States: 129–56. <a href="https://doi.org/10.1007/978-1-60761-842-3_8">https://doi.org/10.1007/978-1-60761-842-3_8</a>.</p>
</div>
<div id="ref-Kuhlbrandt-2014">
<p>Kühlbrandt, Werner. 2014. “Microscopy: Cryo-Em Enters a New Era.” <em>eLife</em> 3. eLife Sciences Publications, Ltd: e03678. <a href="https://doi.org/10.7554/eLife.03678">https://doi.org/10.7554/eLife.03678</a>.</p>
</div>
<div id="ref-Milne-2013">
<p>Milne, Jacqueline L S, Mario J Borgnia, Alberto Bartesaghi, Erin E H Tran, Lesley A Earl, David M Schauder, Jeffrey Lengyel, Jason Pierson, Ardan Patwardhan, and Sriram Subramaniam. n.d. “Cryo-Electron Microscopy–a Primer for the Non-Microscopist.” <em>The FEBS Journal</em> 280 (1): 28–45. <a href="https://doi.org/10.1111/febs.12078">https://doi.org/10.1111/febs.12078</a>.</p>
</div>
<div id="ref-Nogales-2018">
<p>Nogales, Eva. 2018. “Cryo-Em.” <em>Current Biology</em> 28 (19): R1127–R1128. <a href="https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016">https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016</a>.</p>
</div>
<div id="ref-Parker-2016">
<p>Parker, Joanne L, and Simon Newstead. 2016. “Membrane Protein Crystallisation: Current Trends and Future Perspectives.” <em>Advances in Experimental Medicine and Biology</em> 922. Springer International Publishing: 61–72. <a href="https://doi.org/10.1007/978-3-319-35072-1_5">https://doi.org/10.1007/978-3-319-35072-1_5</a>.</p>
</div>
<div id="ref-Powell-2017">
<p>Powell, Harold R. n.d. “X-Ray Data Processing.” <em>Bioscience Reports</em> 37 (5). Portland Press Ltd.: BSR20170227. <a href="https://doi.org/10.1042/BSR20170227">https://doi.org/10.1042/BSR20170227</a>.</p>
</div>
<div id="ref-Rhodes-2006a">
<p>Rhodes, Gale. 2006. “An Overview of Protein Crystallography.” In <em>Crystallography Made Crystal Clear</em>, edited by Gale Rhodes, 3rd ed., 7–30. Complementary Science. Burlington: Academic Press. <a href="https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0">https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0</a>.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
<div id="ref-Sgro-2018">
<p>Sgro, Germán G, and Tiago R D Costa. 2018. “Cryo-Em Grid Preparation of Membrane Protein Samples for Single Particle Analysis.” <em>Front Mol Biosci</em> 5. Departamento de Bioquı'mica, Instituto de Quı'mica, Universidade de São Paulo, São Paulo, Brazil.; Department of Life Sciences, Imperial College London, MRC Centre for Molecular Microbiology; Infection, London, United Kingdom.: 74. <a href="https://doi.org/10.3389/fmolb.2018.00074">https://doi.org/10.3389/fmolb.2018.00074</a>.</p>
</div>
<div id="ref-Wang-2017">
<p>Wang, Hong-Wei, and Jia-Wei Wang. 2017. “How Cryo-Electron Microscopy and X-Ray Crystallography Complement Each Other.” <em>Protein Science</em> 26 (1): 32–39. <a href="https://doi.org/10.1002/pro.3022">https://doi.org/10.1002/pro.3022</a>.</p>
</div>
<div id="ref-Whitford-2005a">
<p>Whitford, David. 2005. “Physical Methods of Determining the Three-Dimensional Structure of Proteins.” Book. In <em>Proteins: Structure and Function</em>. Sussex, England: John Wiley &amp; Sons.</p>
</div>
<div id="ref-Xiong-2008">
<p>Xiong, Yong. 2008. “From electron microscopy to X-ray crystallography: molecular-replacement case studies.” <em>Acta Crystallographica Section D</em> 64 (1): 76–82. <a href="https://doi.org/10.1107/S090744490705398X">https://doi.org/10.1107/S090744490705398X</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>I was REALLY shocked when when facing the search results on EMDB website. According to reviews written in 2014, 4.5Å was still the best resolution ever achieved at that time, and in less than 6 years cryo-EM is becoming able to provide atomic resolution as does X-ray crystallography!<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>To be published. Preprint available on <a href="https://www.biorxiv.org/content/10.1101/855643v1.full">bioRxiv</a><a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/bacterial-cell-division.html" class="archive-item-link">
                Bacterial Cell Division 
              </a>
              <span class="archive-item-date">
                2020-02-08
              </span>
              <div class="article-summary" style="display: none;">
                
                  discussing both Gram positive and negative bacteria with different morphologies, how they divide and chose examples of antibiotics, comparing their different effects
<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#preparation-for-cell-division"><span class="toc-section-number">2</span> Preparation for Cell Division</a><ul>
<li><a href="#chromosome-replication"><span class="toc-section-number">2.1</span> Chromosome Replication</a></li>
<li><a href="#segregation"><span class="toc-section-number">2.2</span> Chromosome Segregation</a></li>
</ul></li>
<li><a href="#ftsz-and-the-z-ring"><span class="toc-section-number">3</span> FtsZ and the Z-ring</a><ul>
<li><a href="#z-ring-placement"><span class="toc-section-number">3.1</span> Z-ring Placement</a><ul>
<li><a href="#the-min-system"><span class="toc-section-number">3.1.1</span> The Min system</a></li>
<li><a href="#nuclear-occlusion"><span class="toc-section-number">3.1.2</span> Nuclear Occlusion</a></li>
</ul></li>
<li><a href="#other-key-proteins-in-the-divisome"><span class="toc-section-number">3.2</span> Other Key Proteins in the Divisome</a><ul>
<li><a href="#zipa-and-ftsa"><span class="toc-section-number">3.2.1</span> ZipA and FtsA</a></li>
<li><a href="#downstream-proteins-recruited-by-the-ftsz-ring"><span class="toc-section-number">3.2.2</span> Downstream Proteins Recruited by the FtsZ Ring</a></li>
<li><a href="#ftsn"><span class="toc-section-number">3.2.3</span> FtsN</a></li>
</ul></li>
</ul></li>
<li><a href="#alternative-strategies-for-bacterial-cell-division"><span class="toc-section-number">4</span> Alternative Strategies for Bacterial Cell Division</a><ul>
<li><a href="#alternative-strategies-of-binary-division"><span class="toc-section-number">4.1</span> Alternative strategies of Binary Division</a><ul>
<li><a href="#caulobacter-crescentus"><span class="toc-section-number">4.1.1</span> <em>Caulobacter crescentus</em></a></li>
<li><a href="#campylobacter-jejuni-and-magnetotactic-bacterium"><span class="toc-section-number">4.1.2</span> <em>Campylobacter jejuni</em> and Magnetotactic Bacterium</a></li>
<li><a href="#agrobacterium-tumefaciens"><span class="toc-section-number">4.1.3</span> <em>Agrobacterium Tumefaciens</em></a></li>
</ul></li>
<li><a href="#positive-regulation-in-myxococcus-xanthus"><span class="toc-section-number">4.2</span> Positive Regulation in <em>Myxococcus xanthus</em></a></li>
<li><a href="#cell-division-without-ftsz"><span class="toc-section-number">4.3</span> Cell Division without FtsZ</a></li>
</ul></li>
<li><a href="#concluding-remarks"><span class="toc-section-number">5</span> Concluding Remarks</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>The modes of bacterial cell division vary, but the most common one is binary fission <span class="citation">(Stahl 2019)</span>, and its mechanism, especially in rod-shaped model organisms <em>Escherichia coli</em> and <em>Bacillus subtilis</em>, has been studied extensively. Despite the characterisation of most proteins required for divisome, their coordination at the molecular detail remain largely unclear <span class="citation">(Tsang and Bernhardt 2015)</span>. The question of how bacteria divide is further complicated by variations in proteins involved, and even in the physical nature, among different bacterial species <span class="citation">(Eswara and Ramamurthi 2017)</span></p>
<p>This essay introduces FtsZ as a highly conserved protein involved in septum formation and its regulation in model organsisms, and gives examples on diviations from these rules that mainly comes from studies on nonmodel organisms.</p>
</div>
<div id="preparation-for-cell-division" class="section level1">
<h1><span class="header-section-number">2</span> Preparation for Cell Division</h1>
<p>The chromosome must be replicated once and only once prior to cell devision, and it must be ensured that each daughter cell receives one copy of the chromosome.</p>
<div id="chromosome-replication" class="section level2">
<h2><span class="header-section-number">2.1</span> Chromosome Replication</h2>
<p>Chromosome replication is principally regulated by DnaA in <em>E. coli</em>. DnaA initiates replication by binding its active, GTP-bound form (DnaA-ATP) to <em>OriC</em>. Its inactivation upon initiation of replication ensures one round of replication per cell cycle<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Such mechanisms include competition for <em>OriC</em> binding by SeqA, repression of <em>dnaA</em> expression (also by SeqA), and inactivation of DnaA-ATP by the ATPase HdaA.</p>
</div>
<div id="segregation" class="section level2">
<h2><span class="header-section-number">2.2</span> Chromosome Segregation</h2>
<p>Segregation of chromosomes and plasmids is achieved while they are being replicated by using the Par system in some bacteria such as the budding bacteria <em>Caulobacter crescentus</em>, as illustrated in Figure <a href="#fig:cc-par">2.1</a>. popZ proteins anchors the chromosome at its <em>parS</em> sequence (near <em>oriC</em>) to the old pole, which is mediated by parB. As replication starts, more parB binds to <em>parS</em> sequence of the newly synthesised chromosome, and is pulled to the new cell pole by the ATPase activity of ParA.</p>
<div class="figure"><span id="fig:cc-par"></span>
<img src="../img/cc-par.jpg" alt="The Par system segregates replicated chromosomes in Caulobacter crescentus." width="100%" />
<p class="caption">
Figure 2.1: The Par system segregates replicated chromosomes in Caulobacter crescentus.
</p>
</div>
<p>The mechanism of chromosome segregation in <em>E. coli</em>, which lacks Par, is not well characterised.</p>
</div>
</div>
<div id="ftsz-and-the-z-ring" class="section level1">
<h1><span class="header-section-number">3</span> FtsZ and the Z-ring</h1>
<p>Central to almost all bactrial division is the FtsZ protein. FtsZ is a homolog of tubulin found in most bacteria and some archaea. Like tubulin, FtsZ is a GTPase and can polymerise into filaments. During cell division, FtsZ filaments assemble into a ring (Z-ring) at the site where the septum forms. The FtsZ ring then recruits other proteins of the divisome, allowing division to proceed. The Z-ring contracts as the bacterium divides, forming an invagination which finally divides the bacterium in two.</p>
<p>The GTPase activity accounts for the rapid turnover dynamics of the FtsZ filaments that form the Z-ring. It has also been proposed that FtsZ’s GTPase activity energises the constriction of the ring and the invagination of the cell membrane and the cell wall, but this notion is challenged by the observation of mutants that are severely impoaired in GTP hydrolysis still capable of dividing, and mathematical modelling that shows force generation could be achieved independenly of GTPase activity, but instead with hydrolysis <span class="citation">(Adams and Errington 2009)</span>.</p>
<div id="z-ring-placement" class="section level2">
<h2><span class="header-section-number">3.1</span> Z-ring Placement</h2>
<p>The site of cell division in rod-shaped bacteria is usually at the centre of the cell, which is dependent on the corrent placement of the Z-ring. As shown in Figure <a href="#fig:no-min">3.1</a>, this is typically achieved using two mechanisms, nucleoid occlusion (NO), and the Min system, which are well characterised in <em>E. coli</em> and <em>B. subtilis</em>. It should be noted, however, that there is increasing evidence showing Z-ring positioning may be determined by other factors, and Min and NO systems may primarily function to ensure the efficient utilisation of this site. Such evidence include the observation of precise midcell Z-ring formation in Min- and Noc-null <em>B. subtilis</em> by <span class="citation">Rodrigues and Harry (2012)</span>.</p>
<div id="the-min-system" class="section level3">
<h3><span class="header-section-number">3.1.1</span> The Min system</h3>
<p>The MinC, MinD and MinE proteins interact to prevent Z-ring formation at the poles <span class="citation">(Ghosal et al. 2014; Ramm, Heermann, and Schwille 2019)</span>. MinC is the effector of the system owing to its inhibitory effect on FtsZ ring assembly. MinD is an ATPase which recruit MinC to the membrane. MinCD complex together inhibit Z-ring formation. MinE is an ATPase-activating protein which has an anti-MinCD domain and a topological specificity domain (which confines its anti-MinCD activity only at midcell). The three proteins MinCDE form an oscillation cycle from pole to pole (driven by MinDE), causing the time-averaged concentration of MinCD complexes to be lowest at the midcell, which favours Z-ring formation there. <em>B. subtilis</em> also has a Min system but it is not involved in the placement of the Z-ring.</p>
</div>
<div id="nuclear-occlusion" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Nuclear Occlusion</h3>
<p>Nuclear occlusion prevents Z-ring formation atop of nucleoids. It is mediated by SlmA in <em>E. coli</em> and Noc in <em>B subtilis</em>. Neither are normally essential in their respective organisms, but both are synthetically lethal with mutatations of Min, due to chaotic FtsZ assembly.</p>
<p>According to <span class="citation">Adams, Wu, and Errington (2015)</span>, Noc has a sequence-specific DNA-binding domain and a membrane-associating domain. Noc is thought to oligomerise using their two dimerisation domains, bind to DNA at specific sequences, and then insert into the inner leaflet of the cytoplasmic membrane.</p>
<div class="figure"><span id="fig:no-min"></span>
<img src="../img/no-min.jpg" alt="Negative regulation by nuclear occulation and the Min system ensures formation of the FtsZ ring in E coli." width="100%" />
<p class="caption">
Figure 3.1: Negative regulation by nuclear occulation and the Min system ensures formation of the FtsZ ring in E coli.
</p>
</div>
</div>
</div>
<div id="other-key-proteins-in-the-divisome" class="section level2">
<h2><span class="header-section-number">3.2</span> Other Key Proteins in the Divisome</h2>
<p>A fully functional divisome requires a combination of proteins in addition to the FtsZ ring. Most of these proteins have close association with the Z ring. In <em>E. coli</em>, It was thought that the recruitment of these proteins appears to have a linear dependecy pathway which starts with FtsA/ZipA and ends with FtsN, but a recent review by <span class="citation">Du and Lutkenhaus (2017)</span> shows evidence for the noon-sequential assembly of the divisome.</p>
<div id="zipa-and-ftsa" class="section level3">
<h3><span class="header-section-number">3.2.1</span> ZipA and FtsA</h3>
<p>ZipA is acts to tether the FtsZ ring to the cytoplasmic membrane. It is not widely conserved outside of Gammaproteobacteria, and, in <em>E. coli</em>, its necessity in cell division is bypassed by a gain-of-function mutation in FtsA. FtsA, which is more widely conserved in bacteria, also anchors the Z ring to the membrane and coordinates its initial assembly <span class="citation">(Adams and Errington 2009)</span>.</p>
<p>FtsA is related to actin, and, like actin, it binds ATP and is found to reversibly polymerise into corkscrew-like helices in <em>Streptococcus pneumoniae</em>. Its ATPase activity is reported in <em>B. subtilis</em> but not in <em>S. pneumoniae</em>. Mutagenesis studies suggest that nucleotide binding may be required for FtsA’s ability to interact with itself and with FtsZ, <span class="citation">(Pichoff and Lutkenhaus 2007)</span> but its exact role is unknown.</p>
<p>FtsA differs from actin in that it is missing the 1B domain and it has an unrelated subdomain called 1C. The 1C domain is involved in recruiting downstream components of the divisome to the Z ring.</p>
</div>
<div id="downstream-proteins-recruited-by-the-ftsz-ring" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Downstream Proteins Recruited by the FtsZ Ring</h3>
<p>Upon assembly of the FtsZ ring, other downstream proteins, notably those involved in peptidoglycan biosynthesis, are recruited to form the divisome.</p>
<p>FtsI, also known as penicillin binding protein 3 (PBP3), is one such protein in <em>E. coli</em>. It is required specifically for peptidoglycan synthesis at the septum <span class="citation">(Chen and Beckwith 2001)</span>. Its recruitment is dependent on FtsK, which appears to be a bifunctional protein, with the C-terminal domain facilitating segregation of chromosome and the N-terminal domain carrying out a necessary, but undefined, function in septum development.</p>
<p>A summary of the divisome components is shown in Figure <a href="#fig:ffts-components">3.2</a></p>
<div class="figure"><span id="fig:ffts-components"></span>
<img src="../img/ffts-components.png" alt="Components of the divisome." width="100%" />
<p class="caption">
Figure 3.2: Components of the divisome.
</p>
</div>
</div>
<div id="ftsn" class="section level3">
<h3><span class="header-section-number">3.2.3</span> FtsN</h3>
<p>In <em>E. coli</em>, FtsN is the last component of recruited to the divisome. Once it arrives at the Z ring, the divisome begin to actively synthesising septal peptidoglycan to divide the cell. It has been suggest that FtsA-FtsN interaction may trigger cell constriction in a positive feedback loop <span class="citation">(Tsang and Bernhardt 2015)</span>, but the exact mechanism remain elusive.</p>
</div>
</div>
</div>
<div id="alternative-strategies-for-bacterial-cell-division" class="section level1">
<h1><span class="header-section-number">4</span> Alternative Strategies for Bacterial Cell Division</h1>
<div id="alternative-strategies-of-binary-division" class="section level2">
<h2><span class="header-section-number">4.1</span> Alternative strategies of Binary Division</h2>
<p>Binary division means dividing a bacterial cell into daughter cells with equal volume and physiology, which requires the FtsZ ring to form at the midcell. Apart form the NO and Min systems described above, there are other strategies to achieve this.</p>
<div class="figure"><span id="fig:binary-division-alternative"></span>
<img src="../img/binary-division-alternative.jpg" alt="Alternative strategies of Binary Division" width="100%" />
<p class="caption">
Figure 4.1: Alternative strategies of Binary Division
</p>
</div>
<div id="caulobacter-crescentus" class="section level3">
<h3><span class="header-section-number">4.1.1</span> <em>Caulobacter crescentus</em></h3>
<p>The budding bacteria <em>C. crescentus</em> uses the MipZ gradient to regulate FtsZ ring placement (Figure <a href="#fig:binary-division-alternative">4.1</a> A). MipZ is translocated from the old pole to the new pole during chromosome replication and segregation, by directly interacting with the ParB-<em>parS</em> complex (see section <a href="#segregation">2.2</a>). MipZ displaces FtsZ from polar regions, thus permitting ring formation only at midcell.</p>
</div>
<div id="campylobacter-jejuni-and-magnetotactic-bacterium" class="section level3">
<h3><span class="header-section-number">4.1.2</span> <em>Campylobacter jejuni</em> and Magnetotactic Bacterium</h3>
<p>The amphitrichous<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> bacteria <em>Campylobacter jejuni</em> uses the ATPase FlhG, which is a MinD/ParA homologue and a known regulator of flagellar number, to locate the Z ring placement, by expelling FtsZ from the two longitudinal poles where flagella grow.</p>
<p>By constrast, in the monotrichohus magnetotactic bacterium of the <em>Gammaproteobacteria</em> class, flagellum formation in the daughter cell is dictated by the septum (Figure <a href="#fig:binary-division-alternative">4.1</a> B and C).</p>
</div>
<div id="agrobacterium-tumefaciens" class="section level3">
<h3><span class="header-section-number">4.1.3</span> <em>Agrobacterium Tumefaciens</em></h3>
<p>Most bacteria that exhibit binary division elongate their cells from the middle or at several locations perpenticular to the long axis (e.g. in <em>E. coli</em>, which is mediated by MreB). However, <em>Agrobacterium Tumefaciens</em>, which elongates unidirectionally, can still achhive binary division. As shown in Figure <a href="#fig:binary-division-alternative">4.1</a> D, after division, FtsA and FtsZ stay at the newly formed growth pole and facilitate polar growth. Upon elongation, they relocate to the midcell to initiate division.</p>
</div>
</div>
<div id="positive-regulation-in-myxococcus-xanthus" class="section level2">
<h2><span class="header-section-number">4.2</span> Positive Regulation in <em>Myxococcus xanthus</em></h2>
<p>The NO and Min systems described above are both negative regulatory mechanisms, where regions of low activity of relevant proteins allows for foramtion of FtsZ ring. There is evidence in some nonmodel organisms of positive regulation, where FtsZ ring formation is promoted by presence of certain proteins. This is exemplified by the deltaproteobacterium <em>M. xanthus</em>, in which the ParA-like protein PomZ localises to midcell in an FtsZ-independent manner before FtsZ ring formation.</p>
</div>
<div id="cell-division-without-ftsz" class="section level2">
<h2><span class="header-section-number">4.3</span> Cell Division without FtsZ</h2>
<p>In some species of Mycoplasma (which are well known for their small genomes), the FtsZ homologue is absent <span class="citation">(Lluch-Senar, Querol, and Piñol 2010)</span>. Although the cell division mechanisms in these organisms remain elusive, they lead to reassessments on the role of FtsZ in FtsZ-dependent cell division. For example, as reviewed by <span class="citation">Xiao and Goley (2016)</span>, it is likely that the main driving force for membrane invagination and constriction is not directly provided by FtsZ, but by the peptidoglycan synthesis.</p>
</div>
</div>
<div id="concluding-remarks" class="section level1">
<h1><span class="header-section-number">5</span> Concluding Remarks</h1>
<p>There is an extensive repertoire of cell division strategies based on distinct families of proteins in the domain of bacteria. The lack of universal conservation and the ease of horizontal gene transfer among bacteria make cell division a suboptimal target of antibiotics. Indeed, most available antibiotics related to bacterial cell division target cell wall synthesis, and there is little progress made on developing drugs that interferes with other components of bacterial divisome.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Adams-2009">
<p>Adams, David W., and Jeff Errington. 2009. “Bacterial Cell Division: Assembly, Maintenance and Disassembly of the Z Ring.” <em>Nature Reviews Microbiology</em> 7 (9): 642–53. <a href="https://doi.org/10.1038/nrmicro2198">https://doi.org/10.1038/nrmicro2198</a>.</p>
</div>
<div id="ref-Adams-2015">
<p>Adams, David William, Ling Juan Wu, and Jeff Errington. 2015. “Nucleoid Occlusion Protein Noc Recruits Dna to the Bacterial Cell Membrane.” <em>EMBO J</em> 34 (4): 491–501. <a href="https://doi.org/10.15252/embj.201490177">https://doi.org/10.15252/embj.201490177</a>.</p>
</div>
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Chen-2001">
<p>Chen, Joseph C., and Jon Beckwith. 2001. “FtsQ, Ftsl and Ftsi Require Ftsk, but Not Ftsn, for Co-Localization with Ftsz During Escherichia Coli Cell Division.” <em>Molecular Microbiology</em> 42 (2): 395–413. <a href="https://doi.org/10.1046/j.1365-2958.2001.02640.x">https://doi.org/10.1046/j.1365-2958.2001.02640.x</a>.</p>
</div>
<div id="ref-Du-2017">
<p>Du, Shishen, and Joe Lutkenhaus. 2017. “Assembly and Activation of the Escherichia Coli Divisome.” <em>Molecular Microbiology</em> 105 (2): 177–87. <a href="https://doi.org/10.1111/mmi.13696">https://doi.org/10.1111/mmi.13696</a>.</p>
</div>
<div id="ref-Eswara-2017">
<p>Eswara, Prahathees J., and Kumaran S. Ramamurthi. 2017. “Bacterial Cell Division: Nonmodels Poised to Take the Spotlight.” <em>Annual Review of Microbiology</em> 71 (1): 393–411. <a href="https://doi.org/10.1146/annurev-micro-102215-095657">https://doi.org/10.1146/annurev-micro-102215-095657</a>.</p>
</div>
<div id="ref-Ghosal-2014">
<p>Ghosal, Debnath, Daniel Trambaiolo, Linda A. Amos, and Jan Löwe. 2014. “MinCD Cell Division Proteins Form Alternating Copolymeric Cytomotive Filaments.” <em>Nature Communications</em> 5 (1): 5341. <a href="https://doi.org/10.1038/ncomms6341">https://doi.org/10.1038/ncomms6341</a>.</p>
</div>
<div id="ref-Lluch-Senar-2010">
<p>Lluch-Senar, Maria, Enrique Querol, and Jaume Piñol. 2010. “Cell Division in a Minimal Bacterium in the Absence of ftsZ.” <em>Molecular Microbiology</em> 78 (2): 278–89. <a href="https://doi.org/10.1111/j.1365-2958.2010.07306.x">https://doi.org/10.1111/j.1365-2958.2010.07306.x</a>.</p>
</div>
<div id="ref-Pichoff-2007">
<p>Pichoff, Sebastien, and Joe Lutkenhaus. 2007. “Identification of a Region of Ftsa Required for Interaction with Ftsz.” <em>Molecular Microbiology</em> 64 (4): 1129–38. <a href="https://doi.org/10.1111/j.1365-2958.2007.05735.x">https://doi.org/10.1111/j.1365-2958.2007.05735.x</a>.</p>
</div>
<div id="ref-Ramm-2019">
<p>Ramm, Beatrice, Tamara Heermann, and Petra Schwille. 2019. “The E. Coli Mincde System in the Regulation of Protein Patterns and Gradients.” <em>Cellular and Molecular Life Sciences</em> 76 (21): 4245–73. <a href="https://doi.org/10.1007/s00018-019-03218-x">https://doi.org/10.1007/s00018-019-03218-x</a>.</p>
</div>
<div id="ref-Rodrigues-2012">
<p>Rodrigues, Christopher D. A., and Elizabeth J. Harry. 2012. “The Min System and Nucleoid Occlusion Are Not Required for Identifying the Division Site in Bacillus Subtilis but Ensure Its Efficient Utilization.” <em>PLOS Genetics</em> 8 (3). Public Library of Science: 1–20. <a href="https://doi.org/10.1371/journal.pgen.1002561">https://doi.org/10.1371/journal.pgen.1002561</a>.</p>
</div>
<div id="ref-Stahl-2019a">
<p>Stahl, Michael T Madigan; Kelly S Bender; Daniel Hezekiah Buckley; W Matthew Sattley; David Allan. 2019. “Molecular Biology of Microbial Growth.” In <em>Brock Biology of Microorganisms</em>, Fifteenth edition, Global edition. Pearson.</p>
</div>
<div id="ref-Tsang-2015">
<p>Tsang, Mary-Jane, and Thomas G Bernhardt. 2015. “Guiding Divisome Assembly and Controlling Its Activity.” <em>Current Opinion in Microbiology</em> 24: 60–65. <a href="https://doi.org/https://doi.org/10.1016/j.mib.2015.01.002">https://doi.org/https://doi.org/10.1016/j.mib.2015.01.002</a>.</p>
</div>
<div id="ref-Xiao-2016">
<p>Xiao, Jie, and Erin D Goley. 2016. “Redefining the Roles of the Ftsz-Ring in Bacterial Cytokinesis.” <em>Current Opinion in Microbiology</em> 34: 90–96. <a href="https://doi.org/https://doi.org/10.1016/j.mib.2016.08.008">https://doi.org/https://doi.org/10.1016/j.mib.2016.08.008</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Some fast-growing bacteria may afford multiple rounds of replication on a single chromosome, and this makes it possible for their division to outpace DNA replication.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>one flagellum at each pole<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/isoelectric-point-calculation.html" class="archive-item-link">
                Isoelectric Point Calculation 
              </a>
              <span class="archive-item-date">
                2020-02-07
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>By rearranging the Henderson-Hasselbalch equation:</p>
<p><span class="math display">\[
\text{pH}=\text{pK}_\text{a} + \log\left(\dfrac{\text{[A}^{-}\text{]}}{\text{[HA]}}\right)
\]</span></p>
<p>we can get the ratio between an acid and its conjugate base:</p>
<p><span class="math display">\[
\frac{\text{[HA]}}{\text{[A}^{-}\text{]}}=10^{(\text{pK}_\text{a}-\text{pH})}
\]</span></p>
<p>and between an base and its conjugate acid:</p>
<p><span class="math display">\[
\frac{\text{[B]}}{\text{[BH}^+\text{]}} = 10^{(\text{pH}-\text{pK}_\text{a})}
\]</span></p>
<p>Thus, the proportion of deprotonated acid is calculated as follows:</p>
<p><span class="math display">\[
\dfrac{\text{[A}^{-}\text{]}}{\text{[A]}_\text{total}}=\dfrac{\text{[A}^{-}\text{]}}{\text{[HA]}+\text{[A}^{-}\text{]}}=\dfrac{1}{1+\frac{\text{[HA]}}{\text{[A}^{-}\text{]}}}=\dfrac{1}{1+10^{(\text{pK}_\text{a}-\text{pH})}}
\]</span></p>
<p>Similarly, for basic species:</p>
<p><span class="math display">\[
\dfrac{\text{[BH}^+\text{]}}{\text{[B]}_\text{total}}=\dfrac{\text{[BH}^+\text{]}}{\text{[B]}+\text{[BH}^+\text{]}}=\dfrac{1}{1+\frac{\text{[B]}}{\text{[BH}^+\text{]}}}=\dfrac{1}{1+10^{(\text{pH}-\text{pK}_\text{a})}}
\]</span></p>
<p><a href="http://fields.scripps.edu/DTASelect/20010710-pI-Algorithm.pdf" class="uri">http://fields.scripps.edu/DTASelect/20010710-pI-Algorithm.pdf</a></p>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../notes/molecular-cloning.html" class="archive-item-link">
                Molecular Cloning 
              </a>
              <span class="archive-item-date">
                2020-02-04
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="about-the-frame" class="section level1">
<h1>About the frame</h1>
<p>Sometimes we want to use the start/stop codon in the vector for our protein production, especially when tag sequences need to be included. In these cases, we need to check our inserted GOI is in frame with the start/codon on the vector. Frame mismatch can be corrected by adding 1 or 2 random nucleotide(s) to the primer (forward primer to match vector start condon and reverse primer to match the stop codon) between the GOI sequence and the restriction site sequence.</p>
<p>Some vectors have an N-terminal tag and stop codons in all three frames (e.g. <code>TGA C TGA C TGA</code>)</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/sequence-alignment.html" class="archive-item-link">
                Sequence Alignment 
              </a>
              <span class="archive-item-date">
                2020-02-03
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="sequence-alignment-in-r" class="section level1">
<h1>Sequence Alignment in R</h1>
<div id="packages" class="section level2">
<h2>Packages</h2>
<pre class="r"><code>library(Biostrings)
library(seqinr)</code></pre>
</div>
<div id="dna-sequence" class="section level2">
<h2>DNA sequence</h2>
<pre class="r"><code>myScoringMat &lt;- nucleotideSubstitutionMatrix(match = 1, mismatch = -1, baseOnly = TRUE)
dnaSeq1 &lt;- &quot;GAATTCGGCTA&quot; # can be a plain string
dnaSeq2 &lt;- DNAString(&quot;GATTACCTA&quot;) # or an XString
gapOpen &lt;- 2
gapExtend &lt;- 1
dnaAln &lt;- pairwiseAlignment(dnaSeq1, dnaSeq2, substitutionMatrix = myScoringMat, gapOpening = gapOpen, gapExtension = gapExtend, type = &#39;global&#39;, scoreOnly=FALSE)   
dnaAln</code></pre>
</div>
<div id="amino-acid-sequence" class="section level2">
<h2>Amino acid sequence</h2>
<p>Here is a list of built-in amino acid substitution matrices, which can be shown with <code>data(package=&quot;Biostrings&quot;)</code>:</p>
<pre><code>BLOSUM100
BLOSUM45
BLOSUM50
BLOSUM62
BLOSUM80
PAM120
PAM250
PAM30
PAM40
PAM70</code></pre>
<pre class="r"><code>aaSeq1 &lt;- &#39;PAWHEAE&#39;
aaSeq2 &lt;- &#39;HEAGAWGHE&#39;
aaAln &lt;- pairwiseAlignment(aaSeq1, aaSeq2, substitutionMatrix=&quot;BLOSUM62&quot;, gapOpening=gapOpen, gapExtension=gapExtend, type=&#39;global&#39;, scoreOnly=FALSE)
aaAln</code></pre>
<pre class="r"><code>ncov &lt;- readDNAStringSet(&#39;./src/2019-ncov.fasta&#39;, &#39;fasta&#39;)
sars &lt;- readDNAStringSet(&#39;./src/sars.fasta&#39;, &#39;fasta&#39;)</code></pre>
<pre class="r"><code>virusAln &lt;- pairwiseAlignment(ncov, sars, substitutionMatrix = myScoringMat, gapOpening = gapOpen, gapExtension = gapExtend, type = &#39;global&#39;, scoreOnly=FALSE)</code></pre>
<pre class="r"><code>ncov1 &lt;- seqinr::read.fasta(&#39;./src/2019-ncov.fasta&#39;, seqtype = &#39;DNA&#39;)
sars1 &lt;- seqinr::read.fasta(&#39;./src/sars.fasta&#39;, seqtype = &#39;DNA&#39;)</code></pre>
<pre class="r"><code>library(ape)
myset &lt;- c(&quot;U15717&quot;, &quot;U15718&quot;, &quot;U15719&quot;, &quot;U15720&quot;, &quot;U15721&quot;,&quot;U15722&quot;, &quot;U15723&quot;, &quot;U15724&quot;)
myseqs &lt;- read.GenBank(myset)
mydist &lt;- dist.dna(myseqs)
plot(myphylo, type=&quot;phylogram&quot;, edge.color=&quot;red&quot;, cex=1, edge.width=1,main=&quot;(A) Phylogram&quot;)</code></pre>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/protein-purification-practical.html" class="archive-item-link">
                Protein Purification Practical 
              </a>
              <span class="archive-item-date">
                2020-02-02
              </span>
              <div class="article-summary" style="display: none;">
                
                


<pre class="r"><code>absorbance &lt;- 
  c(0.086, 0.142, 0.548, 0.976, 1.419, 1.294, 
    1.161, 1.472, 1.800, 2.003, 1.777, 1.463, 
    1.430, 1.486, 1.538, 1.254, 0.820, 0.503, 
    0.388, 0.235, 0.204, 0.163, 0.148, 0.151,
    0.169, 0.221, 0.381, 0.701, 0.954, 0.917,
    0.651, 0.423, 0.274, 0.191, 0.034, 0.031
  )
volume &lt;- 1:36
plot(volume, absorbance, xlab = &#39;volume (ml)&#39;, type = &#39;p&#39;, 
     ylim=c(0, 2.2), yaxs=&#39;i&#39;)
lines(volume, absorbance)
lines(c(5, 5), c(0.0, 1.419), col=&#39;red&#39;)
lines(c(10, 10), c(0.0, 2.003), col=&#39;red&#39;)
lines(c(15, 15), c(0.0, 1.538), col=&#39;red&#39;)
lines(c(29, 29), c(0.0, 0.954), col=&#39;red&#39;)
text(4, 1.5, &#39;ferritin (450 kDa)&#39;)
text(10, 2.1, expression(paste(gamma, &#39;-globulin (155 kDa)&#39;)))
text(17, 1.6, &#39;serum albumin (67 kDa)&#39;)
text(29, 1.05, &#39;cytochrome-c (14 kDa)&#39;)</code></pre>
<p><img src="../lab/2020-02-02-protein-purification-practical_files/figure-html/unnamed-chunk-1-1.png" width="672" /></p>
<pre class="r"><code>volume &lt;- c(5, 10, 15, 29) + 44
log_mw &lt;- log(c(450, 155, 67, 14), 10)
plot(volume, log_mw, 
     xlab=&#39;elution volume (ml)&#39;, ylab=&#39;log(molecular weight) (kDa)&#39;)
(mod &lt;- lm(log_mw ~ volume))</code></pre>
<pre><code>## 
## Call:
## lm(formula = log_mw ~ volume)
## 
## Coefficients:
## (Intercept)       volume  
##     5.52272     -0.06075</code></pre>
<pre class="r"><code>abline(mod, col=&#39;red&#39;)
(mod1 &lt;- lm(log_mw[-4] ~ volume[-4]))</code></pre>
<pre><code>## 
## Call:
## lm(formula = log_mw[-4] ~ volume[-4])
## 
## Coefficients:
## (Intercept)   volume[-4]  
##     6.68975     -0.08271</code></pre>
<pre class="r"><code>abline(mod1, col=&#39;blue&#39;)
points(c(67, 60), log(c(25.8, 29.1), 10), col=&#39;red&#39;)</code></pre>
<p><img src="../lab/2020-02-02-protein-purification-practical_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<pre class="r"><code>par(mar = c(5,5,2,5))
f &lt;- c(13, 11, 11, 12, 12, 15, 14, 14, 16, 16, 14, 17, 
       16, 16, 16, 17, 17, 18, 15, 17, 16, 15, 14, 15, 
       19, 22, 24, 32, 41, 28, 34, 37, 75, 76, 79, 65, 
       341, 608, 1554, 3107, 2264, 1768, 2196, 1424, 33)
v &lt;- 28:72
plot(v, f, type=&#39;p&#39;, col=&#39;red&#39;, ylab=&#39;Fluoresence (arbitrary units)&#39;, xlab=&#39;elution volume (mL)&#39;)
lines(v, f, col=&#39;red&#39;)

A &lt;- c(0.086, 0.072, 0.067, 0.067, 0.067, 0.079, 0.073, 0.065, 0.068, 
       0.070, 0.065, 0.064, 0.080, 0.067, 0.073, 0.079, 0.077, 0.074,
       0.078, 0.075, 0.082, 0.078, 0.076, 0.085, 0.081, 0.080, 0.081,
       0.099, 0.118, 0.122, 0.111, 0.146, 0.180, 0.140, 0.126, 0.117, 
       0.141, 0.125, 0.145, 0.196, 0.138, 0.161, 0.176, 0.150, 0.071)
par(new = T)
plot(v, A, axes=F, xlab=NA, ylab=NA, col=&#39;blue&#39;, pch=4)
lines(v, A, col=&#39;blue&#39;)
axis(side = 4)
mtext(side = 4, line = 3, &#39;Absorbance (arbitrary units)&#39;)
legend(&#39;topleft&#39;,
       legend=c(&#39;fluoresence&#39;, &#39;absorbance&#39;),
       lty=c(1, 1), pch=c(1, 4), col=c(&#39;red&#39;, &#39;blue&#39;))
abline(v = c(60, 67, 70))
axis(side = 1, at = 67,labels = T)</code></pre>
<p><img src="../lab/2020-02-02-protein-purification-practical_files/figure-html/3C-1.png" width="672" /></p>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../fun/vim.html" class="archive-item-link">
                Vim 
              </a>
              <span class="archive-item-date">
                2020-02-01
              </span>
              <div class="article-summary" style="display: none;">
                
                


<pre><code>B = Life Technologies
C = Minotech Biotechnology
E = Agilent Technologies
I = SibEnzyme Ltd.
J = Nippon Gene Co., Ltd.
K = Takara Bio Inc.
M = Roche Applied Science
N = New England Biolabs
O = Toyobo Biochemicals
Q = Molecular Biology Resources - CHIMERx
R = Promega Corporation
S = Sigma Chemical Corporation
V = Vivantis Technologies
X = EURx Ltd.
Y = SinaClon BioScience Co.</code></pre>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../notes/techniques.html" class="archive-item-link">
                Techniques 
              </a>
              <span class="archive-item-date">
                2020-01-29
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="protein-structure" class="section level1">
<h1>Protein Structure</h1>
<div id="homology-modelling" class="section level2">
<h2>Homology Modelling</h2>
<div id="swiss-model" class="section level3">
<h3>Swiss Model</h3>
<div id="model-evaluation" class="section level5">
<h5>Model Evaluation</h5>
<ul>
<li>GMQE (global model quality estimation)
<ul>
<li>0-1, higher better</li>
<li>combines properties from the target-alignment and the template search method</li>
<li>replecting the expected accuracy of a model built with that alignment and the template and the coverage of the target</li>
<li>computed before modelling and after modelling. The latter also takes into account the QMEAN</li>
</ul></li>
<li>QMEAN
<ul>
<li>QMEAN z score: around 0: ideal; &lt; -4.0: low quality</li>
<li>QMEAN is a composite estimator based on different <strong>geometrical properties</strong> and provides both global and local absolute quality estimates on the basis of <em>one single model</em></li>
<li>the QMEAN z-score provides an estimate of the ‘degree of nativeness’ of the structural features observed in the model on a global scale. It indicates whether the QMEAN score of the model is comparable to what one expect from experimental structures of similar size</li>
<li>4 terms of the global QMEAN quality score:
<ul>
<li>the interaction potential between Cβ atoms only</li>
<li>and between all atoms</li>
<li>solvation potential</li>
<li>torsion angle potential</li>
</ul></li>
<li>the accuracy of local QMEAN scores is enhanced by QMEANDisCo
<ul>
<li>below 0.6: low quality</li>
<li>if downloaded, local score is reported in the B-factor column of the PDB file</li>
<li>can be visualised by choosing the colour scheme ‘QMEAN’</li>
<li>QMEANDisco assesses the consistency of observed interatomic distances in the model with ensemble information extracted from experimentally determined protein structures that are homologous to the target sequence. The ‘local quality’ plot shows, for each residue of the model (x-axis), the expected similarity to the native structure (y-axis)</li>
<li>in the ‘comparison’ plot, model quality scores are related to scores obtained for experimental structures of similar size. The x-axis shows protein length and the y-axis shows the normalized QMEAN score. Every dot represents an experimentally determined structure. The mean and standard deviation of the experimental structures around the x-location of the star (the model) are the basis to calculate the QMEAN z-score of the model.</li>
</ul></li>
</ul></li>
</ul>
</div>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/mcb-toolbox.html" class="archive-item-link">
                MCB Toolbox 
              </a>
              <span class="archive-item-date">
                2020-01-23
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>For the second tutorial, I want you to try some proper molecular biology using online tools, just as we do every day in the lab. Try it out on your own first, though you may wish to discuss any issues you may be having with each other. Please bring files/printouts and if you have one, your laptop to the tutorial. You will find NCBI BLAST useful and various links within www.expasy.org (e.g. In proteomics section, Clustal, T-coffee, Translate, prot param etc). You may also like to download ApE or Snapgene viewer from the web but make sure you scan with e.g. Sophos before you open them. You can do full in silico cloning in ApE (but only in the expensive paid-for version of Snapgene) but Snapgene (even free viewer) offers much better visualization so you may want to use both but for different features. For structural productions, Swissprot is convenient, then for mutagenesis, move to Pymol or equivalent.</p>
<pre><code>Mystery gene (lower case = non-coding, upper case = coding)

GTATTTTGCGTCCATTTCGATCAAAAGACTATTTCTCATTTCCTCCCACGATTTTAGATAATAATATTATCTTATGTCATGCACAGATGATAAGTGATGATGACGATCTACCATCTACTCGCCCGGGATCTGTGAATGAGGAATTACCAGAAACCGAACCCGAAGATAATGATGAGTTGCCTGAAACAGAACCTGAAAGCGATTCCGATAAACCTACCGTAACCTCGAATAAAACAGAAAACCAAGTTGCTGATGAAGATTATGATTCATTCGACGATTTTGTGCCCAGTCAAACACACACAGCCTCCAAAATACCTGTAAAAAATAAACGAGCCAAAAAGTGCACTGTAGAATCTGATTCATCATCTTCGGATGATTCCGATCAAGGAGATGATTGTGAATTTATCCCAGCTTGTGATGAGACACAGGAAGTTCCGAAAATCAAAAGAGGATACACTCTGAGAACTCGAGCAAGTGTAAAGAACAAATGTGATGATTCATGGGATGATGGAATAGACGAAGAAGATGTCTCAAAAAGATCAGAAGACACGTTAAATGATTCATTTGTTGATCCTGAATTCATGGATTCTGTTCTAGATAATCAATTAACGATCAAAGGCAAAAAGCAATTTCTCGATGATGGAGAGTTTTTCACAGACCGGAATGTTCCTCAGATTGATGAAGCTACAAAAATGAAGTGGGCATCAATGACGTCACCTCCTCAAGAAGCTTTGAACGCATTGAACGAATTCTTCGGTCATAAAGGATTCCGAGAAAAGCAGTGGGATGTTGTCAGAAATGTTTTGGGAGGAAAAGACCAATTTGTTCTTATGTCCACTGGTTATGGTAAAAGTGTATGTTATCAGCTACCATCACTTCTTCTCAATTCGATGACTGTCGTGGTATCTCCATTAATTTCATTGATGAATGATCAAGTAACTACATTGGTTTCTAAAGGTATTGATGCAGTGAAACTAGATGGACATTCTACACAAATTGAATGGGATCAAGTTGCGAATAATATGCACCGAATTAGGTTCATCTACATGTCACCTGAAATGGTTACGAGCCAAAAGGGTTTGGAATTATTAACTTCTTGCCGAAAACATATCTCCCTCCTCGCTATTGATGAAGCTCATTGTGTTTCTCAATGGGGACACGACTTTCGAAACTCGTACAGGCATCTCGCAGAAATTAGAAACCGATCTGATCTATGCAATATTCCAATGATTGCTCTTACCGCTACTGCCACAGTTAGAGTTCGTGATGACGTCATTGCTAATTTAAGACTCCGCAAGCCATTAATCACAACTACGTCGTTTGATAGAAAGAATCTCTACATTTCTGTGCATTCTTCAAAGGACATGGCTGAAGATTTAGGATTATTCATGAAAACCGATGAAGTTAAAGGAAGACACTTTGGTGGACCTACTATTATTTATTGCCAAACGAAACAAATGGTCGATGATGTGAACTGTGTTTTGAGAAGAATCGGAGTTCGTTCTGCTCATTATCACGCAGGACTCACTAAAAATCAACGAGAAAAAGCACACACCGATTTTATGAGAGATAAGATTACAACAATCGTTGCGACAGTTGCATTTGGTATGGGAATTGACAAACCCGACGTTCGAAATGTGATTCATTACGGATGCCCGAACAATATCGAATCATATTATCAAGAAATCGGAAGAGCTGGTCGAGATGGATCTCCAAGTATTTGTCGTGTATTCTGGGCTCCGAAAGATTTGAATACTATAAAATTTAAACTTCGAAATTCGCAGCAAAAAGAAGAAGTAGTTGAAAATCTTACAATGATGCTAAGACAACTCGAGTTGGTTCTGACAACCGTTGGATGTAGAAGATACCAACTTCTGAAGCACTTTGACCCATCATACGCGAAACCTCCAACTATGCAAGCTGATTGTTGTGATAGATGTACTGAAATGCTCAATGGAAATCAAGATTCATCATCCAGTATTGTTGATGTTACAACAGAATCGAAGTGGTTGTTTCAAGTTATTAACGAAATGTACAACGGGAAAACTGGTATCGGAAAACCAATCGAATTTCTGAGGGGATCGAGTAAAGAAGACTGGCGAATCAAGACCACATCTCAACAAAAATTGTTTGGAATTGGAAAACATATTCCTGATAAATGGTGGAAAGCACTTGCAGCATCACTTCGAATTGCTGGTTATCTTGGAGAAGTTAGGCTGATGCAAATGAAATTTGGAAGTTGTATCACTTTGTCCGAACTCGGGGAACGATGGCTTTTGACTGGAAAAGAGATGAAAATCGATGCGACACCGATTTTATTGCAAGGGAAGAAAGAAAAAGCCGCACCTTCAACTGTCCCCGGAGCTTCAAGATCTCAGTCAACTAAATCAAGTACAGAGATTCCAACCAAGATTCTCGGAGCGAATAAGATTCGTGAATACGAGCCTGCAAATGAAAACGAGCAGCTGATGAACTTGAAAAAGCAAGAAGTCACTGGTCTTCCAGAGAAGATTGATCAACTGCGCTCTCGTCTTGACGACATTCGTGTAGGAATTGCAAACATGCATGAAGTAGCACCATTCCAAATTGTATCGAATACTGTTCTTGATTGTTTTGCCAACTTGAGACCTACCTCAGCCTCGAATCTCGAAATGATTGATGGAATGTCGGCTCAGCAGAAATCTAGATACGGAAAACGATTTGTCGATTGTGTTGTACAATTTTCAAAGGAAACTGGCATTGCAACAAACGTCAATGCCAACGATATGATACCCCCTGAACTTATTTCAAAAATGCAGAAAGTTCTCTCGGATGCGGTGAGAAGAGTATACACAGAGCATCTTATTTCGAGATCCACTGCGAAAGAAGTGGCAACTGCTCGAGGAATTAGTGAGGGTACTGTATATTCATATCTCGCGATGGCGGTAGAAAAAGGATTACCTCTTCACTTAGACAAGTTAAATGTCTCCAGAAAGAATATTGCAATGGCTCTAAATGCAGTTAGAGTACATTTAGGATCAAATGTTGCCGTACTGACACCATGGGTTGAAGCTATGGGAGTCGTACCTGATTTTAATCAGTTGAAATTGATCCGGGCAATTCTTATTTACGAATATGGATTGGATACGAGTGAGAACCAAGAGAAGCCAGACATCCAATCTATGCCGTCCACTTCAAATCCATCCACCATCAAAACAGTTCCATCAACACCTTCATCCTCTCTCAGAGCTCCTCCATTGAAGAAATTCAAACTTTAAATTTCTTATATTTTTTTTAATCTGAATATCATAACTATACGGTCTCTTATTTTCAATCTCATCCCTGTCTATATGAATGCTGGTTATTCGATTTTCACTTATGAATTTTTAAAAACACTT</code></pre>
<pre><code>pIVEX2.3d Vector Sequence

TCGCGCGTTTCGGTGATGACGGTGAAAACCTCTGACACATGCAGCTCCCGGAGACGGTCACAGCTTGTCTGTAAGCGGATGCCGGGAGCAGACAAGCCCGTCAGGGCGCGTCAGCGGGTGTTGGCGGGTGTCGGGGCTGGCTTAACTATGCGGCATCAGAGCAGATTGTACTGAGAGTGCACCATATATGCGGTGTGAAATACCGCACAGATGCGTAAGGAGAAAATACCGCATCAGGCGCCATTCGCCATTCAGGCTGCGCAACTGTTGGGAAGGGCGATCGGTGCGGGCCTCTTCGCTATTACGCCAGCTGGCGAAAGGGGGATGTGCTGCAAGGCGATTAAGTTGGGTAACGCCAGGGTTTTCCCAGTCACGACGTTGTAAAACGACGGCCAGTGCCAAGCTTGCATGCAAGGAGATGGCGCCCAACAGTCCCCCGGCCACGGGGCCTGCCACCATACCCACGCCGAAACAAGCGCTCATGAGCCCGAAGTGGCGAGCCCGATCTTCCCCATCGGTGATGTCGGCGATATAGGCGCCAGCAACCGCACCTGTGGCGCCGGTGATGCCGGCCACGATGCGTCCGGCGTAGAGGATCGAGATCTCGATCCCGCGAAATTAATACGACTCACTATAGGGAGACCACAACGGTTTCCCTCTAGAAATAATTTTGTTTAACTTTAAGAAGGAGATATACCATGGCACATATGAGCGGCCGCGTCGACTCGAGCGAGCTCCCGGGGGGGGTTCTCATCATCATCATCATCATTAATAAAAGGGCGAATTCCAGCACACTGGCGGCCGTTACTAGTGGATCCGGCTGCTAACAAAGCCCGAAAGGAAGCTGAGTTGGCTGCTGCCACCGCTGAGCAATAACTAGCATAACCCCTTGGGGCCTCTAAACGGGTCTTGAGGGGTTTTTTGCTGAAAGGAGGAACTATATCCGGATATCCACAGGACGGGTGTGGTCGCCATGATCGCGTAGTCGATAGTGGCTCCAAGTAGCGAAGCGAGCAGGACTGGGCGGCGGCCAAAGCGGTCGGACAGTGCTCCGAGAACGGGTGCGCATAGAAATTGCATCAACGCATATAGCGCTAGCAGCACGCCATAGTGACTGGCGATGCTGTCGGAATGGACGATATCCCGCAAGAGGCCCGGCAGTACCGGCATAACCAAGCCTATGCCTACAGCATCCAGGGTGACGGTGCCGAGGATGACGATGAGCGCATTGTTAGATTTCATACACGGTGCCTGACTGCGTTAGCAATTTAACTGTGATAAACTACCGCATTAAAGCTTATCGATGATAAGCTGTCAAACATGAGAATTCGTAATCATGTCATAGCTGTTTCCTGTGTGAAATTGTTATCCGCTCACAATTCCACACAACATACGAGCCGGAAGCATAAAGTGTAAAGCCTGGGGTGCCTAATGAGTGAGCTAACTCACATTAATTGCGTTGCGCTCACTGCCCGCTTTCCAGTCGGGAAACCTGTCGTGCCAGCTGCATTAATGAATCGGCCAACGCGCGGGGAGAGGCGGTTTGCGTATTGGGCGCTCTTCCGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAAAGGCGGTAATACGGTTATCCACAGAATCAGGGGATAACGCAGGAAAGAACATGTGAGCAAAAGGCCAGCAAAAGGCCAGGAACCGTAAAAAGGCCGCGTTGCTGGCGTTTTTCCATAGGCTCCGCCCCCCTGACGAGCATCACAAAAATCGACGCTCAAGTCAGAGGTGGCGAAACCCGACAGGACTATAAAGATACCAGGCGTTTCCCCCTGGAAGCTCCCTCGTGCGCTCTCCTGTTCCGACCCTGCCGCTTACCGGATACCTGTCCGCCTTTCTCCCTTCGGGAAGCGTGGCGCTTTCTCATAGCTCACGCTGTAGGTATCTCAGTTCGGTGTAGGTCGTTCGCTCCAAGCTGGGCTGTGTGCACGAACCCCCCGTTCAGCCCGACCGCTGCGCCTTATCCGGTAACTATCGTCTTGAGTCCAACCCGGTAAGACACGACTTATCGCCACTGGCAGCAGCCACTGGTAACAGGATTAGCAGAGCGAGGTATGTAGGCGGTGCTACAGAGTTCTTGAAGTGGTGGCCTAACTACGGCTACACTAGAAGGACAGTATTTGGTATCTGCGCTCTGCTGAAGCCAGTTACCTTCGGAAAAAGAGTTGGTAGCTCTTGATCCGGCAAACAAACCACCGCTGGTAGCGGTGGTTTTTTTGTTTGCAAGCAGCAGATTACGCGCAGAAAAAAAGGATCTCAAGAAGATCCTTTGATCTTTTCTACGGGGTCTGACGCTCAGTGGAACGAAAACTCACGTTAAGGGATTTTGGTCATGAGATTATCAAAAAGGATCTTCACCTAGATCCTTTTAAATTAAAAATGAAGTTTTAAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAATCAGTGAGGCACCTATCTCAGCGATCTGTCTATTTCGTTCATCCATAGTTGCCTGACTCCCCGTCGTGTAGATAACTACGATACGGGAGGGCTTACCATCTGGCCCCAGTGCTGCAATGATACCGCGAGACCCACGCTCACCGGCTCCAGATTTATCAGCAATAAACCAGCCAGCCGGAAGGGCCGAGCGCAGAAGTGGTCCTGCAACTTTATCCGCCTCCATCCAGTCTATTAATTGTTGCCGGGAAGCTAGAGTAAGTAGTTCGCCAGTTAATAGTTTGCGCAACGTTGTTGCCATTGCTACAGGCATCGTGGTGTCACGCTCGTCGTTTGGTATGGCTTCATTCAGCTCCGGTTCCCAACGATCAAGGCGAGTTACATGATCCCCCATGTTGTGCAAAAAAGCGGTTAGCTCCTTCGGTCCTCCGATCGTTGTCAGAAGTAAGTTGGCCGCAGTGTTATCACTCATGGTTATGGCAGCACTGCATAATTCTCTTACTGTCATGCCATCCGTAAGATGCTTTTCTGTGACTGGTGAGTACTCAACCAAGTCATTCTGAGAATAGTGTATGCGGCGACCGAGTTGCTCTTGCCCGGCGTCAATACGGGATAATACCGCGCCACATAGCAGAACTTTAAAAGTGCTCATCATTGGAAAACGTTCTTCGGGGCGAAAACTCTCAAGGATCTTACCGCTGTTGAGATCCAGTTCGATGTAACCCACTCGTGCACCCAACTGATCTTCAGCATCTTTTACTTTCACCAGCGTTTCTGGGTGAGCAAAAACAGGAAGGCAAAATGCCGCAAAAAAGGGAATAAGGGCGACACGGAAATGTTGAATACTCATACTCTTCCTTTTTCAATATTATTGAAGCATTTATCAGGGTTATTGTCTCATGAGCGGATACATATTTGAATGTATTTAGAAAAATAAACAAATAGGGGTTCCGCGCACATTTCCCCGAAAAGTGCCACCTGACGTCTAAGAAACCATTATTATCATGACATTAACCTATAAAAATAGGCGTATCACGAGGCCCTTTCGTC</code></pre>
<ol style="list-style-type: decimal">
<li>Use the information in the attached DNA file (mystery gene) to identify the encoded protein and the species.
<ul>
<li>Caenorhabditis elegans Probable Werner syndrome ATP-dependent helicase homolog 1 (wrn-1) (UniProt accession code: <a href="https://www.uniprot.org/uniprot/Q19546">Q19546</a>)</li>
</ul></li>
<li>Draw a schematic diagram of the protein, noting the number of amino acids, predicted Mw and pI of the encoded protein, and annotate on functional domains (e.g. in ppt).
<ul>
<li>1056 amino acids; MW = 118522.73 Da (average mass)/118447.63 Da (monoisotopic mass); pI = 6.08 (predicted using ExPASy <a href="https://web.expasy.org/compute_pi/">‘Compute pI/Mw tool’</a>)</li>
<li><a href="https://swissmodel.expasy.org/repository/uniprot/Q19546">Automated homology modelling in the SWISS-MODEL Repository</a> uses 1oyy.1.A, 4cdg.1.A and 4cgz.1.A as templates to obtain models with global QMEANs of -3.35, -3.57 and -4.19, respectively</li>
<li>Manual (interactive) modelling using 4q47.1.A (DrRecQ catalytic core (Dr = <em>Deinococcus radiodurans</em>)) achieved a better QMEAN of -3.09. The structure is shown in Fig. <a href="#fig:wrn-model">1</a> (highlighting functional domains) and in Fig. <a href="#fig:wrn-b-factor">2</a> (showing local quality (QMEAN) scores). The annotations on functional domains are based on <span class="citation">Chen et al. (2014)</span>.</li>
<li>Update on 2 Feb: yet a higher QEMAN of -2.90 is achieved with 2wwy.1. 2wwy.1 is the structure of human RECQ-like helicase in complex with DNA substrate<span class="citation">(Pike et al. 2015)</span>. The homology model based on this is shown in Fig. <a href="#fig:recq-in-action">3</a>, and the PyMOL Session (<code>.pse</code>) file can be downloaded <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/model/tianyi-wrn.pse">here</a>.</li>
<li>As shown in Fig. <a href="#fig:wrn-org-cox">4</a>, WRN is unique in the family of RecQ helicases in possessing an 3’-to-5’ exonuclease domain<span class="citation">(Cox and Mason 2013)</span>. In <em>C. elegans</em>, the exonuclease and the helicase are encoded in separate genes. The two experimental structures used for modelling (4q47.1.A (DrRecQ) and 2wwy.1 (human RECQ1)) and the query <em>C. elegans</em> wrn homolog all represent only the helicase domain and the RQC (WH) domain.</li>
</ul></li>
</ol>
<div class="figure"><span id="fig:wrn-model"></span>
<img src="../img/wrn-model.png" alt="A homology model of *C. elegans* wrn-1 based on DrRecQ catalytic core (PDB accession code: 4Q47). The N-terminal part contains the common core structure of the RecQ helicase family, which comprises two RecA-like domains, D1 and D2, each composed of a central β-sheet surrounded by α-helices. The conserved sequence motifs at the interface of D1 and D2 are responsible for ATP binding and hydrolysis. The following Zn-binding domain stabilizes the protein structure. The C-terminus consists of a WH domain (a.k.a RQC domain), which directly participates in DNA binding and base pair separation. [@Chen-2014]" width="100%" />
<p class="caption">
Figure 1: A homology model of <em>C. elegans</em> wrn-1 based on DrRecQ catalytic core (PDB accession code: 4Q47). The N-terminal part contains the common core structure of the RecQ helicase family, which comprises two RecA-like domains, D1 and D2, each composed of a central β-sheet surrounded by α-helices. The conserved sequence motifs at the interface of D1 and D2 are responsible for ATP binding and hydrolysis. The following Zn-binding domain stabilizes the protein structure. The C-terminus consists of a WH domain (a.k.a RQC domain), which directly participates in DNA binding and base pair separation. <span class="citation">(Chen et al. 2014)</span>
</p>
</div>
<div class="figure"><span id="fig:wrn-b-factor"></span>
<img src="../img/wrn-b-factor.png" alt="The same homology model, showing local QMEAN scores (b-factor). Compared to green and thin regions, red and thick regions have local geometrical properties deviating further from those of experimentally determined structures. The Zn-binding domain and WH domain are more consistent with experimental structures than the two RecA-like domains." width="100%" />
<p class="caption">
Figure 2: The same homology model, showing local QMEAN scores (b-factor). Compared to green and thin regions, red and thick regions have local geometrical properties deviating further from those of experimentally determined structures. The Zn-binding domain and WH domain are more consistent with experimental structures than the two RecA-like domains.
</p>
</div>
<div class="figure"><span id="fig:recq-in-action"></span>
<img src="../img/recq-in-action.png" alt="WRN catalytic core (RECQ) in action. This structure is based on the structure of a dimer of human RECQ1 helicase in complex with tailed duplux DNA solved by @Pike-2015. The monomer on the bottom shows the b-factor (local QMEAN) of the homology model with C. elegans wrn-1, where red indicates high and green indicates low scores. The monomer on the top shows the subunits of WRN catalytic core (RECQ), where D1, D2, Zn-binding, and WH (RQC) domains are coloured blue, red, yellow and green, respectively, similar to Fig. 1. DNA is coloured cyan. Ligands (2 zinc ions, 3 sulfate ions and 2 ethanediol molecules) are not removed. " width="100%" />
<p class="caption">
Figure 3: WRN catalytic core (RECQ) in action. This structure is based on the structure of a dimer of human RECQ1 helicase in complex with tailed duplux DNA solved by <span class="citation">Pike et al. (2015)</span>. The monomer on the bottom shows the b-factor (local QMEAN) of the homology model with C. elegans wrn-1, where red indicates high and green indicates low scores. The monomer on the top shows the subunits of WRN catalytic core (RECQ), where D1, D2, Zn-binding, and WH (RQC) domains are coloured blue, red, yellow and green, respectively, similar to Fig. 1. DNA is coloured cyan. Ligands (2 zinc ions, 3 sulfate ions and 2 ethanediol molecules) are not removed.
</p>
</div>
<div class="figure"><span id="fig:wrn-org-cox"></span>
<img src="../img/wrn-org-cox.png" alt="Domain organisation of human WRN and its homologs in *D. melanogaster* and *C. elegans*. Adapted from @Cox-2013." width="100%" />
<p class="caption">
Figure 4: Domain organisation of human WRN and its homologs in <em>D. melanogaster</em> and <em>C. elegans</em>. Adapted from <span class="citation">Cox and Mason (2013)</span>.
</p>
</div>
<ol start="3" style="list-style-type: decimal">
<li>Find homologues in human, Xenopus and mice and give the protein names (Uniprot should help)
<ul>
<li>Use <a href="https://www.uniprot.org/blast/">UniProt BLAST</a></li>
<li>Query: Q19546 (UniProt ID)</li>
<li>Relevant Results:
<ul>
<li>Werner syndrome ATP-dependent helicase (Homo sapiens) (<a href="https://www.uniprot.org/uniprot/Q14191">Q14191</a>; WRN_HUMAN)</li>
<li>Werner syndrome ATP-dependent helicase homolog (Mus musculus (<a href="https://www.uniprot.org/uniprot/O09053">O09053</a>; WRN_MOUSE)</li>
<li>Werner syndrome ATP-dependent helicase homolog (Xenopus laevis) (<a href="https://www.uniprot.org/uniprot/O93530">O93530</a>; WRN_XENLA)</li>
</ul></li>
</ul></li>
<li>Show homology between your mystery protein and the human, mouse and frog homologues.
<ul>
<li>Sequences aligned using UniProt alignment tool (accessible via the BLAST result page)</li>
<li>results displayed using SnapGene Viewer (<a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/clustal.pdf">view PDF</a>)</li>
<li>UniProt uses the ‘clustalo’ program (MSA algorithm) to align</li>
<li>I also used the ‘muscle’ program (MUltiple Sequence Comparison by Log-Expectation) in the command line to produce a different result, and standardised its output using BioPython for visualisation in SnapGene (<a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/muscle.pdf">view PDF</a>)</li>
</ul>
<pre class="bash"><code>$ cat ce.fasta hs.fasta mm.fasta xl.fasta | muscle -out muscle.fasta</code></pre>
<pre class="python"><code>from Bio import AlignIO
muscle = AlignIO.read(&#39;resources/wrn/muscle.fasta&#39;, &#39;fasta&#39;)
AlignIO.write(muscle, &#39;muscle.aln&#39;, &#39;clustal&#39;)</code></pre>
<p>With help of the <code>seqin</code> R package, the pairwise genetic differences between these 4 protein sequences can be calculated:</p>
<pre class="r"><code>aln &lt;- seqinr::read.alignment(&#39;./resources/wrn/clustal.aln&#39;, &#39;clustal&#39;)
alnDist &lt;- seqinr::dist.alignment(aln)
alnDist</code></pre>
<pre><code>##                     SP|Q19546|WRN_CAEEL SP|Q14191|WRN_HUMAN SP|O09053|WRN_MOUSE
## SP|Q14191|WRN_HUMAN           0.5710006                                        
## SP|O09053|WRN_MOUSE           0.5782777           0.3408993                    
## SP|O93530|WRN_XENLA           0.5903942           0.4396192           0.4499939</code></pre>
<p>These can be visulised by a phylogenetic tree produced by the <code>ape</code> package, as shown in Fig. <a href="#fig:phylo">5</a>.</p>
<pre class="r"><code>mydist &lt;- seqinr::dist.alignment(aln)
mytree &lt;- ape::nj(mydist)
myrootedtree &lt;- root(mytree, &#39;SP|Q19546|WRN_CAEEL&#39;, resolve.root=TRUE)
ape::plot.phylo(myrootedtree, type=&quot;p&quot;)</code></pre>
<div class="figure"><span id="fig:phylo"></span>
<img src="../tutorial/2020-01-23-mcb-toolbox_files/figure-html/phylo-1.png" alt="The phylogenetic relationship between the WRN homologs in 4 organisms" width="100%" />
<p class="caption">
Figure 5: The phylogenetic relationship between the WRN homologs in 4 organisms
</p>
</div></li>
<li>Design PCR primers to amplify the coding region, incorporating suitable restriction enzyme sites in each primer so that you can clone easily into the vector pIVEX2.3d (though there are many online primer design sites, I find this is best done manually to get a feel for it, but these days I use SnapGene (viewer version is free….). Hint: we want to keep the start codon ATG so chose an enzyme that contains ATG in its recognition site for the forward primer.
<ul>
<li>I opened the <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/original.dna">original sequence</a> with SnapGene. In the ‘Sequence’ view, I displayed translated amino acids with ‘Frame +1 only’ and thus located the stop codon (the start codon is obviously +1)<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Then I used the ‘Enzymes’ view to obtain a list of noncutters. By comparing this list with the MCS of the <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/vector.dna">piVEX2.3d</a>, I selected <em>Nde</em>I and <em>Sal</em>I as the RE sites to be used<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The <em>Sal</em>I site is not in frame with the C-terminal His tag and stop codon, so I included an additional random nucleotide 3’ to the RE site. The stop codon is excluded from the reverse primer because it is provided in the vector.</li>
<li>forward: <code>NN CAT ATGATAAGTGATGATGACGATCTACC</code>
<ul>
<li><code>CATATG</code> is <em>Nde</em>I restriction site. It ends with <code>ATG</code>, so adding <code>CAT</code> instead of <code>CATATG</code> to the 5’ suffices.</li>
<li>31-mer; 39% GC (&gt;40% if <code>NN</code> = <code>GC</code>); 26 annealed bases; T<sub>m</sub> = 56ºC<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></li>
</ul></li>
<li>reverse: <code>NN GTCGAC N AAGTTTGAATTTCTTCAATGGAGGAGC</code>
<ul>
<li><code>GTCGAC</code> is <em>Sal</em>I restriction site</li>
<li>36-mer; 43% GC; 28 annealed bases; T<sub>m</sub> = 58ºC</li>
</ul></li>
<li><em>In silico</em> PCR is done in SnapGene, and the resulting DNA can be downloaded <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/Amplified1.dna">here</a></li>
</ul></li>
<li><em>In silico</em> clone this into the vector pIVEX2.3d (file attached - .dna is a snap gene format, docx is of course in word). Why might we want to use this vector?
<ul>
<li>This vector has a C-terminal 6-His tag which allows efficient purification by affinity chromatography</li>
<li>The MCS is associated with the T7 promoter. T7 RNAP is highly processive, so consecutive expression is expected.</li>
</ul></li>
<li>Check that your new construct allows correct protein expression in frame with the hexahistidine tag of the vector (SnapGene is good for this, but you can also do it in a few more steps using ApE plus translation apps via Expasy)
<ul>
<li>Yes. (<a href="./resources/wrn/in-silico-cloning/Cloned1.dna.pdf">view result as PDF</a> or <a href="./resources/wrn/in-silico-cloning/Cloned1.dna">.dna file</a>)</li>
<li>Serendipitously, the MCS has an ATG in frame with our insert and the His-tag/STOP codon, so translation will probably start more upstream than we expected, which will add 3 additional amino acid residues to the N-terminus of the product. This ATG was not in frame with the His-tag before insertion, and thus unspotted by SnapGene or me.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></li>
</ul>
<div class="figure"><span id="fig:clone-serendipity"></span>
<img src="../img/clone-serendipity.png" alt="An additional in-frame start codon (red) upstream to the start codon of the insert (blue)." width="100%" />
<p class="caption">
Figure 6: An additional in-frame start codon (red) upstream to the start codon of the insert (blue).
</p>
</div></li>
<li>Predict the protein fold of the new protein (eg Swissprot). Show the effect of predicted enzyme active site mutations on the structure (eg new file in swissport, but Pymol is good for this).
<ul>
<li>Swiss-Model shown in Q2.</li>
</ul></li>
</ol>
<div id="bonus" class="section level3">
<h3>Bonus: Tianyi’s Restriction Site Finder</h3>
<p>See Fig. <a href="#fig:tianyi-re-001-1">7</a> and <a href="#fig:tianyi-re-001-2">8</a>.</p>
<p>This tool takes in two sequences (plain text or FASTA or SnapGene <code>.dna</code> file), one representing the cloning site and the other representing the GOI, and finds the restriction sites inside the cloning site but not in the GOI, and label them with their positions in the cloning site and their commecial suppliers.</p>
<p>The web interface is built upon the <a href="https://www.djangoproject.com">Django</a> framework and the data come from the <code>Restriction</code> module of <a href="https://biopython.org">BioPython</a>, which is (sort of) a wrapper around NEB’s <a href="http://rebase.neb.com/rebase/rebase.html">REBASE</a>.</p>
<div class="figure"><span id="fig:tianyi-re-001-1"></span>
<img src="../img/tianyi-re-001-1.png" alt="Restriction site finder by Tianyi. Query page." width="100%" />
<p class="caption">
Figure 7: Restriction site finder by Tianyi. Query page.
</p>
</div>
<div class="figure"><span id="fig:tianyi-re-001-2"></span>
<img src="../img/tianyi-re--001-2.png" alt="Restriction site finder by Tianyi. Result page." width="100%" />
<p class="caption">
Figure 8: Restriction site finder by Tianyi. Result page.
</p>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<div id="refs" class="references">
<div id="ref-Chen-2014">
<p>Chen, Sheng-Chia, Chi-Hung Huang, Chia Shin Yang, Tzong-Der Way, Ming-Chung Chang, and Yeh Chen. 2014. “Crystal Structure of Deinococcus Radiodurans Recq Helicase Catalytic Core Domain: The Interdomain Flexibility.” <em>BioMed Research International</em> 2014. Hindawi.</p>
</div>
<div id="ref-Cox-2013">
<p>Cox, Lynne S., and Penelope A. Mason. 2013. “The Role of Wrn Helicase/Exonuclease in Dna Replication.” In <em>The Mechanisms of Dna Replication</em>, edited by David Stuart. Rijeka: IntechOpen. <a href="https://doi.org/10.5772/51520">https://doi.org/10.5772/51520</a>.</p>
</div>
<div id="ref-Pike-2015">
<p>Pike, Ashley C. W., Shivasankari Gomathinayagam, Paolo Swuec, Matteo Berti, Ying Zhang, Christina Schnecke, Francesca Marino, et al. 2015. “Human Recq1 Helicase-Driven Dna Unwinding, Annealing, and Branch Migration: Insights from Dna Complex Structures.” <em>Proceedings of the National Academy of Sciences</em> 112 (14). National Academy of Sciences: 4286–91. <a href="https://doi.org/10.1073/pnas.1417594112">https://doi.org/10.1073/pnas.1417594112</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Why does this sequence start with ATG but not end with a stop codon? Are the additional downstream nucleotides for the convinience of primer design (is it worthwhile to include additional upstream nucleotides)?<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Is there a more automatic way of choosing RE sites? I wish I could select the MCS sequence of the vector and the GOI sequence, and then the intersection between the cutting sites of the former and the noncutting sites of the latter is computed automatically. Is this functionality hidden somewhere in SnapGene, or is it available elsewhere? If not I’ll try to make it! (A prototype is shown in Fig. <a href="#fig:tianyi-re-001-1">7</a> and <a href="#fig:tianyi-re-001-2">8</a>)<a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>When calculating %GC and T<sub>m</sub>, do we need to include the 5’ additional bases used for RE cutting/frame adjustment? In SnapGene it seems that these additional bases are included in the calculation of %GC but not T<sub>m</sub>. In practice, isn’t the annealing region the complete length of the primers for most DNA strands after a few rounds of synthesis? Do we need to increase the annealing temperature then?<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>It would be nice if it can highlight all ATGs regardless of the frame. I think additional in-frame ATGs can cause inclusion of unwanted aa residues and addtional out-of-frame ATGs can ruin the expression of GOI, especially in eukaryotes, because their ribosomes start translation at the first encountered ATG. It this correct?<a href="#fnref4" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../special/wuhan-coronavirus.html" class="archive-item-link">
                2019–20 outbreak of novel coronavirus &#39;2019-nCoV&#39; 
              </a>
              <span class="archive-item-date">
                2020-01-21
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="about-the-virus" class="section level1">
<h1>About the virus</h1>
<p>See <a href="https://en.wikipedia.org/wiki/2019–20_outbreak_of_novel_coronavirus">Wikipedia</a></p>
</div>
<div id="statistics" class="section level1">
<h1>Statistics</h1>
<details>
<summary>Source Code</summary>
<p>
<pre class="r"><code>library(tidyverse)
library(lubridate)
library(scales)
library(RColorBrewer)
library(maps)
library(mapdata)
library(maptools)

if(&#39;content&#39; %in% dir()){setwd(&#39;content/special/data&#39;)}

source(&#39;ncov_commons.R&#39;)

# read data ---------------------------------------------------------------

read_ncov &lt;- function(sheet, caseType, fn = &#39;wuhan.xlsx&#39;){
  readxl::read_excel(fn, sheet = sheet) %&gt;%
  replace(is.na(.), 0) %&gt;% 
  mutate(date = as_date(date)) %&gt;% 
  mutate_at(-1, cumsum) %&gt;% 
  gather(location, !!caseType, -1) %&gt;% 
  mutate(location=as_factor(location))
}

calc_ncov_params &lt;- function(df){
  mutate(df,
         current = all-death-cure,
         deathRate = death/all,
         cureRate = cure/all)
}

ncov &lt;- read_ncov(&#39;incidence-wiki&#39;, &#39;all&#39;) %&gt;% 
  left_join(read_ncov(&#39;death-wiki&#39;, &#39;death&#39;)) %&gt;% 
  left_join(read_ncov(&#39;cure-wiki&#39;, &#39;cure&#39;)) %&gt;% 
  calc_ncov_params()

ncovByHubei &lt;- 
  bind_rows(
    ncov %&gt;% # hubei
      filter(location==&#39;湖北省&#39;) %&gt;% 
      select(-location) %&gt;% 
      add_column(type=&#39;湖北&#39;), 
    ncov %&gt;% # non-hubei
      filter(location!=&#39;湖北省&#39;) %&gt;% 
      group_by(date) %&gt;% 
      summarise(all = sum(all, na.rm = TRUE),
                death = sum(death, na.rm = TRUE),
                cure = sum(cure, na.rm = TRUE)) %&gt;% 
      calc_ncov_params() %&gt;% 
      add_column(type=&#39;非湖北&#39;)
    )


typeColorPairs = c(cure=&#39;palegreen3&#39;, death=&#39;grey&#39;, current=&#39;khaki&#39;)

hubeiContrastColor = c(湖北=&quot;orangered2&quot;, 非湖北=&#39;seagreen3&#39;)

{ # death and cure rates
ncov %&gt;% 
  ggplot(aes(date, deathRate))+
  geom_line(aes(group=location, alpha=log(current)))+
  geom_line(data = ncovByHubei, aes(color=type), size=1.1)+
  scale_alpha_continuous(guide=FALSE)+
  scale_y_continuous(limits = c(0,0.15), minor_breaks = seq(0, 0.15, 0.01), labels = scales::percent)+
  theme_default+
  dateScale+
  labs(title=&#39;死亡率趋势图&#39;, x=&#39;日期&#39;,y=&#39;死亡率&#39;,
       color = &#39;地区&#39;,
       caption = &#39;注：黑/灰色线为各省/市/区死亡率趋势，累计病例数表现为线条深浅。&#39;)+
  scale_color_manual(values=hubeiContrastColor)

ggsave(&#39;img/china_death_rate.png&#39;, width = 13, height = 8)

ncov %&gt;% 
  ggplot(aes(date, cureRate))+
  geom_line(aes(group=location, alpha=log(current)))+
  geom_line(data = ncovByHubei, aes(color=type), size=1.1)+
  scale_alpha_continuous(guide=FALSE)+
  scale_y_continuous(limits = c(0,0.9), breaks = seq(0,0.8,0.2), labels = scales::percent)+
  theme_default+
  dateScale+
  labs(title=&#39;治愈率趋势图&#39;, x=&#39;日期&#39;,y=&#39;治愈率&#39;,
       color = &#39;地区&#39;,
       caption = &#39;注：黑/灰色线为各省/市/区治愈率趋势，累计病例数表现为线条深浅。&#39;)+
  scale_color_manual(values=hubeiContrastColor)

ggsave(&#39;img/china_cure_rate.png&#39;, width = 13, height = 8)
}

{ # 治愈死亡现存趋势
(p &lt;- ncov %&gt;%
  mutate(location = fct_reorder(location, -all, min)) %&gt;% 
  select(-all) %&gt;% 
  gather(type, cases, 3:5) %&gt;% 
  mutate(type=factor(type, levels=names(typeColorPairs)))%&gt;% 
  ggplot(aes(date, cases, fill=type))+
  geom_col()+
  scale_fill_manual(labels=c(&#39;治愈&#39;,&#39;死亡&#39;,&#39;现存&#39;), values = typeColorPairs)+
  labs(title = &#39;中国疫情趋势图&#39;, fill = &#39;图例&#39;, x=&#39;日期&#39;, y=&#39;病例数&#39;)+
  theme_date+
  dateScale+
  scale_y_continuous(expand = c(0, 1))
)

ggsave(&#39;img/china_cure_death_current_all.png&#39;, width = 13, height = 8)

p + facet_wrap(~location, scales = &#39;free_y&#39;)+
  theme(axis.ticks.x = element_blank(),
        axis.text.x = element_blank())+
  labs(title = &#39;中国各省市区疫情趋势图&#39;, fill = &#39;图例&#39;)

ggsave(&#39;img/china_cure_death_current_facet.png&#39;, width = 13, height = 8)
}

{ # 现存
ncov %&gt;% 
    ggplot(aes(date, current, fill=fct_reorder(location, -current, last)))+ 
    geom_area(position = position_stack())+
    dateScale+
    theme_date+
    fill_province+
    scale_y_continuous(breaks = seq(0, 70000, 10000))+
    labs(fill=&#39;省/市/区&#39;, x = &#39;日期&#39;, y=&#39;现存病例数&#39;, 
         title = &#39;中国现存病例趋势图&#39;)
ggsave(&#39;img/china_current.png&#39;, width = 13, height = 8)
}

setwd(&#39;../../..&#39;)</code></pre>
</p>
</details>
<div id="number-of-cases-reported-by-the-government" class="section level2">
<h2>Number of cases reported by the government</h2>
<p><img src="data/img/china_current.png" width="100%" /></p>
</div>
<div id="incidence-death-and-cure-over-time" class="section level2">
<h2>Incidence, Death and Cure Over Time</h2>
<p><img src="data/img/china_cure_death_current_all.png" width="100%" /></p>
<p><img src="data/img/china_cure_death_current_facet.png" width="100%" /></p>
</div>
<div id="death-and-cure-rates" class="section level2">
<h2>Death and Cure Rates</h2>
<p><img src="data/img/china_death_rate.png" width="100%" /></p>
<p><img src="data/img/china_cure_rate.png" width="100%" /></p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/vectors-used-in-bacterial-and-eukaryotic-host-cells.html" class="archive-item-link">
                Vectors Used in Bacterial and Eukaryotic Host Cells 
              </a>
              <span class="archive-item-date">
                2020-01-20
              </span>
              <div class="article-summary" style="display: none;">
                
                  What are the key similarities and differences between vectors used in bacterial and eukaryotic host cells? What can such vectors be used for?<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#the-structure-of-the-vectors"><span class="toc-section-number">1</span> The structure of the vectors</a><ul>
<li><a href="#simple-plasmids"><span class="toc-section-number">1.1</span> Simple plasmids</a></li>
<li><a href="#others"><span class="toc-section-number">1.2</span> Other bacterial vectors</a></li>
<li><a href="#other-eukaryotic-vectors"><span class="toc-section-number">1.3</span> Other Eukaryotic Vectors</a></li>
</ul></li>
<li><a href="#vector-construction"><span class="toc-section-number">2</span> Vector Construction</a><ul>
<li><a href="#simple-plasmids-for-both-bacteria-and-eukaryotes"><span class="toc-section-number">2.1</span> Simple plasmids (for both bacteria and eukaryotes)</a></li>
<li><a href="#other-vectors-for-bacteria"><span class="toc-section-number">2.2</span> Other vectors for bacteria</a></li>
<li><a href="#viral-vectors-for-eukaryotes"><span class="toc-section-number">2.3</span> Viral vectors for eukaryotes</a></li>
</ul></li>
<li><a href="#transformationtransfection"><span class="toc-section-number">3</span> Transformation/Transfection</a><ul>
<li><a href="#transformation-of-bacterial-cells"><span class="toc-section-number">3.1</span> Transformation of bacterial cells</a><ul>
<li><a href="#heat-shock-transformation"><span class="toc-section-number">3.1.1</span> Heat-shock transformation</a></li>
<li><a href="#electroporation"><span class="toc-section-number">3.1.2</span> Electroporation</a></li>
<li><a href="#other-methods"><span class="toc-section-number">3.1.3</span> Other methods</a></li>
</ul></li>
<li><a href="#transfection-of-eukaryotic-cells"><span class="toc-section-number">3.2</span> Transfection of Eukaryotic Cells</a><ul>
<li><a href="#physical-methods"><span class="toc-section-number">3.2.1</span> Physical methods</a></li>
<li><a href="#chemical-methods"><span class="toc-section-number">3.2.2</span> Chemical methods</a></li>
<li><a href="#stab-tf"><span class="toc-section-number">3.2.3</span> Biological methods</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>What are the key similarities and differences between vectors used in bacterial and eukaryotic host cells? What can such vectors be used for?</strong></p>
<p>There are various ways by which genes can be introduced into a host organism and expressed. The choice of source genetic material, vector, and the method of transformation/transfection for a particular experiment are largely dependent on the physiology of the host organism, the size of the transgene, and the purpose of the study. However, it is useful to catagorise these strategies into bacteria-compatible ones and eukaryote-compatible ones.</p>
<p>Most molecular cloning experiments begin with the production of a recombinant plasmid containing the gene of interest (GOI). For bacterial hosts, the recombinant plasmid is directly introduced into the cells. Bacteria naturally possess the machinary for replicating plasmids, which allows inheritance of the GOI in their progenies. Eukaryotic cells cannot reliably replicate plasmids, therefore introducing plasmids into eukaryotic cells normally results in <em>transient transfection</em>, which means the GOI is expressed in successfully tranfected cells but not in their progenies. In order to produce a <em>stable cell line</em>, the GOI must be integrated into the genome of the host cell (by homologous recombination), which is an extremely rare event in nature but can be induced artificially, for example by viral/bacterial infection or by designer nucleases such as TALEN, ZFN and CRISPR-Cas9. In order to be compatible with the different transformation/tranfection strategies and the physiology of the host cells, the molecular structure of the plasmids (or vectors of other kinds) vary.</p>
<div id="the-structure-of-the-vectors" class="section level1">
<h1><span class="header-section-number">1</span> The structure of the vectors</h1>
<div id="simple-plasmids" class="section level2">
<h2><span class="header-section-number">1.1</span> Simple plasmids</h2>
<p>Plasmids are extrachromosomal self-replicating cytoplasmic (usually circular) DNA elements found in prokaryotes and, less commonly, in eukaryotes. They come in various forms, from simple plasmids used for direct transformation and fosmids for phage transduction. Plasmids optimised for expressing recombinant proteins share several features, as illustrated in Fig. <a href="#fig:pet">1.1</a> and <a href="#fig:pcmv-gfp-lc3b">1.2</a>:</p>
<ul>
<li>An origin of replication (<em>ori</em>), which allows it to be duplicated in bacterial cells. The type of origin of replication affects the copy number. Eukaryotic vectors generally carry two <em>ori</em>s, one bacterial (e.g. pUC <em>ori</em>) and one viral (e.g. SV40 <em>ori</em>). The bacterial <em>ori</em> is for amplifying the plasmid in bacteria before transfection, and the viral <em>ori</em> allows for episomal amplification of plasmids in the eukaryotic host.<br />
</li>
<li>A cloning site where the GOI is inserted. Typically it is crowded with restriction enzyme cutting sites (compatible with RE/ligase dependent cloning), but it can also be made compatible with other cloning methods, such as the LIC-compatible pBLIC-puro<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> plasmid developed by <span class="citation">Patel et al. (2012)</span>.</li>
<li>A promoter and associated operator/enhancer. The promoter sequences for bacteria and eukaryotes are different (-35/-10 consensus sequence v.s. TATA box) For a bacterial vector, the promoter is usually controlled by an inducible operator so that protein expression is induced manually only when the optimal bacterial density is reached (e.g. <em>lac</em> operator induced by IPTG). In contrast, for a eukaryotic vector, the promoter is usually made as processive as possible, because the natural rate of transcription is generally low, and cell growth and division are not important (they do not serve to amplify the plasmid but to express the protein). For example, the pEF-BOS vector developed by <span class="citation">Mizushima and Nagata (1990)</span> contains the promoter of EF-1<span class="math inline">\(\alpha\)</span> (EF-1<span class="math inline">\(\alpha\)</span> is one of the most abundant proteins in eukaryotic cells, so its promoter is highly processive). Recently, promoters of viral origin (e.g. CMV, SV40) are gaining popularity.</li>
<li>A transcription terminator/polyA signal. The mechanisms of transcription termination in bacteria and eukaryotes differ. In bacteria, the terminator sequence consists of a symmetric DNA sequence, and its transcribed product folds into a hairpin structure, releasing RNA polymerase (RNAP) from DNA. In eukaryotes, CstF and CPSF bound on RNAP C-terminal domain recognise 3’-end processing (polyadenylation) sequence (AAUAAA and GU-rich sequence) and induce cleavage and polyadenylation.</li>
<li>A selectable marker. Usually an antibiotic resistance gene (bacteria and eukaryotes are sensitive to different antibiotics). Fluorophore genes are also common in eukaryotic vectors.</li>
<li>Some plasmids possess a start codon and associated Shine-Dalgarno sequence (for bacteria) or Kozak sequence (for eukaryotes).</li>
<li>Some plasmids have tag sequences near the start or end of the ORF for easy purification of the recombinant protein by affinity chromatography (e.g. hex-His, GST, HSV) and associated cleavage sequence (e.g. TEV) for easy removal of the tag.</li>
</ul>
<div class="figure"><span id="fig:pet"></span>
<img src="../img/pETBlue.jpg" alt="The bacterial expression vector pETBlue-2" width="100%" />
<p class="caption">
Figure 1.1: The bacterial expression vector pETBlue-2
</p>
</div>
<div class="figure"><span id="fig:pcmv-gfp-lc3b"></span>
<img src="../img/pcmv-gfp-lc3b.jpg" alt="The eukaryotic expression vector pCMV-GFP-LC3" width="100%" />
<p class="caption">
Figure 1.2: The eukaryotic expression vector pCMV-GFP-LC3
</p>
</div>
</div>
<div id="others" class="section level2">
<h2><span class="header-section-number">1.2</span> Other bacterial vectors</h2>
<p>There are other forms of circular DNA, each with different disirable properties (larger insertion size, low copy number, phage-compatibility, etc.). They are sometimes called ‘high capacity vectors’ because their insertion sizes are greater than simple plasmids.</p>
<p>Bacterial artificial chromosomes (BACs) contain some regions derived from a special plasmid called the F (fertility) factor: the region containing the origin of replication as well as genes that ensured its precise segregation during bacterial cell division. A great advantage of BAC vectors is the large insertion size (100-200 kb). But the very large insertion can be a problem, in that it cannot be manipulated by restriction enzymes. Instead, in vivo homologous recombination-based strategies are used to insert DNA fragments into BACs. (Fig. <a href="#fig:bac-mech">1.3</a>)</p>
<div class="figure"><span id="fig:bac-mech"></span>
<img src="../img/bac-mech.png" alt="Making a BAC" width="50%" />
<p class="caption">
Figure 1.3: Making a BAC
</p>
</div>
<p>Fosmids are hybrids of <span class="math inline">\(\lambda\)</span> phage DNA and bacterial F plasmid DNA. Fosmids are packaged into <span class="math inline">\(\lambda\)</span> phage particles, which delivers them into bacterial cells. Due to the presence of the F plasmid origins of replications, fosmids are maintained at a very low copy number (normally single copy).</p>
<p>Fig. <a href="#fig:plasmid-derivatives">1.4</a> is a schematic of 6 high capacity vectors <span class="citation">(Saraswathy and Ramalingam 2011)</span>.</p>
<div class="figure"><span id="fig:plasmid-derivatives"></span>
<img src="../img/plasmid-derivatives.png" alt="Schematic of 6 high capacity vectors." width="50%" />
<p class="caption">
Figure 1.4: Schematic of 6 high capacity vectors.
</p>
</div>
</div>
<div id="other-eukaryotic-vectors" class="section level2">
<h2><span class="header-section-number">1.3</span> Other Eukaryotic Vectors</h2>
<p>Yeast artificial chromosome (YAC) has a cloning capacity up to 3000 kbp. It is introduced into the yeast cells by electroporation, then it is maintained as a linear DNA-like chromosome. It is replicated along with other chromosomes in yeast and its copy number of one is maintained after cell divition.</p>
<p>Some viruses (e.g. Adenovirus, Lentivirus and Baculovirus) and bacteria (e.g. <em>Agrobacterium tumefaciens</em>) are reliable vectors for stable transfection of eukaryotic cells (see section <a href="#stab-tf">3.2.3</a>)</p>
</div>
</div>
<div id="vector-construction" class="section level1">
<h1><span class="header-section-number">2</span> Vector Construction</h1>
<div id="simple-plasmids-for-both-bacteria-and-eukaryotes" class="section level2">
<h2><span class="header-section-number">2.1</span> Simple plasmids (for both bacteria and eukaryotes)</h2>
<p>Traditionally, restriction endonucleases and T4 ligase are used to insert GOI into pladmids. This method has several drawbacks:</p>
<ul>
<li>the association between the short 2-4 nt overhangs (sticky ends) is weak</li>
<li>each DNA insert has to be inspected for any internal restriction sites</li>
<li>due to the above two reasons, it has very limited capability of constructing multi-fragment plasmids</li>
<li>incomplete DNA digestion and poor ligation yields</li>
<li>unwanted amino acids can be introduced to the expressed protein</li>
<li>the GOI can only be cloned in the vector position where the selected restriction site is present</li>
</ul>
<p>Ligation-independent cloning (LIC) overcomes many of the problems described above. In a T4 DNA polymerase-dependent approach, the sticky sequences are made by the 3’-to-5’ exonuclease activity of T4 polymerase. These sequences are long, allowing formation of stable recombinant plasmid without <em>in vitro</em> ligase treatment. The nicks in sugar-phosphate backbone are later fixed by the host’s ligase. As T4 polymerase (exonuclease activity) always proceeds from 3’ ends, any internal sticky sequences will not be disrupting and thus the custom stiky sequence can be used for any DNA inserts.</p>
<p>Fig. <a href="#fig:lic">2.1</a> compares the mechanisms of the traditional RE/ligase-based cloning and the T4 polymerase-dependent LIC approach.</p>
<div class="figure"><span id="fig:lic"></span>
<img src="../img/lc-lic.jpg" alt="Left, traditional cloning based on restriction endonuclease and ligase; right, ligase-independent cloning based on T4 polymerase" width="100%" />
<p class="caption">
Figure 2.1: Left, traditional cloning based on restriction endonuclease and ligase; right, ligase-independent cloning based on T4 polymerase
</p>
</div>
<p>There are also other alternatives to the traditional method:</p>
<ul>
<li>the Gateway cloning system exploits the site-specific recombination system utilized by bacteriophage lambda to shuttle sequences between plasmids bearing flanking compatible recombination attachment (att) sites</li>
<li>Gibson assembly is a molecular cloning method which allows for ‘stitching’ multiple overlapping DNA fragments together in a single, isothermal reaction, using exonuclease, polymerase, and ligase.</li>
<li>TOPO cloning expoits Taq polymerase’s feature that adds non-specific A to the 3’ end, and uses topoisomerase I to hold vectors open and to promote ligation</li>
</ul>
</div>
<div id="other-vectors-for-bacteria" class="section level2">
<h2><span class="header-section-number">2.2</span> Other vectors for bacteria</h2>
<p>BACs and fosmids: see section <a href="#others">1.2</a></p>
</div>
<div id="viral-vectors-for-eukaryotes" class="section level2">
<h2><span class="header-section-number">2.3</span> Viral vectors for eukaryotes</h2>
<p>See section <a href="#lenti">3.2.3.1</a></p>
</div>
</div>
<div id="transformationtransfection" class="section level1">
<h1><span class="header-section-number">3</span> Transformation/Transfection</h1>
<div id="transformation-of-bacterial-cells" class="section level2">
<h2><span class="header-section-number">3.1</span> Transformation of bacterial cells</h2>
<div id="heat-shock-transformation" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Heat-shock transformation</h3>
<p>Heat-shock transformation is used for small vectors such as plasmids. The general procedure is as follows:</p>
<ol style="list-style-type: decimal">
<li>Host cells are incubated in a solution containing divalent cations (typically CaCl<sub>2</sub>) on ice.</li>
<li>CaCl<sub>2</sub> partially disrupts the cell membrane, which allows the recombinant DNA enter the host cell. Such cells are called competent cells.</li>
<li>Cells are exposed to a heat pulse (heat shock), and the thermal imbalance causes the entry of DNA through disrupted plasma membrane.</li>
</ol>
</div>
<div id="electroporation" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Electroporation</h3>
<p>Electroporation can be used for larger vectors such as BAC and PAC<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The general procedure is as follows:</p>
<ol style="list-style-type: decimal">
<li>Host cells are placed into a cuvette, together with the vector. The cuvette is connect to electrodes.</li>
<li>The cuvette containing the mixture is subjected to intense electric pulses (2500 V/cm for bacteria, lower for animal and plant cells) each lasting for only a few milliseconds</li>
<li>Most cells would die under such treatment, but for those survived, their membranes are polarised by the electric field and are disrupted, and DNA enters through the pores. Finally, the membrane reseals after the treatment.</li>
</ol>
</div>
<div id="other-methods" class="section level3">
<h3><span class="header-section-number">3.1.3</span> Other methods</h3>
<p>Cosmid, fosmid and PAC vectors are introduced into bacterial cells via bacteriophages. (See section <a href="#others">1.2</a>)</p>
</div>
</div>
<div id="transfection-of-eukaryotic-cells" class="section level2">
<h2><span class="header-section-number">3.2</span> Transfection of Eukaryotic Cells</h2>
<p>Transfection of eukaryotic cells can be transient or stable. Transient gene expression is generally used in academic settings, when the product of the GOI (usually a protein) is used for short-term research purposes. Stable transfection is usually used in indusdries, but it is also used in academia for specific purposes.</p>
<div id="physical-methods" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Physical methods</h3>
<p>In general, physical methods work by briefly pushing holes through the memrbane. The three specific methods are:</p>
<ul>
<li>electroporation, which is described above as a transformation method for bacteria, can also be applied to eukaryotic cells. The mechanism is the same but the voltage used for eukaryotic cells are lower.</li>
<li>microinjection entails injection of recombinant DNA (or RNA) into the cytoplasm or nucleus. This is method particularly convinient for large eukaryotic cells, such as the syncytial gonadal cell of <em>Caenorhabditis elegans</em> and the fertilised egg of <em>Mus musculus</em>.</li>
<li>biolistic delivery of tungsten- or gold-coated DNA (or RNA) by gene gun is mostly used with plant cells.</li>
</ul>
</div>
<div id="chemical-methods" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Chemical methods</h3>
<p>In general, chemical methods work by sheding the negative charge on DNA so as to apprach and cross the plasma membrane, which is also negatively charged.</p>
<ul>
<li>CaPO<sub>4</sub> co-precipitation involves mixing DNA with CaCl<sub>2</sub> in a phosphate solution to generate a CaPO<sub>4</sub>-DNA co-precipitate, which is then added to cultured cells. DNA enters the cell by endocytosis.</li>
<li>the cationic diethylaminoethyl(DEAE)-dextran work similarly by associating and neutralising negatively charged DNA</li>
<li>cationic lipids work similarly by forming DNA-lipid complexes (DNA is not encapsulated in liposomes)</li>
<li>By far, the most cost-effective chemical carrier is polyethylenimine (PEI), which works similarly by forming DNA-PEI complexes <span class="citation">(L’Abbé et al. 2018)</span></li>
</ul>
</div>
<div id="stab-tf" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Biological methods</h3>
<p>The physical and chemical methods described above are generally used for transient transfection. Although they may produce stably transfected cells (when the rare homologous recombination events occur), the probability is very low and the GOI is inserted randomly and is thus not expressed with greatest efficiency (the chances are low that the GOI is inserted into a highly ‘transcriptionally open’ region).</p>
<p>Biological methods significantly improve the chance of integration of GOI into the host’s genome, and can even direct the site of insertion.</p>
<div id="lenti" class="section level4">
<h4><span class="header-section-number">3.2.3.1</span> Lentiviral transduction as an example of viral transfection <span class="citation">(Merten, Hebben, and Bovolenta 2016)</span></h4>
<p>Lentiviruses (LV) carry a genome made of (+) strand RNA. Upon infection, the viral RNA genome is reverse-transcribed and the cDNA is stably integrated into the host’s genome. Unlike <span class="math inline">\(\gamma\)</span>-retroviruses, LVs exploit active nuclear transport and thus do not rely on cell division to access the nucleus. Fig. <a href="#fig:lentivirus-3-gen">3.1</a> is an example of the third generation HIV-1 vector system.</p>
<div class="figure"><span id="fig:lentivirus-3-gen"></span>
<img src="../img/lentivirus-3-gen.jpg" alt="Production of recombinant LV particles and their infection. Production of recombinant LV particles is usually accomplished by transfection of HEK293T cells with a set of three helper plasmids and additionally a LV vector plasmid that encodes the GOI as well as optional reporter (e.g resistance, fluorophore) genes. LV particles released from HEK helper cells infect target cells and integrate the GOI into their genome." width="100%" />
<p class="caption">
Figure 3.1: Production of recombinant LV particles and their infection. Production of recombinant LV particles is usually accomplished by transfection of HEK293T cells with a set of three helper plasmids and additionally a LV vector plasmid that encodes the GOI as well as optional reporter (e.g resistance, fluorophore) genes. LV particles released from HEK helper cells infect target cells and integrate the GOI into their genome.
</p>
</div>
</div>
<div id="ti-plasmid-and-agrobacterium-tumefaciens-infection" class="section level4">
<h4><span class="header-section-number">3.2.3.2</span> Ti plasmid and <em>Agrobacterium tumefaciens</em> infection</h4>
<p><em>Agrobacterium tumefaciens</em> is the bacterium that causes crown gall disease in plants. When the it infects a plant cell, a part of its Ti plasmid, called T-DNA, is transferred and inserted (at random) into the genome of the plant cell. Thus, the T-DNA can be replaced with any GOI for stable transfection of plant cells.</p>
</div>
<div id="crispr-cas9-transfection" class="section level4">
<h4><span class="header-section-number">3.2.3.3</span> CRISPR-Cas9 Transfection</h4>
<p>The CRISPR-Cas9 system comprises a short guide RNA (gRNA) and the Cas9 nuclease. The gRNA guides Cas9 to a specific gene locus that is complementary to its crRNA portion, then Cas9 makes a double strand break, which is dependent on the PAM (protospacer adjacent motif) sequence immediately downstream of the recognition sequence.</p>
<p>Following DNA cleavage, the break is repaired by either non-homologous end joining (NHEJ) or homology directed repair (HDR). It is the HDR mechanism that may cause integration of GOI into the cleavage site.</p>
<div class="figure"><span id="fig:crispr"></span>
<img src="../img/crispr.jpg" alt="gRNA directed DNA cleavage by Cas9" width="100%" />
<p class="caption">
Figure 3.2: gRNA directed DNA cleavage by Cas9
</p>
</div>
</div>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Balbas-2012">
<p>Balbás, Paulina, and Argelia Lorence. 2012. <em>Recombinant Gene Expression: Reviews and Protocols</em>. Book. 2nd ed. Vol. 267. Methods in Molecular Biology. Springer.</p>
</div>
<div id="ref-Griffiths-2015a">
<p>Griffiths, Anthony J. F., Susan R. Wessler, Sean B. Carroll, and John Doebley. 2015. “Gene Isolation and Manipulation.” In <em>Introduction to Genetic Analysis</em>, 11th ed. W.H. Freeman.</p>
</div>
<div id="ref-Hacker-2018">
<p>Hacker, David L. 2018. <em>Recombinant Protein Expression in Mammalian Cells: Methods and Protocols</em>. Book. Vol. 1850. Methods in Molecularbiology. Humana Press. <a href="https://doi.org/10.1007/978-1-4939-8730-6">https://doi.org/10.1007/978-1-4939-8730-6</a>.</p>
</div>
<div id="ref-LAbbe-2018">
<p>L’Abbé, Denis, Louis Bisson, Christian Gervais, Eric Grazzini, and Yves Durocher. 2018. “Transient Gene Expression in Suspension Hek293-Ebna1 Cells.” In <em>Recombinant Protein Expression in Mammalian Cells: Methods and Protocols</em>, edited by David L. Hacker. Vol. 1850. Humana Press.</p>
</div>
<div id="ref-Merten-2016">
<p>Merten, Otto-Wilhelm, Matthias Hebben, and Chiara Bovolenta. 2016. “Production of Lentiviral Vectors.” <em>Molecular Therapy - Methods &amp; Clinical Development</em> 3: 16017. <a href="https://doi.org/https://doi.org/10.1038/mtm.2016.17">https://doi.org/https://doi.org/10.1038/mtm.2016.17</a>.</p>
</div>
<div id="ref-Mizushima-1990">
<p>Mizushima, S, and S Nagata. 1990. “PEF-Bos, a Powerful Mammalian Expression Vector.” <em>Nucleic Acids Res</em> 18 (17). Osaka Bioscience Institute, Japan.: 5322. <a href="https://doi.org/10.1093/nar/18.17.5322">https://doi.org/10.1093/nar/18.17.5322</a>.</p>
</div>
<div id="ref-Patel-2012">
<p>Patel, Asmita, Anisleidys Muñoz, Katherine Halvorsen, and Priyamvada Rai. 2012. “Creation and Validation of a Ligation-Independent Cloning (Lic) Retroviral Vector for Stable Gene Transduction in Mammalian Cells.” <em>BMC Biotechnology</em> 12 (1): 3. <a href="https://doi.org/10.1186/1472-6750-12-3">https://doi.org/10.1186/1472-6750-12-3</a>.</p>
</div>
<div id="ref-Saraswathy-2011a">
<p>Saraswathy, Nachimuthu, and Ponnusamy Ramalingam. 2011. “High Capacity Vectors.” In <em>Concepts and Techniques in Genomics and Proteomics</em>, 49–56. Woodhead Publishing. <a href="https://doi.org/https://doi.org/10.1533/9781908818058.49">https://doi.org/https://doi.org/10.1533/9781908818058.49</a>.</p>
</div>
<div id="ref-Singha-2017">
<p>Singha, Tapan Kumar, Pooja Gulati, Aparajita Mohanty, Yogender Pal Khasa, Rajeev Kumar Kapoor, and Sanjay Kumar. 2017. “Efficient Genetic Approaches for Improvement of Plasmid Based Expression of Recombinant Protein in Escherichia Coli: A Review.” Journal Article. <em>Process Biochemistry</em> 55: 17–31. <a href="https://doi.org/https://doi.org/10.1016/j.procbio.2017.01.026">https://doi.org/https://doi.org/10.1016/j.procbio.2017.01.026</a>.</p>
</div>
<div id="ref-Valla-2014">
<p>Valla, Svein, and Rahmi Lale. 2014. <em>DNA Cloning and Assembly Methods</em>. Book. Vol. 1116. Methods in Molecular Biology. Humana Press. <a href="https://doi.org/10.1007/978-1-62703-764-8">https://doi.org/10.1007/978-1-62703-764-8</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>pBLIC-puro was a gift from Priyamvada Rai (Addgene plasmid # 45197 ; <a href="http://n2t.net/addgene:45197" class="uri">http://n2t.net/addgene:45197</a> ; RRID:Addgene_45197)<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>P1-derived artificial chromosome<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img
              src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>