<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>2-Trinity - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="2-Trinity - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../term/2-trinity/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">2-Trinity</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../tutorial/redox-based-modulation-of-enzyme-activity.html" class="archive-item-link">
                Redox-Based Modulation of Enzyme Activity 
              </a>
              <span class="archive-item-date">
                2020-06-17
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>Presentation see <a href="../thioredoxin-presentation/">here</a></p>
<!--
Thioredoxins are kept in the reduced state by the flavoenzyme thioredoxin reductase in a NADPH-dependent reaction. 

Plants have an unusually complex complement of Trxs composed of six well-defined types (Trxs f, m, x, y, h, and o) that reside in different cell compartments and function in an array of processes. In 2010 it was discovered for the first time that thioredoxin proteins are able to move from cell to cell, representing a novel form of cellular communication in plants.

Thioredoxins are small ubiquitous proteins with two redox-active cysteine rediuses separated by a pair of amino acids (CXXC)

TRXs *m, f, x, y, z* isoforms are located in the chloroplast, whereas TRXs *o* are mitochondrial and TRX *h* proteins are found in different subcellular compartments.

TRX *h* is the largest TRX gene family in plants, comprising nine different isoforms in Arabidopsis. First isolated from spinach leaves; outisde of the chloroplast. Lack of a transit peptide; initially assumed as solely cytosolic proteins. However, alternative locations were also observed for some of them: plasma membrane, endomembranes (ER & Golgi), mitochondria, and the nucleus. Highly redundant -> difficult to unequivocally determine the function of each TRX *h* isoform

Besides the plastid and mitochondria, Trxs are also found in the cytosol, ER and nucleus of plant cells. Little is known about the function of nucleoredoxins (Nrx) and ER-Trxs, the function of cytosolic Trx *h* has been well documented.

Trx regulates photosynthesis and the TCA cycle in chloroplasts and mitochondria, respectively.

Extraplastidial Trxs hhave been shown to be important in the regulation of different processes such as seed germination, seedling establishment, C, N, and mitochondrial metabolism as well as for plant stress tolerance.

# Trx reduction systems

In extraplastidial compartments the reducing power for Trx reduction is provided by NADPH via NADPH-Trx reductase A (NTRA) and NTRB.

The cytosolic h-type thioredoxins act as key regulators of seed germination and are recycled by NADPH-dependent thioredoxin reductase

## In seed

Seeds develop in a metabolically active state but become quiescent during maturatiion and drying. This period is characterised by low water content and low metabolic activity as well as the non-enzymatic accumulation of reactive oxygen species (roS) that oxidise proteins and RNA. Oxidation of the thiol groups of cysteine -> changes in protein structure -> lowering solubility and modifying enzyme function -> protection against denaturation and proteolysis as the seed matures and dries.

Under favourable conditions, the seed germinates and thhe seedling draws on stored reserves for growth and development. Large increase in ROS produced as side-products of metabolism. ROS have important signalling roles in germination (but also cause oxidative stress if present in high amounts). The Trx system represents an important waay to regulate ROS in seeds.

## NTR

The NTRA and NTRB are found in both cytosol and mitochondrion, while thhe first has also been found in the nucleuus. NTRA and NTRB are mainly expressed in the cytosol and mitochondrion, respectively.

The NADPH used by the NTR/Trx system is generated by the OPPP through the activity of G6PDH and 6-PGDH, while the souruce of NADPH in the mitoochondion is throuugh the activity of IDH or NADHk. 


# Methods

## Alkylation Labelling


@Kobrehel-1992 applied the fluorescnet alkylating reagent monobromobimane (mBBr) to label free thiol groups in proteins following reduction by Thx. -> discovered that gliadins and glutenin are reduced in vivo during germination

Buchanan and co-workers pioneered the use of proteomics for Trx target identiﬁcation and applied the ﬂuorescent alkylating reagent monobromobimane (mBBr) to label free sulfhydryl (SH) groups in proteins following reduction by Trx [37]. This approach can be applied not only to in vitro but to in vivo reactions as well [2,40,45,46]. To identify targets from peanut seeds, mBBr-labelled proteins were separated by two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) and ﬂuorescent spots were subjected to trypsin digestion followed by either Edman sequencing or mass spectrometry for protein identiﬁcation

Lane 1, Control: no addition [1.1]; lane 2, GSH/GR/NADPH: GSH, glutathione re- ductase(fromspinachleaves),andNADPH [1.2];lane3,NGS: NADPH, GSH, glutathione reductase (from spinach leaves), and glutaredoxin (from E. coli) [1.6]; lane 4, NTS: NADPH, NTR, and thioredoxin (both proteins from E. coli) [3.9]; lane 5, MET/T(Ec): ,B- mercaptoethanolandthioredoxin(E.co/i)[1.8];lane6,DTT [2.5]; lane7,DTT/T(Ec):DTT andthioredoxin(E.co/i)[3.0];lane8,DTT/ T(W): same as lane 7 except with wheat thioredoxin h [3.6]; lane 9, NGS,-Gliadin: same as lane 3 except without the gliadin protein fraction [0]; lane 10, NTS,-Gliadin: same as lane 4 except without thegliadinproteinfraction[0].

The above findings demonstrate that thioredoxin specifically reduces components of the wheat gliadin and glutenin fractions when tested in vitro. The results, however, provide no indication as to whether these proteins are reduced in vivo during germination-a question that, to our knowledge, has not been previously addressed (cf. 27).

To answer this question, we applied the mBBr/SDS-PAGE technique to monitor the reduction status of proteins in the germinating seed. We observed that reduction of components in the Osborne fractions increased progressively with time and reached a peak after 2 to 3 d germination (Fig. 4). The observed increase in reduction ranged from twofold with the gliadins to threefold with the albumin/globulins and fivefold with the glutenins. The net redox change was most apparent in the gliadins and glutenins found to undergo maximal reduction by thioredoxin, i.e. the 30- to 50-kD gliadins and 30- to 60-kD glutenins (cf. Figs. 1 and 3). The redox changes taking place in these fractions during germination are shown in Figure 5.

## CXXS

@Verdoucq-1999

In the thiol-disulfide exchange reaction, Trx forms a transient intermolecular disufide bond with the target protein involving the more N-terminal cysteine in the active site of Trx (CXXC). Then, the other cysteine (CXXC) attacks the disulﬁde bond of the Trx-target intermediate and releases the reduced target and oxidized Trx.

If instead this second cysteine is replaced with a serine residue through site-directed mutagenesis the Trx loses its capability to reduce the intermolecular disulﬁde bond and the covalent Trx-target complex will be stabilized. This strategy was first applied by Verdoucq et al. [49] who constructed a monocysteinic Trx mutant of A. thaliana Trxh3. The protein was heterologously expressed in Saccharomyces cerevisae and it was successfully stabilized in a complex with a peroxiredoxin target protein.

The concept was further developed by Motohashi et al. [50] who immobilized a monocysteinic Trx m on a resin designed to trap targets on a column. Non-targets were removed with the column wash and the trapped targets could subsequently be eluted with a chemical reductant such as DTT (Fig. 3). This approach has been used to identify target proteins from plant mitochondria [51–53], chloroplasts [54], cytosolic extracts [55], and has also been combined with the thiol alkylation approach described above to identify putative target proteins, for example in seeds from M. truncatula [40].

Improved by @Motohashi-2001 (initially applied to Trx m) 
Sepharose 4B resin

@Yamazaki-2004 Target Proteins of the Cytosolic Thioredoxins in Arabidopsis thaliana.
The obtained proteins were separated by two-dimensional gel electrophoresis and analyzed by matrix-assisted laser desorption ionization time-of-flight mass spectrometry. 

The results in this study and recent proteomics studies on the target proteins of chloroplast thioredoxin indicate the versatility and the physiological significance of thioredoxin as reductant in plant cell.

## Differential Labelling of Recombinant BASI (IAA and 4-VP)

- for the well studied chloroplastic thiioredoxins, target disulphide bonds in target enzymes were mainly idenitified by using strategies based on site directed mutagenesis of target proteins
- hoowever, to produce recombinant target proteiins and their mutants would be entremely time-consumiing considering the large number of potential target proteins of thioredoxin h that have been identifieid
- Using mass spectrometry techniques, peptide fragments containing modifications are recognised in hte peptide mass maps according to an exact mass shift from the expected thhiioretical values.
- the thiol and disulphiide forms of cysteine side chains, however, are usually not distinguishhed in MS because complete reduction and alkylation of cysteines are performed during regular sample preparation
- A novel strategy is developed by @Maeda-2005.
- Trypsin digesting
- According to the mass difference of cysteines modified by these two reagents, conversion of disulphides to thiols catalysed by HvTrxh1 was shown in the peptide mass maps of recombinant BASI.

# Rice

Cereals store reserve proteins in the endosperm during maturation of grain. In germination and early seedling development, these proteins are digested to provide nitrogen for growth. In rice and certain other seeds, the major storage proteins are 11S globulins housed in the protein body of the endosperm. Generally, 11S globulins are hexamers composed of identical subunits, each containing two polypeptide chains (Hara et al. 1976, Müntz 1998). The acidic chain, with one presumed intramolecular disulfide bond, is connected to the basic chain by an intermolecular disulfide bond (Staswick et al. 1984, Lawrence et al. 1994) (Fig. 1). The basic chain also contains a free sulfhydryl group. In rice, the proteins corresponding to 11S globulins, the glutelins, are unique in that they exist as large macromolecular complexes formed by disulfide and hydrophobic interactions of acidic and basic polypeptides (Katsube et al. 1999).
-->
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Geigenberger-2017" class="csl-entry">
Geigenberger, Peter, Ina Thormählen, Danilo M. Daloso, and Alisdair R. Fernie. 2017. <span>“The Unprecedented Versatility of the Plant‎ Thioredoxin System.”</span> <em>Trends in Plant Science</em> 22 (3): 249–62. https://doi.org/<a href="https://doi.org/10.1016/j.tplants.2016.12.008">https://doi.org/10.1016/j.tplants.2016.12.008</a>.
</div>
<div id="ref-Hagglund-2016" class="csl-entry">
Hägglund, Per, Christine Finnie, Hiroyuki Yano, Azar Shahpiri, Bob B. Buchanan, Anette Henriksen, and Birte Svensson. 2016. <span>“Seed Thioredoxin h.”</span> <em>Biochimica Et Biophysica Acta (BBA) - Proteins and Proteomics</em> 1864 (8): 974–82. https://doi.org/<a href="https://doi.org/10.1016/j.bbapap.2016.02.014">https://doi.org/10.1016/j.bbapap.2016.02.014</a>.
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/improving-photosynthesis-by-photorespiratory-bypasses.html" class="archive-item-link">
                Improving Photosynthesis by Photorespiratory Bypasses 
              </a>
              <span class="archive-item-date">
                2020-06-09
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#attempts-to-inhibit-photorespiration-and-improve-specificity-of-rubisco"><span class="toc-section-number">1</span> Attempts to Inhibit Photorespiration and Improve Specificity of Rubisco</a></li>
<li><a href="#bypassing-photorespiration"><span class="toc-section-number">2</span> Bypassing Photorespiration</a><ul>
<li><a href="#implementing-the-bacterial-glycerate-pathway"><span class="toc-section-number">2.1</span> Implementing The Bacterial Glycerate Pathway</a></li>
<li><a href="#inhibition-of-the-glycolate-transporter"><span class="toc-section-number">2.2</span> Inhibition of the Glycolate Transporter</a></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">3</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>Discuss the feasibility and desirability of improving photosynthesis in higher plants through the use of photorespiratory bypasses.</strong></p>
<p>Plants and algae use the Calvin cycle to capture inorganic carbon (as CO<sub>2</sub>) and use ATP and NADPH generated in light reactions to convert it to organic carbon. This process, known as photosynthesis, is the predominant way to replenish organic carbon in the biosphere, and is the basis for human food production. Despite its crucial role, the key enzyme in the pathway that attaches the inorganic CO<sub>2</sub> to the organic sugar 1,5-bisphosphate (RuBP), called Rubisco, is not efficient, in that it can also use O<sub>2</sub> instead of CO<sub>2</sub> as the substrate, which leads to production of one molecule each of 3-PGA and 2-phosphoglycerate (2-PG). Plants use a pathway known as photorespiration to remove the toxic 2-PG and to salvage some (75%) carbon lost from the Calvin cycle by converting two 2-PG to one 3-PGA while releasing CO<sub>2</sub> (Figure <a href="#fig:pr-bypass-native">0.1</a> ). Photorespiration is one of the major factors contributing to inefficiency of photosynthesis because 1) much CO<sub>2</sub> released in mitochondria is escaped into the atmosphere and not re-assimilated in the Calvin cycle, resulting in the reduction of the amount of Calvin cycle intermediates; 2) O<sub>2</sub> directly competes with CO<sub>2</sub> for Rubisco reaction (in ambient atmosphere and at 25<sup>o</sup>C, 25% of the Rubisco reactions in an average C3 plant uses O<sub>2</sub>); 3) in photorespiration, ammonia is released and needs to be reassimilated at the expense of one molecule of ATP and two reducing equivalents (reduced ferredoxins) per molecule of ammonia; 4) reducing power is dissipated in the oxidation of glycolate using molecular oxygen; 5) ATP is also consumed for phosphorylating glycerate in the final step in the photorespiration pathway (because the phosphate group on 2-PG is removed by phosphatase and is thus not conserved). The total direct energy cost for converting two 2PG to one 3PGA is 3.5 ATP and 2 NADPH. The rate of photorespiration is increased by the ratio of O<sub>2</sub> concentration to CO<sub>2</sub> concentration experienced by Rubisco (in C<sub>3</sub> plants, this is similar to atmospheric ratio), and is thus increased by high temperature, as solubulity of CO<sub>2</sub> decreases faster than does O<sub>2</sub> with increasing temperature.</p>
<div class="figure"><span id="fig:pr-bypass-native"></span>
<img src="../img/pr-bypass-native.png" alt="The native photorespiration pathway." width="100%" />
<p class="caption">
Figure 0.1: The native photorespiration pathway.
</p>
</div>
<p>Several groups of photosynthetic organisms independently evolved strategies to minimise photorespiration and are thus able to grow in hot and/or CO<sub>2</sub>-deficient conditions. For example, C<sub>4</sub> and CAM plants uses PEP carboxylase, which uses <span class="math inline">\(HCO_3^-\)</span> instead of CO<sub>2</sub> as the substrate, to perform an initial carboxylation, which results in production of 4-carbon malate/oxaloacetate that is later used to release CO<sub>2</sub> for Rubisco reaction. The initial carboxylation and later decarboxylation is separated spatially in C<sub>4</sub> plants and temporally in CAM plants. In aquatics environments where CO<sub>2</sub> concentration is low, cyanobacteria and green algae uses bicarbonate pumps to actively increase intracellular CO<sub>2</sub> (<span class="math inline">\(HCO_3^-\)</span>) concentration, and their Rubisco is in close proximity with carbonic anhydrase, which converts <span class="math inline">\(HCO_3^-\)</span> to CO<sub>2</sub>.</p>
<p>The vast majority of land plants (especially crops), however, adopts the C<sub>3</sub> metabolism scheme, which means they do not actively concentrate CO<sub>2</sub> and are susceptible to reduction of photosynthesis efficiency by photorespiration. Thus, it has been believed that reduction in photorespiration can improve photosynthesis and hence crop yield.</p>
<div id="attempts-to-inhibit-photorespiration-and-improve-specificity-of-rubisco" class="section level1">
<h1><span class="header-section-number">1</span> Attempts to Inhibit Photorespiration and Improve Specificity of Rubisco</h1>
<p>Initial attempts to reduce the effect of photorespiration include inhibition, knock-down and knock-out of the enzymes involved in photorespiration. The relevant genes are identified by screening for mutants that have impaired photorespiratory function. However, most of such mutants exhibited chlorosis and stunted growth. The rationale is simple: reduction of photorespiratory enzyme activity not only does not reduce the production of 2-PG by Rubisco, but also causes the toxic molecules resulted from partial metabolism of 2-PG to accumulate. Also, it has been suggested that photorespiratory enzymes may be also involved in other pathways related to serine biosynthesis and nitrogen metabolism.</p>
<p>Researches that attempted for a Rubisco enzyme with a greater specificity for CO<sub>2</sub> were also conducted. However, it was found that there is a inverse relationship between specificity and rate of reaction. As adaptations to achieve maximal rate of photosynthesis in different environments, Rubisco with high specificity and low turnover rate are typically found in C<sub>3</sub> plants that grow under high temperature/low CO<sub>2</sub>, and Rubisco with high turnover and low specificity are generally found in C<sub>3</sub> plants that grow under lower temperature, as well as in organisms with a CO<sub>2</sub>-concentrating mechanism (C<sub>4</sub>/CAM/algae/cyanobacteria).</p>
</div>
<div id="bypassing-photorespiration" class="section level1">
<h1><span class="header-section-number">2</span> Bypassing Photorespiration</h1>
<p>As attempts to improve Rubisco specificity and inhibit photorespiration directly both did not yield much outcome, photorespiratory bypasses becomes the new hotspot for improving photosynthesis. In general, photorespiratory bypasses are ‘designed photorespiratory pathways’ that aim to minimise energy comsumption and CO<sub>2</sub> loss to the atmosphere. Successful bypasses usually involve confining photorespiration to less compartments (ideally in chloroplast only), preventing lost of reducing power via oxidase, preventing release of NH<sub>3</sub>, and reducing the number of steps of reactions.</p>
<div id="implementing-the-bacterial-glycerate-pathway" class="section level2">
<h2><span class="header-section-number">2.1</span> Implementing The Bacterial Glycerate Pathway</h2>
<p>Many bacteria possess a simple and efficient pathway for converting glyoxylate to glycerate. This pathway involves two steps: 1) condensation of two glyoxylate to tartronate semialdehyde (by glyoxylate carboxyligase (GCL)) while releasing CO<sub>2</sub>; 2) reduction of tartronate semialdehyde by tartronate semialdehyde reductase (requiring one NADH) to glycerate. Compared with the native photorespiratory pathway (which converts glyoxylate to glycine in peroxisome and decarboxylates glycine in the mitochondrion), NH<sub>3</sub> release is prevented and, if this pathway can be implemented in the chloroplast, CO<sub>2</sub> release is localised to the chloroplast and is more likely to be re-assimilated by Rubisco instead of escaping to the atmosphere.</p>
<p>To implement the pathway in the chloroplast, a targeting sequence (e.g. Arabidopsis Rubisco small subunit (RbcS) or phosphoglucomutase transit peptide sequence) needs to be added to the N terminus of the gene constructs of the desired enzymes. In addition, the enzyme(s) that converts glycolate to glyoxylate also need(s) to be imported to the chloroplast. In normal plants, this conversion is catalysed by glycolate oxidase in peroxisome, and the same enzyme can be targeted to the chloroplast. However, glycolate oxidase is inefficient, as it uses O<sub>2</sub> as the hydrogen acceptor, which wastes reducing power. In addition, its reaction generates H<sub>2</sub>O<sub>2</sub>, which needs to be broken down by catalase. A more efficient bacterial enzyme, glycolate dehydrogenase (GDH), can be used to replace glycolate oxidase’s role. GDH uses NAD<sup>+</sup> instead of O<sub>2</sub> as the hydrogen acceptor, which prevents production of toxic H<sub>2</sub>O<sub>2</sub> and preserves reducing power.</p>
<p>According to the scheme described above, <span class="citation">Kebeish et al. (2007)</span> generated plants transformed with five chloroplast-targeted bacterial genes encoding glycolate dehydrogenase, glyoxylate carboligase and tartronic semialdehyde reductase (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 1). In these plants, some glycolate is successfully converted directly to glycerate in chloroplasts, and reduces, but not eliminate, flux of photorespiratory metabolites through peroxisomes and mitochondria. The transgenic plants grew faster, produced more shoot and root biomass, and <a href="https://www.nature.com/articles/nbt1299/tables/1">the content of soluble sugars (fructose, glucose and sucrose) increases significantly</a>.</p>
<div class="figure"><span id="fig:pr-bypass-3mech"></span>
<img src="../img/pr-bypass-3mech.png" alt="Three photorespiratory bypasses. The loop represents the native pathway." width="100%" />
<p class="caption">
Figure 2.1: Three photorespiratory bypasses. The loop represents the native pathway.
</p>
</div>
<p><span class="citation">Carvalho et al. (n.d.)</span> designed a similar photorespiratory bypass (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 3). It also uses tartronate semialdehyde (TSA) as the intermediate and thus prevents NH<sub>3</sub> production. However, the CO<sub>2</sub>-releasing TSA synthesis reaction occurs in the peroxisome rather than in the chloroplast and therefore this mechanism does not have the additional benefit of reducing CO<sub>2</sub> loss. In the peroxisome, TSA is converted to hydroxypyruvate by hydroxypyruvate isomerase (HYI), and the rest is identical to the native pathway. Unfortunately, all plants fail to express HYI, and for those with GCL expressed, chlorosis and stunted growth are exhibited, similar to photorespiration mutants. (I think one statement regarding this piece of research at the end of page 808 of the review written by <span class="citation">Weber and Bar-Even (2019)</span> is not appropriate, as HYI was not even expressed)</p>
<p><span class="citation">Maier et al. (2012)</span> used a completely different approach. Instead of taking shortcuts within the native photorespiratory pathway with 3-PGA being the final product, they introduced glycolate oxidase, catalase and malate synthase into the chloroplast, which, together with pyruvate dehydrogenase and malic enzyme which are natively present in the chloroplast, completely oxidise glycolate into two CO<sub>2</sub> (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 2), while producing NADPH and NADH. Although this pathway requires more energy than the native pathway to restore the ‘<em>status quo ante</em>’ according to the calculation (see <span class="citation">Peterhansel, Blume, and Offermann (n.d.)</span>, Table 2), and is shown in computational modelling to result in a 31% <em>decrease</em> in photosynthetic efficiency compared to WT <span class="citation">(Xin et al. 2015)</span>, experimental studies on <em>Arabidopsiis thaliana</em> surprisingly revealed an increase in biomass in these transgenic plants. A modified version of this approach carried out later on tobacco plants by <span class="citation">South et al. (2019)</span> were shown to be even more successful than Kebeish’s approach which was calculated to be the most energy-efficient. The success of Maier’s method may be explained by thhe fact that, in hot climates (where sunlight is intense), ATP and NADPH supply from light reactions is often not limiting, so an improved energy balance in photorespiration does not necessarily improve the rate of carbon fixation. Instead, the benefit brought about by the rapid CO<sub>2</sub> generation in this bypass is more important. In addition, it is argued that the ‘wasteful’ dissipation of reducing power by glycolate oxidase may actually be beneficial in avoiding production of reactive oxygen species (ROS) in high light intensity, when the output of light reactions is in large excess. This also explains why higher plants prefer glycolate oxidase rather than glycolate dehydrogenase.</p>
</div>
<div id="inhibition-of-the-glycolate-transporter" class="section level2">
<h2><span class="header-section-number">2.2</span> Inhibition of the Glycolate Transporter</h2>
<p>In the two successful photorespiratory bypasses described above, glycolate metabolism inside chloroplast competes with its export via the glycolate exporter PLGG. Therefore, repression of this transporter is expected to increase the flux into the bypasses and thus further increase the efficiency. <span class="citation">South et al. (2019)</span> studied the effect of RNAi inhibition of <em>PLGG1</em> in these two methods as well as a ‘combined’ scheme of the two. The constructs and mechanisms of action are shown in Figure <a href="#fig:pr-bypasses-south-2019">2.2</a>. Both AP1 and AP3 (but not AP2) showed significant increase in dry weight biomass, by 13% and 18%, respectively. When the <em>PLGG1</em> RNAi module is added, the benefit of AP1 is lost, while that of AP3 increases to 24% (<span class="citation">Weber and Bar-Even (2019)</span> implied, on page 808 when citing this work, that inhibition of <em>PLGG</em> always increases efficiency, which is not correct).</p>
<div class="figure"><span id="fig:pr-bypasses-south-2019"></span>
<img src="../img/pr-bypasses-south-2019.png" alt="South's (2019) three photorespiratory bypass constructs. AP1: five bacterial genes encoding glycolate dehydrogenase (GDH), glyoxylate carboligase and tartronic semialdehyde reductase (same as Kebeish (2007)); AP2: glycolate oxidase (GOX), malate synthase and catalase (CAT) (same as Maier (2012)); AP3: same as AP2, except that GOX+CAT is replaced by GDH." width="100%" />
<p class="caption">
Figure 2.2: South’s (2019) three photorespiratory bypass constructs. AP1: five bacterial genes encoding glycolate dehydrogenase (GDH), glyoxylate carboligase and tartronic semialdehyde reductase (same as Kebeish (2007)); AP2: glycolate oxidase (GOX), malate synthase and catalase (CAT) (same as Maier (2012)); AP3: same as AP2, except that GOX+CAT is replaced by GDH.
</p>
</div>
<p>Elimination of the AP1 enhancements by the <em>PLGG1</em> RNAi module implies that this introduced pathway ‘may not have had sufficient kinetic capacity to handle the full glycolate flux under high rates of Rubisco oxygenation’, according to <span class="citation">South et al. (2019)</span>. This is in accordance with the aforementioned statement that the efficiency of photorespiratory bypasses in accelerating carbon fixation does not depend on energy economy (conservation of ATP and NADPH), but instead on the rate of conversion of 2-PG into TCA cycle intermediates (CO<sub>2</sub> and/or 3-PGA). That is to say, the real culprit for photorespiration-induced reduction in carbon fixation activity is not loss of energy in the form of ATP and NADPH, but is likely to be the inefficiency of removing 2-PG, as well as the low CO<sub>2</sub>:O<sub>2</sub> ratio. Thus, in South’s AP3 construct where 2-PG is efficiently cleared up and CO<sub>2</sub> is produced in a large amount, the most prominent increase in carbon fixation and biomass is observed.</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">3</span> Conclusion</h1>
<p>Photorespiration has long been a target for improving photosynthesis and crop yield. Several photorespiratory bypasses, which provide alternative, more efficient pathways to complement or replace the native pathway, have proved successful for this goal in model plants. Much is yet to be known about the mechanisms by which these bypasses accelerates photosynthesis, but recent experiments suggests that, in contrast to the traditional view that photorespiration is energy-wasting and should be avoided, increasing the flux through the photorespiration pathway might instead speed up carbon fixation. Also, conclusion obtained from computational modelling should be viewed with care, at least in photorespiratory studies, as the seemingly sensible assumptions they make often results in significant deviations from the experimental results.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Carvalho-2011">
<p>Carvalho, Josirley de F C, Pippa J Madgwick, Stephen J Powers, Alfred J Keys, Peter J Lea, and Martin A J Parry. n.d. “An Engineered Pathway for Glyoxylate Metabolism in Tobacco Plants Aimed to Avoid the Release of Ammonia in Photorespiration.” <em>BMC Biotechnol</em> 11. Embrapa Soybean, Londrina, Paraná, Brazil, Rodovia Carlos Strass, Distrito da Warta, Londrina PR, Brasil.: 111. <a href="https://doi.org/10.1186/1472-6750-11-111">https://doi.org/10.1186/1472-6750-11-111</a>.</p>
</div>
<div id="ref-Kebeish-2007">
<p>Kebeish, Rashad, Markus Niessen, Krishnaveni Thiruveedhi, Rafijul Bari, Heinz-Josef Hirsch, Ruben Rosenkranz, Norma Stäbler, Barbara Schönfeld, Fritz Kreuzaler, and Christoph Peterhänsel. 2007. “Chloroplastic Photorespiratory Bypass Increases Photosynthesis and Biomass Production in Arabidopsis Thaliana.” <em>Nat Biotechnol</em> 25 (5). RWTH Aachen, Institute of Biology I, Worringer Weg 1, 52056 Aachen, Germany.: 593–99. <a href="https://doi.org/10.1038/nbt1299">https://doi.org/10.1038/nbt1299</a>.</p>
</div>
<div id="ref-Maier-2012">
<p>Maier, Alexandra, Holger Fahnenstich, Susanne Von Caemmerer, Martin Engqvist, Andreas Weber, Ulf-Ingo Flügge, and Veronica Maurino. 2012. “Transgenic Introduction of a Glycolate Oxidative Cycle into a. Thaliana Chloroplasts Leads to Growth Improvement.” <em>Frontiers in Plant Science</em> 3: 38. <a href="https://doi.org/10.3389/fpls.2012.00038">https://doi.org/10.3389/fpls.2012.00038</a>.</p>
</div>
<div id="ref-Peterhansel-2013">
<p>Peterhansel, Christoph, Christian Blume, and Sascha Offermann. n.d. “Photorespiratory Bypasses: How Can They Work?” <em>J Exp Bot</em> 64 (3). Leibniz University Hannover, Institute of Botany, Herrenhaeuser Straße 2, 30419 Hannover, Germany. cp@botanik.uni-hannover.de: 709–15. <a href="https://doi.org/10.1093/jxb/ers247">https://doi.org/10.1093/jxb/ers247</a>.</p>
</div>
<div id="ref-South-2019">
<p>South, Paul F, Amanda P Cavanagh, Helen W Liu, and Donald R Ort. 2019. “Synthetic Glycolate Metabolism Pathways Stimulate Crop Growth and Productivity in the Field.” <em>Science</em> 363 (6422). Global Change; Photosynthesis Research Unit, United States Department of Agriculture-Agricultural Research Service, Urbana, IL 61801, USA.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Department of Crop Sciences, University of Illinois, Urbana, IL 61801, USA.; Global Change; Photosynthesis Research Unit, United States Department of Agriculture-Agricultural Research Service, Urbana, IL 61801, USA. d-ort@illinois.edu.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Department of Crop Sciences, University of Illinois, Urbana, IL 61801, USA.; Department of Plant Biology, University of Illinois, Urbana, IL 61801, USA. <a href="https://doi.org/10.1126/science.aat9077">https://doi.org/10.1126/science.aat9077</a>.</p>
</div>
<div id="ref-Weber-2019">
<p>Weber, Andreas P.M., and Arren Bar-Even. 2019. “Update: Improving the Efficiency of Photosynthetic Carbon Reactions.” <em>Plant Physiology</em> 179 (3). American Society of Plant Biologists: 803–12. <a href="https://doi.org/10.1104/pp.18.01521">https://doi.org/10.1104/pp.18.01521</a>.</p>
</div>
<div id="ref-Xin-2015">
<p>Xin, Chang-Peng, Danny Tholen, Vincent Devloo, and Xin-Guang Zhu. 2015. “The Benefits of Photorespiratory Bypasses: How Can They Work?” <em>Plant Physiology</em> 167 (2). American Society of Plant Biologists: 574–85. <a href="https://doi.org/10.1104/pp.114.248013">https://doi.org/10.1104/pp.114.248013</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/computational-genomics-practical-tt-2020.html" class="archive-item-link">
                Computational Genomics Practical (TT 2020) 
              </a>
              <span class="archive-item-date">
                2020-06-08
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#quiz-5"><span class="toc-section-number">1</span> Quiz 5</a>
<ul>
<li><a href="#heatmaps-using-ggplot2-in-r"><span class="toc-section-number">1.1</span> Heatmaps (using <strong>ggplot2</strong> in R)</a></li>
<li><a href="#heatmaps-in-python-with-seaborn"><span class="toc-section-number">1.2</span> Heatmaps in Python (with <strong>seaborn</strong>)</a></li>
<li><a href="#average-gene-profile"><span class="toc-section-number">1.3</span> Average Gene Profile</a></li>
</ul></li>
<li><a href="#quiz-8"><span class="toc-section-number">2</span> Quiz 8</a>
<ul>
<li><a href="#matlab"><span class="toc-section-number">2.1</span> Matlab</a></li>
<li><a href="#using-r"><span class="toc-section-number">2.2</span> Using R</a></li>
</ul></li>
</ul>
</div>

<div id="quiz-5" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Quiz 5</h1>
<div id="heatmaps-using-ggplot2-in-r" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Heatmaps (using <strong>ggplot2</strong> in R)</h2>
<p>Loading Packages</p>
<pre class="r"><code>library(R.matlab)</code></pre>
<pre><code>## R.matlab v3.6.2 (2018-09-26) successfully loaded. See ?R.matlab for help.</code></pre>
<pre><code>## 
## Attaching package: &#39;R.matlab&#39;</code></pre>
<pre><code>## The following objects are masked from &#39;package:base&#39;:
## 
##     getOption, isOpen</code></pre>
<pre class="r"><code>library(reshape2)
library(tidyverse)</code></pre>
<pre><code>## ── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>## ✓ ggplot2 3.3.2     ✓ purrr   0.3.4
## ✓ tibble  3.0.4     ✓ dplyr   1.0.2
## ✓ tidyr   1.1.2     ✓ stringr 1.4.0
## ✓ readr   1.4.0     ✓ forcats 0.5.0</code></pre>
<pre><code>## ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
## x dplyr::filter() masks stats::filter()
## x dplyr::lag()    masks stats::lag()</code></pre>
<p>Loading Data</p>
<pre class="r"><code>setwd(&#39;/Users/tianyishi/Documents/GitHub/ox/content/lab/src/Y2T3W7-genomics&#39;)
H3K4me3 &lt;- readMat(&#39;H3K4me3_ChIP_seq_glucose_gene_levels.mat&#39;)$H3K4me3.ChIP.seq.glucose.gene.levels
MNase &lt;- readMat(&#39;MNase_seq_glucose_gene_levels.mat&#39;)$MNase.seq.glucose.gene.levels
NET &lt;- readMat(&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;)$NET.seq.glucose.gene.levels.sense.strand
NET[NET &lt; -0.5 | NET &gt; 5] = NA</code></pre>
<pre class="r"><code>plot_heatmap &lt;- function(mat, title) {
  return(
    mat %&gt;% 
    melt() %&gt;% 
    ggplot(aes(Var2, Var1, fill=value))+
      geom_tile()+
      scale_x_continuous(expand = c(0,0), breaks = c(1, 501, 1001, 1501), labels = c(&#39;-500&#39;, &#39;TSS&#39;, &#39;+500&#39;, &#39;+1000&#39;))+
      scale_y_continuous(expand = c(0,0))+
      scale_fill_gradient(low=&quot;blue&quot;, high = &quot;yellow&quot;)+
      labs(title=title,
           x=&#39;Position relative to TSS (bp)&#39;,
           y=&#39;Gene number&#39;)
  )
}</code></pre>
<pre class="r"><code>plot_heatmap(H3K4me3, &#39;H3K4me3 ChIP-seq Glucose&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<pre class="r"><code>plot_heatmap(MNase, &quot;MNase&quot;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-2.png" width="672" /></p>
<pre class="r"><code>plot_heatmap(NET, &#39;NET&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-3.png" width="672" /></p>
</div>
<div id="heatmaps-in-python-with-seaborn" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Heatmaps in Python (with <strong>seaborn</strong>)</h2>
<p>Plotting above heatmaps using <strong>ggplot2</strong> in R is very slow. Using <strong>seaborn</strong> in Python is faster (plots not displayed):</p>
<pre class="python"><code>from scipy.io import loadmat
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

sns.set()

MNase = np.array(loadmat(&quot;MNase_seq_glucose_gene_levels.mat&quot;)[&quot;MNase_seq_glucose_gene_levels&quot;])
H3K4me3 = np.array(loadmat(&quot;H3K4me3_ChIP_seq_glucose_gene_levels.mat&quot;)[&quot;H3K4me3_ChIP_seq_glucose_gene_levels&quot;])
NET = readMat(&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;)[&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;]

def plot_heatmap(mat, title):
    p = sns.heatmap(mat)
    p.set(
        title=title,
        xlabel=&#39;Position relative to TSS (bp)&#39;, ylabel=&#39;Gene number&#39;,
        xticks=[0, 500, 1000, 1500],
        xticklabels=[&quot;-500&quot;, &quot;TSS&quot;, &quot;+500&quot;, &quot;+1000&quot;],
        yticks=[0, 999, 1999, 2999, 3999, 4999],
        yticklabels=[1, 1000, 2000, 3000, 4000, 5000],
    )
    return p

plot_heatmap(MNase, &quot;MNase seq Glucose&quot;)
plt.show()
plot_heatmap(H3K4me3, &#39;H3K4me3 ChIP-seq Glucose&#39;)
plt.show()
plot_heatmap(NET, &#39;NET-seq Glucose sense strand&#39;)
plt.show()</code></pre>
</div>
<div id="average-gene-profile" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Average Gene Profile</h2>
<pre class="r"><code>xylabs &lt;- labs(x=&#39;Position relative to TSS (bp)&#39;, y=&#39;average NET-seq level&#39;)
xaxis &lt;- scale_x_continuous(breaks = c(0, 500, 1000, 1500), labels = c(&#39;-500&#39;, &#39;TSS&#39;, &#39;+500&#39;, &#39;+1000&#39;))

# Average gene profile MNase-seq Glucose
tibble(x=0:1500, y=apply(MNase, 2, mean)) %&gt;% 
  ggplot(aes(x, y))+
  geom_line()+
  xaxis+
  xylabs+
  labs(title=&#39;Average gene profile MNase-seq Glucose&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<pre class="r"><code># NET-seq data highest and lowest H3K4me3 comparison
tibble() %&gt;% 
  bind_rows(tibble(x=0:1500, y=apply(NET[1:501,], 2, function(x){mean(x, na.rm = TRUE)})) %&gt;% add_column(`H3K4me3 level`=&#39;high&#39;)) %&gt;% 
  bind_rows(tibble(x=0:1500, y=apply(NET[4501:5000,], 2, function(x){mean(x, na.rm = TRUE)})) %&gt;% add_column(`H3K4me3 level`=&#39;low&#39;)) %&gt;%
  ggplot(aes(x, y, color=`H3K4me3 level`))+
    geom_line()+
    xylabs+xaxis+
    labs(title=&#39;NET-seq data highest and lowest H3K4me3 comparison&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-6-2.png" width="672" /></p>
</div>
</div>
<div id="quiz-8" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Quiz 8</h1>
<div id="matlab" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Matlab</h2>
<pre class="matlab"><code>pos=yeast_gene_positions(:,[1,2,5])
% three colomns correspond to chromosome number, direction and TSS
avgs = []
i = 1
for p = pos.&#39;
  chr = NET_seq_data{p(1) + 16 * (1 - p(2))};
  if p(2)==1
    seq = chr(p(3):(p(3) + 299), :);
  else
    seq = chr((p(3) - 299):p(3), :);
  end
  avgs(i,:) = mean(seq);
  i = i + 1;
end
corr(avgs,yeast_gene_measurements(:,1))</code></pre>
</div>
<div id="using-r" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Using R</h2>
<pre class="r"><code>setwd(&#39;/Users/tianyishi/Documents/GitHub/ox/content/lab/src/Y2T3W7-genomics&#39;)
pos &lt;- readMat(&#39;yeast_gene_positions.mat&#39;)$yeast.gene.positions[,c(1,2,5)]
net &lt;- lapply(readMat(&#39;NET_seq_genome_wide_data.mat&#39;)$NET.seq.data, as_vector)
clc &lt;- apply(pos, 1, function(x){
  mean(net[[x[1] + (1-x[2])*16]][x[3]:(x[3]+ifelse(x[2]==0, -1, 1)*299)])
})
exp &lt;- readMat(&#39;yeast_gene_measurements.mat&#39;)$yeast.gene.measurements[,1]

plot(clc, exp)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-8-1.png" width="672" /></p>
<pre class="r"><code>cor(clc, exp)</code></pre>
<pre><code>## [1] 0.8281284</code></pre>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/genetic-manipulation-of-the-calvin-cycle.html" class="archive-item-link">
                Genetic Manipulation of the Calvin Cycle 
              </a>
              <span class="archive-item-date">
                2020-06-02
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a><ul>
<li><a href="#rubisco-does-not-exert-much-control-on-the-flux-of-the-calvin-cycle-unless-in-special-conditions"><span class="toc-section-number">1.1</span> Rubisco Does not Exert Much Control on the Flux of the Calvin Cycle Unless in Special Conditions</a></li>
</ul></li>
<li><a href="#other-enzymes-that-catalyse-irreversible-reactions"><span class="toc-section-number">2</span> Other Enzymes that Catalyse Irreversible Reactions</a></li>
<li><a href="#plastid-aldolase-as-an-non-regulated-enzyme-exerts-a-greater-control-than-rubisco-on-the-rate-of-photosynthesis"><span class="toc-section-number">3</span> Plastid Aldolase as an ‘Non-Regulated’ Enzyme Exerts a Greater Control than Rubisco on the Rate of Photosynthesis</a></li>
<li><a href="#transketolase-directly-controls-not-only-calvin-cycle-but-also-other-pathways"><span class="toc-section-number">4</span> Transketolase Directly Controls not Only Calvin Cycle but Also Other Pathways</a></li>
<li><a href="#conclusion"><span class="toc-section-number">5</span> Conclusion</a></li>
</ul>
</div>

<p><strong>To what extent has genetic manipulation of the Calvin cycle forced the reappraisal of our understanding of the control of metabolic pathways in plants? What do studies of these transgenic plants reveal about the integration of metabolism?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Genetic manipulation, particularly graded knock-down of enzymes in the Calvin cycle, has been exploited in the quest for understanding the control of the rate of carbon fixation step of photosynthesis. While initially it was thought that the flux of carbon fixation is limited by a few ‘regulated’ enzymes, with Rubisco being the most promising candidate, many unexpected experimental results revealed that, in reality, the flux of the Calvin cycle is influenced by multifarious factors, some of which are even due to other carbohydrate-unrelated pathways.</p>
<div id="rubisco-does-not-exert-much-control-on-the-flux-of-the-calvin-cycle-unless-in-special-conditions" class="section level2">
<h2><span class="header-section-number">1.1</span> Rubisco Does not Exert Much Control on the Flux of the Calvin Cycle Unless in Special Conditions</h2>
<p>Rubisco catalyses the carbon fixation step in the Calvin cycle and was initially widely thought to act as a control point on the rate of photosynthesis. Indeed, the enzymatic activity of Rubisco is modulated by a number of ways, such as activation by resersible carbamylation by CO<sub>2</sub> and by Rubisco activase, and inhibition by a number of sugars resulting from Rubisco’s side reactions.</p>
<p><span class="citation">1</span> produced a series of tobacco plants (<em>Nicotiana tabacum</em>) that exhibit a range of reduced amounts of Rubisco by Agrobacterium-mediated transformation of plants with antisense mRNA to the gene for the small subunit of Rubisco (<em>rbcS</em>). Such ‘antisense’ plants were used by<span class="citation"><sup>2</sup></span> to examine the control exerted by Rubisco on the rate of photosynthesis. They found that in optimum enviornmental conditions the amount of Rubisco in a leaf could be reduced by more than one-third before any significant effect on the rate of photosynthesis, with <span class="math inline">\(C^J\)</span> = 0.05-0.15 (very small). It was shown that reduction of enzyme amount is compensated for by an increase in Rubisco activation (from about 55% to almost 100%) due to 1) an increase of substrates (ribulose 1,5-bisphosphate and CO<sub>2</sub>) and decrease of products (3-phosphoglycerate), and 2) an increase of ATP/ADP ratio in the chloroplast stroma. Also, Rubisco is produced in large excess in WT plants, also explaining the lack of impact of its knock-down.</p>
<p>However, when plants were grown in low light and are then suddenly exposed to high light intensity, there was a near-proportional relation between the amount of Rubisco and the rate of photosynthesis<span class="citation"><sup>3</sup></span>. A similar result can be obtained with low CO<sub>2</sub> concentration. These experiments show that the contribution of Rubisco to the control of photosynthesis depends on both current and past conditions (CO<sub>2</sub> concentration and light intensity).</p>
</div>
</div>
<div id="other-enzymes-that-catalyse-irreversible-reactions" class="section level1">
<h1><span class="header-section-number">2</span> Other Enzymes that Catalyse Irreversible Reactions</h1>
<p>Apart from Rubisco, three other enzymes in the Calvin cycle catalyse irreversible reactions: (stromal) fructose 1,6-bisphosphatase (FBPase), sedoheptulose 1,7-bisphosphatase (SBPase) and phosphoribulokinase (PRK). These three enzymes are subjected to a same set of ‘fine’ regulatory mechanisms, all of which are exploited by the light reaction to activate the Calvin cycle:</p>
<ul>
<li><strong>pH.</strong> These enzymes have a relatively sharp pH optimium at around 8.2.</li>
<li><strong>Mg<sup>2+</sup></strong> stimulation.</li>
<li><strong>Redox state.</strong> These enzymes are inactive when disulfide bridges form between certain cysteine residues, and are activated by (reduced) thioredoxin, which reduces these disulfide bonds.</li>
</ul>
<p>Analysis of plant strains with each of these three enzymes knocked down reavealed that transformants with decreased SBPase activity show a significant but still non-proportional inhibition of photosynthesis<span class="citation"><sup>4</sup></span> in ambient conditions, while the effects of FBPase and PRK knock-down are marginal. Therefore, enzymes with similar regulatory properties does not indicate they have similar flux-control capabilities in a metabolic pathway.</p>
</div>
<div id="plastid-aldolase-as-an-non-regulated-enzyme-exerts-a-greater-control-than-rubisco-on-the-rate-of-photosynthesis" class="section level1">
<h1><span class="header-section-number">3</span> Plastid Aldolase as an ‘Non-Regulated’ Enzyme Exerts a Greater Control than Rubisco on the Rate of Photosynthesis</h1>
<p>In the Calvin cycle, aldolase carries out the synthesis of fructose 1,6-bisphohsphate (FBP) and sedoheptulose 1,7-bisphohsphate (SBP), both of which are close to equilibrium. Unlike the four enzymes described previously, aldolase’s activity is devoid of any regulatory properties and thus its rate is solely determined by the concentrations of substrates and products.</p>
<p>However, similar knock-down experiments targeting aldolase showed unexpected results, in that reduction in aldolase was found to significantly reduce the rate of photosynthesis<span class="citation"><sup>5</sup></span>. A 30% decrease of aldolase activity in potato transformants led to a small (5–10%) inhibition of ambient photosynthesis, and reduction below 30% of the wild-type activity led to a severe inhibition. The experiment is repeated with different light intensities and CO<sub>2</sub> concentrations, and it was found that the inhibition is smallest in low light and highest in high light and elevated CO<sub>2</sub> (with <span class="math inline">\(C^J\)</span> of about 0.18 and 0.56, respectively).</p>
<p>Decreased expression of aldolase inhibits photosynthesis for different reasons in low and high light. In low light, decreased expression of aldolase led to an accumulation of its substrate, triose phosphates and a depletion of of its product, which in turn causes the depletion of RuBP. In high light (plus elevated CO<sub>2</sub>), however, the triose phosphates remained very low, RuBP remained high, and PGA was higher in the transformants than in wild-type plants. This is because high irradiation inhibits starch synthesis and hence accumulation of phosphorylated intermediates, which leads to P<sub>i</sub>-limitation, which in turn restricts ATP regeneration by the light reaction. The decreased ATP concentration thhen limits conversion of 3PGA to GAP, causing 3PGA to accumulate, which in turn results in product inibition of Rubisco. Of course, the two mechanisms are not mutually exclusive, and the relative importance of each vary gradually according to the light intensity.</p>
<p>It is important to note here that, to fully rationalise the difference in the effectiveness of inhibition of the Calvin cycle in different conditions, the effect of the conditions on other pathways (starch synthesis) in the metabolic network need also be considered.</p>
</div>
<div id="transketolase-directly-controls-not-only-calvin-cycle-but-also-other-pathways" class="section level1">
<h1><span class="header-section-number">4</span> Transketolase Directly Controls not Only Calvin Cycle but Also Other Pathways</h1>
<p>Perturbation of the enzymes involved in the central carbohydrate metabolism pathways can directly influence the activity of other pathways. Transketolase is such an example.</p>
<p>In the Calvin cycle, transketolase catalyses the reactions 1) <span class="math inline">\(F6P + GAP \rightarrow E4P + X5P\)</span> and 2) <span class="math inline">\(S7P + GAP \rightarrow X5P + R5P\)</span>, both of which are reactions close to equilibrium. Tobacco tranformant with decreased expression of plastid transketolase were also produced. Like aldolase, this ‘non-regulated’ enzyme also has a relatively high flux-control coefficient, especially in saturating light and CO<sub>2</sub>. As expected, loss of transketolase activity results in an increase in the amounts of its substrate (F6P) and decrease of its products, ultimately leading to a decrease of RuBP concentration. However, it was also observed that there was a significant decrease in the levels of aromatic amino acids, intermediates of the phenylpropanoid pathway, and secondary products such as chlorogenic acid and lignin. A plausible explanation for these observations is that the flux into the shikimic acid pathway (which synthesises aromatic pathways) is limited by the decreasing erythrose 4-phosphate (one product of transketolase) concentration.</p>
<p>The multiple consequences of reducing transketolase activity highlight the extent of integration within thhe central metabolic pathways and the potential difficulties in attepting to modify flux through a specific section of the metabolic network.</p>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">5</span> Conclusion</h1>
<p>Genetic manipulation (mainly knock-outs) of some enzymes of the Calvin cycle qualitatively revealed some of the logic behind the regulation of (plant) metabolism in general: 1) loss of regulated enzymes such as Rubisco are compensated by increase in activity, 2) regulated enzymes are often produced in excess so that minor reduction in their amount does not affect the flux at all, 3) non-regulated enzymes such as aldolase are not produced in excess, and their loss restricts the flux more effectively, 4) the extent and the mechanisms of influence on Calvin cycle flux by reduction of an enzyme is affected by enviornmental conditions and the developmental history of the plant, 5) perturbation of a single reaction/enzyme can influence other, even distantly-related metabolic pathways, due to the plethora of interconnected components, 6) the flux through the Calvin cycle is determined by multiple enzymes.</p>
<p>Lots of effort has been made in modelling plant metabolism pathways such as the Calvin cycle (in order to ‘improve’ them), which proves extremely difficult. The Calvin cycle does not exist in isolation, so natually its flux is also influenced by other pathways in the complicated plant metabolism network, through shared intermediates/enzymes/regulatory molcules. Therefore, the precise prediction of its activity, in principle, also requires modelling of other, even distantly related pathways. However, it is impossible to presisely predict the behaviour of the entire metabolic network without knowing all the details (kinetic properties under different conditions, concentrations of enzymes and substrates, etc.) of every reaction in the network. Even if precise kinetic data of enzymes and concentrations of all substances can be obtained, further challenges will be encountered when considering <em>in vivo</em> modelling: unlike the test tube where all substances are relatively evenly distributed and there is only one single isolated compartment, the plant cells are extensively compartmentalised (many metabolic pathways take place in more than one compartments), and substances (substrate or enzymes) may not be evenly distributed and instead localise to specific regions of each compartment, thus altering the effective concentration. In addition, the effects of the internal and external enviornment (e.g. signalling molecules, light intensity) on gene expression of enzymes are also difficult to simulate.</p>
<p>References</p>
<div id="refs" class="references">
<div id="ref-Rodermel-1988">
<p>1. Rodermel, S. R., Abbott, M. S. &amp; Bogorad, L. Nuclear-organelle interactions: Nuclear antisense gene inhibits ribulose bisphosphate carboxylase enzyme levels in transformed tobacco plants. <em>Cell</em> <strong>55</strong>, 673–681.</p>
</div>
<div id="ref-Quick-1991">
<p>2. Quick, W. P. <em>et al.</em> Decreased ribulose-1,5-bisphosphate carboxylase-oxygenase in transgenic tobacco transformed with &quot;antisense&quot; rbcS : I. Impact on photosynthesis in ambient growth conditions. <em>Planta</em> <strong>183</strong>, 542–554.</p>
</div>
<div id="ref-Quick-1992">
<p>3. Quick, W. P. <em>et al.</em> Decreased ribulose-1,5-bisphosphate carboxylase-oxygenase in transgenic tobacco transformed with &quot;antisense&quot; rbcS : IV. Impact on photosynthesis in conditions of altered nitrogen supply. <em>Planta</em> <strong>188</strong>, 522–531.</p>
</div>
<div id="ref-Harrison-1997">
<p>4. Harrison, E. P., Willingham, N. M., Lloyd, J. C. &amp; Raines, C. A. Reduced sedoheptulose-1,7-bisphosphatase levels in transgenic tobacco lead to decreased photosynthetic capacity and altered carbohydrate accumulation. <em>Planta</em> <strong>204</strong>, 27–36 (1997).</p>
</div>
<div id="ref-Haake-1998">
<p>5. Haake, V., Zrenner, R., Sonnewald, U. &amp; Stitt, M. A moderate decrease of plastid aldolase activity inhibits photosynthesis, alters the levels of sugars and starch, and inhibits growth of potato plants. <em>The Plant Journal</em> <strong>14</strong>, 147–157 (1998).</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/coordination-of-ps-i-and-ps-ii.html" class="archive-item-link">
                Coordination of PS I and PS II 
              </a>
              <span class="archive-item-date">
                2020-05-27
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#background"><span class="toc-section-number">1</span> Background</a></li>
<li><a href="#why-is-coordination-between-photosystems-i-and-ii-needed"><span class="toc-section-number">2</span> Why is Coordination between Photosystems I and II Needed</a></li>
<li><a href="#achieve"><span class="toc-section-number">3</span> How is Coordination between PS-I and PS-II Achieved</a><ul>
<li><a href="#the-role-of-the-plastoquinone-pool"><span class="toc-section-number">3.1</span> The Role of the Plastoquinone Pool</a></li>
<li><a href="#redistribution"><span class="toc-section-number">3.2</span> Reversible Redistribution of LHC-II</a></li>
<li><a href="#transcriptional"><span class="toc-section-number">3.3</span> Transcriptional Regulation</a><ul>
<li><a href="#how-does-sig1-phosphorylation-confer-phhotosystem-promoter-specificity"><span class="toc-section-number">3.3.1</span> How does SIG1 phosphorylation confer phhotosystem promoter specificity?</a></li>
</ul></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">4</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>Why do plants co-ordinate the relative activities of Photosystems I and II and how is this co-ordination achieved?</strong></p>
<div id="background" class="section level1">
<h1><span class="header-section-number">1</span> Background</h1>
<p>Photosystems I and II are photochemical complexes that harvest light energy to drive endergonic redox reactions. Their operation in series (PS-II then PS-II), known as the Z scheme, results in generation of oxygen from water via photolysis, ATP production via electron transport chain (ETC) and production of NADPH (reduction of NADP<sup>+</sup>), which is essential for providing reducing power in anabolic reactions such as photosynthesis, by ferredoxin-NADP<sup>+</sup> reductase. PS-I can also operate alone, resulting in cyclic photophosphorylation, generating ATP only.</p>
<p>Each photosystem exists on the thylakiod membrane (PS-II mainly in grana lamellae; PS-I mainly in stroma lamellae) as a multisubunit pigment-protein complex (or a dimer thereof, in the case of PS-II) comprising a reaction centre (RC) core containing a pair of chlorophyll <em>a</em> molecules, which is surrounded by other accessory proteins as well as variable number of antenna complexes ((trimers of) LHC-II/LHC-I; also known as chlorophyll <em>a/b</em> light-harvesting complex ) containing accessory pigments (e.g. chlorophylls and carotenoids) that pass light energy to the reaction centre via resonance energy transfer. The reaction centre in PS-II preferentially absorbs red light at wavelength 680 nm and is thus referred to as P680; the reaction centre in PS-I, known as P700, preferentially absorbs far-red light.</p>
<p>The activities of two types of photosystems is determined both by the external factors, i.e. intensity and wavelength of light, as well as the the intrinsic factors which can be regulated by the plant cells, namely the number of PS-II/PS-I cores and the antenna complexes (and hence the amount of accessory pigments) associated with them. Such regulation can affect the absolute activities of PS-II and PS-I as well as their activities relative to each other.</p>
</div>
<div id="why-is-coordination-between-photosystems-i-and-ii-needed" class="section level1">
<h1><span class="header-section-number">2</span> Why is Coordination between Photosystems I and II Needed</h1>
<p>Because the PS-I and PS-II complexes are not physically coupled (so their abundance can differ) and their light absorption properties differ (PS-I prefers longer wavelength), their activities are not directly related and are often different. The activities of the two photosystems relative to each other can be regulated using the mechanisms described in Section <a href="#achieve">3</a>, but before that, the benefits of this regulation are considered here:</p>
<ol style="list-style-type: decimal">
<li><strong>Stoichiometry adjustment.</strong> PS-II and PS-I activities should ideally be equal, in order to maximise the efficiency of light utilisation for production of NADPH in vectorised electron flow, which would otherwise be limited by the slower photosystem. There is a need for this kind of adjustment when there is a change in light quality. Upon illumination by far-red light that preferentially activates PS-I (PS-I light), PS-I activity is greater than PS-II, so PS-II activity shhould be improved relative to PS-I. Conversely, illumination by PS-II light leads to hyperactive PS-II, and its activity should be reduced relative to PS-II. These two states are referred to state 1 and 2, respectively, and state transition is the process by which the plant adjusts relative activity of PS-II and PS-I in response to a change in state.</li>
<li><strong>Protection against photoinhibition.</strong> If PS-II activity exceeds PS-I activity, the rate of production of reactive oxygen species (ROS) is increased. However, recent studies on stn7 mutants argue against this role. Photodamage of PS-II is unavoidable even under nomal conditions, and it is repaired efficiently in healthy plants.</li>
<li><strong>Meeting increased requirement for ATP.</strong> If PS-I activity exceeds PS-II activity, the excess activity in PS-I results in cyclic electron flow (CEF), generating additional ATP. Theoretically, this may be exploited by the plant to meet increased demand for ATP. However, recent work revealed that CEF is controlled by the redox power and is independent of state transition: state 2 is neither sufficient nor required to promote CEF <span class="citation">(Takahashi et al. 2013)</span>.</li>
</ol>
<p>The discussion below, therefore, focuses on the role of photosystem coordination in stoichiometry adjustment.</p>
</div>
<div id="achieve" class="section level1">
<h1><span class="header-section-number">3</span> How is Coordination between PS-I and PS-II Achieved</h1>
<div id="the-role-of-the-plastoquinone-pool" class="section level2">
<h2><span class="header-section-number">3.1</span> The Role of the Plastoquinone Pool</h2>
<p>The oxidative status of the plastoquinone (PQ) pool reflects the relative activity of PS-II and PS-I, and is involved in the signalling pathway of both short-term and long-term coordination (described in Section <a href="#redistribution">3.2</a> and <a href="#transcriptional">3.3</a>, respectively).</p>
<p>Upon illumination by PS-I light (in state 1), PS-I activity is greater than PS-II, leading to oxidation of the plastoquinone pool (most plastoquinone molecules are in the oxidised form, PQ). Conversely, the plastoquinone pool becomes reduced upon preferential excitation of PS-II (Figure <a href="#fig:plastoquinone">3.1</a>).</p>
<div class="figure"><span id="fig:plastoquinone"></span>
<img src="../img/ps-i-light-ps-2-light-states.png" alt="Preferential activation of either PS-I or PS-II causes a shift in the oxidation state of the plastoquinone (PQ) pool." width="100%" />
<p class="caption">
Figure 3.1: Preferential activation of either PS-I or PS-II causes a shift in the oxidation state of the plastoquinone (PQ) pool.
</p>
</div>
</div>
<div id="redistribution" class="section level2">
<h2><span class="header-section-number">3.2</span> Reversible Redistribution of LHC-II</h2>
<p>LHC-II plays an important role in the coordination between PS-I and PS-II. The structure of one LHC-II protein has been determined by a combination of electron microscopy and X-ray crystallography (reviewed by <span class="citation">Barros and Kühlbrandt (2009)</span>). It contains three membrane-spaning <span class="math inline">\(\alpha\)</span>-helices, and binds about 15 chlorophyll <em>a</em> and <em>b</em> molecules plus a few carotenoids. LHC-II complexes of different subtypes (of which Lhcb1, 2 and 3 are the most common) spontaneously assemble into trimers (can be heterogeneous), which then associate with PS-II and PS-I to increase the efficiency of their corresponding photochemical reaction.</p>
<div class="figure"><span id="fig:lhcii-conf-change"></span>
<img src="../img/lhcii-conf-change.png" alt="Phosphorylation by STN7 causes conformational changes in LHC-II. Top: stroma; bottom: thylakoid lumen." width="100%" />
<p class="caption">
Figure 3.2: Phosphorylation by STN7 causes conformational changes in LHC-II. Top: stroma; bottom: thylakoid lumen.
</p>
</div>
<p>The LHC-II subtypes Lhcb1 and Lhcb2 (but not Lhcb3) can be phosphorylated by a specific protein kinase called STN7 (a.k.a. Stt7) at a threonine residue near the N-terminus <span class="citation">(Mullet 1983)</span>. This happens during transition to state 2, when the STN7 kinase, which is associated with a specific region of the Rieske protein of the cytochrome <em>b<sub>6</sub>f</em> complex (Cyt<em>b<sub>6</sub>f</em>), is activated upon binding of PQH<sub>2</sub> to the Q<sub>o</sub> site of Cyt<em>b<sub>6</sub>f</em>. Phosphorylation on LHC-II causes a conformational change (Figure <a href="#fig:lhcii-conf-change">3.2</a>), reduces its affinity to PS-II and increases its affinity to PS-I. The increase in affinity to PS-I is due to not only the conformational change, but also the tight association between the phosphate group and a specific arginine residue in PsaL, a protein that facilitates docking of LHC-II to PS-I core complex. As a result, LHC-II migrates from PS-II towards PS-I, thus enhancing efficiency of PS-I relative to PS-II (Figure <a href="#fig:stn7-tap38-pq-lhcii">3.3</a>). This is supported by the findings that plants without either STN7 <span class="citation">(Bellafiore et al. 2005)</span> or Lhcb1+Lhcb2 <span class="citation">(Andersson et al. 2003)</span> do not undergo state transitions. It was later shown by <span class="citation">Longoni et al. (2015)</span> that only Lhcb2 phosphorylation is relevant to state transition (Lhcb1 phosphorylation causes its exclusion from the complex). The N-terminal region of phosphorylated Lhcb2 interacts with PsaL, PsaH and PsaO proteins, and PsaO bridges PsaA (PSI reaction centre) and Lhcb2 through contacts within the membrane and on the stromal and luminal surface. PsaO is associated with two chlorophyll molecules which facilitates resonance energy transfer from LHC-II to PsaA.</p>
<p>The phosphorylation can be reversed by a specific kinase called TAP38 (a.k.a PPH1). When PS-I is over-excited relative to PS-II, STN7 is inactive, and TAP38 dephosphorylates LHC-II and causes them to return to PS-II (Figure <a href="#fig:stn7-tap38-pq-lhcii">3.3</a>).</p>
<div class="figure"><span id="fig:stn7-tap38-pq-lhcii"></span>
<img src="../img/stn7-tap38-pq-lhcii.png" alt="Oxidative status of plastoquinone determines the phosphorylation state of LHC-II, which in turn regulates relative activity of PS-I and PS-II." width="100%" />
<p class="caption">
Figure 3.3: Oxidative status of plastoquinone determines the phosphorylation state of LHC-II, which in turn regulates relative activity of PS-I and PS-II.
</p>
</div>
<!-- under High light conditions by response to FdH~2~) -->
</div>
<div id="transcriptional" class="section level2">
<h2><span class="header-section-number">3.3</span> Transcriptional Regulation</h2>
<p>Regulation of relative activity of PS-I and PS-II can also be achieved by controlling the amount of transcription of PS-I genes (e.g. <em>psaA</em>) and PS-II genes (<em>psbA</em>).</p>
<p><span class="citation">Puthiyaveetil et al. (2008)</span> described a sensor kinase called chloroplast sensor kinase (CSK) that couples oxidation state of PQ to transcriptional control of psbA. As shown in Figure <a href="#fig:csk">3.4</a>, PS-I light causes the PQ pool to be oxidised, and then CSK, which is activated by PQ (the PQ analogue, DBMIB (2,5-dibromo-3-methyl-5-isopropyl-<em>p</em>-benzoquinone) binds to CSK similarly), autophosphorylates itself, and then phosphorylates sigma factor 1 (SIG1) subunit of PEP (RNA polymerase). This results in specific repression of PS-I genes (<em>psaA</em>), thus decreasing the stoichiometry of PS-I to PS-I. Conversely, CSK is inactive with a reduced PQ pool under PS-II light, and SIG1 phosphatase removes the repression of PS-I gene transcription, thus increasing the stoichiometry of PS-I to PS-II.</p>
<div class="figure"><span id="fig:csk"></span>
<img src="../img/psi-ii-csk-tx.png" alt="Transcriptional control of PS-I genes mediated by PQ, CSK, SIG1 and PEP." width="100%" />
<p class="caption">
Figure 3.4: Transcriptional control of PS-I genes mediated by PQ, CSK, SIG1 and PEP.
</p>
</div>
<div id="how-does-sig1-phosphorylation-confer-phhotosystem-promoter-specificity" class="section level3">
<h3><span class="header-section-number">3.3.1</span> How does SIG1 phosphorylation confer phhotosystem promoter specificity?</h3>
<p>Bacterial sigma factors contain the region 1.1 at the extreme N-terminus. Region 1.1 is a poorly-conserved, acidic region, which is associated with the positively charged catalytic cavity of RNA polymerase in the closed (non-transcribing) conformation. To begin transcription, (negatively-charged) DNA must replace region 1.1, and the efficiency of this replacement depends on how well the -10 and -35 elements of the promoter match the consensus sequence.</p>
<p>Chloroplast SIG1 also has an N-terminal un-conserved region (UCR), but it contains both acidic and basic patches. It is proposed that, in PS-II light, the basic patch and the acidic patch of the un-phosphorylated SIG1 form a complex through electrostatic interactions, thus preventing the association of the acidic patch with the basic catalytic cavity.
In PS-I light, phosphorylation with the basic patch disfavours its complex formation with the acidic patches, thus allowing the acidic patch to associate with the catalytic cavity, where it acts as the gatekeeper and discriminates between strong and weak promoters.</p>
<p>By comparing the -35 and -10 regions of the promoters of <em>psaA</em> and <em>psbA</em> to the consensus sequence, it can be deduced that <em>psbA</em> (PS-II gene) has a stronger promoter and is therefore preferentially transcribed over PS-I under PS-I light, when SIG1 is phosphorylated and the discrimination mechanism is functioning.</p>
</div>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">4</span> Conclusion</h1>
<p>Plants need to coordinate the efficiency of PS-II and PS-I in order to maximise the utilisation of light energy in linear electron flow. Upon change in light quality, activities of PS-II and PS-I are transiently unsyncronised, leading to change in oxidative state of PQ (i.e. accumulation of PQ or PQH<sub>2</sub>). This triggers two mechanisms used to regain the balance. First, PQH<sub>2</sub> binding to Cyt<em>b<sub>6</sub>f</em> activates STN7, which in turn causes phosphorylation of LHC-II and its migration from PS-II to PS-I. Second, PQ activates CSK, which phosphorylates SIG1, causing PS-II genes to be preferentially transcribed.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Andersson-2003">
<p>Andersson, Jenny, Mark Wentworth, Robin G Walters, Caroline A Howard, Alexander V Ruban, Peter Horton, and Stefan Jansson. 2003. “Absence of the Lhcb1 and Lhcb2 Proteins of the Light-Harvesting Complex of Photosystem Ii - Effects on Photosynthesis, Grana Stacking and Fitness.” <em>Plant J</em> 35 (3). UmeåPlant Science Centre, Department of Plant Physiology, UmeåUniversity, S-901 87 Umeå, Sweden. jenny.andersson@plantphys.umu.se: 350–61. <a href="https://doi.org/10.1046/j.1365-313x.2003.01811.x">https://doi.org/10.1046/j.1365-313x.2003.01811.x</a>.</p>
</div>
<div id="ref-Barros-2009">
<p>Barros, Tiago, and Werner Kühlbrandt. 2009. “Crystallisation, Structure and Function of Plant Light-Harvesting Complex Ii.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1787 (6): 753–72. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2009.03.012">https://doi.org/https://doi.org/10.1016/j.bbabio.2009.03.012</a>.</p>
</div>
<div id="ref-Bellafiore-2005">
<p>Bellafiore, Stéphane, Frédy Barneche, Gilles Peltier, and Jean-David Rochaix. 2005. “State Transitions and Light Adaptation Require Chloroplast Thylakoid Protein Kinase Stn7.” <em>Nature</em> 433 (7028). Department of Molecular Biology, University of Geneva, 30, Quai Ernest Ansermet, 1211 Geneva, Switzerland.: 892–95. <a href="https://doi.org/10.1038/nature03286">https://doi.org/10.1038/nature03286</a>.</p>
</div>
<div id="ref-Buchanan-2015">
<p>Buchanan, Bob B., Wilhelm Gruissem, and Russel L. Jones. 2015. <em>Biochemistry &amp; Molecular Biology of Plants</em>. John Wiley &amp; Sons.</p>
</div>
<div id="ref-Holloszy-1996">
<p>Holloszy, John O., and Wendy M. Kohrt. 1996. “Regulation of Carbohydrate and Fat Metabolism During and After Exercise.” <em>Annual Review of Nutrition</em> 16 (1): 121–38. <a href="https://doi.org/10.1146/annurev.nu.16.070196.001005">https://doi.org/10.1146/annurev.nu.16.070196.001005</a>.</p>
</div>
<div id="ref-Horowitz-2000">
<p>Horowitz, Jeffrey F, and Samuel Klein. 2000. “Lipid metabolism during endurance exercise.” <em>The American Journal of Clinical Nutrition</em> 72 (2): 558S–563S. <a href="https://doi.org/10.1093/ajcn/72.2.558S">https://doi.org/10.1093/ajcn/72.2.558S</a>.</p>
</div>
<div id="ref-Jensen-2012">
<p>Jensen, Thomas E., and Erik A. Richter. 2012. “Regulation of Glucose and Glycogen Metabolism During and After Exercise.” <em>The Journal of Physiology</em> 590 (5): 1069–76. <a href="https://doi.org/10.1113/jphysiol.2011.224972">https://doi.org/10.1113/jphysiol.2011.224972</a>.</p>
</div>
<div id="ref-Longoni-2015">
<p>Longoni, Paolo, Damien Douchi, Federica Cariti, Geoffrey Fucile, and Michel Goldschmidt-Clermont. 2015. “Phosphorylation of the Light-Harvesting Complex Ii Isoform Lhcb2 Is Central to State Transitions.” <em>Plant Physiol</em> 169 (4): 2874–83. <a href="https://doi.org/10.1104/pp.15.01498">https://doi.org/10.1104/pp.15.01498</a>.</p>
</div>
<div id="ref-Maarbjerg-2011">
<p>Maarbjerg, S. J., L. Sylow, and E. A. Richter. 2011. “Current Understanding of Increased Insulin Sensitivity After Exercise – Emerging Candidates.” <em>Acta Physiologica</em> 202 (3): 323–35. <a href="https://doi.org/10.1111/j.1748-1716.2011.02267.x">https://doi.org/10.1111/j.1748-1716.2011.02267.x</a>.</p>
</div>
<div id="ref-Mittendorfer-2003">
<p>Mittendorfer, Bettina, and Samuel Klein. 2003. “Physiological Factors That Regulate the Use of Endogenous Fat and Carbohydrate Fuels During Endurance Exercise.” <em>Nutrition Research Reviews</em> 16 (1). Cambridge University Press: 97–108. <a href="https://doi.org/10.1079/NRR200357">https://doi.org/10.1079/NRR200357</a>.</p>
</div>
<div id="ref-Mullet-1983">
<p>Mullet, J E. 1983. “The Amino Acid Sequence of the Polypeptide Segment Which Regulates Membrane Adhesion (Grana Stacking) in Chloroplasts.” <em>J Biol Chem</em> 258 (16): 9941–8.</p>
</div>
<div id="ref-Puthiyaveetil-2008">
<p>Puthiyaveetil, Sujith, T Anthony Kavanagh, Peter Cain, James A Sullivan, Christine A Newell, John C Gray, Colin Robinson, Mark van der Giezen, Matthew B Rogers, and John F Allen. 2008. “The Ancestral Symbiont Sensor Kinase Csk Links Photosynthesis with Gene Expression in Chloroplasts.” <em>Proc Natl Acad Sci U S A</em> 105 (29): 10061–6. <a href="https://doi.org/10.1073/pnas.0803928105">https://doi.org/10.1073/pnas.0803928105</a>.</p>
</div>
<div id="ref-Richter-2009">
<p>Richter, Erik A., and Neil B. Ruderman. 2009. “AMPK and the biochemistry of exercise: implications for human health and disease.” <em>Biochemical Journal</em> 418 (2): 261–75. <a href="https://doi.org/10.1042/BJ20082055">https://doi.org/10.1042/BJ20082055</a>.</p>
</div>
<div id="ref-Taiz-2010">
<p>Taiz, Lincoln, and Eduardo Zeiger. 2010. <em>Plant Physiology</em>. Sinauer Associates.</p>
</div>
<div id="ref-Takahashi-2013">
<p>Takahashi, Hiroko, Sophie Clowez, Francis-André Wollman, Olivier Vallon, and Fabrice Rappaport. 2013. “Cyclic Electron Flow Is Redox-Controlled but Independent of State Transition.” <em>Nat Commun</em> 4. Institut de Biologie Physico-Chimique, UMR 7141 CNRS-UPMC, 13 rue P et M Curie, 75005 Paris, France.: 1954. <a href="https://doi.org/10.1038/ncomms2954">https://doi.org/10.1038/ncomms2954</a>.</p>
</div>
<div id="ref-Thompson-2012">
<p>Thompson, Dylan, Fredrik Karpe, Max Lafontan, and Keith Frayn. 2012. “Physical Activity and Exercise in the Regulation of Human Adipose Tissue Physiology.” <em>Physiological Reviews</em> 92 (1): 157–91. <a href="https://doi.org/10.1152/physrev.00012.2011">https://doi.org/10.1152/physrev.00012.2011</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/sensing-energy-status-during-exercise.html" class="archive-item-link">
                Fuel Selection During Exercise 
              </a>
              <span class="archive-item-date">
                2020-05-25
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#overview"><span class="toc-section-number">2</span> Overview</a><ul>
<li><a href="#glycogen-and-glucose"><span class="toc-section-number">2.1</span> Glycogen and Glucose</a><ul>
<li><a href="#blood-glucose"><span class="toc-section-number">2.1.1</span> Blood Glucose</a></li>
<li><a href="#regulation-of-glycogenolysis"><span class="toc-section-number">2.1.2</span> Regulation of Glycogenolysis</a></li>
</ul></li>
<li><a href="#fatty-acids"><span class="toc-section-number">2.2</span> Fatty Acids</a><ul>
<li><a href="#adipose-tissue-lipolysis-and-plasma-nefa"><span class="toc-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</a></li>
<li><a href="#intramuscular-triacylglycerol-imtg"><span class="toc-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</a></li>
<li><a href="#plasma-tg-vldl-tg"><span class="toc-section-number">2.2.3</span> Plasma TG (VLDL-TG)</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p><strong>What effects do the intensity and duration of exercise have on fuel selection by mammalian muscle?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>During exercise, glucose and fatty acids are the fuels used by the muscle. Their origin and relative contribution depend on the intensity and duration of exercise, as well as on characteristics of individuals such as the level of physical training.</p>
</div>
<div id="overview" class="section level1">
<h1><span class="header-section-number">2</span> Overview</h1>
<p>As shown in Figure 1, muscle contraction during exercise is powered by oxidation of glucose and fatty acids derived from a number of sources.</p>
<div class="figure"><span id="fig:powering-the-muscle"></span>
<img src="../img/powering-the-muscle.png" alt="Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6)." width="100%" />
<p class="caption">
Figure 2.1: Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6).
</p>
</div>
<p>Immediately after the onset of exercise (0-30 s) energy is derived predominantly from anaerobic respiration. After this period, both excercise intensity and duration determine the choice of fuel used for oxidation. As shown in Figure 2, during the first 30 minutes of a low-intensity exercise bout, the vast majority of fuel used by muscle is plasma NEFA (non-esterified fatty acids) originated from adipose lipolysis with tiny contributions from IMTG (intramuscular triglyceride) and plasma glucose. For moderately intense exercise, the contribution from fat is lower: plasma NEFA and IMTG each represent about 25% of total energy expenditure, and glycogen and plasma glucose account for the remaining 40% and 10%. For high-intensity exercise, the contribution from muscle glycogen is even higher (60%), and fat and glucose represents 30% and 10% of the total, respectively. Overall, as exercise intensity increases, carbohydrates (especially muscle glycogen) progressively becomes the preferred fuel over triglyceride-derived fuel. As for TG-derived fuel, the ratio of IMTG to plasma NEFA is greatest during moderately intense exercise and lowest during low-intensity excercise. These are explained in detail later.</p>
<p>Figure 2 also shows that, during prolonged (low- or moderate-intensity) exercise, the contribution from plasma NEFA increases and the contribution from other fuels decreases, regardless of exercise intensity. This is due to the large amount of TG reserve in adipose tissue and the gradual depletion of other fuels.</p>
<div class="figure"><span id="fig:exercise-choice-of-fuel"></span>
<img src="../img/exercise-choice-of-fuel.png" alt="Effects of exercise intensity and duration on choice of substrates. 'Percentage of VO~2max~', where VO~2max~ is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO~2max~ is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity." width="100%" />
<p class="caption">
Figure 2.2: Effects of exercise intensity and duration on choice of substrates. ‘Percentage of VO<sub>2max</sub>’, where VO<sub>2max</sub> is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO<sub>2max</sub> is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity.
</p>
</div>
<div id="glycogen-and-glucose" class="section level2">
<h2><span class="header-section-number">2.1</span> Glycogen and Glucose</h2>
<div id="blood-glucose" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Blood Glucose</h3>
<p>During exercise, catecholamines stimulates pancreatic <span class="math inline">\(\alpha\)</span> cells to secrete glucagon, which in turn promtotes hepatic glycogenolysis and gluconeogenesis. Both processes release glucose into the bloodstream.</p>
<p>Blood glucose is the predominant source of carbohydrate for oxidation in muscle. Its absolute amount of usage increases as excercise intensity increases (though proportionally remains largely unchanged), which is mediated by an increaed sensitivity to insulin (in part due to utilisation of IMTG) and thus increased translocation of GLUT4 to the cell surface membrane.</p>
</div>
<div id="regulation-of-glycogenolysis" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Regulation of Glycogenolysis</h3>
<p>Glycogen phosphorylase (GP) and phosphorylase kinase (PhK) are central to the regulation of glycogenolysis. Activated PhK phosphorylates GP, thus converting it from the inactive <em>b</em> form to the active <em>a</em> form. GP in its <em>a</em> form then phosphorolyses glycogen, releasing glucose 1-phosphase available for oxidation.</p>
<p>PhK is activated (1) by phosphorylation on <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> subunits by PKA and (2) by binding of Ca<sup>2+</sup> to <span class="math inline">\(\delta\)</span> subunit. PKA is activated upon catecholamine (mostly adrenaline) binding to cell surface receptors (via G<sub>s</sub>, adenylyl cyclase, and cAMP), and Ca<sup>2+</sup> is released upon muscle contraction (stimulated by acetylcholine).</p>
<p>Ca<sup>2+</sup> released upon muscle contraction results in a burst of glycogenolysis. The large amount of glycogen breakdown exceeds muscle’s energy requirement and capacity of TCA cycle, resulting in accumulation of lactate. This probably explains the short period of anaerobic respiration at the onset of exercise. After a few minutes, the activation of phosphorylase reverses by a mechanism that is not fully understood.</p>
<p>Myocytes also respond, via <span class="math inline">\(\beta\)</span> adrenergic receptors, to catecholamines that are produced proportionally to exercise intensity. This results in activation of PKA (via G<sub>s</sub>, AC and cAMP), which in turn activates glycogenolysis.</p>
<p>The rate of glycogenolysis also depends on availability of P<sub>i</sub>, the substrate of glycogen phosphorylase. At rest and during mild exercise, fatty acid oxidation is capable of generating sufficient ATP, resulting in low P<sub>i</sub> concentration. Availability of P<sub>i</sub> increases as exercise intensity increases because the rate of ATP hydrolysis exceeds the rate of convertion from ADP and P<sub>i</sub> back to ATP by oxidation of fatty acids (and plasma glucose).</p>
</div>
</div>
<div id="fatty-acids" class="section level2">
<h2><span class="header-section-number">2.2</span> Fatty Acids</h2>
<p>During exercise, the usage of fatty acids has an important role of delaying the onset of glycogen depletion and hypoglycemia. Its contribution to total energy expenditure increases as excercise proceeds. However, its usage is limited during strenuous exercise.</p>
<div id="adipose-tissue-lipolysis-and-plasma-nefa" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</h3>
<p>Compared to resting conditions, adipose tissue lipolysis occurs at a higher rate during mild- or moderate-intensity exercise. This is mainly mediated by increased <span class="math inline">\(\beta\)</span>-adrenergic receptor activation by catecholamines released by the adrenal gland upon sympathetic nerve stimulation. Exercise also increases blood flow in adipose tissue (which prevents toxic regional accumulation of fatty acids) and in muscle, thus promoting transport of NEFA to myocytes. Passive uptake of NEFA by myocytes, which is proportional to plasma NEFA concentration, is thus increased. AMPK, which is activated upon transition from rest to exercise and during prolonged exercise, phosphorylates ACC2 and malonyl-CoA decarboxylase, inhibiting the former and activating the latter, thus resulting in a decrease of malonyl-CoA concentration and hence releasing CPT-1 from inhibition. This promotes transport of fatty acids into mitochondria for <span class="math inline">\(\beta\)</span>-oxidation in muscle cells.</p>
<p>As the exercise intensity continue to increase, however, oxidation of NEFA derived from adipose tissue lipolysis in muscle decreases. This is in part caused by decreased rate of lipolysis due to lactate (through activation of an orphan GPCR, GPR81). Lactate can also increase re-esterification of fatty acids to TG in the liver. Raising plasma fatty acid concentrations by intravenously infusing a lipid emulsion and heparin during the exercise bout increases plasma NEFA oxidation by about 30% but does not completely restore it to the rate observed during moderate-intensity exercise. Thus, high-intensity exercise also decreases the capacity of skeletal muscle to oxidise fatty acids, which is in part caused by increased glycogen catabolism. As more acetyl-CoA concentration are produced via glycogen catabolism, they are converted to malonyl-CoA by ACC2 located on the outer mitochondrial membrane, and the malonyl-CoA then inhibits CPT-1 and hence fatty acid oxidation.</p>
</div>
<div id="intramuscular-triacylglycerol-imtg" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</h3>
<p>Within myocytes, there are lipid droplets that stores TG. Their usage during exercise can be calculated by subtracting plasma fatty acid oxidation (determined by isotope tracers) from whole-body fat oxidation (determined by indirect calorimetry).</p>
<p>IMTG use is stimulated by catecholamines via <span class="math inline">\(\beta\)</span> adrenergic receptors. This is supported by the finding that pharmarcological blockage of <span class="math inline">\(\beta_1 + \beta_2\)</span> prevents the usage of IMTG.</p>
</div>
<div id="plasma-tg-vldl-tg" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Plasma TG (VLDL-TG)</h3>
<p>The liver takes up plasma NEFA and glycerol and use some of them to resynthesise TG. These TG are packaged in VLDL and released into the bloodstream. Muscle can break down some VLDL by lipoprotein lipase and oxidise the resulting fatty acids. However, this only contributes to a small fraction of energy usage during exercise and might be more important for replenishing IMTG stores after exercise.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/the-aetiology-of-diabetes-mellitus-and-insulin-resistance.html" class="archive-item-link">
                Insulin and Ketone Bodies 
              </a>
              <span class="archive-item-date">
                2020-05-18
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#regulation-of-ketogenesis-by-insulin"><span class="toc-section-number">2</span> Regulation of Ketogenesis by Insulin</a><ul>
<li><a href="#adipocyte-lipolysis"><span class="toc-section-number">2.1</span> Adipocyte Lipolysis</a></li>
<li><a href="#regulation-of-cpt1"><span class="toc-section-number">2.2</span> Regulation of CPT1</a></li>
<li><a href="#channeling-of-acetyl-coa-into-ketogenesis"><span class="toc-section-number">2.3</span> Channeling of Acetyl-CoA into Ketogenesis</a></li>
<li><a href="#mitochondrial-hmg-coa-synthase"><span class="toc-section-number">2.4</span> Mitochondrial HMG-CoA Synthase</a></li>
</ul></li>
<li><a href="#diabetes-and-ketone-bodies"><span class="toc-section-number">3</span> Diabetes and Ketone Bodies</a><ul>
<li><a href="#aetiology-of-diabetes"><span class="toc-section-number">3.1</span> Aetiology of Diabetes</a></li>
<li><a href="#overproduction-of-ketone-bodies"><span class="toc-section-number">3.2</span> Overproduction of Ketone Bodies</a></li>
<li><a href="#consequences-of-ketoacidosis"><span class="toc-section-number">3.3</span> Consequences of Ketoacidosis</a></li>
</ul></li>
</ul>
</div>

<p><strong>How does insulin regulate the pathways for the conversion of body fat into ketone bodies? Why are ketone bodies overproduced in uncontrolled diabetes and what are the metabolic consequences.</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Production of ketone bodies (ketogenesis) occurs in the liver (and the kidneys) as an response to prevent hypoglycemia during starvation, and this process is normally suppressed and stimulated by insulin and glucagon, respectively. In addition to its physiological role in fasting, ketogenesis also has high activity in diabetes patients, which may lead to the potentially lethal condition called diabetic ketoacidosis.</p>
</div>
<div id="regulation-of-ketogenesis-by-insulin" class="section level1">
<h1><span class="header-section-number">2</span> Regulation of Ketogenesis by Insulin</h1>
<p>Insulin suppresses ketogenesis by several mechanisms.</p>
<p>The steps of ketogenesis are breifly shown in Figure <a href="#fig:ketogenesis-brief-diagram">2.1</a>. There are three control points by which ketogenesis can be regulated by insulin: (1) adipocyte lipolysis, (2) mitochondrial fatty acid entry in ketogenic cells (mainly hepatocytes), (3) channeling of Acetyl-CoA (AcCoA) into the ketogenic pathway, and (4) mitochondrial 3-hydroxy-3-methylglutaryl-CoA synthase (HMG-CoA synthase).</p>
<div class="figure"><span id="fig:ketogenesis-brief-diagram"></span>
<img src="../img/ketogenesis-brief-diagram.jpg" alt="Production of ketone bodies from adipose-derived fatty acids." width="100%" />
<p class="caption">
Figure 2.1: Production of ketone bodies from adipose-derived fatty acids.
</p>
</div>
<div id="adipocyte-lipolysis" class="section level2">
<h2><span class="header-section-number">2.1</span> Adipocyte Lipolysis</h2>
<p>The major source molecules for ketogenesis are fatty acids obtained via lipolysis in white adipose tissues. Lipolysis involves the sequential hydrolysis of triacylglycerol (TAG) into 3 non-esterified fatty acids (NEFA) and glycerol by adipocyte triglyceride lipase (ATGL), hormone-sensitive lipase (HSL) and monoglyceride lipase (MGL). Activation of ATGL and HSL are dependent on PKA, which is activated by cAMP that are produced when catecholamines bind to cell-surface <span class="math inline">\(\beta\)</span> adrenergic receptors.</p>
<p>Insulin is a potent inhibitor of lipolysis. It activates PKB via IRS and PI3K, and PKB in turn phosphorylates and activates phosphodiesterase 3B (PDE-3B). PDB-3B decyclises cAMP, thus diminishing PKA, and thus lipolytic, activity (Figure <a href="#fig:ketogenesis-insulin-detail">2.2</a>). During starvation, insulin levels are low, and adipocyte lipolysis is released from inhibition. Adipocytes are then able to release NEFA into bloodstream.</p>
<div class="figure"><span id="fig:ketogenesis-insulin-detail"></span>
<img src="../img/ketogenesis-insulin-detail.jpg" alt="Left, regulation of lipolysis by insulin in adipocytes; right, regulation of uptake of long-chain fatty acids into mitochondria by insulin." width="100%" />
<p class="caption">
Figure 2.2: Left, regulation of lipolysis by insulin in adipocytes; right, regulation of uptake of long-chain fatty acids into mitochondria by insulin.
</p>
</div>
</div>
<div id="regulation-of-cpt1" class="section level2">
<h2><span class="header-section-number">2.2</span> Regulation of CPT1</h2>
<p>Once NEFA molecules arrive at hepatocytes, they are first broken down into AcCoA via <span class="math inline">\(\beta\)</span>-oxidation before conversion into ketone bodies. Long chain fatty acids needs the carnitine shuttle to enter mitochondria where <span class="math inline">\(\beta\)</span>-oxidation takes place, and thus the first enzyme of this shuttling system, carnitine palmitoyl transferase 1 (CPT1) is the rate limiting step and is under control. CPT1 is inhibited by malonyl CoA, which is the lipogenetic precursor produced by acetyl-CoA carboxylase (ACC). Insulin activates protein phosphotase 2A (PP2A, which is also activated by xylulose 5-phosphate during fed state), which dephosphorylates and activates ACC, thus promoting lipogenesis and suppressing <span class="math inline">\(\beta\)</span>-oxidation and ketogenesis (Figure <a href="#fig:ketogenesis-insulin-detail">2.2</a>).</p>
<p>During starvation when insulin levels are low, the high fatty acid concentration in hepatocytes activates the transcription factor PPAR<span class="math inline">\(\alpha\)</span> (activated by direct binding of fatty acids to PPAR<span class="math inline">\(\alpha\)</span>), which in turn promotes transcription of several enzymes involved in lipid catabolism, including CPT1.</p>
</div>
<div id="channeling-of-acetyl-coa-into-ketogenesis" class="section level2">
<h2><span class="header-section-number">2.3</span> Channeling of Acetyl-CoA into Ketogenesis</h2>
<p>AcCoA produced from NEFA via <span class="math inline">\(\beta\)</span>-oxidation can either enter the Krebs cycle or the ketogenesis pathway. When the rate of production of AcCoA exceeds the capacity of citrate synthesis (when there is insufficient oxaloacetate), the surplus becomes the substrate for ketogenesis. This situation occurs during starvation, when insulin levels are low and glucagon levels are high, and gluconeogenesis active. Glucagon has several mechanisms to promote gluconeogenesis (and suppress glycolysis). For example, it activates PKA, which in turn activates the transcription factor CREB, which promotes transcription of gluconeogenenic enzymes such as PEP carboxykinase and glucose 6-phosphatase. Insulin, on the contary, suppresses gluconeogenesis. For instance, it activates PKB, which in turn phosphorylates and thus inactivates FOXO1, which is a transcription factor that promotes the transcription of a similar set of gluconeogenic enzymes. Insulin inhibits gluconeogenesis and thus ketogenesis.</p>
</div>
<div id="mitochondrial-hmg-coa-synthase" class="section level2">
<h2><span class="header-section-number">2.4</span> Mitochondrial HMG-CoA Synthase</h2>
<p>Mitochondrial HMG-CoA catalyses the irreversible step in ketogenesis, and therefore is a site of control.</p>
<p>Insulin promotes glycolysis and increases flux through the Krebs cycle. One of the Krebs cycle intermediate, succinyl-CoA, inactivates mitochondrial HMG-CoA by succinylation. Glucagon, on the contrary, activates gluconeogenesis, which causes depletion of Krebs cycle intermediates including succinyl-CoA, thus activating ketogenesis.</p>
<p>In addition, the transcription factor PPAR<span class="math inline">\(\alpha\)</span> described previously also activates transcription of HMG-CoA synthase.</p>
</div>
</div>
<div id="diabetes-and-ketone-bodies" class="section level1">
<h1><span class="header-section-number">3</span> Diabetes and Ketone Bodies</h1>
<div id="aetiology-of-diabetes" class="section level2">
<h2><span class="header-section-number">3.1</span> Aetiology of Diabetes</h2>
<p>Diabetes is characterised by insufficient insulin activity, which could be caused by either lack of insulin production or insulin resistance in target cells. Most diabetes cases fall into two categories, type 1 and type 2.</p>
<ul>
<li>Type 1 diabetes is caused by autoimmunity agaist pancreatic <span class="math inline">\(\beta\)</span> cells, which leads to their death and inadequacy of insulin. The primary risk factor for type 1 diabetes is genetic, although several environmental risk factors, such as infection by viruses that show tropism to pancreatic islets.</li>
<li>Type 2 diabetes begins with insulin resistance, but reduction of insulin production and number of <span class="math inline">\(\beta\)</span> cells may also be implicated as the disease advances. Type 2 diabetes also has genetic risk factors, but environmental factors, such as sedentarity and obesity, are more important.</li>
</ul>
<p>Whichever the type, ketogenesis is hyperactive in diabetes patients (although generally more severe in type 1) due to inadequate insulin activity.</p>
</div>
<div id="overproduction-of-ketone-bodies" class="section level2">
<h2><span class="header-section-number">3.2</span> Overproduction of Ketone Bodies</h2>
<p>Given the fact that insulin suppresses ketogenesis and insulin activity is decreased in diabetic patients, it is easy to comprehend why ketone bodies are overproduced in diabetic individuals.</p>
<p>Lack of insulin activity upon adipocytes leads to uncontrolled lipolysis and subsequent release of non-esterified fatty acids. As fatty acids enter the liver, they are shuttled into mitochondria, broken down into acetyl-CoA via <span class="math inline">\(\beta\)</span>-oxidation and converted to ketone bodies. This process is facilitated by the low insulin activity, which activates gluconeogenesis (e.g. via FOXO1-induced transcription of gluconeogenenic enzymes and phosphorylation of PFK2/FBPase2 bifunctional enzyme), which in turn releases the rate-limiting enzyme, HMG-CoA from succinylation inhibition and reduces usage of acetyl-CoA in Krebs cycle. The low insulin activity also inhibits ACC, thus activating CPT1 and hence <span class="math inline">\(\beta\)</span>-oxidation. The high fatty acid concentration also leads to activation of PPAR<span class="math inline">\(\alpha\)</span>, which promotes transcription of ketogenic enzymes such as CPT-1 and HMG-CoA synthatase.</p>
</div>
<div id="consequences-of-ketoacidosis" class="section level2">
<h2><span class="header-section-number">3.3</span> Consequences of Ketoacidosis</h2>
<p>Diabetic ketoacidosis (DKA) the most common acute emergency in diabetes patients. It is a potentially lethal condition that is characterised by lowered blood pH due to accumulation of acidic ketone bodies (acetoacetate and 3-hydroxybutyrate). DKA is caused by lack of insulin and thus occurs mostly in uncontrolled type 1 diabetes (DKA is not commonly seen in type 2 diabetes, because even a tiny amount of insulin activity would inhibit ketogenesis; however, there is a small proportion of “ketosis-prone type 2 diabetes”). Often, DKA is triggered by stress or illness, which are associated with increase of ketogenesis-promoting hormones, glucagon and catecholamines.</p>
<p>Ketoacidosis has several consequences, as described below.</p>
<ul>
<li>Kussmaul breathing: initally, bicarbonate ions buffur protons dissociated from ketone bodies, but this buffering system quickly becomes overwhelmed. To compensate, blood CO<sub>2</sub> concentration is reduced by deep, regular breaths known as Kussmaul breathing.</li>
<li>Hypovolaemia: hyperglycemia and ketonaemia results in excretion of glucose and ketone bodies (which normally should be absorbed) in urine. Since they are osmotically active, their presence in urine causes osmotic diuresis, which further leads to hypovolaemia (decrease in body fluid volume).</li>
<li>Electrolyte disturvance: as the the plasma pH falls, extracellular protons are exchanged for intracellular potassium ions. In addition, owing to hypovolaemia, the aldosterone concentration increases to retain water, but this hormone also leads to excretion of K<sup>+</sup> and retention of Na<sup>+</sup>.</li>
<li>The low plasma pH impair with many biological functions, especially in the brain, where potentially life-threatening coma and cerebral oedema may occur.</li>
</ul>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/glucagon-and-insulin.html" class="archive-item-link">
                Glucagon and Insulin 
              </a>
              <span class="archive-item-date">
                2020-05-11
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#initial-steps-of-glucagon-and-insulin-signalling"><span class="toc-section-number">2</span> Initial Steps of Glucagon and Insulin Signalling</a><ul>
<li><a href="#insulin-signalling"><span class="toc-section-number">2.1</span> Insulin Signalling</a></li>
<li><a href="#glucagon-signalling"><span class="toc-section-number">2.2</span> Glucagon Signalling</a></li>
</ul></li>
<li><a href="#glucose-metabolism"><span class="toc-section-number">3</span> Glucose Metabolism</a><ul>
<li><a href="#glycogen-metabolism-in-liver"><span class="toc-section-number">3.1</span> Glycogen Metabolism in Liver</a><ul>
<li><a href="#glucagon-activates-glycogenolysis-in-liver"><span class="toc-section-number">3.1.1</span> Glucagon Activates Glycogenolysis in Liver</a></li>
<li><a href="#insulin-activates-glycogen-synthesis-in-liver"><span class="toc-section-number">3.1.2</span> Insulin Activates Glycogen Synthesis in Liver</a></li>
</ul></li>
<li><a href="#glycolysis-and-gluconeogeneis-in-liver"><span class="toc-section-number">3.2</span> Glycolysis and Gluconeogeneis in Liver</a><ul>
<li><a href="#short-term-control-via-pfk2pbpase2-bifunctional-enzyme"><span class="toc-section-number">3.2.1</span> Short-Term Control via PFK2/PBPase2 Bifunctional Enzyme</a></li>
<li><a href="#long-term-transcriptional-control"><span class="toc-section-number">3.2.2</span> Long-Term Transcriptional Control</a></li>
</ul></li>
<li><a href="#glucose-uptake-into-muscle-cells"><span class="toc-section-number">3.3</span> Glucose Uptake into Muscle Cells</a></li>
</ul></li>
<li><a href="#lipid-metabolism"><span class="toc-section-number">4</span> Lipid Metabolism</a><ul>
<li><a href="#lipolysis-in-adipocytes"><span class="toc-section-number">4.1</span> Lipolysis in Adipocytes</a></li>
<li><a href="#tag-synthesis-in-liver"><span class="toc-section-number">4.2</span> TAG Synthesis in Liver</a></li>
<li><a href="#tag-re-synthesis-in-adipocytes"><span class="toc-section-number">4.3</span> TAG (Re-)Synthesis in Adipocytes</a></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">5</span> Conclusion</a></li>
</ul>
</div>

<p><strong>“Glucagon and insulin affect the body in opposite ways.” To what extent is this statement true?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Glucagon and insulin are peptide hormones secreted by the endocrine cells (<span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> cells, respectively) of the islets of Langerhans in pancreas. Although these two hormones differ in their target tissues and mechanisms of action, the metabolic changes they bring about, at the whole-body level, have opposite effects, which helps to maintain energy homeostasis. Specifically, insulin results in adaptation to high blood glucose concentration and high energy status, which occurs after a meal, while glucagon results in adaptation to low blood glucose concentration.</p>
</div>
<div id="initial-steps-of-glucagon-and-insulin-signalling" class="section level1">
<h1><span class="header-section-number">2</span> Initial Steps of Glucagon and Insulin Signalling</h1>
<p>Glucagon and insulin each has diverse effects in different tissues, but the initial steps are common. To avoid repetition, these step are described in this section.</p>
<div id="insulin-signalling" class="section level2">
<h2><span class="header-section-number">2.1</span> Insulin Signalling</h2>
<p>The insulin receptor belongs to the receptor tyrosine kinase (RTK) family. Upon insulin binding, the receptor dimer trans-autophosphorylates each other on tyrosine residues (pY). IRS1 (insulin receptor substrate 1) binds to pY via its SH2 (src homology 2) domain. Insulin receptor phosphorylates IRS1 on tyrosine, and then PI3K (phosphoinositide 3 kinase) binds to pY of IRS1 via SH2. PI3K converts PIP2 (PtdIns(4,5)P2) to PIP3 (PtdIns(3,4,5)P3), then PIP3 serves as the docking site for PDK1 (phosphoinositide dependent kinase 1) and PKB (Akt). PDK1 and mTOR2C activates PKB by phosphorylation, and PKB then phosphorylates other protein targets to bring about most insulin-induced metabolic changes. Among these effects, a significant porportion is mediated by PP1 (protein phosphatase 1), which is activated by PKB (Figure <a href="#fig:insulin-overview">2.1</a>).</p>
<div class="figure"><span id="fig:insulin-overview"></span>
<img src="../img/insulin-overview.png" alt="Overview of insulin signalling." width="100%" />
<p class="caption">
Figure 2.1: Overview of insulin signalling.
</p>
</div>
</div>
<div id="glucagon-signalling" class="section level2">
<h2><span class="header-section-number">2.2</span> Glucagon Signalling</h2>
<p>Glucagon receptors are G-protein coupled receptors (GPCR). Upon glucagon binding, the receptor acts as a guanine nucleotide exchange factor and replace GDP bound to the G protein with GTP. The activated heterotrimeric G protein dissociates into <span class="math inline">\(\beta\gamma\)</span> and <span class="math inline">\(\alpha\)</span> subunits. Some glucagon receptors are associated with G<sub>s</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates adenylyl cyclase, which produces cAMP, which in turn activates protein kinase A (PKA). Other glucagon receptors are associated with G<sub>q</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates phospholipase C, which converts PIP2 (phosphatidylinositol bisphosphate) to IP3 (inositol trisphosphate) and DAG (diacylglycerol). IP3 stimulates Ca<sup>2+</sup> release from the smooth endoplasmic reticulum. PKA and Ca<sup>2+</sup> then activates downstream proteins (Figure <a href="#fig:glucagon-liver-gs-gq">2.2</a> ).</p>
<div class="figure"><span id="fig:glucagon-liver-gs-gq"></span>
<img src="../img/glucagon-liver-gs-gq.jpg" alt="Overview of glucagon signalling." width="100%" />
<p class="caption">
Figure 2.2: Overview of glucagon signalling.
</p>
</div>
</div>
</div>
<div id="glucose-metabolism" class="section level1">
<h1><span class="header-section-number">3</span> Glucose Metabolism</h1>
<p>Maintaining a glucose concentration of around 5 mmol/L is crucial to human survival. This glucose homeostatis is maintained by glucagon and insulin, where the former results in net increase in glucose concentration and the latter results in net decrease in glucose concentration.</p>
<div id="glycogen-metabolism-in-liver" class="section level2">
<h2><span class="header-section-number">3.1</span> Glycogen Metabolism in Liver</h2>
<p>In liver, net synthesis of glycogen (in order to lower down blood glucose concentration) is stimulated by insulin and net consumption is promoted by glucagon.</p>
<div id="glucagon-activates-glycogenolysis-in-liver" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Glucagon Activates Glycogenolysis in Liver</h3>
<p>As described previously, glucagon signalling results in activation of PKA and release of Ca<sup>2+</sup>. PKA phosphorylates and thus activates phosphorylase b kinase. Ca<sup>2+</sup> binds to phosphorylase b kinase, also activating it. Insulin inhibits glycogenolysis via PP1, which dephosphorylates phosphorylase kinase. Glucagon counteracts the effect of insulin in another way: PKA phosphorylates the R (regulatory) subunit of PP1, causing the dissociation of the C (catalytic) subunit, thus inactivating PP1 (Figure <a href="#fig:glucagon-insulin-glycogenolysis-simple">3.1</a> ).</p>
<div class="figure"><span id="fig:glucagon-insulin-glycogenolysis-simple"></span>
<img src="../img/glucagon-insulin-glycogenolysis-simple.png" alt="Effect of glucagon and insulin on hepatic glycogenolysis" width="100%" />
<p class="caption">
Figure 3.1: Effect of glucagon and insulin on hepatic glycogenolysis
</p>
</div>
</div>
<div id="insulin-activates-glycogen-synthesis-in-liver" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Insulin Activates Glycogen Synthesis in Liver</h3>
<p>Insulin activates glycogen synthesis in hepatocytes. Glycogen is synthesised by glycogen synthase (and branching enzymes), whose active (a) form is unphosphorylated. It is normally phosphorylated (inactivated) by caesin kinase 2 (CK2) and glycogen synthase kinase 3 (GSK3). Insulin-activated PKB phosphorylates (inavtivates) GSK3 and hence releases GS from inhibition (Figure <a href="#fig:insulin-gsk3">3.2</a>).</p>
<div class="figure"><span id="fig:insulin-gsk3"></span>
<img src="../img/insulin-gsk3.jpg" alt="Insulin stimulates glycogen synthesis in the liver." width="100%" />
<p class="caption">
Figure 3.2: Insulin stimulates glycogen synthesis in the liver.
</p>
</div>
</div>
</div>
<div id="glycolysis-and-gluconeogeneis-in-liver" class="section level2">
<h2><span class="header-section-number">3.2</span> Glycolysis and Gluconeogeneis in Liver</h2>
<p>Insulin activates glycolysis and inhibits gluconeogenesis in liver. Glucagon has opposite effects.</p>
<div id="short-term-control-via-pfk2pbpase2-bifunctional-enzyme" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Short-Term Control via PFK2/PBPase2 Bifunctional Enzyme</h3>
<p>In liver, fructose 2,6-bisphosphate (F26BP) the key regulator of glycolysis and gluconeogenesis. It is an allosteric regulator of PFK-1 and FBPase-1 and have opposite effects on them. It significantly reduces the inhibitory effect of ATP on PFK-1—without F26BP, PFK-1 essentially has no activity in normal hepatocytes because hepatic ATP concentration is very high. In contrast, F26BP is an allosteric inhibitor of FBPase-1.</p>
<p>The PFK2/PBPase2 bifunctional enzyme (BFE) converts between fructose 6-phosphate (F6P) and F26BP. PFK2 is active in its unphosphorylated form while FBPase2 is active in its phosphorylated form. Insulin activates PP1, which dephosphorylates BFE (PP2A has the same effect), resulting in net production of F26BP, thus activating glycolysis and inhibiting gluconeogenesis. Glucagon activates PKA, which in turn phosphorylates BFE, reducing amount of F26BP and hence deactivating glycolysis and de-inhibiting (activating) gluconeogenesis (Figure <a href="#fig:hepatic-f26bp">3.3</a> ).</p>
<div class="figure"><span id="fig:hepatic-f26bp"></span>
<img src="../img/hepatic-f26bp.png" alt="Regulation of glycolysis and gluconeogenesis via F26BP in liver." width="100%" />
<p class="caption">
Figure 3.3: Regulation of glycolysis and gluconeogenesis via F26BP in liver.
</p>
</div>
<p>The isoform of pyruvate kinase (PK) present in liver is also phosphorylated and inactivated by PKA, and this also contributes to glucagon-induced inhibition of glycolysis.</p>
</div>
<div id="long-term-transcriptional-control" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Long-Term Transcriptional Control</h3>
<p>Glucagon-triggered rise in cAMP concentration activates CREB (cAMP response element binding protein), which turns on synthesis of glucose 6-phosphatase and PEP carboxykinase, two key enzymes involved in gluconeogenesis.</p>
<p>Insulin activates PKB, which phosphorylates and inactivates other transcription factors, such as FOXO1, and thus reduces transcription of gluconeogenic enzymes.</p>
</div>
</div>
<div id="glucose-uptake-into-muscle-cells" class="section level2">
<h2><span class="header-section-number">3.3</span> Glucose Uptake into Muscle Cells</h2>
<p>Insulin sigalling in muscle cells results in activation of PKB, which in turn causes translocation of GLUT4-containing vesicles to the cell surface. This significantly increases the rate of influx of glucose into muscle cells, thus reducing blood glucose concentration.</p>
</div>
</div>
<div id="lipid-metabolism" class="section level1">
<h1><span class="header-section-number">4</span> Lipid Metabolism</h1>
<p>Glucagon is released when glucose is not abundant, thus its overall effect is to promote mobilisation and utilisation of the alternative fuel, triacylglycerides (TAG). Insulin has the opposite effect: it is released after a meal, when the nutrient concentration in blood is high, and it supresses TAG usage and promotes TAG storage.</p>
<div id="lipolysis-in-adipocytes" class="section level2">
<h2><span class="header-section-number">4.1</span> Lipolysis in Adipocytes</h2>
<p>Lipolysis is the process by which triacylglycerol molecules (TAG) stored in the lipid droplet of adipocytes in white adipose tissues are hydrolysed to 3 free (non-esterified) fatty acid molecules and 1 glycerol molecule and released into the bloodstream, resulting in net increase in free fatty acid concentration in blood plasma.</p>
<p>As shown in Figure <a href="#fig:adipocyte-positive">4.1</a>, PKA is central to the activation of the lipolytic pathway. Catecholamines (adrenaline and noradrenaline), being the major activators of lipolysis, activates PKA via <span class="math inline">\(\beta\)</span> adrenergic receptor, G<sub>s</sub>, AC and cAMP. Glucagon has also been shown to activate lipolysis <em>in vitro</em> (via PKA), but there is no clear evidence that it acts <em>in vivo</em>.</p>
<div class="figure"><span id="fig:adipocyte-positive"></span>
<img src="../img/adipocyte-positive.png" alt="Activation of lipolysis by catecholamines (adrenaline and noradrenaline) and, with less evidence, by glucagon. PKA is activated via the GPCR-G~s~-AC-cAMP pathway described previously. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL. TAG is hydrolysed to 3 fatty acids and a glycerol molecule stepwisely by ATGL, HSL and MGL." width="100%" />
<p class="caption">
Figure 4.1: Activation of lipolysis by catecholamines (adrenaline and noradrenaline) and, with less evidence, by glucagon. PKA is activated via the GPCR-G<sub>s</sub>-AC-cAMP pathway described previously. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL. TAG is hydrolysed to 3 fatty acids and a glycerol molecule stepwisely by ATGL, HSL and MGL.
</p>
</div>
<p>Insulin inhibits lipolysis via PKB and PDE-3B, resulting in net decrease in blood fatty acid concentration (Figure <a href="#fig:lipolysis-insulin-simple">4.2</a>).</p>
<div class="figure"><span id="fig:lipolysis-insulin-simple"></span>
<img src="../img/lipolysis-insulin-simple.jpg" alt="Inhibition of lipolysis by insulin. Insulin activates PKB, which phosphorylates and thus activates the phosphodiesterase PDE-3B. PDE-3B de-cyclise cAMP, to AMP, thus reducing PKA activity and the lipolytic pathway downstream of PKA." width="100%" />
<p class="caption">
Figure 4.2: Inhibition of lipolysis by insulin. Insulin activates PKB, which phosphorylates and thus activates the phosphodiesterase PDE-3B. PDE-3B de-cyclise cAMP, to AMP, thus reducing PKA activity and the lipolytic pathway downstream of PKA.
</p>
</div>
</div>
<div id="tag-synthesis-in-liver" class="section level2">
<h2><span class="header-section-number">4.2</span> TAG Synthesis in Liver</h2>
<p>TAG synthesis in liver is promoted after a meal. This is mainly mediated by de-phosphorylation (activation) of ACC (acetyl-CoA carboxylase) by PP2A as a result of elevated glucose concentration and pentose phosphate pathway activity. (Some books, such as Lehninger’s, say that insulin-activated PP1 also de-phosphorylates ACC). The product of ACC, malonyl CoA, inhibits CPT1 (carnitine palmitoyl transferase) and hence oxidation of fatty acids.</p>
<p>Long term regulation of lipogenesis is achieved via the transcription factors ChREBP and SREBP-1c. These two TFs both up-regulates enzymes involved in lipogenesis, but their regulation differ. ChREBP is activated by xylulose 5-phosphate, which happens when there is a large flux through the pentose phosphate pathway due to high glucose concentration. This effect is reduced by cAMP, whose production is induced by glucagon. In contrast, SREBP-1c is activated by atypical PKC (aPKC), which is another protein kinase activated by insulin (Figure <a href="#fig:chrebp-srebp1c">4.3</a> ).</p>
<div class="figure"><span id="fig:chrebp-srebp1c"></span>
<img src="../img/chrebp-srebp1c.jpg" alt="Long-term regulation of lipogenesis via ChREBP and SREBP-1c." width="100%" />
<p class="caption">
Figure 4.3: Long-term regulation of lipogenesis via ChREBP and SREBP-1c.
</p>
</div>
</div>
<div id="tag-re-synthesis-in-adipocytes" class="section level2">
<h2><span class="header-section-number">4.3</span> TAG (Re-)Synthesis in Adipocytes</h2>
<p>During fed state, adipocytes, with help from endothelial cells, break down chylomicrons (from the intestine) and VLDLs (from the liver), hydrolyse the containing triacylglycerol molecules, take up the resulting fatty acids, and re-synthesise the TAG. Adipocytes do not express glycerol kinase, thus they have to obtain glycogen 3-phosphate, one of the substrate in the first step of TAG synthesis, from dihydroxyacetone phosphte (by glycerol 3-phosphate), which is a glycolytic intermediate. Insulin promotes glucose uptake and hence glycolysis in adipocytes, thus promoting TAG resynthesis.</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">5</span> Conclusion</h1>
<p>Although glucagon and insulin usually exert their effects on different enzymes, and sometimes in different tissues, the net results of their activity are opposite, which ultimately leads to energy homeostasis, where energy consumption and concentration of glucose in blood remain relatively constant while the pattern of food intake is changing.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/supplying-fuels-to-the-brain-during-starvation.html" class="archive-item-link">
                Supplying Fuels to the Brain During Starvation 
              </a>
              <span class="archive-item-date">
                2020-05-03
              </span>
              <div class="article-summary" style="display: none;">
                
                


<!--nocite: |
  @Altarejos-2011, Bezaire-2009, Chakravarty-2005, Elia-1991, Frayn-2010, Fukao-2004, Holness-2003, Hue-2009, Hutton-2009, Lass-2011, Nuttall-2008, Okar-2001, Randle-1998, Rider-2004, Rufer-2009, Saggerson-2008, Soeters-2012, Stanley-2006, Sugden-2006, Thomson-2009, Yang-2009-->
<p><strong>Title: As starvation proceeds, how does the supply of fuels to the brain change? How is this change related to a requirement to conserve body protein?</strong></p>
<p>Due to the blood-brain barrier, The human brain can only harness energy from catabolising either glucose or ketone bodies. Under normal circumstances, glucose of dietary origin is the predominant source of fuel for the brain. As starvation begins (during the postabsorptive phase), astrocytes in the brain break down their stored glycogen to supply fuel in the form of lactate to neurons, which is followed by hepatic glycogenolysis that continue to provide glucose for the brain for a short period. As starvation proceeds, upon depletion of glycogen, muscle protein is broken down for liver gluconeogenesis, which continues to provide glucose for the brain. Meanwhile, triacylglycerol-derived ketone bodies synthesised by the liver will represent an increasingly larger proportion of fuel for the brain.</p>
<p>As shown in Figure <a href="#fig:starve-stages">1</a>, starvation can be divided into 4 stages, each having a different strategy for supplying energy to the brain.</p>
<div class="figure"><span id="fig:starve-stages"></span>
<img src="../img/starve-stages.jpg" alt="Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds." width="100%" />
<p class="caption">
Figure 1: Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds.
</p>
</div>
<div id="stage-i-absoption-and-ii-post-absorption-glycogenolysis" class="section level1">
<h1>Stage I (Absoption) and II (Post-absorption): Glycogenolysis</h1>
<p>Soon after a meal, glucose obtained from food is absorbed from the gut into the bloodstream. The abundant glucose is the exclusive fuel used by the brain. The high blood glucose level also stimulates the <span class="math inline">\(\beta\)</span> cells in the islets of Langerhans in the pancreas to release insulin, which in turn promotes uptake and storage of glucose (as glycogen, in liver, muscle, actrocytes, and other tissues) and triacylglycerol (in lipid droplets in adipocytes).</p>
<p>During the post-absorptive phase, blood glucose concentration drops. Islets of Langerhans respond by reducing production of insulin in <span class="math inline">\(\beta\)</span> cells and increasing production of glucagon in <span class="math inline">\(\alpha\)</span> cells.</p>
<p>The principal effect of glucagon is to promote glycogenolysis in liver.</p>
<p>On the plasma membrane of hepatocytes, glucagon binds to its receptors, which are G-protein coupled receptors (GPCR), on hepatocyte cell surface membrane. Upon glucagon binding, the receptor acts as a guanine nucleotide exchange factor and replace GDP bound to the G protein with GTP. The activated heterotrimeric G protein dissociates into <span class="math inline">\(\beta\gamma\)</span> and <span class="math inline">\(\alpha\)</span> subunits.</p>
<p>Some glucagon receptors are associated with G<sub>s</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates adenylyl cyclase, which produces cAMP, which in turn activates protein kinase A (PKA). PKA phosphorylates phosphorylase kinase, thus activating it. Other glucagon receptors are associated with G<sub>q</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates phospholipase C, which converts PIP2 (phosphatidylinositol bisphosphate) to IP3 (inositol trisphosphate) and DAG (diacylglycerol). IP3 stimulates Ca<sup>2+</sup> release from the smooth endoplasmic reticulum, then Ca<sup>2+</sup> binds to phosphorylase kinase, also activating it.</p>
<p>Activated phosphorylase kinase phosphorylates glycogen phosphorylase <em>b</em> (inactive form), converting it to glycogen phosphorylase <em>a</em> (active form). Then, with help from the debranching enzyme, glycogen phosphorylase <em>a</em> break down glycogen into glucose 1-phosphate, which can be converted by phosphoglucose mutase to glucose 6-phosphate, which is then dephosphorylated by glucose 6-phosphatase. The resulting glucose is released into the bloodstream via GLUT2 transporter down the concentration gradient.</p>
<p>Glycogenolysis occurs not only in liver but also in astrocytes. Unlike liver glycogenolysis, however, 1-phosphoglucose in astrocytes are converted to lactate instead of glucose. Lactate is transported via monocarboxylate transporter (MCT) and supplied to neurons.</p>
<p>During this period, the brain still uses glucose-derived fuel (glucose and lactate) as the sole fuel at the same rate, so do other parts of the body.</p>
</div>
<div id="stage-iii-early-starvation-gluconeogenesis" class="section level1">
<h1>Stage III (Early Starvation): Gluconeogenesis</h1>
<p>As the glycogen storage become depleted, most tissues start to use fatty acids released by white adipose tissue as the fuel, but the brain continue to use, almost exclusively, glucose, which are now derived from gluconeogenesis.</p>
<p>In the human body, the principal substrates for gluconeogenesis are lactate, glucogenic amino acids and glycerol. During starvation, lactate is not an important gluconeogenic substrate for net gain of glucose, because it is produced mostly by anaerobic glycolysis of glucose itself.</p>
<p>During starvation, body protein is the only source of amino acids for gluconeogenesis. These amino acids are initially supplied by hepatic proteolysis and later predominantly provided by muscle proteins. Although the composition of muscle proteins is rather heterogenous, the majority of the release amino acids are alanine and glutamine. Most alanine is taken up by the liver for gluconeogenesis, while the role of glutamine is more varied—it is used by the kidney for acid-base balance during starvation, for example.</p>
<p>Glycerol, the other source molecule for gluconeogenesis during starvation, is produced by the adipocytes in the white adipose tissue in a process known as lipolysis (fat mobilisation). Lipolysis involves the sequential removal of the three fatty acids of a TAG molecule by adipose triglyceride lipase (ATGL), hormone sensitive hormone (HSL), and monoglyceride lipase (MGL), and the removal of the first two fatty acids by ATGL and HSL is under hormonal control (MGL is always active).</p>
<p>Catecholamines (adrenaline/noradrenaline), which are released upon activation of the sympathetic nervous system at the early stages of starvation, are the major activators of lipolysis. They do so via the GPCR-G<sub>s</sub>-AC-cAMP-PKA pathway, similar to that used by glucagon. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL.</p>
<p>Insulin is the major inhibitor of lipolysis. Insulin activates PKB (Akt), which in turn activates phosphodiesterase 3B (PDE-3B). PDE-3B de-cyclise cAMP, thus antagonising the PKA activity. Thus, during starvation, the low insulin level means de-inhibition of lipolysis.</p>
<p>The resulting glycerol and fatty acids are all released into the circulation, the former being used as a substrate for gluconeogenesis and the latter being used as the principal fuel by most tissues and well as a substrate for ketone bodies synthesis in the liver.</p>
<p>There are a number of mechanisms by which the liver ensures that gluconeogenenic precursors are used for gluconeogenesis rather than oxidation. For example, the large amount of acetyl-CoA obtained via <span class="math inline">\(\beta\)</span>-oxidation inhibits pyruvate dehydrogenase complex, the activity of which would result in irreversible loss of gluconeogenic precursors. As an another example, the low insulin level has a long-term effect on gluconeogenesis by altering gene expression. Insulin leads to activation of PKB, and PKB inactivates (by phosphorylation) the transcription factor FOXO1 that promotes transcription of gluconeogenenic enzymes such as PEP carboxykinase. Thus, the low insulin level during starvation effectively activates the transcription of these gluconeogenic enzymes.</p>
</div>
<div id="stage-iv-prolonged-starvation-ketone-body-synthesis" class="section level1">
<h1>Stage IV (Prolonged Starvation): Ketone Body Synthesis</h1>
<p>During prolonged starvation, the brain gradually adopts ketone bodies (acetoacetate and 3-hydroxybutyrate) as an alternative fuel.</p>
<p>The ultimate origin of ketone bodies is the TAG stored in the white adipose tissue. As described above, low insulin level allows continuous lipolysis, resulting in release of fatty acids and glycerol into the bloodstream. Many fatty acid molecules are absorbed by adipocytes, in which they undergo <span class="math inline">\(\beta\)</span>-oxidation in mitochondria, producing acetyl-CoA in the liver. A part of acetyl-CoA is used for respiration, and the excess is converted either to acetoacetate in a 3-step sequence of reactions involving thiolase, HMG-CoA synthetase and HMG-CoA lyase or, with an additional reduction step, to 3-hydroxybutyrate (in mitochondria). Ketone bodies are released via the monocarboxylate transporter (MCT).</p>
<p>The ketone bodies can cross the blood-brain barrier and reach the neurons (via MCT). In the mitochondria of neurons, ketone bodies are converted back to acetyl-CoA, which is used for respiration. The large amount of acetyl-CoA obtained from ketone bodies inhibits pyruvate dehydrogenase complex and thus restricts glucose utilisation in much the same way acetyl-CoA derived from <span class="math inline">\(\beta\)</span>-oxidation in other tissues suppress glucose oxidation.</p>
<p>This gradual switching from glucose to ketone bodies as the fuel in the brain is important to the conservation of body proteins. As described previously, gluconeogenesis is the only source of glucose during starvation, and amino acids obtained from muscle proteolysis represent the majority of gluconeogenic precursors during starvation. Thus, conservation of body protein during starvation is equivalent to minimising the comsumption of glucose as fuel, specifically in the brain.</p>
<p>Prolonged starvation is accompanied by a gradual decrease in concentration of the active thyroid hormone, triiodothyronine (T3) over time, which in turn leads to a decrease in basal metabolic rate in most tissues. This reduces the overall consumption of energy of the body and thus also contributes to the sparing of muscle protein.</p>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p>During a period of starvation, the origin and identity of fuel molecules supplied to the brain (neurons) change, according to this order: astrocyte glycogenolysis, liver glycogenolysis, gluconeogenesis, ketone body synthesis. At the later stages, gluconeogenesis and ketone body synthesis coexist, but the amount of ketone body synthesis gradually increase, in order to spare the functional muscle proteins. This metabolic adaptation to starvation is largely under hormonal control, where insulin, glucagon, thyroid hormone and catecholamines play the most promiment roles.</p>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>