<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>3-Michaelmas - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="3-Michaelmas - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />









<link href="../term/3-michaelmas/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content term" role="main">
  

  <article class="article">
    
      <h1 class="article-title">3-Michaelmas</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="../tutorial/insertion-of-membrane-proteins.html" class="archive-item-link">
                Insertion of Membrane Proteins 
              </a>
              <span class="archive-item-date">
                2020-12-03
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon"><span class="toc-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</a></li>
<li><a href="#single-pass-proteins"><span class="toc-section-number">2</span> Single-Pass Proteins</a>
<ul>
<li><a href="#type-i-proteins"><span class="toc-section-number">2.1</span> Type I Proteins</a></li>
<li><a href="#type-ii-and-iii-proteins"><span class="toc-section-number">2.2</span> Type II and III Proteins</a></li>
</ul></li>
<li><a href="#multi-pass-proteins"><span class="toc-section-number">3</span> Multi-Pass Proteins</a></li>
<li><a href="#insertases"><span class="toc-section-number">4</span> Insertases</a>
<ul>
<li><a href="#insertion-of-tail-anchored-proteins-by-the-get-system"><span class="toc-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</a></li>
<li><a href="#the-roles-of-yidc"><span class="toc-section-number">4.2</span> The Roles of YidC</a></li>
</ul></li>
<li><a href="#beta-barrel-proteins"><span class="toc-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>The majority of integral membrane proteins in an eukaryotic cell are made of one or more <span class="math inline">\(\alpha\)</span>-helices and are first inserted co-translationally via the Sec61 translocon into the endoplasmic reticulum (ER) membrane, assembled there, and then dispatched, via the golgi apparatus and vesicles, to their final destinations. Bacteria use the SecYEG translocon, which is a homolog of Sec61, to insert membrane proteins into their plasma membrane. There are also translocons specific to chroloplasts (TIC-TOC), mitochondria (TIM-TOM) and peroxisomes (PEX), but this essay will focus on the Sec61 translocon.</p>
<div id="targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</h1>
<p>The common co-translationally inserted membrane proteins are targeted to the Sec61 translocon on the ER membrane via the cytosolic signal recognition particle (SRP) and the SRP receptor near the translocon.</p>
<p>SRP is a ribonucleoprotein particle made up of 6 proteins bound to a 300-nucleotide RNA acting as a scaffold. The M and Alu domains of SRP are critical for its functions. The M domain contains a cleft whose inner surface is lined by methionine and other hydrophobic side chains. This cleft binds to a hydrophobic segment of the nascent peptide as soon as it emerges from the exit tunnel. Meanwhile, the Alu domain blocks the elongation factor 2 (eEF2) binding site on the ribosome, pausing translation.</p>
<p>Usually it is the first hydrophobic segment that is recognised by SRP. This segment can either be a pure ER-targeting signal sequence or be the first TM helix itself (see next section). Recognition by SRP not only serves to target the ribosome-nascent protein to the ER but also prevents exposure of the hydrophobic segment to the aqueous environment.</p>
<p>SRP and SRP receptor each has a GTP binding site. When their GTP-bound forms associate with each other, they form two GTPase active sites. The GTPase activity is inhibited by signal peptide binding. Once the nascent chain has left SRP and enters the translocon, the two GTP molecules are hydrolysed, causing dissociation of SRP and its receptor. GTP replaces GDP on both proteins, and they are ready to initiate another round of ER targeting.</p>
<p>The Sec61 translocon is made up of 3 subunits: Sec61<span class="math inline">\(\alpha\)</span>, an integral membrane protein composed of two 5-helix bundles that constitutes the central channel and two smaller proteins, Sec61<span class="math inline">\(\beta\)</span> and Sec61<span class="math inline">\(\gamma\)</span>. The central channel through Sec61<span class="math inline">\(\alpha\)</span> is sealed by a plug made of a helical peptide that only opens during traslocation. The channel can also open laterally by hinging apart the two 5-helix bundles to expose a hydrophobic binding pocket for signal sequences and/or TM helices of the nascent peptide. The verticle opening allows elongation of the nascent peptide chain through the central pore, and the lateral opening allows attachment of signal sequences and exit of TM helices into the membrane (Figure <a href="#fig:sec61">1.1</a>).</p>
<div class="figure"><span id="fig:sec61"></span>
<img src="../img/sec61.png" alt="The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation." width="100%" />
<p class="caption">
Figure 1.1: The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation.
</p>
</div>
</div>
<div id="single-pass-proteins" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Single-Pass Proteins</h1>
<p>A single-pass membrane protein use a single 20-25 residue long <span class="math inline">\(\alpha\)</span>-helix made mainly of hydrophobic amino acids to traverse the membrane. The <span class="math inline">\(\alpha\)</span>-helical structure satisfies hydrogen bonding between main chain carbonyl and amino groups and exposes hydrophobic side chains, which form favourable hydrophobic interactions with the aliphatic core of the membrane bilayer. Co-translational insertion of this helix is a natural strategy to minimise unfavourable interation between the hydrophobic helix and the aqueous cytosolic environment and between the hydrophilic regions of the peptide surrounding the helix and the hydrophobic core of the bilayer.</p>
<p>Single-pass proteins are classified into three types according to the mechanism by which they are inserted. All of them are described in the following sections.</p>
<div id="type-i-proteins" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Type I Proteins</h2>
<p>Type I membrane proteins, like soluble proteins to be translocated into the ER lumen, contain an N-terminal ER-targeting signal sequence to be cleaved. The signal sequence is recognised by SRP, causing the ribosome-nascent peptide complex to be targeted to the Sec61 translocon. Sec61<span class="math inline">\(\alpha\)</span> opens the lateral gate, and the signal peptide fits into the exposed hydrophobic bindng pocket. The sequence following the signal peptide displaces the plug and inserts into the central channel, resulting in the conformation shown in Figure <a href="#fig:mem-insert-type-i">2.1</a>. Signal peptidase cleaves off the signal peptide once it recognises a specific sequence on the C-terminal end of the signal peptide. After cleavage, translation continues and the newly synthesised sequence is threaded through the channel and enters the ER lumen, until another hydrophobic segment is encountered. This segment is the TM helix of the nascent protein, and it acts as a <em>stop-transfer signal</em> (or “stop-transfer anchor sequence”) by triggering the lateral opening of Sec61<span class="math inline">\(\alpha\)</span> and thus allowing this helix to move into the membrane. The C-terminus continues to be synthesised and loops out on the cytosolic side of the membrane. Eventually, the hydrophic N- and C- termini are on the luminal and the cytosolic face, respectively.</p>
<div class="figure"><span id="fig:mem-insert-type-i"></span>
<img src="../img/mem-insert-type-i.png" alt="Insertion of type I membrane proteins." width="100%" />
<p class="caption">
Figure 2.1: Insertion of type I membrane proteins.
</p>
</div>
</div>
<div id="type-ii-and-iii-proteins" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Type II and III Proteins</h2>
<p>Type II and III proteins do not contain a cleavable signal sequence, and they have opposite orientations: the N-terminus is on the cytosolic and the luminal face in type II and type III proteins, respectively.</p>
<p>Type II and III proteins use the sequence of their TM helix as an ER-targeting signal (“start-transfer singal”, or “signal anchor sequence”), and the targeting is also mediated by SRP and SRP receptor. The distribution of positively charged residues around the helix dictates its orientation: the positively charged residues tend to remain on the cytosolic side of the membrane, thus in type II proteins where the N-terminal side has more positively charged residues, the helix adopts the orientation with the N-terminus facing the cytosolic side; and in type III proteins where the C-terminal side has more positively charged residues the opposite orientation is adopted (<br />
Figure <a href="#fig:mem-insert-type-iii-iii">2.2</a> ).</p>

<div class="figure"><span id="fig:mem-insert-type-iii-iii"></span>
<img src="../img/mem-insert-type-iii-iii.png" alt="Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61\(\alpha\) results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol." width="100%" />
<p class="caption">
Figure 2.2: Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61<span class="math inline">\(\alpha\)</span> results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol.
</p>
</div>
</div>
</div>
<div id="multi-pass-proteins" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Multi-Pass Proteins</h1>
<p>Multi-pass membrane proteins (a.k.a. type IV proteins) contain more than one TM <span class="math inline">\(\alpha\)</span> helices and traverse the membrane multiple times. Since the adjacent helices are in opposite orientations, insertion of multipass membrane proteins can be viewed as an alternation of insertion of a type I helix and insertion of a type II helix, which is dictated by the TM helix sequences that alternatingly act as start-transfer signals (signal anchor sequences) and stop-transfer signals (stop-transfer anchor sequences).</p>
<p>In multipass proteins with N-terminus in the cytosol (type IV-A), the TM helix sequence that first emerges from the ribosome (the one closest to the N-terminus) functions in the same way as the start transfer sequence (signal-anchor sequence) of the type II protein, which interacts with SRP to target the protein to the translocon, opens the translocon channel, and allow growing polypeptide chain to pass through the channel. Unlike type II proteins, as the chain grows, the translocon will encounter another hydrophobic TM helix sequence, which then functions as the stop-transfer signal (stop-transfer anchor sequence) in the same way as in type I proteins: the channel is closed, and this helix is moved via the lateral gate to the bilayer, and the nascent chain continues to grow into the cytosol. If a third TM helix sequence is encountered, it will function as another type II-like start-transfer signal, but this time the insertion does not depend on SRP and SRP receptor. This cycle continues until translation completes.</p>
<p>The insertion of multipass proteins with N-terminus in the ER lumen (type IV-B) occurs in a similar manner. The only difference is the first TM helix sequence, which now functions in the same way as the start-transfer sequence of the type III protein, resulting in a closed translocon and the growing peptide to be synthesised in the cytosol. Then, the second, third, fourth TM helix sequences functions in the same way as the first, second, and third TM helix sequences in a type IV-A protein and so on.</p>
<div class="figure"><span id="fig:type-iv-a-b"></span>
<img src="../img/type-iv-a-b.png" alt="Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall." width="100%" />
<p class="caption">
Figure 3.1: Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall.
</p>
</div>
</div>
<div id="insertases" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Insertases</h1>
<p>Some membrane proteins are inserted into the lipid bilayer via Sec-independent mechanisms, which are mediated by factors generally termed ‘insertases’. The bacterial (YidC), mitochondrial (Oxa1), and chrloroplast (Alb3) insertases are evolutionarily related, whereas the ER membrane contains 3 seemingly unrelated insertases: Get1/2 complex, the ER membrane-protein complex (EMC), and TMCO1.</p>
<div id="insertion-of-tail-anchored-proteins-by-the-get-system" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</h2>
<p>Tail-anchored proteins also use an <span class="math inline">\(\alpha\)</span>-helix to traverse the membrane, but this helix is located near the C-terminus. Because there are only few, if any, residues succeeding the C-terminal helix, translation terminates while the helix have not yet emerged from the exit tunnel, so recognition by SRP is not possible, and its insertion into ER membrane depends on a post-translational pathway involving Get1, Get2, and Get3 proteins.</p>
<p>In this pathway, the protein is completely synthesised and released into the cytosol. ATP-bound Get3 binds to the hydrophobic C-terminal tail of the protein (facilitated by some other proteins), and then docks onto the dimeric Get1/Get2 receptor on the ER membrane. Accompanying the hydrolysis of ATP by Get3, the Get1/Get2 complex facilitates the insertion of the tail into the ER membrane. Finally, ATP displaces ADP and Get3 is released back to the cytosol.</p>
</div>
<div id="the-roles-of-yidc" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> The Roles of YidC</h2>
<p>YidC contains 5 TM helices arranged to form a partially hydrophilic groove that is open towards both the lipid bilayer and the cytosol (Figure <a href="#fig:yidc">4.1</a>). Cross-linking experiments suggest that this groove operates as a binding site for TM helices (of the protein to be inserted). YidC alone can mediate insertion of a small subset of small (single- or double-pass) proteins in either co-translationally or post-translationally, and it is also suggested that YidC may act as a chaperone in conjunction with the Sec translocon. Specifically, it may shield the hydrophilic surface of TM helice that line polar cavities/channels in the final structure.</p>

<div class="figure"><span id="fig:yidc"></span>
<img src="../img/yidc.png" alt="The structure of Bacillus halodurans Yidc." width="100%" />
<p class="caption">
Figure 4.1: The structure of <em>Bacillus halodurans</em> Yidc.
</p>
</div>
</div>
</div>
<div id="beta-barrel-proteins" class="section level1" number="5">
<h1><span class="header-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</h1>
<p>In Gram-negative bacteria, intergral OM (outer membrane) <span class="math inline">\(\beta\)</span>-barrel proteins (OMPs) are translocated in an unfolded form across the IM (inner membrane), ferried to the OM via periplasmic chaperones, and integrated into the OM by the <span class="math inline">\(\beta\)</span>-barrel assembly machine (Bam) multiprotein complex.</p>
<p>In mitochondria, the precursors of nucleus-encoded <span class="math inline">\(\beta\)</span>-barrel proteins are transferred from the TOM complex, with the help of chaperones, to the SAM (sorting and assembly machinery) complex. The SAM complex, being homologous to the Bam complex, mediates the assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins in a similar manner (Figure <a href="#fig:insert-beta-barrel">5.1</a>).</p>

<div class="figure"><span id="fig:insert-beta-barrel"></span>
<img src="../img/insert-beta-barrel.png" alt="Insertion of \(\beta\)-barrel proteins into bacterial or mitochondrial outer membrane." width="100%" />
<p class="caption">
Figure 5.1: Insertion of <span class="math inline">\(\beta\)</span>-barrel proteins into bacterial or mitochondrial outer membrane.
</p>
</div>
<p>The exact mechanism of assembly and insertion is not fully understood, but a recent proposed mechanism is shown in Figure <a href="#fig:insert-beta-barrel-mech">5.2</a> <span class="citation">(Ricci and Silhavy 2019)</span>.</p>

<div class="figure"><span id="fig:insert-beta-barrel-mech"></span>
<img src="../img/insert-beta-barrel-mech.png" alt="Proposed Bam-assisted assembly and insertion of \(\beta\)-barrel proteins. (a) Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). (b) The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the \(\beta\)-1 and \(\beta\)-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. (c) The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, (d) Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. (e) The folded client protein is released into the membrane, and the Bam complex restores to its idle state." width="100%" />
<p class="caption">
Figure 5.2: <strong>Proposed Bam-assisted assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins.</strong> <strong>(a)</strong> Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). <strong>(b)</strong> The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the <span class="math inline">\(\beta\)</span>-1 and <span class="math inline">\(\beta\)</span>-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. <strong>(c)</strong> The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, <strong>(d)</strong> Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. <strong>(e)</strong> The folded client protein is released into the membrane, and the Bam complex restores to its idle state.
</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Cymer-2015">
<p>Cymer, Florian, Gunnar von Heijne, and Stephen H. White. 2015. “Mechanisms of Integral Membrane Protein Insertion and Folding.” <em>Journal of Molecular Biology</em> 427 (5): 999–1022. <a href="https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014">https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014</a>.</p>
</div>
<div id="ref-Guna-2018">
<p>Guna, Alina, and Ramanujan S. Hegde. n.d. “Transmembrane Domain Recognition During Membrane Protein Biogenesis and Quality Control.” <em>Current Biology</em> 28 (8): R498–R511. <a href="https://doi.org/10.1016/j.cub.2018.02.004">https://doi.org/10.1016/j.cub.2018.02.004</a>.</p>
</div>
<div id="ref-Lodish-2016">
<p>Lodish, Harvey F., Arnold Berk, Chris Kaiser, Monty Krieger, Anthony Bretscher, Hidde L. Ploegh, Angelika Amon, and Kelsey C Martin. 2016. <em>Molecular Cell Biology</em>. Book. 8th ed. New York: W.H. Freeman.</p>
</div>
<div id="ref-Ricci-2019">
<p>Ricci, Dante P., and Thomas J. Silhavy. 2019. “Outer Membrane Protein Insertion by the β-Barrel Assembly Machine.” In <em>Protein Secretion in Bacteria</em>, 91–101. American Society of Microbiology. <a href="https://www.asmscience.org/content/book/10.1128/9781683670285.chap8">https://www.asmscience.org/content/book/10.1128/9781683670285.chap8</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/protein-nucleic-acid-interactions.html" class="archive-item-link">
                Protein-Nucleic Acid Interactions 
              </a>
              <span class="archive-item-date">
                2020-11-27
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#an-overview-of-mechanisms-of-protein-dna-interactions"><span class="toc-section-number">2</span> An Overview of Mechanisms of Protein-DNA Interactions</a></li>
<li><a href="#thermodynamics-of-protein-dna-binding"><span class="toc-section-number">3</span> Thermodynamics of Protein-DNA Binding</a></li>
<li><a href="#base-readout-in-the-major-groove"><span class="toc-section-number">4</span> Base Readout in the Major Groove</a></li>
<li><a href="#sr"><span class="toc-section-number">5</span> Shape Readout</a>
<ul>
<li><a href="#ele"><span class="toc-section-number">5.1</span> Minor Groove Narrowing</a></li>
<li><a href="#kinks"><span class="toc-section-number">5.2</span> Kinks</a></li>
</ul></li>
<li><a href="#the-nucleosome-as-an-example-of-non-specific-binding"><span class="toc-section-number">6</span> The Nucleosome as an Example of Non-Specific Binding</a></li>
<li><a href="#question-2"><span class="toc-section-number">7</span> (Question 2)</a>
<ul>
<li><a href="#a-bhlh-motif"><span class="toc-section-number">7.1</span> (a) bHLH Motif</a></li>
<li><a href="#b-leucine-zipper"><span class="toc-section-number">7.2</span> (b) Leucine Zipper</a></li>
<li><a href="#c-zinc-finger"><span class="toc-section-number">7.3</span> (c) Zinc Finger</a></li>
<li><a href="#d-tata-box-binding-protein"><span class="toc-section-number">7.4</span> (d) TATA box binding protein</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Protein-DNA interactions have profound impact on the expression and regulation of genes. Many DNA-binding proteins, especially transcription factors, are able to recognise and bind to a specific short sequence of DNA, while others, typically those involved in routine tasks such as DNA damage repair (e.g. glycosylase) and DNA packaging (e.g. histone) show less specificity.</p>
</div>
<div id="an-overview-of-mechanisms-of-protein-dna-interactions" class="section level1" number="2">
<h1><span class="header-section-number">2</span> An Overview of Mechanisms of Protein-DNA Interactions</h1>
<p>Binding of proteins to DNA results from the favourable interactions between short DNA sequences and the amino acid side chains of specific DNA-binding motifs in proteins. In many DNA-binding proteins, the affinity and specificity of binding is also enhanced by flexible segments outside the globular core that mediate specific and nonspecific interactions but are sometimes not included in the definition of DNA binding domains. The majority of DNA-binding domains are made of either <span class="math inline">\(\alpha\)</span>-helices, <span class="math inline">\(\beta\)</span>-sheets, or both. While they can be classified into different types of motifs, any one type of motif can be used in multiple ways to interact with DNA, depending on the protein and the binding site.</p>
<p>There are multiple mechanisms by which specificity is achieved. First, specific DNA base pairs and amino acid side chains can establish shape complementarity, resulting in hydrogen bonds or favourble polar, electrostatic, or hydrophobic interactions. This is commonly referred to as <em>direct readout</em> or <em>base readout</em>. These interactions usually more frequently in the the more accessible major groove of DNA than in the minor groove. However, there is no simple one-to-one relationship between DNA and protein sequences, which means this mechanism itself is not sufficient to accout for the specificity. The second mechanism, <em>shape readout</em>, involves recognition of a sequence-dependent DNA shape, for example an overall bend, a segment of narrow minor groove, or a kinked base pair step.</p>
<p>Base readout and shape readout can be further devided into different modes. These mechanisms rarely exists in isolation, and protein-DNA recognition uses a continuum of mechanisms that depend on the structural features and flexibility of both DNA and protein.</p>
</div>
<div id="thermodynamics-of-protein-dna-binding" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Thermodynamics of Protein-DNA Binding</h1>
<p>All protein-DNA complexes can gain favourable <span class="math inline">\(\Delta H\)</span> from direct protein-DNA recognition interactions and favourable <span class="math inline">\(\Delta S\)</span> from water release. However, in systems with strongly distorted DNA, the loss of favourable interaction due to the molecular strain of base pair destacking is so large so that the net <span class="math inline">\(\Delta H\)</span> becomes unfavourable. This is compensated by a more positive <span class="math inline">\(\Delta S\)</span> as the strained interface experiences less immobilisation (Figure <a href="#fig:pnipni-thermodynamics">3.1</a>).</p>

<div class="figure"><span id="fig:pnipni-thermodynamics"></span>
<img src="../img/pnipni-thermodynamics.png" alt="The relative enthalphic and entropic contributions to binding free energy (\(\Delta G_\text{free}\)) of a few site-specific DNA-binding proteins with similar \(\Delta G_\text{free}\) (about -11.7 kcal/mol). Taken from Jen-Jacobson, Engler, and Jacobson (2000)" width="100%" />
<p class="caption">
Figure 3.1: The relative enthalphic and entropic contributions to binding free energy (<span class="math inline">\(\Delta G_\text{free}\)</span>) of a few site-specific DNA-binding proteins with similar <span class="math inline">\(\Delta G_\text{free}\)</span> (about -11.7 kcal/mol). Taken from <span class="citation">Jen-Jacobson, Engler, and Jacobson (2000)</span>
</p>
</div>
</div>
<div id="base-readout-in-the-major-groove" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Base Readout in the Major Groove</h1>
<p>The major groove is an ideal site for forming specific interactions between DNA bases and amino acid side chains because the four possible base pairs have a unique pattern of hydrogen bond donors and acceptors (and the 5-methyl of thymine) in the major groove. Proteins can also form hydrogen bonds with bases in the minor groove, but this contributes much less to the specificity (because the hydrogen bonding pattern cannot distinguish between AT from TA or CG from GC) (<a href="#fig:pni-major-minor-base">4.1</a>). Many DNA-binding motifs, including HTH, zinc finger and Leucine zipper, forms hydrogen bonds in the major groove.</p>
<div class="figure"><span id="fig:pni-major-minor-base"></span>
<img src="../img/pni-major-minor-base.png" alt="Hydrogen bonding patterns in the major and minor groove of AT and CG base pairs." width="100%" />
<p class="caption">
Figure 4.1: Hydrogen bonding patterns in the major and minor groove of AT and CG base pairs.
</p>
</div>
<p>Specificity not only depends on the number of contacts formed but also on the uniqueness of the hydrogen bonding geometry. Bidentate hydrogen bonds (2 H-bonds with different donors and receptors, which can be formed, for example, between the two NH/NH<sub>2</sub> groups and the O and N of guanine) have the highest degree of specificity followed by bifurcated hydrogen bonds (two H-bonds share one donor) and single hydrogen bonds.</p>
<p>Water molecules can be found at some protein-DNA binding sites. While water molecules can allow flexibility and mediate non-specific binding, they are also present in many specific protein-DNA compelxes, such as Trp repressor-DNA, where their positions are highly ordered and serve to bridge the hydrogen bonds.</p>
</div>
<div id="sr" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Shape Readout</h1>
<p>In physiological conditions, most DNA exists in its B form with well-defined geometries. However, due to the subtle differences between the chemical properties of the four bases, the DNA shape varies in a sequence-dependent manner, and this structural variation is important for protein-DNA recognition.</p>
<p>The B-DNA conformation is largely stabilised by the stacking energy between adjacent base pairs. Thus regions with weak stacking energy, which were found to be AT-AT and AT-TA stacks, have a stronger deformability (i.e. propensity to deviate from the B-DNA conformation). Specifically, DNA sequences of at least four consecutive A-T base pairs without an intervening TpA step are called “A-tracts”, which not only improves deformability but also has implications on electrostatic potentials, which are described in Section <a href="#ele">5.1</a></p>
<p>The following table summarises the relationship between several sequence elements and their impact on the structural property of DNA:</p>
<table>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th>Sequence Element</th>
<th>Structural Property</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>A-tract</td>
<td>narrow minor groove, bending, rigid for <span class="math inline">\(\ge\)</span> bp</td>
</tr>
<tr class="even">
<td>TATA box</td>
<td>high deformability</td>
</tr>
<tr class="odd">
<td>YpR step (especially TpA)</td>
<td>compresses major groove, high deformability, ‘hinge’ step, kinking</td>
</tr>
<tr class="even">
<td>RpY step</td>
<td>compresses minor groove, low deformability</td>
</tr>
</tbody>
</table>
<div id="ele" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Minor Groove Narrowing</h2>
<p><span class="citation">Rohs et al. (2009)</span> reported that the binding of arginine residues to narrow (<span class="math inline">\(&lt; 5\)</span> angstroms) minor grooves is a common mode of protein-DNA recognition (Figure <a href="#fig:pni-narrow-minor">5.1</a>). These narrow groove are usually associates with A-tracts, which estabilish a connection between DNA sequence and shape. Arginines often insert into the minor groove as part of short sequence motifs that vary among different proteins, e.g. Arg-Gln-Arg in the Hox protein SCR, Arg-Lys-Lys-Arg in POU homeodomains, thus providing specificity.</p>
<p>The narrowing results in a more negative electrostatic potential in the minor groove, which promotes the binding of positively charged arginine. The preference of arginine over lysine can be explained by the greater energetic cost of removing a charged lysine from water due to lysine’s smaller radius of the charged group (or greater charge density) compared to arginine.</p>

<div class="figure"><span id="fig:pni-narrow-minor"></span>
<img src="../img/pni-narrow-minor.png" alt="Amino acid frequencies in minor grooves. Taken from Rohs et al. (2009)" width="100%" />
<p class="caption">
Figure 5.1: Amino acid frequencies in minor grooves. Taken from <span class="citation">Rohs et al. (2009)</span>
</p>
</div>
</div>
<div id="kinks" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Kinks</h2>
<p>YpR steps (especially TpA steps) have a strong tendency to form kinks that disrupt the linearity of the double helix. They can contribute to binding specificity by optimising protein-DNA contacts. The binding site of the catabolite activator protein (CAP), for example, shows dramatic kinks at two CpA steps, which along with two additional smaller kinks cause an overall bending of the DNA of about 90 degrees around the protein.</p>
</div>
</div>
<div id="the-nucleosome-as-an-example-of-non-specific-binding" class="section level1" number="6">
<h1><span class="header-section-number">6</span> The Nucleosome as an Example of Non-Specific Binding</h1>
<p>Histones are the ubiquitous nuclear protein that mainly serve to compact DNA. A nucleosome is formed by 147 base pairs of DNA wrapping around a histone octamer, and adjacent nucleosomes are joined by a linker sequence (about 50 bp long in humans). Since nucleosome are so prevalent, the histone-DNA interaction is often considered as non-specific, and its association with DNA is thought to be due to the electrostatic attraction between the abundant lysine and arginine residues on the surface of histoens and the negatively charged phosphate backbone of DNA. However, histones have preferred DNA regions to bind to.</p>
<p>Nucleosomes are often formed in regions enriched with A-tracts for two reasons. First, A-tracts offer high deformability, which is required to bend DNA and wrap it around histone octamers. Second, histones are enriched with arginine residues, which recognise and penetrateinto the narrow minor grooves formed by A-tracts, thus providing important stabilising interaction (Section <a href="#ele">5.1</a>).</p>
</div>
<div id="question-2" class="section level1" number="7">
<h1><span class="header-section-number">7</span> (Question 2)</h1>
<div id="a-bhlh-motif" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> (a) bHLH Motif</h2>
<p>The basic helix-loop-helix (bHLH) motif consists of two <span class="math inline">\(\alpha\)</span>-helices separated by a loop. One of the helices (usually the longer one) contain basic residues (arginine and lysine) that bind to the major groove of DNA. The flexible loop and the other helix are involved in dimerisation. Many bHTH proteins occur as heterodimers (e.g. Myc/MAX), and their activity is hightly regulated by the dimerisation of subunits.</p>
</div>
<div id="b-leucine-zipper" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> (b) Leucine Zipper</h2>
<p>The leucine zipper motif consists of a dimer of <span class="math inline">\(\alpha\)</span>-helices. The dimerisation is driven by interactions between the hydrophobic side chains that cover their inner surfaces. Specifically, each helix has a periodic repetition of leucine residues at every seventh position. Since every turn in an <span class="math inline">\(\alpha\)</span>-helix contains 3.6 amino acids—two turns contain 7.2, which is slightly more than 7, the helices coil around one another in a left-handed sense. The N-terminal DNA-binding domains of each helix protrudes into the major groove of DNA, which together recognise a 8-bp long sequence.</p>
</div>
<div id="c-zinc-finger" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> (c) Zinc Finger</h2>
<p>The zinc finger contains a short <span class="math inline">\(\alpha\)</span>-helix, a two-stranded antiparallel <span class="math inline">\(\beta\)</span>-sheet, and a Zn<sup>2+</sup> ion coordinated by cysteine and histine residues. The zinc ion serves to stabilise the overall structure, while the helix make contact with DNA’s major groove. A typical zinc finger protein, e.g. Zif268, contains a chain of 3 zinc-finger modules that coil in a right-handed sense, so as to follow the curve within the major groove. Each finger recognises 3 base pairs, and thus a protein with 3 zinc finger domains recognise a continuous sequence of 9 base pairs.</p>
</div>
<div id="d-tata-box-binding-protein" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> (d) TATA box binding protein</h2>
<p>TATA box binding proteins (TBP) use a ten-stranded <span class="math inline">\(\beta\)</span>-sheet to recognise DNA by binding in the minor groove. Insertion of the concave <span class="math inline">\(\beta\)</span>-sheet into the groove requires substantial DNA distortion. The flexibility intrinsic to TpA steps (Section <a href="#sr">5</a>) in the TATA sequence faciliates formation of kinks when TBP binds. In addition, yeast TBP-TATA structure shows that the kinks in the first and last base pair step (<strong>TA</strong>TATA<strong>AA</strong>) are stabilised through intercalations with phenylalanine residues.</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Calladine-2004">
<p>Calladine, Chris R., Horace Drew, Ben Luisi, and Andrew Travers. 2004. <em>Understanding Dna-the Molecule and How It Works</em>. 3rd ed. Academic Press. <a href="http://gen.lib.rus.ec/book/index.php?md5=c6b8d89cd8868255bc9ebd4cf51e1917">http://gen.lib.rus.ec/book/index.php?md5=c6b8d89cd8868255bc9ebd4cf51e1917</a>.</p>
</div>
<div id="ref-Jen-Jacobson-2000">
<p>Jen-Jacobson, Linda, Lisa E. Engler, and Lewis A. Jacobson. 2000. “Structural and Thermodynamic Strategies for Site-Specific Dna Binding Proteins.” <em>Structure</em> 8 (10): 1015–23. <a href="https://doi.org/https://doi.org/10.1016/S0969-2126(00)00501-3">https://doi.org/https://doi.org/10.1016/S0969-2126(00)00501-3</a>.</p>
</div>
<div id="ref-Rohs-2009">
<p>Rohs, Remo, Sean M. West, Alona Sosinsky, Peng Liu, Richard S. Mann, and Barry Honig. 2009. “The Role of Dna Shape in Protein–Dna Recognition.” <em>Nature</em> 461 (7268): 1248–53. <a href="https://doi.org/10.1038/nature08473">https://doi.org/10.1038/nature08473</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/metabolic-modelling.html" class="archive-item-link">
                Metabolic Modelling 
              </a>
              <span class="archive-item-date">
                2020-11-26
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#the-haldane-relationship"><span class="toc-section-number">1</span> The Haldane Relationship</a></li>
<li><a href="#control-points-in-a-simple-linear-pathway"><span class="toc-section-number">2</span> Control Points in A Simple Linear Pathway</a></li>
<li><a href="#linear-pathway-with-negative-feedback"><span class="toc-section-number">3</span> Linear Pathway with Negative Feedback</a>
<ul>
<li><a href="#re-analyse-the-effect-of-k_i-with-constant-flux"><span class="toc-section-number">3.1</span> Re-analyse the effect of <span class="math inline">\(K_i\)</span> with constant flux</a>
<ul>
<li><a href="#repeat-with-model-212"><span class="toc-section-number">3.1.1</span> Repeat with Model 212</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<ul>
<li><strong>Model 112 was used.</strong></li>
</ul>
<div id="the-haldane-relationship" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Haldane Relationship</h1>
<p>Using the “Parameter Scan” utility in Copasi (Figure <a href="#fig:copasi-q1-scan">1.1</a>), I collected steady state concentrations of ‘A’ and ‘B’ in <span class="math inline">\(5^4 = 625\)</span> combinations of <span class="math inline">\(V_{max(f)}\)</span>, <span class="math inline">\(K_{m(p)}\)</span>, <span class="math inline">\(V_{max(r)}\)</span>, and <span class="math inline">\(K_{m(s)}\)</span>, where each parameter vary from <span class="math inline">\(1\times10^{-2}\)</span> to <span class="math inline">\(1\times10^{2}\)</span> in a logarithmic scale.</p>
<div class="figure"><span id="fig:copasi-q1-scan"></span>
<img src="../img/copasi-q1-scan.png" alt="&quot;Parameter Scan&quot; setup" width="100%" />
<p class="caption">
Figure 1.1: “Parameter Scan” setup
</p>
</div>
<p>Here I read the data, and add two columns, where <code>keq</code> represents the experimental <span class="math inline">\(K_\text{eq}\)</span>:</p>
<p><span class="math inline">\(K_\text{eq} = \dfrac{\text{[B]}}{\text{[A]}}\)</span></p>
<p>and <code>keq_calc</code> represents the <span class="math inline">\(K_\text{eq}\)</span> calculated from the Haldane relationship:</p>
<p><span class="math inline">\(K_\text{eq} = \dfrac{V_{max(f)}\cdot K_{m(p)}}{V_{max(r)}\cdot K_{m(s)}}\)</span></p>
<pre class="r"><code>q1 &lt;- read_tsv(&#39;metabolic modelling/q1.txt&#39;, col_names = c(&#39;vf&#39;, &#39;kmp&#39;, &#39;vr&#39;, &#39;kms&#39;, &#39;A&#39;, &#39;B&#39;))
q1 &lt;- q1 %&gt;% mutate(
  keq = B/A,
  keq_calc = (vf * kmp) / (vr * kms),
  keq_diff = keq - keq_calc
)
q1</code></pre>
<pre><code>## # A tibble: 625 x 9
##       vf   kmp     vr   kms     A        B      keq keq_calc  keq_diff
##    &lt;dbl&gt; &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt; &lt;dbl&gt;    &lt;dbl&gt;    &lt;dbl&gt;    &lt;dbl&gt;     &lt;dbl&gt;
##  1  0.01  0.01   0.01  0.01 1     1         1         1       0.      
##  2  0.01  0.01   0.1   0.01 1.82  0.182     0.1       0.1     0.      
##  3  0.01  0.01   1     0.01 1.98  0.0198    0.01      0.01    0.      
##  4  0.01  0.01  10     0.01 2.00  0.00200   0.001     0.001   0.      
##  5  0.01  0.01 100     0.01 2.00  0.000200  0.00010   0.0001 -1.36e-20
##  6  0.1   0.01   0.01  0.01 0.182 1.82     10.0      10      -1.78e-15
##  7  0.1   0.01   0.1   0.01 1     1         1         1       0.      
##  8  0.1   0.01   1     0.01 1.82  0.182     0.1       0.1     0.      
##  9  0.1   0.01  10     0.01 1.98  0.0198    0.01      0.01    0.      
## 10  0.1   0.01 100     0.01 2.00  0.00200   0.001     0.001   0.      
## # … with 615 more rows</code></pre>
<p>Plotting <code>keq_calc</code> against <code>keq</code> shows that <span class="math inline">\(K_\text{eq}\)</span> calculated in these two ways are equal, as the linear regression line has a gradient of 1 and passes through the origin.</p>
<pre class="r"><code>q1 %&gt;% ggplot(aes(keq, keq_calc)) +
  geom_point() +
  geom_smooth()+
  scale_x_log10()+
  scale_y_log10()</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<pre class="r"><code>mod &lt;- lm(q1$keq ~ q1$keq_calc)
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = q1$keq ~ q1$keq_calc)
## 
## Residuals:
##        Min         1Q     Median         3Q        Max 
## -1.148e-08 -4.480e-10 -4.480e-10 -4.190e-10  2.880e-07 
## 
## Coefficients:
##              Estimate Std. Error   t value Pr(&gt;|t|)    
## (Intercept) 7.078e-10  4.630e-10 1.529e+00    0.127    
## q1$keq_calc 1.000e+00  1.135e-16 8.814e+15   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 1.156e-08 on 623 degrees of freedom
## Multiple R-squared:      1,  Adjusted R-squared:      1 
## F-statistic: 7.769e+31 on 1 and 623 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>The Haldane relationship shows that <span class="math inline">\(K_\text{eq}\)</span> is proportional to <span class="math inline">\(V_{max(f)}\)</span> and <span class="math inline">\(K_{m(p)}\)</span>, and inversely proportional to <span class="math inline">\(V_{max(r)}\)</span> and <span class="math inline">\(K_{m(s)}\)</span>, and this can be illustrated using a facetted heatmap:</p>

<pre class="r"><code>q1 %&gt;% ggplot(aes(x = vf, y = kmp, fill = log10(keq))) +
  geom_tile()+
  facet_grid((1/vr)~(1/kms), as.table = FALSE, labeller = label_both)+
  scale_x_log10(labels = plain, expand = c(0, 0))+
  scale_y_log10(labels = plain, expand = c(0, 0))+
  scale_fill_gradient2(low = &#39;blue&#39;, mid = &#39;yellow&#39;, high = &#39;red&#39;)+
  labs(title = &quot;Variation of Keq with vf, kmp, vr, and kms&quot;)+
  theme(axis.text.x = element_text(angle = 90))</code></pre>
<div class="figure"><span id="fig:q1"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q1-1.png" alt="Variation of \(K_\text{eq}\) with \(V_{max(f)}\), \(K_{m(p)}\), \(V_{max(r)}\), and \(K_{m(s)}\)" width="672" />
<p class="caption">
Figure 1.2: Variation of <span class="math inline">\(K_\text{eq}\)</span> with <span class="math inline">\(V_{max(f)}\)</span>, <span class="math inline">\(K_{m(p)}\)</span>, <span class="math inline">\(V_{max(r)}\)</span>, and <span class="math inline">\(K_{m(s)}\)</span>
</p>
</div>
<p>The pattern in each grid shows that <span class="math inline">\(K_\text{eq}\)</span> is proportional to <span class="math inline">\(V_{max(f)}\)</span> and <span class="math inline">\(K_{m(p)}\)</span>, and the pattern across the grids shows that it is inversely proportional to <span class="math inline">\(V_{max(r)}\)</span> and <span class="math inline">\(K_{m(s)}\)</span>.</p>
</div>
<div id="control-points-in-a-simple-linear-pathway" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Control Points in A Simple Linear Pathway</h1>
<p>I use the “Parameter Scan” function to vary the <span class="math inline">\(K_\text{eq}\)</span> of reaction 3 (that catalyses the reversible conversion between C and D) in the range <span class="math inline">\(10^{-6}\)</span> to <span class="math inline">\(10^6\)</span> in a logarithmic scale (13 samples). The variables being recorded are <span class="math inline">\(K_\text{eq}\)</span> and <span class="math inline">\(J\)</span> (flux control coefficient) of reaction 3, and the concentrations of C and D. Then, the reaction quotient, <span class="math inline">\(Q\)</span> (a.k.a. mass action ratio), of each row is calculated as:</p>
<p><span class="math display">\[Q = \dfrac{\text{[D]}}{\text{[C]}}\]</span></p>
<pre class="r"><code>q2_scan_r3 &lt;- read_tsv(&#39;metabolic modelling/Q2.txt&#39;)
q2_scan_r3 &lt;- q2_scan_r3 %&gt;% mutate(q = d/c)
q2_scan_r3</code></pre>
<pre><code>## # A tibble: 13 x 5
##            j     keq     c         d          q
##        &lt;dbl&gt;   &lt;dbl&gt; &lt;dbl&gt;     &lt;dbl&gt;      &lt;dbl&gt;
##  1 0.0000489 1.00e-6 100.   0.000100 0.00000100
##  2 0.0000489 1.00e-5 100    0.001    0.00001   
##  3 0.00470   1.00e-4  99.1  0.00987  0.0000996 
##  4 0.0353    1.00e-3  92.2  0.0885   0.000960  
##  5 0.119     1.00e-2  69.0  0.556    0.00806   
##  6 0.210     1.00e-1  44.5  2.35     0.0528    
##  7 0.309     1.00e+0  32.0  6.77     0.212     
##  8 0.369     1.00e+1  28.6 10.0      0.350     
##  9 0.380     1.00e+2  28.2 10.6      0.378     
## 10 0.381     1.00e+3  28.1 10.7      0.381     
## 11 0.381     1.00e+4  28.1 10.7      0.381     
## 12 0.381     1.00e+5  28.1 10.7      0.381     
## 13 0.381     1.00e+6  28.1 10.7      0.381</code></pre>
<p>Figure <a href="#fig:q2-keq">2.1</a> shows the variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with its <span class="math inline">\(K_\text{eq}\)</span>. The plot shows that a high <span class="math inline">\(K_\text{eq}\)</span>, i.e. high irreversibility, is correlated with a high <span class="math inline">\(J\)</span>, and in the intermediate range <span class="math inline">\(J\)</span> varies linearly with <span class="math inline">\(\ln(K_\text{eq})\)</span>, i.e. varies linearly with <span class="math inline">\(\Delta G = -RT\ln(K_\text{eq})\)</span></p>

<pre class="r"><code>q2_scan_r3 %&gt;% ggplot(aes(log(keq), j)) +
  geom_point()</code></pre>
<div class="figure"><span id="fig:q2-keq"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2-keq-1.png" alt="Variation of the flux control coefficient, \(J\), of reaction 3, with its \(K_\text{eq}\)" width="672" />
<p class="caption">
Figure 2.1: Variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with its <span class="math inline">\(K_\text{eq}\)</span>
</p>
</div>
<p>Figure <a href="#fig:q2-displacement">2.2</a> shows the variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3 with <span class="math inline">\(Q/K_\text{eq}\)</span>, which is a measure of displacement of the reaction from the equilibrium. A <span class="math inline">\(Q/K_\text{eq}\)</span> close to 1 indicates the reaction is close to equilibrium.
The plot shows that when reaction is further displaced from the equilibrium, the its flux control coefficient is higher.</p>

<pre class="r"><code>q2_scan_r3 %&gt;% ggplot(aes(q/keq, j)) +
  geom_point()+
  geom_smooth(method = &#39;lm&#39;, size = 0.5)</code></pre>
<div class="figure"><span id="fig:q2-displacement"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2-displacement-1.png" alt="Variation of the flux control coefficient, \(J\), of reaction 3, with \(Q/K_\text{eq}\)" width="672" />
<p class="caption">
Figure 2.2: Variation of the flux control coefficient, <span class="math inline">\(J\)</span>, of reaction 3, with <span class="math inline">\(Q/K_\text{eq}\)</span>
</p>
</div>
<p>A linear regression analysis shows that there is a strong linear correlation between <span class="math inline">\(J\)</span> and <span class="math inline">\(Q/K_\text{eq}\)</span>, with <span class="math inline">\(p = 4.02\times10^{-6} &lt; 10^{-5}\)</span></p>
<pre class="r"><code>mod &lt;- with(q2_scan_r3, lm(j ~ q/keq))
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = j ~ q/keq)
## 
## Residuals:
##       Min        1Q    Median        3Q       Max 
## -0.054438 -0.019991 -0.012931  0.001452  0.108459 
## 
## Coefficients:
##               Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)  5.448e-02  2.221e-02   2.453   0.0341 *  
## q            8.960e-01  9.010e-02   9.945 1.67e-06 ***
## q:keq       -4.196e-08  1.569e-07  -0.267   0.7946    
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.05374 on 10 degrees of freedom
## Multiple R-squared:  0.9167, Adjusted R-squared:    0.9 
## F-statistic:    55 on 2 and 10 DF,  p-value: 4.02e-06</code></pre>
<p>In order to vary <span class="math inline">\(Q/K_\text{eq}\)</span> without directly varying <span class="math inline">\(K_\text{eq}\)</span>, and to see the effect not only on reaction 3 but also on all other reactions, I vary <span class="math inline">\(V_f\)</span>, not <span class="math inline">\(K_\text{eq}\)</span>, of reaction 3 from <span class="math inline">\(10^{-6}\)</span> to <span class="math inline">\(10^6\)</span> with 1000 intervals, and collected the flux control coefficients of all 6 reactions, the <span class="math inline">\(K_\text{eq}\)</span> of reaction 3, as well as concentrations of all species, which are then used to calculate the mass action ratio of each reaction. Then, the displacement of each reaction is calculated. The resulting dataframe contains three columns: 1) the reaction number; 2) the flux control coefficient and 3) displacement from equilibrium of this reaction</p>
<pre class="r"><code>q2_scan_r3_vf &lt;- read_tsv(&#39;metabolic modelling/Q2-1.txt&#39;)
q2_scan_r3_vf &lt;- q2_scan_r3_vf %&gt;% 
  mutate(
    d1 = (B/A) / 10,
    d2 = (C/B) / 1,
    d3 = (D/C) / 0.5,
    d4 = (E/D) / 5,
    d5 = (F/E) / 2,
    d6 = (G/F) / 10,
    c1 = A + B,
    c2 = B + C,
    c3 = C + D,
    c4 = D + E,
    c5 = E + F,
    c6 = F + G,
  ) %&gt;% select(!(1:7))
q2_scan_r3_vf_cleaned &lt;- tibble(
  reaction = integer(),
  displacement = double(),
  j = double()
  )
for (i in 1:6) {
  q2_scan_r3_vf_cleaned &lt;- add_row(
    q2_scan_r3_vf_cleaned, 
    reaction = i, j = q2_scan_r3_vf[[paste0(&#39;j&#39;, i)]], displacement = q2_scan_r3_vf[[paste0(&#39;d&#39;, i)]])
}
q2_scan_r3_vf_cleaned </code></pre>
<pre><code>## # A tibble: 6,006 x 3
##    reaction displacement            j
##       &lt;int&gt;        &lt;dbl&gt;        &lt;dbl&gt;
##  1        1         1.00 0.0000000343
##  2        1         1.00 0.0000000353
##  3        1         1.00 0.0000000363
##  4        1         1.00 0.0000000373
##  5        1         1.00 0.0000000383
##  6        1         1.00 0.0000000394
##  7        1         1.00 0.0000000405
##  8        1         1.00 0.0000000416
##  9        1         1.00 0.0000000428
## 10        1         1.00 0.0000000440
## # … with 5,996 more rows</code></pre>
<p>Then, for all reactions, <span class="math inline">\(J\)</span> versus displacement from equilibrium is plotted:</p>
<pre class="r"><code>q2_scan_r3_vf_cleaned %&gt;% ggplot(aes(displacement, j)) + 
  geom_point()+
  facet_wrap(~reaction, scales = &#39;free&#39;, labeller = label_both)+
  xlab(&quot;MAR/Keq&quot;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q2_2-1.png" width="672" /></p>
<p>The plots shows that, when disturbing <span class="math inline">\(V_{max(f)}\)</span> of reaction 3, the disequilibrium ratio of all reactions are also altered. Not only reaction 3 but also all other reactions follow the rule that, the flux control coefficent of a reaction increases with the extent of displacement from the equilibrium. However, it is hard to predict <span class="math inline">\(J\)</span> given only the value of <span class="math inline">\(Q/K_\text{eq}\)</span>, as different reactions show different patterns of variations, so <span class="math inline">\(Q/K_\text{eq}\)</span> (displacement from equilibrium) is not a robust indicator of <span class="math inline">\(J\)</span>.</p>
</div>
<div id="linear-pathway-with-negative-feedback" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Linear Pathway with Negative Feedback</h1>
<p>I chose reaction 2 to be the one sensitive to the inhibitor, and varied <span class="math inline">\(K_i\)</span> from <span class="math inline">\(10^{-18}\)</span> to <span class="math inline">\(10^{18}\)</span> with 1000 intervals, and recorded it along with flux control coefficients of all reactions. The results are shown in Figure <a href="#fig:q3">3.1</a>.</p>

<pre class="r"><code>q3 &lt;- read_tsv(&#39;metabolic modelling/Q3.txt&#39;)
q3 &lt;- q3 %&gt;% gather(reaction, &quot;j&quot;, -ki)
q3 %&gt;% ggplot(aes(ki, j)) +
  geom_point()+
  facet_wrap(~reaction, labeller = label_both)+
  scale_x_log10()</code></pre>
<div class="figure"><span id="fig:q3"></span>
<img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/q3-1.png" alt="Effect of changing \(K_i\) of reaction 2 on the flux control coefficient of all reactions. Lower \(K_i\) means higher binding affinity to the inhibitor." width="672" />
<p class="caption">
Figure 3.1: Effect of changing <span class="math inline">\(K_i\)</span> of reaction 2 on the flux control coefficient of all reactions. Lower <span class="math inline">\(K_i\)</span> means higher binding affinity to the inhibitor.
</p>
</div>
<p>The plots show that, the flux control coefficient (<span class="math inline">\(J\)</span>) of reaction 2 increases as the binding affinity (i.e. sensitivity) to the inhibitor of the enzyme involved in this step increases (i.e. as <span class="math inline">\(K_i\)</span> decreases). <span class="math inline">\(J\)</span> of the upsteam reaction 1 also inceases slightly. For all downstream reactions, <span class="math inline">\(J\)</span> decreases.</p>
<div id="re-analyse-the-effect-of-k_i-with-constant-flux" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Re-analyse the effect of <span class="math inline">\(K_i\)</span> with constant flux</h2>
<p>The flux and values of <span class="math inline">\(J\)</span> when <span class="math inline">\(K_\text{i} = 10^{-18}\)</span> and when <span class="math inline">\(K_\text{i} = 10^18\)</span> are shown below:</p>
<pre class="r"><code>ki_low = 1e-18
flux_low = 0.476572
j_low = c(0.203839, 0.717872,   0.0342001,  0.0340061,  0.00278022, 0.007302)
ki_high = 1e18
flux_high = 1.07393
j_high = c(0.110937,    0.0822319,  0.278976,   0.319753,   0.0478304,  0.160271)</code></pre>
<p><span class="math inline">\(V_f\)</span> of reaction 2 are optimised so that the flux when <span class="math inline">\(K_\text{i} = 10^{-18}\)</span> is 1.07393 (the same as when <span class="math inline">\(K_\text{i} = 10^{18}\)</span>)</p>
<pre><code>Optimization Result:

    Objective Function Value:   1.07393
    Function Evaluations:   248
    CPU Time [s]:   0.061
    Evaluations/Second [1/s]:   4065.57

    (R2).Vf: 45.0229</code></pre>
<div class="figure"><span id="fig:copasi-q32"></span>
<img src="../img/copasi-q32.png" alt="Adjusting Vf" width="100%" />
<p class="caption">
Figure 3.2: Adjusting Vf
</p>
</div>
<p>I adjusted the <span class="math inline">\(V_f\)</span> of reaction 2 from 5 to 45.0229 (<a href="#fig:copasi-q32">3.2</a>), verified that the flux is 1.07393 (the same as in the state with negligible inhibition), and the values of <span class="math inline">\(J\)</span> are:</p>
<pre class="r"><code>j_low &lt;-    c(0.144048, 0.0858949,  0.266261,   0.30518,    0.0456503,  0.152966)</code></pre>
<p>which can be directly compared to the <span class="math inline">\(J\)</span> values in the uninhibited state:</p>
<pre class="r"><code>tibble(
  ki =c(rep(&quot;low (1e-18)&quot;, 6), rep(&quot;high (1e18)&quot;, 6)),
  reaction = rep(as.character(1:6), 2),
  j = c(j_low, j_high)
) %&gt;% ggplot(aes(reaction, j, fill = ki)) +
  geom_col(position = &#39;dodge&#39;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
<p>The plot shows that, when the flux is made constant, a lower <span class="math inline">\(K_i\)</span> (higher affinity of inhibitor binding) increases the flux control coefficent of reaction 2 and the upstream reaction 1, and decreases that of downstream reactions, which is consistent with the previous experiment. However, the amount of change is not as much as previously modelled.</p>
<div id="repeat-with-model-212" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Repeat with Model 212</h3>
<p>I repeat the last analysis with model 212, this time choosing reaction 1 as the one to be affected by the inhibitor.</p>
<pre class="r"><code>ki_low = 1e-16
j_low = c(0.982287, 0.0109131,  0.00433655, 0.00237021, 6.9396e-05, 2.40539e-05)
flux_low = 0.091902
ki_high = 1e16
flux_high = 0.807846
j_high = c(0.437942,    0.31832,    0.104533,   0.12481,    0.0105218,  0.00387277)</code></pre>
<pre><code>Optimization Result:

    Objective Function Value:   0.807846
    Function Evaluations:   82
    CPU Time [s]:   0.018
    Evaluations/Second [1/s]:   4555.56

    (R1).Vf: 25.7611</code></pre>
<p>I adjusted the <span class="math inline">\(V_f\)</span> of reaction 1 from 5 to 25.7611 (<a href="#fig:copasi-q32">3.2</a>), verified that the flux is 0.807846 (the same as in the state with negligible inhibition), and the values of <span class="math inline">\(J\)</span> are:</p>
<pre class="r"><code>j_low &lt;-    c(0.479732, 0.294652,   0.0967611,  0.11553,    0.00973947, 0.00358483)</code></pre>
<p>comparing to the <span class="math inline">\(J\)</span> values in the uninhibited state:</p>
<pre class="r"><code>tibble(
  ki =c(rep(&quot;low (1e-16)&quot;, 6), rep(&quot;high (1e16)&quot;, 6)),
  reaction = rep(as.character(1:6), 2),
  j = c(j_low, j_high)
) %&gt;% ggplot(aes(reaction, j, fill = ki)) +
  geom_col(position = &#39;dodge&#39;)</code></pre>
<p><img src="../lab/2020-11-26-metabolic-modelling_files/figure-html/unnamed-chunk-12-1.png" width="672" /></p>
<p>This time only reaction 1 has an increased <span class="math inline">\(J\)</span>, and all other downstream reactions have an lowered <span class="math inline">\(J\)</span>, which is consistent with previous observations.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/gene-expression-practical-paper-6-questions.html" class="archive-item-link">
                Gene Expression Practical Paper 6 Questions 
              </a>
              <span class="archive-item-date">
                2020-11-22
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<p><strong>(a) Assign each lac mutation to one of the strains 1 to 5. Explain your reasoning in each case.</strong></p>
<table>
<thead>
<tr class="header">
<th></th>
<th align="center">1</th>
<th align="center">2</th>
<th align="center">3</th>
<th align="center">4</th>
<th align="center">5</th>
<th align="center">WT</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>glycerol</td>
<td align="center">+</td>
<td align="center">−</td>
<td align="center">+</td>
<td align="center">−</td>
<td align="center">−</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td>glycerol+IPTG</td>
<td align="center">+</td>
<td align="center">−</td>
<td align="center">+</td>
<td align="center">+</td>
<td align="center">−</td>
<td align="center">+</td>
</tr>
<tr class="odd">
<td>glucose+IPTG</td>
<td align="center">−</td>
<td align="center">−</td>
<td align="center">−</td>
<td align="center">+</td>
<td align="center">−</td>
<td align="center">-</td>
</tr>
<tr class="even">
<td>×WT hybrid + glycerol</td>
<td align="center">IPTG</td>
<td align="center">IPTG</td>
<td align="center">+</td>
<td align="center">IPTG</td>
<td align="center">IPTG</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td>×lacY<sup>−</sup> hybrid</td>
<td align="center"></td>
<td align="center">IPTG</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">×</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<ul>
<li>Strain 1: <em>lacI<sup>-</sup></em></li>
<li>Strain 2: <em>lacZ<sup>-</sup></em></li>
<li>Strain 3: <em>lacO<sup>c</sup></em></li>
<li>Strain 4: mutation that causes RNAP to bind more easily</li>
<li>Strain 5: <em>lacP<sup>-</sup></em></li>
</ul>
<p>Strains 1 and 3 are both sensitive to catabolite repression (by glucose) but insensitive to induction by IPTG (transcription occurs as long as glucose is not present), which can be either due to <em>lacI<sup>-</sup></em> or <em>lacO<sup>c</sup></em>. In the diploid with WT, strain 1 becomes sensitive to IPTG, which means strain 1 had a <em>lacI<sup>-</sup></em> mutation, and the <em>lacI<sup>-</sup></em> of the newly introduced WT allele produces LacI proteins which are effective on both <em>lac</em> operons. Strain 3 has a <em>lacO<sup>c</sup></em> mutation, thus this operon is continuously being transcribed regardless of the newly introduced WT allele (LacI is present, but this mutant operon is insensitive to it)</p>
<p>In the last experiment, the introduced plasmid has <em>lacY<sup>-</sup></em> mutation and therefore cannot produce <span class="math inline">\(\beta\)</span>-galactoside permease, this causes strain 5 unable to survive with only lactose. Thus, strain 5 has the <em>lacP<sup>-</sup></em> mutation, which originally was able to produce neither permease or galactosidase (because RNAP cannot bind to the promoter to initiate transcription). With plasmid introduced, only galactosidase but not permease is produced, which is not sufficient to catabolise lactose.</p>
<p>Strain 2 contains a <em>lacZ<sup>-</sup></em> mutation, which make it unable to produce galactoside but able to produce permease. Thus, <span class="math inline">\(\beta\)</span>-galactoside catabolism activity is rescued by the introduction of the <em>lacY<sup>-</sup></em> pladmid in the last experiment, which allows galactosidase to be produced.</p>
<p><strong>What is a likely explanation for the phenotype displayed by the strain containing the uncharacterized mutation? In which part of the lac region is the uncharacterized mutation likely to be found?</strong></p>
<p>Strain 4 shows insensitivity to catabolite repression (IPTG induces transcription regardless of the presence of glucose). Thus, it may contain a mutation in the promoter region that causes RNA polymerase to bind more easily, without activation by CAP-cAMP.</p>
<p><strong>The mutation in the catabolite repression-insensitive strain used in the practical has been determined to be within the lac operon and is shown below. The sequence alteration involved (substitution of GT in the wild-type sequence with AA) is highlighted by arrows (▼). How could this sequence alteration lead to the observed mutant phenotype?</strong></p>
<p>ApA and ApT steps have smaller base stacking energies than TpT steps, and there are 2 hydrogen bonds between a AT base pair, which is weaker than the 3 hydrogen bonds between a CG base pair. These makes the sequence <code>AAT</code> easier to deform than <code>GTT</code>. Higher deformability makes a the mutant promoter stronger (i.e. higher affinity for RNA polymerase), so RNAP can bind to it regardless of the binding of CAP-cAMP.</p>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../worksheet/data-handling-biophysical-methods.html" class="archive-item-link">
                Data Handling — Biophysical Methods 
              </a>
              <span class="archive-item-date">
                2020-11-20
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#enzyme"><span class="toc-section-number">1</span> Enzyme</a></li>
<li><a href="#atp-synthase"><span class="toc-section-number">2</span> ATP Synthase</a></li>
<li><a href="#fluorescence-spectroscopy"><span class="toc-section-number">3</span> Fluorescence spectroscopy</a></li>
<li><a href="#high-resolution-nuclear-magnetic-resonance-spectroscopy."><span class="toc-section-number">4</span> High-resolution Nuclear Magnetic Resonance Spectroscopy.</a></li>
<li><a href="#electrospray-mass-spectrometry."><span class="toc-section-number">5</span> Electrospray mass spectrometry.</a></li>
<li><a href="#analytical-ultracentrifugation-equilibrium-sedimentation."><span class="toc-section-number">6</span> Analytical ultracentrifugation: equilibrium sedimentation.</a></li>
</ul>
</div>

<div id="enzyme" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Enzyme</h1>
</div>
<div id="atp-synthase" class="section level1" number="2">
<h1><span class="header-section-number">2</span> ATP Synthase</h1>
</div>
<div id="fluorescence-spectroscopy" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Fluorescence spectroscopy</h1>
</div>
<div id="high-resolution-nuclear-magnetic-resonance-spectroscopy." class="section level1" number="4">
<h1><span class="header-section-number">4</span> High-resolution Nuclear Magnetic Resonance Spectroscopy.</h1>
</div>
<div id="electrospray-mass-spectrometry." class="section level1" number="5">
<h1><span class="header-section-number">5</span> Electrospray mass spectrometry.</h1>
</div>
<div id="analytical-ultracentrifugation-equilibrium-sedimentation." class="section level1" number="6">
<h1><span class="header-section-number">6</span> Analytical ultracentrifugation: equilibrium sedimentation.</h1>
<p><span class="math display">\[
M = \dfrac{2RT}{(1-v\rho)\omega^2}\dfrac{d(\ln(c))}{dr^2}
\]</span></p>
<p><span class="math display">\[
M = \dfrac{2\times 8.314\times298.15}{(1-0.74)\times27000^2}\dfrac{d(\ln(c))}{dr^2} = 2.6156\times10^{-5}\dfrac{d(\ln(c))}{dr^2}
\]</span></p>
<pre class="r"><code>r = c(6.918, 6.923, 6.924, 6.929, 6.932, 6.935, 6.942, 6.943, 6.945, 
6.952, 6.955, 6.962, 6.964, 6.968, 6.973, 6.977, 6.978, 6.983, 
6.985, 6.987, 6.992, 6.994, 6.998, 7.002, 7.004, 7.006, 7.009, 
7.013, 7.016, 7.019, 7.022, 7.024, 7.026, 7.031, 7.033, 7.038, 
7.042, 7.045, 7.047, 7.05, 7.053, 7.059, 7.061, 7.062, 7.065, 
7.069, 7.07, 7.075, 7.077, 7.081, 7.084, 7.087, 7.09, 7.095, 
7.098, 7.102, 7.106, 7.109, 7.11, 7.114, 7.116, 7.121, 7.123, 
7.131, 7.137, 7.139, 7.141, 7.145, 7.146, 7.148, 7.15, 7.156, 
7.158, 7.163, 7.166, 7.17)
od &lt;- c(0.0645, 0.0658, 0.0731, 0.07, 0.0726, 0.0721, 0.0762, 0.0755, 
0.0807, 0.082, 0.0809, 0.0914, 0.0909, 0.0969, 0.0979, 0.0983, 
0.1054, 0.1038, 0.095, 0.1105, 0.1087, 0.1045, 0.1181, 0.1189, 
0.1224, 0.1302, 0.1187, 0.133, 0.1306, 0.1413, 0.1378, 0.1371, 
0.1333, 0.1521, 0.1481, 0.1578, 0.1498, 0.1495, 0.1605, 0.1699, 
0.1664, 0.1757, 0.1802, 0.1834, 0.1882, 0.1856, 0.195, 0.2039, 
0.1999, 0.1874, 0.2191, 0.2143, 0.215, 0.2186, 0.2283, 0.2347, 
0.2677, 0.2532, 0.2596, 0.262, 0.2812, 0.271, 0.2816, 0.2841, 
0.31, 0.3141, 0.3336, 0.3147, 0.334, 0.3528, 0.3548, 0.3572, 
0.3752, 0.373, 0.3863, 0.3792)</code></pre>
<p>Prepare data <span class="math inline">\(\ln(c)\)</span> and <span class="math inline">\(r^2\)</span>:</p>
<pre class="r"><code># divide 2 by 100 to convert cm to m
r2 = (r/100)^2; lnc = log(od)</code></pre>
<pre class="r"><code>mod &lt;- lm(lnc~r2)
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = lnc ~ r2)
## 
## Residuals:
##       Min        1Q    Median        3Q       Max 
## -0.102145 -0.020829  0.001427  0.022606  0.074815 
## 
## Coefficients:
##              Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)  -26.9712     0.2063  -130.7   &lt;2e-16 ***
## r2          5065.5170    41.5559   121.9   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.03669 on 74 degrees of freedom
## Multiple R-squared:  0.995,  Adjusted R-squared:  0.995 
## F-statistic: 1.486e+04 on 1 and 74 DF,  p-value: &lt; 2.2e-16</code></pre>
<pre class="r"><code># only gradient is important</code></pre>
<p>Plot it:</p>
<pre class="r"><code>plot(r2, lnc)
abline(-26.9711515, 5065.51703)</code></pre>
<p><img src="../worksheet/2020-11-20-data-handling-biophysical-methods_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<pre class="r"><code>2.6156e-5 * 5065.51703</code></pre>
<pre><code>## [1] 0.1324937</code></pre>
<pre class="r"><code>#2.6156e-5 * 506551.703</code></pre>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/protein-protein-interactions.html" class="archive-item-link">
                Protein-Protein Interactions 
              </a>
              <span class="archive-item-date">
                2020-11-18
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#properties-of-ppi"><span class="toc-section-number">2</span> Properties of PPI</a>
<ul>
<li><a href="#reversibility"><span class="toc-section-number">2.1</span> Reversibility</a></li>
<li><a href="#prop"><span class="toc-section-number">2.2</span> Properties of the Binding Interfaces</a></li>
<li><a href="#roles-of-water-molecules"><span class="toc-section-number">2.3</span> Roles of Water Molecules</a></li>
<li><a href="#kd"><span class="toc-section-number">2.4</span> Kinetics and Thermodynamics</a></li>
</ul></li>
<li><a href="#studying-ppis"><span class="toc-section-number">3</span> Studying PPIs</a>
<ul>
<li><a href="#determine-kd"><span class="toc-section-number">3.1</span> Determining K<sub>d</sub> (and rate constants)</a>
<ul>
<li><a href="#surface-plasmon-resonance-spr"><span class="toc-section-number">3.1.1</span> Surface Plasmon resonance (SPR)</a></li>
<li><a href="#fluorescence-anisotropy"><span class="toc-section-number">3.1.2</span> Fluorescence anisotropy</a></li>
<li><a href="#isothermal-titration-calorimetry-itc"><span class="toc-section-number">3.1.3</span> Isothermal titration calorimetry (ITC)</a></li>
</ul></li>
<li><a href="#mechanistic-studies"><span class="toc-section-number">3.2</span> Mechanistic Studies</a>
<ul>
<li><a href="#alanine-scanning"><span class="toc-section-number">3.2.1</span> Alanine Scanning</a></li>
<li><a href="#ext"><span class="toc-section-number">3.2.2</span> Extent of Exchengeability of Amino Acids</a></li>
<li><a href="#directed-evolution"><span class="toc-section-number">3.2.3</span> Directed Evolution</a></li>
<li><a href="#comp"><span class="toc-section-number">3.2.4</span> Computational Approaches</a></li>
<li><a href="#connectivity-map-reveals-modularity"><span class="toc-section-number">3.2.5</span> Connectivity Map Reveals Modularity</a></li>
<li><a href="#x-ray-crystallography"><span class="toc-section-number">3.2.6</span> X-Ray Crystallography</a></li>
</ul></li>
</ul></li>
<li><a href="#why-designing-small-molecule-ppi-inhibitors-is-difficult"><span class="toc-section-number">4</span> Why Designing Small Molecule PPI Inhibitors is Difficult</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>How and why do proteins form specific complexes with each other? How can such protein-protein interactions (PPIs) be investigated experimentally, and which problems are associated with designing small molecules to disrupt PPIs?</strong></p>
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Specific protein-protein interactions (PPIs) are critical to numerous biological processes, including cell-cell recognition, immune response, and signal transduction. An understanding of PPIs not only helps to elucidate the detailed roles and to predict the behaviour of proteins in a physiological context but also aids structure-based drug design.</p>
</div>
<div id="properties-of-ppi" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Properties of PPI</h1>
<div id="reversibility" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Reversibility</h2>
<p>Protein-protein interactions can be stable (permanent) or transient. Stable interactions are involved in the assembly of proteins made of multi-subunit complexes such as haemoglobin, which non-reversible in normal physiological conditions. Transient interactions, on the other hand, are reversible, and it is this property that make them act like molecular switches that play versatile roles in controlling cellular processes.</p>
</div>
<div id="prop" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Properties of the Binding Interfaces</h2>
<p>Protein-protein interaction interfaces often have a large surface area (1000-2000 Å<sup>2</sup>) and are relatively flat compared to the deep cavities that typically bind small molecules.
On a binding interface, some residues, known as “hotspots”, contribute to the overall affinity more than other residues.</p>
</div>
<div id="roles-of-water-molecules" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Roles of Water Molecules</h2>
<p>Crystal structures frequently reveal water molecules within PPI interfaces. These water molecules play multifaceted roles in the stability of PPI, e.g. offsetting unfavourable electrostatic interactions, bridging two distant residues via H-bonds.</p>
</div>
<div id="kd" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Kinetics and Thermodynamics</h2>
<p>Reversible PPIs have two important parameters: affinity and specificity. While affinity ranges from as low as millimolar to as high as femtomolar, it is important that the specificity, i.e. the relative affinity of a protein to its cognate binding partner compared to non-cognate ones, is high.</p>
<p>Reversible PPIs can the considered as a simple balance of association and dissociation reactions, with rate constants being <span class="math inline">\(k_{\text{on}}\)</span> and <span class="math inline">\(k_{\text{off}}\)</span>.</p>
<p><span class="math display">\[
\text{A+B}\mathrel{\mathop{\rightleftarrows}^{k_{\text{on}}}_{k_{\text{off}}}} \text{AB}
\]</span></p>
<p>The affinity is usually defined by the dissociation constant:</p>
<p><span class="math display">\[
K_\text{d} = \dfrac{\text{[A][B]}}{\text{[AB]}} = \dfrac{k_{\text{off}}}{k_{\text{on}}}
\]</span></p>
<p>where [A], [B], and [AB] are the concentrations of each species at equilibrium.</p>
<p><span class="math inline">\(K_\text{d}\)</span> can be converted to <span class="math inline">\(\Delta G\)</span> and vice versa:</p>
<p><span class="math display">\[
\Delta G = -RT\ln{(K_\text{d})}
\]</span></p>
<p>In addition to the simple single-step model, <span class="citation">Keeble and Kleanthous (2005)</span> suggested that relatively low affinity PPIs may be better modelled with a two-step induced-fit mechanism involving an unstable intermediate, where electrostatics drives the fast first step (supported by strong dependence on ionic strength) and rigid body rotation occurs in the slow second step:</p>
<p><span class="math display">\[
\text{A+B} \mathrel{\mathop{\rightleftarrows}^{k_{1}}_{k_{-1}}} \text{AB}^\text{*} \mathrel{\mathop{\rightleftarrows}^{k_{2}}_{k_{-2}}} \text{AB}
\]</span></p>
</div>
</div>
<div id="studying-ppis" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Studying PPIs</h1>
<div id="determine-kd" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Determining K<sub>d</sub> (and rate constants)</h2>
<p>Many experimental methods can be used used for studying thermodynamics and kinetics of PPIs (the frequent tasks are determining <span class="math inline">\(K_\text{d}\)</span> and rate constants). Most of them assumes the simple single-step association-dissociation model (Section <a href="#kd">2.4</a>). Three methods are described in this essay.</p>
<div id="surface-plasmon-resonance-spr" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Surface Plasmon resonance (SPR)</h3>
<p>Surface plasmon resonance (Figure <a href="#fig:spr">3.1</a>) can be used to measure both <span class="math inline">\(K_\text{d}\)</span> and <span class="math inline">\(k_\text{on}\)</span> and <span class="math inline">\(k_\text{off}\)</span> rates. How <span class="math inline">\(K_\text{d}\)</span> can be calculated is explained in Equation <a href="#eq:spr-kd">(3.1)</a>.</p>

<div class="figure" style="text-align: c"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, \(\theta_\text{spr}\), is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. \(\Delta\theta_\text{spr}\) is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region). In an SPR experiment, the protein acting as the “bait” is immobilised on the sensor surface, and a the analyte containing the other protein (acting as the “prey”) is passed through the cell. If binding occurs between the two proteins, \(\Delta\theta_\text{spr}\) would increase. Then, non-specific binding is washed off by buffer, and \(\Delta\theta_\text{spr}\) would decrease and \(\Delta\theta_\text{spr}\). Finally, regeneration solution is applied to remove all binding and reset \(\Delta\theta_\text{spr}\) to zero." width="100%" />
<p class="caption">
Figure 3.1: A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, <span class="math inline">\(\theta_\text{spr}\)</span>, is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. <span class="math inline">\(\Delta\theta_\text{spr}\)</span> is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region). In an SPR experiment, the protein acting as the “bait” is immobilised on the sensor surface, and a the analyte containing the other protein (acting as the “prey”) is passed through the cell. If binding occurs between the two proteins, <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would increase. Then, non-specific binding is washed off by buffer, and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would decrease and <span class="math inline">\(\Delta\theta_\text{spr}\)</span>. Finally, regeneration solution is applied to remove all binding and reset <span class="math inline">\(\Delta\theta_\text{spr}\)</span> to zero.
</p>
</div>
<p><span class="math display" id="eq:spr-kd">\[\begin{equation}
\begin{split}
\overbrace{k_{\text{on}}\text{[A][B]}}^\text{rate of association} &amp; = \overbrace{k_{\text{off}}\text{[AB]}}^\text{rate of dissociation} \\
k_{\text{on}}\text{[A]([B]}_{\text{max}}-\text{[AB])} &amp; = k_{\text{off}}\text{[AB]} \\
\text{[AB]} &amp; = \dfrac{k_{\text{on}}\text{[A][B]}_{\text{max}}}{k_{\text{on}}\text{[A]} + k_{\text{off}}} \\
\text{[AB]} &amp; = \dfrac{\text{[A][B]}_{\text{max}}}{\text{[A]} + K_{\text{d}}}
\end{split}
\tag{3.1}
\end{equation}\]</span></p>
<p>In the equation, A is the protein in the analyte, whose concentration is kept constant, and B is the immobilised protein. Since the <span class="math inline">\(\Delta\theta_\text{spr} \propto \text{[AB]}\)</span> (intensity of the signal is proportional to concentration of protein-protein complexes), we can work out <span class="math inline">\(K_\text{d}\)</span> from our initial concentrations of A and B.</p>
<p>SPR was used in the early kinetic analysis of hGH binding (<span class="citation">Wells (1996)</span>).</p>
</div>
<div id="fluorescence-anisotropy" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Fluorescence anisotropy</h3>
<p>Fluorescence anisotropy is based on the phenomenon that, if fluorophores are excited with plane polarized light and the fluorescence is observed through analyzing polarizers, the fluorescence is also polarised.</p>
<p>The fluorescence anisotropy is defined as <span class="math inline">\(A=\dfrac{I_\parallel - I_\bot}{I_\parallel+2_{\bot}}\)</span>, where <span class="math inline">\(I_\parallel\)</span> and <span class="math inline">\(I_{\bot}\)</span> are the fluorescence intensities polarised parallel and perpendicular to the direction of the excitation beam. <span class="math inline">\(A\)</span> is a direct measure of the molecular rotation in solution and can be used to study complex formation, as a macromolecule will rotate more slowly when it is in a complex thatn when it is alone.</p>
<p>Fluorescence anisotropy is more accurate than SPR for measuring ultra-high affinity interactions, and were used to to study ColE DNase-Im interactions (<span class="citation">Papadakos, Wojdyla, and Kleanthous (2012)</span>).</p>
</div>
<div id="isothermal-titration-calorimetry-itc" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Isothermal titration calorimetry (ITC)</h3>
<p>ITC measures heat changes when a complex is formed at constant temperature. In ITC, an insulated reaction cell containing protein is kept at a temperature (usually 8<span class="math inline">\(^\circ\text{C}\)</span> above the environment) which is equal to the temperature of a reference cell, and the reference cell is kept at a constant temperature by a thermostat. Then, increasing amounts of ligand is added into the chamber, and they form complexes with the protein, which can be exothermic or endothermic. The heat change is compensated by a power supply, which can be converted to <span class="math inline">\(\Delta H\)</span> of the reaction. As more ligands are added, proteins become saturated and <span class="math inline">\(\Delta H\)</span> approaches zero. The raw data obtained (power supplied to compensate the heat change caused by each addition of ligands) can be integrated and corrected to give a plot of <span class="math inline">\(\Delta H\)</span> against the molar ratio of the ligand and the protein, and <span class="math inline">\(\Delta H\)</span>, K<sub>d</sub> and stoichiometry can be inferred from the curve (Figure <a href="#fig:itc">3.2</a>).</p>
<div class="figure" style="text-align: c"><span id="fig:itc"></span>
<img src="../img/itc.jpg" alt="ITC data manipulation" width="100%" />
<p class="caption">
Figure 3.2: ITC data manipulation
</p>
</div>
</div>
</div>
<div id="mechanistic-studies" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Mechanistic Studies</h2>
<p>The main theme of mechanistic studies is to find out the “hotspot” residues or regions that are main contributors to the affnity of PPI interfaces, and to attempt to generalise this knowledge in order to predict the affnity of any given PPI interfaces.</p>
<div id="alanine-scanning" class="section level3" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Alanine Scanning</h3>
<p>Alanine scanning is a mutagenesis technique in which mutants are made by substituting alanine for each of the residues in a ‘reactive region’, in this case the PPI interface. By comparing the PPI affininy of each mutant to the wild type (i.e. calculating <span class="math inline">\(\Delta \Delta G\)</span>), the contribution of each residue in binding can be assessed, and this reveals “hotspots” representing important residues.</p>
</div>
<div id="ext" class="section level3" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> Extent of Exchengeability of Amino Acids</h3>
<p>Mutational analysis is most often restricted to alanine substitution and this does not provide an comprehensive view of the allowed amino acid space at each position. This limitation is especially significant in the analysis of PPI interfaces, which, unlike enzyme active sites, the specific orientation and chemical reactivity are less important.</p>
<p><span class="citation">Pál et al. (2006)</span> introduced any one of the 20 natrual amino acids at all 35 hGH-hGHR binding interface positions and obtained surprising results. They verified that, the interface was highly adaptable to mutations, either from a structural or functional point of view. Whereas some of the alanine scanning hotspots showed high specificity agianst substitution, others did not, and some highly specific positions were not hotspots at all.</p>
</div>
<div id="directed-evolution" class="section level3" number="3.2.3">
<h3><span class="header-section-number">3.2.3</span> Directed Evolution</h3>
<p>Directed evolution is an efficient approach to probe sequence and structure space in a PPI. Phage display is a specific implementation of it.</p>
<p>The phage display technique is summarised as follows and illustrated in Figure <a href="#fig:phage-sel">3.3</a>:</p>
<ol style="list-style-type: decimal">
<li>Generate a randomised library of mutants. This can be done using error-prone PCR (ep-PCR). In PPI studies usually the randomised mutations are generated only across small sections of protein sequence that are form of the PPI interface.</li>
<li>The randomised mutant DNA are ligated with a phage coat protein gene and the hybrid is then used to transduce <em>E. coli.</em> cells.</li>
<li>The phage library is amplified by <em>E. coli.</em> cells, and phage particles are produced.</li>
<li>The “bait” protein is immobilised, and the mixture of phage particles displaying different mutant proteins are added. Then, those with low binding affinity are washes away, and the remaining phages and collected, amplified, and used in the next round of selection.</li>
<li>Repeated cycles of selection (a.k.a. “panning”) will identify the mutant proteins with the highest binding affinity to the “bait” protein. The complexes formed by these proteins with the “bait” protein can then be used for detailed mechanistic studies of PPI.</li>
</ol>
<div class="figure" style="text-align: c"><span id="fig:phage-sel"></span>
<img src="../img/phage-sel.png" alt="Directed evolution with phage display." width="100%" />
<p class="caption">
Figure 3.3: Directed evolution with phage display.
</p>
</div>
</div>
<div id="comp" class="section level3" number="3.2.4">
<h3><span class="header-section-number">3.2.4</span> Computational Approaches</h3>
<p>Computational methods can speed up the quest for high-affinity PPIs. These methods are based on rotamer libraries, which summarise the existing knowledge of the experimentally determined structures quantitatively. Rotamers are picked from the library (hotspot constraints may be applied) and grafted onto the scaffold of known structure, then the fitness is assessed using a scoring function, which unfortunately often fails to predict the actual experimental results. As the number rotamers in the library grows and algorithms improve, computational methods are expected to provide more accurate predictions.</p>
</div>
<div id="connectivity-map-reveals-modularity" class="section level3" number="3.2.5">
<h3><span class="header-section-number">3.2.5</span> Connectivity Map Reveals Modularity</h3>
<p><span class="citation">Reichmann et al. (2005)</span> analysed the TEM1–BLIP complex by drawing a connectivity map, which is build from the physical interactions between the proteins (hydrogen bonds, van der Waals interactions, etc.), and showed that the interface can be divided into 6 clusters.
The change <span class="math inline">\(\Delta \Delta G\)</span> on different clusters was found to be additive, whereas mutations within the same cluster caused complex energetic and structural consequences.
Therefore, a PPI interface can be seen as a group of “hot regions”, where each region contribute relatively independently to the total binding affinity, but within each region the contributions from its component amino acids are cooperative.</p>
<div class="figure" style="text-align: c"><span id="fig:ppi"></span>
<img src="../img/ppi.png" alt="Connectivity map of the TEM1–BLIP complex." width="100%" />
<p class="caption">
Figure 3.4: Connectivity map of the TEM1–BLIP complex.
</p>
</div>
</div>
<div id="x-ray-crystallography" class="section level3" number="3.2.6">
<h3><span class="header-section-number">3.2.6</span> X-Ray Crystallography</h3>
<p>X-ray crystallography (XRC) studies provide structural basis for PPI interfaces, which not only facilitates the analysis of individual cases but also help to improve the scoring algorithms for computational methods.</p>
<p>Crystallisation tends to be difficult for low affnity PPIs due to their unstable nature. To overcome this, correctly positioned cysteines can be introduced into each of the binding partners at the interface, which could stabilise the complex, facilitating crystallisation. This technique is known as “disulfide trapping”.</p>
</div>
</div>
</div>
<div id="why-designing-small-molecule-ppi-inhibitors-is-difficult" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Why Designing Small Molecule PPI Inhibitors is Difficult</h1>
<p>PPI interfaces are usually large and flat (Section <a href="#prop">2.2</a>), and often tolerent to a small number of mutations (Section <a href="#ext">3.2.2</a>). Thus, the “druggable” targets are usually restricted to the local non-flat regions (i.e. ‘pockets’) on PPI interfaces that are enriched with “hotspot” residues. A large surface area of PPI interface, which is more tolerent to local non-favourable interactions, also makes developing inhibitors more difficult. In addition, in the case of developing non-peptide inhibitors, there is less existing knowledge on the structures of small molecules and their interactions with proteins, meaning the computational modelling is less accurate.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Arkin-2014">
<p>Arkin, Michelle R, Yinyan Tang, and James A Wells. 2014. “Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing Toward the Reality.” <em>Chem Biol</em> 21 (9): 1102–14. <a href="https://doi.org/10.1016/j.chembiol.2014.09.001">https://doi.org/10.1016/j.chembiol.2014.09.001</a>.</p>
</div>
<div id="ref-Keeble-2005">
<p>Keeble, Anthony H, and Colin Kleanthous. 2005. “The Kinetic Basis for Dual Recognition in Colicin Endonuclease-Immunity Protein Complexes.” <em>J Mol Biol</em> 352 (3): 656–71. <a href="https://doi.org/10.1016/j.jmb.2005.07.035">https://doi.org/10.1016/j.jmb.2005.07.035</a>.</p>
</div>
<div id="ref-Papadakos-2012">
<p>Papadakos, Grigorios, Justyna A. Wojdyla, and Colin Kleanthous. 2012. “Nuclease Colicins and Their Immunity Proteins.” <em>Quarterly Reviews of Biophysics</em> 45 (1): 57–103. <a href="https://doi.org/10.1017/S0033583511000114">https://doi.org/10.1017/S0033583511000114</a>.</p>
</div>
<div id="ref-Pal-2006">
<p>Pál, Gábor, Jean-Louis K Kouadio, Dean R Artis, Anthony A Kossiakoff, and Sachdev S Sidhu. 2006. “Comprehensive and Quantitative Mapping of Energy Landscapes for Protein-Protein Interactions by Rapid Combinatorial Scanning.” <em>J Biol Chem</em> 281 (31): 22378–85. <a href="https://doi.org/10.1074/jbc.M603826200">https://doi.org/10.1074/jbc.M603826200</a>.</p>
</div>
<div id="ref-Reichmann-2005">
<p>Reichmann, D, O Rahat, S Albeck, R Meged, O Dym, and G Schreiber. 2005. “The Modular Architecture of Protein-Protein Binding Interfaces.” <em>Proc Natl Acad Sci U S A</em> 102 (1): 57–62. <a href="https://doi.org/10.1073/pnas.0407280102">https://doi.org/10.1073/pnas.0407280102</a>.</p>
</div>
<div id="ref-Wells-1996">
<p>Wells, J A. 1996. “Binding in the Growth Hormone Receptor Complex.” <em>Proceedings of the National Academy of Sciences</em> 93 (1): 1–6. <a href="https://doi.org/10.1073/pnas.93.1.1">https://doi.org/10.1073/pnas.93.1.1</a>.</p>
</div>
<div id="ref-Wienken-2010">
<p>Wienken, Christoph J., Philipp Baaske, Ulrich Rothbauer, Dieter Braun, and Stefan Duhr. 2010. “Protein-Binding Assays in Biological Liquids Using Microscale Thermophoresis.” <em>Nature Communications</em> 1 (1): 100. <a href="https://doi.org/10.1038/ncomms1093">https://doi.org/10.1038/ncomms1093</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/membrane-protein-structure-and-stability.html" class="archive-item-link">
                Membrane Protein: Structure and Stability 
              </a>
              <span class="archive-item-date">
                2020-11-12
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#the-biomembrane-environment"><span class="toc-section-number">2</span> The Biomembrane Environment</a></li>
<li><a href="#the-thermodynamic-basis-of-secondary-structure-formation"><span class="toc-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</a></li>
<li><a href="#glycine-and-proline-in-tm-helices"><span class="toc-section-number">4</span> Glycine and Proline in TM Helices</a></li>
<li><a href="#snorkel"><span class="toc-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</a></li>
<li><a href="#interactions-with-lipids"><span class="toc-section-number">6</span> Interactions with Lipids</a></li>
</ul>
</div>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Many proteins that perform vital functions are membrane proteins. These include transport proteins (channels, transporters and pumps), cell adhesion molecules, and proteins that transduces energy in the electron transport chain. Despite their diverse roles, the same set of biophysical and biochemical rules govern their structural stability. Membrane proteins fall into two broad categories: 1) integeral proteins, which are embedded within a membrane and can only be isolated with detergents, and 2) peripheral proteins, which are associated with the surface of a membrane and can be removed without detergents. The focus of this essay is on how integral proteins interact with the membrane to maintain their structural stability.</p>
</div>
<div id="the-biomembrane-environment" class="section level1" number="2">
<h1><span class="header-section-number">2</span> The Biomembrane Environment</h1>
<p>The basic structure of a biomembrane is a bilayer of amphipathic phospholipids and sphingolipids, where the polar head groups of lipids align the surfaces (each ~10 angstroms thick) and the acyl chains of lipid tails occupy the hydrophobic core (~ 30 angstroms thick). A bitopic or polytopic integral membrane protein spans the membrane entirely one or more times (Figure <a href="#fig:lipid-bilayer-diagram">2.1</a>).</p>
<div class="figure"><span id="fig:lipid-bilayer-diagram"></span>
<img src="../img/lipid-bilayer-diagram.png" alt="(a) a schematic showing a integral protein embedded within a biomembrane. (b)" width="100%" />
<p class="caption">
Figure 2.1: (a) a schematic showing a integral protein embedded within a biomembrane. (b)
</p>
</div>
<p>The hydrophobic core region has a dielectric constant (<span class="math inline">\(\epsilon_{r}\)</span>) of about 2, which is much lower than <span class="math inline">\(\epsilon_{r}\)</span> of water (about 80), which means the same pair of charges separated by the same distance experience more electrostatic force in the membrane core than in water, according to the Coulomb’s law:</p>
<p><span class="math display">\[F = \dfrac{q_1 q_2}{4\pi \epsilon_0 \epsilon_r r^2}\]</span></p>
<p>This has several implications on acidic and basic amino acid residues. First, the carboxyl group of acidic side chains are more difficult to dissociate (i.e. p<span class="math inline">\(K_a\)</span>s are shifted up), meaning they tend to remain in the uncharged (–COOH) form. Second, in <span class="math inline">\(\alpha\)</span>-helical bundles, oppositely charged residues in adjacent helices associated more strongly than they do in aqueous environment. Third, this facilitates the snorkelling of lysine and arginine residues near the interface (Section <a href="#snorkel">5</a>).</p>
</div>
<div id="the-thermodynamic-basis-of-secondary-structure-formation" class="section level1" number="3">
<h1><span class="header-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</h1>
<p>Transmembrane proteins tend to adopt a conformation that minimises the contact between their polar groups (main chain -NH, C=O, and side chains of polar residues, if any) and the hydrophobic core. This can be explained by simple thermodynamic reasoning.
The water molecules in the aqueous environment outside the membrane are capable of forming relatively strong dipole-dipole interactions and hydrogen bonds (H-bonds) with polar (including charged) amino acids, while the non-polar alkyl groups that occupy the hydrophobic core of the membrane can only provide weak van der Waals interactions.
Thus, if a protein adopts a conformation that exposes many polar groups, the loss of strong interactions with water will make this process very energetically unfavourable (very large <span class="math inline">\(\Delta G\)</span>).
On the contrary, if a protein with a hydrophobic surface were present in an aqueous environment, it would disrupt the dipole-dipole interactions and hydrogen bonding among water and other polar molecules. Thus, its insertion into the membrane is energetically favourable.</p>
<p>By forming either an <span class="math inline">\(\alpha\)</span>-helix or a <span class="math inline">\(\beta\)</span>-sheet (Figure <a href="#fig:bundle-helix">3.1</a>), the hydrogen bonds between main chain -NH and -C=O are maximised, and this is an efficient way to shield the polar groups on the main chain from the hydrophobic core.
Many transmembrane proteins, such as bacteriorhodopsin, contain multiple TM helices, which aggregates to form a <span class="math inline">\(\alpha\)</span>-helix bundle.
A <span class="math inline">\(\beta\)</span>-sheet has to twist and coil to form a ring, called a <span class="math inline">\(\beta\)</span>-barrel, in order to shield the main chain -NH and C=O groups on the first and last strands.
<span class="math inline">\(\beta\)</span>-barrels usually have a even number of strands so that every strand is anti-parallel to adjacent strands, there are exceptions: the voltage-dependent anion channels (VDACs) located on mitochondrial outer membrane has 19 strands, where the first strand is parallel with tha last strand, with weaker hydrogen bonding.
<span class="math inline">\(\beta\)</span>-barrels are less common than <span class="math inline">\(\alpha\)</span>-helix bundles, and they are predominantly present on the outer membrane of Gram-negative bacteria, mitochondria, and chloroplasts.</p>
<div class="figure"><span id="fig:bundle-helix"></span>
<img src="../img/bundle-helix.png" alt="Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively." width="100%" />
<p class="caption">
Figure 3.1: Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively.
</p>
</div>
<p>The transmembrane region of each helix in a <span class="math inline">\(\alpha\)</span>-helical bundle is composed predominantly of hydrophobic amino acid residues (Ala, Leu, Ile, Val, Phe), which are exposed on the surface and allows favourable interaction with the hydrophobic core.
By contrast, each <span class="math inline">\(\beta\)</span>-strand of a <span class="math inline">\(\beta\)</span>-barrel has an alternating pattern of hydrophobic and polar residues.
This is because two adjacent residues in a beta strand have their side chains pointing in opposite directions, and a <span class="math inline">\(\beta\)</span>-barrel formed in this way has an hydrophobic exterior that interacts with the hydrophobic core and an hydrophilic inner surface that is in contact with the aqueous environment.</p>
<p>Thus, TM helices, but not <span class="math inline">\(\beta\)</span>-barrels, can be predicted using a hydropathy plot (Figure <a href="#fig:hydropathy-plot-1">3.2</a>), in which the average hydrophobicity index of a fixed number of consecutive residues (a “window”), <span class="math inline">\(H(i)\)</span>, is plotted against the index (<span class="math inline">\(i\)</span>) of the window, i.e.</p>
<p><span class="math display">\[H(i) = \sum_{i&lt;j&lt;i+k}h(a_j)/k\]</span></p>
<p>for <span class="math inline">\(1 &lt; i &lt; n - k\)</span>, where n is the length (number of residues) of the peptide, <span class="math inline">\(h(a_j)\)</span> is the hydrophobicity index of the <span class="math inline">\(j\)</span>-th amino acid residue and <span class="math inline">\(k\)</span> is the window size.</p>
<div class="figure"><span id="fig:hydropathy-plot-1"></span>
<img src="../img/hydropathy-plot-1.png" alt="A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix." width="100%" />
<p class="caption">
Figure 3.2: A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix.
</p>
</div>
<p>A small number of polar (even charged) residues within a TM helix can be tolerated as long as the overall transmembrane segment is hydrophobic enough.
In addition, in polytopic proteins, polar residues in adjacent helices may help to stabilise each other.
These non-hydrophobic residues often have functional roles, as exemplified by voltage-gated K<sup>+</sup>, Na<sup>+</sup> and Ca<sup>2+</sup> channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
<div class="figure"><span id="fig:kv-mech"></span>
<img src="../img/kv-mech.png" alt="The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker." width="100%" />
<p class="caption">
Figure 3.3: The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker.
</p>
</div>
</div>
<div id="glycine-and-proline-in-tm-helices" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Glycine and Proline in TM Helices</h1>
<p>The side chain of proline forms a pentameric ring with the amine group on the main chain. Thus, in a helix, the amine group of a proline does not have a hydrogen that can be H-bonded to the main chain C=O of the residue above it. This introduces local flexibility within a relatively regid helix and often forms a hinge. Similarly, due to glycine’s small size, it can tolerate a much wider range of dihedral angles than other amino acids, which also makes it able to introduce flexibility. This flexibility is crucial for the function of some proteins, such as voltage-gated ion channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
</div>
<div id="snorkel" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</h1>
<p>Tryptophan and tyrosine are often found in the interfacial region in both <span class="math inline">\(\alpha\)</span>-helical bundle and <span class="math inline">\(\beta\)</span>-barrel proteins. Their hydrophobic aromatic ring interacts with the hydrophobic core, and their polar -NH (of Trp) or -OH (of Tyr) groups interacts with the lipid head groups. Similarly, lysine and arginine extend their side chain towards the surface, so that their positively charged guanidium (of Arg) or amide (of Lys) groups form strong ionic interactions with the negatively charged lipid head groups. This phenomenon is known as “snorkelling.” These interactions helps to lock the membrane in place within the membrane, preventing vertical motions.</p>
</div>
<div id="interactions-with-lipids" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Interactions with Lipids</h1>
<p>Membrane proteins interacts with lipid in various ways, and these interactions contribute to the structural stability and function of the protein to different extents.</p>
<p>A transmembrane protein interacts strongly with the shell (annulus) of lipid that surrounds it. These lipids are called annular lipids and can be distinguished experimentally from the bulk lipids of the bilayer. In addition, there is a third class of lipids, called nonannular lipids (or lipid cofactors), which are tightly bound in crevices or between subunits of the protein and are often crutial for the activity of the protein. Electron paramagnetic resonance (EPR) can be used to measure the mobility of annular lipids, and thus gives information on the selectivity for annular lipids of a protein. Most proteins are found to prefer negatively charged lipids such as phosphatidylserine and phosphatidylinositol.</p>
<p>Strong lipid-protein interactions are also evident form crystal structures, and this interaction can be specific. For example, three cardiolipin (CDL) molecules were found to bind to three sites on the ANT1 transporter. The tendency of CDL to bind to these sites can also be shown in molecular dynamics simulations. The specificity of this binding can also be shown computationally, by calculating the free energy at different protein-lipid separations for CDL and other lipids (Figure <a href="#fig:simu-lipid-protein-binding">6.1</a>).</p>
<div class="figure"><span id="fig:simu-lipid-protein-binding"></span>
<img src="../img/simu-lipid-protein-binding.png" alt="Computational analysis of a protein's preference to bound lipids. CDL binding results in a significantly lower energy state." width="100%" />
<p class="caption">
Figure 6.1: Computational analysis of a protein’s preference to bound lipids. CDL binding results in a significantly lower energy state.
</p>
</div>
<p>Inward-rectifier K<sup>+</sup> channels (K<sub>ir</sub>) are an example where the lipid-protein interaction is directly involved in the protein’s function. K<sub>ir</sub> have a Transmembrane domain (TMD) and an intracellular C-terminal domain (CTD). PIP<sub>2</sub> binding cuases the CTD to move towards the TMD, which in turn causes the helices in the TMD to bend and thus opens the gate (Figure <a href="#fig:kir">6.2</a>).</p>
<div class="figure"><span id="fig:kir"></span>
<img src="../img/kir.png" alt="Lipid-gated Kir channel" width="100%" />
<p class="caption">
Figure 6.2: Lipid-gated Kir channel
</p>
</div>
<p>Many crystal structures of GPCRs reveal cholesterol bound to the protein. Molecular dynamics simulations at different cholesterol concentrations suggests the role of cholesterol in reducing the dynamic flexibility the protein and stabilising a certain conformational state in an allosteric fashion.</p>
<!-- Lipid rafts are stable but transient domains that are enriched in cholesterol and sphingolipids, which makes them thinker than the average membrane thinkness (directly detectable using atomic force microscopy). They are specifically associated with lipid anchored proteins (e.g. GPI-, palmitoyl-, myristoyl-anchored protein) -->
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/gene-control-practical.html" class="archive-item-link">
                Gene Control Practical 
              </a>
              <span class="archive-item-date">
                2020-11-05
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#determine-the-time-course-of-beta-galactosidase-induction"><span class="toc-section-number">1</span> Determine the time course of <span class="math inline">\(\beta\)</span>-galactosidase induction</a>
<ul>
<li><a href="#protocal"><span class="toc-section-number">1.1</span> Protocal</a></li>
<li><a href="#results"><span class="toc-section-number">1.2</span> Results</a></li>
</ul></li>
<li><a href="#identify-the-wt-strain"><span class="toc-section-number">2</span> Identify the WT Strain</a>
<ul>
<li><a href="#protocol"><span class="toc-section-number">2.1</span> Protocol</a></li>
</ul></li>
<li><a href="#devise-an-experimental-protocol-to-show-the-role-played-by-camp-in-catabolite-repression."><span class="toc-section-number">3</span> Devise an experimental protocol to show the role played by cAMP in catabolite repression.</a>
<ul>
<li><a href="#results-part-2-and-3"><span class="toc-section-number">3.1</span> Results (part 2 and 3)</a></li>
</ul></li>
<li><a href="#devise-experiments-to-test-the-following-model."><span class="toc-section-number">4</span> Devise experiments to test the following model.</a>
<ul>
<li><a href="#protocol-1"><span class="toc-section-number">4.1</span> Protocol</a></li>
<li><a href="#results-1"><span class="toc-section-number">4.2</span> Results</a></li>
</ul></li>
</ul>
</div>

<div id="determine-the-time-course-of-beta-galactosidase-induction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Determine the time course of <span class="math inline">\(\beta\)</span>-galactosidase induction</h1>
<div id="protocal" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Protocal</h2>
<p>Trial experiment:</p>
<ul>
<li>Take OD<sub>600</sub> reading in spectrophotometer 1) to verify that the absorbance is around 0.6 (i.e. the cultures are in exponential growth) 2) to estimate the density of <em>E. coli.</em> in solution</li>
<li>Add 9.940 ml (9.9 ml + <span class="math inline">\((20 \times 2)\)</span> <span class="math inline">\(\mu\)</span>l) of stock E. coli solution into each of two flasks labelled <code>+IPTG</code> and <code>-IPTG</code> (control), respectively</li>
<li>at 1 min intervals take 20 <span class="math inline">\(\mu\)</span>l sample and add to a microtube containing 80 <span class="math inline">\(\mu\)</span>l permeabilisation reagent (200 mM Na2HPO4, 20 mM KCl, 2 mM MgSO4, 0.8 mg/mL hexadecyltrimethylammonium bromide (CTAB), 0.4 mg/mL sodium deoxycholate and chloramphenicol. Whirlimix for 5 seconds. (add permeabilisation reagent as soon as a sample is taken out, which contains antibiotics to stop the bacteria from producing more galactosidase (otherwise the bacteria taken out will just continue to produce galactosidase at the same rate); do not add ONPG until all samples are collected, so assays start at the same time)</li>
<li>collect 2 samples, at -2 min and 0 min (for both <code>+IPTG</code> and <code>-IPTG</code>) which represent baseline activity and then:
<ul>
<li>To the remaining 9.9 ml of <em>E. coli.</em> in <code>+IPTG</code> flask, add 0.1 ml of IPTG (stock solution provided at 100 mM) so that the final concentration is 1 mM. This will induce the lac operon by acting as an allolactose mimic. Mix.</li>
<li>To the remaining 9.9 ml of <em>E. coli.</em> in <code>-IPTG</code> flask, add 0.1 ml of salt solution</li>
</ul></li>
<li>Collect 10 additional samples (for each of <code>+IPTG</code> and <code>-IPTG</code>), and setup a blank: 20 <span class="math inline">\(\mu\)</span>l of salt solution plus 80 <span class="math inline">\(\mu\)</span>l permeabilisation reagent. (i.e. measuring 40 samples against one blank).</li>
<li>Add 75 <span class="math inline">\(\mu\)</span>l of ONPG to 75 <span class="math inline">\(\mu\)</span>l of each sample and the blank. Incubate at 37<sup>o</sup>C and start the timer.</li>
<li>Monitor the reaction until a convenient intensity of colour has developed. Take samples out and note the time of incubation.</li>
<li>Read absorption at 415 nm against the blank. The readings should not exceed 1.2 to ensure linearity, so it’s important to stop the reaction before the colour gets too dark (in the previous step).
<ul>
<li>If any of the readings gets beyond 1.2, incubate for less time in the next experiment</li>
</ul></li>
<li>Plot a graph of A<sub>415</sub> against time. Assess the graph:
<ul>
<li>Are there any regions of uncertainty? If yes, take more samples in this region in the next experiemnt.</li>
<li>Has the induction reached completion? If not, take samples in a longer time range (2 hours) in the next experiment.</li>
</ul></li>
</ul>
<p>Optimised experiment:</p>
<p>Perform the same procedure described above with better knowledge on the time of incubation and time range and intervals to take samples.</p>
</div>
<div id="results" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Results</h2>
<pre class="r"><code>control &lt;- c(.060, .076, .077, .079, .080, .082,
             .086, .082, .084, .085, .085, .083)
iptg    &lt;- c(.077, .082, .079, .098, .103, .136,
             .171, .220, .280, .297, .310, .336)
iptg1   &lt;- c(.079, .079, .078, .105, .112, .148,
             .202, .239, .274, .335, .362, .440)
iptg2   &lt;- c(.081, .082, .082, .110, .168, .219,
             .324, .414, .538, .651, .704, .786)
tibble(
  time = rep(seq(-2, 20, 2), 4),
  group = rep(c(&quot;-IPTG&quot;, &quot;+IPTG 1&quot;, &quot;+IPTG 2&quot;, &quot;+IPTG 3&quot;), each = 12),
  absorbance = c(control, iptg, iptg1, iptg2)
) %&gt;% ggplot(aes(time, absorbance)) +
  geom_line(aes(color = group)) +
  ylab(&quot;absorbance (A415)&quot;)+
  geom_vline(xintercept = 0)+
  geom_text(data = tibble(x = 1.4, y = 0.4, label = &#39;+IPTG&#39;),
            aes(x, y, label = label))+
  scale_x_continuous(breaks = seq(-2, 20, 2))</code></pre>
<p><img src="../lab/2020-11-05-gene-control-practical_files/figure-html/unnamed-chunk-1-1.png" width="672" /></p>
<p><strong>Conclusion.</strong> It takes about 4 minutes for the effect of IPTG on increasing galactosidase activity to become evident.</p>
</div>
</div>
<div id="identify-the-wt-strain" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Identify the WT Strain</h1>
<p><em>You are provided with two strains of coli labelled A and B, one of which (‘wild type’) is subject to catabolite repression, and one of which is not. Devise an experimental protocol to determine which strain is the wild type strain.</em></p>
<p>The E.coli strains are already grown on minimal medium containing 1% glycerol as a ‘poor’ carbon source (which should result in a <em>high</em> intracellualr cAMP concentration). Thus, to test for sensitivity of catabolite repression, we add a ‘good’ carbon source — glucose (which should increase cAMP level and repress galactosidase synthesis).</p>
<div id="protocol" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Protocol</h2>
<ul>
<li>Measure OD<sub>600</sub> of both strains.</li>
<li>For each of strains A and B, set up two flasks, <code>+glucose</code> and <code>-glucose</code> (i.e. 4 flasks in total); then:
<ul>
<li>Add 9.440 ml of <em>E. coli</em> to all flasks</li>
<li>collect 2 samples, at -2 min and 0 min which represent baseline activity and then:</li>
<li>add 0.5 ml of glucose solution or glycerol solution to <code>+glucose</code> flasks and <code>-glucose</code> flasks, respectively</li>
<li>collect 3 samples at 2, 4, 6 min and then:</li>
<li>Add 0.1 ml IPTG (100 mM) to all flasks and mix (start induction)</li>
<li>From 8 to 18 min, take 6 samples at 2 min intervals</li>
<li>Add 0.1 ml cAMP to all flasks</li>
<li>Collect samples and set up a blank: 20 <span class="math inline">\(\mu\)</span>l of salt solution plus 80 <span class="math inline">\(\mu\)</span>l permeabilisation reagent. (the blank is used by all samples taken from 4 flasks)</li>
<li>Add 75 <span class="math inline">\(\mu\)</span>l of ONPG to 75 <span class="math inline">\(\mu\)</span>l of each sample and the blank. Incubate at 37<sup>o</sup>C.</li>
<li>Incubate for appropriate amount of time (as determined in part 1)</li>
<li>Read absorption at 415 nm against the blank.</li>
<li>Plot a graph of A<sub>415</sub> against time.</li>
</ul></li>
<li>Compare the plots. The strain unresponsive to catabolite inhibition should give similar plots in both <code>+glucose</code> and <code>-glucose</code> conditions. The WT strain should show lower A<sub>415</sub> (galactosidase activity) when glucose is added, since its galactosidase synthesis is inhibited by the ‘good’ carbon source, pyruvate.</li>
</ul>
<p>The experiment in this part also answers the question in the next part, and results are shown there.</p>
</div>
</div>
<div id="devise-an-experimental-protocol-to-show-the-role-played-by-camp-in-catabolite-repression." class="section level1" number="3">
<h1><span class="header-section-number">3</span> Devise an experimental protocol to show the role played by cAMP in catabolite repression.</h1>
<!-- cAMP production is promoted by a lack of metabolites (or unavailability of 'good' carbon sources), and cAMP in turn allows for galactosidase production via CAP. Here, -->
<p>Since our bacteria are grown on minimal medium containing 1% glycerol as a ‘poor’ carbon source (which should result in a <em>high</em> intracellualr cAMP concentration), we need to first use glucose to inhibit galactosidase production (as described in part 2). Then, we add cAMP. If the added cAMP enhances the production of galactosidase, we can prove that it’s cAMP whose level is decreased during catabolite repression (by addition of a ‘good’ carbon source, pyruvate).</p>
<p>The experimental procedure is described in the previous section.</p>
<div id="results-part-2-and-3" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Results (part 2 and 3)</h2>
<p><strong>Optical density</strong>. OD<sub>600</sub> of strain A and B solutions are 0.309 and 0.505, respectively. Both are within the range in which the bacteria are in exponential growth phase. However, since their densitie differ, when interpreting the result we should not directly compare A to B. Instead, we should assess the effects of glucose and cAMP on each of the strains independently.</p>
<p><strong>Galactosidase activity assay results.</strong> The raw data and the plot are shown below.</p>
<pre class="r"><code># strain A -glucose
Ag = c(.089, .049, .089, .048, .070, .072, .079, .111, 
       .156, .198, .254,  NA, .388, .424, .419, .520)
# strain A +glucose
AG = c(.071, .071, .082, .049, .075, .070,   NA, .102,
       .130, .169, .207, .221, .251, .261, .342, .412)
# strain B -glucose
Bg = c(.087, .086, .097, .073, .091, .089, .099, .156,
       .188, .273, .356, .482, .593, .699, .795, .943)
# strain B +glucose
BG = c(.084, .080, .084, .071, .087, .088, .096, .109,
       .127, .150, .184, .216, .254, .322, .463, .543)
label = tibble (
  x = c(2, 6, 18),
  y = c(0.75, 0.7, 0.75),
  label = c(&quot;+glucose&quot;, &quot;+IPTG&quot;, &quot;+cAMP&quot;)
)
tibble(
  strain = rep(c(&quot;A&quot;, &quot;B&quot;), each = 32),
  glucose = rep(c(&quot;-&quot;, &quot;+&quot;, &quot;-&quot;, &quot;+&quot;), each = 16),
  time = rep(seq(-2, 28, 2), 4),
  absorbance = c(Ag, AG, Bg, BG)
) %&gt;%
  ggplot(aes(time, absorbance)) +
  geom_point(aes(shape = glucose))+
  geom_smooth(aes(linetype = glucose), method = &#39;loess&#39;, formula = &#39;y ~ x&#39;)+
  # label addition of glucose, IPTG and cAMP
  geom_vline(xintercept = c(0, 6, 18))+
  geom_text(data = label, aes(x, y, label = label))+
  # plot strain A and strain B side by side
  facet_grid(~strain)+
  # tweak axes and add title &amp; notes
  scale_x_continuous(breaks = c(0, seq(-2, 30, 4)))+
  labs(
    title = &quot;Effect of glucose and cAMP on galactosidase activity of strains A and B&quot;,
    caption = &quot;N.B. anomalous data values were replaced with NA and are invisible from the plot&quot;
  )</code></pre>
<pre><code>## Warning: Removed 2 rows containing non-finite values (stat_smooth).</code></pre>
<pre><code>## Warning: Removed 2 rows containing missing values (geom_point).</code></pre>
<p><img src="../lab/2020-11-05-gene-control-practical_files/figure-html/unnamed-chunk-2-1.png" width="672" /></p>
<p><strong>Conclusion.</strong> Strain B is WT. In strain B, galactosidase activity was significantly lower in <code>+glucose</code> than in <code>-glucose</code> from <code>t = 6 min</code> to <code>t = 18 min</code> (after IPTG induction, before addition of cAMP). In strain A the difference is less pronounced. Addition of cAMP at <code>t = 18 min</code> markedly increased galactosidase activity in <code>strain B + glucose</code> setup, but had no noticeable effects in the other three setups. This means catabolite repression by glucose is mediated by lowering concentration of cAMP, and thus an addition of cAMP could relieve this repression.</p>
</div>
</div>
<div id="devise-experiments-to-test-the-following-model." class="section level1" number="4">
<h1><span class="header-section-number">4</span> Devise experiments to test the following model.</h1>
<p><span class="math inline">\(\alpha\)</span><em>-keto acids are the long sought `catabolites’ that control cAMP levels and that they act by inhibiting cAMP synthesis</em></p>
<p><em>(Solutions of <span class="math inline">\(\alpha\)</span>-ketoglutarate, pyruvate, and succinate are available in the lab. E. coli is able to take up any of these exogenously added metabolites.)</em></p>
<div id="protocol-1" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Protocol</h2>
<p>Repeat the experimental procedure described in part 2 with only the WT strain (strain B) and replacing glucose with different <span class="math inline">\(\alpha\)</span>-keto acids. The concentration of the ketoacids are 0.5 M. Thus, to make their concentration in the 10 ml reaction mixture be 10 mM, 0.2 mL of each was added to different flasks individually (alpha-ketoglutarate was 1.0 M, so 0.1 mL of it was added instead).</p>
<!-- - Use the WT strain in this set of experiments.
- 4 experiments will be done for each $\alpha$-keto acid. Adding the control, there will be a total of 13 experiments. (if there isn't enough reagent, just scale everything by 1/10)
- Make 1:10 and 1:100 dilutions of each $\alpha$-keto acid.  (*do we need to be quantitative (i.e. comparing the effectiveness of different metabolite)? If this is just a qualitative assessment we don't need multiple dilutions to prove their involvement in catabolite repression and inhibition of cAMP synthesis (we only need 2 experiments for each ketoacid)*)

|            | $\alpha$-ketoglutarate | pyruvate | succinate |
| ---------- | ---------------------- | -------- | --------- |
| 1:1        |                        |          |           |
| 1:10       |                        |          |           |
| 1:100      |                        |          |           |
| 1:1 + cAMP |                        |          |           |

For each cell of the above table:

- Take 9.0 ml of *E. coli* and add 0.9 ml of the dilution.
- Wait for 5 minutes to allow the bacteria adapt to the addition of  $\alpha$-keto acid.
- Add 0.1 ml IPTG (100 mM) and mix (start induction).
- At 1 min intervals take 20 $\mu$l sample and add to a microtube containing 80 $\mu$l permeabilisation reagent
- Collect samples and set up a blank: 20 $\mu$l of uninduced *E. coli* plus 80 $\mu$l permeabilisation reagent. (the blank is only made once and used for all measurements)
- Add 75 $\mu$l of ONPG to 75 $\mu$l of each sample and the blank. Incubate at 37^o^C.
- Incubate for appropriate amount of time (as determined in part 1)
- Read absorption at 415 nm against the blank.
- Plot a graph of A~415~ against time. -->
<p>If the addition of an <span class="math inline">\(\alpha\)</span>-keto acid alone causes the activity of galactosidase to decrease, it can be considered as the evidence that this ketoacid is involved in catabolite repression. If the following addition of cAMP can ‘rescue’ galactosidase activity, then this can be considered as the evidence that this ketoacid inhibits cAMP synthesis.</p>
</div>
<div id="results-1" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Results</h2>
<p><strong>Optical density</strong>. OD<sub>600</sub> = 0.393, which verifies that the bacteria were in exponential growth phase.</p>
<p><strong>Galactosidase activity assay results.</strong> The raw data and the plot are shown below.</p>
<pre class="r"><code># glycerol (control)
gly = c(.075, .076, .076, .076, .077, .078, .116, .214,
        .303, .404, .510, .609, .702, .901, 1.028, 1.173)
# pyruvate
pyr = c(.077, .075, .075, .075, .075, .072, .085, .088,
        .095, .135, .149, .184, .234, .280, .469, .585)
# succinate
suc = c(.074, .076, .075, .074, .078, .080, .083, .120, 
        .152, .204, NA, .307, .408, .554, NA, .762)
# alpha-ketoglutarate
akg = c(.077, .076, .073, .074, .076, .078, .088, .097, 
        .122, .172, .215, .244, .293, .367, .448, .647)
tibble(
  carbon_source = as_factor(rep(c(&quot;glycerol&quot;, &quot;succinate&quot;, &quot;alpha-ketoglutarate&quot;, &quot;pyruvate&quot;), each = 16)),
  time = rep(seq(-2, 28, 2), 4),
  absorbance = c(gly, suc, akg, pyr)
) %&gt;%
  ggplot(aes(time, absorbance)) +
  geom_point(aes(shape = carbon_source))+
  geom_smooth(aes(color = carbon_source), method = &#39;loess&#39;, formula = &#39;y ~ x&#39;)+
  # rates after adding cAMP (tangent lines)
  geom_abline(slope = 0.06314, intercept = c(-0.647, -1.00, -1.20))+
  # label addition of glucose, IPTG and cAMP
  geom_vline(xintercept = c(0, 6, 18))+
  geom_text(data = label, aes(x, y, label = label))+
  # tweak the axes and add title &amp; notes
  scale_x_continuous(breaks = c(0, seq(-2, 30, 4)))+
  labs(
    title = &quot;Effect of different carbon sources on galactosidase activity&quot;,
    caption = &quot;N.B. anomalous data values were replaced with NA and are invisible from the plot&quot;
  )</code></pre>
<pre><code>## Warning: Removed 2 rows containing non-finite values (stat_smooth).</code></pre>
<pre><code>## Warning: Removed 2 rows containing missing values (geom_point).</code></pre>
<p><img src="../lab/2020-11-05-gene-control-practical_files/figure-html/unnamed-chunk-3-1.png" width="672" /></p>
<p><strong>Conclusion.</strong> All three <span class="math inline">\(\alpha\)</span>-keto acids are shown to be capable of catabolite repression. After adding IPTG, the extents to which the <span class="math inline">\(\beta\)</span>-galactosidase activity increases in <em>E. coli</em> provided with <span class="math inline">\(\alpha\)</span>-keto acids are significantly and consistently lower than in <em>E. coli</em> provided with glycerol only. Addition of cAMP has no significant effect on the <code>glycerol</code> control, but was able to improve the rate of increase of <span class="math inline">\(\beta\)</span>-galactosidase activity (gradient on the graph) in groups with <span class="math inline">\(\alpha\)</span>-keto acids to about the same as the <code>glycerol</code> control (shown as tangent lines in the plot). The capability of different <span class="math inline">\(\alpha\)</span>-keto acids can be ranked qualitatively: pyruvate &gt; <span class="math inline">\(\alpha\)</span>-ketoglutarate &gt; succinate.</p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/crystallography-a-case-study.html" class="archive-item-link">
                Crystallography: A Case Study 
              </a>
              <span class="archive-item-date">
                2020-11-04
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>The Ca<sup>2+</sup> selectivity of voltage-gated calcium (Ca<sub>v</sub>) channels remained unclear. They were thought to use a ‘knock-off’ methanism which requires multiple ion-binding cores, but mutational analyses supported a single high-affinity Ca<sup>2+</sup> binding site. This paradox is explained by the mechanism proposed in this article, based on the crystal structures of Ca<sub>v</sub>Ab channels.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<p>In this study, the authors created several mutants of bacterial Na<sub>v</sub> channel Na<sub>v</sub>Ab by site directed mutagenesis using QuickChange at the selectivity filter region<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Some of these mutations changes the activity of the channel from allowing efflux of Na<sup>+</sup> to allowing influx of Ca<sup>2+</sup> to different extents (quantified by the relative permeability of Ca<sup>2+</sup> against Na<sup>+</sup>). Baculovirus were used as the vector to infect <em>Trichopulsia ni</em> cells. Proteins were collected from insect cells, purified, concentrated to ~20mg ml<sup>-1</sup>, and reconstituted into DMPC:CHAPSO bicelles (Figure <a href="#fig:dmpc-chapso">1</a>). Crystals were grown in a hanging-drop vapour-diffusion format.</p>
<div class="figure"><span id="fig:dmpc-chapso"></span>
<img src="../img/dmpc-chapso.png" alt="The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins' natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation." width="100%" />
<p class="caption">
Figure 1: The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins’ natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation.
</p>
</div>
<p>Before X-ray diffraction data collection, crystals were soaked with cryo-protectant solutions containing Ca<sup>2+</sup>, Mn<sup>2+</sup> or Cd<sup>2+</sup> of indicated concentrations. These ions travel through solvent channels within the crystals and chelate with amino acid residues in the selectivity filter region.</p>
<p>X-ray diffraction data were integrated and scaled with the HKL2000 packaged and further processed with the CCP4 package. THe structure of Ca<sub>v</sub>Ab and its derivatives were solved by molecular replacement by using an individual subunit of the Na<sub>v</sub>Ab structure (PDB code 3RVY) as the search template. This choice is natural since the proteins used in this study are derived from Na<sub>v</sub>Ab and differ from it by no more than 3 amino acid residues.</p>
<p>The divalent cations were identified by anomalous difference Fourier maps calculated using data collected at wavelengths of 1.75 Å for Ca<sup>2+</sup>, Cd<sup>2+</sup> and Mn<sup>2+</sup>. Anomalous scattering describes cases where a pair of structure factors <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> does not obey Friedel’s law<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, which occurs when the incident X-ray photons have an energy close to a transition energy of the diffracting atom, resulting in absorption of radiation energy and change in phase (normal scattering does not change the phase). At the wavelengths convinient for diffraction, only atoms heavier than phosphorus or sulfur behave as anomalous scatterers. While anomalous scattering data are frequently used to solve the phase problem (in techniques known as single/multiple wavelength anomalous dispersion (SAD/MAD)), it can also be used (as in this study) to calculate an anomalous difference map after phases are available to show the locations of the heavy atoms.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>Crystallography and NMR System software was used for refinement of coordinates and <em>B</em>-factors. Final models were obtained after several cycle refinement with REFMAC and PHENIX and manual re-building using COOT. The geometries of the final structural models of Ca<sub>v</sub>Ab and its derivatives were verified using PROCHECK.</p>
<p>13 crystal structures of 5 proteins (Na<sub>v</sub>Ab and 4 Ca<sub>v</sub>Ab variants) with the following divalent cation concentrations were determined and deposited into PDB:<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></p>
<ul>
<li>TLESWSM (Na<sub>v</sub>Ab) + Ca<sup>2+</sup> 15mM</li>
<li>(TLE<strong>D</strong>WSM, TLE<strong>D</strong>ES<strong>D</strong>, TL<strong>DD</strong>WSM) + Ca<sup>2+</sup> 15mM</li>
<li>TL<strong>DD</strong>WS<strong>D</strong> + (Ca<sup>2+</sup> 15mM; Mn<sup>2+</sup> 100mM; Cd<sup>2+</sup> 100mM)</li>
<li>TL<strong>DD</strong>WS<strong>N</strong> + Ca<sup>2+</sup> (0.5, 2.5, 5, 10, 15mM)</li>
</ul>
<p>Of the 13 crystal structure models, 4MVR (<sup>175</sup>TLDDWSD<sup>181</sup>) corresponds to the protein with the greatest permeability ratio <em>P</em><sub>Ca</sub>:<em>P</em><sub>Na</sub> , while <a href="https://www.rcsb.org/structure/4MS2">4MS2</a> (<sup>175</sup>TLDDWSN<sup>181</sup>) diffracted to the highest resolution (2.75 Å).</p>
</div>
<div id="analyses-of-results" class="section level1">
<h1>Analyses of Results</h1>
<div id="ca2-binding-sites" class="section level2">
<h2>Ca<sup>2+</sup> Binding Sites</h2>
<p>Using the anomalous diffraction data, the <strong>F</strong><sup>+</sup><sub>Ca</sub> - <strong>F</strong><sup>-</sup><sub>Ca</sub> anomalous difference map was calculated. Two strong peaks followed by a weaker peak were found along the ion-conduction pathway, which correspond to the three Ca<sup>2+</sup> binding sites. They are designated site 1, 2 and 3 from the extracellular side to the intracellular side.</p>
<p>Site 2 is the site with the highest affinity for Ca<sup>2+</sup>. It is surrounded by a total of 8 oxygen atoms, 4 of which coming from the carboxylate of D177 above and the other 4 from the carbonyl of L176 below. Site 1 is coordinated by the plane of 4 carboxyl groups from D178, and site 3 by the plane of 4 carbonyls from T175 (Figure <a href="#fig:cavab-schematic">2</a>). Throughout the selectivity filter, the O-Ca<sup>2+</sup> coordination distances are in the range of 4.0-5.0 Å, which is much longer than the ionic diameter of Ca<sup>2+</sup> (2.28 Å), suggesting that the bound Ca<sup>2+</sup> ion maintains its hydration shell while passing through the pore. Site 3 has the lowest affinity, consistent with its role in exit of Ca<sup>2+</sup> from the selectivity filter into the central cavity. Figure <a href="#fig:coot-4ms2">3</a> shows the molecular model and electron density near the selectivity fileter region of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong> + 15mM Ca<sup>2+</sup>) in COOT<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> (contoured at r.m.s.d = <span class="math inline">\(3.02 \sigma\)</span>).</p>

<div class="figure"><span id="fig:cavab-schematic"></span>
<img src="../img/cavab-schematic.png" alt="A schemetic showing the Ca2+ ion coordination sites of CavAb (TLDDWSD)" width="100%" />
<p class="caption">
Figure 2: A schemetic showing the Ca<sup>2+</sup> ion coordination sites of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>D</strong>)
</p>
</div>
<div class="figure"><span id="fig:coot-4ms2"></span>
<img src="../img/coot-4ms2.png" alt="The selectivity filter region of 4MS2." width="100%" />
<p class="caption">
Figure 3: The selectivity filter region of 4MS2.
</p>
</div>
<p>The relative affinities of the three sites were further confirmed by experiments on Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong>) with varying Ca<sup>2+</sup> concentrations (0.5, 2.5, 5, 10, 15mM): at low Ca<sup>2+</sup> concentration, two strong peaks of approximately equal intensity are found at Site 1 and Site 2; at high concentration the electron density is significantly enhanced in Site 2 and decreased in Site 1. The electron density at Site 3 remains low in all concentrations.</p>
</div>
<div id="ion-permeation-mechanism" class="section level2">
<h2>Ion-Permeation Mechanism</h2>
<p>Based on the properties of the three coordination sites, an ion-permeation mechanism can be deduced. The three coordination sites are separated by a distance of about 4.5 Å, which makes it energetically unfavourable for Ca<sup>2+</sup> to occupy adjacent sites simutaneously. Thus the authors suggested that the selectivity filter oscillates between two states, in which either a single hydrated Ca<sup>2+</sup> occupies Site 2, or two of them each occupies Site 1 and Site 3. The entry of Ca<sup>2+</sup> into Site 1 is promoted by it high extracellular concentration, and the exit of Ca<sup>2+</sup> is facilitated by the low affinity of Site 3.</p>
<p>Notably, this mechanism suggests that Ca<sup>2+</sup> ions are kept hydrated during its passage through the selectivity filter. This is very different from the mechanism by which potassium channels achive selectivity, which requires K<sup>+</sup> ions’ hydration shell to be removed.</p>
</div>
<div id="roles-of-key-selectivity-filter-residues" class="section level2">
<h2>Roles of Key Selectivity Filter Residues</h2>
<p>The five variants of Ca<sub>v</sub>Ab the author produced have different Ca<sup>2+</sup> selectivity ratios, which can be partially explained by directly comparing the difference in the arrangement of selectivity filter residues. This comparison is sometimes facilitated by superposition of one structure onto another. For example, the carboxyl group of D177 in TL<strong>DD</strong>WS<strong>D</strong> interacts with the Ca<sup>2+</sup> ion, while the carboxyl group of E177 in TLE<strong>D</strong>WS<strong>D</strong> swings away from the selectivity filter and forms a hydrogen bond with D181 and the main-chain nitrogen atoms of S180.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>specifically, from <sup>175</sup>TLESWSM<sup>181</sup> to TL<strong>DD</strong>WS<strong>D</strong>, TL<strong>DD</strong>WS<strong>N</strong>, TL<strong>DD</strong>WS<strong>M</strong>, TLE<strong>D</strong>WSM, TLE<strong>D</strong>WS<strong>D</strong> and TL<strong>D</strong>SWSM<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Friedel’s law states that <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> have the same amplitude but opposite phase angle.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>This map is automatically calculated in phenix.refine.<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>namely 4MS2, 4MTO, 4MTF, 4MTG, 4MVU, 4MW8, 4MVZ, 4MVM, 4MVO, 4MVQ, 4MW3, 4MVS, 4MVR<a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>COOT version 0.9.1 (packaged with CCP4), on Linux version 5.8.16-2-MANJARO<a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/assessing-structural-models.html" class="archive-item-link">
                Assessing The Quality Structural Models 
              </a>
              <span class="archive-item-date">
                2020-10-27
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#the-essence-of-x-ray-crystallography"><span class="toc-section-number">1</span> The Essence of X-Ray Crystallography</a></li>
<li><a href="#assessing-the-quality-of-a-structure-during-model-building-and-refinement"><span class="toc-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</a>
<ul>
<li><a href="#r-and-rfree"><span class="toc-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></a></li>
<li><a href="#reci"><span class="toc-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</a></li>
<li><a href="#fit"><span class="toc-section-number">2.3</span> Real-Space Refinement: Map Fitting</a></li>
</ul></li>
<li><a href="#assessing-quality-after-automatic-model-building-and-refinement"><span class="toc-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</a>
<ul>
<li><a href="#density-fit-analysis-and-local-geometry-validation"><span class="toc-section-number">3.1</span> Density fit analysis and local geometry validation</a>
<ul>
<li><a href="#structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles"><span class="toc-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</a></li>
<li><a href="#torsional-angles-and-ramachandran-plot"><span class="toc-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</a></li>
<li><a href="#undefined-regions"><span class="toc-section-number">3.1.3</span> Undefined regions</a></li>
<li><a href="#unexplained-density"><span class="toc-section-number">3.1.4</span> Unexplained Density</a></li>
<li><a href="#distortions-due-to-crystal-packing"><span class="toc-section-number">3.1.5</span> Distortions due to crystal packing</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>A high-quality protein structure model explains the experimental observations well, and conforms to physical and biochemical principles. There are some parameters indicating the quality of a structure model. Some of them are used during model building and refinement, while others apply only on a refined structure.</p>
<div id="the-essence-of-x-ray-crystallography" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Essence of X-Ray Crystallography</h1>
<p>The intensity at <span class="math inline">\((h, k, l)\)</span> in the reciprocal space, i.e. the structure factor amplitudes collected in X-ray experiments is directly related to the Fourier transform of the electron density in the real space (<span class="math inline">\(I = \mathbf{F}_{hkl}^2\)</span>), the latter being calculated as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \int_{x}\int_{y}\int_{z}\rho(x, y, z)e^{2\pi i(hx +ky + lz) }\mathrm{d}x \mathrm{d}y \mathrm{d}z
\end{equation}\]</span></p>
<p>In practice, since the number of atoms and reflections (hence <span class="math inline">\((x, y, z)\)</span> or <span class="math inline">\((h, k, l)\)</span> coordinates is limited, its discrete form, which can be efficiently computed, is used:<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \sum_{j}^{n}f_j e^{2\pi i(hx_j +ky_j + lz_j) }
\end{equation}\]</span></p>
<p>where <span class="math inline">\((x_j, y_j, z_j)\)</span> is the coordinates of the <em>j</em>-th atom in real space.</p>
<p>This is also known as the <em>structure factor equation</em>.</p>
<p>Solving the electron density from structure factors involves solving the inverse Fourier transform of the above equation, also known as the <em>electron density equation</em>, in its discrete form:</p>
<p><span class="math display">\[\begin{equation}
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}w_{hkl}|\mathbf{F}_\text{obs}|e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\alpha^\prime_{hkl}\)</span> is the phase at each set of coordinates in the reciprocal space. The phases are not directly available but are necessary (actually more important than intensities) for computing <span class="math inline">\(\rho(x,y,z)\)</span> (electron density).</p>
<p>The phases can be solved either by phasing experiments, in which case the resolution is relatively low but the data is unbiased, or by molecular replacement (MR), in which case the resolution is relatively high but the data suffer from bias (the phases resemble the search model more than the actual structure). Thus, bias minimisation measures such as maxinum likelihood map coefficients are applied before electron density map contrstruction.</p>
</div>
<div id="assessing-the-quality-of-a-structure-during-model-building-and-refinement" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</h1>
<p>Solving the electron density and ultimately the atomic structure from experimental data is an iterative process, starting from a model with low resolution and imperfect phases and progressively refining the model by applying prior knowledge on the information the current model can provide so far.</p>
<p>In general, the first step in refinement is to construct a crude electron density map (Equation (3)) using phase angles (<span class="math inline">\(\alpha^\prime\)</span>) obtained either from MR or experimental phasing. This initial map, <strong>F</strong><sub>0</sub>, is improved using techniques such as <em>density modification, solvent leveling, solvent flattening</em>. At some critical point in the iterative improvement of phases, the map becomes clear enough that a protein chain can be traced through it. From this point on, real-space refinement is also conducted at each round of iteration. Electron density map is obtained (details in Section <a href="#fit">2.3</a>), and filled with atoms, and transformed back to the reciprocal space. This process repeats until converging to a high-quality strucure. <span class="math inline">\(R\)</span> and <span class="math inline">\(R_\text{free}\)</span>, explained below, are used to assess progress in the refinement, and is also a major factor in evaluating the quality of a model.</p>
<div id="r-and-rfree" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></h2>
<p><em>R</em> is a measure of the deviation of calculated intensities from models (details in Section <a href="#reci">2.2</a>) from the observed intensities in the diffraction pattern, defined by the following equation:</p>
<p><span class="math display">\[
R = \dfrac{\sum ||\mathbf{F}_\text{obs}| - |\mathbf{F}_\text{calc}||}{\mathbf{F}_\text{obs}}
\]</span></p>
<p>Since bias can easily be introduced into the <em>R</em> value (especially by overparameterisation, see Section <a href="#reci">2.2</a>) and a reduction of <em>R</em> value sometimes does not improve the actual quality of structure (<span class="citation">Kleywegt and Brünger (n.d.)</span>), a small fraction (typically around 5%<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>) of randomly selected reflections are removed from the data used for refinement. These reflections can then be used to calculate an R factor, denoted as <span class="math inline">\(R_\text{free}\)</span>, whose reduction can be considered as an <em>unbiased estimate</em> of the improvement of the model.</p>
<div class="figure"><span id="fig:coot-phenix"></span>
<img src="../img/coot-phenix.png" alt=" R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine)." width="100%" />
<p class="caption">
Figure 2.1:  R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine).
</p>
</div>
<p>Figure <a href="#fig:coot-phenix">2.1</a> shows the decrease of <em>R</em> and <em>R</em><sub>free</sub> during a refinement task conducted in Phenix.</p>
</div>
<div id="reci" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</h2>
<p>Reciprocal-space refinement involves computerised attempts to improve agreement between <span class="math inline">\(\mathbf{F}_\text{obs}\)</span> and <span class="math inline">\(\mathbf{F}_\text{calc}\)</span> by <em>without</em> consideration of the maps and models. <em>Refinement by least squares</em> is the earliest successful technique and is discussed here.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>The goal of refinement by least squares is, find <span class="math inline">\((x_j, y_j, z_j)\)</span> for all atom <span class="math inline">\(j\)</span> whose <em>expected</em> ( i.e. computed) structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> are as close as possible to <em>observed</em> structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{obs}|\)</span>. Specifically, this means minimising the function <span class="math inline">\(\Phi\)</span>:</p>
<p><span class="math display">\[\begin{equation}
\Phi = \sum_{hkl}(w_{hkl}\mid\mathbf{F}_\text{obs}\mid - \mid\mathbf{F}_\text{calc}\mid)_{hkl}^{2}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(w_{hkl}\)</span> is the weight term that depends on the reliability of the corresponding measured intensity and <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> is a variant form of Equation (2) that can include additional parameters such as <em>B</em>-factor (<span class="math inline">\(B_j\)</span>) and occupancy <span class="math inline">\(n_j\)</span>. An equation with <span class="math inline">\(B_j\)</span> and <span class="math inline">\(n_j\)</span> included can be written as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_\text{calc} = \sum_{j}n_j f_j e^{2\pi i(hx_j +ky_j + lz_j) - B_j[(\sin\theta)/\lambda]^2}
\end{equation}\]</span></p>
<p>Note that the equatin shows that the effect of <em>B</em>-factors depends on the angle of reflection <span class="math inline">\((\sin\theta)/\lambda\)</span>.</p>
<p>Solving the minimum of <span class="math inline">\(\Phi\)</span> analytically is impractical, and instead numerical methods are used, which would lead to a minimum closest to the starting value.To prevent the refinement converging to a local minimum, it is important that the starting parameters be near the <em>global minimum</em>. Is also important <em>not</em> to include too many parameters (such as <em>B</em>-factor) at the initial stages of resolution when the resolution is low, as this would decrease the radius of convergence (Figure <a href="#fig:radius-convergece">2.2</a>).</p>
<div class="figure"><span id="fig:radius-convergece"></span>
<img src="../img/radius-convergece.png" alt="Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps." width="100%" />
<p class="caption">
Figure 2.2: Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps.
</p>
</div>
<p>To minimise the number of parameters used during early stages of refinement (and thus to increase radius of convergence), individual <span class="math inline">\((x,y,z)\)</span> coordinates are actually not used. Instead, only torional angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are allowed to change, and all bond lengths and angles are fixed to their theoretical average, side chains are assumed to be in their preferred conformation, and peptide linkages are fixed to be planar. This strategy is known as <em>restrained</em> reciprocal space refinement. As refinement proceeds, more parameters, from individual <span class="math inline">\((x,y,z)\)</span> coordinates to isotropic B-factors and finally anisotropic factors, can be added into calculation.</p>
</div>
<div id="fit" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Real-Space Refinement: Map Fitting</h2>
<p><em>Map fitting</em> or <em>model building</em> entails building a molecular model that fits realistically into the current electron density contour map.</p>
<p>To reduce the bias (towards <strong>F</strong><sub>calc</sub> (<strong>F</strong><sub>c</sub>)) when constructing the electron density map, Fourier syntheses of <strong>F</strong><sub>obs</sub> and <strong>F</strong><sub>calc</sub> are used.
A Fourier synthesis <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> is calculated as:</p>
<p><span class="math display">\[
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}(m|\mathbf{F}_\text{o}|-|\mathbf{F}_\text{c}|)e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\]</span></p>
<p>and its corresponding electron density map is called an <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> map.</p>
<p>Simply put, the 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map resembles a molecular surface, and a <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map emphasises the error (positive density implies that the unit cell contains more electron density in this region than implied by the model (<strong>F</strong><sub>c</sub>). Near the end of refinement, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map becomes rather empty except in problem areas, which may need to be corrected manually.</p>
<p>Fitting a molecular model into the electron density map depends on prior knowledge, such as average bond lengths and angles, the amino acid sequence of the protein, properties of peptide chains, etc. For example, we know that carboxyl oxygens in adjacent amino acid residues in a <span class="math inline">\(\beta\)</span>-sheet point in opposite directions. Thus, once a <span class="math inline">\(\beta\)</span>-sheet along with one or two carboxyl oxygen are discernible, we can make a sensible guess of the positions of all other carbonyl oxygens.</p>
</div>
</div>
<div id="assessing-quality-after-automatic-model-building-and-refinement" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</h1>
<div id="density-fit-analysis-and-local-geometry-validation" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Density fit analysis and local geometry validation</h2>
<p>Automatic model building and refinement use the decrease of <em>R</em> value as an indicator of progress and terminates when <em>R</em> is considered to be sufficiently low. This may lead to situations where the global <em>R</em> is favourable but local geometry can still be improved.</p>
<p>Local geometry validation programs, such as “Density fit analysis” in Coot (Figure <a href="#fig:coot-density-fit-2">3.1</a>), evaluate the model geomtry on a per-residue basis and flag outliers. These outliers can then be fixed manually. With the aid of electron density contour maps (where model atoms lie outside 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> contours, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> will often show the atoms with negativel contours, with nearby positive contours pointing to correct locations for these atoms).</p>
<div class="figure"><span id="fig:coot-density-fit-2"></span>
<img src="../img/coot-density-fit-2.png" alt="Use the 'Density fit analysis' function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the 'Rotamers' tool to fix a side chain that's pointing the wrong way." width="100%" />
<p class="caption">
Figure 3.1: Use the ‘Density fit analysis’ function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the ‘Rotamers’ tool to fix a side chain that’s pointing the wrong way.
</p>
</div>
<div id="structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</h3>
<p>During late stages of refinement, restraints on structura paramaters such as bond lengths and angles are release such that thay are allowed to change freely as long as <em>R</em> values could be made to decrease. This may lead to unrealistic models with improbable structural parameters. Thus, they need to be validated. By convention, this is done by calculating the root-mean-square deviations (RMSD) of these parameters in the model from the corresponding set of values based opon the geometry of small organic molecules. These values are recorded in PDB files in <code>REMARK 3</code> fields:</p>
<pre><code>REMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       
REMARK   3                 RMSD          COUNT                                  
REMARK   3   BOND      :  0.003           1366                                  
REMARK   3   ANGLE     :  0.675           1846                                  
REMARK   3   CHIRALITY :  0.050            186                                  
REMARK   3   PLANARITY :  0.005            246                                  
REMARK   3   DIHEDRAL  : 15.473            459                                  </code></pre>
<p>Because bond angles, lengths, chirality and planarity are restrained in early stages of refinement, they are less likely to deviate much from ideal values. Dihedral angles, which is allowed to change throughout the refinement, produce much larger RMSD and are worth closer examination.</p>
</div>
<div id="torsional-angles-and-ramachandran-plot" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</h3>
<p>Torsional (dihedral) angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are show much more variation than bond lengths and angles, but only a subset of all possible (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs are allowed so that adjacent amino acid side chains do not clash. Validation of torsional angle is achieved via a lookup table, where the keys are (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs and values are scores. A (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) is considered preferred or allowed if its score is within certain thresholds. Otherwise, it is considered an outlier. Due to glycine’s small size and proline’s cyclic structure, the preferred/allowed regions of their torsinal angle pairs are defined differently, for example in Phenix/cctbx<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>. Torsional angle validation is often visualised with a Ramachandran plot, as shown in Figure <a href="#fig:coot-ramachandran">3.2</a>.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></p>
<div class="figure"><span id="fig:coot-ramachandran"></span>
<img src="../img/coot-ramachandran.png" alt="Validating dihedral angles with Ramachandran plot in Coot" width="100%" />
<p class="caption">
Figure 3.2: Validating dihedral angles with Ramachandran plot in Coot
</p>
</div>
</div>
<div id="undefined-regions" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Undefined regions</h3>
<p>Often, some regions (often the terminal regions) of the protein are highly disordered and produce no detectable diffraction at high resolution and are thus invisible in the electron-density maps. It is not uncommon to omit these residues in the model. In some cases, however, exact coordinates are given for these atoms, along with very high <em>B</em>-factors (above 50 or 60 angstroms), and one must be careful not to over-interprete these regions.</p>
</div>
<div id="unexplained-density" class="section level3" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> Unexplained Density</h3>
<p>Empty electron density may remain after all known contents of the unit cell have been located. This can either due to an artifect of missing Fourier terms, reagents used in purification or crystallisation, or it could be due to previously unknown ligands (cofactors, inhibitors or allosteric effectors).</p>
</div>
<div id="distortions-due-to-crystal-packing" class="section level3" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Distortions due to crystal packing</h3>
<p>Although flexibility of atoms is usually reflected by the <em>B</em>-factor, in certain situations dynamic regions of a protein molecule can be rigidly fixed in a specific conformation as a result of crystal packing interactions. It should be declared in the PDB header that the protein is affected by crystallisation. One can also check whether these interactions are likely to occur by displaying all neighbouring, symmetry-related molecules in the crystal structure and examine if any intermolecular interactions are present that are a result of crystal packing.</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Blow-2002">
<p>Blow, David. 2002. <em>Outline of Crystallography for Biologists</em>. Oxford University Press.</p>
</div>
<div id="ref-Kleywegt-1996">
<p>Kleywegt, G J, and A T Brünger. n.d. “Checking Your Imagination: Applications of the Free R Value.” <em>Structure</em> 4 (8): 897–904. <a href="https://doi.org/10.1016/s0969-2126(96)00097-4">https://doi.org/10.1016/s0969-2126(96)00097-4</a>.</p>
</div>
<div id="ref-rhodes-2006">
<p>Rhodes, Gale. 2006. <em>Crystallography Made Crystal Clear Clear</em>. Academic Press.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Most crystallography softwares seem to use Fast Fourier Trasnform (FFT) algorithms provided by FFTW (<a href="http://www.fftw.org/" class="uri">http://www.fftw.org/</a>) to compute discrete fourier transforms (and its inverse)<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>In PDB files, this fraction is recorded in the field <code>FREE R VALUE TEST SET SIZE</code><a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>modern refinement softwares use techniques such as simulated annealing and beyesian methods to improve the accuracy of refinement<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>In Phenix/cctbx, a lookup table (actually implemented as an array for better performance) is defined for each of the following cases: glycine, cis-proline, trans-proline, pre-proline, isoleucine/valine, and all other amino acid residues: <a href="https://github.com/cctbx/cctbx_project/blob/master/mmtbx/validation/ramachandran/rama8000_tables.h"><code>mmtbx/validation/ramachandran/rama8000_tables.h</code></a><a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>Recently I’ve been developing a PDB file parser. It is now able to parse coordinate information reliably, which allows Ramanchandran plot analysis: <a href="https://github.com/TianyiShi2001/protein" class="uri">https://github.com/TianyiShi2001/protein</a><a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/download-and-sync-with-the-entire-pdb-database.html" class="archive-item-link">
                Download and Sync with the Entire PDB Database 
              </a>
              <span class="archive-item-date">
                2020-10-17
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>All the PDB files are <a href="https://www.rcsb.org/pages/download/ftp">available via PDB’s FTP server</a>. Simply <code>cd</code> to a directory and use <code>wget</code> to download all of them.</p>
<pre class="bash"><code>mkdir pdb; cd pdb; mkdir zipped unzipped; cd zipped
wget ftp://ftp.wwpdb.org/pub/pdb/data/structures/all/pdb/*</code></pre>
<p>Depending on the traffic, The first download will take 2~4 days. After the first download, to sync the local database with the FTP server, add the <code>-N</code> option to <code>wget</code> (essentially this means downloading “new” files only).</p>
<p>The <code>*.ent.gz</code> files downloaded are zipped. You can use <code>gunzip</code> to decompress them to ready-to-use <code>*.ent</code> files (which is written in PDB format), but that will remove the <code>*.gz</code> files, which is inconvenient if you want to keep your database in sync with the remote. Use <code>gzcat</code> (or equivalently <code>gunzip -c</code>) instead to preserve the <code>*.gz</code> files while decompressing:</p>
<pre class="bash"><code>find . -name &quot;*.gz&quot; | xargs -I{} -n1 bash -c &#39;
src={}; dst=../unzipped/$(basename ${src%*.gz})
[ -f $dst ] &amp;&amp; echo &quot;$dst already exists, skipping...&quot; || ( gzcat -cv $src &gt; $dst )&#39;</code></pre>
<p>You can monitor the progress from the verbose output of <code>gzcat</code>:</p>
<pre><code>./pdb1byf.ent.gz:      78.1%
./pdb1byg.ent.gz:      76.3%
./pdb1byh.ent.gz:      77.3%
./pdb1byi.ent.gz:      75.2%
...</code></pre>
<script src="https://gist.github.com/TianyiShi2001/46fcc143b104cd47c7c768f958da2555.js"></script>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/sequence-alignment-gap-penalty.html" class="archive-item-link">
                Sequence Alignment: Gap Penalty 
              </a>
              <span class="archive-item-date">
                2020-08-29
              </span>
              <div class="article-summary" style="display: none;">
                
                




              </div>
            </article>
          
            <article class="archive-item">
              <a href="../bioinformatics/fasta-parser-implemented-in-rust-with-nom.html" class="archive-item-link">
                FASTA Parser Implemented in Rust with Nom 
              </a>
              <span class="archive-item-date">
                2020-08-23
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#parsing-a-single-fasta-record"><span class="toc-section-number">1</span> Parsing a single FASTA Record</a>
<ul>
<li><a href="#representing-the-record-in-rust"><span class="toc-section-number">1.1</span> Representing the Record in Rust</a></li>
<li><a href="#parsing-the-title"><span class="toc-section-number">1.2</span> Parsing the Title</a></li>
</ul></li>
</ul>
</div>

<p>Nom makes it possible to</p>
<div id="parsing-a-single-fasta-record" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Parsing a single FASTA Record</h1>
<p>An example of FASTA record is shown below.</p>
<pre><code>&gt;gi|2765658|emb|Z78533.1|CIZ78533 C.irapeanum 5.8S rRNA gene and ITS1 and ITS2 DNA
CGTAACAAGGTTTCCGTAGGTGAACCTGCGGAAGGATCATTGATGAGACCGTGGAATAAACGATCGAGTG
AATCCGGAGGACCGGTGTACTCAGCTCACCGGGGGCATTGCTCCCGTGGTGACCCTGATTTGTTGTTGGG
CCGCCTCGGGAGCGTCCATGGCGGGTT

&gt;(another FASTA record)</code></pre>
<p>Each record starts with a <code>&gt;</code> character, which is followed by the title on the same line. The title can be further divided into an ID and description, which is separated by a space character. In this case, the ID is ‘gi|2765658|emb|Z78533.1|CIZ78533’ and the description is ‘C.irapeanum 5.8S rRNA gene and ITS1 and ITS2 DNA’.</p>
<p>The sequence starts from the second line. There can be line breaks within the sequence.</p>
<div id="representing-the-record-in-rust" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Representing the Record in Rust</h2>
<p>When we parse data, we <em>parse it into something that is meaningful in that language</em>. In Rust, users can define custom <code>struct</code> and <code>enum</code> data structures, and in this case, a C-like <code>struct</code> is the most appropriate.</p>
<p>To make it simple, we define one two fields, <code>title</code> and <code>seq</code>:</p>
<pre class="rust"><code>fn main(){
#[derive(Debug)]struct Record {  title: String,  seq: String}
}</code></pre>
</div>
<div id="parsing-the-title" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Parsing the Title</h2>
<p>To make it</p>
</div>
</div>

              </div>
            </article>
          
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="../tutorial/antibiotics-and-protein-synthesis.html" class="archive-item-link">
                Antibiotics and Protein Synthesis 
              </a>
              <span class="archive-item-date">
                2019-11-30
              </span>
              <div class="article-summary" style="display: none;">
                
                  Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.<hr>
                
                


<p><strong>Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Protein synthesis occurs in ribosomes and involves 3 phases: initiation, elongation (which is subdivided into 4 steps) and termination. Each phase depends on the correct conformation and/or catalytic activity of different domains of the ribosomal complex and soluble protein factors (IFs, EFs and RFs). The identities and mechanisms of these critical RNA/protein domains (in bacterial cells) can be investigated by genetic mutation and antibiotics, which hinder their normal functions, along with recent advances in biophysical methods, which allows high resolution structures to be determined for ribosomes in different contexts.</p>
</div>
<div id="background-information" class="section level1">
<h1>Background information</h1>
<div id="the-ribosome" class="section level2">
<h2>The Ribosome</h2>
<p>The bacterial 70S ribosome comprises two subunits, the 50S (large) and the 30S (small) subunit. The 30S subunit mediates selection of cognate aminoacyl tRNAs and the 50S subunit contains the peptidyl-transferase centre (PTC), which catalyses addition of new amino acids to the elongating polypeptide chain.</p>
</div>
<div id="the-process-of-protein-synthesis" class="section level2">
<h2>The process of protein synthesis</h2>
<p>The 3 phases of protein synthesis are initiation, elongation and termination.</p>
<p>During initiation, the 3’ end of the 16S rRNA of the 30S subunit base-pairs with the Shine-Dalgarno sequence, which positions the start codon (usually AUG) in the P site for the binding of the initiator tRNA (usually fMet-tRNA) and IF1, 2, 3. Joining of the 50S subunit and dissociations of IFs primes the elongation phase.</p>
<p>The elongation phase, where new amino acids are appended to the growing polypeptide, involves 4 steps:</p>
<ol style="list-style-type: decimal">
<li><strong>Decoding.</strong> An aminoacylated tRNA (aa-tRNA) is delivered to the A-site of the ribosome by elongation factor EF-Tu complexed with GTP. GTP hydrolysis facilitates discrimination between cognate and non-cognate tRNAs.</li>
<li><strong>Peptidyl transfer.</strong> The peptidyl-transferase centre (PTC) on the 50S subunit catalyses the formation of peptide bond between the amino acids attached to the tRNAs in the A- and P-sites by transferring the polypeptide chain from the P-site tRNA to the aa-tRNA in the A-site.</li>
<li><strong>Translocation.</strong> EF-G catalyses the movement tRNAs from the A and P sites to the P and E sites</li>
<li>the growing polypeptide chain passes through an exit tunnel</li>
</ol>
<p>Finally, when the stop codon (UAA/UAG/UGA) is encountered, it is recognised by release factors (RF1/RF2) that forces hydrolysis of the peptidyl-tRNA bond in the P-site, thus releasing the polypeptide chain from the ribosome.</p>
</div>
</div>
<div id="mutations" class="section level1">
<h1>Mutations</h1>
<p>Mutation in different components of protein synthesis (tRNA, ribosomal proteins, rRNAs) can all lead to decreased fidelity of this process.</p>
<p>The tRNA needs to undergo a structual transition that involves a 30<span class="math inline">\(^\circ\)</span> bend in order to achieve the A/T (decoding) conformation. Mutagenesis studies showed that two mutations in the D-stem, A9C and G24A, promote miscoding and these two bases are critical for distortion of the cognate tRNA that is required for decoding.</p>
<p>Mutagenesis studies also help to identify ribosomal proteins that are critical in the decoding process. The mutations can be classified into two types according to their implication on the speed and accuracy/robustness of translation: <em>ram</em> (ribosome ambiguity) mutations improve speed but has a higher frequency of failure in rejecting non-cognate tRNAs; in contrast, ribosomes with <em>str</em> (stringency) mutations are slower and more accurate, but they reject some cognate aa-tRNAs and thus waste GTP. Most of such mutations are found in the ribosomal proteins S12, S5 and S4, which are close to the decoding site, and these proteins have an important effect on codon/anticodon interactions.</p>
</div>
<div id="antibiotics" class="section level1">
<h1>Antibiotics</h1>
<p>The antibiotic puromycin can be used to demonstrate the existence of A and P sites in the ribosome and the ATP/GTP-independent nature of the peptidyl transfer reaction.</p>
<p>Puromycin (PM) is an aa-tRNA analogue which is composed of an nucleoside and an bound amino acid. Unlike the normal aa-tRNA where the 5’ OH of the nucleoside (adenosine) is linked to the rest of the tRNA, PM has a free 5’ OH. Therefore, PM can fit into the peptidyl transfer centre (PTC) and accept the polypeptide chain in the peptidyl tranfer reaction, but the new peptide with PM attached is released immediately because PM does not have an attached tRNA to anchor it to the ribosome.</p>
<p>The details of the experiment is shown below and in Figure <a href="#fig:puromycin">1</a>:</p>
<ul>
<li>Take a cell free system containing: ribosomes, Mg<sup>2+</sup>-containing buffer, poly-U mRNA, <sup>14</sup>C-Phe-tRNA, ATP/GTP-regenerating system and add varying concentrations of soluble cell extract.</li>
<li>This yields ribosomes carrying polyPhe-tRNA (not free polyPhe or poly-Phe-tRNA unbound to ribosomes). [with prior knowledge, this is because there are no stop codons on the mRNA and hence RFs are not recruited to terminate translation]</li>
<li>the ribosomes are washed to remove soluble proteins and ATP/GTP.</li>
<li>then they are incubated with puromycin (PM)</li>
<li>about 50% of the ribosomes release <sup>14</sup> labelled polyPhe as polyPhe-PM [As shown in Figure <a href="#fig:puromycin">1</a>, the convertion from state 2 to 3 and from state 3 to 1 requires GTP, and the convertion from state 1 to 2 (peptidyl transfer) does not. Therefore, when GTP is removed, protein synthesis is either trapped in state 2 or 3]</li>
<li>without a tRNA, PM cannot bind to the ribosome (at A/P sites) so the compound with polyPhe is released</li>
</ul>
<p>Conclusion: ribosome itself catalyses the formation of the peptide bond without energy supplying molecules (ATP/GTP).</p>
<ul>
<li>now wash the ribosomes to remove the PM</li>
<li>add soluble cell extract and GTP, then more PM</li>
<li>the remaining 50% ribosomes release polyPhe-PM</li>
</ul>
<p>Conclusion:</p>
<ol style="list-style-type: decimal">
<li>ribosomes have a ‘PM-reactive’ site (P site) and a ‘PM-unreactive site’ (A site)</li>
<li>soluble cell extract can translocate from RM-unreactive to PM-reactive site (driven by EF-G)</li>
</ol>
<div class="figure"><span id="fig:puromycin"></span>
<img src="../img/puromycin.jpg" alt="The puromycin experiment." width="100%" />
<p class="caption">
Figure 1: The puromycin experiment.
</p>
</div>
</div>
<div id="summary" class="section level1">
<h1>Summary</h1>
<p>Mutagenesis and ribosome-targeting antibiotics are valuable tools for studying the mechanisms of protein synthesis in the ribosome. These methods are usually aided by cryo-EM and X-ray crystallography to probe into the 3D molecular details.</p>
<p>Mutations in tRNA, rRNA and associated proteins give information on their specific roles and behaviours during protein synthesis. Antibiotics with various (and usually predictable) chemical properties is another way of studying the mechanisms of protein synthesis. Apart from the puromycin (which targets the peptidyl transfer centre) mentioned in this essay, there is a wide range of antibiotics that target different steps of protein synthesis. For example, streptomycins interferes with the delivery of tRNAs to the A-site and neomycin hinders translocation following peptidyl transfer. High resolution crystal structures were obtained for many of these antibiotics in complex with the ribosome (or part of ribosome). These structures are indicative of functionally important sites in the ribosome and they often shed light on the mechanisms.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/rna-splicing.html" class="archive-item-link">
                RNA Splicing 
              </a>
              <span class="archive-item-date">
                2019-11-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What are the roles of small nuclear RNAs (SnRNAs) in pre-mRNA splicing and what are the mechanisms that control alternative splicing?<hr>
                
                


<div id="overview" class="section level1">
<h1>Overview</h1>
<p>RNA splicing by spliceosomes removes introns from nascent pre-mRNA and joins protein-coding exon sequences, which matures the mRNA and makes it ready for translation. By chosing variable splice sites, different mature mRNAs (and hence proteins) can be produced from the same pre-mRNA.</p>
</div>
<div id="structure-of-the-intron" class="section level1">
<h1>Structure of the Intron</h1>
<p>Splice sites (i.e. exon-intron junctions) have conserved features. As shown in Figure <a href="#fig:intron">1</a>, most introns have an invariant GU at the 5’ end (5’ splice site) and AG at the 3’ end (3’ splice site), their flanking sequences are variable but the bases are found at frequencies higher than expected based on a random distribution. The branch point A is also invariant, and most introns also have a polypyrimidine tract between the branch point and the 3’ splice site. These ‘standard’ pre-mRNAs are spliced by snRNPs U1, U2, U4, U5, and U6. A small fraction of pre-mRNA does not follow the ‘GU-AG’ rule and have 5’-AU and AC-3’ instead, which are spliced by four low-abundance snRNPs together with U5.</p>
<div class="figure"><span id="fig:intron"></span>
<img src="../img/intron.jpg" alt="The structure of a typical intron" width="100%" />
<p class="caption">
Figure 1: The structure of a typical intron
</p>
</div>
</div>
<div id="spliceosome-assembly-and-the-splicing-process" class="section level1">
<h1>Spliceosome Assembly and the Splicing Process</h1>
<p>The spliceosome is a large and dynamic RNP machine whose size is comparable to a ribosome. The 5 snRNPs that constitute the spliceosome are U1, U2, U4, U5, and U6, each of which is a association of an snRNA with proteins. The key catalytic activity is attributed to RNA, so the spliceosome is an ribozyme. A spliceosome is assembled whenever an intron is to be removed and its components are recycled after splicing. The series of events are shown in Figure 2.</p>
<div class="figure"><span id="fig:spliceosome-assembly"></span>
<img src="../img/spliceosome-assembly.jpg" alt="Spliceosome assembly and the splicing process" width="100%" />
<p class="caption">
Figure 2: Spliceosome assembly and the splicing process
</p>
</div>
<ol style="list-style-type: decimal">
<li>U1 base-pairs with the 5’ splice site; SF1 (splicing factor 1) binds to the branch-point A; U2AF (U2 auxilliary factor) associates with the polypyrimidine tract and 3’ splice site. Then, U2 replaces SF1 and U2AF.</li>
<li>The U4/U6.U5 tri-snRNP complex joins, forming the spliceosome</li>
<li>Rearrangement of base-paring interactions leads to the catalytically active conformation. U6 dissociates from U4 and replaces U1. U4 is also released.</li>
<li>The catalytic core of U6/U2 catalyses the first transesterification reaction (ligating 5’ guanosine to the 2’ OH of the branch point adenosine)</li>
<li>Further rearrangements take place, and the second transesterification reaction occurs (joining the two exons)</li>
</ol>
<p>The two sequential transesterification reactions <em>per se</em> do not require energy input, as there is no net bond breaking (two phosphodiester bonds are hydrolysed and another two are formed), but ATP/GTP hydrolysis is used to drive structural transitions and as a ‘proofreading’ mechanism.</p>
<p>As shown in Figure <a href="#fig:splice-detailed-mech">3</a>, In the catalitically active state, U6 partially base-pairs with U2 and U6 forms an internal stem loop structure with three nucleotides coordinated to two Mg<sup>2+</sup> ions. Mg<sup>2+</sup> ions stabilise the transitions states of the two transesterification reactions and thus achieve catalysis. The key bases coordinated to Mg<sup>2+</sup> are confirmed by substitution of oxygen for sulfur atoms at various locations.</p>
<div class="figure"><span id="fig:splice-detailed-mech"></span>
<img src="../img/splice-detailed-mech.jpg" alt="The transesterification reactions in detail" width="100%" />
<p class="caption">
Figure 3: The transesterification reactions in detail
</p>
</div>
</div>
<div id="alternative-splicing" class="section level1">
<h1>Alternative Splicing</h1>
<p>pre-mRNAs contain <em>cis</em>-acting regulatory sequences (either splicing enhancers or silencers) to which splicing factors can bind and thus regulate splicing (Figure <a href="#fig:alternative-splicing">4</a>). RNA-RNA base pairing, which can occur either in <em>cis</em> or <em>trans</em>, can also specify splice site use.</p>
<p>RNA-binding proteins/RNPs that regulate splicing can be classified into 3 classes: the classical hnRNPs, serine-arginine repeat (SR) proteins, and tissue-specific RNA-binding proteins. Generally, hnRNPs inhibit splicing from nearby splice sites by interacting with splicing silencers while SR proteins promotes splicing by binding to enhancers.</p>
<div class="figure"><span id="fig:alternative-splicing"></span>
<img src="../img/alternative-splicing.jpg" alt="Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers." width="100%" />
<p class="caption">
Figure 4: Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers.
</p>
</div>
<p>RNA-RNA recognition can also control splice site choice, as exemplified by the exon 6 cluster of the <em>Drosophila</em> DSCAM gene, as shown in Figure <a href="#fig:alternative-splicing-rna-rna">5</a>.</p>
<div class="figure"><span id="fig:alternative-splicing-rna-rna"></span>
<img src="../img/alternative-splicing-rna-rna.jpg" alt="hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream." width="100%" />
<p class="caption">
Figure 5: hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream.
</p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/nmr.html" class="archive-item-link">
                NMR 
              </a>
              <span class="archive-item-date">
                2019-11-16
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p><strong>Write brief notes on the chemical shift, the nuclear Overhauser enhancement effect (NOE), the relaxation times T1 and T2, and the spin-spin coupling constant (J) with an emphasis on how these can provide information about the structure and dynamics of biological molecules.</strong></p>
<ul>
<li>Chemical Shift: NMR applies RF wave to induce resonance with the precession frequency, <span class="math inline">\(\omega=\gamma (B_0-B_S)\)</span>, where <span class="math inline">\(B_0\)</span> is the constant magnetic field strength on the <span class="math inline">\(z\)</span> axis, <span class="math inline">\(B_S\)</span> is the magnetic field strength shilded by electrons around the nucleus and <span class="math inline">\(\gamma\)</span> is the magnetogyric ratio (depending of the identity of the nucleus). Nuclei in different chemical environment have different <span class="math inline">\(B_S\)</span> and thus different <span class="math inline">\(\omega\)</span>, and the chemical shift, <span class="math inline">\(\delta\)</span>, is defined as <span class="math display">\[\delta=\dfrac{\nu - \nu_\text{reference}}{\nu_\text{o}}\times10^6\]</span> When a protein is folded, the chemical environment of protons is highly variable and the NMR signal is noisy and has many peaks. In contrast, when a protein is unfolded, the chemical shift tend to be less variable, and fewer peaks with greater intensities are observed.</li>
<li>NOE: When two nuclei are close (&lt; 5-6 angstroms) to each other, selective irradiation of one spin by <span class="math inline">\(B_1\)</span> field causes intensity changes of the other spin. Two dimensional methods detect NOE more easily. NOE is the main source of information for macromolecular structure determination by NMR.</li>
<li>T<sub>1</sub> &amp; T<sub>2</sub> relaxation: The applied RF wave creates <span class="math inline">\(B_1\)</span> field which transiently reorient the net magnetisation, <span class="math inline">\(M\)</span>. The decay rate of <span class="math inline">\(M_{xy}\propto\dfrac{1}{\text{T}_2}\)</span>; decay rate of <span class="math inline">\(M_z\propto\dfrac{1}{\text{T}_1}\)</span>. T<sub>1</sub> relaxation is along the <span class="math inline">\(B_0\)</span> field (longitudinal) and T<sub>2</sub> relaxation is in the <span class="math inline">\(xy\)</span> plane (transverse). Relaxation rates give information about molecular dynamics and distances. T2 can be measured from the decay rate of the FID and linewidths and T1 can be measured by an inversion-recovery experiment.</li>
<li>Spin-spin coupling constant: The spin states of two protons on two covalently bonded atoms affect the nuclear energy levels of each other, so the peak split in a neighbour-dependent way (<span class="math inline">\(n\)</span> adjacent protons give <span class="math inline">\(n+1\)</span> multiplets). Coupling strength gives bond-angle information for biomolecules. For example, splitting of H<sup>N</sup> due to H<span class="math inline">\(^\alpha\)</span> (<span class="math inline">\(^3J_{\text{NH}\alpha\text{CH}}\)</span>) depends on <span class="math inline">\(\Phi\)</span> angle.</li>
</ul>
<p><strong>Describe how NMR can be used to measure the pK values of ionizable groups in proteins. Why is chemical shift sensitive to ionization state? How can <sup>31</sup>P NMR be used to measure intracellular pH? Over what pH range is this effective?</strong></p>
<ul>
<li>The chemical shift a proton depends on its chemical environment, which includes the protonation state of adjacent acidic or basic sites. Therefore, gradual alteration of pH causes changes in the chemical shifts that can be plotted against pH. The p<span class="math inline">\(K_\text{a}\)</span> is the pH that corresponds to the inflection point of the resulting sigmoidal curve.</li>
<li><em>In vivo</em> phosphate concentration is about constant. In pH range 5-8, the two dominant forms of phosphate, <span class="math inline">\(\text{H}_2\text{PO}_4^-\)</span> and <span class="math inline">\(\text{HPO}_4^{2-}\)</span> have different chemical shifts for <sup>31</sup>P in NMR, so the chemical shift can be related to pH: at higher pH, the <span class="math inline">\(\text{HPO}_4^{2-}\)</span> form is dominant, and its phosphate is more deshielded, leading to larger chemical shift</li>
</ul>
<p><strong>How does multidimensional NMR differ from basic 1-D NMR? What type of information is contained in the 2-D NMR experiments known as NOESY, TOCSY and HSQC?</strong></p>
<p>As shown in Figure <a href="#fig:2d3dnmr">1</a>:</p>
<ul>
<li>The 1D spectrum is obtained by Fourier transformation of a single transient decaying signal. By collecting a series of transients separated by incrementing lengths of ‘evolution time’, <span class="math inline">\(t_1\)</span>, a 2D spectrum can be obtained. Similarly, 3D spectra are made with an additional period of ‘evolution time’, <span class="math inline">\(t_2\)</span>. Multidimensional NMR clearly shows connections (either <em>J</em>-coupling or NOE) between nuclei as ‘cross peaks’.</li>
<li>COSY (correlated spectroscopy) spectra detect through-bond correlations (<em>J</em>-couplings)</li>
<li>NOESY spectra has an extra 90<span class="math inline">\(^\circ\)</span> pulse and a fixed delay, <span class="math inline">\(\tau_m\)</span> inserted in the sequence and this allows detection of nuclear Overhauster effect (NOE).</li>
<li>HSQC (heteronuclear single quantum coherence spectroscopy) gives a 2D spectrum with one axis for <sup>1</sup>H and another for a heteroatom (usually <sup>15</sup>N or <sup>13</sup>C).</li>
</ul>
<div class="figure"><span id="fig:2d3dnmr"></span>
<img src="../img/2d3dnmr.jpg" alt="Schematic of 1D, 2D and 3D NMR, and sample " width="1993" />
<p class="caption">
Figure 1: Schematic of 1D, 2D and 3D NMR, and sample
</p>
</div>
<p><strong>Outline the application of NMR spectroscopy to the determination of protein 3-D structure in solution. What are the advantages and disadvantages of this technique compared to X-ray crystallography?</strong></p>
<ul>
<li>Application of NMR
<ul>
<li>Experimental restraints: structure-related data derived from NMR spectra
<ul>
<li><sup>1</sup>H-<sup>1</sup>H NOEs: each NOESY cross peak gives a specific pairwise distance information</li>
<li>coupling constants <span class="math inline">\(J\)</span> values can help to determine some dihedral angles</li>
<li>chimical shifts: can be used to search in the database</li>
<li>residual dipolar coupling: gives information about the direction of specific groups with respect to B<sub>0</sub>; gives long-range order information which complements the short-range information given by NOEs.</li>
<li>solvent H-exchange: slow H-D exchange suggests that the proton is in a defined region of secondary structure</li>
</ul></li>
<li>Calculation of structures is done by incorporating various experimental restraints into a molecular dynamics (MD) simulation protocol. MD calculates position and velocity of atoms in a series of small steps using Newtonian mechanics. Extra energy terms are added to take account of observed constraints</li>
</ul></li>
<li>Compared to X-ray crystallography:
<ul>
<li>Disadvantages:
<ul>
<li>NMR is relatively laborious and slow</li>
<li>restricted to small proteins</li>
</ul></li>
<li>Advantages:
<ul>
<li>No need for crystallization; only a pure solution is needed. Can be used on proteins that do not crystallize (e.g. those containing intrinsically unstructured regions)</li>
<li>can be used to study dynamics</li>
<li>site-specific info on ligand binding</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p><strong>Describe methods to monitor protein-folding pathways during the formation of a globular protein from an extended chain. Place particular emphasis on the role of NMR but also show how other methods provide complementary information. </strong></p>
<div id="nmr" class="section level2">
<h2>NMR</h2>
<p>Protein folding is a multistep process that proceeds through intermediate states and studying these intermediates is difficult due to their transient nature, low populations under non-denaturing conditions, and difficulty of their isolation.</p>
<p>Over the past two decades, many NMR methods have been developed and some became invaluable tools in studying molecular details of protein folding. Tansverse relaxation optimized spectroscopy (TROSY) is the first method developed for probing of dynamics and interactions of large (up to 1MDa) protein assemblies. Relaxation dispersion (RD) and saturation transfer (ST) methods provide a detailed look into the pathways of biomolecular processes, allowing studies of transient intermediates during protein folding.</p>
<p>RD and ST methods allow studying the minor nonnative conformations during protein folding with short lifespans (often <span class="math inline">\(\mu\)</span>s-ms). The two major types of RD experiments include (1) Carr-Purcell-Meiboom-Gill (CPMG) methods that exploit modulation of R<sub>ex</sub> by a sequence of evenly spaced refocusing pulses, and (2) rotating frame <span class="math inline">\(\text{R}_{1\rho}\)</span> relaxation experiments that use modulation of R<sub>ex</sub> by an on- or off-resonance continuous wave (CW) RF field. The ST experiments exploit modulation of R<sub>ex</sub> by a weak RF field, and are conceptually similar to off-resonance <span class="math inline">\(\text{R}_{1\rho}\)</span> measurements. Transient nonnative protein states can be studied by other NMR experiments, such as paramagnetic relaxation enhancement.</p>
<p>Dynamics of protein folding, on the time scale of seconds, can be studied by other methods. Changes in NMR spectra can be monitored in real-time. Logitudinal magnetization (ZZ) exchange experiments can probe interconversions between states with comparable populations. Hydrogen exchange measurements detect evanescent populations of disordered nonnative states transiently sampled by proteins under native-like conditions.</p>
</div>
<div id="other-methods" class="section level2">
<h2>Other Methods</h2>
<p>There are some methods that can assess the extent of folding qualitatively, but they do not give as much quantitative information as NMR does.</p>
<div id="cd-circular-dichroism" class="section level3">
<h3>CD (Circular Dichroism)</h3>
<p>Plane-polarized light is the sum of two circularly polarized beams, L and R, which are rotating in opposite directions. CD arises from differential absorption of L and R component beams in a smaple of chiral molecules. CD is detected by a double beam instrument with separate L and R paths (which are usually produced by high frequency modulation of a photoelastic modulator (PEM) device). The high intensity light source is produced by synchrotron.</p>
<p>The CD spectrum shows the variation of <span class="math inline">\(\Delta\epsilon\)</span> (difference between the extinction coefficient of the L and R beams) with wavelength (<span class="math inline">\(\lambda\)</span>). Because of the chiral nature of amino acids (except glycine), peptides are optically active. The CD spectra in the 170-250 nm region are distinct for proteins with different structure. Standard curves for different secondary structures (<span class="math inline">\(\alpha\)</span>-helix, <span class="math inline">\(\beta\)</span>-sheet, <span class="math inline">\(\beta\)</span>-turn, and random coil) are available (Figure <a href="#fig:cd">2</a>) and the proportion of different secondary structures of a protein is calculated by linear combination of the standard curves.</p>
<div class="figure"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="CD components and standard curves" width="100%" />
<p class="caption">
Figure 2: CD components and standard curves
</p>
</div>
</div>
</div>
<div id="dsc-differential-scanning-calorimetry" class="section level2">
<h2>DSC (Differential Scanning Calorimetry)</h2>
<p>DSC determines the heat capacity (the difference between the heat capacity in the sample cell and in the reference cell), C<sub>p</sub>, of a molecule in aqueous solution, as a function of temperature. This is done by increasing cell temperatures while keeping the two cells at the same temperature and recording the power supply throughout the experiment. The variation of <span class="math inline">\(C_\text{p}\)</span> with temperature is plotted. On such a curve, there is a peak if the sample molecule undergoes state changes within the temperature (Figure <a href="#fig:dsc">3</a>). For a protein, the peak corresponds to the melting temperature, <span class="math inline">\(T_\text{m}\)</span>, which occurs when the concentrations of its native state ([N]) and the denatured state ([D]) are equal. Stabilization of protein, for example by altering the pH and forming complexes, increases <span class="math inline">\(T_\text{m}\)</span></p>
<div class="figure"><span id="fig:dsc"></span>
<img src="../img/dsc.jpg" alt="Principles of DSC" width="100%" />
<p class="caption">
Figure 3: Principles of DSC
</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Campbell-2012">
<p>Campbell, Iain D. 2012. <em>Biophysical Techniques</em>. Oxford University Press.</p>
</div>
<div id="ref-Zhuravleva-2017">
<p>Zhuravleva, Anastasia, and Dmitry M. Korzhnev. 2017. “Protein Folding by Nmr.” <em>Progress in Nuclear Magnetic Resonance Spectroscopy</em> 100: 52–77. <a href="https://doi.org/https://doi.org/10.1016/j.pnmrs.2016.10.002">https://doi.org/https://doi.org/10.1016/j.pnmrs.2016.10.002</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/biophysical-techniques.html" class="archive-item-link">
                Biophysical Techniques 
              </a>
              <span class="archive-item-date">
                2019-11-12
              </span>
              <div class="article-summary" style="display: none;">
                
                  You have expressed and purified a protein in a recombinant system. How would you confirm that the protein was the one expected from the DNA sequence. (ESI-MS) How would you determine that the protein was folded and its solution behaviour. (CD, Thermofluor, Intrinsic fluorescence, DSC, SEC MALS, AUC, DLS) How can we quantify the binding between two macromolecules? Discuss how you could use the following techniques to determine a KD for an interaction? What are their limitations in practice? What would decide which techniques you could apply to an interaction? (Surface Plasmon resonance, Fluorimetry/Fluorescence anisotropy, Microscale thermophoresis (MST), Isothermal titration calorimetry (ITC), Stopped flow, NMR, AUC) References: Use Campbell&#39;s Biophysical Techniques; for MST use Wienken et al 2010 Nature Communications 10.1038/ncomms1093
<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#confirmation-of-dna-sequence"><span class="toc-section-number">1</span> Confirmation of DNA Sequence</a><ul>
<li><a href="#esi-ms-electrospray-ionization-mass-spectrometry"><span class="toc-section-number">1.1</span> ESI-MS (Electrospray Ionization Mass Spectrometry)</a></li>
</ul></li>
<li><a href="#assess-protein-folding-and-solubility"><span class="toc-section-number">2</span> Assess Protein Folding and Solubility</a><ul>
<li><a href="#cd-circular-dichroism"><span class="toc-section-number">2.1</span> CD (Circular Dichroism)</a></li>
<li><a href="#dsc-differential-scanning-calorimetry"><span class="toc-section-number">2.2</span> DSC (Differential Scanning Calorimetry)</a></li>
<li><a href="#dls-dynamic-light-scattering"><span class="toc-section-number">2.3</span> DLS (Dynamic Light Scattering)</a></li>
<li><a href="#auc-analytical-ultracentrifuge"><span class="toc-section-number">2.4</span> AUC (Analytical Ultracentrifuge)</a></li>
</ul></li>
<li><a href="#determining-kd"><span class="toc-section-number">3</span> Determining K<sub>d</sub></a><ul>
<li><a href="#fluorimetryfluorescence-anisotropy"><span class="toc-section-number">3.1</span> Fluorimetry/Fluorescence anisotropy</a></li>
<li><a href="#surface-plasmon-resonance"><span class="toc-section-number">3.2</span> Surface Plasmon resonance</a></li>
<li><a href="#isothermal-titration-calorimetry-itc"><span class="toc-section-number">3.3</span> Isothermal titration calorimetry (ITC)</a></li>
<li><a href="#mst-microscale-thermophoresis"><span class="toc-section-number">3.4</span> MST (Microscale Thermophoresis)</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="confirmation-of-dna-sequence" class="section level1">
<h1><span class="header-section-number">1</span> Confirmation of DNA Sequence</h1>
<div id="esi-ms-electrospray-ionization-mass-spectrometry" class="section level2">
<h2><span class="header-section-number">1.1</span> ESI-MS (Electrospray Ionization Mass Spectrometry)</h2>
<p>In mass spectrometry, the analytes are first ionised in vacuum and these charged molecules are passed into a electric/magnetic field, and their path through the field can be used to deduce their mass/charge (m/z) ratio (using deflection, TOF, quadrupole, or other analyzers).</p>
<p>Traditional ionisation methods were not suitable for biological macromolecules because the need for heating (or other treatment) to achieve gas phase would cause rapid decomposition of the molecules. Later, MALDI (matrix-assisted laser desorption/ionisation) and ESI (Electrospray Ionization) techniques were deveoped and they are suitable for this purpose. In MALDI, proteins are placed in a light-absorbing matrix, then ionisation and desorption is triggered by a short pulse of laser light. In ESI, a solution of analytes is passed through a charged needle kept at a high electrical potential, dispersing the solution into a microdroplets. The solvent around macromolecules rapidly evaporates, leaving charged molecules in gas phase. ESI can directly accept inputs from many other purification methods such as SDS-PAGE and chromatography and in addition it is good for detecting native states and different conformations.</p>
<p>The sequence of a protein is often too long to be obtained by MS at one time, so protease can be used to break peptides into small fragments which are sequenced individually. In addition, tandem MS (MS/MS) is can be used, in which one peptide is first analysed by one mass analyzer (MS1), and then further fragmented by a ‘collision gas’ such as He and Ar, and the m/z ratios of these fragments are analysed by MS2. This produces a spectrum with many peaks (greater m/z correspond to longer fragments), and the successive peaks differ by the m/z of a particular amino acid in the original peptide. This information can be used to deduce the original sequence.</p>
</div>
</div>
<div id="assess-protein-folding-and-solubility" class="section level1">
<h1><span class="header-section-number">2</span> Assess Protein Folding and Solubility</h1>
<div id="cd-circular-dichroism" class="section level2">
<h2><span class="header-section-number">2.1</span> CD (Circular Dichroism)</h2>
<p>Plane-polarized light is the sum of two circularly polarized beams, L and R, which are rotating in opposite directions. CD arises from differential absorption of L and R component beams in a smaple of chiral molecules.</p>
<p>CD is detected by a double beam instrument with separate L and R paths (which are usually produced by high frequency modulation of a photoelastic modulator (PEM) device). The high intensity light source is produced by synchrotron.</p>
<p>The CD spectrum shows the variation of <span class="math inline">\(\Delta\epsilon\)</span> (difference between the extinction coefficient of the L and R beams) with wavelength (<span class="math inline">\(\lambda\)</span>).</p>
<p>Because of the chiral nature of amino acids (except glycine), peptides are optically active. The CD spectra in the 170-250 nm region are distinct for proteins with different structure. Standard curves for different secondary structures (<span class="math inline">\(\alpha\)</span>-helix, <span class="math inline">\(\beta\)</span>-sheet, <span class="math inline">\(\beta\)</span>-turn, and random coil) are available (Figure <a href="#fig:cd">2.1</a>) and the proportion of different secondary structures of a protein is calculated by linear combination of the standard curves.</p>
<div class="figure" style="text-align: c"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="CD components and standard curves" width="100%" />
<p class="caption">
Figure 2.1: CD components and standard curves
</p>
</div>
</div>
<div id="dsc-differential-scanning-calorimetry" class="section level2">
<h2><span class="header-section-number">2.2</span> DSC (Differential Scanning Calorimetry)</h2>
<p>DSC determines the heat capacity (the difference between the heat capacity in the sample cell and in the reference cell), C<sub>p</sub>, of a molecule in aqueous solution, as a function of temperature. This is done by increasing cell temperatures while keeping the two cells at the same temperature and recording the power supply throughout the experiment. The variation of <span class="math inline">\(C_\text{p}\)</span> with temperature is plotted. On such a curve, there is a peak if the sample molecule undergoes state changes within the temperature (Figure <a href="#fig:dsc">2.2</a>). For a protein, the peak corresponds to the melting temperature, <span class="math inline">\(T_\text{m}\)</span>, which occurs when the concentrations of its native state ([N]) and the denatured state ([D]) are equal. Stabilization of protein, for example by altering the pH and forming complexes, increases <span class="math inline">\(T_\text{m}\)</span></p>
<div class="figure" style="text-align: c"><span id="fig:dsc"></span>
<img src="../img/dsc.jpg" alt="Principles of DSC" width="100%" />
<p class="caption">
Figure 2.2: Principles of DSC
</p>
</div>
</div>
<div id="dls-dynamic-light-scattering" class="section level2">
<h2><span class="header-section-number">2.3</span> DLS (Dynamic Light Scattering)</h2>
<p>Random motion of macromolecules in a suspension causes fluctuations in local concentration and thus local variarions in refractive index and intensity of the scattered light. These time-dependent fluctuations can be analyzed by a coherent laser source. The observed fluctuations give rise to diffusion coefficients, <span class="math inline">\(D\)</span>, which is related to the size of the molecules (large particles diffuse more slowly). DSL can measure polydispersity and the presence of aggregates in protein samples.</p>
</div>
<div id="auc-analytical-ultracentrifuge" class="section level2">
<h2><span class="header-section-number">2.4</span> AUC (Analytical Ultracentrifuge)</h2>
<p>An AUC is a specialised ultracentrifuge equipped with absorbance and interference detection systems. Each cell contains a sample meniscus and a reference cavity, allowing the absorbance of the solvent to be corrected.</p>
<p>Two types of experiments are typically performed by AUC: sedimentation velocity and sedimentation equilibrium. Sedimentation velocity is performed at high speed, which depletes particles from the centrifuge cell and creates a pellet at the bottom of the cell. It can give information on molecular shape, mass, and interactions with themselves and with other components.</p>
<p>Sedimentation equilibrium is perfomed at low speed and does not create a pellet, and the sedimentation of molecules down the centrifuge cell is balanced by their diffusion back up the cell. The main application of sedimentation equilibrium is the detection of complexes and self-association and the quantification of binding between species</p>
</div>
</div>
<div id="determining-kd" class="section level1">
<h1><span class="header-section-number">3</span> Determining K<sub>d</sub></h1>
<div id="fluorimetryfluorescence-anisotropy" class="section level2">
<h2><span class="header-section-number">3.1</span> Fluorimetry/Fluorescence anisotropy</h2>
<p>If fluorophores are excited with plane polarized light and the fluorescence is observed through analyzing polarizers, the fluorescence is also polarised.</p>
<p>The fluorescence anisotropy is defined as <span class="math inline">\(A=\dfrac{I_\parallel - I_\bot}{I_\parallel+2_{\bot}}\)</span>, where <span class="math inline">\(I_\parallel\)</span> and <span class="math inline">\(I_{\bot}\)</span> are the fluorescence intensities polarised parallel and perpendicular to the direction of the excitation beam. <span class="math inline">\(A\)</span> is a direct measure of the molecular rotation in solution and can be used to study complex formation, as a macromolecule will rotate more slowly when it is in a complex thatn when it is alone.</p>
</div>
<div id="surface-plasmon-resonance" class="section level2">
<h2><span class="header-section-number">3.2</span> Surface Plasmon resonance</h2>
<p>Surface plasmon resonance (SPR) is used for measuring molecular interactions between a pair of molecules.</p>
<p>A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, <span class="math inline">\(\theta_\text{spr}\)</span>, is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. <span class="math inline">\(\Delta\theta_\text{spr}\)</span> is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region).</p>
<p>In an SPR experiment, one type of ligand is immobilised at the sensor surface, and a the analyte is passed through the cell. If the ligand binds to any binding partner in the analyte, <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would increase. Then, non-specific binding is washed off by buffer, and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would decrease and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> due to specific binding can be found. Finally, regeneration solution is applied to remove all binding and reset <span class="math inline">\(\Delta\theta_\text{spr}\)</span> to zero. (Figure <a href="#fig:spr">3.1</a>)</p>
<div class="figure" style="text-align: c"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="Principles of SPR" width="100%" />
<p class="caption">
Figure 3.1: Principles of SPR
</p>
</div>
<p>SPR can be used to determine <span class="math inline">\(K_d\)</span> of complex formation, which equals <span class="math inline">\(\dfrac{k_{-1}}{k_1}\)</span> where <span class="math inline">\(k_1\)</span> is the rate of association and <span class="math inline">\(k_{-1}\)</span> is the rate of dissociation. <span class="math inline">\(k_1\)</span> and <span class="math inline">\(k_{-1}\)</span> can be deduced from the plot.</p>
</div>
<div id="isothermal-titration-calorimetry-itc" class="section level2">
<h2><span class="header-section-number">3.3</span> Isothermal titration calorimetry (ITC)</h2>
<p>ITC measures heat changes when a complex is formed at constant temperature.</p>
<p>In ITC, an insulated reaction cell containing protein is kept at a temperature (usually 8<span class="math inline">\(^\circ\text{C}\)</span> above the environment) which is equal to the temperature of a reference cell, and the reference cell is kept at a constant temperature by a thermostat. Then, increasing amounts of ligand is added into the chamber, and they form complexes with the protein, which can be exothermic or endothermic. The heat change is compensated by a power supply, which can be converted to <span class="math inline">\(\Delta H\)</span> of the reaction. As more ligands are added, proteins become saturated and <span class="math inline">\(\Delta H\)</span> approaches zero. The raw data obtained (power supplied to compensate the heat change caused by each addition of ligands) can be integrated and corrected to give a plot of <span class="math inline">\(\Delta H\)</span> against the molar ratio of the ligand and the protein, and <span class="math inline">\(\Delta H\)</span>, K<sub>d</sub> and stoichiometry can be inferred from the curve. Subsequently, <span class="math inline">\(\Delta G\)</span> and <span class="math inline">\(\Delta S\)</span> can also be calculated (Figure <a href="#fig:itc">3.2</a>).</p>
<div class="figure" style="text-align: c"><span id="fig:itc"></span>
<img src="../img/itc.jpg" alt="ITC data manipulation" width="100%" />
<p class="caption">
Figure 3.2: ITC data manipulation
</p>
</div>
</div>
<div id="mst-microscale-thermophoresis" class="section level2">
<h2><span class="header-section-number">3.4</span> MST (Microscale Thermophoresis)</h2>
<p>MST is a relatively new method for analysing interactions of proteins or small molecules in complex bioliquids such as blood serum or cell lysate. The technique depends on the phenomenon that molecules move within temperature gradients (thermophoresis).</p>
<p>The instrument uses an IR laser to create a temperature gradient, and the movement of the molecules within this gradient is monitored by fluoresence. The fluoresence can either be intrinsic (due to tryptophan) or extrinsic (attached dye or fluorescent protein).</p>
<p>Compared with traditional methods for studying protein interactions, MST has several advantages:</p>
<ul>
<li>A minuscule amount of sample (a few <span class="math inline">\(\mu\)</span>l) is needed,</li>
<li>no limitations on size and affinity</li>
<li>no limitations on buffer; tolerates impurity; can be used in in complex bioliquids such as blood serum or cell lysate</li>
<li>no need for immobilization</li>
</ul>
<div class="figure" style="text-align: c"><span id="fig:mst"></span>
<img src="../img/mst.jpg" alt="Sample data produced by MST" width="100%" />
<p class="caption">
Figure 3.3: Sample data produced by MST
</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Wienken-2010">
<p>Wienken, Christoph J., Philipp Baaske, Ulrich Rothbauer, Dieter Braun, and Stefan Duhr. 2010. “Protein-Binding Assays in Biological Liquids Using Microscale Thermophoresis.” <em>Nature Communications</em> 1 (1): 100. <a href="https://doi.org/10.1038/ncomms1093">https://doi.org/10.1038/ncomms1093</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/paper-vi-1.html" class="archive-item-link">
                Paper VI 2011 
              </a>
              <span class="archive-item-date">
                2019-11-09
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="alzheimers-disease" class="section level1">
<h1>Alzheimer’s Disease</h1>
<div id="effect-of-ampk-on-secreted-abeta-levels-in-neuronal-cultures" class="section level2">
<h2>Effect of AMPK on secreted A<span class="math inline">\(\beta\)</span> levels in neuronal cultures</h2>
<ol style="list-style-type: lower-alpha">
<li>In ELISA (enzyme-linked immunosorbant assay), antibodies are used to bind an antigen of interest (usually protein) with high specificity, and the presence and concentration of the antigen is reported by the extent of reaction of the enzyme conjugated with the antibody. Using two types of antibodies binding to different regions of the antigen allows sandwich ELISA (see below), which has greater specificity. Horseradish peroxidase converts its substrate to a coloured product, and the intensity of the colour, which is related to the concentration of A<span class="math inline">\(\beta\)</span> peptide (the antigen), can be measured by a spectrophotometer as absorbance of EM wave at a certain visible wavelength (the relationship is not always linear so a calibration is needed).</li>
</ol>
<p><img src="../img/elisa-sandwich.jpg" width="100%" /></p>
<ol start="2" style="list-style-type: lower-alpha">
<li>By plotting ‘measured value’ against the number of dilution (the power to which <span class="math inline">\(\frac{1}{2}\)</span> is raised), it can be shown that the realtionship between them is roughly linear when measured value is around 100.</li>
</ol>
<pre class="r"><code>standard &lt;- c(206, 215, 201, 199, 205, 197, 199, 202, 190, 120, 67, 41, 27, 13, 8, 8)
untreated &lt;- c(211, 197, 200, 202, 198, 195, 197, 164, 88, 47, 27, 15, 6, 9, 7, 7)
AICAR &lt;- c(207, 202, 205, 199, 189, 200, 183, 101, 58, 30, 18, 12, 7, 7, 8, 6)
pow &lt;- 0:15

df &lt;- tibble(pow, standard, untreated, AICAR)
df %&gt;% gather(standard, untreated, AICAR, key = &#39;treatment&#39;, value = &#39;measured value&#39;) %&gt;% 
  ggplot(aes(pow, `measured value`, color = treatment, shape = treatment, linetype=treatment))+
  geom_point()+xlab(&#39;number of dilution (i.e. number of times the concentration is halved)&#39;)+
  geom_smooth(se=FALSE, method=&#39;loess&#39;, span=0.4, size=0.5)+
  geom_hline(yintercept = 100)+
  scale_x_continuous(breaks = 0:15)</code></pre>
<pre><code>## `geom_smooth()` using formula &#39;y ~ x&#39;</code></pre>
<p><img src="../tutorial/2019-11-09-paper-vi-1_files/figure-html/unnamed-chunk-3-1.png" width="100%" /></p>
<p>Reading off from the plot:</p>
<ul>
<li>for the standard solution, a <span class="math inline">\(\dfrac{1}{2^{9.5}}\)</span> dilution corresponds to 100 units. Thus, 100 units correspond to <span class="math inline">\(10\times \dfrac{1}{2^{9.5}}=0.01381\text{ ng/ml}\)</span></li>
<li>for the untreated sample, a <span class="math inline">\(\dfrac{1}{2^{7.9}}\)</span> dilution corresponds to 100 units, or <span class="math inline">\(0.0138\text{ ng/ml}\)</span>, so the undiluted sample has a A<span class="math inline">\(\beta\)</span> concentration of <span class="math inline">\(0.01381\times2^{7.9}=3.299\text{ ng/ml}\)</span>. The volume of 10<sup>6</sup> cells is 100<span class="math inline">\(\mu\)</span>l, so the level of secretion is <span class="math inline">\(3.299\times0.100\times 1000=329\text{ pg/10}^6\text{ cells/day}\)</span></li>
<li>for the sample with AICAR, the undiluted sample has a A<span class="math inline">\(\beta\)</span> concentration of <span class="math inline">\(0.01381\times2^{7.2}=2.031\text{ ng/ml}\)</span>. The level of secretion is <span class="math inline">\(2.031\times0.100\times 1000=203\text{ pg/10}^6\text{ cells/day}\)</span></li>
</ul>
<ol start="3" style="list-style-type: lower-alpha">
<li>Upon activation by AICAR, AMPK reduces secretion of A<span class="math inline">\(\beta\)</span></li>
</ol>
</div>
<div id="effect-of-added-abeta-on-ampk-activity" class="section level2">
<h2>Effect of added A<span class="math inline">\(\beta\)</span> on AMPK activity</h2>
<ol start="4" style="list-style-type: lower-alpha">
<li><p>Triton X-100 is a detegent, disrupting the cell membrane and lysing the cell; PMSF inhibits serine proteases, preventing degradation of target protein; protein A, which is immobilised on sepharose, binds specifically to the F<sub>c</sub> region of anti-AMPK antibody</p></li>
<li><p>This ensures that all measured enzymatic activity is due to AMPK but not other kinases that would be present in the whole cell extract.</p></li>
<li>Substituting <span class="math inline">\(t_{\frac{1}{2}}=14\text{ days}\)</span> and <span class="math inline">\(t = 14\)</span> into <span class="math inline">\(A=A_0e^{-kt}=A_0e^{-\dfrac{\ln{2}t}{t_{\frac{1}{2}}}}=A_0\times(2)^{-\dfrac{t}{t_{\frac{1}{2}}}}\)</span>, <span class="math inline">\(A=A_0\times(2)^{-\frac{9}{14}}=0.6404A_0\)</span>, i.e. <span class="math inline">\(A_0=1.5614A\)</span>, where <span class="math inline">\(A_0\)</span> is the original activity and <span class="math inline">\(A\)</span> is the activity after 9 days. <span class="math display">\[\begin{aligned}A_0 \text{ cpm}\div80\%\div(2.2\times10^{12})\text{ cpm/Ci}\div(0.5\times10^{-3})\text{ Ci/nmol}\div(50\times10^{-3}\text{ mg cell protein})\div20\text{ min}\div\frac{50}{50+5+5}\\=x \text{ nmol substrate phosphorylated/min/mg cell protein}\end{aligned}\]</span> where x is the final answer. This simplifies to <span class="math display">\[x=1.3636\times10^{-9}A_0=2.129\times10^{-9}A\]</span>
<ul>
<li>Activity (+A<span class="math inline">\(\beta\)</span>): <span class="math inline">\(2.129\times10^{-9}\times(11800-278)=2.45\times10^{-5} \text{ nmol substrate phosphorylated/min/mg cell protein}\)</span></li>
<li>Activity (no A<span class="math inline">\(\beta\)</span>): <span class="math inline">\(2.129\times10^{-9}\times(2940-278)=5.67\times10^{-6} \text{ nmol substrate phosphorylated/min/mg cell protein}\)</span></li>
</ul></li>
<li><p>AMPK activity increases as A<span class="math inline">\(\beta\)</span> is added, and AMPK may help to degrade A<span class="math inline">\(\beta\)</span> according to (c).</p></li>
</ol>
</div>
<div id="association-of-tau-protein-with-microtubules" class="section level2">
<h2>Association of <span class="math inline">\(\tau\)</span> protein with microtubules</h2>
<ol start="8" style="list-style-type: lower-alpha">
<li>tau alone is soluble and exists solely in the supernatant. Unphosphorylated tau binds to microtubules strongly and exists entirely in the pallet (insoluble). Phosphorylated tau has less affinity to microtubules and most of them are found in the supernatant.</li>
</ol>
</div>
<div id="phosphorylation-sites-on-tau" class="section level2">
<h2>Phosphorylation sites on tau</h2>
<ol style="list-style-type: lower-roman">
<li><ul>
<li>Induce mutations on different positions and observe their effect on phosphorylation. The mutated position(s) can be determined by DNA sequencing and the effect of phosphorylation can be determined by isoelectric focusing (phosphorylated protein has a more negative charge).</li>
<li>Fragment the protein and test each piece for phosphorylation.</li>
</ul></li>
</ol>
<ol start="10" style="list-style-type: lower-alpha">
<li>These phosphorylation sites are all located on the MT binding domain (235-368) and thus phosphorylation can directly (at least sterically) hinder binding to MT. In addition, MT binding domain contains many postively charged lysine (K) residues and few negatively charged (D and E) residues. This suggest that the binding between tau and MT is electrostatic (+ve on tau, -ve on MT). Phosphorylation adds -ve charge on tau and thus hinders electrostatic binding.</li>
</ol>
</div>
</div>
<div id="notch-receptor" class="section level1">
<h1>Notch Receptor</h1>
<ol style="list-style-type: lower-alpha">
<li><p>The primer sequences are:</p>
<pre><code>forward primer
5&#39; GGATCC CAGGACGTGGATGAGTGCTCGCTGGG 3&#39; # 33 nt
reverse primer
5&#39; GGATCC TCA CTGGCACAGATGCCCAGTGAAGCCC 3&#39; # 34 nt</code></pre>
<p>To produce a standalone peptide, a start codon (ATG) and a stop codon (TGG) is needed. The stop codon is included in the reverse primer (<code>5'TCA3'</code> which correspond to <code>5'TGA3'</code> in the forward direction). The start codon is defined in the plasmid and sets the reading frame. The restriction site chosen must ensure correct reading frame alignment of the gene of interest, and Bam HI does this job (while some others, such as Sal I, does not), so I chose it, and its sequence, <code>GGATCC</code>, is included at the 5’ end of both primers
<img src="../img/sticky.jpg" width="100%" />
Regarding restriction sites, Pst I and Sph I must not be chosen because they are contained within the gene of interest and applying these restriction enzymes will break the gene. The remaining part of the primer corresponds to the gene of interest, and the 3’ ends are chosen to include 3-4 consecutive G/C to ensure high annealing stability, facilitating DNA polymerase binding.</p></li>
<li><p>Bam HI is applied both to the plasmid and the PCR product, creating sticky ends and allowing them to join. The resulting set of plasmids (some are recombinant and some are not) are transformed into bacteria by heat shock (by disrupting the cell membrane using CaCl<sub>2</sub> and exposing cells to thermal imbalance) or electroporation (by using short pulses of high-voltage electric currents). Ampicilin is added to kill cells without gaining the plasmid, and after that, usually (when a lacZ<span class="math inline">\(\alpha\)</span> gene is incorporated into the cloning site in the reverse direction) X-gal is used to test whether the cells with plasmids contains the gene of interest (blue-white screening).
<img src="../img/clone.jpg" width="100%" /></p></li>
<li><p>As illustrated in the plasmid map, the expression of inserted gene is controlled by the <em>Lac</em> operator. When uninduced, lac repressor protein due to <em>LacI</em> expression binds to <em>Lac</em> operator, preventing expression of inserted gene. IPTG can be used to induce expression. IPTG is an analogue of lactose, which activates <em>Lac</em> inducible operon by deactivating the lac repressor. IPTG is not metabolised by <em>E. coli</em> so it can be maintained at a constant concentration.</p></li>
<li><p>The protein product contains N-terminal hexahistidine tag which binds to Nickle ions with high affinity. Affinity chromatography using nickle-bound resin can be used to purify the protein. After that, size-exclusion chromatography may also be done to remove any possible aggregates of this peptide or complexes of this peptide with other species.</p></li>
<li><p>Run a SDS-PAGE, stain with Commassie blue and there should be only one band.</p></li>
<li><p>This extracellular protein is likely to contain disulfide bridges, which would not form in the normal reducing intracellular environment of bacteria. Some strains of <em>E. coli</em>, such as <em>Origami</em>, have mutant thioredoxin reductase and/or glutathione reductase an thus have a oxidising cytoplasm, and they can be used to express proteins with disulfide bridges.</p></li>
<li><ul>
<li>From the blot developed with biotin binding protein, A alone produce a band while B or C alone does not. This indicates that A must be a potential ligand (because it, being alone and biotin-tagged, is pulled down by biotin-binding Streptavidin).</li>
<li>From the blot developed with His tag binding antibody, only A+B produces a band. As A is the potential ligand, B must be Notch that is pulled down along with A. C must not be Notch because A+C does not produce a band, so C is another potential ligand.</li>
<li>C alone does not produce a band in the biotin-specific blot possibly because the biotin tag possibly because C’s tertiary structure prevents biotin binding</li>
</ul></li>
<li><p>The potential ligand A binds to Notch (B), so A+B produces a band on the His-specific blot. It is uncertain whether A interacts with C. A and C are both potential ligands and will not produce a band on the His-specific blot. A+C produces a band on the biotin-specific blot and this may be solely due to A (which itself alone produces the band). A might bind to C without making C’s biotin available for binding, or A might not bind to C, or A might bind to C, making its own biotin unavailable and C’s biotin available, so that a band with equal thickness is produced.</p></li>
<li><ul>
<li>Fluoresence resonance energy transfer (FRET), in which the potential ligand and Notch are each tagged with a fluorophore, where one type of fluorophore, when excited, can transfer energy to the other type of fluorophore through nonradiative dipole-dipole coupling and hence induce fluorescence of the acceptor fluorophore. The energy transfer is only possible when two fluorophores are very close to each other (when their associated proteins are bound to each other). If excitation of the donor fluorophore results in emission from the acceptor fluorophore, it indicates complex formation.</li>
<li>As shown in Figure <a href="#fig:spr">1</a>, surface plasmon resonance can also be used to monitor complex formation.</li>
<li>MST (Microscale Thermophoresis) allows detection of complex formation in complex bioliquids such as blood serum or cell lysate.</li>
</ul></li>
</ol>
<div class="figure"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="Principles of SPR" width="100%" />
<p class="caption">
Figure 1: Principles of SPR
</p>
</div>
<div style="page-break-after: always;"></div>
</div>
<div id="compound-a" class="section level1">
<h1>Compound A</h1>
<ol style="list-style-type: lower-alpha">
<li><p>Toa1 decreases generation time and decreases lifespan.</p></li>
<li><p>No effect on both.</p></li>
<li><p>As a control group.</p></li>
<li><p>DMSO reduces lifespan.</p></li>
<li><p>Regarding lifespan in DMSO, when either Toa1 or Abp1 (or both) is absent, lifespan is unresponsive to compound A. Among these three conditions, when Toa1 is present and Abp1 is absent, lifespan is constantly short; when Toa is absent, lifespan is constantly long. This suggest that Toa1 reduces lifespan, and Abp1 itself has no effect on the lifespan when Toa1 is absent.
When both proteins are present, lifespan increases with concentration of A and then decreases. This suggest that, at low concentrations of A, Abp1 binds to A, and the binding of Toa1 to Abp1-A complex reduces Toa1’s ability to shorten lifespan. At high concentrations of A, Abp1 becomes less effective in deactivating Toa1, possibly because Abp1 forms aggregates.</p></li>
<li><p>Mutation of ATG8 always leads to short lifespan and generation time, and mutation of SCH9 always leads to increased lifespan and generation time, regardless of the conditions of ABP1 and TOA1, and they all made the lifespan insensitive to the concentration of compound A. When mutations of ATG8 and SCH9 occur simutaneously, the effect of ATG8 loss dominates and results in short lifespan &amp; generation time.
Therefore, SCH9 may reduce lifespan and generation time and ATG8 opposes SCH9’s activity. This explains <span class="math inline">\(sch9\Delta\)</span>, <span class="math inline">\(atg8\Delta\)</span> and <span class="math inline">\(sch9\Delta atg8\Delta\)</span>. In addition, Toa1’s ability to reduce lifespan is dependent on SCH9. In <span class="math inline">\(sch9\Delta abp1\Delta\)</span> and <span class="math inline">\(sch9\Delta toa1\Delta\)</span>, the life-span reduction cannot be done without SCH9. In <span class="math inline">\(abp1\Delta atg8\Delta\)</span>, lifespan reduction is caused by Toa1+SCH9 and in <span class="math inline">\(toa1\Delta atg8\Delta\)</span> caused by SCH9, and no ATG8 was present to oppose this effect.</p></li>
<li><p>In the western blot data, for Sch9 5SA, the purpose of changing serine to alanine is usually to test for phosphorylation, and in this case the band is shorter than WT, indicating there is phosphorylation on Sch9. Also Sch9 5SA band is similar in length to <span class="math inline">\(toa1\Delta\)</span>, which implies that toa1 loss has an similar effect to making Sch9 unavailable for phosphorylation. Thus Toa1 may phosphorylate Sch9, and this is in accordance with an observation in (f): ‘toa1’s ability to reduce lifespan is dependent on sch9’. Sch9 5SA with phosphatase further reduces the signal, and this indicates that Sch9 has other phosphorylation sites (apart from the site where 5 S was changed to A i.e. target of toa1) being phosphorylated.</p></li>
<li><p>WT + compound A produced a low signal comparable to the WT + phosphate band. Thus compound A may hamper phosphorylation of Sch9 (by directly hindering phosphorylation, or acting as a phosphatase). As toa1 is ‘target of A’ and toa1 is the kinase that phosphorylates Sch9, compound A is very likely to inhibit toa1.</p></li>
<li><p>see Figure (g)</p></li>
<li>Purify the proteins and study their interactions <em>in vitro</em>.
<ul>
<li>to confirm phosphorylation of Sch9 by Toa1: mix Sch9, Toa1 and ATP (control group: Sch9 and ATP only) and run isoelectric focusing (phosphorylated Sch9 has a more negative charge)</li>
<li>to confirm the role of compound A: mix compound A, Sch9, Toa1 and ATP (control group: buffer+Sch9+Toa1+ATP) and run isoelectric focusing.</li>
</ul></li>
<li><p>In strains with Sch92D3E but not WT Sch9, the lifespan is constantly reduced and this effect is unaffected by compound A.</p></li>
<li><p>See Figure <a href="#fig:fuckyoucompoundA">2</a>.</p></li>
</ol>
<div class="figure"><span id="fig:fuckyoucompoundA"></span>
<img src="../img/fuckyoucompoundA.jpg" alt="When compound A concentration increases, TOA is is inhibited and less Sch9 is phosphorylated, so there is less lifespan-reduction effect due to phosphorylated Sch9, and hence the lifespan increases. When the concentration of A gets too high, however, ATG8 phosphatase is expressed to dephosphorylate Sch9 and thus decrease lifespan." width="100%" />
<p class="caption">
Figure 2: When compound A concentration increases, TOA is is inhibited and less Sch9 is phosphorylated, so there is less lifespan-reduction effect due to phosphorylated Sch9, and hence the lifespan increases. When the concentration of A gets too high, however, ATG8 phosphatase is expressed to dephosphorylate Sch9 and thus decrease lifespan.
</p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/protein-purification.html" class="archive-item-link">
                Protein Expression and Purification 
              </a>
              <span class="archive-item-date">
                2019-11-02
              </span>
              <div class="article-summary" style="display: none;">
                
                  You want to over-express a protein secreted by the malarial parasite. It is cysteine-rich and has no known structure. Outline an expression and purification strategy, using both traditional and high-throughput methods, to produce a sample suitable for protein crystallography. Include a description of how you would assess the quality of the purified protein. OR What strategies could you employ during protein expression and purification to minimise heterogeneity in a sample required for structural studies?<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#title">Title</a></li>
<li><a href="#overview"><span class="toc-section-number">1</span> Overview</a></li>
<li><a href="#protein-expression"><span class="toc-section-number">2</span> Protein Expression</a><ul>
<li><a href="#design"><span class="toc-section-number">2.1</span> Identifying and amplifying the gene</a></li>
<li><a href="#choosing-the-host-organism-and-the-vector"><span class="toc-section-number">2.2</span> Choosing the host organism and the vector</a><ul>
<li><a href="#bacteria-and-plasmid-vectors"><span class="toc-section-number">2.2.1</span> Bacteria and plasmid vectors</a></li>
<li><a href="#eukaryotic-hosts"><span class="toc-section-number">2.2.2</span> Eukaryotic hosts</a></li>
<li><a href="#high-throughput-strategy"><span class="toc-section-number">2.2.3</span> High-throughput strategy</a></li>
</ul></li>
<li><a href="#lic"><span class="toc-section-number">2.3</span> Constructing the vector</a></li>
<li><a href="#transformationtransfection"><span class="toc-section-number">2.4</span> Transformation/Transfection</a></li>
</ul></li>
<li><a href="#protein-extraction-and-purification"><span class="toc-section-number">3</span> Protein Extraction and Purification</a><ul>
<li><a href="#extraction"><span class="toc-section-number">3.1</span> Extraction</a><ul>
<li><a href="#cell-lysis"><span class="toc-section-number">3.1.1</span> Cell lysis</a></li>
<li><a href="#removing-cellular-components-and-debris"><span class="toc-section-number">3.1.2</span> Removing cellular components and debris</a></li>
</ul></li>
<li><a href="#purification"><span class="toc-section-number">3.2</span> Purification</a><ul>
<li><a href="#salting-out-and-dialysis"><span class="toc-section-number">3.2.1</span> Salting out and dialysis</a></li>
<li><a href="#chromatography"><span class="toc-section-number">3.2.2</span> Chromatography</a></li>
<li><a href="#monitor"><span class="toc-section-number">3.2.3</span> Monitoring the progress of purification</a></li>
</ul></li>
<li><a href="#preparation-for-crystallography"><span class="toc-section-number">3.3</span> Preparation for crystallography</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="title" class="section level2 unnumbered">
<h2>Title</h2>
<p><strong>You want to over-express a protein secreted by the malarial parasite. It is cysteine-rich and has no known structure. Outline an expression and purification strategy, using both traditional and high-throughput methods, to produce a sample suitable for protein crystallography. Include a description of how you would assess the quality of the purified protein.</strong></p>
</div>
<div id="overview" class="section level1">
<h1><span class="header-section-number">1</span> Overview</h1>
<p>The general workflow for obtaining a sample of (malarial) protein is summarised as follows:</p>
<ol style="list-style-type: decimal">
<li>Identifying and amplifying the gene of interest (GOI)</li>
<li>Constructing the vector</li>
<li>Transformation/transfection of host cells</li>
<li>Screening for the most viable (high yield and solubility) transformants</li>
<li>Extracting and purifying proteins</li>
</ol>
<p>For crystallography studies, a protein sample with high purity, solubility and yield must be obtained. The expression and purification strategies depend on the characteristics of the protein of interest, such as size, shape, intrinsic solubility, requirement of post-translational modifications and/or cofactors, presence of disulfide bridges, <em>in vitro</em> stability.</p>
</div>
<div id="protein-expression" class="section level1">
<h1><span class="header-section-number">2</span> Protein Expression</h1>
<div id="design" class="section level2">
<h2><span class="header-section-number">2.1</span> Identifying and amplifying the gene</h2>
<p>If the coding sequence of the protein is unknown but the protein is purified, mass spectrometry (or chemical methods such as Edman degradation) can be used to determine the amino acid sequence of the protein, which can then be used to search for the the corresponding DNA sequence.</p>
<p>Usually, instead of using the full DNA sequence, cDNA is used in cloning because it does not contain introns. cDNA is made by:</p>
<ol style="list-style-type: decimal">
<li>extracting total RNA from cells by TRIzol and then isolate the mature mRNA by affinity chromatography (with poly-T coated resins which binds to poly-A tails of mature mRNA)</li>
<li>reverse transcription of the template DNA strand by viral reverse transcriptase (RT) followed by RNA degradation (often by the RNAse H activity of RT)</li>
<li>synthesis of the coding strand by DNA polymerase</li>
</ol>
<p>Often, the latter two steps are done repeatedly in RT-PCR to amplify the DNA fragment, if the sequence of the GOI is known and specific<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> primers are designed. The primers include additional restriction enzyme cleavage sites for the convinience of traditional ligation-dependent cloning, or other sequences for ligation-independent cloning (see Section <a href="#lic">2.3</a>).</p>
<p>If the protein is already known to have a match in malarial cDNA library, we can skip most of the above steps and PCR-amplify the cDNA of interest from the library directly.</p>
<p>When we know the sequence of GOI, we can do a BLAST to find possible homologous proteins, and study any relavant scientific literature. The structure of some homologous proteins might have been solved previously and we can refer to their expression and purification strategies, which might prevent some waste of time on trial-and-errors. Bioinformatic analyses, such as disorder prediction using the RONN algorithm and generation of 3D homology models using the Phyre2 server, might also be helpful.</p>
<p>Codon optimisation might also be needed. Codon usage can vary significantly between species as well as between genome types such as nuclear DNA and mitochondrial DNA. For example, the frequencies of codons AGG, AGA, and CGA (which code for arginine) in H. sapiens are 11.4, 11.5 and 6.3, respectively, while the corresponding frequencies in E. coli are 1.2, 2.1 and 3.6. Codons that occur in high frequency in <em>Plasmodium</em> but in low frequencies in <em>E. coli</em> would result in a condition where the pool of tRNA for that codon will be so low as to become depleted. When the rare tRNAs are depleted to produce the recombinant protein, proliferation of the host cells is restricted, leading to low yield. This problem may be partially solved by the use of an inducible expression vector (e.g. IPTG), but when the foreign gene contains a large number of rare codons, this is not enough. There are two further approaches to improve the yield. First, the rare codons in the foreign genes can be substituted with prevalent codons, but this is not always reliable. Second, host cells can be transformed with the genes that code for the rare tRNAs. This approach is reliable and several such cell lines are commercially available. Custom host cells can also be made using plasmids.</p>
<p>We might fail in crystallising the single full-length protein (often because of low solubility). In that case, we can try co-expressing another protein or add an appropriate cofactor and crystallising the resulting complex. If this still fails, we might consider truncating the GOI, e.g. removing a very hydrophobic region that results in low solubility. In high-throughput approaches, such as the pipeline adopted by Oxford Protein Production Facility (OPPF), many variants are constructed in parallel to maximise output.</p>
</div>
<div id="choosing-the-host-organism-and-the-vector" class="section level2">
<h2><span class="header-section-number">2.2</span> Choosing the host organism and the vector</h2>
<div id="bacteria-and-plasmid-vectors" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Bacteria and plasmid vectors</h3>
<p>Typically and traditionally, the gene of interest (GOI) is incorporated into a plasmid vector via a ligation-dependent mechanism, then the plasmid is introduced into bacterial (<em>E. coli</em>) cells for expression. There are also ligation-independent methods for making vectors, which is more robust and easy to use (only need to design one ‘adaptor’ parts of the primers once), making it suitable for high-throughput methods.</p>
<p>Bacteria-based protein expression is time- and cost-efficient, but it has two major limitations:</p>
<ol style="list-style-type: decimal">
<li>the insertion size of a plasmid is small, typically 2-10 kb (bacterial artificial chromosomes (BAC) with greater insertion sizes can be used, but the procedures are more complex)</li>
<li>Bacteria might be unable provide the appropriate protein folding environment (including chaperones) and/or post-translational modifications that can be crucial to protein’s functions.</li>
</ol>
<p>Given the protein of interest is cysteine-rich, it is susceptible that disulfide bridges are present if this protein is also extracellular. The normal intracellular reducing environment disfavours disulfide bridge formation, but <em>Origami</em> and related strains have an oxidising intracellular environment due to mutant thioredoxin reductase and glutathione reductase, and they are used to express proteins that require disulfide bonds to achieve their correctly-folded conformation.</p>
<p>The plasmid should contain selectable markers for easy identification of successful transformants. The combination of <em>Amp<sup>R</sup></em> and <em>lacZ</em><span class="math inline">\(\alpha\)</span> is a common choice, as explained in Figure <a href="#fig:pet">2.1</a>.</p>
<p>In addition, the <strong>promoter</strong> should be strong and tightly regulated with minimum or complete lack of basal level transcription under uninduced conditions. The IPTG inducible T7 promoter system (as in pET plasmid) is one of the common choices. IPTG is an analog of allolactose, which can bind to and silence <em>lac</em> inhibitor, de-inhibiting the <em>lac</em> operator. IPTG is not digested by the bacteria, and thus its constant concentration continuously induces protein expression. The <em>lac</em> operator may be also made symmetric, strictly preventing transcription when uninduced.</p>
<p>Some commerially available plamids also include tags added to the N- or C-terminus of the insertion site (multiple cloning site), which aids protein purification. This can be a polyhistidine tag, a fusion protein (e.g. GST) or a short recognisable peptide, and often a cleavage sequence is also introduced for removal of the tags.</p>
<div class="figure"><span id="fig:pet"></span>
<img src="../img/pETBlue.jpg" alt="The pETBlue-2 plasmid vector" width="100%" />
<p class="caption">
Figure 2.1: The pETBlue-2 plasmid vector
</p>
</div>
<p>Plasmid-based bacterial transformation is usually permanent because plasmids can replicate themselves and be distributed into daughter cells after cell division.</p>
</div>
<div id="eukaryotic-hosts" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Eukaryotic hosts</h3>
<p>If the protein is incompatible for expression in bacteria, an eukaryotic host is chosen. This is typically yeast, insect, or human cells.</p>
<p>Transfection of eukaryotic cells can be transient or permanent. Eukaryotic cells do not transcribe plasmids, and the DNA fragment of interest must integrates into the host cell’s genome to produce a stable cell line–which is a rare event and takes more time to achieve. If we had to use a eukaryotic host, transient transfection may be used first to screen for the most viable construct, then a stable cell line is made with this construct for higher yield and convience for later use.</p>
</div>
<div id="high-throughput-strategy" class="section level3">
<h3><span class="header-section-number">2.2.3</span> High-throughput strategy</h3>
<p>High-throughput protein production aims to use robots to automate the steps in making expression constructs and screening them.</p>
<p>Take the OPPF pipeline for example, when the GOI is an eukaryotic protein, they test expressions with different vector constructs (with varying amino acid start and end points, fusion partners, and plasmids) in both <em>E. coli</em> and insect cells in parallel (Figure <a href="#fig:htp-ox">2.2</a>). The expresison screening is automated and performed in 96-well format.</p>
<div class="figure"><span id="fig:htp-ox"></span>
<img src="../img/htp-ox.png" alt="SDS-PAGE gel after NiNTA magnetic bead purification showing expression of constructs for DDR1 in (A) *E. coli* and (B) insect cells via the baculovirus system. The names of constructs correspond to amino acid start and end points and the vector being used (pOPINE, pOPINF or pOPINJ, which confers a C-terminal His-tag, an N-terminal His-tag, or an N-terminal His-GST tag, respectively)" width="100%" />
<p class="caption">
Figure 2.2: SDS-PAGE gel after NiNTA magnetic bead purification showing expression of constructs for DDR1 in (A) <em>E. coli</em> and (B) insect cells via the baculovirus system. The names of constructs correspond to amino acid start and end points and the vector being used (pOPINE, pOPINF or pOPINJ, which confers a C-terminal His-tag, an N-terminal His-tag, or an N-terminal His-GST tag, respectively)
</p>
</div>
<p>they also used ligation-independent cloning methods (Section <a href="#lic">2.3</a>) to reduce variability and thus simplify the workflow.</p>
</div>
</div>
<div id="lic" class="section level2">
<h2><span class="header-section-number">2.3</span> Constructing the vector</h2>
<p>The DNA fragment we obtained should be incorporated into a vector before they can be introduced into the host cell. Usually, a recombinant plasmid is made first and amplified in bacteria. Then, if necessary, it can be transferred into other vectors such as viruses and liposomes, depending on the strategies. Here I describe two ways of making recombinant plasmids: traditional restriction enzyme and ligation-based plasmid cloning, and T4 polymerase-based ligation-independent cloning (LIC).</p>
<div class="figure"><span id="fig:lic"></span>
<img src="../img/lc-lic.jpg" alt="Left, traditional cloning based on restriction enzyme and ligase; right, ligase-independent cloning based on T4 polymerase" width="100%" />
<p class="caption">
Figure 2.3: Left, traditional cloning based on restriction enzyme and ligase; right, ligase-independent cloning based on T4 polymerase
</p>
</div>
<p>As shown in Figure <a href="#fig:lic">2.3</a>, in the ligation-dependent method, the PCR product contains flanking restriction sites (came from the designed primers) to be recognised by a restriction enzyme (e.g. EcoRI). The plasmid also contain the restriction site for EcoRI (located in the multiple cloning site, see Figure <a href="#fig:pet">2.1</a>). Cleavage by EcoRI results in complementary ‘sticky ends’, which facilitate annealing of GOI to the plasmid. The sticky sequences are shart (usually 3-5 bp), thus, to overcome low stability, ligation must be done immediately.</p>
<p>The ligation-based cloning are not proper for high throughput protein production (HTPP) projects due to several disadvantages:</p>
<ul>
<li>incomplete DNA digestion and poor ligation yields</li>
<li>each GOI has to be inspected for any internal restriction sites (for exclusion from the choices of the ligation sequence)</li>
<li>unwanted amino acids can be introduced to the expressed protein</li>
<li>the GOI can only be cloned in the vector position where the selected restriction site is present</li>
</ul>
<p>Ligation-independent cloning (LIC) overcomes many of the problems described above. In a T4 DNA polymerase-dependent approach, the sticky sequences are made by the 3’-to-5’ exonuclease activity of T4 polymerase. These sequences are long, allowing formation of stable recombinant plasmid without <em>in vitro</em> ligase treatment. The nicks in sugar-phosphate backbone are later fixed by host ligase.</p>
<p>The obvious elegance of T4-LIC is that once the sticky sequence is designed and constructed, it can be used for any GOI. As T4 polymerase (exonuclease activity) always proceeds from 3’ ends, any internal sticky sequences will not be disrupting.</p>
</div>
<div id="transformationtransfection" class="section level2">
<h2><span class="header-section-number">2.4</span> Transformation/Transfection</h2>
<p>Transformation is the process of delivering GOI-containing foreign DNA into host cells. For eukaryotic cells, this is more often known as transfection (because ‘transformtion’ has other meanings). Many chemical/physical transformation/transfection methods are generally done by making transient holes on host cell plasma membrane (technically, ‘inducing competence’ in host cells). Here are two examples.</p>
<p>Heat-shock transformation is used for small vectors such as plasmids. The general procedures are:</p>
<ol style="list-style-type: decimal">
<li>Host cells are incubated in a solution containing divalent cations (typically CaCl<sub>2</sub>) on ice.</li>
<li>CaCl<sub>2</sub> partially disrupts the cell membrane, which allows the recombinant DNA enter the host cell. Such cells are called competent cells.</li>
<li>Cells are exposed to a heat pulse (heat shock), and the thermal imbalance causes the entry of DNA through disrupted plasma membrane.</li>
</ol>
<p>Electroporation can be used for larger vectors such as BAC and PAC<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The general procedures are:</p>
<ol style="list-style-type: decimal">
<li>Host cells are placed into a cuvette, together with the vector. The cuvette is connect to electrodes.</li>
<li>The cuvette containing the mixture is subjected to intense electric pulses (2500 V/cm for bacteria, lower for animal and plant cells) each lasting for only a few milliseconds</li>
<li>Most cells would die under such treatment, but for those survived, their membranes are polarised by the electric field and are disrupted, and DNA enters through the pores. Finally, the membrane reseals after the treatment.</li>
</ol>
<p>Depending on the identity of the host cell and the vector, other methods are also available. These are not directly relevant with our goal of expressing a malarial protein, so here is a simple listing:</p>
<ul>
<li>cosmid and fosmid vectors are introduced into bacterial cells via bacteriophages</li>
<li>calcium phosphate co-precipitation for mammalian cells</li>
<li>microinjection for animal cells</li>
<li>microprojectile bombardment for plant cells</li>
<li>adenovirus and lentivirus vectors for mammalian cells</li>
<li><em>Agrobacterium</em>-mediated transformation for plant cells</li>
<li>genome editing techniques based on ‘designer nucleases’ such as ZFN, TALEN and CRISPR-Cas9 with homology directed repair</li>
</ul>
</div>
</div>
<div id="protein-extraction-and-purification" class="section level1">
<h1><span class="header-section-number">3</span> Protein Extraction and Purification</h1>
<p>After proliferation of transformed/transfected cells, we can extract and purify the protein fo interest.</p>
<div id="extraction" class="section level2">
<h2><span class="header-section-number">3.1</span> Extraction</h2>
<div id="cell-lysis" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Cell lysis</h3>
<p>The first step is to lyse, or homogenise the cells to release the protein. To do this, cells can be subjected to osmotic shock or ultrasonic vibration, forced through a small orifice, or ground up in a blender. (if a membrane protein is to be extracted, detergents are normally used, although detergent-free methods have been developed recently)</p>
</div>
<div id="removing-cellular-components-and-debris" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Removing cellular components and debris</h3>
<p>Repeated centrifugation, each time with a higher speed (i.e. differential centrifugation), removes many impurities with large sizes. Organelles (and small vesicles and microsomes) sediment (forming pallets) while proteins remain in the solution (as supernatant).</p>
<p>A finer degree of separation can be achieved by layering the homogenate in a thin band on top of a salt solution that fills a centrifuge tube. When centrifuged, the various components in the mixture move as a series of distinct bands through the solution, each at a different rate, in a process called velocity sedimentation. A glucose gradient is established to protect the bands from convective mixing.</p>
</div>
</div>
<div id="purification" class="section level2">
<h2><span class="header-section-number">3.2</span> Purification</h2>
<p>Classical methods for separating proteins depend on variable protein properties, including solubility, size, charge, and binding affinity. In most cases, protein mixtures are sequentially subjected to different separation methods, each based on a different property. After each step of purification, the fractions are assayed (see Section <a href="#monitor">3.2.3</a>).</p>
<div id="salting-out-and-dialysis" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Salting out and dialysis</h3>
<p>The addition of certain salts in the right amount can selectively precipitate the protein of interest, while others remaining in solution. The precipitation is removed by centrifugation.</p>
<p>Dialysis is then used to remove the salt from the solution containing the protein of interest. The protein mixture is place inside a dialysis bag (i.e. made of size-selective permeable membrane) and placed in a buffer solution with low salt concentration. Salt in the protein-containing solution then diffuse out, leaving proteins inside the bag.</p>
</div>
<div id="chromatography" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Chromatography</h3>
<p>In column chromatography, the solution containing proteins is passed through a column containing a solid matrix (resin). Different proteins are retarded to different extents by their interaction with the matrix, and they can be collected separately as they flow out of the bottom of the column. Depending on the choice of matrix, proteins can be separated according to their charge (ion-exchange chromatography),nhydrophobicity (hydrophobic chromatography), size (gel-filtration/size-exclusion chromatography), or ability to bind to particular small molecules or to other macromolecules (affinity chromatography). The last two are the most important for high throughput protein purification are affinity chromatography and size-exclusion chromatography.</p>
<p>Affinity chromatography extract protein according to the tags we attached to target proteins. For example, His-tagged proteins bind to Nickle-coated resin and HSV epitope tagged proteins bind to resin coated with corresponding antibodies (polyHis-Ni interactions often have lower specificity compared to enzyme-substrate and epitope-antibody interactions)</p>
<p>Size exclusion chromatography cleans abnormal protein aggregates based on their large size.</p>
<p>A problem experienced by column chromatographic methods is diffusional spreading (i.e. proteins that are going down faster also diffuse upwards, mixing with proteins going slower), which reduces the resolution. The degree of diffusional spreading increases with time during which proteins stay in the column. HPLC (high-performance liquid chromatography) solves this problem. It makes use of high-pressure pumps that speed the movement down the column, so that the time of travelling, and hence the diffusional spreading, is greatly reduced, leading to higher resolution.</p>
</div>
<div id="monitor" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Monitoring the progress of purification</h3>
<p>As purification progresses:</p>
<ol style="list-style-type: decimal">
<li>the total amount of protein should decrease as unwanted proteins are removed and some target proteins are lost</li>
<li>amount of target proteins should also decrease because they are unavoidably lost e.g. by nonspecific attachment to the purification apparatus or washed away</li>
<li>the proportion of the target protein (i.e. purity) should increase as unwanted proteins are removed</li>
</ol>
<p>The simplest way to monitor these changes is running a SDS-PAGE gel electrophoresis after each purification step. The bands should progressively decrease to one (which correspond to the target protein), and the thickness of this band should increase because the concentration of target protein increases (its absolute amount decreases, but as we removed the fractions without the target protein in chromatography, its concentration should increase).</p>
<p>To assess whether the protein is correctly folded and functional, specific assays are used For example, if the target protein is an enzyme, it can be assayed using its substrate; if it is a protein which binds to another molecule, their interaction can be probed by surface plasmon resonance. Circular dichroism also helps to assess the foldedness of the target protein.</p>
</div>
</div>
<div id="preparation-for-crystallography" class="section level2">
<h2><span class="header-section-number">3.3</span> Preparation for crystallography</h2>
<p>For crystallisation, the protein sample must be homogenous and correctly folded. They can be checked by the methods descibed in Section <a href="#monitor">3.2.3</a>.</p>
<p>If the affinity tag needs to be removed (which is often required for structual studies), a protease cleavage site is often incorporated before or after the fusion tag and the cleavage can be conducted either in solution following purification (the protease themselves are tagged) or immediately after enzyme capture in situ on the chromatography resin itself. AKTA Express (GE Healthcare) is an elegant procedure for on-column cleavage coupled to multidimensional chromatography that is highly amenable to high throughput protein purification.</p>
<p>There are three methods to grow protein crystalls, namely ‘hanging drop’, ‘sitting drop’, and microdialysis, but the underlying principles are similar. The protein is first dissolved in a ‘crystallisation cocktail’ droplet and concentrated to 2-50 mg/ml, then it is allowed to equilibrate with a more concentrated reservoir solution of the cocktail with a volume ratio of 1:1. As the protein droplet becomes supersaturated, it <em>may</em> start to crystalise. Robots are commonly used for automatic screeening and optimisation of crystallisation conditions. If crystallisation fails under all conditions, we can try co-crystalising with a ligand. If this still fails, we might consider using a truncated protein, as described in Section <a href="#design">2.1</a>.</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Doyle-2008">
<p>Doyle, Sharon A. 2008. <em>HIgh Throughput Protein Expression and Purification: Methods and Protocols</em>. Edited by John W. Walker. Methods in Molecular Biology 498. Humana Press.</p>
</div>
<div id="ref-ThermoFisher-2019">
<p>ThermoFisher. 2019. <a href="https://www.thermofisher.com/uk/en/home/references/gibco-cell-culture-basics/transfection-basics/gene-delivery-technologies.html">https://www.thermofisher.com/uk/en/home/references/gibco-cell-culture-basics/transfection-basics/gene-delivery-technologies.html</a>.</p>
</div>
<div id="ref-Vicentelli-2019">
<p>Vicentelli, Renaud. 2019. <em>HIgh-Throughput Protein Production and Purification: Methods and Protocols</em>. Edited by John M. Walker. Vol. 2025. Methods in Molecular Biology. New York, NY: Humana Press.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>a specific primer will bind the mRNA of interest but not any other mRNAs. Online tools such as Primer3 and Primer-BLAST (NCBI) aids design of specific primers.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>P1-derived artificial chromosome<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/protein-structure-and-function.html" class="archive-item-link">
                Protein Structure and Function 
              </a>
              <span class="archive-item-date">
                2019-10-24
              </span>
              <div class="article-summary" style="display: none;">
                
                  Choose 3 structural papers. They can be crystallography, NMR, CryoEM etc. Explain why the protein is important and what the structure tells you about function (why you find it interesting) but most importantly go to the methods section (it may be supplementary information especially if you choose high profile paper) and explain how the protein was purified. The techniques will be those covered in Matt Higgins&#39; lecture series. Bring notes no essay.<hr>
                
                


<div id="principles-of-x-ray-crystallography" class="section level1">
<h1>Principles of X-ray crystallography</h1>
<div id="scattering" class="section level2">
<h2>Scattering</h2>
<ul>
<li>When light hits matter…
<ul>
<li>Vibration<span class="math inline">\(\rightarrow\)</span>scattering (all directions)</li>
<li>Energy level transitions<span class="math inline">\(\rightarrow\)</span>absorption and emission (fluorescence)</li>
<li>Photochemical reactions (e.g. photosynthesis)</li>
</ul></li>
<li>scattering can give rise to refraction and diffraction</li>
<li>experiments on scattering
<ul>
<li>turbidity (reduction in intensity)</li>
<li>angular dependence</li>
<li>changes in <span class="math inline">\(\lambda\)</span></li>
</ul></li>
</ul>
</div>
<div id="wnt3-fz8-complex-hirai-2019" class="section level2">
<h2>Wnt3-Fz8 Complex <span class="citation">(Hirai et al. 2019)</span></h2>
<ul>
<li>crystals of lysine-methylated and deglycosylated human Wnt3 (hWnt3)-mFz8 CRD complex were obtained by X-ray crystal structure was solved by molecular replacement and refined to a resolution of 2.8Å.</li>
</ul>
<div id="difficulties-in-crystallisation-and-their-solutions" class="section level3">
<h3>Difficulties in crystallisation and their solutions</h3>
<ul>
<li>Strong hydrophobic property of Wnt proteins caused by a covalent lipid modification</li>
</ul>
<p>Optimisation and chemical modifications conducted to ensure high expression yields, enhanced solubility and sample homogeneity</p>
<div id="solubility" class="section level4">
<h4>Solubility</h4>
<ul>
<li>Failed attempts on making crystallization constructs
<ul>
<li>afamin can solubilize Wnt proteins; when coexpressed and complexed, Wnt3 and 3a can be purified to homogeneity. However, diffraction-quality crystals could not be obtained after repeated trials.</li>
<li>coexpression with mFz8 CRD after Janda’s success; but found purified Wnt3a-Fz8 CRD complex still remained hydrophobic and required detergents during concentration, which hampered crystallisation</li>
</ul></li>
<li>Successful: N-terminal truncation of Wnts to mimic the cleavage of the N-terminal peptide by a metalloprotease Tiki, which has been reported to reduce the overall hydrophobicity
<ul>
<li>N-terminal 20 residues were removed from hWnt3/mWnt3a constructs</li>
<li>PA-hWnt3(<span class="math inline">\(\Delta\)</span>N)/PA-mWnt3a(<span class="math inline">\(\Delta\)</span>N) coexpressed with mFz8 CRD C-terminally fused with modified human Fc.</li>
<li>confirmed that complexes were fully soluble in aqueous buffer and could be concentrated to &gt; 5 mg/ml without detergents</li>
</ul></li>
</ul>
</div>
<div id="optimisation" class="section level4">
<h4>Optimisation</h4>
<ul>
<li>Initially (following Janda) attached normal (with hinge region) human IgG1 Fc to the C-terminal of mFz8 CRD, intervened by a TEV protease cleavage sequence for the later Fc removal
<ul>
<li>although complex formed with high yield, it could not be cleaved at all; different linker lengths showed no improvements</li>
</ul></li>
<li>decided to use IdeS protease to remove Fc.</li>
</ul>
<div id="refs" class="references">
<div id="ref-Hirai-2019">
<p>Hirai, Hidenori, Kyoko Matoba, Emiko Mihara, Takao Arimori, and Junichi Takagi. 2019. “Crystal Structure of a Mammalian Wnt–Frizzled Complex.” <em>Nature Structural &amp; Molecular Biology</em> 26 (5): 372–79. <a href="https://doi.org/10.1038/s41594-019-0216-z">https://doi.org/10.1038/s41594-019-0216-z</a>.</p>
</div>
</div>
</div>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/principle-of-electron-transfer-and-proton-transport.html" class="archive-item-link">
                Principle of electron transfer and proton transport 
              </a>
              <span class="archive-item-date">
                2019-10-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p><strong>Using the physical chemistry principles, discuss factors which are contributed in the favourability of an electron transfer reactions between cofactors.</strong></p>
<p>The reduction potential, <span class="math inline">\(E\)</span>, is used to quantify favourability of gaining electrons (getting reduced). <span class="math inline">\(E^{\circ\prime}\)</span> is the reduction potential measured under biological standard conditions (i.e. pH = 7, oxidised and reduced species are 1M, 298K and 1 atm). When concentrations of the oxidised and reduced species differ:</p>
<p><span class="math display">\[E=E^{\circ\prime} + \dfrac{RT}{zF}\ln\left(\dfrac{\text{[reduced]}}{\text{[oxidised]}}\right)\]</span></p>
<p>For two half equations, A<sup>+</sup> + <span class="math inline">\(e^-\rightarrow\)</span> A and <span class="math inline">\(\text{B} + e^- \rightarrow\)</span> B<sup>-</sup> with <span class="math inline">\(E\)</span> of <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> respectively, a full reaction equation can be constructed, which can be A<sup>+</sup> + B<sup>-</sup> <span class="math inline">\(\rightarrow\)</span> A + B (with <span class="math inline">\(E_\text{r}\)</span> = x - y) or the reverse (with <span class="math inline">\(E_\text{r}\)</span> = y - x); the one with positive <span class="math inline">\(E_\text{r}\)</span> (reaction potential) will actually occur.</p>
<p>Electron transfers occur because electrons are moving down to a lower energy level. In simple cases, atoms with greater electronegativity have greater affinity for electrons and thus a more positive reduction potential, and <em>vice versa</em>. This is the case for strong oxidants F<sub>2</sub> and Cl<sub>2</sub> (electronegative; highly positive <span class="math inline">\(E\)</span>) and strong reductants Li and Na (electropositive, highly negative <span class="math inline">\(E\)</span>). In most cases, however, other factors, such as change in state, conformation, or interactions with water, will significantly affect the reduction potential. For example, Cu and Ag is less electronegative than hydrogen, but their monovalent ions are more easily reduced than H<sup>+</sup>. That is mainly because the formation of metallic solid upon reduction is highly exergonic. Similarly, although the atoms involved in the redox reactions in the repiratory chain are not variable (Cu and Fe), they have different <span class="math inline">\(E\)</span> due to the enviornment (e.g. proximity to the P/N face, identity of ligands).</p>
<p>The series of cofactors along the electron transport chain have a generally steadily increasing <span class="math inline">\(E^{\circ\prime}\)</span>, so electrons are moved toward increasingly lower energy levels. There are some exceptions to this trend, but <span class="math inline">\(E_\text{r}\)</span> would be made positive in these cases by making a concentration difference (usually by means of ‘pulling’ from the next redox centre with favourable <span class="math inline">\(E\)</span>).</p>
<p>While the thermodynamic feasibility of the redox reactions is solely determined by the difference in the reduction potentials (which is proportional to Gibbs free energy), the kinetics (rates) are affected by more factors.</p>
<p>Electrons are passed from one centre to another by <em>tunnelling</em>. Specifically, the wave function for an electron in the donor shows there is a probability that the electron will be found at an acceptor some distance away. <em>Tunnelling</em> is independent of temperature and orientation of chemical groups, but is instead influenced by 1) the distance between the electron donor and acceptor, 2) the redox potential difference, <span class="math inline">\(E\)</span>, and 3) the ‘reorganisation energy’, <span class="math inline">\(\lambda\)</span>.</p>
<p>Within each respiratory complex, the redox centres are close to each other—no more than 14Å apart (the maxium 14Å is found between N5 and N6a Fe-S clusters in complex I). Ubiquinone has binding sites in complex I (II) and III in close proximity to the redox centres from which it receives and to which it delivers electrons, and the same is true for cyt <em>c</em> (which binds to complex III and IV). This arrangement makes the numerous electron transfer steps not the limiting factor in the respiratory chain.</p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p><strong>What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.</strong></p>
<div id="two-proton-translocation-models-loop-and-pump" class="section level2">
<h2>Two Proton Translocation Models: Loop and Pump</h2>
<p>Mechanisms of coupling redox reactions to proton translocation can be classified into two models: the chemical loop, and the conformational pump.</p>
<p>In the loop model shown in Figure <a href="#fig:loop">1</a>, the hydrogens (protons plus electrons) from the substrate (AH<sub>2</sub>) first reduce an mediator molecule (B) on the N side, and this mediator is reoxidised, forcing protons to be released to the P side via one pathway and electrons to flow back to the N side via another pathway. The electrons then reduce another species (C) using protons from the N side. This mechanism is employed in the respiratory chain, where ubiquinone serves as the ‘mediator’ molecule and ETC components from Rieske Fe-S protein in complex III to Cu<sub>B</sub> centre in complex IV as the electron pathway. The number of protons translocated in this way has to follow a fixed stoichiometry (i.e. 1 proton per electron). (Not shown in this diagram, the reduction of Q near the N face driven by QH<sub>2</sub> oxidation near the P face in the Q cycle is also an example of this mechanism.)</p>
<div class="figure" style="text-align: c"><span id="fig:loop"></span>
<img src="../img/etc-mech-1.png" alt="The loop model. Left, generic form; right, simplified example found in the respiratory chain." width="100%" />
<p class="caption">
Figure 1: The loop model. Left, generic form; right, simplified example found in the respiratory chain.
</p>
</div>
<p>The key idea of the pump mechanism is the conformational change driven by redox reactions. Conformational change can lead to alternating proton accessibility from the two faces and differential pKa values of proton binding sites, resulting in proton translocation (explained in Figure <a href="#fig:pump">2</a>). The number of protons pumped in this way does not have to follow a fixed stoichiometry within the thermodynamic constraints (i.e. <em>E</em><sub>r</sub>/n must be greater than ∆p). This mechanism is thought to be employed in complex I and IV of the respiratory chain. However, recent studies have shown that complex IV may adopt a unique mechanism, as explained later in this essay.</p>
<div class="figure" style="text-align: c"><span id="fig:pump"></span>
<img src="../img/etc-mech-2.jpg" alt="The conformational pump mechanism. When the &quot;pump&quot; protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation." width="100%" />
<p class="caption">
Figure 2: The conformational pump mechanism. When the “pump” protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation.
</p>
</div>
</div>
<div id="complex-i" class="section level2">
<h2>Complex I</h2>
<p>As shown in Figure <a href="#fig:3m9s">3</a>, bacterial complex I (from <em>Thermus thermophilus</em>) comprises two domains. The hydrophobic, transmembrane domain (a.k.a. membrane arm) has a proton-translocating P-module and subunit NuoH. The hydrophilic domain (a.k.a. peripheral domain) has an NADH-oxidising dehydrogenase module (N-module) which feeds electrons to the electron-transferring chain of Fe-S clusters, and the connecting Q-module, which reduces ubiquinone. Mammalian mitochondrial complex I is similar in gross organisation and function, although it possesses more subunits.</p>
<div class="figure" style="text-align: c"><span id="fig:3m9s"></span>
<img src="../img/3M9S.png" alt="Annotated structure of bacterial complex I. PDB accession code: 3M9S" width="100%" />
<p class="caption">
Figure 3: Annotated structure of bacterial complex I. PDB accession code: 3M9S
</p>
</div>
<div id="hydrophilic-domain-peripheral-arm" class="section level3">
<h3>Hydrophilic domain (peripheral arm)</h3>
<p>All FMN and the seven (bacteria have additional two redundant ones) Fe-S centres reside in the hydrophilic domain. FMN, which is located at the extremity of the hydrophilic arm, accepts two electrons from NADH<sub>2</sub> and transfers them one by one to the nearest Fe-S centre (N3). Electrons then flow from N3 through a series of isopotential Fe-S centres and finally to N2, whose <span class="math inline">\(E^\circ\)</span> is significantly less negative (-100 mV) <span class="citation">(Sazanov and Hinchliffe 2006)</span>. The ubiquinone is then reduced by two one-electron gaining from N2; the existence of at least two forms of semiquinone is identified by electron paramagnetic resonance.</p>
</div>
<div id="hydrophobic-domain-membrane-arm" class="section level3">
<h3>Hydrophobic domain (membrane arm)</h3>
<p>The hydrophobic domain comprises subunits NuoAHJKLMN, of which NuoLMN are homologous to bacterial Mrp Na<sup>+</sup>/H<sup>+</sup> antiporters (characterised by the conserved 14 transmembrane helices).</p>
<p>At the centre of each antiporter-like subunit (NuoLMN), two transmembrane helices are ‘broken’ and each have a kink known as a ‘π-bulge’, and there is a lysine residue in this region (Glu407 for NuoM) critical to proton pumping, confirmed by mutagenesis studies. There are two symmetry-related plausible proton channels lined by hydrophilic/charged residues, one connecting the central lysine on one broken TM to the P face and the other connecting lysine of the other TM to the N face.</p>
<div class="figure" style="text-align: c"><span id="fig:lys"></span>
<img src="../img/membrane-arm.png" alt="Plausible proton paths" width="100%" />
<p class="caption">
Figure 4: Plausible proton paths
</p>
</div>
<p>As shown in Figure <a href="#fig:proton-path">5</a>, NuoL/M/N may work by using conformational changes (induced from the hydrophilic domain) to transfer a proton from the lysines on the helix with N half channel to the other lysine (or Glu in NuoM) on the helix with P half channel, and if the pK<sub>a</sub>s of the two lysines differ, this leads to unidirectional proton transport. This mechanism is in accordance with the general scheme shown in Figure <a href="#fig:pump">2</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:proton-path"></span>
<img src="../img/proton-path.png" alt="Schematic diagram of proton translocation in complex I" width="100%" />
<p class="caption">
Figure 5: Schematic diagram of proton translocation in complex I
</p>
</div>
<p>The three antiporter-related subunits are thought to be responsible for the translocation of 3 protons per 2 <span class="math inline">\(e^-\)</span>. An additional channel is found at the interface of subunits NuoN,K,J,A (according to <span class="citation">Efremov and Sazanov (2011)</span> and <span class="citation">Sazanov (2015)</span> or NuoH,J,K according to <span class="citation">Hirst (2013)</span> and <span class="citation">Nicholls and Ferguson (2013)</span>) and they are thought to act together to pump another proton with a similar mechanism involving protonation/deprotonation of lysine.</p>
</div>
<div id="coupling" class="section level3">
<h3>Coupling</h3>
<p>It is debated how the redox reactions in the hydrophilic domain triggers conformational change, and how this conformational change is transmitted along the hydrophobic domain.</p>
<p>Some features of the Fe-S centre N2 (the last Fe-S centre in complex I) led to hypotheses in which N2 plays a major role in pumping protons<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><sup>,</sup><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, but currently, the mainstream idea is that the initial conformational change in the hydrophilic domain is contributed mostly by the reduction of ubiquinone.</p>
<p>As claimed by <span class="citation">Wirth et al. (2016)</span>, reduction of ubiquinone is a two-step process, where the Q binding site cycles between a conformation that permits electron transfer (E-state), and a conformation that permits proton transfer (P-state) onto intermediates of ubiquinone. The conformational change associated with this controlled process is assumed to initiate proton translocation.</p>
<p>It was originally suggested that the horizontal HL helix (see Figure <a href="#fig:3m9s">3</a>) in NuoL could move horizontally like a piston to transmit conformational change along the hydrophobic domain<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>, but now, after realising the similarity between NuoH and NuoL/M/N, it is thought that conformational changes associated with π-bulges in the mid-membrane parts are transmitted laterally to NuoN/M/L <span class="citation">(Nicholls and Ferguson 2013)</span>, and, as suggested by <span class="citation">Wirth et al. (2016)</span>, this is achieved by electrostatic coupling in the central hydrophilic axis.</p>
<p>As I mentioned last week <span class="citation">(Shi 2019)</span>, as there is no complete evidence demonstrating how complex I proton pump works, other hypotheses exists, including a radical one recently proposed by <span class="citation">Morelli et al. (2019)</span>, which emphasises the elusiveness of the ‘proton entrance half channel’ (from the middle Lys to the N face) contrasted by the clarity of the ‘proton exit half channel’ (to the P face). Together with another piece of evidence that an obvious proton tunnelling is found at the centre of complex I, Morelli drafted an lateral mechanism of proton circuiting, as shown in Figure <a href="#fig:Morelli2019">6</a></p>
<div class="figure" style="text-align: c"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side" width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side
</p>
</div>
</div>
</div>
<div id="complex-iii-cyt-bc1-complex" class="section level2">
<h2>Complex III (cyt <em>bc<sub>1</sub></em> complex)</h2>
<p>Complex III is a homodimer, with each monomer consisting of three subunits: cytochrome b, the Rieske iron-sulfur protein, and cytochrome c1. Each monomer has two binding sites for ubiquinone, called Q<sup>N</sup> and Q<sup>P</sup>, indicating their proximities to two opposite faces.</p>
<p>Uniquinone oxidation and proton translocation is coupled in the Q cycle, as illustrated in Figure <a href="#fig:ciii">7</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:ciii"></span>
<img src="../img/complexIII.png" alt="The Q cycle. Left, stage 1; right, stage 2." width="100%" />
<p class="caption">
Figure 7: The Q cycle. Left, stage 1; right, stage 2.
</p>
</div>
<p>In stage 1, QH<sub>2</sub> binds to Q<sub>N</sub> of one dimer and a Q binds to the Q<sub>P</sub> of another dimer. The QH<sub>2</sub> at Q<sub>N</sub> is oxidised by two electron acceptors in two steps: the first electron is accepted by the Rieske iron-sulfur protein and passed though cytochrome c<sub>1</sub> and finally to cytochrome c, and second other electron is accepted by cytochrome b, passed through its two hemes (b<sub>H</sub> and b<sub>L</sub>) and finally to the Q at Q<sub>P</sub> on the other monomer, forming semiquinone. The two protons of QH<sub>2</sub> are released into the P face (the molecular details is unclear).</p>
<p>In stage 2, the same process is repeated—one QH<sub>2</sub> is oxidised to Q, its protons released and electrons passed 1) onto one cytochrome c and 2) to the other side—but this time the electron acceptor on the other side is semiquinone. Acceptance of an electron by semiquinone, and the addition of two protons from the matrix, produces QH2.</p>
</div>
<div id="complex-iv" class="section level2">
<h2>Complex IV</h2>
<p>Complex IV (cytochrome oxidase) accepts electrons from cytochrome c to reduce oxygen to water and pumps 2 electrons to the P face.</p>
<p>Previous X-ray studies suggested that complex IV exists as dimers with each monomer comprising 13 subunits <span class="citation">(Tsukihara et al. 1996)</span>. However, it was recently shown by <span class="citation">Zong et al. (2018)</span> that complex IV is actually a 14-subunit monomer, and dimerisation was an artifect due to the dissociation of NDUFA4 subunit during the detergent-based purification steps before crystallisation (NDUFA4 hampers dimerisation).</p>
<div id="reduction-of-oxygen" class="section level3">
<h3>Reduction of oxygen</h3>
<p>The key catalytic activities are found in subunit I and II, as shown in Fig. <a href="#fig:complexIV">8</a></p>
<div class="figure" style="text-align: c"><span id="fig:complexIV"></span>
<img src="../img/complexIV.png" alt="Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II." width="100%" />
<p class="caption">
Figure 8: Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II.
</p>
</div>
<p>Each cytochrome c first donates one electron to Cu<sub>A</sub> located in the globular domain of subunit II. Cu<sub>A</sub> is a binulear centre, but it undergoes one-electron redox reactions. The electron is then passed onto haem <em>a</em>, and then to haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binuclear centre, where an oxygen molecule is bound. 4 electrons from cyt c and 4 protons from the N phase are required to reduce one oxygen molecule to two water molecules. Cu<sub>B</sub> has three His ligands, and the forth empty coordination site mediates oxygen reduction by holding the intermediates (its malfunction leads to generation of ROS).</p>
</div>
<div id="pump-mechanism" class="section level3">
<h3>Pump mechanism</h3>
<p>For each pair of electrons, 2 H<sup>+</sup> from N phase is consumed to make water and another 2 H<sup>+</sup> is pumped intot he P phase.</p>
<p>According to a recent review by <span class="citation">Wikström and Sharma (2018)</span>, the favoured charge-conpensation mechanism is as follows: (see Figure <a href="#fig:civ-p">9</a>)</p>
<ol style="list-style-type: decimal">
<li>when an electron is transferred from haem <em>a</em> to the haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binulear centre (BNC), one proton from Glu242 is loaded onto the ‘proton loading site’ (PLS<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>). This is a purely electrostatic event (not acid-base, as pK<sub>a</sub> of PLS is not high enough).</li>
<li>reprotonated Glu242 transfers another proton to BNC, annihilating the -ve charge in BNC, so H<sup>+</sup> on PLS can leave. There must be a barrier preventing H<sup>+</sup> flowing back to the N side, which can be achieved by raising the energy of the transition state of proton transfer between the Glu242 and PLS.</li>
</ol>
<div class="figure" style="text-align: c"><span id="fig:civ-p"></span>
<img src="../img/civ-p.png" alt="Charge-compensation mechanism of proton pumping in complex IV." width="70%" />
<p class="caption">
Figure 9: Charge-compensation mechanism of proton pumping in complex IV.
</p>
</div>
</div>
</div>
<div id="components-of-pmf-in-chloroplasts-and-mitochondria" class="section level2">
<h2>Components of pmf in chloroplasts and mitochondria</h2>
<p>In mitochondria, the major component of protomotive force (<span class="math inline">\(\Delta p\)</span>) is the membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), while in chloroplasts it is <span class="math inline">\(\Delta\)</span>pH. This is caused by the physiology of these organelles and the electrical properties of membranes.</p>
<p>In chloroplasts, when protons are being pumped into the thylakoid space, the developed membrane potential is balanced out by efflux of K<sup>+</sup> and influx of Cl<sup>–</sup> through voltage-gated channels, such as KEA3 K<sup>+</sup>/H<sup>+</sup> antiporter and voltage-gated VCCN1 Cl<sup>–</sup> channel, so it results in a pure pH difference. TPK3 was originally thought to be one of such channels but was recently shown to be not critical <span class="citation">(Höhner et al. 2019)</span>. There are not many enzymes in the thylakoid lumen, so a low pH is not unsafe. In addition, the high H<sup>+</sup> concentration has regulatroy functions. <span class="citation">(Höhner et al. 2019)</span></p>
<p>If mitochondria were allowed to develop such a pH difference (very basic matrix), this would denature the numerous enzymes, such as those involved in <span class="math inline">\(\beta\)</span>-oxidation and TCA cycle, that operate inside the matrix. In mitochondria, there are not such ion channels to counteract change in <span class="math inline">\(\Delta \Psi\)</span>, so proton translocation results in charge imbalance, leading to <span class="math inline">\(\Delta \Psi\)</span> change<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a>, and a significant membrane potential can be established with only minuscule amount of charge movement.This can be explained by the low capacitance of the membrane. Capacitance is the amount of charge separation needed to develop unit voltage difference, i.e.</p>
<p><span class="math display">\[C=\dfrac{Q}{V}\]</span></p>
<p>Given a constant <span class="math inline">\(V\)</span> (<span class="math inline">\(\Delta \Psi\)</span>), lower capacitance means smaller charge difference is required to build up that voltage difference.</p>
<p>Figure <a href="#fig:capacitance">10</a> shows a simple parallel-plate capacitor, which is analogous to mitochondrial inner membrane.</p>
<div class="figure" style="text-align: c"><span id="fig:capacitance"></span>
<img src="img/capacitance.png" alt="Illustration of a simple parallel-plate capacitor" width="70%" />
<p class="caption">
Figure 10: Illustration of a simple parallel-plate capacitor
</p>
</div>
<p>The electrical capacitance of such a plate can be calculated as:</p>
<p><span class="math display">\[C = \dfrac{\epsilon A}{d}\]</span></p>
<p>where <span class="math inline">\(\epsilon\)</span> is the permittivity of the plate, <span class="math inline">\(A\)</span> is the surface area and <span class="math inline">\(d\)</span> is the distance separating charges. For mitochondrial inner membrane, although <span class="math inline">\(\dfrac{A}{d}\)</span> is large, <span class="math inline">\(\epsilon\)</span> is very small, so the overall capacitance is small.</p>
<div id="a-failed-quantitative-analysis" class="section level4">
<h4>A failed quantitative analysis</h4>
<p>According to <span class="citation">Liang et al. (2017)</span>, the cell membrane capacitance is aroud 10 mF/m<sup>2</sup>, and in the <em>Bioenergetics</em> book <span class="citation">(Nicholls and Ferguson 2013)</span>, the capacitance of the mitochondrial inner membrane takes the same valu. According to a model built by <span class="citation">Mannella, Lederer, and Jafri (2013)</span>, one mitochondria with matrix volume of <span class="math inline">\(9.6\times10^4\)</span> nm^3 would have a inner membrane (cristae) surface area of <span class="math inline">\(5.76\times10^6\)</span> nm<sup>2</sup>.</p>
<p>With these values, to build up a <span class="math inline">\(\Delta \Psi\)</span> of -140 mV, the charge required is:</p>
<p><span class="math display">\[Q=CV=0.01\text{F m}^{-2} \times 5.76 \times 10^{-12}\text{m}^{-2}\times 0.14\text{V}=8.1 \times 10^{-15} \text{ C}\]</span></p>
<p>which corresponds to <span class="math inline">\(8.1 \times 10^{-15}/96450=8.4 \times 10^{-20}\)</span> moles of H<sup>+</sup>, or <span class="math inline">\(8.4 \times 10^{-20}/(9.6\times 10^{-20})=0.875\)</span> M (concentration), which is not realistic…</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Efremov-2011">
<p>Efremov, Rouslan G., and Leonid A. Sazanov. 2011. “Structure of the Membrane Domain of Respiratory Complex I.” <em>Nature</em> 476 (7361): 414–20. <a href="https://doi.org/10.1038/nature10330">https://doi.org/10.1038/nature10330</a>.</p>
</div>
<div id="ref-Hirst-2013">
<p>Hirst, Judy. 2013. “Mitochondrial Complex I.” <em>Annual Review of Biochemistry</em> 82 (1). Annual Reviews: 551–75. <a href="https://doi.org/10.1146/annurev-biochem-070511-103700">https://doi.org/10.1146/annurev-biochem-070511-103700</a>.</p>
</div>
<div id="ref-Hohner-2019">
<p>Höhner, Ricarda, Viviana Correa Galvis, Deserah D. Strand, Carsten Völkner, Moritz Krämer, Michaela Messer, Firdevs Dinc, et al. 2019. “Photosynthesis in Arabidopsis Is Unaffected by the Function of the Vacuolar K<span class="math inline">\(&lt;\)</span>sup<span class="math inline">\(&gt;\)</span>+<span class="math inline">\(&lt;\)</span>/Sup<span class="math inline">\(&gt;\)</span> Channel Tpk3.” <em>Plant Physiology</em> 180 (3): 1322. <a href="https://doi.org/10.1104/pp.19.00255">https://doi.org/10.1104/pp.19.00255</a>.</p>
</div>
<div id="ref-Liang:2017">
<p>Liang, Wenfeng, Yuliang Zhao, Lianqing Liu, Yuechao Wang, Wen Jung Li, and Gwo-Bin Lee. 2017. “Determination of Cell Membrane Capacitance and Conductance via Optically Induced Electrokinetics.” <em>Biophysical Journal</em> 113 (7): 1531–9. <a href="https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006">https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006</a>.</p>
</div>
<div id="ref-Mannella-2013">
<p>Mannella, Carmen A., W. Jonathan Lederer, and M. Saleet Jafri. 2013. “The Connection Between Inner Membrane Topology and Mitochondrial Function.” <em>Journal of Molecular and Cellular Cardiology</em> 62: 51–57. <a href="https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001">https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Nelson-2017">
<p>Nelson, David L., and Michael M. Cox. 2017. <em>Lehninger Principles of Biochemistry</em>. Book. 7th ed. W. H. Freeman. <a href="https://search.library.wisc.edu/catalog/999964334502121">https://search.library.wisc.edu/catalog/999964334502121</a>.</p>
</div>
<div id="ref-Nicholls-2013">
<p>Nicholls, David G., and Stuart J. Ferguson. 2013. <em>Bioenergetics</em>. Book. 4th ed. Amsterdam: Academic Press, Elsevier.</p>
</div>
<div id="ref-Roessler-2010">
<p>Roessler, Maxie M., Martin S. King, Alan J. Robinson, Fraser A. Armstrong, Jeffrey Harmer, and Judy Hirst. 2010. “Direct Assignment of Epr Spectra to Structurally Defined Iron-Sulfur Clusters in Complex I by Double Electron–Electron Resonance.” <em>Proceedings of the National Academy of Sciences</em> 107 (5): 1930. <a href="https://doi.org/10.1073/pnas.0908050107">https://doi.org/10.1073/pnas.0908050107</a>.</p>
</div>
<div id="ref-Sazanov-2015">
<p>Sazanov, Leonid A. 2015. “A Giant Molecular Proton Pump: Structure and Mechanism of Respiratory Complex I.” <em>Nature Reviews Molecular Cell Biology</em> 16. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved. SN -: 375 EP. <a href="https://doi.org/10.1038/nrm3997">https://doi.org/10.1038/nrm3997</a>.</p>
</div>
<div id="ref-Sazanov-2006">
<p>Sazanov, Leonid A., and Philip Hinchliffe. 2006. “Structure of the Hydrophilic Domain of Respiratory Complex I from Thermus Thermophilus.” <em>Science</em> 311 (5766): 1430. <a href="https://doi.org/10.1126/science.1123809">https://doi.org/10.1126/science.1123809</a>.</p>
</div>
<div id="ref-Shi-2019">
<p>Shi, Tianyi. 2019. “Principle of Chemiosmotic Mechanism.” 2019.</p>
</div>
<div id="ref-Tsukihara-1996">
<p>Tsukihara, Tomitake, Hiroshi Aoyama, Eiki Yamashita, Takashi Tomizaki, Hiroshi Yamaguchi, Kyoko Shinzawa-Itoh, Ryosuke Nakashima, Rieko Yaono, and Shinya Yoshikawa. 1996. “The Whole Structure of the 13-Subunit Oxidized Cytochrome c Oxidase at 2.8 Å.” <em>Science</em> 272 (5265): 1136. <a href="https://doi.org/10.1126/science.272.5265.1136">https://doi.org/10.1126/science.272.5265.1136</a>.</p>
</div>
<div id="ref-Verkhovskaya-2013">
<p>Verkhovskaya, Marina, and Dmitry A. Bloch. 2013. “Energy-Converting Respiratory Complex I: On the Way to the Molecular Mechanism of the Proton Pump.” <em>The International Journal of Biochemistry &amp; Cell Biology</em> 45 (2): 491–511. <a href="https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024">https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024</a>.</p>
</div>
<div id="ref-Wikstrom-2018">
<p>Wikström, Mårten, and Vivek Sharma. 2018. “Proton Pumping by Cytochrome c Oxidase – a 40 year Anniversary.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1859 (9): 692–98. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009">https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009</a>.</p>
</div>
<div id="ref-Wirth-2016">
<p>Wirth, Christophe, Ulrich Brandt, Carola Hunte, and Volker Zickermann. 2016. “Structure and Function of Mitochondrial Complex I.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1857 (7): 902–14. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013">https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013</a>.</p>
</div>
<div id="ref-Zong-2018">
<p>Zong, Shuai, Meng Wu, Jinke Gu, Tianya Liu, Runyu Guo, and Maojun Yang. 2018. “Structure of the Intact 14-Subunit Human Cytochrome c Oxidase.” <em>Cell Research</em> 28 (10): 1026–34. <a href="https://doi.org/10.1038/s41422-018-0071-1">https://doi.org/10.1038/s41422-018-0071-1</a>.</p>
</div>
<div id="ref-Zwicker-2006">
<p>Zwicker, K., A. Galkin, S. Drose, L. Grgic, S. Kerscher, and U. Brandt. 2006. “The Redox-Bohr Group Associated with Iron-Sulfur Cluster N2 of Complex I.” Journal Article. <em>J Biol Chem</em> 281 (32): 23013–7. <a href="https://doi.org/10.1074/jbc.M603442200">https://doi.org/10.1074/jbc.M603442200</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>In some papers <span class="citation">(Hirst 2013; Verkhovskaya and Bloch 2013)</span> it was suggested that coupling of electron transport to movement of protons depends on the significantly exergonic reduction of the N2 centre by the roughly isopotential wire of Fe-S centres that delivers electrons from NADH, and the large E<sub>m,7</sub> span from N3 to N2 must provide the majority of the driving force for the proton pumping events associated with reduction and reoxidation of N2. This may be incorrect, as newest measurements <span class="citation">(Sazanov 2015)</span> of redox potentials of the full Fe-S series (N3-N1b-N4-N5-N6a-N6b-N2) shows that N3, N4 and N6 are about equipotential at about -250 mV, while N1b, N5 and N6b actually have lower potentials; and such an arrangement is claimed by <span class="citation">Roessler et al. (2010)</span> to be energy efficient.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>The mid-point reduction potential of N2 is shown to be pH-dependent, so that E<sub>acid</sub> &gt; E<sub>alkaline</sub>, and indeed a proton binding site is present there: His226 in <em>Y. lipolytica</em>, and protons are more likely to bind when N2 is reduced. However, shifting the midpoint potential to a more negative and pH insensitive value by exchanging His226 with a methionine affected the functioning of complex I including proton pumping only marginally. <span class="citation">(Zwicker et al. 2006)</span><a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>The alternative role of HL would be to stabilise ends of the helices while permitting movement of the middle hydrophilic axis.<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>Although the identity of PLS is uncertain, many studies indicates that it may be the A-propionate substituent of haem <em>a</em><sub>3</sub>.<a href="#fnref4" class="footnote-back">↩</a></p></li>
<li id="fn5"><p>To be precise, when ∆p = 0, there is still (negative) membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), and H<sup>+</sup> equilibrate under this <span class="math inline">\(\Delta \Psi\)</span>, which results in a lower pH inside (more concentrated H<sup>+</sup>). The <span class="math inline">\(\Delta \Psi\)</span> that contributes to ∆p is actually the <span class="math inline">\(change\)</span> in <span class="math inline">\(\Delta \Psi\)</span> caused by proton translocation.<a href="#fnref5" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../lab/2019-10-16-specific-radioactivity.html" class="archive-item-link">
                Specific Radioactivity 
              </a>
              <span class="archive-item-date">
                2019-10-20
              </span>
              <div class="article-summary" style="display: none;">
                
                  Specific radioactivity of ATP and glucose.<hr>
                
                


<div id="TOC">
<ul>
<li><a href="#aim"><span class="toc-section-number">1</span> Aim</a></li>
<li><a href="#reagents-and-apparatus-provided"><span class="toc-section-number">2</span> Reagents and Apparatus Provided</a></li>
<li><a href="#method"><span class="toc-section-number">3</span> Method</a><ul>
<li><a href="#method-atp"><span class="toc-section-number">3.1</span> ATP concentration</a><ul>
<li><a href="#dilution"><span class="toc-section-number">3.1.1</span> Dilution</a></li>
<li><a href="#results"><span class="toc-section-number">3.1.2</span> Results</a></li>
</ul></li>
<li><a href="#method-glucose"><span class="toc-section-number">3.2</span> Glucose concentration</a><ul>
<li><a href="#mechanism"><span class="toc-section-number">3.2.1</span> Mechanism</a></li>
<li><a href="#determine-the-volume-of-enzyme"><span class="toc-section-number">3.2.2</span> Determine the volume of enzyme</a></li>
<li><a href="#determining-lambda_textmax"><span class="toc-section-number">3.2.3</span> Determining <span class="math inline">\(\lambda_\text{max}\)</span></a></li>
<li><a href="#calibration"><span class="toc-section-number">3.2.4</span> Calibration</a></li>
<li><a href="#sample-assay"><span class="toc-section-number">3.2.5</span> Sample Assay</a></li>
<li><a href="#results-1"><span class="toc-section-number">3.2.6</span> Results</a></li>
</ul></li>
<li><a href="#radioactivity-counting-virtual"><span class="toc-section-number">3.3</span> Radioactivity Counting (Virtual)</a></li>
</ul></li>
<li><a href="#our-results"><span class="toc-section-number">4</span> Our Results</a><ul>
<li><a href="#our-atp"><span class="toc-section-number">4.1</span> ATP</a><ul>
<li><a href="#distinguish"><span class="toc-section-number">4.1.1</span> Distinguishing between 4.0mM and 4.5mM ATP</a></li>
</ul></li>
<li><a href="#our-glucose"><span class="toc-section-number">4.2</span> Glucose</a><ul>
<li><a href="#glucose-calibration-results"><span class="toc-section-number">4.2.1</span> Calibration</a></li>
<li><a href="#our-glucose-sample"><span class="toc-section-number">4.2.2</span> Sample result</a></li>
<li><a href="#distinguish-glucose"><span class="toc-section-number">4.2.3</span> Distinguishing between 4.0mM and 4.5mM glucose</a></li>
</ul></li>
</ul></li>
<li><a href="#class-results"><span class="toc-section-number">5</span> Class Results</a><ul>
<li><a href="#dataremoved"><span class="toc-section-number">5.1</span> Data</a></li>
<li><a href="#preliminary-graphical-analysis"><span class="toc-section-number">5.2</span> Preliminary graphical analysis</a></li>
<li><a href="#class-atp"><span class="toc-section-number">5.3</span> ATP concentration</a><ul>
<li><a href="#grouping-and-grouped-confidence-interval-of-mean"><span class="toc-section-number">5.3.1</span> Grouping and grouped confidence interval of mean</a></li>
<li><a href="#test-for-significance-of-the-difference-in-concentration-between-the-two-groups"><span class="toc-section-number">5.3.2</span> Test for significance of the difference in concentration between the two groups</a></li>
<li><a href="#test-for-diviation-from-given-concentrations"><span class="toc-section-number">5.3.3</span> Test for diviation from given concentrations</a></li>
</ul></li>
<li><a href="#class-glucose"><span class="toc-section-number">5.4</span> Glucose</a><ul>
<li><a href="#grouping-and-grouped-confidence-interval-of-mean-1"><span class="toc-section-number">5.4.1</span> Grouping and grouped confidence interval of mean</a></li>
<li><a href="#test-for-significance-of-the-difference-in-concentration-between-the-two-groups-1"><span class="toc-section-number">5.4.2</span> Test for significance of the difference in concentration between the two groups</a></li>
<li><a href="#test-for-diviation-from-given-concentrations-1"><span class="toc-section-number">5.4.3</span> Test for diviation from given concentrations</a></li>
</ul></li>
<li><a href="#radioactivity"><span class="toc-section-number">5.5</span> Radioactivity</a><ul>
<li><a href="#results-of-the-sample"><span class="toc-section-number">5.5.1</span> Results of the sample</a></li>
<li><a href="#correction-for-overlap"><span class="toc-section-number">5.5.2</span> Correction for overlap</a></li>
<li><a href="#specific-radioactivity"><span class="toc-section-number">5.5.3</span> Specific Radioactivity</a></li>
</ul></li>
</ul></li>
<li><a href="#appendix"><span class="toc-section-number">6</span> Appendix</a><ul>
<li><a href="#temp"><span class="toc-section-number">6.1</span> People should report all measurements to get a true 95% confidence interval</a></li>
<li><a href="#the-mathematical-computation-behind-r-functions-used-in-this-report"><span class="toc-section-number">6.2</span> The mathematical computation behind R functions used in this report</a></li>
</ul></li>
</ul>
</div>

<div id="aim" class="section level1">
<h1><span class="header-section-number">1</span> Aim</h1>
<ul>
<li>Given a sample containing (fake) radioactive <sup>14</sup>C glucose and <sup>3</sup>H ATP:
<ul>
<li>determine concentration of glucose and ATP.
<ul>
<li>Given that both concentrations lie between 1mM and 10mM.</li>
<li>Given that <span class="math inline">\(\epsilon^{260}_\text{ATP}=1.57\times10^4 \text{ mol}^{-1}\text{ dm}^3\text{ dm}^{-1}\)</span></li>
</ul></li>
<li>determine radioactivity of the sample (virtual practical)</li>
<li>and hence determine the specific radioactivity of ATP and glucose</li>
</ul></li>
<li>23 samples have glucose and ATP concentrations falling into two distinct classes. They are independently measured by 23 pairs of students. Determine these concentrations using the class results and assess precision and accuracy.</li>
</ul>
</div>
<div id="reagents-and-apparatus-provided" class="section level1">
<h1><span class="header-section-number">2</span> Reagents and Apparatus Provided</h1>
<ul>
<li>15mL sample solution</li>
<li>0.1 M Na<sub>3</sub>PO<sub>4</sub> buffer at pH 7.0</li>
<li>1mM ABTS</li>
<li>Glucose Oxidase (100 U/mL) and peroxidase (20 U/mL)</li>
<li>Solid glucose</li>
<li>Spectrophotometer</li>
</ul>
</div>
<div id="method" class="section level1">
<h1><span class="header-section-number">3</span> Method</h1>
<div id="method-atp" class="section level2">
<h2><span class="header-section-number">3.1</span> ATP concentration</h2>
<div id="dilution" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Dilution</h3>
<div id="determining-scale-of-dilution" class="section level4">
<h4><span class="header-section-number">3.1.1.1</span> Determining scale of dilution</h4>
<p>Given that 1mM &lt; <a href="#our-atp">ATP</a> &lt; 10mM and that <span class="math inline">\(\epsilon^{260}_\text{ATP}=1.57\times10^4 \text{ mol}^{-1}\text{ dm}^3\text{ dm}^{-1}\)</span>, according to <span class="math inline">\(A^{260}=\epsilon^{260}cl\)</span>:</p>
<p>if no dilution:</p>
<p><span class="math display">\[
\begin{aligned}
1.57\times10^4\times0.001&lt;&amp;A^{260}&lt;1.57\times10^4\times0.01 \\
15.7&lt;&amp;A^{260}_\text{raw}&lt;157 \\
\end{aligned}
\]</span></p>
<p>clearly, a 1:100 dilution would be very likely to produce a sensible <span class="math inline">\(A^{260}\)</span>:</p>
<p><span class="math display">\[0.157&lt;A^{260}_\text{1:100 dilution}&lt;1.57\]</span></p>
</div>
<div id="making-dilution" class="section level4">
<h4><span class="header-section-number">3.1.1.2</span> Making dilution</h4>
<p>5mL of 1:100 diluton is made serially by first fixing 500 <span class="math inline">\(\mu\text{L}\)</span> sample with 4.5mL buffer then mixing 500 <span class="math inline">\(\mu\text{L}\)</span> of the resulting 1:10 dilution with another 4.5mL buffer.</p>
<p>A preliminary test gives <span class="math inline">\(A^{260}_\text{1:100 dilution}=0.503\)</span>, which is within the range where the spectrophotometer is accurate, so we made 4 more 1:100 dilutions using the method described above.</p>
</div>
</div>
<div id="results" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Results</h3>
<p>See section <a href="#our-atp">4.1</a></p>
</div>
</div>
<div id="method-glucose" class="section level2">
<h2><span class="header-section-number">3.2</span> Glucose concentration</h2>
<div id="mechanism" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Mechanism</h3>
<p>Glucose itself does not have convenient absorption. Thus, glucose oxidase, ABTS and peroxidase is added to make a green solution (glucose is first oxidised by oxidase to produce gluconic acid and H<sub>2</sub>O<sub>2</sub>; peroxidase then catalyses the reaction between H<sub>2</sub>O<sub>2</sub> and ABTS to produce a green dye).</p>
</div>
<div id="determine-the-volume-of-enzyme" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Determine the volume of enzyme</h3>
<p>We want the reaction to ‘virtually’ complete (i.e. &gt; 99% conversion of substrate) in 10 min.</p>
<p>For an enzyme reaction:</p>
<p><span class="math display">\[-\dfrac{ds}{dt}=v-\dfrac{V_\text{max}s}{K_m+S}\]</span></p>
<p>As the reaction approaches completion, <span class="math inline">\(s&lt;&lt;K_m\)</span>:</p>
<p><span class="math display">\[-\dfrac{ds}{dt}=\dfrac{V_\text{max}s}{K_m}\]</span></p>
<p>Integrating to determine the total time for a given change in <span class="math inline">\(s\)</span>:</p>
<p><span class="math display">\[
\begin{aligned}
-\int_{s_0}^{s_1}{\dfrac{ds}{s}}&amp;=\dfrac{V_\text{max}}{K_m}\int_0^t dt \\
\ln\dfrac{s_0}{s_1}&amp;=\dfrac{V_\text{max}t}{K_m}
\end{aligned}
\]</span></p>
<p>For a decrease in <span class="math inline">\(s\)</span> from 100% to 1% in 10 min:</p>
<p><span class="math display">\[
\begin{aligned}
\ln\dfrac{100}{1}&amp;=\dfrac{V_\text{max}\times 10}{K_m} \\
V_\text{max}&amp;=0.4605K_m \text{ (mol L}^{-1}\text{min}^{-1}\text{)}
\end{aligned}
\]</span></p>
<p>Our assay volume will be 4.0mL, so the amound of enzyme, in U (<span class="math inline">\(\mu\text{mol min}^{-1}\)</span>), will be</p>
<p><span class="math display">\[\text{amount of enzyme in U}=0.4605\times K_m \times 4.0 \times 10^{-3}\times10^6=1842K_m\]</span></p>
<p>where <span class="math inline">\(K_m\)</span> has unit M.</p>
<p>For GOD, <span class="math inline">\(K_m=1\text{ mM}\)</span>, and the provided enzyme concentration is 100U/mL:</p>
<p><span class="math display">\[
\begin{aligned}
K_m&amp;=1\text{ mM} \\
\text{minimum amount of enzyme}&amp;=1842 \times 1 \times 10^{-3}=1.84 \text{ U} \\
\text{minimum volume of enzyme}&amp;=1.84 \div 100=0.0184 \text{ mL}
\end{aligned}
\]</span></p>
<p>For POD, <span class="math inline">\(K_m=100\text{ }\mu\text{M}\)</span>, and the provided enzyme concentration is 20U/mL:</p>
<p><span class="math display">\[
\begin{aligned}
K_m&amp;=100\text{ }\mu\text{M} \\
\text{minimum amount of enzyme}&amp;=1842 \times 100 \times 10^{-6}=0.184 \text{ U} \\
\text{minimum volume of enzyme}&amp;=0.184 \div 20=0.0092 \text{ mL}
\end{aligned}
\]</span></p>
<p>We decided to use an excess amount of enzyme (i.e. 0.1mL for both). This guarantees that the reactions will complete within 10 min, and makes it easier to make up the 4mL total volume for assay (see section <a href="#glu-assay">3.2.4.2</a>).</p>
</div>
<div id="determining-lambda_textmax" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Determining <span class="math inline">\(\lambda_\text{max}\)</span></h3>
<p>Measuring absorbance near <span class="math inline">\(\lambda_\text{max}\)</span> reduces error. An 100nmol-glucose-containing (i.e. with 1mL 0.1mM glucose solution, see section <a href="#glu-assay">3.2.4.2</a>) assay result is used. The absorbance in the range 350nm to 450nm is measured, and <span class="math inline">\(\lambda_\text{max} = 418\text{ nm}\)</span>. We then used this wavelength for all glucose assays.</p>
</div>
<div id="calibration" class="section level3">
<h3><span class="header-section-number">3.2.4</span> Calibration</h3>
<p>Calibrate <span class="math inline">\(A^{418}\)</span> with known glucose concentrations, then we can map <span class="math inline">\(A^{418}_\text{sample}\)</span> to the calibration curve to determine sample glucose concentration.</p>
<div id="dilution-1" class="section level4">
<h4><span class="header-section-number">3.2.4.1</span> Dilution</h4>
<ul>
<li>0.18 g glucose is dissolved in 100mL buffer, which gives 10mM glucose solution (which is handy, as the maximum possible concentration of glucose in the sample is also 10mM)</li>
<li>100mL 0.1mM glucose solution is made by mixing 1mL 10mM glucose solution with 99mL buffer (1:100 dilution).</li>
</ul>
</div>
<div id="glu-assay" class="section level4">
<h4><span class="header-section-number">3.2.4.2</span> Assay</h4>
<p>Assays are made on 4mL mixtures with varying amount of glucose, according to the following table:</p>
<table>
<thead>
<tr class="header">
<th align="center">ABTS</th>
<th align="center">Buffer</th>
<th align="center">0.1mM glucose</th>
<th align="center">H<sub>2</sub>O</th>
<th align="center">GOD</th>
<th align="center">POD</th>
<th align="center">nmol glucose</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">1.0mL</td>
<td align="center">0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">100</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.8mL</td>
<td align="center">0.2mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">80</td>
</tr>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.6mL</td>
<td align="center">0.4mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">60</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.4mL</td>
<td align="center">0.6mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">40</td>
</tr>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.2mL</td>
<td align="center">0.8mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">20</td>
</tr>
<tr class="even">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">0mL</td>
<td align="center">1.0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center">0</td>
</tr>
</tbody>
</table>
<p>Then the mixture is incubated at 37<span class="math inline">\(^\circ\text{C}\)</span> for 10 minutes (in a theromostatically-controlled water bath), then <span class="math inline">\(A^{418}\)</span> is measured.</p>
<p>Every row is repeated 3 times.</p>
</div>
</div>
<div id="sample-assay" class="section level3">
<h3><span class="header-section-number">3.2.5</span> Sample Assay</h3>
<p>The sample is diluted 1:100 by mixing 1mL sample with 99mL buffer.</p>
<p>4mL assays are made on 1mL 1:100 dilution 3 times, according to the following table (identical to the assays used for calibration except that an unknown concentration is used this time)</p>
<table>
<thead>
<tr class="header">
<th align="center">ABTS</th>
<th align="center">Buffer</th>
<th align="center">diluted sample</th>
<th align="center">GOD</th>
<th align="center">POD</th>
<th align="center"></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">2mL</td>
<td align="center">0.8mL</td>
<td align="center">1.0mL</td>
<td align="center">0.1mL</td>
<td align="center">0.1mL</td>
<td align="center"></td>
</tr>
</tbody>
</table>
</div>
<div id="results-1" class="section level3">
<h3><span class="header-section-number">3.2.6</span> Results</h3>
<p>See section <a href="#our-glucose">4.2</a></p>
</div>
</div>
<div id="radioactivity-counting-virtual" class="section level2">
<h2><span class="header-section-number">3.3</span> Radioactivity Counting (Virtual)</h2>
<p>0.2mL sample is mixed with 1.8mL scintillation fluid in a polyvial and placed into the scintillation counter. The count time is 2min. The results are shown in section <a href="#radioactivity">5.5</a>.</p>
</div>
</div>
<div id="our-results" class="section level1">
<h1><span class="header-section-number">4</span> Our Results</h1>
<div id="our-atp" class="section level2">
<h2><span class="header-section-number">4.1</span> ATP</h2>
<p>(Methods in section <a href="#method-atp">3.1</a>)</p>
<pre class="r"><code>A &lt;- c(0.503, 0.513, 0.508, 0.525, 0.516)
c_ATP &lt;- A/1.57e4*100*1000
kable(tibble(`$A^{260}_\\text{1:100 dilution}$`=A, `[ATP] of raw sample (mM)`=c_ATP))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right"><span class="math inline">\(A^{260}_\text{1:100 dilution}\)</span></th>
<th align="right"><a href="#our-atp">ATP</a> of raw sample (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">0.503</td>
<td align="right">3.203822</td>
</tr>
<tr class="even">
<td align="right">0.513</td>
<td align="right">3.267516</td>
</tr>
<tr class="odd">
<td align="right">0.508</td>
<td align="right">3.235669</td>
</tr>
<tr class="even">
<td align="right">0.525</td>
<td align="right">3.343949</td>
</tr>
<tr class="odd">
<td align="right">0.516</td>
<td align="right">3.286624</td>
</tr>
</tbody>
</table>
<p>Statistical summary of <a href="#our-atp">ATP</a> in sample:</p>
<pre class="r"><code>x &lt;- c_ATP; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 3.26751592 0.05309978 0.02374695 2.77644511 0.06593209</code></pre>
<p>Which gives 95% confidence interval: <span class="math inline">\(\text{[ATP]}=3.268\pm0.054\)</span> mM</p>
<div id="distinguish" class="section level3">
<h3><span class="header-section-number">4.1.1</span> Distinguishing between 4.0mM and 4.5mM ATP</h3>
<p>The percentage error (coefficient of variance) of this experiment is:</p>
<p><span class="math display">\[c_v=\dfrac{s_x}{\bar{x}}=\dfrac{0.054}{3.268}=0.0165\]</span></p>
<p>for <span class="math inline">\(\bar{x}\)</span> = 4.0 and 4.5, <span class="math inline">\(s_x=c_v\bar{x}\)</span> = 0.0661 and 0.0744, respectively.</p>
<p>To distinguish between the two, two sample t test is used. If <span class="math inline">\(n\)</span> measurements are made on each concentration, then</p>
<p><span class="math display">\[t=\dfrac{\bar{x_1}-\bar{x_2}}{\sqrt{\dfrac{s_1^2}{n}+\dfrac{s_2^2}{n}}}=\dfrac{4.5-4.0}{\sqrt{\dfrac{0.0661^2+0.0744^2}{n}}}=\sqrt{25.26n}\]</span></p>
<p>then we can get a list of <span class="math inline">\(p\)</span> values for each <span class="math inline">\(n\)</span>:</p>
<pre class="r"><code>n &lt;- 2:8
df &lt;- n - 1
t &lt;- sqrt(25.26 * n)
p &lt;- (1 - pt(t, df)) * 2
names(p) &lt;- paste(&#39;n =&#39;, n)
round(p, 5)</code></pre>
<pre><code>##   n = 2   n = 3   n = 4   n = 5   n = 6   n = 7   n = 8 
## 0.08898 0.01294 0.00210 0.00036 0.00006 0.00001 0.00000</code></pre>
<p>as n increases, <span class="math inline">\(p\)</span> decreases. When n = 3, <span class="math inline">\(p\)</span> is already less than 0.05.</p>
</div>
</div>
<div id="our-glucose" class="section level2">
<h2><span class="header-section-number">4.2</span> Glucose</h2>
<p>(Methods in section <a href="#method-glucose">3.2</a>)</p>
<div id="glucose-calibration-results" class="section level3">
<h3><span class="header-section-number">4.2.1</span> Calibration</h3>
<p>The results are shown below:</p>
<pre class="r"><code>A1 &lt;- c(0.952, 0.815, 0.709, 0.496, 0.272, 0)
A2 &lt;- c(1.223, 0.969, 0.704, 0.415, 0.222, 0)
A3 &lt;- c(1.194, 0.919, 0.682, 0.452, 0.236, 0)
A &lt;- c(A1, A2, A3)
glucose = c(100, 80, 60, 40, 20, 0)
kable(tibble(`nmol glucose in 4mL assay` = glucose, `$A^{418}_1$` = A1,
             `$A^{418}_2$` = A2, `$A^{418}_3$` = A3))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right">nmol glucose in 4mL assay</th>
<th align="right"><span class="math inline">\(A^{418}_1\)</span></th>
<th align="right"><span class="math inline">\(A^{418}_2\)</span></th>
<th align="right"><span class="math inline">\(A^{418}_3\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">100</td>
<td align="right">0.952</td>
<td align="right">1.223</td>
<td align="right">1.194</td>
</tr>
<tr class="even">
<td align="right">80</td>
<td align="right">0.815</td>
<td align="right">0.969</td>
<td align="right">0.919</td>
</tr>
<tr class="odd">
<td align="right">60</td>
<td align="right">0.709</td>
<td align="right">0.704</td>
<td align="right">0.682</td>
</tr>
<tr class="even">
<td align="right">40</td>
<td align="right">0.496</td>
<td align="right">0.415</td>
<td align="right">0.452</td>
</tr>
<tr class="odd">
<td align="right">20</td>
<td align="right">0.272</td>
<td align="right">0.222</td>
<td align="right">0.236</td>
</tr>
<tr class="even">
<td align="right">0</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
<td align="right">0.000</td>
</tr>
</tbody>
</table>
<p>To fit a straight line, linear regression is used. The coefficients are shown below:</p>
<pre class="r"><code>summary(lm(A ~ 0 + rep(glucose, 3)))</code></pre>
<pre><code>## 
## Call:
## lm(formula = A ~ 0 + rep(glucose, 3))
## 
## Residuals:
##       Min        1Q    Median        3Q       Max 
## -0.181273 -0.000982  0.005691  0.039277  0.089727 
## 
## Coefficients:
##                  Estimate Std. Error t value Pr(&gt;|t|)    
## rep(glucose, 3) 0.0113327  0.0002387   47.48   &lt;2e-16 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.06132 on 17 degrees of freedom
## Multiple R-squared:  0.9925, Adjusted R-squared:  0.9921 
## F-statistic:  2254 on 1 and 17 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>which shows there is a linear relationship <span class="math inline">\(A^{418}=0.01133n_\text{glucose}\)</span> with <span class="math inline">\(R^2=0.9921\)</span>.</p>
<p>Then we can plot it (CI added).</p>
<pre class="r"><code>ggplot(data.frame(A, glucose), aes(glucose, A))+
  geom_point()+
  geom_smooth(formula = y ~ 0 + x, method = &#39;lm&#39;)+
  labs(x = &#39;nmol glucose in 4mL assay&#39;, y = expression(A ^ 418),
       title = &#39;Calibrationn Curve for 4mL essay&#39;)</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-7-1.png" width="60%" style="display: block; margin: auto;" /></p>
</div>
<div id="our-glucose-sample" class="section level3">
<h3><span class="header-section-number">4.2.2</span> Sample result</h3>
<p>Three independent measurements of <span class="math inline">\(A^{418}_\text{sample}\)</span> gives 0.863, 0.872 and 0.871. According to the linear relationship <span class="math inline">\(A^{418}=0.01133n_\text{glucose}\)</span> we can calculate the amount (in nmol) of glucose in 1mL 1:100 dilution, then the concentration of the raw sample can be calculated easily:</p>
<pre class="r"><code>A &lt;- c(0.863, 0.872, 0.871) # sample absorbance
n &lt;- A/0.01133 # nmol glucose in 1 mL 1:100 dilution
(c_glucose_sample &lt;- n * 1e-9 # converting to mol/mL (1:100 dilution)
       / 1e-3 # converting to M (1:100 dilution)
       * 100 # in M (raw sample)
       * 1e3) # converting to mM (raw sample)</code></pre>
<pre><code>## [1] 7.616946 7.696381 7.687555</code></pre>
<p>Statistical summary:</p>
<pre class="r"><code>x &lt;- c_glucose_sample; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t * sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 7.66696087 0.04353824 0.02513682 4.30265273 0.10815499</code></pre>
<p>Which gives 95% confidence interval: <span class="math inline">\(\text{[glucose]}=7.667\pm0.057\)</span> mM</p>
</div>
<div id="distinguish-glucose" class="section level3">
<h3><span class="header-section-number">4.2.3</span> Distinguishing between 4.0mM and 4.5mM glucose</h3>
<p>Similar to the calculation for ATP shown in section <a href="#distinguish">4.1.1</a>, first calculate the percentage error (coefficient of variance) of this experiment:</p>
<p><span class="math display">\[c_v=\dfrac{s_x}{\bar{x}}=\dfrac{0.057}{7.667}=0.00743\]</span></p>
<p>for <span class="math inline">\(\bar{x}\)</span> = 4.0 and 4.5, <span class="math inline">\(s_x=c_v\bar{x}\)</span> = 0.0297 and 0.0335, respectively.</p>
<p>To distinguish between the two, two sample t test is used. If <span class="math inline">\(n\)</span> measurements are made on each concentration, then</p>
<p><span class="math display">\[t=\dfrac{\bar{x_1}-\bar{x_2}}{\sqrt{\dfrac{s_1^2}{n}+\dfrac{s_2^2}{n}}}=\dfrac{4.5-4.0}{\sqrt{\dfrac{0.0297^2+0.0335^2}{n}}}=\sqrt{124.7n}\]</span></p>
<p>then we can get a list of <span class="math inline">\(p\)</span> values for each <span class="math inline">\(n\)</span>:</p>
<pre class="r"><code>n &lt;- 2:8
df &lt;- n - 1
t &lt;- sqrt(124.7 * n)
p &lt;- (1 - pt(t, df)) * 2
names(p) &lt;- paste(&#39;n =&#39;, n)
round(p, 5)</code></pre>
<pre><code>##   n = 2   n = 3   n = 4   n = 5   n = 6   n = 7   n = 8 
## 0.04026 0.00266 0.00020 0.00002 0.00000 0.00000 0.00000</code></pre>
<p>as n increases, <span class="math inline">\(p\)</span> decreases. When n = 2, <span class="math inline">\(p\)</span> is already less than 0.05.</p>
<div id="what-difference-would-it-make-to-your-assay-if-you-doubled-the-concentrations-of-abts-god-and-pod" class="section level4">
<h4><span class="header-section-number">4.2.3.1</span> What difference would it make to your assay if you doubled the concentrations of ABTS, GOD and POD?</h4>
<p>Time required for incubation will be shorter and the final intensity of colors (and hence absorption) will be the same.</p>
</div>
</div>
</div>
</div>
<div id="class-results" class="section level1">
<h1><span class="header-section-number">5</span> Class Results</h1>
<div id="dataremoved" class="section level2">
<h2><span class="header-section-number">5.1</span> Data<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></h2>
<pre class="r"><code>ATP &lt;- c(3.93, 4.04, 3.01, 3.94, 3.20, 3.15, 2.81,       
         4.07, 3.17, 3.05, 4.20, 3.15, 4.17, 3.05,       
         3.90, 2.97, 3.11, 4.13, 3.27, 3.85, 2.92)
ATP_sd &lt;- c(0.164, 0.082, 0.080, 0.080, 0.140, 0.126, 0.139,       
            0.071, 0.140, 0.142, 0.002, 0.012, 0.240, 0.033,       
            0.051, 0.091, 0.187, 0.160, 0.053, 5.e-4, 0.218)
glucose &lt;- c(6.89, 6.92, 8.36, 7.38, 7.10, 7.65, 7.70,         
             9.09, 7.34, 7.60, 8.92, 8.80, 8.50, 6.60,       
             3.81, 9.80, 7.73, 9.80, 7.67, 8.50, 8.63)
glucose_sd &lt;- c(0.0880, 0.0104, 0.1560, 0.0150, 0.0603, 0.5900, 1.4580,        
                0.0470, 0.1640, 0.2810, 0.2500, 0.2000, 0.0010, 0.1180,       
                0.0170, 0.0425, 0.1500, 0.0530, 0.0440, 0.1400, 0.4800)
class_results &lt;- tibble(ATP, ATP_sd, glucose, glucose_sd)
kable(tibble(`ATP conc`=ATP, `ATP sd`=ATP_sd, 
             `Glucose conc`=glucose, `Glucose sd`=glucose_sd))</code></pre>
<table>
<thead>
<tr class="header">
<th align="right">ATP conc</th>
<th align="right">ATP sd</th>
<th align="right">Glucose conc</th>
<th align="right">Glucose sd</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">3.93</td>
<td align="right">0.1640</td>
<td align="right">6.89</td>
<td align="right">0.0880</td>
</tr>
<tr class="even">
<td align="right">4.04</td>
<td align="right">0.0820</td>
<td align="right">6.92</td>
<td align="right">0.0104</td>
</tr>
<tr class="odd">
<td align="right">3.01</td>
<td align="right">0.0800</td>
<td align="right">8.36</td>
<td align="right">0.1560</td>
</tr>
<tr class="even">
<td align="right">3.94</td>
<td align="right">0.0800</td>
<td align="right">7.38</td>
<td align="right">0.0150</td>
</tr>
<tr class="odd">
<td align="right">3.20</td>
<td align="right">0.1400</td>
<td align="right">7.10</td>
<td align="right">0.0603</td>
</tr>
<tr class="even">
<td align="right">3.15</td>
<td align="right">0.1260</td>
<td align="right">7.65</td>
<td align="right">0.5900</td>
</tr>
<tr class="odd">
<td align="right">2.81</td>
<td align="right">0.1390</td>
<td align="right">7.70</td>
<td align="right">1.4580</td>
</tr>
<tr class="even">
<td align="right">4.07</td>
<td align="right">0.0710</td>
<td align="right">9.09</td>
<td align="right">0.0470</td>
</tr>
<tr class="odd">
<td align="right">3.17</td>
<td align="right">0.1400</td>
<td align="right">7.34</td>
<td align="right">0.1640</td>
</tr>
<tr class="even">
<td align="right">3.05</td>
<td align="right">0.1420</td>
<td align="right">7.60</td>
<td align="right">0.2810</td>
</tr>
<tr class="odd">
<td align="right">4.20</td>
<td align="right">0.0020</td>
<td align="right">8.92</td>
<td align="right">0.2500</td>
</tr>
<tr class="even">
<td align="right">3.15</td>
<td align="right">0.0120</td>
<td align="right">8.80</td>
<td align="right">0.2000</td>
</tr>
<tr class="odd">
<td align="right">4.17</td>
<td align="right">0.2400</td>
<td align="right">8.50</td>
<td align="right">0.0010</td>
</tr>
<tr class="even">
<td align="right">3.05</td>
<td align="right">0.0330</td>
<td align="right">6.60</td>
<td align="right">0.1180</td>
</tr>
<tr class="odd">
<td align="right">3.90</td>
<td align="right">0.0510</td>
<td align="right">3.81</td>
<td align="right">0.0170</td>
</tr>
<tr class="even">
<td align="right">2.97</td>
<td align="right">0.0910</td>
<td align="right">9.80</td>
<td align="right">0.0425</td>
</tr>
<tr class="odd">
<td align="right">3.11</td>
<td align="right">0.1870</td>
<td align="right">7.73</td>
<td align="right">0.1500</td>
</tr>
<tr class="even">
<td align="right">4.13</td>
<td align="right">0.1600</td>
<td align="right">9.80</td>
<td align="right">0.0530</td>
</tr>
<tr class="odd">
<td align="right">3.27</td>
<td align="right">0.0530</td>
<td align="right">7.67</td>
<td align="right">0.0440</td>
</tr>
<tr class="even">
<td align="right">3.85</td>
<td align="right">0.0005</td>
<td align="right">8.50</td>
<td align="right">0.1400</td>
</tr>
<tr class="odd">
<td align="right">2.92</td>
<td align="right">0.2180</td>
<td align="right">8.63</td>
<td align="right">0.4800</td>
</tr>
</tbody>
</table>
</div>
<div id="preliminary-graphical-analysis" class="section level2">
<h2><span class="header-section-number">5.2</span> Preliminary graphical analysis</h2>
<p>Density plots and histograms are used to examine the distribution of measurements:</p>
<pre class="r"><code>op &lt;- par(mfrow=c(2,2))
for(i in c(&#39;ATP&#39;, &#39;glucose&#39;)){
  main = paste0(&#39;[&#39;, i, &#39;] in 21 samples:&#39;); xlab &lt;- paste(i, &#39;concentration&#39;)
  {
    plot(density(class_results[[i]], bw = c(ATP = 0.1, glucose = 0.2)[i]), 
         main = paste(main, &#39;density plot&#39;), xlab = xlab) 
    rug(class_results[[i]])
  }
  {
    hist(class_results[[i]], breaks = 10, main = paste(main, &#39;histogram&#39;), xlab = xlab)
  }
}</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-12-1.png" width="100%" /></p>
<p>Clealy, <a href="#our-atp">ATP</a> has a clear bimodal distribution, with two peaks at around 3.1 and 4.0.</p>
<p>The distibution of <a href="#class-glucose">glucose</a> is affected by two outliers (the maximum and minimum). After their removal, a bimodal distribution, which peaks at 7.6 and 8.8, is also produced:</p>
<pre class="r"><code>op &lt;- par(mfrow=c(1,2))
glucose &lt;- glucose[glucose != max(glucose) &amp; glucose != min(glucose)]
main = &#39;[glucose] corrected&#39;; xlab &lt;- &#39;glucose concentration&#39;
{
  plot(density(glucose, bw = c(ATP = 0.1, glucose = 0.2)[i]), 
       main = paste(main, &#39;density plot&#39;), xlab = xlab) 
  rug(glucose)
}
{
  hist(glucose, breaks = 10, main = paste(main, &#39;histogram&#39;), xlab = xlab)
}</code></pre>
<p><img src="../lab/2019-10-16-specific-radioactivity_files/figure-html/unnamed-chunk-14-1.png" width="100%" height="60%" /></p>
</div>
<div id="class-atp" class="section level2">
<h2><span class="header-section-number">5.3</span> ATP concentration</h2>
<div id="grouping-and-grouped-confidence-interval-of-mean" class="section level3">
<h3><span class="header-section-number">5.3.1</span> Grouping and grouped confidence interval of mean</h3>
<p>According to the plots, evidently there are two ATP concentrations, so we need to divide the 21 measurements into two groups, and calculate their mean and CI separately.</p>
<pre class="r"><code>ATP1 &lt;- ATP[ATP &lt; 3.5] # less than 3.5mM, around 3.0mM
ATP2 &lt;- ATP[ATP &gt; 3.5] # more than 3.5mM, around 4.0mM</code></pre>
<p>For the <a href="#our-atp">ATP</a> around 3.0:</p>
<pre class="r"><code>x &lt;- ATP1; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 3.07166667 0.12995337 0.03751431 2.20098516 0.08256843</code></pre>
<p>95% confidence interval: <span class="math inline">\(\text{[ATP]}=3.07\pm0.08\)</span> mM</p>
<p>For the <a href="#our-atp">ATP</a> around 4.0:</p>
<pre class="r"><code>x &lt;- ATP2; n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))</code></pre>
<pre><code>##       mean         sd         SE          t       t*SE 
## 4.02555556 0.12620530 0.04206843 2.30600414 0.09700998</code></pre>
<p>95% confidence interval: <span class="math inline">\(\text{[ATP]}=4.03\pm0.10\)</span> mM</p>
</div>
<div id="test-for-significance-of-the-difference-in-concentration-between-the-two-groups" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Test for significance of the difference in concentration between the two groups</h3>
<p>To test there is a significant difference between the two concentrations, t-test<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> is used:</p>
<pre class="r"><code>t.test(ATP1, ATP2)</code></pre>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  ATP1 and ATP2
## t = -16.923, df = 17.66, p-value = 2.346e-12
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -1.0724720 -0.8353058
## sample estimates:
## mean of x mean of y 
##  3.071667  4.025556</code></pre>
<p><span class="math inline">\(p=2.3\times10^{-12}&lt;&lt;0.05\)</span>; the two concentrations are significantly different</p>
</div>
<div id="test-for-diviation-from-given-concentrations" class="section level3">
<h3><span class="header-section-number">5.3.3</span> Test for diviation from given concentrations</h3>
<p>The solutions were actually made up to be 3.0mM and 4.0mM ATP. Were they made up correctly? We need two one-sample t tests using the given concentrations as the population mean, <span class="math inline">\(\mu\)</span>:</p>
<p>3.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 3.0mM)</p>
<pre class="r"><code>t.test(ATP1, mu=3.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  ATP1
## t = 1.9104, df = 11, p-value = 0.08249
## alternative hypothesis: true mean is not equal to 3
## 95 percent confidence interval:
##  2.989098 3.154235
## sample estimates:
## mean of x 
##  3.071667</code></pre>
<p><span class="math inline">\(p=0.082&gt;0.05\)</span>, failed to reject hypothesis. Solution made up correctly.</p>
<p>4.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 4.0mM)</p>
<pre class="r"><code>t.test(ATP2, mu=4.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  ATP2
## t = 0.60748, df = 8, p-value = 0.5604
## alternative hypothesis: true mean is not equal to 4
## 95 percent confidence interval:
##  3.928546 4.122566
## sample estimates:
## mean of x 
##  4.025556</code></pre>
<p><span class="math inline">\(p=0.56&gt;0.05\)</span>, failed to reject hypothesis. Solution made up correctly.</p>
</div>
</div>
<div id="class-glucose" class="section level2">
<h2><span class="header-section-number">5.4</span> Glucose</h2>
<div id="grouping-and-grouped-confidence-interval-of-mean-1" class="section level3">
<h3><span class="header-section-number">5.4.1</span> Grouping and grouped confidence interval of mean</h3>
<p>By observing the plots, the boundary between the two concentrations of glucose is 8.0mM, so we can divide them into two groups:</p>
<pre class="r"><code>glucose1 &lt;- glucose[glucose &lt; 8.0]
glucose2 &lt;- glucose[glucose &gt; 8.0]</code></pre>
<p>And calculate their 95% CIs separately:</p>
<pre class="r"><code>lapply(list(glucose1=glucose1, glucose2=glucose2), function(x){
  n &lt;- length(x); df &lt;- n - 1; t = qt(0.975, df)
  c(mean=mean(x), sd=sd(x), SE=sd(x)/sqrt(n), t = t, `t*SE` = t*sd(x)/sqrt(n))
})</code></pre>
<pre><code>## $glucose1
##      mean        sd        SE         t      t*SE 
## 7.3254545 0.3921317 0.1182322 2.2281389 0.2634377 
## 
## $glucose2
##      mean        sd        SE         t      t*SE 
## 8.6857143 0.2612698 0.0987507 2.4469119 0.2416343</code></pre>
<p>The confidence intervals are: <span class="math inline">\(7.33\pm0.27\)</span> mM and <span class="math inline">\(8.69\pm0.22\)</span> mM</p>
</div>
<div id="test-for-significance-of-the-difference-in-concentration-between-the-two-groups-1" class="section level3">
<h3><span class="header-section-number">5.4.2</span> Test for significance of the difference in concentration between the two groups</h3>
<p>t-test performed to test for significance:</p>
<pre class="r"><code>t.test(glucose1, glucose2)  </code></pre>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  glucose1 and glucose2
## t = -8.8301, df = 15.912, p-value = 1.573e-07
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -1.686972 -1.033548
## sample estimates:
## mean of x mean of y 
##  7.325455  8.685714</code></pre>
<p><span class="math inline">\(p=1.57\times 10^{-7}&lt;&lt;0.05\)</span>, which shows the two concentrations are significantly different.</p>
</div>
<div id="test-for-diviation-from-given-concentrations-1" class="section level3">
<h3><span class="header-section-number">5.4.3</span> Test for diviation from given concentrations</h3>
<p>The solutions were actually made up to be 7.0mM and 9.0mM ATP. Were they made up correctly? We need two one-sample t tests using the given concentrations as the population mean, <span class="math inline">\(\mu\)</span>:</p>
<p>7.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 7.0mM)</p>
<pre class="r"><code>t.test(glucose1, mu=7.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  glucose1
## t = 2.7527, df = 10, p-value = 0.02038
## alternative hypothesis: true mean is not equal to 7
## 95 percent confidence interval:
##  7.062017 7.588892
## sample estimates:
## mean of x 
##  7.325455</code></pre>
<p><span class="math inline">\(p=0.020&lt;0.05\)</span>, reject null hypothesis. Solution made up incorrectly.</p>
<p>9.0mM: (<span class="math inline">\(H_0\)</span>: true concentration is 9.0mM)</p>
<pre class="r"><code>t.test(glucose2, mu=9.0)</code></pre>
<pre><code>## 
##  One Sample t-test
## 
## data:  glucose2
## t = -3.1826, df = 6, p-value = 0.01901
## alternative hypothesis: true mean is not equal to 9
## 95 percent confidence interval:
##  8.444080 8.927349
## sample estimates:
## mean of x 
##  8.685714</code></pre>
<p><span class="math inline">\(p=0.019&lt;0.05\)</span>, reject null hypothesis. Solution made up incorrectly.</p>
</div>
</div>
<div id="radioactivity" class="section level2">
<h2><span class="header-section-number">5.5</span> Radioactivity</h2>
<div id="results-of-the-sample" class="section level3">
<h3><span class="header-section-number">5.5.1</span> Results of the sample</h3>
<p>Counts in the H channel and C channel are shown below:</p>
<pre class="r"><code>H &lt;- c(14178.5, 13963, 14065, 14356.5, 13809, 14157.5, 13824.5, 14032, 14094.5, 14014)
C &lt;- c(3411, 3472.5, 3386.5, 3516, 3375.5, 3418.5, 3438, 3464.5, 3513.5, 3438.5)</code></pre>
<p>Statistical summaries are calculated:</p>
<pre class="r"><code>n &lt;- 10; df &lt;- n-1; t = qt(0.975, df)
lapply(list(`H Channel`=H, `C Channel`=C), function(i){
  c(mean=mean(i), sd=sd(i), SE=sd(i)/sqrt(n), t = t, `t*SE` = t*sd(i)/sqrt(n))
})</code></pre>
<pre><code>## $`H Channel`
##         mean           sd           SE            t         t*SE 
## 14049.450000   164.091176    51.890186     2.262157   117.383756 
## 
## $`C Channel`
##        mean          sd          SE           t        t*SE 
## 3443.450000   48.359447   15.292600    2.262157   34.594264</code></pre>
<p>95% confidence intervals:</p>
<ul>
<li>H channel: <span class="math inline">\(14049.5\pm 117.38\)</span> cpm</li>
<li>C channel: <span class="math inline">\(3443.5\pm 34.59\)</span> cpm</li>
</ul>
</div>
<div id="correction-for-overlap" class="section level3">
<h3><span class="header-section-number">5.5.2</span> Correction for overlap</h3>
<p>Let counts due to <sup>3</sup>H = <span class="math inline">\(X\)</span> and counts due to <sup>14</sup>C = <span class="math inline">\(Y\)</span>. Let <span class="math inline">\(\alpha\)</span> be the fraction in H channel of all counts due to <sup>3</sup>H and <span class="math inline">\(\beta\)</span> be the fraction in C channel of all counts due to <sup>14</sup>C, so that for a sample with mixed isotopes:</p>
<p><span class="math display">\[H = \alpha X + (1-\beta)Y\]</span>
<span class="math display">\[C = (1-\alpha)X+\beta Y\]</span></p>
<p><span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> can be determined by counting pure <sup>3</sup>H and <sup>14</sup>C, respectively. Here are the results:</p>
<pre class="r"><code>c(H_in_H = mean(c(464814.5, 462463.5, 464775.5)),
  H_in_C = mean(c(14825.5, 15057.5, 14210.0)),
  C_in_H = mean(c(10295, 9842.5, 9922.5)),
  C_in_C = mean(c(47989, 48868.5, 49133.5)))</code></pre>
<pre><code>##    H_in_H    H_in_C    C_in_H    C_in_C 
## 464017.83  14697.67  10020.00  48663.67</code></pre>
<p>which gives:</p>
<p><span class="math display">\[
\begin{aligned}
 \alpha &amp;= 464017.83/(464017.83+14697.67)=0.9693 \\
 \beta &amp;= 48663.67/(48663.67+10020.00)=0.8293
\end{aligned}
\]</span></p>
<p>Plugging in sample results:</p>
<p><span class="math display">\[
\begin{aligned}
  H &amp;= \alpha X + (1-\beta)Y = 0.9693X+(1-0.8293)Y=14049.5(\pm117.38) \\
  C &amp;= (1-\alpha)X+\beta Y = (1-0.9693)X+0.8293Y = 3443.5(\pm34.59)
\end{aligned}
\]</span></p>
<p>Simplifying…</p>
<p><span class="math display">\[
\begin{aligned}
  0.9693X+0.1707Y&amp;=14049.5(\pm117.38) \\
  0.0307X+0.8293Y &amp;= 3443.5(\pm34.59)
\end{aligned}
\]</span></p>
<p>Solving for X: (note that uncertainties are added)</p>
<p><span class="math display">\[
\begin{aligned}
  4.7075X+0.8293Y &amp;=60721.7(\pm507.31) \\
  0.0307X+0.8293Y &amp;= 3443.5(\pm34.59) \\
  4.6769X &amp;= 57278.2(\pm541.9) \\
X &amp;= 12247.2(\pm115.9)
\end{aligned}
\]</span></p>
<p>Solving for Y:</p>
<p><span class="math display">\[
\begin{aligned}
  0.9693X+0.1707Y&amp;=14049.5(\pm117.38) \\
  0.9693X+26.18Y &amp;= 108722.6(\pm1058.66) \\
  26.01Y &amp;= 94673.1(\pm1176.04) \\
  Y &amp;= 3639.4(\pm45.2)
\end{aligned}
\]</span></p>
<p>which gives:</p>
<p><span class="math display">\[
\begin{aligned}
  \text{actual H (ATP) counts in 0.2mL sample}&amp;=12247.2(\pm115.9) \\
  \text{actual C (glucose) counts in 0.2mL sample} &amp;= 3639.4(\pm45.2)
\end{aligned}
\]</span></p>
</div>
<div id="specific-radioactivity" class="section level3">
<h3><span class="header-section-number">5.5.3</span> Specific Radioactivity</h3>
<p>The radioactivity counting were done on 0.2mL sample. Our ATP concentration (section <a href="#our-atp">4.1</a>) is <span class="math inline">\(\text{[ATP]}=3.268\pm0.054\)</span> mM and glucose concentration (section <a href="#our-glucose-sample">4.2.2</a>) is <span class="math inline">\(\text{[glucose]}=7.667\pm0.057\)</span> mM. Thus, the specific radioactivity of ATP and glucose are:</p>
<p><span class="math display">\[
\begin{aligned}
  \text{ATP: }12247.2(\pm115.9)/[0.2\times10^{-3}\times3.268\times 10^{-3}(\pm0.054\times 10^{-3})]&amp;= 1.87(\pm0.05)\times 10^{10} \text{ cpm mol}^{-1}\\
  \text{glucose: } 3639.4(\pm45.2)/[0.2\times10^{-3}\times7.667\times 10^{-3}(\pm0.057\times 10^{-3})]&amp;= 2.37(\pm0.05) \times 10^{9} \text{ cpm mol}^{-1}
\end{aligned}
\]</span></p>
<p>(the <span class="math inline">\(\pm0.05\)</span> comes from the maximal possible numerator (e.g. <span class="math inline">\(12247.2 + 115.9\)</span> for ATP) over the least possible denominator (e.g. <span class="math inline">\((3.268-0.054)\times10^{-3}\)</span> for ATP))</p>
</div>
</div>
</div>
<div id="appendix" class="section level1">
<h1><span class="header-section-number">6</span> Appendix</h1>
<div id="temp" class="section level2">
<h2><span class="header-section-number">6.1</span> People should report all measurements to get a true 95% confidence interval</h2>
<p>The statistical computations perfomed in section <a href="#class-atp">5.3</a> and section <a href="#class-glucose">5.4</a> for class results are actually improper.</p>
<p>Let’s say 3 people are measuring a concentration whose true value is 4.0. Here are their results:</p>
<pre class="r"><code>c1 &lt;- rnorm(5, 4.0, 3) # 5 times, sd about 3
c2 &lt;- rnorm(6, 4.0, 2) # 6 times, sd about 2
c3 &lt;- rnorm(7, 4.0, 1) # 7 times, sd about 1</code></pre>
<p>To calculate the true mean and sd, we need to combine their results i.e. to get list of 15 independent measurements:</p>
<p>True mean:</p>
<pre class="r"><code>mean(c(c1,c2,c3))</code></pre>
<pre><code>## [1] 4.509869</code></pre>
<p>but if we take the mean of the mean (which in this practical we are supposed to do):</p>
<pre class="r"><code>mean(c(mean(c1),
       mean(c2),
       mean(c3)))</code></pre>
<pre><code>## [1] 4.564576</code></pre>
<p>This gives a different mean from the true mean!</p>
<p>similarly, the true sd calculation is not possible if only the sd of each person’s results are given:</p>
<p>true sd:</p>
<pre class="r"><code>sd(c(c1, c2, c3))</code></pre>
<pre><code>## [1] 1.948202</code></pre>
<p>we might be supposed to calculate:</p>
<pre class="r"><code>mean(c(sd(c1), sd(c2), sd(c3)))</code></pre>
<pre><code>## [1] 1.742614</code></pre>
<p>(see: <a href="https://bit.ly/2Wj505F" class="uri">https://bit.ly/2Wj505F</a> )</p>
<p>What I did was treating each person’s mean as one observation:</p>
<pre class="r"><code>c(
  mean = mean(c(mean(c1),
       mean(c2),
       mean(c3))),
  sd = sd(c(mean(c1),
     mean(c2),
     mean(c3)))
)</code></pre>
<pre><code>##      mean        sd 
## 4.5645764 0.8848836</code></pre>
</div>
<div id="the-mathematical-computation-behind-r-functions-used-in-this-report" class="section level2">
<h2><span class="header-section-number">6.2</span> The mathematical computation behind R functions used in this report</h2>
<p>Statistical computations are done automatically by R. For completeness, I include here the actual mathematical operations performed by the mathematical functions used in this report.</p>
<p>For an R vector <code>x &lt;- c(x1, x2, x3, x4, x5, ... xn)</code>:</p>
<p><code>mean(x)</code> = <span class="math inline">\(\bar{x}=\dfrac{\sum{x}}{n}\)</span>
<code>sd(x)</code> = <span class="math inline">\(s_x=\sqrt{\dfrac{\sum{(x-\bar{x})^2}}{n-1}}=\sqrt{\dfrac{\sum{x^2}-\dfrac{(\sum x)^2}{n}}{n-1}}\)</span></p>
<p><code>lm(y ~ x)</code> produces a linear model <span class="math inline">\(y = b_0 + b_1x\)</span></p>
<p><span class="math display">\[b_1=\dfrac{s_{xy}}{s_x}=\dfrac{\sum{(x-\bar{x})}\sum{(y-\bar{y})}}{\sum{(x-\bar{x})^2}} = \dfrac{n\sum{xy}-\sum x \sum y}{n\sum{x^2}-(\sum x)^2}\]</span></p>
<p>and</p>
<p><span class="math display">\[b_0 = \bar{y} - b_1\bar{x} = \dfrac{\sum y - b_1 \sum x}{n}\]</span></p>
<p>However, I forced intercept to be at the origin (<code>lm(y ~ 0 + x)</code>), so actually <span class="math inline">\(b_1\)</span> is calculated as:</p>
<p><span class="math display">\[b_1=\dfrac{\sum xy}{\sum x^2}\]</span></p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>2 pairs of students produced incomplete results, so only 21/23 sets of results are given.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Specifically, Welch’s two sample t-test with unequal variances<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../worksheet/radioactivity-worksheet.html" class="archive-item-link">
                Radioactivity Worksheet 
              </a>
              <span class="archive-item-date">
                2019-10-16
              </span>
              <div class="article-summary" style="display: none;">
                
                



<div id="guinea-pig" class="section level1">
<h1>Guinea pig</h1>
<p><strong>A guinea pig was given a single injection of <sup>24</sup>NaCl (0.5ml of 0.5<span class="math inline">\(\mu\)</span>Ci/ml). Subsequently, 0.2 ml samples of blood were taken at intervals and counted for radioactivity. The following results were obtained. The efficiency of counting <sup>24</sup>Na is 30%.</strong></p>

<p><strong>(a) Given that the radioactive half-life of <sup>24</sup>Na is 15h, calculate the biological half life of Na<sup>+</sup> in the guinea pig body. Why are these values different?</strong></p>
<p>Radioactive decay is a first-order process, so the rate of decay is proportional to the amount of the decaying substance. This can be expressed with a differential equation: <span class="math inline">\(-\frac{dA}{dt}=kA\)</span>, which can be integrated to give:</p>
<p><span class="math display">\[\ln{A} = \ln{A_0}-kt\]</span></p>
<p>where <span class="math inline">\(A_0\)</span> is the initial radioactivity, <span class="math inline">\(t\)</span> is the time elapsed, <span class="math inline">\(k\)</span> is the decay constant, and <span class="math inline">\(A\)</span> is the radioactivity at time <span class="math inline">\(t\)</span>. A linear relationship exists between <span class="math inline">\(\ln{A}\)</span> and <span class="math inline">\(t\)</span>:</p>
<pre class="r"><code>t &lt;- c(1, 3, 5, 10, 16, 24) # in hours
A &lt;- c(3604, 2928, 2376, 1412, 756, 329)/0.3 # corrected for efficiency
mod &lt;- lm(log(A) ~ t)
summary(mod)</code></pre>
<pre><code>## 
## Call:
## lm(formula = log(A) ~ t)
## 
## Residuals:
##          1          2          3          4          5          6 
##  2.578e-05  4.796e-04 -2.429e-04 -2.077e-04 -3.928e-04  3.380e-04 
## 
## Coefficients:
##               Estimate Std. Error t value Pr(&gt;|t|)    
## (Intercept)  9.498e+00  2.503e-04   37948  &lt; 2e-16 ***
## t           -1.041e-01  1.972e-05   -5280 7.72e-15 ***
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1
## 
## Residual standard error: 0.0003878 on 4 degrees of freedom
## Multiple R-squared:      1,  Adjusted R-squared:      1 
## F-statistic: 2.787e+07 on 1 and 4 DF,  p-value: 7.722e-15</code></pre>
<pre class="r"><code>plot(t, log(A), xlab = &#39;t (h)&#39;)
abline(coef(mod))</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-4-1.png" width="100%" /></p>
<p>from linear regression, <span class="math inline">\(A_0=e^{9.498}=13333 \text{ cpm}\)</span>, and <span class="math inline">\(k=0.104 \text{ h}^{-1}\)</span></p>
<p>half life (total): <span class="math inline">\(t_{\frac{1}{2}}=\dfrac{\ln{2}}{k}=6.66 \text{ h}\)</span></p>
<p>Na<sup>+</sup> in biological systems is continuously being excreted, and excretion is generally also a first-order process. So the result is that the overall loss of radioactivity is still a first-order process with a fixed, but shorter half life (i.e. larger <span class="math inline">\(k\)</span>).</p>
<p><span class="math display">\[k_{\text{total}}=k_{\text{radioactivity}}+k_{\text{biological}}\]</span></p>
<p><span class="math display">\[\dfrac{\ln2}{6.66}=\dfrac{\ln{2}}{15}+\dfrac{\ln{2}}{t_{\frac{1}{2}{\text{bioogical}}}}\]</span></p>
<p><span class="math display">\[t_{\frac{1}{2}{\text{bioogical}}}=12.0\]</span></p>
<p><strong>(b) Assuming that Na<sup>+</sup> is essentially excluded from cell water (in the short term), calculate the volume of extracellular fluid in the guinea pig.</strong></p>
<p>from (a), <span class="math inline">\(A_0=e^{9.498}=13333.3 \text{ cpm}\)</span> (in 0.2ml sample), which gives a radioactivity per volume (of blood sample at t = 0): <span class="math inline">\(13333\div0.2=66666\text{ cpm ml}^{-1}\)</span></p>
<p>The total radioactivity is <span class="math inline">\(0.5\text{ ml}\times0.5\times10^{-6}\text{ Ci}\times2.2\times10^{12} = 550000\text{ cpm}\)</span></p>
<p>Volume: <span class="math inline">\(550000/66666=8.25\text{ ml}\)</span></p>
</div>
<div id="sulphate-transport-in-penicillium-chrysogenum" class="section level1">
<h1>Sulphate transport in <em>Penicillium chrysogenum</em></h1>
<p><strong>An experiment was carried out to investigate sulphate transport in the fungus Penicillium chrysogenum. From a culture of an ATP sulphurylase deficient mutant, 1g (wet weight of mycelium was taken and suspended in 100 ml buffer. At t=0, 1ml of 1mM K<sub>2</sub><sup>35</sup>SO<sub>4</sub> was added to the stirred suspension.</strong></p>
<p><strong>(a) 5ml aliquots were taken at 30s intervals, and the mycelium filtered, washed and counted for radioactivity. The following results were obtained.</strong></p>
<pre><code>## $`Cpm on filter after:`
##   30s   60s   90s  120s 
## 10600 21000 31200 41900</code></pre>
<p>To establish the specific radioactivity of the added sulphate, 10<span class="math inline">\(\mu\)</span>l of the stock 1mM K<sub>2</sub><sup>35</sup>SO<sub>4</sub> was also counted, yielding a value of 12,000 cpm.</p>
<p>Calculate the sulphate transport rate in terms of <span class="math inline">\(\mu\)</span>mol/min/g dry wt, assuming that the mycelium contained 15% (by weight) dry matter.</p>
<p>The measurement are made on 5mL aliquots. To calculate the cpm in the 100ml reaction mixture , multiply cpm values by 20. Then do a linear regression between cpm (of all fungus cells) and time elapsed.</p>
<pre class="r"><code>t &lt;- c(0.5, 1.0, 1.5, 2.0) # in minutes
A &lt;- 20 * c(10600, 21000, 31200, 41900)
plot(t, A, xlab = &#39;t (min)&#39;, ylab = &#39;A (cpm)&#39;); abline(lm(A~t))</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-6-1.png" width="100%" /></p>
<pre class="r"><code>coef(lm(A~t))</code></pre>
<pre><code>## (Intercept)           t 
##        3000      416400</code></pre>
<p>the coefficients gives <span class="math inline">\(A=3000 + 416400t\)</span> where <span class="math inline">\(t\)</span> (time elapsed) is in minutes and <span class="math inline">\(A\)</span> in cpm. The slope is the change in cpm per minute (<span class="math inline">\(\dfrac{dA}{dt}=416400\text{ cpm min}^{-1}\)</span>)</p>
<p>From the data of pure K<sub>2</sub>SO~4, the specific radioactivity of K<sub>2</sub>SO~4 is</p>
<p><span class="math inline">\(12000\text{ cpm}\div(10\times10^{-6}\text{ L}\times1\times10^{-3}\text{ M}=1.2\times10^{12}\text{ cpm mol}^{-1})\)</span></p>
<p><span class="math display">\[416400\text{ cpm min}^{-1}/(1.2\times10^{12}\text{ cpm mol}^{-1})\times10^6=0.347\text{ }\mu\text{mol min}^{-1}\]</span></p>
<p>1g mycelium is equivalent to 0.15g dry weight. Transport rate:</p>
<p><span class="math display">\[0.347/0.15=2.31\text{ }\mu\text{mol/min/g dry wt}\]</span></p>
<p><strong>(b) Final samples were taken after 3h and 4h. At these times, the mycelium was too ‘hot’ to count accurately, but the medium contained 220 cpm/ml at both times. Estimate the equilibrium constant for the transport of sulphate in this organism, and comment on this value in terms of the free energy of transport.</strong></p>
<p>Amount of sulfate in the medium at equilibrium: <span class="math inline">\(220\text{ cpm ml}^{-1}\times100\text{ ml}\div(1.2\times10^{12}\text{ cpm mol}^{-1})=1.833\times10^{-8}\text{ mol}\)</span></p>
<p>Total amount of sulfate: <span class="math inline">\(1\times10^{-3}\text{ L}\times1\times10^{-3}\text{ M}=1\times10^{-6}\text{ mol}\)</span></p>
<p>Amount of sulfate in fungi cells at equilibrium: <span class="math inline">\(1\times10^{-6}-1.833\times10^{-8}=9.817\times10^{-7}\)</span></p>
<p>For the reaction X<sub>out</sub><span class="math inline">\(\rightarrow\)</span>X<sub>in</sub>:</p>
<p><span class="math display">\[K=\dfrac{9.817\times10^{-7}}{1.833\times10^{-8}}=53.5\]</span></p>
<p><span class="math display">\[G^\circ=-RT\ln{K}=-9.8\text{ kJ mol}^{-1}\]</span></p>
<p>Sulfate transport into fungi cells is thermodynamically favourable, but the free energy change is not very large.</p>
<p><strong>(c) How would these observations differ if a wild type organism, containing ATP sulphurylase, was used in these experiments?</strong></p>
<p>Once ATP is sulfated, the sulfate groups are no longer free and will not affect the equilibrium between free sulfate ions on both sides. As ATP sulphurylase constantly consumes sulfate inside the cells, the equilibrium is never achieved and <span class="math inline">\(G=G^\circ+RT\ln Q\)</span> will always be negative. Finally, almost all sulfate will enter the cells (and being added to ATP) and this is limited by the equilibrium between unsulfated and sulfated ATP.</p>
</div>
<div id="rnadna-ratio" class="section level1">
<h1>RNA:DNA ratio</h1>
<p><strong>Mutant bacteria can be exploited to provide insight into the ratio of RNA to DNA within cells, if they rely on the uptake of radioactive nutrients. In one such study, a mutant E. coli with a mutation in the enzyme that synthesizes orotic acid and consequently requires uridine for growth was used. This mutant was grown in minimal medium supplemented with <sup>14</sup>C uridine at a specific activity of 30 mCi/mmol. Ribosomal RNA was isolated from the culture.</strong></p>
<p>The base composition of this RNA in mmol/200g was : Uracil = 128.4; Guanine = 189.0; Cytosine =132.4, Adenine = 163.4.</p>
<p><strong>(a) Calculate the specific activity of the RNA in mCi/mg (Hint: think about pyrimidine synthesis when working out the specific activity of the RNA)</strong></p>
<p>Purine synthesis and pyrimidine synthesis use different pathways. In this mutant <em>E. coli</em> uncapable of orotic acid synthesis, uracil and cytosine in RNA are derived from the radioactive <sup>14</sup>C. Both uracil and cytosine molecules have 4 carbon atoms, so both have a specific activity of 30 mCi/mmol.</p>
<p><span class="math display">\[(128.4+132.4)\div(200\times 1000)=1.304\times10^{-3}\text{ mmol pyrimidine/mg RNA}\]</span></p>
<p><span class="math display">\[1.304\times10^{-3}\times30=0.0391\text{ mCi/mg RNA}\]</span></p>
<p><strong>To determine the proportion of DNA complementary to the ribosomal DNA, Membrane filters were charged with 10g denatured <em>E. coli</em> DNA, and incubated with different concentrations of ribosomal RNA labelled as in part (a). A blank filter was also included in each incubation. After thorough washing and treatment with RNAase to remove unhybridised RNA, the radioactivity bound to the filters was determined by liquid scintillation counting. The results are shown in Table 1. 1 mCi = 2.2 x 10<sup>9</sup> dpm. The efficiency of counting is 80%.</strong></p>

<p><strong>(b) Determine the proportion of DNA complementary to the ribosomal RNA</strong></p>
<p>According to the table, when binding is saturated (all 10g DNA is bound complementary RNA), the cpm is around (2828-130) or (2830-140).</p>
<p><span class="math display">\[[(2928-130)+(2830-140)]\div2\div0.8=3367.5\text{ cpm}\]</span></p>
<p><span class="math display">\[3367.5\div (2.2\times10^9)=1.531\text{ mCi}\]</span></p>
<p><span class="math display">\[1.531\text{ mCi}\div0.03912\text{ mCi/mg RNA}=3.91\times10^{-5}\text{ mg} = 3.91\times10^{-8}\text{ g RNA}\]</span></p>
<p><span class="math display">\[3.91\times10^{-8}\text{ g RNA}/10\text{ g DNA}=3.91\times10^{-9}\]</span></p>
<p><strong>(c) If the total chain length of the two ribosomal RNA species is 1.5 x 10<sup>6</sup> Da, and the DNA content of a bacterial cell containing a single chromosome is 2.0 x10<sup>9</sup> Da, how many copies of the genes coding for ribosomal RNA are present in the bacterial chromosome? Speculate why the answer is not one.</strong></p>
<p>Ratio of single rRNA copy to single chromosome:</p>
<p><span class="math display">\[(1.5\times 10^6)\div(2.0\times10^9)=7.5\times10^{-4}\]</span></p>
<p>The result of (b) should be greater than <span class="math inline">\(7.5\times10^{-4}\)</span>, but it was not, so the calculation would be invalid.</p>
</div>
<div id="ip3-receptor" class="section level1">
<h1>IP<sub>3</sub> Receptor</h1>
<p><strong>An analogue of myo-inositol 1,4,5 trisphosphate (IP<sub>3</sub>) in which sulphur atoms replaced one of the oxygen atoms on each phosphate (IPS<sub>3</sub>) was used to probe membrane bound IP<sub>3</sub> receptors in a membrane fraction of rat cerebellum. Cerebellar membranes (80<span class="math inline">\(\mu\)</span>g protein) were incubated in a volume of 200<span class="math inline">\(\mu\)</span>l with increasing concentrations (0.01-2.5 <span class="math inline">\(\mu\)</span>M) of <sup>35</sup>SIPS3 (7x10<sup>9</sup> dpm/<span class="math inline">\(\mu\)</span>mol). After 30 mins, bound and free ligand were separated by centrifugation, and radioactivity measured in the pellet. To allow for non-specific binding, a parallel run was carried out at each IPS<sub>3</sub> concentration with 0.5mM IP<sub>3</sub> also present. The results obtained were as follows:</strong></p>

<p><strong>(a) Draw the structure of IP<sub>3</sub>, and briefly outline its role as an intracellular messenger.</strong></p>
<p><img src="../img/ip3.jpg" width="100%" /></p>
<p>Phospholipase C (PLC) cleaves PIP<sub>2</sub> (phosphatidylinositol 4, 5-bisphosphate) into IP<sub>3</sub> and DAG (diacylglycerol). This occurs, for example, in GPCR pathway (G<sub>s<span class="math inline">\(\alpha\)</span></sub> activates PLC-<span class="math inline">\(\beta\)</span>) and in tyrosine kinase pathway (the SH2 domain of PLC-<span class="math inline">\(\gamma\)</span> docks it to phosphorylated tyrosines in RTK intracellular domain).</p>
<p>IP<sub>3</sub> binds to its receptors located on ER membrane, which triggers release of Ca<sup>2+</sup> ions. Ca<sup>2+</sup> then activates downstream signalling pathways, such as those mediated by calmodulin.</p>
<p><strong>(b) Determine the dissociation constant for the specific binding of IPS3 to the IP3 receptor.</strong></p>
<p>The ratio of IPS<sub>3</sub>-receptor complex concentration to total receptor concentration under different IPS<sub>3</sub> concentrations can be used to calculate <span class="math inline">\(K_d\)</span>. Here’s the proof:</p>
<p>For the equilibrium: <span class="math inline">\(\text{PL}\leftrightarrow\text{P + L}\)</span>, <span class="math inline">\(K_d = \dfrac{\text{[P][L]}}{\text{[PL]}}\)</span>, i.e. <span class="math inline">\(\text{[PL]}=\dfrac{\text{[P][L]}}{K_d}\)</span></p>
<p>the ratio thus simplifies to:</p>
<p><span class="math display">\[\dfrac{\text{[PL]}}{\text{[P]}_\text{total}}=\dfrac{\text{[PL]}}{\text{[PL]}+\text{[P]}} = \dfrac{\dfrac{\text{[P][L]}}{K_d}}{\dfrac{\text{[P][L]}}{K_d}+\text{[P]}}=\dfrac{\text{[L]}}{\text{[L]}+K_d}\]</span></p>
<p>Taking double reciprocal:</p>
<p><span class="math display">\[\dfrac{1}{\text{[PL]}} = \dfrac{1}{\text{[P]}_\text{total}} + \dfrac{K_d}{\text{[P]}_\text{total}\text{[L]}}\]</span></p>
<p>Plotting <span class="math inline">\(\dfrac{1}{\text{[PL]}}\)</span> against <span class="math inline">\(\dfrac{1}{\text{[L]}}\)</span>, the intercept is <span class="math inline">\(\dfrac{1}{\text{[P]}_\text{total}}\)</span> and the gradient is <span class="math inline">\(\dfrac{K_d}{\text{[P]}_\text{total}}\)</span></p>
<p>The cpm due to specific (real PL) binding is first calculated:</p>
<pre class="r"><code>all_cpm &lt;- c(2770, 5385, 10196, 18587, 31694, 48712, 65930, 83190, 106190)
non_specific &lt;- c(110, 205, 396, 807, 1594, 3212, 6430, 12770, 25690)
(specific &lt;- all_cpm - non_specific) # unit: cpm</code></pre>
<pre><code>## [1]  2660  5180  9800 17780 30100 45500 59500 70420 80500</code></pre>
<p>and the concentrations of PL are:</p>
<pre class="r"><code>(PL &lt;- specific/7e9/200) # in M</code></pre>
<pre><code>## [1] 1.90e-09 3.70e-09 7.00e-09 1.27e-08 2.15e-08 3.25e-08 4.25e-08 5.03e-08
## [9] 5.75e-08</code></pre>
<p>the corresponding concentrations of L are:</p>
<pre class="r"><code>(L &lt;- c(.01, .02, .04, .08, .16, .32, .64, 1.28, 2.56)/1e6) # in M</code></pre>
<pre><code>## [1] 1.00e-08 2.00e-08 4.00e-08 8.00e-08 1.60e-07 3.20e-07 6.40e-07 1.28e-06
## [9] 2.56e-06</code></pre>
<p>Linear modelling of <span class="math inline">\(\dfrac{1}{\text{[PL]}}\)</span> against <span class="math inline">\(\dfrac{1}{\text{[L]}}\)</span>:</p>
<pre class="r"><code>y &lt;- 1/PL; x &lt;- 1/L
(coef &lt;- coef(lm(y~x)))</code></pre>
<pre><code>##  (Intercept)            x 
## 1.516132e+07 5.109047e+00</code></pre>
<pre class="r"><code>plot(1/L, 1/PL)
abline(coef = coef)</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-15-1.png" width="100%" /></p>
<p><span class="math inline">\(\dfrac{1}{\text{[P]}_\text{total}} = 1.516\times10^7\)</span>, <span class="math inline">\(\text{[P]}_\text{total}=6.596\times10^{-8}\text{ M}\)</span></p>
<p><span class="math inline">\(\dfrac{K_d}{\text{[P]}_\text{total}}=5.109\)</span>, <span class="math inline">\(K_d=6.596\times10^{-8}\times5.109=3.370\times10^{-7} \text{ M}\)</span></p>
<p><strong>(c) Determine the density of IP3 receptors in the membrane preparation used.</strong></p>
<p>From (b), <span class="math inline">\(\text{[P]}_\text{total}=6.596\times10^{-8}\text{ M}\)</span></p>
<p>Given mass of total protein is 80<span class="math inline">\(\mu\)</span>g and the volume is 200<span class="math inline">\(\mu\)</span>l, the density is:</p>
<p><span class="math display">\[80\times10^{-6}\div(200\times10^{-6}\times 6.596\times10^{-8})=6.06\times10^6 \text{ Da}\]</span></p>
</div>
<div id="ppiase" class="section level1">
<h1>PP<sub>i</sub>ase</h1>
<p><strong>Cytosolic pyrophosphatase (PP<sub>i</sub>ase) (molecular weight 71,000 Da) was purified from yeast, and the activity of the final preparation measured by following the release of <sup>32</sup>PP<sub>i</sub> from <sup>32</sup>PP<sub>i</sub>. Each reaction tube contained 0.5ml buffer (pH 7.4) and <sup>32</sup>PP<sub>i</sub> (400,000 cpm), together with unlabelled PP<sub>i</sub> to the final concentration shown below. (The concentration of <sup>32</sup>PP<sub>i</sub> can be assumed to be negligible.) The reaction was started by the addition of 2<span class="math inline">\(\mu\)</span>l (0.2ng) enzyme and stopped after 1 min by the addition of trichloroacetic acid. <sup>32</sup>PP<sub>i</sub> was then separated from the remaining <sup>32</sup>PP<sub>i</sub> and measured by scintillation counting.</strong></p>

<p>Take the Lineweaver-Burk equation (double reciprocal of the Michaelis-Mention equation):</p>
<p><span class="math display">\[\dfrac{1}{v}=\dfrac{K_m}{V_{\text{max}}\text{[S]}}+\dfrac{1}{V_{\text{max}}}\]</span></p>
<p>In this equation, <span class="math inline">\(v\)</span> refers to the total rate of hydrolysis of both PP<sub>i</sub> and <sup>32</sup>PP<sub>i</sub>, and [S] is the total concentration of PP<sub>i</sub> and <sup>32</sup>PP<sub>i</sub>.</p>
<p>Let concentration of <sup>32</sup>PP<sub>i</sub> be <span class="math inline">\(c_0\)</span> (which is a constant), the concentration of PP<sub>i</sub> be <span class="math inline">\(c^\text{apparent}\)</span>, the ‘apparent’ rate that is due to the hydrolysis of radioactive <sup>32</sup>PP<sub>i</sub> be <span class="math inline">\(v^\text{apparent}\)</span>, then:</p>
<p><span class="math display">\[\text{[S]}=c_0 + c^\text{apparent}\]</span></p>
<p>and because radioactivity (isotopes) does not affect chemical reactivity, the ratio of the apparent rate to the total rate should simply equal to the ratio of the concentration of radioactive <sup>32</sup>PP<sub>i</sub> to the total concentration of PP<sub>i</sub></p>
<p><span class="math display">\[\dfrac{v^\text{apparent}}{v}= \dfrac{c_0}{c^\text{apparent}+c_0}\]</span></p>
<p>Substituting into the Lineweaver-Burk equation:</p>
<p><span class="math display">\[\dfrac{1}{\left(\dfrac{v^\text{apparent}}{\left(\dfrac{c_0}{c^\text{apparent} + c_0}\right)}\right)}=\dfrac{K_m}{V_{\text{max}}( c^\text{apparent}+c_0)}+\dfrac{1}{V_{\text{max}}}\]</span></p>
<p>Rearranging and simplifying:</p>
<p><span class="math display">\[\dfrac{1}{v^\text{apparent}}=\dfrac{K_m+c_0}{V_{\text{max}}c_0}+\dfrac{c^\text{apparent}}{V_{\text{max}}c_0}\]</span></p>
<p>Obviously, there should be a linear relationship–<span class="math inline">\(\dfrac{1}{v^\text{apparent}}\)</span> against <span class="math inline">\(c^\text{apparent}\)</span>, with a gradient of <span class="math inline">\(\dfrac{1}{V_{\text{max}}c_0}\)</span> and an intercept of <span class="math inline">\(\dfrac{K_m+c_0}{V_{\text{max}}c_0}\)</span></p>
<p>Linear modelling:</p>
<pre class="r"><code>capp &lt;- c(.24, .48, .96, 1.92, 3.84, 7.68, 15.4, 30.7, 61.4, 122.9)
vapp &lt;- c(9274, 9351, 8590, 7392, 5784, 4061, 2502, 1426, 778, 437)
x &lt;- capp; y &lt;- 1/vapp
(coef &lt;- coef(lm(y~x)))</code></pre>
<pre><code>##  (Intercept)            x 
## 1.133102e-04 1.803159e-05</code></pre>
<pre class="r"><code>plot(capp, 1/vapp)
abline(coef = coef)</code></pre>
<p><img src="../worksheet/2019-10-16-radioactivity-worksheet-mt-2019_files/figure-html/unnamed-chunk-18-1.png" width="100%" /></p>
<p><span class="math inline">\(\dfrac{1}{V_{\text{max}}c_0}=1.803\times10^{-5}\rightarrow V_{\text{max}}c_0=55458\text{ cpm min}^{-1}\mu\text{M}\)</span></p>
<p><span class="math inline">\(\dfrac{K_m+c_0}{V_{\text{max}}c_0}=1.133\times10^{-4}\text{ cpm}^{-1}\text{ min}\)</span>.</p>
<p><span class="math inline">\(K_m+c_0=1.133\times10^{-4}\times 55458=6.283\text{ }\mu\text{M}\)</span></p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="../tutorial/principle-of-chemiosmotic-mechanism.html" class="archive-item-link">
                Principle of chemiosmotic mechanism 
              </a>
              <span class="archive-item-date">
                2019-10-15
              </span>
              <div class="article-summary" style="display: none;">
                
                  What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p class="topic">
<strong>Explain the difference between electrochemical gradient, pH gradient and membrane potentials. How can they be measured and how are these related to chemiosmotic mechanism.</strong>
</p>
<p>The electrochemical gradient describes the difference in chemical stability of ions (typically protons), between two aqueous compartments separated by a non-conducting and non-permeable membrane (typically a energy-transducing membrane such as mitochondrial inner membrane). It is quantified by the difference in Gibbs’ free energy, <span class="math inline">\(\Delta G\)</span>, or its molar equivalent, <em>chemical potential</em>, <span class="math inline">\(\Delta\mu\)</span>, of the ion species (proton) between the two compartments.</p>
<p>Usually, electrochemical gradient is used in studying processes coupled with <span class="math inline">\(\text{H}^+\)</span> movement across a membrane, where one side is positively charged and has a high <span class="math inline">\(\left[\text{H}^+\right]\)</span> and the other side with -ve charge and low <span class="math inline">\(\left[\text{H}^+\right]\)</span>. These are denoted by P and N sides, respectively, and the ‘gradients’ are often defined as the value of the corresponding parameter on the N side minus its value on the P side.</p>
<p>The electrochemical gradient is the sum of pH gradient (concentration gradient) and membrane potential (electrical gradient).</p>
<p>pH gradient is the (log-transformed) difference in the concentration of protons, <span class="math inline">\(\left[\text{H}^+\right]\)</span>, across the membrane. Protons tend to move down its concentration gradient to maximise the entropy of the whole system, and this tendency, i.e. potetial energy is the “chemical” part of the electrochemical potential. Intracellular/intra-orgenellar pH can be measured with pH-sensitive microelectrodes, nuclear magnetic resonance, or pH-sensitive fluorescent proteins <span class="citation">(Loiselle and Casey 2010)</span>. Low-RMM weak acids and bases can equilibrate across the membrane independent of the membrane potential. When there is an pH gradient, however, weak acids accumulate in the basic compartment and bases in the acidic compartment, and this property is used to measure pH gradient.</p>
<p>The membrane potential is the electrical potential difference across the membrane. According to Coulomb’s law, postively charge protons is repelled by the positively charged P side while attracted towards the N side, and this also results in a potential energy, which adds to the electrochemical gradient as the “electro-” part. Membrane potential is measured with microelectrodes.</p>
<p>Considering only the effect of concentration and electrical potential, the electrochemical potential for the proton on either side is defined as follows:</p>
<p><span class="math display">\[\bar{\mu}_{\text{H}^+} = \bar{\mu}_{\text{H}^+}^\circ + RT\ln\left[{\text{H}^+}\right] + z_{\text{H}^+}F\Psi\]</span></p>
<p>where <span class="math inline">\(\bar{\mu}_{\text{H}^+}^\circ\)</span> is the chemical potential under standard states, <span class="math inline">\(RT\ln\left[{\text{H}^+}\right]\)</span> and <span class="math inline">\(z_{\text{H}^+}F\Psi\)</span> are the additional chemical potential due to concentration and electrical potential, respectively.</p>
<p>The electrochemical gradient is the difference of the electrochemical potential between two compartments:</p>
<p><span class="math display">\[\Delta\bar{\mu} = \bar{\mu}_{\text{N}} - \bar{\mu}_{\text{P}} = \left(RT\ln\left[{\text{H}^+}\right]_{\text{N}} + z_{\text{H}^+}F\Psi_{\text{N}}\right) - \left(RT\ln\left[{\text{H}^+}\right]_{\text{P}} + z_{\text{H}^+}F\Psi_{\text{P}}\right)\]</span></p>
<p><span class="math display">\[ = -\ln{10}RT\left(\text{pH}_\text{N}-\text{pH}_\text{P}\right) + F(\Psi_{\text{N}}-\Psi_{\text{P}})=F\Delta\Psi- 2.3RT\Delta\text{pH} \text{  (J mol}^{-1}\text{)}\]</span></p>
<p>or it can be expressed as ‘protomotive force’ in (milli)volts:</p>
<p><span class="math display">\[\Delta\text{p}=\Delta\Psi-\dfrac{2.3RT}{F}\Delta\text{pH}\]</span></p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p class="topic">
<strong>What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?</strong> <br><sub><em>The essay MUST be equipped with at least 3 annotated diagrams and the length should be 1500-1750 words. Remember to include at least one KEY experimental evidence in the essay.</em></sub>
</p>
<div id="clues-leading-to-the-original-chemiosmosis-hypothesis" class="section level2">
<h2>Clues leading to the original chemiosmosis hypothesis</h2>
<p>The mechanism of oxidative phosphorylation were initially thought to be analogous to substrate-level phosphotylation, that the redox reactions create ‘high-energy’ intermediates, which in turn adds P<sub>i</sub> to ADP. In 1961, Peter Mitchell listed 6 facts to question this chemical mechanism hypothesis:</p>
<ol style="list-style-type: decimal">
<li>The ‘high-energy’ intermediate is not found.</li>
<li>The close association between phosphorylation and membranous structrures is not explained.</li>
<li>P/O ratio varies–it does not follow a fixed stoichiometry.</li>
<li>ATP hydrolysis outside mitochondria promotes NAD reduction inside, accentuated by succinate oxidation. (this observation is probably irrelevant or wrong?)</li>
<li>Uncoupling can be caused by reagents with distinct chemical properties–so they are unlikely to act on a single ‘intermediate’.</li>
<li>Unexplained swelling and shrinkage accompany phosphorylation.</li>
</ol>
<p>Trying to explain these phenomena, he proposed his initial chemiosmosis mechanism, with following 3 features:</p>
<ol style="list-style-type: decimal">
<li>A membrane located, reversible and anisotropic ‘ATPase’</li>
<li>An electron and hydrogen translocation system</li>
<li>A charge impermeable membrane</li>
</ol>
<p>According to this model, oxidation of substrates does not occur in a single compartment, but it instead allow protons and electrons to go in different directions, which in turn creates a concentration gradient of H<sup>+</sup> and OH<sup>-</sup>, as well as a membrane potential. ATP synthesis is driven by the one-way movement of OH<sup>-</sup> down its electrochemical gradient.</p>
<p>Mitchell’s origin model was rather different from the current model, as depicted in figure <a href="#fig:mitchell1961">1</a>, but it provided decent explanations for the phenomena listed above. (In his later review <span class="citation">(Mitchell 1966)</span>, there were some amendments made on his original model, but I have no time to read it)</p>
<div class="figure"><span id="fig:mitchell1961"></span>
<img src="img/mitchell1961.jpg" alt="Left, Mitchell's original chemiosmosis model; right, the current model" width="100%" />
<p class="caption">
Figure 1: Left, Mitchell’s original chemiosmosis model; right, the current model
</p>
</div>
<p>The principal idea was that, energy released from ‘asymmetric’ redox reactions can be used to establish a rather ‘physical’ form of energy (not the common chemical bond energy), the electrochemical gradient, and this is the first coupling. The second coupling is between the synthesis of ATP and the vectorised diffusion of some species promoted by the electrochemical gradient established by the first coupling.</p>
<p>It should be noted that the conception of first coupling was not new. Before the 1960s, it was known that redox reactions can create a electrical potential between two compartments, and it had been noted that some inhibitors of ion transport also has an effect (either stimulatory or inhibitory) on respiratory rate <span class="citation">(Robertson 1960)</span> (Fig. <a href="#fig:fig1">2</a>).</p>
<div class="figure"><span id="fig:fig1"></span>
<img src="img/chemiosmosis-inhibitor.png" alt="Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors" width="100%" />
<p class="caption">
Figure 2: Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors
</p>
</div>
</div>
<div id="proving-proton-pumping-by-the-redox-chain-and-determining-ho-ratio" class="section level2">
<h2>Proving proton pumping by the redox chain and determining H<sup>+</sup>/O ratio</h2>
<p>The ‘first coupling’ of the current chemiosmosis model means the pumping of protons into the P compartment powered by stepwise redox reactions along the ETC, with the overall oxidation of NADH or FADH<sub>2</sub> as substrate by molecular oxygen.</p>
<p>In actively respiring mitochondria, the proton concentrations (i.e. pH) on either side of the membrane is constant because the rate at which protons are pumped into the P compartment and that flowing back to the N compartment are equal. This makes it difficult to detect the presence of proton flux.</p>
<p><span class="citation">Mitchell and Moyle (1967)</span> devised the following procedure to solve the problem (Fig. <a href="#fig:mitchell1967a">3</a>):</p>
<ol style="list-style-type: decimal">
<li>mitochondria are incubated in anaerobic conditions, allowing H<sup>+</sup> to equilibrate (i.e. making <span class="math inline">\(\Delta p\)</span> fall back to zero)</li>
<li>a pH-sensitive glass electrode is used to monitor pH of the suspension (i.e. the environment outside the mitochondria)</li>
<li>a known quantity of O<sub>2</sub> is introduced into the suspension by ‘injecting pulses of air-saturated 150mM KCI solution.’</li>
<li>by measuring the pH change, the amount of protons pumped out of the mitochondria can be calculated, and with the known amount of O<sub>2</sub>, the H<sup>+</sup>/O ratio can be calculated.</li>
</ol>
<div class="figure"><span id="fig:mitchell1967a"></span>
<img src="img/mitchell1967a.jpg" alt="Determining H/O ratio (Mitchell 1967a)" width="100%" />
<p class="caption">
Figure 3: Determining H/O ratio (Mitchell 1967a)
</p>
</div>
<p>Additional procedures are needed to minimise error, and they’re described in <em>Bioenergetics</em> pp59-60</p>
</div>
<div id="proving-the-atp-synthesis-is-driven-by-the-electrochemical-gradient" class="section level2">
<h2>Proving the ATP synthesis is driven by the electrochemical gradient</h2>
<p>The electrochemical gradients is the sum of concentration gradient and electrical potential gradient, and each of them are shown to drive ATP synthesis.</p>
<p><span class="citation">Jagendorf and Uribe (1966)</span> showed ATP synthesis can be induced in dark, by artificially creating a pH difference across the thylakoid membrane in chloroplasts (Fig <a href="#fig:Sone">4</a>). In this experiment, broken chloroplasts (extracted from spinach) are first incubated in an acidic solution, allowing the thylakoid space to achieve the enviornmental pH of 4.0 (different acids are used to verify its non-specificity). Next, chloroplasts are exposed to pH 8.0 to develop the proton gradient across the thylakoid membrane. While moving out from the thylakoid space, protons drive ATP synthase. The amount of ATP produced is then determined by firefly luciferase assay. By repeating with different pHs, the authors concluded that the rate of ATP synthesis is positively correlated with the pH difference between two phases.</p>
<p><span class="citation">Sone et al. (1977)</span> used reconsitituted vesicles, made from purified <span class="math inline">\(\text{F}_o\text{F}_1\)</span> complex and the membrane of a thermophilic bacterium, to achieve ATP synthesis by induced membrane potential (Fig <a href="#fig:Sone">4</a>). They first incubated mitochondria with valinomycin at a fixed pH, then exposed the suspension to KCl. Valinomycin-mediated entry of K<sup>+</sup> shifts the membrane potential so the vesicle interior becomes more positively charged. Consequently, H<sup>+</sup> moves out and drives ATP synthase.</p>
<div class="figure"><span id="fig:Sone"></span>
<img src="img/sone.jpg" alt="Left, Jagendorf (2016); right, Sone (1977)" width="100%" />
<p class="caption">
Figure 4: Left, Jagendorf (2016); right, Sone (1977)
</p>
</div>
</div>
<div id="monitoring-the-protomotive-force-and-proton-flux-in-different-conditions" class="section level2">
<h2>Monitoring the protomotive force and proton flux in different conditions</h2>
<p>If the chemiosmotic model is corrent, it will have some properties resembling those of an electric circuit–the current (<span class="math inline">\(I\)</span>) and electromotive force (<span class="math inline">\(V\)</span>) are mirrored by the proton flux (<span class="math inline">\(J_{\text{H}^+}\)</span>) and the protomotive force (<span class="math inline">\(\Delta\text{p}\)</span>), respectively. These two terms can be monitored in different states of the proton circuit, and the results are in accordance with the chemiosmotic model.</p>
<div class="figure"><span id="fig:circuit-analogy"></span>
<img src="img/circuit_analogy.jpg" alt="Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state." width="100%" />
<p class="caption">
Figure 5: Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state.
</p>
</div>
<p>The different states shown in Fig <a href="#fig:circuit-analogy">5</a> are explained below:</p>
<ol style="list-style-type: decimal">
<li>When ETC substrate is absent, there is absolutely no oxygen consumption. No active forces act protons so they are allowed to equilibrate, and therefore the <span class="math inline">\(\Delta\text{p}\)</span> is zero. This resembles an electrical circuit without a power source.</li>
<li>When substrate (e.g. succinate) is provided, it is oxidised by ETC and oxygen is consumed. ETC starts to pump H<sup>+</sup> into the P face, building up the electrochemical gradient If the membrane was perfectly impermeable to H<sup>+</sup>, ETC would quickly stop as its provided energy for proton efflux balances the electrochemical gradient which favours protons’ influx, and there would be no oxygen consumption. However, the membrane is ‘leaky’ (and there are other processes causing influx of H<sup>+</sup>), causing constant influx of H<sup>+</sup>, which in turn allows H<sup>+</sup> to be steadily, but slowly, pumped to the P face, which is accompanied by oxygen usage. This resembles more of a closed circuit where energy is dissipated slowly through a high-resistance resistor than of an ‘open circuit’ as described in some books.</li>
<li>When ADP is added, ATP synthase can use the proton gradient to synthesise ATP. The proton flux through ATP synthase is huge, so the oxygen concentration (whose gradient parallels proton flux) decreases steeply. (I can’t give an rigorous mathematical explanation on the decrease of <span class="math inline">\(\Delta\text{p}\)</span>)</li>
<li>ATP is used up, and the condition is identical to state 2.</li>
<li>FCCP short-circuits the proton circuit by allowing H<sup>+</sup> quickly flowing back to the N face without doing ‘useful work’ (driving ATP synthase). The proton influx is so large that it outstrips the maximal efflux powered by ETC, so <span class="math inline">\(\Delta\text{p}\)</span> quickly falls back to zero.</li>
</ol>
</div>
<div id="lateral-proton-current" class="section level2">
<h2>Lateral proton current?</h2>
<p>The widely accepted view that ETC pumps protons transversally from the N to the P face may need updating for the following reasons stated by <span class="citation">Morelli et al. (2019)</span>:</p>
<ol style="list-style-type: decimal">
<li>H<sup>+</sup> almost never exist as isolated protons. Instead, they bind to water to form H<sub>3</sub>O<sup>+</sup>. Free protons in a mitochondrial periplasmic space are too few (fewer than 10) to drive ATP synthase.</li>
<li>Free protons are destructive for biological membranes</li>
<li>The pH inside mitochondria was shown to be higher by 0.5 units from what was previously believed <span class="citation">(Żurawik et al. 2016)</span></li>
<li>Phospholipid membranes are intrinsically (significantly) permeable to protons</li>
</ol>
<p>According to the classic view of proton pumping, complex I should have a continuous channel traversing the membrane. However, based on comprehensive X-ray studies, the ‘proton entrance half channel’ is not identified with certainty, while the ‘proton exit half channel’ is clearly identifiable, and an obvious proton tunnelling is found at the centre of complex I. This leads to an lateral mechanism of proton circuiting, as shown below:</p>
<div class="figure"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019)." width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019).
</p>
</div>
</div>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Jagendorf:1966">
<p>Jagendorf, A T, and E Uribe. 1966. “ATP Formation Caused by Acid-Base Transition of Spinach Chloroplasts.” <em>Proceedings of the National Academy of Sciences</em> 55 (1). National Academy of Sciences: 170–77. <a href="https://doi.org/10.1073/pnas.55.1.170">https://doi.org/10.1073/pnas.55.1.170</a>.</p>
</div>
<div id="ref-Loiselle:2010">
<p>Loiselle, Frederick B., and Joseph R. Casey. 2010. “Measurement of Intracellular pH.” In <em>Membrane Transporters in Drug Discovery and Development</em>, edited by Qing Yan. Vol. 637. Methods in Molecular Biology. Humana Press.</p>
</div>
<div id="ref-Mitchell:1966">
<p>Mitchell, Peter. 1966. “CHEMIOSMOTIC Coupling in Oxidative and Photosynthetic Phosphorylation.” Journal Article. <em>Biological Reviews</em> 41 (3): 445–501. <a href="https://doi.org/10.1111/j.1469-185X.1966.tb01501.x">https://doi.org/10.1111/j.1469-185X.1966.tb01501.x</a>.</p>
</div>
<div id="ref-Mitchell:1967a">
<p>Mitchell, Peter, and Jennifer Moyle. 1967. “Respiration-driven proton translocation in rat liver mitochondria.” <em>Biochemical Journal</em> 105 (3): 1147–62. <a href="https://doi.org/10.1042/bj1051147">https://doi.org/10.1042/bj1051147</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Robertson:1960">
<p>Robertson, R. N. 1960. “ION Transport and Respiration.” Journal Article. <em>Biological Reviews</em> 35 (2): 231–64. <a href="https://doi.org/10.1111/j.1469-185X.1960.tb01415.x">https://doi.org/10.1111/j.1469-185X.1960.tb01415.x</a>.</p>
</div>
<div id="ref-Sone:1977">
<p>Sone, N, M Yoshida, H Hirata, and Y Kagawa. 1977. “Adenosine Triphosphate Synthesis by Electrochemical Proton Gradient in Vesicles Reconstituted from Purified Adenosine Triphosphatase and Phospholipids of Thermophilic Bacterium.” <em>Journal of Biological Chemistry</em> 252: 2956–60.</p>
</div>
<div id="ref-Zurawik:2016">
<p>Żurawik, Tomasz Michał, Adam Pomorski, Agnieszka Belczyk-Ciesielska, Grażyna Goch, Katarzyna Niedźwiedzka, Róża Kucharczyk, Artur Krężel, and Wojciech Bal. 2016. “Revisiting Mitochondrial pH with an Improved Algorithm for Calibration of the Ratiometric 5(6)-Carboxy-Snarf-1 Probe Reveals Anticooperative Reaction with H+ Ions and Warrants Further Studies of Organellar pH.” <em>PLOS ONE</em> 11 (8). Public Library of Science: 1–17. <a href="https://doi.org/10.1371/journal.pone.0161353">https://doi.org/10.1371/journal.pone.0161353</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="../js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>