<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>Computational Genomics Practical (TT 2020) - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="Computational Genomics Practical (TT 2020) - Tianyi Shi (Hideyuki)">


<link href='../favicon.ico' rel='icon' type='image/x-icon' />















<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="../css/fonts.css" media="all">
<link rel="stylesheet" href="../css/main.css" media="all">
<script src=../js/main.js></script>

</head>

<body>
  <div class="wrapper single">
    <header class="header">
      <nav id="navbar">
  <a href=../ class="nav-logo">
    <img src="../images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="../cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="../tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="../cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="../tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    
<main class="content" role="main">

  <article class="article">
    
    <span class="article-duration">487 words</span>
    


    
    <h1 class="article-title">Computational Genomics Practical (TT 2020)</h1>
    <span class="article-date">2020-06-08</span>

    <p class="link-to-pdf"><a href="2020-06-08-computational-genomics-practical-tt-2020.pdf "> PDF Version Here</a></p>
    

    <div class="article-content">
      
<script src="../rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="../rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="../rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#quiz-5"><span class="toc-section-number">1</span> Quiz 5</a>
<ul>
<li><a href="#heatmaps-using-ggplot2-in-r"><span class="toc-section-number">1.1</span> Heatmaps (using <strong>ggplot2</strong> in R)</a></li>
<li><a href="#heatmaps-in-python-with-seaborn"><span class="toc-section-number">1.2</span> Heatmaps in Python (with <strong>seaborn</strong>)</a></li>
<li><a href="#average-gene-profile"><span class="toc-section-number">1.3</span> Average Gene Profile</a></li>
</ul></li>
<li><a href="#quiz-8"><span class="toc-section-number">2</span> Quiz 8</a>
<ul>
<li><a href="#matlab"><span class="toc-section-number">2.1</span> Matlab</a></li>
<li><a href="#using-r"><span class="toc-section-number">2.2</span> Using R</a></li>
</ul></li>
</ul>
</div>

<div id="quiz-5" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Quiz 5</h1>
<div id="heatmaps-using-ggplot2-in-r" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Heatmaps (using <strong>ggplot2</strong> in R)</h2>
<p>Loading Packages</p>
<pre class="r"><code>library(R.matlab)</code></pre>
<pre><code>## R.matlab v3.6.2 (2018-09-26) successfully loaded. See ?R.matlab for help.</code></pre>
<pre><code>## 
## Attaching package: &#39;R.matlab&#39;</code></pre>
<pre><code>## The following objects are masked from &#39;package:base&#39;:
## 
##     getOption, isOpen</code></pre>
<pre class="r"><code>library(reshape2)
library(tidyverse)</code></pre>
<pre><code>## ── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──</code></pre>
<pre><code>## ✓ ggplot2 3.3.2     ✓ purrr   0.3.4
## ✓ tibble  3.0.4     ✓ dplyr   1.0.2
## ✓ tidyr   1.1.2     ✓ stringr 1.4.0
## ✓ readr   1.4.0     ✓ forcats 0.5.0</code></pre>
<pre><code>## ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
## x dplyr::filter() masks stats::filter()
## x dplyr::lag()    masks stats::lag()</code></pre>
<p>Loading Data</p>
<pre class="r"><code>setwd(&#39;/Users/tianyishi/Documents/GitHub/ox/content/lab/src/Y2T3W7-genomics&#39;)
H3K4me3 &lt;- readMat(&#39;H3K4me3_ChIP_seq_glucose_gene_levels.mat&#39;)$H3K4me3.ChIP.seq.glucose.gene.levels
MNase &lt;- readMat(&#39;MNase_seq_glucose_gene_levels.mat&#39;)$MNase.seq.glucose.gene.levels
NET &lt;- readMat(&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;)$NET.seq.glucose.gene.levels.sense.strand
NET[NET &lt; -0.5 | NET &gt; 5] = NA</code></pre>
<pre class="r"><code>plot_heatmap &lt;- function(mat, title) {
  return(
    mat %&gt;% 
    melt() %&gt;% 
    ggplot(aes(Var2, Var1, fill=value))+
      geom_tile()+
      scale_x_continuous(expand = c(0,0), breaks = c(1, 501, 1001, 1501), labels = c(&#39;-500&#39;, &#39;TSS&#39;, &#39;+500&#39;, &#39;+1000&#39;))+
      scale_y_continuous(expand = c(0,0))+
      scale_fill_gradient(low=&quot;blue&quot;, high = &quot;yellow&quot;)+
      labs(title=title,
           x=&#39;Position relative to TSS (bp)&#39;,
           y=&#39;Gene number&#39;)
  )
}</code></pre>
<pre class="r"><code>plot_heatmap(H3K4me3, &#39;H3K4me3 ChIP-seq Glucose&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<pre class="r"><code>plot_heatmap(MNase, &quot;MNase&quot;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-2.png" width="672" /></p>
<pre class="r"><code>plot_heatmap(NET, &#39;NET&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-4-3.png" width="672" /></p>
</div>
<div id="heatmaps-in-python-with-seaborn" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Heatmaps in Python (with <strong>seaborn</strong>)</h2>
<p>Plotting above heatmaps using <strong>ggplot2</strong> in R is very slow. Using <strong>seaborn</strong> in Python is faster (plots not displayed):</p>
<pre class="python"><code>from scipy.io import loadmat
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

sns.set()

MNase = np.array(loadmat(&quot;MNase_seq_glucose_gene_levels.mat&quot;)[&quot;MNase_seq_glucose_gene_levels&quot;])
H3K4me3 = np.array(loadmat(&quot;H3K4me3_ChIP_seq_glucose_gene_levels.mat&quot;)[&quot;H3K4me3_ChIP_seq_glucose_gene_levels&quot;])
NET = readMat(&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;)[&#39;NET_seq_glucose_gene_levels_sense_strand.mat&#39;]

def plot_heatmap(mat, title):
    p = sns.heatmap(mat)
    p.set(
        title=title,
        xlabel=&#39;Position relative to TSS (bp)&#39;, ylabel=&#39;Gene number&#39;,
        xticks=[0, 500, 1000, 1500],
        xticklabels=[&quot;-500&quot;, &quot;TSS&quot;, &quot;+500&quot;, &quot;+1000&quot;],
        yticks=[0, 999, 1999, 2999, 3999, 4999],
        yticklabels=[1, 1000, 2000, 3000, 4000, 5000],
    )
    return p

plot_heatmap(MNase, &quot;MNase seq Glucose&quot;)
plt.show()
plot_heatmap(H3K4me3, &#39;H3K4me3 ChIP-seq Glucose&#39;)
plt.show()
plot_heatmap(NET, &#39;NET-seq Glucose sense strand&#39;)
plt.show()</code></pre>
</div>
<div id="average-gene-profile" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Average Gene Profile</h2>
<pre class="r"><code>xylabs &lt;- labs(x=&#39;Position relative to TSS (bp)&#39;, y=&#39;average NET-seq level&#39;)
xaxis &lt;- scale_x_continuous(breaks = c(0, 500, 1000, 1500), labels = c(&#39;-500&#39;, &#39;TSS&#39;, &#39;+500&#39;, &#39;+1000&#39;))

# Average gene profile MNase-seq Glucose
tibble(x=0:1500, y=apply(MNase, 2, mean)) %&gt;% 
  ggplot(aes(x, y))+
  geom_line()+
  xaxis+
  xylabs+
  labs(title=&#39;Average gene profile MNase-seq Glucose&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<pre class="r"><code># NET-seq data highest and lowest H3K4me3 comparison
tibble() %&gt;% 
  bind_rows(tibble(x=0:1500, y=apply(NET[1:501,], 2, function(x){mean(x, na.rm = TRUE)})) %&gt;% add_column(`H3K4me3 level`=&#39;high&#39;)) %&gt;% 
  bind_rows(tibble(x=0:1500, y=apply(NET[4501:5000,], 2, function(x){mean(x, na.rm = TRUE)})) %&gt;% add_column(`H3K4me3 level`=&#39;low&#39;)) %&gt;%
  ggplot(aes(x, y, color=`H3K4me3 level`))+
    geom_line()+
    xylabs+xaxis+
    labs(title=&#39;NET-seq data highest and lowest H3K4me3 comparison&#39;)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-6-2.png" width="672" /></p>
</div>
</div>
<div id="quiz-8" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Quiz 8</h1>
<div id="matlab" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Matlab</h2>
<pre class="matlab"><code>pos=yeast_gene_positions(:,[1,2,5])
% three colomns correspond to chromosome number, direction and TSS
avgs = []
i = 1
for p = pos.&#39;
  chr = NET_seq_data{p(1) + 16 * (1 - p(2))};
  if p(2)==1
    seq = chr(p(3):(p(3) + 299), :);
  else
    seq = chr((p(3) - 299):p(3), :);
  end
  avgs(i,:) = mean(seq);
  i = i + 1;
end
corr(avgs,yeast_gene_measurements(:,1))</code></pre>
</div>
<div id="using-r" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Using R</h2>
<pre class="r"><code>setwd(&#39;/Users/tianyishi/Documents/GitHub/ox/content/lab/src/Y2T3W7-genomics&#39;)
pos &lt;- readMat(&#39;yeast_gene_positions.mat&#39;)$yeast.gene.positions[,c(1,2,5)]
net &lt;- lapply(readMat(&#39;NET_seq_genome_wide_data.mat&#39;)$NET.seq.data, as_vector)
clc &lt;- apply(pos, 1, function(x){
  mean(net[[x[1] + (1-x[2])*16]][x[3]:(x[3]+ifelse(x[2]==0, -1, 1)*299)])
})
exp &lt;- readMat(&#39;yeast_gene_measurements.mat&#39;)$yeast.gene.measurements[,1]

plot(clc, exp)</code></pre>
<p><img src="../lab/2020-06-08-computational-genomics-practical-tt-2020_files/figure-html/unnamed-chunk-8-1.png" width="672" /></p>
<pre class="r"><code>cor(clc, exp)</code></pre>
<pre><code>## [1] 0.8281284</code></pre>
</div>
</div>

    </div>
  </article>
  <div id="disqus_thread"></div>

  <script>

    

    var disqus_config = function () {
      this.page.url = "https://tianyishi2001.github.io/ox/lab/computational-genomics-practical-tt-2020.html"
      this.page.identifier = "computational-genomics-practical-tt-2020"
    };

    (function () { 
      var d = document, s = d.createElement('script');
      s.src = 'https://tianyishi2001.disqus.com/embed.js';
      s.setAttribute('data-timestamp', +new Date());
      (d.head || d.body).appendChild(s);
    })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by
      Disqus.</a></noscript>
</main>


    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="../index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img
              src="../images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
      <p>COPYRIGHT © 2020 by Tianyi Shi. All rights reserved.</p>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="../js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>