<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>Notes - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="Notes - Tianyi Shi (Hideyuki)">


<link href='./favicon.ico' rel='icon' type='image/x-icon' />









<link href="./notes/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="./css/fonts.css" media="all">
<link rel="stylesheet" href="./css/main.css" media="all">
<script src=./js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=./ class="nav-logo">
    <img src="./images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="./cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="./tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="./cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="./tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content section" role="main">
  

  <article class="article">
    
      <h1 class="article-title">Notes</h1>
    
    <div class="article-content">
      <p>These are notes.</p>

    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="./notes/rust-notes.html" class="archive-item-link">
                Rust Notes 
              </a>
              <span class="archive-item-date">
                2020-02-29
              </span>
              <div class="article-summary" style="display: none;">
                
                


<pre class="rust"><code>fn main() {
    let number_list = vec![34, 50, 25, 100, 65];

    print_max(&amp;number_list);
    print_max_1(&amp;number_list);
    print_max_2(number_list)
}

fn print_max(number_list: &amp;[u16]) {
    let mut largest = &amp;number_list[0];
    for number in number_list.iter() {
        if number &gt; largest {
            largest = number;
        }
    }
    println!(&quot;The largest number is {}&quot;, largest);
}

fn print_max_1(number_list: &amp;[u16]) {
    let mut largest = &amp;number_list[0];
    for number in number_list {
        if number &gt; largest {
            largest = number;
        }
    }
    println!(&quot;The largest number is {}&quot;, largest);
} 

fn print_max_2(number_list: Vec&lt;u16&gt;) {
    let mut largest = number_list[0];
    for number in number_list {
        if number &gt; largest {
            largest = number;
        }
    }
    println!(&quot;The largest number is {}&quot;, largest);
} </code></pre>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./notes/molecular-cloning.html" class="archive-item-link">
                Molecular Cloning 
              </a>
              <span class="archive-item-date">
                2020-02-04
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="about-the-frame" class="section level1">
<h1>About the frame</h1>
<p>Sometimes we want to use the start/stop codon in the vector for our protein production, especially when tag sequences need to be included. In these cases, we need to check our inserted GOI is in frame with the start/codon on the vector. Frame mismatch can be corrected by adding 1 or 2 random nucleotide(s) to the primer (forward primer to match vector start condon and reverse primer to match the stop codon) between the GOI sequence and the restriction site sequence.</p>
<p>Some vectors have an N-terminal tag and stop codons in all three frames (e.g. <code>TGA C TGA C TGA</code>)</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./notes/techniques.html" class="archive-item-link">
                Techniques 
              </a>
              <span class="archive-item-date">
                2020-01-29
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="protein-structure" class="section level1">
<h1>Protein Structure</h1>
<div id="homology-modelling" class="section level2">
<h2>Homology Modelling</h2>
<div id="swiss-model" class="section level3">
<h3>Swiss Model</h3>
<div id="model-evaluation" class="section level5">
<h5>Model Evaluation</h5>
<ul>
<li>GMQE (global model quality estimation)
<ul>
<li>0-1, higher better</li>
<li>combines properties from the target-alignment and the template search method</li>
<li>replecting the expected accuracy of a model built with that alignment and the template and the coverage of the target</li>
<li>computed before modelling and after modelling. The latter also takes into account the QMEAN</li>
</ul></li>
<li>QMEAN
<ul>
<li>QMEAN z score: around 0: ideal; &lt; -4.0: low quality</li>
<li>QMEAN is a composite estimator based on different <strong>geometrical properties</strong> and provides both global and local absolute quality estimates on the basis of <em>one single model</em></li>
<li>the QMEAN z-score provides an estimate of the ‘degree of nativeness’ of the structural features observed in the model on a global scale. It indicates whether the QMEAN score of the model is comparable to what one expect from experimental structures of similar size</li>
<li>4 terms of the global QMEAN quality score:
<ul>
<li>the interaction potential between Cβ atoms only</li>
<li>and between all atoms</li>
<li>solvation potential</li>
<li>torsion angle potential</li>
</ul></li>
<li>the accuracy of local QMEAN scores is enhanced by QMEANDisCo
<ul>
<li>below 0.6: low quality</li>
<li>if downloaded, local score is reported in the B-factor column of the PDB file</li>
<li>can be visualised by choosing the colour scheme ‘QMEAN’</li>
<li>QMEANDisco assesses the consistency of observed interatomic distances in the model with ensemble information extracted from experimentally determined protein structures that are homologous to the target sequence. The ‘local quality’ plot shows, for each residue of the model (x-axis), the expected similarity to the native structure (y-axis)</li>
<li>in the ‘comparison’ plot, model quality scores are related to scores obtained for experimental structures of similar size. The x-axis shows protein length and the y-axis shows the normalized QMEAN score. Every dot represents an experimentally determined structure. The mean and standard deviation of the experimental structures around the x-location of the star (the model) are the basis to calculate the QMEAN z-score of the model.</li>
</ul></li>
</ul></li>
</ul>
</div>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./notes/reading-lists.html" class="archive-item-link">
                Reading Lists 
              </a>
              <span class="archive-item-date">
                2020-01-19
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="paper-i" class="section level2">
<h2>Paper I</h2>
<div id="x-ray-and-em" class="section level3">
<h3>X-ray and EM</h3>
<p> “Biomolecular Crystallography” by Bernard Rupp (preferably 2 nd Edition) Garland Science ISBN 978-0815340812  “Outline of Crystallography for Biologists” by David Blow Oxford University Press, ISBN 978-0198510512  “Crystallography Made Crystal Clear” by Gale Rhodes Academic Press ISBN 978-0125870733  Milne JL, Borgnia MJ, Bartesaghi A, Tran EE, Earl LA, Schauder DM, Lengyel J, Pierson J, Patwardhan A, Subramaniam S. Cryo-electron microscopy–a primer for the non-microscopist. FEBS J. 2013 Jan;280(1):28-45. doi:</p>
<p>10.1111/febs.12078.</p>
<p> Kühlbrandt W. Cryo-EM enters a new era. Elife. 2014 Aug 13;3:e03678. doi:</p>
<p>10.7554/eLife.03678  Yong Xiong (2007) From electron microscopy to X-ray crystallography:</p>
<p>molecular-replacement case studies. Elife. 2014 Aug 13;3:e03678.</p>
</div>
</div>
<div id="paper-vi" class="section level2">
<h2>Paper VI</h2>
<div id="ht-2020-protein-purification" class="section level3">
<h3>HT 2020: Protein Purification</h3>
<ul>
<li>‘Methods in Enzymology’ volumes 22, 34, and 104 are devoted to techniques of protein purification.</li>
<li>‘Protein Purification Methods: A Practical Approach’ IRL Press, Eds ELV Harris and S Angal</li>
</ul>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="./js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="./index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="./%20images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="./js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>