---
title: "Research Proposal: Elucidating the Functions of Polycomb Group Proteins in the Context of Neurodevelopment"
author: "Tianyi Shi"
date: '2022-08-21'
bibliography: [bib/proposal.bib]
header-includes:
  # converts hyperlinks to footnotes
  - \newcommand{\hrefootnote}[2]{#1\footnote{\url{#2}}}
  # prevents figures less than 80% long taking up their own page
  - \renewcommand{\floatpagefraction}{.8}%
  - \usepackage{float}
documentclass: article
output: 
  # enables table references etc.
  bookdown::pdf_document2:
    keep_tex: true
    toc: false
    number_sections: false
    # better handling of non-English letters
    # latex_engine: xelatex
    # includes:
      # in_header: "preamble.tex"
  blogdown::html_page:
    number_sections: true
    toc: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Abstract 

The Polycomb repressive complexes PRC1 and PRC2 formed by Polycomb group (PcG) proteins are epigenetic regulators that are involved in the maintenance of gene repression patterns. Although their histone modification activities (H3K27me3 by PRC2 and H2A119ub by PRC1) have been well defined and their ability to drive formation of compacted chromatin regions (PcG domains) has been unveiled using chromatin conformation capture (3C) based technologies, the exact mechanism(s) by which they achieve gene repression are not fully understood. Although the studies based on mouse embryonic stem cells (mESCs) unraveled many mechanistic details which leads to a refined model of hierarchical recruitment and feedforward loops that underscores the central role of variant PRC1-mediated H2A ubiquitylation in gene repression, studies on partially and terminally differentiated cells, such as neural precursor cells and neurons, revealed alternative mechanisms of PcG function that deviate from the model developed on mESCs. This proposal reviews some of these discoveries focusing on the context of neurodevelopment, highlights the aspects in these studies that require further investigation, and suggests potential methods to do so.

# Main

## PcG Proteins and PRCs

Polycomb group (PcG) proteins are a group of evolutionarily conserved epigenetic modulators that mainly serve to maintain the repressed state of genes in different contexts, most notably embryonic development and adult stem cell maintenance. PcG proteins assemble into either Polycomb repressive complex 1 (PRC1), which is characterized by its core components Ring1B/Ring1A (which deposits H2AK119ub) and PCGF2/4/1/3/5/6, or PRC2, whose core components comprise EZH1/2 (which deposits H3K27me1/2/3), EED, SUZ12 and RBBP4/7. PRC1 is further divided into PHC1/2/3- and Cbx2/4/6/7/8-containing canonical PRC1 (cPRC1) which has low Ring1 ubiquitylation activity and RYBP/YAF2-containing variant PRC1 (vPRC1) which has pronounced Ring1 ubiquitylation activity [@Fursova-2019]. The current knowledge about these PcG proteins and the different PRCs they constitute has recently been extensively reviewed by @Blackledge-2021 and @Kim-2022.

## Ubiquitylation (vPRC1)-dependent and ubiquitylation-independent (cPRC1-dependent) Functions of PcG

It has been well established that the E3 ubiquitin ligase Ring1B (or Ring1A) in PRC1 deposits H2Aub (H2AK119Ub in *Mus musculus*), and its activity is more pronounced in vPRC1 than in cPRC1 [@Kim-2022; @Fursova-2019]. What remains controversial are the function(s) of H2A ubiquitylation and its dispensability for PcG functions.

Early studies on the roles of Ring1-mediated H2A ubiquitylation (e.g. @Kundu-2017) were perplexed by the use of the I53A mutant of Ring1B, which does not completely abolish ubiquitylation activity, and by not knocking out Ring1A, which would compensate for Ring1B deficiency. @Tsuboi-2018 developed a *bona fide* ubiquitylation-deficient Ring1B I53A/D56K double mutant, whose complete inactivity was also independently confirmed by @Blackledge-2020. Using this genuine ubiquitylation-deficient mutant together with Ring1A KO, @Blackledge-2020 showed that ubiquitylation activity of PRC1 is essential for PcG domain formation (see also the following section) as well as PcG-mediated gene repression in ESCs and their results strongly substantiates the hierarchical recruitment model in which H2AK119ub deposited by vPRC1 recruits PRC2, and H2K27me3 deposited by PRC2 recruits cPRC1, which then drive long-range interactions between PcG target sites, contributing to PcG domain formation and hence gene repression (Figure 4H of @Blackledge-2020; Figure 2 of @Kim-2022). In this study, PRC2.2 occupancy at PcG target sites was found to be reduced upon ubiquitylation abolishement, which is expected because of the PRC2.2-specific component JARID2 acting as a reader for H2AK119ub. Interestingly however, PRC2.1 occupancy was also reduced upon despite of the H2A119ub-independent DNA-binding activity conferred by PCL1/2/3 proteins. Thus the mechanism by which PRC2.1 responds to (loss of) ubiquitylation needs further clarification.

@Fursova-2019 pinpointed the critical role of vPRC1s in both genome-wide pervasive and high-intensity local H2A ubiquitylation and demonstrated that vPRC1-mediated ubiquitylation (but not cPRC1) is vital for recruitment of PRC2 (and thus PcG domain formation) and gene repression, but this study was also confined to mESCs.

@Tsuboi-2018 were the first to demonstrate a ubiquitylation-independent role of PRC1 in gene suppression. They showed that, in Ring1A KO mouse neural precursor cells (NPCs), after conditional knockout of floxed Ring1B, ubiquitylation-deficient Ring1B introduced by retrovirus was able to rescue the repression of neurogenic genes *Neurog1*, *Fezf2*, *Lef1* and *Tcfap2c* to a similar extent as did transfected WT Ring1B in the astrogliogenic phase but not the neurogenic phase, and this rescue of repression was observed for *Hox* genes in both neurogenic and astrogliogenic NPCs. This study further established the critical roles of the SAM domain of Phc2 and histone deacetylation in the adoption of ubiquitylation-independent functioning of PRC1. Given the central role of vPRC1 in ubiquitylation [@Fursova-2019] and the elevated levels of cPRC1-specific Phc2 at neurogenic genes found in this study (Fig. 4A of @Tsuboi-2018), this study implied a switching from vPRC1-centered mode of repression to a cPRC-centered mode of repression. The preferential recruitment of cPRC1 could be due to the recognition of PRC2-catalyzed H3K27me3 by the chromodomain of the cPRC-specific Cbx subunit (Fig. 3 of @Blackledge-2021), and this is consistent with the necessity for NuRD/HDAC-mediated H3K27me3 deacetylation [@Tsuboi-2018]. However, this need for deacetylation seemed to hold true only for the *Neurog1* gene, and Mbd3 (part of NuRD/HDAC) KO did not significantly affect the global H3K27me3 level [@Tsuboi-2018]. The mechanisms (especially the role of PRC2/H3K27me3) and generalizability of this mode of repression thus needs further investigation. A first step could be assessing PRC2/H3K27me3 occupancy at the loci of interest using ChIP-qPCR. Genome-wide techniques including RNA-seq, ChIP-seq (or better CUT&RUN or CUT&Tag) and Hi-C will be powerful tools to test the generalizability of this mode of repression.

Despite various developmental abnormalities (notably homeotic transforamtion) upon disruption of cPRC1 components in vivo being reported e.g. by @Isono-2013, @Lau-2017, @Wani-2016, many studies, especially those focusing on mESCs (e.g. @Fursova-2019), concluded that cPRC1 contributes little, if at all, to gene repression. This paradox is at least partially explained by a model in which PcG proteins have both acute and long-term mechanisms of gene repression, with vPRC1-mediated ubiquitylation providing the former and cPRC1 and PRC2 mainly contributing to the latter (reviewed by @Blackledge-2021).

In the study conducted by @Fursova-2019, combined removal of vPRC1-specific PCGF1/3/5/6 largely recapitulated the effect of Ring1A/B removal in terms of the scale and magnitude of gene derepression and loss of ubiquitylation, whereas removal of cPRC1-specific PCGF2/4 had minimal effects. Interestingly, in this study, 137 genes seemed to be sensitive to PCGF2 removal (exhibited additional reactivation after PCGF1/3/5 knockout and displayed further reduction in Suz12 occupancy, H2AK119ub and H3K27me3). Also, with simultaneous PCGF1/3/5/6 knockdown, despite the large-scale gene derepression accompanied by the loss of Ring1B and Suz12 occupancy as well as loss of H2AK119ub and H3K27me3 at PRC1-bound sites, Ring1B occupancy and H2AK119ub was retained to some extent at 354 PRC1 target gene promoters and Suz12 and H3K27me3 were largely unaffected. These genes, like those PCGF2-sensitive genes, were located at the large Polycomb chromatin domains with extremely high levels of Ring1B, and the results indicate that PCGF2-containing PRC1 contributes to ubiquitylation and gene repression at these specific loci, leading to the following questions: What are the identities of the PCGF2-containing PRC1 in these loci? (Are they cPRC1 or the rare PCGF2-containing vPRC1? Do they associate with special auxiliary factors?) What drives the formation of these loci with extremely high Ring1B occupancy? Are there any special features about the genes in these loci? How do these loci change in differentiated cell types? Conducting similar studies in other cell types such as NPCs, combined with techniques to visualize subnuclear chromatin domains such as Hi-C, might provide insight into these issues.

Recently, @Zhou-2022 discovered a role of rixosome-mediated RNA degradation in PcG-mediated gene repression and chromatin compaction. Rixosome directly associates Ring1B and promotes degradation of nascent transcripts. Could this mechanism be critical for ubiquitylation-independent PRC1-mediated gene repression? If that is the case, why did PCGF2/4 KO have negligible effects on gene repression in mESCs despite causing significant reduction in Ring1B occupancy [@Fursova-2019]? A simple answer to the latter question could be that the remaining vPRC1 is able recruit enough rixosome required for repression, but it could also be that the PRC1-rixosome co-operation is more prominant in differentiated cells (human embryonic kidney cells in the case of @Zhou-2022) than in pluripotent ESCs (as in @Fursova-2019). The affinity of rixosome to vPRC1 vs cPRC1 as well as the significant of PRC1-rixosome co-operation in different cell types are thus worth further investigation.

## The Mechanisms of PcG-mediated Higher-Order Chromatin Interactions and its Relationship With Gene Repression

PRC1 and PRC2, along with H2AK119ub and H3K27me catalyzed by them, often colocalize at inactive gene promoters on CpG islands, forming so-called PcG domains [@Blackledge-2021; @Kim-2022]. Distant PcG-bound loci often associate with each other, and this kind of long-range chromatin interactions leads to formation of subnuclear structures known as Polycomb bodies. It has been well established that PRC1 and PRC2 are both readers of the histone modifications of themselves and of each other, thereby creating a self-reinforcing positive feedback loop that promotes formation of PcG domains (Fig. 3 of @Blackledge-2021). vPRC1-mediated ubiquitylation in particular plays a crucial role in the initiation and maintenance of PcG domains, at least in mESCs [@Fursova-2019; @Blackledge-2021]. PcG domain maintenance is also thought to be relying on the self-polymerization activity of the SAM domain of Phc1/2 [@Isono-2013; @Kundu-2017] and nucleosome bridging activity of Cbx2 [@Lau-2017], both of which are components of cPRC1. Such capabilities of Phc2 and Cbx2 (but not necessarily Polycomb bodies) were demonstrated to contribute to gene repression and embryonic development (disruption of which causes gene derepression and homeotic transformation) in *Drosophila* [@Wani-2016] and mice [@Isono-2013; @Lau-2017].

@Isono-2013 showed that in mouse embryonic fibroblasts (MEFs), knockin of *Phc2*^L307R^ with a defective SAM domain resulted in perturbation of PRC1 clustering and de-condensation of the *Hoxb* gene cluster (examined by immunofluorescence) and derepression of some of the genes in the cluster (*Hoxb4* and *Hoxb13* but not *Hoxb3* and *Hoxb9*) as well as a fraction (12%) of other genes co-occupied by PRC1 and H3K27me3. The decrease in PRC1 occupancy caused by Phc2^L307R^ also caused reduction of PRC2 (Ezh2 and Suz12) and H3K27me3 at genes co-occupied by PRC1 and PRC2, but the mechanism by which this occurs requires further clarification, since purturbation of only cPRC1 should not have an acute effect on PRC2/H3K27me3, at least in mESCs [@Fursova-2019]. It would be interesting to see whether this differential effect of cPRC1 purturbation is due to different cell types (MEF vs mESC) or due to purturbation of PcG bodies. It might be that the both are true, since Cbx7, which is expressed specifically in ESCs, has marginal neucleosome compaction activity, and only when Cbx2 with high neucleosome compaction is expressed as cells differentiate (Fig. 3 of @Kim-2022) could the formation of PcG bodies have a significant contribution to recruitment of PRC2 and gene repression. However, contrary to this hypothesis, when spatial chromatin interactions were assessed using a 3C-based technology called 5C (Chromosome Conformation Capture Carbon Copy) in mESCs, @Kundu-2017 revealed PcG domains that were distinct from and smaller than topologically associated domains (TADs) and were dependent on the SAM domain of Phc1 (of cPRC1), which actually diminished upon neural differentiation. This notion is corroborated by @Bonev-2017, who showed that, in the context of neurodevelopment, the strong PcG-mediated interactions in mESCs become disrupted with neural differentiation, as assessed by Hi-C (the genome-wide version of 3C), although the interactions strengthened in a small fraction of sites, such as *Skap2* in close contact with *HoxA*. It is worth to check whether genes whose repression is supposed to be dependent on cPRC1 or PcG bodies, such as *Neurog1* supposed by @Tsuboi-2018, reside in such uncommon sites. @Tsuboi-2018 proposed a model which hypothesized that initial ubiquitylation-dependent repression by (v)PRC1 serves to maintain differentiation potential such that the target genes are temporarily repressed and poised for activation in response to appropriate cues, whereas ubiquitylation-independent repression by (c)PRC1 serves to restrict the differentiation potential by robustly repressing target genes upon cell fate commitment. In relation to @Bonev-2017 and @Kundu-2017, here I extend this model by suggesting that there are two different types of PcG bodies (domains), one being vPRC1 (ubiquitylation) dependent and the other one being ubiquitylation independent but cPRC1-dependent, that are responsible for these two different modes of PRC1-dependent repression, and Cbx7 vs Cbx2 might be one of the key factors that discriminate these two types of PcG bodies.

While @Bonev-2017 and @Kundu-2017 both used 3C-based methods (5C or Hi-C) to reveal the presence of Polycomb bodies and, to some extent, their changes upon neural differentiation, their relationship with transcriptional activity was not systematically assessed, thus warranting further investigation. In fact, even for TADs, which are the first type of chromatin domain discovered by Hi-C and are ubiquitous throughout the genome, their influence on gene transcription is still not fully understood and is under active research [@Beagan-2020]. The upcoming methodologies for studying TADs might be adapted to the studies on PcG bodies in order to further dissect their roles in gene expression regulation. 

Recently, a comprehensive epigenome atlas of NPCs within the embryonic mouse forebrain has been constructed, which involved assessment of the transcriptome (scRNA-seq), chromatin accessibility (scATAC-seq), histone modifications (CUT&Tag and CUT&RUN), promoter-enhancer interactions (Capture-C) and high-order chromatin structure (Hi-C) in the medial/caudal/lateral ganglionic eminence (MGE, CGE, LGE) and cortex that are undergoing neurogenesis (at embryonic day 12.5 (E12.5); also E14.5 for cortex) [@Rhodes-2022]. These data, which are accessible via public databases, will serve as an important reference for our specialized studies relationship between PcG bodies and transcriptional regulation in the context of neurodevelopment.

## Diverse PcG Mechanisms in Neurodevelopment

Studies on murine PcG have already led to a refined model of the mechanism of action of PcG-mediated gene repression featuring hierarchical recruitment and positive feedback (Figure 2 of @Kim-2022), but the evidence that support this model mostly come from studies using mESCs. It is becoming increasingly evident that, the choice of paralogs of PRC components and incorporation of auxiliary proteins allow assembly of a wide range of PRC subcomplexes, and these subcomplexes can display cell type-specificity and thus may contribute to cell type-specific mechanisms of PcG-mediated repression [@Kim-2022]. This idea is well illustrated by the examples presented in previous sections as well as in other studies, such as a gene-activating role of vPRC1 during skin development [@Cohen-2018] and in the CNS [@Gao-2014], as well as genes being differentially repressed by PRC2.1 and PRC2.2 during NPC differentiation [@Petracovici-2021].

In addition to the gene repression mechanisms of the PcG proteins *per se*, its diversified regulatory mechanisms, many of which are yet to be uncovered, are also indispensable for the varying roles for PcG in a wide range of developmental contexts. 
Whereas PRC2 (Ezh2)-mediated H3K27me3 was shown to repress *Wif1* and *Dkk2* and thus activates *Wnt* signaling in the developing midbrain at ~E11.5, which is essential for the maintenance of midbrain identity (prevention of rostralization) [@Zemke-2015], Ring1B promotes ventral identity in the early stage telecephalon (before E9.0) by direct repression of BMP and Wnt signalling outside of the dorsal midline due to its higher occupancy at *Bmp* and *Wnt* genes [@Eto-2020]. Looking forward, the spatiotemporal differences in PcG recruitment and activity might also play important roles in other neurodevelopment-related contexts, and single cell epigenome profiling methods such as scCUT&Tag [@Bartosovic-2021] are ideal for accelerating their discovery. Combined with other single-cell technologies, the mechanisms that govern the development of the intricate brain will be unveiled at an unprecedented rate.

# References