<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="generator" content="Hugo 0.101.0" />


<title>Tutorials - Tianyi Shi (Hideyuki)</title>
<meta property="og:title" content="Tutorials - Tianyi Shi (Hideyuki)">


<link href='./favicon.ico' rel='icon' type='image/x-icon' />









<link href="./tutorial/index.xml" rel="alternate" type="application/rss+xml" title="Tianyi Shi (Hideyuki)" />







<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
    integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
    crossorigin="anonymous"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
    crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
    crossorigin="anonymous"></script>


<link rel="stylesheet" href="./css/fonts.css" media="all">
<link rel="stylesheet" href="./css/main.css" media="all">
<script src=./js/main.js></script>

</head>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-FPDHCSDNTQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag() { dataLayer.push(arguments); }
  gtag('js', new Date());

  gtag('config', 'G-FPDHCSDNTQ');
</script>

<body>
  <div class="wrapper list">
    <header class="header">
      <nav id="navbar">
  <a href=./ class="nav-logo">
    <img src="./images/logo.png" width="50"
      height="50" alt="Logo">
  </a>

  <p class="nav-title">Tianyi's Work at Oxford</p>
  <div class="menu">
    <ul>
      
      <li><a href="./cv.pdf">CV</a></li>
      
      <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
      
      <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
      
      <li><a href="./tutorial.html">Oxford</a></li>
      
      
    </ul>
  </div>
</nav>
<div class="nav-button" id="nav-button" onclick="toggleNav(this)" title="Open menu">
  <div class="bar bar1"></div>
  <div class="bar bar2"></div>
  <div class="bar bar3"></div>
</div>
<nav id="main-nav" class="overlay" style="height: 0%;">
  <ul class="overlay-content">
    
    <li><a href="./cv.pdf">CV</a></li>
    
    <li><a href="mailto:ShiTianyi2001@outlook.com">Email</a></li>
    
    <li><a href="https://github.com/TianyiShi2001">GitHub</a></li>
    
    <li><a href="./tutorial.html">Oxford</a></li>
    
  </ul>
</nav>

    </header>

    

<main class="content section" role="main">
  

  <article class="article">
    
      <h1 class="article-title">Tutorials</h1>
    
    <div class="article-content">
      
    </div>
  </article>

  <div class="archive">
    
    
    
      
      <h2 class="archive-title">2020</h2>
          
            <article class="archive-item">
              <a href="./tutorial/insertion-of-membrane-proteins.html" class="archive-item-link">
                Insertion of Membrane Proteins 
              </a>
              <span class="archive-item-date">
                2020-12-03
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon"><span class="toc-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</a></li>
<li><a href="#single-pass-proteins"><span class="toc-section-number">2</span> Single-Pass Proteins</a>
<ul>
<li><a href="#type-i-proteins"><span class="toc-section-number">2.1</span> Type I Proteins</a></li>
<li><a href="#type-ii-and-iii-proteins"><span class="toc-section-number">2.2</span> Type II and III Proteins</a></li>
</ul></li>
<li><a href="#multi-pass-proteins"><span class="toc-section-number">3</span> Multi-Pass Proteins</a></li>
<li><a href="#insertases"><span class="toc-section-number">4</span> Insertases</a>
<ul>
<li><a href="#insertion-of-tail-anchored-proteins-by-the-get-system"><span class="toc-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</a></li>
<li><a href="#the-roles-of-yidc"><span class="toc-section-number">4.2</span> The Roles of YidC</a></li>
</ul></li>
<li><a href="#beta-barrel-proteins"><span class="toc-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>The majority of integral membrane proteins in an eukaryotic cell are made of one or more <span class="math inline">\(\alpha\)</span>-helices and are first inserted co-translationally via the Sec61 translocon into the endoplasmic reticulum (ER) membrane, assembled there, and then dispatched, via the golgi apparatus and vesicles, to their final destinations. Bacteria use the SecYEG translocon, which is a homolog of Sec61, to insert membrane proteins into their plasma membrane. There are also translocons specific to chroloplasts (TIC-TOC), mitochondria (TIM-TOM) and peroxisomes (PEX), but this essay will focus on the Sec61 translocon.</p>
<div id="targeting-the-protein-to-the-er-membrane-srp-and-the-sec61-translocon" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Targeting the Protein to the ER Membrane: SRP and the Sec61 Translocon</h1>
<p>The common co-translationally inserted membrane proteins are targeted to the Sec61 translocon on the ER membrane via the cytosolic signal recognition particle (SRP) and the SRP receptor near the translocon.</p>
<p>SRP is a ribonucleoprotein particle made up of 6 proteins bound to a 300-nucleotide RNA acting as a scaffold. The M and Alu domains of SRP are critical for its functions. The M domain contains a cleft whose inner surface is lined by methionine and other hydrophobic side chains. This cleft binds to a hydrophobic segment of the nascent peptide as soon as it emerges from the exit tunnel. Meanwhile, the Alu domain blocks the elongation factor 2 (eEF2) binding site on the ribosome, pausing translation.</p>
<p>Usually it is the first hydrophobic segment that is recognised by SRP. This segment can either be a pure ER-targeting signal sequence or be the first TM helix itself (see next section). Recognition by SRP not only serves to target the ribosome-nascent protein to the ER but also prevents exposure of the hydrophobic segment to the aqueous environment.</p>
<p>SRP and SRP receptor each has a GTP binding site. When their GTP-bound forms associate with each other, they form two GTPase active sites. The GTPase activity is inhibited by signal peptide binding. Once the nascent chain has left SRP and enters the translocon, the two GTP molecules are hydrolysed, causing dissociation of SRP and its receptor. GTP replaces GDP on both proteins, and they are ready to initiate another round of ER targeting.</p>
<p>The Sec61 translocon is made up of 3 subunits: Sec61<span class="math inline">\(\alpha\)</span>, an integral membrane protein composed of two 5-helix bundles that constitutes the central channel and two smaller proteins, Sec61<span class="math inline">\(\beta\)</span> and Sec61<span class="math inline">\(\gamma\)</span>. The central channel through Sec61<span class="math inline">\(\alpha\)</span> is sealed by a plug made of a helical peptide that only opens during traslocation. The channel can also open laterally by hinging apart the two 5-helix bundles to expose a hydrophobic binding pocket for signal sequences and/or TM helices of the nascent peptide. The verticle opening allows elongation of the nascent peptide chain through the central pore, and the lateral opening allows attachment of signal sequences and exit of TM helices into the membrane (Figure <a href="#fig:sec61">1.1</a>).</p>
<div class="figure"><span id="fig:sec61"></span>
<img src="../img/sec61.png" alt="The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation." width="100%" />
<p class="caption">
Figure 1.1: The structure and mode of action of the Sec61 translocon. Left: the closed conformation. Right: the open conformation.
</p>
</div>
</div>
<div id="single-pass-proteins" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Single-Pass Proteins</h1>
<p>A single-pass membrane protein use a single 20-25 residue long <span class="math inline">\(\alpha\)</span>-helix made mainly of hydrophobic amino acids to traverse the membrane. The <span class="math inline">\(\alpha\)</span>-helical structure satisfies hydrogen bonding between main chain carbonyl and amino groups and exposes hydrophobic side chains, which form favourable hydrophobic interactions with the aliphatic core of the membrane bilayer. Co-translational insertion of this helix is a natural strategy to minimise unfavourable interation between the hydrophobic helix and the aqueous cytosolic environment and between the hydrophilic regions of the peptide surrounding the helix and the hydrophobic core of the bilayer.</p>
<p>Single-pass proteins are classified into three types according to the mechanism by which they are inserted. All of them are described in the following sections.</p>
<div id="type-i-proteins" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Type I Proteins</h2>
<p>Type I membrane proteins, like soluble proteins to be translocated into the ER lumen, contain an N-terminal ER-targeting signal sequence to be cleaved. The signal sequence is recognised by SRP, causing the ribosome-nascent peptide complex to be targeted to the Sec61 translocon. Sec61<span class="math inline">\(\alpha\)</span> opens the lateral gate, and the signal peptide fits into the exposed hydrophobic bindng pocket. The sequence following the signal peptide displaces the plug and inserts into the central channel, resulting in the conformation shown in Figure <a href="#fig:mem-insert-type-i">2.1</a>. Signal peptidase cleaves off the signal peptide once it recognises a specific sequence on the C-terminal end of the signal peptide. After cleavage, translation continues and the newly synthesised sequence is threaded through the channel and enters the ER lumen, until another hydrophobic segment is encountered. This segment is the TM helix of the nascent protein, and it acts as a <em>stop-transfer signal</em> (or “stop-transfer anchor sequence”) by triggering the lateral opening of Sec61<span class="math inline">\(\alpha\)</span> and thus allowing this helix to move into the membrane. The C-terminus continues to be synthesised and loops out on the cytosolic side of the membrane. Eventually, the hydrophic N- and C- termini are on the luminal and the cytosolic face, respectively.</p>
<div class="figure"><span id="fig:mem-insert-type-i"></span>
<img src="../img/mem-insert-type-i.png" alt="Insertion of type I membrane proteins." width="100%" />
<p class="caption">
Figure 2.1: Insertion of type I membrane proteins.
</p>
</div>
</div>
<div id="type-ii-and-iii-proteins" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Type II and III Proteins</h2>
<p>Type II and III proteins do not contain a cleavable signal sequence, and they have opposite orientations: the N-terminus is on the cytosolic and the luminal face in type II and type III proteins, respectively.</p>
<p>Type II and III proteins use the sequence of their TM helix as an ER-targeting signal (“start-transfer singal”, or “signal anchor sequence”), and the targeting is also mediated by SRP and SRP receptor. The distribution of positively charged residues around the helix dictates its orientation: the positively charged residues tend to remain on the cytosolic side of the membrane, thus in type II proteins where the N-terminal side has more positively charged residues, the helix adopts the orientation with the N-terminus facing the cytosolic side; and in type III proteins where the C-terminal side has more positively charged residues the opposite orientation is adopted (<br />
Figure <a href="#fig:mem-insert-type-iii-iii">2.2</a> ).</p>

<div class="figure"><span id="fig:mem-insert-type-iii-iii"></span>
<img src="../img/mem-insert-type-iii-iii.png" alt="Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61\(\alpha\) results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol." width="100%" />
<p class="caption">
Figure 2.2: Insertion of type II and III membrane proteins. In type II proteins, the orientation of the signal anchor sequence in the hydrophibic binding pocket of the laterally opened Sec61<span class="math inline">\(\alpha\)</span> results in the opening of the pore in the vertical direction (in the same way as type I proteins, but the signal peptide is not cleaved off), allowing the remaining C-terminal portion of the polypeptide to thread through the pore and enter the ER lumen. In type III proteins, the orientation of the signal anchor sequence causes the channel to close, and the C-terminal portion is synthesised in the cytosol.
</p>
</div>
</div>
</div>
<div id="multi-pass-proteins" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Multi-Pass Proteins</h1>
<p>Multi-pass membrane proteins (a.k.a. type IV proteins) contain more than one TM <span class="math inline">\(\alpha\)</span> helices and traverse the membrane multiple times. Since the adjacent helices are in opposite orientations, insertion of multipass membrane proteins can be viewed as an alternation of insertion of a type I helix and insertion of a type II helix, which is dictated by the TM helix sequences that alternatingly act as start-transfer signals (signal anchor sequences) and stop-transfer signals (stop-transfer anchor sequences).</p>
<p>In multipass proteins with N-terminus in the cytosol (type IV-A), the TM helix sequence that first emerges from the ribosome (the one closest to the N-terminus) functions in the same way as the start transfer sequence (signal-anchor sequence) of the type II protein, which interacts with SRP to target the protein to the translocon, opens the translocon channel, and allow growing polypeptide chain to pass through the channel. Unlike type II proteins, as the chain grows, the translocon will encounter another hydrophobic TM helix sequence, which then functions as the stop-transfer signal (stop-transfer anchor sequence) in the same way as in type I proteins: the channel is closed, and this helix is moved via the lateral gate to the bilayer, and the nascent chain continues to grow into the cytosol. If a third TM helix sequence is encountered, it will function as another type II-like start-transfer signal, but this time the insertion does not depend on SRP and SRP receptor. This cycle continues until translation completes.</p>
<p>The insertion of multipass proteins with N-terminus in the ER lumen (type IV-B) occurs in a similar manner. The only difference is the first TM helix sequence, which now functions in the same way as the start-transfer sequence of the type III protein, resulting in a closed translocon and the growing peptide to be synthesised in the cytosol. Then, the second, third, fourth TM helix sequences functions in the same way as the first, second, and third TM helix sequences in a type IV-A protein and so on.</p>
<div class="figure"><span id="fig:type-iv-a-b"></span>
<img src="../img/type-iv-a-b.png" alt="Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall." width="100%" />
<p class="caption">
Figure 3.1: Multipass membrane proteins have multiple TM helices which alternatingly function as start-transfer signals (signal-anchor sequences, SA) and stop-transfer signals (stop-transfer anchor sequences, STA). The distribution of charge usually follows the same rule as in type II and III proteins, so that in the final conformation, the cytosolic side has more positively charged residues overall.
</p>
</div>
</div>
<div id="insertases" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Insertases</h1>
<p>Some membrane proteins are inserted into the lipid bilayer via Sec-independent mechanisms, which are mediated by factors generally termed ‘insertases’. The bacterial (YidC), mitochondrial (Oxa1), and chrloroplast (Alb3) insertases are evolutionarily related, whereas the ER membrane contains 3 seemingly unrelated insertases: Get1/2 complex, the ER membrane-protein complex (EMC), and TMCO1.</p>
<div id="insertion-of-tail-anchored-proteins-by-the-get-system" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Insertion of Tail-anchored Proteins by the Get system</h2>
<p>Tail-anchored proteins also use an <span class="math inline">\(\alpha\)</span>-helix to traverse the membrane, but this helix is located near the C-terminus. Because there are only few, if any, residues succeeding the C-terminal helix, translation terminates while the helix have not yet emerged from the exit tunnel, so recognition by SRP is not possible, and its insertion into ER membrane depends on a post-translational pathway involving Get1, Get2, and Get3 proteins.</p>
<p>In this pathway, the protein is completely synthesised and released into the cytosol. ATP-bound Get3 binds to the hydrophobic C-terminal tail of the protein (facilitated by some other proteins), and then docks onto the dimeric Get1/Get2 receptor on the ER membrane. Accompanying the hydrolysis of ATP by Get3, the Get1/Get2 complex facilitates the insertion of the tail into the ER membrane. Finally, ATP displaces ADP and Get3 is released back to the cytosol.</p>
</div>
<div id="the-roles-of-yidc" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> The Roles of YidC</h2>
<p>YidC contains 5 TM helices arranged to form a partially hydrophilic groove that is open towards both the lipid bilayer and the cytosol (Figure <a href="#fig:yidc">4.1</a>). Cross-linking experiments suggest that this groove operates as a binding site for TM helices (of the protein to be inserted). YidC alone can mediate insertion of a small subset of small (single- or double-pass) proteins in either co-translationally or post-translationally, and it is also suggested that YidC may act as a chaperone in conjunction with the Sec translocon. Specifically, it may shield the hydrophilic surface of TM helice that line polar cavities/channels in the final structure.</p>

<div class="figure"><span id="fig:yidc"></span>
<img src="../img/yidc.png" alt="The structure of Bacillus halodurans Yidc." width="100%" />
<p class="caption">
Figure 4.1: The structure of <em>Bacillus halodurans</em> Yidc.
</p>
</div>
</div>
</div>
<div id="beta-barrel-proteins" class="section level1" number="5">
<h1><span class="header-section-number">5</span> <span class="math inline">\(\beta\)</span>-barrel Proteins</h1>
<p>In Gram-negative bacteria, intergral OM (outer membrane) <span class="math inline">\(\beta\)</span>-barrel proteins (OMPs) are translocated in an unfolded form across the IM (inner membrane), ferried to the OM via periplasmic chaperones, and integrated into the OM by the <span class="math inline">\(\beta\)</span>-barrel assembly machine (Bam) multiprotein complex.</p>
<p>In mitochondria, the precursors of nucleus-encoded <span class="math inline">\(\beta\)</span>-barrel proteins are transferred from the TOM complex, with the help of chaperones, to the SAM (sorting and assembly machinery) complex. The SAM complex, being homologous to the Bam complex, mediates the assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins in a similar manner (Figure <a href="#fig:insert-beta-barrel">5.1</a>).</p>

<div class="figure"><span id="fig:insert-beta-barrel"></span>
<img src="../img/insert-beta-barrel.png" alt="Insertion of \(\beta\)-barrel proteins into bacterial or mitochondrial outer membrane." width="100%" />
<p class="caption">
Figure 5.1: Insertion of <span class="math inline">\(\beta\)</span>-barrel proteins into bacterial or mitochondrial outer membrane.
</p>
</div>
<p>The exact mechanism of assembly and insertion is not fully understood, but a recent proposed mechanism is shown in Figure <a href="#fig:insert-beta-barrel-mech">5.2</a> <span class="citation">(Ricci and Silhavy 2019)</span>.</p>

<div class="figure"><span id="fig:insert-beta-barrel-mech"></span>
<img src="../img/insert-beta-barrel-mech.png" alt="Proposed Bam-assisted assembly and insertion of \(\beta\)-barrel proteins. (a) Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). (b) The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the \(\beta\)-1 and \(\beta\)-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. (c) The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, (d) Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. (e) The folded client protein is released into the membrane, and the Bam complex restores to its idle state." width="100%" />
<p class="caption">
Figure 5.2: <strong>Proposed Bam-assisted assembly and insertion of <span class="math inline">\(\beta\)</span>-barrel proteins.</strong> <strong>(a)</strong> Unfolded precursor protein (red) stablised by chaperons (green) are transferred to the Bam complex (blue). <strong>(b)</strong> The five periplasmic POTRA domains of BamA is thought to nucleate the early formation of OMP secondary structure and, together with Bam lipoproteins, form a ring that act as a scaffold for the client protein. The hydrogen bonds between the <span class="math inline">\(\beta\)</span>-1 and <span class="math inline">\(\beta\)</span>-16 strands of BamA are metastable, allowing transient lateral opening of the barrel through this seam. <strong>(c)</strong> The Bam complex prevents aggregation, protects the client protein from proteolysis, and lowers the kinetic barrier to OM integration to accelerate OMP folding along the native pathway, <strong>(d)</strong> Bam destabilises the membrane locally, which facilitates the insertion of the client protein into the membrane. <strong>(e)</strong> The folded client protein is released into the membrane, and the Bam complex restores to its idle state.
</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Cymer-2015">
<p>Cymer, Florian, Gunnar von Heijne, and Stephen H. White. 2015. “Mechanisms of Integral Membrane Protein Insertion and Folding.” <em>Journal of Molecular Biology</em> 427 (5): 999–1022. <a href="https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014">https://doi.org/https://doi.org/10.1016/j.jmb.2014.09.014</a>.</p>
</div>
<div id="ref-Guna-2018">
<p>Guna, Alina, and Ramanujan S. Hegde. n.d. “Transmembrane Domain Recognition During Membrane Protein Biogenesis and Quality Control.” <em>Current Biology</em> 28 (8): R498–R511. <a href="https://doi.org/10.1016/j.cub.2018.02.004">https://doi.org/10.1016/j.cub.2018.02.004</a>.</p>
</div>
<div id="ref-Lodish-2016">
<p>Lodish, Harvey F., Arnold Berk, Chris Kaiser, Monty Krieger, Anthony Bretscher, Hidde L. Ploegh, Angelika Amon, and Kelsey C Martin. 2016. <em>Molecular Cell Biology</em>. Book. 8th ed. New York: W.H. Freeman.</p>
</div>
<div id="ref-Ricci-2019">
<p>Ricci, Dante P., and Thomas J. Silhavy. 2019. “Outer Membrane Protein Insertion by the β-Barrel Assembly Machine.” In <em>Protein Secretion in Bacteria</em>, 91–101. American Society of Microbiology. <a href="https://www.asmscience.org/content/book/10.1128/9781683670285.chap8">https://www.asmscience.org/content/book/10.1128/9781683670285.chap8</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/protein-nucleic-acid-interactions.html" class="archive-item-link">
                Protein-Nucleic Acid Interactions 
              </a>
              <span class="archive-item-date">
                2020-11-27
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#an-overview-of-mechanisms-of-protein-dna-interactions"><span class="toc-section-number">2</span> An Overview of Mechanisms of Protein-DNA Interactions</a></li>
<li><a href="#thermodynamics-of-protein-dna-binding"><span class="toc-section-number">3</span> Thermodynamics of Protein-DNA Binding</a></li>
<li><a href="#base-readout-in-the-major-groove"><span class="toc-section-number">4</span> Base Readout in the Major Groove</a></li>
<li><a href="#sr"><span class="toc-section-number">5</span> Shape Readout</a>
<ul>
<li><a href="#ele"><span class="toc-section-number">5.1</span> Minor Groove Narrowing</a></li>
<li><a href="#kinks"><span class="toc-section-number">5.2</span> Kinks</a></li>
</ul></li>
<li><a href="#the-nucleosome-as-an-example-of-non-specific-binding"><span class="toc-section-number">6</span> The Nucleosome as an Example of Non-Specific Binding</a></li>
<li><a href="#question-2"><span class="toc-section-number">7</span> (Question 2)</a>
<ul>
<li><a href="#a-bhlh-motif"><span class="toc-section-number">7.1</span> (a) bHLH Motif</a></li>
<li><a href="#b-leucine-zipper"><span class="toc-section-number">7.2</span> (b) Leucine Zipper</a></li>
<li><a href="#c-zinc-finger"><span class="toc-section-number">7.3</span> (c) Zinc Finger</a></li>
<li><a href="#d-tata-box-binding-protein"><span class="toc-section-number">7.4</span> (d) TATA box binding protein</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Protein-DNA interactions have profound impact on the expression and regulation of genes. Many DNA-binding proteins, especially transcription factors, are able to recognise and bind to a specific short sequence of DNA, while others, typically those involved in routine tasks such as DNA damage repair (e.g. glycosylase) and DNA packaging (e.g. histone) show less specificity.</p>
</div>
<div id="an-overview-of-mechanisms-of-protein-dna-interactions" class="section level1" number="2">
<h1><span class="header-section-number">2</span> An Overview of Mechanisms of Protein-DNA Interactions</h1>
<p>Binding of proteins to DNA results from the favourable interactions between short DNA sequences and the amino acid side chains of specific DNA-binding motifs in proteins. In many DNA-binding proteins, the affinity and specificity of binding is also enhanced by flexible segments outside the globular core that mediate specific and nonspecific interactions but are sometimes not included in the definition of DNA binding domains. The majority of DNA-binding domains are made of either <span class="math inline">\(\alpha\)</span>-helices, <span class="math inline">\(\beta\)</span>-sheets, or both. While they can be classified into different types of motifs, any one type of motif can be used in multiple ways to interact with DNA, depending on the protein and the binding site.</p>
<p>There are multiple mechanisms by which specificity is achieved. First, specific DNA base pairs and amino acid side chains can establish shape complementarity, resulting in hydrogen bonds or favourble polar, electrostatic, or hydrophobic interactions. This is commonly referred to as <em>direct readout</em> or <em>base readout</em>. These interactions usually more frequently in the the more accessible major groove of DNA than in the minor groove. However, there is no simple one-to-one relationship between DNA and protein sequences, which means this mechanism itself is not sufficient to accout for the specificity. The second mechanism, <em>shape readout</em>, involves recognition of a sequence-dependent DNA shape, for example an overall bend, a segment of narrow minor groove, or a kinked base pair step.</p>
<p>Base readout and shape readout can be further devided into different modes. These mechanisms rarely exists in isolation, and protein-DNA recognition uses a continuum of mechanisms that depend on the structural features and flexibility of both DNA and protein.</p>
</div>
<div id="thermodynamics-of-protein-dna-binding" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Thermodynamics of Protein-DNA Binding</h1>
<p>All protein-DNA complexes can gain favourable <span class="math inline">\(\Delta H\)</span> from direct protein-DNA recognition interactions and favourable <span class="math inline">\(\Delta S\)</span> from water release. However, in systems with strongly distorted DNA, the loss of favourable interaction due to the molecular strain of base pair destacking is so large so that the net <span class="math inline">\(\Delta H\)</span> becomes unfavourable. This is compensated by a more positive <span class="math inline">\(\Delta S\)</span> as the strained interface experiences less immobilisation (Figure <a href="#fig:pnipni-thermodynamics">3.1</a>).</p>

<div class="figure"><span id="fig:pnipni-thermodynamics"></span>
<img src="../img/pnipni-thermodynamics.png" alt="The relative enthalphic and entropic contributions to binding free energy (\(\Delta G_\text{free}\)) of a few site-specific DNA-binding proteins with similar \(\Delta G_\text{free}\) (about -11.7 kcal/mol). Taken from Jen-Jacobson, Engler, and Jacobson (2000)" width="100%" />
<p class="caption">
Figure 3.1: The relative enthalphic and entropic contributions to binding free energy (<span class="math inline">\(\Delta G_\text{free}\)</span>) of a few site-specific DNA-binding proteins with similar <span class="math inline">\(\Delta G_\text{free}\)</span> (about -11.7 kcal/mol). Taken from <span class="citation">Jen-Jacobson, Engler, and Jacobson (2000)</span>
</p>
</div>
</div>
<div id="base-readout-in-the-major-groove" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Base Readout in the Major Groove</h1>
<p>The major groove is an ideal site for forming specific interactions between DNA bases and amino acid side chains because the four possible base pairs have a unique pattern of hydrogen bond donors and acceptors (and the 5-methyl of thymine) in the major groove. Proteins can also form hydrogen bonds with bases in the minor groove, but this contributes much less to the specificity (because the hydrogen bonding pattern cannot distinguish between AT from TA or CG from GC) (<a href="#fig:pni-major-minor-base">4.1</a>). Many DNA-binding motifs, including HTH, zinc finger and Leucine zipper, forms hydrogen bonds in the major groove.</p>
<div class="figure"><span id="fig:pni-major-minor-base"></span>
<img src="../img/pni-major-minor-base.png" alt="Hydrogen bonding patterns in the major and minor groove of AT and CG base pairs." width="100%" />
<p class="caption">
Figure 4.1: Hydrogen bonding patterns in the major and minor groove of AT and CG base pairs.
</p>
</div>
<p>Specificity not only depends on the number of contacts formed but also on the uniqueness of the hydrogen bonding geometry. Bidentate hydrogen bonds (2 H-bonds with different donors and receptors, which can be formed, for example, between the two NH/NH<sub>2</sub> groups and the O and N of guanine) have the highest degree of specificity followed by bifurcated hydrogen bonds (two H-bonds share one donor) and single hydrogen bonds.</p>
<p>Water molecules can be found at some protein-DNA binding sites. While water molecules can allow flexibility and mediate non-specific binding, they are also present in many specific protein-DNA compelxes, such as Trp repressor-DNA, where their positions are highly ordered and serve to bridge the hydrogen bonds.</p>
</div>
<div id="sr" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Shape Readout</h1>
<p>In physiological conditions, most DNA exists in its B form with well-defined geometries. However, due to the subtle differences between the chemical properties of the four bases, the DNA shape varies in a sequence-dependent manner, and this structural variation is important for protein-DNA recognition.</p>
<p>The B-DNA conformation is largely stabilised by the stacking energy between adjacent base pairs. Thus regions with weak stacking energy, which were found to be AT-AT and AT-TA stacks, have a stronger deformability (i.e. propensity to deviate from the B-DNA conformation). Specifically, DNA sequences of at least four consecutive A-T base pairs without an intervening TpA step are called “A-tracts”, which not only improves deformability but also has implications on electrostatic potentials, which are described in Section <a href="#ele">5.1</a></p>
<p>The following table summarises the relationship between several sequence elements and their impact on the structural property of DNA:</p>
<table>
<colgroup>
<col width="27%" />
<col width="72%" />
</colgroup>
<thead>
<tr class="header">
<th>Sequence Element</th>
<th>Structural Property</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>A-tract</td>
<td>narrow minor groove, bending, rigid for <span class="math inline">\(\ge\)</span> bp</td>
</tr>
<tr class="even">
<td>TATA box</td>
<td>high deformability</td>
</tr>
<tr class="odd">
<td>YpR step (especially TpA)</td>
<td>compresses major groove, high deformability, ‘hinge’ step, kinking</td>
</tr>
<tr class="even">
<td>RpY step</td>
<td>compresses minor groove, low deformability</td>
</tr>
</tbody>
</table>
<div id="ele" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Minor Groove Narrowing</h2>
<p><span class="citation">Rohs et al. (2009)</span> reported that the binding of arginine residues to narrow (<span class="math inline">\(&lt; 5\)</span> angstroms) minor grooves is a common mode of protein-DNA recognition (Figure <a href="#fig:pni-narrow-minor">5.1</a>). These narrow groove are usually associates with A-tracts, which estabilish a connection between DNA sequence and shape. Arginines often insert into the minor groove as part of short sequence motifs that vary among different proteins, e.g. Arg-Gln-Arg in the Hox protein SCR, Arg-Lys-Lys-Arg in POU homeodomains, thus providing specificity.</p>
<p>The narrowing results in a more negative electrostatic potential in the minor groove, which promotes the binding of positively charged arginine. The preference of arginine over lysine can be explained by the greater energetic cost of removing a charged lysine from water due to lysine’s smaller radius of the charged group (or greater charge density) compared to arginine.</p>

<div class="figure"><span id="fig:pni-narrow-minor"></span>
<img src="../img/pni-narrow-minor.png" alt="Amino acid frequencies in minor grooves. Taken from Rohs et al. (2009)" width="100%" />
<p class="caption">
Figure 5.1: Amino acid frequencies in minor grooves. Taken from <span class="citation">Rohs et al. (2009)</span>
</p>
</div>
</div>
<div id="kinks" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Kinks</h2>
<p>YpR steps (especially TpA steps) have a strong tendency to form kinks that disrupt the linearity of the double helix. They can contribute to binding specificity by optimising protein-DNA contacts. The binding site of the catabolite activator protein (CAP), for example, shows dramatic kinks at two CpA steps, which along with two additional smaller kinks cause an overall bending of the DNA of about 90 degrees around the protein.</p>
</div>
</div>
<div id="the-nucleosome-as-an-example-of-non-specific-binding" class="section level1" number="6">
<h1><span class="header-section-number">6</span> The Nucleosome as an Example of Non-Specific Binding</h1>
<p>Histones are the ubiquitous nuclear protein that mainly serve to compact DNA. A nucleosome is formed by 147 base pairs of DNA wrapping around a histone octamer, and adjacent nucleosomes are joined by a linker sequence (about 50 bp long in humans). Since nucleosome are so prevalent, the histone-DNA interaction is often considered as non-specific, and its association with DNA is thought to be due to the electrostatic attraction between the abundant lysine and arginine residues on the surface of histoens and the negatively charged phosphate backbone of DNA. However, histones have preferred DNA regions to bind to.</p>
<p>Nucleosomes are often formed in regions enriched with A-tracts for two reasons. First, A-tracts offer high deformability, which is required to bend DNA and wrap it around histone octamers. Second, histones are enriched with arginine residues, which recognise and penetrateinto the narrow minor grooves formed by A-tracts, thus providing important stabilising interaction (Section <a href="#ele">5.1</a>).</p>
</div>
<div id="question-2" class="section level1" number="7">
<h1><span class="header-section-number">7</span> (Question 2)</h1>
<div id="a-bhlh-motif" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> (a) bHLH Motif</h2>
<p>The basic helix-loop-helix (bHLH) motif consists of two <span class="math inline">\(\alpha\)</span>-helices separated by a loop. One of the helices (usually the longer one) contain basic residues (arginine and lysine) that bind to the major groove of DNA. The flexible loop and the other helix are involved in dimerisation. Many bHTH proteins occur as heterodimers (e.g. Myc/MAX), and their activity is hightly regulated by the dimerisation of subunits.</p>
</div>
<div id="b-leucine-zipper" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> (b) Leucine Zipper</h2>
<p>The leucine zipper motif consists of a dimer of <span class="math inline">\(\alpha\)</span>-helices. The dimerisation is driven by interactions between the hydrophobic side chains that cover their inner surfaces. Specifically, each helix has a periodic repetition of leucine residues at every seventh position. Since every turn in an <span class="math inline">\(\alpha\)</span>-helix contains 3.6 amino acids—two turns contain 7.2, which is slightly more than 7, the helices coil around one another in a left-handed sense. The N-terminal DNA-binding domains of each helix protrudes into the major groove of DNA, which together recognise a 8-bp long sequence.</p>
</div>
<div id="c-zinc-finger" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> (c) Zinc Finger</h2>
<p>The zinc finger contains a short <span class="math inline">\(\alpha\)</span>-helix, a two-stranded antiparallel <span class="math inline">\(\beta\)</span>-sheet, and a Zn<sup>2+</sup> ion coordinated by cysteine and histine residues. The zinc ion serves to stabilise the overall structure, while the helix make contact with DNA’s major groove. A typical zinc finger protein, e.g. Zif268, contains a chain of 3 zinc-finger modules that coil in a right-handed sense, so as to follow the curve within the major groove. Each finger recognises 3 base pairs, and thus a protein with 3 zinc finger domains recognise a continuous sequence of 9 base pairs.</p>
</div>
<div id="d-tata-box-binding-protein" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> (d) TATA box binding protein</h2>
<p>TATA box binding proteins (TBP) use a ten-stranded <span class="math inline">\(\beta\)</span>-sheet to recognise DNA by binding in the minor groove. Insertion of the concave <span class="math inline">\(\beta\)</span>-sheet into the groove requires substantial DNA distortion. The flexibility intrinsic to TpA steps (Section <a href="#sr">5</a>) in the TATA sequence faciliates formation of kinks when TBP binds. In addition, yeast TBP-TATA structure shows that the kinks in the first and last base pair step (<strong>TA</strong>TATA<strong>AA</strong>) are stabilised through intercalations with phenylalanine residues.</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Calladine-2004">
<p>Calladine, Chris R., Horace Drew, Ben Luisi, and Andrew Travers. 2004. <em>Understanding Dna-the Molecule and How It Works</em>. 3rd ed. Academic Press. <a href="http://gen.lib.rus.ec/book/index.php?md5=c6b8d89cd8868255bc9ebd4cf51e1917">http://gen.lib.rus.ec/book/index.php?md5=c6b8d89cd8868255bc9ebd4cf51e1917</a>.</p>
</div>
<div id="ref-Jen-Jacobson-2000">
<p>Jen-Jacobson, Linda, Lisa E. Engler, and Lewis A. Jacobson. 2000. “Structural and Thermodynamic Strategies for Site-Specific Dna Binding Proteins.” <em>Structure</em> 8 (10): 1015–23. <a href="https://doi.org/https://doi.org/10.1016/S0969-2126(00)00501-3">https://doi.org/https://doi.org/10.1016/S0969-2126(00)00501-3</a>.</p>
</div>
<div id="ref-Rohs-2009">
<p>Rohs, Remo, Sean M. West, Alona Sosinsky, Peng Liu, Richard S. Mann, and Barry Honig. 2009. “The Role of Dna Shape in Protein–Dna Recognition.” <em>Nature</em> 461 (7268): 1248–53. <a href="https://doi.org/10.1038/nature08473">https://doi.org/10.1038/nature08473</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/protein-protein-interactions.html" class="archive-item-link">
                Protein-Protein Interactions 
              </a>
              <span class="archive-item-date">
                2020-11-18
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#properties-of-ppi"><span class="toc-section-number">2</span> Properties of PPI</a>
<ul>
<li><a href="#reversibility"><span class="toc-section-number">2.1</span> Reversibility</a></li>
<li><a href="#prop"><span class="toc-section-number">2.2</span> Properties of the Binding Interfaces</a></li>
<li><a href="#roles-of-water-molecules"><span class="toc-section-number">2.3</span> Roles of Water Molecules</a></li>
<li><a href="#kd"><span class="toc-section-number">2.4</span> Kinetics and Thermodynamics</a></li>
</ul></li>
<li><a href="#studying-ppis"><span class="toc-section-number">3</span> Studying PPIs</a>
<ul>
<li><a href="#determine-kd"><span class="toc-section-number">3.1</span> Determining K<sub>d</sub> (and rate constants)</a>
<ul>
<li><a href="#surface-plasmon-resonance-spr"><span class="toc-section-number">3.1.1</span> Surface Plasmon resonance (SPR)</a></li>
<li><a href="#fluorescence-anisotropy"><span class="toc-section-number">3.1.2</span> Fluorescence anisotropy</a></li>
<li><a href="#isothermal-titration-calorimetry-itc"><span class="toc-section-number">3.1.3</span> Isothermal titration calorimetry (ITC)</a></li>
</ul></li>
<li><a href="#mechanistic-studies"><span class="toc-section-number">3.2</span> Mechanistic Studies</a>
<ul>
<li><a href="#alanine-scanning"><span class="toc-section-number">3.2.1</span> Alanine Scanning</a></li>
<li><a href="#ext"><span class="toc-section-number">3.2.2</span> Extent of Exchengeability of Amino Acids</a></li>
<li><a href="#directed-evolution"><span class="toc-section-number">3.2.3</span> Directed Evolution</a></li>
<li><a href="#comp"><span class="toc-section-number">3.2.4</span> Computational Approaches</a></li>
<li><a href="#connectivity-map-reveals-modularity"><span class="toc-section-number">3.2.5</span> Connectivity Map Reveals Modularity</a></li>
<li><a href="#x-ray-crystallography"><span class="toc-section-number">3.2.6</span> X-Ray Crystallography</a></li>
</ul></li>
</ul></li>
<li><a href="#why-designing-small-molecule-ppi-inhibitors-is-difficult"><span class="toc-section-number">4</span> Why Designing Small Molecule PPI Inhibitors is Difficult</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>How and why do proteins form specific complexes with each other? How can such protein-protein interactions (PPIs) be investigated experimentally, and which problems are associated with designing small molecules to disrupt PPIs?</strong></p>
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Specific protein-protein interactions (PPIs) are critical to numerous biological processes, including cell-cell recognition, immune response, and signal transduction. An understanding of PPIs not only helps to elucidate the detailed roles and to predict the behaviour of proteins in a physiological context but also aids structure-based drug design.</p>
</div>
<div id="properties-of-ppi" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Properties of PPI</h1>
<div id="reversibility" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Reversibility</h2>
<p>Protein-protein interactions can be stable (permanent) or transient. Stable interactions are involved in the assembly of proteins made of multi-subunit complexes such as haemoglobin, which non-reversible in normal physiological conditions. Transient interactions, on the other hand, are reversible, and it is this property that make them act like molecular switches that play versatile roles in controlling cellular processes.</p>
</div>
<div id="prop" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Properties of the Binding Interfaces</h2>
<p>Protein-protein interaction interfaces often have a large surface area (1000-2000 Å<sup>2</sup>) and are relatively flat compared to the deep cavities that typically bind small molecules.
On a binding interface, some residues, known as “hotspots”, contribute to the overall affinity more than other residues.</p>
</div>
<div id="roles-of-water-molecules" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Roles of Water Molecules</h2>
<p>Crystal structures frequently reveal water molecules within PPI interfaces. These water molecules play multifaceted roles in the stability of PPI, e.g. offsetting unfavourable electrostatic interactions, bridging two distant residues via H-bonds.</p>
</div>
<div id="kd" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Kinetics and Thermodynamics</h2>
<p>Reversible PPIs have two important parameters: affinity and specificity. While affinity ranges from as low as millimolar to as high as femtomolar, it is important that the specificity, i.e. the relative affinity of a protein to its cognate binding partner compared to non-cognate ones, is high.</p>
<p>Reversible PPIs can the considered as a simple balance of association and dissociation reactions, with rate constants being <span class="math inline">\(k_{\text{on}}\)</span> and <span class="math inline">\(k_{\text{off}}\)</span>.</p>
<p><span class="math display">\[
\text{A+B}\mathrel{\mathop{\rightleftarrows}^{k_{\text{on}}}_{k_{\text{off}}}} \text{AB}
\]</span></p>
<p>The affinity is usually defined by the dissociation constant:</p>
<p><span class="math display">\[
K_\text{d} = \dfrac{\text{[A][B]}}{\text{[AB]}} = \dfrac{k_{\text{off}}}{k_{\text{on}}}
\]</span></p>
<p>where [A], [B], and [AB] are the concentrations of each species at equilibrium.</p>
<p><span class="math inline">\(K_\text{d}\)</span> can be converted to <span class="math inline">\(\Delta G\)</span> and vice versa:</p>
<p><span class="math display">\[
\Delta G = -RT\ln{(K_\text{d})}
\]</span></p>
<p>In addition to the simple single-step model, <span class="citation">Keeble and Kleanthous (2005)</span> suggested that relatively low affinity PPIs may be better modelled with a two-step induced-fit mechanism involving an unstable intermediate, where electrostatics drives the fast first step (supported by strong dependence on ionic strength) and rigid body rotation occurs in the slow second step:</p>
<p><span class="math display">\[
\text{A+B} \mathrel{\mathop{\rightleftarrows}^{k_{1}}_{k_{-1}}} \text{AB}^\text{*} \mathrel{\mathop{\rightleftarrows}^{k_{2}}_{k_{-2}}} \text{AB}
\]</span></p>
</div>
</div>
<div id="studying-ppis" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Studying PPIs</h1>
<div id="determine-kd" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Determining K<sub>d</sub> (and rate constants)</h2>
<p>Many experimental methods can be used used for studying thermodynamics and kinetics of PPIs (the frequent tasks are determining <span class="math inline">\(K_\text{d}\)</span> and rate constants). Most of them assumes the simple single-step association-dissociation model (Section <a href="#kd">2.4</a>). Three methods are described in this essay.</p>
<div id="surface-plasmon-resonance-spr" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Surface Plasmon resonance (SPR)</h3>
<p>Surface plasmon resonance (Figure <a href="#fig:spr">3.1</a>) can be used to measure both <span class="math inline">\(K_\text{d}\)</span> and <span class="math inline">\(k_\text{on}\)</span> and <span class="math inline">\(k_\text{off}\)</span> rates. How <span class="math inline">\(K_\text{d}\)</span> can be calculated is explained in Equation <a href="#eq:spr-kd">(3.1)</a>.</p>

<div class="figure" style="text-align: c"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, \(\theta_\text{spr}\), is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. \(\Delta\theta_\text{spr}\) is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region). In an SPR experiment, the protein acting as the “bait” is immobilised on the sensor surface, and a the analyte containing the other protein (acting as the “prey”) is passed through the cell. If binding occurs between the two proteins, \(\Delta\theta_\text{spr}\) would increase. Then, non-specific binding is washed off by buffer, and \(\Delta\theta_\text{spr}\) would decrease and \(\Delta\theta_\text{spr}\). Finally, regeneration solution is applied to remove all binding and reset \(\Delta\theta_\text{spr}\) to zero." width="100%" />
<p class="caption">
Figure 3.1: A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, <span class="math inline">\(\theta_\text{spr}\)</span>, is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. <span class="math inline">\(\Delta\theta_\text{spr}\)</span> is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region). In an SPR experiment, the protein acting as the “bait” is immobilised on the sensor surface, and a the analyte containing the other protein (acting as the “prey”) is passed through the cell. If binding occurs between the two proteins, <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would increase. Then, non-specific binding is washed off by buffer, and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would decrease and <span class="math inline">\(\Delta\theta_\text{spr}\)</span>. Finally, regeneration solution is applied to remove all binding and reset <span class="math inline">\(\Delta\theta_\text{spr}\)</span> to zero.
</p>
</div>
<p><span class="math display" id="eq:spr-kd">\[\begin{equation}
\begin{split}
\overbrace{k_{\text{on}}\text{[A][B]}}^\text{rate of association} &amp; = \overbrace{k_{\text{off}}\text{[AB]}}^\text{rate of dissociation} \\
k_{\text{on}}\text{[A]([B]}_{\text{max}}-\text{[AB])} &amp; = k_{\text{off}}\text{[AB]} \\
\text{[AB]} &amp; = \dfrac{k_{\text{on}}\text{[A][B]}_{\text{max}}}{k_{\text{on}}\text{[A]} + k_{\text{off}}} \\
\text{[AB]} &amp; = \dfrac{\text{[A][B]}_{\text{max}}}{\text{[A]} + K_{\text{d}}}
\end{split}
\tag{3.1}
\end{equation}\]</span></p>
<p>In the equation, A is the protein in the analyte, whose concentration is kept constant, and B is the immobilised protein. Since the <span class="math inline">\(\Delta\theta_\text{spr} \propto \text{[AB]}\)</span> (intensity of the signal is proportional to concentration of protein-protein complexes), we can work out <span class="math inline">\(K_\text{d}\)</span> from our initial concentrations of A and B.</p>
<p>SPR was used in the early kinetic analysis of hGH binding (<span class="citation">Wells (1996)</span>).</p>
</div>
<div id="fluorescence-anisotropy" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Fluorescence anisotropy</h3>
<p>Fluorescence anisotropy is based on the phenomenon that, if fluorophores are excited with plane polarized light and the fluorescence is observed through analyzing polarizers, the fluorescence is also polarised.</p>
<p>The fluorescence anisotropy is defined as <span class="math inline">\(A=\dfrac{I_\parallel - I_\bot}{I_\parallel+2_{\bot}}\)</span>, where <span class="math inline">\(I_\parallel\)</span> and <span class="math inline">\(I_{\bot}\)</span> are the fluorescence intensities polarised parallel and perpendicular to the direction of the excitation beam. <span class="math inline">\(A\)</span> is a direct measure of the molecular rotation in solution and can be used to study complex formation, as a macromolecule will rotate more slowly when it is in a complex thatn when it is alone.</p>
<p>Fluorescence anisotropy is more accurate than SPR for measuring ultra-high affinity interactions, and were used to to study ColE DNase-Im interactions (<span class="citation">Papadakos, Wojdyla, and Kleanthous (2012)</span>).</p>
</div>
<div id="isothermal-titration-calorimetry-itc" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Isothermal titration calorimetry (ITC)</h3>
<p>ITC measures heat changes when a complex is formed at constant temperature. In ITC, an insulated reaction cell containing protein is kept at a temperature (usually 8<span class="math inline">\(^\circ\text{C}\)</span> above the environment) which is equal to the temperature of a reference cell, and the reference cell is kept at a constant temperature by a thermostat. Then, increasing amounts of ligand is added into the chamber, and they form complexes with the protein, which can be exothermic or endothermic. The heat change is compensated by a power supply, which can be converted to <span class="math inline">\(\Delta H\)</span> of the reaction. As more ligands are added, proteins become saturated and <span class="math inline">\(\Delta H\)</span> approaches zero. The raw data obtained (power supplied to compensate the heat change caused by each addition of ligands) can be integrated and corrected to give a plot of <span class="math inline">\(\Delta H\)</span> against the molar ratio of the ligand and the protein, and <span class="math inline">\(\Delta H\)</span>, K<sub>d</sub> and stoichiometry can be inferred from the curve (Figure <a href="#fig:itc">3.2</a>).</p>
<div class="figure" style="text-align: c"><span id="fig:itc"></span>
<img src="../img/itc.jpg" alt="ITC data manipulation" width="100%" />
<p class="caption">
Figure 3.2: ITC data manipulation
</p>
</div>
</div>
</div>
<div id="mechanistic-studies" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Mechanistic Studies</h2>
<p>The main theme of mechanistic studies is to find out the “hotspot” residues or regions that are main contributors to the affnity of PPI interfaces, and to attempt to generalise this knowledge in order to predict the affnity of any given PPI interfaces.</p>
<div id="alanine-scanning" class="section level3" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Alanine Scanning</h3>
<p>Alanine scanning is a mutagenesis technique in which mutants are made by substituting alanine for each of the residues in a ‘reactive region’, in this case the PPI interface. By comparing the PPI affininy of each mutant to the wild type (i.e. calculating <span class="math inline">\(\Delta \Delta G\)</span>), the contribution of each residue in binding can be assessed, and this reveals “hotspots” representing important residues.</p>
</div>
<div id="ext" class="section level3" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> Extent of Exchengeability of Amino Acids</h3>
<p>Mutational analysis is most often restricted to alanine substitution and this does not provide an comprehensive view of the allowed amino acid space at each position. This limitation is especially significant in the analysis of PPI interfaces, which, unlike enzyme active sites, the specific orientation and chemical reactivity are less important.</p>
<p><span class="citation">Pál et al. (2006)</span> introduced any one of the 20 natrual amino acids at all 35 hGH-hGHR binding interface positions and obtained surprising results. They verified that, the interface was highly adaptable to mutations, either from a structural or functional point of view. Whereas some of the alanine scanning hotspots showed high specificity agianst substitution, others did not, and some highly specific positions were not hotspots at all.</p>
</div>
<div id="directed-evolution" class="section level3" number="3.2.3">
<h3><span class="header-section-number">3.2.3</span> Directed Evolution</h3>
<p>Directed evolution is an efficient approach to probe sequence and structure space in a PPI. Phage display is a specific implementation of it.</p>
<p>The phage display technique is summarised as follows and illustrated in Figure <a href="#fig:phage-sel">3.3</a>:</p>
<ol style="list-style-type: decimal">
<li>Generate a randomised library of mutants. This can be done using error-prone PCR (ep-PCR). In PPI studies usually the randomised mutations are generated only across small sections of protein sequence that are form of the PPI interface.</li>
<li>The randomised mutant DNA are ligated with a phage coat protein gene and the hybrid is then used to transduce <em>E. coli.</em> cells.</li>
<li>The phage library is amplified by <em>E. coli.</em> cells, and phage particles are produced.</li>
<li>The “bait” protein is immobilised, and the mixture of phage particles displaying different mutant proteins are added. Then, those with low binding affinity are washes away, and the remaining phages and collected, amplified, and used in the next round of selection.</li>
<li>Repeated cycles of selection (a.k.a. “panning”) will identify the mutant proteins with the highest binding affinity to the “bait” protein. The complexes formed by these proteins with the “bait” protein can then be used for detailed mechanistic studies of PPI.</li>
</ol>
<div class="figure" style="text-align: c"><span id="fig:phage-sel"></span>
<img src="../img/phage-sel.png" alt="Directed evolution with phage display." width="100%" />
<p class="caption">
Figure 3.3: Directed evolution with phage display.
</p>
</div>
</div>
<div id="comp" class="section level3" number="3.2.4">
<h3><span class="header-section-number">3.2.4</span> Computational Approaches</h3>
<p>Computational methods can speed up the quest for high-affinity PPIs. These methods are based on rotamer libraries, which summarise the existing knowledge of the experimentally determined structures quantitatively. Rotamers are picked from the library (hotspot constraints may be applied) and grafted onto the scaffold of known structure, then the fitness is assessed using a scoring function, which unfortunately often fails to predict the actual experimental results. As the number rotamers in the library grows and algorithms improve, computational methods are expected to provide more accurate predictions.</p>
</div>
<div id="connectivity-map-reveals-modularity" class="section level3" number="3.2.5">
<h3><span class="header-section-number">3.2.5</span> Connectivity Map Reveals Modularity</h3>
<p><span class="citation">Reichmann et al. (2005)</span> analysed the TEM1–BLIP complex by drawing a connectivity map, which is build from the physical interactions between the proteins (hydrogen bonds, van der Waals interactions, etc.), and showed that the interface can be divided into 6 clusters.
The change <span class="math inline">\(\Delta \Delta G\)</span> on different clusters was found to be additive, whereas mutations within the same cluster caused complex energetic and structural consequences.
Therefore, a PPI interface can be seen as a group of “hot regions”, where each region contribute relatively independently to the total binding affinity, but within each region the contributions from its component amino acids are cooperative.</p>
<div class="figure" style="text-align: c"><span id="fig:ppi"></span>
<img src="../img/ppi.png" alt="Connectivity map of the TEM1–BLIP complex." width="100%" />
<p class="caption">
Figure 3.4: Connectivity map of the TEM1–BLIP complex.
</p>
</div>
</div>
<div id="x-ray-crystallography" class="section level3" number="3.2.6">
<h3><span class="header-section-number">3.2.6</span> X-Ray Crystallography</h3>
<p>X-ray crystallography (XRC) studies provide structural basis for PPI interfaces, which not only facilitates the analysis of individual cases but also help to improve the scoring algorithms for computational methods.</p>
<p>Crystallisation tends to be difficult for low affnity PPIs due to their unstable nature. To overcome this, correctly positioned cysteines can be introduced into each of the binding partners at the interface, which could stabilise the complex, facilitating crystallisation. This technique is known as “disulfide trapping”.</p>
</div>
</div>
</div>
<div id="why-designing-small-molecule-ppi-inhibitors-is-difficult" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Why Designing Small Molecule PPI Inhibitors is Difficult</h1>
<p>PPI interfaces are usually large and flat (Section <a href="#prop">2.2</a>), and often tolerent to a small number of mutations (Section <a href="#ext">3.2.2</a>). Thus, the “druggable” targets are usually restricted to the local non-flat regions (i.e. ‘pockets’) on PPI interfaces that are enriched with “hotspot” residues. A large surface area of PPI interface, which is more tolerent to local non-favourable interactions, also makes developing inhibitors more difficult. In addition, in the case of developing non-peptide inhibitors, there is less existing knowledge on the structures of small molecules and their interactions with proteins, meaning the computational modelling is less accurate.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Arkin-2014">
<p>Arkin, Michelle R, Yinyan Tang, and James A Wells. 2014. “Small-Molecule Inhibitors of Protein-Protein Interactions: Progressing Toward the Reality.” <em>Chem Biol</em> 21 (9): 1102–14. <a href="https://doi.org/10.1016/j.chembiol.2014.09.001">https://doi.org/10.1016/j.chembiol.2014.09.001</a>.</p>
</div>
<div id="ref-Keeble-2005">
<p>Keeble, Anthony H, and Colin Kleanthous. 2005. “The Kinetic Basis for Dual Recognition in Colicin Endonuclease-Immunity Protein Complexes.” <em>J Mol Biol</em> 352 (3): 656–71. <a href="https://doi.org/10.1016/j.jmb.2005.07.035">https://doi.org/10.1016/j.jmb.2005.07.035</a>.</p>
</div>
<div id="ref-Papadakos-2012">
<p>Papadakos, Grigorios, Justyna A. Wojdyla, and Colin Kleanthous. 2012. “Nuclease Colicins and Their Immunity Proteins.” <em>Quarterly Reviews of Biophysics</em> 45 (1): 57–103. <a href="https://doi.org/10.1017/S0033583511000114">https://doi.org/10.1017/S0033583511000114</a>.</p>
</div>
<div id="ref-Pal-2006">
<p>Pál, Gábor, Jean-Louis K Kouadio, Dean R Artis, Anthony A Kossiakoff, and Sachdev S Sidhu. 2006. “Comprehensive and Quantitative Mapping of Energy Landscapes for Protein-Protein Interactions by Rapid Combinatorial Scanning.” <em>J Biol Chem</em> 281 (31): 22378–85. <a href="https://doi.org/10.1074/jbc.M603826200">https://doi.org/10.1074/jbc.M603826200</a>.</p>
</div>
<div id="ref-Reichmann-2005">
<p>Reichmann, D, O Rahat, S Albeck, R Meged, O Dym, and G Schreiber. 2005. “The Modular Architecture of Protein-Protein Binding Interfaces.” <em>Proc Natl Acad Sci U S A</em> 102 (1): 57–62. <a href="https://doi.org/10.1073/pnas.0407280102">https://doi.org/10.1073/pnas.0407280102</a>.</p>
</div>
<div id="ref-Wells-1996">
<p>Wells, J A. 1996. “Binding in the Growth Hormone Receptor Complex.” <em>Proceedings of the National Academy of Sciences</em> 93 (1): 1–6. <a href="https://doi.org/10.1073/pnas.93.1.1">https://doi.org/10.1073/pnas.93.1.1</a>.</p>
</div>
<div id="ref-Wienken-2010">
<p>Wienken, Christoph J., Philipp Baaske, Ulrich Rothbauer, Dieter Braun, and Stefan Duhr. 2010. “Protein-Binding Assays in Biological Liquids Using Microscale Thermophoresis.” <em>Nature Communications</em> 1 (1): 100. <a href="https://doi.org/10.1038/ncomms1093">https://doi.org/10.1038/ncomms1093</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/membrane-protein-structure-and-stability.html" class="archive-item-link">
                Membrane Protein: Structure and Stability 
              </a>
              <span class="archive-item-date">
                2020-11-12
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#the-biomembrane-environment"><span class="toc-section-number">2</span> The Biomembrane Environment</a></li>
<li><a href="#the-thermodynamic-basis-of-secondary-structure-formation"><span class="toc-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</a></li>
<li><a href="#glycine-and-proline-in-tm-helices"><span class="toc-section-number">4</span> Glycine and Proline in TM Helices</a></li>
<li><a href="#snorkel"><span class="toc-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</a></li>
<li><a href="#interactions-with-lipids"><span class="toc-section-number">6</span> Interactions with Lipids</a></li>
</ul>
</div>

<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Many proteins that perform vital functions are membrane proteins. These include transport proteins (channels, transporters and pumps), cell adhesion molecules, and proteins that transduces energy in the electron transport chain. Despite their diverse roles, the same set of biophysical and biochemical rules govern their structural stability. Membrane proteins fall into two broad categories: 1) integeral proteins, which are embedded within a membrane and can only be isolated with detergents, and 2) peripheral proteins, which are associated with the surface of a membrane and can be removed without detergents. The focus of this essay is on how integral proteins interact with the membrane to maintain their structural stability.</p>
</div>
<div id="the-biomembrane-environment" class="section level1" number="2">
<h1><span class="header-section-number">2</span> The Biomembrane Environment</h1>
<p>The basic structure of a biomembrane is a bilayer of amphipathic phospholipids and sphingolipids, where the polar head groups of lipids align the surfaces (each ~10 angstroms thick) and the acyl chains of lipid tails occupy the hydrophobic core (~ 30 angstroms thick). A bitopic or polytopic integral membrane protein spans the membrane entirely one or more times (Figure <a href="#fig:lipid-bilayer-diagram">2.1</a>).</p>
<div class="figure"><span id="fig:lipid-bilayer-diagram"></span>
<img src="../img/lipid-bilayer-diagram.png" alt="(a) a schematic showing a integral protein embedded within a biomembrane. (b)" width="100%" />
<p class="caption">
Figure 2.1: (a) a schematic showing a integral protein embedded within a biomembrane. (b)
</p>
</div>
<p>The hydrophobic core region has a dielectric constant (<span class="math inline">\(\epsilon_{r}\)</span>) of about 2, which is much lower than <span class="math inline">\(\epsilon_{r}\)</span> of water (about 80), which means the same pair of charges separated by the same distance experience more electrostatic force in the membrane core than in water, according to the Coulomb’s law:</p>
<p><span class="math display">\[F = \dfrac{q_1 q_2}{4\pi \epsilon_0 \epsilon_r r^2}\]</span></p>
<p>This has several implications on acidic and basic amino acid residues. First, the carboxyl group of acidic side chains are more difficult to dissociate (i.e. p<span class="math inline">\(K_a\)</span>s are shifted up), meaning they tend to remain in the uncharged (–COOH) form. Second, in <span class="math inline">\(\alpha\)</span>-helical bundles, oppositely charged residues in adjacent helices associated more strongly than they do in aqueous environment. Third, this facilitates the snorkelling of lysine and arginine residues near the interface (Section <a href="#snorkel">5</a>).</p>
</div>
<div id="the-thermodynamic-basis-of-secondary-structure-formation" class="section level1" number="3">
<h1><span class="header-section-number">3</span> The Thermodynamic Basis of Secondary Structure Formation</h1>
<p>Transmembrane proteins tend to adopt a conformation that minimises the contact between their polar groups (main chain -NH, C=O, and side chains of polar residues, if any) and the hydrophobic core. This can be explained by simple thermodynamic reasoning.
The water molecules in the aqueous environment outside the membrane are capable of forming relatively strong dipole-dipole interactions and hydrogen bonds (H-bonds) with polar (including charged) amino acids, while the non-polar alkyl groups that occupy the hydrophobic core of the membrane can only provide weak van der Waals interactions.
Thus, if a protein adopts a conformation that exposes many polar groups, the loss of strong interactions with water will make this process very energetically unfavourable (very large <span class="math inline">\(\Delta G\)</span>).
On the contrary, if a protein with a hydrophobic surface were present in an aqueous environment, it would disrupt the dipole-dipole interactions and hydrogen bonding among water and other polar molecules. Thus, its insertion into the membrane is energetically favourable.</p>
<p>By forming either an <span class="math inline">\(\alpha\)</span>-helix or a <span class="math inline">\(\beta\)</span>-sheet (Figure <a href="#fig:bundle-helix">3.1</a>), the hydrogen bonds between main chain -NH and -C=O are maximised, and this is an efficient way to shield the polar groups on the main chain from the hydrophobic core.
Many transmembrane proteins, such as bacteriorhodopsin, contain multiple TM helices, which aggregates to form a <span class="math inline">\(\alpha\)</span>-helix bundle.
A <span class="math inline">\(\beta\)</span>-sheet has to twist and coil to form a ring, called a <span class="math inline">\(\beta\)</span>-barrel, in order to shield the main chain -NH and C=O groups on the first and last strands.
<span class="math inline">\(\beta\)</span>-barrels usually have a even number of strands so that every strand is anti-parallel to adjacent strands, there are exceptions: the voltage-dependent anion channels (VDACs) located on mitochondrial outer membrane has 19 strands, where the first strand is parallel with tha last strand, with weaker hydrogen bonding.
<span class="math inline">\(\beta\)</span>-barrels are less common than <span class="math inline">\(\alpha\)</span>-helix bundles, and they are predominantly present on the outer membrane of Gram-negative bacteria, mitochondria, and chloroplasts.</p>
<div class="figure"><span id="fig:bundle-helix"></span>
<img src="../img/bundle-helix.png" alt="Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively." width="100%" />
<p class="caption">
Figure 3.1: Schematic representation of an alpha helix and a beta barrel. Blue and red lines represent hydrophobic and hydrophilic side chains, respectively.
</p>
</div>
<p>The transmembrane region of each helix in a <span class="math inline">\(\alpha\)</span>-helical bundle is composed predominantly of hydrophobic amino acid residues (Ala, Leu, Ile, Val, Phe), which are exposed on the surface and allows favourable interaction with the hydrophobic core.
By contrast, each <span class="math inline">\(\beta\)</span>-strand of a <span class="math inline">\(\beta\)</span>-barrel has an alternating pattern of hydrophobic and polar residues.
This is because two adjacent residues in a beta strand have their side chains pointing in opposite directions, and a <span class="math inline">\(\beta\)</span>-barrel formed in this way has an hydrophobic exterior that interacts with the hydrophobic core and an hydrophilic inner surface that is in contact with the aqueous environment.</p>
<p>Thus, TM helices, but not <span class="math inline">\(\beta\)</span>-barrels, can be predicted using a hydropathy plot (Figure <a href="#fig:hydropathy-plot-1">3.2</a>), in which the average hydrophobicity index of a fixed number of consecutive residues (a “window”), <span class="math inline">\(H(i)\)</span>, is plotted against the index (<span class="math inline">\(i\)</span>) of the window, i.e.</p>
<p><span class="math display">\[H(i) = \sum_{i&lt;j&lt;i+k}h(a_j)/k\]</span></p>
<p>for <span class="math inline">\(1 &lt; i &lt; n - k\)</span>, where n is the length (number of residues) of the peptide, <span class="math inline">\(h(a_j)\)</span> is the hydrophobicity index of the <span class="math inline">\(j\)</span>-th amino acid residue and <span class="math inline">\(k\)</span> is the window size.</p>
<div class="figure"><span id="fig:hydropathy-plot-1"></span>
<img src="../img/hydropathy-plot-1.png" alt="A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix." width="100%" />
<p class="caption">
Figure 3.2: A schematic hydropathy plot. Every peak in the plot represents a highly hydrophobic local region in the peptide and thus indicates a potential transmembrane helix.
</p>
</div>
<p>A small number of polar (even charged) residues within a TM helix can be tolerated as long as the overall transmembrane segment is hydrophobic enough.
In addition, in polytopic proteins, polar residues in adjacent helices may help to stabilise each other.
These non-hydrophobic residues often have functional roles, as exemplified by voltage-gated K<sup>+</sup>, Na<sup>+</sup> and Ca<sup>2+</sup> channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
<div class="figure"><span id="fig:kv-mech"></span>
<img src="../img/kv-mech.png" alt="The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker." width="100%" />
<p class="caption">
Figure 3.3: The gating mechanism of voltage-gated potassium channel. The S4 helix (red) is arginine-rich and thus positively charged. A linker connects S4 and S5 helix, the latter being a part of the pore domain. In resting state, the interior (bottom) of the cell is negatively charged relative to the exterior (top). A change in voltage causes the S4 helix to move upwards due to electrostatic force, and it pulls the gate open via the S4-S5 linker.
</p>
</div>
</div>
<div id="glycine-and-proline-in-tm-helices" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Glycine and Proline in TM Helices</h1>
<p>The side chain of proline forms a pentameric ring with the amine group on the main chain. Thus, in a helix, the amine group of a proline does not have a hydrogen that can be H-bonded to the main chain C=O of the residue above it. This introduces local flexibility within a relatively regid helix and often forms a hinge. Similarly, due to glycine’s small size, it can tolerate a much wider range of dihedral angles than other amino acids, which also makes it able to introduce flexibility. This flexibility is crucial for the function of some proteins, such as voltage-gated ion channels (Figure <a href="#fig:kv-mech">3.3</a>).</p>
</div>
<div id="snorkel" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Tryptophan and Tyrosine at the Interface and Lysine/Arginine Snorkelling</h1>
<p>Tryptophan and tyrosine are often found in the interfacial region in both <span class="math inline">\(\alpha\)</span>-helical bundle and <span class="math inline">\(\beta\)</span>-barrel proteins. Their hydrophobic aromatic ring interacts with the hydrophobic core, and their polar -NH (of Trp) or -OH (of Tyr) groups interacts with the lipid head groups. Similarly, lysine and arginine extend their side chain towards the surface, so that their positively charged guanidium (of Arg) or amide (of Lys) groups form strong ionic interactions with the negatively charged lipid head groups. This phenomenon is known as “snorkelling.” These interactions helps to lock the membrane in place within the membrane, preventing vertical motions.</p>
</div>
<div id="interactions-with-lipids" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Interactions with Lipids</h1>
<p>Membrane proteins interacts with lipid in various ways, and these interactions contribute to the structural stability and function of the protein to different extents.</p>
<p>A transmembrane protein interacts strongly with the shell (annulus) of lipid that surrounds it. These lipids are called annular lipids and can be distinguished experimentally from the bulk lipids of the bilayer. In addition, there is a third class of lipids, called nonannular lipids (or lipid cofactors), which are tightly bound in crevices or between subunits of the protein and are often crutial for the activity of the protein. Electron paramagnetic resonance (EPR) can be used to measure the mobility of annular lipids, and thus gives information on the selectivity for annular lipids of a protein. Most proteins are found to prefer negatively charged lipids such as phosphatidylserine and phosphatidylinositol.</p>
<p>Strong lipid-protein interactions are also evident form crystal structures, and this interaction can be specific. For example, three cardiolipin (CDL) molecules were found to bind to three sites on the ANT1 transporter. The tendency of CDL to bind to these sites can also be shown in molecular dynamics simulations. The specificity of this binding can also be shown computationally, by calculating the free energy at different protein-lipid separations for CDL and other lipids (Figure <a href="#fig:simu-lipid-protein-binding">6.1</a>).</p>
<div class="figure"><span id="fig:simu-lipid-protein-binding"></span>
<img src="../img/simu-lipid-protein-binding.png" alt="Computational analysis of a protein's preference to bound lipids. CDL binding results in a significantly lower energy state." width="100%" />
<p class="caption">
Figure 6.1: Computational analysis of a protein’s preference to bound lipids. CDL binding results in a significantly lower energy state.
</p>
</div>
<p>Inward-rectifier K<sup>+</sup> channels (K<sub>ir</sub>) are an example where the lipid-protein interaction is directly involved in the protein’s function. K<sub>ir</sub> have a Transmembrane domain (TMD) and an intracellular C-terminal domain (CTD). PIP<sub>2</sub> binding cuases the CTD to move towards the TMD, which in turn causes the helices in the TMD to bend and thus opens the gate (Figure <a href="#fig:kir">6.2</a>).</p>
<div class="figure"><span id="fig:kir"></span>
<img src="../img/kir.png" alt="Lipid-gated Kir channel" width="100%" />
<p class="caption">
Figure 6.2: Lipid-gated Kir channel
</p>
</div>
<p>Many crystal structures of GPCRs reveal cholesterol bound to the protein. Molecular dynamics simulations at different cholesterol concentrations suggests the role of cholesterol in reducing the dynamic flexibility the protein and stabilising a certain conformational state in an allosteric fashion.</p>
<!-- Lipid rafts are stable but transient domains that are enriched in cholesterol and sphingolipids, which makes them thinker than the average membrane thinkness (directly detectable using atomic force microscopy). They are specifically associated with lipid anchored proteins (e.g. GPI-, palmitoyl-, myristoyl-anchored protein) -->
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/crystallography-a-case-study.html" class="archive-item-link">
                Crystallography: A Case Study 
              </a>
              <span class="archive-item-date">
                2020-11-04
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>The Ca<sup>2+</sup> selectivity of voltage-gated calcium (Ca<sub>v</sub>) channels remained unclear. They were thought to use a ‘knock-off’ methanism which requires multiple ion-binding cores, but mutational analyses supported a single high-affinity Ca<sup>2+</sup> binding site. This paradox is explained by the mechanism proposed in this article, based on the crystal structures of Ca<sub>v</sub>Ab channels.</p>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<p>In this study, the authors created several mutants of bacterial Na<sub>v</sub> channel Na<sub>v</sub>Ab by site directed mutagenesis using QuickChange at the selectivity filter region<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Some of these mutations changes the activity of the channel from allowing efflux of Na<sup>+</sup> to allowing influx of Ca<sup>2+</sup> to different extents (quantified by the relative permeability of Ca<sup>2+</sup> against Na<sup>+</sup>). Baculovirus were used as the vector to infect <em>Trichopulsia ni</em> cells. Proteins were collected from insect cells, purified, concentrated to ~20mg ml<sup>-1</sup>, and reconstituted into DMPC:CHAPSO bicelles (Figure <a href="#fig:dmpc-chapso">1</a>). Crystals were grown in a hanging-drop vapour-diffusion format.</p>
<div class="figure"><span id="fig:dmpc-chapso"></span>
<img src="../img/dmpc-chapso.png" alt="The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins' natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation." width="100%" />
<p class="caption">
Figure 1: The DMPC:CHAPSO bicelle. Bicelles are disks with a planar region formed by long chain lipid phospholipids and a rim composed of short chain phospholipids or detergent molecules. It provides an environment that resembles a lipid bilayer where membrane proteins’ natively resides. Membrane proteins can be incorporated into bicelles for functional studies as well as crystallisation.
</p>
</div>
<p>Before X-ray diffraction data collection, crystals were soaked with cryo-protectant solutions containing Ca<sup>2+</sup>, Mn<sup>2+</sup> or Cd<sup>2+</sup> of indicated concentrations. These ions travel through solvent channels within the crystals and chelate with amino acid residues in the selectivity filter region.</p>
<p>X-ray diffraction data were integrated and scaled with the HKL2000 packaged and further processed with the CCP4 package. THe structure of Ca<sub>v</sub>Ab and its derivatives were solved by molecular replacement by using an individual subunit of the Na<sub>v</sub>Ab structure (PDB code 3RVY) as the search template. This choice is natural since the proteins used in this study are derived from Na<sub>v</sub>Ab and differ from it by no more than 3 amino acid residues.</p>
<p>The divalent cations were identified by anomalous difference Fourier maps calculated using data collected at wavelengths of 1.75 Å for Ca<sup>2+</sup>, Cd<sup>2+</sup> and Mn<sup>2+</sup>. Anomalous scattering describes cases where a pair of structure factors <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> does not obey Friedel’s law<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, which occurs when the incident X-ray photons have an energy close to a transition energy of the diffracting atom, resulting in absorption of radiation energy and change in phase (normal scattering does not change the phase). At the wavelengths convinient for diffraction, only atoms heavier than phosphorus or sulfur behave as anomalous scatterers. While anomalous scattering data are frequently used to solve the phase problem (in techniques known as single/multiple wavelength anomalous dispersion (SAD/MAD)), it can also be used (as in this study) to calculate an anomalous difference map after phases are available to show the locations of the heavy atoms.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>Crystallography and NMR System software was used for refinement of coordinates and <em>B</em>-factors. Final models were obtained after several cycle refinement with REFMAC and PHENIX and manual re-building using COOT. The geometries of the final structural models of Ca<sub>v</sub>Ab and its derivatives were verified using PROCHECK.</p>
<p>13 crystal structures of 5 proteins (Na<sub>v</sub>Ab and 4 Ca<sub>v</sub>Ab variants) with the following divalent cation concentrations were determined and deposited into PDB:<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></p>
<ul>
<li>TLESWSM (Na<sub>v</sub>Ab) + Ca<sup>2+</sup> 15mM</li>
<li>(TLE<strong>D</strong>WSM, TLE<strong>D</strong>ES<strong>D</strong>, TL<strong>DD</strong>WSM) + Ca<sup>2+</sup> 15mM</li>
<li>TL<strong>DD</strong>WS<strong>D</strong> + (Ca<sup>2+</sup> 15mM; Mn<sup>2+</sup> 100mM; Cd<sup>2+</sup> 100mM)</li>
<li>TL<strong>DD</strong>WS<strong>N</strong> + Ca<sup>2+</sup> (0.5, 2.5, 5, 10, 15mM)</li>
</ul>
<p>Of the 13 crystal structure models, 4MVR (<sup>175</sup>TLDDWSD<sup>181</sup>) corresponds to the protein with the greatest permeability ratio <em>P</em><sub>Ca</sub>:<em>P</em><sub>Na</sub> , while <a href="https://www.rcsb.org/structure/4MS2">4MS2</a> (<sup>175</sup>TLDDWSN<sup>181</sup>) diffracted to the highest resolution (2.75 Å).</p>
</div>
<div id="analyses-of-results" class="section level1">
<h1>Analyses of Results</h1>
<div id="ca2-binding-sites" class="section level2">
<h2>Ca<sup>2+</sup> Binding Sites</h2>
<p>Using the anomalous diffraction data, the <strong>F</strong><sup>+</sup><sub>Ca</sub> - <strong>F</strong><sup>-</sup><sub>Ca</sub> anomalous difference map was calculated. Two strong peaks followed by a weaker peak were found along the ion-conduction pathway, which correspond to the three Ca<sup>2+</sup> binding sites. They are designated site 1, 2 and 3 from the extracellular side to the intracellular side.</p>
<p>Site 2 is the site with the highest affinity for Ca<sup>2+</sup>. It is surrounded by a total of 8 oxygen atoms, 4 of which coming from the carboxylate of D177 above and the other 4 from the carbonyl of L176 below. Site 1 is coordinated by the plane of 4 carboxyl groups from D178, and site 3 by the plane of 4 carbonyls from T175 (Figure <a href="#fig:cavab-schematic">2</a>). Throughout the selectivity filter, the O-Ca<sup>2+</sup> coordination distances are in the range of 4.0-5.0 Å, which is much longer than the ionic diameter of Ca<sup>2+</sup> (2.28 Å), suggesting that the bound Ca<sup>2+</sup> ion maintains its hydration shell while passing through the pore. Site 3 has the lowest affinity, consistent with its role in exit of Ca<sup>2+</sup> from the selectivity filter into the central cavity. Figure <a href="#fig:coot-4ms2">3</a> shows the molecular model and electron density near the selectivity fileter region of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong> + 15mM Ca<sup>2+</sup>) in COOT<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> (contoured at r.m.s.d = <span class="math inline">\(3.02 \sigma\)</span>).</p>

<div class="figure"><span id="fig:cavab-schematic"></span>
<img src="../img/cavab-schematic.png" alt="A schemetic showing the Ca2+ ion coordination sites of CavAb (TLDDWSD)" width="100%" />
<p class="caption">
Figure 2: A schemetic showing the Ca<sup>2+</sup> ion coordination sites of Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>D</strong>)
</p>
</div>
<div class="figure"><span id="fig:coot-4ms2"></span>
<img src="../img/coot-4ms2.png" alt="The selectivity filter region of 4MS2." width="100%" />
<p class="caption">
Figure 3: The selectivity filter region of 4MS2.
</p>
</div>
<p>The relative affinities of the three sites were further confirmed by experiments on Ca<sub>v</sub>Ab (TL<strong>DD</strong>WS<strong>N</strong>) with varying Ca<sup>2+</sup> concentrations (0.5, 2.5, 5, 10, 15mM): at low Ca<sup>2+</sup> concentration, two strong peaks of approximately equal intensity are found at Site 1 and Site 2; at high concentration the electron density is significantly enhanced in Site 2 and decreased in Site 1. The electron density at Site 3 remains low in all concentrations.</p>
</div>
<div id="ion-permeation-mechanism" class="section level2">
<h2>Ion-Permeation Mechanism</h2>
<p>Based on the properties of the three coordination sites, an ion-permeation mechanism can be deduced. The three coordination sites are separated by a distance of about 4.5 Å, which makes it energetically unfavourable for Ca<sup>2+</sup> to occupy adjacent sites simutaneously. Thus the authors suggested that the selectivity filter oscillates between two states, in which either a single hydrated Ca<sup>2+</sup> occupies Site 2, or two of them each occupies Site 1 and Site 3. The entry of Ca<sup>2+</sup> into Site 1 is promoted by it high extracellular concentration, and the exit of Ca<sup>2+</sup> is facilitated by the low affinity of Site 3.</p>
<p>Notably, this mechanism suggests that Ca<sup>2+</sup> ions are kept hydrated during its passage through the selectivity filter. This is very different from the mechanism by which potassium channels achive selectivity, which requires K<sup>+</sup> ions’ hydration shell to be removed.</p>
</div>
<div id="roles-of-key-selectivity-filter-residues" class="section level2">
<h2>Roles of Key Selectivity Filter Residues</h2>
<p>The five variants of Ca<sub>v</sub>Ab the author produced have different Ca<sup>2+</sup> selectivity ratios, which can be partially explained by directly comparing the difference in the arrangement of selectivity filter residues. This comparison is sometimes facilitated by superposition of one structure onto another. For example, the carboxyl group of D177 in TL<strong>DD</strong>WS<strong>D</strong> interacts with the Ca<sup>2+</sup> ion, while the carboxyl group of E177 in TLE<strong>D</strong>WS<strong>D</strong> swings away from the selectivity filter and forms a hydrogen bond with D181 and the main-chain nitrogen atoms of S180.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>specifically, from <sup>175</sup>TLESWSM<sup>181</sup> to TL<strong>DD</strong>WS<strong>D</strong>, TL<strong>DD</strong>WS<strong>N</strong>, TL<strong>DD</strong>WS<strong>M</strong>, TLE<strong>D</strong>WSM, TLE<strong>D</strong>WS<strong>D</strong> and TL<strong>D</strong>SWSM<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Friedel’s law states that <span class="math inline">\(\mathbf{F}_{hkl}\)</span> and <span class="math inline">\(\mathbf{F}_{\overline{hkl}}\)</span> have the same amplitude but opposite phase angle.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>This map is automatically calculated in phenix.refine.<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>namely 4MS2, 4MTO, 4MTF, 4MTG, 4MVU, 4MW8, 4MVZ, 4MVM, 4MVO, 4MVQ, 4MW3, 4MVS, 4MVR<a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>COOT version 0.9.1 (packaged with CCP4), on Linux version 5.8.16-2-MANJARO<a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/assessing-structural-models.html" class="archive-item-link">
                Assessing The Quality Structural Models 
              </a>
              <span class="archive-item-date">
                2020-10-27
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/header-attrs/header-attrs.js"></script>
<link href="./rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="./rmarkdown-libs/anchor-sections/anchor-sections.js"></script>

<div id="TOC">
<ul>
<li><a href="#the-essence-of-x-ray-crystallography"><span class="toc-section-number">1</span> The Essence of X-Ray Crystallography</a></li>
<li><a href="#assessing-the-quality-of-a-structure-during-model-building-and-refinement"><span class="toc-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</a>
<ul>
<li><a href="#r-and-rfree"><span class="toc-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></a></li>
<li><a href="#reci"><span class="toc-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</a></li>
<li><a href="#fit"><span class="toc-section-number">2.3</span> Real-Space Refinement: Map Fitting</a></li>
</ul></li>
<li><a href="#assessing-quality-after-automatic-model-building-and-refinement"><span class="toc-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</a>
<ul>
<li><a href="#density-fit-analysis-and-local-geometry-validation"><span class="toc-section-number">3.1</span> Density fit analysis and local geometry validation</a>
<ul>
<li><a href="#structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles"><span class="toc-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</a></li>
<li><a href="#torsional-angles-and-ramachandran-plot"><span class="toc-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</a></li>
<li><a href="#undefined-regions"><span class="toc-section-number">3.1.3</span> Undefined regions</a></li>
<li><a href="#unexplained-density"><span class="toc-section-number">3.1.4</span> Unexplained Density</a></li>
<li><a href="#distortions-due-to-crystal-packing"><span class="toc-section-number">3.1.5</span> Distortions due to crystal packing</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p>A high-quality protein structure model explains the experimental observations well, and conforms to physical and biochemical principles. There are some parameters indicating the quality of a structure model. Some of them are used during model building and refinement, while others apply only on a refined structure.</p>
<div id="the-essence-of-x-ray-crystallography" class="section level1" number="1">
<h1><span class="header-section-number">1</span> The Essence of X-Ray Crystallography</h1>
<p>The intensity at <span class="math inline">\((h, k, l)\)</span> in the reciprocal space, i.e. the structure factor amplitudes collected in X-ray experiments is directly related to the Fourier transform of the electron density in the real space (<span class="math inline">\(I = \mathbf{F}_{hkl}^2\)</span>), the latter being calculated as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \int_{x}\int_{y}\int_{z}\rho(x, y, z)e^{2\pi i(hx +ky + lz) }\mathrm{d}x \mathrm{d}y \mathrm{d}z
\end{equation}\]</span></p>
<p>In practice, since the number of atoms and reflections (hence <span class="math inline">\((x, y, z)\)</span> or <span class="math inline">\((h, k, l)\)</span> coordinates is limited, its discrete form, which can be efficiently computed, is used:<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_{hkl} = \sum_{j}^{n}f_j e^{2\pi i(hx_j +ky_j + lz_j) }
\end{equation}\]</span></p>
<p>where <span class="math inline">\((x_j, y_j, z_j)\)</span> is the coordinates of the <em>j</em>-th atom in real space.</p>
<p>This is also known as the <em>structure factor equation</em>.</p>
<p>Solving the electron density from structure factors involves solving the inverse Fourier transform of the above equation, also known as the <em>electron density equation</em>, in its discrete form:</p>
<p><span class="math display">\[\begin{equation}
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}w_{hkl}|\mathbf{F}_\text{obs}|e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(\alpha^\prime_{hkl}\)</span> is the phase at each set of coordinates in the reciprocal space. The phases are not directly available but are necessary (actually more important than intensities) for computing <span class="math inline">\(\rho(x,y,z)\)</span> (electron density).</p>
<p>The phases can be solved either by phasing experiments, in which case the resolution is relatively low but the data is unbiased, or by molecular replacement (MR), in which case the resolution is relatively high but the data suffer from bias (the phases resemble the search model more than the actual structure). Thus, bias minimisation measures such as maxinum likelihood map coefficients are applied before electron density map contrstruction.</p>
</div>
<div id="assessing-the-quality-of-a-structure-during-model-building-and-refinement" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Assessing the Quality of A Structure During Model Building and Refinement</h1>
<p>Solving the electron density and ultimately the atomic structure from experimental data is an iterative process, starting from a model with low resolution and imperfect phases and progressively refining the model by applying prior knowledge on the information the current model can provide so far.</p>
<p>In general, the first step in refinement is to construct a crude electron density map (Equation (3)) using phase angles (<span class="math inline">\(\alpha^\prime\)</span>) obtained either from MR or experimental phasing. This initial map, <strong>F</strong><sub>0</sub>, is improved using techniques such as <em>density modification, solvent leveling, solvent flattening</em>. At some critical point in the iterative improvement of phases, the map becomes clear enough that a protein chain can be traced through it. From this point on, real-space refinement is also conducted at each round of iteration. Electron density map is obtained (details in Section <a href="#fit">2.3</a>), and filled with atoms, and transformed back to the reciprocal space. This process repeats until converging to a high-quality strucure. <span class="math inline">\(R\)</span> and <span class="math inline">\(R_\text{free}\)</span>, explained below, are used to assess progress in the refinement, and is also a major factor in evaluating the quality of a model.</p>
<div id="r-and-rfree" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> <em>R</em> and <em>R</em><sub>free</sub></h2>
<p><em>R</em> is a measure of the deviation of calculated intensities from models (details in Section <a href="#reci">2.2</a>) from the observed intensities in the diffraction pattern, defined by the following equation:</p>
<p><span class="math display">\[
R = \dfrac{\sum ||\mathbf{F}_\text{obs}| - |\mathbf{F}_\text{calc}||}{\mathbf{F}_\text{obs}}
\]</span></p>
<p>Since bias can easily be introduced into the <em>R</em> value (especially by overparameterisation, see Section <a href="#reci">2.2</a>) and a reduction of <em>R</em> value sometimes does not improve the actual quality of structure (<span class="citation">Kleywegt and Brünger (n.d.)</span>), a small fraction (typically around 5%<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>) of randomly selected reflections are removed from the data used for refinement. These reflections can then be used to calculate an R factor, denoted as <span class="math inline">\(R_\text{free}\)</span>, whose reduction can be considered as an <em>unbiased estimate</em> of the improvement of the model.</p>
<div class="figure"><span id="fig:coot-phenix"></span>
<img src="../img/coot-phenix.png" alt=" R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine)." width="100%" />
<p class="caption">
Figure 2.1:  R and R-free values decreases as refinement proceeds (left: Coot; right: phenix.refine).
</p>
</div>
<p>Figure <a href="#fig:coot-phenix">2.1</a> shows the decrease of <em>R</em> and <em>R</em><sub>free</sub> during a refinement task conducted in Phenix.</p>
</div>
<div id="reci" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Reciprocal-Space Refinement: Refinement by least squares</h2>
<p>Reciprocal-space refinement involves computerised attempts to improve agreement between <span class="math inline">\(\mathbf{F}_\text{obs}\)</span> and <span class="math inline">\(\mathbf{F}_\text{calc}\)</span> by <em>without</em> consideration of the maps and models. <em>Refinement by least squares</em> is the earliest successful technique and is discussed here.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>The goal of refinement by least squares is, find <span class="math inline">\((x_j, y_j, z_j)\)</span> for all atom <span class="math inline">\(j\)</span> whose <em>expected</em> ( i.e. computed) structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> are as close as possible to <em>observed</em> structure factor amplitudes, <span class="math inline">\(|\mathbf{F}_\text{obs}|\)</span>. Specifically, this means minimising the function <span class="math inline">\(\Phi\)</span>:</p>
<p><span class="math display">\[\begin{equation}
\Phi = \sum_{hkl}(w_{hkl}\mid\mathbf{F}_\text{obs}\mid - \mid\mathbf{F}_\text{calc}\mid)_{hkl}^{2}
\end{equation}\]</span></p>
<p>where <span class="math inline">\(w_{hkl}\)</span> is the weight term that depends on the reliability of the corresponding measured intensity and <span class="math inline">\(|\mathbf{F}_\text{calc}|\)</span> is a variant form of Equation (2) that can include additional parameters such as <em>B</em>-factor (<span class="math inline">\(B_j\)</span>) and occupancy <span class="math inline">\(n_j\)</span>. An equation with <span class="math inline">\(B_j\)</span> and <span class="math inline">\(n_j\)</span> included can be written as:</p>
<p><span class="math display">\[\begin{equation}
\mathbf{F}_\text{calc} = \sum_{j}n_j f_j e^{2\pi i(hx_j +ky_j + lz_j) - B_j[(\sin\theta)/\lambda]^2}
\end{equation}\]</span></p>
<p>Note that the equatin shows that the effect of <em>B</em>-factors depends on the angle of reflection <span class="math inline">\((\sin\theta)/\lambda\)</span>.</p>
<p>Solving the minimum of <span class="math inline">\(\Phi\)</span> analytically is impractical, and instead numerical methods are used, which would lead to a minimum closest to the starting value.To prevent the refinement converging to a local minimum, it is important that the starting parameters be near the <em>global minimum</em>. Is also important <em>not</em> to include too many parameters (such as <em>B</em>-factor) at the initial stages of resolution when the resolution is low, as this would decrease the radius of convergence (Figure <a href="#fig:radius-convergece">2.2</a>).</p>
<div class="figure"><span id="fig:radius-convergece"></span>
<img src="../img/radius-convergece.png" alt="Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps." width="100%" />
<p class="caption">
Figure 2.2: Adding number of parameters improves precision of refinement, but makes it more unlikely to reach the global minimum from a given point. Thus, refinement starts with a small number of parameters, and more parameters are only added after the success of previous lower-resolution refinement steps.
</p>
</div>
<p>To minimise the number of parameters used during early stages of refinement (and thus to increase radius of convergence), individual <span class="math inline">\((x,y,z)\)</span> coordinates are actually not used. Instead, only torional angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are allowed to change, and all bond lengths and angles are fixed to their theoretical average, side chains are assumed to be in their preferred conformation, and peptide linkages are fixed to be planar. This strategy is known as <em>restrained</em> reciprocal space refinement. As refinement proceeds, more parameters, from individual <span class="math inline">\((x,y,z)\)</span> coordinates to isotropic B-factors and finally anisotropic factors, can be added into calculation.</p>
</div>
<div id="fit" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Real-Space Refinement: Map Fitting</h2>
<p><em>Map fitting</em> or <em>model building</em> entails building a molecular model that fits realistically into the current electron density contour map.</p>
<p>To reduce the bias (towards <strong>F</strong><sub>calc</sub> (<strong>F</strong><sub>c</sub>)) when constructing the electron density map, Fourier syntheses of <strong>F</strong><sub>obs</sub> and <strong>F</strong><sub>calc</sub> are used.
A Fourier synthesis <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> is calculated as:</p>
<p><span class="math display">\[
\rho(x, y, z) = \dfrac{1}{V}\sum_{h}\sum_{k}\sum_{l}(m|\mathbf{F}_\text{o}|-|\mathbf{F}_\text{c}|)e^{-2\pi i(hx + ky + lz - \alpha^\prime_{hkl})}
\]</span></p>
<p>and its corresponding electron density map is called an <em>m</em><strong>F</strong><sub>o</sub> - <em>n</em><strong>F</strong><sub>c</sub> map.</p>
<p>Simply put, the 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map resembles a molecular surface, and a <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map emphasises the error (positive density implies that the unit cell contains more electron density in this region than implied by the model (<strong>F</strong><sub>c</sub>). Near the end of refinement, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> map becomes rather empty except in problem areas, which may need to be corrected manually.</p>
<p>Fitting a molecular model into the electron density map depends on prior knowledge, such as average bond lengths and angles, the amino acid sequence of the protein, properties of peptide chains, etc. For example, we know that carboxyl oxygens in adjacent amino acid residues in a <span class="math inline">\(\beta\)</span>-sheet point in opposite directions. Thus, once a <span class="math inline">\(\beta\)</span>-sheet along with one or two carboxyl oxygen are discernible, we can make a sensible guess of the positions of all other carbonyl oxygens.</p>
</div>
</div>
<div id="assessing-quality-after-automatic-model-building-and-refinement" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Assessing Quality After Automatic Model Building and Refinement</h1>
<div id="density-fit-analysis-and-local-geometry-validation" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Density fit analysis and local geometry validation</h2>
<p>Automatic model building and refinement use the decrease of <em>R</em> value as an indicator of progress and terminates when <em>R</em> is considered to be sufficiently low. This may lead to situations where the global <em>R</em> is favourable but local geometry can still be improved.</p>
<p>Local geometry validation programs, such as “Density fit analysis” in Coot (Figure <a href="#fig:coot-density-fit-2">3.1</a>), evaluate the model geomtry on a per-residue basis and flag outliers. These outliers can then be fixed manually. With the aid of electron density contour maps (where model atoms lie outside 2<strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> contours, the <strong>F</strong><sub>o</sub> - <strong>F</strong><sub>c</sub> will often show the atoms with negativel contours, with nearby positive contours pointing to correct locations for these atoms).</p>
<div class="figure"><span id="fig:coot-density-fit-2"></span>
<img src="../img/coot-density-fit-2.png" alt="Use the 'Density fit analysis' function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the 'Rotamers' tool to fix a side chain that's pointing the wrong way." width="100%" />
<p class="caption">
Figure 3.1: Use the ‘Density fit analysis’ function to evaluate model geometry on a per-residue basis and plot a histogram that shows outliers, then use the ‘Rotamers’ tool to fix a side chain that’s pointing the wrong way.
</p>
</div>
<div id="structural-parameters-bond-length-bond-angle-chirality-planarity-and-dihedral-angles" class="section level3" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Structural parameters: bond length, bond angle, chirality, planarity, and dihedral angles</h3>
<p>During late stages of refinement, restraints on structura paramaters such as bond lengths and angles are release such that thay are allowed to change freely as long as <em>R</em> values could be made to decrease. This may lead to unrealistic models with improbable structural parameters. Thus, they need to be validated. By convention, this is done by calculating the root-mean-square deviations (RMSD) of these parameters in the model from the corresponding set of values based opon the geometry of small organic molecules. These values are recorded in PDB files in <code>REMARK 3</code> fields:</p>
<pre><code>REMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       
REMARK   3                 RMSD          COUNT                                  
REMARK   3   BOND      :  0.003           1366                                  
REMARK   3   ANGLE     :  0.675           1846                                  
REMARK   3   CHIRALITY :  0.050            186                                  
REMARK   3   PLANARITY :  0.005            246                                  
REMARK   3   DIHEDRAL  : 15.473            459                                  </code></pre>
<p>Because bond angles, lengths, chirality and planarity are restrained in early stages of refinement, they are less likely to deviate much from ideal values. Dihedral angles, which is allowed to change throughout the refinement, produce much larger RMSD and are worth closer examination.</p>
</div>
<div id="torsional-angles-and-ramachandran-plot" class="section level3" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Torsional Angles and Ramachandran Plot</h3>
<p>Torsional (dihedral) angles <span class="math inline">\(\psi\)</span> and <span class="math inline">\(\phi\)</span> are show much more variation than bond lengths and angles, but only a subset of all possible (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs are allowed so that adjacent amino acid side chains do not clash. Validation of torsional angle is achieved via a lookup table, where the keys are (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) pairs and values are scores. A (<span class="math inline">\(\phi\)</span>, <span class="math inline">\(\psi\)</span>) is considered preferred or allowed if its score is within certain thresholds. Otherwise, it is considered an outlier. Due to glycine’s small size and proline’s cyclic structure, the preferred/allowed regions of their torsinal angle pairs are defined differently, for example in Phenix/cctbx<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>. Torsional angle validation is often visualised with a Ramachandran plot, as shown in Figure <a href="#fig:coot-ramachandran">3.2</a>.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></p>
<div class="figure"><span id="fig:coot-ramachandran"></span>
<img src="../img/coot-ramachandran.png" alt="Validating dihedral angles with Ramachandran plot in Coot" width="100%" />
<p class="caption">
Figure 3.2: Validating dihedral angles with Ramachandran plot in Coot
</p>
</div>
</div>
<div id="undefined-regions" class="section level3" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Undefined regions</h3>
<p>Often, some regions (often the terminal regions) of the protein are highly disordered and produce no detectable diffraction at high resolution and are thus invisible in the electron-density maps. It is not uncommon to omit these residues in the model. In some cases, however, exact coordinates are given for these atoms, along with very high <em>B</em>-factors (above 50 or 60 angstroms), and one must be careful not to over-interprete these regions.</p>
</div>
<div id="unexplained-density" class="section level3" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> Unexplained Density</h3>
<p>Empty electron density may remain after all known contents of the unit cell have been located. This can either due to an artifect of missing Fourier terms, reagents used in purification or crystallisation, or it could be due to previously unknown ligands (cofactors, inhibitors or allosteric effectors).</p>
</div>
<div id="distortions-due-to-crystal-packing" class="section level3" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Distortions due to crystal packing</h3>
<p>Although flexibility of atoms is usually reflected by the <em>B</em>-factor, in certain situations dynamic regions of a protein molecule can be rigidly fixed in a specific conformation as a result of crystal packing interactions. It should be declared in the PDB header that the protein is affected by crystallisation. One can also check whether these interactions are likely to occur by displaying all neighbouring, symmetry-related molecules in the crystal structure and examine if any intermolecular interactions are present that are a result of crystal packing.</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references hanging-indent">
<div id="ref-Blow-2002">
<p>Blow, David. 2002. <em>Outline of Crystallography for Biologists</em>. Oxford University Press.</p>
</div>
<div id="ref-Kleywegt-1996">
<p>Kleywegt, G J, and A T Brünger. n.d. “Checking Your Imagination: Applications of the Free R Value.” <em>Structure</em> 4 (8): 897–904. <a href="https://doi.org/10.1016/s0969-2126(96)00097-4">https://doi.org/10.1016/s0969-2126(96)00097-4</a>.</p>
</div>
<div id="ref-rhodes-2006">
<p>Rhodes, Gale. 2006. <em>Crystallography Made Crystal Clear Clear</em>. Academic Press.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Most crystallography softwares seem to use Fast Fourier Trasnform (FFT) algorithms provided by FFTW (<a href="http://www.fftw.org/" class="uri">http://www.fftw.org/</a>) to compute discrete fourier transforms (and its inverse)<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>In PDB files, this fraction is recorded in the field <code>FREE R VALUE TEST SET SIZE</code><a href="#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>modern refinement softwares use techniques such as simulated annealing and beyesian methods to improve the accuracy of refinement<a href="#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>In Phenix/cctbx, a lookup table (actually implemented as an array for better performance) is defined for each of the following cases: glycine, cis-proline, trans-proline, pre-proline, isoleucine/valine, and all other amino acid residues: <a href="https://github.com/cctbx/cctbx_project/blob/master/mmtbx/validation/ramachandran/rama8000_tables.h"><code>mmtbx/validation/ramachandran/rama8000_tables.h</code></a><a href="#fnref4" class="footnote-back">↩︎</a></p></li>
<li id="fn5"><p>Recently I’ve been developing a PDB file parser. It is now able to parse coordinate information reliably, which allows Ramanchandran plot analysis: <a href="https://github.com/TianyiShi2001/protein" class="uri">https://github.com/TianyiShi2001/protein</a><a href="#fnref5" class="footnote-back">↩︎</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/redox-based-modulation-of-enzyme-activity.html" class="archive-item-link">
                Redox-Based Modulation of Enzyme Activity 
              </a>
              <span class="archive-item-date">
                2020-06-17
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>Presentation see <a href="./thioredoxin-presentation/">here</a></p>
<!--
Thioredoxins are kept in the reduced state by the flavoenzyme thioredoxin reductase in a NADPH-dependent reaction. 

Plants have an unusually complex complement of Trxs composed of six well-defined types (Trxs f, m, x, y, h, and o) that reside in different cell compartments and function in an array of processes. In 2010 it was discovered for the first time that thioredoxin proteins are able to move from cell to cell, representing a novel form of cellular communication in plants.

Thioredoxins are small ubiquitous proteins with two redox-active cysteine rediuses separated by a pair of amino acids (CXXC)

TRXs *m, f, x, y, z* isoforms are located in the chloroplast, whereas TRXs *o* are mitochondrial and TRX *h* proteins are found in different subcellular compartments.

TRX *h* is the largest TRX gene family in plants, comprising nine different isoforms in Arabidopsis. First isolated from spinach leaves; outisde of the chloroplast. Lack of a transit peptide; initially assumed as solely cytosolic proteins. However, alternative locations were also observed for some of them: plasma membrane, endomembranes (ER & Golgi), mitochondria, and the nucleus. Highly redundant -> difficult to unequivocally determine the function of each TRX *h* isoform

Besides the plastid and mitochondria, Trxs are also found in the cytosol, ER and nucleus of plant cells. Little is known about the function of nucleoredoxins (Nrx) and ER-Trxs, the function of cytosolic Trx *h* has been well documented.

Trx regulates photosynthesis and the TCA cycle in chloroplasts and mitochondria, respectively.

Extraplastidial Trxs hhave been shown to be important in the regulation of different processes such as seed germination, seedling establishment, C, N, and mitochondrial metabolism as well as for plant stress tolerance.

# Trx reduction systems

In extraplastidial compartments the reducing power for Trx reduction is provided by NADPH via NADPH-Trx reductase A (NTRA) and NTRB.

The cytosolic h-type thioredoxins act as key regulators of seed germination and are recycled by NADPH-dependent thioredoxin reductase

## In seed

Seeds develop in a metabolically active state but become quiescent during maturatiion and drying. This period is characterised by low water content and low metabolic activity as well as the non-enzymatic accumulation of reactive oxygen species (roS) that oxidise proteins and RNA. Oxidation of the thiol groups of cysteine -> changes in protein structure -> lowering solubility and modifying enzyme function -> protection against denaturation and proteolysis as the seed matures and dries.

Under favourable conditions, the seed germinates and thhe seedling draws on stored reserves for growth and development. Large increase in ROS produced as side-products of metabolism. ROS have important signalling roles in germination (but also cause oxidative stress if present in high amounts). The Trx system represents an important waay to regulate ROS in seeds.

## NTR

The NTRA and NTRB are found in both cytosol and mitochondrion, while thhe first has also been found in the nucleuus. NTRA and NTRB are mainly expressed in the cytosol and mitochondrion, respectively.

The NADPH used by the NTR/Trx system is generated by the OPPP through the activity of G6PDH and 6-PGDH, while the souruce of NADPH in the mitoochondion is throuugh the activity of IDH or NADHk. 


# Methods

## Alkylation Labelling


@Kobrehel-1992 applied the fluorescnet alkylating reagent monobromobimane (mBBr) to label free thiol groups in proteins following reduction by Thx. -> discovered that gliadins and glutenin are reduced in vivo during germination

Buchanan and co-workers pioneered the use of proteomics for Trx target identiﬁcation and applied the ﬂuorescent alkylating reagent monobromobimane (mBBr) to label free sulfhydryl (SH) groups in proteins following reduction by Trx [37]. This approach can be applied not only to in vitro but to in vivo reactions as well [2,40,45,46]. To identify targets from peanut seeds, mBBr-labelled proteins were separated by two-dimensional polyacrylamide gel electrophoresis (2D-PAGE) and ﬂuorescent spots were subjected to trypsin digestion followed by either Edman sequencing or mass spectrometry for protein identiﬁcation

Lane 1, Control: no addition [1.1]; lane 2, GSH/GR/NADPH: GSH, glutathione re- ductase(fromspinachleaves),andNADPH [1.2];lane3,NGS: NADPH, GSH, glutathione reductase (from spinach leaves), and glutaredoxin (from E. coli) [1.6]; lane 4, NTS: NADPH, NTR, and thioredoxin (both proteins from E. coli) [3.9]; lane 5, MET/T(Ec): ,B- mercaptoethanolandthioredoxin(E.co/i)[1.8];lane6,DTT [2.5]; lane7,DTT/T(Ec):DTT andthioredoxin(E.co/i)[3.0];lane8,DTT/ T(W): same as lane 7 except with wheat thioredoxin h [3.6]; lane 9, NGS,-Gliadin: same as lane 3 except without the gliadin protein fraction [0]; lane 10, NTS,-Gliadin: same as lane 4 except without thegliadinproteinfraction[0].

The above findings demonstrate that thioredoxin specifically reduces components of the wheat gliadin and glutenin fractions when tested in vitro. The results, however, provide no indication as to whether these proteins are reduced in vivo during germination-a question that, to our knowledge, has not been previously addressed (cf. 27).

To answer this question, we applied the mBBr/SDS-PAGE technique to monitor the reduction status of proteins in the germinating seed. We observed that reduction of components in the Osborne fractions increased progressively with time and reached a peak after 2 to 3 d germination (Fig. 4). The observed increase in reduction ranged from twofold with the gliadins to threefold with the albumin/globulins and fivefold with the glutenins. The net redox change was most apparent in the gliadins and glutenins found to undergo maximal reduction by thioredoxin, i.e. the 30- to 50-kD gliadins and 30- to 60-kD glutenins (cf. Figs. 1 and 3). The redox changes taking place in these fractions during germination are shown in Figure 5.

## CXXS

@Verdoucq-1999

In the thiol-disulfide exchange reaction, Trx forms a transient intermolecular disufide bond with the target protein involving the more N-terminal cysteine in the active site of Trx (CXXC). Then, the other cysteine (CXXC) attacks the disulﬁde bond of the Trx-target intermediate and releases the reduced target and oxidized Trx.

If instead this second cysteine is replaced with a serine residue through site-directed mutagenesis the Trx loses its capability to reduce the intermolecular disulﬁde bond and the covalent Trx-target complex will be stabilized. This strategy was first applied by Verdoucq et al. [49] who constructed a monocysteinic Trx mutant of A. thaliana Trxh3. The protein was heterologously expressed in Saccharomyces cerevisae and it was successfully stabilized in a complex with a peroxiredoxin target protein.

The concept was further developed by Motohashi et al. [50] who immobilized a monocysteinic Trx m on a resin designed to trap targets on a column. Non-targets were removed with the column wash and the trapped targets could subsequently be eluted with a chemical reductant such as DTT (Fig. 3). This approach has been used to identify target proteins from plant mitochondria [51–53], chloroplasts [54], cytosolic extracts [55], and has also been combined with the thiol alkylation approach described above to identify putative target proteins, for example in seeds from M. truncatula [40].

Improved by @Motohashi-2001 (initially applied to Trx m) 
Sepharose 4B resin

@Yamazaki-2004 Target Proteins of the Cytosolic Thioredoxins in Arabidopsis thaliana.
The obtained proteins were separated by two-dimensional gel electrophoresis and analyzed by matrix-assisted laser desorption ionization time-of-flight mass spectrometry. 

The results in this study and recent proteomics studies on the target proteins of chloroplast thioredoxin indicate the versatility and the physiological significance of thioredoxin as reductant in plant cell.

## Differential Labelling of Recombinant BASI (IAA and 4-VP)

- for the well studied chloroplastic thiioredoxins, target disulphide bonds in target enzymes were mainly idenitified by using strategies based on site directed mutagenesis of target proteins
- hoowever, to produce recombinant target proteiins and their mutants would be entremely time-consumiing considering the large number of potential target proteins of thioredoxin h that have been identifieid
- Using mass spectrometry techniques, peptide fragments containing modifications are recognised in hte peptide mass maps according to an exact mass shift from the expected thhiioretical values.
- the thiol and disulphiide forms of cysteine side chains, however, are usually not distinguishhed in MS because complete reduction and alkylation of cysteines are performed during regular sample preparation
- A novel strategy is developed by @Maeda-2005.
- Trypsin digesting
- According to the mass difference of cysteines modified by these two reagents, conversion of disulphides to thiols catalysed by HvTrxh1 was shown in the peptide mass maps of recombinant BASI.

# Rice

Cereals store reserve proteins in the endosperm during maturation of grain. In germination and early seedling development, these proteins are digested to provide nitrogen for growth. In rice and certain other seeds, the major storage proteins are 11S globulins housed in the protein body of the endosperm. Generally, 11S globulins are hexamers composed of identical subunits, each containing two polypeptide chains (Hara et al. 1976, Müntz 1998). The acidic chain, with one presumed intramolecular disulfide bond, is connected to the basic chain by an intermolecular disulfide bond (Staswick et al. 1984, Lawrence et al. 1994) (Fig. 1). The basic chain also contains a free sulfhydryl group. In rice, the proteins corresponding to 11S globulins, the glutelins, are unique in that they exist as large macromolecular complexes formed by disulfide and hydrophobic interactions of acidic and basic polypeptides (Katsube et al. 1999).
-->
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-Geigenberger-2017" class="csl-entry">
Geigenberger, Peter, Ina Thormählen, Danilo M. Daloso, and Alisdair R. Fernie. 2017. <span>“The Unprecedented Versatility of the Plant‎ Thioredoxin System.”</span> <em>Trends in Plant Science</em> 22 (3): 249–62. https://doi.org/<a href="https://doi.org/10.1016/j.tplants.2016.12.008">https://doi.org/10.1016/j.tplants.2016.12.008</a>.
</div>
<div id="ref-Hagglund-2016" class="csl-entry">
Hägglund, Per, Christine Finnie, Hiroyuki Yano, Azar Shahpiri, Bob B. Buchanan, Anette Henriksen, and Birte Svensson. 2016. <span>“Seed Thioredoxin h.”</span> <em>Biochimica Et Biophysica Acta (BBA) - Proteins and Proteomics</em> 1864 (8): 974–82. https://doi.org/<a href="https://doi.org/10.1016/j.bbapap.2016.02.014">https://doi.org/10.1016/j.bbapap.2016.02.014</a>.
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/improving-photosynthesis-by-photorespiratory-bypasses.html" class="archive-item-link">
                Improving Photosynthesis by Photorespiratory Bypasses 
              </a>
              <span class="archive-item-date">
                2020-06-09
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#attempts-to-inhibit-photorespiration-and-improve-specificity-of-rubisco"><span class="toc-section-number">1</span> Attempts to Inhibit Photorespiration and Improve Specificity of Rubisco</a></li>
<li><a href="#bypassing-photorespiration"><span class="toc-section-number">2</span> Bypassing Photorespiration</a><ul>
<li><a href="#implementing-the-bacterial-glycerate-pathway"><span class="toc-section-number">2.1</span> Implementing The Bacterial Glycerate Pathway</a></li>
<li><a href="#inhibition-of-the-glycolate-transporter"><span class="toc-section-number">2.2</span> Inhibition of the Glycolate Transporter</a></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">3</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>Discuss the feasibility and desirability of improving photosynthesis in higher plants through the use of photorespiratory bypasses.</strong></p>
<p>Plants and algae use the Calvin cycle to capture inorganic carbon (as CO<sub>2</sub>) and use ATP and NADPH generated in light reactions to convert it to organic carbon. This process, known as photosynthesis, is the predominant way to replenish organic carbon in the biosphere, and is the basis for human food production. Despite its crucial role, the key enzyme in the pathway that attaches the inorganic CO<sub>2</sub> to the organic sugar 1,5-bisphosphate (RuBP), called Rubisco, is not efficient, in that it can also use O<sub>2</sub> instead of CO<sub>2</sub> as the substrate, which leads to production of one molecule each of 3-PGA and 2-phosphoglycerate (2-PG). Plants use a pathway known as photorespiration to remove the toxic 2-PG and to salvage some (75%) carbon lost from the Calvin cycle by converting two 2-PG to one 3-PGA while releasing CO<sub>2</sub> (Figure <a href="#fig:pr-bypass-native">0.1</a> ). Photorespiration is one of the major factors contributing to inefficiency of photosynthesis because 1) much CO<sub>2</sub> released in mitochondria is escaped into the atmosphere and not re-assimilated in the Calvin cycle, resulting in the reduction of the amount of Calvin cycle intermediates; 2) O<sub>2</sub> directly competes with CO<sub>2</sub> for Rubisco reaction (in ambient atmosphere and at 25<sup>o</sup>C, 25% of the Rubisco reactions in an average C3 plant uses O<sub>2</sub>); 3) in photorespiration, ammonia is released and needs to be reassimilated at the expense of one molecule of ATP and two reducing equivalents (reduced ferredoxins) per molecule of ammonia; 4) reducing power is dissipated in the oxidation of glycolate using molecular oxygen; 5) ATP is also consumed for phosphorylating glycerate in the final step in the photorespiration pathway (because the phosphate group on 2-PG is removed by phosphatase and is thus not conserved). The total direct energy cost for converting two 2PG to one 3PGA is 3.5 ATP and 2 NADPH. The rate of photorespiration is increased by the ratio of O<sub>2</sub> concentration to CO<sub>2</sub> concentration experienced by Rubisco (in C<sub>3</sub> plants, this is similar to atmospheric ratio), and is thus increased by high temperature, as solubulity of CO<sub>2</sub> decreases faster than does O<sub>2</sub> with increasing temperature.</p>
<div class="figure"><span id="fig:pr-bypass-native"></span>
<img src="../img/pr-bypass-native.png" alt="The native photorespiration pathway." width="100%" />
<p class="caption">
Figure 0.1: The native photorespiration pathway.
</p>
</div>
<p>Several groups of photosynthetic organisms independently evolved strategies to minimise photorespiration and are thus able to grow in hot and/or CO<sub>2</sub>-deficient conditions. For example, C<sub>4</sub> and CAM plants uses PEP carboxylase, which uses <span class="math inline">\(HCO_3^-\)</span> instead of CO<sub>2</sub> as the substrate, to perform an initial carboxylation, which results in production of 4-carbon malate/oxaloacetate that is later used to release CO<sub>2</sub> for Rubisco reaction. The initial carboxylation and later decarboxylation is separated spatially in C<sub>4</sub> plants and temporally in CAM plants. In aquatics environments where CO<sub>2</sub> concentration is low, cyanobacteria and green algae uses bicarbonate pumps to actively increase intracellular CO<sub>2</sub> (<span class="math inline">\(HCO_3^-\)</span>) concentration, and their Rubisco is in close proximity with carbonic anhydrase, which converts <span class="math inline">\(HCO_3^-\)</span> to CO<sub>2</sub>.</p>
<p>The vast majority of land plants (especially crops), however, adopts the C<sub>3</sub> metabolism scheme, which means they do not actively concentrate CO<sub>2</sub> and are susceptible to reduction of photosynthesis efficiency by photorespiration. Thus, it has been believed that reduction in photorespiration can improve photosynthesis and hence crop yield.</p>
<div id="attempts-to-inhibit-photorespiration-and-improve-specificity-of-rubisco" class="section level1">
<h1><span class="header-section-number">1</span> Attempts to Inhibit Photorespiration and Improve Specificity of Rubisco</h1>
<p>Initial attempts to reduce the effect of photorespiration include inhibition, knock-down and knock-out of the enzymes involved in photorespiration. The relevant genes are identified by screening for mutants that have impaired photorespiratory function. However, most of such mutants exhibited chlorosis and stunted growth. The rationale is simple: reduction of photorespiratory enzyme activity not only does not reduce the production of 2-PG by Rubisco, but also causes the toxic molecules resulted from partial metabolism of 2-PG to accumulate. Also, it has been suggested that photorespiratory enzymes may be also involved in other pathways related to serine biosynthesis and nitrogen metabolism.</p>
<p>Researches that attempted for a Rubisco enzyme with a greater specificity for CO<sub>2</sub> were also conducted. However, it was found that there is a inverse relationship between specificity and rate of reaction. As adaptations to achieve maximal rate of photosynthesis in different environments, Rubisco with high specificity and low turnover rate are typically found in C<sub>3</sub> plants that grow under high temperature/low CO<sub>2</sub>, and Rubisco with high turnover and low specificity are generally found in C<sub>3</sub> plants that grow under lower temperature, as well as in organisms with a CO<sub>2</sub>-concentrating mechanism (C<sub>4</sub>/CAM/algae/cyanobacteria).</p>
</div>
<div id="bypassing-photorespiration" class="section level1">
<h1><span class="header-section-number">2</span> Bypassing Photorespiration</h1>
<p>As attempts to improve Rubisco specificity and inhibit photorespiration directly both did not yield much outcome, photorespiratory bypasses becomes the new hotspot for improving photosynthesis. In general, photorespiratory bypasses are ‘designed photorespiratory pathways’ that aim to minimise energy comsumption and CO<sub>2</sub> loss to the atmosphere. Successful bypasses usually involve confining photorespiration to less compartments (ideally in chloroplast only), preventing lost of reducing power via oxidase, preventing release of NH<sub>3</sub>, and reducing the number of steps of reactions.</p>
<div id="implementing-the-bacterial-glycerate-pathway" class="section level2">
<h2><span class="header-section-number">2.1</span> Implementing The Bacterial Glycerate Pathway</h2>
<p>Many bacteria possess a simple and efficient pathway for converting glyoxylate to glycerate. This pathway involves two steps: 1) condensation of two glyoxylate to tartronate semialdehyde (by glyoxylate carboxyligase (GCL)) while releasing CO<sub>2</sub>; 2) reduction of tartronate semialdehyde by tartronate semialdehyde reductase (requiring one NADH) to glycerate. Compared with the native photorespiratory pathway (which converts glyoxylate to glycine in peroxisome and decarboxylates glycine in the mitochondrion), NH<sub>3</sub> release is prevented and, if this pathway can be implemented in the chloroplast, CO<sub>2</sub> release is localised to the chloroplast and is more likely to be re-assimilated by Rubisco instead of escaping to the atmosphere.</p>
<p>To implement the pathway in the chloroplast, a targeting sequence (e.g. Arabidopsis Rubisco small subunit (RbcS) or phosphoglucomutase transit peptide sequence) needs to be added to the N terminus of the gene constructs of the desired enzymes. In addition, the enzyme(s) that converts glycolate to glyoxylate also need(s) to be imported to the chloroplast. In normal plants, this conversion is catalysed by glycolate oxidase in peroxisome, and the same enzyme can be targeted to the chloroplast. However, glycolate oxidase is inefficient, as it uses O<sub>2</sub> as the hydrogen acceptor, which wastes reducing power. In addition, its reaction generates H<sub>2</sub>O<sub>2</sub>, which needs to be broken down by catalase. A more efficient bacterial enzyme, glycolate dehydrogenase (GDH), can be used to replace glycolate oxidase’s role. GDH uses NAD<sup>+</sup> instead of O<sub>2</sub> as the hydrogen acceptor, which prevents production of toxic H<sub>2</sub>O<sub>2</sub> and preserves reducing power.</p>
<p>According to the scheme described above, <span class="citation">Kebeish et al. (2007)</span> generated plants transformed with five chloroplast-targeted bacterial genes encoding glycolate dehydrogenase, glyoxylate carboligase and tartronic semialdehyde reductase (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 1). In these plants, some glycolate is successfully converted directly to glycerate in chloroplasts, and reduces, but not eliminate, flux of photorespiratory metabolites through peroxisomes and mitochondria. The transgenic plants grew faster, produced more shoot and root biomass, and <a href="https://www.nature.com/articles/nbt1299/tables/1">the content of soluble sugars (fructose, glucose and sucrose) increases significantly</a>.</p>
<div class="figure"><span id="fig:pr-bypass-3mech"></span>
<img src="../img/pr-bypass-3mech.png" alt="Three photorespiratory bypasses. The loop represents the native pathway." width="100%" />
<p class="caption">
Figure 2.1: Three photorespiratory bypasses. The loop represents the native pathway.
</p>
</div>
<p><span class="citation">Carvalho et al. (n.d.)</span> designed a similar photorespiratory bypass (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 3). It also uses tartronate semialdehyde (TSA) as the intermediate and thus prevents NH<sub>3</sub> production. However, the CO<sub>2</sub>-releasing TSA synthesis reaction occurs in the peroxisome rather than in the chloroplast and therefore this mechanism does not have the additional benefit of reducing CO<sub>2</sub> loss. In the peroxisome, TSA is converted to hydroxypyruvate by hydroxypyruvate isomerase (HYI), and the rest is identical to the native pathway. Unfortunately, all plants fail to express HYI, and for those with GCL expressed, chlorosis and stunted growth are exhibited, similar to photorespiration mutants. (I think one statement regarding this piece of research at the end of page 808 of the review written by <span class="citation">Weber and Bar-Even (2019)</span> is not appropriate, as HYI was not even expressed)</p>
<p><span class="citation">Maier et al. (2012)</span> used a completely different approach. Instead of taking shortcuts within the native photorespiratory pathway with 3-PGA being the final product, they introduced glycolate oxidase, catalase and malate synthase into the chloroplast, which, together with pyruvate dehydrogenase and malic enzyme which are natively present in the chloroplast, completely oxidise glycolate into two CO<sub>2</sub> (Figure <a href="#fig:pr-bypass-3mech">2.1</a>, 2), while producing NADPH and NADH. Although this pathway requires more energy than the native pathway to restore the ‘<em>status quo ante</em>’ according to the calculation (see <span class="citation">Peterhansel, Blume, and Offermann (n.d.)</span>, Table 2), and is shown in computational modelling to result in a 31% <em>decrease</em> in photosynthetic efficiency compared to WT <span class="citation">(Xin et al. 2015)</span>, experimental studies on <em>Arabidopsiis thaliana</em> surprisingly revealed an increase in biomass in these transgenic plants. A modified version of this approach carried out later on tobacco plants by <span class="citation">South et al. (2019)</span> were shown to be even more successful than Kebeish’s approach which was calculated to be the most energy-efficient. The success of Maier’s method may be explained by thhe fact that, in hot climates (where sunlight is intense), ATP and NADPH supply from light reactions is often not limiting, so an improved energy balance in photorespiration does not necessarily improve the rate of carbon fixation. Instead, the benefit brought about by the rapid CO<sub>2</sub> generation in this bypass is more important. In addition, it is argued that the ‘wasteful’ dissipation of reducing power by glycolate oxidase may actually be beneficial in avoiding production of reactive oxygen species (ROS) in high light intensity, when the output of light reactions is in large excess. This also explains why higher plants prefer glycolate oxidase rather than glycolate dehydrogenase.</p>
</div>
<div id="inhibition-of-the-glycolate-transporter" class="section level2">
<h2><span class="header-section-number">2.2</span> Inhibition of the Glycolate Transporter</h2>
<p>In the two successful photorespiratory bypasses described above, glycolate metabolism inside chloroplast competes with its export via the glycolate exporter PLGG. Therefore, repression of this transporter is expected to increase the flux into the bypasses and thus further increase the efficiency. <span class="citation">South et al. (2019)</span> studied the effect of RNAi inhibition of <em>PLGG1</em> in these two methods as well as a ‘combined’ scheme of the two. The constructs and mechanisms of action are shown in Figure <a href="#fig:pr-bypasses-south-2019">2.2</a>. Both AP1 and AP3 (but not AP2) showed significant increase in dry weight biomass, by 13% and 18%, respectively. When the <em>PLGG1</em> RNAi module is added, the benefit of AP1 is lost, while that of AP3 increases to 24% (<span class="citation">Weber and Bar-Even (2019)</span> implied, on page 808 when citing this work, that inhibition of <em>PLGG</em> always increases efficiency, which is not correct).</p>
<div class="figure"><span id="fig:pr-bypasses-south-2019"></span>
<img src="../img/pr-bypasses-south-2019.png" alt="South's (2019) three photorespiratory bypass constructs. AP1: five bacterial genes encoding glycolate dehydrogenase (GDH), glyoxylate carboligase and tartronic semialdehyde reductase (same as Kebeish (2007)); AP2: glycolate oxidase (GOX), malate synthase and catalase (CAT) (same as Maier (2012)); AP3: same as AP2, except that GOX+CAT is replaced by GDH." width="100%" />
<p class="caption">
Figure 2.2: South’s (2019) three photorespiratory bypass constructs. AP1: five bacterial genes encoding glycolate dehydrogenase (GDH), glyoxylate carboligase and tartronic semialdehyde reductase (same as Kebeish (2007)); AP2: glycolate oxidase (GOX), malate synthase and catalase (CAT) (same as Maier (2012)); AP3: same as AP2, except that GOX+CAT is replaced by GDH.
</p>
</div>
<p>Elimination of the AP1 enhancements by the <em>PLGG1</em> RNAi module implies that this introduced pathway ‘may not have had sufficient kinetic capacity to handle the full glycolate flux under high rates of Rubisco oxygenation’, according to <span class="citation">South et al. (2019)</span>. This is in accordance with the aforementioned statement that the efficiency of photorespiratory bypasses in accelerating carbon fixation does not depend on energy economy (conservation of ATP and NADPH), but instead on the rate of conversion of 2-PG into TCA cycle intermediates (CO<sub>2</sub> and/or 3-PGA). That is to say, the real culprit for photorespiration-induced reduction in carbon fixation activity is not loss of energy in the form of ATP and NADPH, but is likely to be the inefficiency of removing 2-PG, as well as the low CO<sub>2</sub>:O<sub>2</sub> ratio. Thus, in South’s AP3 construct where 2-PG is efficiently cleared up and CO<sub>2</sub> is produced in a large amount, the most prominent increase in carbon fixation and biomass is observed.</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">3</span> Conclusion</h1>
<p>Photorespiration has long been a target for improving photosynthesis and crop yield. Several photorespiratory bypasses, which provide alternative, more efficient pathways to complement or replace the native pathway, have proved successful for this goal in model plants. Much is yet to be known about the mechanisms by which these bypasses accelerates photosynthesis, but recent experiments suggests that, in contrast to the traditional view that photorespiration is energy-wasting and should be avoided, increasing the flux through the photorespiration pathway might instead speed up carbon fixation. Also, conclusion obtained from computational modelling should be viewed with care, at least in photorespiratory studies, as the seemingly sensible assumptions they make often results in significant deviations from the experimental results.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Carvalho-2011">
<p>Carvalho, Josirley de F C, Pippa J Madgwick, Stephen J Powers, Alfred J Keys, Peter J Lea, and Martin A J Parry. n.d. “An Engineered Pathway for Glyoxylate Metabolism in Tobacco Plants Aimed to Avoid the Release of Ammonia in Photorespiration.” <em>BMC Biotechnol</em> 11. Embrapa Soybean, Londrina, Paraná, Brazil, Rodovia Carlos Strass, Distrito da Warta, Londrina PR, Brasil.: 111. <a href="https://doi.org/10.1186/1472-6750-11-111">https://doi.org/10.1186/1472-6750-11-111</a>.</p>
</div>
<div id="ref-Kebeish-2007">
<p>Kebeish, Rashad, Markus Niessen, Krishnaveni Thiruveedhi, Rafijul Bari, Heinz-Josef Hirsch, Ruben Rosenkranz, Norma Stäbler, Barbara Schönfeld, Fritz Kreuzaler, and Christoph Peterhänsel. 2007. “Chloroplastic Photorespiratory Bypass Increases Photosynthesis and Biomass Production in Arabidopsis Thaliana.” <em>Nat Biotechnol</em> 25 (5). RWTH Aachen, Institute of Biology I, Worringer Weg 1, 52056 Aachen, Germany.: 593–99. <a href="https://doi.org/10.1038/nbt1299">https://doi.org/10.1038/nbt1299</a>.</p>
</div>
<div id="ref-Maier-2012">
<p>Maier, Alexandra, Holger Fahnenstich, Susanne Von Caemmerer, Martin Engqvist, Andreas Weber, Ulf-Ingo Flügge, and Veronica Maurino. 2012. “Transgenic Introduction of a Glycolate Oxidative Cycle into a. Thaliana Chloroplasts Leads to Growth Improvement.” <em>Frontiers in Plant Science</em> 3: 38. <a href="https://doi.org/10.3389/fpls.2012.00038">https://doi.org/10.3389/fpls.2012.00038</a>.</p>
</div>
<div id="ref-Peterhansel-2013">
<p>Peterhansel, Christoph, Christian Blume, and Sascha Offermann. n.d. “Photorespiratory Bypasses: How Can They Work?” <em>J Exp Bot</em> 64 (3). Leibniz University Hannover, Institute of Botany, Herrenhaeuser Straße 2, 30419 Hannover, Germany. cp@botanik.uni-hannover.de: 709–15. <a href="https://doi.org/10.1093/jxb/ers247">https://doi.org/10.1093/jxb/ers247</a>.</p>
</div>
<div id="ref-South-2019">
<p>South, Paul F, Amanda P Cavanagh, Helen W Liu, and Donald R Ort. 2019. “Synthetic Glycolate Metabolism Pathways Stimulate Crop Growth and Productivity in the Field.” <em>Science</em> 363 (6422). Global Change; Photosynthesis Research Unit, United States Department of Agriculture-Agricultural Research Service, Urbana, IL 61801, USA.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Department of Crop Sciences, University of Illinois, Urbana, IL 61801, USA.; Global Change; Photosynthesis Research Unit, United States Department of Agriculture-Agricultural Research Service, Urbana, IL 61801, USA. d-ort@illinois.edu.; Carl R. Woese Institute for Genomic Biology, University of Illinois, Urbana, IL 61801, USA.; Department of Crop Sciences, University of Illinois, Urbana, IL 61801, USA.; Department of Plant Biology, University of Illinois, Urbana, IL 61801, USA. <a href="https://doi.org/10.1126/science.aat9077">https://doi.org/10.1126/science.aat9077</a>.</p>
</div>
<div id="ref-Weber-2019">
<p>Weber, Andreas P.M., and Arren Bar-Even. 2019. “Update: Improving the Efficiency of Photosynthetic Carbon Reactions.” <em>Plant Physiology</em> 179 (3). American Society of Plant Biologists: 803–12. <a href="https://doi.org/10.1104/pp.18.01521">https://doi.org/10.1104/pp.18.01521</a>.</p>
</div>
<div id="ref-Xin-2015">
<p>Xin, Chang-Peng, Danny Tholen, Vincent Devloo, and Xin-Guang Zhu. 2015. “The Benefits of Photorespiratory Bypasses: How Can They Work?” <em>Plant Physiology</em> 167 (2). American Society of Plant Biologists: 574–85. <a href="https://doi.org/10.1104/pp.114.248013">https://doi.org/10.1104/pp.114.248013</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/genetic-manipulation-of-the-calvin-cycle.html" class="archive-item-link">
                Genetic Manipulation of the Calvin Cycle 
              </a>
              <span class="archive-item-date">
                2020-06-02
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a><ul>
<li><a href="#rubisco-does-not-exert-much-control-on-the-flux-of-the-calvin-cycle-unless-in-special-conditions"><span class="toc-section-number">1.1</span> Rubisco Does not Exert Much Control on the Flux of the Calvin Cycle Unless in Special Conditions</a></li>
</ul></li>
<li><a href="#other-enzymes-that-catalyse-irreversible-reactions"><span class="toc-section-number">2</span> Other Enzymes that Catalyse Irreversible Reactions</a></li>
<li><a href="#plastid-aldolase-as-an-non-regulated-enzyme-exerts-a-greater-control-than-rubisco-on-the-rate-of-photosynthesis"><span class="toc-section-number">3</span> Plastid Aldolase as an ‘Non-Regulated’ Enzyme Exerts a Greater Control than Rubisco on the Rate of Photosynthesis</a></li>
<li><a href="#transketolase-directly-controls-not-only-calvin-cycle-but-also-other-pathways"><span class="toc-section-number">4</span> Transketolase Directly Controls not Only Calvin Cycle but Also Other Pathways</a></li>
<li><a href="#conclusion"><span class="toc-section-number">5</span> Conclusion</a></li>
</ul>
</div>

<p><strong>To what extent has genetic manipulation of the Calvin cycle forced the reappraisal of our understanding of the control of metabolic pathways in plants? What do studies of these transgenic plants reveal about the integration of metabolism?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Genetic manipulation, particularly graded knock-down of enzymes in the Calvin cycle, has been exploited in the quest for understanding the control of the rate of carbon fixation step of photosynthesis. While initially it was thought that the flux of carbon fixation is limited by a few ‘regulated’ enzymes, with Rubisco being the most promising candidate, many unexpected experimental results revealed that, in reality, the flux of the Calvin cycle is influenced by multifarious factors, some of which are even due to other carbohydrate-unrelated pathways.</p>
<div id="rubisco-does-not-exert-much-control-on-the-flux-of-the-calvin-cycle-unless-in-special-conditions" class="section level2">
<h2><span class="header-section-number">1.1</span> Rubisco Does not Exert Much Control on the Flux of the Calvin Cycle Unless in Special Conditions</h2>
<p>Rubisco catalyses the carbon fixation step in the Calvin cycle and was initially widely thought to act as a control point on the rate of photosynthesis. Indeed, the enzymatic activity of Rubisco is modulated by a number of ways, such as activation by resersible carbamylation by CO<sub>2</sub> and by Rubisco activase, and inhibition by a number of sugars resulting from Rubisco’s side reactions.</p>
<p><span class="citation">1</span> produced a series of tobacco plants (<em>Nicotiana tabacum</em>) that exhibit a range of reduced amounts of Rubisco by Agrobacterium-mediated transformation of plants with antisense mRNA to the gene for the small subunit of Rubisco (<em>rbcS</em>). Such ‘antisense’ plants were used by<span class="citation"><sup>2</sup></span> to examine the control exerted by Rubisco on the rate of photosynthesis. They found that in optimum enviornmental conditions the amount of Rubisco in a leaf could be reduced by more than one-third before any significant effect on the rate of photosynthesis, with <span class="math inline">\(C^J\)</span> = 0.05-0.15 (very small). It was shown that reduction of enzyme amount is compensated for by an increase in Rubisco activation (from about 55% to almost 100%) due to 1) an increase of substrates (ribulose 1,5-bisphosphate and CO<sub>2</sub>) and decrease of products (3-phosphoglycerate), and 2) an increase of ATP/ADP ratio in the chloroplast stroma. Also, Rubisco is produced in large excess in WT plants, also explaining the lack of impact of its knock-down.</p>
<p>However, when plants were grown in low light and are then suddenly exposed to high light intensity, there was a near-proportional relation between the amount of Rubisco and the rate of photosynthesis<span class="citation"><sup>3</sup></span>. A similar result can be obtained with low CO<sub>2</sub> concentration. These experiments show that the contribution of Rubisco to the control of photosynthesis depends on both current and past conditions (CO<sub>2</sub> concentration and light intensity).</p>
</div>
</div>
<div id="other-enzymes-that-catalyse-irreversible-reactions" class="section level1">
<h1><span class="header-section-number">2</span> Other Enzymes that Catalyse Irreversible Reactions</h1>
<p>Apart from Rubisco, three other enzymes in the Calvin cycle catalyse irreversible reactions: (stromal) fructose 1,6-bisphosphatase (FBPase), sedoheptulose 1,7-bisphosphatase (SBPase) and phosphoribulokinase (PRK). These three enzymes are subjected to a same set of ‘fine’ regulatory mechanisms, all of which are exploited by the light reaction to activate the Calvin cycle:</p>
<ul>
<li><strong>pH.</strong> These enzymes have a relatively sharp pH optimium at around 8.2.</li>
<li><strong>Mg<sup>2+</sup></strong> stimulation.</li>
<li><strong>Redox state.</strong> These enzymes are inactive when disulfide bridges form between certain cysteine residues, and are activated by (reduced) thioredoxin, which reduces these disulfide bonds.</li>
</ul>
<p>Analysis of plant strains with each of these three enzymes knocked down reavealed that transformants with decreased SBPase activity show a significant but still non-proportional inhibition of photosynthesis<span class="citation"><sup>4</sup></span> in ambient conditions, while the effects of FBPase and PRK knock-down are marginal. Therefore, enzymes with similar regulatory properties does not indicate they have similar flux-control capabilities in a metabolic pathway.</p>
</div>
<div id="plastid-aldolase-as-an-non-regulated-enzyme-exerts-a-greater-control-than-rubisco-on-the-rate-of-photosynthesis" class="section level1">
<h1><span class="header-section-number">3</span> Plastid Aldolase as an ‘Non-Regulated’ Enzyme Exerts a Greater Control than Rubisco on the Rate of Photosynthesis</h1>
<p>In the Calvin cycle, aldolase carries out the synthesis of fructose 1,6-bisphohsphate (FBP) and sedoheptulose 1,7-bisphohsphate (SBP), both of which are close to equilibrium. Unlike the four enzymes described previously, aldolase’s activity is devoid of any regulatory properties and thus its rate is solely determined by the concentrations of substrates and products.</p>
<p>However, similar knock-down experiments targeting aldolase showed unexpected results, in that reduction in aldolase was found to significantly reduce the rate of photosynthesis<span class="citation"><sup>5</sup></span>. A 30% decrease of aldolase activity in potato transformants led to a small (5–10%) inhibition of ambient photosynthesis, and reduction below 30% of the wild-type activity led to a severe inhibition. The experiment is repeated with different light intensities and CO<sub>2</sub> concentrations, and it was found that the inhibition is smallest in low light and highest in high light and elevated CO<sub>2</sub> (with <span class="math inline">\(C^J\)</span> of about 0.18 and 0.56, respectively).</p>
<p>Decreased expression of aldolase inhibits photosynthesis for different reasons in low and high light. In low light, decreased expression of aldolase led to an accumulation of its substrate, triose phosphates and a depletion of of its product, which in turn causes the depletion of RuBP. In high light (plus elevated CO<sub>2</sub>), however, the triose phosphates remained very low, RuBP remained high, and PGA was higher in the transformants than in wild-type plants. This is because high irradiation inhibits starch synthesis and hence accumulation of phosphorylated intermediates, which leads to P<sub>i</sub>-limitation, which in turn restricts ATP regeneration by the light reaction. The decreased ATP concentration thhen limits conversion of 3PGA to GAP, causing 3PGA to accumulate, which in turn results in product inibition of Rubisco. Of course, the two mechanisms are not mutually exclusive, and the relative importance of each vary gradually according to the light intensity.</p>
<p>It is important to note here that, to fully rationalise the difference in the effectiveness of inhibition of the Calvin cycle in different conditions, the effect of the conditions on other pathways (starch synthesis) in the metabolic network need also be considered.</p>
</div>
<div id="transketolase-directly-controls-not-only-calvin-cycle-but-also-other-pathways" class="section level1">
<h1><span class="header-section-number">4</span> Transketolase Directly Controls not Only Calvin Cycle but Also Other Pathways</h1>
<p>Perturbation of the enzymes involved in the central carbohydrate metabolism pathways can directly influence the activity of other pathways. Transketolase is such an example.</p>
<p>In the Calvin cycle, transketolase catalyses the reactions 1) <span class="math inline">\(F6P + GAP \rightarrow E4P + X5P\)</span> and 2) <span class="math inline">\(S7P + GAP \rightarrow X5P + R5P\)</span>, both of which are reactions close to equilibrium. Tobacco tranformant with decreased expression of plastid transketolase were also produced. Like aldolase, this ‘non-regulated’ enzyme also has a relatively high flux-control coefficient, especially in saturating light and CO<sub>2</sub>. As expected, loss of transketolase activity results in an increase in the amounts of its substrate (F6P) and decrease of its products, ultimately leading to a decrease of RuBP concentration. However, it was also observed that there was a significant decrease in the levels of aromatic amino acids, intermediates of the phenylpropanoid pathway, and secondary products such as chlorogenic acid and lignin. A plausible explanation for these observations is that the flux into the shikimic acid pathway (which synthesises aromatic pathways) is limited by the decreasing erythrose 4-phosphate (one product of transketolase) concentration.</p>
<p>The multiple consequences of reducing transketolase activity highlight the extent of integration within thhe central metabolic pathways and the potential difficulties in attepting to modify flux through a specific section of the metabolic network.</p>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">5</span> Conclusion</h1>
<p>Genetic manipulation (mainly knock-outs) of some enzymes of the Calvin cycle qualitatively revealed some of the logic behind the regulation of (plant) metabolism in general: 1) loss of regulated enzymes such as Rubisco are compensated by increase in activity, 2) regulated enzymes are often produced in excess so that minor reduction in their amount does not affect the flux at all, 3) non-regulated enzymes such as aldolase are not produced in excess, and their loss restricts the flux more effectively, 4) the extent and the mechanisms of influence on Calvin cycle flux by reduction of an enzyme is affected by enviornmental conditions and the developmental history of the plant, 5) perturbation of a single reaction/enzyme can influence other, even distantly-related metabolic pathways, due to the plethora of interconnected components, 6) the flux through the Calvin cycle is determined by multiple enzymes.</p>
<p>Lots of effort has been made in modelling plant metabolism pathways such as the Calvin cycle (in order to ‘improve’ them), which proves extremely difficult. The Calvin cycle does not exist in isolation, so natually its flux is also influenced by other pathways in the complicated plant metabolism network, through shared intermediates/enzymes/regulatory molcules. Therefore, the precise prediction of its activity, in principle, also requires modelling of other, even distantly related pathways. However, it is impossible to presisely predict the behaviour of the entire metabolic network without knowing all the details (kinetic properties under different conditions, concentrations of enzymes and substrates, etc.) of every reaction in the network. Even if precise kinetic data of enzymes and concentrations of all substances can be obtained, further challenges will be encountered when considering <em>in vivo</em> modelling: unlike the test tube where all substances are relatively evenly distributed and there is only one single isolated compartment, the plant cells are extensively compartmentalised (many metabolic pathways take place in more than one compartments), and substances (substrate or enzymes) may not be evenly distributed and instead localise to specific regions of each compartment, thus altering the effective concentration. In addition, the effects of the internal and external enviornment (e.g. signalling molecules, light intensity) on gene expression of enzymes are also difficult to simulate.</p>
<p>References</p>
<div id="refs" class="references">
<div id="ref-Rodermel-1988">
<p>1. Rodermel, S. R., Abbott, M. S. &amp; Bogorad, L. Nuclear-organelle interactions: Nuclear antisense gene inhibits ribulose bisphosphate carboxylase enzyme levels in transformed tobacco plants. <em>Cell</em> <strong>55</strong>, 673–681.</p>
</div>
<div id="ref-Quick-1991">
<p>2. Quick, W. P. <em>et al.</em> Decreased ribulose-1,5-bisphosphate carboxylase-oxygenase in transgenic tobacco transformed with &quot;antisense&quot; rbcS : I. Impact on photosynthesis in ambient growth conditions. <em>Planta</em> <strong>183</strong>, 542–554.</p>
</div>
<div id="ref-Quick-1992">
<p>3. Quick, W. P. <em>et al.</em> Decreased ribulose-1,5-bisphosphate carboxylase-oxygenase in transgenic tobacco transformed with &quot;antisense&quot; rbcS : IV. Impact on photosynthesis in conditions of altered nitrogen supply. <em>Planta</em> <strong>188</strong>, 522–531.</p>
</div>
<div id="ref-Harrison-1997">
<p>4. Harrison, E. P., Willingham, N. M., Lloyd, J. C. &amp; Raines, C. A. Reduced sedoheptulose-1,7-bisphosphatase levels in transgenic tobacco lead to decreased photosynthetic capacity and altered carbohydrate accumulation. <em>Planta</em> <strong>204</strong>, 27–36 (1997).</p>
</div>
<div id="ref-Haake-1998">
<p>5. Haake, V., Zrenner, R., Sonnewald, U. &amp; Stitt, M. A moderate decrease of plastid aldolase activity inhibits photosynthesis, alters the levels of sugars and starch, and inhibits growth of potato plants. <em>The Plant Journal</em> <strong>14</strong>, 147–157 (1998).</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/coordination-of-ps-i-and-ps-ii.html" class="archive-item-link">
                Coordination of PS I and PS II 
              </a>
              <span class="archive-item-date">
                2020-05-27
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#background"><span class="toc-section-number">1</span> Background</a></li>
<li><a href="#why-is-coordination-between-photosystems-i-and-ii-needed"><span class="toc-section-number">2</span> Why is Coordination between Photosystems I and II Needed</a></li>
<li><a href="#achieve"><span class="toc-section-number">3</span> How is Coordination between PS-I and PS-II Achieved</a><ul>
<li><a href="#the-role-of-the-plastoquinone-pool"><span class="toc-section-number">3.1</span> The Role of the Plastoquinone Pool</a></li>
<li><a href="#redistribution"><span class="toc-section-number">3.2</span> Reversible Redistribution of LHC-II</a></li>
<li><a href="#transcriptional"><span class="toc-section-number">3.3</span> Transcriptional Regulation</a><ul>
<li><a href="#how-does-sig1-phosphorylation-confer-phhotosystem-promoter-specificity"><span class="toc-section-number">3.3.1</span> How does SIG1 phosphorylation confer phhotosystem promoter specificity?</a></li>
</ul></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">4</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>Why do plants co-ordinate the relative activities of Photosystems I and II and how is this co-ordination achieved?</strong></p>
<div id="background" class="section level1">
<h1><span class="header-section-number">1</span> Background</h1>
<p>Photosystems I and II are photochemical complexes that harvest light energy to drive endergonic redox reactions. Their operation in series (PS-II then PS-II), known as the Z scheme, results in generation of oxygen from water via photolysis, ATP production via electron transport chain (ETC) and production of NADPH (reduction of NADP<sup>+</sup>), which is essential for providing reducing power in anabolic reactions such as photosynthesis, by ferredoxin-NADP<sup>+</sup> reductase. PS-I can also operate alone, resulting in cyclic photophosphorylation, generating ATP only.</p>
<p>Each photosystem exists on the thylakiod membrane (PS-II mainly in grana lamellae; PS-I mainly in stroma lamellae) as a multisubunit pigment-protein complex (or a dimer thereof, in the case of PS-II) comprising a reaction centre (RC) core containing a pair of chlorophyll <em>a</em> molecules, which is surrounded by other accessory proteins as well as variable number of antenna complexes ((trimers of) LHC-II/LHC-I; also known as chlorophyll <em>a/b</em> light-harvesting complex ) containing accessory pigments (e.g. chlorophylls and carotenoids) that pass light energy to the reaction centre via resonance energy transfer. The reaction centre in PS-II preferentially absorbs red light at wavelength 680 nm and is thus referred to as P680; the reaction centre in PS-I, known as P700, preferentially absorbs far-red light.</p>
<p>The activities of two types of photosystems is determined both by the external factors, i.e. intensity and wavelength of light, as well as the the intrinsic factors which can be regulated by the plant cells, namely the number of PS-II/PS-I cores and the antenna complexes (and hence the amount of accessory pigments) associated with them. Such regulation can affect the absolute activities of PS-II and PS-I as well as their activities relative to each other.</p>
</div>
<div id="why-is-coordination-between-photosystems-i-and-ii-needed" class="section level1">
<h1><span class="header-section-number">2</span> Why is Coordination between Photosystems I and II Needed</h1>
<p>Because the PS-I and PS-II complexes are not physically coupled (so their abundance can differ) and their light absorption properties differ (PS-I prefers longer wavelength), their activities are not directly related and are often different. The activities of the two photosystems relative to each other can be regulated using the mechanisms described in Section <a href="#achieve">3</a>, but before that, the benefits of this regulation are considered here:</p>
<ol style="list-style-type: decimal">
<li><strong>Stoichiometry adjustment.</strong> PS-II and PS-I activities should ideally be equal, in order to maximise the efficiency of light utilisation for production of NADPH in vectorised electron flow, which would otherwise be limited by the slower photosystem. There is a need for this kind of adjustment when there is a change in light quality. Upon illumination by far-red light that preferentially activates PS-I (PS-I light), PS-I activity is greater than PS-II, so PS-II activity shhould be improved relative to PS-I. Conversely, illumination by PS-II light leads to hyperactive PS-II, and its activity should be reduced relative to PS-II. These two states are referred to state 1 and 2, respectively, and state transition is the process by which the plant adjusts relative activity of PS-II and PS-I in response to a change in state.</li>
<li><strong>Protection against photoinhibition.</strong> If PS-II activity exceeds PS-I activity, the rate of production of reactive oxygen species (ROS) is increased. However, recent studies on stn7 mutants argue against this role. Photodamage of PS-II is unavoidable even under nomal conditions, and it is repaired efficiently in healthy plants.</li>
<li><strong>Meeting increased requirement for ATP.</strong> If PS-I activity exceeds PS-II activity, the excess activity in PS-I results in cyclic electron flow (CEF), generating additional ATP. Theoretically, this may be exploited by the plant to meet increased demand for ATP. However, recent work revealed that CEF is controlled by the redox power and is independent of state transition: state 2 is neither sufficient nor required to promote CEF <span class="citation">(Takahashi et al. 2013)</span>.</li>
</ol>
<p>The discussion below, therefore, focuses on the role of photosystem coordination in stoichiometry adjustment.</p>
</div>
<div id="achieve" class="section level1">
<h1><span class="header-section-number">3</span> How is Coordination between PS-I and PS-II Achieved</h1>
<div id="the-role-of-the-plastoquinone-pool" class="section level2">
<h2><span class="header-section-number">3.1</span> The Role of the Plastoquinone Pool</h2>
<p>The oxidative status of the plastoquinone (PQ) pool reflects the relative activity of PS-II and PS-I, and is involved in the signalling pathway of both short-term and long-term coordination (described in Section <a href="#redistribution">3.2</a> and <a href="#transcriptional">3.3</a>, respectively).</p>
<p>Upon illumination by PS-I light (in state 1), PS-I activity is greater than PS-II, leading to oxidation of the plastoquinone pool (most plastoquinone molecules are in the oxidised form, PQ). Conversely, the plastoquinone pool becomes reduced upon preferential excitation of PS-II (Figure <a href="#fig:plastoquinone">3.1</a>).</p>
<div class="figure"><span id="fig:plastoquinone"></span>
<img src="../img/ps-i-light-ps-2-light-states.png" alt="Preferential activation of either PS-I or PS-II causes a shift in the oxidation state of the plastoquinone (PQ) pool." width="100%" />
<p class="caption">
Figure 3.1: Preferential activation of either PS-I or PS-II causes a shift in the oxidation state of the plastoquinone (PQ) pool.
</p>
</div>
</div>
<div id="redistribution" class="section level2">
<h2><span class="header-section-number">3.2</span> Reversible Redistribution of LHC-II</h2>
<p>LHC-II plays an important role in the coordination between PS-I and PS-II. The structure of one LHC-II protein has been determined by a combination of electron microscopy and X-ray crystallography (reviewed by <span class="citation">Barros and Kühlbrandt (2009)</span>). It contains three membrane-spaning <span class="math inline">\(\alpha\)</span>-helices, and binds about 15 chlorophyll <em>a</em> and <em>b</em> molecules plus a few carotenoids. LHC-II complexes of different subtypes (of which Lhcb1, 2 and 3 are the most common) spontaneously assemble into trimers (can be heterogeneous), which then associate with PS-II and PS-I to increase the efficiency of their corresponding photochemical reaction.</p>
<div class="figure"><span id="fig:lhcii-conf-change"></span>
<img src="../img/lhcii-conf-change.png" alt="Phosphorylation by STN7 causes conformational changes in LHC-II. Top: stroma; bottom: thylakoid lumen." width="100%" />
<p class="caption">
Figure 3.2: Phosphorylation by STN7 causes conformational changes in LHC-II. Top: stroma; bottom: thylakoid lumen.
</p>
</div>
<p>The LHC-II subtypes Lhcb1 and Lhcb2 (but not Lhcb3) can be phosphorylated by a specific protein kinase called STN7 (a.k.a. Stt7) at a threonine residue near the N-terminus <span class="citation">(Mullet 1983)</span>. This happens during transition to state 2, when the STN7 kinase, which is associated with a specific region of the Rieske protein of the cytochrome <em>b<sub>6</sub>f</em> complex (Cyt<em>b<sub>6</sub>f</em>), is activated upon binding of PQH<sub>2</sub> to the Q<sub>o</sub> site of Cyt<em>b<sub>6</sub>f</em>. Phosphorylation on LHC-II causes a conformational change (Figure <a href="#fig:lhcii-conf-change">3.2</a>), reduces its affinity to PS-II and increases its affinity to PS-I. The increase in affinity to PS-I is due to not only the conformational change, but also the tight association between the phosphate group and a specific arginine residue in PsaL, a protein that facilitates docking of LHC-II to PS-I core complex. As a result, LHC-II migrates from PS-II towards PS-I, thus enhancing efficiency of PS-I relative to PS-II (Figure <a href="#fig:stn7-tap38-pq-lhcii">3.3</a>). This is supported by the findings that plants without either STN7 <span class="citation">(Bellafiore et al. 2005)</span> or Lhcb1+Lhcb2 <span class="citation">(Andersson et al. 2003)</span> do not undergo state transitions. It was later shown by <span class="citation">Longoni et al. (2015)</span> that only Lhcb2 phosphorylation is relevant to state transition (Lhcb1 phosphorylation causes its exclusion from the complex). The N-terminal region of phosphorylated Lhcb2 interacts with PsaL, PsaH and PsaO proteins, and PsaO bridges PsaA (PSI reaction centre) and Lhcb2 through contacts within the membrane and on the stromal and luminal surface. PsaO is associated with two chlorophyll molecules which facilitates resonance energy transfer from LHC-II to PsaA.</p>
<p>The phosphorylation can be reversed by a specific kinase called TAP38 (a.k.a PPH1). When PS-I is over-excited relative to PS-II, STN7 is inactive, and TAP38 dephosphorylates LHC-II and causes them to return to PS-II (Figure <a href="#fig:stn7-tap38-pq-lhcii">3.3</a>).</p>
<div class="figure"><span id="fig:stn7-tap38-pq-lhcii"></span>
<img src="../img/stn7-tap38-pq-lhcii.png" alt="Oxidative status of plastoquinone determines the phosphorylation state of LHC-II, which in turn regulates relative activity of PS-I and PS-II." width="100%" />
<p class="caption">
Figure 3.3: Oxidative status of plastoquinone determines the phosphorylation state of LHC-II, which in turn regulates relative activity of PS-I and PS-II.
</p>
</div>
<!-- under High light conditions by response to FdH~2~) -->
</div>
<div id="transcriptional" class="section level2">
<h2><span class="header-section-number">3.3</span> Transcriptional Regulation</h2>
<p>Regulation of relative activity of PS-I and PS-II can also be achieved by controlling the amount of transcription of PS-I genes (e.g. <em>psaA</em>) and PS-II genes (<em>psbA</em>).</p>
<p><span class="citation">Puthiyaveetil et al. (2008)</span> described a sensor kinase called chloroplast sensor kinase (CSK) that couples oxidation state of PQ to transcriptional control of psbA. As shown in Figure <a href="#fig:csk">3.4</a>, PS-I light causes the PQ pool to be oxidised, and then CSK, which is activated by PQ (the PQ analogue, DBMIB (2,5-dibromo-3-methyl-5-isopropyl-<em>p</em>-benzoquinone) binds to CSK similarly), autophosphorylates itself, and then phosphorylates sigma factor 1 (SIG1) subunit of PEP (RNA polymerase). This results in specific repression of PS-I genes (<em>psaA</em>), thus decreasing the stoichiometry of PS-I to PS-I. Conversely, CSK is inactive with a reduced PQ pool under PS-II light, and SIG1 phosphatase removes the repression of PS-I gene transcription, thus increasing the stoichiometry of PS-I to PS-II.</p>
<div class="figure"><span id="fig:csk"></span>
<img src="../img/psi-ii-csk-tx.png" alt="Transcriptional control of PS-I genes mediated by PQ, CSK, SIG1 and PEP." width="100%" />
<p class="caption">
Figure 3.4: Transcriptional control of PS-I genes mediated by PQ, CSK, SIG1 and PEP.
</p>
</div>
<div id="how-does-sig1-phosphorylation-confer-phhotosystem-promoter-specificity" class="section level3">
<h3><span class="header-section-number">3.3.1</span> How does SIG1 phosphorylation confer phhotosystem promoter specificity?</h3>
<p>Bacterial sigma factors contain the region 1.1 at the extreme N-terminus. Region 1.1 is a poorly-conserved, acidic region, which is associated with the positively charged catalytic cavity of RNA polymerase in the closed (non-transcribing) conformation. To begin transcription, (negatively-charged) DNA must replace region 1.1, and the efficiency of this replacement depends on how well the -10 and -35 elements of the promoter match the consensus sequence.</p>
<p>Chloroplast SIG1 also has an N-terminal un-conserved region (UCR), but it contains both acidic and basic patches. It is proposed that, in PS-II light, the basic patch and the acidic patch of the un-phosphorylated SIG1 form a complex through electrostatic interactions, thus preventing the association of the acidic patch with the basic catalytic cavity.
In PS-I light, phosphorylation with the basic patch disfavours its complex formation with the acidic patches, thus allowing the acidic patch to associate with the catalytic cavity, where it acts as the gatekeeper and discriminates between strong and weak promoters.</p>
<p>By comparing the -35 and -10 regions of the promoters of <em>psaA</em> and <em>psbA</em> to the consensus sequence, it can be deduced that <em>psbA</em> (PS-II gene) has a stronger promoter and is therefore preferentially transcribed over PS-I under PS-I light, when SIG1 is phosphorylated and the discrimination mechanism is functioning.</p>
</div>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">4</span> Conclusion</h1>
<p>Plants need to coordinate the efficiency of PS-II and PS-I in order to maximise the utilisation of light energy in linear electron flow. Upon change in light quality, activities of PS-II and PS-I are transiently unsyncronised, leading to change in oxidative state of PQ (i.e. accumulation of PQ or PQH<sub>2</sub>). This triggers two mechanisms used to regain the balance. First, PQH<sub>2</sub> binding to Cyt<em>b<sub>6</sub>f</em> activates STN7, which in turn causes phosphorylation of LHC-II and its migration from PS-II to PS-I. Second, PQ activates CSK, which phosphorylates SIG1, causing PS-II genes to be preferentially transcribed.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Andersson-2003">
<p>Andersson, Jenny, Mark Wentworth, Robin G Walters, Caroline A Howard, Alexander V Ruban, Peter Horton, and Stefan Jansson. 2003. “Absence of the Lhcb1 and Lhcb2 Proteins of the Light-Harvesting Complex of Photosystem Ii - Effects on Photosynthesis, Grana Stacking and Fitness.” <em>Plant J</em> 35 (3). UmeåPlant Science Centre, Department of Plant Physiology, UmeåUniversity, S-901 87 Umeå, Sweden. jenny.andersson@plantphys.umu.se: 350–61. <a href="https://doi.org/10.1046/j.1365-313x.2003.01811.x">https://doi.org/10.1046/j.1365-313x.2003.01811.x</a>.</p>
</div>
<div id="ref-Barros-2009">
<p>Barros, Tiago, and Werner Kühlbrandt. 2009. “Crystallisation, Structure and Function of Plant Light-Harvesting Complex Ii.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1787 (6): 753–72. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2009.03.012">https://doi.org/https://doi.org/10.1016/j.bbabio.2009.03.012</a>.</p>
</div>
<div id="ref-Bellafiore-2005">
<p>Bellafiore, Stéphane, Frédy Barneche, Gilles Peltier, and Jean-David Rochaix. 2005. “State Transitions and Light Adaptation Require Chloroplast Thylakoid Protein Kinase Stn7.” <em>Nature</em> 433 (7028). Department of Molecular Biology, University of Geneva, 30, Quai Ernest Ansermet, 1211 Geneva, Switzerland.: 892–95. <a href="https://doi.org/10.1038/nature03286">https://doi.org/10.1038/nature03286</a>.</p>
</div>
<div id="ref-Buchanan-2015">
<p>Buchanan, Bob B., Wilhelm Gruissem, and Russel L. Jones. 2015. <em>Biochemistry &amp; Molecular Biology of Plants</em>. John Wiley &amp; Sons.</p>
</div>
<div id="ref-Holloszy-1996">
<p>Holloszy, John O., and Wendy M. Kohrt. 1996. “Regulation of Carbohydrate and Fat Metabolism During and After Exercise.” <em>Annual Review of Nutrition</em> 16 (1): 121–38. <a href="https://doi.org/10.1146/annurev.nu.16.070196.001005">https://doi.org/10.1146/annurev.nu.16.070196.001005</a>.</p>
</div>
<div id="ref-Horowitz-2000">
<p>Horowitz, Jeffrey F, and Samuel Klein. 2000. “Lipid metabolism during endurance exercise.” <em>The American Journal of Clinical Nutrition</em> 72 (2): 558S–563S. <a href="https://doi.org/10.1093/ajcn/72.2.558S">https://doi.org/10.1093/ajcn/72.2.558S</a>.</p>
</div>
<div id="ref-Jensen-2012">
<p>Jensen, Thomas E., and Erik A. Richter. 2012. “Regulation of Glucose and Glycogen Metabolism During and After Exercise.” <em>The Journal of Physiology</em> 590 (5): 1069–76. <a href="https://doi.org/10.1113/jphysiol.2011.224972">https://doi.org/10.1113/jphysiol.2011.224972</a>.</p>
</div>
<div id="ref-Longoni-2015">
<p>Longoni, Paolo, Damien Douchi, Federica Cariti, Geoffrey Fucile, and Michel Goldschmidt-Clermont. 2015. “Phosphorylation of the Light-Harvesting Complex Ii Isoform Lhcb2 Is Central to State Transitions.” <em>Plant Physiol</em> 169 (4): 2874–83. <a href="https://doi.org/10.1104/pp.15.01498">https://doi.org/10.1104/pp.15.01498</a>.</p>
</div>
<div id="ref-Maarbjerg-2011">
<p>Maarbjerg, S. J., L. Sylow, and E. A. Richter. 2011. “Current Understanding of Increased Insulin Sensitivity After Exercise – Emerging Candidates.” <em>Acta Physiologica</em> 202 (3): 323–35. <a href="https://doi.org/10.1111/j.1748-1716.2011.02267.x">https://doi.org/10.1111/j.1748-1716.2011.02267.x</a>.</p>
</div>
<div id="ref-Mittendorfer-2003">
<p>Mittendorfer, Bettina, and Samuel Klein. 2003. “Physiological Factors That Regulate the Use of Endogenous Fat and Carbohydrate Fuels During Endurance Exercise.” <em>Nutrition Research Reviews</em> 16 (1). Cambridge University Press: 97–108. <a href="https://doi.org/10.1079/NRR200357">https://doi.org/10.1079/NRR200357</a>.</p>
</div>
<div id="ref-Mullet-1983">
<p>Mullet, J E. 1983. “The Amino Acid Sequence of the Polypeptide Segment Which Regulates Membrane Adhesion (Grana Stacking) in Chloroplasts.” <em>J Biol Chem</em> 258 (16): 9941–8.</p>
</div>
<div id="ref-Puthiyaveetil-2008">
<p>Puthiyaveetil, Sujith, T Anthony Kavanagh, Peter Cain, James A Sullivan, Christine A Newell, John C Gray, Colin Robinson, Mark van der Giezen, Matthew B Rogers, and John F Allen. 2008. “The Ancestral Symbiont Sensor Kinase Csk Links Photosynthesis with Gene Expression in Chloroplasts.” <em>Proc Natl Acad Sci U S A</em> 105 (29): 10061–6. <a href="https://doi.org/10.1073/pnas.0803928105">https://doi.org/10.1073/pnas.0803928105</a>.</p>
</div>
<div id="ref-Richter-2009">
<p>Richter, Erik A., and Neil B. Ruderman. 2009. “AMPK and the biochemistry of exercise: implications for human health and disease.” <em>Biochemical Journal</em> 418 (2): 261–75. <a href="https://doi.org/10.1042/BJ20082055">https://doi.org/10.1042/BJ20082055</a>.</p>
</div>
<div id="ref-Taiz-2010">
<p>Taiz, Lincoln, and Eduardo Zeiger. 2010. <em>Plant Physiology</em>. Sinauer Associates.</p>
</div>
<div id="ref-Takahashi-2013">
<p>Takahashi, Hiroko, Sophie Clowez, Francis-André Wollman, Olivier Vallon, and Fabrice Rappaport. 2013. “Cyclic Electron Flow Is Redox-Controlled but Independent of State Transition.” <em>Nat Commun</em> 4. Institut de Biologie Physico-Chimique, UMR 7141 CNRS-UPMC, 13 rue P et M Curie, 75005 Paris, France.: 1954. <a href="https://doi.org/10.1038/ncomms2954">https://doi.org/10.1038/ncomms2954</a>.</p>
</div>
<div id="ref-Thompson-2012">
<p>Thompson, Dylan, Fredrik Karpe, Max Lafontan, and Keith Frayn. 2012. “Physical Activity and Exercise in the Regulation of Human Adipose Tissue Physiology.” <em>Physiological Reviews</em> 92 (1): 157–91. <a href="https://doi.org/10.1152/physrev.00012.2011">https://doi.org/10.1152/physrev.00012.2011</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/sensing-energy-status-during-exercise.html" class="archive-item-link">
                Fuel Selection During Exercise 
              </a>
              <span class="archive-item-date">
                2020-05-25
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#overview"><span class="toc-section-number">2</span> Overview</a><ul>
<li><a href="#glycogen-and-glucose"><span class="toc-section-number">2.1</span> Glycogen and Glucose</a><ul>
<li><a href="#blood-glucose"><span class="toc-section-number">2.1.1</span> Blood Glucose</a></li>
<li><a href="#regulation-of-glycogenolysis"><span class="toc-section-number">2.1.2</span> Regulation of Glycogenolysis</a></li>
</ul></li>
<li><a href="#fatty-acids"><span class="toc-section-number">2.2</span> Fatty Acids</a><ul>
<li><a href="#adipose-tissue-lipolysis-and-plasma-nefa"><span class="toc-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</a></li>
<li><a href="#intramuscular-triacylglycerol-imtg"><span class="toc-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</a></li>
<li><a href="#plasma-tg-vldl-tg"><span class="toc-section-number">2.2.3</span> Plasma TG (VLDL-TG)</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p><strong>What effects do the intensity and duration of exercise have on fuel selection by mammalian muscle?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>During exercise, glucose and fatty acids are the fuels used by the muscle. Their origin and relative contribution depend on the intensity and duration of exercise, as well as on characteristics of individuals such as the level of physical training.</p>
</div>
<div id="overview" class="section level1">
<h1><span class="header-section-number">2</span> Overview</h1>
<p>As shown in Figure 1, muscle contraction during exercise is powered by oxidation of glucose and fatty acids derived from a number of sources.</p>
<div class="figure"><span id="fig:powering-the-muscle"></span>
<img src="../img/powering-the-muscle.png" alt="Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6)." width="100%" />
<p class="caption">
Figure 2.1: Sources of fuel during exercise. (1) white adipose tissue lipolysis; (2) intramuscular triglycerides (IMTG); (3) plasma TG (mainly contained in VLDL synthesised by the liver); (4) mucle glycogenolysis; blood glucose derived from liver glycogenolysis (5) and gluconeogenesis (6).
</p>
</div>
<p>Immediately after the onset of exercise (0-30 s) energy is derived predominantly from anaerobic respiration. After this period, both excercise intensity and duration determine the choice of fuel used for oxidation. As shown in Figure 2, during the first 30 minutes of a low-intensity exercise bout, the vast majority of fuel used by muscle is plasma NEFA (non-esterified fatty acids) originated from adipose lipolysis with tiny contributions from IMTG (intramuscular triglyceride) and plasma glucose. For moderately intense exercise, the contribution from fat is lower: plasma NEFA and IMTG each represent about 25% of total energy expenditure, and glycogen and plasma glucose account for the remaining 40% and 10%. For high-intensity exercise, the contribution from muscle glycogen is even higher (60%), and fat and glucose represents 30% and 10% of the total, respectively. Overall, as exercise intensity increases, carbohydrates (especially muscle glycogen) progressively becomes the preferred fuel over triglyceride-derived fuel. As for TG-derived fuel, the ratio of IMTG to plasma NEFA is greatest during moderately intense exercise and lowest during low-intensity excercise. These are explained in detail later.</p>
<p>Figure 2 also shows that, during prolonged (low- or moderate-intensity) exercise, the contribution from plasma NEFA increases and the contribution from other fuels decreases, regardless of exercise intensity. This is due to the large amount of TG reserve in adipose tissue and the gradual depletion of other fuels.</p>
<div class="figure"><span id="fig:exercise-choice-of-fuel"></span>
<img src="../img/exercise-choice-of-fuel.png" alt="Effects of exercise intensity and duration on choice of substrates. 'Percentage of VO~2max~', where VO~2max~ is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO~2max~ is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity." width="100%" />
<p class="caption">
Figure 2.2: Effects of exercise intensity and duration on choice of substrates. ‘Percentage of VO<sub>2max</sub>’, where VO<sub>2max</sub> is the highest rate of ATP generation via oxidative phosphorylation attainable, is a normalised parameter used to measure relative exercise intensity. Exercise requiring less than 30% VO<sub>2max</sub> is considered low-intensity; 50%-70%, moderate intensity; 70-85%, high-intensity.
</p>
</div>
<div id="glycogen-and-glucose" class="section level2">
<h2><span class="header-section-number">2.1</span> Glycogen and Glucose</h2>
<div id="blood-glucose" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Blood Glucose</h3>
<p>During exercise, catecholamines stimulates pancreatic <span class="math inline">\(\alpha\)</span> cells to secrete glucagon, which in turn promtotes hepatic glycogenolysis and gluconeogenesis. Both processes release glucose into the bloodstream.</p>
<p>Blood glucose is the predominant source of carbohydrate for oxidation in muscle. Its absolute amount of usage increases as excercise intensity increases (though proportionally remains largely unchanged), which is mediated by an increaed sensitivity to insulin (in part due to utilisation of IMTG) and thus increased translocation of GLUT4 to the cell surface membrane.</p>
</div>
<div id="regulation-of-glycogenolysis" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Regulation of Glycogenolysis</h3>
<p>Glycogen phosphorylase (GP) and phosphorylase kinase (PhK) are central to the regulation of glycogenolysis. Activated PhK phosphorylates GP, thus converting it from the inactive <em>b</em> form to the active <em>a</em> form. GP in its <em>a</em> form then phosphorolyses glycogen, releasing glucose 1-phosphase available for oxidation.</p>
<p>PhK is activated (1) by phosphorylation on <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> subunits by PKA and (2) by binding of Ca<sup>2+</sup> to <span class="math inline">\(\delta\)</span> subunit. PKA is activated upon catecholamine (mostly adrenaline) binding to cell surface receptors (via G<sub>s</sub>, adenylyl cyclase, and cAMP), and Ca<sup>2+</sup> is released upon muscle contraction (stimulated by acetylcholine).</p>
<p>Ca<sup>2+</sup> released upon muscle contraction results in a burst of glycogenolysis. The large amount of glycogen breakdown exceeds muscle’s energy requirement and capacity of TCA cycle, resulting in accumulation of lactate. This probably explains the short period of anaerobic respiration at the onset of exercise. After a few minutes, the activation of phosphorylase reverses by a mechanism that is not fully understood.</p>
<p>Myocytes also respond, via <span class="math inline">\(\beta\)</span> adrenergic receptors, to catecholamines that are produced proportionally to exercise intensity. This results in activation of PKA (via G<sub>s</sub>, AC and cAMP), which in turn activates glycogenolysis.</p>
<p>The rate of glycogenolysis also depends on availability of P<sub>i</sub>, the substrate of glycogen phosphorylase. At rest and during mild exercise, fatty acid oxidation is capable of generating sufficient ATP, resulting in low P<sub>i</sub> concentration. Availability of P<sub>i</sub> increases as exercise intensity increases because the rate of ATP hydrolysis exceeds the rate of convertion from ADP and P<sub>i</sub> back to ATP by oxidation of fatty acids (and plasma glucose).</p>
</div>
</div>
<div id="fatty-acids" class="section level2">
<h2><span class="header-section-number">2.2</span> Fatty Acids</h2>
<p>During exercise, the usage of fatty acids has an important role of delaying the onset of glycogen depletion and hypoglycemia. Its contribution to total energy expenditure increases as excercise proceeds. However, its usage is limited during strenuous exercise.</p>
<div id="adipose-tissue-lipolysis-and-plasma-nefa" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Adipose Tissue Lipolysis and Plasma NEFA</h3>
<p>Compared to resting conditions, adipose tissue lipolysis occurs at a higher rate during mild- or moderate-intensity exercise. This is mainly mediated by increased <span class="math inline">\(\beta\)</span>-adrenergic receptor activation by catecholamines released by the adrenal gland upon sympathetic nerve stimulation. Exercise also increases blood flow in adipose tissue (which prevents toxic regional accumulation of fatty acids) and in muscle, thus promoting transport of NEFA to myocytes. Passive uptake of NEFA by myocytes, which is proportional to plasma NEFA concentration, is thus increased. AMPK, which is activated upon transition from rest to exercise and during prolonged exercise, phosphorylates ACC2 and malonyl-CoA decarboxylase, inhibiting the former and activating the latter, thus resulting in a decrease of malonyl-CoA concentration and hence releasing CPT-1 from inhibition. This promotes transport of fatty acids into mitochondria for <span class="math inline">\(\beta\)</span>-oxidation in muscle cells.</p>
<p>As the exercise intensity continue to increase, however, oxidation of NEFA derived from adipose tissue lipolysis in muscle decreases. This is in part caused by decreased rate of lipolysis due to lactate (through activation of an orphan GPCR, GPR81). Lactate can also increase re-esterification of fatty acids to TG in the liver. Raising plasma fatty acid concentrations by intravenously infusing a lipid emulsion and heparin during the exercise bout increases plasma NEFA oxidation by about 30% but does not completely restore it to the rate observed during moderate-intensity exercise. Thus, high-intensity exercise also decreases the capacity of skeletal muscle to oxidise fatty acids, which is in part caused by increased glycogen catabolism. As more acetyl-CoA concentration are produced via glycogen catabolism, they are converted to malonyl-CoA by ACC2 located on the outer mitochondrial membrane, and the malonyl-CoA then inhibits CPT-1 and hence fatty acid oxidation.</p>
</div>
<div id="intramuscular-triacylglycerol-imtg" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Intramuscular Triacylglycerol (IMTG)</h3>
<p>Within myocytes, there are lipid droplets that stores TG. Their usage during exercise can be calculated by subtracting plasma fatty acid oxidation (determined by isotope tracers) from whole-body fat oxidation (determined by indirect calorimetry).</p>
<p>IMTG use is stimulated by catecholamines via <span class="math inline">\(\beta\)</span> adrenergic receptors. This is supported by the finding that pharmarcological blockage of <span class="math inline">\(\beta_1 + \beta_2\)</span> prevents the usage of IMTG.</p>
</div>
<div id="plasma-tg-vldl-tg" class="section level3">
<h3><span class="header-section-number">2.2.3</span> Plasma TG (VLDL-TG)</h3>
<p>The liver takes up plasma NEFA and glycerol and use some of them to resynthesise TG. These TG are packaged in VLDL and released into the bloodstream. Muscle can break down some VLDL by lipoprotein lipase and oxidise the resulting fatty acids. However, this only contributes to a small fraction of energy usage during exercise and might be more important for replenishing IMTG stores after exercise.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/the-aetiology-of-diabetes-mellitus-and-insulin-resistance.html" class="archive-item-link">
                Insulin and Ketone Bodies 
              </a>
              <span class="archive-item-date">
                2020-05-18
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#regulation-of-ketogenesis-by-insulin"><span class="toc-section-number">2</span> Regulation of Ketogenesis by Insulin</a><ul>
<li><a href="#adipocyte-lipolysis"><span class="toc-section-number">2.1</span> Adipocyte Lipolysis</a></li>
<li><a href="#regulation-of-cpt1"><span class="toc-section-number">2.2</span> Regulation of CPT1</a></li>
<li><a href="#channeling-of-acetyl-coa-into-ketogenesis"><span class="toc-section-number">2.3</span> Channeling of Acetyl-CoA into Ketogenesis</a></li>
<li><a href="#mitochondrial-hmg-coa-synthase"><span class="toc-section-number">2.4</span> Mitochondrial HMG-CoA Synthase</a></li>
</ul></li>
<li><a href="#diabetes-and-ketone-bodies"><span class="toc-section-number">3</span> Diabetes and Ketone Bodies</a><ul>
<li><a href="#aetiology-of-diabetes"><span class="toc-section-number">3.1</span> Aetiology of Diabetes</a></li>
<li><a href="#overproduction-of-ketone-bodies"><span class="toc-section-number">3.2</span> Overproduction of Ketone Bodies</a></li>
<li><a href="#consequences-of-ketoacidosis"><span class="toc-section-number">3.3</span> Consequences of Ketoacidosis</a></li>
</ul></li>
</ul>
</div>

<p><strong>How does insulin regulate the pathways for the conversion of body fat into ketone bodies? Why are ketone bodies overproduced in uncontrolled diabetes and what are the metabolic consequences.</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Production of ketone bodies (ketogenesis) occurs in the liver (and the kidneys) as an response to prevent hypoglycemia during starvation, and this process is normally suppressed and stimulated by insulin and glucagon, respectively. In addition to its physiological role in fasting, ketogenesis also has high activity in diabetes patients, which may lead to the potentially lethal condition called diabetic ketoacidosis.</p>
</div>
<div id="regulation-of-ketogenesis-by-insulin" class="section level1">
<h1><span class="header-section-number">2</span> Regulation of Ketogenesis by Insulin</h1>
<p>Insulin suppresses ketogenesis by several mechanisms.</p>
<p>The steps of ketogenesis are breifly shown in Figure <a href="#fig:ketogenesis-brief-diagram">2.1</a>. There are three control points by which ketogenesis can be regulated by insulin: (1) adipocyte lipolysis, (2) mitochondrial fatty acid entry in ketogenic cells (mainly hepatocytes), (3) channeling of Acetyl-CoA (AcCoA) into the ketogenic pathway, and (4) mitochondrial 3-hydroxy-3-methylglutaryl-CoA synthase (HMG-CoA synthase).</p>
<div class="figure"><span id="fig:ketogenesis-brief-diagram"></span>
<img src="../img/ketogenesis-brief-diagram.jpg" alt="Production of ketone bodies from adipose-derived fatty acids." width="100%" />
<p class="caption">
Figure 2.1: Production of ketone bodies from adipose-derived fatty acids.
</p>
</div>
<div id="adipocyte-lipolysis" class="section level2">
<h2><span class="header-section-number">2.1</span> Adipocyte Lipolysis</h2>
<p>The major source molecules for ketogenesis are fatty acids obtained via lipolysis in white adipose tissues. Lipolysis involves the sequential hydrolysis of triacylglycerol (TAG) into 3 non-esterified fatty acids (NEFA) and glycerol by adipocyte triglyceride lipase (ATGL), hormone-sensitive lipase (HSL) and monoglyceride lipase (MGL). Activation of ATGL and HSL are dependent on PKA, which is activated by cAMP that are produced when catecholamines bind to cell-surface <span class="math inline">\(\beta\)</span> adrenergic receptors.</p>
<p>Insulin is a potent inhibitor of lipolysis. It activates PKB via IRS and PI3K, and PKB in turn phosphorylates and activates phosphodiesterase 3B (PDE-3B). PDB-3B decyclises cAMP, thus diminishing PKA, and thus lipolytic, activity (Figure <a href="#fig:ketogenesis-insulin-detail">2.2</a>). During starvation, insulin levels are low, and adipocyte lipolysis is released from inhibition. Adipocytes are then able to release NEFA into bloodstream.</p>
<div class="figure"><span id="fig:ketogenesis-insulin-detail"></span>
<img src="../img/ketogenesis-insulin-detail.jpg" alt="Left, regulation of lipolysis by insulin in adipocytes; right, regulation of uptake of long-chain fatty acids into mitochondria by insulin." width="100%" />
<p class="caption">
Figure 2.2: Left, regulation of lipolysis by insulin in adipocytes; right, regulation of uptake of long-chain fatty acids into mitochondria by insulin.
</p>
</div>
</div>
<div id="regulation-of-cpt1" class="section level2">
<h2><span class="header-section-number">2.2</span> Regulation of CPT1</h2>
<p>Once NEFA molecules arrive at hepatocytes, they are first broken down into AcCoA via <span class="math inline">\(\beta\)</span>-oxidation before conversion into ketone bodies. Long chain fatty acids needs the carnitine shuttle to enter mitochondria where <span class="math inline">\(\beta\)</span>-oxidation takes place, and thus the first enzyme of this shuttling system, carnitine palmitoyl transferase 1 (CPT1) is the rate limiting step and is under control. CPT1 is inhibited by malonyl CoA, which is the lipogenetic precursor produced by acetyl-CoA carboxylase (ACC). Insulin activates protein phosphotase 2A (PP2A, which is also activated by xylulose 5-phosphate during fed state), which dephosphorylates and activates ACC, thus promoting lipogenesis and suppressing <span class="math inline">\(\beta\)</span>-oxidation and ketogenesis (Figure <a href="#fig:ketogenesis-insulin-detail">2.2</a>).</p>
<p>During starvation when insulin levels are low, the high fatty acid concentration in hepatocytes activates the transcription factor PPAR<span class="math inline">\(\alpha\)</span> (activated by direct binding of fatty acids to PPAR<span class="math inline">\(\alpha\)</span>), which in turn promotes transcription of several enzymes involved in lipid catabolism, including CPT1.</p>
</div>
<div id="channeling-of-acetyl-coa-into-ketogenesis" class="section level2">
<h2><span class="header-section-number">2.3</span> Channeling of Acetyl-CoA into Ketogenesis</h2>
<p>AcCoA produced from NEFA via <span class="math inline">\(\beta\)</span>-oxidation can either enter the Krebs cycle or the ketogenesis pathway. When the rate of production of AcCoA exceeds the capacity of citrate synthesis (when there is insufficient oxaloacetate), the surplus becomes the substrate for ketogenesis. This situation occurs during starvation, when insulin levels are low and glucagon levels are high, and gluconeogenesis active. Glucagon has several mechanisms to promote gluconeogenesis (and suppress glycolysis). For example, it activates PKA, which in turn activates the transcription factor CREB, which promotes transcription of gluconeogenenic enzymes such as PEP carboxykinase and glucose 6-phosphatase. Insulin, on the contary, suppresses gluconeogenesis. For instance, it activates PKB, which in turn phosphorylates and thus inactivates FOXO1, which is a transcription factor that promotes the transcription of a similar set of gluconeogenic enzymes. Insulin inhibits gluconeogenesis and thus ketogenesis.</p>
</div>
<div id="mitochondrial-hmg-coa-synthase" class="section level2">
<h2><span class="header-section-number">2.4</span> Mitochondrial HMG-CoA Synthase</h2>
<p>Mitochondrial HMG-CoA catalyses the irreversible step in ketogenesis, and therefore is a site of control.</p>
<p>Insulin promotes glycolysis and increases flux through the Krebs cycle. One of the Krebs cycle intermediate, succinyl-CoA, inactivates mitochondrial HMG-CoA by succinylation. Glucagon, on the contrary, activates gluconeogenesis, which causes depletion of Krebs cycle intermediates including succinyl-CoA, thus activating ketogenesis.</p>
<p>In addition, the transcription factor PPAR<span class="math inline">\(\alpha\)</span> described previously also activates transcription of HMG-CoA synthase.</p>
</div>
</div>
<div id="diabetes-and-ketone-bodies" class="section level1">
<h1><span class="header-section-number">3</span> Diabetes and Ketone Bodies</h1>
<div id="aetiology-of-diabetes" class="section level2">
<h2><span class="header-section-number">3.1</span> Aetiology of Diabetes</h2>
<p>Diabetes is characterised by insufficient insulin activity, which could be caused by either lack of insulin production or insulin resistance in target cells. Most diabetes cases fall into two categories, type 1 and type 2.</p>
<ul>
<li>Type 1 diabetes is caused by autoimmunity agaist pancreatic <span class="math inline">\(\beta\)</span> cells, which leads to their death and inadequacy of insulin. The primary risk factor for type 1 diabetes is genetic, although several environmental risk factors, such as infection by viruses that show tropism to pancreatic islets.</li>
<li>Type 2 diabetes begins with insulin resistance, but reduction of insulin production and number of <span class="math inline">\(\beta\)</span> cells may also be implicated as the disease advances. Type 2 diabetes also has genetic risk factors, but environmental factors, such as sedentarity and obesity, are more important.</li>
</ul>
<p>Whichever the type, ketogenesis is hyperactive in diabetes patients (although generally more severe in type 1) due to inadequate insulin activity.</p>
</div>
<div id="overproduction-of-ketone-bodies" class="section level2">
<h2><span class="header-section-number">3.2</span> Overproduction of Ketone Bodies</h2>
<p>Given the fact that insulin suppresses ketogenesis and insulin activity is decreased in diabetic patients, it is easy to comprehend why ketone bodies are overproduced in diabetic individuals.</p>
<p>Lack of insulin activity upon adipocytes leads to uncontrolled lipolysis and subsequent release of non-esterified fatty acids. As fatty acids enter the liver, they are shuttled into mitochondria, broken down into acetyl-CoA via <span class="math inline">\(\beta\)</span>-oxidation and converted to ketone bodies. This process is facilitated by the low insulin activity, which activates gluconeogenesis (e.g. via FOXO1-induced transcription of gluconeogenenic enzymes and phosphorylation of PFK2/FBPase2 bifunctional enzyme), which in turn releases the rate-limiting enzyme, HMG-CoA from succinylation inhibition and reduces usage of acetyl-CoA in Krebs cycle. The low insulin activity also inhibits ACC, thus activating CPT1 and hence <span class="math inline">\(\beta\)</span>-oxidation. The high fatty acid concentration also leads to activation of PPAR<span class="math inline">\(\alpha\)</span>, which promotes transcription of ketogenic enzymes such as CPT-1 and HMG-CoA synthatase.</p>
</div>
<div id="consequences-of-ketoacidosis" class="section level2">
<h2><span class="header-section-number">3.3</span> Consequences of Ketoacidosis</h2>
<p>Diabetic ketoacidosis (DKA) the most common acute emergency in diabetes patients. It is a potentially lethal condition that is characterised by lowered blood pH due to accumulation of acidic ketone bodies (acetoacetate and 3-hydroxybutyrate). DKA is caused by lack of insulin and thus occurs mostly in uncontrolled type 1 diabetes (DKA is not commonly seen in type 2 diabetes, because even a tiny amount of insulin activity would inhibit ketogenesis; however, there is a small proportion of “ketosis-prone type 2 diabetes”). Often, DKA is triggered by stress or illness, which are associated with increase of ketogenesis-promoting hormones, glucagon and catecholamines.</p>
<p>Ketoacidosis has several consequences, as described below.</p>
<ul>
<li>Kussmaul breathing: initally, bicarbonate ions buffur protons dissociated from ketone bodies, but this buffering system quickly becomes overwhelmed. To compensate, blood CO<sub>2</sub> concentration is reduced by deep, regular breaths known as Kussmaul breathing.</li>
<li>Hypovolaemia: hyperglycemia and ketonaemia results in excretion of glucose and ketone bodies (which normally should be absorbed) in urine. Since they are osmotically active, their presence in urine causes osmotic diuresis, which further leads to hypovolaemia (decrease in body fluid volume).</li>
<li>Electrolyte disturvance: as the the plasma pH falls, extracellular protons are exchanged for intracellular potassium ions. In addition, owing to hypovolaemia, the aldosterone concentration increases to retain water, but this hormone also leads to excretion of K<sup>+</sup> and retention of Na<sup>+</sup>.</li>
<li>The low plasma pH impair with many biological functions, especially in the brain, where potentially life-threatening coma and cerebral oedema may occur.</li>
</ul>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/glucagon-and-insulin.html" class="archive-item-link">
                Glucagon and Insulin 
              </a>
              <span class="archive-item-date">
                2020-05-11
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#initial-steps-of-glucagon-and-insulin-signalling"><span class="toc-section-number">2</span> Initial Steps of Glucagon and Insulin Signalling</a><ul>
<li><a href="#insulin-signalling"><span class="toc-section-number">2.1</span> Insulin Signalling</a></li>
<li><a href="#glucagon-signalling"><span class="toc-section-number">2.2</span> Glucagon Signalling</a></li>
</ul></li>
<li><a href="#glucose-metabolism"><span class="toc-section-number">3</span> Glucose Metabolism</a><ul>
<li><a href="#glycogen-metabolism-in-liver"><span class="toc-section-number">3.1</span> Glycogen Metabolism in Liver</a><ul>
<li><a href="#glucagon-activates-glycogenolysis-in-liver"><span class="toc-section-number">3.1.1</span> Glucagon Activates Glycogenolysis in Liver</a></li>
<li><a href="#insulin-activates-glycogen-synthesis-in-liver"><span class="toc-section-number">3.1.2</span> Insulin Activates Glycogen Synthesis in Liver</a></li>
</ul></li>
<li><a href="#glycolysis-and-gluconeogeneis-in-liver"><span class="toc-section-number">3.2</span> Glycolysis and Gluconeogeneis in Liver</a><ul>
<li><a href="#short-term-control-via-pfk2pbpase2-bifunctional-enzyme"><span class="toc-section-number">3.2.1</span> Short-Term Control via PFK2/PBPase2 Bifunctional Enzyme</a></li>
<li><a href="#long-term-transcriptional-control"><span class="toc-section-number">3.2.2</span> Long-Term Transcriptional Control</a></li>
</ul></li>
<li><a href="#glucose-uptake-into-muscle-cells"><span class="toc-section-number">3.3</span> Glucose Uptake into Muscle Cells</a></li>
</ul></li>
<li><a href="#lipid-metabolism"><span class="toc-section-number">4</span> Lipid Metabolism</a><ul>
<li><a href="#lipolysis-in-adipocytes"><span class="toc-section-number">4.1</span> Lipolysis in Adipocytes</a></li>
<li><a href="#tag-synthesis-in-liver"><span class="toc-section-number">4.2</span> TAG Synthesis in Liver</a></li>
<li><a href="#tag-re-synthesis-in-adipocytes"><span class="toc-section-number">4.3</span> TAG (Re-)Synthesis in Adipocytes</a></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">5</span> Conclusion</a></li>
</ul>
</div>

<p><strong>“Glucagon and insulin affect the body in opposite ways.” To what extent is this statement true?</strong></p>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>Glucagon and insulin are peptide hormones secreted by the endocrine cells (<span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span> cells, respectively) of the islets of Langerhans in pancreas. Although these two hormones differ in their target tissues and mechanisms of action, the metabolic changes they bring about, at the whole-body level, have opposite effects, which helps to maintain energy homeostasis. Specifically, insulin results in adaptation to high blood glucose concentration and high energy status, which occurs after a meal, while glucagon results in adaptation to low blood glucose concentration.</p>
</div>
<div id="initial-steps-of-glucagon-and-insulin-signalling" class="section level1">
<h1><span class="header-section-number">2</span> Initial Steps of Glucagon and Insulin Signalling</h1>
<p>Glucagon and insulin each has diverse effects in different tissues, but the initial steps are common. To avoid repetition, these step are described in this section.</p>
<div id="insulin-signalling" class="section level2">
<h2><span class="header-section-number">2.1</span> Insulin Signalling</h2>
<p>The insulin receptor belongs to the receptor tyrosine kinase (RTK) family. Upon insulin binding, the receptor dimer trans-autophosphorylates each other on tyrosine residues (pY). IRS1 (insulin receptor substrate 1) binds to pY via its SH2 (src homology 2) domain. Insulin receptor phosphorylates IRS1 on tyrosine, and then PI3K (phosphoinositide 3 kinase) binds to pY of IRS1 via SH2. PI3K converts PIP2 (PtdIns(4,5)P2) to PIP3 (PtdIns(3,4,5)P3), then PIP3 serves as the docking site for PDK1 (phosphoinositide dependent kinase 1) and PKB (Akt). PDK1 and mTOR2C activates PKB by phosphorylation, and PKB then phosphorylates other protein targets to bring about most insulin-induced metabolic changes. Among these effects, a significant porportion is mediated by PP1 (protein phosphatase 1), which is activated by PKB (Figure <a href="#fig:insulin-overview">2.1</a>).</p>
<div class="figure"><span id="fig:insulin-overview"></span>
<img src="../img/insulin-overview.png" alt="Overview of insulin signalling." width="100%" />
<p class="caption">
Figure 2.1: Overview of insulin signalling.
</p>
</div>
</div>
<div id="glucagon-signalling" class="section level2">
<h2><span class="header-section-number">2.2</span> Glucagon Signalling</h2>
<p>Glucagon receptors are G-protein coupled receptors (GPCR). Upon glucagon binding, the receptor acts as a guanine nucleotide exchange factor and replace GDP bound to the G protein with GTP. The activated heterotrimeric G protein dissociates into <span class="math inline">\(\beta\gamma\)</span> and <span class="math inline">\(\alpha\)</span> subunits. Some glucagon receptors are associated with G<sub>s</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates adenylyl cyclase, which produces cAMP, which in turn activates protein kinase A (PKA). Other glucagon receptors are associated with G<sub>q</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates phospholipase C, which converts PIP2 (phosphatidylinositol bisphosphate) to IP3 (inositol trisphosphate) and DAG (diacylglycerol). IP3 stimulates Ca<sup>2+</sup> release from the smooth endoplasmic reticulum. PKA and Ca<sup>2+</sup> then activates downstream proteins (Figure <a href="#fig:glucagon-liver-gs-gq">2.2</a> ).</p>
<div class="figure"><span id="fig:glucagon-liver-gs-gq"></span>
<img src="../img/glucagon-liver-gs-gq.jpg" alt="Overview of glucagon signalling." width="100%" />
<p class="caption">
Figure 2.2: Overview of glucagon signalling.
</p>
</div>
</div>
</div>
<div id="glucose-metabolism" class="section level1">
<h1><span class="header-section-number">3</span> Glucose Metabolism</h1>
<p>Maintaining a glucose concentration of around 5 mmol/L is crucial to human survival. This glucose homeostatis is maintained by glucagon and insulin, where the former results in net increase in glucose concentration and the latter results in net decrease in glucose concentration.</p>
<div id="glycogen-metabolism-in-liver" class="section level2">
<h2><span class="header-section-number">3.1</span> Glycogen Metabolism in Liver</h2>
<p>In liver, net synthesis of glycogen (in order to lower down blood glucose concentration) is stimulated by insulin and net consumption is promoted by glucagon.</p>
<div id="glucagon-activates-glycogenolysis-in-liver" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Glucagon Activates Glycogenolysis in Liver</h3>
<p>As described previously, glucagon signalling results in activation of PKA and release of Ca<sup>2+</sup>. PKA phosphorylates and thus activates phosphorylase b kinase. Ca<sup>2+</sup> binds to phosphorylase b kinase, also activating it. Insulin inhibits glycogenolysis via PP1, which dephosphorylates phosphorylase kinase. Glucagon counteracts the effect of insulin in another way: PKA phosphorylates the R (regulatory) subunit of PP1, causing the dissociation of the C (catalytic) subunit, thus inactivating PP1 (Figure <a href="#fig:glucagon-insulin-glycogenolysis-simple">3.1</a> ).</p>
<div class="figure"><span id="fig:glucagon-insulin-glycogenolysis-simple"></span>
<img src="../img/glucagon-insulin-glycogenolysis-simple.png" alt="Effect of glucagon and insulin on hepatic glycogenolysis" width="100%" />
<p class="caption">
Figure 3.1: Effect of glucagon and insulin on hepatic glycogenolysis
</p>
</div>
</div>
<div id="insulin-activates-glycogen-synthesis-in-liver" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Insulin Activates Glycogen Synthesis in Liver</h3>
<p>Insulin activates glycogen synthesis in hepatocytes. Glycogen is synthesised by glycogen synthase (and branching enzymes), whose active (a) form is unphosphorylated. It is normally phosphorylated (inactivated) by caesin kinase 2 (CK2) and glycogen synthase kinase 3 (GSK3). Insulin-activated PKB phosphorylates (inavtivates) GSK3 and hence releases GS from inhibition (Figure <a href="#fig:insulin-gsk3">3.2</a>).</p>
<div class="figure"><span id="fig:insulin-gsk3"></span>
<img src="../img/insulin-gsk3.jpg" alt="Insulin stimulates glycogen synthesis in the liver." width="100%" />
<p class="caption">
Figure 3.2: Insulin stimulates glycogen synthesis in the liver.
</p>
</div>
</div>
</div>
<div id="glycolysis-and-gluconeogeneis-in-liver" class="section level2">
<h2><span class="header-section-number">3.2</span> Glycolysis and Gluconeogeneis in Liver</h2>
<p>Insulin activates glycolysis and inhibits gluconeogenesis in liver. Glucagon has opposite effects.</p>
<div id="short-term-control-via-pfk2pbpase2-bifunctional-enzyme" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Short-Term Control via PFK2/PBPase2 Bifunctional Enzyme</h3>
<p>In liver, fructose 2,6-bisphosphate (F26BP) the key regulator of glycolysis and gluconeogenesis. It is an allosteric regulator of PFK-1 and FBPase-1 and have opposite effects on them. It significantly reduces the inhibitory effect of ATP on PFK-1—without F26BP, PFK-1 essentially has no activity in normal hepatocytes because hepatic ATP concentration is very high. In contrast, F26BP is an allosteric inhibitor of FBPase-1.</p>
<p>The PFK2/PBPase2 bifunctional enzyme (BFE) converts between fructose 6-phosphate (F6P) and F26BP. PFK2 is active in its unphosphorylated form while FBPase2 is active in its phosphorylated form. Insulin activates PP1, which dephosphorylates BFE (PP2A has the same effect), resulting in net production of F26BP, thus activating glycolysis and inhibiting gluconeogenesis. Glucagon activates PKA, which in turn phosphorylates BFE, reducing amount of F26BP and hence deactivating glycolysis and de-inhibiting (activating) gluconeogenesis (Figure <a href="#fig:hepatic-f26bp">3.3</a> ).</p>
<div class="figure"><span id="fig:hepatic-f26bp"></span>
<img src="../img/hepatic-f26bp.png" alt="Regulation of glycolysis and gluconeogenesis via F26BP in liver." width="100%" />
<p class="caption">
Figure 3.3: Regulation of glycolysis and gluconeogenesis via F26BP in liver.
</p>
</div>
<p>The isoform of pyruvate kinase (PK) present in liver is also phosphorylated and inactivated by PKA, and this also contributes to glucagon-induced inhibition of glycolysis.</p>
</div>
<div id="long-term-transcriptional-control" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Long-Term Transcriptional Control</h3>
<p>Glucagon-triggered rise in cAMP concentration activates CREB (cAMP response element binding protein), which turns on synthesis of glucose 6-phosphatase and PEP carboxykinase, two key enzymes involved in gluconeogenesis.</p>
<p>Insulin activates PKB, which phosphorylates and inactivates other transcription factors, such as FOXO1, and thus reduces transcription of gluconeogenic enzymes.</p>
</div>
</div>
<div id="glucose-uptake-into-muscle-cells" class="section level2">
<h2><span class="header-section-number">3.3</span> Glucose Uptake into Muscle Cells</h2>
<p>Insulin sigalling in muscle cells results in activation of PKB, which in turn causes translocation of GLUT4-containing vesicles to the cell surface. This significantly increases the rate of influx of glucose into muscle cells, thus reducing blood glucose concentration.</p>
</div>
</div>
<div id="lipid-metabolism" class="section level1">
<h1><span class="header-section-number">4</span> Lipid Metabolism</h1>
<p>Glucagon is released when glucose is not abundant, thus its overall effect is to promote mobilisation and utilisation of the alternative fuel, triacylglycerides (TAG). Insulin has the opposite effect: it is released after a meal, when the nutrient concentration in blood is high, and it supresses TAG usage and promotes TAG storage.</p>
<div id="lipolysis-in-adipocytes" class="section level2">
<h2><span class="header-section-number">4.1</span> Lipolysis in Adipocytes</h2>
<p>Lipolysis is the process by which triacylglycerol molecules (TAG) stored in the lipid droplet of adipocytes in white adipose tissues are hydrolysed to 3 free (non-esterified) fatty acid molecules and 1 glycerol molecule and released into the bloodstream, resulting in net increase in free fatty acid concentration in blood plasma.</p>
<p>As shown in Figure <a href="#fig:adipocyte-positive">4.1</a>, PKA is central to the activation of the lipolytic pathway. Catecholamines (adrenaline and noradrenaline), being the major activators of lipolysis, activates PKA via <span class="math inline">\(\beta\)</span> adrenergic receptor, G<sub>s</sub>, AC and cAMP. Glucagon has also been shown to activate lipolysis <em>in vitro</em> (via PKA), but there is no clear evidence that it acts <em>in vivo</em>.</p>
<div class="figure"><span id="fig:adipocyte-positive"></span>
<img src="../img/adipocyte-positive.png" alt="Activation of lipolysis by catecholamines (adrenaline and noradrenaline) and, with less evidence, by glucagon. PKA is activated via the GPCR-G~s~-AC-cAMP pathway described previously. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL. TAG is hydrolysed to 3 fatty acids and a glycerol molecule stepwisely by ATGL, HSL and MGL." width="100%" />
<p class="caption">
Figure 4.1: Activation of lipolysis by catecholamines (adrenaline and noradrenaline) and, with less evidence, by glucagon. PKA is activated via the GPCR-G<sub>s</sub>-AC-cAMP pathway described previously. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL. TAG is hydrolysed to 3 fatty acids and a glycerol molecule stepwisely by ATGL, HSL and MGL.
</p>
</div>
<p>Insulin inhibits lipolysis via PKB and PDE-3B, resulting in net decrease in blood fatty acid concentration (Figure <a href="#fig:lipolysis-insulin-simple">4.2</a>).</p>
<div class="figure"><span id="fig:lipolysis-insulin-simple"></span>
<img src="../img/lipolysis-insulin-simple.jpg" alt="Inhibition of lipolysis by insulin. Insulin activates PKB, which phosphorylates and thus activates the phosphodiesterase PDE-3B. PDE-3B de-cyclise cAMP, to AMP, thus reducing PKA activity and the lipolytic pathway downstream of PKA." width="100%" />
<p class="caption">
Figure 4.2: Inhibition of lipolysis by insulin. Insulin activates PKB, which phosphorylates and thus activates the phosphodiesterase PDE-3B. PDE-3B de-cyclise cAMP, to AMP, thus reducing PKA activity and the lipolytic pathway downstream of PKA.
</p>
</div>
</div>
<div id="tag-synthesis-in-liver" class="section level2">
<h2><span class="header-section-number">4.2</span> TAG Synthesis in Liver</h2>
<p>TAG synthesis in liver is promoted after a meal. This is mainly mediated by de-phosphorylation (activation) of ACC (acetyl-CoA carboxylase) by PP2A as a result of elevated glucose concentration and pentose phosphate pathway activity. (Some books, such as Lehninger’s, say that insulin-activated PP1 also de-phosphorylates ACC). The product of ACC, malonyl CoA, inhibits CPT1 (carnitine palmitoyl transferase) and hence oxidation of fatty acids.</p>
<p>Long term regulation of lipogenesis is achieved via the transcription factors ChREBP and SREBP-1c. These two TFs both up-regulates enzymes involved in lipogenesis, but their regulation differ. ChREBP is activated by xylulose 5-phosphate, which happens when there is a large flux through the pentose phosphate pathway due to high glucose concentration. This effect is reduced by cAMP, whose production is induced by glucagon. In contrast, SREBP-1c is activated by atypical PKC (aPKC), which is another protein kinase activated by insulin (Figure <a href="#fig:chrebp-srebp1c">4.3</a> ).</p>
<div class="figure"><span id="fig:chrebp-srebp1c"></span>
<img src="../img/chrebp-srebp1c.jpg" alt="Long-term regulation of lipogenesis via ChREBP and SREBP-1c." width="100%" />
<p class="caption">
Figure 4.3: Long-term regulation of lipogenesis via ChREBP and SREBP-1c.
</p>
</div>
</div>
<div id="tag-re-synthesis-in-adipocytes" class="section level2">
<h2><span class="header-section-number">4.3</span> TAG (Re-)Synthesis in Adipocytes</h2>
<p>During fed state, adipocytes, with help from endothelial cells, break down chylomicrons (from the intestine) and VLDLs (from the liver), hydrolyse the containing triacylglycerol molecules, take up the resulting fatty acids, and re-synthesise the TAG. Adipocytes do not express glycerol kinase, thus they have to obtain glycogen 3-phosphate, one of the substrate in the first step of TAG synthesis, from dihydroxyacetone phosphte (by glycerol 3-phosphate), which is a glycolytic intermediate. Insulin promotes glucose uptake and hence glycolysis in adipocytes, thus promoting TAG resynthesis.</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">5</span> Conclusion</h1>
<p>Although glucagon and insulin usually exert their effects on different enzymes, and sometimes in different tissues, the net results of their activity are opposite, which ultimately leads to energy homeostasis, where energy consumption and concentration of glucose in blood remain relatively constant while the pattern of food intake is changing.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/supplying-fuels-to-the-brain-during-starvation.html" class="archive-item-link">
                Supplying Fuels to the Brain During Starvation 
              </a>
              <span class="archive-item-date">
                2020-05-03
              </span>
              <div class="article-summary" style="display: none;">
                
                


<!--nocite: |
  @Altarejos-2011, Bezaire-2009, Chakravarty-2005, Elia-1991, Frayn-2010, Fukao-2004, Holness-2003, Hue-2009, Hutton-2009, Lass-2011, Nuttall-2008, Okar-2001, Randle-1998, Rider-2004, Rufer-2009, Saggerson-2008, Soeters-2012, Stanley-2006, Sugden-2006, Thomson-2009, Yang-2009-->
<p><strong>Title: As starvation proceeds, how does the supply of fuels to the brain change? How is this change related to a requirement to conserve body protein?</strong></p>
<p>Due to the blood-brain barrier, The human brain can only harness energy from catabolising either glucose or ketone bodies. Under normal circumstances, glucose of dietary origin is the predominant source of fuel for the brain. As starvation begins (during the postabsorptive phase), astrocytes in the brain break down their stored glycogen to supply fuel in the form of lactate to neurons, which is followed by hepatic glycogenolysis that continue to provide glucose for the brain for a short period. As starvation proceeds, upon depletion of glycogen, muscle protein is broken down for liver gluconeogenesis, which continues to provide glucose for the brain. Meanwhile, triacylglycerol-derived ketone bodies synthesised by the liver will represent an increasingly larger proportion of fuel for the brain.</p>
<p>As shown in Figure <a href="#fig:starve-stages">1</a>, starvation can be divided into 4 stages, each having a different strategy for supplying energy to the brain.</p>
<div class="figure"><span id="fig:starve-stages"></span>
<img src="../img/starve-stages.jpg" alt="Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds." width="100%" />
<p class="caption">
Figure 1: Left: Origin of blood glucose and rate of glucose utilisation at different stages of starvation. Right: Concentrations of fuel molecules in blood plasma as starvtion proceeds.
</p>
</div>
<div id="stage-i-absoption-and-ii-post-absorption-glycogenolysis" class="section level1">
<h1>Stage I (Absoption) and II (Post-absorption): Glycogenolysis</h1>
<p>Soon after a meal, glucose obtained from food is absorbed from the gut into the bloodstream. The abundant glucose is the exclusive fuel used by the brain. The high blood glucose level also stimulates the <span class="math inline">\(\beta\)</span> cells in the islets of Langerhans in the pancreas to release insulin, which in turn promotes uptake and storage of glucose (as glycogen, in liver, muscle, actrocytes, and other tissues) and triacylglycerol (in lipid droplets in adipocytes).</p>
<p>During the post-absorptive phase, blood glucose concentration drops. Islets of Langerhans respond by reducing production of insulin in <span class="math inline">\(\beta\)</span> cells and increasing production of glucagon in <span class="math inline">\(\alpha\)</span> cells.</p>
<p>The principal effect of glucagon is to promote glycogenolysis in liver.</p>
<p>On the plasma membrane of hepatocytes, glucagon binds to its receptors, which are G-protein coupled receptors (GPCR), on hepatocyte cell surface membrane. Upon glucagon binding, the receptor acts as a guanine nucleotide exchange factor and replace GDP bound to the G protein with GTP. The activated heterotrimeric G protein dissociates into <span class="math inline">\(\beta\gamma\)</span> and <span class="math inline">\(\alpha\)</span> subunits.</p>
<p>Some glucagon receptors are associated with G<sub>s</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates adenylyl cyclase, which produces cAMP, which in turn activates protein kinase A (PKA). PKA phosphorylates phosphorylase kinase, thus activating it. Other glucagon receptors are associated with G<sub>q</sub>, and their <span class="math inline">\(\alpha\)</span> subunit activates phospholipase C, which converts PIP2 (phosphatidylinositol bisphosphate) to IP3 (inositol trisphosphate) and DAG (diacylglycerol). IP3 stimulates Ca<sup>2+</sup> release from the smooth endoplasmic reticulum, then Ca<sup>2+</sup> binds to phosphorylase kinase, also activating it.</p>
<p>Activated phosphorylase kinase phosphorylates glycogen phosphorylase <em>b</em> (inactive form), converting it to glycogen phosphorylase <em>a</em> (active form). Then, with help from the debranching enzyme, glycogen phosphorylase <em>a</em> break down glycogen into glucose 1-phosphate, which can be converted by phosphoglucose mutase to glucose 6-phosphate, which is then dephosphorylated by glucose 6-phosphatase. The resulting glucose is released into the bloodstream via GLUT2 transporter down the concentration gradient.</p>
<p>Glycogenolysis occurs not only in liver but also in astrocytes. Unlike liver glycogenolysis, however, 1-phosphoglucose in astrocytes are converted to lactate instead of glucose. Lactate is transported via monocarboxylate transporter (MCT) and supplied to neurons.</p>
<p>During this period, the brain still uses glucose-derived fuel (glucose and lactate) as the sole fuel at the same rate, so do other parts of the body.</p>
</div>
<div id="stage-iii-early-starvation-gluconeogenesis" class="section level1">
<h1>Stage III (Early Starvation): Gluconeogenesis</h1>
<p>As the glycogen storage become depleted, most tissues start to use fatty acids released by white adipose tissue as the fuel, but the brain continue to use, almost exclusively, glucose, which are now derived from gluconeogenesis.</p>
<p>In the human body, the principal substrates for gluconeogenesis are lactate, glucogenic amino acids and glycerol. During starvation, lactate is not an important gluconeogenic substrate for net gain of glucose, because it is produced mostly by anaerobic glycolysis of glucose itself.</p>
<p>During starvation, body protein is the only source of amino acids for gluconeogenesis. These amino acids are initially supplied by hepatic proteolysis and later predominantly provided by muscle proteins. Although the composition of muscle proteins is rather heterogenous, the majority of the release amino acids are alanine and glutamine. Most alanine is taken up by the liver for gluconeogenesis, while the role of glutamine is more varied—it is used by the kidney for acid-base balance during starvation, for example.</p>
<p>Glycerol, the other source molecule for gluconeogenesis during starvation, is produced by the adipocytes in the white adipose tissue in a process known as lipolysis (fat mobilisation). Lipolysis involves the sequential removal of the three fatty acids of a TAG molecule by adipose triglyceride lipase (ATGL), hormone sensitive hormone (HSL), and monoglyceride lipase (MGL), and the removal of the first two fatty acids by ATGL and HSL is under hormonal control (MGL is always active).</p>
<p>Catecholamines (adrenaline/noradrenaline), which are released upon activation of the sympathetic nervous system at the early stages of starvation, are the major activators of lipolysis. They do so via the GPCR-G<sub>s</sub>-AC-cAMP-PKA pathway, similar to that used by glucagon. Activated PKA has two roles. First, it phosphorylates HSL and increases enzymatic activity thereof. Second, it phosphorylates perilipin, which is the protein residing on the membrane of the lipid droplet (LD) providing protection against ATGL and HSL. The phosphate groups induce binding of HSL to perilipin, effectively translocating HSL from the cytosol to the LD. Phosphorylation also release CGI-58 from perilipin, and the free CGI-58 activates ATGL.</p>
<p>Insulin is the major inhibitor of lipolysis. Insulin activates PKB (Akt), which in turn activates phosphodiesterase 3B (PDE-3B). PDE-3B de-cyclise cAMP, thus antagonising the PKA activity. Thus, during starvation, the low insulin level means de-inhibition of lipolysis.</p>
<p>The resulting glycerol and fatty acids are all released into the circulation, the former being used as a substrate for gluconeogenesis and the latter being used as the principal fuel by most tissues and well as a substrate for ketone bodies synthesis in the liver.</p>
<p>There are a number of mechanisms by which the liver ensures that gluconeogenenic precursors are used for gluconeogenesis rather than oxidation. For example, the large amount of acetyl-CoA obtained via <span class="math inline">\(\beta\)</span>-oxidation inhibits pyruvate dehydrogenase complex, the activity of which would result in irreversible loss of gluconeogenic precursors. As an another example, the low insulin level has a long-term effect on gluconeogenesis by altering gene expression. Insulin leads to activation of PKB, and PKB inactivates (by phosphorylation) the transcription factor FOXO1 that promotes transcription of gluconeogenenic enzymes such as PEP carboxykinase. Thus, the low insulin level during starvation effectively activates the transcription of these gluconeogenic enzymes.</p>
</div>
<div id="stage-iv-prolonged-starvation-ketone-body-synthesis" class="section level1">
<h1>Stage IV (Prolonged Starvation): Ketone Body Synthesis</h1>
<p>During prolonged starvation, the brain gradually adopts ketone bodies (acetoacetate and 3-hydroxybutyrate) as an alternative fuel.</p>
<p>The ultimate origin of ketone bodies is the TAG stored in the white adipose tissue. As described above, low insulin level allows continuous lipolysis, resulting in release of fatty acids and glycerol into the bloodstream. Many fatty acid molecules are absorbed by adipocytes, in which they undergo <span class="math inline">\(\beta\)</span>-oxidation in mitochondria, producing acetyl-CoA in the liver. A part of acetyl-CoA is used for respiration, and the excess is converted either to acetoacetate in a 3-step sequence of reactions involving thiolase, HMG-CoA synthetase and HMG-CoA lyase or, with an additional reduction step, to 3-hydroxybutyrate (in mitochondria). Ketone bodies are released via the monocarboxylate transporter (MCT).</p>
<p>The ketone bodies can cross the blood-brain barrier and reach the neurons (via MCT). In the mitochondria of neurons, ketone bodies are converted back to acetyl-CoA, which is used for respiration. The large amount of acetyl-CoA obtained from ketone bodies inhibits pyruvate dehydrogenase complex and thus restricts glucose utilisation in much the same way acetyl-CoA derived from <span class="math inline">\(\beta\)</span>-oxidation in other tissues suppress glucose oxidation.</p>
<p>This gradual switching from glucose to ketone bodies as the fuel in the brain is important to the conservation of body proteins. As described previously, gluconeogenesis is the only source of glucose during starvation, and amino acids obtained from muscle proteolysis represent the majority of gluconeogenic precursors during starvation. Thus, conservation of body protein during starvation is equivalent to minimising the comsumption of glucose as fuel, specifically in the brain.</p>
<p>Prolonged starvation is accompanied by a gradual decrease in concentration of the active thyroid hormone, triiodothyronine (T3) over time, which in turn leads to a decrease in basal metabolic rate in most tissues. This reduces the overall consumption of energy of the body and thus also contributes to the sparing of muscle protein.</p>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p>During a period of starvation, the origin and identity of fuel molecules supplied to the brain (neurons) change, according to this order: astrocyte glycogenolysis, liver glycogenolysis, gluconeogenesis, ketone body synthesis. At the later stages, gluconeogenesis and ketone body synthesis coexist, but the amount of ketone body synthesis gradually increase, in order to spare the functional muscle proteins. This metabolic adaptation to starvation is largely under hormonal control, where insulin, glucagon, thyroid hormone and catecholamines play the most promiment roles.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/how-has-ras-signalling-been-studied-in-c-elegans.html" class="archive-item-link">
                How Has Ras Signalling Been Studied in C. Elegans 
              </a>
              <span class="archive-item-date">
                2020-03-03
              </span>
              <div class="article-summary" style="display: none;">
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#the-big-picture-components-of-the-c.-elegans-egf-ras-mapk-pathway-involved-in-vulva-induction"><span class="toc-section-number">2</span> The Big Picture: Components of the <em>C. elegans</em> EGF-Ras-MAPK Pathway Involved in Vulva Induction</a></li>
<li><a href="#techniques-for-studying-egfr-ras-mapk-signalling-and-vulva-induction"><span class="toc-section-number">3</span> Techniques for Studying EGFR-Ras-MAPK Signalling and Vulva Induction</a><ul>
<li><a href="#forward-genetics-and-reverse-genetics"><span class="toc-section-number">3.1</span> Forward Genetics and Reverse Genetics</a><ul>
<li><a href="#in-early-experiments-forward-genetics-methods-were-used-entensively"><span class="toc-section-number">3.1.1</span> In Early Experiments, Forward Genetics Methods were Used Entensively</a></li>
<li><a href="#rnai-and-reverse-genetics-since-1998"><span class="toc-section-number">3.1.2</span> RNAi and Reverse Genetics Since 1998</a></li>
</ul></li>
<li><a href="#observing-cell-fates"><span class="toc-section-number">3.2</span> Observing Cell Fates</a><ul>
<li><a href="#microscopic-analysis"><span class="toc-section-number">3.2.1</span> Microscopic Analysis</a></li>
<li><a href="#transcriptional-reporters"><span class="toc-section-number">3.2.2</span> Transcriptional Reporters</a></li>
</ul></li>
<li><a href="#cross-species-genomic-analysis-reveals-homology-and-helps-to-elucidate-roles-of-proteins-with-greater-accuracy"><span class="toc-section-number">3.3</span> Cross-Species Genomic Analysis Reveals Homology and Helps to Elucidate Roles of Proteins with Greater Accuracy</a></li>
<li><a href="#studying-egfr-localisation-and-trafficking"><span class="toc-section-number">3.4</span> Studying EGFR Localisation and Trafficking</a><ul>
<li><a href="#immunostaining"><span class="toc-section-number">3.4.1</span> Immunostaining</a></li>
<li><a href="#live-imaging"><span class="toc-section-number">3.4.2</span> Live Imaging</a></li>
</ul></li>
</ul></li>
<li><a href="#conclusion"><span class="toc-section-number">4</span> Conclusion</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>The <em>Caenorhabditis elegans</em> vulva is one of the best studied models of organogeneis. Genetic screening revealed three signalling pathways, EGFR-Ras-MAPK, Notch and Wnt<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>, that are critical to the fate determination of the 6 vulval precursor cells (VPCs), P(3-8)p.</p>
<p>VPCs adopt either 1<span class="math inline">\(^\circ\)</span> or 2<span class="math inline">\(^\circ\)</span> fate upon receiving appropriate signals, or 3<span class="math inline">\(^\circ\)</span> fate if undinduced. The 8 descentdents of the 1<span class="math inline">\(^\circ\)</span> VPC and the 14 descendents (7 each) of the 2<span class="math inline">\(^\circ\)</span> VPCs give rise to the vulva, while 3<span class="math inline">\(^\circ\)</span> VPCs produces epidermal cells which fuse with the syncytial epidermis hyp7.</p>
<p>In the L1 stage, a Wnt signal form the posterior body region selects the six Pn.p cells to become the VPCs and form the vulval competentce group. Then in the L2 stage, LIN-3 (EGF) signal from the anchor cell (AC) in the somatic gonad selects P6.p to be the single 1<span class="math inline">\(^\circ\)</span> VPC<span class="citation"><sup>1,2</sup></span>, and contact-dependent LIN-12 (Notch) signalling causes the flanking P5.p and P7.p to become 2<span class="math inline">\(^\circ\)</span> VPCs. These precise signalling events always results in an invariant spatial pattern of VPCs in wild-type (WT) <em>C. elegans</em>, namely 3<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-1<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>.</p>
<p>This essay illustrates how the LET-60 (Ras) signalling has been studied in <em>C. elegans</em> vulval development, specifically the 1<span class="math inline">\(^\circ\)</span> fate determination, according to these selected work and reviews: <span class="citation">3</span>; <span class="citation">4</span>; <span class="citation">5</span>; <span class="citation">6</span>; <span class="citation">7</span>; <span class="citation">8</span>. A recent review article by <span class="citation">9</span> and a book chapter by <span class="citation">10</span> are also taken into account (the latter was the most helpful).</p>
</div>
<div id="the-big-picture-components-of-the-c.-elegans-egf-ras-mapk-pathway-involved-in-vulva-induction" class="section level1">
<h1><span class="header-section-number">2</span> The Big Picture: Components of the <em>C. elegans</em> EGF-Ras-MAPK Pathway Involved in Vulva Induction</h1>
<p>Figure <a href="#fig:c-ele-valva-big">2.1</a> gives an overview of the pathways by which EGF-Ras-MAPK signalling leads to the 1<span class="math inline">\(^\circ\)</span> fate in P6.p and 2<span class="math inline">\(^\circ\)</span> fate in flanking P5.p and P7.p cells.</p>
<div class="figure"><span id="fig:c-ele-valva-big"></span>
<img src="../img/c-ele-valva-big.jpg" alt="P6.p is the VPC closest to the AC, and it also expresses the highest levels of LET-23 on its basolateral membrane. Thus, P6.p receives most LIN-3 signalling and commits to the 1\(^\circ\) fate. Downstram of the LET-23 (EGFR) RTK, a canocical Ras-MAPK pathway transduces the signal into the nucleus. The scaffold proteins LIN-2 (CASK), LIN-7 (Veli), and LIN-10 (Mint-1) form a complex which helps to maintain basolateral localisation of LET-23. The SEM-5 (Grb2) adaptor protein11 uses its SH2 domain to anchor to LET-23 at phosphorylated tyrosines. The guanine exchange factor (GEF) SOS-112 then binds to SEM-5 (at SEM-5’s two SH3 domains), and it activates the membrane-anchored LET-60 (Ras)3. GTP-bound LET-60 then activates the LIN-45 (Raf)13, MEK-214 and MPK-1 (MAPK) kinase cascade, which is facilitated by the scaffold protein KSR. LIN-1 and forkhead LIN-31 transcription factors are the targets of MPK-115. In their unphosphorylated state, LIN-1 and LIN-31 forms a complex which represses 1\(^\circ\)-specific transcription of lin-39 and egl-1716. Phosphorylation makes the complex to dissociate, and phospho-LIN-31 is turned into an transcriptional activator. The 1\(^\circ\) P6.p also produces LAG-2 (Delta), which is the ligand for LIN-12 (Notch), thus transducing lateral signals to P5.p and P7.p. Notch signalling inhibits MAKP, and LET-60 in P5.p and P7.p cells uses a RAL-1 dependent pathway to commit to 2\(^\circ\) fate. A number of modifiers and negative regulators of the EGFR pathway has been identified, which include the tyrosine kinase Ark-117, the RTK phosphatase Dep-118. Adapted from 10 with reference to 19 and 20." width="100%" />
<p class="caption">
Figure 2.1: P6.p is the VPC closest to the AC, and it also expresses the highest levels of LET-23 on its basolateral membrane. Thus, P6.p receives most LIN-3 signalling and commits to the 1<span class="math inline">\(^\circ\)</span> fate. Downstram of the LET-23 (EGFR) RTK, a canocical Ras-MAPK pathway transduces the signal into the nucleus. The scaffold proteins LIN-2 (CASK), LIN-7 (Veli), and LIN-10 (Mint-1) form a complex which helps to maintain basolateral localisation of LET-23. The SEM-5 (Grb2) adaptor protein<span class="citation"><sup>11</sup></span> uses its SH2 domain to anchor to LET-23 at phosphorylated tyrosines. The guanine exchange factor (GEF) SOS-1<span class="citation"><sup>12</sup></span> then binds to SEM-5 (at SEM-5’s two SH3 domains), and it activates the membrane-anchored LET-60 (Ras)<span class="citation"><sup>3</sup></span>. GTP-bound LET-60 then activates the LIN-45 (Raf)<span class="citation"><sup>13</sup></span>, MEK-2<span class="citation"><sup>14</sup></span> and MPK-1 (MAPK) kinase cascade, which is facilitated by the scaffold protein KSR. LIN-1 and <em>forkhead</em> LIN-31 transcription factors are the targets of MPK-1<span class="citation"><sup>15</sup></span>. In their unphosphorylated state, LIN-1 and LIN-31 forms a complex which represses 1<span class="math inline">\(^\circ\)</span>-specific transcription of <em>lin-39</em> and <em>egl-17</em><span class="citation"><sup>16</sup></span>. Phosphorylation makes the complex to dissociate, and phospho-LIN-31 is turned into an transcriptional activator. The 1<span class="math inline">\(^\circ\)</span> P6.p also produces LAG-2 (Delta), which is the ligand for LIN-12 (Notch), thus transducing lateral signals to P5.p and P7.p. Notch signalling inhibits MAKP, and LET-60 in P5.p and P7.p cells uses a RAL-1 dependent pathway to commit to 2<span class="math inline">\(^\circ\)</span> fate. A number of modifiers and negative regulators of the EGFR pathway has been identified, which include the tyrosine kinase Ark-1<span class="citation"><sup>17</sup></span>, the RTK phosphatase Dep-1<span class="citation"><sup>18</sup></span>. Adapted from <span class="citation">10</span> with reference to <span class="citation">19</span> and <span class="citation">20</span>.
</p>
</div>

</div>
<div id="techniques-for-studying-egfr-ras-mapk-signalling-and-vulva-induction" class="section level1">
<h1><span class="header-section-number">3</span> Techniques for Studying EGFR-Ras-MAPK Signalling and Vulva Induction</h1>
<div id="forward-genetics-and-reverse-genetics" class="section level2">
<h2><span class="header-section-number">3.1</span> Forward Genetics and Reverse Genetics</h2>
<div id="in-early-experiments-forward-genetics-methods-were-used-entensively" class="section level3">
<h3><span class="header-section-number">3.1.1</span> In Early Experiments, Forward Genetics Methods were Used Entensively</h3>
<p>Studies on the molecular mechanisms of <em>C. elegans</em> development date back to the 1980s, when genetic manipulating techniques, especially gene cloning, were emerging, but the complete genome sequencing (of <em>C. elegans</em> and other model organisms) had yet to start. Therefore, those pioineering studies relied heavily on forward genetics, where worms were subjected to mutagens, and abnomal phenotypes were identified, isolated, amplified (by self-fertilisation) and the respective genotypes were determined. Differential interference contrast (DIC)/Nomarski microscopes were commonly used to examine the phenotypes because of their superior imaging quuality for live and unstained samples. There were also cellular methods for manipulating gene expression (which especially aided in eludidating cell-cell interactions). First, observing fates of the cells among varying individuals. Second, using laser beams to ablate cells suspected to interact with the cell of interest. Third, isolation of cells of interest, or rearrangement of their surrounding cells.</p>
<p>These extensive genetic screens led to identification of a number of genes that cause abormal cell lineage (the <em>lin#</em> genes) phenotypes, notably vulvaless (Vul)/Egg-laying defective (Egl) or multivulva (Muv) (Figure <a href="#fig:egl-muv-c-ele-vulva">3.1</a>). Epistasis analyses of these phenotypes grouped these genes into distinct pathways (notably EGFR-Ras-MAPK and Notch) and helped elucidating where each component functions in the respetive pathway. The general idea is that, if the phenotype of gene A masks the phenotype of gene B, then gene A is likely to act downstream of gene B.</p>

<div class="figure"><span id="fig:egl-muv-c-ele-vulva"></span>
<img src="../img/egl-muv-c-ele-vulva.png" alt="Egg-laying defective (Egl) and Multivulva (Muv) phenotypes caused by LET-23 EGFR-Ras-MAPK signalling mutants. (a): Wild type. Eggs are neatly arranged in the uterus. (b) and (c): Defective LIN-10 (which helps to localise LET-23) can lead to disorganisation of eggs (b) and the bag-of-worms phenotype (c), in which self-fertilised eggs hatch and larvae fills up their mother’s carcass. (d): let-60 gain-of-function results in Ras signalling in all VPCs, resulting in a Muv phenotype. Reproduced from 10." width="100%" />
<p class="caption">
Figure 3.1: Egg-laying defective (Egl) and Multivulva (Muv) phenotypes caused by LET-23 EGFR-Ras-MAPK signalling mutants. (a): Wild type. Eggs are neatly arranged in the uterus. (b) and (c): Defective LIN-10 (which helps to localise LET-23) can lead to disorganisation of eggs (b) and the bag-of-worms phenotype (c), in which self-fertilised eggs hatch and larvae fills up their mother’s carcass. (d): <em>let-60</em> gain-of-function results in Ras signalling in all VPCs, resulting in a Muv phenotype. Reproduced from <span class="citation">10</span>.
</p>
</div>
<p>Figure <a href="#fig:vul-Muv-simplest">3.2</a> shows the earliest expetiments that demonstrated the role of the AC and the LET-60 protein in vulval induction.</p>

<div class="figure"><span id="fig:vul-Muv-simplest"></span>
<img src="../img/vul-mul-simplest.jpg" alt="(a) in wild-type cells, signals from the AC results in the invariant VPC pattern 3\(^\circ\)-3\(^\circ\)-2\(^\circ\)-1\(^\circ\)-2\(^\circ\)-3\(^\circ\); (b) ablation of the AC renders a Vul phenotype, where all 6 VPCs adopts the 3\(^\circ\) fate, this shows that the AC positively regulates vulval induction; (c) inactive LET-60 (Ras) also renders a Vul phenotype even in the presence of the AC, therefore LET-60 should be downstream of the signal from the AC; (d) LET-60 hyperactivity leads to a Muv phenotype, further demonstrating LET-60 as a positive regulator of vulval induction." width="100%" />
<p class="caption">
Figure 3.2: (a) in wild-type cells, signals from the AC results in the invariant VPC pattern 3<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-1<span class="math inline">\(^\circ\)</span>-2<span class="math inline">\(^\circ\)</span>-3<span class="math inline">\(^\circ\)</span>; (b) ablation of the AC renders a Vul phenotype, where all 6 VPCs adopts the 3<span class="math inline">\(^\circ\)</span> fate, this shows that the AC positively regulates vulval induction; (c) inactive LET-60 (Ras) also renders a Vul phenotype even in the presence of the AC, therefore LET-60 should be downstream of the signal from the AC; (d) LET-60 hyperactivity leads to a Muv phenotype, further demonstrating LET-60 as a positive regulator of vulval induction.
</p>
</div>
</div>
<div id="rnai-and-reverse-genetics-since-1998" class="section level3">
<h3><span class="header-section-number">3.1.2</span> RNAi and Reverse Genetics Since 1998</h3>
<p>Early mutagenesis studies relied on chance to generate individuals with mutations for certain genes, which is laborious and not specific—usually some genes/DNA regions other than the genes of interest are also mutated, and studies on them can lead to confounding results.</p>
<p>Reverse genetics in <em>C. elegans</em> has become much more prevalent since 1998, when <em>C. elegans</em> became the first multicellular eukaryote to have its genome sequenced<span class="citation"><sup>21</sup></span>, and it was shown that introdcution of dsRNA into worms results in inactivation of an endogenous gene with corresponding sequence<span class="citation"><sup>22</sup></span>. This technique, known as RNA interference (RNAi), enables rapid, site-specific gene inactivation and has been rapidly embraced as a reverse-genetic tool and has dramatically accelerated the pace at which new gene functions are discovered.</p>
</div>
</div>
<div id="observing-cell-fates" class="section level2">
<h2><span class="header-section-number">3.2</span> Observing Cell Fates</h2>
<div id="microscopic-analysis" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Microscopic Analysis</h3>
<p>Observing Val and Muv phenotypes is generally sufficient for ordering the components of the Ras pathway, but cell-cell interaction analyses require the ability to identify differentiated or undifferentiated individual VPCs. The boundaries between cells are not visible by DIC optics and thus cells are identified by the location, size, and morphology of the nucleus and nucleoli. Undifferentiated VPCs at the Pn.p stage each have a characteristic oblong nucleus with a round nucleolus.</p>
</div>
<div id="transcriptional-reporters" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Transcriptional Reporters</h3>
<p>Several transcriptional targets of LET-23 (EGFR) signalling in P6.p, such as <em>egl-17</em> and <em>lag-2</em>, have been identified and can be used as transcriptional reporters to identify vulval cell fates. By using a highly sentitive Ras-responsive reporter gene, <em>egl-17::CFP</em>, <span class="citation">8</span> showed that the Ras pathway is transiently activated in P5.p and P7.p, and that LIN-12 (Notch) activation in these two cells antagonises Ras signalling.</p>
<p>Single molecule fluorescence <em>in situ</em> hybridisation (smFISH) can also be used to monitor expression of cell fate determinants. It reveals subtle changes in gene expression and has been used, for example by <span class="citation">23</span>, to visualise the dynamics of Ras-induced <em>lag-2</em> expression, which demonstrates the cellular control of EGF-Ras induced gene expression by changing sensitivity to the EGF gradient .</p>
</div>
</div>
<div id="cross-species-genomic-analysis-reveals-homology-and-helps-to-elucidate-roles-of-proteins-with-greater-accuracy" class="section level2">
<h2><span class="header-section-number">3.3</span> Cross-Species Genomic Analysis Reveals Homology and Helps to Elucidate Roles of Proteins with Greater Accuracy</h2>
<p>As sequencing techniques mature, complete genome sequences of more organisms are becoming available. This is accompanied by advance in computing power and bioinformatic algorithms, and they allow identification of a significant amount of homology among species.</p>
<p>The studies on <em>C. elegans</em> Ras signalling also benefitted from these developments. For example, MPK-1 was initially identified as a suppressor of Ras and was named SUR-1 before being identified as an Erk1/2 homologue. The scaffold protein KSR-1 was originally named SUR-3 due to the same reason. With homology available, one can easily guess the function of a novel gene/protein identified in one organism, if its sequence is found homologous with another gene with known functions.</p>
</div>
<div id="studying-egfr-localisation-and-trafficking" class="section level2">
<h2><span class="header-section-number">3.4</span> Studying EGFR Localisation and Trafficking</h2>
<p>The LET-23 EGFR upstream of LET-60 Ras has a polarised localisation in VPCs: vuval induction requires that EGFR to be expressed on the basolateral membrane. Immunostaining and live imaging can be used to monitor LET-23 trafficking and identify regulators involved.</p>
<div id="immunostaining" class="section level3">
<h3><span class="header-section-number">3.4.1</span> Immunostaining</h3>
<p>Immunostaining can be used to visualise subcellular localisation of components of regulators of the EDGR-Ras-MAPK pathway. This technique was used by <span class="citation">24</span> to establish the role of LIN-2/7/10 complex binding to LET-23 C-terminuc in regulating LET-23 basolateral localisation (Figure <a href="#fig:c-ele-vulva-immunostaining">3.3</a>).</p>

<div class="figure"><span id="fig:c-ele-vulva-immunostaining"></span>
<img src="../img/c-ele-vulva-immunostaining.png" alt="Top: LET-23 are localised to the basolateral membrane of VPCs in wild-type C. elegans; Middle: lin-7 mutant shows mislocalisation of LET-23 to the apical membrane; Bottom: let-23(sy1) mutant, in which a premature stop codon removes 6 C-terminal amino acids of LET-23, similarly resulted in mislocalisation of LET-23. LET-23 staining is shownin green, and staining for the cell junctions using the monoclonal antibody MH27 is shown in red. Reproduced from 24." width="100%" />
<p class="caption">
Figure 3.3: Top: LET-23 are localised to the basolateral membrane of VPCs in wild-type <em>C. elegans</em>; Middle: <em>lin-7</em> mutant shows mislocalisation of LET-23 to the apical membrane; Bottom: <em>let-23(sy1)</em> mutant, in which a premature stop codon removes 6 C-terminal amino acids of LET-23, similarly resulted in mislocalisation of LET-23. LET-23 staining is shownin green, and staining for the cell junctions using the monoclonal antibody MH27 is shown in red. Reproduced from <span class="citation">24</span>.
</p>
</div>
</div>
<div id="live-imaging" class="section level3">
<h3><span class="header-section-number">3.4.2</span> Live Imaging</h3>
<p>Modern imaging techniques, notably confocal and epifluorescence microscopy, allow the subcellular localisation and trafficking of LET-23 to be observed <em>in vivo</em>, in real time. These studies often involve fusion proteins with fluorescence tags. For example, <span class="citation">25</span> used time-lapse imaging and fluorescence recovery after photobleaching to identify ERM-1 (the Ezrin homologue) as a temporal regulator of LET-23 localisation.</p>
</div>
</div>
</div>
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">4</span> Conclusion</h1>
<p>The understanding of the EGFR-Ras-MAPK signalling pathway is importnat because of its involvement in many human diseases, especially cancer. Vulva induction in <em>C. elegans</em> has been a valuable <em>in vivo</em> model for elucidating the identities and mechanisms of the components and regulators in this pathway. Studies on <em>C. elegans</em> traditionally relied on forward genetics, but new experimental techniques and bioinformatic are now solving previously unanswered questions in this field.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Hill-1992">
<p>1. Hill, R. &amp; Sternberg, P. The gene lin-3 encodes an inductive signal for vulval development in c. Elegans. <em>Nature</em> <strong>358</strong>, 470–476 (1992).</p>
</div>
<div id="ref-Sternberg-1989">
<p>2. Sternberg, P. W. &amp; Horvitz, H. The combined action of two intercellular signaling pathways specifies three cell fates during vulval induction in c. Elegans. <em>Cell</em> <strong>58</strong>, 679–693 (1989).</p>
</div>
<div id="ref-Beitel-1990">
<p>3. Beitel, G. J., Clark, S. G. &amp; Horvitz, H. R. Caenorhabditis elegans ras gene let-60 acts as a switch in the pathway of vulval induction. <em>Nature</em> <strong>348</strong>, 503–509 (1990).</p>
</div>
<div id="ref-Thomas-1990">
<p>4. Thomas, J. H., Stern, M. J. &amp; Horvitz, H. R. Cell interactions coordinate the development of the c. Elegans egg-laying system. <em>Cell</em> <strong>62</strong>, 1041–1052 (1990).</p>
</div>
<div id="ref-Sternberg-1998">
<p>5. Sternberg, P. W. &amp; Han, M. Genetics of ras signaling in c. Elegans. <em>Trends Genet</em> <strong>14</strong>, 466–472 (1998).</p>
</div>
<div id="ref-Kornfeld-1997">
<p>6. Kornfeld, K. Vulval development in caenorhabditis elegans. <em>Trends Genet</em> <strong>13</strong>, 55–61 (1997).</p>
</div>
<div id="ref-Sundaram-2004">
<p>7. Sundaram, M. V. Vulval development: The battle between ras and notch. <em>Current Biology</em> <strong>14</strong>, R311–R313 (2004).</p>
</div>
<div id="ref-Yoo-2004">
<p>8. Yoo, A., Bais, C. &amp; Greenwald, I. Crosstalk between the egfr and lin-12/notch pathways in c. Elegans vulval development. <em>Science</em> <strong>303</strong>, 663–666 (2004).</p>
</div>
<div id="ref-Schmid-2015">
<p>9. Schmid, T. &amp; Hajnal, A. Signal transduction during c. Elegans vulval development: A neverending story. <em>Current Opinion in Genetics &amp; Development</em> <strong>32</strong>, 1–9 (2015).</p>
</div>
<div id="ref-Gauthier-2017">
<p>10. Gauthier, K. &amp; Rocheleau, C. E. C. Elegans vulva induction: An in vivo model to study epidermal growth factor receptor signaling and trafficking. in <em>ErbB receptor signaling: Methods and protocols</em> (ed. Wang, Z.) vol. 1652 43–61 (Humana Press, 2017).</p>
</div>
<div id="ref-Clark-1992">
<p>11. Clark, S., Stern, M. &amp; Horvritz, H. C. Elegans cell-signalling gene sem-5 encodes a protein with sh2 and sh3 domains. <em>Nature</em> <strong>356</strong>, 340–344 (1992).</p>
</div>
<div id="ref-Chang-2000">
<p>12. Chang, C., Hopper, N. &amp; Sternberg, P. Caenorhabditis elegans sos-1 is necessary for multiple ras-mediated developmental signals. <em>EMBO Journal</em> <strong>19</strong>, 3283–3294 (2000).</p>
</div>
<div id="ref-Han-1993">
<p>13. Han, M., Golden, A., Han, Y. &amp; Sternberg, P. C. Elegans lin-45 raf gene participates in let-60 ras-stimulated vulval differentiation. <em>Nature</em> <strong>363</strong>, 133–140 (1993).</p>
</div>
<div id="ref-Kornfeld-1995">
<p>14. Kornfeld, K., Guan, K. L. &amp; Horvitz, H. R. The caenorhabditis elegans gene mek-2 is required for vulval induction and encodes a protein similar to the protein kinase mek. <em>Genes Dev</em> <strong>9</strong>, 756–768 (1995).</p>
</div>
<div id="ref-Lackner-1994">
<p>15. Lackner, M. R., Kornfeld, K., Miller, L. M., Horvitz, H. R. &amp; Kim, S. K. A map kinase homolog, mpk-1, is involved in ras-mediated induction of vulval cell fates in caenorhabditis elegans. <em>Genes Dev</em> <strong>8</strong>, 160–173 (1994).</p>
</div>
<div id="ref-Leight-2005">
<p>16. Leight, E. R., Glossip, D. &amp; Kornfeld, K. Sumoylation of lin-1 promotes transcriptional repression and inhibition of vulval cell fates. <em>Development</em> <strong>132</strong>, 1047–1056 (2005).</p>
</div>
<div id="ref-Hopper-2000">
<p>17. Hopper, N. A., Lee, J. &amp; Sternberg, P. W. ARK-1 inhibits EGFR signaling in <em>C. elegans</em>. <em>Molecular Cell</em> <strong>6</strong>, 65–75 (2000).</p>
</div>
<div id="ref-Berset-2005">
<p>18. Berset, T. A., Hoier, E. F. &amp; Hajnal, A. The c. Elegans homolog of the mammalian tumor suppressor dep-1/scc1 inhibits egfr signaling to regulate binary cell fate decisions. <em>Genes Dev</em> <strong>19</strong>, 1328–1340 (2005).</p>
</div>
<div id="ref-Alberts-2014">
<p>19. Alberts, B. <em>et al.</em> <em>Molecular biology of the cell</em>. (Garland Science, 2014).</p>
</div>
<div id="ref-Sternberg-2005">
<p>20. Sternberg, P. W. Vulval development. in <em>WormBook</em> (ed. Research Community, T. C. elegans) (WormBook, 2005). doi:<a href="https://doi.org/doi/10.1895/wormbook.1.6.1">doi/10.1895/wormbook.1.6.1</a>.</p>
</div>
<div id="ref-Stein-1999">
<p>21. Stein, L. D. Internet access to the c. Elegans genome. <em>Trends in Genetics</em> <strong>15</strong>, 425–427 (1999).</p>
</div>
<div id="ref-Fire-1998">
<p>22. Fire, A. <em>et al.</em> Potent and specific genetic interference by double-stranded rna in caenorhabditis elegans. <em>Nature</em> <strong>391</strong>, 806–811 (1998).</p>
</div>
<div id="ref-Zon-2015">
<p>23. Zon, J. S. van, Kienle, S., Huelsz-Prince, G., Barkoulas, M. &amp; Oudenaarden, A. van. Cells change their sensitivity to an egf morphogen gradient to control egf-induced gene expression. <em>Nature Communications</em> <strong>6</strong>, 7053 (2015).</p>
</div>
<div id="ref-Kaech-1998">
<p>24. Kaech, S. M., Whitfield, C. W. &amp; Kim, S. K. The lin-2/lin-7/lin-10 complex mediates basolateral membrane localization of the c. Elegans egf receptor let-23 in vulval epithelial cells. <em>Cell</em> <strong>94</strong>, 761–771 (1998).</p>
</div>
<div id="ref-Haag-2014">
<p>25. Haag, P. A. B., Andrea AND Gutierrez. An in vivo egf receptor localization screen in c. Elegans identifies the ezrin homolog erm-1 as a temporal regulator of signaling. <em>PLOS Genetics</em> <strong>10</strong>, 1–13 (2014).</p>
</div>
<div id="ref-Gilbert-2016">
<p>26. Gilbert, S. F. &amp; Barresi, M. J. F. <em>Developmental biology</em>. (Sinauer Associates, 2016).</p>
</div>
<div id="ref-Slack-2018">
<p>27. Slack, J. M. W. <em>Essential developmental biology</em>. (WIley-Blackwell, 2018).</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>In this essay, the names of the actual <em>C. elegans</em> proteins and their (more commonly known) vertebrate/<em>Drosophila</em> homologues are used interchangeably. Here is a translation between them: (<em>Drosophila</em>: <em>C. elegans</em>) EGF: LIN-3; EGFR: LET-23; Grb2: SEM-5; Ras: LET-60; Raf: LIN-45; MEK: MEK-2; MAPK/Erk: MPK-1; Notch: LIN-12; Frizzled and Ryk (WNT-receptors): LIN-17 and LIN-18. <em>C. elegans</em> protein naming follows the <a href="https://wormbase.org//about/userguide/nomenclature#634c2j7f9gi1e05kabmlh8d--10">WormBase convention</a><a href="#fnref1" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/what-is-the-point-of-studying-developmental-biology.html" class="archive-item-link">
                What is the Point of Studying Developmental Biology 
              </a>
              <span class="archive-item-date">
                2020-02-26
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p><strong>Title: What is the point of studying developmental biology?</strong></p>
<p><a href="https://tianyishi2001.github.io/ox/developmental-biology-presentation/#/">Link to presentation</a></p>
<p>The main theme of developmental biology is the fate determination of cells, their division, differention and death, as well as the resulting higher level processes, including morphogenesis and organ formation. I’ll illustrate how this knowledge can help with preventive and public healthcare as well as modern medical research.</p>
<div id="prenatal-care-preventing-teratogenesis" class="section level1">
<h1>Prenatal Care (Preventing Teratogenesis)</h1>
<div id="teratogen-and-congenital-anomaly" class="section level2">
<h2>Teratogen and congenital anomaly</h2>
<p>According to the US’ National Research Council, ~3% of live births suffer from major developmental aberrations, ~70% of neonatal deaths are due to developmental defects.</p>
<ul>
<li>exogenous agents that cause birth defects are teratogens, they can be chemicals, radiations, or viral infections, and so on</li>
<li>thalidomide
<ul>
<li>used to treat anxiety</li>
<li>malformed arms and legs</li>
<li>thee first major evidence that drugs could induce congenital anomalies</li>
<li>led to the development of greater drug regulation and monitoring in many countries</li>
</ul></li>
<li>alcohol interferes with fetal neural development and is shown to cause mental retardation in the newborn
<ul>
<li>fetal alcohol syndrome</li>
<li>abnormal facial development</li>
<li>most prevalent type of congenital mental retardation syndrome in the US</li>
</ul></li>
<li>in pregnant women, Zika virus directly infects the neural progenitor cells of the fetal cortex (vertically transmitted infection), resulting in the death of these cells</li>
<li>Model organisms such as Xenopus and Denio, are often used to screen compound that have a high probability of being hazardous, because the early development of these organisms rely on the same basic paracrine factors and TFs as humans do.
<ul>
<li>studies on model organisms have revealed three probable pathways for alcohol teratogenesis. first, generation of superoxide redicals; second, downregulation of Sonic hedgehog; third, intereference of the cell adhesion molecule L1</li>
</ul></li>
</ul>
</div>
<div id="nutrients" class="section level2">
<h2>Nutrients</h2>
<p>to sustain normal development of the fetus, healthy diet is needed. This is quite obvious, however, there is one vitamin, that cannot be obtained with enough amount just from food, it is folic acid</p>
<ul>
<li>Folic acid
<ul>
<li>it is found to significantly reduce the chance of <strong>neural tube defects in infants</strong>, but only at high doses</li>
<li>it’s recommended, by NHS, that all pregnant women should take a daily supplement of 400 migrograms of folic acid before they’re pregnant and during the first 12 weeeks of pregnancy, when the baby’s spine is developing</li>
</ul></li>
</ul>
</div>
</div>
<div id="stem-cell" class="section level1">
<h1>Stem Cell</h1>
<p>an important subdomain of developmental biology is stem cell biology, which has many applications in research and medicine. here I’ll focus on pluripotent stem cells</p>
<div id="es-cells" class="section level2">
<h2>ES Cells</h2>
<ul>
<li>ES cells are the natural source of pluripotent cells</li>
<li>ICM (embryoblast) and trophoblast</li>
<li>the study of the normal sequence of embryonic inductions yielded the methods for induced differentiation of these cells</li>
<li>derived from the inner cell mass of blastocysts from IVF clinic</li>
<li>harvesting human ES cells means destroying human embryos (5 days after fertilisation)</li>
<li>raised ethical controversy</li>
</ul>
</div>
<div id="ips-cells" class="section level2">
<h2>iPS cells</h2>
<ul>
<li>Reverting to undifferentated state
<ul>
<li>functionallty equivalent to embryonic stem cells</li>
<li>capable of differentiating into almost all cell types</li>
</ul></li>
<li>Shinya Yamanaka factors: Oct4, Klf4, Sox2, c-Myc
<ul>
<li>screened 22 factors</li>
</ul></li>
<li>Regenerative Medicine
<ul>
<li>SMA clinical trial in Japan</li>
</ul></li>
<li>Organoids
<ul>
<li>An organoid is a miniaturized and simplified version of an organ produced in vitro in three dimensions that shows realistic micro-anatomy</li>
</ul></li>
</ul>
</div>
</div>
<div id="cancer-as-a-disease-of-development" class="section level1">
<h1>Cancer as a Disease of Development</h1>
<ul>
<li>carcinogenesis can be viewed as a disease of development because it involes aberrations of the processes that underlie cell division, differentiation and morphogenesis</li>
<li>once it was thought that carcinogenesis and metastasis were caused by cells that had acquired mutations enabling them to proliferate independent of the enviornment.</li>
<li>but it turns out that this is not the complete explanation</li>
<li>cancer cells can actuallt modify their environment, turning it into a cancer-promoting niche</li>
<li>this means the progression of many cancers depends on reciprocal interactions between hte cancer cells and the supporting cells of their tissue environment</li>
<li><p>Indeed, carcinogenesis appears to recapitulate steps of normal development, including the formation of a niche in which to proliferate, using the same or closely related signalling pathways</p></li>
<li>Defects in cell-cell conmmunication
<ul>
<li>studies have shown that tumours can be cuased by altering hte structure of the surrounding tissue, and that these tumors can be suppressed by restoring an appropriate tissue envionment</li>
<li>althogh 80% human tumors are from epithelial cells, the cells that carcinogens act on are often not the epithelial cells themselves, but the mesenchymal stromal cells that surround and sustain the epithelia</li>
<li>there was a study in which normal and carcinogen-treated epithelia and mesenchyme in rat mammary glands are <strong>recombined</strong>, and it turns out that, tumor growth occurred not in carcinogen-treated epithelia, but in epithelia placed in combination with carcinogen-treated mesenchyme</li>
<li>in this case, the mesenchyme fails to give instructions to the epithelia to form normals structures, and epithelial cells exhibited a loose control of cell proliferation</li>
</ul></li>
<li>Defects in paracrine pathways
<ul>
<li>Several key signaling pathways involved in embryogenesis, such as Hedgehog, Notch and Wnt, also have crucial roles in carcinogenesis when improperly activated in adults through sporadic mutatiions or other mechanisms</li>
<li>many tumors, secrete the paracrine signalling factor Shh, which can act in two ways</li>
<li><strong>the same chemicals that can cause teratogenesis by blocking a pathway in embryonic development may be useful in blocking the activation of cancer cells</strong> (cancer stem cells, as I will describe next)</li>
<li>Cyclopamine and other antagonists of the Shh pathway, for instance, appear to be useful in preventing hte generation and proliferation of meedulloblastoma stem cells</li>
</ul></li>
<li>Cancer stem cells
<ul>
<li>an aspect of viewing cancers as diseases of development is that the properties of tumors may emerge because of a population of cells that are analogous to adult stem cells</li>
<li>this is shown in studies on rat intestinal adenomas</li>
<li>the lumen of the mice intestine is made up of villi and crypts</li>
<li>at the bottom of the crypts there reside two important cells, Lrg5+ cells and Paneth cells, Lgr5+ is the stem cell and Paneth cell is the</li>
<li>lineage tracing revealed that
<ul>
<li>the stem cells of rat intestinal adenomas also expresses Lgr5 and has the same interact with P cell as normal stem cells do</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Gilbert-2016">
<p>Gilbert, Scott F., and Michael J. F. Barresi. 2016. <em>Developmental Biology</em>. 11th ed. Sinauer Associates.</p>
</div>
<div id="ref-Slack-2018">
<p>Slack, Jonathan M. W. 2018. <em>Essential Developmental Biology</em>. 3rd Ser. WIley-Blackwell.</p>
</div>
<div id="ref-Song-2011">
<p>Song, Wen AND Wei, Zhou AND Yue. 2011. “Sonic Hedgehog Pathway Is Essential for Maintenance of Cancer Stem-Like Cells in Human Gastric Cancer.” <em>PLOS ONE</em> 6 (3). Public Library of Science: 1–13. <a href="https://doi.org/10.1371/journal.pone.0017687">https://doi.org/10.1371/journal.pone.0017687</a>.</p>
</div>
<div id="ref-Takeo-2018">
<p>Takeo, Makoto, and Takashi Tsuji. 2018. “Organ Regeneration Based on Developmental Biology: Past and Future.” <em>Current Opinion in Genetics &amp; Development</em> 52: 42–47. <a href="https://doi.org/https://doi.org/10.1016/j.gde.2018.05.008">https://doi.org/https://doi.org/10.1016/j.gde.2018.05.008</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/three-biophysical-methods.html" class="archive-item-link">
                Three Biophysical Methods 
              </a>
              <span class="archive-item-date">
                2020-02-22
              </span>
              <div class="article-summary" style="display: none;">
                
                
<script src="./rmarkdown-libs/htmlwidgets/htmlwidgets.js"></script>
<script src="./rmarkdown-libs/plotly-binding/plotly.js"></script>
<script src="./rmarkdown-libs/typedarray/typedarray.min.js"></script>
<script src="./rmarkdown-libs/jquery/jquery.min.js"></script>
<link href="./rmarkdown-libs/crosstalk/css/crosstalk.css" rel="stylesheet" />
<script src="./rmarkdown-libs/crosstalk/js/crosstalk.min.js"></script>
<link href="./rmarkdown-libs/plotly-htmlwidgets-css/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="./rmarkdown-libs/plotly-main/plotly-latest.min.js"></script>

<div id="TOC">
<ul>
<li><a href="#circular-dichorism-cd"><span class="toc-section-number">1</span> Circular Dichorism (CD)</a><ul>
<li><a href="#physical-principles"><span class="toc-section-number">1.1</span> Physical Principles</a></li>
<li><a href="#experimental-setup"><span class="toc-section-number">1.2</span> Experimental Setup</a></li>
<li><a href="#ss"><span class="toc-section-number">1.3</span> Application of CD</a></li>
<li><a href="#other-cd-based-techniques"><span class="toc-section-number">1.4</span> Other CD-based Techniques</a></li>
</ul></li>
<li><a href="#small-angle-x-ray-scattering-saxs"><span class="toc-section-number">2</span> Small-angle X-ray Scattering (SAXS)</a><ul>
<li><a href="#physical-principles-1"><span class="toc-section-number">2.1</span> Physical Principles</a></li>
<li><a href="#experimentation-and-data-processing"><span class="toc-section-number">2.2</span> Experimentation and Data Processing</a></li>
</ul></li>
<li><a href="#analytical-ultracentrifuge-auc"><span class="toc-section-number">3</span> Analytical Ultracentrifuge (AUC)</a><ul>
<li><a href="#choice-of-optical-system"><span class="toc-section-number">3.1</span> Choice of Optical System</a></li>
<li><a href="#experiments"><span class="toc-section-number">3.2</span> Experiments</a><ul>
<li><a href="#sedimentation-velocity-sv"><span class="toc-section-number">3.2.1</span> Sedimentation Velocity (SV)</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="circular-dichorism-cd" class="section level1">
<h1><span class="header-section-number">1</span> Circular Dichorism (CD)</h1>
<p>Reference: <span class="citation">Ranjbar and Gill (2009)</span>; <span class="citation">Kelly, Jess, and Price (2005)</span>;<span class="citation">Whitmore and Wallace (2008)</span></p>
<p>Circular dichorism measures the difference in absorption of left-hand and right-hand circularly polarised light by optically active (chiral) molecules (usually biological macromolecules). It serves as a rapid way to assess structural information on proteins, carbohydrates, nucleic acids, pharmaceuticals, liquiud crystals, etc., in the solution state.</p>
<div id="physical-principles" class="section level2">
<h2><span class="header-section-number">1.1</span> Physical Principles</h2>
<p>Electromagnetic waves (EM waves) has an electrical (E) component and a magnetic (M) component perpendicular to each other and both are perpendicular to the axis of propagation. Usually, only the E component is depicted for simplicity.</p>
<p>Most light sources have E components in all orientations, and when they pass through a slit, they become plane polarised, and can be represented by a simple sinusoidal wave. Adding up two plane polarised light perpendicular to each other produces interesting resultant waves, which can be visualised with <strong>plotly</strong> in R <span class="citation">(Sievert 2018)</span> (which is an uncommon use ‘invented’ by me<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>). Assuming we are superposing two sinusoidal waves of the same amplitude and frequency with various phase differences, we can then compute the pattern of the resultant wave as a function of the phase difference:</p>
<pre class="r"><code>library(plotly)
library(tibble)

calc_waves &lt;- function(phase_diff = 0){
  # compute y and z values for given x values of the waves 
  # first arbitrarily choose evenly-spaced points on the x_axis
  x_axis = x=seq(1,20,0.1) 
  # compute the y, z values of the first sin wave on the xy plane
  sin1 &lt;- tibble(x=x_axis,y=sin(x),z=0)
  # compute the y, z values of the second sin wave on the xz plane, 
  # taking the phase difference into account
  sin2 &lt;- tibble(x=x_axis,y=0,z=sin(x + phase_diff))
  # compute the axes of the resultant wave
  # each y and z value is the sum of corresponding values in the 2 component waves
  resultant &lt;- tibble(x=x_axis, y = sin1$y+sin2$y, z = sin1$z+sin2$z)
  
  return(list(sin1, sin2, resultant))
} 

plot_waves &lt;- function(wave_list, components = c(&#39;sin1&#39;, &#39;sin2&#39;), resultant = &#39;resultant&#39;){
  # plotting
  plot_ly() %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=components[1], data = wave_list[[1]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;) %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=components[2], data = wave_list[[2]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;) %&gt;% 
    add_trace(x=~x, y=~y, z=~z, name=resultant, data = wave_list[[3]], type=&#39;scatter3d&#39;, mode=&#39;lines&#39;)
}</code></pre>
<p>When phase difference is zero, the resultant wave is also sinusoidal, and the plane in which it resides is oriented at 45<span class="math inline">\(^\circ\)</span> to each component wave, as shown in Fig. <a href="#fig:si">1.1</a> (<em>this is an interactive HTML widget so you can rotate, drag and zoom! Unfortunately it cannot be rendered in LaTeX, so please visit ../tutorial/three-biophysical-methods.html if you are reeading the LaTeX PDF output</em>).</p>
<pre class="r"><code>calc_waves(phase_diff=0) %&gt;% plot_waves()</code></pre>
<div class="figure"><span id="fig:si"></span>
<div id="htmlwidget-1" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1">{"x":{"visdat":{"629ff04c89d":["function () ","plotlyVisDat"],"629f30ec31f":["function () ","data"],"629f3e2c29ca":["function () ","data"],"629f456baca9":["function () ","data"]},"cur_data":"629f456baca9","attrs":{"629f30ec31f":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f3e2c29ca":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f456baca9":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"name":"resultant","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.1: 0 phase difference results in a sinusoidal wave
</p>
</div>
<p>As shown in <a href="#fig:lh">1.2</a>, When the phase difference is <span class="math inline">\(\pi/2\)</span>, the resultant wave is helical and is said to be <strong>circularly polarised</strong> (for other phase differences, the pattern is helical but ellipised.</p>
<pre class="r"><code>L &lt;- calc_waves(phase_diff=pi/2)
plot_waves(L, resultant = &#39;resultant: left circularly polarised&#39;)</code></pre>
<div class="figure"><span id="fig:lh"></span>
<div id="htmlwidget-2" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-2">{"x":{"visdat":{"629f2de37aaa":["function () ","plotlyVisDat"],"629f318a466f":["function () ","data"],"629f6d163ad1":["function () ","data"],"629f4e6fa74a":["function () ","data"]},"cur_data":"629f4e6fa74a","attrs":{"629f318a466f":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f6d163ad1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f4e6fa74a":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant: left circularly polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"resultant: left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.2: Formation of left circularly polarised wave.
</p>
</div>
<pre class="r"><code>L &lt;- L[[3]] # the third tibble represents the resultant, which will be used later</code></pre>
<p>Note the handedness of the resultant wave: when the phase difference is <span class="math inline">\(\pi/2\)</span>, it is left-handed and it becomes right-handed when the phase difference is <span class="math inline">\(-\pi/2\)</span>. The latter is drawn in Fig. <a href="#fig:rh">1.3</a>.</p>
<pre class="r"><code>R &lt;- calc_waves(phase_diff=-pi/2)
plot_waves(R, resultant = &#39;resultant: right circularly polarised&#39;)</code></pre>
<div class="figure"><span id="fig:rh"></span>
<div id="htmlwidget-3" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-3">{"x":{"visdat":{"629f25307c5d":["function () ","plotlyVisDat"],"629f1234d0be":["function () ","data"],"629f49747348":["function () ","data"],"629f7c3ca1a4":["function () ","data"]},"cur_data":"629f7c3ca1a4","attrs":{"629f1234d0be":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin1","type":"scatter3d","mode":"lines","inherit":true},"629f49747348":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"sin2","type":"scatter3d","mode":"lines","inherit":true},"629f7c3ca1a4":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"resultant: right circularly polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"name":"sin1","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"sin2","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"resultant: right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.3: Formation of right circularly polarised wave.
</p>
</div>
<pre class="r"><code>R &lt;- R[[3]]</code></pre>
<p>As shown in Fig. <a href="#fig:plane-polarised">1.4</a> If we add up the left and right circularly polarised light, the resultant is a plane polarised light. Thus it can be said that a plane polarised light can be viewed as being made up of two circularly polarised components of equal magnitude and frequency.</p>
<pre class="r"><code>plot_waves(list(L, R, 
  resultant=tibble(x=L$x, y = L$y+R$y, z = L$z+R$z)),
  components = c(&#39;left circularly polarised&#39;, &#39;right circularly polarised&#39;),
  resultant = &#39;plane polarised&#39;)</code></pre>
<div class="figure"><span id="fig:plane-polarised"></span>
<div id="htmlwidget-4" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-4">{"x":{"visdat":{"629f2122d62c":["function () ","plotlyVisDat"],"629f761af3b2":["function () ","data"],"629f637567b1":["function () ","data"],"629f30eac87a":["function () ","data"]},"cur_data":"629f30eac87a","attrs":{"629f761af3b2":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"left circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f637567b1":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"right circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f30eac87a":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"plane polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[1.68294196961579,1.78241472012287,1.86407817193445,1.92711637083439,1.97089945997692,1.99498997320811,1.99914720608301,1.98332962090494,1.94769526175639,1.89260017537483,1.81859485365136,1.72641873329775,1.61699280763918,1.49141042435344,1.3509263611023,1.19694428820791,1.03100274364293,0.85475976046766,0.66997630031181,0.478498658427964,0.282240016119734,0.083161324866581,-0.11674828685516,-0.315491388286497,-0.511082204053663,-0.70156645537924,-0.885040886589705,-1.05967228181699,-1.22371578188544,-1.37553231836795,-1.51360499061586,-1.63655422212882,-1.74315154482718,-1.83233187349891,-1.90320414777903,-1.95506023533019,-1.98738200726693,-1.9998465151282,-1.99232921767168,-1.96490522524866,-1.91784854932628,-1.85162936465546,-1.76690931144031,-1.6645348844478,-1.54552897511197,-1.41108065114078,-1.26253327574464,-1.10137108519528,-0.929204358827513,-0.747753329660472,-0.558830996397852,-0.36432500854419,-0.166178805634993,0.0336278009687012,0.233098409700987,0.430239976175631,0.623082727026757,0.809699841233197,0.988226702277218,1.1568795287764,1.31397319743758,1.45793808025175,1.58733572769831,1.70087324125713,1.79741619162325,1.87599995354948,1.93583934406297,1.976336467754,1.99708669074921,1.99788268367954,1.97871649324676,1.93977962169017,1.88146111335955,1.80434366751259,1.70919781617656,1.59697422524698,1.46879419574822,1.32593846016437,1.16983438578352,1.00204171291577,0.824236970483513,0.638196724698704,0.445779828200492,0.248908847014123,0.0495508509067155,-0.150302240923619,-0.348653562445959,-0.543521252821888,-0.732958258503857,-0.915071787550643,-1.08804222177874,-1.25014129778576,-1.39974937518709,-1.53537161952716,-1.65565293817131,-1.75939151994334,-1.84555084322561,-1.91327003254038,-1.96187246013298,-1.99087250661275,-1.99998041310141,-1.98910517640798,-1.95835545830263,-1.90803849980418,-1.83865705132935,-1.75090434937686,-1.64565718993742,-1.52396716783806,-1.38705016955424,-1.23627422447407,-1.07314583600087,-0.8992949290692,-0.716458564473654,-0.526463582731602,-0.331208350896619,-0.132643794702401,0.0672460944422769,0.266464082839888,0.463019650203078,0.654948878275386,0.840334073653282,1.01732292874475,1.18414702941445,1.3391395243932,1.4807517799049,1.60756885310324,1.71832362971299,1.81190948461693,1.88739133888821,1.94401500278995,1.98121471138974,1.99861877749584,1.99605330543272,1.97354392854923,1.93131555309856,1.86979011104937,1.78958234428101,1.69149366228587,1.57650413475063,1.44576269902395,1.30057568031423,1.14239373931997,0.972797377707596,0.793481146261224,0.606236713491405,0.412934963875593,0.215507304598885,0.0159263675718711,-0.183813700455363,-0.381717162748379,-0.575806633330131,-0.764142834368018,-0.944843972796939,-1.11610454257356,-1.27621336469589,-1.42357068473825,-1.5567041570686,-1.6742835560395,-1.77513406716301,-1.85824802546874,-1.92279498375911,-1.96813001016329,-1.99380013208319,-1.99954886214602,-1.98531876094127,-1.95125201093632,-1.89768899583625,-1.82516489958237,-1.73440435897116,-1.62631422332297,-1.50197449354335,-1.362627531111,-1.20966564481257,-1.04461717925346,-0.869131244143787,-0.684961236939225,-0.493947323473242,-0.297998051628398,-0.0990712817567348,0.10084537561363,0.299754419325905,0.49566841596592,0.686629857639791,0.870730720745786,1.0461315303154,1.2110797394392,1.36392724013627,1.5031468307043,1.62734747501421,1.73528820128334,1.82589050145526],"z":[1.11022302462516e-16,1.11022302462516e-16,-1.11022302462516e-16,3.33066907387547e-16,1.11022302462516e-16,1.2490009027033e-16,1.21430643318376e-16,1.11022302462516e-16,3.33066907387547e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,3.33066907387547e-16,0,0,1.11022302462516e-16,0,0,0,1.11022302462516e-16,-1.11022302462516e-16,0,-3.33066907387547e-16,-1.11022302462516e-16,2.22044604925031e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-5.55111512312578e-17,-1.38777878078145e-16,-1.2490009027033e-16,-1.21430643318376e-16,-1.2490009027033e-16,-1.38777878078145e-16,-1.11022302462516e-16,-1.11022302462516e-16,-5.55111512312578e-17,-1.11022302462516e-16,-1.11022302462516e-16,0,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,0,0,0,3.33066907387547e-16,-2.22044604925031e-16,0,1.11022302462516e-16,1.11022302462516e-16,6.66133814775094e-16,-5.55111512312578e-16,1.11022302462516e-16,1.66533453693773e-16,1.66533453693773e-16,1.0547118733939e-15,-7.7715611723761e-16,1.2490009027033e-16,1.2490009027033e-16,1.11022302462516e-16,8.32667268468867e-17,1.66533453693773e-16,1.66533453693773e-16,1.11022302462516e-16,0,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,0,1.11022302462516e-16,1.11022302462516e-16,0,0,0,-1.11022302462516e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.38777878078145e-16,-1.2490009027033e-16,-1.22298005056365e-16,-1.2490009027033e-16,-1.38777878078145e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,-1.11022302462516e-16,0,0,0,0,0,0,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,0,2.22044604925031e-16,2.22044604925031e-16,2.22044604925031e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.11022302462516e-16,1.17961196366423e-16,1.2490009027033e-16,1.11022302462516e-16,1.11022302462516e-16,-1.55431223447522e-15,-1.49880108324396e-15,2.22044604925031e-16,1.66533453693773e-15,1.11022302462516e-16,-9.99200722162641e-16,-8.88178419700125e-16,1.11022302462516e-16,7.7715611723761e-16,0,-3.33066907387547e-16,-2.22044604925031e-16,0,-1.11022302462516e-16,-1.11022302462516e-16,5.55111512312578e-16,6.66133814775094e-16,7.7715611723761e-16,-1.11022302462516e-15,1.11022302462516e-15,1.22124532708767e-15,1.33226762955019e-15,1.33226762955019e-15,-1.66533453693773e-15,1.55431223447522e-15,1.60982338570648e-15,1.63757896132211e-15,-1.90125692967058e-15,-1.90125692967058e-15,-1.88737914186277e-15,1.60982338570648e-15,3.21964677141295e-15,-3.38618022510673e-15,-3.21964677141295e-15,-2.99760216648792e-15,2.55351295663786e-15,2.44249065417534e-15,-2.22044604925031e-15,-1.88737914186277e-15,-1.55431223447522e-15,1.11022302462516e-15,7.7715611723761e-16,-5.55111512312578e-16,-2.22044604925031e-16,2.22044604925031e-16,-5.55111512312578e-16,-8.88178419700125e-16,1.22124532708767e-15,1.66533453693773e-15,1.88737914186277e-15,-2.1094237467878e-15,-2.33146835171283e-15,2.77555756156289e-15,2.99760216648792e-15,3.21964677141295e-15,-3.10862446895044e-15],"name":"plane polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.4: Addition of two identical waves of oppsite handedness results in a plane polarised wave.
</p>
</div>
<p>CD measures differential absorption of these components. If the two components are absorbed to the same extent, clearly the resultant will still be planar as shown above. If there is differential absorption, as I simulate in Fig. <a href="#fig:diff-absorb">1.5</a>, the resultant wave will be elliptical polarised. Here the R component is absorbed more than the L component, and the resultant ellipsed wave is left-handed. The CD signal (formally <span class="math inline">\(\Delta A=\Delta A_\text{L}-\Delta A_\text{R}\)</span>) is generally reported in terms of the ellipticity, <span class="math inline">\(\theta\)</span>, of this resultant wave. <span class="math inline">\(\theta=\arctan(b/a)\)</span> where b and a are the minor and major axes of the ellipse. <span class="math inline">\(\theta\)</span> can be easily converted to <span class="math inline">\(\Delta A\)</span> by the simple relationship: <span class="math inline">\(\theta=32.98\Delta A\)</span>. The CD spectrum is obtained when the CD signal <span class="math inline">\(\theta\)</span> or <span class="math inline">\(\Delta A\)</span> is measured as a function of wavelength.</p>
<pre class="r"><code>L1 &lt;- L %&gt;% mutate(y = 0.9 * y, z = 0.9 * z) # 10% absorption of L
R1 &lt;- R %&gt;% mutate(y = 0.7 * y, z = 0.7 * z) # 30% absorption of R
plot_waves(list(L, R, 
  resultant=tibble(x=L1$x, y = L1$y+R1$y, z = L1$z+R1$z)),
  components = c(&#39;left circularly polarised&#39;, &#39;right circularly polarised&#39;),
  resultant = &#39;elliptical polarised&#39;)</code></pre>
<div class="figure"><span id="fig:diff-absorb"></span>
<div id="htmlwidget-5" style="width:100%;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-5">{"x":{"visdat":{"629f7add96f6":["function () ","plotlyVisDat"],"629f69e22f7e":["function () ","data"],"629f9c2f81":["function () ","data"],"629fdf2be77":["function () ","data"]},"cur_data":"629fdf2be77","attrs":{"629f69e22f7e":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"left circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629f9c2f81":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"right circularly polarised","type":"scatter3d","mode":"lines","inherit":true},"629fdf2be77":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"x":{},"y":{},"z":{},"name":"elliptical polarised","type":"scatter3d","mode":"lines","inherit":true}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"x"},"yaxis":{"title":"y"},"zaxis":{"title":"z"}},"hovermode":"closest","showlegend":true},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[0.54030230586814,0.453596121425577,0.362357754476673,0.267498828624588,0.169967142900241,0.070737201667703,-0.0291995223012888,-0.128844494295525,-0.227202094693087,-0.323289566863503,-0.416146836547142,-0.504846104599857,-0.588501117255346,-0.666276021279824,-0.737393715541246,-0.801143615546934,-0.856888753368947,-0.904072142017061,-0.942222340668658,-0.970958165149591,-0.989992496600445,-0.999135150273279,-0.998294775794753,-0.987479769908865,-0.966798192579461,-0.936456687290796,-0.896758416334147,-0.848100031710408,-0.790967711914416,-0.72593230420014,-0.653643620863612,-0.574823946533269,-0.490260821340699,-0.400799172079975,-0.307332869978419,-0.21079579943078,-0.112152526935055,-0.0123886634628906,0.0874989834394472,0.186512369422576,0.283662185463226,0.377977742712981,0.468516671300377,0.554374336179161,0.634692875942635,0.70866977429126,0.77556587851025,0.83471278483916,0.885519516941319,0.927478430744036,0.960170286650366,0.983268438442585,0.996542097023217,0.999858636383415,0.993184918758193,0.976587625728023,0.95023259195853,0.914383148235319,0.869397490349825,0.815725100125357,0.753902254343305,0.684546666442807,0.608351314532254,0.526077517381105,0.43854732757439,0.346635317835026,0.251259842582256,0.153373862037864,0.0539554205626489,-0.0460021256395369,-0.145500033808613,-0.243544153735793,-0.339154860983834,-0.431376844970621,-0.519288654116686,-0.602011902684824,-0.678720047320014,-0.748646645597399,-0.811093014061656,-0.865435209241112,-0.911130261884677,-0.947721602131112,-0.974843621404164,-0.992225325452603,-0.999693042035206,-0.997172156196378,-0.984687855794127,-0.96236487983131,-0.930426272104753,-0.889191152625361,-0.839071529076453,-0.780568180169184,-0.714265652027199,-0.640826417594993,-0.560984257427229,-0.475536927995993,-0.385338190771828,-0.291289281721344,-0.194329906455335,-0.0954288510009507,0.00442569798805072,0.104236026865699,0.203004863818752,0.299745343277015,0.393490866347891,0.483304758753006,0.568289629767975,0.647596338653877,0.720432478990839,0.786070296141039,0.843853958732492,0.893206111509323,0.933633644074638,0.96473261788661,0.986192302278864,0.997798279178581,0.999434585501005,0.991084871814253,0.972832565697435,0.944860038159861,0.907446781450196,0.860966616462306,0.80588395764045,0.742749172703669,0.672193083553468,0.594920663309892,0.511703992453148,0.423374544450664,0.330814877949047,0.234949818539823,0.136737218207834,0.0371583847908247,-0.0627917229240835,-0.162114436499718,-0.259817356213756,-0.354924266788707,-0.446484891412269,-0.533584386589119,-0.615352482954719,-0.690972180719126,-0.759687912858822,-0.82081309449267,-0.873736983011081,-0.917930780414293,-0.95295291688718,-0.978453462818885,-0.994177625183816,-0.99996829334934,-0.995767608873289,-0.981617543606384,-0.957659480323384,-0.924132800073129,-0.881372490362232,-0.82980579807065,-0.769947960542071,-0.702397057502712,-0.627828035246385,-0.546985962794231,-0.460678587411364,-0.369768263863172,-0.275163338051595,-0.177809071123114,-0.078678194731842,0.0212388081736442,0.120943599928476,0.219439963211461,0.315743754919245,0.408892739398878,0.497956202788414,0.582044252402124,0.660316708244081,0.731991497808948,0.796352470291922,0.852756552130872,0.900640172384769,0.939524893748257,0.96902219293905,0.988837342694146,0.99877235658721,0.998727967243502,0.988704618186669,0.96880245940721,0.939220346696871,0.900253854747305,0.852292323865463,0.795814969813943,0.731386095645496,0.659649453373463,0.581321811814437,0.497185794871204,0.40808206181339],"name":"left circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(31,119,180,1)","line":{"color":"rgba(31,119,180,1)"}},"error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"line":{"color":"rgba(31,119,180,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[0.841470984807897,0.891207360061435,0.932039085967226,0.963558185417193,0.98544972998846,0.997494986604054,0.999573603041505,0.991664810452469,0.973847630878195,0.946300087687414,0.909297426825682,0.863209366648874,0.80849640381959,0.74570521217672,0.675463180551151,0.598472144103956,0.515501371821464,0.42737988023383,0.334988150155905,0.239249329213982,0.141120008059867,0.0415806624332905,-0.0583741434275801,-0.157745694143249,-0.255541102026832,-0.35078322768962,-0.442520443294852,-0.529836140908493,-0.611857890942719,-0.687766159183974,-0.756802495307928,-0.81827711106441,-0.871575772413588,-0.916165936749455,-0.951602073889516,-0.977530117665097,-0.993691003633464,-0.999923257564101,-0.996164608835841,-0.982452612624332,-0.958924274663138,-0.925814682327732,-0.883454655720153,-0.832267442223901,-0.772764487555987,-0.705540325570392,-0.631266637872321,-0.550685542597638,-0.464602179413757,-0.373876664830236,-0.279415498198926,-0.182162504272095,-0.0830894028174964,0.0168139004843506,0.116549204850494,0.215119988087816,0.311541363513379,0.404849920616598,0.494113351138609,0.5784397643882,0.656986598718789,0.728969040125876,0.793667863849153,0.850436620628565,0.898708095811627,0.937999976774739,0.967919672031487,0.988168233877,0.998543345374605,0.998941341839772,0.989358246623382,0.969889810845086,0.940730556679773,0.902171833756293,0.854598908088281,0.79848711262349,0.734397097874112,0.662969230082183,0.584917192891762,0.501020856457885,0.412118485241757,0.319098362349352,0.222889914100246,0.124454423507062,0.0247754254533578,-0.0751511204618093,-0.17432678122298,-0.271760626410944,-0.366479129251928,-0.457535893775321,-0.54402111088937,-0.625070648892882,-0.699874687593544,-0.767685809763582,-0.827826469085654,-0.87969575997167,-0.922775421612807,-0.956635016270188,-0.980936230066492,-0.995436253306377,-0.999990206550703,-0.994552588203989,-0.979177729151317,-0.954019249902089,-0.919328525664676,-0.875452174688429,-0.822828594968708,-0.761983583919032,-0.693525084777122,-0.618137112237033,-0.536572918000435,-0.4496474645346,-0.358229282236827,-0.263231791365801,-0.165604175448309,-0.0663218973512007,0.0336230472211385,0.133232041419944,0.231509825101539,0.327474439137693,0.420167036826641,0.508661464372375,0.592073514707224,0.669569762196602,0.740375889952449,0.803784426551621,0.859161814856497,0.905954742308463,0.943695669444105,0.972007501394976,0.99060735569487,0.999309388747918,0.998026652716362,0.986771964274613,0.965657776549278,0.934895055524683,0.894791172140503,0.845746831142933,0.788252067375316,0.722881349511976,0.650287840157117,0.571196869659987,0.486398688853798,0.396740573130612,0.303118356745702,0.206467481937797,0.107753652299442,0.00796318378593557,-0.0919068502276816,-0.190858581374189,-0.287903316665065,-0.382071417184009,-0.472421986398469,-0.558052271286779,-0.638106682347947,-0.711785342369123,-0.778352078534299,-0.837141778019749,-0.887567033581505,-0.929124012734368,-0.961397491879557,-0.984065005081643,-0.996900066041596,-0.999774431073011,-0.992659380470633,-0.975626005468158,-0.948844497918124,-0.912582449791185,-0.867202179485581,-0.813157111661486,-0.750987246771676,-0.6813137655555,-0.604832822406284,-0.522308589626732,-0.434565622071894,-0.342480618469613,-0.246973661736621,-0.148999025814199,-0.0495356408783674,0.0504226878068148,0.149877209662952,0.24783420798296,0.343314928819895,0.435365360372893,0.523065765157699,0.605539869719601,0.681963620068135,0.751573415352148,0.813673737507105,0.867644100641669,0.912945250727628],"z":[-0.54030230586814,-0.453596121425577,-0.362357754476674,-0.267498828624587,-0.169967142900241,-0.0707372016677029,0.0291995223012889,0.128844494295525,0.227202094693087,0.323289566863503,0.416146836547142,0.504846104599858,0.588501117255346,0.666276021279824,0.737393715541246,0.801143615546934,0.856888753368947,0.904072142017061,0.942222340668658,0.970958165149591,0.989992496600445,0.999135150273279,0.998294775794753,0.987479769908865,0.966798192579461,0.936456687290796,0.896758416334147,0.848100031710408,0.790967711914417,0.72593230420014,0.653643620863612,0.574823946533269,0.490260821340699,0.400799172079975,0.307332869978419,0.21079579943078,0.112152526935055,0.0123886634628905,-0.0874989834394473,-0.186512369422576,-0.283662185463226,-0.377977742712981,-0.468516671300377,-0.554374336179161,-0.634692875942635,-0.70866977429126,-0.77556587851025,-0.83471278483916,-0.885519516941319,-0.927478430744036,-0.960170286650366,-0.983268438442585,-0.996542097023217,-0.999858636383415,-0.993184918758193,-0.976587625728023,-0.950232591958529,-0.914383148235319,-0.869397490349825,-0.815725100125357,-0.753902254343305,-0.684546666442806,-0.608351314532254,-0.526077517381105,-0.43854732757439,-0.346635317835026,-0.251259842582255,-0.153373862037864,-0.0539554205626488,0.046002125639537,0.145500033808614,0.243544153735793,0.339154860983835,0.431376844970621,0.519288654116686,0.602011902684824,0.678720047320014,0.748646645597399,0.811093014061656,0.865435209241112,0.911130261884677,0.947721602131112,0.974843621404164,0.992225325452604,0.999693042035206,0.997172156196378,0.984687855794127,0.96236487983131,0.930426272104753,0.889191152625361,0.839071529076452,0.780568180169184,0.714265652027199,0.640826417594993,0.560984257427229,0.475536927995993,0.385338190771828,0.291289281721344,0.194329906455335,0.0954288510009506,-0.00442569798805085,-0.104236026865699,-0.203004863818752,-0.299745343277015,-0.393490866347891,-0.483304758753006,-0.568289629767975,-0.647596338653877,-0.720432478990839,-0.786070296141039,-0.843853958732492,-0.893206111509323,-0.933633644074638,-0.96473261788661,-0.986192302278864,-0.997798279178581,-0.999434585501005,-0.991084871814253,-0.972832565697435,-0.944860038159861,-0.907446781450196,-0.860966616462306,-0.80588395764045,-0.742749172703669,-0.672193083553468,-0.594920663309892,-0.511703992453147,-0.423374544450664,-0.330814877949047,-0.234949818539823,-0.136737218207834,-0.0371583847908246,0.0627917229240836,0.162114436499718,0.259817356213756,0.354924266788705,0.446484891412267,0.533584386589119,0.615352482954721,0.690972180719126,0.759687912858821,0.820813094492669,0.873736983011081,0.917930780414293,0.95295291688718,0.978453462818884,0.994177625183815,0.99996829334934,0.995767608873289,0.981617543606384,0.957659480323385,0.92413280007313,0.881372490362233,0.829805798070649,0.769947960542072,0.702397057502714,0.627828035246386,0.546985962794233,0.460678587411362,0.369768263863173,0.275163338051597,0.177809071123116,0.0786781947318401,-0.0212388081736461,-0.120943599928478,-0.219439963211459,-0.315743754919242,-0.408892739398882,-0.497956202788417,-0.582044252402127,-0.660316708244079,-0.731991497808945,-0.796352470291924,-0.852756552130874,-0.900640172384771,-0.939524893748255,-0.969022192939049,-0.988837342694146,-0.99877235658721,-0.998727967243501,-0.98870461818667,-0.968802459407211,-0.93922034669687,-0.900253854747304,-0.852292323865461,-0.795814969813945,-0.731386095645498,-0.659649453373461,-0.581321811814434,-0.497185794871201,-0.408082061813394],"name":"right circularly polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(255,127,14,1)","line":{"color":"rgba(255,127,14,1)"}},"error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"line":{"color":"rgba(255,127,14,1)"},"frame":null},{"x":[1,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,2,2.1,2.2,2.3,2.4,2.5,2.6,2.7,2.8,2.9,3,3.1,3.2,3.3,3.4,3.5,3.6,3.7,3.8,3.9,4,4.1,4.2,4.3,4.4,4.5,4.6,4.7,4.8,4.9,5,5.1,5.2,5.3,5.4,5.5,5.6,5.7,5.8,5.9,6,6.1,6.2,6.3,6.4,6.5,6.6,6.7,6.8,6.9,7,7.1,7.2,7.3,7.4,7.5,7.6,7.7,7.8,7.9,8,8.1,8.2,8.3,8.4,8.5,8.6,8.7,8.8,8.9,9,9.1,9.2,9.3,9.4,9.5,9.6,9.7,9.8,9.9,10,10.1,10.2,10.3,10.4,10.5,10.6,10.7,10.8,10.9,11,11.1,11.2,11.3,11.4,11.5,11.6,11.7,11.8,11.9,12,12.1,12.2,12.3,12.4,12.5,12.6,12.7,12.8,12.9,13,13.1,13.2,13.3,13.4,13.5,13.6,13.7,13.8,13.9,14,14.1,14.2,14.3,14.4,14.5,14.6,14.7,14.8,14.9,15,15.1,15.2,15.3,15.4,15.5,15.6,15.7,15.8,15.9,16,16.1,16.2,16.3,16.4,16.5,16.6,16.7,16.8,16.9,17,17.1,17.2,17.3,17.4,17.5,17.6,17.7,17.8,17.9,18,18.1,18.2,18.3,18.4,18.5,18.6,18.7,18.8,18.9,19,19.1,19.2,19.3,19.4,19.5,19.6,19.7,19.8,19.9,20],"y":[1.34635357569263,1.4259317760983,1.49126253754756,1.54169309666751,1.57671956798154,1.59599197856649,1.59931776486641,1.58666369672395,1.55815620940511,1.51408014029986,1.45487588292109,1.3811349866382,1.29359424611134,1.19312833948275,1.08074108888184,0.95755543056633,0.824802194914343,0.683807808374128,0.535981040249448,0.382798926742371,0.225792012895788,0.0665290598932648,-0.0933986294841281,-0.252393110629198,-0.408865763242931,-0.561253164303392,-0.708032709271764,-0.847737825453589,-0.978972625508351,-1.10042585469436,-1.21088399249269,-1.30924337770306,-1.39452123586174,-1.46586549879913,-1.52256331822323,-1.56404818826416,-1.58990560581354,-1.59987721210256,-1.59386337413734,-1.57192418019893,-1.53427883946102,-1.48130349172437,-1.41352744915224,-1.33162790755824,-1.23642318008958,-1.12886452091263,-1.01002662059571,-0.88109686815622,-0.74336348706201,-0.598202663728378,-0.447064797118281,-0.291460006835352,-0.132943044507994,0.026902240774961,0.18647872776079,0.344191980940505,0.498466181621406,0.647759872986557,0.790581361821774,0.92550362302112,1.05117855795006,1.1663504642014,1.26986858215865,1.3606985930057,1.4379329532986,1.50079996283958,1.54867147525038,1.5810691742032,1.59766935259937,1.59830614694364,1.58297319459741,1.55182369735214,1.50516889068764,1.44347493401007,1.36735825294125,1.27757938019758,1.17503535659858,1.06075076813149,0.935867508626819,0.801633370332615,0.659389576386811,0.510557379758963,0.356623862560393,0.199127077611299,0.0396406807253724,-0.120241792738895,-0.278922849956767,-0.434817002257511,-0.586366606803085,-0.732057430040514,-0.870433777422992,-1.00011303822861,-1.11979950014967,-1.22829729562173,-1.32452235053705,-1.40751321595467,-1.47644067458049,-1.5306160260323,-1.56949796810639,-1.5926980052902,-1.59998433048113,-1.59128414112638,-1.56668436664211,-1.52643079984334,-1.47092564106348,-1.40072347950149,-1.31652575194993,-1.21917373427045,-1.1096401356434,-0.989019379579253,-0.858516668800696,-0.71943594325536,-0.573166851578923,-0.421170866185282,-0.264966680717295,-0.106115035761921,0.0537968755538215,0.21317126627191,0.370415720162462,0.523959102620309,0.672267258922626,0.8138583429958,0.947317623531559,1.07131161951456,1.18460142392392,1.28605508248259,1.37465890377039,1.44952758769354,1.50991307111057,1.55521200223196,1.58497176911179,1.59889502199667,1.59684264434618,1.57883514283938,1.54505244247884,1.49583208883949,1.43166587542481,1.35319492982869,1.26120330780051,1.15661015921916,1.04046054425139,0.913914991455979,0.778237902166077,0.634784917008979,0.484989370793124,0.330347971100475,0.172405843679108,0.0127410940574969,-0.147050960364291,-0.305373730198703,-0.460645306664104,-0.611314267494415,-0.755875178237551,-0.892883634058847,-1.02097069175672,-1.1388565477906,-1.24536332565488,-1.3394268448316,-1.42010725373041,-1.48659842037499,-1.53823598700729,-1.57450400813063,-1.59504010566655,-1.59963908971682,-1.58825500875301,-1.56100160874905,-1.518151196669,-1.4601319196659,-1.38752348717693,-1.30105137865838,-1.20157959483468,-1.0901020248888,-0.967732515850055,-0.83569374340277,-0.69530499531503,-0.54796898955138,-0.395157858778593,-0.238398441302718,-0.0792570254053879,0.0806763004909036,0.239803535460724,0.396534732772736,0.549303886111833,0.696584576596629,0.836905224252319,0.968863791551361,1.09114179210902,1.20251746456344,1.30187798001137,1.38823056102667,1.4607124011642],"z":[0.108060461173628,0.0907192242851156,0.0724715508953346,0.0534997657249178,0.0339934285800483,0.0141474403335407,-0.00583990446025767,-0.0257688988591049,-0.0454404189386172,-0.0646579133727006,-0.0832293673094284,-0.100969220919971,-0.117700223451069,-0.133255204255965,-0.147478743108249,-0.160228723109387,-0.171377750673789,-0.180814428403412,-0.188444468133732,-0.194191633029918,-0.197998499320089,-0.199827030054656,-0.199658955158951,-0.197495953981773,-0.193359638515892,-0.187291337458159,-0.17935168326683,-0.169620006342082,-0.158193542382883,-0.145186460840028,-0.130728724172722,-0.114964789306654,-0.09805216426814,-0.080159834415995,-0.0614665739956839,-0.0421591598861561,-0.0224305053870111,-0.00247773269257821,0.0174997966878894,0.037302473884515,0.0567324370926452,0.0755955485425961,0.0937033342600754,0.110874867235832,0.126938575188527,0.141733954858252,0.15511317570205,0.166942556967832,0.177103903388264,0.185495686148807,0.192034057330073,0.196653687688517,0.199308419404643,0.199971727276683,0.198636983751639,0.195317525145605,0.190046518391706,0.182876629647064,0.173879498069965,0.163145020025072,0.150780450868661,0.136909333288562,0.12167026290645,0.105215503476221,0.0877094655148782,0.0693270635670053,0.0502519685164519,0.0306747724075722,0.0107910841125299,-0.00920042512790729,-0.0291000067617226,-0.0487088307471585,-0.0678309721967668,-0.0862753689941241,-0.103857730823337,-0.120402380536965,-0.135744009464003,-0.14972932911948,-0.162218602812331,-0.173087041848222,-0.182226052376935,-0.189544320426222,-0.194968724280833,-0.198445065090521,-0.199938608407041,-0.199434431239276,-0.196937571158825,-0.192472975966262,-0.186085254420951,-0.177838230525072,-0.167814305815291,-0.156113636033837,-0.14285313040544,-0.128165283518999,-0.112196851485446,-0.0951073855991986,-0.0770676381543657,-0.0582578563442689,-0.0388659812910671,-0.0190857702001902,0.000885139597610059,0.0208472053731396,0.0406009727637503,0.0599490686554029,0.0786981732695781,0.0966609517506011,0.113657925953595,0.129519267730775,0.144086495798168,0.157214059228208,0.168770791746498,0.178641222301865,0.186726728814928,0.192946523577322,0.197238460455773,0.199559655835716,0.199886917100201,0.198216974362851,0.194566513139487,0.188972007631972,0.181489356290039,0.172193323292461,0.16117679152809,0.148549834540734,0.134438616710694,0.118984132661979,0.10234079849063,0.0846749088901329,0.0661629755898095,0.0469899637079647,0.0273474436415668,0.00743167695816502,-0.0125583445848166,-0.0324228872999436,-0.0519634712427511,-0.0709848533577424,-0.0892969782824548,-0.106716877317824,-0.123070496590943,-0.138194436143825,-0.151937582571765,-0.164162618898535,-0.174747396602216,-0.183586156082858,-0.190590583377436,-0.195690692563777,-0.198835525036763,-0.199993658669868,-0.199153521774658,-0.196323508721277,-0.191531896064676,-0.184826560014625,-0.176274498072446,-0.165961159614131,-0.153989592108413,-0.140479411500542,-0.125565607049276,-0.109397192558845,-0.0921357174822741,-0.0739536527726332,-0.0550326676103179,-0.0355618142246217,-0.0157356389463697,0.0042477616347275,0.0241887199856939,0.0438879926422933,0.0631487509838513,0.0817785478797733,0.0995912405576805,0.116408850480423,0.132063341648818,0.146398299561791,0.159270494058383,0.170551310426173,0.180128034476953,0.187904978749652,0.193804438587811,0.197767468538829,0.199754471317442,0.199745593448701,0.197740923637333,0.193760491881441,0.187844069339375,0.180050770949462,0.170458464773094,0.159162993962787,0.146277219129098,0.131929890674695,0.11626436236289,0.099437158974243,0.081616412362676],"name":"elliptical polarised","type":"scatter3d","mode":"lines","marker":{"color":"rgba(44,160,44,1)","line":{"color":"rgba(44,160,44,1)"}},"error_y":{"color":"rgba(44,160,44,1)"},"error_x":{"color":"rgba(44,160,44,1)"},"line":{"color":"rgba(44,160,44,1)"},"frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 1.5: Differential absorption of circularly polarised light. Here 10% of the L component and 30% of the R component is absorbed (90% and 70% transmitted, respectively).
</p>
</div>
<p>A CD signal will be observed when a chromophore is chiral (either intrinsically chiral, bonded to a chiral atom, or due to asymmetric enviornment). In proteins, such chromophores include the peptide bond (absorption below 240 nm), aromatic amino acid side chains (absorption in the range 260~320 nm) and disulphide bonds (weak absorptoin around 260 nm).</p>
</div>
<div id="experimental-setup" class="section level2">
<h2><span class="header-section-number">1.2</span> Experimental Setup</h2>
<p>The EM wave used in CD is UV, usually with <span class="math inline">\(\lambda\)</span> in the range 170~320 nm. Traditionally Xe arc lamps have been used as the light source of UV, but they can hardly achieve a wavelength below 180 nm. Now, high frequency UV can be generated by modern synchrotrons, and extending of CD data into the far UV region improves reliability of secondary structure prediction (see Section <a href="#ss">1.3</a>).</p>
<p>There are various methods by which the CD effect can be measured in a spectropolarimeter:</p>
<ul>
<li>modulation, in which the incident readiation is continuously switched between the L and R ocmponents</li>
<li>direct subtraction, in whiich the absorbances of the 2 components are measured separately and subtracted from each other</li>
<li>ellipsometric, in which the ellipticity of the transmitted radiation is measured</li>
</ul>
<p>The modulation method is most commonly used. The experimental setup is described as follows:</p>
<ul>
<li>plane polarised light is split into the L and R components by passage through a modulator subjected to an alternating electric field</li>
<li>the modulator normally consists of a piezoelectric quartz crystal and a thin plate of isotropic material (e.g. fused silica) tightly coupled to the crystal.</li>
<li>the alternating electric field induces structural changes in the quartz crystal; these make the plate transmit circularly polarised light at the extremes of the field</li>
<li>as the transmitted radiation of switched between L and R components, these are detecred in turn by the photomultiplier.</li>
</ul>
</div>
<div id="ss" class="section level2">
<h2><span class="header-section-number">1.3</span> Application of CD</h2>
<p>Far UV and near UV regions of CD give different information about a protein.</p>
<p>Absorption in the far UV region (170-250 nm) is mainly due to the <span class="math inline">\(\pi\rightarrow\pi^*\)</span> and <span class="math inline">\(n\rightarrow\pi^*\)</span> transitions in the peptide bonds, which is dependent on <span class="math inline">\(\Psi\)</span> and <span class="math inline">\(\Phi\)</span> torsional angles. Thus different types of secondary structures, such as <span class="math inline">\(\alpha\)</span>-helices, <span class="math inline">\(\beta\)</span>-sheets, <span class="math inline">\(\beta\)</span>-turns, each have their characteristic CD spectrum (Fig. <a href="#fig:cd">1.6</a>). These standard curves can be linearly combined to estimate the proportion of each secondary structure in a protein of interest.</p>
<div class="figure"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="Experimental setup of CD; characteristic curves of some secondary structures" width="100%" />
<p class="caption">
Figure 1.6: Experimental setup of CD; characteristic curves of some secondary structures
</p>
</div>
<p>The spectra in the near-UV region (260-320 nm) arise from the aromatic amino acids, each with a characteristic CD profile. The actual shape of the near UV CD spectrum will depend on the number of each type of aromic amino acid present, their mobility, their residing environment. Thus it can serve as a fingerprint of the of the tertiary structure of a protein.</p>
<div class="figure"><span id="fig:cd-aromatic"></span>
<img src="../img/cd-aromatic.png" alt="The near UV CD spectrum of a dehydroquinase, labelled with contribution from each aromatic amino acid side chain." width="100%" />
<p class="caption">
Figure 1.7: The near UV CD spectrum of a dehydroquinase, labelled with contribution from each aromatic amino acid side chain.
</p>
</div>
<p>By combining the far- and near-UV CD spectra, we can obtain a summary of the overall structural features of the protein of interest. Although it gives little insight into the precise 3D structure of the protein, it serves at a rapid way to detect conformational differences between two similar proteins <em>in solution</em>. Specifically, it can be used to:</p>
<ul>
<li>monitor the progress of protein folding (especially the detectiion of molten globule-like structures)</li>
<li>compare the wild type and mutant forms of a protein</li>
<li>confirm a modification (tagging) will not affect the protein’s native conformation and normal function</li>
<li>assess thermal stability (unfolding at high temperature)</li>
<li>show the formation of amyloid <span class="math inline">\(\beta\)</span> protein in Alzheimer disease <span class="citation">(Barrow et al. 1992)</span></li>
</ul>
<p>CD can not only be applied to proteins but also to other chiral molecules. Such applications include:</p>
<ul>
<li>determination of nucleic acid conformations (A-RNA, A-DNA, B-DNA, Z-DNA)</li>
<li>determination of nucleic acid-ligand interactions, e.g. between cationic porphyrins and DNA <span class="citation">(Pasternack 2003)</span></li>
<li>conformational study of biomolecular interaction with nanoparticles, where the degree of protein or nucleic acid denaturation is estimated <span class="citation">(Liu and Webster 2007)</span></li>
</ul>
</div>
<div id="other-cd-based-techniques" class="section level2">
<h2><span class="header-section-number">1.4</span> Other CD-based Techniques</h2>
<p>The experiment described above is the conventional electronic circular dichorism (ECD). During the past decades, many other CD-based techniques have been developed to solve more specific questions. These include magnetic CDs (MCD, magnetic vibrational circular dichroism (MVCD), XMCD), fluorescence detected CD (FDCD), near-infrared CD (NIR-CD), vibrational CDs (VCD, FTIR-VCD), HPLC-CD, stopped-flow CD, and synchrotron radiation CD (SRCD).</p>
</div>
</div>
<div id="small-angle-x-ray-scattering-saxs" class="section level1">
<h1><span class="header-section-number">2</span> Small-angle X-ray Scattering (SAXS)</h1>
<p>Reference: <span class="citation">Kachala, Valentini, and Svergun (2015)</span>, <span class="citation">Kikhney and Svergun (2015)</span></p>
<p>Small-angle X-ray Scattering (SAXS) detects the X-ray scattering pattern of macromolecules in the solution state. Information about particle shape and size can then be obtained from the angular dependence of scattering.</p>
<p>SAXS provides low resolution information on the structure, conformation and assembly state of proteins, nucleic acids and various macromolecular complexes. Importantly, it offers powerful means for the quantitative analysis of flexible systems, including intrinsically disordered proteins (IDPs) and multi-domain proteins with ﬂexible linkers.</p>
<div id="physical-principles-1" class="section level2">
<h2><span class="header-section-number">2.1</span> Physical Principles</h2>
<p>The the X-ray used in SAXS has wavelength of about the same size as the macromolecules in the sample, which allows formation of interference pattern (Fig. <a href="#fig:saxs-principle-1">2.1</a>).</p>
<div class="figure"><span id="fig:saxs-principle-1"></span>
<img src="../img/saxs-principle-1.jpg" alt="(A) for an array (usually a crystal) of small scatterers whose size is small compared to the wavelength of the incident light, an interference pattern can be observed due to the fixed path differences (and hence phase differences) between any two scatterers; (B) in the solution state of such small particles, their random orientation and movement make scattering to occur in all directions (so that a pattern cannot be observed); (C) when the size of the scatterers is comparable to or greater than the wavelength, angle-dependent scatterings due the nuclei/electrons of pairs of atoms within individual particles can thus be observed (larger angle, larger path difference)." width="100%" />
<p class="caption">
Figure 2.1: (A) for an array (usually a crystal) of small scatterers whose size is small compared to the wavelength of the incident light, an interference pattern can be observed due to the fixed path differences (and hence phase differences) between any two scatterers; (B) in the solution state of such small particles, their random orientation and movement make scattering to occur in all directions (so that a pattern cannot be observed); (C) when the size of the scatterers is comparable to or greater than the wavelength, angle-dependent scatterings due the nuclei/electrons of pairs of atoms within individual particles can thus be observed (larger angle, larger path difference).
</p>
</div>
</div>
<div id="experimentation-and-data-processing" class="section level2">
<h2><span class="header-section-number">2.2</span> Experimentation and Data Processing</h2>
<p>SAXS uses a collimated monochromatic X-ray beam to illuminate the sample, and the intensity of the scattered X-rays a is recorded. The scattering of the pure solvent is also collected and subtracted from the sample solution scattering. The resulting 2D scattering pattern is translated into a 1D <em>I</em> vs. <em>q</em> relationship (where <em>I</em> is intensity and <span class="math inline">\(q = 4\pi\sin(\theta/2)/\lambda\)</span>), and the data is transformed and plotted in a variety of ways.</p>
<p>Several parameters can be calculated from SAXS data, including molecular weight, excluded particle volume, maximum dimension <span class="math inline">\(D_\text{max}\)</span> and the radius of gyration <span class="math inline">\(R_g\)</span>.</p>
<p><span class="math inline">\(R_g\)</span> can be directly extracted from SAXS data using the Guinier approximation, which states that, when the incident angle is small (approaching 0), the angular dependence of scattering can be described by the equation <span class="math inline">\(I(q)=I_z\exp(-q^2R_g^2/3)\)</span>, where <span class="math inline">\(R_g\)</span> is the radius of gyration of the particle. When <span class="math inline">\(\ln(I(q))\)</span> is plotted against <span class="math inline">\(q^2\)</span> (<span class="math inline">\(\ln(I(q)) = \ln{I_0}-(R_g^2/3)q^2\)</span>), <span class="math inline">\(R_g^2/3\)</span> is the slope of the resulting straight line.</p>
<p><span class="math inline">\(R_g\)</span> provides a measure of the overall size of the macromolecule. It is the average root-mean-square distance to the centre of density in the molecule weighted by the scattering length density.</p>
<p>Some plots can emphasised sample flexibility, e.g. a Kranky plot (<span class="math inline">\(IQ^2\)</span> agianst <span class="math inline">\(Q\)</span>) can help identify an unfolded protein, as shown in Fig. <a href="#fig:Kratky">2.2</a></p>
<div class="figure"><span id="fig:Kratky"></span>
<img src="../img/Kratky.png" alt="The Kratky plot" width="100%" />
<p class="caption">
Figure 2.2: The Kratky plot
</p>
</div>
<p><span class="math inline">\(I(q)\)</span> can be considered as a reciprocal space as a Fourier transform of <span class="math inline">\(p(r)\)</span>, which is the distribution of distances between pairs of atoms in real space. These two are related to each other by the equation:</p>
<p><span class="math display">\[p(r)=\dfrac{r^2}{2\pi^2}\int_0^\infty\dfrac{q^2I(q)\sin{(qr)}}{qr}ds\]</span></p>
<p><span class="math inline">\(p(r)\)</span> can be obtained from experimental data by indirect Fourier transformation. One such distribution is shown in Fig. <a href="#fig:saxs-pr">2.3</a>.</p>
<div class="figure"><span id="fig:saxs-pr"></span>
<img src="../img/saxs-pr.png" alt="Distance distribution function *p(r)* vs. r. Globular compact particles have a more symmetric distribution while unfolded particles have an skewed distribution with an extended tail." width="100%" />
<p class="caption">
Figure 2.3: Distance distribution function <em>p(r)</em> vs. r. Globular compact particles have a more symmetric distribution while unfolded particles have an skewed distribution with an extended tail.
</p>
</div>
<p>To produce meaningful results, SAXS requires that the samples to be monodisperse (non-aggregated), which can be verified by dynamic light scattering (DLS) or analytical centrifugation (AUC).</p>
</div>
</div>
<div id="analytical-ultracentrifuge-auc" class="section level1">
<h1><span class="header-section-number">3</span> Analytical Ultracentrifuge (AUC)</h1>
<p>Reference: <span class="citation">Harding et al. (2015)</span>; <span class="citation">Uchiyama, Noda, and Krayukhina (2018)</span>; <span class="citation">Unzai (2018)</span>; <span class="citation">Harding and Rowe (2010)</span></p>
<p>The analytical centrifuge (AUC) is a high speed (up to 60000 rpm) ultracentrifuge equipped with absorbance and interference detection systems, which allow the analysis of the redistribution of macromolecular solute under the influence of a centrifugal field. AUC is a convenient, matrix-free solution technique without requirement for immobilisation, columns, or membranes.</p>
<p>Typical AUC experiments can be classified into two types:</p>
<ul>
<li><strong>sedimentation velocity experiment:</strong> record the <em>change in concentration distribution</em> over time (performed at high speed)</li>
<li><strong>sedimentation equilibrium experiment:</strong> record the <em>steady state distribution</em> of the macromolecular solute following equiilibration of centrifugal and diffusive forces (performed at lower speed)</li>
</ul>
<p>These two methods can give information about a wide range of parameters of protein-protein interactions, including stoichiometry, reversibility, strength and, in some cases, dynamics.</p>
<p>Apart from protein-protein interactions, AUC is now applied to study interactions of a wider range of macromolecules, such as protein-like carbohydrate assocaition, carbohydrate-protein association (polysaccharide-gliadin), and nucleic acid protein (G-duplexes) interactions.</p>
<div id="choice-of-optical-system" class="section level2">
<h2><span class="header-section-number">3.1</span> Choice of Optical System</h2>
<p>Depending on the strengths of the interaction probed (and the concentration of the sample), different optical systems are used:</p>
<ul>
<li>for weak interactions (<span class="math inline">\(10^{-4}&lt;K_\text{d}&lt;10^{-1}\)</span>), higher concentrations (&gt;5 mg/ml) are required and the <em>Rayleigh interference</em> optical system is the most appropriate</li>
<li>for moderate-strength interactions (<span class="math inline">\(10^{-7}&lt;K_\text{d}&lt;10^{-4}\)</span>, 0.1~0.5 mg/ml), either <em>interference</em> optics or <em>UV absorption</em> optics can be used</li>
<li>for strong interactions (<span class="math inline">\(K_\text{d}&lt;10^{-7}\)</span>), <em>dye-labelled</em> proteins and <em>fluorescence optics</em> are necessary</li>
</ul>
</div>
<div id="experiments" class="section level2">
<h2><span class="header-section-number">3.2</span> Experiments</h2>
<div id="sedimentation-velocity-sv" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Sedimentation Velocity (SV)</h3>
<p>In the SV experiment, measurements on the absorption (<span class="math inline">\(A\)</span>, which is proportional to local concentration) at different radial distances (<span class="math inline">\(r\)</span>) from the rotation centre are made at fixed time intervals (<span class="math inline">\(t\)</span>), producing a series of s-shaped curves that shift to higher <span class="math inline">\(r\)</span> values as sedimentation proceeds (Fig. <a href="#fig:auc-sv-data">3.1</a>.</p>
<p>The spinning rotor generates a sedimentation force on a particle of <span class="math inline">\(m\omega^2r\)</span> (<span class="math inline">\(m\)</span> = particle mass; <span class="math inline">\(\omega\)</span> = angular velocity; <span class="math inline">\(r\)</span> = distance from the centre of rotation). In solution, the particle displaces solvent, so the sedimentation force acts on an effective mass, <span class="math inline">\(m_\text{eff}=m(1-\bar v\rho)\)</span> that is less than <span class="math inline">\(m\)</span> (<span class="math inline">\(\rho\)</span> = solvent density; <span class="math inline">\(\bar v\)</span> = partial-specific volume (in ml/g)). <span class="math inline">\(\bar v\)</span> is usually calculated (for proteins, <span class="math inline">\(\bar v\)</span> lies in the range 0.70-0.75, leading to <span class="math inline">\((1-\bar v\rho)\)</span> of around 0.27 (in water)). At terminal velocity (acceleration = 0), the sedimentation force is balanced by the frictional force and the velocity can be described by the equation <span class="math inline">\(v=m(1-\bar{v}\rho)\omega^2r/f\)</span>, where <span class="math inline">\(f\)</span> is the friction coefficient, which is related to the diffusion coefficient, <span class="math inline">\(D\)</span>, by <span class="math inline">\(f=RT/N_\text{A}D\)</span>. The equation can thus be arranged to <span class="math inline">\(v=DM(1-\bar{v}\rho)\omega^2r/RT\)</span>. By defining the <em>sedimentation coefficient</em> as <span class="math inline">\(s=v/\omega^2r\)</span>, the <em>Svedberg equation</em> can be written as <span class="math inline">\(s=\dfrac{DM(1-\bar{v}\rho)}{RT}\)</span>.</p>
<p>The sedimentation coefficient <span class="math inline">\(s\)</span>, which is experimentally determined in SV has the unit Svedberg (S) where 1 S = 10<sup>-13</sup> seconds. The greater the molecular weight or more compact/spherical (less friction) the macromolecule is, the larger its <span class="math inline">\(s\)</span> value. Usually the experimentally measured <span class="math inline">\(s_\text{exp}\)</span> is standarised to <span class="math inline">\(s_{20, \text{w}}\)</span>, which is the value that would have been observed in water at 20<span class="math inline">\(^\circ\)</span>C, using the relationship <span class="math inline">\(s_{20, \text{w}}=\dfrac{s_\text{exp}(\eta_\text{exp}(1-\bar{v}\rho_\text{20,w})}{(\eta_\text{20,w}(1-\bar{v}\rho_\text{exp})}\)</span>, where <span class="math inline">\(\eta\)</span> and <span class="math inline">\(\rho\)</span> refer to the viscosity and density of the buffer.</p>
<p>The Lamm equation, derived from Svedberg equation and Fick’s diffusion laws, describes the time dependence of the concentration:</p>
<p><span class="math display">\[\frac{\partial c}{\partial t}=D\left[\left(\frac{\partial^2 c}{\partial r^2}\right)+\frac{1}{r}\left(\frac{\partial c}{\partial r}\right)\right]-s\omega^2\left[r\left(\frac{\partial c}{\partial r}\right)+2c\right]\]</span></p>
<p>Solving this diffential equation leads to <span class="math inline">\(g(s)\)</span>, which is the distribution of <span class="math inline">\(s\)</span>. SEDFIT is one of the softwares specialised in solving this. SEDFIT can either give the uncorrected <span class="math inline">\(g(s)\)</span> versus <span class="math inline">\(s\)</span> profile, or it can give a distribution, known as <span class="math inline">\(c(s)\)</span> vs. <span class="math inline">\(s\)</span>, which has been corrected for diffusion broadening (this assumes all particles have the same frictional ratio <span class="math inline">\(f/f_o\)</span>). The <span class="math inline">\(g(s)\)</span> vs. <span class="math inline">\(s\)</span> or <span class="math inline">\(c(s)\)</span> vs. <span class="math inline">\(s\)</span> can further be converted to a molecular weight distribution, <span class="math inline">\(c(M)\)</span> vs. <span class="math inline">\(M\)</span>, which is analogous to a mass spectrum (Fig. <a href="#fig:auc-sv-data">3.1</a>). SEDFIT is particularly good at evaluating homogeneity/heterogeneity of a prepation. Where the solution is heterogeneous, it can estimate the proportion of each sedimenting species and ascertain whether there is a reversible equilibrium.</p>
<div class="figure"><span id="fig:auc-sv-data"></span>
<img src="../img/auc-sv-data.jpg" alt="Schematic of data processing in SV experiments. The raw data collected is a series of curves recorded at fixed intervals, each showing the variation of the absorbance with the distance from the rotation centre. By solving the Lamm equation, this is transformed to a *g(s)* vs. *s* plot, or *c(s)* vs. *s* plot if it is denoised by SEDFIT. It can be further transformed into a *c(M)* vs. M plot (distribution of molecular weights), assuming constant frictional ratio $f/f_o$" width="100%" />
<p class="caption">
Figure 3.1: Schematic of data processing in SV experiments. The raw data collected is a series of curves recorded at fixed intervals, each showing the variation of the absorbance with the distance from the rotation centre. By solving the Lamm equation, this is transformed to a <em>g(s)</em> vs. <em>s</em> plot, or <em>c(s)</em> vs. <em>s</em> plot if it is denoised by SEDFIT. It can be further transformed into a <em>c(M)</em> vs. M plot (distribution of molecular weights), assuming constant frictional ratio <span class="math inline">\(f/f_o\)</span>
</p>
</div>
<p>SV also gives information about the shape. The friction coefficient, <span class="math inline">\(f\)</span>, can be easily calculated from the terminal velocity (<span class="math inline">\(v=m(1-\bar{v}\rho)\omega^2r/f\)</span>), and the friction ratio, <span class="math inline">\(f/f_o\)</span> (where <span class="math inline">\(f_o=6\pi r\eta\)</span>), shows diviation of the molecular shape from the sphere.</p>
<p>SV can also be applied for interaction analysis, and the simpest case is co-sedimentation. For example, the binding of adenosylcobalamin cofactor to the methylmalonyl-CoA mutase system from <em>Propionibacterium shermanii</em> was demonstrated by AUC with UV-absoption system. At a wavelength selected to detect the ligand only, in the presence of the mutase, all ligands sediment at the same rate as the protein, confirming the ligand is 100% bound.</p>
<p>In SV analysis, the emergence of new peaks at higher concentrations, shifts in the ratios of the peak areas, and/or shifts in peak positions are indicative of protein-protein interactions.</p>
<p>Fig. <a href="#fig:sv-trap">3.2</a> shows an SV experiment which studies the binding between the <em>Bacillus stearothermophilus</em> 11-mer protein TRAP (<em>trp</em> RNA-binding attentuation protein) and Anti-TRAP (AT). As the TRAP:AT ratio increases froom 1:0 to 1:6, the peak representing TRAP shifts to higher <em>s</em>, indicating rapid and reversible binding between TRAP and AT (slow/irreversible interaction would result in separate peaks). The plot also shows that TRAP was saturated with AT at a 1:6 stoichiometry (as increasing ratio to 1:10 did not result in further shifts) and that the TRAP-AT complex was stable to excess AT (no negative feedback loop).</p>
<div class="figure"><span id="fig:sv-trap"></span>
<img src="../img/sv-trap.png" alt="SV analysis of TRAP and AT. The TRAP concentratioin was fixed at 0.5 mg/ml, and varying amounts of AT (TRAP's ligand) were added, with molar ratios of TRAP:AT labelled." width="100%" />
<p class="caption">
Figure 3.2: SV analysis of TRAP and AT. The TRAP concentratioin was fixed at 0.5 mg/ml, and varying amounts of AT (TRAP’s ligand) were added, with molar ratios of TRAP:AT labelled.
</p>
</div>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Barrow-1992">
<p>Barrow, Colin J., Akikazu Yasuda, Peter T.M. Kenny, and Michael G. Zagorski. 1992. “Solution Conformations and Aggregational Properties of Synthetic Amyloid β-Peptides of Alzheimer’s Disease: Analysis of Circular Dichroism Spectra.” <em>Journal of Molecular Biology</em> 225 (4): 1075–93. <a href="https://doi.org/https://doi.org/10.1016/0022-2836(92)90106-T">https://doi.org/https://doi.org/10.1016/0022-2836(92)90106-T</a>.</p>
</div>
<div id="ref-Harding-2015">
<p>Harding, Stephen E, Richard B Gillis, Fahad Almutairi, Tayyibe Erten, M Şamil Kök, and Gary G Adams. 2015. “Recent Advances in the Analysis of Macromolecular Interactions Using the Matrix-Free Method of Sedimentation in the Analytical Ultracentrifuge.” <em>Biology</em> 4 (1). MDPI: 237–50. <a href="https://doi.org/10.3390/biology4010237">https://doi.org/10.3390/biology4010237</a>.</p>
</div>
<div id="ref-Harding-2010">
<p>Harding, Stephen E., and Arthur J. Rowe. 2010. “Insight into protein–protein interactions from analytical ultracentrifugation.” <em>Biochemical Society Transactions</em> 38 (4): 901–7. <a href="https://doi.org/10.1042/BST0380901">https://doi.org/10.1042/BST0380901</a>.</p>
</div>
<div id="ref-Kachala-2015">
<p>Kachala, Michael, Erica Valentini, and Dmitri I Svergun. 2015. “Application of Saxs for the Structural Characterization of Idps.” <em>Advances in Experimental Medicine and Biology</em> 870. United States: 261–89. <a href="https://doi.org/10.1007/978-3-319-20164-1{\_}8">https://doi.org/10.1007/978-3-319-20164-1{\_}8</a>.</p>
</div>
<div id="ref-Kelly-2005">
<p>Kelly, Sharon M, Thomas J Jess, and Nicholas C Price. 2005. “How to Study Proteins by Circular Dichroism.” <em>Biochimica et Biophysica Acta</em> 1751 (2). Netherlands: 119–39. <a href="https://doi.org/10.1016/j.bbapap.2005.06.005">https://doi.org/10.1016/j.bbapap.2005.06.005</a>.</p>
</div>
<div id="ref-Kikhney-2015">
<p>Kikhney, Alexey G., and Dmitri I. Svergun. 2015. “A Practical Guide to Small Angle X-Ray Scattering (Saxs) of Flexible and Intrinsically Disordered Proteins.” <em>FEBS Letters</em> 589 (19, Part A): 2570–7. <a href="https://doi.org/https://doi.org/10.1016/j.febslet.2015.08.027">https://doi.org/https://doi.org/10.1016/j.febslet.2015.08.027</a>.</p>
</div>
<div id="ref-Liu-2007">
<p>Liu, Huinan, and Thomas Jay Webster. 2007. “Nanomedicine for Implants: A Review of Studies and Necessary Experimental Tools.” <em>Biomaterials</em> 28 (2): 354–69. <a href="https://doi.org/https://doi.org/10.1016/j.biomaterials.2006.08.049">https://doi.org/https://doi.org/10.1016/j.biomaterials.2006.08.049</a>.</p>
</div>
<div id="ref-Pasternack-2003">
<p>Pasternack, Robert F. 2003. “Circular Dichroism and the Interactions of Water Soluble Porphyrins with Dna—a Minireview.” <em>Chirality: The Pharmacological, Biological, and Chemical Consequences of Molecular Asymmetry</em> 15 (4). Wiley Online Library: 329–32.</p>
</div>
<div id="ref-Ranjbar-2009">
<p>Ranjbar, Bijan, and Pooria Gill. 2009. “Circular Dichroism Techniques: Biomolecular and Nanostructural Analyses—a Review.” <em>Chemical Biology &amp; Drug Design</em> 74 (2). England: 101–20. <a href="https://doi.org/10.1111/j.1747-0285.2009.00847.x">https://doi.org/10.1111/j.1747-0285.2009.00847.x</a>.</p>
</div>
<div id="ref-Sievert-2018a">
<p>Sievert, Carson. 2018. “Plotly for R.” 2018.</p>
</div>
<div id="ref-Uchiyama-2018">
<p>Uchiyama, Susumu, Masanori Noda, and Elena Krayukhina. 2018. “Sedimentation Velocity Analytical Ultracentrifugation for Characterization of Therapeutic Antibodies.” <em>Biophysical Reviews</em> 10 (2). Springer Berlin Heidelberg: 259–69. <a href="https://doi.org/10.1007/s12551-017-0374-3">https://doi.org/10.1007/s12551-017-0374-3</a>.</p>
</div>
<div id="ref-Unzai-2018">
<p>Unzai, Satoru. 2018. “Analytical Ultracentrifugation in Structural Biology.” <em>Biophysical Reviews</em> 10 (2). Springer Berlin Heidelberg: 229–33. <a href="https://doi.org/10.1007/s12551-017-0340-0">https://doi.org/10.1007/s12551-017-0340-0</a>.</p>
</div>
<div id="ref-Whitmore-2008">
<p>Whitmore, Lee, and B A Wallace. 2008. “Protein Secondary Structure Analyses from Circular Dichroism Spectroscopy: Methods and Reference Databases.” <em>Biopolymers</em> 89 (5). United States: 392–400. <a href="https://doi.org/10.1002/bip.20853">https://doi.org/10.1002/bip.20853</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>suprisingly, this graphing library originally not designed for math plotting worked pretty well, and is much easier to use (and more powerful in terms of interactivity) than other professional math plotting libraries. I even <a href="../bioinformatics/visualizing-protein-structures-using-plotly.html">used <strong>plotly</strong> to plot proteins</a> and it worked quite decently!)<a href="#fnref1" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/electron-microscopy-and-x-ray-crystallography.html" class="archive-item-link">
                Electron Microscopy and X-Ray Crystallography 
              </a>
              <span class="archive-item-date">
                2020-02-13
              </span>
              <div class="article-summary" style="display: none;">
                
                  Learning objectives

To understand the methodological principle of electron microscopy in its different applications (e.g. TEM, SEM, tomography, single-particle), and X-ray crystallography  To be able to evaluate, which method of structural analysis would be most feasible with which kind of biological specimen (e.g. whole cells, viruses, protein complexes, peptides).

To gain a conception of the resolution ranges attainable by EM methods as compared to X-ray crystallography.

To be aware of the computational methods that combine EM maps and crystallogrographic data
<hr>
                
                


<p><strong>Compare the strengths and limitations of Electron Microscopy and X-ray crystallography. Which types of biological specimen can be studied with the respective methods, and how can cryo-EM and X-ray crystallography be combined to obtain novel structural information?</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Being able to solve macromolecular strucutures at atomic resolution, X-ray crystallography has long been the most powerful tool in structural biology. As of Feb 16 2019, 167943 molecular structures have been deposited into the Protein Data Bank (PDB) since its launch in 1971. However, recent improvements in the field of electron microscopy, especially the introduction of the use of cryocooling, have revolutionised the way cellular material is viewed. In conjunction with computational methods, cryo-EM can now solve macromolecular structures at sub-atomic or even atomic resolution. The structures publicised on Electron Microscopy Data Bank (EMDB) has grown exponentially over the last two decades, from only 8 in 2002 to more than 10000 on Feb 16 2020. In this breif narrative, I introduce the basic principles of both methods, their strengths and limitations, and how they can complement each other in modern structural biology studies.</p>
</div>
<div id="x-ray-crystallography" class="section level1">
<h1>X-Ray Crystallography</h1>
<p>In short, structural determination using X-ray crystallography involves making the appropriately-sized crystals of the protein of interest, gaining electron density maps by deciphering X-ray diffraction patterns obtained from these crystals, and filling the map with amino acids (and prosthetic groups or other ligands).</p>
<p>Prior to the X-ray experiment, the protein must be crystallised. This is often the most laborious and time-consuming step, involving numerous trial-and-errors. In modern high throughput methods, a wide range of crystallisation constructs are designed (which involves truncation the protein of interest, expression with a fusion partner, binding with a ligand, use of different detergents, etc.), and robots are used to deploy these crystallisation conditions in multiwell plates and to monitor the growth of crystals. <em>One of the critial factors in the sucess of crystallisation is aqueous solubility, which represents the major bottleneck in solving membrane protein structures <span class="citation">(Parker and Newstead 2016)</span>.</em></p>
<p>In the X-ray experiment, the crytal is mounted on a support which can be rotated in any direction, and is cryocooled in order to minimise radioactive damage and random thermal vibration, thus improving signal-to-noise-ratio (contrast). For every orientation of the crystal, a monochromatic X-ray is applied, and the detector records the position and intensity of the reflections (points) in the diffraction pattern.</p>
<p>Crystallisation is needed for two reasons. First, the diffraction signal given by a single protein molecule is too weak to be detected. Second, incoherent scattering causes serious chemical damage to protein molecules, and if we try to image a single molecule with X-rays, it would be destroyed as soon as it scattered one or two photons and no longer represent the native protein structure. Crystallisation forms an array of a substantial number of protein molecules arranged in the same orientation (so that their diffraction properties are the same) so the radiation damage in a small number of molecules becomes negligible. Every molecule in the array gives the same diffraction signal, thus achieving amplification.</p>
<p>X-rays with wavelengths within the range 0.5-1.6Å are used in crystallography. X-rays above this range are ‘soft X-rays’ that penerate crystals without scattering. Generally, X-rays of shorter wavelengths are more preferable because of two reasons. First, this generally gives better resolution. According to this rearranged form of Bragg’s equation, <span class="math inline">\(\dfrac{n_{max}}{d}\propto\dfrac{1}{\lambda}\)</span>, as wavelength (<span class="math inline">\(\lambda\)</span> decreases), smaller <span class="math inline">\(d\)</span> (distance between crystal lattice planes) can be distinguished with the same <span class="math inline">\(n\)</span> (number of constructive interference). Second, X-rays with shorter <span class="math inline">\(\lambda\)</span> are scattered more, thus producing stronger signals for a fixed amount of sample, which allows crytals of smaller sizes to be studied. However, using X-rays of too short wavelengths (and using too small crystals) have the disadvantage that a greater proportion of proteins would be damaged due to the higher energy of the X-ray and the smaller sample size. Traditional X-rays sources are characteristic radiations from a Cu anode (CuK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=1.54Å) or a Mo anode (MoK<span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\lambda\)</span>=0.71Å) when bombarded by electrons beams at appropriate potential difference from the cathode. Since the 1960s, synchrotrons (e.g. Diamond Light Source in Oxfordshire) have become available as more intense and X-ray sources. In these giant devices, electrons travel on a circular track in vacuum, emitting intense X-rays in tangential directions. Electrical disturbances in synchrotron allow for production of X-rays with any <span class="math inline">\(\lambda\)</span> within the useful range 0.5-1.6Å, and this makes it possible to use multiple anomalous dispersion (MAD) method to solve the phase problem.</p>
<p>The diffraction patterns only give information on the position amplitude of every reflection, but an additional parameter, phase, is required to do the inverse Fourier transform that would give the electron density map. Single/multiple isomorphous replacements (SIR/MIR) using heavy metals represent the earliest attempts to solve the phase problem. Later, with the advent of synchrotrons, multi-wavelength anomalous dispersion (MAD) became the more popular phasing strategy. Recently, thank to the numerous experimental structures solved previously and publicised in PDB, molecular replacement (MR) has become the most efficient way to solve the phase problem.</p>
<p>After solving the first electron density map using inverse Fourier transform, the structure is refined by optimising the parameters of the model to fit the observations in a iterative process.</p>
<p>Today, abundant software packages are available for data collection and processing, structure solution, refinement and validation.</p>
</div>
<div id="cryo-electron-microscopy" class="section level1">
<h1>Cryo-Electron Microscopy</h1>
<p>Electron microscopy (EM) is mechanistically similar to light microscopy (LM). Electron beams (which is analogous to visible light beams in LM) hit the sample, and their path is regulated by a set of electromagnets (analogous to lens in LM) so that they finally converge onto a plane where an enlarged image of the sample can be detected.</p>
<p>Unstained samples have a very poor singal-to-noise ratio (contrast) under EM, and traditionally heavy metal are applied to improve contrast. However, this often leads to unwanted artefacts and can only achieve resolutions at 20-40Å. High electron dose improves contrast but causes damage to the specimen.</p>
<p>Cryo-EM partially solves this problem by reducing the effect of radiation damage using low temperature. In cryo-electron microscopy, protein solutions are applied onto a support grid, and is then plunge-frozen with liquid ethane. The process is so fast that the water adopts a vitreous form instead of crystallises into ice. The vitrified sample is then maintained at low temperature with liquid nitrogen during storage and EM studies.</p>
<p>Another challenge of cryo-EM is the movement of the particles when being hit by the electron beam, which leads to blurred images on conventional CCD/photographic films (because they are less sensitive and need a long time of exposure). This is solved by the more sensive and faster direct electron detectors which can record movies at a rate of many fps. The traces of molecules recorded in the movies can be computationally processed to give much sharper images. The motion is in part due to the different thermal expansion coefficients between the metal grid and the carbon film, and choosing metals that have similar thermal properties with carbon (e.g. titanium, molybdenum or tungsten instead of copper or gold) can minimise this effect <span class="citation">(Sgro and Costa 2018)</span>.</p>
<p>There are two major strategies of constructing 3D models using cryo-EM, namely electron tomography and single-particle cryo-EM. In electron tomography, the specimen is tilted in all directions and respective EM images are recorded, which are combined (similar to CT) into a 3D model. It is commonly used to visualise one-of-a-kind, structurally heterogeneous entities (such as viruses and whole bacterial cells) at resolutions 50-100Å. Single-particle cryo-EM are usually used to study smaller entities such as the ribosome and proteins. In this approach, a large number (tens or hunders of thousands) of 2D images are extracted from EM images. These heterogeneous low-resolution ‘snapshots’ are computationally sorted and aligned (sometimes called <em>in silico</em> purification), and finally used to synthesise the 3D model using Fourier transform.</p>
</div>
<div id="comparison-of-strengths-and-limitations" class="section level1">
<h1>Comparison of Strengths and Limitations</h1>
<p>The most significant advantage of X-ray crystallography is its resolution. 2.05Å is the median resolution for X-ray crystallographic results in the protein data bank (as of May 19, 2019, according to <a href="https://proteopedia.org/wiki/index.php/Resolution">Proteopedia</a>, but I will do the analysis myself on Monday. I have downloaded all the entire PDB repository.). The protein backbone and most sidechains can be identified unambiguously under this resolution. Strikingly<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>, however, this long unrivalled strength of X-ray crystallography is now challeged by cryo-EM—Wu el al.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> claimed cryo-EM solution of mouse heavy chain apoferritin at 1.75Å!</p>
<p>A major advantage of cryo-EM over X-ray crystallography is the ease and speed of sample preparation, as the proteins need not to be crystallised and only a small amount is needed. Another related merit is its forgiveness of heterogeneity, as robust computational methods can ‘purify’ proteins <em>in silico</em>. Furthermore, cryo-EM are suited for studying membrane proteins and multiprotein supra-assemblies/RNP machines, which are difficult to crystallise in their native states. However, it should be noted that during the process of specimen preparation, delicate protein complexes may become associated so that they no longer represent their <em>in vivo</em> state.</p>
</div>
<div id="combining-x-ray-and-cryo-em-studies" class="section level1">
<h1>Combining X-ray and Cryo-EM studies</h1>
<p>Today, it is common to combine the results of X-ray crystallography and cryo-EM studies for structural determination. There are two major ways in which these two methods can complement each other. First, a low-resolution cryo-EM map can provide an overall shape of the macromolecule, whose sub-components are solved at a high resolution by X-ray crystallograhy and docked onto the EM map. Second, the cryo-EM model may help to solve the phase problem in X-ray crystallography by serving as a search model in molecular replacement (MR).</p>
<p>The docking methods can be classified into two categories: rigid-body docking and flexible docking. Both are used to find the optimal position and orientation of sub-component X-ray structures in the cryo-EM map, but the the latter entails additional algorithms such as normal mode analysis and molecular dynamic simulation that introduce minor conformational changes within stereochemistry limits in X-ray structures to minimise local conformational discrepancy between the X-ray and cryo-EM models. The docking is useful to define the protein location and protein-protein interface within a complex and new interaction modes that are not revealed by X-ray crystallography.</p>
<p>The electron density map (strictly speaking, Coulomb potential density map) obtained by Cryo-EM, albeit at a low resolution, has information about the phase, and thus can be used as an initial phasing model for X-ray studies. Once the search EM map has been positioned, theoretical phases can be calculated by Fourier transform up to the EM model resolution.</p>
</div>
<div id="concluding-remarks" class="section level1">
<h1>Concluding Remarks</h1>
<p>The past decade has witnessed a resolution revolution in single particle cryo-EM, making them another powerful tool in solving biological macromolecules after X-ray crystallography. The different perspectives provided by these two methods is helping us to gain a more complete understanding of molecualr mechanisms that underlie the principle of life.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Cheng-2018">
<p>Cheng, Yifan. 2018. “Single-Particle Cryo-Em—How Did It Get Here and Where Will It Go.” <em>Science</em> 361 (6405). American Association for the Advancement of Science: 876–80. <a href="https://doi.org/10.1126/science.aat4346">https://doi.org/10.1126/science.aat4346</a>.</p>
</div>
<div id="ref-Earl-2017">
<p>Earl, Lesley A, Veronica Falconieri, Jacqueline LS Milne, and Sriram Subramaniam. 2017. “Cryo-Em: Beyond the Microscope.” <em>Current Opinion in Structural Biology</em> 46: 71–78. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002">https://doi.org/https://doi.org/10.1016/j.sbi.2017.06.002</a>.</p>
</div>
<div id="ref-Elmlund-2017">
<p>Elmlund, Dominika, Sarah N Le, and Hans Elmlund. 2017. “High-Resolution Cryo-Em: The Nuts and Bolts.” <em>Current Opinion in Structural Biology</em> 46: 1–6. <a href="https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003">https://doi.org/https://doi.org/10.1016/j.sbi.2017.03.003</a>.</p>
</div>
<div id="ref-Jain-2010">
<p>Jain, Deepti, and Valerie Lamour. 2010. “Computational Tools in Protein Crystallography.” <em>Methods in Molecular Biology (Clifton, N.J.)</em> 673. United States: 129–56. <a href="https://doi.org/10.1007/978-1-60761-842-3_8">https://doi.org/10.1007/978-1-60761-842-3_8</a>.</p>
</div>
<div id="ref-Kuhlbrandt-2014">
<p>Kühlbrandt, Werner. 2014. “Microscopy: Cryo-Em Enters a New Era.” <em>eLife</em> 3. eLife Sciences Publications, Ltd: e03678. <a href="https://doi.org/10.7554/eLife.03678">https://doi.org/10.7554/eLife.03678</a>.</p>
</div>
<div id="ref-Milne-2013">
<p>Milne, Jacqueline L S, Mario J Borgnia, Alberto Bartesaghi, Erin E H Tran, Lesley A Earl, David M Schauder, Jeffrey Lengyel, Jason Pierson, Ardan Patwardhan, and Sriram Subramaniam. n.d. “Cryo-Electron Microscopy–a Primer for the Non-Microscopist.” <em>The FEBS Journal</em> 280 (1): 28–45. <a href="https://doi.org/10.1111/febs.12078">https://doi.org/10.1111/febs.12078</a>.</p>
</div>
<div id="ref-Nogales-2018">
<p>Nogales, Eva. 2018. “Cryo-Em.” <em>Current Biology</em> 28 (19): R1127–R1128. <a href="https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016">https://doi.org/https://doi.org/10.1016/j.cub.2018.07.016</a>.</p>
</div>
<div id="ref-Parker-2016">
<p>Parker, Joanne L, and Simon Newstead. 2016. “Membrane Protein Crystallisation: Current Trends and Future Perspectives.” <em>Advances in Experimental Medicine and Biology</em> 922. Springer International Publishing: 61–72. <a href="https://doi.org/10.1007/978-3-319-35072-1_5">https://doi.org/10.1007/978-3-319-35072-1_5</a>.</p>
</div>
<div id="ref-Powell-2017">
<p>Powell, Harold R. n.d. “X-Ray Data Processing.” <em>Bioscience Reports</em> 37 (5). Portland Press Ltd.: BSR20170227. <a href="https://doi.org/10.1042/BSR20170227">https://doi.org/10.1042/BSR20170227</a>.</p>
</div>
<div id="ref-Rhodes-2006a">
<p>Rhodes, Gale. 2006. “An Overview of Protein Crystallography.” In <em>Crystallography Made Crystal Clear</em>, edited by Gale Rhodes, 3rd ed., 7–30. Complementary Science. Burlington: Academic Press. <a href="https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0">https://doi.org/https://doi.org/10.1016/B978-012587073-3/50004-0</a>.</p>
</div>
<div id="ref-Rupp-2010">
<p>Rupp, Bernhard. 2010. <em>Biomolecular Crystallography: Principles, Practice, and Application to Structural Biology</em>. Garland Science.</p>
</div>
<div id="ref-Sgro-2018">
<p>Sgro, Germán G, and Tiago R D Costa. 2018. “Cryo-Em Grid Preparation of Membrane Protein Samples for Single Particle Analysis.” <em>Front Mol Biosci</em> 5. Departamento de Bioquı'mica, Instituto de Quı'mica, Universidade de São Paulo, São Paulo, Brazil.; Department of Life Sciences, Imperial College London, MRC Centre for Molecular Microbiology; Infection, London, United Kingdom.: 74. <a href="https://doi.org/10.3389/fmolb.2018.00074">https://doi.org/10.3389/fmolb.2018.00074</a>.</p>
</div>
<div id="ref-Wang-2017">
<p>Wang, Hong-Wei, and Jia-Wei Wang. 2017. “How Cryo-Electron Microscopy and X-Ray Crystallography Complement Each Other.” <em>Protein Science</em> 26 (1): 32–39. <a href="https://doi.org/10.1002/pro.3022">https://doi.org/10.1002/pro.3022</a>.</p>
</div>
<div id="ref-Whitford-2005a">
<p>Whitford, David. 2005. “Physical Methods of Determining the Three-Dimensional Structure of Proteins.” Book. In <em>Proteins: Structure and Function</em>. Sussex, England: John Wiley &amp; Sons.</p>
</div>
<div id="ref-Xiong-2008">
<p>Xiong, Yong. 2008. “From electron microscopy to X-ray crystallography: molecular-replacement case studies.” <em>Acta Crystallographica Section D</em> 64 (1): 76–82. <a href="https://doi.org/10.1107/S090744490705398X">https://doi.org/10.1107/S090744490705398X</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>I was REALLY shocked when when facing the search results on EMDB website. According to reviews written in 2014, 4.5Å was still the best resolution ever achieved at that time, and in less than 6 years cryo-EM is becoming able to provide atomic resolution as does X-ray crystallography!<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>To be published. Preprint available on <a href="https://www.biorxiv.org/content/10.1101/855643v1.full">bioRxiv</a><a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/bacterial-cell-division.html" class="archive-item-link">
                Bacterial Cell Division 
              </a>
              <span class="archive-item-date">
                2020-02-08
              </span>
              <div class="article-summary" style="display: none;">
                
                  discussing both Gram positive and negative bacteria with different morphologies, how they divide and chose examples of antibiotics, comparing their different effects
<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#introduction"><span class="toc-section-number">1</span> Introduction</a></li>
<li><a href="#preparation-for-cell-division"><span class="toc-section-number">2</span> Preparation for Cell Division</a><ul>
<li><a href="#chromosome-replication"><span class="toc-section-number">2.1</span> Chromosome Replication</a></li>
<li><a href="#segregation"><span class="toc-section-number">2.2</span> Chromosome Segregation</a></li>
</ul></li>
<li><a href="#ftsz-and-the-z-ring"><span class="toc-section-number">3</span> FtsZ and the Z-ring</a><ul>
<li><a href="#z-ring-placement"><span class="toc-section-number">3.1</span> Z-ring Placement</a><ul>
<li><a href="#the-min-system"><span class="toc-section-number">3.1.1</span> The Min system</a></li>
<li><a href="#nuclear-occlusion"><span class="toc-section-number">3.1.2</span> Nuclear Occlusion</a></li>
</ul></li>
<li><a href="#other-key-proteins-in-the-divisome"><span class="toc-section-number">3.2</span> Other Key Proteins in the Divisome</a><ul>
<li><a href="#zipa-and-ftsa"><span class="toc-section-number">3.2.1</span> ZipA and FtsA</a></li>
<li><a href="#downstream-proteins-recruited-by-the-ftsz-ring"><span class="toc-section-number">3.2.2</span> Downstream Proteins Recruited by the FtsZ Ring</a></li>
<li><a href="#ftsn"><span class="toc-section-number">3.2.3</span> FtsN</a></li>
</ul></li>
</ul></li>
<li><a href="#alternative-strategies-for-bacterial-cell-division"><span class="toc-section-number">4</span> Alternative Strategies for Bacterial Cell Division</a><ul>
<li><a href="#alternative-strategies-of-binary-division"><span class="toc-section-number">4.1</span> Alternative strategies of Binary Division</a><ul>
<li><a href="#caulobacter-crescentus"><span class="toc-section-number">4.1.1</span> <em>Caulobacter crescentus</em></a></li>
<li><a href="#campylobacter-jejuni-and-magnetotactic-bacterium"><span class="toc-section-number">4.1.2</span> <em>Campylobacter jejuni</em> and Magnetotactic Bacterium</a></li>
<li><a href="#agrobacterium-tumefaciens"><span class="toc-section-number">4.1.3</span> <em>Agrobacterium Tumefaciens</em></a></li>
</ul></li>
<li><a href="#positive-regulation-in-myxococcus-xanthus"><span class="toc-section-number">4.2</span> Positive Regulation in <em>Myxococcus xanthus</em></a></li>
<li><a href="#cell-division-without-ftsz"><span class="toc-section-number">4.3</span> Cell Division without FtsZ</a></li>
</ul></li>
<li><a href="#concluding-remarks"><span class="toc-section-number">5</span> Concluding Remarks</a></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>The modes of bacterial cell division vary, but the most common one is binary fission <span class="citation">(Stahl 2019)</span>, and its mechanism, especially in rod-shaped model organisms <em>Escherichia coli</em> and <em>Bacillus subtilis</em>, has been studied extensively. Despite the characterisation of most proteins required for divisome, their coordination at the molecular detail remain largely unclear <span class="citation">(Tsang and Bernhardt 2015)</span>. The question of how bacteria divide is further complicated by variations in proteins involved, and even in the physical nature, among different bacterial species <span class="citation">(Eswara and Ramamurthi 2017)</span></p>
<p>This essay introduces FtsZ as a highly conserved protein involved in septum formation and its regulation in model organsisms, and gives examples on diviations from these rules that mainly comes from studies on nonmodel organisms.</p>
</div>
<div id="preparation-for-cell-division" class="section level1">
<h1><span class="header-section-number">2</span> Preparation for Cell Division</h1>
<p>The chromosome must be replicated once and only once prior to cell devision, and it must be ensured that each daughter cell receives one copy of the chromosome.</p>
<div id="chromosome-replication" class="section level2">
<h2><span class="header-section-number">2.1</span> Chromosome Replication</h2>
<p>Chromosome replication is principally regulated by DnaA in <em>E. coli</em>. DnaA initiates replication by binding its active, GTP-bound form (DnaA-ATP) to <em>OriC</em>. Its inactivation upon initiation of replication ensures one round of replication per cell cycle<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Such mechanisms include competition for <em>OriC</em> binding by SeqA, repression of <em>dnaA</em> expression (also by SeqA), and inactivation of DnaA-ATP by the ATPase HdaA.</p>
</div>
<div id="segregation" class="section level2">
<h2><span class="header-section-number">2.2</span> Chromosome Segregation</h2>
<p>Segregation of chromosomes and plasmids is achieved while they are being replicated by using the Par system in some bacteria such as the budding bacteria <em>Caulobacter crescentus</em>, as illustrated in Figure <a href="#fig:cc-par">2.1</a>. popZ proteins anchors the chromosome at its <em>parS</em> sequence (near <em>oriC</em>) to the old pole, which is mediated by parB. As replication starts, more parB binds to <em>parS</em> sequence of the newly synthesised chromosome, and is pulled to the new cell pole by the ATPase activity of ParA.</p>
<div class="figure"><span id="fig:cc-par"></span>
<img src="../img/cc-par.jpg" alt="The Par system segregates replicated chromosomes in Caulobacter crescentus." width="100%" />
<p class="caption">
Figure 2.1: The Par system segregates replicated chromosomes in Caulobacter crescentus.
</p>
</div>
<p>The mechanism of chromosome segregation in <em>E. coli</em>, which lacks Par, is not well characterised.</p>
</div>
</div>
<div id="ftsz-and-the-z-ring" class="section level1">
<h1><span class="header-section-number">3</span> FtsZ and the Z-ring</h1>
<p>Central to almost all bactrial division is the FtsZ protein. FtsZ is a homolog of tubulin found in most bacteria and some archaea. Like tubulin, FtsZ is a GTPase and can polymerise into filaments. During cell division, FtsZ filaments assemble into a ring (Z-ring) at the site where the septum forms. The FtsZ ring then recruits other proteins of the divisome, allowing division to proceed. The Z-ring contracts as the bacterium divides, forming an invagination which finally divides the bacterium in two.</p>
<p>The GTPase activity accounts for the rapid turnover dynamics of the FtsZ filaments that form the Z-ring. It has also been proposed that FtsZ’s GTPase activity energises the constriction of the ring and the invagination of the cell membrane and the cell wall, but this notion is challenged by the observation of mutants that are severely impoaired in GTP hydrolysis still capable of dividing, and mathematical modelling that shows force generation could be achieved independenly of GTPase activity, but instead with hydrolysis <span class="citation">(Adams and Errington 2009)</span>.</p>
<div id="z-ring-placement" class="section level2">
<h2><span class="header-section-number">3.1</span> Z-ring Placement</h2>
<p>The site of cell division in rod-shaped bacteria is usually at the centre of the cell, which is dependent on the corrent placement of the Z-ring. As shown in Figure <a href="#fig:no-min">3.1</a>, this is typically achieved using two mechanisms, nucleoid occlusion (NO), and the Min system, which are well characterised in <em>E. coli</em> and <em>B. subtilis</em>. It should be noted, however, that there is increasing evidence showing Z-ring positioning may be determined by other factors, and Min and NO systems may primarily function to ensure the efficient utilisation of this site. Such evidence include the observation of precise midcell Z-ring formation in Min- and Noc-null <em>B. subtilis</em> by <span class="citation">Rodrigues and Harry (2012)</span>.</p>
<div id="the-min-system" class="section level3">
<h3><span class="header-section-number">3.1.1</span> The Min system</h3>
<p>The MinC, MinD and MinE proteins interact to prevent Z-ring formation at the poles <span class="citation">(Ghosal et al. 2014; Ramm, Heermann, and Schwille 2019)</span>. MinC is the effector of the system owing to its inhibitory effect on FtsZ ring assembly. MinD is an ATPase which recruit MinC to the membrane. MinCD complex together inhibit Z-ring formation. MinE is an ATPase-activating protein which has an anti-MinCD domain and a topological specificity domain (which confines its anti-MinCD activity only at midcell). The three proteins MinCDE form an oscillation cycle from pole to pole (driven by MinDE), causing the time-averaged concentration of MinCD complexes to be lowest at the midcell, which favours Z-ring formation there. <em>B. subtilis</em> also has a Min system but it is not involved in the placement of the Z-ring.</p>
</div>
<div id="nuclear-occlusion" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Nuclear Occlusion</h3>
<p>Nuclear occlusion prevents Z-ring formation atop of nucleoids. It is mediated by SlmA in <em>E. coli</em> and Noc in <em>B subtilis</em>. Neither are normally essential in their respective organisms, but both are synthetically lethal with mutatations of Min, due to chaotic FtsZ assembly.</p>
<p>According to <span class="citation">Adams, Wu, and Errington (2015)</span>, Noc has a sequence-specific DNA-binding domain and a membrane-associating domain. Noc is thought to oligomerise using their two dimerisation domains, bind to DNA at specific sequences, and then insert into the inner leaflet of the cytoplasmic membrane.</p>
<div class="figure"><span id="fig:no-min"></span>
<img src="../img/no-min.jpg" alt="Negative regulation by nuclear occulation and the Min system ensures formation of the FtsZ ring in E coli." width="100%" />
<p class="caption">
Figure 3.1: Negative regulation by nuclear occulation and the Min system ensures formation of the FtsZ ring in E coli.
</p>
</div>
</div>
</div>
<div id="other-key-proteins-in-the-divisome" class="section level2">
<h2><span class="header-section-number">3.2</span> Other Key Proteins in the Divisome</h2>
<p>A fully functional divisome requires a combination of proteins in addition to the FtsZ ring. Most of these proteins have close association with the Z ring. In <em>E. coli</em>, It was thought that the recruitment of these proteins appears to have a linear dependecy pathway which starts with FtsA/ZipA and ends with FtsN, but a recent review by <span class="citation">Du and Lutkenhaus (2017)</span> shows evidence for the noon-sequential assembly of the divisome.</p>
<div id="zipa-and-ftsa" class="section level3">
<h3><span class="header-section-number">3.2.1</span> ZipA and FtsA</h3>
<p>ZipA is acts to tether the FtsZ ring to the cytoplasmic membrane. It is not widely conserved outside of Gammaproteobacteria, and, in <em>E. coli</em>, its necessity in cell division is bypassed by a gain-of-function mutation in FtsA. FtsA, which is more widely conserved in bacteria, also anchors the Z ring to the membrane and coordinates its initial assembly <span class="citation">(Adams and Errington 2009)</span>.</p>
<p>FtsA is related to actin, and, like actin, it binds ATP and is found to reversibly polymerise into corkscrew-like helices in <em>Streptococcus pneumoniae</em>. Its ATPase activity is reported in <em>B. subtilis</em> but not in <em>S. pneumoniae</em>. Mutagenesis studies suggest that nucleotide binding may be required for FtsA’s ability to interact with itself and with FtsZ, <span class="citation">(Pichoff and Lutkenhaus 2007)</span> but its exact role is unknown.</p>
<p>FtsA differs from actin in that it is missing the 1B domain and it has an unrelated subdomain called 1C. The 1C domain is involved in recruiting downstream components of the divisome to the Z ring.</p>
</div>
<div id="downstream-proteins-recruited-by-the-ftsz-ring" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Downstream Proteins Recruited by the FtsZ Ring</h3>
<p>Upon assembly of the FtsZ ring, other downstream proteins, notably those involved in peptidoglycan biosynthesis, are recruited to form the divisome.</p>
<p>FtsI, also known as penicillin binding protein 3 (PBP3), is one such protein in <em>E. coli</em>. It is required specifically for peptidoglycan synthesis at the septum <span class="citation">(Chen and Beckwith 2001)</span>. Its recruitment is dependent on FtsK, which appears to be a bifunctional protein, with the C-terminal domain facilitating segregation of chromosome and the N-terminal domain carrying out a necessary, but undefined, function in septum development.</p>
<p>A summary of the divisome components is shown in Figure <a href="#fig:ffts-components">3.2</a></p>
<div class="figure"><span id="fig:ffts-components"></span>
<img src="../img/ffts-components.png" alt="Components of the divisome." width="100%" />
<p class="caption">
Figure 3.2: Components of the divisome.
</p>
</div>
</div>
<div id="ftsn" class="section level3">
<h3><span class="header-section-number">3.2.3</span> FtsN</h3>
<p>In <em>E. coli</em>, FtsN is the last component of recruited to the divisome. Once it arrives at the Z ring, the divisome begin to actively synthesising septal peptidoglycan to divide the cell. It has been suggest that FtsA-FtsN interaction may trigger cell constriction in a positive feedback loop <span class="citation">(Tsang and Bernhardt 2015)</span>, but the exact mechanism remain elusive.</p>
</div>
</div>
</div>
<div id="alternative-strategies-for-bacterial-cell-division" class="section level1">
<h1><span class="header-section-number">4</span> Alternative Strategies for Bacterial Cell Division</h1>
<div id="alternative-strategies-of-binary-division" class="section level2">
<h2><span class="header-section-number">4.1</span> Alternative strategies of Binary Division</h2>
<p>Binary division means dividing a bacterial cell into daughter cells with equal volume and physiology, which requires the FtsZ ring to form at the midcell. Apart form the NO and Min systems described above, there are other strategies to achieve this.</p>
<div class="figure"><span id="fig:binary-division-alternative"></span>
<img src="../img/binary-division-alternative.jpg" alt="Alternative strategies of Binary Division" width="100%" />
<p class="caption">
Figure 4.1: Alternative strategies of Binary Division
</p>
</div>
<div id="caulobacter-crescentus" class="section level3">
<h3><span class="header-section-number">4.1.1</span> <em>Caulobacter crescentus</em></h3>
<p>The budding bacteria <em>C. crescentus</em> uses the MipZ gradient to regulate FtsZ ring placement (Figure <a href="#fig:binary-division-alternative">4.1</a> A). MipZ is translocated from the old pole to the new pole during chromosome replication and segregation, by directly interacting with the ParB-<em>parS</em> complex (see section <a href="#segregation">2.2</a>). MipZ displaces FtsZ from polar regions, thus permitting ring formation only at midcell.</p>
</div>
<div id="campylobacter-jejuni-and-magnetotactic-bacterium" class="section level3">
<h3><span class="header-section-number">4.1.2</span> <em>Campylobacter jejuni</em> and Magnetotactic Bacterium</h3>
<p>The amphitrichous<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> bacteria <em>Campylobacter jejuni</em> uses the ATPase FlhG, which is a MinD/ParA homologue and a known regulator of flagellar number, to locate the Z ring placement, by expelling FtsZ from the two longitudinal poles where flagella grow.</p>
<p>By constrast, in the monotrichohus magnetotactic bacterium of the <em>Gammaproteobacteria</em> class, flagellum formation in the daughter cell is dictated by the septum (Figure <a href="#fig:binary-division-alternative">4.1</a> B and C).</p>
</div>
<div id="agrobacterium-tumefaciens" class="section level3">
<h3><span class="header-section-number">4.1.3</span> <em>Agrobacterium Tumefaciens</em></h3>
<p>Most bacteria that exhibit binary division elongate their cells from the middle or at several locations perpenticular to the long axis (e.g. in <em>E. coli</em>, which is mediated by MreB). However, <em>Agrobacterium Tumefaciens</em>, which elongates unidirectionally, can still achhive binary division. As shown in Figure <a href="#fig:binary-division-alternative">4.1</a> D, after division, FtsA and FtsZ stay at the newly formed growth pole and facilitate polar growth. Upon elongation, they relocate to the midcell to initiate division.</p>
</div>
</div>
<div id="positive-regulation-in-myxococcus-xanthus" class="section level2">
<h2><span class="header-section-number">4.2</span> Positive Regulation in <em>Myxococcus xanthus</em></h2>
<p>The NO and Min systems described above are both negative regulatory mechanisms, where regions of low activity of relevant proteins allows for foramtion of FtsZ ring. There is evidence in some nonmodel organisms of positive regulation, where FtsZ ring formation is promoted by presence of certain proteins. This is exemplified by the deltaproteobacterium <em>M. xanthus</em>, in which the ParA-like protein PomZ localises to midcell in an FtsZ-independent manner before FtsZ ring formation.</p>
</div>
<div id="cell-division-without-ftsz" class="section level2">
<h2><span class="header-section-number">4.3</span> Cell Division without FtsZ</h2>
<p>In some species of Mycoplasma (which are well known for their small genomes), the FtsZ homologue is absent <span class="citation">(Lluch-Senar, Querol, and Piñol 2010)</span>. Although the cell division mechanisms in these organisms remain elusive, they lead to reassessments on the role of FtsZ in FtsZ-dependent cell division. For example, as reviewed by <span class="citation">Xiao and Goley (2016)</span>, it is likely that the main driving force for membrane invagination and constriction is not directly provided by FtsZ, but by the peptidoglycan synthesis.</p>
</div>
</div>
<div id="concluding-remarks" class="section level1">
<h1><span class="header-section-number">5</span> Concluding Remarks</h1>
<p>There is an extensive repertoire of cell division strategies based on distinct families of proteins in the domain of bacteria. The lack of universal conservation and the ease of horizontal gene transfer among bacteria make cell division a suboptimal target of antibiotics. Indeed, most available antibiotics related to bacterial cell division target cell wall synthesis, and there is little progress made on developing drugs that interferes with other components of bacterial divisome.</p>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Adams-2009">
<p>Adams, David W., and Jeff Errington. 2009. “Bacterial Cell Division: Assembly, Maintenance and Disassembly of the Z Ring.” <em>Nature Reviews Microbiology</em> 7 (9): 642–53. <a href="https://doi.org/10.1038/nrmicro2198">https://doi.org/10.1038/nrmicro2198</a>.</p>
</div>
<div id="ref-Adams-2015">
<p>Adams, David William, Ling Juan Wu, and Jeff Errington. 2015. “Nucleoid Occlusion Protein Noc Recruits Dna to the Bacterial Cell Membrane.” <em>EMBO J</em> 34 (4): 491–501. <a href="https://doi.org/10.15252/embj.201490177">https://doi.org/10.15252/embj.201490177</a>.</p>
</div>
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Chen-2001">
<p>Chen, Joseph C., and Jon Beckwith. 2001. “FtsQ, Ftsl and Ftsi Require Ftsk, but Not Ftsn, for Co-Localization with Ftsz During Escherichia Coli Cell Division.” <em>Molecular Microbiology</em> 42 (2): 395–413. <a href="https://doi.org/10.1046/j.1365-2958.2001.02640.x">https://doi.org/10.1046/j.1365-2958.2001.02640.x</a>.</p>
</div>
<div id="ref-Du-2017">
<p>Du, Shishen, and Joe Lutkenhaus. 2017. “Assembly and Activation of the Escherichia Coli Divisome.” <em>Molecular Microbiology</em> 105 (2): 177–87. <a href="https://doi.org/10.1111/mmi.13696">https://doi.org/10.1111/mmi.13696</a>.</p>
</div>
<div id="ref-Eswara-2017">
<p>Eswara, Prahathees J., and Kumaran S. Ramamurthi. 2017. “Bacterial Cell Division: Nonmodels Poised to Take the Spotlight.” <em>Annual Review of Microbiology</em> 71 (1): 393–411. <a href="https://doi.org/10.1146/annurev-micro-102215-095657">https://doi.org/10.1146/annurev-micro-102215-095657</a>.</p>
</div>
<div id="ref-Ghosal-2014">
<p>Ghosal, Debnath, Daniel Trambaiolo, Linda A. Amos, and Jan Löwe. 2014. “MinCD Cell Division Proteins Form Alternating Copolymeric Cytomotive Filaments.” <em>Nature Communications</em> 5 (1): 5341. <a href="https://doi.org/10.1038/ncomms6341">https://doi.org/10.1038/ncomms6341</a>.</p>
</div>
<div id="ref-Lluch-Senar-2010">
<p>Lluch-Senar, Maria, Enrique Querol, and Jaume Piñol. 2010. “Cell Division in a Minimal Bacterium in the Absence of ftsZ.” <em>Molecular Microbiology</em> 78 (2): 278–89. <a href="https://doi.org/10.1111/j.1365-2958.2010.07306.x">https://doi.org/10.1111/j.1365-2958.2010.07306.x</a>.</p>
</div>
<div id="ref-Pichoff-2007">
<p>Pichoff, Sebastien, and Joe Lutkenhaus. 2007. “Identification of a Region of Ftsa Required for Interaction with Ftsz.” <em>Molecular Microbiology</em> 64 (4): 1129–38. <a href="https://doi.org/10.1111/j.1365-2958.2007.05735.x">https://doi.org/10.1111/j.1365-2958.2007.05735.x</a>.</p>
</div>
<div id="ref-Ramm-2019">
<p>Ramm, Beatrice, Tamara Heermann, and Petra Schwille. 2019. “The E. Coli Mincde System in the Regulation of Protein Patterns and Gradients.” <em>Cellular and Molecular Life Sciences</em> 76 (21): 4245–73. <a href="https://doi.org/10.1007/s00018-019-03218-x">https://doi.org/10.1007/s00018-019-03218-x</a>.</p>
</div>
<div id="ref-Rodrigues-2012">
<p>Rodrigues, Christopher D. A., and Elizabeth J. Harry. 2012. “The Min System and Nucleoid Occlusion Are Not Required for Identifying the Division Site in Bacillus Subtilis but Ensure Its Efficient Utilization.” <em>PLOS Genetics</em> 8 (3). Public Library of Science: 1–20. <a href="https://doi.org/10.1371/journal.pgen.1002561">https://doi.org/10.1371/journal.pgen.1002561</a>.</p>
</div>
<div id="ref-Stahl-2019a">
<p>Stahl, Michael T Madigan; Kelly S Bender; Daniel Hezekiah Buckley; W Matthew Sattley; David Allan. 2019. “Molecular Biology of Microbial Growth.” In <em>Brock Biology of Microorganisms</em>, Fifteenth edition, Global edition. Pearson.</p>
</div>
<div id="ref-Tsang-2015">
<p>Tsang, Mary-Jane, and Thomas G Bernhardt. 2015. “Guiding Divisome Assembly and Controlling Its Activity.” <em>Current Opinion in Microbiology</em> 24: 60–65. <a href="https://doi.org/https://doi.org/10.1016/j.mib.2015.01.002">https://doi.org/https://doi.org/10.1016/j.mib.2015.01.002</a>.</p>
</div>
<div id="ref-Xiao-2016">
<p>Xiao, Jie, and Erin D Goley. 2016. “Redefining the Roles of the Ftsz-Ring in Bacterial Cytokinesis.” <em>Current Opinion in Microbiology</em> 34: 90–96. <a href="https://doi.org/https://doi.org/10.1016/j.mib.2016.08.008">https://doi.org/https://doi.org/10.1016/j.mib.2016.08.008</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Some fast-growing bacteria may afford multiple rounds of replication on a single chromosome, and this makes it possible for their division to outpace DNA replication.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>one flagellum at each pole<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/mcb-toolbox.html" class="archive-item-link">
                MCB Toolbox 
              </a>
              <span class="archive-item-date">
                2020-01-23
              </span>
              <div class="article-summary" style="display: none;">
                
                


<p>For the second tutorial, I want you to try some proper molecular biology using online tools, just as we do every day in the lab. Try it out on your own first, though you may wish to discuss any issues you may be having with each other. Please bring files/printouts and if you have one, your laptop to the tutorial. You will find NCBI BLAST useful and various links within www.expasy.org (e.g. In proteomics section, Clustal, T-coffee, Translate, prot param etc). You may also like to download ApE or Snapgene viewer from the web but make sure you scan with e.g. Sophos before you open them. You can do full in silico cloning in ApE (but only in the expensive paid-for version of Snapgene) but Snapgene (even free viewer) offers much better visualization so you may want to use both but for different features. For structural productions, Swissprot is convenient, then for mutagenesis, move to Pymol or equivalent.</p>
<pre><code>Mystery gene (lower case = non-coding, upper case = coding)

GTATTTTGCGTCCATTTCGATCAAAAGACTATTTCTCATTTCCTCCCACGATTTTAGATAATAATATTATCTTATGTCATGCACAGATGATAAGTGATGATGACGATCTACCATCTACTCGCCCGGGATCTGTGAATGAGGAATTACCAGAAACCGAACCCGAAGATAATGATGAGTTGCCTGAAACAGAACCTGAAAGCGATTCCGATAAACCTACCGTAACCTCGAATAAAACAGAAAACCAAGTTGCTGATGAAGATTATGATTCATTCGACGATTTTGTGCCCAGTCAAACACACACAGCCTCCAAAATACCTGTAAAAAATAAACGAGCCAAAAAGTGCACTGTAGAATCTGATTCATCATCTTCGGATGATTCCGATCAAGGAGATGATTGTGAATTTATCCCAGCTTGTGATGAGACACAGGAAGTTCCGAAAATCAAAAGAGGATACACTCTGAGAACTCGAGCAAGTGTAAAGAACAAATGTGATGATTCATGGGATGATGGAATAGACGAAGAAGATGTCTCAAAAAGATCAGAAGACACGTTAAATGATTCATTTGTTGATCCTGAATTCATGGATTCTGTTCTAGATAATCAATTAACGATCAAAGGCAAAAAGCAATTTCTCGATGATGGAGAGTTTTTCACAGACCGGAATGTTCCTCAGATTGATGAAGCTACAAAAATGAAGTGGGCATCAATGACGTCACCTCCTCAAGAAGCTTTGAACGCATTGAACGAATTCTTCGGTCATAAAGGATTCCGAGAAAAGCAGTGGGATGTTGTCAGAAATGTTTTGGGAGGAAAAGACCAATTTGTTCTTATGTCCACTGGTTATGGTAAAAGTGTATGTTATCAGCTACCATCACTTCTTCTCAATTCGATGACTGTCGTGGTATCTCCATTAATTTCATTGATGAATGATCAAGTAACTACATTGGTTTCTAAAGGTATTGATGCAGTGAAACTAGATGGACATTCTACACAAATTGAATGGGATCAAGTTGCGAATAATATGCACCGAATTAGGTTCATCTACATGTCACCTGAAATGGTTACGAGCCAAAAGGGTTTGGAATTATTAACTTCTTGCCGAAAACATATCTCCCTCCTCGCTATTGATGAAGCTCATTGTGTTTCTCAATGGGGACACGACTTTCGAAACTCGTACAGGCATCTCGCAGAAATTAGAAACCGATCTGATCTATGCAATATTCCAATGATTGCTCTTACCGCTACTGCCACAGTTAGAGTTCGTGATGACGTCATTGCTAATTTAAGACTCCGCAAGCCATTAATCACAACTACGTCGTTTGATAGAAAGAATCTCTACATTTCTGTGCATTCTTCAAAGGACATGGCTGAAGATTTAGGATTATTCATGAAAACCGATGAAGTTAAAGGAAGACACTTTGGTGGACCTACTATTATTTATTGCCAAACGAAACAAATGGTCGATGATGTGAACTGTGTTTTGAGAAGAATCGGAGTTCGTTCTGCTCATTATCACGCAGGACTCACTAAAAATCAACGAGAAAAAGCACACACCGATTTTATGAGAGATAAGATTACAACAATCGTTGCGACAGTTGCATTTGGTATGGGAATTGACAAACCCGACGTTCGAAATGTGATTCATTACGGATGCCCGAACAATATCGAATCATATTATCAAGAAATCGGAAGAGCTGGTCGAGATGGATCTCCAAGTATTTGTCGTGTATTCTGGGCTCCGAAAGATTTGAATACTATAAAATTTAAACTTCGAAATTCGCAGCAAAAAGAAGAAGTAGTTGAAAATCTTACAATGATGCTAAGACAACTCGAGTTGGTTCTGACAACCGTTGGATGTAGAAGATACCAACTTCTGAAGCACTTTGACCCATCATACGCGAAACCTCCAACTATGCAAGCTGATTGTTGTGATAGATGTACTGAAATGCTCAATGGAAATCAAGATTCATCATCCAGTATTGTTGATGTTACAACAGAATCGAAGTGGTTGTTTCAAGTTATTAACGAAATGTACAACGGGAAAACTGGTATCGGAAAACCAATCGAATTTCTGAGGGGATCGAGTAAAGAAGACTGGCGAATCAAGACCACATCTCAACAAAAATTGTTTGGAATTGGAAAACATATTCCTGATAAATGGTGGAAAGCACTTGCAGCATCACTTCGAATTGCTGGTTATCTTGGAGAAGTTAGGCTGATGCAAATGAAATTTGGAAGTTGTATCACTTTGTCCGAACTCGGGGAACGATGGCTTTTGACTGGAAAAGAGATGAAAATCGATGCGACACCGATTTTATTGCAAGGGAAGAAAGAAAAAGCCGCACCTTCAACTGTCCCCGGAGCTTCAAGATCTCAGTCAACTAAATCAAGTACAGAGATTCCAACCAAGATTCTCGGAGCGAATAAGATTCGTGAATACGAGCCTGCAAATGAAAACGAGCAGCTGATGAACTTGAAAAAGCAAGAAGTCACTGGTCTTCCAGAGAAGATTGATCAACTGCGCTCTCGTCTTGACGACATTCGTGTAGGAATTGCAAACATGCATGAAGTAGCACCATTCCAAATTGTATCGAATACTGTTCTTGATTGTTTTGCCAACTTGAGACCTACCTCAGCCTCGAATCTCGAAATGATTGATGGAATGTCGGCTCAGCAGAAATCTAGATACGGAAAACGATTTGTCGATTGTGTTGTACAATTTTCAAAGGAAACTGGCATTGCAACAAACGTCAATGCCAACGATATGATACCCCCTGAACTTATTTCAAAAATGCAGAAAGTTCTCTCGGATGCGGTGAGAAGAGTATACACAGAGCATCTTATTTCGAGATCCACTGCGAAAGAAGTGGCAACTGCTCGAGGAATTAGTGAGGGTACTGTATATTCATATCTCGCGATGGCGGTAGAAAAAGGATTACCTCTTCACTTAGACAAGTTAAATGTCTCCAGAAAGAATATTGCAATGGCTCTAAATGCAGTTAGAGTACATTTAGGATCAAATGTTGCCGTACTGACACCATGGGTTGAAGCTATGGGAGTCGTACCTGATTTTAATCAGTTGAAATTGATCCGGGCAATTCTTATTTACGAATATGGATTGGATACGAGTGAGAACCAAGAGAAGCCAGACATCCAATCTATGCCGTCCACTTCAAATCCATCCACCATCAAAACAGTTCCATCAACACCTTCATCCTCTCTCAGAGCTCCTCCATTGAAGAAATTCAAACTTTAAATTTCTTATATTTTTTTTAATCTGAATATCATAACTATACGGTCTCTTATTTTCAATCTCATCCCTGTCTATATGAATGCTGGTTATTCGATTTTCACTTATGAATTTTTAAAAACACTT</code></pre>
<pre><code>pIVEX2.3d Vector Sequence

TCGCGCGTTTCGGTGATGACGGTGAAAACCTCTGACACATGCAGCTCCCGGAGACGGTCACAGCTTGTCTGTAAGCGGATGCCGGGAGCAGACAAGCCCGTCAGGGCGCGTCAGCGGGTGTTGGCGGGTGTCGGGGCTGGCTTAACTATGCGGCATCAGAGCAGATTGTACTGAGAGTGCACCATATATGCGGTGTGAAATACCGCACAGATGCGTAAGGAGAAAATACCGCATCAGGCGCCATTCGCCATTCAGGCTGCGCAACTGTTGGGAAGGGCGATCGGTGCGGGCCTCTTCGCTATTACGCCAGCTGGCGAAAGGGGGATGTGCTGCAAGGCGATTAAGTTGGGTAACGCCAGGGTTTTCCCAGTCACGACGTTGTAAAACGACGGCCAGTGCCAAGCTTGCATGCAAGGAGATGGCGCCCAACAGTCCCCCGGCCACGGGGCCTGCCACCATACCCACGCCGAAACAAGCGCTCATGAGCCCGAAGTGGCGAGCCCGATCTTCCCCATCGGTGATGTCGGCGATATAGGCGCCAGCAACCGCACCTGTGGCGCCGGTGATGCCGGCCACGATGCGTCCGGCGTAGAGGATCGAGATCTCGATCCCGCGAAATTAATACGACTCACTATAGGGAGACCACAACGGTTTCCCTCTAGAAATAATTTTGTTTAACTTTAAGAAGGAGATATACCATGGCACATATGAGCGGCCGCGTCGACTCGAGCGAGCTCCCGGGGGGGGTTCTCATCATCATCATCATCATTAATAAAAGGGCGAATTCCAGCACACTGGCGGCCGTTACTAGTGGATCCGGCTGCTAACAAAGCCCGAAAGGAAGCTGAGTTGGCTGCTGCCACCGCTGAGCAATAACTAGCATAACCCCTTGGGGCCTCTAAACGGGTCTTGAGGGGTTTTTTGCTGAAAGGAGGAACTATATCCGGATATCCACAGGACGGGTGTGGTCGCCATGATCGCGTAGTCGATAGTGGCTCCAAGTAGCGAAGCGAGCAGGACTGGGCGGCGGCCAAAGCGGTCGGACAGTGCTCCGAGAACGGGTGCGCATAGAAATTGCATCAACGCATATAGCGCTAGCAGCACGCCATAGTGACTGGCGATGCTGTCGGAATGGACGATATCCCGCAAGAGGCCCGGCAGTACCGGCATAACCAAGCCTATGCCTACAGCATCCAGGGTGACGGTGCCGAGGATGACGATGAGCGCATTGTTAGATTTCATACACGGTGCCTGACTGCGTTAGCAATTTAACTGTGATAAACTACCGCATTAAAGCTTATCGATGATAAGCTGTCAAACATGAGAATTCGTAATCATGTCATAGCTGTTTCCTGTGTGAAATTGTTATCCGCTCACAATTCCACACAACATACGAGCCGGAAGCATAAAGTGTAAAGCCTGGGGTGCCTAATGAGTGAGCTAACTCACATTAATTGCGTTGCGCTCACTGCCCGCTTTCCAGTCGGGAAACCTGTCGTGCCAGCTGCATTAATGAATCGGCCAACGCGCGGGGAGAGGCGGTTTGCGTATTGGGCGCTCTTCCGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAAAGGCGGTAATACGGTTATCCACAGAATCAGGGGATAACGCAGGAAAGAACATGTGAGCAAAAGGCCAGCAAAAGGCCAGGAACCGTAAAAAGGCCGCGTTGCTGGCGTTTTTCCATAGGCTCCGCCCCCCTGACGAGCATCACAAAAATCGACGCTCAAGTCAGAGGTGGCGAAACCCGACAGGACTATAAAGATACCAGGCGTTTCCCCCTGGAAGCTCCCTCGTGCGCTCTCCTGTTCCGACCCTGCCGCTTACCGGATACCTGTCCGCCTTTCTCCCTTCGGGAAGCGTGGCGCTTTCTCATAGCTCACGCTGTAGGTATCTCAGTTCGGTGTAGGTCGTTCGCTCCAAGCTGGGCTGTGTGCACGAACCCCCCGTTCAGCCCGACCGCTGCGCCTTATCCGGTAACTATCGTCTTGAGTCCAACCCGGTAAGACACGACTTATCGCCACTGGCAGCAGCCACTGGTAACAGGATTAGCAGAGCGAGGTATGTAGGCGGTGCTACAGAGTTCTTGAAGTGGTGGCCTAACTACGGCTACACTAGAAGGACAGTATTTGGTATCTGCGCTCTGCTGAAGCCAGTTACCTTCGGAAAAAGAGTTGGTAGCTCTTGATCCGGCAAACAAACCACCGCTGGTAGCGGTGGTTTTTTTGTTTGCAAGCAGCAGATTACGCGCAGAAAAAAAGGATCTCAAGAAGATCCTTTGATCTTTTCTACGGGGTCTGACGCTCAGTGGAACGAAAACTCACGTTAAGGGATTTTGGTCATGAGATTATCAAAAAGGATCTTCACCTAGATCCTTTTAAATTAAAAATGAAGTTTTAAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAATCAGTGAGGCACCTATCTCAGCGATCTGTCTATTTCGTTCATCCATAGTTGCCTGACTCCCCGTCGTGTAGATAACTACGATACGGGAGGGCTTACCATCTGGCCCCAGTGCTGCAATGATACCGCGAGACCCACGCTCACCGGCTCCAGATTTATCAGCAATAAACCAGCCAGCCGGAAGGGCCGAGCGCAGAAGTGGTCCTGCAACTTTATCCGCCTCCATCCAGTCTATTAATTGTTGCCGGGAAGCTAGAGTAAGTAGTTCGCCAGTTAATAGTTTGCGCAACGTTGTTGCCATTGCTACAGGCATCGTGGTGTCACGCTCGTCGTTTGGTATGGCTTCATTCAGCTCCGGTTCCCAACGATCAAGGCGAGTTACATGATCCCCCATGTTGTGCAAAAAAGCGGTTAGCTCCTTCGGTCCTCCGATCGTTGTCAGAAGTAAGTTGGCCGCAGTGTTATCACTCATGGTTATGGCAGCACTGCATAATTCTCTTACTGTCATGCCATCCGTAAGATGCTTTTCTGTGACTGGTGAGTACTCAACCAAGTCATTCTGAGAATAGTGTATGCGGCGACCGAGTTGCTCTTGCCCGGCGTCAATACGGGATAATACCGCGCCACATAGCAGAACTTTAAAAGTGCTCATCATTGGAAAACGTTCTTCGGGGCGAAAACTCTCAAGGATCTTACCGCTGTTGAGATCCAGTTCGATGTAACCCACTCGTGCACCCAACTGATCTTCAGCATCTTTTACTTTCACCAGCGTTTCTGGGTGAGCAAAAACAGGAAGGCAAAATGCCGCAAAAAAGGGAATAAGGGCGACACGGAAATGTTGAATACTCATACTCTTCCTTTTTCAATATTATTGAAGCATTTATCAGGGTTATTGTCTCATGAGCGGATACATATTTGAATGTATTTAGAAAAATAAACAAATAGGGGTTCCGCGCACATTTCCCCGAAAAGTGCCACCTGACGTCTAAGAAACCATTATTATCATGACATTAACCTATAAAAATAGGCGTATCACGAGGCCCTTTCGTC</code></pre>
<ol style="list-style-type: decimal">
<li>Use the information in the attached DNA file (mystery gene) to identify the encoded protein and the species.
<ul>
<li>Caenorhabditis elegans Probable Werner syndrome ATP-dependent helicase homolog 1 (wrn-1) (UniProt accession code: <a href="https://www.uniprot.org/uniprot/Q19546">Q19546</a>)</li>
</ul></li>
<li>Draw a schematic diagram of the protein, noting the number of amino acids, predicted Mw and pI of the encoded protein, and annotate on functional domains (e.g. in ppt).
<ul>
<li>1056 amino acids; MW = 118522.73 Da (average mass)/118447.63 Da (monoisotopic mass); pI = 6.08 (predicted using ExPASy <a href="https://web.expasy.org/compute_pi/">‘Compute pI/Mw tool’</a>)</li>
<li><a href="https://swissmodel.expasy.org/repository/uniprot/Q19546">Automated homology modelling in the SWISS-MODEL Repository</a> uses 1oyy.1.A, 4cdg.1.A and 4cgz.1.A as templates to obtain models with global QMEANs of -3.35, -3.57 and -4.19, respectively</li>
<li>Manual (interactive) modelling using 4q47.1.A (DrRecQ catalytic core (Dr = <em>Deinococcus radiodurans</em>)) achieved a better QMEAN of -3.09. The structure is shown in Fig. <a href="#fig:wrn-model">1</a> (highlighting functional domains) and in Fig. <a href="#fig:wrn-b-factor">2</a> (showing local quality (QMEAN) scores). The annotations on functional domains are based on <span class="citation">Chen et al. (2014)</span>.</li>
<li>Update on 2 Feb: yet a higher QEMAN of -2.90 is achieved with 2wwy.1. 2wwy.1 is the structure of human RECQ-like helicase in complex with DNA substrate<span class="citation">(Pike et al. 2015)</span>. The homology model based on this is shown in Fig. <a href="#fig:recq-in-action">3</a>, and the PyMOL Session (<code>.pse</code>) file can be downloaded <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/model/tianyi-wrn.pse">here</a>.</li>
<li>As shown in Fig. <a href="#fig:wrn-org-cox">4</a>, WRN is unique in the family of RecQ helicases in possessing an 3’-to-5’ exonuclease domain<span class="citation">(Cox and Mason 2013)</span>. In <em>C. elegans</em>, the exonuclease and the helicase are encoded in separate genes. The two experimental structures used for modelling (4q47.1.A (DrRecQ) and 2wwy.1 (human RECQ1)) and the query <em>C. elegans</em> wrn homolog all represent only the helicase domain and the RQC (WH) domain.</li>
</ul></li>
</ol>
<div class="figure"><span id="fig:wrn-model"></span>
<img src="../img/wrn-model.png" alt="A homology model of *C. elegans* wrn-1 based on DrRecQ catalytic core (PDB accession code: 4Q47). The N-terminal part contains the common core structure of the RecQ helicase family, which comprises two RecA-like domains, D1 and D2, each composed of a central β-sheet surrounded by α-helices. The conserved sequence motifs at the interface of D1 and D2 are responsible for ATP binding and hydrolysis. The following Zn-binding domain stabilizes the protein structure. The C-terminus consists of a WH domain (a.k.a RQC domain), which directly participates in DNA binding and base pair separation. [@Chen-2014]" width="100%" />
<p class="caption">
Figure 1: A homology model of <em>C. elegans</em> wrn-1 based on DrRecQ catalytic core (PDB accession code: 4Q47). The N-terminal part contains the common core structure of the RecQ helicase family, which comprises two RecA-like domains, D1 and D2, each composed of a central β-sheet surrounded by α-helices. The conserved sequence motifs at the interface of D1 and D2 are responsible for ATP binding and hydrolysis. The following Zn-binding domain stabilizes the protein structure. The C-terminus consists of a WH domain (a.k.a RQC domain), which directly participates in DNA binding and base pair separation. <span class="citation">(Chen et al. 2014)</span>
</p>
</div>
<div class="figure"><span id="fig:wrn-b-factor"></span>
<img src="../img/wrn-b-factor.png" alt="The same homology model, showing local QMEAN scores (b-factor). Compared to green and thin regions, red and thick regions have local geometrical properties deviating further from those of experimentally determined structures. The Zn-binding domain and WH domain are more consistent with experimental structures than the two RecA-like domains." width="100%" />
<p class="caption">
Figure 2: The same homology model, showing local QMEAN scores (b-factor). Compared to green and thin regions, red and thick regions have local geometrical properties deviating further from those of experimentally determined structures. The Zn-binding domain and WH domain are more consistent with experimental structures than the two RecA-like domains.
</p>
</div>
<div class="figure"><span id="fig:recq-in-action"></span>
<img src="../img/recq-in-action.png" alt="WRN catalytic core (RECQ) in action. This structure is based on the structure of a dimer of human RECQ1 helicase in complex with tailed duplux DNA solved by @Pike-2015. The monomer on the bottom shows the b-factor (local QMEAN) of the homology model with C. elegans wrn-1, where red indicates high and green indicates low scores. The monomer on the top shows the subunits of WRN catalytic core (RECQ), where D1, D2, Zn-binding, and WH (RQC) domains are coloured blue, red, yellow and green, respectively, similar to Fig. 1. DNA is coloured cyan. Ligands (2 zinc ions, 3 sulfate ions and 2 ethanediol molecules) are not removed. " width="100%" />
<p class="caption">
Figure 3: WRN catalytic core (RECQ) in action. This structure is based on the structure of a dimer of human RECQ1 helicase in complex with tailed duplux DNA solved by <span class="citation">Pike et al. (2015)</span>. The monomer on the bottom shows the b-factor (local QMEAN) of the homology model with C. elegans wrn-1, where red indicates high and green indicates low scores. The monomer on the top shows the subunits of WRN catalytic core (RECQ), where D1, D2, Zn-binding, and WH (RQC) domains are coloured blue, red, yellow and green, respectively, similar to Fig. 1. DNA is coloured cyan. Ligands (2 zinc ions, 3 sulfate ions and 2 ethanediol molecules) are not removed.
</p>
</div>
<div class="figure"><span id="fig:wrn-org-cox"></span>
<img src="../img/wrn-org-cox.png" alt="Domain organisation of human WRN and its homologs in *D. melanogaster* and *C. elegans*. Adapted from @Cox-2013." width="100%" />
<p class="caption">
Figure 4: Domain organisation of human WRN and its homologs in <em>D. melanogaster</em> and <em>C. elegans</em>. Adapted from <span class="citation">Cox and Mason (2013)</span>.
</p>
</div>
<ol start="3" style="list-style-type: decimal">
<li>Find homologues in human, Xenopus and mice and give the protein names (Uniprot should help)
<ul>
<li>Use <a href="https://www.uniprot.org/blast/">UniProt BLAST</a></li>
<li>Query: Q19546 (UniProt ID)</li>
<li>Relevant Results:
<ul>
<li>Werner syndrome ATP-dependent helicase (Homo sapiens) (<a href="https://www.uniprot.org/uniprot/Q14191">Q14191</a>; WRN_HUMAN)</li>
<li>Werner syndrome ATP-dependent helicase homolog (Mus musculus (<a href="https://www.uniprot.org/uniprot/O09053">O09053</a>; WRN_MOUSE)</li>
<li>Werner syndrome ATP-dependent helicase homolog (Xenopus laevis) (<a href="https://www.uniprot.org/uniprot/O93530">O93530</a>; WRN_XENLA)</li>
</ul></li>
</ul></li>
<li>Show homology between your mystery protein and the human, mouse and frog homologues.
<ul>
<li>Sequences aligned using UniProt alignment tool (accessible via the BLAST result page)</li>
<li>results displayed using SnapGene Viewer (<a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/clustal.pdf">view PDF</a>)</li>
<li>UniProt uses the ‘clustalo’ program (MSA algorithm) to align</li>
<li>I also used the ‘muscle’ program (MUltiple Sequence Comparison by Log-Expectation) in the command line to produce a different result, and standardised its output using BioPython for visualisation in SnapGene (<a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/muscle.pdf">view PDF</a>)</li>
</ul>
<pre class="bash"><code>$ cat ce.fasta hs.fasta mm.fasta xl.fasta | muscle -out muscle.fasta</code></pre>
<pre class="python"><code>from Bio import AlignIO
muscle = AlignIO.read(&#39;resources/wrn/muscle.fasta&#39;, &#39;fasta&#39;)
AlignIO.write(muscle, &#39;muscle.aln&#39;, &#39;clustal&#39;)</code></pre>
<p>With help of the <code>seqin</code> R package, the pairwise genetic differences between these 4 protein sequences can be calculated:</p>
<pre class="r"><code>aln &lt;- seqinr::read.alignment(&#39;./resources/wrn/clustal.aln&#39;, &#39;clustal&#39;)
alnDist &lt;- seqinr::dist.alignment(aln)
alnDist</code></pre>
<pre><code>##                     SP|Q19546|WRN_CAEEL SP|Q14191|WRN_HUMAN SP|O09053|WRN_MOUSE
## SP|Q14191|WRN_HUMAN           0.5710006                                        
## SP|O09053|WRN_MOUSE           0.5782777           0.3408993                    
## SP|O93530|WRN_XENLA           0.5903942           0.4396192           0.4499939</code></pre>
<p>These can be visulised by a phylogenetic tree produced by the <code>ape</code> package, as shown in Fig. <a href="#fig:phylo">5</a>.</p>
<pre class="r"><code>mydist &lt;- seqinr::dist.alignment(aln)
mytree &lt;- ape::nj(mydist)
myrootedtree &lt;- root(mytree, &#39;SP|Q19546|WRN_CAEEL&#39;, resolve.root=TRUE)
ape::plot.phylo(myrootedtree, type=&quot;p&quot;)</code></pre>
<div class="figure"><span id="fig:phylo"></span>
<img src="./tutorial/2020-01-23-mcb-toolbox_files/figure-html/phylo-1.png" alt="The phylogenetic relationship between the WRN homologs in 4 organisms" width="100%" />
<p class="caption">
Figure 5: The phylogenetic relationship between the WRN homologs in 4 organisms
</p>
</div></li>
<li>Design PCR primers to amplify the coding region, incorporating suitable restriction enzyme sites in each primer so that you can clone easily into the vector pIVEX2.3d (though there are many online primer design sites, I find this is best done manually to get a feel for it, but these days I use SnapGene (viewer version is free….). Hint: we want to keep the start codon ATG so chose an enzyme that contains ATG in its recognition site for the forward primer.
<ul>
<li>I opened the <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/original.dna">original sequence</a> with SnapGene. In the ‘Sequence’ view, I displayed translated amino acids with ‘Frame +1 only’ and thus located the stop codon (the start codon is obviously +1)<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>. Then I used the ‘Enzymes’ view to obtain a list of noncutters. By comparing this list with the MCS of the <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/vector.dna">piVEX2.3d</a>, I selected <em>Nde</em>I and <em>Sal</em>I as the RE sites to be used<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The <em>Sal</em>I site is not in frame with the C-terminal His tag and stop codon, so I included an additional random nucleotide 3’ to the RE site. The stop codon is excluded from the reverse primer because it is provided in the vector.</li>
<li>forward: <code>NN CAT ATGATAAGTGATGATGACGATCTACC</code>
<ul>
<li><code>CATATG</code> is <em>Nde</em>I restriction site. It ends with <code>ATG</code>, so adding <code>CAT</code> instead of <code>CATATG</code> to the 5’ suffices.</li>
<li>31-mer; 39% GC (&gt;40% if <code>NN</code> = <code>GC</code>); 26 annealed bases; T<sub>m</sub> = 56ºC<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></li>
</ul></li>
<li>reverse: <code>NN GTCGAC N AAGTTTGAATTTCTTCAATGGAGGAGC</code>
<ul>
<li><code>GTCGAC</code> is <em>Sal</em>I restriction site</li>
<li>36-mer; 43% GC; 28 annealed bases; T<sub>m</sub> = 58ºC</li>
</ul></li>
<li><em>In silico</em> PCR is done in SnapGene, and the resulting DNA can be downloaded <a href="https://tianyishi2001.github.io/ox/tutorial/resources/wrn/in-silico-cloning/Amplified1.dna">here</a></li>
</ul></li>
<li><em>In silico</em> clone this into the vector pIVEX2.3d (file attached - .dna is a snap gene format, docx is of course in word). Why might we want to use this vector?
<ul>
<li>This vector has a C-terminal 6-His tag which allows efficient purification by affinity chromatography</li>
<li>The MCS is associated with the T7 promoter. T7 RNAP is highly processive, so consecutive expression is expected.</li>
</ul></li>
<li>Check that your new construct allows correct protein expression in frame with the hexahistidine tag of the vector (SnapGene is good for this, but you can also do it in a few more steps using ApE plus translation apps via Expasy)
<ul>
<li>Yes. (<a href="./resources/wrn/in-silico-cloning/Cloned1.dna.pdf">view result as PDF</a> or <a href="./resources/wrn/in-silico-cloning/Cloned1.dna">.dna file</a>)</li>
<li>Serendipitously, the MCS has an ATG in frame with our insert and the His-tag/STOP codon, so translation will probably start more upstream than we expected, which will add 3 additional amino acid residues to the N-terminus of the product. This ATG was not in frame with the His-tag before insertion, and thus unspotted by SnapGene or me.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></li>
</ul>
<div class="figure"><span id="fig:clone-serendipity"></span>
<img src="../img/clone-serendipity.png" alt="An additional in-frame start codon (red) upstream to the start codon of the insert (blue)." width="100%" />
<p class="caption">
Figure 6: An additional in-frame start codon (red) upstream to the start codon of the insert (blue).
</p>
</div></li>
<li>Predict the protein fold of the new protein (eg Swissprot). Show the effect of predicted enzyme active site mutations on the structure (eg new file in swissport, but Pymol is good for this).
<ul>
<li>Swiss-Model shown in Q2.</li>
</ul></li>
</ol>
<div id="bonus" class="section level3">
<h3>Bonus: Tianyi’s Restriction Site Finder</h3>
<p>See Fig. <a href="#fig:tianyi-re-001-1">7</a> and <a href="#fig:tianyi-re-001-2">8</a>.</p>
<p>This tool takes in two sequences (plain text or FASTA or SnapGene <code>.dna</code> file), one representing the cloning site and the other representing the GOI, and finds the restriction sites inside the cloning site but not in the GOI, and label them with their positions in the cloning site and their commecial suppliers.</p>
<p>The web interface is built upon the <a href="https://www.djangoproject.com">Django</a> framework and the data come from the <code>Restriction</code> module of <a href="https://biopython.org">BioPython</a>, which is (sort of) a wrapper around NEB’s <a href="http://rebase.neb.com/rebase/rebase.html">REBASE</a>.</p>
<div class="figure"><span id="fig:tianyi-re-001-1"></span>
<img src="../img/tianyi-re-001-1.png" alt="Restriction site finder by Tianyi. Query page." width="100%" />
<p class="caption">
Figure 7: Restriction site finder by Tianyi. Query page.
</p>
</div>
<div class="figure"><span id="fig:tianyi-re-001-2"></span>
<img src="../img/tianyi-re--001-2.png" alt="Restriction site finder by Tianyi. Result page." width="100%" />
<p class="caption">
Figure 8: Restriction site finder by Tianyi. Result page.
</p>
</div>
</div>
<div id="references" class="section level2 unnumbered">
<h2>References</h2>
<div id="refs" class="references">
<div id="ref-Chen-2014">
<p>Chen, Sheng-Chia, Chi-Hung Huang, Chia Shin Yang, Tzong-Der Way, Ming-Chung Chang, and Yeh Chen. 2014. “Crystal Structure of Deinococcus Radiodurans Recq Helicase Catalytic Core Domain: The Interdomain Flexibility.” <em>BioMed Research International</em> 2014. Hindawi.</p>
</div>
<div id="ref-Cox-2013">
<p>Cox, Lynne S., and Penelope A. Mason. 2013. “The Role of Wrn Helicase/Exonuclease in Dna Replication.” In <em>The Mechanisms of Dna Replication</em>, edited by David Stuart. Rijeka: IntechOpen. <a href="https://doi.org/10.5772/51520">https://doi.org/10.5772/51520</a>.</p>
</div>
<div id="ref-Pike-2015">
<p>Pike, Ashley C. W., Shivasankari Gomathinayagam, Paolo Swuec, Matteo Berti, Ying Zhang, Christina Schnecke, Francesca Marino, et al. 2015. “Human Recq1 Helicase-Driven Dna Unwinding, Annealing, and Branch Migration: Insights from Dna Complex Structures.” <em>Proceedings of the National Academy of Sciences</em> 112 (14). National Academy of Sciences: 4286–91. <a href="https://doi.org/10.1073/pnas.1417594112">https://doi.org/10.1073/pnas.1417594112</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>Why does this sequence start with ATG but not end with a stop codon? Are the additional downstream nucleotides for the convinience of primer design (is it worthwhile to include additional upstream nucleotides)?<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>Is there a more automatic way of choosing RE sites? I wish I could select the MCS sequence of the vector and the GOI sequence, and then the intersection between the cutting sites of the former and the noncutting sites of the latter is computed automatically. Is this functionality hidden somewhere in SnapGene, or is it available elsewhere? If not I’ll try to make it! (A prototype is shown in Fig. <a href="#fig:tianyi-re-001-1">7</a> and <a href="#fig:tianyi-re-001-2">8</a>)<a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>When calculating %GC and T<sub>m</sub>, do we need to include the 5’ additional bases used for RE cutting/frame adjustment? In SnapGene it seems that these additional bases are included in the calculation of %GC but not T<sub>m</sub>. In practice, isn’t the annealing region the complete length of the primers for most DNA strands after a few rounds of synthesis? Do we need to increase the annealing temperature then?<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>It would be nice if it can highlight all ATGs regardless of the frame. I think additional in-frame ATGs can cause inclusion of unwanted aa residues and addtional out-of-frame ATGs can ruin the expression of GOI, especially in eukaryotes, because their ribosomes start translation at the first encountered ATG. It this correct?<a href="#fnref4" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/vectors-used-in-bacterial-and-eukaryotic-host-cells.html" class="archive-item-link">
                Vectors Used in Bacterial and Eukaryotic Host Cells 
              </a>
              <span class="archive-item-date">
                2020-01-20
              </span>
              <div class="article-summary" style="display: none;">
                
                  What are the key similarities and differences between vectors used in bacterial and eukaryotic host cells? What can such vectors be used for?<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#the-structure-of-the-vectors"><span class="toc-section-number">1</span> The structure of the vectors</a><ul>
<li><a href="#simple-plasmids"><span class="toc-section-number">1.1</span> Simple plasmids</a></li>
<li><a href="#others"><span class="toc-section-number">1.2</span> Other bacterial vectors</a></li>
<li><a href="#other-eukaryotic-vectors"><span class="toc-section-number">1.3</span> Other Eukaryotic Vectors</a></li>
</ul></li>
<li><a href="#vector-construction"><span class="toc-section-number">2</span> Vector Construction</a><ul>
<li><a href="#simple-plasmids-for-both-bacteria-and-eukaryotes"><span class="toc-section-number">2.1</span> Simple plasmids (for both bacteria and eukaryotes)</a></li>
<li><a href="#other-vectors-for-bacteria"><span class="toc-section-number">2.2</span> Other vectors for bacteria</a></li>
<li><a href="#viral-vectors-for-eukaryotes"><span class="toc-section-number">2.3</span> Viral vectors for eukaryotes</a></li>
</ul></li>
<li><a href="#transformationtransfection"><span class="toc-section-number">3</span> Transformation/Transfection</a><ul>
<li><a href="#transformation-of-bacterial-cells"><span class="toc-section-number">3.1</span> Transformation of bacterial cells</a><ul>
<li><a href="#heat-shock-transformation"><span class="toc-section-number">3.1.1</span> Heat-shock transformation</a></li>
<li><a href="#electroporation"><span class="toc-section-number">3.1.2</span> Electroporation</a></li>
<li><a href="#other-methods"><span class="toc-section-number">3.1.3</span> Other methods</a></li>
</ul></li>
<li><a href="#transfection-of-eukaryotic-cells"><span class="toc-section-number">3.2</span> Transfection of Eukaryotic Cells</a><ul>
<li><a href="#physical-methods"><span class="toc-section-number">3.2.1</span> Physical methods</a></li>
<li><a href="#chemical-methods"><span class="toc-section-number">3.2.2</span> Chemical methods</a></li>
<li><a href="#stab-tf"><span class="toc-section-number">3.2.3</span> Biological methods</a></li>
</ul></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<p><strong>What are the key similarities and differences between vectors used in bacterial and eukaryotic host cells? What can such vectors be used for?</strong></p>
<p>There are various ways by which genes can be introduced into a host organism and expressed. The choice of source genetic material, vector, and the method of transformation/transfection for a particular experiment are largely dependent on the physiology of the host organism, the size of the transgene, and the purpose of the study. However, it is useful to catagorise these strategies into bacteria-compatible ones and eukaryote-compatible ones.</p>
<p>Most molecular cloning experiments begin with the production of a recombinant plasmid containing the gene of interest (GOI). For bacterial hosts, the recombinant plasmid is directly introduced into the cells. Bacteria naturally possess the machinary for replicating plasmids, which allows inheritance of the GOI in their progenies. Eukaryotic cells cannot reliably replicate plasmids, therefore introducing plasmids into eukaryotic cells normally results in <em>transient transfection</em>, which means the GOI is expressed in successfully tranfected cells but not in their progenies. In order to produce a <em>stable cell line</em>, the GOI must be integrated into the genome of the host cell (by homologous recombination), which is an extremely rare event in nature but can be induced artificially, for example by viral/bacterial infection or by designer nucleases such as TALEN, ZFN and CRISPR-Cas9. In order to be compatible with the different transformation/tranfection strategies and the physiology of the host cells, the molecular structure of the plasmids (or vectors of other kinds) vary.</p>
<div id="the-structure-of-the-vectors" class="section level1">
<h1><span class="header-section-number">1</span> The structure of the vectors</h1>
<div id="simple-plasmids" class="section level2">
<h2><span class="header-section-number">1.1</span> Simple plasmids</h2>
<p>Plasmids are extrachromosomal self-replicating cytoplasmic (usually circular) DNA elements found in prokaryotes and, less commonly, in eukaryotes. They come in various forms, from simple plasmids used for direct transformation and fosmids for phage transduction. Plasmids optimised for expressing recombinant proteins share several features, as illustrated in Fig. <a href="#fig:pet">1.1</a> and <a href="#fig:pcmv-gfp-lc3b">1.2</a>:</p>
<ul>
<li>An origin of replication (<em>ori</em>), which allows it to be duplicated in bacterial cells. The type of origin of replication affects the copy number. Eukaryotic vectors generally carry two <em>ori</em>s, one bacterial (e.g. pUC <em>ori</em>) and one viral (e.g. SV40 <em>ori</em>). The bacterial <em>ori</em> is for amplifying the plasmid in bacteria before transfection, and the viral <em>ori</em> allows for episomal amplification of plasmids in the eukaryotic host.<br />
</li>
<li>A cloning site where the GOI is inserted. Typically it is crowded with restriction enzyme cutting sites (compatible with RE/ligase dependent cloning), but it can also be made compatible with other cloning methods, such as the LIC-compatible pBLIC-puro<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> plasmid developed by <span class="citation">Patel et al. (2012)</span>.</li>
<li>A promoter and associated operator/enhancer. The promoter sequences for bacteria and eukaryotes are different (-35/-10 consensus sequence v.s. TATA box) For a bacterial vector, the promoter is usually controlled by an inducible operator so that protein expression is induced manually only when the optimal bacterial density is reached (e.g. <em>lac</em> operator induced by IPTG). In contrast, for a eukaryotic vector, the promoter is usually made as processive as possible, because the natural rate of transcription is generally low, and cell growth and division are not important (they do not serve to amplify the plasmid but to express the protein). For example, the pEF-BOS vector developed by <span class="citation">Mizushima and Nagata (1990)</span> contains the promoter of EF-1<span class="math inline">\(\alpha\)</span> (EF-1<span class="math inline">\(\alpha\)</span> is one of the most abundant proteins in eukaryotic cells, so its promoter is highly processive). Recently, promoters of viral origin (e.g. CMV, SV40) are gaining popularity.</li>
<li>A transcription terminator/polyA signal. The mechanisms of transcription termination in bacteria and eukaryotes differ. In bacteria, the terminator sequence consists of a symmetric DNA sequence, and its transcribed product folds into a hairpin structure, releasing RNA polymerase (RNAP) from DNA. In eukaryotes, CstF and CPSF bound on RNAP C-terminal domain recognise 3’-end processing (polyadenylation) sequence (AAUAAA and GU-rich sequence) and induce cleavage and polyadenylation.</li>
<li>A selectable marker. Usually an antibiotic resistance gene (bacteria and eukaryotes are sensitive to different antibiotics). Fluorophore genes are also common in eukaryotic vectors.</li>
<li>Some plasmids possess a start codon and associated Shine-Dalgarno sequence (for bacteria) or Kozak sequence (for eukaryotes).</li>
<li>Some plasmids have tag sequences near the start or end of the ORF for easy purification of the recombinant protein by affinity chromatography (e.g. hex-His, GST, HSV) and associated cleavage sequence (e.g. TEV) for easy removal of the tag.</li>
</ul>
<div class="figure"><span id="fig:pet"></span>
<img src="../img/pETBlue.jpg" alt="The bacterial expression vector pETBlue-2" width="100%" />
<p class="caption">
Figure 1.1: The bacterial expression vector pETBlue-2
</p>
</div>
<div class="figure"><span id="fig:pcmv-gfp-lc3b"></span>
<img src="../img/pcmv-gfp-lc3b.jpg" alt="The eukaryotic expression vector pCMV-GFP-LC3" width="100%" />
<p class="caption">
Figure 1.2: The eukaryotic expression vector pCMV-GFP-LC3
</p>
</div>
</div>
<div id="others" class="section level2">
<h2><span class="header-section-number">1.2</span> Other bacterial vectors</h2>
<p>There are other forms of circular DNA, each with different disirable properties (larger insertion size, low copy number, phage-compatibility, etc.). They are sometimes called ‘high capacity vectors’ because their insertion sizes are greater than simple plasmids.</p>
<p>Bacterial artificial chromosomes (BACs) contain some regions derived from a special plasmid called the F (fertility) factor: the region containing the origin of replication as well as genes that ensured its precise segregation during bacterial cell division. A great advantage of BAC vectors is the large insertion size (100-200 kb). But the very large insertion can be a problem, in that it cannot be manipulated by restriction enzymes. Instead, in vivo homologous recombination-based strategies are used to insert DNA fragments into BACs. (Fig. <a href="#fig:bac-mech">1.3</a>)</p>
<div class="figure"><span id="fig:bac-mech"></span>
<img src="../img/bac-mech.png" alt="Making a BAC" width="50%" />
<p class="caption">
Figure 1.3: Making a BAC
</p>
</div>
<p>Fosmids are hybrids of <span class="math inline">\(\lambda\)</span> phage DNA and bacterial F plasmid DNA. Fosmids are packaged into <span class="math inline">\(\lambda\)</span> phage particles, which delivers them into bacterial cells. Due to the presence of the F plasmid origins of replications, fosmids are maintained at a very low copy number (normally single copy).</p>
<p>Fig. <a href="#fig:plasmid-derivatives">1.4</a> is a schematic of 6 high capacity vectors <span class="citation">(Saraswathy and Ramalingam 2011)</span>.</p>
<div class="figure"><span id="fig:plasmid-derivatives"></span>
<img src="../img/plasmid-derivatives.png" alt="Schematic of 6 high capacity vectors." width="50%" />
<p class="caption">
Figure 1.4: Schematic of 6 high capacity vectors.
</p>
</div>
</div>
<div id="other-eukaryotic-vectors" class="section level2">
<h2><span class="header-section-number">1.3</span> Other Eukaryotic Vectors</h2>
<p>Yeast artificial chromosome (YAC) has a cloning capacity up to 3000 kbp. It is introduced into the yeast cells by electroporation, then it is maintained as a linear DNA-like chromosome. It is replicated along with other chromosomes in yeast and its copy number of one is maintained after cell divition.</p>
<p>Some viruses (e.g. Adenovirus, Lentivirus and Baculovirus) and bacteria (e.g. <em>Agrobacterium tumefaciens</em>) are reliable vectors for stable transfection of eukaryotic cells (see section <a href="#stab-tf">3.2.3</a>)</p>
</div>
</div>
<div id="vector-construction" class="section level1">
<h1><span class="header-section-number">2</span> Vector Construction</h1>
<div id="simple-plasmids-for-both-bacteria-and-eukaryotes" class="section level2">
<h2><span class="header-section-number">2.1</span> Simple plasmids (for both bacteria and eukaryotes)</h2>
<p>Traditionally, restriction endonucleases and T4 ligase are used to insert GOI into pladmids. This method has several drawbacks:</p>
<ul>
<li>the association between the short 2-4 nt overhangs (sticky ends) is weak</li>
<li>each DNA insert has to be inspected for any internal restriction sites</li>
<li>due to the above two reasons, it has very limited capability of constructing multi-fragment plasmids</li>
<li>incomplete DNA digestion and poor ligation yields</li>
<li>unwanted amino acids can be introduced to the expressed protein</li>
<li>the GOI can only be cloned in the vector position where the selected restriction site is present</li>
</ul>
<p>Ligation-independent cloning (LIC) overcomes many of the problems described above. In a T4 DNA polymerase-dependent approach, the sticky sequences are made by the 3’-to-5’ exonuclease activity of T4 polymerase. These sequences are long, allowing formation of stable recombinant plasmid without <em>in vitro</em> ligase treatment. The nicks in sugar-phosphate backbone are later fixed by the host’s ligase. As T4 polymerase (exonuclease activity) always proceeds from 3’ ends, any internal sticky sequences will not be disrupting and thus the custom stiky sequence can be used for any DNA inserts.</p>
<p>Fig. <a href="#fig:lic">2.1</a> compares the mechanisms of the traditional RE/ligase-based cloning and the T4 polymerase-dependent LIC approach.</p>
<div class="figure"><span id="fig:lic"></span>
<img src="../img/lc-lic.jpg" alt="Left, traditional cloning based on restriction endonuclease and ligase; right, ligase-independent cloning based on T4 polymerase" width="100%" />
<p class="caption">
Figure 2.1: Left, traditional cloning based on restriction endonuclease and ligase; right, ligase-independent cloning based on T4 polymerase
</p>
</div>
<p>There are also other alternatives to the traditional method:</p>
<ul>
<li>the Gateway cloning system exploits the site-specific recombination system utilized by bacteriophage lambda to shuttle sequences between plasmids bearing flanking compatible recombination attachment (att) sites</li>
<li>Gibson assembly is a molecular cloning method which allows for ‘stitching’ multiple overlapping DNA fragments together in a single, isothermal reaction, using exonuclease, polymerase, and ligase.</li>
<li>TOPO cloning expoits Taq polymerase’s feature that adds non-specific A to the 3’ end, and uses topoisomerase I to hold vectors open and to promote ligation</li>
</ul>
</div>
<div id="other-vectors-for-bacteria" class="section level2">
<h2><span class="header-section-number">2.2</span> Other vectors for bacteria</h2>
<p>BACs and fosmids: see section <a href="#others">1.2</a></p>
</div>
<div id="viral-vectors-for-eukaryotes" class="section level2">
<h2><span class="header-section-number">2.3</span> Viral vectors for eukaryotes</h2>
<p>See section <a href="#lenti">3.2.3.1</a></p>
</div>
</div>
<div id="transformationtransfection" class="section level1">
<h1><span class="header-section-number">3</span> Transformation/Transfection</h1>
<div id="transformation-of-bacterial-cells" class="section level2">
<h2><span class="header-section-number">3.1</span> Transformation of bacterial cells</h2>
<div id="heat-shock-transformation" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Heat-shock transformation</h3>
<p>Heat-shock transformation is used for small vectors such as plasmids. The general procedure is as follows:</p>
<ol style="list-style-type: decimal">
<li>Host cells are incubated in a solution containing divalent cations (typically CaCl<sub>2</sub>) on ice.</li>
<li>CaCl<sub>2</sub> partially disrupts the cell membrane, which allows the recombinant DNA enter the host cell. Such cells are called competent cells.</li>
<li>Cells are exposed to a heat pulse (heat shock), and the thermal imbalance causes the entry of DNA through disrupted plasma membrane.</li>
</ol>
</div>
<div id="electroporation" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Electroporation</h3>
<p>Electroporation can be used for larger vectors such as BAC and PAC<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The general procedure is as follows:</p>
<ol style="list-style-type: decimal">
<li>Host cells are placed into a cuvette, together with the vector. The cuvette is connect to electrodes.</li>
<li>The cuvette containing the mixture is subjected to intense electric pulses (2500 V/cm for bacteria, lower for animal and plant cells) each lasting for only a few milliseconds</li>
<li>Most cells would die under such treatment, but for those survived, their membranes are polarised by the electric field and are disrupted, and DNA enters through the pores. Finally, the membrane reseals after the treatment.</li>
</ol>
</div>
<div id="other-methods" class="section level3">
<h3><span class="header-section-number">3.1.3</span> Other methods</h3>
<p>Cosmid, fosmid and PAC vectors are introduced into bacterial cells via bacteriophages. (See section <a href="#others">1.2</a>)</p>
</div>
</div>
<div id="transfection-of-eukaryotic-cells" class="section level2">
<h2><span class="header-section-number">3.2</span> Transfection of Eukaryotic Cells</h2>
<p>Transfection of eukaryotic cells can be transient or stable. Transient gene expression is generally used in academic settings, when the product of the GOI (usually a protein) is used for short-term research purposes. Stable transfection is usually used in indusdries, but it is also used in academia for specific purposes.</p>
<div id="physical-methods" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Physical methods</h3>
<p>In general, physical methods work by briefly pushing holes through the memrbane. The three specific methods are:</p>
<ul>
<li>electroporation, which is described above as a transformation method for bacteria, can also be applied to eukaryotic cells. The mechanism is the same but the voltage used for eukaryotic cells are lower.</li>
<li>microinjection entails injection of recombinant DNA (or RNA) into the cytoplasm or nucleus. This is method particularly convinient for large eukaryotic cells, such as the syncytial gonadal cell of <em>Caenorhabditis elegans</em> and the fertilised egg of <em>Mus musculus</em>.</li>
<li>biolistic delivery of tungsten- or gold-coated DNA (or RNA) by gene gun is mostly used with plant cells.</li>
</ul>
</div>
<div id="chemical-methods" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Chemical methods</h3>
<p>In general, chemical methods work by sheding the negative charge on DNA so as to apprach and cross the plasma membrane, which is also negatively charged.</p>
<ul>
<li>CaPO<sub>4</sub> co-precipitation involves mixing DNA with CaCl<sub>2</sub> in a phosphate solution to generate a CaPO<sub>4</sub>-DNA co-precipitate, which is then added to cultured cells. DNA enters the cell by endocytosis.</li>
<li>the cationic diethylaminoethyl(DEAE)-dextran work similarly by associating and neutralising negatively charged DNA</li>
<li>cationic lipids work similarly by forming DNA-lipid complexes (DNA is not encapsulated in liposomes)</li>
<li>By far, the most cost-effective chemical carrier is polyethylenimine (PEI), which works similarly by forming DNA-PEI complexes <span class="citation">(L’Abbé et al. 2018)</span></li>
</ul>
</div>
<div id="stab-tf" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Biological methods</h3>
<p>The physical and chemical methods described above are generally used for transient transfection. Although they may produce stably transfected cells (when the rare homologous recombination events occur), the probability is very low and the GOI is inserted randomly and is thus not expressed with greatest efficiency (the chances are low that the GOI is inserted into a highly ‘transcriptionally open’ region).</p>
<p>Biological methods significantly improve the chance of integration of GOI into the host’s genome, and can even direct the site of insertion.</p>
<div id="lenti" class="section level4">
<h4><span class="header-section-number">3.2.3.1</span> Lentiviral transduction as an example of viral transfection <span class="citation">(Merten, Hebben, and Bovolenta 2016)</span></h4>
<p>Lentiviruses (LV) carry a genome made of (+) strand RNA. Upon infection, the viral RNA genome is reverse-transcribed and the cDNA is stably integrated into the host’s genome. Unlike <span class="math inline">\(\gamma\)</span>-retroviruses, LVs exploit active nuclear transport and thus do not rely on cell division to access the nucleus. Fig. <a href="#fig:lentivirus-3-gen">3.1</a> is an example of the third generation HIV-1 vector system.</p>
<div class="figure"><span id="fig:lentivirus-3-gen"></span>
<img src="../img/lentivirus-3-gen.jpg" alt="Production of recombinant LV particles and their infection. Production of recombinant LV particles is usually accomplished by transfection of HEK293T cells with a set of three helper plasmids and additionally a LV vector plasmid that encodes the GOI as well as optional reporter (e.g resistance, fluorophore) genes. LV particles released from HEK helper cells infect target cells and integrate the GOI into their genome." width="100%" />
<p class="caption">
Figure 3.1: Production of recombinant LV particles and their infection. Production of recombinant LV particles is usually accomplished by transfection of HEK293T cells with a set of three helper plasmids and additionally a LV vector plasmid that encodes the GOI as well as optional reporter (e.g resistance, fluorophore) genes. LV particles released from HEK helper cells infect target cells and integrate the GOI into their genome.
</p>
</div>
</div>
<div id="ti-plasmid-and-agrobacterium-tumefaciens-infection" class="section level4">
<h4><span class="header-section-number">3.2.3.2</span> Ti plasmid and <em>Agrobacterium tumefaciens</em> infection</h4>
<p><em>Agrobacterium tumefaciens</em> is the bacterium that causes crown gall disease in plants. When the it infects a plant cell, a part of its Ti plasmid, called T-DNA, is transferred and inserted (at random) into the genome of the plant cell. Thus, the T-DNA can be replaced with any GOI for stable transfection of plant cells.</p>
</div>
<div id="crispr-cas9-transfection" class="section level4">
<h4><span class="header-section-number">3.2.3.3</span> CRISPR-Cas9 Transfection</h4>
<p>The CRISPR-Cas9 system comprises a short guide RNA (gRNA) and the Cas9 nuclease. The gRNA guides Cas9 to a specific gene locus that is complementary to its crRNA portion, then Cas9 makes a double strand break, which is dependent on the PAM (protospacer adjacent motif) sequence immediately downstream of the recognition sequence.</p>
<p>Following DNA cleavage, the break is repaired by either non-homologous end joining (NHEJ) or homology directed repair (HDR). It is the HDR mechanism that may cause integration of GOI into the cleavage site.</p>
<div class="figure"><span id="fig:crispr"></span>
<img src="../img/crispr.jpg" alt="gRNA directed DNA cleavage by Cas9" width="100%" />
<p class="caption">
Figure 3.2: gRNA directed DNA cleavage by Cas9
</p>
</div>
</div>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Alberts-2014">
<p>Alberts, Bruce, Alexander Johnson, Julian Lewis, David Morgan, Martin Raff, Keith Roberts, and Peter Walter. 2014. <em>Molecular Biology of the Cell</em>. Book. 6th ed. Garland Science.</p>
</div>
<div id="ref-Balbas-2012">
<p>Balbás, Paulina, and Argelia Lorence. 2012. <em>Recombinant Gene Expression: Reviews and Protocols</em>. Book. 2nd ed. Vol. 267. Methods in Molecular Biology. Springer.</p>
</div>
<div id="ref-Griffiths-2015a">
<p>Griffiths, Anthony J. F., Susan R. Wessler, Sean B. Carroll, and John Doebley. 2015. “Gene Isolation and Manipulation.” In <em>Introduction to Genetic Analysis</em>, 11th ed. W.H. Freeman.</p>
</div>
<div id="ref-Hacker-2018">
<p>Hacker, David L. 2018. <em>Recombinant Protein Expression in Mammalian Cells: Methods and Protocols</em>. Book. Vol. 1850. Methods in Molecularbiology. Humana Press. <a href="https://doi.org/10.1007/978-1-4939-8730-6">https://doi.org/10.1007/978-1-4939-8730-6</a>.</p>
</div>
<div id="ref-LAbbe-2018">
<p>L’Abbé, Denis, Louis Bisson, Christian Gervais, Eric Grazzini, and Yves Durocher. 2018. “Transient Gene Expression in Suspension Hek293-Ebna1 Cells.” In <em>Recombinant Protein Expression in Mammalian Cells: Methods and Protocols</em>, edited by David L. Hacker. Vol. 1850. Humana Press.</p>
</div>
<div id="ref-Merten-2016">
<p>Merten, Otto-Wilhelm, Matthias Hebben, and Chiara Bovolenta. 2016. “Production of Lentiviral Vectors.” <em>Molecular Therapy - Methods &amp; Clinical Development</em> 3: 16017. <a href="https://doi.org/https://doi.org/10.1038/mtm.2016.17">https://doi.org/https://doi.org/10.1038/mtm.2016.17</a>.</p>
</div>
<div id="ref-Mizushima-1990">
<p>Mizushima, S, and S Nagata. 1990. “PEF-Bos, a Powerful Mammalian Expression Vector.” <em>Nucleic Acids Res</em> 18 (17). Osaka Bioscience Institute, Japan.: 5322. <a href="https://doi.org/10.1093/nar/18.17.5322">https://doi.org/10.1093/nar/18.17.5322</a>.</p>
</div>
<div id="ref-Patel-2012">
<p>Patel, Asmita, Anisleidys Muñoz, Katherine Halvorsen, and Priyamvada Rai. 2012. “Creation and Validation of a Ligation-Independent Cloning (Lic) Retroviral Vector for Stable Gene Transduction in Mammalian Cells.” <em>BMC Biotechnology</em> 12 (1): 3. <a href="https://doi.org/10.1186/1472-6750-12-3">https://doi.org/10.1186/1472-6750-12-3</a>.</p>
</div>
<div id="ref-Saraswathy-2011a">
<p>Saraswathy, Nachimuthu, and Ponnusamy Ramalingam. 2011. “High Capacity Vectors.” In <em>Concepts and Techniques in Genomics and Proteomics</em>, 49–56. Woodhead Publishing. <a href="https://doi.org/https://doi.org/10.1533/9781908818058.49">https://doi.org/https://doi.org/10.1533/9781908818058.49</a>.</p>
</div>
<div id="ref-Singha-2017">
<p>Singha, Tapan Kumar, Pooja Gulati, Aparajita Mohanty, Yogender Pal Khasa, Rajeev Kumar Kapoor, and Sanjay Kumar. 2017. “Efficient Genetic Approaches for Improvement of Plasmid Based Expression of Recombinant Protein in Escherichia Coli: A Review.” Journal Article. <em>Process Biochemistry</em> 55: 17–31. <a href="https://doi.org/https://doi.org/10.1016/j.procbio.2017.01.026">https://doi.org/https://doi.org/10.1016/j.procbio.2017.01.026</a>.</p>
</div>
<div id="ref-Valla-2014">
<p>Valla, Svein, and Rahmi Lale. 2014. <em>DNA Cloning and Assembly Methods</em>. Book. Vol. 1116. Methods in Molecular Biology. Humana Press. <a href="https://doi.org/10.1007/978-1-62703-764-8">https://doi.org/10.1007/978-1-62703-764-8</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>pBLIC-puro was a gift from Priyamvada Rai (Addgene plasmid # 45197 ; <a href="http://n2t.net/addgene:45197" class="uri">http://n2t.net/addgene:45197</a> ; RRID:Addgene_45197)<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>P1-derived artificial chromosome<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
      
      <h2 class="archive-title">2019</h2>
          
            <article class="archive-item">
              <a href="./tutorial/antibiotics-and-protein-synthesis.html" class="archive-item-link">
                Antibiotics and Protein Synthesis 
              </a>
              <span class="archive-item-date">
                2019-11-30
              </span>
              <div class="article-summary" style="display: none;">
                
                  Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.<hr>
                
                


<p><strong>Explain how antibiotics and genetic mutations have helped in our understanding of the mechanism of protein synthesis on the ribosome.</strong></p>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<p>Protein synthesis occurs in ribosomes and involves 3 phases: initiation, elongation (which is subdivided into 4 steps) and termination. Each phase depends on the correct conformation and/or catalytic activity of different domains of the ribosomal complex and soluble protein factors (IFs, EFs and RFs). The identities and mechanisms of these critical RNA/protein domains (in bacterial cells) can be investigated by genetic mutation and antibiotics, which hinder their normal functions, along with recent advances in biophysical methods, which allows high resolution structures to be determined for ribosomes in different contexts.</p>
</div>
<div id="background-information" class="section level1">
<h1>Background information</h1>
<div id="the-ribosome" class="section level2">
<h2>The Ribosome</h2>
<p>The bacterial 70S ribosome comprises two subunits, the 50S (large) and the 30S (small) subunit. The 30S subunit mediates selection of cognate aminoacyl tRNAs and the 50S subunit contains the peptidyl-transferase centre (PTC), which catalyses addition of new amino acids to the elongating polypeptide chain.</p>
</div>
<div id="the-process-of-protein-synthesis" class="section level2">
<h2>The process of protein synthesis</h2>
<p>The 3 phases of protein synthesis are initiation, elongation and termination.</p>
<p>During initiation, the 3’ end of the 16S rRNA of the 30S subunit base-pairs with the Shine-Dalgarno sequence, which positions the start codon (usually AUG) in the P site for the binding of the initiator tRNA (usually fMet-tRNA) and IF1, 2, 3. Joining of the 50S subunit and dissociations of IFs primes the elongation phase.</p>
<p>The elongation phase, where new amino acids are appended to the growing polypeptide, involves 4 steps:</p>
<ol style="list-style-type: decimal">
<li><strong>Decoding.</strong> An aminoacylated tRNA (aa-tRNA) is delivered to the A-site of the ribosome by elongation factor EF-Tu complexed with GTP. GTP hydrolysis facilitates discrimination between cognate and non-cognate tRNAs.</li>
<li><strong>Peptidyl transfer.</strong> The peptidyl-transferase centre (PTC) on the 50S subunit catalyses the formation of peptide bond between the amino acids attached to the tRNAs in the A- and P-sites by transferring the polypeptide chain from the P-site tRNA to the aa-tRNA in the A-site.</li>
<li><strong>Translocation.</strong> EF-G catalyses the movement tRNAs from the A and P sites to the P and E sites</li>
<li>the growing polypeptide chain passes through an exit tunnel</li>
</ol>
<p>Finally, when the stop codon (UAA/UAG/UGA) is encountered, it is recognised by release factors (RF1/RF2) that forces hydrolysis of the peptidyl-tRNA bond in the P-site, thus releasing the polypeptide chain from the ribosome.</p>
</div>
</div>
<div id="mutations" class="section level1">
<h1>Mutations</h1>
<p>Mutation in different components of protein synthesis (tRNA, ribosomal proteins, rRNAs) can all lead to decreased fidelity of this process.</p>
<p>The tRNA needs to undergo a structual transition that involves a 30<span class="math inline">\(^\circ\)</span> bend in order to achieve the A/T (decoding) conformation. Mutagenesis studies showed that two mutations in the D-stem, A9C and G24A, promote miscoding and these two bases are critical for distortion of the cognate tRNA that is required for decoding.</p>
<p>Mutagenesis studies also help to identify ribosomal proteins that are critical in the decoding process. The mutations can be classified into two types according to their implication on the speed and accuracy/robustness of translation: <em>ram</em> (ribosome ambiguity) mutations improve speed but has a higher frequency of failure in rejecting non-cognate tRNAs; in contrast, ribosomes with <em>str</em> (stringency) mutations are slower and more accurate, but they reject some cognate aa-tRNAs and thus waste GTP. Most of such mutations are found in the ribosomal proteins S12, S5 and S4, which are close to the decoding site, and these proteins have an important effect on codon/anticodon interactions.</p>
</div>
<div id="antibiotics" class="section level1">
<h1>Antibiotics</h1>
<p>The antibiotic puromycin can be used to demonstrate the existence of A and P sites in the ribosome and the ATP/GTP-independent nature of the peptidyl transfer reaction.</p>
<p>Puromycin (PM) is an aa-tRNA analogue which is composed of an nucleoside and an bound amino acid. Unlike the normal aa-tRNA where the 5’ OH of the nucleoside (adenosine) is linked to the rest of the tRNA, PM has a free 5’ OH. Therefore, PM can fit into the peptidyl transfer centre (PTC) and accept the polypeptide chain in the peptidyl tranfer reaction, but the new peptide with PM attached is released immediately because PM does not have an attached tRNA to anchor it to the ribosome.</p>
<p>The details of the experiment is shown below and in Figure <a href="#fig:puromycin">1</a>:</p>
<ul>
<li>Take a cell free system containing: ribosomes, Mg<sup>2+</sup>-containing buffer, poly-U mRNA, <sup>14</sup>C-Phe-tRNA, ATP/GTP-regenerating system and add varying concentrations of soluble cell extract.</li>
<li>This yields ribosomes carrying polyPhe-tRNA (not free polyPhe or poly-Phe-tRNA unbound to ribosomes). [with prior knowledge, this is because there are no stop codons on the mRNA and hence RFs are not recruited to terminate translation]</li>
<li>the ribosomes are washed to remove soluble proteins and ATP/GTP.</li>
<li>then they are incubated with puromycin (PM)</li>
<li>about 50% of the ribosomes release <sup>14</sup> labelled polyPhe as polyPhe-PM [As shown in Figure <a href="#fig:puromycin">1</a>, the convertion from state 2 to 3 and from state 3 to 1 requires GTP, and the convertion from state 1 to 2 (peptidyl transfer) does not. Therefore, when GTP is removed, protein synthesis is either trapped in state 2 or 3]</li>
<li>without a tRNA, PM cannot bind to the ribosome (at A/P sites) so the compound with polyPhe is released</li>
</ul>
<p>Conclusion: ribosome itself catalyses the formation of the peptide bond without energy supplying molecules (ATP/GTP).</p>
<ul>
<li>now wash the ribosomes to remove the PM</li>
<li>add soluble cell extract and GTP, then more PM</li>
<li>the remaining 50% ribosomes release polyPhe-PM</li>
</ul>
<p>Conclusion:</p>
<ol style="list-style-type: decimal">
<li>ribosomes have a ‘PM-reactive’ site (P site) and a ‘PM-unreactive site’ (A site)</li>
<li>soluble cell extract can translocate from RM-unreactive to PM-reactive site (driven by EF-G)</li>
</ol>
<div class="figure"><span id="fig:puromycin"></span>
<img src="../img/puromycin.jpg" alt="The puromycin experiment." width="100%" />
<p class="caption">
Figure 1: The puromycin experiment.
</p>
</div>
</div>
<div id="summary" class="section level1">
<h1>Summary</h1>
<p>Mutagenesis and ribosome-targeting antibiotics are valuable tools for studying the mechanisms of protein synthesis in the ribosome. These methods are usually aided by cryo-EM and X-ray crystallography to probe into the 3D molecular details.</p>
<p>Mutations in tRNA, rRNA and associated proteins give information on their specific roles and behaviours during protein synthesis. Antibiotics with various (and usually predictable) chemical properties is another way of studying the mechanisms of protein synthesis. Apart from the puromycin (which targets the peptidyl transfer centre) mentioned in this essay, there is a wide range of antibiotics that target different steps of protein synthesis. For example, streptomycins interferes with the delivery of tRNAs to the A-site and neomycin hinders translocation following peptidyl transfer. High resolution crystal structures were obtained for many of these antibiotics in complex with the ribosome (or part of ribosome). These structures are indicative of functionally important sites in the ribosome and they often shed light on the mechanisms.</p>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/rna-splicing.html" class="archive-item-link">
                RNA Splicing 
              </a>
              <span class="archive-item-date">
                2019-11-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What are the roles of small nuclear RNAs (SnRNAs) in pre-mRNA splicing and what are the mechanisms that control alternative splicing?<hr>
                
                


<div id="overview" class="section level1">
<h1>Overview</h1>
<p>RNA splicing by spliceosomes removes introns from nascent pre-mRNA and joins protein-coding exon sequences, which matures the mRNA and makes it ready for translation. By chosing variable splice sites, different mature mRNAs (and hence proteins) can be produced from the same pre-mRNA.</p>
</div>
<div id="structure-of-the-intron" class="section level1">
<h1>Structure of the Intron</h1>
<p>Splice sites (i.e. exon-intron junctions) have conserved features. As shown in Figure <a href="#fig:intron">1</a>, most introns have an invariant GU at the 5’ end (5’ splice site) and AG at the 3’ end (3’ splice site), their flanking sequences are variable but the bases are found at frequencies higher than expected based on a random distribution. The branch point A is also invariant, and most introns also have a polypyrimidine tract between the branch point and the 3’ splice site. These ‘standard’ pre-mRNAs are spliced by snRNPs U1, U2, U4, U5, and U6. A small fraction of pre-mRNA does not follow the ‘GU-AG’ rule and have 5’-AU and AC-3’ instead, which are spliced by four low-abundance snRNPs together with U5.</p>
<div class="figure"><span id="fig:intron"></span>
<img src="../img/intron.jpg" alt="The structure of a typical intron" width="100%" />
<p class="caption">
Figure 1: The structure of a typical intron
</p>
</div>
</div>
<div id="spliceosome-assembly-and-the-splicing-process" class="section level1">
<h1>Spliceosome Assembly and the Splicing Process</h1>
<p>The spliceosome is a large and dynamic RNP machine whose size is comparable to a ribosome. The 5 snRNPs that constitute the spliceosome are U1, U2, U4, U5, and U6, each of which is a association of an snRNA with proteins. The key catalytic activity is attributed to RNA, so the spliceosome is an ribozyme. A spliceosome is assembled whenever an intron is to be removed and its components are recycled after splicing. The series of events are shown in Figure 2.</p>
<div class="figure"><span id="fig:spliceosome-assembly"></span>
<img src="../img/spliceosome-assembly.jpg" alt="Spliceosome assembly and the splicing process" width="100%" />
<p class="caption">
Figure 2: Spliceosome assembly and the splicing process
</p>
</div>
<ol style="list-style-type: decimal">
<li>U1 base-pairs with the 5’ splice site; SF1 (splicing factor 1) binds to the branch-point A; U2AF (U2 auxilliary factor) associates with the polypyrimidine tract and 3’ splice site. Then, U2 replaces SF1 and U2AF.</li>
<li>The U4/U6.U5 tri-snRNP complex joins, forming the spliceosome</li>
<li>Rearrangement of base-paring interactions leads to the catalytically active conformation. U6 dissociates from U4 and replaces U1. U4 is also released.</li>
<li>The catalytic core of U6/U2 catalyses the first transesterification reaction (ligating 5’ guanosine to the 2’ OH of the branch point adenosine)</li>
<li>Further rearrangements take place, and the second transesterification reaction occurs (joining the two exons)</li>
</ol>
<p>The two sequential transesterification reactions <em>per se</em> do not require energy input, as there is no net bond breaking (two phosphodiester bonds are hydrolysed and another two are formed), but ATP/GTP hydrolysis is used to drive structural transitions and as a ‘proofreading’ mechanism.</p>
<p>As shown in Figure <a href="#fig:splice-detailed-mech">3</a>, In the catalitically active state, U6 partially base-pairs with U2 and U6 forms an internal stem loop structure with three nucleotides coordinated to two Mg<sup>2+</sup> ions. Mg<sup>2+</sup> ions stabilise the transitions states of the two transesterification reactions and thus achieve catalysis. The key bases coordinated to Mg<sup>2+</sup> are confirmed by substitution of oxygen for sulfur atoms at various locations.</p>
<div class="figure"><span id="fig:splice-detailed-mech"></span>
<img src="../img/splice-detailed-mech.jpg" alt="The transesterification reactions in detail" width="100%" />
<p class="caption">
Figure 3: The transesterification reactions in detail
</p>
</div>
</div>
<div id="alternative-splicing" class="section level1">
<h1>Alternative Splicing</h1>
<p>pre-mRNAs contain <em>cis</em>-acting regulatory sequences (either splicing enhancers or silencers) to which splicing factors can bind and thus regulate splicing (Figure <a href="#fig:alternative-splicing">4</a>). RNA-RNA base pairing, which can occur either in <em>cis</em> or <em>trans</em>, can also specify splice site use.</p>
<p>RNA-binding proteins/RNPs that regulate splicing can be classified into 3 classes: the classical hnRNPs, serine-arginine repeat (SR) proteins, and tissue-specific RNA-binding proteins. Generally, hnRNPs inhibit splicing from nearby splice sites by interacting with splicing silencers while SR proteins promotes splicing by binding to enhancers.</p>
<div class="figure"><span id="fig:alternative-splicing"></span>
<img src="../img/alternative-splicing.jpg" alt="Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers." width="100%" />
<p class="caption">
Figure 4: Splicing control by cis-regulatory sequences. ISE/ESE, intronic/exonic splicing enhancers; ISS/ESS, intronic/exonic splicing silencers.
</p>
</div>
<p>RNA-RNA recognition can also control splice site choice, as exemplified by the exon 6 cluster of the <em>Drosophila</em> DSCAM gene, as shown in Figure <a href="#fig:alternative-splicing-rna-rna">5</a>.</p>
<div class="figure"><span id="fig:alternative-splicing-rna-rna"></span>
<img src="../img/alternative-splicing-rna-rna.jpg" alt="hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream." width="100%" />
<p class="caption">
Figure 5: hrp36 binds to all axon6 variants and represses their inclusion in the mature mRNA. When the cis-acting RNA selector sequence upstream of a specific exon (in this case 6.6) pairs with the docking site, it results in activation of the exon immediately downstream.
</p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/nmr.html" class="archive-item-link">
                NMR 
              </a>
              <span class="archive-item-date">
                2019-11-16
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p><strong>Write brief notes on the chemical shift, the nuclear Overhauser enhancement effect (NOE), the relaxation times T1 and T2, and the spin-spin coupling constant (J) with an emphasis on how these can provide information about the structure and dynamics of biological molecules.</strong></p>
<ul>
<li>Chemical Shift: NMR applies RF wave to induce resonance with the precession frequency, <span class="math inline">\(\omega=\gamma (B_0-B_S)\)</span>, where <span class="math inline">\(B_0\)</span> is the constant magnetic field strength on the <span class="math inline">\(z\)</span> axis, <span class="math inline">\(B_S\)</span> is the magnetic field strength shilded by electrons around the nucleus and <span class="math inline">\(\gamma\)</span> is the magnetogyric ratio (depending of the identity of the nucleus). Nuclei in different chemical environment have different <span class="math inline">\(B_S\)</span> and thus different <span class="math inline">\(\omega\)</span>, and the chemical shift, <span class="math inline">\(\delta\)</span>, is defined as <span class="math display">\[\delta=\dfrac{\nu - \nu_\text{reference}}{\nu_\text{o}}\times10^6\]</span> When a protein is folded, the chemical environment of protons is highly variable and the NMR signal is noisy and has many peaks. In contrast, when a protein is unfolded, the chemical shift tend to be less variable, and fewer peaks with greater intensities are observed.</li>
<li>NOE: When two nuclei are close (&lt; 5-6 angstroms) to each other, selective irradiation of one spin by <span class="math inline">\(B_1\)</span> field causes intensity changes of the other spin. Two dimensional methods detect NOE more easily. NOE is the main source of information for macromolecular structure determination by NMR.</li>
<li>T<sub>1</sub> &amp; T<sub>2</sub> relaxation: The applied RF wave creates <span class="math inline">\(B_1\)</span> field which transiently reorient the net magnetisation, <span class="math inline">\(M\)</span>. The decay rate of <span class="math inline">\(M_{xy}\propto\dfrac{1}{\text{T}_2}\)</span>; decay rate of <span class="math inline">\(M_z\propto\dfrac{1}{\text{T}_1}\)</span>. T<sub>1</sub> relaxation is along the <span class="math inline">\(B_0\)</span> field (longitudinal) and T<sub>2</sub> relaxation is in the <span class="math inline">\(xy\)</span> plane (transverse). Relaxation rates give information about molecular dynamics and distances. T2 can be measured from the decay rate of the FID and linewidths and T1 can be measured by an inversion-recovery experiment.</li>
<li>Spin-spin coupling constant: The spin states of two protons on two covalently bonded atoms affect the nuclear energy levels of each other, so the peak split in a neighbour-dependent way (<span class="math inline">\(n\)</span> adjacent protons give <span class="math inline">\(n+1\)</span> multiplets). Coupling strength gives bond-angle information for biomolecules. For example, splitting of H<sup>N</sup> due to H<span class="math inline">\(^\alpha\)</span> (<span class="math inline">\(^3J_{\text{NH}\alpha\text{CH}}\)</span>) depends on <span class="math inline">\(\Phi\)</span> angle.</li>
</ul>
<p><strong>Describe how NMR can be used to measure the pK values of ionizable groups in proteins. Why is chemical shift sensitive to ionization state? How can <sup>31</sup>P NMR be used to measure intracellular pH? Over what pH range is this effective?</strong></p>
<ul>
<li>The chemical shift a proton depends on its chemical environment, which includes the protonation state of adjacent acidic or basic sites. Therefore, gradual alteration of pH causes changes in the chemical shifts that can be plotted against pH. The p<span class="math inline">\(K_\text{a}\)</span> is the pH that corresponds to the inflection point of the resulting sigmoidal curve.</li>
<li><em>In vivo</em> phosphate concentration is about constant. In pH range 5-8, the two dominant forms of phosphate, <span class="math inline">\(\text{H}_2\text{PO}_4^-\)</span> and <span class="math inline">\(\text{HPO}_4^{2-}\)</span> have different chemical shifts for <sup>31</sup>P in NMR, so the chemical shift can be related to pH: at higher pH, the <span class="math inline">\(\text{HPO}_4^{2-}\)</span> form is dominant, and its phosphate is more deshielded, leading to larger chemical shift</li>
</ul>
<p><strong>How does multidimensional NMR differ from basic 1-D NMR? What type of information is contained in the 2-D NMR experiments known as NOESY, TOCSY and HSQC?</strong></p>
<p>As shown in Figure <a href="#fig:2d3dnmr">1</a>:</p>
<ul>
<li>The 1D spectrum is obtained by Fourier transformation of a single transient decaying signal. By collecting a series of transients separated by incrementing lengths of ‘evolution time’, <span class="math inline">\(t_1\)</span>, a 2D spectrum can be obtained. Similarly, 3D spectra are made with an additional period of ‘evolution time’, <span class="math inline">\(t_2\)</span>. Multidimensional NMR clearly shows connections (either <em>J</em>-coupling or NOE) between nuclei as ‘cross peaks’.</li>
<li>COSY (correlated spectroscopy) spectra detect through-bond correlations (<em>J</em>-couplings)</li>
<li>NOESY spectra has an extra 90<span class="math inline">\(^\circ\)</span> pulse and a fixed delay, <span class="math inline">\(\tau_m\)</span> inserted in the sequence and this allows detection of nuclear Overhauster effect (NOE).</li>
<li>HSQC (heteronuclear single quantum coherence spectroscopy) gives a 2D spectrum with one axis for <sup>1</sup>H and another for a heteroatom (usually <sup>15</sup>N or <sup>13</sup>C).</li>
</ul>
<div class="figure"><span id="fig:2d3dnmr"></span>
<img src="../img/2d3dnmr.jpg" alt="Schematic of 1D, 2D and 3D NMR, and sample " width="1993" />
<p class="caption">
Figure 1: Schematic of 1D, 2D and 3D NMR, and sample
</p>
</div>
<p><strong>Outline the application of NMR spectroscopy to the determination of protein 3-D structure in solution. What are the advantages and disadvantages of this technique compared to X-ray crystallography?</strong></p>
<ul>
<li>Application of NMR
<ul>
<li>Experimental restraints: structure-related data derived from NMR spectra
<ul>
<li><sup>1</sup>H-<sup>1</sup>H NOEs: each NOESY cross peak gives a specific pairwise distance information</li>
<li>coupling constants <span class="math inline">\(J\)</span> values can help to determine some dihedral angles</li>
<li>chimical shifts: can be used to search in the database</li>
<li>residual dipolar coupling: gives information about the direction of specific groups with respect to B<sub>0</sub>; gives long-range order information which complements the short-range information given by NOEs.</li>
<li>solvent H-exchange: slow H-D exchange suggests that the proton is in a defined region of secondary structure</li>
</ul></li>
<li>Calculation of structures is done by incorporating various experimental restraints into a molecular dynamics (MD) simulation protocol. MD calculates position and velocity of atoms in a series of small steps using Newtonian mechanics. Extra energy terms are added to take account of observed constraints</li>
</ul></li>
<li>Compared to X-ray crystallography:
<ul>
<li>Disadvantages:
<ul>
<li>NMR is relatively laborious and slow</li>
<li>restricted to small proteins</li>
</ul></li>
<li>Advantages:
<ul>
<li>No need for crystallization; only a pure solution is needed. Can be used on proteins that do not crystallize (e.g. those containing intrinsically unstructured regions)</li>
<li>can be used to study dynamics</li>
<li>site-specific info on ligand binding</li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p><strong>Describe methods to monitor protein-folding pathways during the formation of a globular protein from an extended chain. Place particular emphasis on the role of NMR but also show how other methods provide complementary information. </strong></p>
<div id="nmr" class="section level2">
<h2>NMR</h2>
<p>Protein folding is a multistep process that proceeds through intermediate states and studying these intermediates is difficult due to their transient nature, low populations under non-denaturing conditions, and difficulty of their isolation.</p>
<p>Over the past two decades, many NMR methods have been developed and some became invaluable tools in studying molecular details of protein folding. Tansverse relaxation optimized spectroscopy (TROSY) is the first method developed for probing of dynamics and interactions of large (up to 1MDa) protein assemblies. Relaxation dispersion (RD) and saturation transfer (ST) methods provide a detailed look into the pathways of biomolecular processes, allowing studies of transient intermediates during protein folding.</p>
<p>RD and ST methods allow studying the minor nonnative conformations during protein folding with short lifespans (often <span class="math inline">\(\mu\)</span>s-ms). The two major types of RD experiments include (1) Carr-Purcell-Meiboom-Gill (CPMG) methods that exploit modulation of R<sub>ex</sub> by a sequence of evenly spaced refocusing pulses, and (2) rotating frame <span class="math inline">\(\text{R}_{1\rho}\)</span> relaxation experiments that use modulation of R<sub>ex</sub> by an on- or off-resonance continuous wave (CW) RF field. The ST experiments exploit modulation of R<sub>ex</sub> by a weak RF field, and are conceptually similar to off-resonance <span class="math inline">\(\text{R}_{1\rho}\)</span> measurements. Transient nonnative protein states can be studied by other NMR experiments, such as paramagnetic relaxation enhancement.</p>
<p>Dynamics of protein folding, on the time scale of seconds, can be studied by other methods. Changes in NMR spectra can be monitored in real-time. Logitudinal magnetization (ZZ) exchange experiments can probe interconversions between states with comparable populations. Hydrogen exchange measurements detect evanescent populations of disordered nonnative states transiently sampled by proteins under native-like conditions.</p>
</div>
<div id="other-methods" class="section level2">
<h2>Other Methods</h2>
<p>There are some methods that can assess the extent of folding qualitatively, but they do not give as much quantitative information as NMR does.</p>
<div id="cd-circular-dichroism" class="section level3">
<h3>CD (Circular Dichroism)</h3>
<p>Plane-polarized light is the sum of two circularly polarized beams, L and R, which are rotating in opposite directions. CD arises from differential absorption of L and R component beams in a smaple of chiral molecules. CD is detected by a double beam instrument with separate L and R paths (which are usually produced by high frequency modulation of a photoelastic modulator (PEM) device). The high intensity light source is produced by synchrotron.</p>
<p>The CD spectrum shows the variation of <span class="math inline">\(\Delta\epsilon\)</span> (difference between the extinction coefficient of the L and R beams) with wavelength (<span class="math inline">\(\lambda\)</span>). Because of the chiral nature of amino acids (except glycine), peptides are optically active. The CD spectra in the 170-250 nm region are distinct for proteins with different structure. Standard curves for different secondary structures (<span class="math inline">\(\alpha\)</span>-helix, <span class="math inline">\(\beta\)</span>-sheet, <span class="math inline">\(\beta\)</span>-turn, and random coil) are available (Figure <a href="#fig:cd">2</a>) and the proportion of different secondary structures of a protein is calculated by linear combination of the standard curves.</p>
<div class="figure"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="CD components and standard curves" width="100%" />
<p class="caption">
Figure 2: CD components and standard curves
</p>
</div>
</div>
</div>
<div id="dsc-differential-scanning-calorimetry" class="section level2">
<h2>DSC (Differential Scanning Calorimetry)</h2>
<p>DSC determines the heat capacity (the difference between the heat capacity in the sample cell and in the reference cell), C<sub>p</sub>, of a molecule in aqueous solution, as a function of temperature. This is done by increasing cell temperatures while keeping the two cells at the same temperature and recording the power supply throughout the experiment. The variation of <span class="math inline">\(C_\text{p}\)</span> with temperature is plotted. On such a curve, there is a peak if the sample molecule undergoes state changes within the temperature (Figure <a href="#fig:dsc">3</a>). For a protein, the peak corresponds to the melting temperature, <span class="math inline">\(T_\text{m}\)</span>, which occurs when the concentrations of its native state ([N]) and the denatured state ([D]) are equal. Stabilization of protein, for example by altering the pH and forming complexes, increases <span class="math inline">\(T_\text{m}\)</span></p>
<div class="figure"><span id="fig:dsc"></span>
<img src="../img/dsc.jpg" alt="Principles of DSC" width="100%" />
<p class="caption">
Figure 3: Principles of DSC
</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Campbell-2012">
<p>Campbell, Iain D. 2012. <em>Biophysical Techniques</em>. Oxford University Press.</p>
</div>
<div id="ref-Zhuravleva-2017">
<p>Zhuravleva, Anastasia, and Dmitry M. Korzhnev. 2017. “Protein Folding by Nmr.” <em>Progress in Nuclear Magnetic Resonance Spectroscopy</em> 100: 52–77. <a href="https://doi.org/https://doi.org/10.1016/j.pnmrs.2016.10.002">https://doi.org/https://doi.org/10.1016/j.pnmrs.2016.10.002</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/biophysical-techniques.html" class="archive-item-link">
                Biophysical Techniques 
              </a>
              <span class="archive-item-date">
                2019-11-12
              </span>
              <div class="article-summary" style="display: none;">
                
                  You have expressed and purified a protein in a recombinant system. How would you confirm that the protein was the one expected from the DNA sequence. (ESI-MS) How would you determine that the protein was folded and its solution behaviour. (CD, Thermofluor, Intrinsic fluorescence, DSC, SEC MALS, AUC, DLS) How can we quantify the binding between two macromolecules? Discuss how you could use the following techniques to determine a KD for an interaction? What are their limitations in practice? What would decide which techniques you could apply to an interaction? (Surface Plasmon resonance, Fluorimetry/Fluorescence anisotropy, Microscale thermophoresis (MST), Isothermal titration calorimetry (ITC), Stopped flow, NMR, AUC) References: Use Campbell&#39;s Biophysical Techniques; for MST use Wienken et al 2010 Nature Communications 10.1038/ncomms1093
<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#confirmation-of-dna-sequence"><span class="toc-section-number">1</span> Confirmation of DNA Sequence</a><ul>
<li><a href="#esi-ms-electrospray-ionization-mass-spectrometry"><span class="toc-section-number">1.1</span> ESI-MS (Electrospray Ionization Mass Spectrometry)</a></li>
</ul></li>
<li><a href="#assess-protein-folding-and-solubility"><span class="toc-section-number">2</span> Assess Protein Folding and Solubility</a><ul>
<li><a href="#cd-circular-dichroism"><span class="toc-section-number">2.1</span> CD (Circular Dichroism)</a></li>
<li><a href="#dsc-differential-scanning-calorimetry"><span class="toc-section-number">2.2</span> DSC (Differential Scanning Calorimetry)</a></li>
<li><a href="#dls-dynamic-light-scattering"><span class="toc-section-number">2.3</span> DLS (Dynamic Light Scattering)</a></li>
<li><a href="#auc-analytical-ultracentrifuge"><span class="toc-section-number">2.4</span> AUC (Analytical Ultracentrifuge)</a></li>
</ul></li>
<li><a href="#determining-kd"><span class="toc-section-number">3</span> Determining K<sub>d</sub></a><ul>
<li><a href="#fluorimetryfluorescence-anisotropy"><span class="toc-section-number">3.1</span> Fluorimetry/Fluorescence anisotropy</a></li>
<li><a href="#surface-plasmon-resonance"><span class="toc-section-number">3.2</span> Surface Plasmon resonance</a></li>
<li><a href="#isothermal-titration-calorimetry-itc"><span class="toc-section-number">3.3</span> Isothermal titration calorimetry (ITC)</a></li>
<li><a href="#mst-microscale-thermophoresis"><span class="toc-section-number">3.4</span> MST (Microscale Thermophoresis)</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="confirmation-of-dna-sequence" class="section level1">
<h1><span class="header-section-number">1</span> Confirmation of DNA Sequence</h1>
<div id="esi-ms-electrospray-ionization-mass-spectrometry" class="section level2">
<h2><span class="header-section-number">1.1</span> ESI-MS (Electrospray Ionization Mass Spectrometry)</h2>
<p>In mass spectrometry, the analytes are first ionised in vacuum and these charged molecules are passed into a electric/magnetic field, and their path through the field can be used to deduce their mass/charge (m/z) ratio (using deflection, TOF, quadrupole, or other analyzers).</p>
<p>Traditional ionisation methods were not suitable for biological macromolecules because the need for heating (or other treatment) to achieve gas phase would cause rapid decomposition of the molecules. Later, MALDI (matrix-assisted laser desorption/ionisation) and ESI (Electrospray Ionization) techniques were deveoped and they are suitable for this purpose. In MALDI, proteins are placed in a light-absorbing matrix, then ionisation and desorption is triggered by a short pulse of laser light. In ESI, a solution of analytes is passed through a charged needle kept at a high electrical potential, dispersing the solution into a microdroplets. The solvent around macromolecules rapidly evaporates, leaving charged molecules in gas phase. ESI can directly accept inputs from many other purification methods such as SDS-PAGE and chromatography and in addition it is good for detecting native states and different conformations.</p>
<p>The sequence of a protein is often too long to be obtained by MS at one time, so protease can be used to break peptides into small fragments which are sequenced individually. In addition, tandem MS (MS/MS) is can be used, in which one peptide is first analysed by one mass analyzer (MS1), and then further fragmented by a ‘collision gas’ such as He and Ar, and the m/z ratios of these fragments are analysed by MS2. This produces a spectrum with many peaks (greater m/z correspond to longer fragments), and the successive peaks differ by the m/z of a particular amino acid in the original peptide. This information can be used to deduce the original sequence.</p>
</div>
</div>
<div id="assess-protein-folding-and-solubility" class="section level1">
<h1><span class="header-section-number">2</span> Assess Protein Folding and Solubility</h1>
<div id="cd-circular-dichroism" class="section level2">
<h2><span class="header-section-number">2.1</span> CD (Circular Dichroism)</h2>
<p>Plane-polarized light is the sum of two circularly polarized beams, L and R, which are rotating in opposite directions. CD arises from differential absorption of L and R component beams in a smaple of chiral molecules.</p>
<p>CD is detected by a double beam instrument with separate L and R paths (which are usually produced by high frequency modulation of a photoelastic modulator (PEM) device). The high intensity light source is produced by synchrotron.</p>
<p>The CD spectrum shows the variation of <span class="math inline">\(\Delta\epsilon\)</span> (difference between the extinction coefficient of the L and R beams) with wavelength (<span class="math inline">\(\lambda\)</span>).</p>
<p>Because of the chiral nature of amino acids (except glycine), peptides are optically active. The CD spectra in the 170-250 nm region are distinct for proteins with different structure. Standard curves for different secondary structures (<span class="math inline">\(\alpha\)</span>-helix, <span class="math inline">\(\beta\)</span>-sheet, <span class="math inline">\(\beta\)</span>-turn, and random coil) are available (Figure <a href="#fig:cd">2.1</a>) and the proportion of different secondary structures of a protein is calculated by linear combination of the standard curves.</p>
<div class="figure" style="text-align: c"><span id="fig:cd"></span>
<img src="../img/cd.jpg" alt="CD components and standard curves" width="100%" />
<p class="caption">
Figure 2.1: CD components and standard curves
</p>
</div>
</div>
<div id="dsc-differential-scanning-calorimetry" class="section level2">
<h2><span class="header-section-number">2.2</span> DSC (Differential Scanning Calorimetry)</h2>
<p>DSC determines the heat capacity (the difference between the heat capacity in the sample cell and in the reference cell), C<sub>p</sub>, of a molecule in aqueous solution, as a function of temperature. This is done by increasing cell temperatures while keeping the two cells at the same temperature and recording the power supply throughout the experiment. The variation of <span class="math inline">\(C_\text{p}\)</span> with temperature is plotted. On such a curve, there is a peak if the sample molecule undergoes state changes within the temperature (Figure <a href="#fig:dsc">2.2</a>). For a protein, the peak corresponds to the melting temperature, <span class="math inline">\(T_\text{m}\)</span>, which occurs when the concentrations of its native state ([N]) and the denatured state ([D]) are equal. Stabilization of protein, for example by altering the pH and forming complexes, increases <span class="math inline">\(T_\text{m}\)</span></p>
<div class="figure" style="text-align: c"><span id="fig:dsc"></span>
<img src="../img/dsc.jpg" alt="Principles of DSC" width="100%" />
<p class="caption">
Figure 2.2: Principles of DSC
</p>
</div>
</div>
<div id="dls-dynamic-light-scattering" class="section level2">
<h2><span class="header-section-number">2.3</span> DLS (Dynamic Light Scattering)</h2>
<p>Random motion of macromolecules in a suspension causes fluctuations in local concentration and thus local variarions in refractive index and intensity of the scattered light. These time-dependent fluctuations can be analyzed by a coherent laser source. The observed fluctuations give rise to diffusion coefficients, <span class="math inline">\(D\)</span>, which is related to the size of the molecules (large particles diffuse more slowly). DSL can measure polydispersity and the presence of aggregates in protein samples.</p>
</div>
<div id="auc-analytical-ultracentrifuge" class="section level2">
<h2><span class="header-section-number">2.4</span> AUC (Analytical Ultracentrifuge)</h2>
<p>An AUC is a specialised ultracentrifuge equipped with absorbance and interference detection systems. Each cell contains a sample meniscus and a reference cavity, allowing the absorbance of the solvent to be corrected.</p>
<p>Two types of experiments are typically performed by AUC: sedimentation velocity and sedimentation equilibrium. Sedimentation velocity is performed at high speed, which depletes particles from the centrifuge cell and creates a pellet at the bottom of the cell. It can give information on molecular shape, mass, and interactions with themselves and with other components.</p>
<p>Sedimentation equilibrium is perfomed at low speed and does not create a pellet, and the sedimentation of molecules down the centrifuge cell is balanced by their diffusion back up the cell. The main application of sedimentation equilibrium is the detection of complexes and self-association and the quantification of binding between species</p>
</div>
</div>
<div id="determining-kd" class="section level1">
<h1><span class="header-section-number">3</span> Determining K<sub>d</sub></h1>
<div id="fluorimetryfluorescence-anisotropy" class="section level2">
<h2><span class="header-section-number">3.1</span> Fluorimetry/Fluorescence anisotropy</h2>
<p>If fluorophores are excited with plane polarized light and the fluorescence is observed through analyzing polarizers, the fluorescence is also polarised.</p>
<p>The fluorescence anisotropy is defined as <span class="math inline">\(A=\dfrac{I_\parallel - I_\bot}{I_\parallel+2_{\bot}}\)</span>, where <span class="math inline">\(I_\parallel\)</span> and <span class="math inline">\(I_{\bot}\)</span> are the fluorescence intensities polarised parallel and perpendicular to the direction of the excitation beam. <span class="math inline">\(A\)</span> is a direct measure of the molecular rotation in solution and can be used to study complex formation, as a macromolecule will rotate more slowly when it is in a complex thatn when it is alone.</p>
</div>
<div id="surface-plasmon-resonance" class="section level2">
<h2><span class="header-section-number">3.2</span> Surface Plasmon resonance</h2>
<p>Surface plasmon resonance (SPR) is used for measuring molecular interactions between a pair of molecules.</p>
<p>A surface plasmon is an electron oscillation generated at a surface interface between a metal and a dielectric. A plasmon resonance occurs when EM wave in visible light couples optimally with the oscillating electrons in the metal, and this results in a maximal reduction in the reflected light intensity. The resonance angle, <span class="math inline">\(\theta_\text{spr}\)</span>, is found by changing the angle of incidence of the light beam, giving a dip in a plot of intensity against angle. <span class="math inline">\(\Delta\theta_\text{spr}\)</span> is sensitive to changes in the refractive index of the medium near the metal surface and this is a measure of the mass change at the sensor surface (in the evanescent region).</p>
<p>In an SPR experiment, one type of ligand is immobilised at the sensor surface, and a the analyte is passed through the cell. If the ligand binds to any binding partner in the analyte, <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would increase. Then, non-specific binding is washed off by buffer, and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> would decrease and <span class="math inline">\(\Delta\theta_\text{spr}\)</span> due to specific binding can be found. Finally, regeneration solution is applied to remove all binding and reset <span class="math inline">\(\Delta\theta_\text{spr}\)</span> to zero. (Figure <a href="#fig:spr">3.1</a>)</p>
<div class="figure" style="text-align: c"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="Principles of SPR" width="100%" />
<p class="caption">
Figure 3.1: Principles of SPR
</p>
</div>
<p>SPR can be used to determine <span class="math inline">\(K_d\)</span> of complex formation, which equals <span class="math inline">\(\dfrac{k_{-1}}{k_1}\)</span> where <span class="math inline">\(k_1\)</span> is the rate of association and <span class="math inline">\(k_{-1}\)</span> is the rate of dissociation. <span class="math inline">\(k_1\)</span> and <span class="math inline">\(k_{-1}\)</span> can be deduced from the plot.</p>
</div>
<div id="isothermal-titration-calorimetry-itc" class="section level2">
<h2><span class="header-section-number">3.3</span> Isothermal titration calorimetry (ITC)</h2>
<p>ITC measures heat changes when a complex is formed at constant temperature.</p>
<p>In ITC, an insulated reaction cell containing protein is kept at a temperature (usually 8<span class="math inline">\(^\circ\text{C}\)</span> above the environment) which is equal to the temperature of a reference cell, and the reference cell is kept at a constant temperature by a thermostat. Then, increasing amounts of ligand is added into the chamber, and they form complexes with the protein, which can be exothermic or endothermic. The heat change is compensated by a power supply, which can be converted to <span class="math inline">\(\Delta H\)</span> of the reaction. As more ligands are added, proteins become saturated and <span class="math inline">\(\Delta H\)</span> approaches zero. The raw data obtained (power supplied to compensate the heat change caused by each addition of ligands) can be integrated and corrected to give a plot of <span class="math inline">\(\Delta H\)</span> against the molar ratio of the ligand and the protein, and <span class="math inline">\(\Delta H\)</span>, K<sub>d</sub> and stoichiometry can be inferred from the curve. Subsequently, <span class="math inline">\(\Delta G\)</span> and <span class="math inline">\(\Delta S\)</span> can also be calculated (Figure <a href="#fig:itc">3.2</a>).</p>
<div class="figure" style="text-align: c"><span id="fig:itc"></span>
<img src="../img/itc.jpg" alt="ITC data manipulation" width="100%" />
<p class="caption">
Figure 3.2: ITC data manipulation
</p>
</div>
</div>
<div id="mst-microscale-thermophoresis" class="section level2">
<h2><span class="header-section-number">3.4</span> MST (Microscale Thermophoresis)</h2>
<p>MST is a relatively new method for analysing interactions of proteins or small molecules in complex bioliquids such as blood serum or cell lysate. The technique depends on the phenomenon that molecules move within temperature gradients (thermophoresis).</p>
<p>The instrument uses an IR laser to create a temperature gradient, and the movement of the molecules within this gradient is monitored by fluoresence. The fluoresence can either be intrinsic (due to tryptophan) or extrinsic (attached dye or fluorescent protein).</p>
<p>Compared with traditional methods for studying protein interactions, MST has several advantages:</p>
<ul>
<li>A minuscule amount of sample (a few <span class="math inline">\(\mu\)</span>l) is needed,</li>
<li>no limitations on size and affinity</li>
<li>no limitations on buffer; tolerates impurity; can be used in in complex bioliquids such as blood serum or cell lysate</li>
<li>no need for immobilization</li>
</ul>
<div class="figure" style="text-align: c"><span id="fig:mst"></span>
<img src="../img/mst.jpg" alt="Sample data produced by MST" width="100%" />
<p class="caption">
Figure 3.3: Sample data produced by MST
</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Wienken-2010">
<p>Wienken, Christoph J., Philipp Baaske, Ulrich Rothbauer, Dieter Braun, and Stefan Duhr. 2010. “Protein-Binding Assays in Biological Liquids Using Microscale Thermophoresis.” <em>Nature Communications</em> 1 (1): 100. <a href="https://doi.org/10.1038/ncomms1093">https://doi.org/10.1038/ncomms1093</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/paper-vi-1.html" class="archive-item-link">
                Paper VI 2011 
              </a>
              <span class="archive-item-date">
                2019-11-09
              </span>
              <div class="article-summary" style="display: none;">
                
                


<div id="alzheimers-disease" class="section level1">
<h1>Alzheimer’s Disease</h1>
<div id="effect-of-ampk-on-secreted-abeta-levels-in-neuronal-cultures" class="section level2">
<h2>Effect of AMPK on secreted A<span class="math inline">\(\beta\)</span> levels in neuronal cultures</h2>
<ol style="list-style-type: lower-alpha">
<li>In ELISA (enzyme-linked immunosorbant assay), antibodies are used to bind an antigen of interest (usually protein) with high specificity, and the presence and concentration of the antigen is reported by the extent of reaction of the enzyme conjugated with the antibody. Using two types of antibodies binding to different regions of the antigen allows sandwich ELISA (see below), which has greater specificity. Horseradish peroxidase converts its substrate to a coloured product, and the intensity of the colour, which is related to the concentration of A<span class="math inline">\(\beta\)</span> peptide (the antigen), can be measured by a spectrophotometer as absorbance of EM wave at a certain visible wavelength (the relationship is not always linear so a calibration is needed).</li>
</ol>
<p><img src="../img/elisa-sandwich.jpg" width="100%" /></p>
<ol start="2" style="list-style-type: lower-alpha">
<li>By plotting ‘measured value’ against the number of dilution (the power to which <span class="math inline">\(\frac{1}{2}\)</span> is raised), it can be shown that the realtionship between them is roughly linear when measured value is around 100.</li>
</ol>
<pre class="r"><code>standard &lt;- c(206, 215, 201, 199, 205, 197, 199, 202, 190, 120, 67, 41, 27, 13, 8, 8)
untreated &lt;- c(211, 197, 200, 202, 198, 195, 197, 164, 88, 47, 27, 15, 6, 9, 7, 7)
AICAR &lt;- c(207, 202, 205, 199, 189, 200, 183, 101, 58, 30, 18, 12, 7, 7, 8, 6)
pow &lt;- 0:15

df &lt;- tibble(pow, standard, untreated, AICAR)
df %&gt;% gather(standard, untreated, AICAR, key = &#39;treatment&#39;, value = &#39;measured value&#39;) %&gt;% 
  ggplot(aes(pow, `measured value`, color = treatment, shape = treatment, linetype=treatment))+
  geom_point()+xlab(&#39;number of dilution (i.e. number of times the concentration is halved)&#39;)+
  geom_smooth(se=FALSE, method=&#39;loess&#39;, span=0.4, size=0.5)+
  geom_hline(yintercept = 100)+
  scale_x_continuous(breaks = 0:15)</code></pre>
<pre><code>## `geom_smooth()` using formula &#39;y ~ x&#39;</code></pre>
<p><img src="./tutorial/2019-11-09-paper-vi-1_files/figure-html/unnamed-chunk-3-1.png" width="100%" /></p>
<p>Reading off from the plot:</p>
<ul>
<li>for the standard solution, a <span class="math inline">\(\dfrac{1}{2^{9.5}}\)</span> dilution corresponds to 100 units. Thus, 100 units correspond to <span class="math inline">\(10\times \dfrac{1}{2^{9.5}}=0.01381\text{ ng/ml}\)</span></li>
<li>for the untreated sample, a <span class="math inline">\(\dfrac{1}{2^{7.9}}\)</span> dilution corresponds to 100 units, or <span class="math inline">\(0.0138\text{ ng/ml}\)</span>, so the undiluted sample has a A<span class="math inline">\(\beta\)</span> concentration of <span class="math inline">\(0.01381\times2^{7.9}=3.299\text{ ng/ml}\)</span>. The volume of 10<sup>6</sup> cells is 100<span class="math inline">\(\mu\)</span>l, so the level of secretion is <span class="math inline">\(3.299\times0.100\times 1000=329\text{ pg/10}^6\text{ cells/day}\)</span></li>
<li>for the sample with AICAR, the undiluted sample has a A<span class="math inline">\(\beta\)</span> concentration of <span class="math inline">\(0.01381\times2^{7.2}=2.031\text{ ng/ml}\)</span>. The level of secretion is <span class="math inline">\(2.031\times0.100\times 1000=203\text{ pg/10}^6\text{ cells/day}\)</span></li>
</ul>
<ol start="3" style="list-style-type: lower-alpha">
<li>Upon activation by AICAR, AMPK reduces secretion of A<span class="math inline">\(\beta\)</span></li>
</ol>
</div>
<div id="effect-of-added-abeta-on-ampk-activity" class="section level2">
<h2>Effect of added A<span class="math inline">\(\beta\)</span> on AMPK activity</h2>
<ol start="4" style="list-style-type: lower-alpha">
<li><p>Triton X-100 is a detegent, disrupting the cell membrane and lysing the cell; PMSF inhibits serine proteases, preventing degradation of target protein; protein A, which is immobilised on sepharose, binds specifically to the F<sub>c</sub> region of anti-AMPK antibody</p></li>
<li><p>This ensures that all measured enzymatic activity is due to AMPK but not other kinases that would be present in the whole cell extract.</p></li>
<li>Substituting <span class="math inline">\(t_{\frac{1}{2}}=14\text{ days}\)</span> and <span class="math inline">\(t = 14\)</span> into <span class="math inline">\(A=A_0e^{-kt}=A_0e^{-\dfrac{\ln{2}t}{t_{\frac{1}{2}}}}=A_0\times(2)^{-\dfrac{t}{t_{\frac{1}{2}}}}\)</span>, <span class="math inline">\(A=A_0\times(2)^{-\frac{9}{14}}=0.6404A_0\)</span>, i.e. <span class="math inline">\(A_0=1.5614A\)</span>, where <span class="math inline">\(A_0\)</span> is the original activity and <span class="math inline">\(A\)</span> is the activity after 9 days. <span class="math display">\[\begin{aligned}A_0 \text{ cpm}\div80\%\div(2.2\times10^{12})\text{ cpm/Ci}\div(0.5\times10^{-3})\text{ Ci/nmol}\div(50\times10^{-3}\text{ mg cell protein})\div20\text{ min}\div\frac{50}{50+5+5}\\=x \text{ nmol substrate phosphorylated/min/mg cell protein}\end{aligned}\]</span> where x is the final answer. This simplifies to <span class="math display">\[x=1.3636\times10^{-9}A_0=2.129\times10^{-9}A\]</span>
<ul>
<li>Activity (+A<span class="math inline">\(\beta\)</span>): <span class="math inline">\(2.129\times10^{-9}\times(11800-278)=2.45\times10^{-5} \text{ nmol substrate phosphorylated/min/mg cell protein}\)</span></li>
<li>Activity (no A<span class="math inline">\(\beta\)</span>): <span class="math inline">\(2.129\times10^{-9}\times(2940-278)=5.67\times10^{-6} \text{ nmol substrate phosphorylated/min/mg cell protein}\)</span></li>
</ul></li>
<li><p>AMPK activity increases as A<span class="math inline">\(\beta\)</span> is added, and AMPK may help to degrade A<span class="math inline">\(\beta\)</span> according to (c).</p></li>
</ol>
</div>
<div id="association-of-tau-protein-with-microtubules" class="section level2">
<h2>Association of <span class="math inline">\(\tau\)</span> protein with microtubules</h2>
<ol start="8" style="list-style-type: lower-alpha">
<li>tau alone is soluble and exists solely in the supernatant. Unphosphorylated tau binds to microtubules strongly and exists entirely in the pallet (insoluble). Phosphorylated tau has less affinity to microtubules and most of them are found in the supernatant.</li>
</ol>
</div>
<div id="phosphorylation-sites-on-tau" class="section level2">
<h2>Phosphorylation sites on tau</h2>
<ol style="list-style-type: lower-roman">
<li><ul>
<li>Induce mutations on different positions and observe their effect on phosphorylation. The mutated position(s) can be determined by DNA sequencing and the effect of phosphorylation can be determined by isoelectric focusing (phosphorylated protein has a more negative charge).</li>
<li>Fragment the protein and test each piece for phosphorylation.</li>
</ul></li>
</ol>
<ol start="10" style="list-style-type: lower-alpha">
<li>These phosphorylation sites are all located on the MT binding domain (235-368) and thus phosphorylation can directly (at least sterically) hinder binding to MT. In addition, MT binding domain contains many postively charged lysine (K) residues and few negatively charged (D and E) residues. This suggest that the binding between tau and MT is electrostatic (+ve on tau, -ve on MT). Phosphorylation adds -ve charge on tau and thus hinders electrostatic binding.</li>
</ol>
</div>
</div>
<div id="notch-receptor" class="section level1">
<h1>Notch Receptor</h1>
<ol style="list-style-type: lower-alpha">
<li><p>The primer sequences are:</p>
<pre><code>forward primer
5&#39; GGATCC CAGGACGTGGATGAGTGCTCGCTGGG 3&#39; # 33 nt
reverse primer
5&#39; GGATCC TCA CTGGCACAGATGCCCAGTGAAGCCC 3&#39; # 34 nt</code></pre>
<p>To produce a standalone peptide, a start codon (ATG) and a stop codon (TGG) is needed. The stop codon is included in the reverse primer (<code>5'TCA3'</code> which correspond to <code>5'TGA3'</code> in the forward direction). The start codon is defined in the plasmid and sets the reading frame. The restriction site chosen must ensure correct reading frame alignment of the gene of interest, and Bam HI does this job (while some others, such as Sal I, does not), so I chose it, and its sequence, <code>GGATCC</code>, is included at the 5’ end of both primers
<img src="../img/sticky.jpg" width="100%" />
Regarding restriction sites, Pst I and Sph I must not be chosen because they are contained within the gene of interest and applying these restriction enzymes will break the gene. The remaining part of the primer corresponds to the gene of interest, and the 3’ ends are chosen to include 3-4 consecutive G/C to ensure high annealing stability, facilitating DNA polymerase binding.</p></li>
<li><p>Bam HI is applied both to the plasmid and the PCR product, creating sticky ends and allowing them to join. The resulting set of plasmids (some are recombinant and some are not) are transformed into bacteria by heat shock (by disrupting the cell membrane using CaCl<sub>2</sub> and exposing cells to thermal imbalance) or electroporation (by using short pulses of high-voltage electric currents). Ampicilin is added to kill cells without gaining the plasmid, and after that, usually (when a lacZ<span class="math inline">\(\alpha\)</span> gene is incorporated into the cloning site in the reverse direction) X-gal is used to test whether the cells with plasmids contains the gene of interest (blue-white screening).
<img src="../img/clone.jpg" width="100%" /></p></li>
<li><p>As illustrated in the plasmid map, the expression of inserted gene is controlled by the <em>Lac</em> operator. When uninduced, lac repressor protein due to <em>LacI</em> expression binds to <em>Lac</em> operator, preventing expression of inserted gene. IPTG can be used to induce expression. IPTG is an analogue of lactose, which activates <em>Lac</em> inducible operon by deactivating the lac repressor. IPTG is not metabolised by <em>E. coli</em> so it can be maintained at a constant concentration.</p></li>
<li><p>The protein product contains N-terminal hexahistidine tag which binds to Nickle ions with high affinity. Affinity chromatography using nickle-bound resin can be used to purify the protein. After that, size-exclusion chromatography may also be done to remove any possible aggregates of this peptide or complexes of this peptide with other species.</p></li>
<li><p>Run a SDS-PAGE, stain with Commassie blue and there should be only one band.</p></li>
<li><p>This extracellular protein is likely to contain disulfide bridges, which would not form in the normal reducing intracellular environment of bacteria. Some strains of <em>E. coli</em>, such as <em>Origami</em>, have mutant thioredoxin reductase and/or glutathione reductase an thus have a oxidising cytoplasm, and they can be used to express proteins with disulfide bridges.</p></li>
<li><ul>
<li>From the blot developed with biotin binding protein, A alone produce a band while B or C alone does not. This indicates that A must be a potential ligand (because it, being alone and biotin-tagged, is pulled down by biotin-binding Streptavidin).</li>
<li>From the blot developed with His tag binding antibody, only A+B produces a band. As A is the potential ligand, B must be Notch that is pulled down along with A. C must not be Notch because A+C does not produce a band, so C is another potential ligand.</li>
<li>C alone does not produce a band in the biotin-specific blot possibly because the biotin tag possibly because C’s tertiary structure prevents biotin binding</li>
</ul></li>
<li><p>The potential ligand A binds to Notch (B), so A+B produces a band on the His-specific blot. It is uncertain whether A interacts with C. A and C are both potential ligands and will not produce a band on the His-specific blot. A+C produces a band on the biotin-specific blot and this may be solely due to A (which itself alone produces the band). A might bind to C without making C’s biotin available for binding, or A might not bind to C, or A might bind to C, making its own biotin unavailable and C’s biotin available, so that a band with equal thickness is produced.</p></li>
<li><ul>
<li>Fluoresence resonance energy transfer (FRET), in which the potential ligand and Notch are each tagged with a fluorophore, where one type of fluorophore, when excited, can transfer energy to the other type of fluorophore through nonradiative dipole-dipole coupling and hence induce fluorescence of the acceptor fluorophore. The energy transfer is only possible when two fluorophores are very close to each other (when their associated proteins are bound to each other). If excitation of the donor fluorophore results in emission from the acceptor fluorophore, it indicates complex formation.</li>
<li>As shown in Figure <a href="#fig:spr">1</a>, surface plasmon resonance can also be used to monitor complex formation.</li>
<li>MST (Microscale Thermophoresis) allows detection of complex formation in complex bioliquids such as blood serum or cell lysate.</li>
</ul></li>
</ol>
<div class="figure"><span id="fig:spr"></span>
<img src="../img/spr.jpg" alt="Principles of SPR" width="100%" />
<p class="caption">
Figure 1: Principles of SPR
</p>
</div>
<div style="page-break-after: always;"></div>
</div>
<div id="compound-a" class="section level1">
<h1>Compound A</h1>
<ol style="list-style-type: lower-alpha">
<li><p>Toa1 decreases generation time and decreases lifespan.</p></li>
<li><p>No effect on both.</p></li>
<li><p>As a control group.</p></li>
<li><p>DMSO reduces lifespan.</p></li>
<li><p>Regarding lifespan in DMSO, when either Toa1 or Abp1 (or both) is absent, lifespan is unresponsive to compound A. Among these three conditions, when Toa1 is present and Abp1 is absent, lifespan is constantly short; when Toa is absent, lifespan is constantly long. This suggest that Toa1 reduces lifespan, and Abp1 itself has no effect on the lifespan when Toa1 is absent.
When both proteins are present, lifespan increases with concentration of A and then decreases. This suggest that, at low concentrations of A, Abp1 binds to A, and the binding of Toa1 to Abp1-A complex reduces Toa1’s ability to shorten lifespan. At high concentrations of A, Abp1 becomes less effective in deactivating Toa1, possibly because Abp1 forms aggregates.</p></li>
<li><p>Mutation of ATG8 always leads to short lifespan and generation time, and mutation of SCH9 always leads to increased lifespan and generation time, regardless of the conditions of ABP1 and TOA1, and they all made the lifespan insensitive to the concentration of compound A. When mutations of ATG8 and SCH9 occur simutaneously, the effect of ATG8 loss dominates and results in short lifespan &amp; generation time.
Therefore, SCH9 may reduce lifespan and generation time and ATG8 opposes SCH9’s activity. This explains <span class="math inline">\(sch9\Delta\)</span>, <span class="math inline">\(atg8\Delta\)</span> and <span class="math inline">\(sch9\Delta atg8\Delta\)</span>. In addition, Toa1’s ability to reduce lifespan is dependent on SCH9. In <span class="math inline">\(sch9\Delta abp1\Delta\)</span> and <span class="math inline">\(sch9\Delta toa1\Delta\)</span>, the life-span reduction cannot be done without SCH9. In <span class="math inline">\(abp1\Delta atg8\Delta\)</span>, lifespan reduction is caused by Toa1+SCH9 and in <span class="math inline">\(toa1\Delta atg8\Delta\)</span> caused by SCH9, and no ATG8 was present to oppose this effect.</p></li>
<li><p>In the western blot data, for Sch9 5SA, the purpose of changing serine to alanine is usually to test for phosphorylation, and in this case the band is shorter than WT, indicating there is phosphorylation on Sch9. Also Sch9 5SA band is similar in length to <span class="math inline">\(toa1\Delta\)</span>, which implies that toa1 loss has an similar effect to making Sch9 unavailable for phosphorylation. Thus Toa1 may phosphorylate Sch9, and this is in accordance with an observation in (f): ‘toa1’s ability to reduce lifespan is dependent on sch9’. Sch9 5SA with phosphatase further reduces the signal, and this indicates that Sch9 has other phosphorylation sites (apart from the site where 5 S was changed to A i.e. target of toa1) being phosphorylated.</p></li>
<li><p>WT + compound A produced a low signal comparable to the WT + phosphate band. Thus compound A may hamper phosphorylation of Sch9 (by directly hindering phosphorylation, or acting as a phosphatase). As toa1 is ‘target of A’ and toa1 is the kinase that phosphorylates Sch9, compound A is very likely to inhibit toa1.</p></li>
<li><p>see Figure (g)</p></li>
<li>Purify the proteins and study their interactions <em>in vitro</em>.
<ul>
<li>to confirm phosphorylation of Sch9 by Toa1: mix Sch9, Toa1 and ATP (control group: Sch9 and ATP only) and run isoelectric focusing (phosphorylated Sch9 has a more negative charge)</li>
<li>to confirm the role of compound A: mix compound A, Sch9, Toa1 and ATP (control group: buffer+Sch9+Toa1+ATP) and run isoelectric focusing.</li>
</ul></li>
<li><p>In strains with Sch92D3E but not WT Sch9, the lifespan is constantly reduced and this effect is unaffected by compound A.</p></li>
<li><p>See Figure <a href="#fig:fuckyoucompoundA">2</a>.</p></li>
</ol>
<div class="figure"><span id="fig:fuckyoucompoundA"></span>
<img src="../img/fuckyoucompoundA.jpg" alt="When compound A concentration increases, TOA is is inhibited and less Sch9 is phosphorylated, so there is less lifespan-reduction effect due to phosphorylated Sch9, and hence the lifespan increases. When the concentration of A gets too high, however, ATG8 phosphatase is expressed to dephosphorylate Sch9 and thus decrease lifespan." width="100%" />
<p class="caption">
Figure 2: When compound A concentration increases, TOA is is inhibited and less Sch9 is phosphorylated, so there is less lifespan-reduction effect due to phosphorylated Sch9, and hence the lifespan increases. When the concentration of A gets too high, however, ATG8 phosphatase is expressed to dephosphorylate Sch9 and thus decrease lifespan.
</p>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/protein-purification.html" class="archive-item-link">
                Protein Expression and Purification 
              </a>
              <span class="archive-item-date">
                2019-11-02
              </span>
              <div class="article-summary" style="display: none;">
                
                  You want to over-express a protein secreted by the malarial parasite. It is cysteine-rich and has no known structure. Outline an expression and purification strategy, using both traditional and high-throughput methods, to produce a sample suitable for protein crystallography. Include a description of how you would assess the quality of the purified protein. OR What strategies could you employ during protein expression and purification to minimise heterogeneity in a sample required for structural studies?<hr>
                
                

<div id="TOC">
<ul>
<li><a href="#title">Title</a></li>
<li><a href="#overview"><span class="toc-section-number">1</span> Overview</a></li>
<li><a href="#protein-expression"><span class="toc-section-number">2</span> Protein Expression</a><ul>
<li><a href="#design"><span class="toc-section-number">2.1</span> Identifying and amplifying the gene</a></li>
<li><a href="#choosing-the-host-organism-and-the-vector"><span class="toc-section-number">2.2</span> Choosing the host organism and the vector</a><ul>
<li><a href="#bacteria-and-plasmid-vectors"><span class="toc-section-number">2.2.1</span> Bacteria and plasmid vectors</a></li>
<li><a href="#eukaryotic-hosts"><span class="toc-section-number">2.2.2</span> Eukaryotic hosts</a></li>
<li><a href="#high-throughput-strategy"><span class="toc-section-number">2.2.3</span> High-throughput strategy</a></li>
</ul></li>
<li><a href="#lic"><span class="toc-section-number">2.3</span> Constructing the vector</a></li>
<li><a href="#transformationtransfection"><span class="toc-section-number">2.4</span> Transformation/Transfection</a></li>
</ul></li>
<li><a href="#protein-extraction-and-purification"><span class="toc-section-number">3</span> Protein Extraction and Purification</a><ul>
<li><a href="#extraction"><span class="toc-section-number">3.1</span> Extraction</a><ul>
<li><a href="#cell-lysis"><span class="toc-section-number">3.1.1</span> Cell lysis</a></li>
<li><a href="#removing-cellular-components-and-debris"><span class="toc-section-number">3.1.2</span> Removing cellular components and debris</a></li>
</ul></li>
<li><a href="#purification"><span class="toc-section-number">3.2</span> Purification</a><ul>
<li><a href="#salting-out-and-dialysis"><span class="toc-section-number">3.2.1</span> Salting out and dialysis</a></li>
<li><a href="#chromatography"><span class="toc-section-number">3.2.2</span> Chromatography</a></li>
<li><a href="#monitor"><span class="toc-section-number">3.2.3</span> Monitoring the progress of purification</a></li>
</ul></li>
<li><a href="#preparation-for-crystallography"><span class="toc-section-number">3.3</span> Preparation for crystallography</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>

<div id="title" class="section level2 unnumbered">
<h2>Title</h2>
<p><strong>You want to over-express a protein secreted by the malarial parasite. It is cysteine-rich and has no known structure. Outline an expression and purification strategy, using both traditional and high-throughput methods, to produce a sample suitable for protein crystallography. Include a description of how you would assess the quality of the purified protein.</strong></p>
</div>
<div id="overview" class="section level1">
<h1><span class="header-section-number">1</span> Overview</h1>
<p>The general workflow for obtaining a sample of (malarial) protein is summarised as follows:</p>
<ol style="list-style-type: decimal">
<li>Identifying and amplifying the gene of interest (GOI)</li>
<li>Constructing the vector</li>
<li>Transformation/transfection of host cells</li>
<li>Screening for the most viable (high yield and solubility) transformants</li>
<li>Extracting and purifying proteins</li>
</ol>
<p>For crystallography studies, a protein sample with high purity, solubility and yield must be obtained. The expression and purification strategies depend on the characteristics of the protein of interest, such as size, shape, intrinsic solubility, requirement of post-translational modifications and/or cofactors, presence of disulfide bridges, <em>in vitro</em> stability.</p>
</div>
<div id="protein-expression" class="section level1">
<h1><span class="header-section-number">2</span> Protein Expression</h1>
<div id="design" class="section level2">
<h2><span class="header-section-number">2.1</span> Identifying and amplifying the gene</h2>
<p>If the coding sequence of the protein is unknown but the protein is purified, mass spectrometry (or chemical methods such as Edman degradation) can be used to determine the amino acid sequence of the protein, which can then be used to search for the the corresponding DNA sequence.</p>
<p>Usually, instead of using the full DNA sequence, cDNA is used in cloning because it does not contain introns. cDNA is made by:</p>
<ol style="list-style-type: decimal">
<li>extracting total RNA from cells by TRIzol and then isolate the mature mRNA by affinity chromatography (with poly-T coated resins which binds to poly-A tails of mature mRNA)</li>
<li>reverse transcription of the template DNA strand by viral reverse transcriptase (RT) followed by RNA degradation (often by the RNAse H activity of RT)</li>
<li>synthesis of the coding strand by DNA polymerase</li>
</ol>
<p>Often, the latter two steps are done repeatedly in RT-PCR to amplify the DNA fragment, if the sequence of the GOI is known and specific<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> primers are designed. The primers include additional restriction enzyme cleavage sites for the convinience of traditional ligation-dependent cloning, or other sequences for ligation-independent cloning (see Section <a href="#lic">2.3</a>).</p>
<p>If the protein is already known to have a match in malarial cDNA library, we can skip most of the above steps and PCR-amplify the cDNA of interest from the library directly.</p>
<p>When we know the sequence of GOI, we can do a BLAST to find possible homologous proteins, and study any relavant scientific literature. The structure of some homologous proteins might have been solved previously and we can refer to their expression and purification strategies, which might prevent some waste of time on trial-and-errors. Bioinformatic analyses, such as disorder prediction using the RONN algorithm and generation of 3D homology models using the Phyre2 server, might also be helpful.</p>
<p>Codon optimisation might also be needed. Codon usage can vary significantly between species as well as between genome types such as nuclear DNA and mitochondrial DNA. For example, the frequencies of codons AGG, AGA, and CGA (which code for arginine) in H. sapiens are 11.4, 11.5 and 6.3, respectively, while the corresponding frequencies in E. coli are 1.2, 2.1 and 3.6. Codons that occur in high frequency in <em>Plasmodium</em> but in low frequencies in <em>E. coli</em> would result in a condition where the pool of tRNA for that codon will be so low as to become depleted. When the rare tRNAs are depleted to produce the recombinant protein, proliferation of the host cells is restricted, leading to low yield. This problem may be partially solved by the use of an inducible expression vector (e.g. IPTG), but when the foreign gene contains a large number of rare codons, this is not enough. There are two further approaches to improve the yield. First, the rare codons in the foreign genes can be substituted with prevalent codons, but this is not always reliable. Second, host cells can be transformed with the genes that code for the rare tRNAs. This approach is reliable and several such cell lines are commercially available. Custom host cells can also be made using plasmids.</p>
<p>We might fail in crystallising the single full-length protein (often because of low solubility). In that case, we can try co-expressing another protein or add an appropriate cofactor and crystallising the resulting complex. If this still fails, we might consider truncating the GOI, e.g. removing a very hydrophobic region that results in low solubility. In high-throughput approaches, such as the pipeline adopted by Oxford Protein Production Facility (OPPF), many variants are constructed in parallel to maximise output.</p>
</div>
<div id="choosing-the-host-organism-and-the-vector" class="section level2">
<h2><span class="header-section-number">2.2</span> Choosing the host organism and the vector</h2>
<div id="bacteria-and-plasmid-vectors" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Bacteria and plasmid vectors</h3>
<p>Typically and traditionally, the gene of interest (GOI) is incorporated into a plasmid vector via a ligation-dependent mechanism, then the plasmid is introduced into bacterial (<em>E. coli</em>) cells for expression. There are also ligation-independent methods for making vectors, which is more robust and easy to use (only need to design one ‘adaptor’ parts of the primers once), making it suitable for high-throughput methods.</p>
<p>Bacteria-based protein expression is time- and cost-efficient, but it has two major limitations:</p>
<ol style="list-style-type: decimal">
<li>the insertion size of a plasmid is small, typically 2-10 kb (bacterial artificial chromosomes (BAC) with greater insertion sizes can be used, but the procedures are more complex)</li>
<li>Bacteria might be unable provide the appropriate protein folding environment (including chaperones) and/or post-translational modifications that can be crucial to protein’s functions.</li>
</ol>
<p>Given the protein of interest is cysteine-rich, it is susceptible that disulfide bridges are present if this protein is also extracellular. The normal intracellular reducing environment disfavours disulfide bridge formation, but <em>Origami</em> and related strains have an oxidising intracellular environment due to mutant thioredoxin reductase and glutathione reductase, and they are used to express proteins that require disulfide bonds to achieve their correctly-folded conformation.</p>
<p>The plasmid should contain selectable markers for easy identification of successful transformants. The combination of <em>Amp<sup>R</sup></em> and <em>lacZ</em><span class="math inline">\(\alpha\)</span> is a common choice, as explained in Figure <a href="#fig:pet">2.1</a>.</p>
<p>In addition, the <strong>promoter</strong> should be strong and tightly regulated with minimum or complete lack of basal level transcription under uninduced conditions. The IPTG inducible T7 promoter system (as in pET plasmid) is one of the common choices. IPTG is an analog of allolactose, which can bind to and silence <em>lac</em> inhibitor, de-inhibiting the <em>lac</em> operator. IPTG is not digested by the bacteria, and thus its constant concentration continuously induces protein expression. The <em>lac</em> operator may be also made symmetric, strictly preventing transcription when uninduced.</p>
<p>Some commerially available plamids also include tags added to the N- or C-terminus of the insertion site (multiple cloning site), which aids protein purification. This can be a polyhistidine tag, a fusion protein (e.g. GST) or a short recognisable peptide, and often a cleavage sequence is also introduced for removal of the tags.</p>
<div class="figure"><span id="fig:pet"></span>
<img src="../img/pETBlue.jpg" alt="The pETBlue-2 plasmid vector" width="100%" />
<p class="caption">
Figure 2.1: The pETBlue-2 plasmid vector
</p>
</div>
<p>Plasmid-based bacterial transformation is usually permanent because plasmids can replicate themselves and be distributed into daughter cells after cell division.</p>
</div>
<div id="eukaryotic-hosts" class="section level3">
<h3><span class="header-section-number">2.2.2</span> Eukaryotic hosts</h3>
<p>If the protein is incompatible for expression in bacteria, an eukaryotic host is chosen. This is typically yeast, insect, or human cells.</p>
<p>Transfection of eukaryotic cells can be transient or permanent. Eukaryotic cells do not transcribe plasmids, and the DNA fragment of interest must integrates into the host cell’s genome to produce a stable cell line–which is a rare event and takes more time to achieve. If we had to use a eukaryotic host, transient transfection may be used first to screen for the most viable construct, then a stable cell line is made with this construct for higher yield and convience for later use.</p>
</div>
<div id="high-throughput-strategy" class="section level3">
<h3><span class="header-section-number">2.2.3</span> High-throughput strategy</h3>
<p>High-throughput protein production aims to use robots to automate the steps in making expression constructs and screening them.</p>
<p>Take the OPPF pipeline for example, when the GOI is an eukaryotic protein, they test expressions with different vector constructs (with varying amino acid start and end points, fusion partners, and plasmids) in both <em>E. coli</em> and insect cells in parallel (Figure <a href="#fig:htp-ox">2.2</a>). The expresison screening is automated and performed in 96-well format.</p>
<div class="figure"><span id="fig:htp-ox"></span>
<img src="../img/htp-ox.png" alt="SDS-PAGE gel after NiNTA magnetic bead purification showing expression of constructs for DDR1 in (A) *E. coli* and (B) insect cells via the baculovirus system. The names of constructs correspond to amino acid start and end points and the vector being used (pOPINE, pOPINF or pOPINJ, which confers a C-terminal His-tag, an N-terminal His-tag, or an N-terminal His-GST tag, respectively)" width="100%" />
<p class="caption">
Figure 2.2: SDS-PAGE gel after NiNTA magnetic bead purification showing expression of constructs for DDR1 in (A) <em>E. coli</em> and (B) insect cells via the baculovirus system. The names of constructs correspond to amino acid start and end points and the vector being used (pOPINE, pOPINF or pOPINJ, which confers a C-terminal His-tag, an N-terminal His-tag, or an N-terminal His-GST tag, respectively)
</p>
</div>
<p>they also used ligation-independent cloning methods (Section <a href="#lic">2.3</a>) to reduce variability and thus simplify the workflow.</p>
</div>
</div>
<div id="lic" class="section level2">
<h2><span class="header-section-number">2.3</span> Constructing the vector</h2>
<p>The DNA fragment we obtained should be incorporated into a vector before they can be introduced into the host cell. Usually, a recombinant plasmid is made first and amplified in bacteria. Then, if necessary, it can be transferred into other vectors such as viruses and liposomes, depending on the strategies. Here I describe two ways of making recombinant plasmids: traditional restriction enzyme and ligation-based plasmid cloning, and T4 polymerase-based ligation-independent cloning (LIC).</p>
<div class="figure"><span id="fig:lic"></span>
<img src="../img/lc-lic.jpg" alt="Left, traditional cloning based on restriction enzyme and ligase; right, ligase-independent cloning based on T4 polymerase" width="100%" />
<p class="caption">
Figure 2.3: Left, traditional cloning based on restriction enzyme and ligase; right, ligase-independent cloning based on T4 polymerase
</p>
</div>
<p>As shown in Figure <a href="#fig:lic">2.3</a>, in the ligation-dependent method, the PCR product contains flanking restriction sites (came from the designed primers) to be recognised by a restriction enzyme (e.g. EcoRI). The plasmid also contain the restriction site for EcoRI (located in the multiple cloning site, see Figure <a href="#fig:pet">2.1</a>). Cleavage by EcoRI results in complementary ‘sticky ends’, which facilitate annealing of GOI to the plasmid. The sticky sequences are shart (usually 3-5 bp), thus, to overcome low stability, ligation must be done immediately.</p>
<p>The ligation-based cloning are not proper for high throughput protein production (HTPP) projects due to several disadvantages:</p>
<ul>
<li>incomplete DNA digestion and poor ligation yields</li>
<li>each GOI has to be inspected for any internal restriction sites (for exclusion from the choices of the ligation sequence)</li>
<li>unwanted amino acids can be introduced to the expressed protein</li>
<li>the GOI can only be cloned in the vector position where the selected restriction site is present</li>
</ul>
<p>Ligation-independent cloning (LIC) overcomes many of the problems described above. In a T4 DNA polymerase-dependent approach, the sticky sequences are made by the 3’-to-5’ exonuclease activity of T4 polymerase. These sequences are long, allowing formation of stable recombinant plasmid without <em>in vitro</em> ligase treatment. The nicks in sugar-phosphate backbone are later fixed by host ligase.</p>
<p>The obvious elegance of T4-LIC is that once the sticky sequence is designed and constructed, it can be used for any GOI. As T4 polymerase (exonuclease activity) always proceeds from 3’ ends, any internal sticky sequences will not be disrupting.</p>
</div>
<div id="transformationtransfection" class="section level2">
<h2><span class="header-section-number">2.4</span> Transformation/Transfection</h2>
<p>Transformation is the process of delivering GOI-containing foreign DNA into host cells. For eukaryotic cells, this is more often known as transfection (because ‘transformtion’ has other meanings). Many chemical/physical transformation/transfection methods are generally done by making transient holes on host cell plasma membrane (technically, ‘inducing competence’ in host cells). Here are two examples.</p>
<p>Heat-shock transformation is used for small vectors such as plasmids. The general procedures are:</p>
<ol style="list-style-type: decimal">
<li>Host cells are incubated in a solution containing divalent cations (typically CaCl<sub>2</sub>) on ice.</li>
<li>CaCl<sub>2</sub> partially disrupts the cell membrane, which allows the recombinant DNA enter the host cell. Such cells are called competent cells.</li>
<li>Cells are exposed to a heat pulse (heat shock), and the thermal imbalance causes the entry of DNA through disrupted plasma membrane.</li>
</ol>
<p>Electroporation can be used for larger vectors such as BAC and PAC<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>. The general procedures are:</p>
<ol style="list-style-type: decimal">
<li>Host cells are placed into a cuvette, together with the vector. The cuvette is connect to electrodes.</li>
<li>The cuvette containing the mixture is subjected to intense electric pulses (2500 V/cm for bacteria, lower for animal and plant cells) each lasting for only a few milliseconds</li>
<li>Most cells would die under such treatment, but for those survived, their membranes are polarised by the electric field and are disrupted, and DNA enters through the pores. Finally, the membrane reseals after the treatment.</li>
</ol>
<p>Depending on the identity of the host cell and the vector, other methods are also available. These are not directly relevant with our goal of expressing a malarial protein, so here is a simple listing:</p>
<ul>
<li>cosmid and fosmid vectors are introduced into bacterial cells via bacteriophages</li>
<li>calcium phosphate co-precipitation for mammalian cells</li>
<li>microinjection for animal cells</li>
<li>microprojectile bombardment for plant cells</li>
<li>adenovirus and lentivirus vectors for mammalian cells</li>
<li><em>Agrobacterium</em>-mediated transformation for plant cells</li>
<li>genome editing techniques based on ‘designer nucleases’ such as ZFN, TALEN and CRISPR-Cas9 with homology directed repair</li>
</ul>
</div>
</div>
<div id="protein-extraction-and-purification" class="section level1">
<h1><span class="header-section-number">3</span> Protein Extraction and Purification</h1>
<p>After proliferation of transformed/transfected cells, we can extract and purify the protein fo interest.</p>
<div id="extraction" class="section level2">
<h2><span class="header-section-number">3.1</span> Extraction</h2>
<div id="cell-lysis" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Cell lysis</h3>
<p>The first step is to lyse, or homogenise the cells to release the protein. To do this, cells can be subjected to osmotic shock or ultrasonic vibration, forced through a small orifice, or ground up in a blender. (if a membrane protein is to be extracted, detergents are normally used, although detergent-free methods have been developed recently)</p>
</div>
<div id="removing-cellular-components-and-debris" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Removing cellular components and debris</h3>
<p>Repeated centrifugation, each time with a higher speed (i.e. differential centrifugation), removes many impurities with large sizes. Organelles (and small vesicles and microsomes) sediment (forming pallets) while proteins remain in the solution (as supernatant).</p>
<p>A finer degree of separation can be achieved by layering the homogenate in a thin band on top of a salt solution that fills a centrifuge tube. When centrifuged, the various components in the mixture move as a series of distinct bands through the solution, each at a different rate, in a process called velocity sedimentation. A glucose gradient is established to protect the bands from convective mixing.</p>
</div>
</div>
<div id="purification" class="section level2">
<h2><span class="header-section-number">3.2</span> Purification</h2>
<p>Classical methods for separating proteins depend on variable protein properties, including solubility, size, charge, and binding affinity. In most cases, protein mixtures are sequentially subjected to different separation methods, each based on a different property. After each step of purification, the fractions are assayed (see Section <a href="#monitor">3.2.3</a>).</p>
<div id="salting-out-and-dialysis" class="section level3">
<h3><span class="header-section-number">3.2.1</span> Salting out and dialysis</h3>
<p>The addition of certain salts in the right amount can selectively precipitate the protein of interest, while others remaining in solution. The precipitation is removed by centrifugation.</p>
<p>Dialysis is then used to remove the salt from the solution containing the protein of interest. The protein mixture is place inside a dialysis bag (i.e. made of size-selective permeable membrane) and placed in a buffer solution with low salt concentration. Salt in the protein-containing solution then diffuse out, leaving proteins inside the bag.</p>
</div>
<div id="chromatography" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Chromatography</h3>
<p>In column chromatography, the solution containing proteins is passed through a column containing a solid matrix (resin). Different proteins are retarded to different extents by their interaction with the matrix, and they can be collected separately as they flow out of the bottom of the column. Depending on the choice of matrix, proteins can be separated according to their charge (ion-exchange chromatography),nhydrophobicity (hydrophobic chromatography), size (gel-filtration/size-exclusion chromatography), or ability to bind to particular small molecules or to other macromolecules (affinity chromatography). The last two are the most important for high throughput protein purification are affinity chromatography and size-exclusion chromatography.</p>
<p>Affinity chromatography extract protein according to the tags we attached to target proteins. For example, His-tagged proteins bind to Nickle-coated resin and HSV epitope tagged proteins bind to resin coated with corresponding antibodies (polyHis-Ni interactions often have lower specificity compared to enzyme-substrate and epitope-antibody interactions)</p>
<p>Size exclusion chromatography cleans abnormal protein aggregates based on their large size.</p>
<p>A problem experienced by column chromatographic methods is diffusional spreading (i.e. proteins that are going down faster also diffuse upwards, mixing with proteins going slower), which reduces the resolution. The degree of diffusional spreading increases with time during which proteins stay in the column. HPLC (high-performance liquid chromatography) solves this problem. It makes use of high-pressure pumps that speed the movement down the column, so that the time of travelling, and hence the diffusional spreading, is greatly reduced, leading to higher resolution.</p>
</div>
<div id="monitor" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Monitoring the progress of purification</h3>
<p>As purification progresses:</p>
<ol style="list-style-type: decimal">
<li>the total amount of protein should decrease as unwanted proteins are removed and some target proteins are lost</li>
<li>amount of target proteins should also decrease because they are unavoidably lost e.g. by nonspecific attachment to the purification apparatus or washed away</li>
<li>the proportion of the target protein (i.e. purity) should increase as unwanted proteins are removed</li>
</ol>
<p>The simplest way to monitor these changes is running a SDS-PAGE gel electrophoresis after each purification step. The bands should progressively decrease to one (which correspond to the target protein), and the thickness of this band should increase because the concentration of target protein increases (its absolute amount decreases, but as we removed the fractions without the target protein in chromatography, its concentration should increase).</p>
<p>To assess whether the protein is correctly folded and functional, specific assays are used For example, if the target protein is an enzyme, it can be assayed using its substrate; if it is a protein which binds to another molecule, their interaction can be probed by surface plasmon resonance. Circular dichroism also helps to assess the foldedness of the target protein.</p>
</div>
</div>
<div id="preparation-for-crystallography" class="section level2">
<h2><span class="header-section-number">3.3</span> Preparation for crystallography</h2>
<p>For crystallisation, the protein sample must be homogenous and correctly folded. They can be checked by the methods descibed in Section <a href="#monitor">3.2.3</a>.</p>
<p>If the affinity tag needs to be removed (which is often required for structual studies), a protease cleavage site is often incorporated before or after the fusion tag and the cleavage can be conducted either in solution following purification (the protease themselves are tagged) or immediately after enzyme capture in situ on the chromatography resin itself. AKTA Express (GE Healthcare) is an elegant procedure for on-column cleavage coupled to multidimensional chromatography that is highly amenable to high throughput protein purification.</p>
<p>There are three methods to grow protein crystalls, namely ‘hanging drop’, ‘sitting drop’, and microdialysis, but the underlying principles are similar. The protein is first dissolved in a ‘crystallisation cocktail’ droplet and concentrated to 2-50 mg/ml, then it is allowed to equilibrate with a more concentrated reservoir solution of the cocktail with a volume ratio of 1:1. As the protein droplet becomes supersaturated, it <em>may</em> start to crystalise. Robots are commonly used for automatic screeening and optimisation of crystallisation conditions. If crystallisation fails under all conditions, we can try co-crystalising with a ligand. If this still fails, we might consider using a truncated protein, as described in Section <a href="#design">2.1</a>.</p>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Doyle-2008">
<p>Doyle, Sharon A. 2008. <em>HIgh Throughput Protein Expression and Purification: Methods and Protocols</em>. Edited by John W. Walker. Methods in Molecular Biology 498. Humana Press.</p>
</div>
<div id="ref-ThermoFisher-2019">
<p>ThermoFisher. 2019. <a href="https://www.thermofisher.com/uk/en/home/references/gibco-cell-culture-basics/transfection-basics/gene-delivery-technologies.html">https://www.thermofisher.com/uk/en/home/references/gibco-cell-culture-basics/transfection-basics/gene-delivery-technologies.html</a>.</p>
</div>
<div id="ref-Vicentelli-2019">
<p>Vicentelli, Renaud. 2019. <em>HIgh-Throughput Protein Production and Purification: Methods and Protocols</em>. Edited by John M. Walker. Vol. 2025. Methods in Molecular Biology. New York, NY: Humana Press.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>a specific primer will bind the mRNA of interest but not any other mRNAs. Online tools such as Primer3 and Primer-BLAST (NCBI) aids design of specific primers.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>P1-derived artificial chromosome<a href="#fnref2" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/protein-structure-and-function.html" class="archive-item-link">
                Protein Structure and Function 
              </a>
              <span class="archive-item-date">
                2019-10-24
              </span>
              <div class="article-summary" style="display: none;">
                
                  Choose 3 structural papers. They can be crystallography, NMR, CryoEM etc. Explain why the protein is important and what the structure tells you about function (why you find it interesting) but most importantly go to the methods section (it may be supplementary information especially if you choose high profile paper) and explain how the protein was purified. The techniques will be those covered in Matt Higgins&#39; lecture series. Bring notes no essay.<hr>
                
                


<div id="principles-of-x-ray-crystallography" class="section level1">
<h1>Principles of X-ray crystallography</h1>
<div id="scattering" class="section level2">
<h2>Scattering</h2>
<ul>
<li>When light hits matter…
<ul>
<li>Vibration<span class="math inline">\(\rightarrow\)</span>scattering (all directions)</li>
<li>Energy level transitions<span class="math inline">\(\rightarrow\)</span>absorption and emission (fluorescence)</li>
<li>Photochemical reactions (e.g. photosynthesis)</li>
</ul></li>
<li>scattering can give rise to refraction and diffraction</li>
<li>experiments on scattering
<ul>
<li>turbidity (reduction in intensity)</li>
<li>angular dependence</li>
<li>changes in <span class="math inline">\(\lambda\)</span></li>
</ul></li>
</ul>
</div>
<div id="wnt3-fz8-complex-hirai-2019" class="section level2">
<h2>Wnt3-Fz8 Complex <span class="citation">(Hirai et al. 2019)</span></h2>
<ul>
<li>crystals of lysine-methylated and deglycosylated human Wnt3 (hWnt3)-mFz8 CRD complex were obtained by X-ray crystal structure was solved by molecular replacement and refined to a resolution of 2.8Å.</li>
</ul>
<div id="difficulties-in-crystallisation-and-their-solutions" class="section level3">
<h3>Difficulties in crystallisation and their solutions</h3>
<ul>
<li>Strong hydrophobic property of Wnt proteins caused by a covalent lipid modification</li>
</ul>
<p>Optimisation and chemical modifications conducted to ensure high expression yields, enhanced solubility and sample homogeneity</p>
<div id="solubility" class="section level4">
<h4>Solubility</h4>
<ul>
<li>Failed attempts on making crystallization constructs
<ul>
<li>afamin can solubilize Wnt proteins; when coexpressed and complexed, Wnt3 and 3a can be purified to homogeneity. However, diffraction-quality crystals could not be obtained after repeated trials.</li>
<li>coexpression with mFz8 CRD after Janda’s success; but found purified Wnt3a-Fz8 CRD complex still remained hydrophobic and required detergents during concentration, which hampered crystallisation</li>
</ul></li>
<li>Successful: N-terminal truncation of Wnts to mimic the cleavage of the N-terminal peptide by a metalloprotease Tiki, which has been reported to reduce the overall hydrophobicity
<ul>
<li>N-terminal 20 residues were removed from hWnt3/mWnt3a constructs</li>
<li>PA-hWnt3(<span class="math inline">\(\Delta\)</span>N)/PA-mWnt3a(<span class="math inline">\(\Delta\)</span>N) coexpressed with mFz8 CRD C-terminally fused with modified human Fc.</li>
<li>confirmed that complexes were fully soluble in aqueous buffer and could be concentrated to &gt; 5 mg/ml without detergents</li>
</ul></li>
</ul>
</div>
<div id="optimisation" class="section level4">
<h4>Optimisation</h4>
<ul>
<li>Initially (following Janda) attached normal (with hinge region) human IgG1 Fc to the C-terminal of mFz8 CRD, intervened by a TEV protease cleavage sequence for the later Fc removal
<ul>
<li>although complex formed with high yield, it could not be cleaved at all; different linker lengths showed no improvements</li>
</ul></li>
<li>decided to use IdeS protease to remove Fc.</li>
</ul>
<div id="refs" class="references">
<div id="ref-Hirai-2019">
<p>Hirai, Hidenori, Kyoko Matoba, Emiko Mihara, Takao Arimori, and Junichi Takagi. 2019. “Crystal Structure of a Mammalian Wnt–Frizzled Complex.” <em>Nature Structural &amp; Molecular Biology</em> 26 (5): 372–79. <a href="https://doi.org/10.1038/s41594-019-0216-z">https://doi.org/10.1038/s41594-019-0216-z</a>.</p>
</div>
</div>
</div>
</div>
</div>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/principle-of-electron-transfer-and-proton-transport.html" class="archive-item-link">
                Principle of electron transfer and proton transport 
              </a>
              <span class="archive-item-date">
                2019-10-23
              </span>
              <div class="article-summary" style="display: none;">
                
                  What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p><strong>Using the physical chemistry principles, discuss factors which are contributed in the favourability of an electron transfer reactions between cofactors.</strong></p>
<p>The reduction potential, <span class="math inline">\(E\)</span>, is used to quantify favourability of gaining electrons (getting reduced). <span class="math inline">\(E^{\circ\prime}\)</span> is the reduction potential measured under biological standard conditions (i.e. pH = 7, oxidised and reduced species are 1M, 298K and 1 atm). When concentrations of the oxidised and reduced species differ:</p>
<p><span class="math display">\[E=E^{\circ\prime} + \dfrac{RT}{zF}\ln\left(\dfrac{\text{[reduced]}}{\text{[oxidised]}}\right)\]</span></p>
<p>For two half equations, A<sup>+</sup> + <span class="math inline">\(e^-\rightarrow\)</span> A and <span class="math inline">\(\text{B} + e^- \rightarrow\)</span> B<sup>-</sup> with <span class="math inline">\(E\)</span> of <span class="math inline">\(x\)</span> and <span class="math inline">\(y\)</span> respectively, a full reaction equation can be constructed, which can be A<sup>+</sup> + B<sup>-</sup> <span class="math inline">\(\rightarrow\)</span> A + B (with <span class="math inline">\(E_\text{r}\)</span> = x - y) or the reverse (with <span class="math inline">\(E_\text{r}\)</span> = y - x); the one with positive <span class="math inline">\(E_\text{r}\)</span> (reaction potential) will actually occur.</p>
<p>Electron transfers occur because electrons are moving down to a lower energy level. In simple cases, atoms with greater electronegativity have greater affinity for electrons and thus a more positive reduction potential, and <em>vice versa</em>. This is the case for strong oxidants F<sub>2</sub> and Cl<sub>2</sub> (electronegative; highly positive <span class="math inline">\(E\)</span>) and strong reductants Li and Na (electropositive, highly negative <span class="math inline">\(E\)</span>). In most cases, however, other factors, such as change in state, conformation, or interactions with water, will significantly affect the reduction potential. For example, Cu and Ag is less electronegative than hydrogen, but their monovalent ions are more easily reduced than H<sup>+</sup>. That is mainly because the formation of metallic solid upon reduction is highly exergonic. Similarly, although the atoms involved in the redox reactions in the repiratory chain are not variable (Cu and Fe), they have different <span class="math inline">\(E\)</span> due to the enviornment (e.g. proximity to the P/N face, identity of ligands).</p>
<p>The series of cofactors along the electron transport chain have a generally steadily increasing <span class="math inline">\(E^{\circ\prime}\)</span>, so electrons are moved toward increasingly lower energy levels. There are some exceptions to this trend, but <span class="math inline">\(E_\text{r}\)</span> would be made positive in these cases by making a concentration difference (usually by means of ‘pulling’ from the next redox centre with favourable <span class="math inline">\(E\)</span>).</p>
<p>While the thermodynamic feasibility of the redox reactions is solely determined by the difference in the reduction potentials (which is proportional to Gibbs free energy), the kinetics (rates) are affected by more factors.</p>
<p>Electrons are passed from one centre to another by <em>tunnelling</em>. Specifically, the wave function for an electron in the donor shows there is a probability that the electron will be found at an acceptor some distance away. <em>Tunnelling</em> is independent of temperature and orientation of chemical groups, but is instead influenced by 1) the distance between the electron donor and acceptor, 2) the redox potential difference, <span class="math inline">\(E\)</span>, and 3) the ‘reorganisation energy’, <span class="math inline">\(\lambda\)</span>.</p>
<p>Within each respiratory complex, the redox centres are close to each other—no more than 14Å apart (the maxium 14Å is found between N5 and N6a Fe-S clusters in complex I). Ubiquinone has binding sites in complex I (II) and III in close proximity to the redox centres from which it receives and to which it delivers electrons, and the same is true for cyt <em>c</em> (which binds to complex III and IV). This arrangement makes the numerous electron transfer steps not the limiting factor in the respiratory chain.</p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p><strong>What mechanisms are employed by mitochondrial electron transfer proteins to generate a proton-motive force (pmf)? Using the concept of membrane capacitance, explain why the dominant component of the pmf in mitochondria is an electrical potential rather than a pH gradient.</strong></p>
<div id="two-proton-translocation-models-loop-and-pump" class="section level2">
<h2>Two Proton Translocation Models: Loop and Pump</h2>
<p>Mechanisms of coupling redox reactions to proton translocation can be classified into two models: the chemical loop, and the conformational pump.</p>
<p>In the loop model shown in Figure <a href="#fig:loop">1</a>, the hydrogens (protons plus electrons) from the substrate (AH<sub>2</sub>) first reduce an mediator molecule (B) on the N side, and this mediator is reoxidised, forcing protons to be released to the P side via one pathway and electrons to flow back to the N side via another pathway. The electrons then reduce another species (C) using protons from the N side. This mechanism is employed in the respiratory chain, where ubiquinone serves as the ‘mediator’ molecule and ETC components from Rieske Fe-S protein in complex III to Cu<sub>B</sub> centre in complex IV as the electron pathway. The number of protons translocated in this way has to follow a fixed stoichiometry (i.e. 1 proton per electron). (Not shown in this diagram, the reduction of Q near the N face driven by QH<sub>2</sub> oxidation near the P face in the Q cycle is also an example of this mechanism.)</p>
<div class="figure" style="text-align: c"><span id="fig:loop"></span>
<img src="../img/etc-mech-1.png" alt="The loop model. Left, generic form; right, simplified example found in the respiratory chain." width="100%" />
<p class="caption">
Figure 1: The loop model. Left, generic form; right, simplified example found in the respiratory chain.
</p>
</div>
<p>The key idea of the pump mechanism is the conformational change driven by redox reactions. Conformational change can lead to alternating proton accessibility from the two faces and differential pKa values of proton binding sites, resulting in proton translocation (explained in Figure <a href="#fig:pump">2</a>). The number of protons pumped in this way does not have to follow a fixed stoichiometry within the thermodynamic constraints (i.e. <em>E</em><sub>r</sub>/n must be greater than ∆p). This mechanism is thought to be employed in complex I and IV of the respiratory chain. However, recent studies have shown that complex IV may adopt a unique mechanism, as explained later in this essay.</p>
<div class="figure" style="text-align: c"><span id="fig:pump"></span>
<img src="../img/etc-mech-2.jpg" alt="The conformational pump mechanism. When the &quot;pump&quot; protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation." width="100%" />
<p class="caption">
Figure 2: The conformational pump mechanism. When the “pump” protein is reduced, the proton binding site is exposed to the N face and with high pKa, promoting proton binding; upon oxidation, the proton binding site is open to the P face, and pKa drops, promoting proton dissociation.
</p>
</div>
</div>
<div id="complex-i" class="section level2">
<h2>Complex I</h2>
<p>As shown in Figure <a href="#fig:3m9s">3</a>, bacterial complex I (from <em>Thermus thermophilus</em>) comprises two domains. The hydrophobic, transmembrane domain (a.k.a. membrane arm) has a proton-translocating P-module and subunit NuoH. The hydrophilic domain (a.k.a. peripheral domain) has an NADH-oxidising dehydrogenase module (N-module) which feeds electrons to the electron-transferring chain of Fe-S clusters, and the connecting Q-module, which reduces ubiquinone. Mammalian mitochondrial complex I is similar in gross organisation and function, although it possesses more subunits.</p>
<div class="figure" style="text-align: c"><span id="fig:3m9s"></span>
<img src="../img/3M9S.png" alt="Annotated structure of bacterial complex I. PDB accession code: 3M9S" width="100%" />
<p class="caption">
Figure 3: Annotated structure of bacterial complex I. PDB accession code: 3M9S
</p>
</div>
<div id="hydrophilic-domain-peripheral-arm" class="section level3">
<h3>Hydrophilic domain (peripheral arm)</h3>
<p>All FMN and the seven (bacteria have additional two redundant ones) Fe-S centres reside in the hydrophilic domain. FMN, which is located at the extremity of the hydrophilic arm, accepts two electrons from NADH<sub>2</sub> and transfers them one by one to the nearest Fe-S centre (N3). Electrons then flow from N3 through a series of isopotential Fe-S centres and finally to N2, whose <span class="math inline">\(E^\circ\)</span> is significantly less negative (-100 mV) <span class="citation">(Sazanov and Hinchliffe 2006)</span>. The ubiquinone is then reduced by two one-electron gaining from N2; the existence of at least two forms of semiquinone is identified by electron paramagnetic resonance.</p>
</div>
<div id="hydrophobic-domain-membrane-arm" class="section level3">
<h3>Hydrophobic domain (membrane arm)</h3>
<p>The hydrophobic domain comprises subunits NuoAHJKLMN, of which NuoLMN are homologous to bacterial Mrp Na<sup>+</sup>/H<sup>+</sup> antiporters (characterised by the conserved 14 transmembrane helices).</p>
<p>At the centre of each antiporter-like subunit (NuoLMN), two transmembrane helices are ‘broken’ and each have a kink known as a ‘π-bulge’, and there is a lysine residue in this region (Glu407 for NuoM) critical to proton pumping, confirmed by mutagenesis studies. There are two symmetry-related plausible proton channels lined by hydrophilic/charged residues, one connecting the central lysine on one broken TM to the P face and the other connecting lysine of the other TM to the N face.</p>
<div class="figure" style="text-align: c"><span id="fig:lys"></span>
<img src="../img/membrane-arm.png" alt="Plausible proton paths" width="100%" />
<p class="caption">
Figure 4: Plausible proton paths
</p>
</div>
<p>As shown in Figure <a href="#fig:proton-path">5</a>, NuoL/M/N may work by using conformational changes (induced from the hydrophilic domain) to transfer a proton from the lysines on the helix with N half channel to the other lysine (or Glu in NuoM) on the helix with P half channel, and if the pK<sub>a</sub>s of the two lysines differ, this leads to unidirectional proton transport. This mechanism is in accordance with the general scheme shown in Figure <a href="#fig:pump">2</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:proton-path"></span>
<img src="../img/proton-path.png" alt="Schematic diagram of proton translocation in complex I" width="100%" />
<p class="caption">
Figure 5: Schematic diagram of proton translocation in complex I
</p>
</div>
<p>The three antiporter-related subunits are thought to be responsible for the translocation of 3 protons per 2 <span class="math inline">\(e^-\)</span>. An additional channel is found at the interface of subunits NuoN,K,J,A (according to <span class="citation">Efremov and Sazanov (2011)</span> and <span class="citation">Sazanov (2015)</span> or NuoH,J,K according to <span class="citation">Hirst (2013)</span> and <span class="citation">Nicholls and Ferguson (2013)</span>) and they are thought to act together to pump another proton with a similar mechanism involving protonation/deprotonation of lysine.</p>
</div>
<div id="coupling" class="section level3">
<h3>Coupling</h3>
<p>It is debated how the redox reactions in the hydrophilic domain triggers conformational change, and how this conformational change is transmitted along the hydrophobic domain.</p>
<p>Some features of the Fe-S centre N2 (the last Fe-S centre in complex I) led to hypotheses in which N2 plays a major role in pumping protons<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><sup>,</sup><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>, but currently, the mainstream idea is that the initial conformational change in the hydrophilic domain is contributed mostly by the reduction of ubiquinone.</p>
<p>As claimed by <span class="citation">Wirth et al. (2016)</span>, reduction of ubiquinone is a two-step process, where the Q binding site cycles between a conformation that permits electron transfer (E-state), and a conformation that permits proton transfer (P-state) onto intermediates of ubiquinone. The conformational change associated with this controlled process is assumed to initiate proton translocation.</p>
<p>It was originally suggested that the horizontal HL helix (see Figure <a href="#fig:3m9s">3</a>) in NuoL could move horizontally like a piston to transmit conformational change along the hydrophobic domain<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>, but now, after realising the similarity between NuoH and NuoL/M/N, it is thought that conformational changes associated with π-bulges in the mid-membrane parts are transmitted laterally to NuoN/M/L <span class="citation">(Nicholls and Ferguson 2013)</span>, and, as suggested by <span class="citation">Wirth et al. (2016)</span>, this is achieved by electrostatic coupling in the central hydrophilic axis.</p>
<p>As I mentioned last week <span class="citation">(Shi 2019)</span>, as there is no complete evidence demonstrating how complex I proton pump works, other hypotheses exists, including a radical one recently proposed by <span class="citation">Morelli et al. (2019)</span>, which emphasises the elusiveness of the ‘proton entrance half channel’ (from the middle Lys to the N face) contrasted by the clarity of the ‘proton exit half channel’ (to the P face). Together with another piece of evidence that an obvious proton tunnelling is found at the centre of complex I, Morelli drafted an lateral mechanism of proton circuiting, as shown in Figure <a href="#fig:Morelli2019">6</a></p>
<div class="figure" style="text-align: c"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side" width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019). Protons are only pumped from the middle to the P side, then trasferred through the hydrophilic heads of phospholipids to ATP synthase. After a complete turn in ATP synthase, protons move back through channelling in the middle of the membrane to complex I, but not to the N side
</p>
</div>
</div>
</div>
<div id="complex-iii-cyt-bc1-complex" class="section level2">
<h2>Complex III (cyt <em>bc<sub>1</sub></em> complex)</h2>
<p>Complex III is a homodimer, with each monomer consisting of three subunits: cytochrome b, the Rieske iron-sulfur protein, and cytochrome c1. Each monomer has two binding sites for ubiquinone, called Q<sup>N</sup> and Q<sup>P</sup>, indicating their proximities to two opposite faces.</p>
<p>Uniquinone oxidation and proton translocation is coupled in the Q cycle, as illustrated in Figure <a href="#fig:ciii">7</a>.</p>
<div class="figure" style="text-align: c"><span id="fig:ciii"></span>
<img src="../img/complexIII.png" alt="The Q cycle. Left, stage 1; right, stage 2." width="100%" />
<p class="caption">
Figure 7: The Q cycle. Left, stage 1; right, stage 2.
</p>
</div>
<p>In stage 1, QH<sub>2</sub> binds to Q<sub>N</sub> of one dimer and a Q binds to the Q<sub>P</sub> of another dimer. The QH<sub>2</sub> at Q<sub>N</sub> is oxidised by two electron acceptors in two steps: the first electron is accepted by the Rieske iron-sulfur protein and passed though cytochrome c<sub>1</sub> and finally to cytochrome c, and second other electron is accepted by cytochrome b, passed through its two hemes (b<sub>H</sub> and b<sub>L</sub>) and finally to the Q at Q<sub>P</sub> on the other monomer, forming semiquinone. The two protons of QH<sub>2</sub> are released into the P face (the molecular details is unclear).</p>
<p>In stage 2, the same process is repeated—one QH<sub>2</sub> is oxidised to Q, its protons released and electrons passed 1) onto one cytochrome c and 2) to the other side—but this time the electron acceptor on the other side is semiquinone. Acceptance of an electron by semiquinone, and the addition of two protons from the matrix, produces QH2.</p>
</div>
<div id="complex-iv" class="section level2">
<h2>Complex IV</h2>
<p>Complex IV (cytochrome oxidase) accepts electrons from cytochrome c to reduce oxygen to water and pumps 2 electrons to the P face.</p>
<p>Previous X-ray studies suggested that complex IV exists as dimers with each monomer comprising 13 subunits <span class="citation">(Tsukihara et al. 1996)</span>. However, it was recently shown by <span class="citation">Zong et al. (2018)</span> that complex IV is actually a 14-subunit monomer, and dimerisation was an artifect due to the dissociation of NDUFA4 subunit during the detergent-based purification steps before crystallisation (NDUFA4 hampers dimerisation).</p>
<div id="reduction-of-oxygen" class="section level3">
<h3>Reduction of oxygen</h3>
<p>The key catalytic activities are found in subunit I and II, as shown in Fig. <a href="#fig:complexIV">8</a></p>
<div class="figure" style="text-align: c"><span id="fig:complexIV"></span>
<img src="../img/complexIV.png" alt="Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II." width="100%" />
<p class="caption">
Figure 8: Schematic diagram of complex IV with emphasis on the electron-transport and catalytic activities of subunit I and II.
</p>
</div>
<p>Each cytochrome c first donates one electron to Cu<sub>A</sub> located in the globular domain of subunit II. Cu<sub>A</sub> is a binulear centre, but it undergoes one-electron redox reactions. The electron is then passed onto haem <em>a</em>, and then to haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binuclear centre, where an oxygen molecule is bound. 4 electrons from cyt c and 4 protons from the N phase are required to reduce one oxygen molecule to two water molecules. Cu<sub>B</sub> has three His ligands, and the forth empty coordination site mediates oxygen reduction by holding the intermediates (its malfunction leads to generation of ROS).</p>
</div>
<div id="pump-mechanism" class="section level3">
<h3>Pump mechanism</h3>
<p>For each pair of electrons, 2 H<sup>+</sup> from N phase is consumed to make water and another 2 H<sup>+</sup> is pumped intot he P phase.</p>
<p>According to a recent review by <span class="citation">Wikström and Sharma (2018)</span>, the favoured charge-conpensation mechanism is as follows: (see Figure <a href="#fig:civ-p">9</a>)</p>
<ol style="list-style-type: decimal">
<li>when an electron is transferred from haem <em>a</em> to the haem <em>a</em><sub>3</sub>-Cu<sub>B</sub> binulear centre (BNC), one proton from Glu242 is loaded onto the ‘proton loading site’ (PLS<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a>). This is a purely electrostatic event (not acid-base, as pK<sub>a</sub> of PLS is not high enough).</li>
<li>reprotonated Glu242 transfers another proton to BNC, annihilating the -ve charge in BNC, so H<sup>+</sup> on PLS can leave. There must be a barrier preventing H<sup>+</sup> flowing back to the N side, which can be achieved by raising the energy of the transition state of proton transfer between the Glu242 and PLS.</li>
</ol>
<div class="figure" style="text-align: c"><span id="fig:civ-p"></span>
<img src="../img/civ-p.png" alt="Charge-compensation mechanism of proton pumping in complex IV." width="70%" />
<p class="caption">
Figure 9: Charge-compensation mechanism of proton pumping in complex IV.
</p>
</div>
</div>
</div>
<div id="components-of-pmf-in-chloroplasts-and-mitochondria" class="section level2">
<h2>Components of pmf in chloroplasts and mitochondria</h2>
<p>In mitochondria, the major component of protomotive force (<span class="math inline">\(\Delta p\)</span>) is the membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), while in chloroplasts it is <span class="math inline">\(\Delta\)</span>pH. This is caused by the physiology of these organelles and the electrical properties of membranes.</p>
<p>In chloroplasts, when protons are being pumped into the thylakoid space, the developed membrane potential is balanced out by efflux of K<sup>+</sup> and influx of Cl<sup>–</sup> through voltage-gated channels, such as KEA3 K<sup>+</sup>/H<sup>+</sup> antiporter and voltage-gated VCCN1 Cl<sup>–</sup> channel, so it results in a pure pH difference. TPK3 was originally thought to be one of such channels but was recently shown to be not critical <span class="citation">(Höhner et al. 2019)</span>. There are not many enzymes in the thylakoid lumen, so a low pH is not unsafe. In addition, the high H<sup>+</sup> concentration has regulatroy functions. <span class="citation">(Höhner et al. 2019)</span></p>
<p>If mitochondria were allowed to develop such a pH difference (very basic matrix), this would denature the numerous enzymes, such as those involved in <span class="math inline">\(\beta\)</span>-oxidation and TCA cycle, that operate inside the matrix. In mitochondria, there are not such ion channels to counteract change in <span class="math inline">\(\Delta \Psi\)</span>, so proton translocation results in charge imbalance, leading to <span class="math inline">\(\Delta \Psi\)</span> change<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a>, and a significant membrane potential can be established with only minuscule amount of charge movement.This can be explained by the low capacitance of the membrane. Capacitance is the amount of charge separation needed to develop unit voltage difference, i.e.</p>
<p><span class="math display">\[C=\dfrac{Q}{V}\]</span></p>
<p>Given a constant <span class="math inline">\(V\)</span> (<span class="math inline">\(\Delta \Psi\)</span>), lower capacitance means smaller charge difference is required to build up that voltage difference.</p>
<p>Figure <a href="#fig:capacitance">10</a> shows a simple parallel-plate capacitor, which is analogous to mitochondrial inner membrane.</p>
<div class="figure" style="text-align: c"><span id="fig:capacitance"></span>
<img src="img/capacitance.png" alt="Illustration of a simple parallel-plate capacitor" width="70%" />
<p class="caption">
Figure 10: Illustration of a simple parallel-plate capacitor
</p>
</div>
<p>The electrical capacitance of such a plate can be calculated as:</p>
<p><span class="math display">\[C = \dfrac{\epsilon A}{d}\]</span></p>
<p>where <span class="math inline">\(\epsilon\)</span> is the permittivity of the plate, <span class="math inline">\(A\)</span> is the surface area and <span class="math inline">\(d\)</span> is the distance separating charges. For mitochondrial inner membrane, although <span class="math inline">\(\dfrac{A}{d}\)</span> is large, <span class="math inline">\(\epsilon\)</span> is very small, so the overall capacitance is small.</p>
<div id="a-failed-quantitative-analysis" class="section level4">
<h4>A failed quantitative analysis</h4>
<p>According to <span class="citation">Liang et al. (2017)</span>, the cell membrane capacitance is aroud 10 mF/m<sup>2</sup>, and in the <em>Bioenergetics</em> book <span class="citation">(Nicholls and Ferguson 2013)</span>, the capacitance of the mitochondrial inner membrane takes the same valu. According to a model built by <span class="citation">Mannella, Lederer, and Jafri (2013)</span>, one mitochondria with matrix volume of <span class="math inline">\(9.6\times10^4\)</span> nm^3 would have a inner membrane (cristae) surface area of <span class="math inline">\(5.76\times10^6\)</span> nm<sup>2</sup>.</p>
<p>With these values, to build up a <span class="math inline">\(\Delta \Psi\)</span> of -140 mV, the charge required is:</p>
<p><span class="math display">\[Q=CV=0.01\text{F m}^{-2} \times 5.76 \times 10^{-12}\text{m}^{-2}\times 0.14\text{V}=8.1 \times 10^{-15} \text{ C}\]</span></p>
<p>which corresponds to <span class="math inline">\(8.1 \times 10^{-15}/96450=8.4 \times 10^{-20}\)</span> moles of H<sup>+</sup>, or <span class="math inline">\(8.4 \times 10^{-20}/(9.6\times 10^{-20})=0.875\)</span> M (concentration), which is not realistic…</p>
</div>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Efremov-2011">
<p>Efremov, Rouslan G., and Leonid A. Sazanov. 2011. “Structure of the Membrane Domain of Respiratory Complex I.” <em>Nature</em> 476 (7361): 414–20. <a href="https://doi.org/10.1038/nature10330">https://doi.org/10.1038/nature10330</a>.</p>
</div>
<div id="ref-Hirst-2013">
<p>Hirst, Judy. 2013. “Mitochondrial Complex I.” <em>Annual Review of Biochemistry</em> 82 (1). Annual Reviews: 551–75. <a href="https://doi.org/10.1146/annurev-biochem-070511-103700">https://doi.org/10.1146/annurev-biochem-070511-103700</a>.</p>
</div>
<div id="ref-Hohner-2019">
<p>Höhner, Ricarda, Viviana Correa Galvis, Deserah D. Strand, Carsten Völkner, Moritz Krämer, Michaela Messer, Firdevs Dinc, et al. 2019. “Photosynthesis in Arabidopsis Is Unaffected by the Function of the Vacuolar K<span class="math inline">\(&lt;\)</span>sup<span class="math inline">\(&gt;\)</span>+<span class="math inline">\(&lt;\)</span>/Sup<span class="math inline">\(&gt;\)</span> Channel Tpk3.” <em>Plant Physiology</em> 180 (3): 1322. <a href="https://doi.org/10.1104/pp.19.00255">https://doi.org/10.1104/pp.19.00255</a>.</p>
</div>
<div id="ref-Liang:2017">
<p>Liang, Wenfeng, Yuliang Zhao, Lianqing Liu, Yuechao Wang, Wen Jung Li, and Gwo-Bin Lee. 2017. “Determination of Cell Membrane Capacitance and Conductance via Optically Induced Electrokinetics.” <em>Biophysical Journal</em> 113 (7): 1531–9. <a href="https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006">https://doi.org/https://doi.org/10.1016/j.bpj.2017.08.006</a>.</p>
</div>
<div id="ref-Mannella-2013">
<p>Mannella, Carmen A., W. Jonathan Lederer, and M. Saleet Jafri. 2013. “The Connection Between Inner Membrane Topology and Mitochondrial Function.” <em>Journal of Molecular and Cellular Cardiology</em> 62: 51–57. <a href="https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001">https://doi.org/https://doi.org/10.1016/j.yjmcc.2013.05.001</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Nelson-2017">
<p>Nelson, David L., and Michael M. Cox. 2017. <em>Lehninger Principles of Biochemistry</em>. Book. 7th ed. W. H. Freeman. <a href="https://search.library.wisc.edu/catalog/999964334502121">https://search.library.wisc.edu/catalog/999964334502121</a>.</p>
</div>
<div id="ref-Nicholls-2013">
<p>Nicholls, David G., and Stuart J. Ferguson. 2013. <em>Bioenergetics</em>. Book. 4th ed. Amsterdam: Academic Press, Elsevier.</p>
</div>
<div id="ref-Roessler-2010">
<p>Roessler, Maxie M., Martin S. King, Alan J. Robinson, Fraser A. Armstrong, Jeffrey Harmer, and Judy Hirst. 2010. “Direct Assignment of Epr Spectra to Structurally Defined Iron-Sulfur Clusters in Complex I by Double Electron–Electron Resonance.” <em>Proceedings of the National Academy of Sciences</em> 107 (5): 1930. <a href="https://doi.org/10.1073/pnas.0908050107">https://doi.org/10.1073/pnas.0908050107</a>.</p>
</div>
<div id="ref-Sazanov-2015">
<p>Sazanov, Leonid A. 2015. “A Giant Molecular Proton Pump: Structure and Mechanism of Respiratory Complex I.” <em>Nature Reviews Molecular Cell Biology</em> 16. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved. SN -: 375 EP. <a href="https://doi.org/10.1038/nrm3997">https://doi.org/10.1038/nrm3997</a>.</p>
</div>
<div id="ref-Sazanov-2006">
<p>Sazanov, Leonid A., and Philip Hinchliffe. 2006. “Structure of the Hydrophilic Domain of Respiratory Complex I from Thermus Thermophilus.” <em>Science</em> 311 (5766): 1430. <a href="https://doi.org/10.1126/science.1123809">https://doi.org/10.1126/science.1123809</a>.</p>
</div>
<div id="ref-Shi-2019">
<p>Shi, Tianyi. 2019. “Principle of Chemiosmotic Mechanism.” 2019.</p>
</div>
<div id="ref-Tsukihara-1996">
<p>Tsukihara, Tomitake, Hiroshi Aoyama, Eiki Yamashita, Takashi Tomizaki, Hiroshi Yamaguchi, Kyoko Shinzawa-Itoh, Ryosuke Nakashima, Rieko Yaono, and Shinya Yoshikawa. 1996. “The Whole Structure of the 13-Subunit Oxidized Cytochrome c Oxidase at 2.8 Å.” <em>Science</em> 272 (5265): 1136. <a href="https://doi.org/10.1126/science.272.5265.1136">https://doi.org/10.1126/science.272.5265.1136</a>.</p>
</div>
<div id="ref-Verkhovskaya-2013">
<p>Verkhovskaya, Marina, and Dmitry A. Bloch. 2013. “Energy-Converting Respiratory Complex I: On the Way to the Molecular Mechanism of the Proton Pump.” <em>The International Journal of Biochemistry &amp; Cell Biology</em> 45 (2): 491–511. <a href="https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024">https://doi.org/https://doi.org/10.1016/j.biocel.2012.08.024</a>.</p>
</div>
<div id="ref-Wikstrom-2018">
<p>Wikström, Mårten, and Vivek Sharma. 2018. “Proton Pumping by Cytochrome c Oxidase – a 40 year Anniversary.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1859 (9): 692–98. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009">https://doi.org/https://doi.org/10.1016/j.bbabio.2018.03.009</a>.</p>
</div>
<div id="ref-Wirth-2016">
<p>Wirth, Christophe, Ulrich Brandt, Carola Hunte, and Volker Zickermann. 2016. “Structure and Function of Mitochondrial Complex I.” <em>Biochimica et Biophysica Acta (BBA) - Bioenergetics</em> 1857 (7): 902–14. <a href="https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013">https://doi.org/https://doi.org/10.1016/j.bbabio.2016.02.013</a>.</p>
</div>
<div id="ref-Zong-2018">
<p>Zong, Shuai, Meng Wu, Jinke Gu, Tianya Liu, Runyu Guo, and Maojun Yang. 2018. “Structure of the Intact 14-Subunit Human Cytochrome c Oxidase.” <em>Cell Research</em> 28 (10): 1026–34. <a href="https://doi.org/10.1038/s41422-018-0071-1">https://doi.org/10.1038/s41422-018-0071-1</a>.</p>
</div>
<div id="ref-Zwicker-2006">
<p>Zwicker, K., A. Galkin, S. Drose, L. Grgic, S. Kerscher, and U. Brandt. 2006. “The Redox-Bohr Group Associated with Iron-Sulfur Cluster N2 of Complex I.” Journal Article. <em>J Biol Chem</em> 281 (32): 23013–7. <a href="https://doi.org/10.1074/jbc.M603442200">https://doi.org/10.1074/jbc.M603442200</a>.</p>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol>
<li id="fn1"><p>In some papers <span class="citation">(Hirst 2013; Verkhovskaya and Bloch 2013)</span> it was suggested that coupling of electron transport to movement of protons depends on the significantly exergonic reduction of the N2 centre by the roughly isopotential wire of Fe-S centres that delivers electrons from NADH, and the large E<sub>m,7</sub> span from N3 to N2 must provide the majority of the driving force for the proton pumping events associated with reduction and reoxidation of N2. This may be incorrect, as newest measurements <span class="citation">(Sazanov 2015)</span> of redox potentials of the full Fe-S series (N3-N1b-N4-N5-N6a-N6b-N2) shows that N3, N4 and N6 are about equipotential at about -250 mV, while N1b, N5 and N6b actually have lower potentials; and such an arrangement is claimed by <span class="citation">Roessler et al. (2010)</span> to be energy efficient.<a href="#fnref1" class="footnote-back">↩</a></p></li>
<li id="fn2"><p>The mid-point reduction potential of N2 is shown to be pH-dependent, so that E<sub>acid</sub> &gt; E<sub>alkaline</sub>, and indeed a proton binding site is present there: His226 in <em>Y. lipolytica</em>, and protons are more likely to bind when N2 is reduced. However, shifting the midpoint potential to a more negative and pH insensitive value by exchanging His226 with a methionine affected the functioning of complex I including proton pumping only marginally. <span class="citation">(Zwicker et al. 2006)</span><a href="#fnref2" class="footnote-back">↩</a></p></li>
<li id="fn3"><p>The alternative role of HL would be to stabilise ends of the helices while permitting movement of the middle hydrophilic axis.<a href="#fnref3" class="footnote-back">↩</a></p></li>
<li id="fn4"><p>Although the identity of PLS is uncertain, many studies indicates that it may be the A-propionate substituent of haem <em>a</em><sub>3</sub>.<a href="#fnref4" class="footnote-back">↩</a></p></li>
<li id="fn5"><p>To be precise, when ∆p = 0, there is still (negative) membrane potential (<span class="math inline">\(\Delta \Psi\)</span>), and H<sup>+</sup> equilibrate under this <span class="math inline">\(\Delta \Psi\)</span>, which results in a lower pH inside (more concentrated H<sup>+</sup>). The <span class="math inline">\(\Delta \Psi\)</span> that contributes to ∆p is actually the <span class="math inline">\(change\)</span> in <span class="math inline">\(\Delta \Psi\)</span> caused by proton translocation.<a href="#fnref5" class="footnote-back">↩</a></p></li>
</ol>
</div>

              </div>
            </article>
          
            <article class="archive-item">
              <a href="./tutorial/principle-of-chemiosmotic-mechanism.html" class="archive-item-link">
                Principle of chemiosmotic mechanism 
              </a>
              <span class="archive-item-date">
                2019-10-15
              </span>
              <div class="article-summary" style="display: none;">
                
                  What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?<hr>
                
                


<div id="saq" class="section level1">
<h1>SAQ</h1>
<p class="topic">
<strong>Explain the difference between electrochemical gradient, pH gradient and membrane potentials. How can they be measured and how are these related to chemiosmotic mechanism.</strong>
</p>
<p>The electrochemical gradient describes the difference in chemical stability of ions (typically protons), between two aqueous compartments separated by a non-conducting and non-permeable membrane (typically a energy-transducing membrane such as mitochondrial inner membrane). It is quantified by the difference in Gibbs’ free energy, <span class="math inline">\(\Delta G\)</span>, or its molar equivalent, <em>chemical potential</em>, <span class="math inline">\(\Delta\mu\)</span>, of the ion species (proton) between the two compartments.</p>
<p>Usually, electrochemical gradient is used in studying processes coupled with <span class="math inline">\(\text{H}^+\)</span> movement across a membrane, where one side is positively charged and has a high <span class="math inline">\(\left[\text{H}^+\right]\)</span> and the other side with -ve charge and low <span class="math inline">\(\left[\text{H}^+\right]\)</span>. These are denoted by P and N sides, respectively, and the ‘gradients’ are often defined as the value of the corresponding parameter on the N side minus its value on the P side.</p>
<p>The electrochemical gradient is the sum of pH gradient (concentration gradient) and membrane potential (electrical gradient).</p>
<p>pH gradient is the (log-transformed) difference in the concentration of protons, <span class="math inline">\(\left[\text{H}^+\right]\)</span>, across the membrane. Protons tend to move down its concentration gradient to maximise the entropy of the whole system, and this tendency, i.e. potetial energy is the “chemical” part of the electrochemical potential. Intracellular/intra-orgenellar pH can be measured with pH-sensitive microelectrodes, nuclear magnetic resonance, or pH-sensitive fluorescent proteins <span class="citation">(Loiselle and Casey 2010)</span>. Low-RMM weak acids and bases can equilibrate across the membrane independent of the membrane potential. When there is an pH gradient, however, weak acids accumulate in the basic compartment and bases in the acidic compartment, and this property is used to measure pH gradient.</p>
<p>The membrane potential is the electrical potential difference across the membrane. According to Coulomb’s law, postively charge protons is repelled by the positively charged P side while attracted towards the N side, and this also results in a potential energy, which adds to the electrochemical gradient as the “electro-” part. Membrane potential is measured with microelectrodes.</p>
<p>Considering only the effect of concentration and electrical potential, the electrochemical potential for the proton on either side is defined as follows:</p>
<p><span class="math display">\[\bar{\mu}_{\text{H}^+} = \bar{\mu}_{\text{H}^+}^\circ + RT\ln\left[{\text{H}^+}\right] + z_{\text{H}^+}F\Psi\]</span></p>
<p>where <span class="math inline">\(\bar{\mu}_{\text{H}^+}^\circ\)</span> is the chemical potential under standard states, <span class="math inline">\(RT\ln\left[{\text{H}^+}\right]\)</span> and <span class="math inline">\(z_{\text{H}^+}F\Psi\)</span> are the additional chemical potential due to concentration and electrical potential, respectively.</p>
<p>The electrochemical gradient is the difference of the electrochemical potential between two compartments:</p>
<p><span class="math display">\[\Delta\bar{\mu} = \bar{\mu}_{\text{N}} - \bar{\mu}_{\text{P}} = \left(RT\ln\left[{\text{H}^+}\right]_{\text{N}} + z_{\text{H}^+}F\Psi_{\text{N}}\right) - \left(RT\ln\left[{\text{H}^+}\right]_{\text{P}} + z_{\text{H}^+}F\Psi_{\text{P}}\right)\]</span></p>
<p><span class="math display">\[ = -\ln{10}RT\left(\text{pH}_\text{N}-\text{pH}_\text{P}\right) + F(\Psi_{\text{N}}-\Psi_{\text{P}})=F\Delta\Psi- 2.3RT\Delta\text{pH} \text{  (J mol}^{-1}\text{)}\]</span></p>
<p>or it can be expressed as ‘protomotive force’ in (milli)volts:</p>
<p><span class="math display">\[\Delta\text{p}=\Delta\Psi-\dfrac{2.3RT}{F}\Delta\text{pH}\]</span></p>
</div>
<div id="essay" class="section level1">
<h1>Essay</h1>
<p class="topic">
<strong>What is the evidence that the chemiosmotic mechanism applies to oxidative phosphorylation and photophosphorylation?</strong> <br><sub><em>The essay MUST be equipped with at least 3 annotated diagrams and the length should be 1500-1750 words. Remember to include at least one KEY experimental evidence in the essay.</em></sub>
</p>
<div id="clues-leading-to-the-original-chemiosmosis-hypothesis" class="section level2">
<h2>Clues leading to the original chemiosmosis hypothesis</h2>
<p>The mechanism of oxidative phosphorylation were initially thought to be analogous to substrate-level phosphotylation, that the redox reactions create ‘high-energy’ intermediates, which in turn adds P<sub>i</sub> to ADP. In 1961, Peter Mitchell listed 6 facts to question this chemical mechanism hypothesis:</p>
<ol style="list-style-type: decimal">
<li>The ‘high-energy’ intermediate is not found.</li>
<li>The close association between phosphorylation and membranous structrures is not explained.</li>
<li>P/O ratio varies–it does not follow a fixed stoichiometry.</li>
<li>ATP hydrolysis outside mitochondria promotes NAD reduction inside, accentuated by succinate oxidation. (this observation is probably irrelevant or wrong?)</li>
<li>Uncoupling can be caused by reagents with distinct chemical properties–so they are unlikely to act on a single ‘intermediate’.</li>
<li>Unexplained swelling and shrinkage accompany phosphorylation.</li>
</ol>
<p>Trying to explain these phenomena, he proposed his initial chemiosmosis mechanism, with following 3 features:</p>
<ol style="list-style-type: decimal">
<li>A membrane located, reversible and anisotropic ‘ATPase’</li>
<li>An electron and hydrogen translocation system</li>
<li>A charge impermeable membrane</li>
</ol>
<p>According to this model, oxidation of substrates does not occur in a single compartment, but it instead allow protons and electrons to go in different directions, which in turn creates a concentration gradient of H<sup>+</sup> and OH<sup>-</sup>, as well as a membrane potential. ATP synthesis is driven by the one-way movement of OH<sup>-</sup> down its electrochemical gradient.</p>
<p>Mitchell’s origin model was rather different from the current model, as depicted in figure <a href="#fig:mitchell1961">1</a>, but it provided decent explanations for the phenomena listed above. (In his later review <span class="citation">(Mitchell 1966)</span>, there were some amendments made on his original model, but I have no time to read it)</p>
<div class="figure"><span id="fig:mitchell1961"></span>
<img src="img/mitchell1961.jpg" alt="Left, Mitchell's original chemiosmosis model; right, the current model" width="100%" />
<p class="caption">
Figure 1: Left, Mitchell’s original chemiosmosis model; right, the current model
</p>
</div>
<p>The principal idea was that, energy released from ‘asymmetric’ redox reactions can be used to establish a rather ‘physical’ form of energy (not the common chemical bond energy), the electrochemical gradient, and this is the first coupling. The second coupling is between the synthesis of ATP and the vectorised diffusion of some species promoted by the electrochemical gradient established by the first coupling.</p>
<p>It should be noted that the conception of first coupling was not new. Before the 1960s, it was known that redox reactions can create a electrical potential between two compartments, and it had been noted that some inhibitors of ion transport also has an effect (either stimulatory or inhibitory) on respiratory rate <span class="citation">(Robertson 1960)</span> (Fig. <a href="#fig:fig1">2</a>).</p>
<div class="figure"><span id="fig:fig1"></span>
<img src="img/chemiosmosis-inhibitor.png" alt="Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors" width="100%" />
<p class="caption">
Figure 2: Correlation between the effect on oxygen uptake and on ion accumulation of some inhibitors
</p>
</div>
</div>
<div id="proving-proton-pumping-by-the-redox-chain-and-determining-ho-ratio" class="section level2">
<h2>Proving proton pumping by the redox chain and determining H<sup>+</sup>/O ratio</h2>
<p>The ‘first coupling’ of the current chemiosmosis model means the pumping of protons into the P compartment powered by stepwise redox reactions along the ETC, with the overall oxidation of NADH or FADH<sub>2</sub> as substrate by molecular oxygen.</p>
<p>In actively respiring mitochondria, the proton concentrations (i.e. pH) on either side of the membrane is constant because the rate at which protons are pumped into the P compartment and that flowing back to the N compartment are equal. This makes it difficult to detect the presence of proton flux.</p>
<p><span class="citation">Mitchell and Moyle (1967)</span> devised the following procedure to solve the problem (Fig. <a href="#fig:mitchell1967a">3</a>):</p>
<ol style="list-style-type: decimal">
<li>mitochondria are incubated in anaerobic conditions, allowing H<sup>+</sup> to equilibrate (i.e. making <span class="math inline">\(\Delta p\)</span> fall back to zero)</li>
<li>a pH-sensitive glass electrode is used to monitor pH of the suspension (i.e. the environment outside the mitochondria)</li>
<li>a known quantity of O<sub>2</sub> is introduced into the suspension by ‘injecting pulses of air-saturated 150mM KCI solution.’</li>
<li>by measuring the pH change, the amount of protons pumped out of the mitochondria can be calculated, and with the known amount of O<sub>2</sub>, the H<sup>+</sup>/O ratio can be calculated.</li>
</ol>
<div class="figure"><span id="fig:mitchell1967a"></span>
<img src="img/mitchell1967a.jpg" alt="Determining H/O ratio (Mitchell 1967a)" width="100%" />
<p class="caption">
Figure 3: Determining H/O ratio (Mitchell 1967a)
</p>
</div>
<p>Additional procedures are needed to minimise error, and they’re described in <em>Bioenergetics</em> pp59-60</p>
</div>
<div id="proving-the-atp-synthesis-is-driven-by-the-electrochemical-gradient" class="section level2">
<h2>Proving the ATP synthesis is driven by the electrochemical gradient</h2>
<p>The electrochemical gradients is the sum of concentration gradient and electrical potential gradient, and each of them are shown to drive ATP synthesis.</p>
<p><span class="citation">Jagendorf and Uribe (1966)</span> showed ATP synthesis can be induced in dark, by artificially creating a pH difference across the thylakoid membrane in chloroplasts (Fig <a href="#fig:Sone">4</a>). In this experiment, broken chloroplasts (extracted from spinach) are first incubated in an acidic solution, allowing the thylakoid space to achieve the enviornmental pH of 4.0 (different acids are used to verify its non-specificity). Next, chloroplasts are exposed to pH 8.0 to develop the proton gradient across the thylakoid membrane. While moving out from the thylakoid space, protons drive ATP synthase. The amount of ATP produced is then determined by firefly luciferase assay. By repeating with different pHs, the authors concluded that the rate of ATP synthesis is positively correlated with the pH difference between two phases.</p>
<p><span class="citation">Sone et al. (1977)</span> used reconsitituted vesicles, made from purified <span class="math inline">\(\text{F}_o\text{F}_1\)</span> complex and the membrane of a thermophilic bacterium, to achieve ATP synthesis by induced membrane potential (Fig <a href="#fig:Sone">4</a>). They first incubated mitochondria with valinomycin at a fixed pH, then exposed the suspension to KCl. Valinomycin-mediated entry of K<sup>+</sup> shifts the membrane potential so the vesicle interior becomes more positively charged. Consequently, H<sup>+</sup> moves out and drives ATP synthase.</p>
<div class="figure"><span id="fig:Sone"></span>
<img src="img/sone.jpg" alt="Left, Jagendorf (2016); right, Sone (1977)" width="100%" />
<p class="caption">
Figure 4: Left, Jagendorf (2016); right, Sone (1977)
</p>
</div>
</div>
<div id="monitoring-the-protomotive-force-and-proton-flux-in-different-conditions" class="section level2">
<h2>Monitoring the protomotive force and proton flux in different conditions</h2>
<p>If the chemiosmotic model is corrent, it will have some properties resembling those of an electric circuit–the current (<span class="math inline">\(I\)</span>) and electromotive force (<span class="math inline">\(V\)</span>) are mirrored by the proton flux (<span class="math inline">\(J_{\text{H}^+}\)</span>) and the protomotive force (<span class="math inline">\(\Delta\text{p}\)</span>), respectively. These two terms can be monitored in different states of the proton circuit, and the results are in accordance with the chemiosmotic model.</p>
<div class="figure"><span id="fig:circuit-analogy"></span>
<img src="img/circuit_analogy.jpg" alt="Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state." width="100%" />
<p class="caption">
Figure 5: Circuit Analogy. Top left, proton gradient; top right, oxygen concentration, whose gradient reflects proton flux; below, illustrations of each numbered state.
</p>
</div>
<p>The different states shown in Fig <a href="#fig:circuit-analogy">5</a> are explained below:</p>
<ol style="list-style-type: decimal">
<li>When ETC substrate is absent, there is absolutely no oxygen consumption. No active forces act protons so they are allowed to equilibrate, and therefore the <span class="math inline">\(\Delta\text{p}\)</span> is zero. This resembles an electrical circuit without a power source.</li>
<li>When substrate (e.g. succinate) is provided, it is oxidised by ETC and oxygen is consumed. ETC starts to pump H<sup>+</sup> into the P face, building up the electrochemical gradient If the membrane was perfectly impermeable to H<sup>+</sup>, ETC would quickly stop as its provided energy for proton efflux balances the electrochemical gradient which favours protons’ influx, and there would be no oxygen consumption. However, the membrane is ‘leaky’ (and there are other processes causing influx of H<sup>+</sup>), causing constant influx of H<sup>+</sup>, which in turn allows H<sup>+</sup> to be steadily, but slowly, pumped to the P face, which is accompanied by oxygen usage. This resembles more of a closed circuit where energy is dissipated slowly through a high-resistance resistor than of an ‘open circuit’ as described in some books.</li>
<li>When ADP is added, ATP synthase can use the proton gradient to synthesise ATP. The proton flux through ATP synthase is huge, so the oxygen concentration (whose gradient parallels proton flux) decreases steeply. (I can’t give an rigorous mathematical explanation on the decrease of <span class="math inline">\(\Delta\text{p}\)</span>)</li>
<li>ATP is used up, and the condition is identical to state 2.</li>
<li>FCCP short-circuits the proton circuit by allowing H<sup>+</sup> quickly flowing back to the N face without doing ‘useful work’ (driving ATP synthase). The proton influx is so large that it outstrips the maximal efflux powered by ETC, so <span class="math inline">\(\Delta\text{p}\)</span> quickly falls back to zero.</li>
</ol>
</div>
<div id="lateral-proton-current" class="section level2">
<h2>Lateral proton current?</h2>
<p>The widely accepted view that ETC pumps protons transversally from the N to the P face may need updating for the following reasons stated by <span class="citation">Morelli et al. (2019)</span>:</p>
<ol style="list-style-type: decimal">
<li>H<sup>+</sup> almost never exist as isolated protons. Instead, they bind to water to form H<sub>3</sub>O<sup>+</sup>. Free protons in a mitochondrial periplasmic space are too few (fewer than 10) to drive ATP synthase.</li>
<li>Free protons are destructive for biological membranes</li>
<li>The pH inside mitochondria was shown to be higher by 0.5 units from what was previously believed <span class="citation">(Żurawik et al. 2016)</span></li>
<li>Phospholipid membranes are intrinsically (significantly) permeable to protons</li>
</ol>
<p>According to the classic view of proton pumping, complex I should have a continuous channel traversing the membrane. However, based on comprehensive X-ray studies, the ‘proton entrance half channel’ is not identified with certainty, while the ‘proton exit half channel’ is clearly identifiable, and an obvious proton tunnelling is found at the centre of complex I. This leads to an lateral mechanism of proton circuiting, as shown below:</p>
<div class="figure"><span id="fig:Morelli2019"></span>
<img src="img/Morelli2019.png" alt="A possible lateral proton circuit suggested by Morelli (2019)." width="100%" />
<p class="caption">
Figure 6: A possible lateral proton circuit suggested by Morelli (2019).
</p>
</div>
</div>
</div>
<div id="bibliography" class="section level1 unnumbered">
<h1>Bibliography</h1>
<div id="refs" class="references">
<div id="ref-Jagendorf:1966">
<p>Jagendorf, A T, and E Uribe. 1966. “ATP Formation Caused by Acid-Base Transition of Spinach Chloroplasts.” <em>Proceedings of the National Academy of Sciences</em> 55 (1). National Academy of Sciences: 170–77. <a href="https://doi.org/10.1073/pnas.55.1.170">https://doi.org/10.1073/pnas.55.1.170</a>.</p>
</div>
<div id="ref-Loiselle:2010">
<p>Loiselle, Frederick B., and Joseph R. Casey. 2010. “Measurement of Intracellular pH.” In <em>Membrane Transporters in Drug Discovery and Development</em>, edited by Qing Yan. Vol. 637. Methods in Molecular Biology. Humana Press.</p>
</div>
<div id="ref-Mitchell:1966">
<p>Mitchell, Peter. 1966. “CHEMIOSMOTIC Coupling in Oxidative and Photosynthetic Phosphorylation.” Journal Article. <em>Biological Reviews</em> 41 (3): 445–501. <a href="https://doi.org/10.1111/j.1469-185X.1966.tb01501.x">https://doi.org/10.1111/j.1469-185X.1966.tb01501.x</a>.</p>
</div>
<div id="ref-Mitchell:1967a">
<p>Mitchell, Peter, and Jennifer Moyle. 1967. “Respiration-driven proton translocation in rat liver mitochondria.” <em>Biochemical Journal</em> 105 (3): 1147–62. <a href="https://doi.org/10.1042/bj1051147">https://doi.org/10.1042/bj1051147</a>.</p>
</div>
<div id="ref-Morelli:2019">
<p>Morelli, Alessandro Maria, Silvia Ravera, Daniela Calzia, and Isabella Panfoli. 2019. “An Update of the Chemiosmotic Theory as Suggested by Possible Proton Currents Inside the Coupling Membrane.” Journal Article. <em>Open Biology</em> 9 (4): 180221. <a href="https://doi.org/10.1098/rsob.180221">https://doi.org/10.1098/rsob.180221</a>.</p>
</div>
<div id="ref-Robertson:1960">
<p>Robertson, R. N. 1960. “ION Transport and Respiration.” Journal Article. <em>Biological Reviews</em> 35 (2): 231–64. <a href="https://doi.org/10.1111/j.1469-185X.1960.tb01415.x">https://doi.org/10.1111/j.1469-185X.1960.tb01415.x</a>.</p>
</div>
<div id="ref-Sone:1977">
<p>Sone, N, M Yoshida, H Hirata, and Y Kagawa. 1977. “Adenosine Triphosphate Synthesis by Electrochemical Proton Gradient in Vesicles Reconstituted from Purified Adenosine Triphosphatase and Phospholipids of Thermophilic Bacterium.” <em>Journal of Biological Chemistry</em> 252: 2956–60.</p>
</div>
<div id="ref-Zurawik:2016">
<p>Żurawik, Tomasz Michał, Adam Pomorski, Agnieszka Belczyk-Ciesielska, Grażyna Goch, Katarzyna Niedźwiedzka, Róża Kucharczyk, Artur Krężel, and Wojciech Bal. 2016. “Revisiting Mitochondrial pH with an Improved Algorithm for Calibration of the Ratiometric 5(6)-Carboxy-Snarf-1 Probe Reveals Anticooperative Reaction with H+ Ions and Warrants Further Studies of Organellar pH.” <em>PLOS ONE</em> 11 (8). Public Library of Science: 1–17. <a href="https://doi.org/10.1371/journal.pone.0161353">https://doi.org/10.1371/journal.pone.0161353</a>.</p>
</div>
</div>
</div>

              </div>
            </article>
          
      
    
  </div>

</main>

<script src="./js/list.js"></script>



    <footer class="footer">
      <ul class="footer-links">
        <li>
          <a href="./index.xml" type="application/rss+xml" target="_blank">RSS feed</a>
        </li>
        <li>
          <a href="https://gohugo.io/" class="footer-links-kudos">Made with <img src="./images/hugo-logo.png" alt="Img link to Hugo website" width="22" height="22"></a>
        </li>
      </ul>
      
    </footer>

  </div>
  


<link href='//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css' rel='stylesheet'
    type='text/css' />

<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>



<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/r.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/yaml.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/python.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/json.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/go.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/rust.min.js"></script>
<script>hljs.configure({ languages: [] }); hljs.initHighlightingOnLoad();</script>
  
<script src="./js/math-code.js"></script>
<script async src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"></script>


  
  
</body>

</html>